



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>MicroRNAs (miRNAs) are a group of endogenous non-coding small RNAs that have critical regulatory functions in almost all known biological processes at the post-transcriptional level in a variety of organisms. The oriental fruit moth <em>Grapholita molesta</em> is one of the most serious pests in orchards worldwide and threatens the production of Rosacea fruits. In this study, a <em>de novo</em> small RNA library constructed from mixed stages of <em>G</em>. <em>molesta</em> was sequenced through Illumina sequencing platform and a total of 536 mature miRNAs consisting of 291 conserved and 245 novel miRNAs were identified. Most of the conserved and novel miRNAs were detected with moderate abundance. The miRNAs in the same cluster normally showed correlated expressional profiles. A comparative analysis of the 79 conserved miRNA families within 31 arthropod species indicated that these miRNA families were more conserved among insects and within orders of closer phylogenetic relationships. The KEGG pathway analysis and network prediction of target genes indicated that the complex composed of miRNAs, clock genes and developmental regulation genes may play vital roles to regulate the developmental circadian rhythm of <em>G</em>. <em>molesta</em>. Furthermore, based on the sRNA library of <em>G</em>. <em>molesta</em>, suitable reference genes were selected and validated for study of miRNA transcriptional profile in <em>G</em>. <em>molesta</em> under two biotic and six abiotic experimental conditions. This study systematically documented the miRNA profile in <em>G</em>. <em>molesta</em>, which could lay a foundation for further understanding of the regulatory roles of miRNAs in the development and metabolism in this pest and might also suggest clues to the development of genetic-based techniques for agricultural pest control.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wang X, Li Y, Zhang J, Zhang Q, Liu X, Li Z (2017) <em>De novo</em> characterization of microRNAs in oriental fruit moth <em>Grapholita molesta</em> and selection of reference genes for normalization of microRNA expression. PLoS ONE 12(2):
           e0171120.
        
        https://doi.org/10.1371/journal.pone.0171120</p><p><strong>Editor: </strong>Youjun Zhang, Chinese Academy of Agricultural Sciences, CHINA</p><p><strong>Received: </strong>August 16, 2016; <strong>Accepted: </strong>January 16, 2017; <strong>Published: </strong> February 3, 2017</p><p><strong>Copyright: </strong> © 2017 Wang et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>All relevant data are within the paper and its Supporting Information files.</p><p><strong>Funding: </strong>This study was supported by Chinese Universities Scientific Fund (No. 2014XJ018) and Modern Agro-industry Technology Research System Grants (No. CARS-29-08). The funders had no role in study design, data collection and analysis, decision to publish, and preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>MicroRNAs (miRNAs) are non-coding small RNAs and are usually 21–24 nucleotide (nt) in lengths [<a href="#pone.0171120.ref001" class="ref-tip">1</a>,<a href="#pone.0171120.ref002" class="ref-tip">2</a>]. Accumulative reports suggest that miRNAs function as important gene expression regulators in almost all known physiological and pathophysiological process in varieties of eukaryotes and viruses [<a href="#pone.0171120.ref003" class="ref-tip">3</a>–<a href="#pone.0171120.ref005" class="ref-tip">5</a>]. Through the specific recognition between the miRNA seed region (nucleotides 2–8 from the 5’ en≈d) and the target sequences of mRNA existing in the 3’, 5’–untranslated region (3’UTR, 5’UTR) or the open reading frame, the finely tuning results of miRNAs normally lead to the degradation [<a href="#pone.0171120.ref006" class="ref-tip">6</a>], transcriptional or translational repression [<a href="#pone.0171120.ref007" class="ref-tip">7</a>,<a href="#pone.0171120.ref008" class="ref-tip">8</a>] or even transcriptional enhancement of the target mRNA [<a href="#pone.0171120.ref009" class="ref-tip">9</a>,<a href="#pone.0171120.ref010" class="ref-tip">10</a>].</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Since miRNA lin-4 was firstly discovered in <em>Caenorhabditis elegans</em> for regulation of development timing [<a href="#pone.0171120.ref011" class="ref-tip">11</a>], with the development of next generation sequencing platforms, thousands of miRNAs have been identified in human, animals, plants, and viruses [<a href="#pone.0171120.ref012" class="ref-tip">12</a>] and are available on miRBase (<a href="http://www.mirbase.org/">http://www.mirbase.org/</a>). Until now, 35828 mature miRNAs have been identified from 220 species, in which 26 species belong to insects (<a href="http://www.mirbase.org/">http://www.mirbase.org/</a>). Due to the absence of complete genome sequences, whereas, most of the available sequences of insect miRNAs were determined from model organisms, including 15 species of Diptera (12 <em>Drosophila</em>, <em>Aedes aegypti</em>, <em>Anopheles gambiae</em> and <em>Culex quingquefasciatus</em>), 4 species of Hymenoptera (<em>Apis mellifera</em> and 3 <em>Nasonia</em>), 4 species of Lepidoptera (<em>Bombyx mori</em>, <em>Manduca sexta</em>, <em>Heliconius melpomene</em> and <em>Plutella xylostella</em>), 1 Hemiptera species (<em>Acyrthosiphon pisum</em>), 1Orthoptera species (<em>Locusta migratoria</em>), and 1 Coleoptera species (<em>Tribolium castaneum</em>).</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>miRNA play various regulatory roles in insects, mainly in developmental regulations of germ cell line [<a href="#pone.0171120.ref013" class="ref-tip">13</a>], wing formation [<a href="#pone.0171120.ref014" class="ref-tip">14</a>,<a href="#pone.0171120.ref015" class="ref-tip">15</a>], muscle development [<a href="#pone.0171120.ref007" class="ref-tip">7</a>], neurogenesis [<a href="#pone.0171120.ref016" class="ref-tip">16</a>], apoptosis [<a href="#pone.0171120.ref017" class="ref-tip">17</a>], phenotypic plasticity [<a href="#pone.0171120.ref018" class="ref-tip">18</a>,<a href="#pone.0171120.ref019" class="ref-tip">19</a>], reproduction [<a href="#pone.0171120.ref018" class="ref-tip">18</a>], metamorphosis [<a href="#pone.0171120.ref020" class="ref-tip">20</a>], behavior [<a href="#pone.0171120.ref021" class="ref-tip">21</a>,<a href="#pone.0171120.ref022" class="ref-tip">22</a>], stress resistance [<a href="#pone.0171120.ref023" class="ref-tip">23</a>] and host-pathogen interaction [<a href="#pone.0171120.ref024" class="ref-tip">24</a>,<a href="#pone.0171120.ref025" class="ref-tip">25</a>]. For the multiply regulatory feature, miRNAs are usually expressed in a precisely temporal and spatial pattern [<a href="#pone.0171120.ref026" class="ref-tip">26</a>–<a href="#pone.0171120.ref029" class="ref-tip">29</a>]. Thus, accurate quantifications of miRNA profiles in specific experimental conditions are essential to their functional detection. Up to date, three technologies, microarray [<a href="#pone.0171120.ref030" class="ref-tip">30</a>], bead-based flow cytometry [<a href="#pone.0171120.ref031" class="ref-tip">31</a>], and quantitative real time PCR (qPCR) [<a href="#pone.0171120.ref032" class="ref-tip">32</a>,<a href="#pone.0171120.ref033" class="ref-tip">33</a>] have been developed to study expressional profiles of miRNAs. qPCR, for its sensitivity, flexibility and convenience, has been widely applied to measure miRNA expression levels in diverse organisms. A large number of studies have revealed that appropriate normalizers are critical to the accuracy of gene expression evaluation [<a href="#pone.0171120.ref034" class="ref-tip">34</a>–<a href="#pone.0171120.ref036" class="ref-tip">36</a>].</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>The oriental fruit moth, <em>Grapholita molesta</em> (Busck) (Lepidoptera: Tortricidae) is a worldwide pest of stone and pome fruits in most temperate fruit-growing regions [<a href="#pone.0171120.ref037" class="ref-tip">37</a>] and prefers the new shoots and fruits of host plants within the family Rosacea for boring and internal-feeding [<a href="#pone.0171120.ref038" class="ref-tip">38</a>,<a href="#pone.0171120.ref039" class="ref-tip">39</a>]. <em>Grapholita molesta</em> normally has three to five generations per season. The adults migrate from peach orchards to pear or apple orchards. Therefore, outbreaks often occur in the mixed-planting orchards, which result in substantial losses in yields and incomes [<a href="#pone.0171120.ref040" class="ref-tip">40</a>,<a href="#pone.0171120.ref041" class="ref-tip">41</a>]. Nowadays, the main management method is to use insecticides for egg and neonate control. Unfortunately, insecticide resistance has occurred in <em>G</em>. <em>molesta</em> [<a href="#pone.0171120.ref042" class="ref-tip">42</a>]. An auxiliary technique, pheromone-based mating disruption of adults, is effective but with high management costs [<a href="#pone.0171120.ref043" class="ref-tip">43</a>]. A better understanding about the regulatory mechanisms of the metabolism, development and metamorphosis of <em>G</em>. <em>molesta</em> would shed light on the development of new techniques for efficient management of this borer pest.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5" class="link-target"></a><p>In the present study, on the basis of the Illumina small RNA (sRNA) library sequencing platform, the miRNA transcriptome of <em>G</em>. <em>molesta</em> was sequenced and performed with systematically <em>in silico</em> analysis in reference to the transcriptome of <em>G</em>. <em>molesta</em> and the genome of <em>Danaus plexippus</em>. We also validated the expressional stability of the candidate reference genes and made selection of suitable normalizers under different biotic and abiotic experimental conditions. Our results would provide useful information for revealing the miRNA-involved regulatory mechanism of the development and metabolism in <em>G</em>. <em>molesta</em> and would also supply a reference for deeper understanding of the expressional profile and regulatory function of miRNA in other non-model insects and agricultural insect pests.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Materials and methods"></a><h2>Materials and methods</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Ethics statement"></a>
<h3>Ethics statement</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>In this study, the larvae of oriental fruit moth <em>G</em>. <em>molesta</em> were originally collected in the Institute of Pomology in Liaoning province, China. No permissions were required for the insect collection, as the orchards are experimental plots belonging to the Institute of Pomology in Liaoning province. The “List of Protected Animals in China” excludes insects.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Insect rearing"></a>
<h3>Insect rearing</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p><em>Grapholita molesta</em> used in this study were reared under laboratory conditions (26 ± 1°C, 70 ± 10%, 15 h L: 9 h D) on fresh apples and artificial diet over four years as our previous study [<a href="#pone.0171120.ref044" class="ref-tip">44</a>]. Different instars of <em>G</em>. <em>molesta</em> were carefully picked out from rotten apples with soft tweezers. Five larvae of the same instar were reared together in one finger-shaped glass tube (5.5 cm in length ×2.2 cm in diameter) on artificial diet jelly. After pupation, pupae were collected from the cotton-wool tampon on the top of the tubes, and newly-emerged adults were transferred to beakers (1 L in volume) with fresh apples inside for egg laying. Cotton balls soaked with 10% honey solution were provided for nutrient supplement.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Sample collection and total RNA extraction for sRNA library construction"></a>
<h3>Sample collection and total RNA extraction for sRNA library construction</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Healthy samples of <em>G</em>. <em>molesta</em>, including eggs, the first to fifth instars, prepupae, pupae, female and male adults were collected and snap-frozen in liquid nitrogen. Three hundred eggs, one hundred first and second instars and ten pooled frozen insects of other developmental stages were respectively ground into powder in liquid nitrogen with cooled mortars and pestles. Aliquots of the powdered samples were separately transferred into 1.5 ml RNase-Free Eppendorf tubes. The total RNA samples were isolated with Trizol reagent (Takara, Kyoto, Japan) according to the instruction. The quality and quantity of RNA extracts were then measured with Nanodrop 2000 (Thermo Fisher Scientific, Waltham, MA, USA) and Bioanalyzer 2100 (Agilent, Santa Clara, CA, USA).</p>
</div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="sRNA library construction and sequencing"></a>
<h3>sRNA library construction and sequencing</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>RNA extracts from each stage of <em>G</em>. <em>molesta</em> were mixed in an equal amount and then 1 μg of the resulting RNA mix was used for construction of sRNA library using TruSeq Small RNA Sample Prep Kit (Illumina, San Diego, CA, USA). Briefly, sRNAs of 15–30 bp were fractionated on 15% PAGE (polyacrylamide gel electrophoresis); 5’ and 3’ RNA adaptors were then ligated to the recovered sRNA; cDNAs were finally obtained after reverse transcription. After PCR amplification, 6% polyacrylamide TBE (Tris-borate-EDTA) gel purification and cluster formation, the resulting cDNAs were subjected to the single-end sequencing for 50 cycles (1 bp reads per cycle) on an Illumina Hiseq2500 (Illumina, San Diego, CA, USA) in LC-BIO (Hangzhou, China).</p>
</div>

<div id="section5" class="section toc-section"><a id="sec007" name="sec007" class="link-target" title="Sequence filter and annotation"></a>
<h3>Sequence filter and annotation</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>The raw reads were subjected to the Illumina pipeline filter (Solexa 0.3) for quality assessment. Clean reads were then obtained through data process with the program ACGT101-miR (LC Sciences, Houston, Texas, USA) after removing adapter dimers, junk reads and common non-coding RNA families (rRNA, tRNA, snRNA, snoRNA). Finally, the unique reads were screened out by eliminating the repeated sRNAs from the clean reads database with single sequence left.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>To identify miRNAs, the unique sequence dataset with length of 18–26 bp were mapped in the miRBase (release 21, <a href="http://www.mirbase.org/">http://www.mirbase.org/</a>) by BLAST search, and length variations at both 3’ and 5’ ends and one mismatch inside the sequence were allowed in the alignment. The unique reads that mapped to the known miRNAs and/or pre-miRNAs of specific species were determined to be conserved miRNAs. The remaining sequences were further mapped against the transcriptome of <em>G</em>. <em>molesta</em> (unpublished data) and genome of highly homological lepidopteran <em>D</em>. <em>plexippus</em> (downloaded from MonarchBase website: <a href="http://monarchbase.umassmed.edu/">http://monarchbase.umassmed.edu</a>). The reads that could map to the transcriptome and/or genome were further subjected to the secondary hairpin prediction using RNAfold software (<a href="http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi">http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi</a>). The sequences of transcriptome/genome that 20 bp upstream and 60 bp downstream of the mapped sequences and the sequences 60 bp upstream and 20 bp downstream of the mapped sequences were both extracted as candidate precursors for secondary structure prediction. The criteria for secondary structure prediction were: (1) number of nucleotides in one bulge in stem (≤ 12); (2) number of base pairs in the stem region of the predicted hairpin (≥ 16); (3) cut of free energy (kCal/mol ≤ 15); (4) length of hairpin (up and down stems + terminal loop ≥ 50); (5) length of hairpin loop (≤ 20); (6) number of nucleotides in one bulge in mature region (≤ 8); (7) number of biased errors in one bulge in mature region (≤ 4); (8) number of biased bulges in mature region (≤ 2); (9) number of errors in mature region (≤ 7); (10) number of base pairs in the mature region of the predicted hairpin (≥ 12); (11) percent of mature in stem (≥ 80%). The reads with precursors that satisfied all of the above standards and with more than 10 reads account were determined as novel miRNAs. The secondary structure of novel miRNAs were then constructed using srna-workbench V2.5.0 (<a href="http://srna-workbench.cmp.uea.ac.uk/downloadspage/">http://srna-workbench.cmp.uea.ac.uk/downloadspage/</a>) with the precursor sequence and the brief-code of the corresponding secondary structure of each novel miRNA.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="Validation of the Illumina sequencing"></a>
<h3>Validation of the Illumina sequencing</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>To validate the Illumina sequencing, 10 conserved miRNAs that have been reported with relative stable expression in other organisms [<a href="#pone.0171120.ref045" class="ref-tip">45</a>–<a href="#pone.0171120.ref047" class="ref-tip">47</a>] and 10 novel identified miRNAs with high abundance in sRNA library of <em>G</em>. <em>molesta</em> (<strong><a href="#pone.0171120.s004">S1 Table</a></strong>) were selected and their expression level in the sample for construction of sRNA library were checked using qPCR. What’s more, the 20 selected miRNAs were respectively cloned into pMD18-T (Takara) and were further validated with Sanger sequencing by Beijing Genomics Institute (BGI).</p>
</div>

<div id="section7" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Cluster and conservation analysis"></a>
<h3>Cluster and conservation analysis</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>Cluster analysis was conducted among miRNAs at the distance of 3 kb, 5 kb, 10 kb and 50 kb within the genome of <em>D</em>. <em>plexippus</em>. One cluster was determined when at least two miRNAs could be found in a specific distance range.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2" class="link-target"></a><p>The miRNA sequences of arthropods, <em>C</em>. <em>elegans</em> and <em>Homo sapiens</em> have been downloaded from miRBase (release 21). Conservation of miRNA families among <em>G</em>. <em>molesta</em> and other arthropods was compared through BLASTn analysis. Phylogenetic relationships among 27 insect species belonging to 6 orders in Hexapoda and 4 species in the other three phyla of arthropods have been analyzed. The topology tree was constructed according to previous reports [<a href="#pone.0171120.ref048" class="ref-tip">48</a>,<a href="#pone.0171120.ref049" class="ref-tip">49</a>].</p>
</div>

<div id="section8" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Target prediction and KEGG enrichment analysis"></a>
<h3>Target prediction and KEGG enrichment analysis</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>Putative target genes of miRNAs in <em>G</em>. <em>molesta</em> were concurrently predicted by TargetScan 50 and miRanda 3.3a through identification of miRNA binding sties on the 3’- UTR sequences of <em>G</em>. <em>molesta</em> transcriptome. The target genes with context score percentile less than 50 were then eliminated by TargetScan. The ones with max energy greater than -10 were screened out by miRanda. Finally, the target genes simultaneously recommended by the two algorithms were defined as the final prediction.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2" class="link-target"></a><p>Functional pathway determination of target genes was performed through the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analysis against the public database (<a href="http://www.genome.jp/kegg">http://www.genome.jp/kegg</a>) [<a href="#pone.0171120.ref050" class="ref-tip">50</a>]. Pathways with <em>P–</em>value of fisher’s exact test ≤ 0.05 were determined to be significantly enriched pathways for metabolism or signal transduction. For elucidation of the interactions between miRNAs and target genes in metabolisms and signal transductions, networks of miRNA-KEGG in <em>G</em>. <em>molesta</em> were further analyzed with miRNA-mRNA target sequence analysis technique and were descripted with Cytoscape 2.8.3.</p>
</div>

<div id="section9" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="Sample preparation and candidate sRNA selection for reference gene selection"></a>
<h3>Sample preparation and candidate sRNA selection for reference gene selection</h3>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1" class="link-target"></a><p>Eight experimental conditions had been considered for study on expressional stability of sRNAs in <em>G</em>. <em>molesta</em>, including two intrinsic biotic conditions (developmental stage and tissue) and six exogenous stress conditions (temperature, photoperiod, starvation, JH injection, dsRNA injection and insecticide).</p>
<a id="article1.body1.sec2.sec9.p2" name="article1.body1.sec2.sec9.p2" class="link-target"></a><p>For samples of different developmental stages, 300 eggs, 200 first instars, 30 second instars, 10 third instars, 10 fourth instars, 7 fifth instars, 10 prepupae, 10 female pupae, 10 male pupae, 10 female adults (3 d old post-emergence) and 10 male adults (3 d old post-emergence) of <em>G</em>. <em>molesta</em> have been collected respectively as one replicate. For sample preparation of different tissues, 20 fifth instars (2 d old post-molt) and 10 adults (3 d old post-emergence) were dissected under a binocular microscope in 10 mM cold phosphate buffered saline (PBS, pH 7.8) as one replicate; after rinsed in PBS, samples of each tissue, including 9 tissues of larvae (head, cuticle, foregut, midgut, hindgut, Malpighian tubule, fatbody and ventral nerve cord) and 4 tissues of adults (head, thorax, abdomen and leg) were obtained respectively from pooled dissections.</p>
<a id="article1.body1.sec2.sec9.p3" name="article1.body1.sec2.sec9.p3" class="link-target"></a><p>For sample preparation of <em>G</em>. <em>molesta</em> subjected to stresses of varying temperature, photoperiod and food deprivation, the third instars were treated under nine temperature conditions (4, 26 and 40°C for 2, 12 and 24 h, respectively), eight photoperiod conditions (24 h L: 0 h D, 0 h L: 24 h D, 14 h L: 10 h D and 10 h L: 14 h D for 1 d and 2 d, respectively), and four starvation conditions (starved for 12 h, 24 h, 48 h and 48 h followed by 24 h refeeding). For insect samples pretreated with double strand RNA (dsRNA) and juvenile hormone (JH) injection, the fifth instars were respectively injected with artificially synthesized dsRNA of <em>met</em> (encoding putative JH receptor Methoprene-tolerant) and chemical reagent JH III (Sigma-Aldrich, Milwaukee, WI, USA). DsRNA of <em>egfp</em> (encoding Enhanced Green Fluorescent Protein) and chemical reagent dimethyl sulphoxide (DMSO, solvent of JH III, Sigma-Aldrich, Milwaukee, WI, USA) were injected as control treatments accordingly. DsRNAs of <em>met</em> and <em>egfp</em> were artificially prepared with MEGAscript RNAi kit (Ambion) with primer pairs listed in <strong><a href="#pone.0171120.s005">S2 Table</a></strong>. For stress induced by insecticide, the third instars were topically applied with LD<sub>50</sub> of 2.5% β-cypermethrin (1 μL of 1500 times dilution per larva, or 0.625 ng per larvae) calculated using PoloPlus<sup>TM</sup> software (LeOra software, Berkeley, CA, USA) and were then sampled, respectively, at 12 h, 24 h and 48 h post application. Ten insects pretreated under each exogenous stress condition were collected as one replicate. Three replicates were collected for all of the eight treatments.</p>
<a id="article1.body1.sec2.sec9.p4" name="article1.body1.sec2.sec9.p4" class="link-target"></a><p>All insect samples were snap frozen in liquid nitrogen and stored at -80°C. Total RNA of each sample was then extracted with Trizol reagent as above and checked with Nanodrop 2000 (Thermo Fisher Scientific, Waltham, MA, USA). Subsequently, cDNAs of sRNAs were synthesized after reverse transcription from the qualified RNA with A260/280 value of 1.8–2.3 and A260/230 value ≥ 2.0 using miScript II RT Kit (Qiangen, Dusseldorf, Germany). Universal reverse primer (<span class="monospace">GAATCGAGCACCAGTTACGC</span>) was used for cDNA synthesis according to the manufacture’s protocol. All of the resulting cDNA samples were stored at −20°C for sRNA expressional analysis.</p>
<a id="article1.body1.sec2.sec9.p5" name="article1.body1.sec2.sec9.p5" class="link-target"></a><p>For reference RNAs selection, 12 sRNAs, including two commonly used normalizer (small nuclear RNA <em>U6</em> and small ribosomal RNA <em>5SrRNA</em>) for miRNA quantification in mammals and plants [<a href="#pone.0171120.ref051" class="ref-tip">51</a>–<a href="#pone.0171120.ref054" class="ref-tip">54</a>] and 10 miRNAs (bmo-miR-2b-3p_1ss22GC, pxy-mir-6497-p3_1ss7CT, hsa-miR-16-5p, bmo-miR-281-3p_L-2R+2, bmo-miR-279a_R+2, bmo-miR-9a-5p, bmo-miR-998_R+2, bmo-miR-305-5p_R+1, bmo-let-7-5p and mse-miR-92a) which have been used for normalization in previous reports [<a href="#pone.0171120.ref045" class="ref-tip">45</a>–<a href="#pone.0171120.ref047" class="ref-tip">47</a>] and showed abundant expression in the sRNA library of <em>G</em>. <em>molesta</em> have been selected as candidate reference genes for miRNA profile study of <em>G</em>. <em>molesta</em>.</p>
</div>

<div id="section10" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="Quantitative PCR analysis"></a>
<h3>Quantitative PCR analysis</h3>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1" class="link-target"></a><p>qPCR with high sensitivity and specificity was adopted for validation of sRNA library sequencing and quantification of selected candidate reference genes through measurement of mature sRNA expression.</p>
<a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2" class="link-target"></a><p>The forward primers for qPCR amplification of miRNAs were designed according to the specific sequences of miRNAs with some modifications at 5’ ends for adjustment of GC contents (<strong><a href="#pone.0171120.s004">S1 Table</a></strong>). The reverse primer for qRT-PCR analysis was the universal primer used for cDNA synthesis above. All primers were synthesized commercially (Sangon Biotechnology, Shanghai, China) and diluted to 10 μM. qPCR was conducted on a Bio-Rad CFX Connect<sup>TM</sup> Real-Time PCR Detection System (Hercules, CA, US) using miScript SYBR® Green PCR kit (Qiagen). The amplification was performed following the program as: 94°C for 15 min for denaturation, 40 cycles of 95°C for 15 s, 55°C for 30 s, 72°C for 30 s for collection of amplification signal and 70°C for 30 s for reaction termination. The dissociation protocol was also carried out for melting curve analysis on the specific amplification. The amplification efficiency (E%) and correlation coefficient (R<sup>2</sup>) for amplification of each sRNA were calculated according to the standard curve generated from the five 5-fold serial dilution points of cDNAs and their corresponding C<sub>q</sub> values [<a href="#pone.0171120.ref035" class="ref-tip">35</a>,<a href="#pone.0171120.ref055" class="ref-tip">55</a>].</p>
</div>

<div id="section11" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="Statistical analysis for selection and validation of reference genes"></a>
<h3>Statistical analysis for selection and validation of reference genes</h3>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1" class="link-target"></a><p>Global mean that was defined as the mean C<sub>q</sub> value of all expressed sRNAs in a given sample [<a href="#pone.0171120.ref034" class="ref-tip">34</a>] was calculated. The Spearman’s rank correlation coefficient <em>r</em><sub><em>s</em></sub> between C<sub>q</sub> value of each candidate reference gene and the corresponding global mean was then respectively analyzed under each experimental condition [<a href="#pone.0171120.ref056" class="ref-tip">56</a>]. The expressional stabilities of the candidate reference genes were preliminarily ranked according to their <em>r</em><sub><em>s</em></sub> values and the sRNA with <em>r</em><sub><em>s</em></sub> value closer to 1 was considered more stable.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2" class="link-target"></a><p>The expressional stability of candidate sRNAs with <em>r</em><sub><em>s</em></sub> value &gt; 0 were further assessed using commonly used software geNorm [<a href="#pone.0171120.ref057" class="ref-tip">57</a>]. The linear scale expression quantity of each candidate gene was calculated by the equation 2<sup>(-ΔCq)</sup> (according to the handbook of geNorm<sup>TM</sup> kit with perfect probe, PrimerDesign Ltd.) and loaded in the Excel-based geNorm software. The optimal number of reference genes used under each experimental condition was also recommended by geNorm through pairwise variation analysis.</p>
<a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3" class="link-target"></a><p>To validate the selected reference genes, transcriptional level of let-7 in <em>G</em>. <em>molesta</em> at different developmental stages were detected and compared with different normalizers, including the single one best recommendation hsa16 (NF1), the recommended combination hsa16 + <em>U6</em> + bmo2b (NF(1–3)), one least stable miRNA bmo281 (NF8) ranked by geNorm, and single mse92a (NF12) with the lowest <em>r</em><sub><em>s</em></sub> value. Relative expression levels of let-7 were calculated according to the ΔΔCq method. The geometric mean calculated from the C<sub>q</sub> values of the three sRNAs (hsa16, <em>U6</em>, bmo2b) were used as normalization factor for NF(1–3). For each specific developmental stage, the significant difference among the expression levels of let-7 calculated according to different normalization factors were statistically analyzed using one-way ANOVA with SPSS Statistics 17.0 (SPSS Inc., Chicago, IL, USA). Specifically, Tukey test was adopted for analysis of gene expression in <em>G</em>. <em>molesta</em> of fifth instar, male pupa, female pupa, and female adul. Games-Howell test was performed in analysis of gene expression in <em>G</em>. <em>molesta</em> of first instar, second instar, fourth instar and prepupa. For expressional analysis in <em>G</em>. <em>molesta</em> of third instar and male adult, Games-Howell test was also applied after square root transformation of the relative gene expression values.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec014" name="sec014" data-toc="sec014" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec015" name="sec015" class="link-target" title="Profile of sRNA library"></a>
<h3>Profile of sRNA library</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>In order to understand the miRNA profile, a sRNA library was constructed with a mixture of RNA extracted from <em>G</em>. <em>molesta</em> at varying developmental stages. A total of 16,305,575 raw reads were obtained, including 2,219,608 unique sequences. After trimming of the adaptor and junk reads, filtering out the Rfam reads (rRNA, tRNA, snoRNA, snRNA and other Rfam RNA), and removing the repeats, a total number of 5,998,101 unique reads were finally obtained for further analysis (<strong><a href="#pone-0171120-t001">Table 1</a></strong>).</p>
<a class="link-target" id="pone-0171120-t001" name="pone-0171120-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.t001" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Overview of sRNA sequencing data.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.t001">
              https://doi.org/10.1371/journal.pone.0171120.t001</a></p></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>The length of sequenced sRNAs ranged from 18–26 nucleotides (nt). The length of the total clean reads mainly showed two peaks, with the highest one at 26 nt and the other at 22 nt (<strong><a href="#pone-0171120-g001">Fig 1A</a></strong>). The number of the unique clean reads increased with the lengths of the reads (<strong><a href="#pone-0171120-g001">Fig 1A</a></strong>), and the highest number of miRNAs were found with length of 22 nt (<strong><a href="#pone-0171120-g001">Fig 1B</a></strong>).</p>
<a class="link-target" id="pone-0171120-g001" name="pone-0171120-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.g001" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Length distribution of sRNAs in the sRNA library of <em>G</em>. <em>molesta</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p><strong>A</strong>, The total number of clean and unique reads of sRNAs in lengths of 18–26 nt; <strong>B</strong>, the number of unique miRNAs ranging from 18–26 nt.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.g001">
              https://doi.org/10.1371/journal.pone.0171120.g001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec016" name="sec016" class="link-target" title="Identification of conserved and novel miRNAs"></a>
<h3>Identification of conserved and novel miRNAs</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>After all of the unique clean reads were subjected to the alignment in the miRBase 21.0, transcriptome of <em>G</em>. <em>molesta</em> and the genome of <em>D</em>. <em>plexippus</em>, and were performed with the analysis of the precursors’ hairpin, a total of 536 mature miRNAs were obtained in the sequenced sRNA library of <em>G</em>. <em>molesta</em> (<strong><a href="#pone.0171120.s006">S3 Table</a></strong>). Among the defined miRNAs, 38.06% were mapped to the transcriptome of <em>G</em>. <em>molesta</em> (2.94% conserved and 97.6% novel) (<strong><a href="#pone.0171120.s007">S4 Table</a></strong> and <strong><a href="#pone.0171120.s001">S1 Fig</a></strong>), and 27.24% were mapped to the genome of <em>D</em>. <em>plexippus</em> (67.81% conserved and 32.19% novel) (<strong><a href="#pone.0171120.s008">S5 Table</a></strong> and <strong><a href="#pone.0171120.s001">S1 Fig</a></strong>). The identified 536 miRNAs were composed of 291 conserved and 245 novel miRNAs which were, respectively, belonged to 206 conserved and 230 novel seed-based miRNA families (<strong><a href="#pone.0171120.s009">S6 Table</a></strong>).</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>Among the 291 conserved miRNAs, only 40 miRNA showed high expression levels (&gt; 2000 reads) (<strong><a href="#pone-0171120-t002">Table 2</a></strong>), and miR-10, miR-8, miR-276, miR-14 and miR-281 were identified as the most abundantly expressed conserved miRNAs in <em>G</em>. <em>molesta</em>. Majority of the conserved miRNA showed middle (132 miRNAs with 10–1999 reads) to low (119 miRNAs with &lt; 10 reads) expressional levels (<strong><a href="#pone.0171120.s007">S4</a> </strong>and <strong><a href="#pone.0171120.s008">S5</a> Tables</strong>). In order to detect the conservation of miRNAs among <em>G</em>. <em>molesta</em> and other arthropods, 79 miRNA families identified in the sRNA library of <em>G</em>. <em>molesta</em> were compared within 31 arthropods, and the phylogenetic relationship among these arthropods were constructed with <em>C</em>. <em>elegans</em> and <em>H</em>. <em>sapiens</em> as outgroup taxa (<strong><a href="#pone-0171120-g002">Fig 2</a></strong>). miRNAs were relatively conserved in the same order, and the miRNAs were more conserved in orders with closer phylogenetic relationships. Insects appeared having more expressed miRNAs than those in other arthropods, Nematoda and Chordata, and lepidopterans seemed owing the most abundant miRNAs in Hexapoda. In the present study, miR-9 was conserved in all of the insect species, and miR-745 was conserved in Lepidoptera but absent in other species.</p>
<a class="link-target" id="pone-0171120-g002" name="pone-0171120-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.g002" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Conservation analysis of 79 miRNA famililes within 33 species.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>Conservation analysis of the 79 miRNA families identified from the sRNA library of <em>G</em>. <em>molesta</em> was conducted among 33 species belonging to the three phyla, Arthropoda, Nematoda and Chordata. The miRNA information of the other 32 species was retrieved from the miRBase. Colored box indicated the presence of the conserved miRNA family and the same color indicated species belonging to the same order of insect in Hexapoda, the same class in Arthropoda except for the Hexapoda species, <em>C</em>. <em>elegans</em> in Nematoda, and <em>Homo sampiens</em> in Chordata. The phylogenetic tree was constructed according to previous reports [<a href="#pone.0171120.ref048" class="ref-tip">48</a>,<a href="#pone.0171120.ref049" class="ref-tip">49</a>].</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.g002">
              https://doi.org/10.1371/journal.pone.0171120.g002</a></p></div><a class="link-target" id="pone-0171120-t002" name="pone-0171120-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.t002" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> The most abundant conserved miRNAs in the sRNA library of <em>G</em>. <em>molesta</em>.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.t002">
              https://doi.org/10.1371/journal.pone.0171120.t002</a></p></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3" class="link-target"></a><p>Most (82.04%) of the identified novel miRNAs showed low expressional levels (&lt; 10 reads) (<strong><a href="#pone.0171120.s007">S4</a> </strong>and <strong><a href="#pone.0171120.s008">S5</a> Tables</strong>), 39 (15.92%) of the novel miRNAs showed middle abundance (10–1999 reads) (<strong><a href="#pone-0171120-t003">Table 3</a></strong>), only 5 novel miRNAs exhibited with high abundance (&gt; 2000 reads) (<strong><a href="#pone-0171120-t003">Table 3</a></strong>). The 5 highly expressed novel miRNAs were gmo-miR-PC-5p-15_50867 (127,819 reads), gmo-miR-PC-3p-598_1629 (17,872 reads), gmo-miR-PC-3p-674_1493 (17,424 reads), gmo-miR-PC-5p-1258_867 (14,307 reads) and gmo-miR-PC-5p-1018_1028 (9,152 reads), all of which accounted about 96.94% of all the identified novel miRNA reads. The secondary hairpin structures of the 44 novel miRNAs with middle to high expressed levels in the sRNA library of <em>G</em>. <em>molesta</em> were all predicted (<strong><a href="#pone-0171120-g003">Fig 3</a></strong>). The 44 miRNAs could be divided into three types: (1) 10 miRNAs derived from either the 5’ or 3’ mature miRNAs of the 5 corresponding precursors, (2) 27 miRNAs only from the 5’ mature miRNAs, and (3) 17 miRNAs only from the 3’ mature miRNAs (<strong><a href="#pone-0171120-t003">Table 3</a> </strong>and <strong><a href="#pone-0171120-g003">Fig 3</a></strong>).</p>
<a class="link-target" id="pone-0171120-g003" name="pone-0171120-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.g003" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Predicted secondary structures of the 44 novel miRNAs with middle to high abundance in the sRNA library of <em>G</em>. <em>molesta</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1" class="link-target"></a><p>The mature miRNAs located in the 5’ arm are shaded in yellow, whereas the mature miRNAs located in the 3’ arm are shaded in blue.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.g003">
              https://doi.org/10.1371/journal.pone.0171120.g003</a></p></div><a class="link-target" id="pone-0171120-t003" name="pone-0171120-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.t003" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Novel miRNAs in <em>G</em>. <em>molesta</em>.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.t003">
              https://doi.org/10.1371/journal.pone.0171120.t003</a></p></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4" class="link-target"></a><p>In order to verify the Illumina sequencing, the expressional quantity and sequence accuracy of the 10 conserved and 10 novel miRNA determined in the sRNA library of <em>G</em>. <em>molesta</em> were validated with the qPCR method and Sanger sequencing. The result of qPCR demonstrated that the abundance of most selected miRNAs showed concordant expressional profiles with the corresponding reads accounts in sRNA library, although bmo9a exhibited very low expressional level in the qPCR test (<strong><a href="#pone-0171120-g004">Fig 4</a></strong>). Sequence alignment illustrated that most of the miRNA sequences sequenced by Illumina platform were in accordance with those sequenced by Sanger method. The substitutions at the 5’end produced by the artificially added bases in the forward primer and the differences of one or two bases at the 3’ end resulted from sequencing deviation all have no influence on annotation and target prediction of miRNAs (<strong><a href="#pone.0171120.s010">S7 Table</a></strong>). Therefore, Illumina sequencing is a reliable platform for the overall understanding of miRNA profile in <em>G</em>. <em>molesta</em>.</p>
<a class="link-target" id="pone-0171120-g004" name="pone-0171120-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.g004" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Comparative analysis of qPCR detection with the expressional profiling in sRNA sequencing.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1" class="link-target"></a><p>The <em>y</em>-axis indicates the abundance of miRNAs in the sRNA library and the relative transcriptional level of miRNAs detected using qPCR. The <em>x</em>-axis represents the specific assessed miRNAs, as 1: let-7, 2: mse92a, 3: bmo998, 4: pxy-6497-p5, 5: pxy-6497-p3, 6: bmo2b, 7: bmo281, 8: bmo279a, 9: bmo9a, 10: bmo305, 11: PC-5P-671, 12: PC-5P-1276, 13: PC-3P-1433, 14: PC-3P-4029, 15: PC-3P-16268, 16: PC-5P-35148, 17: PC-5P-15, 18: PC-5P-1018, 19: PC-5P-1258, 20: PC-3P-598.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.g004">
              https://doi.org/10.1371/journal.pone.0171120.g004</a></p></div></div>

<div id="section3" class="section toc-section"><a id="sec017" name="sec017" class="link-target" title="Cluster of miRNAs"></a>
<h3>Cluster of miRNAs</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>miRNAs that were transcribed from the same primary miRNA (pri-miRNA) frequently exhibit clustered distribution in the genome and were usually expressed as a polycistronic transcript with co-regulation activity in biological networks. The analysis of the miRNA cluster referred to the genome of <em>D</em>. <em>plexippus</em> (<strong><a href="#pone-0171120-t004">Table 4</a></strong>) revealed that more miRNAs in <em>G</em>. <em>molesta</em> were linked within 10–50 kb genomic distances, with 28.92% of miRNAs clustered within 50 kb and 26.51% miRNAs closely linked within 10 kb. There were 22.89% and 21.69% miRNAs respectively clustered within 5 kb and 3 kb genomic distance. The mean number of miRNAs per cluster was 3.2, 2.9, 2.7 and 2.8 correspondingly in 50 kb, 10 kb, 5kb and 3 kb genomic distances. At the same genomic distance, a positive and correlated expression relationship has been normally detected among miRNAs within the same cluster ID, or at least two of which appeared with similar expressional profiles. However, the expression patterns (reads account) between distance-neighboring miRNAs of the same cluster varied among different clusters.</p>
<a class="link-target" id="pone-0171120-t004" name="pone-0171120-t004"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.t004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.t004" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.t004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.t004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 4. </span> miRNA cluster analysis in <em>G</em>. <em>molesta</em>.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.t004">
              https://doi.org/10.1371/journal.pone.0171120.t004</a></p></div></div>

<div id="section4" class="section toc-section"><a id="sec018" name="sec018" class="link-target" title="KEGG analysis of predicted target genes"></a>
<h3>KEGG analysis of predicted target genes</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>KEGG is a database that helps understanding the molecular interaction and reaction networks in cells and organisms (KEGG PATHWAY Database: <a href="http://www.kegg.jp/kegg/pathway.html">http://www.kegg.jp/kegg/pathway.html</a>). In this study, the KEGG analysis of predicted target genes of miRNAs was used to increase our understanding about the biological functions of the identified miRNAs in the metabolism and development of <em>G</em>. <em>molesta</em>. A total of 1,896 target genes of the miRNAs in the <em>G</em>. <em>molesta</em> were significantly enriched in the KEGG analysis, among which 410 genes were matched to 16 KEGG pathways, including 10 pathways of amino acid metabolism and protein processing (ko00270, ko00330, ko00650, ko00250, ko00280, ko00640, ko00350, ko00410, ko04964 and ko04141), 2 pathways involved in nucleotide repairing (ko03420) and mRNA surveillance (ko03015), 1 pathway of plant-pathogen interaction (ko04626), 2 pathways participating in development regulation (ko04350 and ko04391), and 1 pathway involved in circadian rhythm (ko04710) (<strong><a href="#pone.0171120.s011">S8 Table</a></strong>). The further gene network analysis demonstrated that many conserved and novel miRNAs in <em>G</em>. <em>molesta</em> participated in the circadian rhythm through networks among clock genes (<em>cry1</em>, <em>clock</em> and <em>per</em>) and genes involving in growth and development regulation (<em>TGF-beta</em>, <em>skp1</em>, <em>fbw1B</em>, and <em>cul1</em>) (<strong><a href="#pone-0171120-g005">Fig 5</a></strong>).</p>
<a class="link-target" id="pone-0171120-g005" name="pone-0171120-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.g005" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Network prediction of the circadian rhythm pathway in <em>G</em>. <em>molesta</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1" class="link-target"></a><p>The miRNAs in magenta and cyan circles respectively represent the conserved and novel miRNAs. Genes in yellow diamonds are clock genes and those involved in growth and development regulation. The regulatory network of miRNAs and their target genes for the circadian rhythm in <em>G</em>. <em>molesta</em> are linked with blue lines.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.g005">
              https://doi.org/10.1371/journal.pone.0171120.g005</a></p></div></div>

<div id="section5" class="section toc-section"><a id="sec019" name="sec019" class="link-target" title="Expressional stability of candidate reference genes under different experimental conditions"></a>
<h3>Expressional stability of candidate reference genes under different experimental conditions</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>A universal method, mean expression value normalization (global mean), was adopted for preliminarily assess the expressional stabilities of the candidate reference genes. This method demonstrated with better reduction of technical variation and more accurate appreciation of biological change [<a href="#pone.0171120.ref034" class="ref-tip">34</a>]. After calculation and comparison of the Spearman’s rank correlation coefficient <em>r</em><sub><em>s</em></sub>, eight candidate reference sRNAs (bmo-miR-2b-3p_1ss22GC, pxy-mir-6497-p3_1ss7CT, hsa-miR-16-5p, bmo-miR-281-3p_L-2R+2, <em>U6</em> snRNA, bmo-miR-279a_R+2, <em>5S</em> rRNA and bmo-miR-9a-5p) were preliminarily identified by their positive correlation with the global mean expressions (<strong><a href="#pone-0171120-t005">Table 5</a></strong>), and were further evaluated with geNorm software.</p>
<a class="link-target" id="pone-0171120-t005" name="pone-0171120-t005"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.t005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.t005" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.t005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.t005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 5. </span> Information of candidate reference genes used for qRT-PCR.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.t005">
              https://doi.org/10.1371/journal.pone.0171120.t005</a></p></div><a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2" class="link-target"></a><p>geNorm software ranks the reference gene according to their M values (average expression stability) and provides optimal recommendation of the number of reference genes based on the pairwise variation (V) analysis. In this study, the expressional stabilities of the 8 candidate reference genes were ranked under eight different experimental conditions (<strong><a href="#pone.0171120.s002">S2 Fig</a></strong>). The candidate reference, bmo2b, was ranked with high stability under seven experimental conditions, except for temperature treatment; <em>U6</em> and bmo9a were all ranked as stable references under four experimental conditions. In combination with the V value (<strong><a href="#pone.0171120.s003">S3 Fig</a></strong>), the recommendations of internal normalizers for analysis of miRNA expressional profiles of <em>G</em>. <em>molesta</em> using qRT-PCR were determined for each of the eight specific experimental conditions. For intrinsic biotic conditions, 3 sRNAs (hsa16, <em>U6</em> and bmo2b) used together were recommended for study of different developmental stages. Combined usage of 6 sRNAs (hsa16, bmo2b, <em>U6</em>, <em>5S</em>, bmo279 and bmo9a) was recommended for miRNA quantification in different tissues (<strong><a href="#pone-0171120-t006">Table 6</a></strong>). For different exogenous stresses, it is recommended 2 sRNAs (bmo281 and <em>U6</em>) for temperature changes, 3 miRNAs (bmo2b, bmo279 and hsa16) for photoperiod treatment, 2 miRNAs (bmo279 and bmo9a) for starvation, 2 miRNAs (bmo2b and bmo9a) for JH exposure, 2 miRNAs (bmo2b and bmo281) for dsRNA treatment, and 2 sRNAs (bmo281 and <em>U6</em>) for insecticide application (<strong><a href="#pone-0171120-t006">Table 6</a></strong>).</p>
<a class="link-target" id="pone-0171120-t006" name="pone-0171120-t006"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.t006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.t006" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.t006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.t006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 6. </span> Recommended reference genes for expressional qualification of microRNA in <em>G</em>. <em>molesta</em> under different experimental conditions.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.t006">
              https://doi.org/10.1371/journal.pone.0171120.t006</a></p></div></div>

<div id="section6" class="section toc-section"><a id="sec020" name="sec020" class="link-target" title="Validation of reference gene selection"></a>
<h3>Validation of reference gene selection</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>To evaluate the performance of selected reference genes, the expression level of let-7, which participates in development regulation in insects [<a href="#pone.0171120.ref058" class="ref-tip">58</a>,<a href="#pone.0171120.ref059" class="ref-tip">59</a>] and showed dynamic expressional profiles in different developmental stages of <em>Bombyx mori</em> [<a href="#pone.0171120.ref060" class="ref-tip">60</a>] was examined in different developmental stages of <em>G</em>. <em>molesta</em>. Similar expression profiles of let-7 in different developmental stages of <em>G</em>. <em>molesta</em> were detected when single best recommended reference NF(1) (hsa16) and the optimal recommendation NF(1–3) (hsa16 + <em>U6</em> + bmo2b) were used as normalization factors, only with different expression patterns in the second <em>vs</em> third instars and fifth instar <em>vs</em> male pupa (<strong><a href="#pone-0171120-t007">Table 7</a></strong>). However, the use of single least stable reference NF(8) and NF(12), respectively, showed significant down-regulation and up-regulation of let-7 within all tested developmental stages of <em>G</em>. <em>molesta</em> (<strong><a href="#pone-0171120-t007">Table 7</a></strong>).</p>
<a class="link-target" id="pone-0171120-t007" name="pone-0171120-t007"></a><div class="figure" data-doi="10.1371/journal.pone.0171120.t007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0171120.t007" data-doi="info:doi/10.1371/journal.pone.0171120" data-uri="info:doi/10.1371/journal.pone.0171120.t007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0171120.t007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0171120.t007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0171120.t007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0171120.t007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 7. </span> Expressional profiles of let-7 in <em>G</em>. <em>molesta</em> quantified according to different normalizers.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0171120.t007">
              https://doi.org/10.1371/journal.pone.0171120.t007</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="sec021" name="sec021" data-toc="sec021" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>The miRNA information of several Lepidopteran insects has been identified with reference to their genomic information [<a href="#pone.0171120.ref020" class="ref-tip">20</a>,<a href="#pone.0171120.ref061" class="ref-tip">61</a>] or genome sequences of the model insect silkworm (<em>B</em>. <em>mori</em>) [<a href="#pone.0171120.ref062" class="ref-tip">62</a>,<a href="#pone.0171120.ref063" class="ref-tip">63</a>]. In the present study, a pooled sRNA library of <em>G</em>. <em>molesta</em> was prepared from mixed developmental stages of oriental fruit moth, and was further analyzed in reference to its own transcriptome and the genome of <em>D</em>. <em>plexippus</em> which was found with the highest similarity to the transcripts of <em>G</em>. <em>molesta</em> in the phylogenetic conservation analysis (unpublished data). A total of 536 mature miRNAs composed of 291 previously reported and 245 novel ones were finally identified in <em>G</em>. <em>molesta</em>. In previous studies of Lepidopteran insects, 55 conserved and 202 novel miRNAs were found in <em>B</em>. <em>mori</em> [<a href="#pone.0171120.ref061" class="ref-tip">61</a>], 163 conserved and 13 novel miRNAs were identified in <em>M</em>. <em>sexta</em> [<a href="#pone.0171120.ref020" class="ref-tip">20</a>], and 97, 91 and 69 conserved together with 1, 8 and 383 novel ones were respectively identified in the miRNA analyses of <em>Helicoverpa armigera</em>, <em>Spodoptera litura</em> and <em>P</em>. <em>xylostella</em> [<a href="#pone.0171120.ref062" class="ref-tip">62</a>,<a href="#pone.0171120.ref063" class="ref-tip">63</a>]. With the cumulative availability of miRNA data and the increase of sequencing depth, more information of mature miRNA would be uncovered and more novel miRNA could be identified in lepidopterans, which would provide useful repertoires for clarification of the modulation complex in insects and other organisms.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>Among the 291 conserved miRNAs identified in the sRNA transcriptome of <em>G</em>. <em>molesta</em>, 40 miRNAs showed highly expressed levels, which probably play critical roles in various regulatory processes in the oriental fruit moth. The miRNAs with the highest account of reads in this study including miR-10, miR-8, miR-281, and miR-263 were also identified as the most abundantly expressed ones in <em>A</em>. <em>gambiae</em> [<a href="#pone.0171120.ref064" class="ref-tip">64</a>]. miR-10 and miR-8 were found with high abundance in the digestive tract of green bottle fly maggot <em>Lucilia sericata</em> and both were considered involved in secretions [<a href="#pone.0171120.ref065" class="ref-tip">65</a>]. A study on <em>S</em>. <em>exigua</em> demonstrated that miR-10 also participated in developmental regulation since oral feeding using synthetic miRNA mimics of miR-10-1a resulted in suppressed growth and increased mortality [<a href="#pone.0171120.ref066" class="ref-tip">66</a>]. miR-8 showed pleiotropic regulatory functions in insects, including neurogenesis through targeting <em>atrophin</em> mRNA or regulating synaptic activity [<a href="#pone.0171120.ref067" class="ref-tip">67</a>,<a href="#pone.0171120.ref068" class="ref-tip">68</a>], homeostasis of innate immunity by altering the expression of antimicrobial peptides [<a href="#pone.0171120.ref069" class="ref-tip">69</a>], and development tuning via responding to ecdysone [<a href="#pone.0171120.ref070" class="ref-tip">70</a>]. miR-281 is another multi-functional miRNA in insects. It is involved in regulation of long chain fatty acid synthesis and cuticle formation in <em>M</em>. <em>sexta</em> [<a href="#pone.0171120.ref071" class="ref-tip">71</a>], highly expressed in Malpighian tubules and participating in developmental regulation through targeting and suppressing the transcription of ecdysone receptor–B (EcRB) in <em>B</em>. <em>mori</em> [<a href="#pone.0171120.ref072" class="ref-tip">72</a>], and facilitating the replication of dengue virus (DENV-2) in vector mosquito <em>Aedes albopictus</em> by specifically induced in female midgut upon DENV-2 infection [<a href="#pone.0171120.ref073" class="ref-tip">73</a>]. The conserved miR-263 was abundantly expressed in pupae and may participate in temporal regulation during silkworm development [<a href="#pone.0171120.ref074" class="ref-tip">74</a>]. The other two miRNAs, miR-276 and miR-14 that showed reads account at top levels in the sRNA library of <em>G</em>. <em>molesta</em>, were also found with regulatory effects in the development, reproduction and immunity of insects. miR-276 was noted as one of the miRNAs for identifying reproductive stages of <em>A</em>. <em>mellifera</em> independent of caste [<a href="#pone.0171120.ref075" class="ref-tip">75</a>] and was found promoting egg-hatching synchrony of locusts through up-regulation of the target, a transcription coactivator gene <em>brahma</em> (<em>brm</em>) [<a href="#pone.0171120.ref010" class="ref-tip">10</a>]. In studies of <em>Drosophila</em>, miR-14 has been considered as an important regulator in the development process, mainly through negative modulation of the Hedgehog signaling [<a href="#pone.0171120.ref076" class="ref-tip">76</a>], positive regulation of the autoregulatory loop of ecdysone signaling [<a href="#pone.0171120.ref077" class="ref-tip">77</a>] and autophagy regulation during salivary gland cell death by targeting 1,4,5-triphospate (IP3) signaling [<a href="#pone.0171120.ref078" class="ref-tip">78</a>]. miR-14 was also found involved in the interactions of insect and virus, e.g. <em>H</em>. <em>armigera</em> single nucleopolyhedrovirus (HaSNPV) manipulated EcR transcription in <em>H</em>. <em>armigera</em> through down regulation of miR-14 in host insect [<a href="#pone.0171120.ref079" class="ref-tip">79</a>] and miR-14 in the white-backed planthopper <em>Sogatella furcifera</em> may participate in the immune response against the southern rice black-streaked dwarf virus vectored by the planthopper [<a href="#pone.0171120.ref080" class="ref-tip">80</a>].</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>Analysis about organisms of different kingdoms demonstrated that large fractions of miRNAs share conservation among closely-related species and even species with distant phylogenetic relationships, suggesting evolutionarily conserved regulatory functions across species [<a href="#pone.0171120.ref081" class="ref-tip">81</a>]. In the present study, the 79 miRNA families found in the sRNA library of <em>G</em>. <em>molesta</em> were detected to be conserved among the Lepidoptera insects, yet most of the miRNA families numbered from miR-1175 to miR-8506 were absent in other species. More miRNAs among the 79 families were detected to be present in Hexapoda species, whereas notably less number of miRNA families were found in species in Brachiopoda, Chiopoda and Arachinida, although they all belong to Arthropods. Let-7 and miR-124 were conserved among Arthropoda, Nematoda and Chordata species in our analysis. Previous analyses suggested that many miRNA families, such as let-7, miR-10, miR-99 and miR-125 were highly conserved among animal species [<a href="#pone.0171120.ref046" class="ref-tip">46</a>], whereas miR-99 was not found in the sRNA library of <em>G</em>. <em>molesta</em>, while miR-10 and miR-125 were conserved in most species but absent in <em>C</em>. <em>elegans</em>. Numerous studies have demonstrated that miRNAs showed widely conservative evolution, and thus have been proposed to be used as additional sequence markers in combination of mitochondrial or gnomic DNA for evolutionary modeling and phylogenetic analysis among various taxa [<a href="#pone.0171120.ref082" class="ref-tip">82</a>,<a href="#pone.0171120.ref083" class="ref-tip">83</a>]. Insects have been considered as an excellent model for study about regulatory functions of miRNAs in conserved regulatory pathways between vertebrates and invertebrates [<a href="#pone.0171120.ref003" class="ref-tip">3</a>]. Besides of sequence conservation, miRNAs were also demonstrated with spatial and temporal expression conservation. A comparative study among bilaterian animals suggested that the tissue specificity of ancient microRNAs was highly conserved [<a href="#pone.0171120.ref084" class="ref-tip">84</a>]. A recent study on <em>Drosophila</em> discovered that the temporal expression of orthologous microRNAs could be more conserved than their sequences and the hourglass pattern of miRNA expression are highly similar, not only among the miRNAs with highly conserved sequences, but also for rapidly evolving orthologs [<a href="#pone.0171120.ref085" class="ref-tip">85</a>]. Therefore, as an important regulation component at posttranscriptional level between mRNA and gene-encoding protein, miRNA and its expression pattern were considered critical in understanding the evolution of developmental gene expression.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Similar to previous studies in planthopper <em>L</em>. <em>striatellus</em> [<a href="#pone.0171120.ref046" class="ref-tip">46</a>] and diamondback moth <em>P</em>. <em>xylostella</em> [<a href="#pone.0171120.ref063" class="ref-tip">63</a>], the abundances of novel miRNAs were also found very low in <em>G</em>. <em>molesta</em> and the 5 novel miRNAs with the highest abundance may play important roles in the development of <em>G</em>. <em>molesta</em>. The secondary structure prediction demonstrated that only a small fraction of mature ones were generated from both arms and most of the mature novel miRNAs were produced from either the 3’ or 5’ arms of the hairpins with a slight bias toward 5’ arm usage (5p/3p proportion was around 1.59). The arm usage preference differed in various <em>Diptera</em>, such as 5’ arm-bias in <em>Drosophila</em> [<a href="#pone.0171120.ref086" class="ref-tip">86</a>], but 3’ arm-bias in <em>A</em>. <em>gambiae</em> [<a href="#pone.0171120.ref064" class="ref-tip">64</a>]. Studies in mammalians illustrated that selection and usage of preferred arm could be dynamically regulated in a development and tissue specific pattern [<a href="#pone.0171120.ref087" class="ref-tip">87</a>]. The study in <em>A</em>. <em>gambiae</em> revealed that arm-switching event could happen after blood feeding [<a href="#pone.0171120.ref064" class="ref-tip">64</a>], all of which suggest that miRNAs are expressed under a condition-specific manner and arm switching can significantly enrich the regulatory capacity of miRNA.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>As the next-generation sequencing (NGS) method, Illumina sequencing enables the high-throughput and comprehensive understanding about the genome, transcriptome, epigenome or microbiome of a variety of organisms [<a href="#pone.0171120.ref088" class="ref-tip">88</a>–<a href="#pone.0171120.ref091" class="ref-tip">91</a>]. However, Illumina sequencing has been found more prone to produce deviation than Sanger technique, as it is vulnerable to be contaminated in the process of library preparation and accompanied with deviations resulted from base bias and subjective supposition in the bio-information analysis. In the present study, qPCR and Sanger sequencing were adopted for validation of the sRNA library of <em>G</em>. <em>molesta</em> sequenced by Illumina platform and concordant results between different methods were detected for most of the miRNAs. The divergences between Illumina sequencing and qPCR were mainly due to the limited replication of Illumina analysis, thus, qPCR validation is indispensible for sophisticated measurement about the expressional pattern of specific miRNA of interest. Alignment detected sequence differences happened in some miRNAs between Sanger and Illumina sequencing results. However, most parts of these miRNA sequences were in agreement between the two methods and the seed regions for target prediction were intact after eliminating the sequence deviations at the 5’ end caused by primer modification for adjustment of GC content. Therefore, Illumina sequencing is a reliable platform for the overall understanding of miRNA profile in <em>G</em>. <em>molesta</em>.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6" class="link-target"></a><p>Cluster analysis is an important part for understanding the biological network and evolutionary conservation of miRNA. In this study, the clusters of the conserved and novel miRNAs determined in <em>G</em>. <em>molesta</em> were analyzed in reference to the genome of <em>D</em>. <em>plexippus</em>. Fifteen clusters were identified separately at 50 kb, 10 kb, 5kb and 3 kb genomic distances. Most of the miRNAs in the same cluster showed concordant expressional abundance within the same genomic distance. A study showed that expressional levels of clustered miRNAs in mosquito libraries were highly correlated and the mature miRNAs in the same cluster exhibited a strong bias towards the same arm selection [<a href="#pone.0171120.ref064" class="ref-tip">64</a>]. However, the arm selection of the miRNAs in the same cluster showed in a more dynamic manner in this study, with only a small bias towards the 5’ arm selection in the same cluster. Conservation of miRNA clusters among related species would provide evolutionary evidence for functional shift of miRNA in insects. A study found that the cluster organized in mir-2/mir-13/mir-71 was highly conserved in insects [<a href="#pone.0171120.ref086" class="ref-tip">86</a>], which was not detected in <em>G</em>. <em>molesta</em>. Cluster analysis is highly dependent upon the corresponding genome sequences, hence in future, with the information of genome sequence of <em>G</em>. <em>molesta</em> and the availability of sequence information in its phylogenetic related species, more useful information of miRNA data would probably be revealed through comparative studies among agricultural pests.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7" class="link-target"></a><p>Target prediction and KEGG pathway analysis are important to understand the regulatory networks of miRNAs. In this study, except for the majority pathways in the processing, repairing and surveillance of nucleic acid, amino acid and protein plus one pathway function in the plant-pathogen interaction, three pathways for controlling the growth and development modulation (including two pathways involved in development regulation and one pathway participating in circadian rhythm) were identified in the sRNA library constructed from different developmental stages of <em>G</em>. <em>molesta</em>. The network prediction of the identified circadian rhythm pathway further revealed the close relationship among a range of miRNAs, clock genes and developmental regulation genes in <em>G</em>. <em>molesta</em>. A majority of insect developmental processes exhibit circadian rhythm (including growth, ecdysis, metamorphosis, diapause, and eclosion) and developmental circadian rhythm can be profoundly impacted by environmental changes, especially the daily or seasonally thermal and photoperiodic alterations [<a href="#pone.0171120.ref092" class="ref-tip">92</a>]. Correspondingly, most organisms have evolved circadian clock to adjust their development and metamorphism rhythm for coping with environmental changes [<a href="#pone.0171120.ref093" class="ref-tip">93</a>]. Recent studies have detected that the miRNA might be an important regulator of circadian rhythmicity at posttranscriptional level in various taxa of organisms [<a href="#pone.0171120.ref094" class="ref-tip">94</a>,<a href="#pone.0171120.ref095" class="ref-tip">95</a>]. The overall understanding about the network of miRNA and their target genes in the developmental regulation of insects would provide useful information for the monitoring and management of agricultural pest.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8" class="link-target"></a><p>Since the discovery about the vital roles of miRNA at post-transcriptional level, precisely expressional profiles of miRNA under specific conditions have been considered as the prerequisite for understanding its regulatory functions. Identical to the quantification of coding genes, using suitable normalizers synchronically processed with tested genes is also considered as an effective way for eliminating the errors from materials and experimental manipulation and realizing accurate evaluation of miRNA profiles [<a href="#pone.0171120.ref096" class="ref-tip">96</a>]. In recent years, numerous studies on selection and validation of reference genes for miRNA quantification have been published being relevant to plants, human, vertebrates and nematode [<a href="#pone.0171120.ref045" class="ref-tip">45</a>,<a href="#pone.0171120.ref047" class="ref-tip">47</a>,<a href="#pone.0171120.ref097" class="ref-tip">97</a>,<a href="#pone.0171120.ref098" class="ref-tip">98</a>], whereas few studies have been reported about insects. In the present study, on the basis of the sRNA library of <em>G</em>. <em>molesta</em>, one small nuclear RNA <em>U6</em>, one small ribosomal RNA <em>5SrRNA</em> and ten miRNAs were preliminarily selected as candidate references according to the previous studies [<a href="#pone.0171120.ref045" class="ref-tip">45</a>–<a href="#pone.0171120.ref047" class="ref-tip">47</a>,<a href="#pone.0171120.ref051" class="ref-tip">51</a>–<a href="#pone.0171120.ref054" class="ref-tip">54</a>] and their stabilities under two biotic and six abiotic conditions were further assessed with global mean method and geNorm algorithm. Normally, four statistical methods, geNorm, Normfinder, delta Ct and Bestkeeper are adopted for expressional stability analysis. Based on different algorithms, however, inconsistent outcomes are usually obtained from the four software. Therefore, the comprehensive ranking of expressional stability is hard to determine from parallel analysis with the four algorithms. In the present study, “global mean”, a universal method for analyzing the transcriptional stability of microRNA [<a href="#pone.0171120.ref034" class="ref-tip">34</a>], was adopted for preliminary screening of the candidate reference genes. A final recommendation of suitable normalizers was then clearly acquired with further evaluation by geNorm. In the assessment of reliable internal controls for miRNA expressional quantification in <em>C</em>. <em>elegans</em>, a genome-wide RNA-seq was conducted for preliminary selection of control miRNAs with minimal variation, and similarly, the final stabilities of candidate reference miRNAs were then ranked by applying the common geNorm logarithm [<a href="#pone.0171120.ref097" class="ref-tip">97</a>]. The comprehensive analysis in the present study detected that intrinsic biotic factors including developmental stages and tissues impacted more on the stability of miRNA expression in <em>G</em>. <em>molesta</em>, as a greater number of reference genes should be used for normalization in samples of development stages or tissues. Similarly, investigation in the Chinese perch revealed that embryonic developmental stage was an important factor to the variability of miRNA expression [<a href="#pone.0171120.ref047" class="ref-tip">47</a>]. In previous studies of miRNA evaluation, <em>U6</em> has always been empirically chosen as an internal control and <em>U6</em> has been recommended as a stable reference gene under half of the tested conditions in our study. However, the expressional stabilities of <em>U6</em> and other small nuclear RNAs were actually not be acceptable as endogenous controls in many experimental conditions [<a href="#pone.0171120.ref045" class="ref-tip">45</a>,<a href="#pone.0171120.ref047" class="ref-tip">47</a>]. The present and previous studies all illustrated that misusage of an internal control gene would lead to notably distorted results and misunderstanding of gene expressional profiles [<a href="#pone.0171120.ref099" class="ref-tip">99</a>].</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9" class="link-target"></a><p>In summary, a total of 536 mature miRNAs including 291 conserved and 245 novel miRNAs was identified in <em>G</em>. <em>molesta</em>. The abundance, conservation and cluster of identified miRNA were analyzed. The KEGG pathway analysis and network prediction of target genes demonstrated that the network composed of miRNAs, clock genes and developmental regulation genes probably play critical roles in the regulation of developmental circadian in <em>G</em>. <em>molesta</em>. Furthermore, suitable reference genes were selected and validated for study on miRNA expressional profile in <em>G</em>. <em>molesta</em> under two biotic and six abiotic experimental conditions. The present study provides an overview of miRNA profile in <em>G</em>. <em>molesta</em> and may serve as a basic reference for evaluation of miRNA abundance in this pest and other insects. Further studies, such as analyzing of the interaction relationship between miRNAs and their targets and deciphering of the regulatory functions and mechanisms of specific miRNAs will shed light on the deeper interpretation of the miRNA-involved post-transcriptional regulation in <em>G</em>. <em>molesta</em> and might also provide a useful foundation for development of new targets or genetic-based techniques for agricultural pest control in the future.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec022" name="sec022" data-toc="sec022" class="link-target" title="Supporting information"></a><h2>Supporting information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0171120"></div><div class="supplementary-material"><a name="pone.0171120.s001" id="pone.0171120.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s001">S1 Fig. </a>Reads frequency of the identified mature miRNAs mapped to different sequence databases.</h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">34.7% was mapped to miRBase, 38.06% was mapped to the transcriptome of <em>G</em>. <em>molesta</em> and 27.24% were mapped to the genome of <em>D</em>. <em>plexippus</em>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s001">https://doi.org/10.1371/journal.pone.0171120.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s002" id="pone.0171120.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s002">S2 Fig. </a>Average expression stability and ranking of eight candidate reference genes under eight different experimental conditions calculated using geNorm.</h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">The average expression stability (M value) was calculated for each candidate and the sRNA with the lowest M value is considered as the most stably expressed reference gene.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s002">https://doi.org/10.1371/journal.pone.0171120.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s003" id="pone.0171120.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s003">S3 Fig. </a>Determination of the optimal number of reference genes for accurate normalization of miRNA transcription under different experimental conditions.</h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">Average pairwise variations (V values) were calculated between the normalization factors NFn and NFn+1, and the addition of reference gene is not required when the V value is below 0.15.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s003">https://doi.org/10.1371/journal.pone.0171120.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s004" id="pone.0171120.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s004">S1 Table. </a>Information of miRNAs for validation of sRNA library sequencing.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s004">https://doi.org/10.1371/journal.pone.0171120.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s005" id="pone.0171120.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s005">S2 Table. </a>Primer information for dsRNA synthesis.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s005">https://doi.org/10.1371/journal.pone.0171120.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s006" id="pone.0171120.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s006">S3 Table. </a>Information of all expressed mature miRNAs predicted in the sRNA library constructed with <em>G</em>. <em>molesta</em> of varying developmental stages.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s006">https://doi.org/10.1371/journal.pone.0171120.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s007" id="pone.0171120.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s007">S4 Table. </a>Predicted <em>G</em>. <em>molesta</em> miRNAs with reference to its own transcriptome data.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s007">https://doi.org/10.1371/journal.pone.0171120.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s008" id="pone.0171120.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s008">S5 Table. </a>Predicted <em>G</em>. <em>molesta</em> miRNAs with reference to the genomic data of <em>D</em>. <em>plexippus</em>.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s008">https://doi.org/10.1371/journal.pone.0171120.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s009" id="pone.0171120.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s009">S6 Table. </a>Seeds-based family distribution of conserved and novel miRNAs identified in <em>G</em>. <em>molesta</em>.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s009">https://doi.org/10.1371/journal.pone.0171120.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s010" id="pone.0171120.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s010">S7 Table. </a>Comparative analysis of the miRNA sequences between Illumina sequencing and Sanger sequencing.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s010">https://doi.org/10.1371/journal.pone.0171120.s010</a></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0171120.s011" id="pone.0171120.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0171120.s011">S8 Table. </a>KEGG significant enrichment analysis for target genes of miRNAs in <em>G</em>. <em>molesta</em>.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0171120.s011">https://doi.org/10.1371/journal.pone.0171120.s011</a></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The authors thank Dr. Huaijiang Zhang (Institute of Pomology in Liaoning province) for the sample collection of <em>G</em>. <em>molesta</em>. Special thanks go to Prof. Qingxiao Li (University of Hawaii at Manoa) for copyediting our manuscript.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>

<ol class="simple">

<li><strong>Conceptualization:</strong> ZL XXL QWZ.</li>

<li><strong>Data curation:</strong> XW ZL.</li>

<li><strong>Formal analysis:</strong> XW ZL.</li>

<li><strong>Funding acquisition:</strong> ZL XXL.</li>

<li><strong>Investigation:</strong> XW YSL ZL.</li>

<li><strong>Methodology:</strong> ZL XXL QWZ.</li>

<li><strong>Project administration:</strong> ZL XXL QWZ.</li>

<li><strong>Resources:</strong> XW YSL JZ XXL ZL.</li>

<li><strong>Supervision:</strong> ZL XXL.</li>

<li><strong>Validation:</strong> XW YSL ZL.</li>

<li><strong>Visualization:</strong> ZL.</li>

<li><strong>Writing – original draft:</strong> ZL.</li>

<li><strong>Writing – review &amp; editing:</strong> ZL XXL YSL.</li>

</ol>
</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0171120.ref001" id="pone.0171120.ref001" class="link-target"></a>Bartel DP. MicroRNA: genomics, biogenesis, mechanism, and function. Cell. 2004; 116: 281–297. pmid:14744438 <ul class="reflinks"><li><a href="#" data-author="Bartel" data-cit="BartelDP.%20MicroRNA%3A%20genomics%2C%20biogenesis%2C%20mechanism%2C%20and%20function.%20Cell.%202004%3B%20116%3A%20281%E2%80%93297.%2014744438" data-title="MicroRNA%3A%20genomics%2C%20biogenesis%2C%20mechanism%2C%20and%20function" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14744438" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA%3A+genomics%2C+biogenesis%2C+mechanism%2C+and+function+Bartel+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0171120.ref002" id="pone.0171120.ref002" class="link-target"></a>Cullen BR. Virus and microRNAs. Nat Genet. 2006; 38: S25–30.  pmid:16736021 <ul class="reflinks"><li><a href="#" data-author="Cullen" data-cit="CullenBR.%20Virus%20and%20microRNAs.%20Nat%20Genet.%202006%3B%2038%3A%20S25%E2%80%9330.%20doi%3A%2010.1038%2Fng1793%2016736021" data-title="Virus%20and%20microRNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16736021" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Virus+and+microRNAs+Cullen+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0171120.ref003" id="pone.0171120.ref003" class="link-target"></a>Asgari S. MicroRNA functions in insects. Insect Biochem Mol Biol. 2013; 43: 388–397.  pmid:23103375 <ul class="reflinks"><li><a href="#" data-author="Asgari" data-cit="AsgariS.%20MicroRNA%20functions%20in%20insects.%20Insect%20Biochem%20Mol%20Biol.%202013%3B%2043%3A%20388%E2%80%93397.%20doi%3A%2010.1016%2Fj.ibmb.2012.10.005%2023103375" data-title="MicroRNA%20functions%20in%20insects" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23103375" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+functions+in+insects+Asgari+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0171120.ref004" id="pone.0171120.ref004" class="link-target"></a>Lucas K, Raikhel A. Insect microRNAs: Biogenesis, expression profiling and biological functions. Insect Biochem Mol Biol. 2013; 43: 24–38.  pmid:23165178 <ul class="reflinks"><li><a href="#" data-author="Lucas" data-cit="LucasK%2C%20RaikhelA.%20Insect%20microRNAs%3A%20Biogenesis%2C%20expression%20profiling%20and%20biological%20functions.%20Insect%20Biochem%20Mol%20Biol.%202013%3B%2043%3A%2024%E2%80%9338.%20doi%3A%2010.1016%2Fj.ibmb.2012.10.009%2023165178" data-title="Insect%20microRNAs%3A%20Biogenesis%2C%20expression%20profiling%20and%20biological%20functions" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23165178" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Insect+microRNAs%3A+Biogenesis%2C+expression+profiling+and+biological+functions+Lucas+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0171120.ref005" id="pone.0171120.ref005" class="link-target"></a>Miska EA. How microRNA control cell division, differentiation and death. Curr Opin Genet Dev. 2005; 15: 563–568.  pmid:16099643 <ul class="reflinks"><li><a href="#" data-author="Miska" data-cit="MiskaEA.%20How%20microRNA%20control%20cell%20division%2C%20differentiation%20and%20death.%20Curr%20Opin%20Genet%20Dev.%202005%3B%2015%3A%20563%E2%80%93568.%20doi%3A%2010.1016%2Fj.gde.2005.08.005%2016099643" data-title="How%20microRNA%20control%20cell%20division%2C%20differentiation%20and%20death" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16099643" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=How+microRNA+control+cell+division%2C+differentiation+and+death+Miska+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0171120.ref006" id="pone.0171120.ref006" class="link-target"></a>Hussain M, Asgari S. Functional analysis of a cellular microRNA in insect host-ascovirus interaction. J Virol. 2010; 84: 612–620.  pmid:19846520 <ul class="reflinks"><li><a href="#" data-author="Hussain" data-cit="HussainM%2C%20AsgariS.%20Functional%20analysis%20of%20a%20cellular%20microRNA%20in%20insect%20host-ascovirus%20interaction.%20J%20Virol.%202010%3B%2084%3A%20612%E2%80%93620.%20doi%3A%2010.1128%2FJVI.01794-09%2019846520" data-title="Functional%20analysis%20of%20a%20cellular%20microRNA%20in%20insect%20host-ascovirus%20interaction" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19846520" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Functional+analysis+of+a+cellular+microRNA+in+insect+host-ascovirus+interaction+Hussain+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0171120.ref007" id="pone.0171120.ref007" class="link-target"></a>Chawla G, Sokol NS. MicroRNAs in <em>Drosophila</em> development. Int Rev Cell Mol Biol. 2011; 286: 1–65.  pmid:21199779 <ul class="reflinks"><li><a href="#" data-author="Chawla" data-cit="ChawlaG%2C%20SokolNS.%20MicroRNAs%20in%20Drosophila%20development.%20Int%20Rev%20Cell%20Mol%20Biol.%202011%3B%20286%3A%201%E2%80%9365.%20doi%3A%2010.1016%2FB978-0-12-385859-7.00001-X%2021199779" data-title="MicroRNAs%20in%20Drosophila%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21199779" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+in+Drosophila+development+Chawla+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0171120.ref008" id="pone.0171120.ref008" class="link-target"></a>Zdanowicz A, Thermann R, Kowalska J, Jemielity J, Duncan K, Preiss T, et al. <em>Drosophila</em> miR2 primarily targets the m7GpppN cap structure for translational repression. Mol Cell. 2009; 35: 881–888.  pmid:19782035 <ul class="reflinks"><li><a href="#" data-author="Zdanowicz" data-cit="ZdanowiczA%2C%20ThermannR%2C%20KowalskaJ%2C%20JemielityJ%2C%20DuncanK%2C%20PreissT%2C%20et%20al.%20Drosophila%20miR2%20primarily%20targets%20the%20m7GpppN%20cap%20structure%20for%20translational%20repression.%20Mol%20Cell.%202009%3B%2035%3A%20881%E2%80%93888.%20doi%3A%2010.1016%2Fj.molcel.2009.09.009%2019782035" data-title="Drosophila%20miR2%20primarily%20targets%20the%20m7GpppN%20cap%20structure%20for%20translational%20repression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19782035" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Drosophila+miR2+primarily+targets+the+m7GpppN+cap+structure+for+translational+repression+Zdanowicz+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0171120.ref009" id="pone.0171120.ref009" class="link-target"></a>Hussain M, Torres S, Schnettler E, Funk A, Grundhoff A, Pijlman GP, et al. West Nile virus encodes a microRNA-like small RNA in the 3’ untranslated region which unregulates GATA4 mRNA and facilitates virus replication in mosquito cells. Nucleic Acids Res. 2012; 40: 2210–2223.  pmid:22080551 <ul class="reflinks"><li><a href="#" data-author="Hussain" data-cit="HussainM%2C%20TorresS%2C%20SchnettlerE%2C%20FunkA%2C%20GrundhoffA%2C%20PijlmanGP%2C%20et%20al.%20West%20Nile%20virus%20encodes%20a%20microRNA-like%20small%20RNA%20in%20the%203%E2%80%99%20untranslated%20region%20which%20unregulates%20GATA4%20mRNA%20and%20facilitates%20virus%20replication%20in%20mosquito%20cells.%20Nucleic%20Acids%20Res.%202012%3B%2040%3A%202210%E2%80%932223.%20doi%3A%2010.1093%2Fnar%2Fgkr848%2022080551" data-title="West%20Nile%20virus%20encodes%20a%20microRNA-like%20small%20RNA%20in%20the%203%E2%80%99%20untranslated%20region%20which%20unregulates%20GATA4%20mRNA%20and%20facilitates%20virus%20replication%20in%20mosquito%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22080551" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=West+Nile+virus+encodes+a+microRNA-like+small+RNA+in+the+3%E2%80%99+untranslated+region+which+unregulates+GATA4+mRNA+and+facilitates+virus+replication+in+mosquito+cells+Hussain+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0171120.ref010" id="pone.0171120.ref010" class="link-target"></a>He J, Chen Q, Wei Y, Jiang F, Yang M, Hao S, et al. MicroRNA-276 promotes egg-hatching synchrony by up-regulating <em>brim</em> in locusts. Proc Natl Acad Sci U S A. 2016; 113: 584–589.  pmid:26729868 <ul class="reflinks"><li><a href="#" data-author="He" data-cit="HeJ%2C%20ChenQ%2C%20WeiY%2C%20JiangF%2C%20YangM%2C%20HaoS%2C%20et%20al.%20MicroRNA-276%20promotes%20egg-hatching%20synchrony%20by%20up-regulating%20brim%20in%20locusts.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202016%3B%20113%3A%20584%E2%80%93589.%20doi%3A%2010.1073%2Fpnas.1521098113%2026729868" data-title="MicroRNA-276%20promotes%20egg-hatching%20synchrony%20by%20up-regulating%20brim%20in%20locusts" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26729868" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-276+promotes+egg-hatching+synchrony+by+up-regulating+brim+in+locusts+He+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0171120.ref011" id="pone.0171120.ref011" class="link-target"></a>Lee RC, Feinbaum RL, Ambros V. The <em>C</em>. <em>elegans</em> heterochronic gene <em>lin-4</em> encodes small RNAs with antisense complimentarity to <em>lin-14</em>. Cell. 1993; 75: 843–854. pmid:8252621 <ul class="reflinks"><li><a href="#" data-author="Lee" data-cit="LeeRC%2C%20FeinbaumRL%2C%20AmbrosV.%20The%20C.%20elegans%20heterochronic%20gene%20lin-4%20encodes%20small%20RNAs%20with%20antisense%20complimentarity%20to%20lin-14.%20Cell.%201993%3B%2075%3A%20843%E2%80%93854.%208252621" data-title="The%20C.%20elegans%20heterochronic%20gene%20lin-4%20encodes%20small%20RNAs%20with%20antisense%20complimentarity%20to%20lin-14" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8252621" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+C.+elegans+heterochronic+gene+lin-4+encodes+small+RNAs+with+antisense+complimentarity+to+lin-14+Lee+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0171120.ref012" id="pone.0171120.ref012" class="link-target"></a>Kozomara A, Griffiths-Jones S. miRBase: integrating microRNA annotation and deep-sequencing data. Nucleic Acids Res. 2011; 39: D152–D157.  pmid:21037258 <ul class="reflinks"><li><a href="#" data-author="Kozomara" data-cit="KozomaraA%2C%20Griffiths-JonesS.%20miRBase%3A%20integrating%20microRNA%20annotation%20and%20deep-sequencing%20data.%20Nucleic%20Acids%20Res.%202011%3B%2039%3A%20D152%E2%80%93D157.%20doi%3A%2010.1093%2Fnar%2Fgkq1027%2021037258" data-title="miRBase%3A%20integrating%20microRNA%20annotation%20and%20deep-sequencing%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21037258" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=miRBase%3A+integrating+microRNA+annotation+and+deep-sequencing+data+Kozomara+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0171120.ref013" id="pone.0171120.ref013" class="link-target"></a>Li Y, Maines JZ, Tastan OY, McKearin DM, Buszczak M. Mei-P26 regulates the maintenance of ovarian germline stem cells by promoting BMP signaling. Development. 2012; 139: 1547–1556.  pmid:22438571 <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiY%2C%20MainesJZ%2C%20TastanOY%2C%20McKearinDM%2C%20BuszczakM.%20Mei-P26%20regulates%20the%20maintenance%20of%20ovarian%20germline%20stem%20cells%20by%20promoting%20BMP%20signaling.%20Development.%202012%3B%20139%3A%201547%E2%80%931556.%20doi%3A%2010.1242%2Fdev.077412%2022438571" data-title="Mei-P26%20regulates%20the%20maintenance%20of%20ovarian%20germline%20stem%20cells%20by%20promoting%20BMP%20signaling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22438571" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mei-P26+regulates+the+maintenance+of+ovarian+germline+stem+cells+by+promoting+BMP+signaling+Li+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0171120.ref014" id="pone.0171120.ref014" class="link-target"></a>Becam I, Rafel N, Hong X, Cohen SM, Milán M. Notch-mediated repression of bantam miRNA contributes to boundary formation in the <em>Drosophila</em> wing. Development. 2011; 138: 3781–3789.  pmid:21795284 <ul class="reflinks"><li><a href="#" data-author="Becam" data-cit="BecamI%2C%20RafelN%2C%20HongX%2C%20CohenSM%2C%20Mil%C3%A1nM.%20Notch-mediated%20repression%20of%20bantam%20miRNA%20contributes%20to%20boundary%20formation%20in%20the%20Drosophila%20wing.%20Development.%202011%3B%20138%3A%203781%E2%80%933789.%20doi%3A%2010.1242%2Fdev.064774%2021795284" data-title="Notch-mediated%20repression%20of%20bantam%20miRNA%20contributes%20to%20boundary%20formation%20in%20the%20Drosophila%20wing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21795284" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Notch-mediated+repression+of+bantam+miRNA+contributes+to+boundary+formation+in+the+Drosophila+wing+Becam+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0171120.ref015" id="pone.0171120.ref015" class="link-target"></a>Surridge AK, Lopez-Gomollon S, Moxon S, Maroja LS, Rathjen T, Nadeau NJ, et al. Characterisation and expression of microRNAs in developing wings of the neotropical butterfly <em>Heliconius melpomene</em>. BMC Genomics. 2011; 12: 62.  pmid:21266089 <ul class="reflinks"><li><a href="#" data-author="Surridge" data-cit="SurridgeAK%2C%20Lopez-GomollonS%2C%20MoxonS%2C%20MarojaLS%2C%20RathjenT%2C%20NadeauNJ%2C%20et%20al.%20Characterisation%20and%20expression%20of%20microRNAs%20in%20developing%20wings%20of%20the%20neotropical%20butterfly%20Heliconius%20melpomene.%20BMC%20Genomics.%202011%3B%2012%3A%2062.%20doi%3A%2010.1186%2F1471-2164-12-62%2021266089" data-title="Characterisation%20and%20expression%20of%20microRNAs%20in%20developing%20wings%20of%20the%20neotropical%20butterfly%20Heliconius%20melpomene" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21266089" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Characterisation+and+expression+of+microRNAs+in+developing+wings+of+the+neotropical+butterfly+Heliconius+melpomene+Surridge+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0171120.ref016" id="pone.0171120.ref016" class="link-target"></a>Sun K, Westholm JO, Tsurudome K, Hagen JW, Lu Y, Kohwi M, et al. Neurophysiological defects and neuronal gene deregulation in <em>Drosophila mir-124</em> mutants. PLoS Genet. 2012; 8: e1002515.  pmid:22347817 <ul class="reflinks"><li><a href="#" data-author="Sun" data-cit="SunK%2C%20WestholmJO%2C%20TsurudomeK%2C%20HagenJW%2C%20LuY%2C%20KohwiM%2C%20et%20al.%20Neurophysiological%20defects%20and%20neuronal%20gene%20deregulation%20in%20Drosophila%20mir-124%20mutants.%20PLoS%20Genet.%202012%3B%208%3A%20e1002515.%20doi%3A%2010.1371%2Fjournal.pgen.1002515%2022347817" data-title="Neurophysiological%20defects%20and%20neuronal%20gene%20deregulation%20in%20Drosophila%20mir-124%20mutants" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22347817" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Neurophysiological+defects+and+neuronal+gene+deregulation+in+Drosophila+mir-124+mutants+Sun+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0171120.ref017" id="pone.0171120.ref017" class="link-target"></a>Xu P, Gue M, Hay BA. MicroRNA and the regulation of cell death. Trends Genet. 2004; 20: 617–624.  pmid:15522457 <ul class="reflinks"><li><a href="#" data-author="Xu" data-cit="XuP%2C%20GueM%2C%20HayBA.%20MicroRNA%20and%20the%20regulation%20of%20cell%20death.%20Trends%20Genet.%202004%3B%2020%3A%20617%E2%80%93624.%20doi%3A%2010.1016%2Fj.tig.2004.09.010%2015522457" data-title="MicroRNA%20and%20the%20regulation%20of%20cell%20death" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15522457" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+and+the+regulation+of+cell+death+Xu+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0171120.ref018" id="pone.0171120.ref018" class="link-target"></a>Legeai F, Rizk G, Walsh T, Edwards O, Gordon K, Lavenier D, et al. Bioinformatic prediction, deep sequencing of microRNAs and expression analysis during phenotypic plasticity in the pea aphid, <em>Acrythosiphon pisum</em>. BMC Genomics. 2010; 11: 281.  pmid:20444247 <ul class="reflinks"><li><a href="#" data-author="Legeai" data-cit="LegeaiF%2C%20RizkG%2C%20WalshT%2C%20EdwardsO%2C%20GordonK%2C%20LavenierD%2C%20et%20al.%20Bioinformatic%20prediction%2C%20deep%20sequencing%20of%20microRNAs%20and%20expression%20analysis%20during%20phenotypic%20plasticity%20in%20the%20pea%20aphid%2C%20Acrythosiphon%20pisum.%20BMC%20Genomics.%202010%3B%2011%3A%20281.%20doi%3A%2010.1186%2F1471-2164-11-281%2020444247" data-title="Bioinformatic%20prediction%2C%20deep%20sequencing%20of%20microRNAs%20and%20expression%20analysis%20during%20phenotypic%20plasticity%20in%20the%20pea%20aphid%2C%20Acrythosiphon%20pisum" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20444247" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bioinformatic+prediction%2C+deep+sequencing+of+microRNAs+and+expression+analysis+during+phenotypic+plasticity+in+the+pea+aphid%2C+Acrythosiphon+pisum+Legeai+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0171120.ref019" id="pone.0171120.ref019" class="link-target"></a>Liu F, Peng W, Li Z, Li W, Li L, Pan J, et al. Next-generation small RNA sequencing for microRNAs profiling in <em>Apis mellifera</em>: comparison between nurses and foragers. Insect Mol Biol. 2012; 21: 297–303.  pmid:22458842 <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuF%2C%20PengW%2C%20LiZ%2C%20LiW%2C%20LiL%2C%20PanJ%2C%20et%20al.%20Next-generation%20small%20RNA%20sequencing%20for%20microRNAs%20profiling%20in%20Apis%20mellifera%3A%20comparison%20between%20nurses%20and%20foragers.%20Insect%20Mol%20Biol.%202012%3B%2021%3A%20297%E2%80%93303.%20doi%3A%2010.1111%2Fj.1365-2583.2012.01135.x%2022458842" data-title="Next-generation%20small%20RNA%20sequencing%20for%20microRNAs%20profiling%20in%20Apis%20mellifera%3A%20comparison%20between%20nurses%20and%20foragers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22458842" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Next-generation+small+RNA+sequencing+for+microRNAs+profiling+in+Apis+mellifera%3A+comparison+between+nurses+and+foragers+Liu+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0171120.ref020" id="pone.0171120.ref020" class="link-target"></a>Zhang X, Zheng Y, Jagadeeswaran G, Ren R, Sunkar R, Jiang H. Identification and development profiling of conserved and novel microRNAs in <em>Manduca sexta</em>. Insect Biochem Mol Biol. 2012; 42: 381–395.  pmid:22406339 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangX%2C%20ZhengY%2C%20JagadeeswaranG%2C%20RenR%2C%20SunkarR%2C%20JiangH.%20Identification%20and%20development%20profiling%20of%20conserved%20and%20novel%20microRNAs%20in%20Manduca%20sexta.%20Insect%20Biochem%20Mol%20Biol.%202012%3B%2042%3A%20381%E2%80%93395.%20doi%3A%2010.1016%2Fj.ibmb.2012.01.006%2022406339" data-title="Identification%20and%20development%20profiling%20of%20conserved%20and%20novel%20microRNAs%20in%20Manduca%20sexta" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22406339" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+development+profiling+of+conserved+and+novel+microRNAs+in+Manduca+sexta+Zhang+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0171120.ref021" id="pone.0171120.ref021" class="link-target"></a>Cayirlioglu P, Kadow IG, Zhan X, Okamura K, Suh GS, Gunning D, et al. Hybrid neurons in a microRNA mutant are putative evolutionary intermediates in insect CO<sub>2</sub> sensory systems. Science. 2008; 319: 1256–1260.  pmid:18309086 <ul class="reflinks"><li><a href="#" data-author="Cayirlioglu" data-cit="CayirliogluP%2C%20KadowIG%2C%20ZhanX%2C%20OkamuraK%2C%20SuhGS%2C%20GunningD%2C%20et%20al.%20Hybrid%20neurons%20in%20a%20microRNA%20mutant%20are%20putative%20evolutionary%20intermediates%20in%20insect%20CO2%20sensory%20systems.%20Science.%202008%3B%20319%3A%201256%E2%80%931260.%20doi%3A%2010.1126%2Fscience.1149483%2018309086" data-title="Hybrid%20neurons%20in%20a%20microRNA%20mutant%20are%20putative%20evolutionary%20intermediates%20in%20insect%20CO2%20sensory%20systems" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18309086" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Hybrid+neurons+in+a+microRNA+mutant+are+putative+evolutionary+intermediates+in+insect+CO2+sensory+systems+Cayirlioglu+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0171120.ref022" id="pone.0171120.ref022" class="link-target"></a>Greenberg JK, Xia J, Zhou X, Thatcher SR, Gu X, Ament SA, et al. Behavioral plasticity in honey bees in associated with differences in brain microRNA transcriptome. Genes brain Behav. 2012; 11: 660–670.  pmid:22409512 <ul class="reflinks"><li><a href="#" data-author="Greenberg" data-cit="GreenbergJK%2C%20XiaJ%2C%20ZhouX%2C%20ThatcherSR%2C%20GuX%2C%20AmentSA%2C%20et%20al.%20Behavioral%20plasticity%20in%20honey%20bees%20in%20associated%20with%20differences%20in%20brain%20microRNA%20transcriptome.%20Genes%20brain%20Behav.%202012%3B%2011%3A%20660%E2%80%93670.%20doi%3A%2010.1111%2Fj.1601-183X.2012.00782.x%2022409512" data-title="Behavioral%20plasticity%20in%20honey%20bees%20in%20associated%20with%20differences%20in%20brain%20microRNA%20transcriptome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22409512" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Behavioral+plasticity+in+honey+bees+in+associated+with+differences+in+brain+microRNA+transcriptome+Greenberg+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0171120.ref023" id="pone.0171120.ref023" class="link-target"></a>Li X, Cassidy JJ, Reinke CA, Fischboeck S, Carthew RW. A microRNA imparts robustness against environmental fluctuation during development. Cell. 2009; 137: 273–283.  pmid:19379693 <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiX%2C%20CassidyJJ%2C%20ReinkeCA%2C%20FischboeckS%2C%20CarthewRW.%20A%20microRNA%20imparts%20robustness%20against%20environmental%20fluctuation%20during%20development.%20Cell.%202009%3B%20137%3A%20273%E2%80%93283.%20doi%3A%2010.1016%2Fj.cell.2009.01.058%2019379693" data-title="A%20microRNA%20imparts%20robustness%20against%20environmental%20fluctuation%20during%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19379693" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+microRNA+imparts+robustness+against+environmental+fluctuation+during+development+Li+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0171120.ref024" id="pone.0171120.ref024" class="link-target"></a>Umbach JL, Kramer MF, Jurak I, Karnowski HW, Coen DM, Cullen BR. MicroRNAs expressed by herpes simplex virus 1 during latent infection regulate viral mRNAs. Nature. 2008; 454: 780–783.  pmid:18596690 <ul class="reflinks"><li><a href="#" data-author="Umbach" data-cit="UmbachJL%2C%20KramerMF%2C%20JurakI%2C%20KarnowskiHW%2C%20CoenDM%2C%20CullenBR.%20MicroRNAs%20expressed%20by%20herpes%20simplex%20virus%201%20during%20latent%20infection%20regulate%20viral%20mRNAs.%20Nature.%202008%3B%20454%3A%20780%E2%80%93783.%20doi%3A%2010.1038%2Fnature07103%2018596690" data-title="MicroRNAs%20expressed%20by%20herpes%20simplex%20virus%201%20during%20latent%20infection%20regulate%20viral%20mRNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18596690" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+expressed+by+herpes+simplex+virus+1+during+latent+infection+regulate+viral+mRNAs+Umbach+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0171120.ref025" id="pone.0171120.ref025" class="link-target"></a>Hussain M, Frentiu FD, Moreira LA, O’Neill SL, Asgari S. Wolbachia utilizes host microRNAs to manipulate host gene expression and facilitate colonization of the dengue vector <em>Aedes aegypti</em>. Proc Natl Acad Sci U S A. 2011; 108:9250–9255.  pmid:21576469 <ul class="reflinks"><li><a href="#" data-author="Hussain" data-cit="HussainM%2C%20FrentiuFD%2C%20MoreiraLA%2C%20O%E2%80%99NeillSL%2C%20AsgariS.%20Wolbachia%20utilizes%20host%20microRNAs%20to%20manipulate%20host%20gene%20expression%20and%20facilitate%20colonization%20of%20the%20dengue%20vector%20Aedes%20aegypti.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202011%3B%20108%3A9250%E2%80%939255.%20doi%3A%2010.1073%2Fpnas.1105469108%2021576469" data-title="Wolbachia%20utilizes%20host%20microRNAs%20to%20manipulate%20host%20gene%20expression%20and%20facilitate%20colonization%20of%20the%20dengue%20vector%20Aedes%20aegypti" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21576469" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Wolbachia+utilizes+host+microRNAs+to+manipulate+host+gene+expression+and+facilitate+colonization+of+the+dengue+vector+Aedes+aegypti+Hussain+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0171120.ref026" id="pone.0171120.ref026" class="link-target"></a>Aboobaker AA, Tomancak P, Patel N, Rubin GM, Lai EC. <em>Drosophila</em> microRNAs exhibit diverse spatial expression patterns during embryonic development. Proc Natl Acad Sci U S A. 2005; 102: 18017–18022.  pmid:16330759 <ul class="reflinks"><li><a href="#" data-author="Aboobaker" data-cit="AboobakerAA%2C%20TomancakP%2C%20PatelN%2C%20RubinGM%2C%20LaiEC.%20Drosophila%20microRNAs%20exhibit%20diverse%20spatial%20expression%20patterns%20during%20embryonic%20development.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202005%3B%20102%3A%2018017%E2%80%9318022.%20doi%3A%2010.1073%2Fpnas.0508823102%2016330759" data-title="Drosophila%20microRNAs%20exhibit%20diverse%20spatial%20expression%20patterns%20during%20embryonic%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16330759" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Drosophila+microRNAs+exhibit+diverse+spatial+expression+patterns+during+embryonic+development+Aboobaker+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0171120.ref027" id="pone.0171120.ref027" class="link-target"></a>Liu S, Gao S, Zhang D, Yin J, Xiang Z, Xia Q. MicroRNAs show diverse and dynamic expression patterns in multiple tissues of <em>Bombyx mori</em>. BMC Genomics. 2010; 11: 85.  pmid:20122259 <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuS%2C%20GaoS%2C%20ZhangD%2C%20YinJ%2C%20XiangZ%2C%20XiaQ.%20MicroRNAs%20show%20diverse%20and%20dynamic%20expression%20patterns%20in%20multiple%20tissues%20of%20Bombyx%20mori.%20BMC%20Genomics.%202010%3B%2011%3A%2085.%20doi%3A%2010.1186%2F1471-2164-11-85%2020122259" data-title="MicroRNAs%20show%20diverse%20and%20dynamic%20expression%20patterns%20in%20multiple%20tissues%20of%20Bombyx%20mori" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20122259" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+show+diverse+and+dynamic+expression+patterns+in+multiple+tissues+of+Bombyx+mori+Liu+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0171120.ref028" id="pone.0171120.ref028" class="link-target"></a>Sempere LF, Sokol NS, Dubrovsky EB, Berger EM, Ambros V. Temporal regulation of microRNA expression in <em>Drosophila melanogaster</em> mediated by hormonal signals and broad-Complex gene activity. Dev Biol. 2003; 259: 9–18. pmid:12812784 <ul class="reflinks"><li><a href="#" data-author="Sempere" data-cit="SempereLF%2C%20SokolNS%2C%20DubrovskyEB%2C%20BergerEM%2C%20AmbrosV.%20Temporal%20regulation%20of%20microRNA%20expression%20in%20Drosophila%20melanogaster%20mediated%20by%20hormonal%20signals%20and%20broad-Complex%20gene%20activity.%20Dev%20Biol.%202003%3B%20259%3A%209%E2%80%9318.%2012812784" data-title="Temporal%20regulation%20of%20microRNA%20expression%20in%20Drosophila%20melanogaster%20mediated%20by%20hormonal%20signals%20and%20broad-Complex%20gene%20activity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12812784" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Temporal+regulation+of+microRNA+expression+in+Drosophila+melanogaster+mediated+by+hormonal+signals+and+broad-Complex+gene+activity+Sempere+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0171120.ref029" id="pone.0171120.ref029" class="link-target"></a>Yu X, Zhou Q, Li S, Luo Q, Cai Y, Lin W, et al. The silkworm (<em>Bombyx mori</em>) microRNAs and their expressions in multiple developmental stages. PLoS ONE. 2008; 3: e2997.  pmid:18714353 <ul class="reflinks"><li><a href="#" data-author="Yu" data-cit="YuX%2C%20ZhouQ%2C%20LiS%2C%20LuoQ%2C%20CaiY%2C%20LinW%2C%20et%20al.%20The%20silkworm%20%28Bombyx%20mori%29%20microRNAs%20and%20their%20expressions%20in%20multiple%20developmental%20stages.%20PLoS%20ONE.%202008%3B%203%3A%20e2997.%20doi%3A%2010.1371%2Fjournal.pone.0002997%2018714353" data-title="The%20silkworm%20%28Bombyx%20mori%29%20microRNAs%20and%20their%20expressions%20in%20multiple%20developmental%20stages" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18714353" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+silkworm+%28Bombyx+mori%29+microRNAs+and+their+expressions+in+multiple+developmental+stages+Yu+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0171120.ref030" id="pone.0171120.ref030" class="link-target"></a>Nelson PT, Baldwin DA, Scearce LM, Oberhotzer JC, Tobias JW, Mourelatos Z. Microarray-based, high-throughput gene expression profiling of microRNAs. Nat Methods. 2004; 1: 155–161.  pmid:15782179 <ul class="reflinks"><li><a href="#" data-author="Nelson" data-cit="NelsonPT%2C%20BaldwinDA%2C%20ScearceLM%2C%20OberhotzerJC%2C%20TobiasJW%2C%20MourelatosZ.%20Microarray-based%2C%20high-throughput%20gene%20expression%20profiling%20of%20microRNAs.%20Nat%20Methods.%202004%3B%201%3A%20155%E2%80%93161.%20doi%3A%2010.1038%2Fnmeth717%2015782179" data-title="Microarray-based%2C%20high-throughput%20gene%20expression%20profiling%20of%20microRNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15782179" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Microarray-based%2C+high-throughput+gene+expression+profiling+of+microRNAs+Nelson+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0171120.ref031" id="pone.0171120.ref031" class="link-target"></a>Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, Peck D, et al. MicroRNA expression profiles classify human cancers. Nature. 2005; 435: 834–838.  pmid:15944708 <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuJ%2C%20GetzG%2C%20MiskaEA%2C%20Alvarez-SaavedraE%2C%20LambJ%2C%20PeckD%2C%20et%20al.%20MicroRNA%20expression%20profiles%20classify%20human%20cancers.%20Nature.%202005%3B%20435%3A%20834%E2%80%93838.%20doi%3A%2010.1038%2Fnature03702%2015944708" data-title="MicroRNA%20expression%20profiles%20classify%20human%20cancers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15944708" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+expression+profiles+classify+human+cancers+Lu+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0171120.ref032" id="pone.0171120.ref032" class="link-target"></a>Chen C, Ridzon DA, Broomer AJ, Zhou Z, Lee DH, Nguyen JT, et al. Real-time quantification of microRNAs by stem-loop RT-PCR. Nucleic Acids Res. 2005; 33: e179.  pmid:16314309 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenC%2C%20RidzonDA%2C%20BroomerAJ%2C%20ZhouZ%2C%20LeeDH%2C%20NguyenJT%2C%20et%20al.%20Real-time%20quantification%20of%20microRNAs%20by%20stem-loop%20RT-PCR.%20Nucleic%20Acids%20Res.%202005%3B%2033%3A%20e179.%20doi%3A%2010.1093%2Fnar%2Fgni178%2016314309" data-title="Real-time%20quantification%20of%20microRNAs%20by%20stem-loop%20RT-PCR" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16314309" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Real-time+quantification+of+microRNAs+by+stem-loop+RT-PCR+Chen+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0171120.ref033" id="pone.0171120.ref033" class="link-target"></a>Wan G, Lim QE, Too HP. High-performance quantification of mature microRNAs by real-time RT-PCR using deoxyuridine-incorporated oligonucleotides and hemi-nested primers. RNA. 2010; 16: 1436–1445.  pmid:20547774 <ul class="reflinks"><li><a href="#" data-author="Wan" data-cit="WanG%2C%20LimQE%2C%20TooHP.%20High-performance%20quantification%20of%20mature%20microRNAs%20by%20real-time%20RT-PCR%20using%20deoxyuridine-incorporated%20oligonucleotides%20and%20hemi-nested%20primers.%20RNA.%202010%3B%2016%3A%201436%E2%80%931445.%20doi%3A%2010.1261%2Frna.2001610%2020547774" data-title="High-performance%20quantification%20of%20mature%20microRNAs%20by%20real-time%20RT-PCR%20using%20deoxyuridine-incorporated%20oligonucleotides%20and%20hemi-nested%20primers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20547774" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=High-performance+quantification+of+mature+microRNAs+by+real-time+RT-PCR+using+deoxyuridine-incorporated+oligonucleotides+and+hemi-nested+primers+Wan+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0171120.ref034" id="pone.0171120.ref034" class="link-target"></a>Mestdagh P, Van Vlierberghe P, De Weer A, Muth D, Westermann F, Speleman F, et al. A novel and universal method for microRNA RT-qPCR data normalization. Genome Biol. 2009; 10: R64.  pmid:19531210 <ul class="reflinks"><li><a href="#" data-author="Mestdagh" data-cit="MestdaghP%2C%20Van%20VlierbergheP%2C%20De%20WeerA%2C%20MuthD%2C%20WestermannF%2C%20SpelemanF%2C%20et%20al.%20A%20novel%20and%20universal%20method%20for%20microRNA%20RT-qPCR%20data%20normalization.%20Genome%20Biol.%202009%3B%2010%3A%20R64.%20doi%3A%2010.1186%2Fgb-2009-10-6-r64%2019531210" data-title="A%20novel%20and%20universal%20method%20for%20microRNA%20RT-qPCR%20data%20normalization" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19531210" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+novel+and+universal+method+for+microRNA+RT-qPCR+data+normalization+Mestdagh+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0171120.ref035" id="pone.0171120.ref035" class="link-target"></a>Yang Q, Li Z, Cao J, Zhang S, Zhang H, Wu X, et al. Selection and assessment of reference genes for quantitative PCR normalization in migratoty locust <em>Locusta migratoria</em> (Orthoptera: Acrididae). PLoS ONE. 2014; 9: e98164.  pmid:24887329 <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangQ%2C%20LiZ%2C%20CaoJ%2C%20ZhangS%2C%20ZhangH%2C%20WuX%2C%20et%20al.%20Selection%20and%20assessment%20of%20reference%20genes%20for%20quantitative%20PCR%20normalization%20in%20migratoty%20locust%20Locusta%20migratoria%20%28Orthoptera%3A%20Acrididae%29.%20PLoS%20ONE.%202014%3B%209%3A%20e98164.%20doi%3A%2010.1371%2Fjournal.pone.0098164%2024887329" data-title="Selection%20and%20assessment%20of%20reference%20genes%20for%20quantitative%20PCR%20normalization%20in%20migratoty%20locust%20Locusta%20migratoria%20%28Orthoptera%3A%20Acrididae%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24887329" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Selection+and+assessment+of+reference+genes+for+quantitative+PCR+normalization+in+migratoty+locust+Locusta+migratoria+%28Orthoptera%3A+Acrididae%29+Yang+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0171120.ref036" id="pone.0171120.ref036" class="link-target"></a>Zhang S, An S, Li Z, Wu F, Yang Q, Liu Y, et al. Identification and validation of reference genes for normalization of gene expression analysis using qRT-PCR in <em>Helicoverpa armigera</em>. Gene. 2015; 555: 393–402.  pmid:25447918 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangS%2C%20AnS%2C%20LiZ%2C%20WuF%2C%20YangQ%2C%20LiuY%2C%20et%20al.%20Identification%20and%20validation%20of%20reference%20genes%20for%20normalization%20of%20gene%20expression%20analysis%20using%20qRT-PCR%20in%20Helicoverpa%20armigera.%20Gene.%202015%3B%20555%3A%20393%E2%80%93402.%20doi%3A%2010.1016%2Fj.gene.2014.11.038%2025447918" data-title="Identification%20and%20validation%20of%20reference%20genes%20for%20normalization%20of%20gene%20expression%20analysis%20using%20qRT-PCR%20in%20Helicoverpa%20armigera" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25447918" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+validation+of+reference+genes+for+normalization+of+gene+expression+analysis+using+qRT-PCR+in+Helicoverpa+armigera+Zhang+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0171120.ref037" id="pone.0171120.ref037" class="link-target"></a>Kirk H, Dorn S, Mazzi D. Worldwide population genetic structure of the oriental fruit moth (<em>Grapholita molesta</em>), a globally invasive pest. BMC Ecol. 2013; 13: 12.  pmid:23531126 <ul class="reflinks"><li><a href="#" data-author="Kirk" data-cit="KirkH%2C%20DornS%2C%20MazziD.%20Worldwide%20population%20genetic%20structure%20of%20the%20oriental%20fruit%20moth%20%28Grapholita%20molesta%29%2C%20a%20globally%20invasive%20pest.%20BMC%20Ecol.%202013%3B%2013%3A%2012.%20doi%3A%2010.1186%2F1472-6785-13-12%2023531126" data-title="Worldwide%20population%20genetic%20structure%20of%20the%20oriental%20fruit%20moth%20%28Grapholita%20molesta%29%2C%20a%20globally%20invasive%20pest" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23531126" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Worldwide+population+genetic+structure+of+the+oriental+fruit+moth+%28Grapholita+molesta%29%2C+a+globally+invasive+pest+Kirk+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0171120.ref038" id="pone.0171120.ref038" class="link-target"></a>Rothschild GHL, Vickers RA. Biology, ecology and control of the oriental fruit moth. In: vander Geest LPS, Evenhuis HH, editors. World crops pest, Tortricid pests their biology, natural enemies and control. Amsterdan: Elsevier Academic Publishers; 1991. pp. 389–412. <ul class="find-nolinks"></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0171120.ref039" id="pone.0171120.ref039" class="link-target"></a>Li G, Zhang Y, Li Y, Wu J, Xu X. Cloning, expression, and functional analysis of three odorant-binding proteins of the oriental fruit moth, <em>Grapholita molesta</em> (Busck) (Lepidoptera: Tortricidae). Arch Insect Biochem Physiol. 2016; 91: 67–87.  pmid:26609640 <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiG%2C%20ZhangY%2C%20LiY%2C%20WuJ%2C%20XuX.%20Cloning%2C%20expression%2C%20and%20functional%20analysis%20of%20three%20odorant-binding%20proteins%20of%20the%20oriental%20fruit%20moth%2C%20Grapholita%20molesta%20%28Busck%29%20%28Lepidoptera%3A%20Tortricidae%29.%20Arch%20Insect%20Biochem%20Physiol.%202016%3B%2091%3A%2067%E2%80%9387.%20doi%3A%2010.1002%2Farch.21309%2026609640" data-title="Cloning%2C%20expression%2C%20and%20functional%20analysis%20of%20three%20odorant-binding%20proteins%20of%20the%20oriental%20fruit%20moth%2C%20Grapholita%20molesta%20%28Busck%29%20%28Lepidoptera%3A%20Tortricidae%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26609640" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cloning%2C+expression%2C+and+functional+analysis+of+three+odorant-binding+proteins+of+the+oriental+fruit+moth%2C+Grapholita+molesta+%28Busck%29+%28Lepidoptera%3A+Tortricidae%29+Li+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0171120.ref040" id="pone.0171120.ref040" class="link-target"></a>Mazzi D, Dorn S. Movement of insect pests in agricultural landscapes. Ann Appl Biol. 2012; 160: 97–113. <ul class="reflinks"><li><a href="#" data-author="Mazzi" data-cit="MazziD%2C%20DornS.%20Movement%20of%20insect%20pests%20in%20agricultural%20landscapes.%20Ann%20Appl%20Biol.%202012%3B%20160%3A%2097%E2%80%93113." data-title="Movement%20of%20insect%20pests%20in%20agricultural%20landscapes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Movement+of+insect+pests+in+agricultural+landscapes+Mazzi+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0171120.ref041" id="pone.0171120.ref041" class="link-target"></a>Myers CT, Hull LA, Krawczyk G. Effects of orchard host plants (apple and peach) on development of oriental fruit moth (Lepidoptera: Tortricidae). J Econ Entomol. 2007; 100: 421–430. pmid:17461067 <ul class="reflinks"><li><a href="#" data-author="Myers" data-cit="MyersCT%2C%20HullLA%2C%20KrawczykG.%20Effects%20of%20orchard%20host%20plants%20%28apple%20and%20peach%29%20on%20development%20of%20oriental%20fruit%20moth%20%28Lepidoptera%3A%20Tortricidae%29.%20J%20Econ%20Entomol.%202007%3B%20100%3A%20421%E2%80%93430.%2017461067" data-title="Effects%20of%20orchard%20host%20plants%20%28apple%20and%20peach%29%20on%20development%20of%20oriental%20fruit%20moth%20%28Lepidoptera%3A%20Tortricidae%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17461067" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Effects+of+orchard+host+plants+%28apple+and+peach%29+on+development+of+oriental+fruit+moth+%28Lepidoptera%3A+Tortricidae%29+Myers+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0171120.ref042" id="pone.0171120.ref042" class="link-target"></a>Kanga LHB, Pree DJ, van Lier JL, Walker GM. Management of insecticide resistance in oriental fruit moth (<em>Grapholita molesta</em>; Lepidoptera: Tortricidae) populations from Ontario. Pest Manag Sci. 2003; 59: 921–927.  pmid:12916773 <ul class="reflinks"><li><a href="#" data-author="Kanga" data-cit="KangaLHB%2C%20PreeDJ%2C%20van%20LierJL%2C%20WalkerGM.%20Management%20of%20insecticide%20resistance%20in%20oriental%20fruit%20moth%20%28Grapholita%20molesta%3B%20Lepidoptera%3A%20Tortricidae%29%20populations%20from%20Ontario.%20Pest%20Manag%20Sci.%202003%3B%2059%3A%20921%E2%80%93927.%20doi%3A%2010.1002%2Fps.702%2012916773" data-title="Management%20of%20insecticide%20resistance%20in%20oriental%20fruit%20moth%20%28Grapholita%20molesta%3B%20Lepidoptera%3A%20Tortricidae%29%20populations%20from%20Ontario" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12916773" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Management+of+insecticide+resistance+in+oriental+fruit+moth+%28Grapholita+molesta%3B+Lepidoptera%3A+Tortricidae%29+populations+from+Ontario+Kanga+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0171120.ref043" id="pone.0171120.ref043" class="link-target"></a>Tollerup KE, Rucker A, Shearer PW. Whole-farm mating disruption to manage <em>Grapholita molesta</em> (Lepidoptera: Tortricidae) in diversified New Jersey Orchards. J Econ Entomol. 2012; 105: 1712–1718. pmid:23156168 <ul class="reflinks"><li><a href="#" data-author="Tollerup" data-cit="TollerupKE%2C%20RuckerA%2C%20ShearerPW.%20Whole-farm%20mating%20disruption%20to%20manage%20Grapholita%20molesta%20%28Lepidoptera%3A%20Tortricidae%29%20in%20diversified%20New%20Jersey%20Orchards.%20J%20Econ%20Entomol.%202012%3B%20105%3A%201712%E2%80%931718.%2023156168" data-title="Whole-farm%20mating%20disruption%20to%20manage%20Grapholita%20molesta%20%28Lepidoptera%3A%20Tortricidae%29%20in%20diversified%20New%20Jersey%20Orchards" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23156168" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Whole-farm+mating+disruption+to+manage+Grapholita+molesta+%28Lepidoptera%3A+Tortricidae%29+in+diversified+New+Jersey+Orchards+Tollerup+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0171120.ref044" id="pone.0171120.ref044" class="link-target"></a>Cao J, Liu Y, Yang Y, Zhang H, Li Z, Yang Q, et al. Molecular characterization and functional analysis of the ultraspiracle (USP) in the oriental fruit moth <em>Grapholita molesta</em> (Lepidoptera: Olethreutidae). Comp Biochem physiol B Biochem Mol Biol. 2015; 190: 54–62.  pmid:26325588 <ul class="reflinks"><li><a href="#" data-author="Cao" data-cit="CaoJ%2C%20LiuY%2C%20YangY%2C%20ZhangH%2C%20LiZ%2C%20YangQ%2C%20et%20al.%20Molecular%20characterization%20and%20functional%20analysis%20of%20the%20ultraspiracle%20%28USP%29%20in%20the%20oriental%20fruit%20moth%20Grapholita%20molesta%20%28Lepidoptera%3A%20Olethreutidae%29.%20Comp%20Biochem%20physiol%20B%20Biochem%20Mol%20Biol.%202015%3B%20190%3A%2054%E2%80%9362.%20doi%3A%2010.1016%2Fj.cbpb.2015.08.008%2026325588" data-title="Molecular%20characterization%20and%20functional%20analysis%20of%20the%20ultraspiracle%20%28USP%29%20in%20the%20oriental%20fruit%20moth%20Grapholita%20molesta%20%28Lepidoptera%3A%20Olethreutidae%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26325588" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+characterization+and+functional+analysis+of+the+ultraspiracle+%28USP%29+in+the+oriental+fruit+moth+Grapholita+molesta+%28Lepidoptera%3A+Olethreutidae%29+Cao+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0171120.ref045" id="pone.0171120.ref045" class="link-target"></a>Torres A, Torres K, Wdowiak P, Paszkowski T, Maciejewski R. Selection and validation of endogenous controls for microRNA expression studies in endometrioid endometrial cancer tissues. Gynecol Oncol. 2013; 130: 588–594.  pmid:23811003 <ul class="reflinks"><li><a href="#" data-author="Torres" data-cit="TorresA%2C%20TorresK%2C%20WdowiakP%2C%20PaszkowskiT%2C%20MaciejewskiR.%20Selection%20and%20validation%20of%20endogenous%20controls%20for%20microRNA%20expression%20studies%20in%20endometrioid%20endometrial%20cancer%20tissues.%20Gynecol%20Oncol.%202013%3B%20130%3A%20588%E2%80%93594.%20doi%3A%2010.1016%2Fj.ygyno.2013.06.026%2023811003" data-title="Selection%20and%20validation%20of%20endogenous%20controls%20for%20microRNA%20expression%20studies%20in%20endometrioid%20endometrial%20cancer%20tissues" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23811003" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Selection+and+validation+of+endogenous+controls+for+microRNA+expression+studies+in+endometrioid+endometrial+cancer+tissues+Torres+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0171120.ref046" id="pone.0171120.ref046" class="link-target"></a>Mahdipour M, van Tol HTA, Stout TAE, Roelen BAJ. Validating reference microRNAs for normalizing qRT-PCR data in bovine oocytes and preimplantation embryos. BMC Dev Biol. 2015; 15: 25.  pmid:26062615 <ul class="reflinks"><li><a href="#" data-author="Mahdipour" data-cit="MahdipourM%2C%20van%20TolHTA%2C%20StoutTAE%2C%20RoelenBAJ.%20Validating%20reference%20microRNAs%20for%20normalizing%20qRT-PCR%20data%20in%20bovine%20oocytes%20and%20preimplantation%20embryos.%20BMC%20Dev%20Biol.%202015%3B%2015%3A%2025.%20doi%3A%2010.1186%2Fs12861-015-0075-8%2026062615" data-title="Validating%20reference%20microRNAs%20for%20normalizing%20qRT-PCR%20data%20in%20bovine%20oocytes%20and%20preimplantation%20embryos" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26062615" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Validating+reference+microRNAs+for+normalizing+qRT-PCR+data+in+bovine+oocytes+and+preimplantation+embryos+Mahdipour+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0171120.ref047" id="pone.0171120.ref047" class="link-target"></a>Zhu X, Li Y, Chen D, Wu P, Yi T, Chen T, et al. Selection of reference genes for microRNA quantitative expression analysis in Chinese perch, <em>Siniperca chuatsi</em>. Int J Mol Sci. 2015; 16: 8310–8323.  pmid:25874758 <ul class="reflinks"><li><a href="#" data-author="Zhu" data-cit="ZhuX%2C%20LiY%2C%20ChenD%2C%20WuP%2C%20YiT%2C%20ChenT%2C%20et%20al.%20Selection%20of%20reference%20genes%20for%20microRNA%20quantitative%20expression%20analysis%20in%20Chinese%20perch%2C%20Siniperca%20chuatsi.%20Int%20J%20Mol%20Sci.%202015%3B%2016%3A%208310%E2%80%938323.%20doi%3A%2010.3390%2Fijms16048310%2025874758" data-title="Selection%20of%20reference%20genes%20for%20microRNA%20quantitative%20expression%20analysis%20in%20Chinese%20perch%2C%20Siniperca%20chuatsi" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25874758" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Selection+of+reference+genes+for+microRNA+quantitative+expression+analysis+in+Chinese+perch%2C+Siniperca+chuatsi+Zhu+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0171120.ref048" id="pone.0171120.ref048" class="link-target"></a>Wheat CW, Wahlberg N. Phylogenomic insights into the cambrian explosion, the colonization of land and the evolution of flight in arthropoda. Syst Biol. 2013; 62: 93–109.  pmid:22949483 <ul class="reflinks"><li><a href="#" data-author="Wheat" data-cit="WheatCW%2C%20WahlbergN.%20Phylogenomic%20insights%20into%20the%20cambrian%20explosion%2C%20the%20colonization%20of%20land%20and%20the%20evolution%20of%20flight%20in%20arthropoda.%20Syst%20Biol.%202013%3B%2062%3A%2093%E2%80%93109.%20doi%3A%2010.1093%2Fsysbio%2Fsys074%2022949483" data-title="Phylogenomic%20insights%20into%20the%20cambrian%20explosion%2C%20the%20colonization%20of%20land%20and%20the%20evolution%20of%20flight%20in%20arthropoda" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22949483" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogenomic+insights+into+the+cambrian+explosion%2C+the+colonization+of+land+and+the+evolution+of+flight+in+arthropoda+Wheat+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0171120.ref049" id="pone.0171120.ref049" class="link-target"></a>Zhou G, Wang T, Lou Y, Cheng J, Zhang H, Xu J. Identification and characterization of microRNAs in small brown planthopper (<em>Laodephax striatellus</em>) by next-generation sequencing. PLoS ONE. 2014; 9: e103041.  pmid:25057821 <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouG%2C%20WangT%2C%20LouY%2C%20ChengJ%2C%20ZhangH%2C%20XuJ.%20Identification%20and%20characterization%20of%20microRNAs%20in%20small%20brown%20planthopper%20%28Laodephax%20striatellus%29%20by%20next-generation%20sequencing.%20PLoS%20ONE.%202014%3B%209%3A%20e103041.%20doi%3A%2010.1371%2Fjournal.pone.0103041%2025057821" data-title="Identification%20and%20characterization%20of%20microRNAs%20in%20small%20brown%20planthopper%20%28Laodephax%20striatellus%29%20by%20next-generation%20sequencing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25057821" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+characterization+of+microRNAs+in+small+brown+planthopper+%28Laodephax+striatellus%29+by+next-generation+sequencing+Zhou+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0171120.ref050" id="pone.0171120.ref050" class="link-target"></a>Kanehisa M, Araki M, Goto S, Hattori M, Hirakawa M, Itoh M, et al. KEGG for linking genomes to life and the environment. Nucl Acids Res. 2008; 36: D480–484.  pmid:18077471 <ul class="reflinks"><li><a href="#" data-author="Kanehisa" data-cit="KanehisaM%2C%20ArakiM%2C%20GotoS%2C%20HattoriM%2C%20HirakawaM%2C%20ItohM%2C%20et%20al.%20KEGG%20for%20linking%20genomes%20to%20life%20and%20the%20environment.%20Nucl%20Acids%20Res.%202008%3B%2036%3A%20D480%E2%80%93484.%20doi%3A%2010.1093%2Fnar%2Fgkm882%2018077471" data-title="KEGG%20for%20linking%20genomes%20to%20life%20and%20the%20environment" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18077471" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=KEGG+for+linking+genomes+to+life+and+the+environment+Kanehisa+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0171120.ref051" id="pone.0171120.ref051" class="link-target"></a>Gharbi S, Shamsara M, Khateri S, Soroush MR, Ghorbanmehr N, Tavallaei M, et al. Identification of reliable reference genes for quantification of microRNAs in serum samples of sulfur mustard-exposed veterans. Cell J. 2015; 17: 494–501. pmid:26464821 <ul class="reflinks"><li><a href="#" data-author="Gharbi" data-cit="GharbiS%2C%20ShamsaraM%2C%20KhateriS%2C%20SoroushMR%2C%20GhorbanmehrN%2C%20TavallaeiM%2C%20et%20al.%20Identification%20of%20reliable%20reference%20genes%20for%20quantification%20of%20microRNAs%20in%20serum%20samples%20of%20sulfur%20mustard-exposed%20veterans.%20Cell%20J.%202015%3B%2017%3A%20494%E2%80%93501.%2026464821" data-title="Identification%20of%20reliable%20reference%20genes%20for%20quantification%20of%20microRNAs%20in%20serum%20samples%20of%20sulfur%20mustard-exposed%20veterans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26464821" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+reliable+reference+genes+for+quantification+of+microRNAs+in+serum+samples+of+sulfur+mustard-exposed+veterans+Gharbi+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0171120.ref052" id="pone.0171120.ref052" class="link-target"></a>Han H, Jo YN, Lee JY, Choi SY, Jeong Y, Yun J, et al. Identification of suitable reference genes for the relative quantification of microRNAs in pleural effusion. Oncol Lett. 2014; 8: 1889–1895.  pmid:25202432 <ul class="reflinks"><li><a href="#" data-author="Han" data-cit="HanH%2C%20JoYN%2C%20LeeJY%2C%20ChoiSY%2C%20JeongY%2C%20YunJ%2C%20et%20al.%20Identification%20of%20suitable%20reference%20genes%20for%20the%20relative%20quantification%20of%20microRNAs%20in%20pleural%20effusion.%20Oncol%20Lett.%202014%3B%208%3A%201889%E2%80%931895.%20doi%3A%2010.3892%2Fol.2014.2404%2025202432" data-title="Identification%20of%20suitable%20reference%20genes%20for%20the%20relative%20quantification%20of%20microRNAs%20in%20pleural%20effusion" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25202432" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+suitable+reference+genes+for+the+relative+quantification+of+microRNAs+in+pleural+effusion+Han+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0171120.ref053" id="pone.0171120.ref053" class="link-target"></a>Peltier HJ, Latham GJ. Normalization of microRNA expression levels in quantitative RT-PCR assays: identification of suitable reference RNA targets in normal and cancerous human solid tissues. RNA. 2008; 14: 844–852.  pmid:18375788 <ul class="reflinks"><li><a href="#" data-author="Peltier" data-cit="PeltierHJ%2C%20LathamGJ.%20Normalization%20of%20microRNA%20expression%20levels%20in%20quantitative%20RT-PCR%20assays%3A%20identification%20of%20suitable%20reference%20RNA%20targets%20in%20normal%20and%20cancerous%20human%20solid%20tissues.%20RNA.%202008%3B%2014%3A%20844%E2%80%93852.%20doi%3A%2010.1261%2Frna.939908%2018375788" data-title="Normalization%20of%20microRNA%20expression%20levels%20in%20quantitative%20RT-PCR%20assays%3A%20identification%20of%20suitable%20reference%20RNA%20targets%20in%20normal%20and%20cancerous%20human%20solid%20tissues" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18375788" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Normalization+of+microRNA+expression+levels+in+quantitative+RT-PCR+assays%3A+identification+of+suitable+reference+RNA+targets+in+normal+and+cancerous+human+solid+tissues+Peltier+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0171120.ref054" id="pone.0171120.ref054" class="link-target"></a>Lin Y, Lai Z. Evaluation of suitable reference genes for normalization of microRNA expression by real-time reverse transcription PCR analysis during longan somatic embryogenesis. Plant physiol Biochem. 2013; 66: 20–25.  pmid:23454294 <ul class="reflinks"><li><a href="#" data-author="Lin" data-cit="LinY%2C%20LaiZ.%20Evaluation%20of%20suitable%20reference%20genes%20for%20normalization%20of%20microRNA%20expression%20by%20real-time%20reverse%20transcription%20PCR%20analysis%20during%20longan%20somatic%20embryogenesis.%20Plant%20physiol%20Biochem.%202013%3B%2066%3A%2020%E2%80%9325.%20doi%3A%2010.1016%2Fj.plaphy.2013.02.002%2023454294" data-title="Evaluation%20of%20suitable%20reference%20genes%20for%20normalization%20of%20microRNA%20expression%20by%20real-time%20reverse%20transcription%20PCR%20analysis%20during%20longan%20somatic%20embryogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23454294" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Evaluation+of+suitable+reference+genes+for+normalization+of+microRNA+expression+by+real-time+reverse+transcription+PCR+analysis+during+longan+somatic+embryogenesis+Lin+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pone.0171120.ref055" id="pone.0171120.ref055" class="link-target"></a>Pfaffl MW, Tichopad A, Progomet C, Neuvians TP. Determination of stable housekeeping genes, differentially regulated target genes and sample integrity: Bestkeeper-Excel-based tool using pair-wise correlations. Biotechnol let. 2004; 26: 509–515. <ul class="reflinks"><li><a href="#" data-author="Pfaffl" data-cit="PfafflMW%2C%20TichopadA%2C%20ProgometC%2C%20NeuviansTP.%20Determination%20of%20stable%20housekeeping%20genes%2C%20differentially%20regulated%20target%20genes%20and%20sample%20integrity%3A%20Bestkeeper-Excel-based%20tool%20using%20pair-wise%20correlations.%20Biotechnol%20let.%202004%3B%2026%3A%20509%E2%80%93515." data-title="Determination%20of%20stable%20housekeeping%20genes%2C%20differentially%20regulated%20target%20genes%20and%20sample%20integrity%3A%20Bestkeeper-Excel-based%20tool%20using%20pair-wise%20correlations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Determination+of+stable+housekeeping+genes%2C+differentially+regulated+target+genes+and+sample+integrity%3A+Bestkeeper-Excel-based+tool+using+pair-wise+correlations+Pfaffl+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pone.0171120.ref056" id="pone.0171120.ref056" class="link-target"></a>Biosen MK, Dehlendorff C, Linnemann D, Schultz NA, Jensen BV, Høgdall EVS, et al. MicroRNA expression in formalin-fixed paraffin-embedded cancer tissue: identifying reference microRNAs and variability. BMC Cancer. 2015; 15: 1024.  pmid:26714641 <ul class="reflinks"><li><a href="#" data-author="Biosen" data-cit="BiosenMK%2C%20DehlendorffC%2C%20LinnemannD%2C%20SchultzNA%2C%20JensenBV%2C%20H%C3%B8gdallEVS%2C%20et%20al.%20MicroRNA%20expression%20in%20formalin-fixed%20paraffin-embedded%20cancer%20tissue%3A%20identifying%20reference%20microRNAs%20and%20variability.%20BMC%20Cancer.%202015%3B%2015%3A%201024.%20doi%3A%2010.1186%2Fs12885-015-2030-2%2026714641" data-title="MicroRNA%20expression%20in%20formalin-fixed%20paraffin-embedded%20cancer%20tissue%3A%20identifying%20reference%20microRNAs%20and%20variability" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26714641" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+expression+in+formalin-fixed+paraffin-embedded+cancer+tissue%3A+identifying+reference+microRNAs+and+variability+Biosen+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pone.0171120.ref057" id="pone.0171120.ref057" class="link-target"></a>Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, De Paepe A, et al. Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome biol. 2002; 3: research0034. pmid:12184808 <ul class="reflinks"><li><a href="#" data-author="Vandesompele" data-cit="VandesompeleJ%2C%20De%20PreterK%2C%20PattynF%2C%20PoppeB%2C%20Van%20RoyN%2C%20De%20PaepeA%2C%20et%20al.%20Accurate%20normalization%20of%20real-time%20quantitative%20RT-PCR%20data%20by%20geometric%20averaging%20of%20multiple%20internal%20control%20genes.%20Genome%20biol.%202002%3B%203%3A%20research0034.%2012184808" data-title="Accurate%20normalization%20of%20real-time%20quantitative%20RT-PCR%20data%20by%20geometric%20averaging%20of%20multiple%20internal%20control%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12184808" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Accurate+normalization+of+real-time+quantitative+RT-PCR+data+by+geometric+averaging+of+multiple+internal+control+genes+Vandesompele+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pone.0171120.ref058" id="pone.0171120.ref058" class="link-target"></a>Rubio M, Belles X. Subtle roles of microRNAs let-7, miR-100 and miR-125 on wing morphogenesis in hemimetabolan metamorphosis. J Insect Physiol. 2013; 59: 1089–1094.  pmid:24071062 <ul class="reflinks"><li><a href="#" data-author="Rubio" data-cit="RubioM%2C%20BellesX.%20Subtle%20roles%20of%20microRNAs%20let-7%2C%20miR-100%20and%20miR-125%20on%20wing%20morphogenesis%20in%20hemimetabolan%20metamorphosis.%20J%20Insect%20Physiol.%202013%3B%2059%3A%201089%E2%80%931094.%20doi%3A%2010.1016%2Fj.jinsphys.2013.09.003%2024071062" data-title="Subtle%20roles%20of%20microRNAs%20let-7%2C%20miR-100%20and%20miR-125%20on%20wing%20morphogenesis%20in%20hemimetabolan%20metamorphosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24071062" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Subtle+roles+of+microRNAs+let-7%2C+miR-100+and+miR-125+on+wing+morphogenesis+in+hemimetabolan+metamorphosis+Rubio+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pone.0171120.ref059" id="pone.0171120.ref059" class="link-target"></a>Sempere LF, Dubrovsky EB, Dubrovskaya VA, Berger EM, Ambros V. The expression of the let-7 small regulatory RNA is controlled by ecdysone during metamorphosis in <em>Drosophila melanogaster</em>. Dev Biol. 2002; 244: 170–179.  pmid:11900466 <ul class="reflinks"><li><a href="#" data-author="Sempere" data-cit="SempereLF%2C%20DubrovskyEB%2C%20DubrovskayaVA%2C%20BergerEM%2C%20AmbrosV.%20The%20expression%20of%20the%20let-7%20small%20regulatory%20RNA%20is%20controlled%20by%20ecdysone%20during%20metamorphosis%20in%20Drosophila%20melanogaster.%20Dev%20Biol.%202002%3B%20244%3A%20170%E2%80%93179.%20doi%3A%2010.1006%2Fdbio.2002.0594%2011900466" data-title="The%20expression%20of%20the%20let-7%20small%20regulatory%20RNA%20is%20controlled%20by%20ecdysone%20during%20metamorphosis%20in%20Drosophila%20melanogaster" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11900466" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+expression+of+the+let-7+small+regulatory+RNA+is+controlled+by+ecdysone+during+metamorphosis+in+Drosophila+melanogaster+Sempere+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span class="order">60.
            </span><a name="pone.0171120.ref060" id="pone.0171120.ref060" class="link-target"></a>Liu S, Xia Q, Zhao P, Cheng T, Hong K, Xiang Z. Characterization and expression patterns of let-7 microRNA in the silkworm (<em>Bombyx mori</em>). BMC Dev Biol. 2007; 7: 88.  pmid:17651473 <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuS%2C%20XiaQ%2C%20ZhaoP%2C%20ChengT%2C%20HongK%2C%20XiangZ.%20Characterization%20and%20expression%20patterns%20of%20let-7%20microRNA%20in%20the%20silkworm%20%28Bombyx%20mori%29.%20BMC%20Dev%20Biol.%202007%3B%207%3A%2088.%20doi%3A%2010.1186%2F1471-213X-7-88%2017651473" data-title="Characterization%20and%20expression%20patterns%20of%20let-7%20microRNA%20in%20the%20silkworm%20%28Bombyx%20mori%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17651473" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+and+expression+patterns+of+let-7+microRNA+in+the+silkworm+%28Bombyx+mori%29+Liu+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span class="order">61.
            </span><a name="pone.0171120.ref061" id="pone.0171120.ref061" class="link-target"></a>Liu S, Li D, Li Q, Zhao P, Xiang Z, Xia Q. MicroRNAs of <em>Bombyx mori</em> identified by solexa sequencing. BMC Genomics. 2010; 11: 148.  pmid:20199675 <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuS%2C%20LiD%2C%20LiQ%2C%20ZhaoP%2C%20XiangZ%2C%20XiaQ.%20MicroRNAs%20of%20Bombyx%20mori%20identified%20by%20solexa%20sequencing.%20BMC%20Genomics.%202010%3B%2011%3A%20148.%20doi%3A%2010.1186%2F1471-2164-11-148%2020199675" data-title="MicroRNAs%20of%20Bombyx%20mori%20identified%20by%20solexa%20sequencing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20199675" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+of+Bombyx+mori+identified+by+solexa+sequencing+Liu+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span class="order">62.
            </span><a name="pone.0171120.ref062" id="pone.0171120.ref062" class="link-target"></a>Ge X, Zhang Y, Jiang J, Zhong Y, Yang X, Li Z, et al. Identification of microRNAs in <em>Helicoverpa armigera</em> and <em>Spodoptera litura</em> based on deep sequencing and homology analysis. Int J Biol Sci. 2013; 9: 1–15.  pmid:23289012 <ul class="reflinks"><li><a href="#" data-author="Ge" data-cit="GeX%2C%20ZhangY%2C%20JiangJ%2C%20ZhongY%2C%20YangX%2C%20LiZ%2C%20et%20al.%20Identification%20of%20microRNAs%20in%20Helicoverpa%20armigera%20and%20Spodoptera%20litura%20based%20on%20deep%20sequencing%20and%20homology%20analysis.%20Int%20J%20Biol%20Sci.%202013%3B%209%3A%201%E2%80%9315.%20doi%3A%2010.7150%2Fijbs.5249%2023289012" data-title="Identification%20of%20microRNAs%20in%20Helicoverpa%20armigera%20and%20Spodoptera%20litura%20based%20on%20deep%20sequencing%20and%20homology%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23289012" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+microRNAs+in+Helicoverpa+armigera+and+Spodoptera+litura+based+on+deep+sequencing+and+homology+analysis+Ge+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref63"><span class="order">63.
            </span><a name="pone.0171120.ref063" id="pone.0171120.ref063" class="link-target"></a>Liang P, Feng B, Zhou X, Gao X. Identification and developmental profiling of microRNAs in diamondback moth, <em>Plutella xylostella</em> (L.). PLoS ONE. 2013; 8: e78787.  pmid:24236051 <ul class="reflinks"><li><a href="#" data-author="Liang" data-cit="LiangP%2C%20FengB%2C%20ZhouX%2C%20GaoX.%20Identification%20and%20developmental%20profiling%20of%20microRNAs%20in%20diamondback%20moth%2C%20Plutella%20xylostella%20%28L.%29.%20PLoS%20ONE.%202013%3B%208%3A%20e78787.%20doi%3A%2010.1371%2Fjournal.pone.0078787%2024236051" data-title="Identification%20and%20developmental%20profiling%20of%20microRNAs%20in%20diamondback%20moth%2C%20Plutella%20xylostella%20%28L.%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24236051" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+developmental+profiling+of+microRNAs+in+diamondback+moth%2C+Plutella+xylostella+%28L.%29+Liang+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref64"><span class="order">64.
            </span><a name="pone.0171120.ref064" id="pone.0171120.ref064" class="link-target"></a>Biryukova I, Ye T, Levashina E. Transcriptome-wide analysis of microRNA expression in the malaria mosquito <em>Anopheles gambiae</em>. BMC Genomics. 2014; 15: 557.  pmid:24997592 <ul class="reflinks"><li><a href="#" data-author="Biryukova" data-cit="BiryukovaI%2C%20YeT%2C%20LevashinaE.%20Transcriptome-wide%20analysis%20of%20microRNA%20expression%20in%20the%20malaria%20mosquito%20Anopheles%20gambiae.%20BMC%20Genomics.%202014%3B%2015%3A%20557.%20doi%3A%2010.1186%2F1471-2164-15-557%2024997592" data-title="Transcriptome-wide%20analysis%20of%20microRNA%20expression%20in%20the%20malaria%20mosquito%20Anopheles%20gambiae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24997592" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptome-wide+analysis+of+microRNA+expression+in+the+malaria+mosquito+Anopheles+gambiae+Biryukova+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref65"><span class="order">65.
            </span><a name="pone.0171120.ref065" id="pone.0171120.ref065" class="link-target"></a>Blenkiron C, Tsai P, Brown LA, Tintinger V, Askelund KJ, Windsor JA, et al. Characterisation of the small RNAs in the biomedically important green-bottle blowfly <em>Lucilia sericata</em>. PLoS ONE. 2015; 10: e0122203.  pmid:25803701 <ul class="reflinks"><li><a href="#" data-author="Blenkiron" data-cit="BlenkironC%2C%20TsaiP%2C%20BrownLA%2C%20TintingerV%2C%20AskelundKJ%2C%20WindsorJA%2C%20et%20al.%20Characterisation%20of%20the%20small%20RNAs%20in%20the%20biomedically%20important%20green-bottle%20blowfly%20Lucilia%20sericata.%20PLoS%20ONE.%202015%3B%2010%3A%20e0122203.%20doi%3A%2010.1371%2Fjournal.pone.0122203%2025803701" data-title="Characterisation%20of%20the%20small%20RNAs%20in%20the%20biomedically%20important%20green-bottle%20blowfly%20Lucilia%20sericata" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25803701" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Characterisation+of+the+small+RNAs+in+the+biomedically+important+green-bottle+blowfly+Lucilia+sericata+Blenkiron+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref66"><span class="order">66.
            </span><a name="pone.0171120.ref066" id="pone.0171120.ref066" class="link-target"></a>Zhang Y, Huang Q, Yin G, Lee S, Jia R, Liu Z, et al. Identification of microRNAs by small RNA deep sequencing for synthetic microRNAs mimics to control <em>Spodoptera exigua</em>. Gene. 2015; 557: 215–221.  pmid:25528266 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangY%2C%20HuangQ%2C%20YinG%2C%20LeeS%2C%20JiaR%2C%20LiuZ%2C%20et%20al.%20Identification%20of%20microRNAs%20by%20small%20RNA%20deep%20sequencing%20for%20synthetic%20microRNAs%20mimics%20to%20control%20Spodoptera%20exigua.%20Gene.%202015%3B%20557%3A%20215%E2%80%93221.%20doi%3A%2010.1016%2Fj.gene.2014.12.038%2025528266" data-title="Identification%20of%20microRNAs%20by%20small%20RNA%20deep%20sequencing%20for%20synthetic%20microRNAs%20mimics%20to%20control%20Spodoptera%20exigua" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25528266" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+microRNAs+by+small+RNA+deep+sequencing+for+synthetic+microRNAs+mimics+to+control+Spodoptera+exigua+Zhang+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref67"><span class="order">67.
            </span><a name="pone.0171120.ref067" id="pone.0171120.ref067" class="link-target"></a>Karres JS, Hilgers V, Carrera I, Treisman J, Cohen SM. The conserved microRNA miR-8 tunes atrophin levels to prevent neurodegeneration in <em>Drosophila</em>. Cell. 2007; 131: 136–145.  pmid:17923093 <ul class="reflinks"><li><a href="#" data-author="Karres" data-cit="KarresJS%2C%20HilgersV%2C%20CarreraI%2C%20TreismanJ%2C%20CohenSM.%20The%20conserved%20microRNA%20miR-8%20tunes%20atrophin%20levels%20to%20prevent%20neurodegeneration%20in%20Drosophila.%20Cell.%202007%3B%20131%3A%20136%E2%80%93145.%20doi%3A%2010.1016%2Fj.cell.2007.09.020%2017923093" data-title="The%20conserved%20microRNA%20miR-8%20tunes%20atrophin%20levels%20to%20prevent%20neurodegeneration%20in%20Drosophila" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17923093" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+conserved+microRNA+miR-8+tunes+atrophin+levels+to+prevent+neurodegeneration+in+Drosophila+Karres+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref68"><span class="order">68.
            </span><a name="pone.0171120.ref068" id="pone.0171120.ref068" class="link-target"></a>Loya CM, Lu CS, Van Vactor D, Fulga TA. Transgenic microRNA inhibition with spatiotemporal specificity in intact organisms. Nat Methods. 2009; 6: 897–903.  pmid:19915559 <ul class="reflinks"><li><a href="#" data-author="Loya" data-cit="LoyaCM%2C%20LuCS%2C%20Van%20VactorD%2C%20FulgaTA.%20Transgenic%20microRNA%20inhibition%20with%20spatiotemporal%20specificity%20in%20intact%20organisms.%20Nat%20Methods.%202009%3B%206%3A%20897%E2%80%93903.%20doi%3A%2010.1038%2Fnmeth.1402%2019915559" data-title="Transgenic%20microRNA%20inhibition%20with%20spatiotemporal%20specificity%20in%20intact%20organisms" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19915559" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Transgenic+microRNA+inhibition+with+spatiotemporal+specificity+in+intact+organisms+Loya+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref69"><span class="order">69.
            </span><a name="pone.0171120.ref069" id="pone.0171120.ref069" class="link-target"></a>Choi IK, Hyun S. Conserved microRNA miR-8 in fat body regulates innate immune homeostasis in <em>Drosophila</em>. Dev Comp Immunol. 2012; 37: 50–54.  pmid:22210547 <ul class="reflinks"><li><a href="#" data-author="Choi" data-cit="ChoiIK%2C%20HyunS.%20Conserved%20microRNA%20miR-8%20in%20fat%20body%20regulates%20innate%20immune%20homeostasis%20in%20Drosophila.%20Dev%20Comp%20Immunol.%202012%3B%2037%3A%2050%E2%80%9354.%20doi%3A%2010.1016%2Fj.dci.2011.12.008%2022210547" data-title="Conserved%20microRNA%20miR-8%20in%20fat%20body%20regulates%20innate%20immune%20homeostasis%20in%20Drosophila" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22210547" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Conserved+microRNA+miR-8+in+fat+body+regulates+innate+immune+homeostasis+in+Drosophila+Choi+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref70"><span class="order">70.
            </span><a name="pone.0171120.ref070" id="pone.0171120.ref070" class="link-target"></a>Jin H, Kim VN, Hyun S. Conserved microRNA miR-8 controls body size in response to steroid signaling in <em>Drosophila</em>. Genes Dev. 2012; 26: 1427–1432.  pmid:22751499 <ul class="reflinks"><li><a href="#" data-author="Jin" data-cit="JinH%2C%20KimVN%2C%20HyunS.%20Conserved%20microRNA%20miR-8%20controls%20body%20size%20in%20response%20to%20steroid%20signaling%20in%20Drosophila.%20Genes%20Dev.%202012%3B%2026%3A%201427%E2%80%931432.%20doi%3A%2010.1101%2Fgad.192872.112%2022751499" data-title="Conserved%20microRNA%20miR-8%20controls%20body%20size%20in%20response%20to%20steroid%20signaling%20in%20Drosophila" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22751499" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Conserved+microRNA+miR-8+controls+body+size+in+response+to+steroid+signaling+in+Drosophila+Jin+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref71"><span class="order">71.
            </span><a name="pone.0171120.ref071" id="pone.0171120.ref071" class="link-target"></a>Zhang X, Zheng Y, Cao X, Ren R, Yu X, Jiang H. Identification and profiling of <em>Manduca sexta</em> microRNAs and their possible roles in regulating specific transcripts in fat body, hemocytes, and midgut. Insect Biochem Mol Biol. 2015; 62: 11–22.  pmid:25196249 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangX%2C%20ZhengY%2C%20CaoX%2C%20RenR%2C%20YuX%2C%20JiangH.%20Identification%20and%20profiling%20of%20Manduca%20sexta%20microRNAs%20and%20their%20possible%20roles%20in%20regulating%20specific%20transcripts%20in%20fat%20body%2C%20hemocytes%2C%20and%20midgut.%20Insect%20Biochem%20Mol%20Biol.%202015%3B%2062%3A%2011%E2%80%9322.%20doi%3A%2010.1016%2Fj.ibmb.2014.08.006%2025196249" data-title="Identification%20and%20profiling%20of%20Manduca%20sexta%20microRNAs%20and%20their%20possible%20roles%20in%20regulating%20specific%20transcripts%20in%20fat%20body%2C%20hemocytes%2C%20and%20midgut" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25196249" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+profiling+of+Manduca+sexta+microRNAs+and+their+possible+roles+in+regulating+specific+transcripts+in+fat+body%2C+hemocytes%2C+and+midgut+Zhang+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref72"><span class="order">72.
            </span><a name="pone.0171120.ref072" id="pone.0171120.ref072" class="link-target"></a>Jiang J, Ge X, Li Z, Wang Y, Song Q, Stanley DW, et al. MicroRNA-281 regulates the expression of ecdysone receptor (EcR) isoform B in the silkworm, <em>Bombyx mori</em>. Insect Biochem Mol Biol. 2013; 43: 692–700.  pmid:23707601 <ul class="reflinks"><li><a href="#" data-author="Jiang" data-cit="JiangJ%2C%20GeX%2C%20LiZ%2C%20WangY%2C%20SongQ%2C%20StanleyDW%2C%20et%20al.%20MicroRNA-281%20regulates%20the%20expression%20of%20ecdysone%20receptor%20%28EcR%29%20isoform%20B%20in%20the%20silkworm%2C%20Bombyx%20mori.%20Insect%20Biochem%20Mol%20Biol.%202013%3B%2043%3A%20692%E2%80%93700.%20doi%3A%2010.1016%2Fj.ibmb.2013.05.002%2023707601" data-title="MicroRNA-281%20regulates%20the%20expression%20of%20ecdysone%20receptor%20%28EcR%29%20isoform%20B%20in%20the%20silkworm%2C%20Bombyx%20mori" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23707601" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-281+regulates+the+expression+of+ecdysone+receptor+%28EcR%29+isoform+B+in+the+silkworm%2C+Bombyx+mori+Jiang+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref73"><span class="order">73.
            </span><a name="pone.0171120.ref073" id="pone.0171120.ref073" class="link-target"></a>Zhou Y, Liu Y, Yan H, Li Y, Zhang H, Xu J, et al. miR-281, an abundant midgut-specific miRNA of the vector mosquito <em>Aedes albopictus</em> enhances dengue virus replication. Parasit Vectors. 2014; 7: 488.  pmid:25331963 <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouY%2C%20LiuY%2C%20YanH%2C%20LiY%2C%20ZhangH%2C%20XuJ%2C%20et%20al.%20miR-281%2C%20an%20abundant%20midgut-specific%20miRNA%20of%20the%20vector%20mosquito%20Aedes%20albopictus%20enhances%20dengue%20virus%20replication.%20Parasit%20Vectors.%202014%3B%207%3A%20488.%20doi%3A%2010.1186%2Fs13071-014-0488-4%2025331963" data-title="miR-281%2C%20an%20abundant%20midgut-specific%20miRNA%20of%20the%20vector%20mosquito%20Aedes%20albopictus%20enhances%20dengue%20virus%20replication" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25331963" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=miR-281%2C+an+abundant+midgut-specific+miRNA+of+the+vector+mosquito+Aedes+albopictus+enhances+dengue+virus+replication+Zhou+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref74"><span class="order">74.
            </span><a name="pone.0171120.ref074" id="pone.0171120.ref074" class="link-target"></a>Jagadeeswaran G, Zheng Y, Sumathipala N, Jiang H, Arrese EL, Soulages JL, et al. Deep sequencing of small RNA libraries reveals dynamic regulation of conserved and novel microRNAs and microRNA-stars during silkworm development. BMC Genomics. 2010; 11: 52.  pmid:20089182 <ul class="reflinks"><li><a href="#" data-author="Jagadeeswaran" data-cit="JagadeeswaranG%2C%20ZhengY%2C%20SumathipalaN%2C%20JiangH%2C%20ArreseEL%2C%20SoulagesJL%2C%20et%20al.%20Deep%20sequencing%20of%20small%20RNA%20libraries%20reveals%20dynamic%20regulation%20of%20conserved%20and%20novel%20microRNAs%20and%20microRNA-stars%20during%20silkworm%20development.%20BMC%20Genomics.%202010%3B%2011%3A%2052.%20doi%3A%2010.1186%2F1471-2164-11-52%2020089182" data-title="Deep%20sequencing%20of%20small%20RNA%20libraries%20reveals%20dynamic%20regulation%20of%20conserved%20and%20novel%20microRNAs%20and%20microRNA-stars%20during%20silkworm%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20089182" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Deep+sequencing+of+small+RNA+libraries+reveals+dynamic+regulation+of+conserved+and+novel+microRNAs+and+microRNA-stars+during+silkworm+development+Jagadeeswaran+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref75"><span class="order">75.
            </span><a name="pone.0171120.ref075" id="pone.0171120.ref075" class="link-target"></a>Macedo LMF, Nunes FMF, Freitas CV, Pires CV, Tanaka ED, Martins JR, et al. MicroRNA signatures characterizing caste-independent ovarian activity in queen and worker honeybees (<em>Apis mellifera</em> L.). Insect Mol Biol. 2016; 25: 216–226.  pmid:26853694 <ul class="reflinks"><li><a href="#" data-author="Macedo" data-cit="MacedoLMF%2C%20NunesFMF%2C%20FreitasCV%2C%20PiresCV%2C%20TanakaED%2C%20MartinsJR%2C%20et%20al.%20MicroRNA%20signatures%20characterizing%20caste-independent%20ovarian%20activity%20in%20queen%20and%20worker%20honeybees%20%28Apis%20mellifera%20L.%29.%20Insect%20Mol%20Biol.%202016%3B%2025%3A%20216%E2%80%93226.%20doi%3A%2010.1111%2Fimb.12214%2026853694" data-title="MicroRNA%20signatures%20characterizing%20caste-independent%20ovarian%20activity%20in%20queen%20and%20worker%20honeybees%20%28Apis%20mellifera%20L.%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26853694" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+signatures+characterizing+caste-independent+ovarian+activity+in+queen+and+worker+honeybees+%28Apis+mellifera+L.%29+Macedo+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref76"><span class="order">76.
            </span><a name="pone.0171120.ref076" id="pone.0171120.ref076" class="link-target"></a>Kim K, Vinayagam A, Perrimon N. A rapid genome-wide microRNA screen identifies miR-14 as a modulator of Hedgehop signaling. Cell Rep. 2014; 7: 2066–2077.  pmid:24931604 <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimK%2C%20VinayagamA%2C%20PerrimonN.%20A%20rapid%20genome-wide%20microRNA%20screen%20identifies%20miR-14%20as%20a%20modulator%20of%20Hedgehop%20signaling.%20Cell%20Rep.%202014%3B%207%3A%202066%E2%80%932077.%20doi%3A%2010.1016%2Fj.celrep.2014.05.025%2024931604" data-title="A%20rapid%20genome-wide%20microRNA%20screen%20identifies%20miR-14%20as%20a%20modulator%20of%20Hedgehop%20signaling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24931604" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+rapid+genome-wide+microRNA+screen+identifies+miR-14+as+a+modulator+of+Hedgehop+signaling+Kim+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref77"><span class="order">77.
            </span><a name="pone.0171120.ref077" id="pone.0171120.ref077" class="link-target"></a>Varghese J, Cohen SM. microRNA miR-14 acts to modulate a positive autoregulatory loop controlling steroid hormone signaling in <em>Drasophila</em>. Genes Dev. 2007; 21: 2277–2282.  pmid:17761811 <ul class="reflinks"><li><a href="#" data-author="Varghese" data-cit="VargheseJ%2C%20CohenSM.%20microRNA%20miR-14%20acts%20to%20modulate%20a%20positive%20autoregulatory%20loop%20controlling%20steroid%20hormone%20signaling%20in%20Drasophila.%20Genes%20Dev.%202007%3B%2021%3A%202277%E2%80%932282.%20doi%3A%2010.1101%2Fgad.439807%2017761811" data-title="microRNA%20miR-14%20acts%20to%20modulate%20a%20positive%20autoregulatory%20loop%20controlling%20steroid%20hormone%20signaling%20in%20Drasophila" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17761811" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=microRNA+miR-14+acts+to+modulate+a+positive+autoregulatory+loop+controlling+steroid+hormone+signaling+in+Drasophila+Varghese+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref78"><span class="order">78.
            </span><a name="pone.0171120.ref078" id="pone.0171120.ref078" class="link-target"></a>Nelson C, Ambros V, Baehrecke EH. miR-14 regualtes autophagy during developmental cell death by targeting <em>ip3-kinase 2</em>. Mol Cell. 2014; 56: 376–388.  pmid:25306920 <ul class="reflinks"><li><a href="#" data-author="Nelson" data-cit="NelsonC%2C%20AmbrosV%2C%20BaehreckeEH.%20miR-14%20regualtes%20autophagy%20during%20developmental%20cell%20death%20by%20targeting%20ip3-kinase%202.%20Mol%20Cell.%202014%3B%2056%3A%20376%E2%80%93388.%20doi%3A%2010.1016%2Fj.molcel.2014.09.011%2025306920" data-title="miR-14%20regualtes%20autophagy%20during%20developmental%20cell%20death%20by%20targeting%20ip3-kinase%202" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25306920" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=miR-14+regualtes+autophagy+during+developmental+cell+death+by+targeting+ip3-kinase+2+Nelson+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref79"><span class="order">79.
            </span><a name="pone.0171120.ref079" id="pone.0171120.ref079" class="link-target"></a>Jayachandran B, Hussain M, Asgari S. Regulation of <em>Helicoverpa armigera</em> ecdysone receptor by miR-14 and its potential link to baculovirus infection. J Invertebr Pathol. 2013; 114: 151–157.  pmid:23899861 <ul class="reflinks"><li><a href="#" data-author="Jayachandran" data-cit="JayachandranB%2C%20HussainM%2C%20AsgariS.%20Regulation%20of%20Helicoverpa%20armigera%20ecdysone%20receptor%20by%20miR-14%20and%20its%20potential%20link%20to%20baculovirus%20infection.%20J%20Invertebr%20Pathol.%202013%3B%20114%3A%20151%E2%80%93157.%20doi%3A%2010.1016%2Fj.jip.2013.07.004%2023899861" data-title="Regulation%20of%20Helicoverpa%20armigera%20ecdysone%20receptor%20by%20miR-14%20and%20its%20potential%20link%20to%20baculovirus%20infection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23899861" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+Helicoverpa+armigera+ecdysone+receptor+by+miR-14+and+its+potential+link+to+baculovirus+infection+Jayachandran+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref80"><span class="order">80.
            </span><a name="pone.0171120.ref080" id="pone.0171120.ref080" class="link-target"></a>Chang Z, Tang N, Wang L, Zhang L, Akinyemi IA, Wu Q. Identification and characterization of microRNAs in the white-backed planthopper, <em>Sogatella furcifera</em>. Insect Sci. 2016; 23: 452–468.  pmid:27060479 <ul class="reflinks"><li><a href="#" data-author="Chang" data-cit="ChangZ%2C%20TangN%2C%20WangL%2C%20ZhangL%2C%20AkinyemiIA%2C%20WuQ.%20Identification%20and%20characterization%20of%20microRNAs%20in%20the%20white-backed%20planthopper%2C%20Sogatella%20furcifera.%20Insect%20Sci.%202016%3B%2023%3A%20452%E2%80%93468.%20doi%3A%2010.1111%2F1744-7917.12343%2027060479" data-title="Identification%20and%20characterization%20of%20microRNAs%20in%20the%20white-backed%20planthopper%2C%20Sogatella%20furcifera" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27060479" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+characterization+of+microRNAs+in+the+white-backed+planthopper%2C+Sogatella+furcifera+Chang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref81"><span class="order">81.
            </span><a name="pone.0171120.ref081" id="pone.0171120.ref081" class="link-target"></a>Carrington JC, Ambros V. Role of microRNAs in plant and animal development. Science. 2003; 301: 336–338.  pmid:12869753 <ul class="reflinks"><li><a href="#" data-author="Carrington" data-cit="CarringtonJC%2C%20AmbrosV.%20Role%20of%20microRNAs%20in%20plant%20and%20animal%20development.%20Science.%202003%3B%20301%3A%20336%E2%80%93338.%20doi%3A%2010.1126%2Fscience.1085242%2012869753" data-title="Role%20of%20microRNAs%20in%20plant%20and%20animal%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869753" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Role+of+microRNAs+in+plant+and+animal+development+Carrington+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref82"><span class="order">82.
            </span><a name="pone.0171120.ref082" id="pone.0171120.ref082" class="link-target"></a>Rota-Stabelli O, Campbell L, Brinkmann H, Edgecombe GD, Longhorn SJ, Peterson KJ, et al. A congruent solution to arthropod phylogeny: phylogenomics, microRNAs and morphology support monophyletic <em>Mandibulata</em>. Proc Biol Sci. 2011; 278: 298–306.  pmid:20702459 <ul class="reflinks"><li><a href="#" data-author="Rota-Stabelli" data-cit="Rota-StabelliO%2C%20CampbellL%2C%20BrinkmannH%2C%20EdgecombeGD%2C%20LonghornSJ%2C%20PetersonKJ%2C%20et%20al.%20A%20congruent%20solution%20to%20arthropod%20phylogeny%3A%20phylogenomics%2C%20microRNAs%20and%20morphology%20support%20monophyletic%20Mandibulata.%20Proc%20Biol%20Sci.%202011%3B%20278%3A%20298%E2%80%93306.%20doi%3A%2010.1098%2Frspb.2010.0590%2020702459" data-title="A%20congruent%20solution%20to%20arthropod%20phylogeny%3A%20phylogenomics%2C%20microRNAs%20and%20morphology%20support%20monophyletic%20Mandibulata" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20702459" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+congruent+solution+to+arthropod+phylogeny%3A+phylogenomics%2C+microRNAs+and+morphology+support+monophyletic+Mandibulata+Rota-Stabelli+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref83"><span class="order">83.
            </span><a name="pone.0171120.ref083" id="pone.0171120.ref083" class="link-target"></a>Wheeler BM, Heimberg AM, Moy VN, Sperling EA, Holstein TW, Heber S, et al. The deep evolution of metazoan microRNAs. Evol Dev. 2009; 11: 50–68.  pmid:19196333 <ul class="reflinks"><li><a href="#" data-author="Wheeler" data-cit="WheelerBM%2C%20HeimbergAM%2C%20MoyVN%2C%20SperlingEA%2C%20HolsteinTW%2C%20HeberS%2C%20et%20al.%20The%20deep%20evolution%20of%20metazoan%20microRNAs.%20Evol%20Dev.%202009%3B%2011%3A%2050%E2%80%9368.%20doi%3A%2010.1111%2Fj.1525-142X.2008.00302.x%2019196333" data-title="The%20deep%20evolution%20of%20metazoan%20microRNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19196333" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+deep+evolution+of+metazoan+microRNAs+Wheeler+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref84"><span class="order">84.
            </span><a name="pone.0171120.ref084" id="pone.0171120.ref084" class="link-target"></a>Christodoulou F, Raible F, Tomer R, Simakov O, Trachana K, Klaus S, et al. Ancient animal microRNAs and the evolution of tissue identity. Nature. 2010; 463: 1084–1088.  pmid:20118916 <ul class="reflinks"><li><a href="#" data-author="Christodoulou" data-cit="ChristodoulouF%2C%20RaibleF%2C%20TomerR%2C%20SimakovO%2C%20TrachanaK%2C%20KlausS%2C%20et%20al.%20Ancient%20animal%20microRNAs%20and%20the%20evolution%20of%20tissue%20identity.%20Nature.%202010%3B%20463%3A%201084%E2%80%931088.%20doi%3A%2010.1038%2Fnature08744%2020118916" data-title="Ancient%20animal%20microRNAs%20and%20the%20evolution%20of%20tissue%20identity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20118916" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Ancient+animal+microRNAs+and+the+evolution+of+tissue+identity+Christodoulou+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref85"><span class="order">85.
            </span><a name="pone.0171120.ref085" id="pone.0171120.ref085" class="link-target"></a>Ninova M, Ronshaugen M, Griffiths-Jones S. Conserved temperal patterns of microRNA expression in <em>Drosophila</em> support a developmental hourglass model. Genome Biol Evol. 2014; 6: 2459–2467.  pmid:25169982 <ul class="reflinks"><li><a href="#" data-author="Ninova" data-cit="NinovaM%2C%20RonshaugenM%2C%20Griffiths-JonesS.%20Conserved%20temperal%20patterns%20of%20microRNA%20expression%20in%20Drosophila%20support%20a%20developmental%20hourglass%20model.%20Genome%20Biol%20Evol.%202014%3B%206%3A%202459%E2%80%932467.%20doi%3A%2010.1093%2Fgbe%2Fevu183%2025169982" data-title="Conserved%20temperal%20patterns%20of%20microRNA%20expression%20in%20Drosophila%20support%20a%20developmental%20hourglass%20model" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25169982" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Conserved+temperal+patterns+of+microRNA+expression+in+Drosophila+support+a+developmental+hourglass+model+Ninova+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref86"><span class="order">86.
            </span><a name="pone.0171120.ref086" id="pone.0171120.ref086" class="link-target"></a>Marco A, Hui J, Ronshaugen M, Griffiths-Jones S. Functional shifts in insect microRNA evolution. Genome Biol Evol. 2010; 2: 686–696.  pmid:20817720 <ul class="reflinks"><li><a href="#" data-author="Marco" data-cit="MarcoA%2C%20HuiJ%2C%20RonshaugenM%2C%20Griffiths-JonesS.%20Functional%20shifts%20in%20insect%20microRNA%20evolution.%20Genome%20Biol%20Evol.%202010%3B%202%3A%20686%E2%80%93696.%20doi%3A%2010.1093%2Fgbe%2Fevq053%2020817720" data-title="Functional%20shifts%20in%20insect%20microRNA%20evolution" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20817720" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Functional+shifts+in+insect+microRNA+evolution+Marco+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref87"><span class="order">87.
            </span><a name="pone.0171120.ref087" id="pone.0171120.ref087" class="link-target"></a>Zhou H, Arcila M, Li Z, Lee E, Henzler C, Liu J, et al. Deep annotation of mouse iso-miR and iso-moR variation. Nucleic Acids Res. 2012; 40: 5864–5875.  pmid:22434881 <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouH%2C%20ArcilaM%2C%20LiZ%2C%20LeeE%2C%20HenzlerC%2C%20LiuJ%2C%20et%20al.%20Deep%20annotation%20of%20mouse%20iso-miR%20and%20iso-moR%20variation.%20Nucleic%20Acids%20Res.%202012%3B%2040%3A%205864%E2%80%935875.%20doi%3A%2010.1093%2Fnar%2Fgks247%2022434881" data-title="Deep%20annotation%20of%20mouse%20iso-miR%20and%20iso-moR%20variation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22434881" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Deep+annotation+of+mouse+iso-miR+and+iso-moR+variation+Zhou+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref88"><span class="order">88.
            </span><a name="pone.0171120.ref088" id="pone.0171120.ref088" class="link-target"></a>Long QX, Hu JL, Huang AL. Deep Sequencing of the Hepatitis B Virus Genome: Analysis of Multiple Samples by Implementation of theIllumina Platform. Methods Mol Biol. 2017;1540: 211–218.  pmid:27975319 <ul class="reflinks"><li><a href="#" data-author="Long" data-cit="LongQX%2C%20HuJL%2C%20HuangAL.%20Deep%20Sequencing%20of%20the%20Hepatitis%20B%20Virus%20Genome%3A%20Analysis%20of%20Multiple%20Samples%20by%20Implementation%20of%20theIllumina%20Platform.%20Methods%20Mol%20Biol.%202017%3B1540%3A%20211%E2%80%93218.%20doi%3A%2010.1007%2F978-1-4939-6700-1_17%2027975319" data-title="Deep%20Sequencing%20of%20the%20Hepatitis%20B%20Virus%20Genome%3A%20Analysis%20of%20Multiple%20Samples%20by%20Implementation%20of%20theIllumina%20Platform" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27975319" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Deep+Sequencing+of+the+Hepatitis+B+Virus+Genome%3A+Analysis+of+Multiple+Samples+by+Implementation+of+theIllumina+Platform+Long+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref89"><span class="order">89.
            </span><a name="pone.0171120.ref089" id="pone.0171120.ref089" class="link-target"></a>Ragupathy R, Ravichandran S, Mahdi MS, Huang D, Domaratzki M, Cloutier S. Deep sequencing of wheat sRNA transcriptome reveals distinct temporal expression pattern of miRNAs in response to heat, light and UV. Sci Rep. 2016; 6: 39373.  pmid:28004741 <ul class="reflinks"><li><a href="#" data-author="Ragupathy" data-cit="RagupathyR%2C%20RavichandranS%2C%20MahdiMS%2C%20HuangD%2C%20DomaratzkiM%2C%20CloutierS.%20Deep%20sequencing%20of%20wheat%20sRNA%20transcriptome%20reveals%20distinct%20temporal%20expression%20pattern%20of%20miRNAs%20in%20response%20to%20heat%2C%20light%20and%20UV.%20Sci%20Rep.%202016%3B%206%3A%2039373.%20doi%3A%2010.1038%2Fsrep39373%2028004741" data-title="Deep%20sequencing%20of%20wheat%20sRNA%20transcriptome%20reveals%20distinct%20temporal%20expression%20pattern%20of%20miRNAs%20in%20response%20to%20heat%2C%20light%20and%20UV" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/28004741" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Deep+sequencing+of+wheat+sRNA+transcriptome+reveals+distinct+temporal+expression+pattern+of+miRNAs+in+response+to+heat%2C+light+and+UV+Ragupathy+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref90"><span class="order">90.
            </span><a name="pone.0171120.ref090" id="pone.0171120.ref090" class="link-target"></a>Andrews SV, Ladd-Acosta C, Feinberg AP, Hansen KD, Fallin MD. "Gap hunting" to characterize clustered probe signals in Illumina methylation array data. Epigenetics Chromatin. 2016; 9: 56.  pmid:27980682 <ul class="reflinks"><li><a href="#" data-author="Andrews" data-cit="AndrewsSV%2C%20Ladd-AcostaC%2C%20FeinbergAP%2C%20HansenKD%2C%20FallinMD.%20%22Gap%20hunting%22%20to%20characterize%20clustered%20probe%20signals%20in%20Illumina%20methylation%20array%20data.%20Epigenetics%20Chromatin.%202016%3B%209%3A%2056.%20doi%3A%2010.1186%2Fs13072-016-0107-z%2027980682" data-title="%22Gap%20hunting%22%20to%20characterize%20clustered%20probe%20signals%20in%20Illumina%20methylation%20array%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27980682" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=%22Gap+hunting%22+to+characterize+clustered+probe+signals+in+Illumina+methylation+array+data+Andrews+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref91"><span class="order">91.
            </span><a name="pone.0171120.ref091" id="pone.0171120.ref091" class="link-target"></a>Yang L, Yang HL, Tu ZC, Wang XL. High-Throughput Sequencing of Microbial Community Diversity and Dynamics during Douchi Fermentation. PLoS ONE. 2016; 11: e0168166.  pmid:27992473 <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangL%2C%20YangHL%2C%20TuZC%2C%20WangXL.%20High-Throughput%20Sequencing%20of%20Microbial%20Community%20Diversity%20and%20Dynamics%20during%20Douchi%20Fermentation.%20PLoS%20ONE.%202016%3B%2011%3A%20e0168166.%20doi%3A%2010.1371%2Fjournal.pone.0168166%2027992473" data-title="High-Throughput%20Sequencing%20of%20Microbial%20Community%20Diversity%20and%20Dynamics%20during%20Douchi%20Fermentation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27992473" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=High-Throughput+Sequencing+of+Microbial+Community+Diversity+and+Dynamics+during+Douchi+Fermentation+Yang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref92"><span class="order">92.
            </span><a name="pone.0171120.ref092" id="pone.0171120.ref092" class="link-target"></a>Meuti ME, Denlinger DL. Evolutionary links between circadian clocks and photoperiodic diapause in insects. Integr Comp Biol. 2013; 53: 131–143.  pmid:23615363 <ul class="reflinks"><li><a href="#" data-author="Meuti" data-cit="MeutiME%2C%20DenlingerDL.%20Evolutionary%20links%20between%20circadian%20clocks%20and%20photoperiodic%20diapause%20in%20insects.%20Integr%20Comp%20Biol.%202013%3B%2053%3A%20131%E2%80%93143.%20doi%3A%2010.1093%2Ficb%2Fict023%2023615363" data-title="Evolutionary%20links%20between%20circadian%20clocks%20and%20photoperiodic%20diapause%20in%20insects" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23615363" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Evolutionary+links+between+circadian+clocks+and+photoperiodic+diapause+in+insects+Meuti+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref93"><span class="order">93.
            </span><a name="pone.0171120.ref093" id="pone.0171120.ref093" class="link-target"></a>Ikeno T, Tanaka SI, Numata H, Goto SG. Photoperiodic diapause under the control of circadian clock genes in an insect. BMC Biol. 2010; 8: 116.  pmid:20815865 <ul class="reflinks"><li><a href="#" data-author="Ikeno" data-cit="IkenoT%2C%20TanakaSI%2C%20NumataH%2C%20GotoSG.%20Photoperiodic%20diapause%20under%20the%20control%20of%20circadian%20clock%20genes%20in%20an%20insect.%20BMC%20Biol.%202010%3B%208%3A%20116.%20doi%3A%2010.1186%2F1741-7007-8-116%2020815865" data-title="Photoperiodic%20diapause%20under%20the%20control%20of%20circadian%20clock%20genes%20in%20an%20insect" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20815865" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Photoperiodic+diapause+under+the+control+of+circadian+clock+genes+in+an+insect+Ikeno+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref94"><span class="order">94.
            </span><a name="pone.0171120.ref094" id="pone.0171120.ref094" class="link-target"></a>Pegoraro M, Tauber E. The role of microRNAs (miRNA) in circadian rhythmicity. J Genet. 2008; 87: 505–511. pmid:19147939 <ul class="reflinks"><li><a href="#" data-author="Pegoraro" data-cit="PegoraroM%2C%20TauberE.%20The%20role%20of%20microRNAs%20%28miRNA%29%20in%20circadian%20rhythmicity.%20J%20Genet.%202008%3B%2087%3A%20505%E2%80%93511.%2019147939" data-title="The%20role%20of%20microRNAs%20%28miRNA%29%20in%20circadian%20rhythmicity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19147939" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+role+of+microRNAs+%28miRNA%29+in+circadian+rhythmicity+Pegoraro+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref95"><span class="order">95.
            </span><a name="pone.0171120.ref095" id="pone.0171120.ref095" class="link-target"></a>Chen W, Liu Z, Li T, Zhang R, Xue Y, Zhong Y, et al. Regulation of <em>Drosophila</em> circadian rhythms by miRNA let-7 is mediated by a regulatory cycle. Nat Commun. 2014; 5: 5549.  pmid:25417916 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenW%2C%20LiuZ%2C%20LiT%2C%20ZhangR%2C%20XueY%2C%20ZhongY%2C%20et%20al.%20Regulation%20of%20Drosophila%20circadian%20rhythms%20by%20miRNA%20let-7%20is%20mediated%20by%20a%20regulatory%20cycle.%20Nat%20Commun.%202014%3B%205%3A%205549.%20doi%3A%2010.1038%2Fncomms6549%2025417916" data-title="Regulation%20of%20Drosophila%20circadian%20rhythms%20by%20miRNA%20let-7%20is%20mediated%20by%20a%20regulatory%20cycle" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25417916" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+Drosophila+circadian+rhythms+by+miRNA+let-7+is+mediated+by+a+regulatory+cycle+Chen+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref96"><span class="order">96.
            </span><a name="pone.0171120.ref096" id="pone.0171120.ref096" class="link-target"></a>Mestdagh P, Hartmann N, Baeriswyl L, Andreasen D, Bernard N, Chen C, et al. Evaluation of quantitative miRNA expression platforms in the microRNA quality control (miRQC) study. Nat Methods. 2014; 11: 809–815.  pmid:24973947 <ul class="reflinks"><li><a href="#" data-author="Mestdagh" data-cit="MestdaghP%2C%20HartmannN%2C%20BaeriswylL%2C%20AndreasenD%2C%20BernardN%2C%20ChenC%2C%20et%20al.%20Evaluation%20of%20quantitative%20miRNA%20expression%20platforms%20in%20the%20microRNA%20quality%20control%20%28miRQC%29%20study.%20Nat%20Methods.%202014%3B%2011%3A%20809%E2%80%93815.%20doi%3A%2010.1038%2Fnmeth.3014%2024973947" data-title="Evaluation%20of%20quantitative%20miRNA%20expression%20platforms%20in%20the%20microRNA%20quality%20control%20%28miRQC%29%20study" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24973947" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Evaluation+of+quantitative+miRNA+expression+platforms+in+the+microRNA+quality+control+%28miRQC%29+study+Mestdagh+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref97"><span class="order">97.
            </span><a name="pone.0171120.ref097" id="pone.0171120.ref097" class="link-target"></a>Kagias K, Podolska A, Pocock R. Reliable reference miRNAs for quantitative gene expression analysis of stress responses in <em>Caenorhabditis elegans</em>. BMC Genomics. 2014; 15: 222.  pmid:24656064 <ul class="reflinks"><li><a href="#" data-author="Kagias" data-cit="KagiasK%2C%20PodolskaA%2C%20PocockR.%20Reliable%20reference%20miRNAs%20for%20quantitative%20gene%20expression%20analysis%20of%20stress%20responses%20in%20Caenorhabditis%20elegans.%20BMC%20Genomics.%202014%3B%2015%3A%20222.%20doi%3A%2010.1186%2F1471-2164-15-222%2024656064" data-title="Reliable%20reference%20miRNAs%20for%20quantitative%20gene%20expression%20analysis%20of%20stress%20responses%20in%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24656064" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Reliable+reference+miRNAs+for+quantitative+gene+expression+analysis+of+stress+responses+in+Caenorhabditis+elegans+Kagias+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref98"><span class="order">98.
            </span><a name="pone.0171120.ref098" id="pone.0171120.ref098" class="link-target"></a>Yang Y, Zhang X, Chen Y, Guo J, Ling H, Gao S, et al. Selection of reference genes for normalization of microRNA expression by RT-qPCR in sugarcane buds under cold stress. Front Plant Sci. 2016; 7: 86.  pmid:26904058 <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangY%2C%20ZhangX%2C%20ChenY%2C%20GuoJ%2C%20LingH%2C%20GaoS%2C%20et%20al.%20Selection%20of%20reference%20genes%20for%20normalization%20of%20microRNA%20expression%20by%20RT-qPCR%20in%20sugarcane%20buds%20under%20cold%20stress.%20Front%20Plant%20Sci.%202016%3B%207%3A%2086.%20doi%3A%2010.3389%2Ffpls.2016.00086%2026904058" data-title="Selection%20of%20reference%20genes%20for%20normalization%20of%20microRNA%20expression%20by%20RT-qPCR%20in%20sugarcane%20buds%20under%20cold%20stress" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26904058" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Selection+of+reference+genes+for+normalization+of+microRNA+expression+by+RT-qPCR+in+sugarcane+buds+under+cold+stress+Yang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref99"><span class="order">99.
            </span><a name="pone.0171120.ref099" id="pone.0171120.ref099" class="link-target"></a>Schmittgen TD, Zakrajsek BA. Effect of experimental treatment on housekeeping gene expression: validation by real-time, quantitative RT-PCR. J Biochem Biophys Methods. 2000; 46: 69–81. pmid:11086195 <ul class="reflinks"><li><a href="#" data-author="Schmittgen" data-cit="SchmittgenTD%2C%20ZakrajsekBA.%20Effect%20of%20experimental%20treatment%20on%20housekeeping%20gene%20expression%3A%20validation%20by%20real-time%2C%20quantitative%20RT-PCR.%20J%20Biochem%20Biophys%20Methods.%202000%3B%2046%3A%2069%E2%80%9381.%2011086195" data-title="Effect%20of%20experimental%20treatment%20on%20housekeeping%20gene%20expression%3A%20validation%20by%20real-time%2C%20quantitative%20RT-PCR" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11086195" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Effect+of+experimental+treatment+on+housekeeping+gene+expression%3A+validation+by+real-time%2C+quantitative+RT-PCR+Schmittgen+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

