



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Lactate production is enhanced by adding calcium carbonate or sodium hydroxide during fermentation. However, <em>Bacillus coagulans</em> 2-6 can produce more than 180 g/L L-lactic acid when calcium lactate is accumulated, but less than 120 g/L L-lactic acid when sodium lactate is formed. The molecular mechanisms by which <em>B</em>. <em>coagulans</em> responds to calcium lactate and sodium lactate remain unclear. In this study, comparative transcriptomic methods based on high-throughput RNA sequencing were applied to study gene expression changes in <em>B</em>. <em>coagulans</em> 2-6 cultured in non-stress, sodium lactate stress and calcium lactate stress conditions. Gene expression profiling identified 712 and 1213 significantly regulated genes in response to calcium lactate stress and sodium lactate stress, respectively. Gene ontology assignments of the differentially expressed genes were performed. KEGG pathway enrichment analysis revealed that ‘ATP-binding cassette transporters’ were significantly affected by calcium lactate stress, and ‘amino sugar and nucleotide sugar metabolism’ was significantly affected by sodium lactate stress. It was also found that lactate fermentation was less affected by calcium lactate stress than by sodium lactate stress. Sodium lactate stress had negative effect on the expression of ‘glycolysis/gluconeogenesis’ genes but positive effect on the expression of ‘citrate cycle (TCA cycle)’ genes. However, calcium lactate stress had positive influence on the expression of ‘glycolysis/gluconeogenesis’ genes and had minor influence on ‘citrate cycle (TCA cycle)’ genes. Thus, our findings offer new insights into the responses of <em>B</em>. <em>coagulans</em> to different lactate stresses. Notably, our RNA-seq dataset constitute a robust database for investigating the functions of genes induced by lactate stress in the future and identify potential targets for genetic engineering to further improve L-lactic acid production by <em>B</em>. <em>coagulans</em>.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Qin J, Wang X, Wang L, Zhu B, Zhang X, Yao Q, et al.  (2015) Comparative Transcriptome Analysis Reveals Different Molecular Mechanisms of <em>Bacillus coagulans</em> 2-6 Response to Sodium Lactate and Calcium Lactate during Lactic Acid Production. PLoS ONE 10(4):
           e0124316.
        
        https://doi.org/10.1371/journal.pone.0124316</p><p><strong>Academic Editor: </strong>Selvakumar Subbian, Public Health Research Institute at RBHS, UNITED STATES</p><p><strong>Received: </strong>December 5, 2014; <strong>Accepted: </strong>February 27, 2015; <strong>Published: </strong> April 15, 2015</p><p><strong>Copyright: </strong> © 2015 Qin et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited</p><p><strong>Data Availability: </strong>Relevant data are within the paper and its Supporting Information files. All sequencing data have been deposited in the NCBI Sequence Read Archive under accession numbers SRX700697 (GY), SRX700698 (CA) and SRX 700710 (NA).</p><p><strong>Funding: </strong>This work was supported by the National Natural Science Foundation of China (31200078), the Shandong Provincial Natural Science Foundation, China (ZR2001CQ025), and the CAS Key Laboratory of Microbial Physiological and Metabolic Engineering, Institute of Microbiology, Chinese Academy of Sciences (KLIM-201302). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Lactic acid has wide applications in the industries of food, medicine, brewing, leather, textiles, environmental protection and agriculture [<a href="#pone.0124316.ref001" class="ref-tip">1</a>–<a href="#pone.0124316.ref003" class="ref-tip">3</a>]. The major industrial application of lactic acid is the production of polylactic acid, a biodegradable polymer. Given the biocompatibility, biodegradability and excellent mechanical properties of polylactic acid, it is considered the most promising novel packaging material of this century. It can replace non-degradable petrochemical materials such as polyethylene, polypropylene and polystyrene. Importantly, widespread use of polylactic acid has the potential to solve ‘white pollution’ and mitigate the current energy crisis [<a href="#pone.0124316.ref002" class="ref-tip">2</a>, <a href="#pone.0124316.ref004" class="ref-tip">4</a>].</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>During fermentative production of lactic acid, a neutraliser such as calcium carbonate or sodium hydroxide is required to maintain neutral or mildly acidic conditions for the fermentation liquid. Sodium lactate, calcium lactate or another lactate can accumulate in the fermentation broth depending on the type of neutraliser selected. High concentrations of lactate result in stress effects on lactic acid producers, and are among the key factors limiting the enhancement of lactic acid yields [<a href="#pone.0124316.ref005" class="ref-tip">5</a>]. Previous studies indicated that the maximum yield of lactic acid can exceed 200 g/L when calcium lactate accumulates in the fermentation broth [<a href="#pone.0124316.ref006" class="ref-tip">6</a>, <a href="#pone.0124316.ref007" class="ref-tip">7</a>]; however, the lactic acid yield is generally less than 125 g/L when sodium lactate is formed [<a href="#pone.0124316.ref005" class="ref-tip">5</a>, <a href="#pone.0124316.ref008" class="ref-tip">8</a>]. Therefore, the influences of lactate on lactic acid production may be different depending on the type of lactic acid formed.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p><em>Bacillus coagulans</em> strains are L-lactic acid producers with industrial potential because they can tolerate high fermentation temperatures, their culture media do not need sterilisation and their products exhibit high optical purity [<a href="#pone.0124316.ref009" class="ref-tip">9</a>, <a href="#pone.0124316.ref010" class="ref-tip">10</a>]. <em>B</em>. <em>coagulans</em> 2–6 can produce more than 180 g/L L-lactic acid with high optical purity when calcium lactate is accumulated, but less than 120 g/L L-lactic acid when sodium lactate is formed [<a href="#pone.0124316.ref005" class="ref-tip">5</a>, <a href="#pone.0124316.ref011" class="ref-tip">11</a>]. Under the same concentration of lactates, sodium lactate results in stronger stress on <em>B</em>. <em>coagulans</em> 2–6 compared with calcium lactate [<a href="#pone.0124316.ref012" class="ref-tip">12</a>]. However, the mechanisms by which <em>B</em>. <em>coagulans</em> strains respond to different lactates remain unclear.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Omics tools are effective for studying the response mechanisms of bacteria to stress conditions. Wang et al. [<a href="#pone.0124316.ref012" class="ref-tip">12</a>] reported a comparative proteomic analysis of <em>B</em>. <em>coagulans</em> 2–6 in response to lactate stress and identified four highly expressed proteins under calcium lactate stress. In the present study, we conducted comparative transcriptomics to study differential expression of genes among <em>B</em>. <em>coagulans</em> 2–6 cultured in normal, sodium lactate stress and calcium lactate stress conditions. Similarities and differences in the response of <em>B</em>. <em>coagulans</em> 2–6 to sodium lactate and calcium lactate were also investigated.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Bacterial strains and cultivation conditions"></a>
<h3>Bacterial strains and cultivation conditions</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>The <em>B</em>. <em>coagulans</em> wild-type strain 2–6 [<a href="#pone.0124316.ref011" class="ref-tip">11</a>, <a href="#pone.0124316.ref013" class="ref-tip">13</a>] was used in this study. It was cultivated at 50°C with shaking at 150 rpm in non-stress medium (GY medium: 40 g/L glucose, 10 g/L yeast extract, 1 g/L CaCl<sub>2</sub>, pH 6.7) until the cell dry weight (CDW) reached 2.0. Then 5 ml bacterial liquid was inoculated into 100 ml fresh non-stress medium and lactate stress medium (GY medium with 0.5 M sodium lactate or 0.25 M calcium lactate), and incubated at 50°C with shaking at 150 rpm until CDW reached 0.4 g/L at late exponential stage of the cell growth as reported before [<a href="#pone.0124316.ref012" class="ref-tip">12</a>]. Cells were harvested for preparation of total RNA.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Library preparation and RNA sequencing (RNA-seq)"></a>
<h3>Library preparation and RNA sequencing (RNA-seq)</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Total RNA of <em>B</em>. <em>coagulans</em> 2–6 was isolated using TRIzol reagent according to the manufacturer’s protocol (Invitrogen, Carlsbad, CA, USA) and then treated with DNase I. The quality and quantity of purified RNA were determined by measuring the absorbance at 260 nm/280 nm (A260/A280) using a Nanodrop ND-1000 spectrophotometer (LabTech, Wilmington, MA, USA). RNA integrity was further verified by electrophoresis through 1.5% (w/v) agarose gel. mRNA was enriched from total RNA using a Ribo-Zero Magnetic Kit (Gram-Positive Bacteria) (EpiCentre, Madison, WY, USA). Synthesis of double-stranded cDNA from the extracted mRNA was done using the Truseq RNA sample prep kit (Illumina, San Diego, CA, USA), followed by 15 cycles of enrichment PCR. Sequencing was carried out on the Illumina HiSeq 2000 platform (Majorbio Bio-Pharm Technology Co., Ltd., Shanghai, China). All sequencing data have been deposited in the NCBI Sequence Read Archive under accession numbers SRX700697 (GY), SRX700698 (CA) and SRX 700710 (NA).</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Transcriptome analysis"></a>
<h3>Transcriptome analysis</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>The raw data were initially processed to obtain clean reads by removing the adapter sequences and low quality bases. The clean reads were then aligned to the reference genome sequence of <em>B</em>. <em>coagulans</em> 2–6 (accession number: NC_015634) using the short sequence alignment software Bowtie 2 [<a href="#pone.0124316.ref014" class="ref-tip">14</a>]. To identify differentially expressed genes (DEGs) between the two different treatment conditions, the expression level for each transcript was calculated by RSEM (<a href="http://deweylab.biostat.wisc.edu/rsem/">http://deweylab.biostat.wisc.edu/rsem/</a>) using the fragments per kilobase of exon per million mapped reads method [<a href="#pone.0124316.ref015" class="ref-tip">15</a>]. The R statistical package software EdgeR (<a href="http://www.bioconductor.org/packages/2.12/bioc/html/edgeR.html">http://www.bioconductor.org/packages/2.12/bioc/html/edgeR.html</a>) was employed to quantify differential gene expression [<a href="#pone.0124316.ref016" class="ref-tip">16</a>]. Genes were defined as differentially expressed if they exhibited two-fold or greater change between the samples under a false discovery rate (FDR) of 5% or less. In addition, functional enrichment analyses, including gene ontology (GO) and Kyoto Encyclopaedia of Genes and Genomes (KEGG), were performed to identify which DEGs were significantly enriched in GO terms and metabolic pathways compared with the whole-transcriptome background using the formula described in previous studies [<a href="#pone.0124316.ref017" class="ref-tip">17</a>, <a href="#pone.0124316.ref018" class="ref-tip">18</a>]. A Bonferroni-corrected P-value ≤ 0.05 was considered statistically significant.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Quantitative real-time PCR (qRT-PCR) verification"></a>
<h3>Quantitative real-time PCR (qRT-PCR) verification</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>Five genes belonging to 5 different operons were chosen for confirmation of RNA-seq data by qRT-PCR. Total RNA was extracted using the EasyPure RNA Purification Kit (TRANSGEN BIOTECH, Beijing, China). Then, the total RNA sample was treated with DNase and qRT-PCR was performed on RNA samples to confirm the absence of DNA contamination using 16S primers [<a href="#pone.0124316.ref019" class="ref-tip">19</a>]. qRT-PCR was performed using the TransScript II Green One-Step qRT-PCR SuperMix (TRANSGEN BIOTECH, Beijing, China) and the Corbett Rotor Gene 3000 (Corbett Robotics, Sydney, Australia) according to the manufacturer’s instructions. The genes and primers used for qRT-PCR are shown in <a href="#pone.0124316.s001">S1 Table</a>. The relative gene expression data were analyzed using the 2<sup>-ΔΔCt</sup> method as described by Tan et al. [<a href="#pone.0124316.ref019" class="ref-tip">19</a>]. Rotor-Gene Real-Time Analysis Software 6.0 was used for data analysis (Corbett Robotics, Sydney, Australia). All qRT-PCR runs were conducted with three biological and three technical replicates.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec007" name="sec007" data-toc="sec007" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="Illumina draft reads"></a>
<h3>Illumina draft reads</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>Large volumes of data were generated with Illumina HiSeq 2000 sequencing of the three <em>B</em>. <em>coagulans</em> cDNA samples, namely, GY (untreated control), CA, (calcium lactate-stressed sample) and NA (sodium lactate-stressed sample). Each cDNA fragment yielded two 101 base pair long paired-end (PE) reads. After removing low quality reads and trimming off adapter sequences, 13,747,126 (GY), 13,464,898 (CA) and 14,974,752 (NA) high-quality, clean PE sequencing reads with a total of 1,334,784,442 (GY), 1,306,406,094 (CA) and 1,456,148,628 (NA) nucleotides were obtained for the three samples. Clean reads were aligned to the reference genome. The mapping rates were 97.9% (GY), 95.9% (CA) and 95.6% (NA).</p>
</div>

<div id="section2" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Gene ontology (GO) functional annotation of transcripts"></a>
<h3>Gene ontology (GO) functional annotation of transcripts</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>GO assignments were used to classify the functions of the <em>B</em>. <em>coagulans</em> genes under lactate stress. Based on sequence homology, 2053 transcript sequences were assigned at least one GO term, including 43 functional groups at the second level (<a href="#pone-0124316-g001">Fig 1</a>). Among these groups, ‘catalytic activity’ (1359 sequences, 66.2%), ‘binding’ (1091 sequences, 53.1%) and ‘single-organism process’ (587 sequences, 28.6%) were the most abundant subcategories for ‘molecular function’. ‘Cell’ (739 sequences, 36.0%), ‘cell part’ (739 sequences, 36.0%) and ‘membrane’ (431 sequences, 21.0%) terms were dominant in ‘cellular component’. ‘Metabolic process’ (1472 sequences, 71.7%), ‘cellular process’ (1396 sequences, 68.0%) and ‘single-organism process’ (587 sequences, 28.6%) were represented the highest under ‘biological process’.</p>
<a class="link-target" id="pone-0124316-g001" name="pone-0124316-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.g001" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Gene Ontology (GO) functional annotation of transcripts.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>All 2,053 transcripts were assigned to at least one GO term and were grouped into three main GO categories and 43 groups, 19 groups in biological process, 12 in cellular component, and 12 in molecular function. The right-hand Y-axis represents the number of genes in a sub-category. The left-hand Y-axis indicates the percentage of a specific sub-category of genes in each main category.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.g001">
              https://doi.org/10.1371/journal.pone.0124316.g001</a></p></div></div>

<div id="section3" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Differentially expressed genes (DEGs) involved in response to lactate stress in B. coagulans"></a>
<h3>Differentially expressed genes (DEGs) involved in response to lactate stress in <em>B</em>. <em>coagulans</em></h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>Comparison of gene expression showed that a total of 1577 unigenes were differentially expressed between any two-way comparison of GY, CA and NA (fold changes ≥2 or ≤−2; FDR&lt;0.05), including 712 isogenes between GY and CA (GY vs CA), 702 isogenes between NA and CA (NA vs CA) and 1213 isogenes between NA and GY (NA vs GY). The amounts of significantly up- and down-regulated genes in NA were 543 and 670, respectively, compared with those in GY. The amounts of significantly up- and down-regulated genes in CA were 398 and 314, respectively, compared with those in GY. Moreover, 364 unigenes were significantly differentially expressed in both CA vs GY and NA vs GY; 121 unigenes were significantly differentially expressed in both CA vs GY and NA vs CA; and 381 unigenes were significantly differentially expressed in both NA vs GY and NA vs CA. Only 92 unigenes were significantly differentially expressed in CA vs GY, NA vs GY and NA vs CA. The numbers of all differentially expressed genes among the three groups of GY, CA and NA are illustrated in <a href="#pone-0124316-g002">Fig 2</a>.</p>
<a class="link-target" id="pone-0124316-g002" name="pone-0124316-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.g002" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Differentially expressed genes that are unique or shared among three groups of GY, NA and CA.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p>NA vs GY refers to the comparison between sodium lactate stressed (NA) and untreated (GY) groups. CA vs GY refers to the comparison between calcium lactate stressed (CA) and GY groups. NA vs CA refers to the comparison between NA and CA groups. Numbers in each section of the figure indicate the numbers of differently expressed genes in the indicated comparison.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.g002">
              https://doi.org/10.1371/journal.pone.0124316.g002</a></p></div></div>

<div id="section4" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="GO assignments of DEGs"></a>
<h3>GO assignments of DEGs</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>GO assignments of DEGs in response to calcium lactate stress and sodium lactate stress were performed. The results are shown in <a href="#pone-0124316-t001">Table 1</a>. The 398 significantly upregulated genes in CA compared with those in GY were assigned 34 GO terms, namely, 14 subcategories of biological process, 9 subcategories of cellular component and 11 subcategories of molecular function. The 314 significantly down-regulated genes in CA compared with those in GY were assigned 34 GO terms, namely, 15 subcategories of biological process, 10 subcategories of cellular component and 9 subcategories of molecular function. The 543 significantly upregulated genes in NA compared with those in GY were assigned 39 GO terms, namely, 17 subcategories of biological process, 12 subcategories of cellular component and 10 subcategories of molecular function. The 670 significantly down-regulated genes in NA compared with those in GY were assigned 40 GO terms, namely, 18 subcategories of biological process, 10 subcategories of cellular component and 12 subcategories of molecular function. The significantly up- or down-regulated genes in the corresponding GO terms are listed in <a href="#pone-0124316-t001">Table 1</a>, <a href="#pone.0124316.s010">S1 Data</a> and <a href="#pone.0124316.s011">S2 Data</a>. Taken together, the results indicate differences in gene expression changes in response to calcium lactate stress and sodium lactate stress in <em>B</em>. <em>coagulans</em> 2–6.</p>
<a class="link-target" id="pone-0124316-t001" name="pone-0124316-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t001" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Gene ontology (GO) assignments of differentially expressed genes.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t001">
              https://doi.org/10.1371/journal.pone.0124316.t001</a></p></div></div>

<div id="section5" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="GO and pathway functional enrichment analysis of DEGs"></a>
<h3>GO and pathway functional enrichment analysis of DEGs</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>GO function and pathway enrichment analyses were performed on all DEGs. According to GO functional enrichment analysis between CA and GY, the GO term ‘response to stress’ was represented the most and included 96 unigenes and 41 significantly differentially expressed unigenes. Analysis between NA and GY revealed one significantly enriched GO term. This GO term was ‘biological process’ (GO: 0008150), which involved 1813 unigenes and 793 significantly differentially expressed unigenes. No significantly enriched GO term was found between NA and CA.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2" class="link-target"></a><p>Kyoto Encyclopaedia of Genes and Genomes (KEGG) enrichment analysis showed that the DEGs between the CA and GY groups were significantly involved in the ‘ATP-binding cassette (ABC) transporters’ (KEGG: ko02010) pathway, which included 96 unigenes and 37 significantly differentially expressed unigenes. The DEGs between the NA and GY groups were significantly involved in the ‘amino sugar and nucleotide sugar metabolism’ pathway, which included 35 unigenes and 26 significantly differentially expressed unigenes. The DEGs between the NA and CA groups were significantly involved in the ‘amino sugar and nucleotide sugar metabolism’ (KEGG: ko00520) pathway, which included 35 unigenes and 22 significantly differentially expressed unigenes.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="Identification and comparison of DEGs involved in ‘ABC transporters’"></a>
<h3>Identification and comparison of DEGs involved in ‘ABC transporters’</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>‘ABC transporters’, which are also called traffic ATPases, use the binding and hydrolysis of ATP to power the translocation of a diverse assortment of substrates across biological membranes, ranging from ions to macromolecules [<a href="#pone.0124316.ref020" class="ref-tip">20</a>]. ‘ABC transporters’ either import or export substrates with relative specificity for a given substrate. Dozens of ABC transporters were identified by genome analysis in <em>Escherichia coli</em> [<a href="#pone.0124316.ref021" class="ref-tip">21</a>] and <em>Bacillus subtilis</em> [<a href="#pone.0124316.ref022" class="ref-tip">22</a>].</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2" class="link-target"></a><p>KEGG pathway enrichment analysis revealed that ‘ABC transporters’ was significantly affected by calcium lactate stress. A total of 10 significantly upregulated genes (<a href="#pone-0124316-t002">Table 2</a>) and 27 significantly down-regulated genes (<a href="#pone.0124316.s002">S2 Table</a>) were involved in ‘ABC transporters’ under calcium lactate stress condition (CA group), compared with those under non-stress condition (GY group). Fourteen significantly upregulated genes (<a href="#pone-0124316-t003">Table 3</a>) and 35 significantly down-regulated genes (<a href="#pone.0124316.s003">S3 Table</a>) were involved in ‘ABC transporters’ under sodium lactate stress condition (NA group) compared with those under non-stress condition (GY group). Six genes, namely, glycine betaine/L-proline ABC transporter ATPase (BCO26_0425), binding-protein-dependent transport system inner membrane protein (BCO26_0553), NMT1/THI5 like domain-containing protein (BCO26_0554), ABC transporter-like protein (BCO26_0555), periplasmic solute binding protein (BCO26_2084) and ABC transporter (BCO26_2523), were significantly upregulated in both the CA and NA groups compared with those in the GY group. Four genes, namely, phosphate binding protein (BCO26_0705), phosphate ABC transporter permease (BCO26_0706, BCO26_0707) and ABC transporter-like protein (BCO26_0743), were significantly up-regulated in CA but not in NA compared with GY. Eight genes, namely binding-protein-dependent transport system inner membrane protein (BCO26_0422), glycine betaine ABC transporter substrate-binding protein (BCO26_0423), binding-protein-dependent transport system inner membrane protein (BCO26_0424), transport system permease (BCO26_0685, BCO26_0686), ABC transporter-like protein (BCO26_0687), periplasmic binding protein (BCO26_0688) and ABC transporter-like protein (BCO26_2588), were significantly upregulated in NA but not in CA compared with GY. The significantly down-regulated genes involved in ‘ABC transporters’ under the two kinds of lactate stress were mostly the same (<a href="#pone.0124316.s002">S2 Table</a> and <a href="#pone.0124316.s003">S3 Table</a>). However, sodium lactate induced more significantly down-regulated genes than calcium lactate (35 to 27) and higher average down-regulation (fold change ranging from 3.72 to 3.06).</p>
<a class="link-target" id="pone-0124316-t002" name="pone-0124316-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t002" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Significantly upregulated genes involved in ‘ABC transporters’ under calcium lactate stress.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t002">
              https://doi.org/10.1371/journal.pone.0124316.t002</a></p></div><a class="link-target" id="pone-0124316-t003" name="pone-0124316-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t003" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Significantly upregulated genes involved in ‘ABC transporters’ under sodium lactate stress.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t003">
              https://doi.org/10.1371/journal.pone.0124316.t003</a></p></div></div>

<div id="section7" class="section toc-section"><a id="sec014" name="sec014" class="link-target" title="Identification and comparison of DEGs involved in ‘amino sugar and nucleotide sugar metabolism’"></a>
<h3>Identification and comparison of DEGs involved in ‘amino sugar and nucleotide sugar metabolism’</h3>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1" class="link-target"></a><p>In nucleotide sugar metabolism, nucleotide sugars act as donors for sugar residues in the glycosylation reactions that produce polysaccharides, which are important constituents of the cell wall [<a href="#pone.0124316.ref023" class="ref-tip">23</a>]. Nucleotide sugars are also intermediates in nucleotide sugar interconversions, which produce some of the activated sugars necessary for glycosylation reactions [<a href="#pone.0124316.ref024" class="ref-tip">24</a>]. KEGG pathway enrichment analysis revealed that ‘amino sugar and nucleotide sugar metabolism’ was significantly affected by sodium lactate stress but not by calcium lactate stress. Fifteen significantly upregulated genes (<a href="#pone-0124316-t004">Table 4</a>) and 11 significantly down-regulated genes (<a href="#pone.0124316.s004">S4 Table</a>) were involved in ‘amino sugar and nucleotide sugar metabolism’ under sodium lactate stress condition (NA group) compared with those under non-stress condition (GY group). Eight significantly upregulated genes and one significantly down-regulated gene were involved in ‘amino sugar and nucleotide sugar metabolism’ under calcium lactate stress condition (CA group) compared with those under non-stress condition (GY group) (<a href="#pone-0124316-t005">Table 5</a>). Seven significantly regulated genes were found to exhibit the same trend under the two types of lactate stress conditions, namely, NAD(P)(+)-protein-arginine ADP-ribosyltransferase (BCO26_0186), glucosamine/fructose-6-phosphate aminotransferase (BCO26_0255), galactokinase (BCO26_0622), UDP-glucose 4-epimerase (BCO26_0623), N-acetylglucosamine-6-phosphate deacetylase (BCO26_0635), glucosamine-6-phosphate isomerise (BCO26_0636) and N-acylglucosamine-6-phosphate 2-epimerase (BCO26_2463). Eight genes were significantly upregulated specifically under sodium lactate stress, namely, UDP-N-acetylglucosamine pyrophosphorylase (BCO26_0036), phosphoglucosamine mutase (BCO26_0177), PTS system glucose subfamily transporter subunit IIA (BCO26_0204), phosphoglucomutase/phosphomannomutase alpha/beta/alpha domain I (BCO26_0507), galactose-1-phosphate uridylyltransferase (BCO26_0624), UDP-N-acetylenolpyruvoylglucosamine reductase (BCO26_1444), ROK family glucokinase (BCO26_1699) and UDP-N-acetylglucosamine 1-carboxyvinyltransferase (BCO26_2818). More genes were significantly down-regulated under sodium lactate stress than under calcium lactate stress (11 to 1). The four subunits of phosphoenolpyruvate-dependent sugar phosphotransferase system (PTS) mannose/fructose/sorbose family transporter (i.e., BCO26_0540, BCO26_0541, BCO26_0542 and BCO26_0543) were significantly down-regulated under sodium lactate stress (<a href="#pone.0124316.s004">S4 Table</a>), whereas the PTS system glucose subfamily transporter subunit IIA (BCO26_0204) was significantly upregulated (<a href="#pone-0124316-t004">Table 4</a>).</p>
<a class="link-target" id="pone-0124316-t004" name="pone-0124316-t004"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t004" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 4. </span> Significantly upregulated genes involved in ‘amino sugar and nucleotide sugar metabolism’ under sodium lactate stress.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t004">
              https://doi.org/10.1371/journal.pone.0124316.t004</a></p></div><a class="link-target" id="pone-0124316-t005" name="pone-0124316-t005"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t005" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 5. </span> Significantly regulated genes involved in ‘amino sugar and nucleotide sugar metabolism’ under calcium lactate stress.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t005">
              https://doi.org/10.1371/journal.pone.0124316.t005</a></p></div></div>

<div id="section8" class="section toc-section"><a id="sec015" name="sec015" class="link-target" title="Identification and comparison of DEGs involved in ‘glycolysis/gluconeogenesis’"></a>
<h3>Identification and comparison of DEGs involved in ‘glycolysis/gluconeogenesis’</h3>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1" class="link-target"></a><p>‘Glycolysis/gluconeogenesis’ is the key pathway for lactic acid production in <em>B</em>. <em>coagulans</em> 2–6. Therefore, we compared DEGs involved in ‘glycolysis/gluconeogenesis’ under lactate stress and non-stress conditions. Thirteen significantly upregulated genes (<a href="#pone-0124316-t006">Table 6</a>) and eight significantly down-regulated genes (<a href="#pone.0124316.s005">S5 Table</a>) were involved in ‘glycolysis/gluconeogenesis’ under sodium lactate stress condition (NA group) compared with those under non-stress condition (GY group). The pyruvate dehydrogenase complex, which mediates the conversion of pyruvic acid to acetyl-CoA, was significantly upregulated under sodium lactate stress compared with non-stress condition. The complex consists of pyruvate dehydrogenase components (BCO26_0962 and BCO26_0963), dihydrolipoamide transacetylase (BCO26_0964) and dihydrolipoamide dehydrogenase (BCO26_0965 and BCO26_1409).</p>
<a class="link-target" id="pone-0124316-t006" name="pone-0124316-t006"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t006" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 6. </span> Significantly upregulated genes involved in ‘glycolysis/gluconeogenesis’ under sodium lactate stress.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t006">
              https://doi.org/10.1371/journal.pone.0124316.t006</a></p></div><a id="article1.body1.sec3.sec8.p2" name="article1.body1.sec3.sec8.p2" class="link-target"></a><p>Eleven significantly upregulated genes (<a href="#pone-0124316-t007">Table 7</a>) and five significantly down-regulated genes (<a href="#pone.0124316.s006">S6 Table</a>) were involved in ‘glycolysis/gluconeogenesis’ under calcium lactate stress condition (CA group) compared with those under non-stress condition (GY group) (<a href="#pone-0124316-t007">Table 7</a>). The key genes of the glycolysis pathway, including L-lactate dehydrogenase (BCO26_0531), pyruvate kinase (BCO26_1967), 6-phosphofructokinase (BCO26_1968), enolase (BCO26_2235), phosphoglycerate mutase (BCO26_2237), phosphoglycerate kinase (BCO26_2239) and fructose-bisphosphate aldolase (BCO26_2530), were all significantly upregulated (<a href="#pone-0124316-t007">Table 7</a>).</p>
<a class="link-target" id="pone-0124316-t007" name="pone-0124316-t007"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t007" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 7. </span> Significantly upregulated genes involved in ‘glycolysis/gluconeogenesis’ under calcium lactate stress.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t007">
              https://doi.org/10.1371/journal.pone.0124316.t007</a></p></div><a id="article1.body1.sec3.sec8.p3" name="article1.body1.sec3.sec8.p3" class="link-target"></a><p>Under the two types of lactate stress conditions, two significantly upregulated genes involved in ‘glycolysis/gluconeogenesis’ were found, namely, alcohol dehydrogenase GroES domain-containing protein (BCO26_1901) and enolase (BCO26_2235). Four key genes of glycolysis (i.e., iron-containing alcohol dehydrogenase, lactate/malate dehydrogenase, AMP-dependent synthetase and ligase and hypothetical protein BCO26_2297) were significantly down-regulated both in CA and NA compared with those in GY. Pyruvate kinase (BCO26_1967) was significantly down-regulated in NA but upregulated in CA compared with that in GY.</p>
</div>

<div id="section9" class="section toc-section"><a id="sec016" name="sec016" class="link-target" title="Identification and comparison of DEGs involved in ‘citrate cycle (TCA cycle)’"></a>
<h3>Identification and comparison of DEGs involved in ‘citrate cycle (TCA cycle)’</h3>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1" class="link-target"></a><p>Analysis of the genome of <em>B</em>. <em>coagulans</em> 2–6 revealed that the strain possessed all components of the TCA cycle. The TCA cycle is an important aerobic pathway in the final steps of the oxidation of carbohydrates and fatty acids. As a critical pathway for energy production in cells, it may be beneficial for <em>B</em>. <em>coagulans</em> strains to resist lactate stress conditions. Our previous study indicated that trace amounts of aeration enhance the resistance of <em>B</em>. <em>coagulans</em> to sodium lactate [<a href="#pone.0124316.ref005" class="ref-tip">5</a>]. Therefore, we compared DEGs involved in ‘citrate cycle (TCA cycle)’ under lactate stress and non-stress conditions.</p>
<a id="article1.body1.sec3.sec9.p2" name="article1.body1.sec3.sec9.p2" class="link-target"></a><p>Seven significantly upregulated genes and seven significantly down-regulated genes were involved in ‘citrate cycle (TCA cycle)’ under sodium lactate stress condition (NA group) compared with those under non-stress condition (GY group) (<a href="#pone-0124316-t008">Table 8</a>). The pyruvate dehydrogenase complex (BCO26_0962, BCO26_0963, BCO26_0964, BCO26_0965 and BCO26_1409) was significantly upregulated as described above. Aconitate hydratase 1 (BCO26_1352), which responds to the conversion of citrate to isocitrate, and class II fumarate hydratase (BCO26_0741), which responds to the conversion of fumarate to L-malate, were also significantly upregulated. Succinyl-CoA synthetase (BCO26_1117 and BCO26_1118), 2-oxoglutarate dehydrogenase (BCO26_1275 and BCO26_1276), dihydrolipoamide dehydrogenase (BCO26_1640), succinate dehydrogenase (or fumarate reductase) cytochrome b subunit, b558 family (BCO26_1913) and isocitrate dehydrogenase (BCO26_1963) were all significantly down-regulated under sodium lactate stress.</p>
<a class="link-target" id="pone-0124316-t008" name="pone-0124316-t008"></a><div class="figure" data-doi="10.1371/journal.pone.0124316.t008"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0124316.t008" data-doi="info:doi/10.1371/journal.pone.0124316" data-uri="info:doi/10.1371/journal.pone.0124316.t008"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0124316.t008" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t008">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0124316.t008">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t008">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0124316.t008">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t008">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0124316.t008">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 8. </span> Significantly regulated genes involved in ‘citrate cycle (TCA cycle)’ under lactate stress.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0124316.t008">
              https://doi.org/10.1371/journal.pone.0124316.t008</a></p></div><a id="article1.body1.sec3.sec9.p3" name="article1.body1.sec3.sec9.p3" class="link-target"></a><p>Only one ‘citrate cycle (TCA cycle)’ gene was found to be significantly upregulated under calcium lactate stress condition (CA group) compared with non-stress condition (GY group) (<a href="#pone-0124316-t008">Table 8</a>). This gene was class II fumarate hydratase (BCO26_0741), which was also expressed at higher level under sodium lactate stress compared to non-stress condition. Therefore, the TCA cycle of <em>B</em>. <em>coagulans</em> 2–6 was less affected by calcium lactate stress compared to sodium lactate stress.</p>
</div>

<div id="section10" class="section toc-section"><a id="sec017" name="sec017" class="link-target" title="Identification of genes with at least five-fold up-regulation under lactate stress"></a>
<h3>Identification of genes with at least five-fold up-regulation under lactate stress</h3>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1" class="link-target"></a><p>The genes with at least five-fold up-regulation under lactate stress were identified because they are potential targets for genetic engineering to further improve <em>B</em>. <em>coagulans</em> as a producer of L-lactic acid. Under sodium lactate stress condition (NA group), 28 genes were upregulated greater than fivefold (<a href="#pone.0124316.s007">S7 Table</a>).</p>
<a id="article1.body1.sec3.sec10.p2" name="article1.body1.sec3.sec10.p2" class="link-target"></a><p>Only three genes were upregulated by greater than fivefold under calcium lactate stress condition (CA group) (<a href="#pone.0124316.s008">S8 Table</a>). Lambda repressor-like DNA-binding domain-containing protein (BCO26_0364) and heat shock protein Hsp20 (BCO26_1778) were both upregulated under the two types of lactate stress. The function of small heat shock proteins is to mediate correct protein folding and coping with stress-induced cell damage (Guzzo 2012). Two of the three small heat shock proteins Hsp20 in <em>B</em>. <em>coagulans</em> 2–6 were significantly upregualted under the two types of lactate stress. Major facilitator superfamily permease (BCO26_0524) was specifically upregulated greater than fivefold under calcium lactate stress (<a href="#pone.0124316.s008">S8 Table</a>). Five genes were specifically upregulated greater than six-fold under sodium lactate stress, namely, hypothetical protein (BCO26_0739), ATPase AAA-2 domain-containing protein (BCO26_0934), hypothetical protein (BCO26_0740), copper ion binding protein (BCO26_2102) and hypothetical protein (BCO26_2103) (<a href="#pone.0124316.s007">S7 Table</a>).</p>
</div>

<div id="section11" class="section toc-section"><a id="sec018" name="sec018" class="link-target" title="Verification of RNA-seq data"></a>
<h3>Verification of RNA-seq data</h3>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1" class="link-target"></a><p>Reliability of the RNA-seq data was assessed by qRT-PCR analysis. In total, 5 genes were selected for validation. Although the magnitude of fold-change in gene expression varied between the two data sets, similar trends were observed in qRT-PCR and RNA-seq results, which supported the validity of the RNA-seq data (<a href="#pone.0124316.s009">S9 Table</a>).</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="sec019" name="sec019" data-toc="sec019" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p><em>B</em>. <em>coagulans</em> strains have attracted significant attention for L-lactic acid production in the past few years [<a href="#pone.0124316.ref005" class="ref-tip">5</a>, <a href="#pone.0124316.ref011" class="ref-tip">11</a>, <a href="#pone.0124316.ref025" class="ref-tip">25</a>–<a href="#pone.0124316.ref028" class="ref-tip">28</a>]. Most of the research on this strain has focused on fermentation process. Qin et al. [<a href="#pone.0124316.ref011" class="ref-tip">11</a>] and Ye et al. [<a href="#pone.0124316.ref028" class="ref-tip">28</a>] reported that Ca(OH)<sub>2</sub> and CaCO<sub>3</sub> are more suitable neutralizing agents than NaOH for getting higher lactic acid titer. They presumed that it was attributable to lower product inhibition by calcium lactate compared to sodium lactate. However, little is known about the molecular mechanisms by which <em>B</em>. <em>coagulans</em> responds to different lactates.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>In this study, the gene expression patterns of <em>B</em>. <em>coagulans</em> 2–6 responding to sodium lactate and calcium lactate were investigated by RNA-seq and comparative transcriptome analysis. Gene ontology assignments and KEGG pathway enrichment analysis of the DEGs were performed. It was found that the influence of sodium lactate on gene expression in <em>B</em>. <em>coagulans</em> 2–6 was greater than that of calcium lactate. Sodium lactate induced significant upregulation of 543 genes and down-regulation of 670 genes, while calcium lactate induced significant upregulation of 398 genes and down-regulation of 314 genes. It was also found that lactate fermentation was less affected by calcium lactate stress than by sodium lactate stress. Key genes of the glycolysis pathway were most significantly upregulated under calcium lactate stress, while L-lactate dehydrogenase (–1.52-fold), pyruvate kinase (–3.98-fold) and 6-phosphofructokinase (–1.41-fold) were down-regulated under sodium lactate stress. Pyruvate dehydrogenase complex, which converts pyruvic acid to acetyl-CoA, was significantly upregulated (2.8-fold) under sodium lactate stress. These changes are expected to have significant influence on the formation of lactic acid from pyruvic acid. Taken together, our findings provide new mechanistic insights into higher lactic acid titer obtained using CaCO<sub>3</sub> as neutralizing agent compared to that obtained using NaOH [<a href="#pone.0124316.ref005" class="ref-tip">5</a>, <a href="#pone.0124316.ref011" class="ref-tip">11</a>].</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>Our results also provide potential targets for genetic engineering to further improve <em>B</em>. <em>coagulans</em> as an L-lactic acid producer. ABC transporters play important roles in response to lactate stress. A large number of the ABC transporters in <em>B</em>. <em>coagulans</em> 2–6 were found to be significantly down-regulated under the two types of lactate stress. We also identified ABC transporters that were significantly upregulated under the two types of lactates stress or specifically upregulated by one lactate. High expression of ABC transporter genes may be of benefit to <em>B</em>. <em>coagulans</em> 2–6 to maintain intercellular homeostasis; as such, these genes are good potential targets for genetic engineering. Furthermore, we identified genes with at least five-fold upregulation under lactate stress. Some of them were well-known stress-induced proteins, such as heat shock protein Hsp20 [<a href="#pone.0124316.ref029" class="ref-tip">29</a>] and major facilitator superfamily protein [<a href="#pone.0124316.ref030" class="ref-tip">30</a>], while others were previously unknown proteins, whose functions could be characterized in future studies.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Wang et al. [<a href="#pone.0124316.ref012" class="ref-tip">12</a>] carried out a comparative proteomic analysis of <em>B</em>. <em>coagulans</em> in response to lactate stress. Using two-dimensional electrophoresis coupled with mass spectrometric identification, they identified four highly expressed proteins under calcium lactate stress. The same four proteins were also identified in this study, which indicates that our results are consistent with previous findings on this topic. In addition, we found that expressions of lactate dehydrogenase (ACR02673), cysteine synthase A (AEH52120) and ribosomal protein L7/L12 (AEH52167) were upregulated under calcium lactate stress but down-regulated under sodium lactate stress. Gene expression of aldo/keto reductase (AEH52329) was upregulated under calcium lactate stress and sodium lactate stress. Thus, our RNA-seq dataset provides a rich account of gene expression changes in <em>B</em>. <em>coagulans</em> in response to lactate stress, which could be highly useful for future efforts to improve lactic acid production by genetic engineering.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec020" name="sec020" data-toc="sec020" class="link-target" title="Conclusions"></a><h2>Conclusions</h2><a id="article1.body1.sec5.p1" name="article1.body1.sec5.p1" class="link-target"></a><p>In summary, RNA-seq technology was employed in this study to characterise the transcriptome of <em>B</em>. <em>coagulans</em> under non-stress, sodium lactate stress and calcium lactate stress conditions. The responses of <em>B</em>. <em>coagulans</em> 2–6 to sodium lactate stress include: more genes changed expression than that in response to calcium lactate stress, significant regulation of ‘amino sugar and nucleotide sugar metabolism’ genes and negative influence on ‘glycolysis/gluconeogenesis’ and ‘citrate cycle (TCA cycle)’ genes. The responses of <em>B</em>. <em>coagulans</em> 2–6 to calcium lactate stress include: less genes changed expression than that in response to sodium lactate stress, significant regulation of ‘ABC transporters’ genes and positive influence on ‘glycolysis/gluconeogenesis’ genes. These results provide new insights into the responses of <em>B</em>. <em>coagulans</em> to different lactate stresses. Notably, our findings not only provide a robust database for investigating the functions of genes induced by lactate stress but also identify potential targets for genetic engineering to further improve L-lactic acid production by <em>B</em>. <em>coagulans</em>.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec021" name="sec021" data-toc="sec021" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0124316"></div><div class="supplementary-material"><a name="pone.0124316.s001" id="pone.0124316.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s001">S1 Table. </a>The primers for qRT-PCR analysis with target gene information.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s001">https://doi.org/10.1371/journal.pone.0124316.s001</a></p><a id="article1.body1.sec6.supplementary-material1.caption1.p1" name="article1.body1.sec6.supplementary-material1.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s002" id="pone.0124316.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s002">S2 Table. </a>Significantly down-regulated genes involved in ‘ABC transporters’ under calcium lactate stress.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s002">https://doi.org/10.1371/journal.pone.0124316.s002</a></p><a id="article1.body1.sec6.supplementary-material2.caption1.p1" name="article1.body1.sec6.supplementary-material2.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s003" id="pone.0124316.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s003">S3 Table. </a>Significantly down-regulated genes involved in ‘ABC transporters’ under sodium lactate stress.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s003">https://doi.org/10.1371/journal.pone.0124316.s003</a></p><a id="article1.body1.sec6.supplementary-material3.caption1.p1" name="article1.body1.sec6.supplementary-material3.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s004" id="pone.0124316.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s004">S4 Table. </a>Significantly down-regulated genes involved in ‘amino sugar and nucleotide sugar metabolism’ under sodium lactate stress.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s004">https://doi.org/10.1371/journal.pone.0124316.s004</a></p><a id="article1.body1.sec6.supplementary-material4.caption1.p1" name="article1.body1.sec6.supplementary-material4.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s005" id="pone.0124316.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s005">S5 Table. </a>Significantly down-regulated genes involved in ‘glycolysis/gluconeogenesis’ under sodium lactate stress.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s005">https://doi.org/10.1371/journal.pone.0124316.s005</a></p><a id="article1.body1.sec6.supplementary-material5.caption1.p1" name="article1.body1.sec6.supplementary-material5.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s006" id="pone.0124316.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s006">S6 Table. </a>Significantly down-regulated genes involved in ‘glycolysis/gluconeogenesis’ under calcium lactate stress.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s006">https://doi.org/10.1371/journal.pone.0124316.s006</a></p><a id="article1.body1.sec6.supplementary-material6.caption1.p1" name="article1.body1.sec6.supplementary-material6.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s007" id="pone.0124316.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s007">S7 Table. </a>Genes with at least fivefold upregulation under sodium lactate stress.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s007">https://doi.org/10.1371/journal.pone.0124316.s007</a></p><a id="article1.body1.sec6.supplementary-material7.caption1.p1" name="article1.body1.sec6.supplementary-material7.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s008" id="pone.0124316.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s008">S8 Table. </a>Genes with at least fivefold upregulation under calcium lactate stress.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s008">https://doi.org/10.1371/journal.pone.0124316.s008</a></p><a id="article1.body1.sec6.supplementary-material8.caption1.p1" name="article1.body1.sec6.supplementary-material8.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s009" id="pone.0124316.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s009">S9 Table. </a>The expression information of selected genes for qRT-PCR under lactate stress condition.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s009">https://doi.org/10.1371/journal.pone.0124316.s009</a></p><a id="article1.body1.sec6.supplementary-material9.caption1.p1" name="article1.body1.sec6.supplementary-material9.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s010" id="pone.0124316.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s010">S1 Data. </a>GO assignments of differentially expressed genes (CA vs GY).</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s010">https://doi.org/10.1371/journal.pone.0124316.s010</a></p><a id="article1.body1.sec6.supplementary-material10.caption1.p1" name="article1.body1.sec6.supplementary-material10.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0124316.s011" id="pone.0124316.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0124316.s011">S2 Data. </a>GO assignments of differentially expressed genes (NA vs GY).</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0124316.s011">https://doi.org/10.1371/journal.pone.0124316.s011</a></p><a id="article1.body1.sec6.supplementary-material11.caption1.p1" name="article1.body1.sec6.supplementary-material11.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div></div>



<div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: JQ. Performed the experiments: JQ XW LW BZ. Analyzed the data: JQ XZ QY. Contributed reagents/materials/analysis tools: JQ. Wrote the paper: JQ PX.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0124316.ref001" id="pone.0124316.ref001" class="link-target"></a>Datta R, Henry M. Lactic acid: recent advances in products, processes and technologies—a review. J Chem Technol Biotechnol. 2006;81: 1119–1129. <ul class="reflinks"><li><a href="#" data-author="Datta" data-cit="DattaR%2C%20HenryM.%20Lactic%20acid%3A%20recent%20advances%20in%20products%2C%20processes%20and%20technologies%E2%80%94a%20review.%20J%20Chem%20Technol%20Biotechnol.%202006%3B81%3A%201119%E2%80%931129." data-title="Lactic%20acid%3A%20recent%20advances%20in%20products%2C%20processes%20and%20technologies%E2%80%94a%20review" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Lactic+acid%3A+recent+advances+in+products%2C+processes+and+technologies%E2%80%94a+review+Datta+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0124316.ref002" id="pone.0124316.ref002" class="link-target"></a>John RP, Nampoothiri KM, Pandey A. Fermentative production of lactic acid from biomass: an overview on process developments and future perspectives. Appl Microbiol Biotechnol. 2007;74: 524–534. pmid:17225102 <ul class="reflinks"><li><a href="#" data-author="John" data-cit="JohnRP%2C%20NampoothiriKM%2C%20PandeyA.%20Fermentative%20production%20of%20lactic%20acid%20from%20biomass%3A%20an%20overview%20on%20process%20developments%20and%20future%20perspectives.%20Appl%20Microbiol%20Biotechnol.%202007%3B74%3A%20524%E2%80%93534.%2017225102" data-title="Fermentative%20production%20of%20lactic%20acid%20from%20biomass%3A%20an%20overview%20on%20process%20developments%20and%20future%20perspectives" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17225102" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Fermentative+production+of+lactic+acid+from+biomass%3A+an+overview+on+process+developments+and+future+perspectives+John+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0124316.ref003" id="pone.0124316.ref003" class="link-target"></a>Okano K, Tanaka T, Ogino C, Fukuda H, Kondo A. Biotechnological production of enantiomeric pure lactic acid from renewable resources—recent achievements, perspectives, and limits. Appl Microbiol Biotechnol. 2010;85: 413–423.  pmid:19826806 <ul class="reflinks"><li><a href="#" data-author="Okano" data-cit="OkanoK%2C%20TanakaT%2C%20OginoC%2C%20FukudaH%2C%20KondoA.%20Biotechnological%20production%20of%20enantiomeric%20pure%20lactic%20acid%20from%20renewable%20resources%E2%80%94recent%20achievements%2C%20perspectives%2C%20and%20limits.%20Appl%20Microbiol%20Biotechnol.%202010%3B85%3A%20413%E2%80%93423.%20doi%3A%2010.1007%2Fs00253-009-2280-5%2019826806" data-title="Biotechnological%20production%20of%20enantiomeric%20pure%20lactic%20acid%20from%20renewable%20resources%E2%80%94recent%20achievements%2C%20perspectives%2C%20and%20limits" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19826806" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Biotechnological+production+of+enantiomeric+pure+lactic+acid+from+renewable+resources%E2%80%94recent+achievements%2C+perspectives%2C+and+limits+Okano+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0124316.ref004" id="pone.0124316.ref004" class="link-target"></a>Corma A, Iborra S, Velty A. Chemical routes for the transformation of biomass into chemicals. Chem Rev. 2007;107: 2411–2502. pmid:17535020 <ul class="reflinks"><li><a href="#" data-author="Corma" data-cit="CormaA%2C%20IborraS%2C%20VeltyA.%20Chemical%20routes%20for%20the%20transformation%20of%20biomass%20into%20chemicals.%20Chem%20Rev.%202007%3B107%3A%202411%E2%80%932502.%2017535020" data-title="Chemical%20routes%20for%20the%20transformation%20of%20biomass%20into%20chemicals" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17535020" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chemical+routes+for+the+transformation+of+biomass+into+chemicals+Corma+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0124316.ref005" id="pone.0124316.ref005" class="link-target"></a>Qin J, Wang X, Zheng Z, Ma C, Tang H, Xu P. Production of L-lactic acid by a thermophilic <em>Bacillus</em> mutant using sodium hydroxide as neutralizing agent. Bioresour Technol. 2010;101: 7570–7576.  pmid:20488697 <ul class="reflinks"><li><a href="#" data-author="Qin" data-cit="QinJ%2C%20WangX%2C%20ZhengZ%2C%20MaC%2C%20TangH%2C%20XuP.%20Production%20of%20L-lactic%20acid%20by%20a%20thermophilic%20Bacillus%20mutant%20using%20sodium%20hydroxide%20as%20neutralizing%20agent.%20Bioresour%20Technol.%202010%3B101%3A%207570%E2%80%937576.%20doi%3A%2010.1016%2Fj.biortech.2010.04.037%2020488697" data-title="Production%20of%20L-lactic%20acid%20by%20a%20thermophilic%20Bacillus%20mutant%20using%20sodium%20hydroxide%20as%20neutralizing%20agent" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20488697" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Production+of+L-lactic+acid+by+a+thermophilic+Bacillus+mutant+using+sodium+hydroxide+as+neutralizing+agent+Qin+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0124316.ref006" id="pone.0124316.ref006" class="link-target"></a>Bai DM, Wei Q, Yan ZH, Zhao XM, Li XG, Xu SM. Fed-batch fermentation of <em>Lactobacillus lactis</em> for hyper-production of L-lactic acid. Biotechnol Lett. 2003;25: 1833–1835. pmid:14677707 <ul class="reflinks"><li><a href="#" data-author="Bai" data-cit="BaiDM%2C%20WeiQ%2C%20YanZH%2C%20ZhaoXM%2C%20LiXG%2C%20XuSM.%20Fed-batch%20fermentation%20of%20Lactobacillus%20lactis%20for%20hyper-production%20of%20L-lactic%20acid.%20Biotechnol%20Lett.%202003%3B25%3A%201833%E2%80%931835.%2014677707" data-title="Fed-batch%20fermentation%20of%20Lactobacillus%20lactis%20for%20hyper-production%20of%20L-lactic%20acid" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14677707" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Fed-batch+fermentation+of+Lactobacillus+lactis+for+hyper-production+of+L-lactic+acid+Bai+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0124316.ref007" id="pone.0124316.ref007" class="link-target"></a>Ding S, Tan T. L-Lactic acid production by <em>Lactobacillus casei</em> fermentation using different fed-batch feeding strategies. Proc Biochem. 2006;41: 1451–1454. <ul class="reflinks"><li><a href="#" data-author="Ding" data-cit="DingS%2C%20TanT.%20L-Lactic%20acid%20production%20by%20Lactobacillus%20casei%20fermentation%20using%20different%20fed-batch%20feeding%20strategies.%20Proc%20Biochem.%202006%3B41%3A%201451%E2%80%931454." data-title="L-Lactic%20acid%20production%20by%20Lactobacillus%20casei%20fermentation%20using%20different%20fed-batch%20feeding%20strategies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=L-Lactic+acid+production+by+Lactobacillus+casei+fermentation+using+different+fed-batch+feeding+strategies+Ding+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0124316.ref008" id="pone.0124316.ref008" class="link-target"></a>Saitoh S, Ishida N, Onishi T, Tokuhiro K, Nagamori E, Kitamoto K, et al. Genetically engineered wine yeast produces a high concentration of L-lactic acid of extremely high optical purity. Appl Environ Microbiol. 2005;71: 2789–2792. pmid:15870375 <ul class="reflinks"><li><a href="#" data-author="Saitoh" data-cit="SaitohS%2C%20IshidaN%2C%20OnishiT%2C%20TokuhiroK%2C%20NagamoriE%2C%20KitamotoK%2C%20et%20al.%20Genetically%20engineered%20wine%20yeast%20produces%20a%20high%20concentration%20of%20L-lactic%20acid%20of%20extremely%20high%20optical%20purity.%20Appl%20Environ%20Microbiol.%202005%3B71%3A%202789%E2%80%932792.%2015870375" data-title="Genetically%20engineered%20wine%20yeast%20produces%20a%20high%20concentration%20of%20L-lactic%20acid%20of%20extremely%20high%20optical%20purity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15870375" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genetically+engineered+wine+yeast+produces+a+high+concentration+of+L-lactic+acid+of+extremely+high+optical+purity+Saitoh+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0124316.ref009" id="pone.0124316.ref009" class="link-target"></a>Budhavaram NK, Fan Z. Production of lactic acid from paper sludge using acid-tolerant, thermophilic <em>Bacillus coagulan</em> strains. Bioresour Technol. 2009;100: 5966–5972.  pmid:19577925 <ul class="reflinks"><li><a href="#" data-author="Budhavaram" data-cit="BudhavaramNK%2C%20FanZ.%20Production%20of%20lactic%20acid%20from%20paper%20sludge%20using%20acid-tolerant%2C%20thermophilic%20Bacillus%20coagulan%20strains.%20Bioresour%20Technol.%202009%3B100%3A%205966%E2%80%935972.%20doi%3A%2010.1016%2Fj.biortech.2009.01.080%2019577925" data-title="Production%20of%20lactic%20acid%20from%20paper%20sludge%20using%20acid-tolerant%2C%20thermophilic%20Bacillus%20coagulan%20strains" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19577925" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Production+of+lactic+acid+from+paper+sludge+using+acid-tolerant%2C+thermophilic+Bacillus+coagulan+strains+Budhavaram+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0124316.ref010" id="pone.0124316.ref010" class="link-target"></a>Michelson T, Kask K, Jõgi E, Talpsep E, Suitso I, Nurk A. L(+)-Lactic acid producer <em>Bacillus coagulans</em> SIM-7 DSM 14043 and its comparison with <em>Lactobacillus delbrueckii</em> ssp. <em>lactis</em> DSM 20073. Enzyme Microb Technol. 2006;39: 861–867. <ul class="reflinks"><li><a href="#" data-author="Michelson" data-cit="MichelsonT%2C%20KaskK%2C%20J%C3%B5giE%2C%20TalpsepE%2C%20SuitsoI%2C%20NurkA.%20L%28%2B%29-Lactic%20acid%20producer%20Bacillus%20coagulans%20SIM-7%20DSM%2014043%20and%20its%20comparison%20with%20Lactobacillus%20delbrueckii%20ssp.%20lactis%20DSM%2020073.%20Enzyme%20Microb%20Technol.%202006%3B39%3A%20861%E2%80%93867." data-title="L%28%2B%29-Lactic%20acid%20producer%20Bacillus%20coagulans%20SIM-7%20DSM%2014043%20and%20its%20comparison%20with%20Lactobacillus%20delbrueckii%20ssp.%20lactis%20DSM%2020073" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=L%28%2B%29-Lactic+acid+producer+Bacillus+coagulans+SIM-7+DSM+14043+and+its+comparison+with+Lactobacillus+delbrueckii+ssp.+lactis+DSM+20073+Michelson+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0124316.ref011" id="pone.0124316.ref011" class="link-target"></a>Qin J, Zhao B, Wang X, Wang L, Yu B, Ma Y, et al. Non-sterilized fermentative production of polymer-grade L-lactic acid by a newly isolated thermophilic strain <em>Bacillus</em> sp. 2–6. PloS ONE. 2009;4: e4359.  pmid:19194504 <ul class="reflinks"><li><a href="#" data-author="Qin" data-cit="QinJ%2C%20ZhaoB%2C%20WangX%2C%20WangL%2C%20YuB%2C%20MaY%2C%20et%20al.%20Non-sterilized%20fermentative%20production%20of%20polymer-grade%20L-lactic%20acid%20by%20a%20newly%20isolated%20thermophilic%20strain%20Bacillus%20sp.%202%E2%80%936.%20PloS%20ONE.%202009%3B4%3A%20e4359.%20doi%3A%2010.1371%2Fjournal.pone.0004359%2019194504" data-title="Non-sterilized%20fermentative%20production%20of%20polymer-grade%20L-lactic%20acid%20by%20a%20newly%20isolated%20thermophilic%20strain%20Bacillus%20sp.%202%E2%80%936" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19194504" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Non-sterilized+fermentative+production+of+polymer-grade+L-lactic+acid+by+a+newly+isolated+thermophilic+strain+Bacillus+sp.+2%E2%80%936+Qin+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0124316.ref012" id="pone.0124316.ref012" class="link-target"></a>Wang X, Qin J, Wang L, Xu P. A comparative proteomic analysis of <em>Bacillus coagulans</em> in response to lactate stress during the production of L-lactic acid. Biotechnol Lett. 2014;36: 2545–2549.  pmid:25214213 <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangX%2C%20QinJ%2C%20WangL%2C%20XuP.%20A%20comparative%20proteomic%20analysis%20of%20Bacillus%20coagulans%20in%20response%20to%20lactate%20stress%20during%20the%20production%20of%20L-lactic%20acid.%20Biotechnol%20Lett.%202014%3B36%3A%202545%E2%80%932549.%20doi%3A%2010.1007%2Fs10529-014-1639-6%2025214213" data-title="A%20comparative%20proteomic%20analysis%20of%20Bacillus%20coagulans%20in%20response%20to%20lactate%20stress%20during%20the%20production%20of%20L-lactic%20acid" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25214213" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+comparative+proteomic+analysis+of+Bacillus+coagulans+in+response+to+lactate+stress+during+the+production+of+L-lactic+acid+Wang+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0124316.ref013" id="pone.0124316.ref013" class="link-target"></a>Su F, Yu B, Sun J, Ou HY, Zhao B, Wang L, et al. Genome sequence of the thermophilic strain <em>Bacillus coagulans</em> 2–6, an efficient producer of high-optical-purity L-lactic acid. J Bacteriol. 2011;193: 4563–4564.  pmid:21705584 <ul class="reflinks"><li><a href="#" data-author="Su" data-cit="SuF%2C%20YuB%2C%20SunJ%2C%20OuHY%2C%20ZhaoB%2C%20WangL%2C%20et%20al.%20Genome%20sequence%20of%20the%20thermophilic%20strain%20Bacillus%20coagulans%202%E2%80%936%2C%20an%20efficient%20producer%20of%20high-optical-purity%20L-lactic%20acid.%20J%20Bacteriol.%202011%3B193%3A%204563%E2%80%934564.%20doi%3A%2010.1128%2FJB.05378-11%2021705584" data-title="Genome%20sequence%20of%20the%20thermophilic%20strain%20Bacillus%20coagulans%202%E2%80%936%2C%20an%20efficient%20producer%20of%20high-optical-purity%20L-lactic%20acid" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21705584" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genome+sequence+of+the+thermophilic+strain+Bacillus+coagulans+2%E2%80%936%2C+an+efficient+producer+of+high-optical-purity+L-lactic+acid+Su+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0124316.ref014" id="pone.0124316.ref014" class="link-target"></a>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2010;9: 357–359. <ul class="reflinks"><li><a href="#" data-author="Langmead" data-cit="LangmeadB%2C%20SalzbergSL.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat%20Methods.%202010%3B9%3A%20357%E2%80%93359." data-title="Fast%20gapped-read%20alignment%20with%20Bowtie%202" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Fast+gapped-read+alignment+with+Bowtie+2+Langmead+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0124316.ref015" id="pone.0124316.ref015" class="link-target"></a>Li B, Dewey CN. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics. 2011;12: 323.  pmid:21816040 <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiB%2C%20DeweyCN.%20RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome.%20BMC%20Bioinformatics.%202011%3B12%3A%20323.%20doi%3A%2010.1186%2F1471-2105-12-323%2021816040" data-title="RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=RSEM%3A+accurate+transcript+quantification+from+RNA-Seq+data+with+or+without+a+reference+genome+Li+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0124316.ref016" id="pone.0124316.ref016" class="link-target"></a>Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26: 139–140.  pmid:19910308 <ul class="reflinks"><li><a href="#" data-author="Robinson" data-cit="RobinsonMD%2C%20McCarthyDJ%2C%20SmythGK.%20edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data.%20Bioinformatics.%202010%3B26%3A%20139%E2%80%93140.%20doi%3A%2010.1093%2Fbioinformatics%2Fbtp616%2019910308" data-title="edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=edgeR%3A+a+Bioconductor+package+for+differential+expression+analysis+of+digital+gene+expression+data+Robinson+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0124316.ref017" id="pone.0124316.ref017" class="link-target"></a>Shen Y, Zhang Y, Chen J, Lin H, Zhao M, Peng H, et al. Genome expression profile analysis reveals important transcripts in maize roots responding to the stress of heavy metal Pb. Physiol Plant. 2013;147: 270–282.  pmid:22747913 <ul class="reflinks"><li><a href="#" data-author="Shen" data-cit="ShenY%2C%20ZhangY%2C%20ChenJ%2C%20LinH%2C%20ZhaoM%2C%20PengH%2C%20et%20al.%20Genome%20expression%20profile%20analysis%20reveals%20important%20transcripts%20in%20maize%20roots%20responding%20to%20the%20stress%20of%20heavy%20metal%20Pb.%20Physiol%20Plant.%202013%3B147%3A%20270%E2%80%93282.%20doi%3A%2010.1111%2Fj.1399-3054.2012.01670.x%2022747913" data-title="Genome%20expression%20profile%20analysis%20reveals%20important%20transcripts%20in%20maize%20roots%20responding%20to%20the%20stress%20of%20heavy%20metal%20Pb" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22747913" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genome+expression+profile+analysis+reveals+important+transcripts+in+maize+roots+responding+to+the+stress+of+heavy+metal+Pb+Shen+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0124316.ref018" id="pone.0124316.ref018" class="link-target"></a>Shi T, Gao Z, Wang L, Zhang Z, Zhuang W, Sun H, et al. Identification of differentially-expressed genes associated with pistil abortion in Japanese apricot by genome-wide transcriptional analysis. PloS ONE. 2012;7: e47810.  pmid:23091648 <ul class="reflinks"><li><a href="#" data-author="Shi" data-cit="ShiT%2C%20GaoZ%2C%20WangL%2C%20ZhangZ%2C%20ZhuangW%2C%20SunH%2C%20et%20al.%20Identification%20of%20differentially-expressed%20genes%20associated%20with%20pistil%20abortion%20in%20Japanese%20apricot%20by%20genome-wide%20transcriptional%20analysis.%20PloS%20ONE.%202012%3B7%3A%20e47810.%20doi%3A%2010.1371%2Fjournal.pone.0047810%2023091648" data-title="Identification%20of%20differentially-expressed%20genes%20associated%20with%20pistil%20abortion%20in%20Japanese%20apricot%20by%20genome-wide%20transcriptional%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23091648" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+differentially-expressed+genes+associated+with+pistil+abortion+in+Japanese+apricot+by+genome-wide+transcriptional+analysis+Shi+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0124316.ref019" id="pone.0124316.ref019" class="link-target"></a>Tan Y, Liu J, Chen X, Zheng H, Li F. RNA-seq-based comparative transcriptome analysis of the syngas-utilizing bacterium <em>Clostridium ljungdahlii</em> DSM 13528 grown autotrophically and heterotrophically. Mol BioSyst. 2013;9: 2775–2784.  pmid:24056499 <ul class="reflinks"><li><a href="#" data-author="Tan" data-cit="TanY%2C%20LiuJ%2C%20ChenX%2C%20ZhengH%2C%20LiF.%20RNA-seq-based%20comparative%20transcriptome%20analysis%20of%20the%20syngas-utilizing%20bacterium%20Clostridium%20ljungdahlii%20DSM%2013528%20grown%20autotrophically%20and%20heterotrophically.%20Mol%20BioSyst.%202013%3B9%3A%202775%E2%80%932784.%20doi%3A%2010.1039%2Fc3mb70232d%2024056499" data-title="RNA-seq-based%20comparative%20transcriptome%20analysis%20of%20the%20syngas-utilizing%20bacterium%20Clostridium%20ljungdahlii%20DSM%2013528%20grown%20autotrophically%20and%20heterotrophically" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24056499" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=RNA-seq-based+comparative+transcriptome+analysis+of+the+syngas-utilizing+bacterium+Clostridium+ljungdahlii+DSM+13528+grown+autotrophically+and+heterotrophically+Tan+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0124316.ref020" id="pone.0124316.ref020" class="link-target"></a>Rees DC, Johnson E, Lewinson O. ABC transporters: the power to change. Nat Rev Mol Cell Biol. 2009;10: 218–227.  pmid:19234479 <ul class="reflinks"><li><a href="#" data-author="Rees" data-cit="ReesDC%2C%20JohnsonE%2C%20LewinsonO.%20ABC%20transporters%3A%20the%20power%20to%20change.%20Nat%20Rev%20Mol%20Cell%20Biol.%202009%3B10%3A%20218%E2%80%93227.%20doi%3A%2010.1038%2Fnrm2646%2019234479" data-title="ABC%20transporters%3A%20the%20power%20to%20change" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19234479" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=ABC+transporters%3A+the+power+to+change+Rees+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0124316.ref021" id="pone.0124316.ref021" class="link-target"></a>Linton KJ, Higgins CF. The <em>Escherichia coli</em> ATP-binding cassette (ABC) proteins. Mol Microbiol. 1998;28: 5–13. pmid:9593292 <ul class="reflinks"><li><a href="#" data-author="Linton" data-cit="LintonKJ%2C%20HigginsCF.%20The%20Escherichia%20coli%20ATP-binding%20cassette%20%28ABC%29%20proteins.%20Mol%20Microbiol.%201998%3B28%3A%205%E2%80%9313.%209593292" data-title="The%20Escherichia%20coli%20ATP-binding%20cassette%20%28ABC%29%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9593292" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+Escherichia+coli+ATP-binding+cassette+%28ABC%29+proteins+Linton+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0124316.ref022" id="pone.0124316.ref022" class="link-target"></a>Quentin Y, Fichant G, Denizot F. Inventory, assembly and analysis of <em>Bacillus subtilis</em> ABC transport systems. J Mol Biol. 1999;287: 467–484. pmid:10092453 <ul class="reflinks"><li><a href="#" data-author="Quentin" data-cit="QuentinY%2C%20FichantG%2C%20DenizotF.%20Inventory%2C%20assembly%20and%20analysis%20of%20Bacillus%20subtilis%20ABC%20transport%20systems.%20J%20Mol%20Biol.%201999%3B287%3A%20467%E2%80%93484.%2010092453" data-title="Inventory%2C%20assembly%20and%20analysis%20of%20Bacillus%20subtilis%20ABC%20transport%20systems" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10092453" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inventory%2C+assembly+and+analysis+of+Bacillus+subtilis+ABC+transport+systems+Quentin+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0124316.ref023" id="pone.0124316.ref023" class="link-target"></a>Reiter WD. Biochemical genetics of nucleotide sugar interconversion reactions. Curr Opin Plant Biol. 2008;11: 236–243.  pmid:18486535 <ul class="reflinks"><li><a href="#" data-author="Reiter" data-cit="ReiterWD.%20Biochemical%20genetics%20of%20nucleotide%20sugar%20interconversion%20reactions.%20Curr%20Opin%20Plant%20Biol.%202008%3B11%3A%20236%E2%80%93243.%20doi%3A%2010.1016%2Fj.pbi.2008.03.009%2018486535" data-title="Biochemical%20genetics%20of%20nucleotide%20sugar%20interconversion%20reactions" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18486535" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Biochemical+genetics+of+nucleotide+sugar+interconversion+reactions+Reiter+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0124316.ref024" id="pone.0124316.ref024" class="link-target"></a>Kilstrup M, Hammer K, Ruhdal Jensen P, Martinussen J. Nucleotide metabolism and its control in lactic acid bacteria. FEMS Microbiol Rev. 2005;29: 555–590. pmid:15935511 <ul class="reflinks"><li><a href="#" data-author="Kilstrup" data-cit="KilstrupM%2C%20HammerK%2C%20Ruhdal%20JensenP%2C%20MartinussenJ.%20Nucleotide%20metabolism%20and%20its%20control%20in%20lactic%20acid%20bacteria.%20FEMS%20Microbiol%20Rev.%202005%3B29%3A%20555%E2%80%93590.%2015935511" data-title="Nucleotide%20metabolism%20and%20its%20control%20in%20lactic%20acid%20bacteria" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15935511" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Nucleotide+metabolism+and+its+control+in+lactic+acid+bacteria+Kilstrup+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0124316.ref025" id="pone.0124316.ref025" class="link-target"></a>Xu K, Xu P. Betaine and beet molasses enhance L-lactic acid production by <em>Bacillus coagulans</em>. PLoS ONE. 2014;9: e100731.  pmid:24956474 <ul class="reflinks"><li><a href="#" data-author="Xu" data-cit="XuK%2C%20XuP.%20Betaine%20and%20beet%20molasses%20enhance%20L-lactic%20acid%20production%20by%20Bacillus%20coagulans.%20PLoS%20ONE.%202014%3B9%3A%20e100731.%20doi%3A%2010.1371%2Fjournal.pone.0100731%2024956474" data-title="Betaine%20and%20beet%20molasses%20enhance%20L-lactic%20acid%20production%20by%20Bacillus%20coagulans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24956474" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Betaine+and+beet+molasses+enhance+L-lactic+acid+production+by+Bacillus+coagulans+Xu+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0124316.ref026" id="pone.0124316.ref026" class="link-target"></a>Xu K, Xu P. Efficient production of L-lactic acid using co-feeding strategy based on cane molasses/glucose carbon sources. Bioresour Technol. 2014;153: 23–29.  pmid:24333698 <ul class="reflinks"><li><a href="#" data-author="Xu" data-cit="XuK%2C%20XuP.%20Efficient%20production%20of%20L-lactic%20acid%20using%20co-feeding%20strategy%20based%20on%20cane%20molasses%2Fglucose%20carbon%20sources.%20Bioresour%20Technol.%202014%3B153%3A%2023%E2%80%9329.%20doi%3A%2010.1016%2Fj.biortech.2013.11.057%2024333698" data-title="Efficient%20production%20of%20L-lactic%20acid%20using%20co-feeding%20strategy%20based%20on%20cane%20molasses%2Fglucose%20carbon%20sources" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24333698" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Efficient+production+of+L-lactic+acid+using+co-feeding+strategy+based+on+cane+molasses%2Fglucose+carbon+sources+Xu+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0124316.ref027" id="pone.0124316.ref027" class="link-target"></a>Wang L, Xue Z, Zhao B, Yu B, Xu P, Ma Y. Jerusalem artichoke powder: A useful material in producing high-optical-purity L-lactate using an efficient sugar-utilizing thermophilic <em>Bacillus coagulans</em> strain. Bioresour Technol. 2013;130: 174–180.  pmid:23306126 <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangL%2C%20XueZ%2C%20ZhaoB%2C%20YuB%2C%20XuP%2C%20MaY.%20Jerusalem%20artichoke%20powder%3A%20A%20useful%20material%20in%20producing%20high-optical-purity%20L-lactate%20using%20an%20efficient%20sugar-utilizing%20thermophilic%20Bacillus%20coagulans%20strain.%20Bioresour%20Technol.%202013%3B130%3A%20174%E2%80%93180.%20doi%3A%2010.1016%2Fj.biortech.2012.11.144%2023306126" data-title="Jerusalem%20artichoke%20powder%3A%20A%20useful%20material%20in%20producing%20high-optical-purity%20L-lactate%20using%20an%20efficient%20sugar-utilizing%20thermophilic%20Bacillus%20coagulans%20strain" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23306126" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Jerusalem+artichoke+powder%3A+A+useful+material+in+producing+high-optical-purity+L-lactate+using+an+efficient+sugar-utilizing+thermophilic+Bacillus+coagulans+strain+Wang+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0124316.ref028" id="pone.0124316.ref028" class="link-target"></a>Ye L, Zhou X, Hudari MSB, Li Z, Wu JC. Highly efficient production of L-lactic acid from xylose by newly isolated <em>Bacillus coagulans</em> C106. Bioresour Technol. 2013;132: 38–44.  pmid:23399496 <ul class="reflinks"><li><a href="#" data-author="Ye" data-cit="YeL%2C%20ZhouX%2C%20HudariMSB%2C%20LiZ%2C%20WuJC.%20Highly%20efficient%20production%20of%20L-lactic%20acid%20from%20xylose%20by%20newly%20isolated%20Bacillus%20coagulans%20C106.%20Bioresour%20Technol.%202013%3B132%3A%2038%E2%80%9344.%20doi%3A%2010.1016%2Fj.biortech.2013.01.011%2023399496" data-title="Highly%20efficient%20production%20of%20L-lactic%20acid%20from%20xylose%20by%20newly%20isolated%20Bacillus%20coagulans%20C106" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23399496" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Highly+efficient+production+of+L-lactic+acid+from+xylose+by+newly+isolated+Bacillus+coagulans+C106+Ye+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0124316.ref029" id="pone.0124316.ref029" class="link-target"></a>Guzzo J. Biotechnical applications of small heat shock proteins from bacteria. Int J Biochem Cell Biol. 2012;44: 1698–1705.  pmid:22706478 <ul class="reflinks"><li><a href="#" data-author="Guzzo" data-cit="GuzzoJ.%20Biotechnical%20applications%20of%20small%20heat%20shock%20proteins%20from%20bacteria.%20Int%20J%20Biochem%20Cell%20Biol.%202012%3B44%3A%201698%E2%80%931705.%20doi%3A%2010.1016%2Fj.biocel.2012.06.007%2022706478" data-title="Biotechnical%20applications%20of%20small%20heat%20shock%20proteins%20from%20bacteria" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22706478" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Biotechnical+applications+of+small+heat+shock+proteins+from+bacteria+Guzzo+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0124316.ref030" id="pone.0124316.ref030" class="link-target"></a>Law CJ, Maloney PC, Wang DN. Ins and outs of major facilitator superfamily antiporters. Annu Rev Microbiol. 2008;62: 289–305.  pmid:18537473 <ul class="reflinks"><li><a href="#" data-author="Law" data-cit="LawCJ%2C%20MaloneyPC%2C%20WangDN.%20Ins%20and%20outs%20of%20major%20facilitator%20superfamily%20antiporters.%20Annu%20Rev%20Microbiol.%202008%3B62%3A%20289%E2%80%93305.%20doi%3A%2010.1146%2Fannurev.micro.61.080706.093329%2018537473" data-title="Ins%20and%20outs%20of%20major%20facilitator%20superfamily%20antiporters" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18537473" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Ins+and+outs+of+major+facilitator+superfamily+antiporters+Law+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

