



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>One of the most important and challenging problems in biomedicine and genomics is how to identify the disease genes. In this study, we developed a computational method to identify colorectal cancer-related genes based on (i) the gene expression profiles, and (ii) the shortest path analysis of functional protein association networks. The former has been used to select differentially expressed genes as disease genes for quite a long time, while the latter has been widely used to study the mechanism of diseases. With the existing protein-protein interaction data from STRING (<span class="underline">S</span>earch <span class="underline">T</span>ool for the <span class="underline">R</span>etrieval of <span class="underline">I</span>nteracting <span class="underline">G</span>enes), a weighted functional protein association network was constructed. By means of the mRMR (<span class="underline">M</span>aximum <span class="underline">R</span>elevance <span class="underline">M</span>inimum <span class="underline">R</span>edundancy) approach, six genes were identified that can distinguish the colorectal tumors and normal adjacent colonic tissues from their gene expression profiles. Meanwhile, according to the shortest path approach, we further found an additional 35 genes, of which some have been reported to be relevant to colorectal cancer and some are very likely to be relevant to it. Interestingly, the genes we identified from both the gene expression profiles and the functional protein association network have more cancer genes than the genes identified from the gene expression profiles alone. Besides, these genes also had greater functional similarity with the reported colorectal cancer genes than the genes identified from the gene expression profiles alone. All these indicate that our method as presented in this paper is quite promising. The method may become a useful tool, or at least plays a complementary role to the existing method, for identifying colorectal cancer genes. It has not escaped our notice that the method can be applied to identify the genes of other diseases as well.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Li B-Q, Huang T, Liu L, Cai Y-D, Chou K-C (2012) Identification of Colorectal Cancer Related Genes with mRMR and Shortest Path in Protein-Protein Interaction Network. PLoS ONE 7(4):
           e33393.
        
        https://doi.org/10.1371/journal.pone.0033393</p><p><strong>Editor: </strong>Paulo Lee Ho, Instituto Butantan, Brazil</p><p><strong>Received: </strong>September 13, 2011; <strong>Accepted: </strong>February 13, 2012; <strong>Published: </strong> April 4, 2012</p><p><strong>Copyright: </strong> © 2012 Li et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the National Basic Research Program of China (2011CB510102, 2011CB510101, 2011CB910200 and 2010CB912702), the Natural Science Foundation of China (90913009), the Chinese Academy of Sciences (KSCX2-EW-R-04), a National High Tech Program Grant 2012AA022802 and the Innovation Program of Shanghai Municipal Education Commission (12ZZ087). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Colorectal cancer (CRC) is one of the most common malignancies in the western countries and a major cause of cancer-related death. Early detection of CRC could reduce the morbidity and improve the prognosis. Therefore, it is of great importance to identify cancer-related genes that could be used as biomarker for early diagnosis.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Recently, with the development of high-throughput biotechnologies, a large amount of biological data has been generated, such as yeast two-hybrid systems, protein complex and gene expression profiles, etc. These data are useful resources for deducing and understanding gene functions <a href="#pone.0033393-Huang1" class="ref-tip">[1]</a>, <a href="#pone.0033393-Huang2" class="ref-tip">[2]</a>, <a href="#pone.0033393-Cai1" class="ref-tip">[3]</a>, <a href="#pone.0033393-Huang3" class="ref-tip">[4]</a>, <a href="#pone.0033393-Huang4" class="ref-tip">[5]</a>, <a href="#pone.0033393-Chen1" class="ref-tip">[6]</a>, <a href="#pone.0033393-Chen2" class="ref-tip">[7]</a>, <a href="#pone.0033393-Liu1" class="ref-tip">[8]</a>. So far the protein-protein interaction (PPI) data has been widely used for gene function prediction with the assumption that interacting proteins share the same or have similar functions and hence may be involved in the same pathway. This “guilty by association” rule was first proposed by Nabieva et al. <a href="#pone.0033393-Nabieva1" class="ref-tip">[9]</a> and can also be used to identify cancer related genes.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>STRING is an online database resource which is an abbreviation for <strong>S</strong>earch <strong>T</strong>ool for the <strong>R</strong>etrieval of <strong>I</strong>nteracting <strong>G</strong>enes <a href="#pone.0033393-Szklarczyk1" class="ref-tip">[10]</a>. It provides both experimental as well as predicted interaction information with a confidence score. Algorithms based on PPI suggest that proteins with short distances to each other in the network are more likely to share the common biological functions <a href="#pone.0033393-Sharan1" class="ref-tip">[11]</a>, <a href="#pone.0033393-Bogdanov1" class="ref-tip">[12]</a>, <a href="#pone.0033393-Kourmpetis1" class="ref-tip">[13]</a>, <a href="#pone.0033393-Ng1" class="ref-tip">[14]</a>, and that interactive neighbors are more likely to have identical biological function than non-interactive ones <a href="#pone.0033393-Karaoz1" class="ref-tip">[15]</a>, <a href="#pone.0033393-Letovsky1" class="ref-tip">[16]</a>. This is because the query protein and its interactive proteins may form a protein complex to perform a particular function or involved in a same pathway.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Although the successful application of the high-throughput data for gene function perdition and identification of novel genes associated with cancers, the errors in the high-throughput data have not been well solved yet. In this paper, we proposed a new method for identifying CRC related genes by integrating gene expression profile and a weighted functional protein association network constructed with PPI data from STRING. This method can make up the defect of only using high-throughput data. Meanwhile, the mRMR (<strong>m</strong>aximum <strong>r</strong>elevance <strong>m</strong>inimum <strong>r</strong>edundancy) algorithm <a href="#pone.0033393-Peng1" class="ref-tip">[17]</a> was utilized to identify six promising candidate genes distinguishing tumor and the normal colorectal samples. The Dijkstra's algorithm <a href="#pone.0033393-Dijkstra1" class="ref-tip">[18]</a> was used to construct the shortest paths between each pair of the six genes. Moreover, additional 35 genes on these shortest paths were also identified and analyzed. For such <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e001" class="inline-graphic"></span> gene thus identified, it was observed that they contained more cancer genes than the genes identified from the gene expression profiles alone. Furthermore, the 41 genes also had greater functional similarity with the reported CRC genes than the genes identified from gene expression profiles alone. It is anticipated that some of the 41 genes thus identified might belong to novel CRC related genes.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Dataset"></a>
<h3>Dataset</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>We used the gene expression data from the colorectal cancer study of Hinoue et al. <a href="#pone.0033393-Hinoue1" class="ref-tip">[19]</a>. The gene expression profiling of 26 colorectal tumors and matched histologically normal adjacent colonic tissue samples were retrieved from NCBI Gene Expression Omnibus (GEO) with the accession number of GSE25070. The gene expression profile was obtained using the Illumina Ref-8 whole-genome expression BeadChip with 24526 probes corresponding to 18491 genes. Signal intensity was log2 transformed and then normalized with RSN (<strong>R</strong>obust <strong>S</strong>pline <strong>N</strong>ormalization) method.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Tissue sample representation"></a>
<h3>Tissue sample representation</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Based on the above, the representation of a tissue sample can be formulated as a 24526-D (dimensional vector), as given by<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e002" class="inline-graphic"><span class="note">(1)</span></span>where <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e003" class="inline-graphic"></span> represents the tissue sample, <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e004" class="inline-graphic"></span> the value of it's <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e005" class="inline-graphic"></span> probe, and <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e006" class="inline-graphic"></span> the transpose matrix (cf. Eq.6 of <a href="#pone.0033393-Chou1" class="ref-tip">[20]</a>).</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Cancer related gene list and two colorectal cancer related gene lists"></a>
<h3>Cancer related gene list and two colorectal cancer related gene lists</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>We compiled three gene lists from public databases and published works to compare with the 41 candidate genes we identified. These three genes lists included one cancer related gene list and two colorectal cancer related gene lists.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2" class="link-target"></a><p>742 cancer-related genes were derived from three sources. First, we obtained 457 cancer-related genes from the Cancer Gene Census of the Sanger Centre. Secondly, we retrieved cancer-related genes from the Atlas of Genetics and Cytogenetic in Oncology <a href="#pone.0033393-Huret1" class="ref-tip">[21]</a>. The third part was collected from the Human Protein Reference Database <a href="#pone.0033393-KeshavaPrasad1" class="ref-tip">[22]</a>. See <a href="#pone.0033393.s001">Supporting Information S1</a>.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3" class="link-target"></a><p>The first colorectal cancer related gene list was retrieved from the study of Sabates-Bellver and coworkers <a href="#pone.0033393-SabatesBellver1" class="ref-tip">[23]</a>. They compared the transcriptomes of 32 adenomas with normal mucosa from the same individuals and identified 438 genes with markedly altered expression in colorectal adenomas compared with normal mucosa with Affymetrix U133 Plus 2.0 array. See <a href="#pone.0033393.s001">Supporting Information S1</a>.</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4" class="link-target"></a><p>The second colorectal cancer related gene list was retrieved form a recent work of Nagaraj et al. <a href="#pone.0033393-Nagaraj1" class="ref-tip">[24]</a>. They proposed a Boolean based systems biology approach with guilt-by-association algorithm to identify novel cancer-associated genes. We compiled all the 134 novel CRC related genes identified in this study. See <a href="#pone.0033393.s001">Supporting Information S1</a>.</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="PPI data from STRING"></a>
<h3>PPI data from STRING</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>The initial weighted PPI network was retrieved from STRING (version 9.0) <a href="#pone.0033393-Szklarczyk1" class="ref-tip">[10]</a> (<a href="http://string.embl.de/">http://string.embl.de/</a>), which is a large database of known and predicted protein interactions. Proteins in the interaction network were represented with nodes, while the interaction between any two proteins therein was represented with an edge. These interactions contain direct (physical) and indirect (functional) interactions, derived from numerous sources such as experimental repositories, computational prediction methods. In the network, each edge is marked with a score to quantify the interaction confidence, i.e., the likelihood that an interaction may occur.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="The mRMR (maximum relevance minimum redundancy) method"></a>
<h3>The mRMR (<span class="underline">m</span>aximum <span class="underline">r</span>elevance <span class="underline">m</span>inimum <span class="underline">r</span>edundancy) method</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>To find the genes that can distinguish colorectal tumors and normal adjacent tissues, we used the mRMR method, which was originally developed by Peng et al. <a href="#pone.0033393-Peng1" class="ref-tip">[17]</a> for analyzing the microarray data. The mRMR method could rank genes according to their relevance to the class of samples concerned, and meanwhile also could take the redundancy of genes into account. Those genes, which have the best trade-off between the maximum relevance to the sample class and the minimum redundancy, were considered as “good” biomarkers.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>Both the relevance and redundancy were quantified by the following mutual information (MI):<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e007" class="inline-graphic"><span class="note">(2)</span></span>where <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e008" class="inline-graphic"></span> and <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e009" class="inline-graphic"></span> are vectors, <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e010" class="inline-graphic"></span> is their joint probabilistic density, and <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e011" class="inline-graphic"></span> and <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e012" class="inline-graphic"></span> are the marginal probabilistic densities.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3" class="link-target"></a><p>To quantify both the relevance and redundancy, let us define <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e013" class="inline-graphic"></span> as the whole gene set, <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e014" class="inline-graphic"></span> as the already-selected gene set containing <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e015" class="inline-graphic"></span> genes and <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e016" class="inline-graphic"></span> as the to-be-selected gene set containing <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e017" class="inline-graphic"></span> genes. The relevance <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e018" class="inline-graphic"></span> between the gene <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e019" class="inline-graphic"></span> in <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e020" class="inline-graphic"></span> and the target <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e021" class="inline-graphic"></span> can be calculated by:<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e022" class="inline-graphic"><span class="note">(3)</span></span>The redundancy <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e023" class="inline-graphic"></span> between the gene <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e024" class="inline-graphic"></span> in <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e025" class="inline-graphic"></span> and all the genes in <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e026" class="inline-graphic"></span> can be calculated by:<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e027" class="inline-graphic"><span class="note">(4)</span></span>In order to obtain the gene <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e028" class="inline-graphic"></span> in <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e029" class="inline-graphic"></span> with the maximum relevance and minimum redundancy, let us combine Eq.3 and Eq.4, as can be formulated as follows:<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e030" class="inline-graphic"><span class="note">(5)</span></span>Given a gene set with <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e031" class="inline-graphic"></span> genes, the mRMR operation for the gene evaluation will continue <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e032" class="inline-graphic"></span> rounds. After these evaluations, the mRMR method will generate a gene set <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e033" class="inline-graphic"></span> as formulated by<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e034" class="inline-graphic"><span class="note">(6)</span></span>where the index <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e035" class="inline-graphic"></span> indicates which round the gene is selected. The smaller the index <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e036" class="inline-graphic"></span> is, the earlier the gene satisfied Eq.5 and the better the gene is.</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Prediction engine"></a>
<h3>Prediction engine</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>In this study, the Nearest Neighbor Algorithm (NNA) <a href="#pone.0033393-Friedman1" class="ref-tip">[25]</a>, <a href="#pone.0033393-Denoeux1" class="ref-tip">[26]</a>, which has been widely used in bioinformatics and computational biology <a href="#pone.0033393-Cai1" class="ref-tip">[3]</a>, <a href="#pone.0033393-Hu1" class="ref-tip">[27]</a>, <a href="#pone.0033393-Hu2" class="ref-tip">[28]</a>, <a href="#pone.0033393-Yang1" class="ref-tip">[29]</a>, <a href="#pone.0033393-Huang5" class="ref-tip">[30]</a>, <a href="#pone.0033393-Hu3" class="ref-tip">[31]</a>, <a href="#pone.0033393-Huang6" class="ref-tip">[32]</a>, <a href="#pone.0033393-Huang7" class="ref-tip">[33]</a>, <a href="#pone.0033393-Cai2" class="ref-tip">[34]</a>, was adopted to predict the class of colorectal tissue samples. The “nearness” was calculated according to the following equation<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e037" class="inline-graphic"><span class="note">(7)</span></span>where <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e038" class="inline-graphic"></span> and <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e039" class="inline-graphic"></span> are two vectors representing two tissue samples, <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e040" class="inline-graphic"></span> is their dot product, <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e041" class="inline-graphic"></span> and <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e042" class="inline-graphic"></span> are their moduluses. The smaller the <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e043" class="inline-graphic"></span>, the more similar the two samples are <a href="#pone.0033393-Chou2" class="ref-tip">[35]</a>. For an intuitive illustration of how NNA works, see Fig.5 of <a href="#pone.0033393-Chou1" class="ref-tip">[20]</a>.</p>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="Performance validation"></a>
<h3>Performance validation</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>The following three cross-validation methods are often used in statistics for validating a statistical prediction method: independent dataset test, subsampling test, and jackknife test <a href="#pone.0033393-Chou3" class="ref-tip">[36]</a>. However, among the three validation methods, the jackknife test is the least arbitrary due to the following facts. (i) For the independent dataset test, although all the samples used to test the predictor are outside the training dataset used to train the prediction engine so as to exclude the “memory” effect or bias, the way of how to select the independent samples to test the predictor could be quite arbitrary unless the number of independent samples is sufficiently large. This kind of arbitrariness might lead to completely opposite conclusions. For instance, the conclusion that a predictor yielded a higher success rate than the other predictor for a given independent testing dataset might become just opposite when tested by another independent testing dataset <a href="#pone.0033393-Chou3" class="ref-tip">[36]</a>. (ii) For the subsampling test, the concrete procedure usually used in literatures is the 5-fold, 7-fold or 10-fold cross-validation. The problem with this kind of subsampling test is that the number of possible selections in dividing a benchmark dataset is extremely large even for a very simple and small dataset, as elucidated in <a href="#pone.0033393-Chou4" class="ref-tip">[37]</a> and demonstrated by Eqs.28–30 in <a href="#pone.0033393-Chou1" class="ref-tip">[20]</a>. Therefore, in any actual subsampling cross-validation tests, only a very tiny fraction of the possible selections are taken into account. Since different selections will always result in different outcomes even for a same benchmark dataset and a same predictor, the subsampling test cannot avoid the arbitrariness either. A test method unable to yield a unique outcome cannot be regarded as a good one. (iii) In the jackknife test, all the samples in the benchmark dataset will be singled out one-by-one and tested by the predictor trained by the remaining samples. During the process of jackknifing, both the training dataset and testing dataset are actually open, and each sample will be in turn moved between the two. The jackknife test can exclude the “memory” effect. Also, the arbitrariness problem as mentioned above for the independent dataset test and subsampling test can be avoided because the outcome obtained by the jackknife test is always unique for a given benchmark dataset. Accordingly, the jackknife test has been widely and increasingly used to inspect the quality of various predictors (see, e.g., <a href="#pone.0033393-Huang5" class="ref-tip">[30]</a>, <a href="#pone.0033393-Hu3" class="ref-tip">[31]</a>, <a href="#pone.0033393-Huang6" class="ref-tip">[32]</a>, <a href="#pone.0033393-Esmaeili1" class="ref-tip">[38]</a>, <a href="#pone.0033393-Georgiou1" class="ref-tip">[39]</a>, <a href="#pone.0033393-Gu1" class="ref-tip">[40]</a>, <a href="#pone.0033393-Chou5" class="ref-tip">[41]</a>, <a href="#pone.0033393-Mohabatkar1" class="ref-tip">[42]</a>, <a href="#pone.0033393-Xiao1" class="ref-tip">[43]</a>, <a href="#pone.0033393-Wu1" class="ref-tip">[44]</a>, <a href="#pone.0033393-Mohabatkar2" class="ref-tip">[45]</a>, <a href="#pone.0033393-Chou6" class="ref-tip">[46]</a>). Accordingly, in this study the jackknife test was also used to examine the quality of the current prediction method.</p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2" class="link-target"></a><p>The prediction accuracy was formulated by<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e044" class="inline-graphic"><span class="note">(8)</span></span>where TP represents the true positive; TN, the true negative; FP, the false positive; and FN, the false negative.</p>
</div>

<div id="section8" class="section toc-section"><a id="s2h" name="s2h" class="link-target" title="Incremental feature selection (IFS)"></a>
<h3>Incremental feature selection (IFS)</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>Based on the ranked genes according to their importance after mRMR evaluation, we used the Incremental Feature Selection (IFS) (see, e.g., <a href="#pone.0033393-Huang1" class="ref-tip">[1]</a>, <a href="#pone.0033393-Huang8" class="ref-tip">[47]</a>) to determine the optimal number of genes as biomarkers. During the IFS procedure, genes in the ranked gene set are added one by one from higher to lower rank. A new gene set is composed when one gene is added. Thus <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e045" class="inline-graphic"></span> gene sets would be composed when given <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e046" class="inline-graphic"></span> ranked genes. The<span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e047" class="inline-graphic"></span> gene set is<a name="" id="" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e048" class="inline-graphic"><span class="note">(9)</span></span>For each of the N gene sets, an NNA predictor was constructed and examined using the jackknife test to the benchmark dataset. By doing so we obtained an IFS table with one column for the index <em>i</em> and another column for the prediction accuracy. Thus, we could obtain the optimal gene set (<span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e049" class="inline-graphic"></span>), with which the predictor would yield the best prediction accuracy.</p>
</div>

<div id="section9" class="section toc-section"><a id="s2i" name="s2i" class="link-target" title="Graph approach and shortest paths tracing"></a>
<h3>Graph approach and shortest paths tracing</h3>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1" class="link-target"></a><p>Graphs are a useful vehicle for studying complex biological systems because they can provide intuitive insights and the overall structure property, as demonstrated by various studies on a series of important biological topics (see, e.g., <a href="#pone.0033393-Chou7" class="ref-tip">[48]</a>, <a href="#pone.0033393-Zhou1" class="ref-tip">[49]</a>, <a href="#pone.0033393-Chou8" class="ref-tip">[50]</a>, <a href="#pone.0033393-Chou9" class="ref-tip">[51]</a>, <a href="#pone.0033393-Althaus1" class="ref-tip">[52]</a>, <a href="#pone.0033393-Chou10" class="ref-tip">[53]</a>, <a href="#pone.0033393-Andraos1" class="ref-tip">[54]</a>, <a href="#pone.0033393-Chou11" class="ref-tip">[55]</a>, <a href="#pone.0033393-Zhou2" class="ref-tip">[56]</a>, <a href="#pone.0033393-Chou12" class="ref-tip">[57]</a>, <a href="#pone.0033393-Zhou3" class="ref-tip">[58]</a>). In this study, we first constructed a graph G(V, E) with the PPI data from STRING. In the graph, an edge was assigned for each pair of genes if they were in interaction with each other. The weight of edge E in graph G was derived from the confidence score according to the equation <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e050" class="inline-graphic"></span>, where <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e051" class="inline-graphic"></span> is the weight in graph G while <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0033393.e052" class="inline-graphic"></span> is the confidence score between two proteins concerned. Thus, we get a functional protein association network with edge weight. Dijkstra's algorithm <a href="#pone.0033393-Dijkstra1" class="ref-tip">[18]</a> was used to find the shortest path from each of the six genes to all the other five genes in the graph. Then we picked out all the genes existing in the shortest paths and rank these genes according to their betweenness.</p>
</div>

<div id="section10" class="section toc-section"><a id="s2j" name="s2j" class="link-target" title="KEGG enrichment analysis"></a>
<h3>KEGG enrichment analysis</h3>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1" class="link-target"></a><p>Functional annotation tool of DAVID <a href="#pone.0033393-Huangda1" class="ref-tip">[59]</a> was used for KEGG pathway enrichment analysis. The enrichment p-value was corrected to control family-wide false discovery rate under certain rate (e.g., ≤0.05) with Benjamin multiple testing correction method <a href="#pone.0033393-Benjamini1" class="ref-tip">[60]</a>. All the genes on the BeadChip were selected as background during the enrichment analysis.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="mRMR results"></a>
<h3>mRMR results</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>The expression profile was retrieved from GEO with the accession number of GSE25070, which contained 52 samples and 24,526 probes and was transformed to a CSV file with 52 rows and 24526 columns as the input of mRMR. Each probe represented a feature and the 26 tumor samples belonged to class 1 while the paired26 paired normal samples belonged to class 2. After running the mRMR software, we obtained two tables (see <a href="#pone.0033393.s002">Supporting Information S2</a>), of which one was called MaxRel table that ranked the probes according to their relevance to the class of samples, and the other called mRMR feature table that listed the probes with the maximum relevance and minimum redundancy to the class of samples.</p>
</div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="Six candidate genes identified by NNA and IFS"></a>
<h3>Six candidate genes identified by NNA and IFS</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>On the basis of the outputs of mRMR, we constructed 1000 feature subsets according to Eq.9. As described in the <a href="#s2">Materials and Methods</a> section, we tested the predictor with one feature, two features, three features, etc., and the IFS result can be found in <a href="#pone.0033393.s003">Supporting Information S3</a>. Shown in <a href="#pone-0033393-g001"><strong>Fig. 1</strong></a> is the IFS curve plotted based on the data of <a href="#pone.0033393.s003">Supporting Information S3</a>. In the IFS curve, the X-axis is the number of probes used for classification, and the Y-axis is the prediction accuracies of the nearest neighbor algorithm evaluated by the jackknife test. The maximum accuracy was 1 when 6 features were included. The optimal probe set included 6 probes corresponding to 6 different genes, which were GUCA2B, PI16, CDH3, SPIB, BEST2, and HMGCLL1 (<a href="#pone-0033393-t001"><strong>Table 1</strong></a>).</p>
<a class="link-target" id="pone-0033393-g001" name="pone-0033393-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.g001" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> IFS curve for the colorectal tumors and matched normal adjacent tissue samples classification.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>In the IFS curve, the X-axis is for the number of probes used for classification, and the Y-axis for the prediction accuracies by the nearest neighbor algorithm (NNA) evaluated by the jackknife (Leave-One-Out) cross-validation test. The peak accuracy was 1 with six probes. The top 6 probes in the mRMR probe list formed the optimal discriminative probe set.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.g001">
              https://doi.org/10.1371/journal.pone.0033393.g001</a></p></div><a class="link-target" id="pone-0033393-t001" name="pone-0033393-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.t001" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> mRMR top six genes.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.t001">
              https://doi.org/10.1371/journal.pone.0033393.t001</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="Shortest paths genes"></a>
<h3>Shortest paths genes</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>Meanwhile, we constructed an undirected graph with the PPI data from STRING. Then we picked out two genes from the six genes identified with the mRMR method as described above, and found out the shortest path between these two genes with the Dijkstra's algorithm. We obtained a total of 15 shortest paths with lowest cost (<a href="#pone.0033393.s004">Supporting Information S4</a>). Shown in <a href="#pone-0033393-g002"><strong>Fig. 2</strong></a> are the 15 shortest paths between the six candidate genes, where the interaction confidence was labeled on the edge for each of the interaction gene pairs. There were a total of 35 genes on the shortest paths and we ranked these genes according to their betweenness (<a href="#pone-0033393-t002"><strong>Table 2</strong></a>). Among these 35 genes, AR has the largest betweenness of 7, meaning that there are 7 shortest paths going through this gene. Accordingly, AR may play an important role in connecting the six candidate genes and hence may be related to CRC. Such a conclusion is fully consistent with the fact that AR protein was found in normal colorectal mucosa as well as in most CRC <a href="#pone.0033393-Catalano1" class="ref-tip">[61]</a>, <a href="#pone.0033393-Castagnetta1" class="ref-tip">[62]</a>, implying that the AR receptor is responsible for the mitogenic effects of the hormone as will be further discussed later.</p>
<a class="link-target" id="pone-0033393-g002" name="pone-0033393-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.g002" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> 15 shortest paths between the six genes identified with mRMR method.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p>The 15 shortest paths between the six candidate genes were identified with Dijkstra's algorithm based on the PPI data from STRING. Yellow roundrect represents the top six candidate genes identified by the mRMR method. Red round represents the 35 genes existing within the range of the shortest paths. Numbers on edges represent the edge weights to quantify the interaction confidence. The smaller the number is, the stronger the interaction between two nodes is. See the text in the Section of “Graph approach and shortest paths tracing” for the quantitative relation of the edge weight with the confidence score between two proteins concerned.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.g002">
              https://doi.org/10.1371/journal.pone.0033393.g002</a></p></div><a class="link-target" id="pone-0033393-t002" name="pone-0033393-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.t002" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Shortest paths genes.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.t002">
              https://doi.org/10.1371/journal.pone.0033393.t002</a></p></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>To test whether our 35 shortest path genes were hubs in the background network or not, we ran a permutation to count the occurrence time of our 35 shortest path genes in the shortest paths between 6 random selected genes when they has greater betweenness than that in our study. We repeated this process 5000 times, and the p-value was calculated as the proportion of occurrence time of the 35 genes in 5000 permutation. For detail, please see <a href="#pone-0033393-t002"><strong>Table 2</strong></a>. There were 10 shortest path genes whose p-values were not significant. TP53 was a star molecular involved in numerous biological processes and nearly related to all kinds of cancers <a href="#pone.0033393-Menendez1" class="ref-tip">[63]</a>. Therefore, it is nothing surprising that TP53 appeared many times in shortest path between 6 randomly picked genes. For EP300, it has been reported that this gene can acetylate TP53 and associated with lots of tumors <a href="#pone.0033393-Gayther1" class="ref-tip">[64]</a>. CTNNB1 and GSK3B belong to the Wnt signaling pathway, the role of which in caners has been well documented <a href="#pone.0033393-Polakis1" class="ref-tip">[65]</a>. For the remaining insignificant 6 genes, their betweennesses in our study were all one (<a href="#pone-0033393-t002"><strong>Table 2</strong></a>), and hence the number of occurrences for these genes in random shortest paths is prone to be greater than one. Most of these insignificant 6 shortest path genes fall behind in <a href="#pone-0033393-t002"><strong>Table 2</strong></a> according to their betweennesses, suggesting that they might not be important. Besides these 10 genes, the remaining 25 shortest path genes in our study were identified to be significant.</p>
</div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="MaxRel table gene KEGG enrichment"></a>
<h3>MaxRel table gene KEGG enrichment</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Using the functional annotation tool of DAVID, the KEGG pathway enrichment analysis was carried out for the genes corresponding to the 1000 probes listed in the MaxRel. The enrichment results showed that these genes were significantly enriched in the energy metabolism pathways, including fatty acid metabolism, pentose and glucuronate interconversions, as well as starch and sucrose metabolism (<a href="#pone-0033393-t003"><strong>Table 3</strong></a>). These results suggested that metabolism of nutrients may play critical role in the tumorigenesis of CRC.</p>
<a class="link-target" id="pone-0033393-t003" name="pone-0033393-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.t003" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> MaxRel table genes KEGG enrichment.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.t003">
              https://doi.org/10.1371/journal.pone.0033393.t003</a></p></div></div>

<div id="section5" class="section toc-section"><a id="s3e" name="s3e" class="link-target" title="Six candidate genes and shortest paths genes of KEGG enrichment"></a>
<h3>Six candidate genes and shortest paths genes of KEGG enrichment</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>The KEGG pathway enrichment analysis was also performed on the 41 genes including the top six genes in the mRMR list and 35 genes in the shortest paths between these six genes with the functional annotation tool of DAVID. The enrichment result thus obtained showed that these genes were significantly enriched in the canonic cancer related pathways, such as prostate cancer, pathways in cancer, Wnt signaling pathway, cell cycle, colorectal cancer, thyroid cancer, and so on. It is instructive to note that among these pathways, some have been proved to be relevant to colorectal cancer including Wnt signaling pathway, cell cycle, colorectal cancer and insulin signaling pathway (<a href="#pone-0033393-t004"><strong>Table 4</strong></a>).</p>
<a class="link-target" id="pone-0033393-t004" name="pone-0033393-t004"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.t004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.t004" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.t004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.t004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 4. </span> mRMR top six genes and shortest path genes KEGG enrichment.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.t004">
              https://doi.org/10.1371/journal.pone.0033393.t004</a></p></div></div>

<div id="section6" class="section toc-section"><a id="s3f" name="s3f" class="link-target" title="Overlap with cancer related gene list and two CRC related gene lists"></a>
<h3>Overlap with cancer related gene list and two CRC related gene lists</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>We compiled 742 cancer-related genes from the following three different sources: Cancer Gene Census from the Sanger Centre, Atlas of Genetics and Cytogenetic in Oncology <a href="#pone.0033393-Huret1" class="ref-tip">[21]</a>, and Human Protein Reference Database <a href="#pone.0033393-KeshavaPrasad1" class="ref-tip">[22]</a>. It was observed that 8 out of the 41 genes identified by us were proven to be cancer-related genes. Also, it was indicated by the Fisher's exact test that these 41 genes were significantly related to cancer (p-value = 0.0001908). See <span class="underline"><a href="#pone.0033393.s005">Supporting Information S5</a></span>.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2" class="link-target"></a><p>Moreover, we collected 438 genes that were differentially expressed between colorectal adenomas and normal mucosa from previous study <a href="#pone.0033393-SabatesBellver1" class="ref-tip">[23]</a>. Interestingly, the aforementioned 41 candidate genes identified by us had an overlap of 4 genes with the 438 genes, and the overlap was quite significant (p-value = 0.01057, Fisher's exact test). See <span class="underline"><a href="#pone.0033393.s005">Supporting Information S5</a></span>.</p>
<a id="article1.body1.sec3.sec6.p3" name="article1.body1.sec3.sec6.p3" class="link-target"></a><p>Recently, the Boolean based systems biology approach was employed to identify 134 novel CRC related genes <a href="#pone.0033393-Nagaraj1" class="ref-tip">[24]</a>, of which three were identified by us in this study and the overlap was significant (p-value = 0.002017, Fisher's exact test). <span class="underline">See <a href="#pone.0033393.s005">Supporting Information S5</a></span>.</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Discussion"></a><h2>Discussion</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="KEGG enrichment of MaxRel genes"></a>
<h3>KEGG enrichment of MaxRel genes</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>The genes corresponding to the 1000 probes listed in the MaxRel table were significantly enriched in the energy metabolism pathways, including fatty acid metabolism, pentose and glucuronate interconversions, as well as starch and sucrose metabolism. It has been shown that diet has an important effect on the CRC development. Our finding is quite consistent with the fact that genetic polymorphisms influencing the metabolism of nutrients play an important role in the etiology of CRC and colorectal adenomatous polyps <a href="#pone.0033393-Castagnetta1" class="ref-tip">[62]</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2" class="link-target"></a><p>Multiple lines of evidences have indicated the implication or involvement of fat in the etiology of CRC <a href="#pone.0033393-Howe1" class="ref-tip">[66]</a>. The crucial role of fatty acids in numerous biological processes suggests that alteration in fatty acid metabolizing genes contributes to colon carcinogenesis <a href="#pone.0033393-Hoeft1" class="ref-tip">[67]</a>. It has been shown that starch and sucrose metabolism and pentose and glucuronateinterconversions were closely related to cancers. Christensen et al. <a href="#pone.0033393-Christensen1" class="ref-tip">[68]</a> demonstrated that starch and sucrose metabolism and pentose and glucuronateinterconversions pathway were hypomethylated in isocitrate dehydrogenase mutant tumors. In addition, these two metabolic pathways were found to be significantly related to the risk of developing estrogen receptor-negative breast cancer <a href="#pone.0033393-Li1" class="ref-tip">[69]</a>.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3" class="link-target"></a><p>A recent CRC disease-specific transcriptome research showed that starch and sucrose metabolism was one of the 7 common pathway significant differentially regulated using two different microarray platforms including Affymetrix HGU133 Plus2.0 array and the CRC disease specific array. Besides, fatty acid metabolism was identified as significantly differentially regulated pathway using colorectal disease specific array <a href="#pone.0033393-Allen1" class="ref-tip">[70]</a>.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="Six candidate genes identified by mRMR, NNA and IFS"></a>
<h3>Six candidate genes identified by mRMR, NNA and IFS</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>In this study, we have identified the following six genes: GUCA2B, PI16, CDH3, SPIB, BEST2, and HMGCLL1. Below, let us briefly discuss their relationships with colorectal cancer.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2" class="link-target"></a><p>GUCA2B (uroguanylin) is an endogenous activator of the guanylate cyclase-2C receptor found to be down regulated 8-fold in adenoma, and its expression is detected in blood and urine <a href="#pone.0033393-Tsukahara1" class="ref-tip">[71]</a>.Therefore, GUCA2B could be regarded as a non-invasive biomarker for the early detection of CRC. In addition, the radio labeled uroguanylin analogs have been used for detection of CRC in vivo <a href="#pone.0033393-Liu2" class="ref-tip">[72]</a>.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3" class="link-target"></a><p>PI16 (Peptidase inhibitor 16) is detected within the testis, prostate, small intestine, colon, and ovary with immunohistochemical analyses <a href="#pone.0033393-Reeves1" class="ref-tip">[73]</a>. Decrease of PI16 level was detected in prostate cancer <a href="#pone.0033393-Reeves1" class="ref-tip">[73]</a> and gastric cancer <a href="#pone.0033393-Cui1" class="ref-tip">[74]</a>. Our result also showed that the expression of PI16 in colorectal adenocarcinoma was significant decreased compared with the adjacent non-tumor colorectal tissue, which was consistent with the result of the research in prostate cancer and gastric cancer. Since PI16 is not well characterized and so far there is no report whatsoever about PI16 in colorectal cancer etiology, our result implied that PI16 may become a promising biomarker for colorectal cancer early diagnosis.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4" class="link-target"></a><p>CDH3 is a classical cadherin, the demethylation of which is frequently detected in the advanced CRC which was associated with the overexpression of CDH3 <a href="#pone.0033393-Hibi1" class="ref-tip">[75]</a>. Besides CRC, CDH3 was also overexpressed in the majority of pancreatic cancer and gastric cancer, but not in their noncancerous counterparts or in normal tissues. Thus CDH3 was regarded as a novel tumor-associated antigen useful for immunotherapy and early diagnosis of gastric cancer and CRC <a href="#pone.0033393-Imai1" class="ref-tip">[76]</a>.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5" class="link-target"></a><p>SPIB is a transcription factor of the E-twenty-six (ETS) family, which is known to act as positive or negative regulators of gene expression. SPIB is an adenoma condition-specific down regulated gene and its expression underwent a striking decrease in CRC tissues indicating that SPIB may serve as potential markers of CRC invasiveness and metastasis <a href="#pone.0033393-Deves1" class="ref-tip">[77]</a>.</p>
<a id="article1.body1.sec4.sec2.p6" name="article1.body1.sec4.sec2.p6" class="link-target"></a><p>BEST2 (also known as VMD2L1) encodes a protein of the bestrophin family. Both RT-PCR analyses and X-gal staining revealed tissue-restricted BEST2 and VMD2L2 abundantly expressed in colon <a href="#pone.0033393-Stohr1" class="ref-tip">[78]</a>, <a href="#pone.0033393-Bakall1" class="ref-tip">[79]</a>. It has been show that BEST2 mediates bicarbonate transport by goblet cells in mouse colon <a href="#pone.0033393-Yu1" class="ref-tip">[80]</a>. Straub et al. <a href="#pone.0033393-vanCriekinge1" class="ref-tip">[81]</a> identified BEST2 as one of the methylation markers for early detection and prognosis of CRC. Therefore, BEST2 was expected to become a therapy target for CRC with demethylation agent.</p>
<a id="article1.body1.sec4.sec2.p7" name="article1.body1.sec4.sec2.p7" class="link-target"></a><p>HMGCLL1 has been show to be related to various cancers, such as pancreatic cancers <a href="#pone.0033393-Jones1" class="ref-tip">[82]</a>, glioblastoma multiforme <a href="#pone.0033393-Parsons1" class="ref-tip">[83]</a>, breast and colorectal cancers <a href="#pone.0033393-Wood1" class="ref-tip">[84]</a>. HMGCLL1 is one of the genes containing somatic mutations in pancreatic cancer <a href="#pone.0033393-Jones1" class="ref-tip">[82]</a>. Though mutation in HMGCLL1 has been reported to be involved in these cancers, the specific mechanisms underlying remain to be elucidated.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="Shortest path genes"></a>
<h3>Shortest path genes</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>We totally identified 35 shortest paths genes. As we can see from <a href="#pone-0033393-t002"><strong>Table 2</strong></a>, some shortest path genes such as TP53, EP300, CTNNB1 and GSK3B were not significant for CRC due to their universality in numerous cancers. However, these genes have been well documented to be relevant to CRC, and also their role in CRC has been well characterized <a href="#pone.0033393-Morikawa1" class="ref-tip">[85]</a>. Besides these genes, most of the other shortest genes listed in <a href="#pone-0033393-t002">Table 2</a> were quite specific to CRC (p-value&lt;0.05). Below, let us focus on the specific genes with the large betweenness values and discuss the relationship of such genes with CRC.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2" class="link-target"></a><p>AR (androgen receptor) is a ligand dependent transcription factor, which is involved in the control of cellular proliferation and differentiation <a href="#pone.0033393-MacLean1" class="ref-tip">[86]</a>. Several studies have provided supporting evidences for its involvement of sex steroid hormones (estrogens and androgens) in the etiology and progression of CRC <a href="#pone.0033393-Slattery1" class="ref-tip">[87]</a>. AR protein has been shown to be expressed in normal colorectal mucosa and in most colorectal cancer <a href="#pone.0033393-Catalano1" class="ref-tip">[61]</a>, <a href="#pone.0033393-Castagnetta1" class="ref-tip">[62]</a>, supporting that CRC expressing the AR receptor may respond to mitogenic effects of the hormone. Moreover, somatic reductions of the androgen receptor CAG repeat occur frequently, through a pathway different from microsatellite instability and early during colon carcinogenesis. Apparent growth selection of cells harboring shortened AR alleles suggests that androgens contribute to colon carcinogenesis in a yet unknown way <a href="#pone.0033393-Catalano1" class="ref-tip">[61]</a>.</p>
<a id="article1.body1.sec4.sec3.p3" name="article1.body1.sec4.sec3.p3" class="link-target"></a><p>TBP (the TATA-binding protein) is a key eukaryotic transcription factor used by all three cellular RNA polymerases. Compared to normal colon epithelium, TBP expression is elevated in the case of human colon carcinomas. Both Ras-dependent and Ras-independent mechanisms mediate the increases of TBP expression in colon carcinoma cell lines. Thus, TBP may be a crucial component in dysregulated signaling for causing tumors <a href="#pone.0033393-Johnson1" class="ref-tip">[88]</a>.</p>
<a id="article1.body1.sec4.sec3.p4" name="article1.body1.sec4.sec3.p4" class="link-target"></a><p>MYOD1 promoter methylation occurs in various malignancies including CRC. MYOD1 promoter methylation was detectable in tumor and normal colorectal samples, but was significantly higher in tumor than in normal mucosa. Patients without MYOD1 hypermethylation showed significantly longer survival than those with hypermethylation. Therefore, MYOD1 hypermethylation plays an important role in CRC and may be a novel prognostic factor <a href="#pone.0033393-Hiranuma1" class="ref-tip">[89]</a>.</p>
<a id="article1.body1.sec4.sec3.p5" name="article1.body1.sec4.sec3.p5" class="link-target"></a><p>HMGCR (3-hydroxy-3-methylglutaryl coenzyme A reductase) is an enzyme that catalyzes the rate-limiting step of cholesterol biosynthesis. HMGCR alternative splicing of exon 13 is not only a biomarker, but also a determinant of statin efficacy, which is a class of cholesterol-lowering drugs that inhibit HMGCR. HMGCR was used not only for the treatment of hypercholesterolemia, but also as a chemopreventive agent for CRC <a href="#pone.0033393-Medina1" class="ref-tip">[90]</a>. A genetic test of HMGCR was utilized to determine in which patients cholesterol-lowering statin drugs might have the most benefit in reducing the risk of CRC. A recent research has found a genetic variant may affect the way of how statins control both colorectal cancer and cardiovascular disease risk <a href="#pone.0033393-Lipkin1" class="ref-tip">[91]</a>.</p>
<a id="article1.body1.sec4.sec3.p6" name="article1.body1.sec4.sec3.p6" class="link-target"></a><p>KLK3 (also known as prostate-specific antigen, PSA) is a kallikrein-like serine protease that is a widely used biomarker for prostate cancer <a href="#pone.0033393-Diamandis1" class="ref-tip">[92]</a>. In addition to prostate cancer, breast, colon, ovarian, liver and kidney tumors can also produce KLK3 <a href="#pone.0033393-Levesque1" class="ref-tip">[93]</a>. Recently, several other members of KLK family like KLK7 have shown promise as potential biomarkers for various cancers including colon cancer <a href="#pone.0033393-Borgono1" class="ref-tip">[94]</a>, <a href="#pone.0033393-Diamandis2" class="ref-tip">[95]</a>, <a href="#pone.0033393-Talieri1" class="ref-tip">[96]</a>. Thus, with the progress of research, KLK3 may become a biomarker for CRC as well.</p>
<a id="article1.body1.sec4.sec3.p7" name="article1.body1.sec4.sec3.p7" class="link-target"></a><p>CHD (Chromodomain helicase DNA-binding protein) is a regulator of the chromatin remodeling process. CHD4 expression was detected in gastric cancers and CRCs by immunohistochemistry. It has been reported that loss of CHD4 expression was observed in 56.4% of the gastric cancers and 55.7% of the CRCs. In addition, Frameshift mutation and loss of expression of CHD genes are common in gastric cancers and CRCs with MSI-H. These alterations might contribute to cancer pathogenesis by deregulating CHD-mediated chromatin remodeling <a href="#pone.0033393-Kim1" class="ref-tip">[97]</a>.</p>
<a id="article1.body1.sec4.sec3.p8" name="article1.body1.sec4.sec3.p8" class="link-target"></a><p>MSMB encodesβ-microsemino protein, which is a proposed biomarker for prostate cancer <a href="#pone.0033393-Thomas1" class="ref-tip">[98]</a>. Genome-wide association studies (GWAS) have identified a variant, rs10993994, on chromosome 10q11 which is associated with prostate cancer risk. So far, there is no report about MSMB in CRC etiology. However, the expression of MSMB was detected in colon epithelial cells by immunohistochemistry <a href="#pone.0033393-Ohkubo1" class="ref-tip">[99]</a>. Thus, it may be a potential biomarker for colorectal cancer diagnosis although it is remained to be verified.</p>
<a id="article1.body1.sec4.sec3.p9" name="article1.body1.sec4.sec3.p9" class="link-target"></a><p>FOXJ3 is a member of Human Forkhead-box (FOX) gene family. It has been shown that genetic and epigenetic changes of FOX family genes as well as alterations occurring in target genes of FOX transcription factors family could lead to human disease including carcinogenesis <a href="#pone.0033393-Katoh1" class="ref-tip">[100]</a>. Recently, Niittymaki et al. <a href="#pone.0033393-Niittymaki1" class="ref-tip">[101]</a> identified a SNP, rs2761880, locates in the binding site of FOXJ3 in CRC. It has been proposed that many of the predisposition loci for CRC are involved in control of gene expression by targeting transcription factor binding sites. In addition, oligonucleotide microarray analysis of distinct gene expression patterns in CRC tissues harboring BRAF and K-ras mutations has shown that FOXJ3 was identified by PAM (Prediction analysis of microarrays) and the jackknife (or leave-one-out) cross validation as candidate to distinguish the mutant groups <a href="#pone.0033393-Kim2" class="ref-tip">[102]</a>.</p>
<a id="article1.body1.sec4.sec3.p10" name="article1.body1.sec4.sec3.p10" class="link-target"></a><p>HDAC1 (Histone deacetylase 1) is involved in tumorigenesis through their regulation of cell proliferation, differentiation and survival. In cancer cells, HDAC1 represses the expression of tumor suppress genes such as p21/WAF1/CIP1 and Bax, leading to aberrant cell proliferation and cell viability <a href="#pone.0033393-Lagger1" class="ref-tip">[103]</a>. HDAC1 and HDAC3 are overexpressed in colon cancer cells and in primary colon cancer, and siRNA (small interfering RNA) mediated silencing of HDAC1 and HDAC3 in colon cancer cells induced apoptosis <a href="#pone.0033393-Thangaraju1" class="ref-tip">[104]</a>.</p>
<a id="article1.body1.sec4.sec3.p11" name="article1.body1.sec4.sec3.p11" class="link-target"></a><p>MEF2C (myocyte enhancer factor 2C) is a member of the MEF2 family of transcription factors. Recently, MEF2C was identified as a potential oncogenic transcription factor associated with CRC <a href="#pone.0033393-Nagaraj1" class="ref-tip">[24]</a>. Besides, it has been shown that MEF2C was hypermethylated. Also, it was indicated by the significantly down-regulated in colon cancer that MEF2C may play a role in CRC etiology <a href="#pone.0033393-Irizarry1" class="ref-tip">[105]</a>.</p>
<a id="article1.body1.sec4.sec3.p12" name="article1.body1.sec4.sec3.p12" class="link-target"></a><p>NR1H3 is a transcription factor involved in lipid homeostasis and inflammation. Recent evidences indicated that miRNAs can bind to the 3′untranslatedregions (UTRs) of mRNAs and regulates their translation. Genetic polymorphisms can locate in miRNA binding sites. Thus, miRNA regulation may be influenced by polymorphisms on the 3′UTRs. NR1H3 was identified as a candidate gene that harboring polymorphic in miRNA target sites which was associated with risk of sporadic CRC <a href="#pone.0033393-Landi1" class="ref-tip">[106]</a>. The specific relationship between NR1H3 and CRC remains to be further elucidated.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="Overlap between selected genes and known cancer genes as well as known CRC related genes"></a>
<h3>Overlap between selected genes and known cancer genes as well as known CRC related genes</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>Statistic test showed that the overlap between the 41 genes identified in our study and the 742 cancer-related genes we compiled was quite significant (p-value = 0.0001908). The KEGG analysis result of such 41 genes also implied that they were significantly enriched in cancer-related pathways (p-value = 8.00E-05). Taken together, it indicated that the 41 genes identified by us were closely associated with cancer. In addition, the overlaps of such 41 candidate genes with the previous (p-value = 0.01057) and recent (p-value = 0.002017) reported CRC biomarkers were significant. This suggested that the 41 candidate genes have the potential to be used as biomarkers for CRC diagnosis.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2" class="link-target"></a><p>In addition, we compared the 41 genes identified by us with the top 41 genes in mRMR feature list and the top 41 differentially expressed genes identified by the traditional t-test method of R language <a href="#pone.0033393-Team1" class="ref-tip">[107]</a>. See the <a href="#pone.0033393.s006">Supporting Information S6</a> for such three sets of 41 genes. As can be seen from there, the 41 genes identified by us contain 8 cancer genes, which is more than 4 (p-value = 0.03965, proportion test) and 2 (p-value = 4.923e-05, proportion test) cancer genes than those contained in the 41 genes identified by mRMR and the 41 genes identified by the t-test, respectively (<a href="#pone-0033393-t005"><strong>Table 5</strong></a>).</p>
<a class="link-target" id="pone-0033393-t005" name="pone-0033393-t005"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.t005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.t005" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.t005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.t005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 5. </span> The overlap between 41 genes identified from three different methods and 742 cancer genes.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.t005">
              https://doi.org/10.1371/journal.pone.0033393.t005</a></p></div></div>

<div id="section5" class="section toc-section"><a id="s4e" name="s4e" class="link-target" title="Functional similarity between selected genes and known CRC related genes"></a>
<h3>Functional similarity between selected genes and known CRC related genes</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>In this study, five gene sets were defined. The first gene set is our 41 selected genes. The second gene set is the top 41 mRMR genes. The third gene set is the top 41 t-test genes that have the smallest t-test p values. The second and third gene sets were from gene expression profiles alone. Our 41 gene were selected based on both gene expression profiles and protein interaction network. The fourth gene set is the 742 cancer genes mentioned above. The fifth gene set is the combined known CRC related genes of 742 cancer related genes, 438 genes from Sabates-Bellver's study <a href="#pone.0033393-SabatesBellver1" class="ref-tip">[23]</a>and 134 colorectal cancer related genes from Nagaraj's study <a href="#pone.0033393-Nagaraj1" class="ref-tip">[24]</a>. These five gene sets can be found in <span class="underline">the <a href="#pone.0033393.s006">Supporting Information S6</a>.</span></p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2" class="link-target"></a><p>To compare the functional similarity between our selected genes and the known CRC related genes, we constructed their functional profiles using the −log10 of the hypergeometric test p value on Gene Ontology (GO) terms <a href="#pone.0033393-Huang1" class="ref-tip">[1]</a>, <a href="#pone.0033393-Huang4" class="ref-tip">[5]</a>, <a href="#pone.0033393-Huang9" class="ref-tip">[108]</a>. Then we calculated the Pearson correlation coefficient of their functional profiles <a href="#pone.0033393-Huang1" class="ref-tip">[1]</a>, <a href="#pone.0033393-Huang10" class="ref-tip">[109]</a>. The functional similarities of the functional profiles for the five gene sets were shown in <a href="#pone-0033393-t006"><strong>Table 6</strong></a>. Our 41 genes had greater functional similarity with the cancer genes and the known CRC genes than the genes identified from gene expression profiles alone: top 41 mRMR genes and top 41 t-test genes. This suggests that the genes selected by our method are more reliable than the genes identified from the gene expression profiles alone. Combining the gene expression profiles and protein interaction network together can improve the identification of disease genes.</p>
<a class="link-target" id="pone-0033393-t006" name="pone-0033393-t006"></a><div class="figure" data-doi="10.1371/journal.pone.0033393.t006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0033393.t006" data-doi="info:doi/10.1371/journal.pone.0033393" data-uri="info:doi/10.1371/journal.pone.0033393.t006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0033393.t006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0033393.t006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0033393.t006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0033393.t006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 6. </span> The functional similarity between our 41 genes and known colorectal cancer genes.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0033393.t006">
              https://doi.org/10.1371/journal.pone.0033393.t006</a></p></div><a id="article1.body1.sec4.sec5.p3" name="article1.body1.sec4.sec5.p3" class="link-target"></a><p>The reason why our method can generate more reliable results is because that the shortest pathway approach integrated here is based on all the information of genes from database, text mining, etc. that is quite stable and can avoid the false positives. In contrast to this, the method based on the gene expression data can cause lots of false positives. It is anticipated that our method may become a useful tool, or at least play a complementary role to the existing method, for identifying colorectal cancer genes.</p>
<a id="article1.body1.sec4.sec5.p4" name="article1.body1.sec4.sec5.p4" class="link-target"></a><p>It is instructive to point out that our method may have some limitations. This is because some hub genes that may simultaneously interact with lots of other genes can also occur in our shortest path and the randomly selected shortest paths, such as TP53 and EP300. Nevertheless, our method can provide a p-value to evaluate the significance that can be used to distinguish the hubs in the network background.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4f" name="s4f" class="link-target" title="Conclusion"></a>
<h3>Conclusion</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>We proposed a novel method to identify cancer related genes. We applied this method on CRC and identified 41 genes which had the most potential to be biomarker for CRC early diagnose. Statistic test and KEGG analysis showed that the 41 candidate genes identified in our study are not only closely related to cancer but also have great potential to become biomarker for CRC diagnosis. In addition, the 41 candidate genes contain more cancer genes than the genes identified from gene expression profiles alone, and functional similarity analysis revealed that our genes had greater functional similarity with the reported CRC genes than the genes identified from gene expression profiles alone. We believe that our method may be helpful (or at least play a stimulative role) for predicting novel cancer related genes, and that it might have the potential applicability for the cancer research.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0033393"></div><div class="supplementary-material"><a name="pone.0033393.s001" id="pone.0033393.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0033393.s001">Supporting Information S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>The cancer-related gene list and the two colorectal cancer-related gene lists.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0033393.s001">https://doi.org/10.1371/journal.pone.0033393.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0033393.s002" id="pone.0033393.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0033393.s002">Supporting Information S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>The MaxRel features table and mRMR features table.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0033393.s002">https://doi.org/10.1371/journal.pone.0033393.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0033393.s003" id="pone.0033393.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0033393.s003">Supporting Information S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Feature numbers and the first order accuracy which the IFS curve plot was based on.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0033393.s003">https://doi.org/10.1371/journal.pone.0033393.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0033393.s004" id="pone.0033393.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0033393.s004">Supporting Information S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>The 15 shortest paths with the lowest cost presented with protein and gene, respectively.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0033393.s004">https://doi.org/10.1371/journal.pone.0033393.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0033393.s005" id="pone.0033393.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0033393.s005">Supporting Information S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>The overlap between the 41 candidate genes and the three other datasets and the corresponding Fisher's exact test.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0033393.s005">https://doi.org/10.1371/journal.pone.0033393.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0033393.s006" id="pone.0033393.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0033393.s006">Supporting Information S6. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Five gene sets.</strong> First gene set is our 41 selected genes. The second gene set is the top 41 mRMR genes. The third gene set is the top 41 t-test genes that have the smallest t-test p values. The second and third gene sets were from gene expression profiles alone. Our 41 gene were selected based on both gene expression profiles and protein interaction network. The fourth gene set is the 742 cancer genes. The fifth gene set is the combined known colorectal cancer related genes.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0033393.s006">https://doi.org/10.1371/journal.pone.0033393.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The authors are very much indebted to the Editor and Reviewers for their constructive comments, which were very helpful for strengthening the presentation of this paper.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: BL TH LL YC KCC. Performed the experiments: BL TH. Analyzed the data: BL. Contributed reagents/materials/analysis tools: BL. Wrote the paper: BL KCC.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0033393-Huang1" id="pone.0033393-Huang1" class="link-target"></a>Huang T, Chen L, Cai Y-D, Chou K-C (2011) Classification and analysis of regulatory pathways using graph property, biochemical and physicochemical property, and functional property. PLoS One  6: e25297.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTChenLCaiY-DChouK-C2011Classification%20and%20analysis%20of%20regulatory%20pathways%20using%20graph%20property%2C%20biochemical%20and%20physicochemical%20property%2C%20and%20functional%20property.PLoS%20One6e25297" data-title="Classification%20and%20analysis%20of%20regulatory%20pathways%20using%20graph%20property%2C%20biochemical%20and%20physicochemical%20property%2C%20and%20functional%20property." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Classification+and+analysis+of+regulatory+pathways+using+graph+property%2C+biochemical+and+physicochemical+property%2C+and+functional+property.+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0033393-Huang2" id="pone.0033393-Huang2" class="link-target"></a>Huang T, Cui W, Hu L, Feng K, Li YX, et al.  (2009) Prediction of pharmacological and xenobiotic responses to drugs based on time course gene expression profiles. PLoS One  4: e8126.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTCuiWHuLFengKLiYX2009Prediction%20of%20pharmacological%20and%20xenobiotic%20responses%20to%20drugs%20based%20on%20time%20course%20gene%20expression%20profiles.PLoS%20One4e8126" data-title="Prediction%20of%20pharmacological%20and%20xenobiotic%20responses%20to%20drugs%20based%20on%20time%20course%20gene%20expression%20profiles." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+pharmacological+and+xenobiotic+responses+to+drugs+based+on+time+course+gene+expression+profiles.+Huang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0033393-Cai1" id="pone.0033393-Cai1" class="link-target"></a>Cai YD, Huang T, Feng KY, Hu L, Xie L (2010) A Unified 35-Gene Signature for both Subtype Classification and Survival Prediction in Diffuse Large B-Cell Lymphomas. PLoS One  5:   <ul class="reflinks"><li><a href="#" data-author="Cai" data-cit="CaiYDHuangTFengKYHuLXieL2010A%20Unified%2035-Gene%20Signature%20for%20both%20Subtype%20Classification%20and%20Survival%20Prediction%20in%20Diffuse%20Large%20B-Cell%20Lymphomas.PLoS%20One5" data-title="A%20Unified%2035-Gene%20Signature%20for%20both%20Subtype%20Classification%20and%20Survival%20Prediction%20in%20Diffuse%20Large%20B-Cell%20Lymphomas." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+Unified+35-Gene+Signature+for+both+Subtype+Classification+and+Survival+Prediction+in+Diffuse+Large+B-Cell+Lymphomas.+Cai+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0033393-Huang3" id="pone.0033393-Huang3" class="link-target"></a>Huang T, Cai Y-D, Chen L, Hu L, Kong X-Y, et al.  (2011) Selection of reprogramming factors of induced pluripotent stem cells based on the protein interaction network and functional profiles. Protein &amp; Peptide Letters.   <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTCaiY-DChenLHuLKongX-Y2011Selection%20of%20reprogramming%20factors%20of%20induced%20pluripotent%20stem%20cells%20based%20on%20the%20protein%20interaction%20network%20and%20functional%20profiles.Protein%20%26%20Peptide%20Letters" data-title="Selection%20of%20reprogramming%20factors%20of%20induced%20pluripotent%20stem%20cells%20based%20on%20the%20protein%20interaction%20network%20and%20functional%20profiles." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Selection+of+reprogramming+factors+of+induced+pluripotent+stem+cells+based+on+the+protein+interaction+network+and+functional+profiles.+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0033393-Huang4" id="pone.0033393-Huang4" class="link-target"></a>Huang T, Shi XH, Wang P, He Z, Feng KY, et al.  (2010) Analysis and prediction of the metabolic stability of proteins based on their sequential features, subcellular locations and interaction networks. PLoS One  5: e10972.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTShiXHWangPHeZFengKY2010Analysis%20and%20prediction%20of%20the%20metabolic%20stability%20of%20proteins%20based%20on%20their%20sequential%20features%2C%20subcellular%20locations%20and%20interaction%20networks.PLoS%20One5e10972" data-title="Analysis%20and%20prediction%20of%20the%20metabolic%20stability%20of%20proteins%20based%20on%20their%20sequential%20features%2C%20subcellular%20locations%20and%20interaction%20networks." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+and+prediction+of+the+metabolic+stability+of+proteins+based+on+their+sequential+features%2C+subcellular+locations+and+interaction+networks.+Huang+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0033393-Chen1" id="pone.0033393-Chen1" class="link-target"></a>Chen L, Cai Y-D, Shi X-H, Huang T (2011) Analysis of metabolic pathway using hybrid properties. Protein &amp; Peptide Letters.   <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenLCaiY-DShiX-HHuangT2011Analysis%20of%20metabolic%20pathway%20using%20hybrid%20properties.Protein%20%26%20Peptide%20Letters" data-title="Analysis%20of%20metabolic%20pathway%20using%20hybrid%20properties." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+of+metabolic+pathway+using+hybrid+properties.+Chen+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0033393-Chen2" id="pone.0033393-Chen2" class="link-target"></a>Chen L, Cai Y-D, Huang T, Zeng W-M (2011) Prediction of metabolic pathway using graph property, chemical functional group and chemical structure. Current Bioinformatics.   <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenLCaiY-DHuangTZengW-M2011Prediction%20of%20metabolic%20pathway%20using%20graph%20property%2C%20chemical%20functional%20group%20and%20chemical%20structure.Current%20Bioinformatics" data-title="Prediction%20of%20metabolic%20pathway%20using%20graph%20property%2C%20chemical%20functional%20group%20and%20chemical%20structure." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+metabolic+pathway+using+graph+property%2C+chemical+functional+group+and+chemical+structure.+Chen+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0033393-Liu1" id="pone.0033393-Liu1" class="link-target"></a>Liu Q, Tan Y, Huang T, Ding G, Tu Z, et al.  (2010) TF-centered downstream gene set enrichment analysis: Inference of causal regulators by integrating TF-DNA interactions and protein post-translational modifications information. BMC Bioinformatics  11: S5.  <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuQTanYHuangTDingGTuZ2010TF-centered%20downstream%20gene%20set%20enrichment%20analysis%3A%20Inference%20of%20causal%20regulators%20by%20integrating%20TF-DNA%20interactions%20and%20protein%20post-translational%20modifications%20information.BMC%20Bioinformatics11S5" data-title="TF-centered%20downstream%20gene%20set%20enrichment%20analysis%3A%20Inference%20of%20causal%20regulators%20by%20integrating%20TF-DNA%20interactions%20and%20protein%20post-translational%20modifications%20information." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=TF-centered+downstream+gene+set+enrichment+analysis%3A+Inference+of+causal+regulators+by+integrating+TF-DNA+interactions+and+protein+post-translational+modifications+information.+Liu+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0033393-Nabieva1" id="pone.0033393-Nabieva1" class="link-target"></a>Nabieva E, Jim K, Agarwal A, Chazelle B, Singh M (2005) Whole-proteome prediction of protein function via graph-theoretic analysis of interaction maps. Bioinformatics  21: Suppl 1i302–310.  <ul class="reflinks"><li><a href="#" data-author="Nabieva" data-cit="NabievaEJimKAgarwalAChazelleBSinghM2005Whole-proteome%20prediction%20of%20protein%20function%20via%20graph-theoretic%20analysis%20of%20interaction%20maps.Bioinformatics21Suppl%201i302310" data-title="Whole-proteome%20prediction%20of%20protein%20function%20via%20graph-theoretic%20analysis%20of%20interaction%20maps." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Whole-proteome+prediction+of+protein+function+via+graph-theoretic+analysis+of+interaction+maps.+Nabieva+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0033393-Szklarczyk1" id="pone.0033393-Szklarczyk1" class="link-target"></a>Szklarczyk D, Franceschini A, Kuhn M, Simonovic M, Roth A, et al.  (2011) The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored. Nucleic Acids Res  39: D561–568.  <ul class="reflinks"><li><a href="#" data-author="Szklarczyk" data-cit="SzklarczykDFranceschiniAKuhnMSimonovicMRothA2011The%20STRING%20database%20in%202011%3A%20functional%20interaction%20networks%20of%20proteins%2C%20globally%20integrated%20and%20scored.Nucleic%20Acids%20Res39D561568" data-title="The%20STRING%20database%20in%202011%3A%20functional%20interaction%20networks%20of%20proteins%2C%20globally%20integrated%20and%20scored." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+STRING+database+in+2011%3A+functional+interaction+networks+of+proteins%2C+globally+integrated+and+scored.+Szklarczyk+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0033393-Sharan1" id="pone.0033393-Sharan1" class="link-target"></a>Sharan R, Ulitsky I, Shamir R (2007) Network-based prediction of protein function. Mol Syst Biol  3: 88.  <ul class="reflinks"><li><a href="#" data-author="Sharan" data-cit="SharanRUlitskyIShamirR2007Network-based%20prediction%20of%20protein%20function.Mol%20Syst%20Biol388" data-title="Network-based%20prediction%20of%20protein%20function." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Network-based+prediction+of+protein+function.+Sharan+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0033393-Bogdanov1" id="pone.0033393-Bogdanov1" class="link-target"></a>Bogdanov P, Singh AK (2010) Molecular function prediction using neighborhood features. IEEE/ACM Trans Comput Biol Bioinform  7: 208–217.  <ul class="reflinks"><li><a href="#" data-author="Bogdanov" data-cit="BogdanovPSinghAK2010Molecular%20function%20prediction%20using%20neighborhood%20features.IEEE%2FACM%20Trans%20Comput%20Biol%20Bioinform7208217" data-title="Molecular%20function%20prediction%20using%20neighborhood%20features." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+function+prediction+using+neighborhood+features.+Bogdanov+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0033393-Kourmpetis1" id="pone.0033393-Kourmpetis1" class="link-target"></a>Kourmpetis YA, van Dijk AD, Bink MC, van Ham RC, ter Braak CJ (2010) Bayesian Markov Random Field analysis for protein function prediction based on network data. PLoS One  5: e9293.  <ul class="reflinks"><li><a href="#" data-author="Kourmpetis" data-cit="KourmpetisYAvan%20DijkADBinkMCvan%20HamRCter%20BraakCJ2010Bayesian%20Markov%20Random%20Field%20analysis%20for%20protein%20function%20prediction%20based%20on%20network%20data.PLoS%20One5e9293" data-title="Bayesian%20Markov%20Random%20Field%20analysis%20for%20protein%20function%20prediction%20based%20on%20network%20data." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Bayesian+Markov+Random+Field+analysis+for+protein+function+prediction+based+on+network+data.+Kourmpetis+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0033393-Ng1" id="pone.0033393-Ng1" class="link-target"></a>Ng KL, Ciou JS, Huang CH (2010) Prediction of protein functions based on function-function correlation relations. Comput Biol Med  40: 300–305.  <ul class="reflinks"><li><a href="#" data-author="Ng" data-cit="NgKLCiouJSHuangCH2010Prediction%20of%20protein%20functions%20based%20on%20function-function%20correlation%20relations.Comput%20Biol%20Med40300305" data-title="Prediction%20of%20protein%20functions%20based%20on%20function-function%20correlation%20relations." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+protein+functions+based+on+function-function+correlation+relations.+Ng+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0033393-Karaoz1" id="pone.0033393-Karaoz1" class="link-target"></a>Karaoz U, Murali TM, Letovsky S, Zheng Y, Ding C, et al.  (2004) Whole-genome annotation by using evidence integration in functional-linkage networks. Proc Natl Acad Sci U S A  101: 2888–2893.  <ul class="reflinks"><li><a href="#" data-author="Karaoz" data-cit="KaraozUMuraliTMLetovskySZhengYDingC2004Whole-genome%20annotation%20by%20using%20evidence%20integration%20in%20functional-linkage%20networks.Proc%20Natl%20Acad%20Sci%20U%20S%20A10128882893" data-title="Whole-genome%20annotation%20by%20using%20evidence%20integration%20in%20functional-linkage%20networks." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Whole-genome+annotation+by+using+evidence+integration+in+functional-linkage+networks.+Karaoz+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0033393-Letovsky1" id="pone.0033393-Letovsky1" class="link-target"></a>Letovsky S, Kasif S (2003) Predicting protein function from protein/protein interaction data: a probabilistic approach. Bioinformatics  19: Suppl 1i197–204.  <ul class="reflinks"><li><a href="#" data-author="Letovsky" data-cit="LetovskySKasifS2003Predicting%20protein%20function%20from%20protein%2Fprotein%20interaction%20data%3A%20a%20probabilistic%20approach.Bioinformatics19Suppl%201i197204" data-title="Predicting%20protein%20function%20from%20protein%2Fprotein%20interaction%20data%3A%20a%20probabilistic%20approach." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Predicting+protein+function+from+protein%2Fprotein+interaction+data%3A+a+probabilistic+approach.+Letovsky+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0033393-Peng1" id="pone.0033393-Peng1" class="link-target"></a>Peng H, Long F, Ding C (2005) Feature selection based on mutual information: criteria of max-dependency, max-relevance, and min-redundancy. IEEE Trans Pattern Anal Mach Intell  27: 1226–1238.  <ul class="reflinks"><li><a href="#" data-author="Peng" data-cit="PengHLongFDingC2005Feature%20selection%20based%20on%20mutual%20information%3A%20criteria%20of%20max-dependency%2C%20max-relevance%2C%20and%20min-redundancy.IEEE%20Trans%20Pattern%20Anal%20Mach%20Intell2712261238" data-title="Feature%20selection%20based%20on%20mutual%20information%3A%20criteria%20of%20max-dependency%2C%20max-relevance%2C%20and%20min-redundancy." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Feature+selection+based+on+mutual+information%3A+criteria+of+max-dependency%2C+max-relevance%2C+and+min-redundancy.+Peng+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0033393-Dijkstra1" id="pone.0033393-Dijkstra1" class="link-target"></a>Dijkstra E (1959) A Note on Two Problems in Connection with Graphs. Numerische Mathematik  1: 269–271.  <ul class="reflinks"><li><a href="#" data-author="Dijkstra" data-cit="DijkstraE1959A%20Note%20on%20Two%20Problems%20in%20Connection%20with%20Graphs.Numerische%20Mathematik1269271" data-title="A%20Note%20on%20Two%20Problems%20in%20Connection%20with%20Graphs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+Note+on+Two+Problems+in+Connection+with+Graphs.+Dijkstra+1959" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0033393-Hinoue1" id="pone.0033393-Hinoue1" class="link-target"></a>Hinoue T, Weisenberger DJ, Lange CP, Shen H, Byun HM, et al.  (2011) Genome-scale analysis of aberrant DNA methylation in colorectal cancer. Genome Res.   <ul class="reflinks"><li><a href="#" data-author="Hinoue" data-cit="HinoueTWeisenbergerDJLangeCPShenHByunHM2011Genome-scale%20analysis%20of%20aberrant%20DNA%20methylation%20in%20colorectal%20cancer.Genome%20Res" data-title="Genome-scale%20analysis%20of%20aberrant%20DNA%20methylation%20in%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-scale+analysis+of+aberrant+DNA+methylation+in+colorectal+cancer.+Hinoue+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0033393-Chou1" id="pone.0033393-Chou1" class="link-target"></a>Chou KC (2011) Some remarks on protein attribute prediction and pseudo amino acid composition (50th Anniversary Year Review). Journal of Theoretical Biology  273: 236–247.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKC2011Some%20remarks%20on%20protein%20attribute%20prediction%20and%20pseudo%20amino%20acid%20composition%20%2850th%20Anniversary%20Year%20Review%29.Journal%20of%20Theoretical%20Biology273236247" data-title="Some%20remarks%20on%20protein%20attribute%20prediction%20and%20pseudo%20amino%20acid%20composition%20%2850th%20Anniversary%20Year%20Review%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Some+remarks+on+protein+attribute+prediction+and+pseudo+amino+acid+composition+%2850th+Anniversary+Year+Review%29.+Chou+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0033393-Huret1" id="pone.0033393-Huret1" class="link-target"></a>Huret JL, Dessen P, Bernheim A (2003) Atlas of Genetics and Cytogenetics in Oncology and Haematology, year 2003. Nucleic Acids Res  31: 272–274.  <ul class="reflinks"><li><a href="#" data-author="Huret" data-cit="HuretJLDessenPBernheimA2003Atlas%20of%20Genetics%20and%20Cytogenetics%20in%20Oncology%20and%20Haematology%2C%20year%202003.Nucleic%20Acids%20Res31272274" data-title="Atlas%20of%20Genetics%20and%20Cytogenetics%20in%20Oncology%20and%20Haematology%2C%20year%202003." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Atlas+of+Genetics+and+Cytogenetics+in+Oncology+and+Haematology%2C+year+2003.+Huret+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0033393-KeshavaPrasad1" id="pone.0033393-KeshavaPrasad1" class="link-target"></a>Keshava Prasad TS, Goel R, Kandasamy K, Keerthikumar S, Kumar S, et al.  (2009) Human Protein Reference Database–2009 update. Nucleic Acids Res  37: D767–772.  <ul class="reflinks"><li><a href="#" data-author="Keshava%20Prasad" data-cit="Keshava%20PrasadTSGoelRKandasamyKKeerthikumarSKumarS2009Human%20Protein%20Reference%20Database%E2%80%932009%20update.Nucleic%20Acids%20Res37D767772" data-title="Human%20Protein%20Reference%20Database%E2%80%932009%20update." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+Protein+Reference+Database%E2%80%932009+update.+Keshava+Prasad+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0033393-SabatesBellver1" id="pone.0033393-SabatesBellver1" class="link-target"></a>Sabates-Bellver J, Van der Flier LG, de Palo M, Cattaneo E, Maake C, et al.  (2007) Transcriptome profile of human colorectal adenomas. Mol Cancer Res  5: 1263–1275.  <ul class="reflinks"><li><a href="#" data-author="Sabates-Bellver" data-cit="Sabates-BellverJVan%20der%20FlierLGde%20PaloMCattaneoEMaakeC2007Transcriptome%20profile%20of%20human%20colorectal%20adenomas.Mol%20Cancer%20Res512631275" data-title="Transcriptome%20profile%20of%20human%20colorectal%20adenomas." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptome+profile+of+human+colorectal+adenomas.+Sabates-Bellver+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0033393-Nagaraj1" id="pone.0033393-Nagaraj1" class="link-target"></a>Nagaraj SH, Reverter A (2011) A Boolean-based systems biology approach to predict novel genes associated with cancer: Application to colorectal cancer. BMC Syst Biol  5: 35.  <ul class="reflinks"><li><a href="#" data-author="Nagaraj" data-cit="NagarajSHReverterA2011A%20Boolean-based%20systems%20biology%20approach%20to%20predict%20novel%20genes%20associated%20with%20cancer%3A%20Application%20to%20colorectal%20cancer.BMC%20Syst%20Biol535" data-title="A%20Boolean-based%20systems%20biology%20approach%20to%20predict%20novel%20genes%20associated%20with%20cancer%3A%20Application%20to%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+Boolean-based+systems+biology+approach+to+predict+novel+genes+associated+with+cancer%3A+Application+to+colorectal+cancer.+Nagaraj+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0033393-Friedman1" id="pone.0033393-Friedman1" class="link-target"></a>Friedman JH, Baskett F, Shustek LJ (1975) An algorithm for finding nearest neighbors. IEEE Transaction on Information Theory  C-24: 1000–1006.  <ul class="reflinks"><li><a href="#" data-author="Friedman" data-cit="FriedmanJHBaskettFShustekLJ1975An%20algorithm%20for%20finding%20nearest%20neighbors.IEEE%20Transaction%20on%20Information%20TheoryC-2410001006" data-title="An%20algorithm%20for%20finding%20nearest%20neighbors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+algorithm+for+finding+nearest+neighbors.+Friedman+1975" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0033393-Denoeux1" id="pone.0033393-Denoeux1" class="link-target"></a>Denoeux T (1995) A k-nearest neighbor classification rule based on Dempster-Shafer theory. IEEE Transactions on Systems, Man and Cybernetics  25: 804–813.  <ul class="reflinks"><li><a href="#" data-author="Denoeux" data-cit="DenoeuxT1995A%20k-nearest%20neighbor%20classification%20rule%20based%20on%20Dempster-Shafer%20theory.IEEE%20Transactions%20on%20Systems%2C%20Man%20and%20Cybernetics25804813" data-title="A%20k-nearest%20neighbor%20classification%20rule%20based%20on%20Dempster-Shafer%20theory." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+k-nearest+neighbor+classification+rule+based+on+Dempster-Shafer+theory.+Denoeux+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0033393-Hu1" id="pone.0033393-Hu1" class="link-target"></a>Hu L, Huang T, Shi X, Lu WC, Cai YD, et al.  (2011) Predicting functions of proteins in mouse based on weighted protein-protein interaction network and protein hybrid properties. PLoS ONE  6: e14556.  <ul class="reflinks"><li><a href="#" data-author="Hu" data-cit="HuLHuangTShiXLuWCCaiYD2011Predicting%20functions%20of%20proteins%20in%20mouse%20based%20on%20weighted%20protein-protein%20interaction%20network%20and%20protein%20hybrid%20properties.PLoS%20ONE6e14556" data-title="Predicting%20functions%20of%20proteins%20in%20mouse%20based%20on%20weighted%20protein-protein%20interaction%20network%20and%20protein%20hybrid%20properties." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Predicting+functions+of+proteins+in+mouse+based+on+weighted+protein-protein+interaction+network+and+protein+hybrid+properties.+Hu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0033393-Hu2" id="pone.0033393-Hu2" class="link-target"></a>Hu L, Zheng L, Wang Z, Li B, Liu L (2011) Using pseudo amino Acid composition to predict protease families by incorporating a series of protein biological features. Protein and Peptide Letters  18: 552–558.  <ul class="reflinks"><li><a href="#" data-author="Hu" data-cit="HuLZhengLWangZLiBLiuL2011Using%20pseudo%20amino%20Acid%20composition%20to%20predict%20protease%20families%20by%20incorporating%20a%20series%20of%20protein%20biological%20features.Protein%20and%20Peptide%20Letters18552558" data-title="Using%20pseudo%20amino%20Acid%20composition%20to%20predict%20protease%20families%20by%20incorporating%20a%20series%20of%20protein%20biological%20features." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Using+pseudo+amino+Acid+composition+to+predict+protease+families+by+incorporating+a+series+of+protein+biological+features.+Hu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0033393-Yang1" id="pone.0033393-Yang1" class="link-target"></a>Yang XY, Shi XH, Meng X, Li XL, Lin K, et al.  (2010) Classification of transcription factors using protein primary structure. Protein &amp; Peptide Letters  17: 899–908.  <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangXYShiXHMengXLiXLLinK2010Classification%20of%20transcription%20factors%20using%20protein%20primary%20structure.Protein%20%26%20Peptide%20Letters17899908" data-title="Classification%20of%20transcription%20factors%20using%20protein%20primary%20structure." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Classification+of+transcription+factors+using+protein+primary+structure.+Yang+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0033393-Huang5" id="pone.0033393-Huang5" class="link-target"></a>Huang T, Chen L, Cai YD, Chou KC (2011) Classification and analysis of regulatory pathways using graph property, biochemical and physicochemical property, and functional property. PLoS ONE  6: e25297.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTChenLCaiYDChouKC2011Classification%20and%20analysis%20of%20regulatory%20pathways%20using%20graph%20property%2C%20biochemical%20and%20physicochemical%20property%2C%20and%20functional%20property.PLoS%20ONE6e25297" data-title="Classification%20and%20analysis%20of%20regulatory%20pathways%20using%20graph%20property%2C%20biochemical%20and%20physicochemical%20property%2C%20and%20functional%20property." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Classification+and+analysis+of+regulatory+pathways+using+graph+property%2C+biochemical+and+physicochemical+property%2C+and+functional+property.+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0033393-Hu3" id="pone.0033393-Hu3" class="link-target"></a>Hu LL, Huang T, Cai YD, Chou KC (2011) Prediction of Body Fluids where Proteins are Secreted into Based on Protein Interaction Network. PLoS One  6: e22989.  <ul class="reflinks"><li><a href="#" data-author="Hu" data-cit="HuLLHuangTCaiYDChouKC2011Prediction%20of%20Body%20Fluids%20where%20Proteins%20are%20Secreted%20into%20Based%20on%20Protein%20Interaction%20Network.PLoS%20One6e22989" data-title="Prediction%20of%20Body%20Fluids%20where%20Proteins%20are%20Secreted%20into%20Based%20on%20Protein%20Interaction%20Network." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+Body+Fluids+where+Proteins+are+Secreted+into+Based+on+Protein+Interaction+Network.+Hu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0033393-Huang6" id="pone.0033393-Huang6" class="link-target"></a>Huang T, Niu S, Xu Z, Huang Y, Kong X, et al.  (2011) Predicting Transcriptional Activity of Multiple Site p53 Mutants Based on Hybrid Properties. PLoS ONE  6: e22940.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTNiuSXuZHuangYKongX2011Predicting%20Transcriptional%20Activity%20of%20Multiple%20Site%20p53%20Mutants%20Based%20on%20Hybrid%20Properties.PLoS%20ONE6e22940" data-title="Predicting%20Transcriptional%20Activity%20of%20Multiple%20Site%20p53%20Mutants%20Based%20on%20Hybrid%20Properties." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Predicting+Transcriptional+Activity+of+Multiple+Site+p53+Mutants+Based+on+Hybrid+Properties.+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0033393-Huang7" id="pone.0033393-Huang7" class="link-target"></a>Huang T, Shi XH, Wang P, He Z, Feng KY, et al.  (2010) Analysis and prediction of the metabolic stability of proteins based on their sequential features, subcellular locations and interaction networks. PLoS ONE  5: e10972.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTShiXHWangPHeZFengKY2010Analysis%20and%20prediction%20of%20the%20metabolic%20stability%20of%20proteins%20based%20on%20their%20sequential%20features%2C%20subcellular%20locations%20and%20interaction%20networks.PLoS%20ONE5e10972" data-title="Analysis%20and%20prediction%20of%20the%20metabolic%20stability%20of%20proteins%20based%20on%20their%20sequential%20features%2C%20subcellular%20locations%20and%20interaction%20networks." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+and+prediction+of+the+metabolic+stability+of+proteins+based+on+their+sequential+features%2C+subcellular+locations+and+interaction+networks.+Huang+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0033393-Cai2" id="pone.0033393-Cai2" class="link-target"></a>Cai Y, Huang T, Hu L, Shi X, Xie L, et al.  (2011) Prediction of lysine ubiquitination with mRMR feature selection and analysis. Amino Acids.   <ul class="reflinks"><li><a href="#" data-author="Cai" data-cit="CaiYHuangTHuLShiXXieL2011Prediction%20of%20lysine%20ubiquitination%20with%20mRMR%20feature%20selection%20and%20analysis.Amino%20Acids" data-title="Prediction%20of%20lysine%20ubiquitination%20with%20mRMR%20feature%20selection%20and%20analysis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+lysine+ubiquitination+with+mRMR+feature+selection+and+analysis.+Cai+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0033393-Chou2" id="pone.0033393-Chou2" class="link-target"></a>Chou KC, Shen HB (2007) Review: Recent progresses in protein subcellular location prediction. Analytical Biochemistry  370: 1–16.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCShenHB2007Review%3A%20Recent%20progresses%20in%20protein%20subcellular%20location%20prediction.Analytical%20Biochemistry370116" data-title="Review%3A%20Recent%20progresses%20in%20protein%20subcellular%20location%20prediction." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Review%3A+Recent+progresses+in+protein+subcellular+location+prediction.+Chou+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0033393-Chou3" id="pone.0033393-Chou3" class="link-target"></a>Chou KC, Zhang CT (1995) Review: Prediction of protein structural classes. Critical Reviews in Biochemistry and Molecular Biology  30: 275–349.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCZhangCT1995Review%3A%20Prediction%20of%20protein%20structural%20classes.Critical%20Reviews%20in%20Biochemistry%20and%20Molecular%20Biology30275349" data-title="Review%3A%20Prediction%20of%20protein%20structural%20classes." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Review%3A+Prediction+of+protein+structural+classes.+Chou+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0033393-Chou4" id="pone.0033393-Chou4" class="link-target"></a>Chou KC, Shen HB (2008) Cell-PLoc: A package of Web servers for predicting subcellular localization of proteins in various organisms (updated version: Cell-PLoc 2.0: An improved package of web-servers for predicting subcellular localization of proteins in various organisms, Natural Science, 2010, 2, 1090–1103). Nature Protocols  3: 153–162.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCShenHB2008Cell-PLoc%3A%20A%20package%20of%20Web%20servers%20for%20predicting%20subcellular%20localization%20of%20proteins%20in%20various%20organisms%20%28updated%20version%3A%20Cell-PLoc%202.0%3A%20An%20improved%20package%20of%20web-servers%20for%20predicting%20subcellular%20localization%20of%20proteins%20in%20various%20organisms%2C%20Natural%20Science%2C%202010%2C%202%2C%201090%E2%80%931103%29.Nature%20Protocols3153162" data-title="Cell-PLoc%3A%20A%20package%20of%20Web%20servers%20for%20predicting%20subcellular%20localization%20of%20proteins%20in%20various%20organisms%20%28updated%20version%3A%20Cell-PLoc%202.0%3A%20An%20improved%20package%20of%20web-servers%20for%20predicting%20subcellular%20localization%20of%20proteins%20in%20various%20organisms%2C%20Natural%20Science%2C%202010%2C%202%2C%201090%E2%80%931103%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cell-PLoc%3A+A+package+of+Web+servers+for+predicting+subcellular+localization+of+proteins+in+various+organisms+%28updated+version%3A+Cell-PLoc+2.0%3A+An+improved+package+of+web-servers+for+predicting+subcellular+localization+of+proteins+in+various+organisms%2C+Natural+Science%2C+2010%2C+2%2C+1090%E2%80%931103%29.+Chou+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0033393-Esmaeili1" id="pone.0033393-Esmaeili1" class="link-target"></a>Esmaeili M, Mohabatkar H, Mohsenzadeh S (2010) Using the concept of Chou's pseudo amino acid composition for risk type prediction of human papillomaviruses. Journal of Theoretical Biology  263: 203–209.  <ul class="reflinks"><li><a href="#" data-author="Esmaeili" data-cit="EsmaeiliMMohabatkarHMohsenzadehS2010Using%20the%20concept%20of%20Chou%27s%20pseudo%20amino%20acid%20composition%20for%20risk%20type%20prediction%20of%20human%20papillomaviruses.Journal%20of%20Theoretical%20Biology263203209" data-title="Using%20the%20concept%20of%20Chou%27s%20pseudo%20amino%20acid%20composition%20for%20risk%20type%20prediction%20of%20human%20papillomaviruses." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Using+the+concept+of+Chou%27s+pseudo+amino+acid+composition+for+risk+type+prediction+of+human+papillomaviruses.+Esmaeili+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0033393-Georgiou1" id="pone.0033393-Georgiou1" class="link-target"></a>Georgiou DN, Karakasidis TE, Nieto JJ, Torres A (2009) Use of fuzzy clustering technique and matrices to classify amino acids and its impact to Chou's pseudo amino acid composition. Journal of Theoretical Biology  257: 17–26.  <ul class="reflinks"><li><a href="#" data-author="Georgiou" data-cit="GeorgiouDNKarakasidisTENietoJJTorresA2009Use%20of%20fuzzy%20clustering%20technique%20and%20matrices%20to%20classify%20amino%20acids%20and%20its%20impact%20to%20Chou%27s%20pseudo%20amino%20acid%20composition.Journal%20of%20Theoretical%20Biology2571726" data-title="Use%20of%20fuzzy%20clustering%20technique%20and%20matrices%20to%20classify%20amino%20acids%20and%20its%20impact%20to%20Chou%27s%20pseudo%20amino%20acid%20composition." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Use+of+fuzzy+clustering+technique+and+matrices+to+classify+amino+acids+and+its+impact+to+Chou%27s+pseudo+amino+acid+composition.+Georgiou+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0033393-Gu1" id="pone.0033393-Gu1" class="link-target"></a>Gu Q, Ding YS, Zhang TL (2010) Prediction of G-Protein-Coupled Receptor Classes in Low Homology Using Chou's Pseudo Amino Acid Composition with Approximate Entropy and Hydrophobicity Patterns. Protein &amp; Peptide Letters  17: 559–567.  <ul class="reflinks"><li><a href="#" data-author="Gu" data-cit="GuQDingYSZhangTL2010Prediction%20of%20G-Protein-Coupled%20Receptor%20Classes%20in%20Low%20Homology%20Using%20Chou%27s%20Pseudo%20Amino%20Acid%20Composition%20with%20Approximate%20Entropy%20and%20Hydrophobicity%20Patterns.Protein%20%26%20Peptide%20Letters17559567" data-title="Prediction%20of%20G-Protein-Coupled%20Receptor%20Classes%20in%20Low%20Homology%20Using%20Chou%27s%20Pseudo%20Amino%20Acid%20Composition%20with%20Approximate%20Entropy%20and%20Hydrophobicity%20Patterns." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+G-Protein-Coupled+Receptor+Classes+in+Low+Homology+Using+Chou%27s+Pseudo+Amino+Acid+Composition+with+Approximate+Entropy+and+Hydrophobicity+Patterns.+Gu+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0033393-Chou5" id="pone.0033393-Chou5" class="link-target"></a>Chou KC, Wu ZC, Xiao X (2012) iLoc-Hum: Using accumulation-label scale to predict subcellular locations of human proteins with both single and multiple sites. Molecular Biosystems  8: 629–641.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCWuZCXiaoX2012iLoc-Hum%3A%20Using%20accumulation-label%20scale%20to%20predict%20subcellular%20locations%20of%20human%20proteins%20with%20both%20single%20and%20multiple%20sites.Molecular%20Biosystems8629641" data-title="iLoc-Hum%3A%20Using%20accumulation-label%20scale%20to%20predict%20subcellular%20locations%20of%20human%20proteins%20with%20both%20single%20and%20multiple%20sites." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=iLoc-Hum%3A+Using+accumulation-label+scale+to+predict+subcellular+locations+of+human+proteins+with+both+single+and+multiple+sites.+Chou+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0033393-Mohabatkar1" id="pone.0033393-Mohabatkar1" class="link-target"></a>Mohabatkar H (2010) Prediction of cyclin proteins using Chou's pseudo amino acid composition. Protein &amp; Peptide Letters  17: 1207–1214.  <ul class="reflinks"><li><a href="#" data-author="Mohabatkar" data-cit="MohabatkarH2010Prediction%20of%20cyclin%20proteins%20using%20Chou%27s%20pseudo%20amino%20acid%20composition.Protein%20%26%20Peptide%20Letters1712071214" data-title="Prediction%20of%20cyclin%20proteins%20using%20Chou%27s%20pseudo%20amino%20acid%20composition." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+cyclin+proteins+using+Chou%27s+pseudo+amino+acid+composition.+Mohabatkar+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0033393-Xiao1" id="pone.0033393-Xiao1" class="link-target"></a>Xiao X, Wu ZC, Chou KC (2011) A multi-label classifier for predicting the subcellular localization of gram-negative bacterial proteins with both single and multiple sites. PLoS ONE  6: e20592.  <ul class="reflinks"><li><a href="#" data-author="Xiao" data-cit="XiaoXWuZCChouKC2011A%20multi-label%20classifier%20for%20predicting%20the%20subcellular%20localization%20of%20gram-negative%20bacterial%20proteins%20with%20both%20single%20and%20multiple%20sites.PLoS%20ONE6e20592" data-title="A%20multi-label%20classifier%20for%20predicting%20the%20subcellular%20localization%20of%20gram-negative%20bacterial%20proteins%20with%20both%20single%20and%20multiple%20sites." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+multi-label+classifier+for+predicting+the+subcellular+localization+of+gram-negative+bacterial+proteins+with+both+single+and+multiple+sites.+Xiao+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0033393-Wu1" id="pone.0033393-Wu1" class="link-target"></a>Wu ZC, Xiao X, Chou KC (2011) iLoc-Plant: a multi-label classifier for predicting the subcellular localization of plant proteins with both single and multiple sites. Molecular Bio Systems  7: 3287–3297.  <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuZCXiaoXChouKC2011iLoc-Plant%3A%20a%20multi-label%20classifier%20for%20predicting%20the%20subcellular%20localization%20of%20plant%20proteins%20with%20both%20single%20and%20multiple%20sites.Molecular%20Bio%20Systems732873297" data-title="iLoc-Plant%3A%20a%20multi-label%20classifier%20for%20predicting%20the%20subcellular%20localization%20of%20plant%20proteins%20with%20both%20single%20and%20multiple%20sites." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=iLoc-Plant%3A+a+multi-label+classifier+for+predicting+the+subcellular+localization+of+plant+proteins+with+both+single+and+multiple+sites.+Wu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0033393-Mohabatkar2" id="pone.0033393-Mohabatkar2" class="link-target"></a>Mohabatkar H, Mohammad Beigi M, Esmaeili A (2011) Prediction of GABA(A) receptor proteins using the concept of Chou's pseudo-amino acid composition and support vector machine. Journal of Theoretical Biology  281: 18–23.  <ul class="reflinks"><li><a href="#" data-author="Mohabatkar" data-cit="MohabatkarHMohammad%20BeigiMEsmaeiliA2011Prediction%20of%20GABA%28A%29%20receptor%20proteins%20using%20the%20concept%20of%20Chou%27s%20pseudo-amino%20acid%20composition%20and%20support%20vector%20machine.Journal%20of%20Theoretical%20Biology2811823" data-title="Prediction%20of%20GABA%28A%29%20receptor%20proteins%20using%20the%20concept%20of%20Chou%27s%20pseudo-amino%20acid%20composition%20and%20support%20vector%20machine." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+GABA%28A%29+receptor+proteins+using+the+concept+of+Chou%27s+pseudo-amino+acid+composition+and+support+vector+machine.+Mohabatkar+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0033393-Chou6" id="pone.0033393-Chou6" class="link-target"></a>Chou KC, Wu ZC, Xiao X (2011) iLoc-Euk: A Multi-Label Classifier for Predicting the Subcellular Localization of Singleplex and Multiplex Eukaryotic Proteins. PLoS One  6: e18258.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCWuZCXiaoX2011iLoc-Euk%3A%20A%20Multi-Label%20Classifier%20for%20Predicting%20the%20Subcellular%20Localization%20of%20Singleplex%20and%20Multiplex%20Eukaryotic%20Proteins.PLoS%20One6e18258" data-title="iLoc-Euk%3A%20A%20Multi-Label%20Classifier%20for%20Predicting%20the%20Subcellular%20Localization%20of%20Singleplex%20and%20Multiplex%20Eukaryotic%20Proteins." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=iLoc-Euk%3A+A+Multi-Label+Classifier+for+Predicting+the+Subcellular+Localization+of+Singleplex+and+Multiplex+Eukaryotic+Proteins.+Chou+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0033393-Huang8" id="pone.0033393-Huang8" class="link-target"></a>Huang T, Cui W, He ZS, Hu L, Liu F, et al.  (2009) Functional association between influenza A (H1N1) virus and human. Biochem Biophys Res Commun  390: 1111–1113.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTCuiWHeZSHuLLiuF2009Functional%20association%20between%20influenza%20A%20%28H1N1%29%20virus%20and%20human.Biochem%20Biophys%20Res%20Commun39011111113" data-title="Functional%20association%20between%20influenza%20A%20%28H1N1%29%20virus%20and%20human." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Functional+association+between+influenza+A+%28H1N1%29+virus+and+human.+Huang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0033393-Chou7" id="pone.0033393-Chou7" class="link-target"></a>Chou KC, Forsen S (1980) Graphical rules for enzyme-catalyzed rate laws. Biochemical Journal  187: 829–835.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCForsenS1980Graphical%20rules%20for%20enzyme-catalyzed%20rate%20laws.Biochemical%20Journal187829835" data-title="Graphical%20rules%20for%20enzyme-catalyzed%20rate%20laws." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Graphical+rules+for+enzyme-catalyzed+rate+laws.+Chou+1980" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0033393-Zhou1" id="pone.0033393-Zhou1" class="link-target"></a>Zhou GP, Deng MH (1984) An extension of Chou's graphic rules for deriving enzyme kinetic equations to systems involving parallel reaction pathways. Biochemical Journal  222: 169–176.  <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouGPDengMH1984An%20extension%20of%20Chou%27s%20graphic%20rules%20for%20deriving%20enzyme%20kinetic%20equations%20to%20systems%20involving%20parallel%20reaction%20pathways.Biochemical%20Journal222169176" data-title="An%20extension%20of%20Chou%27s%20graphic%20rules%20for%20deriving%20enzyme%20kinetic%20equations%20to%20systems%20involving%20parallel%20reaction%20pathways." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+extension+of+Chou%27s+graphic+rules+for+deriving+enzyme+kinetic+equations+to+systems+involving+parallel+reaction+pathways.+Zhou+1984" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0033393-Chou8" id="pone.0033393-Chou8" class="link-target"></a>Chou KC (1989) Graphic rules in steady and non-steady enzyme kinetics. Journal of Biological Chemistry  264: 12074–12079.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKC1989Graphic%20rules%20in%20steady%20and%20non-steady%20enzyme%20kinetics.Journal%20of%20Biological%20Chemistry2641207412079" data-title="Graphic%20rules%20in%20steady%20and%20non-steady%20enzyme%20kinetics." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Graphic+rules+in+steady+and+non-steady+enzyme+kinetics.+Chou+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0033393-Chou9" id="pone.0033393-Chou9" class="link-target"></a>Chou KC (1990) Review: Applications of graph theory to enzyme kinetics and protein folding kinetics. Steady and non-steady state systems. Biophysical Chemistry  35: 1–24.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKC1990Review%3A%20Applications%20of%20graph%20theory%20to%20enzyme%20kinetics%20and%20protein%20folding%20kinetics.%20Steady%20and%20non-steady%20state%20systems.Biophysical%20Chemistry35124" data-title="Review%3A%20Applications%20of%20graph%20theory%20to%20enzyme%20kinetics%20and%20protein%20folding%20kinetics.%20Steady%20and%20non-steady%20state%20systems." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Review%3A+Applications+of+graph+theory+to+enzyme+kinetics+and+protein+folding+kinetics.+Steady+and+non-steady+state+systems.+Chou+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0033393-Althaus1" id="pone.0033393-Althaus1" class="link-target"></a>Althaus IW, Gonzales AJ, Chou JJ, Diebel MR, Chou KC, et al.  (1993) The quinoline U-78036 is a potent inhibitor of HIV-1 reverse transcriptase. Journal of Biological Chemistry  268: 14875–14880.  <ul class="reflinks"><li><a href="#" data-author="Althaus" data-cit="AlthausIWGonzalesAJChouJJDiebelMRChouKC1993The%20quinoline%20U-78036%20is%20a%20potent%20inhibitor%20of%20HIV-1%20reverse%20transcriptase.Journal%20of%20Biological%20Chemistry2681487514880" data-title="The%20quinoline%20U-78036%20is%20a%20potent%20inhibitor%20of%20HIV-1%20reverse%20transcriptase." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+quinoline+U-78036+is+a+potent+inhibitor+of+HIV-1+reverse+transcriptase.+Althaus+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0033393-Chou10" id="pone.0033393-Chou10" class="link-target"></a>Chou KC, Kezdy FJ, Reusser F (1994) Review: Steady-state inhibition kinetics of processive nucleic acid polymerases and nucleases. Analytical Biochemistry  221: 217–230.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCKezdyFJReusserF1994Review%3A%20Steady-state%20inhibition%20kinetics%20of%20processive%20nucleic%20acid%20polymerases%20and%20nucleases.Analytical%20Biochemistry221217230" data-title="Review%3A%20Steady-state%20inhibition%20kinetics%20of%20processive%20nucleic%20acid%20polymerases%20and%20nucleases." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Review%3A+Steady-state+inhibition+kinetics+of+processive+nucleic+acid+polymerases+and+nucleases.+Chou+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0033393-Andraos1" id="pone.0033393-Andraos1" class="link-target"></a>Andraos J (2008) Kinetic plasticity and the determination of product ratios for kinetic schemes leading to multiple products without rate laws: new methods based on directed graphs. Canadian Journal of Chemistry  86: 342–357.  <ul class="reflinks"><li><a href="#" data-author="Andraos" data-cit="AndraosJ2008Kinetic%20plasticity%20and%20the%20determination%20of%20product%20ratios%20for%20kinetic%20schemes%20leading%20to%20multiple%20products%20without%20rate%20laws%3A%20new%20methods%20based%20on%20directed%20graphs.Canadian%20Journal%20of%20Chemistry86342357" data-title="Kinetic%20plasticity%20and%20the%20determination%20of%20product%20ratios%20for%20kinetic%20schemes%20leading%20to%20multiple%20products%20without%20rate%20laws%3A%20new%20methods%20based%20on%20directed%20graphs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Kinetic+plasticity+and+the+determination+of+product+ratios+for+kinetic+schemes+leading+to+multiple+products+without+rate+laws%3A+new+methods+based+on+directed+graphs.+Andraos+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pone.0033393-Chou11" id="pone.0033393-Chou11" class="link-target"></a>Chou KC (2010) Graphic rule for drug metabolism systems. Current Drug Metabolism  11: 369–378.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKC2010Graphic%20rule%20for%20drug%20metabolism%20systems.Current%20Drug%20Metabolism11369378" data-title="Graphic%20rule%20for%20drug%20metabolism%20systems." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Graphic+rule+for+drug+metabolism+systems.+Chou+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pone.0033393-Zhou2" id="pone.0033393-Zhou2" class="link-target"></a>Zhou GP (2011) The disposition of the LZCC protein residues in wenxiang diagram provides new insights into the protein-protein interaction mechanism. Journal of Theoretical Biology  284: 142–148.  <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouGP2011The%20disposition%20of%20the%20LZCC%20protein%20residues%20in%20wenxiang%20diagram%20provides%20new%20insights%20into%20the%20protein-protein%20interaction%20mechanism.Journal%20of%20Theoretical%20Biology284142148" data-title="The%20disposition%20of%20the%20LZCC%20protein%20residues%20in%20wenxiang%20diagram%20provides%20new%20insights%20into%20the%20protein-protein%20interaction%20mechanism." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+disposition+of+the+LZCC+protein+residues+in+wenxiang+diagram+provides+new+insights+into+the+protein-protein+interaction+mechanism.+Zhou+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pone.0033393-Chou12" id="pone.0033393-Chou12" class="link-target"></a>Chou KC, Lin WZ, Xiao X (2011) Wenxiang: a web-server for drawing wenxiang diagrams. Natural Science  3: 862–865.  <ul class="reflinks"><li><a href="#" data-author="Chou" data-cit="ChouKCLinWZXiaoX2011Wenxiang%3A%20a%20web-server%20for%20drawing%20wenxiang%20diagrams.Natural%20Science3862865" data-title="Wenxiang%3A%20a%20web-server%20for%20drawing%20wenxiang%20diagrams." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Wenxiang%3A+a+web-server+for+drawing+wenxiang+diagrams.+Chou+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pone.0033393-Zhou3" id="pone.0033393-Zhou3" class="link-target"></a>Zhou GP (2011) The Structural Determinations of the Leucine Zipper Coiled-Coil Domains of the cGMP-Dependent Protein Kinase I alpha and its Interaction with the Myosin Binding Subunit of the Myosin Light Chains Phosphase. Proteins &amp; Peptide Letters  18: 966–978.  <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouGP2011The%20Structural%20Determinations%20of%20the%20Leucine%20Zipper%20Coiled-Coil%20Domains%20of%20the%20cGMP-Dependent%20Protein%20Kinase%20I%20alpha%20and%20its%20Interaction%20with%20the%20Myosin%20Binding%20Subunit%20of%20the%20Myosin%20Light%20Chains%20Phosphase.Proteins%20%26%20Peptide%20Letters18966978" data-title="The%20Structural%20Determinations%20of%20the%20Leucine%20Zipper%20Coiled-Coil%20Domains%20of%20the%20cGMP-Dependent%20Protein%20Kinase%20I%20alpha%20and%20its%20Interaction%20with%20the%20Myosin%20Binding%20Subunit%20of%20the%20Myosin%20Light%20Chains%20Phosphase." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Structural+Determinations+of+the+Leucine+Zipper+Coiled-Coil+Domains+of+the+cGMP-Dependent+Protein+Kinase+I+alpha+and+its+Interaction+with+the+Myosin+Binding+Subunit+of+the+Myosin+Light+Chains+Phosphase.+Zhou+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pone.0033393-Huangda1" id="pone.0033393-Huangda1" class="link-target"></a>Huang da W, Sherman BT, Lempicki RA (2009) Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nat Protoc  4: 44–57.  <ul class="reflinks"><li><a href="#" data-author="Huang%20da" data-cit="Huang%20daWShermanBTLempickiRA2009Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources.Nat%20Protoc44457" data-title="Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Systematic+and+integrative+analysis+of+large+gene+lists+using+DAVID+bioinformatics+resources.+Huang+da+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span class="order">60.
            </span><a name="pone.0033393-Benjamini1" id="pone.0033393-Benjamini1" class="link-target"></a>Benjamini Y, Yekutieli D (2001) The Control of the False Discovery Rate in Multiple Testing under Dependency. The Annals of Statistics  29: 1165–1188.  <ul class="reflinks"><li><a href="#" data-author="Benjamini" data-cit="BenjaminiYYekutieliD2001The%20Control%20of%20the%20False%20Discovery%20Rate%20in%20Multiple%20Testing%20under%20Dependency.The%20Annals%20of%20Statistics2911651188" data-title="The%20Control%20of%20the%20False%20Discovery%20Rate%20in%20Multiple%20Testing%20under%20Dependency." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Control+of+the+False+Discovery+Rate+in+Multiple+Testing+under+Dependency.+Benjamini+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span class="order">61.
            </span><a name="pone.0033393-Catalano1" id="pone.0033393-Catalano1" class="link-target"></a>Catalano MG, Pfeffer U, Raineri M, Ferro P, Curto A, et al.  (2000) Altered expression of androgen-receptor isoforms in human colon-cancer tissues. Int J Cancer  86: 325–330.  <ul class="reflinks"><li><a href="#" data-author="Catalano" data-cit="CatalanoMGPfefferURaineriMFerroPCurtoA2000Altered%20expression%20of%20androgen-receptor%20isoforms%20in%20human%20colon-cancer%20tissues.Int%20J%20Cancer86325330" data-title="Altered%20expression%20of%20androgen-receptor%20isoforms%20in%20human%20colon-cancer%20tissues." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Altered+expression+of+androgen-receptor+isoforms+in+human+colon-cancer+tissues.+Catalano+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span class="order">62.
            </span><a name="pone.0033393-Castagnetta1" id="pone.0033393-Castagnetta1" class="link-target"></a>Castagnetta L, Traina A, Campisi I, Calabro M, Maratta A, et al.  (2002) Androgen receptor status in nontumoral and malignant human colorectal tissues. Ann N Y Acad Sci  963: 322–325.  <ul class="reflinks"><li><a href="#" data-author="Castagnetta" data-cit="CastagnettaLTrainaACampisiICalabroMMarattaA2002Androgen%20receptor%20status%20in%20nontumoral%20and%20malignant%20human%20colorectal%20tissues.Ann%20N%20Y%20Acad%20Sci963322325" data-title="Androgen%20receptor%20status%20in%20nontumoral%20and%20malignant%20human%20colorectal%20tissues." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Androgen+receptor+status+in+nontumoral+and+malignant+human+colorectal+tissues.+Castagnetta+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref63"><span class="order">63.
            </span><a name="pone.0033393-Menendez1" id="pone.0033393-Menendez1" class="link-target"></a>Menendez D, Inga A, Resnick MA (2009) The expanding universe of p53 targets. Nat Rev Cancer  9: 724–737.  <ul class="reflinks"><li><a href="#" data-author="Menendez" data-cit="MenendezDIngaAResnickMA2009The%20expanding%20universe%20of%20p53%20targets.Nat%20Rev%20Cancer9724737" data-title="The%20expanding%20universe%20of%20p53%20targets." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+expanding+universe+of+p53+targets.+Menendez+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref64"><span class="order">64.
            </span><a name="pone.0033393-Gayther1" id="pone.0033393-Gayther1" class="link-target"></a>Gayther SA, Batley SJ, Linger L, Bannister A, Thorpe K, et al.  (2000) Mutations truncating the EP300 acetylase in human cancers. Nat Genet  24: 300–303.  <ul class="reflinks"><li><a href="#" data-author="Gayther" data-cit="GaytherSABatleySJLingerLBannisterAThorpeK2000Mutations%20truncating%20the%20EP300%20acetylase%20in%20human%20cancers.Nat%20Genet24300303" data-title="Mutations%20truncating%20the%20EP300%20acetylase%20in%20human%20cancers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mutations+truncating+the+EP300+acetylase+in+human+cancers.+Gayther+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref65"><span class="order">65.
            </span><a name="pone.0033393-Polakis1" id="pone.0033393-Polakis1" class="link-target"></a>Polakis P (2000) Wnt signaling and cancer. Genes Dev  14: 1837–1851.  <ul class="reflinks"><li><a href="#" data-author="Polakis" data-cit="PolakisP2000Wnt%20signaling%20and%20cancer.Genes%20Dev1418371851" data-title="Wnt%20signaling%20and%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Wnt+signaling+and+cancer.+Polakis+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref66"><span class="order">66.
            </span><a name="pone.0033393-Howe1" id="pone.0033393-Howe1" class="link-target"></a>Howe GR, Aronson KJ, Benito E, Castelleto R, Cornee J, et al.  (1997) The relationship between dietary fat intake and risk of colorectal cancer: evidence from the combined analysis of 13 case-control studies. Cancer Causes Control  8: 215–228.  <ul class="reflinks"><li><a href="#" data-author="Howe" data-cit="HoweGRAronsonKJBenitoECastelletoRCorneeJ1997The%20relationship%20between%20dietary%20fat%20intake%20and%20risk%20of%20colorectal%20cancer%3A%20evidence%20from%20the%20combined%20analysis%20of%2013%20case-control%20studies.Cancer%20Causes%20Control8215228" data-title="The%20relationship%20between%20dietary%20fat%20intake%20and%20risk%20of%20colorectal%20cancer%3A%20evidence%20from%20the%20combined%20analysis%20of%2013%20case-control%20studies." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+relationship+between+dietary+fat+intake+and+risk+of+colorectal+cancer%3A+evidence+from+the+combined+analysis+of+13+case-control+studies.+Howe+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref67"><span class="order">67.
            </span><a name="pone.0033393-Hoeft1" id="pone.0033393-Hoeft1" class="link-target"></a>Hoeft B, Linseisen J, Beckmann L, Muller-Decker K, Canzian F, et al.  (2010) Polymorphisms in fatty-acid-metabolism-related genes are associated with colorectal cancer risk. Carcinogenesis  31: 466–472.  <ul class="reflinks"><li><a href="#" data-author="Hoeft" data-cit="HoeftBLinseisenJBeckmannLMuller-DeckerKCanzianF2010Polymorphisms%20in%20fatty-acid-metabolism-related%20genes%20are%20associated%20with%20colorectal%20cancer%20risk.Carcinogenesis31466472" data-title="Polymorphisms%20in%20fatty-acid-metabolism-related%20genes%20are%20associated%20with%20colorectal%20cancer%20risk." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Polymorphisms+in+fatty-acid-metabolism-related+genes+are+associated+with+colorectal+cancer+risk.+Hoeft+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref68"><span class="order">68.
            </span><a name="pone.0033393-Christensen1" id="pone.0033393-Christensen1" class="link-target"></a>Christensen BC, Smith AA, Zheng S, Koestler DC, Houseman EA, et al.  (2011) DNA methylation, isocitrate dehydrogenase mutation, and survival in glioma. J Natl Cancer Inst  103: 143–153.  <ul class="reflinks"><li><a href="#" data-author="Christensen" data-cit="ChristensenBCSmithAAZhengSKoestlerDCHousemanEA2011DNA%20methylation%2C%20isocitrate%20dehydrogenase%20mutation%2C%20and%20survival%20in%20glioma.J%20Natl%20Cancer%20Inst103143153" data-title="DNA%20methylation%2C%20isocitrate%20dehydrogenase%20mutation%2C%20and%20survival%20in%20glioma." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+methylation%2C+isocitrate+dehydrogenase+mutation%2C+and+survival+in+glioma.+Christensen+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref69"><span class="order">69.
            </span><a name="pone.0033393-Li1" id="pone.0033393-Li1" class="link-target"></a>Li J, Humphreys K, Darabi H, Rosin G, Hannelius U, et al.  (2010) A genome-wide association scan on estrogen receptor-negative breast cancer. Breast Cancer Res  12: R93.  <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiJHumphreysKDarabiHRosinGHanneliusU2010A%20genome-wide%20association%20scan%20on%20estrogen%20receptor-negative%20breast%20cancer.Breast%20Cancer%20Res12R93" data-title="A%20genome-wide%20association%20scan%20on%20estrogen%20receptor-negative%20breast%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+genome-wide+association+scan+on+estrogen+receptor-negative+breast+cancer.+Li+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref70"><span class="order">70.
            </span><a name="pone.0033393-Allen1" id="pone.0033393-Allen1" class="link-target"></a>Allen WL, Jithesh PV, Oliver GR, Proutski I, Longley DB, et al.  (2010) The colorectal cancer disease-specific transcriptome may facilitate the discovery of more biologically and clinically relevant information. BMC Cancer  10: 687.  <ul class="reflinks"><li><a href="#" data-author="Allen" data-cit="AllenWLJitheshPVOliverGRProutskiILongleyDB2010The%20colorectal%20cancer%20disease-specific%20transcriptome%20may%20facilitate%20the%20discovery%20of%20more%20biologically%20and%20clinically%20relevant%20information.BMC%20Cancer10687" data-title="The%20colorectal%20cancer%20disease-specific%20transcriptome%20may%20facilitate%20the%20discovery%20of%20more%20biologically%20and%20clinically%20relevant%20information." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+colorectal+cancer+disease-specific+transcriptome+may+facilitate+the+discovery+of+more+biologically+and+clinically+relevant+information.+Allen+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref71"><span class="order">71.
            </span><a name="pone.0033393-Tsukahara1" id="pone.0033393-Tsukahara1" class="link-target"></a>Tsukahara H, Sekine K, Uchiyama M, Miura M, Nakazato M, et al.  (2001) Uroguanylin level in umbilical cord blood. Pediatr Int  43: 267–269.  <ul class="reflinks"><li><a href="#" data-author="Tsukahara" data-cit="TsukaharaHSekineKUchiyamaMMiuraMNakazatoM2001Uroguanylin%20level%20in%20umbilical%20cord%20blood.Pediatr%20Int43267269" data-title="Uroguanylin%20level%20in%20umbilical%20cord%20blood." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Uroguanylin+level+in+umbilical+cord+blood.+Tsukahara+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref72"><span class="order">72.
            </span><a name="pone.0033393-Liu2" id="pone.0033393-Liu2" class="link-target"></a>Liu D, Overbey D, Watkinson LD, Daibes-Figueroa S, Hoffman TJ, et al.  (2009) In vivo imaging of human colorectal cancer using radiolabeled analogs of the uroguanylin peptide hormone. Anticancer Res  29: 3777–3783.  <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuDOverbeyDWatkinsonLDDaibes-FigueroaSHoffmanTJ2009In%20vivo%20imaging%20of%20human%20colorectal%20cancer%20using%20radiolabeled%20analogs%20of%20the%20uroguanylin%20peptide%20hormone.Anticancer%20Res2937773783" data-title="In%20vivo%20imaging%20of%20human%20colorectal%20cancer%20using%20radiolabeled%20analogs%20of%20the%20uroguanylin%20peptide%20hormone." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=In+vivo+imaging+of+human+colorectal+cancer+using+radiolabeled+analogs+of+the+uroguanylin+peptide+hormone.+Liu+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref73"><span class="order">73.
            </span><a name="pone.0033393-Reeves1" id="pone.0033393-Reeves1" class="link-target"></a>Reeves JR, Xuan JW, Arfanis K, Morin C, Garde SV, et al.  (2005) Identification, purification and characterization of a novel human blood protein with binding affinity for prostate secretory protein of 94 amino acids. Biochem J  385: 105–114.  <ul class="reflinks"><li><a href="#" data-author="Reeves" data-cit="ReevesJRXuanJWArfanisKMorinCGardeSV2005Identification%2C%20purification%20and%20characterization%20of%20a%20novel%20human%20blood%20protein%20with%20binding%20affinity%20for%20prostate%20secretory%20protein%20of%2094%20amino%20acids.Biochem%20J385105114" data-title="Identification%2C%20purification%20and%20characterization%20of%20a%20novel%20human%20blood%20protein%20with%20binding%20affinity%20for%20prostate%20secretory%20protein%20of%2094%20amino%20acids." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification%2C+purification+and+characterization+of+a+novel+human+blood+protein+with+binding+affinity+for+prostate+secretory+protein+of+94+amino+acids.+Reeves+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref74"><span class="order">74.
            </span><a name="pone.0033393-Cui1" id="pone.0033393-Cui1" class="link-target"></a>Cui J, Chen Y, Chou WC, Sun L, Chen L, et al.  (2011) An integrated transcriptomic and computational analysis for biomarker identification in gastric cancer. Nucleic Acids Res  39: 1197–1207.  <ul class="reflinks"><li><a href="#" data-author="Cui" data-cit="CuiJChenYChouWCSunLChenL2011An%20integrated%20transcriptomic%20and%20computational%20analysis%20for%20biomarker%20identification%20in%20gastric%20cancer.Nucleic%20Acids%20Res3911971207" data-title="An%20integrated%20transcriptomic%20and%20computational%20analysis%20for%20biomarker%20identification%20in%20gastric%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+integrated+transcriptomic+and+computational+analysis+for+biomarker+identification+in+gastric+cancer.+Cui+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref75"><span class="order">75.
            </span><a name="pone.0033393-Hibi1" id="pone.0033393-Hibi1" class="link-target"></a>Hibi K, Goto T, Mizukami H, Kitamura YH, Sakuraba K, et al.  (2009) Demethylation of the CDH3 gene is frequently detected in advanced colorectal cancer. Anticancer Res  29: 2215–2217.  <ul class="reflinks"><li><a href="#" data-author="Hibi" data-cit="HibiKGotoTMizukamiHKitamuraYHSakurabaK2009Demethylation%20of%20the%20CDH3%20gene%20is%20frequently%20detected%20in%20advanced%20colorectal%20cancer.Anticancer%20Res2922152217" data-title="Demethylation%20of%20the%20CDH3%20gene%20is%20frequently%20detected%20in%20advanced%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Demethylation+of+the+CDH3+gene+is+frequently+detected+in+advanced+colorectal+cancer.+Hibi+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref76"><span class="order">76.
            </span><a name="pone.0033393-Imai1" id="pone.0033393-Imai1" class="link-target"></a>Imai K, Hirata S, Irie A, Senju S, Ikuta Y, et al.  (2008) Identification of a novel tumor-associated antigen, cadherin 3/P-cadherin, as a possible target for immunotherapy of pancreatic, gastric, and colorectal cancers. Clin Cancer Res  14: 6487–6495.  <ul class="reflinks"><li><a href="#" data-author="Imai" data-cit="ImaiKHirataSIrieASenjuSIkutaY2008Identification%20of%20a%20novel%20tumor-associated%20antigen%2C%20cadherin%203%2FP-cadherin%2C%20as%20a%20possible%20target%20for%20immunotherapy%20of%20pancreatic%2C%20gastric%2C%20and%20colorectal%20cancers.Clin%20Cancer%20Res1464876495" data-title="Identification%20of%20a%20novel%20tumor-associated%20antigen%2C%20cadherin%203%2FP-cadherin%2C%20as%20a%20possible%20target%20for%20immunotherapy%20of%20pancreatic%2C%20gastric%2C%20and%20colorectal%20cancers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+a+novel+tumor-associated+antigen%2C+cadherin+3%2FP-cadherin%2C+as+a+possible+target+for+immunotherapy+of+pancreatic%2C+gastric%2C+and+colorectal+cancers.+Imai+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref77"><span class="order">77.
            </span><a name="pone.0033393-Deves1" id="pone.0033393-Deves1" class="link-target"></a>Deves C, Renck D, Garicochea B, da Silva VD, Giulianni Lopes T, et al.  (2011) Analysis of select members of the E26 (ETS) transcription factors family in colorectal cancer. Virchows Arch  458: 421–430.  <ul class="reflinks"><li><a href="#" data-author="Deves" data-cit="DevesCRenckDGaricocheaBda%20SilvaVDGiulianni%20LopesT2011Analysis%20of%20select%20members%20of%20the%20E26%20%28ETS%29%20transcription%20factors%20family%20in%20colorectal%20cancer.Virchows%20Arch458421430" data-title="Analysis%20of%20select%20members%20of%20the%20E26%20%28ETS%29%20transcription%20factors%20family%20in%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+of+select+members+of+the+E26+%28ETS%29+transcription+factors+family+in+colorectal+cancer.+Deves+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref78"><span class="order">78.
            </span><a name="pone.0033393-Stohr1" id="pone.0033393-Stohr1" class="link-target"></a>Stohr H, Marquardt A, Nanda I, Schmid M, Weber BH (2002) Three novel human VMD2-like genes are members of the evolutionary highly conserved RFP-TM family. Eur J Hum Genet  10: 281–284.  <ul class="reflinks"><li><a href="#" data-author="Stohr" data-cit="StohrHMarquardtANandaISchmidMWeberBH2002Three%20novel%20human%20VMD2-like%20genes%20are%20members%20of%20the%20evolutionary%20highly%20conserved%20RFP-TM%20family.Eur%20J%20Hum%20Genet10281284" data-title="Three%20novel%20human%20VMD2-like%20genes%20are%20members%20of%20the%20evolutionary%20highly%20conserved%20RFP-TM%20family." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Three+novel+human+VMD2-like+genes+are+members+of+the+evolutionary+highly+conserved+RFP-TM+family.+Stohr+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref79"><span class="order">79.
            </span><a name="pone.0033393-Bakall1" id="pone.0033393-Bakall1" class="link-target"></a>Bakall B, McLaughlin P, Stanton JB, Zhang Y, Hartzell HC, et al.  (2008) Bestrophin-2 is involved in the generation of intraocular pressure. Invest Ophthalmol Vis Sci  49: 1563–1570.  <ul class="reflinks"><li><a href="#" data-author="Bakall" data-cit="BakallBMcLaughlinPStantonJBZhangYHartzellHC2008Bestrophin-2%20is%20involved%20in%20the%20generation%20of%20intraocular%20pressure.Invest%20Ophthalmol%20Vis%20Sci4915631570" data-title="Bestrophin-2%20is%20involved%20in%20the%20generation%20of%20intraocular%20pressure." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Bestrophin-2+is+involved+in+the+generation+of+intraocular+pressure.+Bakall+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref80"><span class="order">80.
            </span><a name="pone.0033393-Yu1" id="pone.0033393-Yu1" class="link-target"></a>Yu K, Lujan R, Marmorstein A, Gabriel S, Hartzell HC (2010) Bestrophin-2 mediates bicarbonate transport by goblet cells in mouse colon. J Clin Invest  120: 1722–1735.  <ul class="reflinks"><li><a href="#" data-author="Yu" data-cit="YuKLujanRMarmorsteinAGabrielSHartzellHC2010Bestrophin-2%20mediates%20bicarbonate%20transport%20by%20goblet%20cells%20in%20mouse%20colon.J%20Clin%20Invest12017221735" data-title="Bestrophin-2%20mediates%20bicarbonate%20transport%20by%20goblet%20cells%20in%20mouse%20colon." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Bestrophin-2+mediates+bicarbonate+transport+by+goblet+cells+in+mouse+colon.+Yu+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref81"><span class="order">81.
            </span><a name="pone.0033393-vanCriekinge1" id="pone.0033393-vanCriekinge1" class="link-target"></a>van Criekinge W, Meijer G, Straub J, De Carvalho BPM (2007) Methylation markers for early detection and prognosis of colon cancers.  International Application No: PCT/US2007/013803.  <ul class="find-nolinks"></ul></li><li id="ref82"><span class="order">82.
            </span><a name="pone.0033393-Jones1" id="pone.0033393-Jones1" class="link-target"></a>Jones S, Zhang X, Parsons DW, Lin JC, Leary RJ, et al.  (2008) Core signaling pathways in human pancreatic cancers revealed by global genomic analyses. Science  321: 1801–1806.  <ul class="reflinks"><li><a href="#" data-author="Jones" data-cit="JonesSZhangXParsonsDWLinJCLearyRJ2008Core%20signaling%20pathways%20in%20human%20pancreatic%20cancers%20revealed%20by%20global%20genomic%20analyses.Science32118011806" data-title="Core%20signaling%20pathways%20in%20human%20pancreatic%20cancers%20revealed%20by%20global%20genomic%20analyses." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Core+signaling+pathways+in+human+pancreatic+cancers+revealed+by+global+genomic+analyses.+Jones+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref83"><span class="order">83.
            </span><a name="pone.0033393-Parsons1" id="pone.0033393-Parsons1" class="link-target"></a>Parsons DW, Jones S, Zhang X, Lin JC, Leary RJ, et al.  (2008) An integrated genomic analysis of human glioblastoma multiforme. Science  321: 1807–1812.  <ul class="reflinks"><li><a href="#" data-author="Parsons" data-cit="ParsonsDWJonesSZhangXLinJCLearyRJ2008An%20integrated%20genomic%20analysis%20of%20human%20glioblastoma%20multiforme.Science32118071812" data-title="An%20integrated%20genomic%20analysis%20of%20human%20glioblastoma%20multiforme." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+integrated+genomic+analysis+of+human+glioblastoma+multiforme.+Parsons+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref84"><span class="order">84.
            </span><a name="pone.0033393-Wood1" id="pone.0033393-Wood1" class="link-target"></a>Wood LD, Parsons DW, Jones S, Lin J, Sjoblom T, et al.  (2007) The genomic landscapes of human breast and colorectal cancers. Science  318: 1108–1113.  <ul class="reflinks"><li><a href="#" data-author="Wood" data-cit="WoodLDParsonsDWJonesSLinJSjoblomT2007The%20genomic%20landscapes%20of%20human%20breast%20and%20colorectal%20cancers.Science31811081113" data-title="The%20genomic%20landscapes%20of%20human%20breast%20and%20colorectal%20cancers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+genomic+landscapes+of+human+breast+and+colorectal+cancers.+Wood+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref85"><span class="order">85.
            </span><a name="pone.0033393-Morikawa1" id="pone.0033393-Morikawa1" class="link-target"></a>Morikawa T, Kuchiba A, Yamauchi M, Meyerhardt JA, Shima K, et al.  (2011) Association of CTNNB1 (beta-catenin) alterations, body mass index, and physical activity with survival in patients with colorectal cancer. JAMA  305: 1685–1694.  <ul class="reflinks"><li><a href="#" data-author="Morikawa" data-cit="MorikawaTKuchibaAYamauchiMMeyerhardtJAShimaK2011Association%20of%20CTNNB1%20%28beta-catenin%29%20alterations%2C%20body%20mass%20index%2C%20and%20physical%20activity%20with%20survival%20in%20patients%20with%20colorectal%20cancer.JAMA30516851694" data-title="Association%20of%20CTNNB1%20%28beta-catenin%29%20alterations%2C%20body%20mass%20index%2C%20and%20physical%20activity%20with%20survival%20in%20patients%20with%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Association+of+CTNNB1+%28beta-catenin%29+alterations%2C+body+mass+index%2C+and+physical+activity+with+survival+in+patients+with+colorectal+cancer.+Morikawa+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref86"><span class="order">86.
            </span><a name="pone.0033393-MacLean1" id="pone.0033393-MacLean1" class="link-target"></a>MacLean HE, Warne GL, Zajac JD (1997) Localization of functional domains in the androgen receptor. J Steroid Biochem Mol Biol  62: 233–242.  <ul class="reflinks"><li><a href="#" data-author="MacLean" data-cit="MacLeanHEWarneGLZajacJD1997Localization%20of%20functional%20domains%20in%20the%20androgen%20receptor.J%20Steroid%20Biochem%20Mol%20Biol62233242" data-title="Localization%20of%20functional%20domains%20in%20the%20androgen%20receptor." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Localization+of+functional+domains+in+the+androgen+receptor.+MacLean+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref87"><span class="order">87.
            </span><a name="pone.0033393-Slattery1" id="pone.0033393-Slattery1" class="link-target"></a>Slattery ML, Sweeney C, Murtaugh M, Ma KN, Wolff RK, et al.  (2005) Associations between ERalpha, ERbeta, and AR genotypes and colon and rectal cancer. Cancer Epidemiol Biomarkers Prev  14: 2936–2942.  <ul class="reflinks"><li><a href="#" data-author="Slattery" data-cit="SlatteryMLSweeneyCMurtaughMMaKNWolffRK2005Associations%20between%20ERalpha%2C%20ERbeta%2C%20and%20AR%20genotypes%20and%20colon%20and%20rectal%20cancer.Cancer%20Epidemiol%20Biomarkers%20Prev1429362942" data-title="Associations%20between%20ERalpha%2C%20ERbeta%2C%20and%20AR%20genotypes%20and%20colon%20and%20rectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Associations+between+ERalpha%2C+ERbeta%2C+and+AR+genotypes+and+colon+and+rectal+cancer.+Slattery+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref88"><span class="order">88.
            </span><a name="pone.0033393-Johnson1" id="pone.0033393-Johnson1" class="link-target"></a>Johnson SA, Dubeau L, Kawalek M, Dervan A, Schonthal AH, et al.  (2003) Increased expression of TATA-binding protein, the central transcription factor, can contribute to oncogenesis. Mol Cell Biol  23: 3043–3051.  <ul class="reflinks"><li><a href="#" data-author="Johnson" data-cit="JohnsonSADubeauLKawalekMDervanASchonthalAH2003Increased%20expression%20of%20TATA-binding%20protein%2C%20the%20central%20transcription%20factor%2C%20can%20contribute%20to%20oncogenesis.Mol%20Cell%20Biol2330433051" data-title="Increased%20expression%20of%20TATA-binding%20protein%2C%20the%20central%20transcription%20factor%2C%20can%20contribute%20to%20oncogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Increased+expression+of+TATA-binding+protein%2C+the+central+transcription+factor%2C+can+contribute+to+oncogenesis.+Johnson+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref89"><span class="order">89.
            </span><a name="pone.0033393-Hiranuma1" id="pone.0033393-Hiranuma1" class="link-target"></a>Hiranuma C, Kawakami K, Oyama K, Ota N, Omura K, et al.  (2004) Hypermethylation of the MYOD1 gene is a novel prognostic factor in patients with colorectal cancer. Int J Mol Med  13: 413–417.  <ul class="reflinks"><li><a href="#" data-author="Hiranuma" data-cit="HiranumaCKawakamiKOyamaKOtaNOmuraK2004Hypermethylation%20of%20the%20MYOD1%20gene%20is%20a%20novel%20prognostic%20factor%20in%20patients%20with%20colorectal%20cancer.Int%20J%20Mol%20Med13413417" data-title="Hypermethylation%20of%20the%20MYOD1%20gene%20is%20a%20novel%20prognostic%20factor%20in%20patients%20with%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hypermethylation+of+the+MYOD1+gene+is+a+novel+prognostic+factor+in+patients+with+colorectal+cancer.+Hiranuma+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref90"><span class="order">90.
            </span><a name="pone.0033393-Medina1" id="pone.0033393-Medina1" class="link-target"></a>Medina MW (2010) The relationship between HMGCR genetic variation, alternative splicing, and statin efficacy. Discov Med  9: 495–499.  <ul class="reflinks"><li><a href="#" data-author="Medina" data-cit="MedinaMW2010The%20relationship%20between%20HMGCR%20genetic%20variation%2C%20alternative%20splicing%2C%20and%20statin%20efficacy.Discov%20Med9495499" data-title="The%20relationship%20between%20HMGCR%20genetic%20variation%2C%20alternative%20splicing%2C%20and%20statin%20efficacy." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+relationship+between+HMGCR+genetic+variation%2C+alternative+splicing%2C+and+statin+efficacy.+Medina+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref91"><span class="order">91.
            </span><a name="pone.0033393-Lipkin1" id="pone.0033393-Lipkin1" class="link-target"></a>Lipkin SM, Chao EC, Moreno V, Rozek LS, Rennert H, et al.  (2010) Genetic variation in 3-hydroxy-3-methylglutaryl CoA reductase modifies the chemopreventive activity of statins for colorectal cancer. Cancer Prev Res (Phila)  3: 597–603.  <ul class="reflinks"><li><a href="#" data-author="Lipkin" data-cit="LipkinSMChaoECMorenoVRozekLSRennertH2010Genetic%20variation%20in%203-hydroxy-3-methylglutaryl%20CoA%20reductase%20modifies%20the%20chemopreventive%20activity%20of%20statins%20for%20colorectal%20cancer.Cancer%20Prev%20Res%20%28Phila%293597603" data-title="Genetic%20variation%20in%203-hydroxy-3-methylglutaryl%20CoA%20reductase%20modifies%20the%20chemopreventive%20activity%20of%20statins%20for%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+variation+in+3-hydroxy-3-methylglutaryl+CoA+reductase+modifies+the+chemopreventive+activity+of+statins+for+colorectal+cancer.+Lipkin+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref92"><span class="order">92.
            </span><a name="pone.0033393-Diamandis1" id="pone.0033393-Diamandis1" class="link-target"></a>Diamandis EP, Yu H (1995) New biological functions of prostate-specific antigen? J Clin Endocrinol Metab  80: 1515–1517.  <ul class="reflinks"><li><a href="#" data-author="Diamandis" data-cit="DiamandisEPYuH1995New%20biological%20functions%20of%20prostate-specific%20antigen%3FJ%20Clin%20Endocrinol%20Metab8015151517" data-title="New%20biological%20functions%20of%20prostate-specific%20antigen%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=New+biological+functions+of+prostate-specific+antigen%3F+Diamandis+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref93"><span class="order">93.
            </span><a name="pone.0033393-Levesque1" id="pone.0033393-Levesque1" class="link-target"></a>Levesque M, Hu H, D'Costa M, Diamandis EP (1995) Prostate-specific antigen expression by various tumors. J Clin Lab Anal  9: 123–128.  <ul class="reflinks"><li><a href="#" data-author="Levesque" data-cit="LevesqueMHuHD%27CostaMDiamandisEP1995Prostate-specific%20antigen%20expression%20by%20various%20tumors.J%20Clin%20Lab%20Anal9123128" data-title="Prostate-specific%20antigen%20expression%20by%20various%20tumors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prostate-specific+antigen+expression+by+various+tumors.+Levesque+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref94"><span class="order">94.
            </span><a name="pone.0033393-Borgono1" id="pone.0033393-Borgono1" class="link-target"></a>Borgono CA, Diamandis EP (2004) The emerging roles of human tissue kallikreins in cancer. Nat Rev Cancer  4: 876–890.  <ul class="reflinks"><li><a href="#" data-author="Borgono" data-cit="BorgonoCADiamandisEP2004The%20emerging%20roles%20of%20human%20tissue%20kallikreins%20in%20cancer.Nat%20Rev%20Cancer4876890" data-title="The%20emerging%20roles%20of%20human%20tissue%20kallikreins%20in%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+emerging+roles+of+human+tissue+kallikreins+in+cancer.+Borgono+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref95"><span class="order">95.
            </span><a name="pone.0033393-Diamandis2" id="pone.0033393-Diamandis2" class="link-target"></a>Diamandis EP, Yousef GM (2002) Human tissue kallikreins: a family of new cancer biomarkers. Clin Chem  48: 1198–1205.  <ul class="reflinks"><li><a href="#" data-author="Diamandis" data-cit="DiamandisEPYousefGM2002Human%20tissue%20kallikreins%3A%20a%20family%20of%20new%20cancer%20biomarkers.Clin%20Chem4811981205" data-title="Human%20tissue%20kallikreins%3A%20a%20family%20of%20new%20cancer%20biomarkers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+tissue+kallikreins%3A+a+family+of+new+cancer+biomarkers.+Diamandis+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref96"><span class="order">96.
            </span><a name="pone.0033393-Talieri1" id="pone.0033393-Talieri1" class="link-target"></a>Talieri M, Mathioudaki K, Prezas P, Alexopoulou DK, Diamandis EP, et al.  (2009) Clinical significance of kallikrein-related peptidase 7 (KLK7) in colorectal cancer. Thromb Haemost  101: 741–747.  <ul class="reflinks"><li><a href="#" data-author="Talieri" data-cit="TalieriMMathioudakiKPrezasPAlexopoulouDKDiamandisEP2009Clinical%20significance%20of%20kallikrein-related%20peptidase%207%20%28KLK7%29%20in%20colorectal%20cancer.Thromb%20Haemost101741747" data-title="Clinical%20significance%20of%20kallikrein-related%20peptidase%207%20%28KLK7%29%20in%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Clinical+significance+of+kallikrein-related+peptidase+7+%28KLK7%29+in+colorectal+cancer.+Talieri+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref97"><span class="order">97.
            </span><a name="pone.0033393-Kim1" id="pone.0033393-Kim1" class="link-target"></a>Kim MS, Chung NG, Kang MR, Yoo NJ, Lee SH (2011) Genetic and expressional alterations of CHD genes in gastric and colorectal cancers. Histopathology  58: 660–668.  <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimMSChungNGKangMRYooNJLeeSH2011Genetic%20and%20expressional%20alterations%20of%20CHD%20genes%20in%20gastric%20and%20colorectal%20cancers.Histopathology58660668" data-title="Genetic%20and%20expressional%20alterations%20of%20CHD%20genes%20in%20gastric%20and%20colorectal%20cancers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+and+expressional+alterations+of+CHD+genes+in+gastric+and+colorectal+cancers.+Kim+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref98"><span class="order">98.
            </span><a name="pone.0033393-Thomas1" id="pone.0033393-Thomas1" class="link-target"></a>Thomas G, Jacobs KB, Yeager M, Kraft P, Wacholder S, et al.  (2008) Multiple loci identified in a genome-wide association study of prostate cancer. Nat Genet  40: 310–315.  <ul class="reflinks"><li><a href="#" data-author="Thomas" data-cit="ThomasGJacobsKBYeagerMKraftPWacholderS2008Multiple%20loci%20identified%20in%20a%20genome-wide%20association%20study%20of%20prostate%20cancer.Nat%20Genet40310315" data-title="Multiple%20loci%20identified%20in%20a%20genome-wide%20association%20study%20of%20prostate%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multiple+loci+identified+in+a+genome-wide+association+study+of+prostate+cancer.+Thomas+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref99"><span class="order">99.
            </span><a name="pone.0033393-Ohkubo1" id="pone.0033393-Ohkubo1" class="link-target"></a>Ohkubo I, Tada T, Ochiai Y, Ueyama H, Eimoto T, et al.  (1995) Human seminal plasma beta-microseminoprotein: its purification, characterization, and immunohistochemical localization. Int J Biochem Cell Biol  27: 603–611.  <ul class="reflinks"><li><a href="#" data-author="Ohkubo" data-cit="OhkuboITadaTOchiaiYUeyamaHEimotoT1995Human%20seminal%20plasma%20beta-microseminoprotein%3A%20its%20purification%2C%20characterization%2C%20and%20immunohistochemical%20localization.Int%20J%20Biochem%20Cell%20Biol27603611" data-title="Human%20seminal%20plasma%20beta-microseminoprotein%3A%20its%20purification%2C%20characterization%2C%20and%20immunohistochemical%20localization." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+seminal+plasma+beta-microseminoprotein%3A+its+purification%2C+characterization%2C+and+immunohistochemical+localization.+Ohkubo+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref100"><span class="order">100.
            </span><a name="pone.0033393-Katoh1" id="pone.0033393-Katoh1" class="link-target"></a>Katoh M (2004) Human FOX gene family (Review). Int J Oncol  25: 1495–1500.  <ul class="reflinks"><li><a href="#" data-author="Katoh" data-cit="KatohM2004Human%20FOX%20gene%20family%20%28Review%29.Int%20J%20Oncol2514951500" data-title="Human%20FOX%20gene%20family%20%28Review%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+FOX+gene+family+%28Review%29.+Katoh+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref101"><span class="order">101.
            </span><a name="pone.0033393-Niittymaki1" id="pone.0033393-Niittymaki1" class="link-target"></a>Niittymaki I, Tuupanen S, Li Y, Jarvinen H, Mecklin JP, et al.  (2011) Systematic search for enhancer elements and somatic allelic imbalance at seven low-penetrance colorectal cancer predisposition loci. BMC Med Genet  12: 23.  <ul class="reflinks"><li><a href="#" data-author="Niittymaki" data-cit="NiittymakiITuupanenSLiYJarvinenHMecklinJP2011Systematic%20search%20for%20enhancer%20elements%20and%20somatic%20allelic%20imbalance%20at%20seven%20low-penetrance%20colorectal%20cancer%20predisposition%20loci.BMC%20Med%20Genet1223" data-title="Systematic%20search%20for%20enhancer%20elements%20and%20somatic%20allelic%20imbalance%20at%20seven%20low-penetrance%20colorectal%20cancer%20predisposition%20loci." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Systematic+search+for+enhancer+elements+and+somatic+allelic+imbalance+at+seven+low-penetrance+colorectal+cancer+predisposition+loci.+Niittymaki+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref102"><span class="order">102.
            </span><a name="pone.0033393-Kim2" id="pone.0033393-Kim2" class="link-target"></a>Kim IJ, Kang HC, Jang SG, Kim K, Ahn SA, et al.  (2006) Oligonucleotide microarray analysis of distinct gene expression patterns in colorectal cancer tissues harboring BRAF and K-ras mutations. Carcinogenesis  27: 392–404.  <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimIJKangHCJangSGKimKAhnSA2006Oligonucleotide%20microarray%20analysis%20of%20distinct%20gene%20expression%20patterns%20in%20colorectal%20cancer%20tissues%20harboring%20BRAF%20and%20K-ras%20mutations.Carcinogenesis27392404" data-title="Oligonucleotide%20microarray%20analysis%20of%20distinct%20gene%20expression%20patterns%20in%20colorectal%20cancer%20tissues%20harboring%20BRAF%20and%20K-ras%20mutations." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Oligonucleotide+microarray+analysis+of+distinct+gene+expression+patterns+in+colorectal+cancer+tissues+harboring+BRAF+and+K-ras+mutations.+Kim+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref103"><span class="order">103.
            </span><a name="pone.0033393-Lagger1" id="pone.0033393-Lagger1" class="link-target"></a>Lagger G, Doetzlhofer A, Schuettengruber B, Haidweger E, Simboeck E, et al.  (2003) The tumor suppressor p53 and histone deacetylase 1 are antagonistic regulators of the cyclin-dependent kinase inhibitor p21/WAF1/CIP1 gene. Mol Cell Biol  23: 2669–2679.  <ul class="reflinks"><li><a href="#" data-author="Lagger" data-cit="LaggerGDoetzlhoferASchuettengruberBHaidwegerESimboeckE2003The%20tumor%20suppressor%20p53%20and%20histone%20deacetylase%201%20are%20antagonistic%20regulators%20of%20the%20cyclin-dependent%20kinase%20inhibitor%20p21%2FWAF1%2FCIP1%20gene.Mol%20Cell%20Biol2326692679" data-title="The%20tumor%20suppressor%20p53%20and%20histone%20deacetylase%201%20are%20antagonistic%20regulators%20of%20the%20cyclin-dependent%20kinase%20inhibitor%20p21%2FWAF1%2FCIP1%20gene." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+tumor+suppressor+p53+and+histone+deacetylase+1+are+antagonistic+regulators+of+the+cyclin-dependent+kinase+inhibitor+p21%2FWAF1%2FCIP1+gene.+Lagger+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref104"><span class="order">104.
            </span><a name="pone.0033393-Thangaraju1" id="pone.0033393-Thangaraju1" class="link-target"></a>Thangaraju M, Carswell KN, Prasad PD, Ganapathy V (2009) Colon cancer cells maintain low levels of pyruvate to avoid cell death caused by inhibition of HDAC1/HDAC3. Biochem J  417: 379–389.  <ul class="reflinks"><li><a href="#" data-author="Thangaraju" data-cit="ThangarajuMCarswellKNPrasadPDGanapathyV2009Colon%20cancer%20cells%20maintain%20low%20levels%20of%20pyruvate%20to%20avoid%20cell%20death%20caused%20by%20inhibition%20of%20HDAC1%2FHDAC3.Biochem%20J417379389" data-title="Colon%20cancer%20cells%20maintain%20low%20levels%20of%20pyruvate%20to%20avoid%20cell%20death%20caused%20by%20inhibition%20of%20HDAC1%2FHDAC3." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Colon+cancer+cells+maintain+low+levels+of+pyruvate+to+avoid+cell+death+caused+by+inhibition+of+HDAC1%2FHDAC3.+Thangaraju+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref105"><span class="order">105.
            </span><a name="pone.0033393-Irizarry1" id="pone.0033393-Irizarry1" class="link-target"></a>Irizarry RA, Ladd-Acosta C, Wen B, Wu Z, Montano C, et al.  (2009) The human colon cancer methylome shows similar hypo- and hypermethylation at conserved tissue-specific CpG island shores. Nat Genet  41: 178–186.  <ul class="reflinks"><li><a href="#" data-author="Irizarry" data-cit="IrizarryRALadd-AcostaCWenBWuZMontanoC2009The%20human%20colon%20cancer%20methylome%20shows%20similar%20hypo-%20and%20hypermethylation%20at%20conserved%20tissue-specific%20CpG%20island%20shores.Nat%20Genet41178186" data-title="The%20human%20colon%20cancer%20methylome%20shows%20similar%20hypo-%20and%20hypermethylation%20at%20conserved%20tissue-specific%20CpG%20island%20shores." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+human+colon+cancer+methylome+shows+similar+hypo-+and+hypermethylation+at+conserved+tissue-specific+CpG+island+shores.+Irizarry+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref106"><span class="order">106.
            </span><a name="pone.0033393-Landi1" id="pone.0033393-Landi1" class="link-target"></a>Landi D, Gemignani F, Naccarati A, Pardini B, Vodicka P, et al.  (2008) Polymorphisms within micro-RNA-binding sites and risk of sporadic colorectal cancer. Carcinogenesis  29: 579–584.  <ul class="reflinks"><li><a href="#" data-author="Landi" data-cit="LandiDGemignaniFNaccaratiAPardiniBVodickaP2008Polymorphisms%20within%20micro-RNA-binding%20sites%20and%20risk%20of%20sporadic%20colorectal%20cancer.Carcinogenesis29579584" data-title="Polymorphisms%20within%20micro-RNA-binding%20sites%20and%20risk%20of%20sporadic%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Polymorphisms+within+micro-RNA-binding+sites+and+risk+of+sporadic+colorectal+cancer.+Landi+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref107"><span class="order">107.
            </span><a name="pone.0033393-Team1" id="pone.0033393-Team1" class="link-target"></a>Team RDC (2008) R: A Language and Environment for Statistical Computing.   <ul class="find-nolinks"></ul></li><li id="ref108"><span class="order">108.
            </span><a name="pone.0033393-Huang9" id="pone.0033393-Huang9" class="link-target"></a>Huang T, Wan S, Xu Z, Zheng Y, Feng KY, et al.  (2011) Analysis and prediction of translation rate based on sequence and functional features of the mRNA. PLoS One  6: e16036.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTWanSXuZZhengYFengKY2011Analysis%20and%20prediction%20of%20translation%20rate%20based%20on%20sequence%20and%20functional%20features%20of%20the%20mRNA.PLoS%20One6e16036" data-title="Analysis%20and%20prediction%20of%20translation%20rate%20based%20on%20sequence%20and%20functional%20features%20of%20the%20mRNA." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+and+prediction+of+translation+rate+based+on+sequence+and+functional+features+of+the+mRNA.+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref109"><span class="order">109.
            </span><a name="pone.0033393-Huang10" id="pone.0033393-Huang10" class="link-target"></a>Huang T, Chen L, Liu X-J, Cai Y-D (2011) Predicting triplet of transcription factor - mediating enzyme - target gene by functional profiles. Neurocomputing  74: 3677–3681.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangTChenLLiuX-JCaiY-D2011Predicting%20triplet%20of%20transcription%20factor%20-%20mediating%20enzyme%20-%20target%20gene%20by%20functional%20profiles.Neurocomputing7436773681" data-title="Predicting%20triplet%20of%20transcription%20factor%20-%20mediating%20enzyme%20-%20target%20gene%20by%20functional%20profiles." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Predicting+triplet+of+transcription+factor+-+mediating+enzyme+-+target+gene+by+functional+profiles.+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

