



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>The cAMP response element binding protein (CREB) plays key roles in differentiation of embryonic skeletal muscle progenitors and survival of adult skeletal muscle. However, little is known about the physiologic signals that activate CREB in normal muscle. Here we show that CREB phosphorylation and target genes are induced after acute muscle injury and during regeneration due to genetic mutation. Activated CREB localizes to both myogenic precursor cells and newly regenerating myofibers within regenerating areas. Moreover, we found that signals from damaged skeletal muscle tissue induce CREB phosphorylation and target gene expression in primary mouse myoblasts. An activated CREB mutant (CREBY134F) potentiates myoblast proliferation as well as expression of early myogenic transcription factors in cultured primary myocytes. Consistently, activated CREB-YF promotes myoblast proliferation after acute muscle injury <em>in vivo</em> and enhances muscle regeneration in dystrophic <em>mdx</em> mice. Our findings reveal a new physiologic function for CREB in contributing to skeletal muscle regeneration.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Stewart R, Flechner L, Montminy M, Berdeaux R (2011) CREB Is Activated by Muscle Injury and Promotes Muscle Regeneration. PLoS ONE 6(9):
           e24714.
        
        https://doi.org/10.1371/journal.pone.0024714</p><p><strong>Editor: </strong>Laszlo Tora, Institute of Genetics and Molecular and Cellular Biology, France</p><p><strong>Received: </strong>May 18, 2011; <strong>Accepted: </strong>August 16, 2011; <strong>Published: </strong> September 13, 2011</p><p><strong>Copyright: </strong> Â© 2011 Stewart et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Funding was provided by the Muscular Dystrophy Association (MDA68640, RB), the NIH National Institute of Diabetes and Digestive and Kidney Diseases (R01 DK049777 and DK083834, MM; DK58337 Texas Medical Center Digestive Diseases Center, RB), the NIH National Institute of Arthritis and Musculoskeletal and Skin Diseases (R01 AR059847, RB) and the University of Texas Health Science Center at Houston. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Vertebrate myogenesis is controlled by cascades of muscle-specific transcription factors, which dictate myogenic specification and differentiation, as well as repair of damaged adult skeletal muscle <a href="#pone.0024714-Charge1" class="ref-tip">[1]</a>. The second messenger cAMP and the cAMP-responsive transcription factor CREB are temporally regulated during myogenesis and required for somite development in mouse embryos <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>, <a href="#pone.0024714-Keren1" class="ref-tip">[3]</a>, <a href="#pone.0024714-Zalin1" class="ref-tip">[4]</a>. Agents that induce cAMP signaling improve muscle strength in humans and mice with muscle disease <a href="#pone.0024714-Lynch1" class="ref-tip">[5]</a>, but little is known about how cAMP-dependent transcription in myogenic precursor cells may contribute to regeneration of damaged adult muscle.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Numerous extracellular signals including those that increase cAMP induce CREB phosphorylation on a conserved serine residue (Ser133) that is required for recruitment of the related histone acetyltransferases CBP/p300 <a href="#pone.0024714-Mayr1" class="ref-tip">[reviewed in 6]</a>. Although it is currently unknown what signals induce CREB(S133) phosphorylation in myoblasts within adult skeletal muscle, genetic studies in mice have shown that CREB activity is required for muscle development and survival. Genetic deletion of <em>Creb</em> or expression of a dominant CREB inhibitor termed A-CREB impairs myotome development in mice, possibly via regulation of the myogenic regulators <em>Pax3</em> and <em>Myf5</em> <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>. Additionally, transgenic expression of A-CREB in mature myofibers causes muscle degeneration <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. CREB promotes survival of differentiated muscle by transcriptional induction of the target gene salt inducible kinase 1 (<em>Sik1</em>), which couples CREB and MEF2 transcription by direct phosphorylation and inhibition of class II histone deacetylases <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. CREB has also been shown to regulate <em>RB</em> <a href="#pone.0024714-Magenta1" class="ref-tip">[8]</a> and <em>follistatin</em> <a href="#pone.0024714-Iezzi1" class="ref-tip">[9]</a> transcription during myogenic differentiation, suggesting that CREB is involved in terminal cell cycle arrest and fusion during myogenesis. Together these findings show that CREB is an important regulator of multiple stages of muscle differentiation and survival, likely via distinct sets of target genes.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Myogenic differentiation not only occurs during muscle development, but also during muscle regeneration <a href="#pone.0024714-Charge1" class="ref-tip">[1]</a>. Upon acute muscle injury or damage due to genetic mutations, resident muscle stem cells, or satellite cells, become activated, proliferate, migrate to the site of damage and fuse with each other and existing myofibers to restore muscle structure. Injured muscle releases numerous signaling molecules including growth factors (HGF, FGFs, PDGF), Wnts, TGF-beta family ligands, and G-protein coupled receptor ligands <a href="#pone.0024714-Charge1" class="ref-tip">[1]</a>, <a href="#pone.0024714-Griffin1" class="ref-tip">[10]</a>. These signals promote regeneration in part by activating quiescent satellite cells and providing homing cues for migrating myoblasts and macrophages. As many of these signals activate CREB <a href="#pone.0024714-Mayr1" class="ref-tip">[6]</a> and CREB activity is required for myogenic differentiation during embryogenesis <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>, CREB is ideally situated to mediate regenerative responses to signals released in damaged skeletal muscle. However, it is still unknown whether CREB activity is dynamically regulated in myoblasts after muscle injury and how CREB contributes to muscle regeneration. Mouse models with persistent CREB inhibition do not permit analysis of CREB action in this dynamic setting, so we tested the hypothesis that CREB activation contributes to regeneration using primary mouse myoblasts and knock-in mice expressing activated CREB. We show that CREB phosphorylation and target genes are activated in response to skeletal muscle injury and that activated CREB drives myoblast proliferation. Moreover, genetic activation of CREB promotes proliferation after acute muscle damage and regeneration in mice with muscular dystrophy. Our data support a model in which CREB promotes satellite cell proliferation and skeletal muscle regeneration after muscle injury.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results and Discussion"></a><h2>Results and Discussion</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="CREB is activated in response to muscle damage"></a>
<h3>CREB is activated in response to muscle damage</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>To characterize the role of CREB in skeletal muscle regeneration, we injected the snake venom component cardiotoxin into mouse gastrocnemius muscles. Cardiotoxin is commonly used to induce muscle regeneration in experimental models <a href="#pone.0024714-Charge1" class="ref-tip">[1]</a>. After cardiotoxin injection, skeletal muscle degeneration and regeneration occur by a well-characterized process of myofiber necrosis, satellite cell activation and differentiation, myoblast proliferation and migration, and eventual myofiber regeneration <a href="#pone.0024714-Shi1" class="ref-tip">[11]</a>. Within three days of cardiotoxin injury, we observed striking induction of phosphorylated CREB(Ser133) in the injured areas (<a href="#pone-0024714-g001">Figure 1A</a> and <a href="#pone.0024714.s001">Figure S1A</a>,B), which are evident as large regions of mononucleate cells. These regions are comprised of proliferating myoblasts <a href="#pone.0024714-Shi1" class="ref-tip">[11]</a> and infiltrating immune cells <a href="#pone.0024714-Tidball1" class="ref-tip">[12]</a>. To monitor CREB transcriptional activity, we quantified mRNAs of two direct CREB target genes in muscle cells, <em>Sik1</em> and <em>Nr4a2</em> <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. Both of these genes have been previously shown to contain consensus CREB binding sites that are occupied by CREB and phospho-CREB in multiple cell types <a href="#pone.0024714-Zhang1" class="ref-tip">[13]</a>, <a href="#pone.0024714-Ravnskjaer1" class="ref-tip">[14]</a>, <a href="#pone.0024714-Conkright1" class="ref-tip">[15]</a>, including C2C12 myoblasts <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. Moreover, acute induction of these genes is blocked in myoblasts and other cell types by the dominant CREB inhibitor A-CREB <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. In skeletal muscle, <em>Sik1</em> mediates CREB-dependent myofiber survival <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. Although <em>Nr4a2</em> is induced by adrenergic agonists in muscle cells <a href="#pone.0024714-Pearen1" class="ref-tip">[reviewed in 16]</a>, the role of this orphan nuclear receptor in muscle physiology is unknown. Consistent with our results showing phosphorylated CREB in injured skeletal muscle tissue, we found that cardiotoxin treatment induced <em>Sik1</em> and <em>Nr4a2</em> mRNAs, which peaked 3 days post-injury (<a href="#pone-0024714-g001">Figure 1B</a> and not shown). This time point corresponds to the period of rapid myoblast proliferation <a href="#pone.0024714-Yan1" class="ref-tip">[17]</a>.</p>
<a class="link-target" id="pone-0024714-g001" name="pone-0024714-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0024714.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0024714.g001" data-doi="info:doi/10.1371/journal.pone.0024714" data-uri="info:doi/10.1371/journal.pone.0024714.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0024714.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> CREB is activated by muscle injury.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>A) Immunohistochemistry with anti-pCREB(S133) antibody shown on uninjured and injured contralateral muscles 3 days after cardiotoxin (CTX) injection; inset, no primary antibody control (npc). B) CREB target gene mRNAs <em>Sik1</em> and <em>Nr4a2</em> in mouse gastrocnemius muscle 3 days after CTX injection. Relative mRNA amounts to 18 S rRNA, normalized to naÃ¯ve muscle. <em>nâ=â</em>2 naÃ¯ve, 3 CTX. * <em>p</em>&lt;.05, ** <em>p</em>&lt;.01. Similar data were obtained on an independent cohort. C) <em>In situ</em> hybridization with <em>Nr4a2</em> antisense riboprobe on uninjured and 3 day CTX-injected contralateral legs; no riboprobe control (blank, inset). D) Phospho-CREB immunohistochemistry and npc on gastrocnemius muscle from a 4-week old <em>Dmd<sup>mdx</sup></em> mouse. E) <em>Sik1</em> mRNA in wild-type and <em>Dmd<sup>mdx</sup></em> gastrocnemius tissue from 4-week old mice. mRNA normalized to <em>Gapdh</em>, averaged by genotype (Â±stdev); <em>nâ=â</em>6 WT, 6 <em>Dmd<sup>mdx</sup></em>; ** <em>p</em>&lt;.01. F) <em>In situ</em> hybridization with antisense <em>Nr4a2</em> riboprobe or blank (control) on <em>Dmd<sup>mdx</sup></em> gastrocnemius at 4 weeks of age. G) Immunofluorescence staining of <em>Dmd<sup>mdx</sup></em> gastrocnemius (4 wk) showing pCREB (green), Pax7 (red) and merge. Filled arrowheads, co-localized Pax7 and pCREB; open arrowheads, Pax7-positive nuclei without high pCREB; dotted circles, centrally nucleated myofibers. Bars, 50 Âµm (A, C, D, F), 10 Âµm (G).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0024714.g001">
              https://doi.org/10.1371/journal.pone.0024714.g001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>Cardiotoxin injection into the gastrocnemius muscle causes local skeletal muscle degeneration and regeneration, leaving a substantial amount of the muscle tissue intact (not shown). To determine whether CREB target gene mRNA induction was localized to the damaged region of the skeletal muscle, we performed <em>in situ</em> hybridization with an antisense probe to the <em>Nr4a2</em> transcript <a href="#pone.0024714-Eichele1" class="ref-tip">[18]</a>. We found that <em>Nr4a2</em> expression was strongly localized to the injured region (<a href="#pone-0024714-g001">Figure 1C</a>), similar to phospho-CREB. Control serial sections incubated with no riboprobe showed no staining (<a href="#pone-0024714-g001">Figure 1C</a>, inset). We confirmed that the probe matched the previously reported pattern of <em>Nr4a2</em> expression in P14.5 mouse embryos <a href="#pone.0024714-Eichele1" class="ref-tip">[not shown and 18]</a>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3" class="link-target"></a><p>We next investigated whether CREB activation is a general feature of regenerating skeletal muscle. Several genetic mutations in mice and humans result in skeletal muscle degeneration and regeneration; many are clinically relevant, such as the missense mutation in the <em>Dystrophin</em> gene that underlies Duchenne's muscular dystrophy <a href="#pone.0024714-Hoffman1" class="ref-tip">[19]</a>, <a href="#pone.0024714-Hoffman2" class="ref-tip">[20]</a>. We therefore tested CREB activity in skeletal muscle of dystrophin-deficient <em>Dmd<sup>mdx</sup></em> mice, a genetic model of muscular dystrophy in which a âcrisisâ of myofiber degeneration and regeneration occurs at 4 weeks of age <a href="#pone.0024714-Bulfield1" class="ref-tip">[21]</a>. Although animals at this age invariably show dramatic muscle degeneration and regeneration, the tissue architecture is heterogeneous within a given muscle. Some regions are highly infiltrated with inflammatory cells and regenerating, while others appear histologically normal. Similar to our findings in cardiotoxin injured muscle, phosphorylated CREB was enriched in regenerating areas of <em>Dmd<sup>mdx</sup></em> skeletal muscle from 4-week old animals (<a href="#pone-0024714-g001">Figure 1D</a>). Moreover, <em>Sik1</em> mRNA was increased in muscle tissue from 4-week old dystrophin-deficient animals compared with age-matched wild-type control mice (<a href="#pone-0024714-g001">Figure 1E</a>). <em>Nr4a2</em> mRNA was also abundant in regenerating regions of skeletal muscle from 28 day-old <em>Dmd<sup>mdx</sup></em> mice (<a href="#pone-0024714-g001">Figure 1F</a>). This pattern was markedly similar to the <em>Nr4a2</em> expression pattern we observed in skeletal muscle after cardiotoxin injury. Thus, CREB phosphorylation and transcriptional activity are enriched in regenerating skeletal muscle after acute injury or degeneration due to genetic mutation.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="CREB is activated in both myogenic precursor cells and regenerating myofibers"></a>
<h3>CREB is activated in both myogenic precursor cells and regenerating myofibers</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Many cell types are activated in regenerating skeletal muscle <a href="#pone.0024714-Charge1" class="ref-tip">[1]</a>, so we performed dual-labeling experiments to identify the cell type in which CREB is activated in injured skeletal muscle. Because cardiotoxin-induced CREB target gene activation coincided with the peak time point of myoblast proliferation, we hypothesized that CREB becomes activated in proliferating myoblasts. The strong unspecific staining with mouse antibodies on necrotic cardiotoxin-injured tissue precluded reliable dual-label analysis in this model (not shown). As expected, we observed enriched phospho-CREB in regenerating areas of <em>Dmd<sup>mdx</sup></em> muscle (4-week old) by immunofluorescence (<a href="#pone-0024714-g001">Figure 1G</a> and <a href="#pone.0024714.s001">Figure S1C</a>). Consistent with our hypothesis, we observed that some, but not all, Pax7-expressing satellite cells in regenerating <em>mdx</em> muscle contained high levels of phosphorylated CREB (<a href="#pone-0024714-g001">Figure 1G</a> and <a href="#pone.0024714.s001">Figure S1C</a>, arrowheads). Phosphorylated CREB was not limited to this cell type. Indeed, we observed strong staining in newly regenerated myofibers, which we identified by the characteristic central nuclei (<a href="#pone-0024714-g001">Figure 1G</a> and <a href="#pone.0024714.s001">Figure S1C</a>, dotted circles). It is possible that the unidentified mononucleate cells with high phospho-CREB staining are differentiating myocytes that lost Pax7 expression or are infiltrating macrophages and neutrophils. The strong phospho-CREB staining in newly regenerated myofibers suggests that CREB may play a role in differentiating myocytes during myofiber regeneration. This notion is consistent with previous observations that CREB activity is elevated during myogenic differentiation of C2C12 myoblasts <a href="#pone.0024714-Magenta1" class="ref-tip">[8]</a>. Further experiments will be required to fully identify the cell type(s) within injured muscle in which CREB and its target genes are activated. Nonetheless, our data show for the first time that CREB phosphorylation is induced in myogenic precursors and nascent myofibers in regenerating skeletal muscle.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Factors released from crushed muscle activate CREB in primary skeletal myoblasts"></a>
<h3>Factors released from crushed muscle activate CREB in primary skeletal myoblasts</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Having observed elevated CREB activity in regenerating skeletal muscle, we hypothesized that after injury, CREB contributes to myoblast activation and/or differentiation. To specifically investigate CREB action in myogenic cell populations, we cultured primary mouse myoblasts, which comprise the cell type that divides and differentiates to form new myofibers. As expected, CREB phosphorylation on Ser133 in these cells was induced by application of forskolin (FSK), which elevates intracellular cAMP, or basic fibroblast growth factor (bFGF), which is a potent myoblast mitogen (<a href="#pone-0024714-g002">Figure 2A</a>). To directly test whether CREB is activated in myoblasts in response to stimuli released from damaged muscle, we treated primary myoblasts with extracts from mouse skeletal muscles crushed <em>ex vivo</em>. Similar preparations have previously been shown to induce satellite cell activation <a href="#pone.0024714-Tatsumi1" class="ref-tip">[22]</a>, proliferation <a href="#pone.0024714-Chen2" class="ref-tip">[23]</a> and migration <a href="#pone.0024714-Griffin1" class="ref-tip">[10]</a>. Crushed muscle extract (CME) rapidly induced CREB phosphorylation in myoblasts (<a href="#pone-0024714-g002">Figure 2B</a> and <a href="#pone.0024714.s002">Figure S2A</a>). Intriguingly, we consistently recovered more total CREB after 5 and 10 minutes of CME treatment compared to untreated control myoblasts (<a href="#pone-0024714-g002">Figure 2B</a> and <a href="#pone.0024714.s002">Figure S2B</a>), although the molecular mechanism is unknown. Acute changes in total CREB levels are not commonly observed upon acute cellular treatments, such as FSK or bFGF (<a href="#pone-0024714-g002">Figure 2A</a>, center). In other cell types, CREB translation <a href="#pone.0024714-Cox1" class="ref-tip">[24]</a> or degradation <a href="#pone.0024714-Garat1" class="ref-tip">[25]</a>, <a href="#pone.0024714-LefrancoisMartinez1" class="ref-tip">[26]</a> can be modulated by extracellular stimuli on the order of hours. Possible regulation of CREB protein abundance in myocytes will be an interesting avenue for further study.</p>
<a class="link-target" id="pone-0024714-g002" name="pone-0024714-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0024714.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0024714.g002" data-doi="info:doi/10.1371/journal.pone.0024714" data-uri="info:doi/10.1371/journal.pone.0024714.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0024714.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> CREB is activated by crushed muscle extract in primary myoblasts.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>A) Phospho-CREB (pCREBS133), pATF1, total CREB and HSP90 in primary skeletal myoblasts treated with FSK/IBMX (cAMP) or bFGF for 10 min. B) pCREB, total CREB and HSP90 in primary skeletal myoblasts treated with crushed muscle extract (CME) for 0, 5 and 10 min. C) Amounts of <em>Sik1</em> and <em>Nr4a2</em> mRNA in primary skeletal myoblasts left untreated or incubated 1 h with PBS or CME. D) A-CREB expression in primary skeletal myoblasts blocks <em>Sik1</em> and <em>Nr4a2</em> induction by CME (1 h). In C and D, mRNA is normalized to <em>Gapdh</em>, represented as fold change above unstimulated (Â± stdev). Representative of 3 experiments. For D, treatments were normalized to GFP, unstimulated for each target gene.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0024714.g002">
              https://doi.org/10.1371/journal.pone.0024714.g002</a></p></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2" class="link-target"></a><p>CREB phosphorylation is necessary but not sufficient for activation of CREB target genes <a href="#pone.0024714-Mayr2" class="ref-tip">[27]</a>. We therefore tested whether CREB phosphorylation in CME-treated myoblasts correlates with increased CREB transcriptional activity. In agreement with increased CREB(S133) phosphorylation, the CREB target genes <em>Sik1</em> and <em>Nr4a2</em> increased in primary myoblasts treated for 1 hour with CME (<a href="#pone-0024714-g002">Figure 2C</a>). We used the dominant-negative CREB polypeptide A-CREB <a href="#pone.0024714-Ahn1" class="ref-tip">[28]</a> to verify that CREB is necessary for <em>Sik1</em> and <em>Nr4a2</em> mRNA activation by crushed muscle extract. Infection of skeletal myoblasts with adenovirus expressing GFP had little effect on activation of <em>Sik1</em> and <em>Nr4a2</em> transcription in response to cAMP-inducing agents or crushed muscle extract (<a href="#pone-0024714-g002">Figure 2D</a> and <a href="#pone.0024714.s003">Figure S3</a>). By contrast, adenoviral delivery of A-CREB severely blunted induction of both genes (<a href="#pone-0024714-g002">Figure 2D</a> and <a href="#pone.0024714.s003">Figure S3</a>). These findings are consistent with activation of CREB target genes after cardiotoxin injection <em>in vivo</em> and support the notion that at least part of the gene induction we observed in injured areas of skeletal muscle occurred in myoblasts. It is unknown what protein or small molecules in CME activates CREB in myoblasts. Numerous growth factors that could induce CREB phosphorylation are present in CME, including basic fibroblast growth factor (bFGF) <a href="#pone.0024714-Chen2" class="ref-tip">[23]</a> and an unidentified G-protein coupled receptor ligand <a href="#pone.0024714-Griffin1" class="ref-tip">[10]</a>. Our data suggest that one of these signals or an unknown factor released from injured skeletal muscle activates CREB in myoblasts within damaged muscle.</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="CREB-YF mice have normal muscle structure"></a>
<h3>CREB-YF mice have normal muscle structure</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>Phosphorylation of CREB on Ser133 promotes recruitment of CBP/p300 and loosening of chromatin at target gene promoters <a href="#pone.0024714-Mayr1" class="ref-tip">[6]</a>. Mutation of the adjacent Tyr134 to Phe (CREBY134F, <a href="#pone-0024714-g003">Figure 3A</a>) results in enhanced PKA phosphorylation and CBP recruitment <a href="#pone.0024714-Du1" class="ref-tip">[29]</a>. CREB(Y134F) therefore acts as a gain-of-function mutant when over-expressed in cells <a href="#pone.0024714-Iezzi1" class="ref-tip">[9]</a>, <a href="#pone.0024714-Jagasia1" class="ref-tip">[30]</a>, <a href="#pone.0024714-Han1" class="ref-tip">[31]</a>. To investigate physiologic effects of activated CREB <em>in vivo</em> without over-expression, we knocked the Y134F mutation into the <em>Creb1</em> locus in mice (<a href="#pone-0024714-g003">Figure 3B</a> and <a href="#pone.0024714.s004">Figure S4</a>). <em>Creb<sup>+/+</sup></em>, <em>Creb<sup>+/YF</sup></em>, and <em>Creb<sup>YF/YF</sup></em> mice were recovered at the expected Mendelian frequency (<a href="#pone.0024714.s008">Table S1</a>). Wild-type and knock-in mice were visually indistinguishable, and CREB-YF homozygotes exhibited no phenotypic abnormalities. We verified that the YF mutation did not alter CREB protein expression in skeletal muscle (<a href="#pone-0024714-g003">Figure 3C</a>). In unchallenged animals, we observed no statistically significant difference in amounts of phospho-CREB between adult <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> skeletal muscles (<a href="#pone-0024714-g003">Figure 3C</a> and <a href="#pone.0024714.s005">Figure S5</a>). Histological analysis revealed no differences between skeletal muscles of WT and YF littermates in terms of size or fiber type distribution (not shown). By contrast, expression of CREB-YF in C2C12 myoblasts induces expression of <em>follistatin</em>, which promotes myoblast fusion and myotube hypertrophy <em>in vitro</em> <a href="#pone.0024714-Iezzi1" class="ref-tip">[9]</a>. Additionally, CREB binds directly to the <em>cyctochrome c</em> <a href="#pone.0024714-Gopalakrishnan1" class="ref-tip">[32]</a> and <em>Ppargc1a</em> (encoding PGC1-alpha) promoters <a href="#pone.0024714-Akimoto1" class="ref-tip">[33]</a>, and forced expression of CREB co-activators promotes mitochondrial biogenesis in cultured myocytes <a href="#pone.0024714-Wu1" class="ref-tip">[34]</a>. Although we measured a modest increase in <em>Ppargc1a</em> mRNA levels in <em>Creb<sup>YF/YF</sup></em> muscle, we observed no difference in myofiber succinate dehydrogenase activity, exercise endurance or voluntary wheel running in these mice (not shown). We conclude that although CREB may be involved in muscle responses to exercise, endogenous expression of CREB-YF does not lead to an increase in CREB activity sufficient to promote muscle hypertrophy or mitochondrial biogenesis <em>in vivo</em>.</p>
<a class="link-target" id="pone-0024714-g003" name="pone-0024714-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0024714.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0024714.g003" data-doi="info:doi/10.1371/journal.pone.0024714" data-uri="info:doi/10.1371/journal.pone.0024714.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0024714.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> CREB-YF enhances myoblast response to cAMP.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p>A) Amino acid sequence of CREB PKA phosphorylation site (Ser133) showing Y134F mutation. B) Targeting strategy for CREB-YF knock-in strain. Mouse <em>Creb1</em> locus with Y134 in exon 5 (top); targeting construct inserts neo cassette in intron 4 and Y134F/<em>StuI</em> mutation (middle); targeted locus contains Y134F mutation. S, B: <em>StuI</em>, <em>BamHI</em> restriction sites. Southern probes indicated. C) Phospho-CREB, CREB and HSP90 protein in gastrocnemius muscle from <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> mice. <em>nâ=â</em>4 per genotype. D) <em>Nr4a2</em> mRNA in primary skeletal myoblasts from wild-type and CREB-YF mice, stimulated with FSK/IBMX for indicated times. <em>Nr4a2</em> normalized to <em>Gapdh</em>, expressed in arbitrary units (A.U.). **, <em>p</em>&lt;.01; *, <em>p</em>&lt;.05 between genotypes. Average of 3 experiments Â± stdev.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0024714.g003">
              https://doi.org/10.1371/journal.pone.0024714.g003</a></p></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>To assay CREB-YF transcriptional activity, we monitored CREB target gene expression in primary myoblasts from <em>Creb</em><sup>+/+</sup> and <em>Creb<sup>YF/YF</sup></em> littermates. Under basal conditions, we observed a statistically significant increase in <em>Nr4a2</em> mRNA levels (<a href="#pone-0024714-g003">Figure 3D</a>). When stimulated with saturating concentrations of FSK, primary myoblasts of both genotypes had identical maximal expression of <em>Nr4a2</em> mRNA (not shown). However, we observed stronger <em>Nr4a2</em> mRNA induction in <em>Creb<sup>YF/YF</sup></em> myoblasts compared to controls upon exposure to a low concentration of FSK (<a href="#pone-0024714-g003">Figure 3D</a>). Thus, CREB(Y134F) is not a constitutively active mutant, but is more sensitive to PKA signaling as previously shown <a href="#pone.0024714-Du1" class="ref-tip">[29]</a>. This is underscored by the fact that <em>Nr4a2</em> mRNA levels decline after prolonged FSK treatment in both <em>Creb</em><sup>+/+</sup> and <em>Creb<sup>YF/YF</sup></em> myoblasts (<a href="#pone-0024714-g003">Figure 3D</a>).</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Activated CREB promotes myoblast proliferation"></a>
<h3>Activated CREB promotes myoblast proliferation</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Because we observed striking induction of CREB activity in regenerating areas of damaged skeletal muscle (<a href="#pone-0024714-g001">Figure 1</a>), we assessed the effects of the CREB-YF gain-of-function mutation on myoblast proliferation and differentiation, two processes that occur as part of muscle regeneration. In growth-curve assays, CREB-YF myoblasts proliferated faster than CREB-WT myoblasts (<a href="#pone-0024714-g004">Figure 4A</a>). Moreover, BrdU-labeling revealed more dividing myoblasts in primary cultures from <em>Creb<sup>YF/YF</sup></em> muscle compared with <em>Creb<sup>+/+</sup></em> control muscle (<a href="#pone-0024714-g004">Figure 4B</a>). This finding is consistent with previous reports showing reduced cell proliferation in somites of <em>Creb<sup>â/â</sup></em> embryos <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>. To explore a potential transcriptional target underlying the enhanced proliferation, we measured expression of CREB target genes implicated in proliferation. We observed an increase in <em>Ccna2</em> mRNA in primary myoblasts from <em>Creb<sup>YF/YF</sup></em> mice (not shown), which is also evident at the protein level (<a href="#pone-0024714-g004">Figure 4C</a>). CREB binds to the mouse <em>Ccna2</em> promoter <a href="#pone.0024714-Song1" class="ref-tip">[35]</a>, suggesting that CREB may directly influence cell cycle progression in myoblasts. Notably, in response to cardiotoxin injection in mouse muscle, <em>Ccna2</em> expression and myoblast proliferation are maximal after 3 days <a href="#pone.0024714-Yan1" class="ref-tip">[17]</a>, coincident with peak expression of the CREB transcriptional targets <em>Sik1</em> and <em>Nr4a2</em> (<a href="#pone-0024714-g001">Figure 1B</a>).</p>
<a class="link-target" id="pone-0024714-g004" name="pone-0024714-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0024714.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0024714.g004" data-doi="info:doi/10.1371/journal.pone.0024714" data-uri="info:doi/10.1371/journal.pone.0024714.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0024714.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Activated CREB promotes myoblast proliferation and differentiation.</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>A) Growth curve assay of <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> myoblasts over 5 days. Arrows, medium added. **, <em>p</em>&lt;.01 between genotypes. B) Percent of BrdU-positive nuclei in asynchronous myoblast cultures. Means of 10 fields among 3 replicates. *, <em>p&lt;</em>.05. C) CyclinA and HSP90 protein in asynchronous myoblasts. D) Myf5 and GAPDH control in differentiating <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> myocytes. E) Muscle-specific proteins in <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> myocytes after the indicated times in differentiation medium (DM). MYOG, myogenin; MHC, myosin heavy chain; GAPDH loading control. F) Phase contrast images of 2 day differentiated <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> myotubes. Bar, 100 Âµm.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0024714.g004">
              https://doi.org/10.1371/journal.pone.0024714.g004</a></p></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>Paradoxically, CREB has also been implicated in myogenic differentiation through multiple transcriptional targets. During development, a non-canonical Wnt1-cAMP-PKA-CREB pathway drives myogenesis in part via activation of <em>Myf5</em> expression <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>. The mouse <em>Myf5</em> promoter contains several conserved CREB binding sites and a TATA box <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>, <a href="#pone.0024714-Zhang1" class="ref-tip">[13]</a>, so it is likely that CREB directly induces <em>Myf5</em> expression. However, CREB binding to the <em>Myf5</em> promoter has not been demonstrated. We tested whether CREB-YF is sufficient to drive <em>Myf5</em> expression and myogenic differentiation in primary myoblasts. In spite of the enhanced proliferation we observed when CREB-YF myoblasts were cultured in growth medium (<a href="#pone-0024714-g004">Figure 4A, B</a>), CREB-YF cells exhibited stronger upregulation of MYF5 protein within 6 hours of incubation in differentiation medium (<a href="#pone-0024714-g004">Figure 4D</a> and <a href="#pone.0024714.s006">Figure S6A</a>). Similarly, Pax7 was transiently upregulated in both the CREB-WT and CREB-YF cells, but to a greater extent in the CREB-YF cells (<a href="#pone-0024714-g004">Figures 4E</a> and <a href="#pone.0024714.s006">Figure S6B</a>). We do not know if the increased abundance of MYF5 and Pax7 proteins in CREB-YF cultures reflects the presence of more progenitor cells due to proliferation or elevated expression due to enhanced CREB activity on the <em>Myf5</em> or <em>Pax3</em> promoters <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3" class="link-target"></a><p>We tested additional cell cycle and myogenic markers in the differentiating CREB-WT and CREB-YF cultures. The cell cycle inhibitor p27 peaked approximately 12 hours later in differentiating CREB-YF myocytes than in CREB-WT controls (<a href="#pone-0024714-g004">Figure 4E</a> and <a href="#pone.0024714.s006">Figure S6C</a>), perhaps reflecting enhanced proliferation in the CREB-YF cultures after plating. Intermediate and late myogenic markers (myogenin, MEF2, myosin heavy chain) increased with similarly delayed kinetics in CREB-YF myotubes compared to controls (<a href="#pone-0024714-g004">Figure 4E</a> and <a href="#pone.0024714.s006">Figure S6D</a>-F). The peak amounts of these proteins were similar between the two genotypes, suggesting that any promotion of myogenesis by CREB occurs early, possibly as a result of <em>Myf5</em> induction. <em>Creb<sup>YF/YF</sup></em> myoblasts ultimately differentiated normally into myotubes that were morphologically indistinguishable from <em>Creb<sup>+/+</sup></em> cells (<a href="#pone-0024714-g004">Figure 4F</a>). The slight differences in myotube density in <a href="#pone-0024714-g004">Figure 4F</a> may have resulted from enhanced proliferation of CREB-YF myoblasts in the 18 hours between plating and addition of differentiation medium. Together, our findings show that activated CREB drives both myoblast proliferation and early steps in myogenic differentiation, perhaps through direct regulation of genes that drive the cell cycle and myogenesis, respectively.</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="CREB promotes proliferation and muscle regeneration in mice"></a>
<h3>CREB promotes proliferation and muscle regeneration in mice</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>We have shown that CREB phosphorylation and CREB target gene expression are elevated in damaged skeletal muscle and that activated CREB promotes myoblast proliferation and expression of early myogenic markers in primary myocytes. Next, we took advantage of our <em>in vivo</em> CREB gain-of-function mouse model to determine how increased CREB activity might affect muscle repair. We first tested whether CREB-YF is sufficient to promote proliferation <em>in vivo</em>, as it did in cultured myoblasts. In juvenile animals, we observed no difference in the number of Pax7-positive nuclei between <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> animals (not shown). Thus, CREB-YF does not increase the number of resident muscle progenitor cells in the postnatal growth period. To test whether CREB-YF promotes proliferation after acute muscle injury, we injected age-matched adult male CREB-WT and CREB-YF animals with cardiotoxin to induce a synchronous wave of muscle degeneration, myoblast proliferation and regeneration. On days 2, 3 and 4 after injury, we administered EdU to label newly synthesized DNA in proliferating cells <a href="#pone.0024714-Lepper1" class="ref-tip">[36]</a>. On day 5 after injury, we collected and analyzed the injured tissue for EdU incorporation in the injured areas (<a href="#pone-0024714-g005">Figure 5A</a>). As expected, both <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> mice had many proliferating cells in the injured regions, which were evident as large areas of mononucleate cells (<a href="#pone-0024714-g005">Figure 5A</a>, dashed outline). The total injured area in fields of <em>Creb<sup>+/+</sup></em> muscle was similar to that in <em>Creb<sup>YF/YF</sup></em> muscle (<a href="#pone.0024714.s007">Figure S7</a>). However, <em>Creb<sup>YF/YF</sup></em> mice had a significant increase in the number of EdU-positive nuclei per injured area relative to <em>Creb<sup>+/+</sup></em> controls (<a href="#pone-0024714-g005">Figure 5B</a>). This finding is consistent with our <em>in vitro</em> data showing elevated proliferation in cultured CREB-YF myoblasts.</p>
<a class="link-target" id="pone-0024714-g005" name="pone-0024714-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0024714.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0024714.g005" data-doi="info:doi/10.1371/journal.pone.0024714" data-uri="info:doi/10.1371/journal.pone.0024714.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0024714.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0024714.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0024714.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0024714.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> CREB promotes proliferation and muscle regeneration in mice.</div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1" class="link-target"></a><p>A) EdU-positive nuclei (magenta) and all nuclei (DAPI, blue) in injured region (below dotted line) of adult male <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> mice 5 days after CTX injection with EdU delivery on days 2â4 after cardiotoxin. B) Average number of EdU-positive nuclei per injured area (mm<sup>2</sup>) in 5 fields per mouse. <em>nâ=â</em>5 mice per genotype, *, <em>p</em>&lt;0.05 by 2-tailed paired <em>t</em>-test. C) Anti-dystrophin immunohistochemistry in gastrocnemius sections from 4-week old CREB-YF-<em>mdx</em> mice of indicated genotypes. Dystrophin, magenta; DAPI, green. D) Confocal micrographs of regenerating myofibers in cross-sections of CREB-YF-<em>mdx</em> gastrocnemius tissue from 4-week old mice visualized by embryonic myosin heavy chain (eMHC, magenta, broken outlines) and central nuclei (DAPI, green, arrow). E) Number of regenerating myofibers per area (Â±stdev) in fields representing an entire cross-section of the gastrocnemius muscle from animals of the indicated genotypes (4-week old). <em>nâ=â</em>6 <em>Dmd<sup>mdx</sup>Creb<sup>+/</sup></em><sup>+</sup>, 5 <em>Dmd<sup>mdx</sup>Creb<sup>YF/YF</sup></em>; *, <em>p</em>&lt;.05. F) Percent regenerating myofiber area (Â± stdev) in gastrocnemius tissue. <em>n</em>â=â3 <em>Dmd<sup>mdx</sup>Creb<sup>+/</sup></em><sup>+</sup>, 4 <em>Dmd<sup>mdx</sup>Creb<sup>YF/YF</sup></em>; *, <em>p</em>&lt;.05. Bars, 50 Âµm.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0024714.g005">
              https://doi.org/10.1371/journal.pone.0024714.g005</a></p></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2" class="link-target"></a><p>We tested effects of the <em>Creb<sup>YF</sup></em> allele on muscle regeneration by crossing the <em>Creb<sup>YF/YF</sup></em> mice with <em>mdx</em> mice. The CREB-YF allele did not restore dystrophin expression (<a href="#pone-0024714-g005">Figure 5C</a>), and degeneration proceeded normally in CREB-YF-<em>mdx</em> mice (plasma creatine kinase: <em>Dmd<sup>mdx</sup>Creb<sup>+/+</sup></em> 1,961Â±950 U/L; <em>Dmd<sup>mdx</sup>Creb<sup>YF/YF</sup></em> 2,295Â±267; <em>pâ=â</em>.52). We visualized regenerating myofibers in muscle sections by embryonic myosin heavy chain (eMHC) staining and centrally located nuclei (<a href="#pone-0024714-g005">Figure 5D</a>) and observed a significant increase in the number of regenerating myofibers from CREB-YF-<em>mdx</em> compared to CREB-WT-<em>mdx</em> controls (<a href="#pone-0024714-g005">Figure 5E</a>). Moreover, the total cross-sectional area of actively regenerating myofibers was â¼30% greater in <em>Dmd<sup>mdx</sup>Creb<sup>YF/YF</sup></em> than control <em>Dmd<sup>mdx</sup>Creb<sup>+/+</sup></em> mice (<a href="#pone-0024714-g005">Figure 5F</a>). These data demonstrate that genetic activation of CREB promotes myofiber regeneration in dystrophic <em>mdx</em> mice. It is possible that the enhanced regeneration we observe results from enhanced proliferation at an earlier time point. Indeed, we noted a significant increase in the proportion of small diameter myofibers in some CREB-YF-<em>mdx</em> mice compared to CREB-WT-<em>mdx</em> controls, but this trend varied among individual animals (not shown). Alternatively, the skew to smaller myofibers could result from a defect in myotube fusion, although we did not observe overt fusion defects in differentiating myocyte cultures. It is also noteworthy that the intensity of myosin heavy chain staining varied within regenerating regions, as shown in <a href="#pone-0024714-g005">Figure 5D</a>. Replacement of the embryonic myosin heavy chain isoform by adult isoforms at later stages of myofiber regeneration could account for the reduced overall intensity <a href="#pone.0024714-Whalen1" class="ref-tip">[37]</a> if CREB-YF-<em>mdx</em> myofibers were regenerating faster than CREB-WT-<em>mdx</em> myofibers. Variance in eMHC intensity did not impact our quantification, as we included all eMHC-positive and centrally nucleated myofibers (even those with no eMHC staining).</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3" class="link-target"></a><p>Because our genetic strategy results in CREB-YF expression in all cells, enhanced CREB activity in other cell types may contribute to the regenerative phenotype. This is particularly true of immune cells in which CREB has multiple roles including activation of cytokine gene expression and promotion of survival, migration and proliferation <a href="#pone.0024714-Wen1" class="ref-tip">[38]</a>. We do not know at which stage in myogenesis CREB acts to promote regeneration. Many nuclei in regenerating <em>mdx</em> muscle have readily detectable phospho-CREB. Some of those nuclei express the satellite cell marker Pax7, but many other nuclei do not (<a href="#pone-0024714-g001">Figure 1G</a> and <a href="#pone.0024714.s001">Figure S1C</a>). The strong phospho-CREB staining in central nuclei of newly regenerated myofibers suggests that perhaps CREB is activated at multiple stages of muscle regeneration. Further studies including lineage-tracing analysis will be necessary to resolve this question.</p>
<a id="article1.body1.sec2.sec6.p4" name="article1.body1.sec2.sec6.p4" class="link-target"></a><p>Our results show that genetic activation of CREB is sufficient to induce both myoblast proliferation and differentiation <em>in vitro</em> and proliferation and muscle regeneration <em>in vivo</em>. It is not known which CREB target genes mediate these phenotypes. The distinct cellular phenotypes we observed in cultured primary myoblasts correlate with differential expression of the cell cycle regulator <em>Ccna2</em> as well as accumulation of the early myogenic marker MYF5, which have been previously implicated as CREB target genes <a href="#pone.0024714-Chen1" class="ref-tip">[2]</a>, <a href="#pone.0024714-Song1" class="ref-tip">[35]</a>. These data suggest that multiple CREB target genes may be involved in the complex process of muscle regeneration. Additional genetic studies will reveal the molecular mechanisms and target genes underlying the effects of CREB on proliferation and regeneration in injured skeletal muscle.</p>
<a id="article1.body1.sec2.sec6.p5" name="article1.body1.sec2.sec6.p5" class="link-target"></a><p>Together our findings support a model in which CREB is activated in myoblasts and regenerating myofibers by stimuli released within damaged skeletal muscle tissue, whereupon CREB promotes myoblast proliferation and contributes to myofiber regeneration. We posit that the mechanisms by which CREB participates in skeletal muscle regeneration are distinct from those in axon regeneration <a href="#pone.0024714-Gao1" class="ref-tip">[39]</a> because muscle regeneration involves activation, proliferation and differentiation of resident quiescent stem cells. We are just beginning to uncover CREB-dependent molecular mechanisms that may contribute to the salutary effects of cAMP signaling on muscle strength <a href="#pone.0024714-Lynch1" class="ref-tip">[5]</a> and possibly regeneraftion in humans and mice with muscular disease.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Methods"></a><h2>Methods</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="Primary skeletal myoblasts"></a>
<h3>Primary skeletal myoblasts</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>Myofiber-associated satellite cells and myoblasts were prepared from neonatal mouse limb muscles as described <a href="#pone.0024714-Liu1" class="ref-tip">[40]</a>. Myoblasts were seeded on Matrigel in DMEM-F10, 20% FCS, 2.5 ng/mL bFGF (Peprotech). Cells were differentiated in DMEM, 2% horse serum.</p>
</div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="Cell treatments"></a>
<h3>Cell treatments</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>Myoblasts were treated with forskolin (FSK, EMD Chemicals)/3-isobutyl-1-methylxanthine (phosphodiesterase inhibitor, IBMX, Sigma) in DMSO at final concentrations of 1 ÂµM FSK/1.8 ÂµM IBMX. For bFGF stimulation, myoblasts were starved for 3 h in DMEM-F10, 3% FBS then treated with recombinant bFGF (2.5 ng/mL). For crushed muscle extract (CME), isolated gastrocnemius muscles of adult mice were crushed with blunt forceps and incubated in PBS with tumbling 1.5 h, 4Â°C. Myoblasts were treated with CME (250 Âµg/mL) or PBS (equal volume). Myoblasts were labeled with BrdU (10 ÂµM, 1 h) and analyzed by immunofluorescence, counting &gt;800 cells per condition among 3 sets of myoblasts.</p>
</div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="Adenovirus"></a>
<h3>Adenovirus</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>Recombinant adenovirus encoding GFP or Flag-A-CREB was propagated in HEK-293 cells and purified by ultracentrifugation on CsCl<sub>2</sub> density gradients and dialyzed extensively into PBS/10% glycerol as described <a href="#pone.0024714-Herzig1" class="ref-tip">[41]</a>. Freshly explanted myoblasts were allowed to adhere overnight and then infected for 6 hours. 24 hours after infection, efficiency was determined by GFP expression. Cells were then treated for 1 hr with FSK/IBMX or CME. The University of Texas Health Institutional Biosafety Committee approved the use of adenovirus (IBC-09-009).</p>
</div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="Protein extracts"></a>
<h3>Protein extracts</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Whole cell extracts were prepared from cells and tissue in RIPA buffer (20 mM HEPES pH7.5, 137 mM NaCl, 0.5% TX-100, 0.1% SDS, 0.5% Na-DOC, 0.5 mM EDTA, 5 mM Na<sub>4</sub>P<sub>2</sub>O<sub>7</sub>, 20 mM beta-glycerol phosphate, 50 mM NaF, 1 mM Na<sub>3</sub>VO<sub>4</sub>, PI cocktail) and concentration measured by BCA assay as described <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. For phospho-CREB and total CREB western blots, samples were resolved on duplicate gels in parallel. pCREB and total CREB were probed on different blots to prevent misinterpretation of data due to residual signal or loss of protein after stripping. HSP90 loading controls were performed for each blot to verify equal loading; shown for the pCREB blots.</p>
</div>

<div id="section5" class="section toc-section"><a id="s3e" name="s3e" class="link-target" title="Gene expression"></a>
<h3>Gene expression</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>cDNA was prepared and analyzed from total RNA by QPCR as described <a href="#pone.0024714-Berdeaux1" class="ref-tip">[7]</a>. Gene-specific RT was used for 18 S rRNA; oligo-dT RT was used for other transcripts. Relative mRNA amounts were normalized in each sample to an internal control and expressed in arbitrary units or fold change. See <a href="#pone.0024714.s009">Table S2</a> for primer sequences.</p>
</div>

<div id="section6" class="section toc-section"><a id="s3f" name="s3f" class="link-target" title="Antibodies"></a>
<h3>Antibodies</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>Rabbit monoclonal phospho-CREB(S133) (87G3), CREB (48H2) and p27 (D69C12), Cell Signaling; mouse monoclonal adult myosin heavy chain (MHC) MF20 <a href="#pone.0024714-Bader1" class="ref-tip">[42]</a>, embryonic (e)MHC F1.652 <a href="#pone.0024714-Webster1" class="ref-tip">[43]</a>, myogenin F5D <a href="#pone.0024714-Wright1" class="ref-tip">[44]</a> and dystrophin MANDRA clone 7A10 <a href="#pone.0024714-Ellis1" class="ref-tip">[45]</a>, Developmental Studies Hybridoma Bank; MEF2 C-21 <a href="#pone.0024714-Karasseva1" class="ref-tip">[46]</a> and HSP90 C-20 <a href="#pone.0024714-Teng1" class="ref-tip">[47]</a>, Santa Cruz; BrdU clone B44 <a href="#pone.0024714-Gratzner1" class="ref-tip">[48]</a>, BD Biosciences; CyclinA2 AF5999, R&amp;D Systems; DyLight549-AffiniPure F(ab')<sub>2</sub> Donkey anti-mouse IgG (715-506-150) and FITC-AffiniPure F(ab')<sub>2</sub> Donkey anti-rabbit IgG (711-096-152), Jackson Immunoresearch.</p>
</div>

<div id="section7" class="section toc-section"><a id="s3g" name="s3g" class="link-target" title="CREB(Y134F) mice"></a>
<h3>CREB(Y134F) mice</h3>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1" class="link-target"></a><p>A targeting vector was constructed from a 10.5-kb genomic fragment encompassing exons 3â5 of mouse <em>Creb1</em>. Site-directed mutagenesis was used to introduce the Y134F mutation. <em>Creb<sup>+/YF</sup></em> mice were backcrossed for 10 generations to C57Bl/6 to obtain <em>Creb</em><sup>+/+</sup> and <em>Creb<sup>YF/YF</sup></em> mice. See <a href="#pone.0024714.s010">Materials and Methods S1</a> for details.</p>
</div>

<div id="section8" class="section toc-section"><a id="s3h" name="s3h" class="link-target" title="Animals"></a>
<h3>Animals</h3>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1" class="link-target"></a><p>Animal experiments performed in this study were specifically approved by the IACUC Committees of the Salk Institute (01-047, 04-042) and the University of Texas Health (HSC-AWC-08-125, HSC-AWC-11-096). Breeding of transgenic rodents was also approved by the UT Health Institutional Biosafety Committee (IBC-09-009). C57Bl/6 were purchased from Harlan; ICR from Taconic; <em>Dmd<sup>mdx</sup></em> from Jackson. Cardiotoxin (10 ÂµM, 25 ÂµL per mouse, Sigma) was injected into gastrocnemius muscles of 8â10 wk mice. <em>Creb<sup>YF/YF</sup></em> males and <em>Dmd<sup>mdx/mdx</sup></em> females were crossed to generate the CREB-YF-<em>mdx</em> strain. Double heterozygous offspring were intercrossed to obtain homozygous mice. Plasma creatine kinase (CK) activity was measured from EDTA-treated tail blood of 3.5-week old mice with an enzymatic assay (Catachem, Inc) prior to isolation of muscles from the animals of desired genotype (28 days old). Genotyping protocols are provided in <a href="#pone.0024714.s010">Materials and Methods S1</a>. CME was obtained from skeletal muscles of adult ICR mice (8â24 weeks).</p>
</div>

<div id="section9" class="section toc-section"><a id="s3i" name="s3i" class="link-target" title="In vivo EdU labeling"></a>
<h3><em>In vivo</em> EdU labeling</h3>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1" class="link-target"></a><p>Male <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>YF/YF</sup></em> mice (8 wk) were anesthetized with isoflurane and gastrocnemius muscles of the right hind limb were treated with topical lidocaine (0.5%) and injected with cardiotoxin (10 ÂµM, 25 ÂµL per mouse, Sigma) on day zero of the experiment. EdU (5 mg/kg in 0.9% saline, Invitrogen) was administered via subcutaneous injection on days 2, 3, and 4 after cardiotoxin injury, and tissue was collected on day 5 of the experiment <a href="#pone.0024714-Lepper1" class="ref-tip">[36]</a>. Ketoprofen (Butler Animal Health Supply) was administered via subcutaneous injection (5 mg/kg, days 0â1; 2 mg/kg days 2â4). Isolated muscles were flash frozen in OCT in liquid nitrogen-cooled isopentane. 8-Âµm fresh frozen sections through the muscle belly were collected, fixed for 10 min in 10% buffered formalin and permeabilized in 0.3% TX-100-PBS for 20 min. EdU-labeled DNA was detected by manufacturer's directions using the Click-It EdU Alexa-647 detection kit (Invitrogen). Sections were washed in 1Ã PBS and counterstained with DAPI (2 Âµg/mL). Confocal images were collected on a Nikon A1R microscope and all visible EdU-positive nuclei in five fields for each of five animals per genotype were counted using Nikon NIS Elements software. Injured areas were identified by visualization of high concentrations of mononucleate cells in the DAPI channel and measured using Nikon NIS elements software. For each field, the number of EdU-positive nuclei was normalized to the injured area, then averaged by genotype (Â±stdev). The average injured area in mm<sup>2</sup> for each genotype was also calculated.</p>
</div>

<div id="section10" class="section toc-section"><a id="s3j" name="s3j" class="link-target" title="Immunohistochemistry"></a>
<h3>Immunohistochemistry</h3>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1" class="link-target"></a><p>Phosphorylated CREB was visualized in 8-Âµm frozen muscle sections prepared as described for EdU detection. After permeabilization and washing, sections were stained with anti-phospho-CREB antibody (1â¶100) using the MOM and ABC-HRP detection kits with NovaRed substrate (Vector Laboratories). Brightfield images were obtained on a Nikon 80i microscope with a color camera and Nikon NIS Elements software using matched light and exposure settings. For fluorescence detection, the blocking and antibody dilution buffer was PBS/10% normal goat serum/0.3% TX-100. Primary antibodies to phospho-CREB and Pax7 (1â¶100) were added for 1 hour. Secondary antibody incubations were sequential: 45 min anti-rabbit (1â¶500) and 10 min anti-mouse (1â¶2000 with 0.05% Tween-20). Sections were washed and counterstained with DAPI (2 Âµg/mL) prior to imaging on a Nikon A1R confocal microscope with sequential laser scanning.</p>
</div>

<div id="section11" class="section toc-section"><a id="s3k" name="s3k" class="link-target" title="In situ hybridization"></a>
<h3><em>In situ</em> hybridization</h3>
<a id="article1.body1.sec3.sec11.p1" name="article1.body1.sec3.sec11.p1" class="link-target"></a><p><em>In situ</em> hybridization on skeletal muscle sections was performed using an automated system as described <a href="#pone.0024714-Yaylaoglu1" class="ref-tip">[49]</a> with 4 Âµg/mL Proteinase K. An antisense digoxygenin-labeled riboprobe complementary to the mouse <em>Nr4a2</em> transcript (nucleotides 337â934) was synthesized by <em>in vitro</em> transcription, used at 300 ng/mL. Negative controls included no probe and sense probe. The sense probe exhibited strong unspecific staining in all tissues examined (embryos and muscle) and was disregarded as an inappropriate control. The <em>Nr4a2</em> antisense probe recognized the reported <em>Nr4a2</em> expression pattern in E14.5 mouse embryos <a href="#pone.0024714-Eichele1" class="ref-tip">[18]</a>, <a href="#pone.0024714-Lein1" class="ref-tip">[50]</a>. Sections were imaged using a Nikon 80i microscope with color camera and Nikon NIS Elements software.</p>
</div>

<div id="section12" class="section toc-section"><a id="s3l" name="s3l" class="link-target" title="Muscle regeneration"></a>
<h3>Muscle regeneration</h3>
<a id="article1.body1.sec3.sec12.p1" name="article1.body1.sec3.sec12.p1" class="link-target"></a><p>Cross-sections of fresh frozen tissue were stained with an eMHC antibody using the MOM detection kit (Jackson). Micrographs of tiled fields representing an entire section through the gastrocnemius muscle belly of each mouse were collected (Nikon A1 confocal). Section area, number of regenerating fibers (eMHC positive or central nucleus) and cross-sectional area of individual regenerating fibers were measured with Adobe Photoshop. Areas of myofibers at oblique angles were omitted. The percent regenerating area was the sum of individual myofiber areas divided by the total section area. Values were averaged for age-matched double homozygous animals irrespective of sex.</p>
</div>

<div id="section13" class="section toc-section"><a id="s3m" name="s3m" class="link-target" title="Statistics"></a>
<h3>Statistics</h3>
<a id="article1.body1.sec3.sec13.p1" name="article1.body1.sec3.sec13.p1" class="link-target"></a><p>Significance between means of groups was determined by the 2-tailed Student's <em>t</em>-test (Excel).</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0024714"></div><div class="supplementary-material"><a name="pone.0024714.s001" id="pone.0024714.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s001">Figure S1. </a></h3><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Localization of phospho-CREB in muscle tissue.</strong> A) Full panel of no primary control (npc) inset shown in <a href="#pone-0024714-g001">Figure 1A</a>. B) Phospho-CREB(S133) staining in contralateral legs of 2 wild-type mice (uninjured and 3d after CTX injury). Bars, 50 Âµm. C) Phospho-CREB (green) and Pax7 (red) staining in 2 <em>Dmd<sup>mdx</sup></em> mice. Filled arrowheads, Pax7-pCREB double positive nuclei; open arrowheads, Pax7-positive nuclei with low pCREB staining; outlines, regenerating myofibers. Bottom row, no primary control (npc) in the same experiment with matched imaging settings on serial sections of muscle from animal #3. Bars, 10 Âµm.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s001">https://doi.org/10.1371/journal.pone.0024714.s001</a></p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s002" id="pone.0024714.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s002">Figure S2. </a></h3><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Quantification of western blots in </strong><a href="#pone-0024714-g002"><strong>Figure 2B</strong></a><strong>.</strong> A) Ratio of pCREB/total CREB. B) Ratio of total CREB/HSP90. Data represent averages of normalized intensity among panel shown in <a href="#pone-0024714-g002">Figure 2B</a> and replicate samples. *, <em>p</em>&lt;.05; ** <em>p</em>&lt;.01. Data are representative of qualitative analysis of 4 independent sets of treated cells.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s002">https://doi.org/10.1371/journal.pone.0024714.s002</a></p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s003" id="pone.0024714.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s003">Figure S3. </a></h3><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>A-CREB blunts expression of cAMP-induced genes in primary myoblasts.</strong> CREB target gene mRNA in Ad-GFP or Ad-ACREB infected primary myoblasts treated 1 h with FSK/IBMX (cAMP). mRNA amounts normalized to <em>Gapdh</em> internal control, expressed as fold difference to GFP, unstimulated control for each target gene. Data are average Â±stdev (measurement error). Data represent 3 independent experiments.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s003">https://doi.org/10.1371/journal.pone.0024714.s003</a></p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s004" id="pone.0024714.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s004">Figure S4. </a></h3><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Southern blot of targeted mouse </strong><strong><em>Creb</em></strong><strong> locus.</strong> Southern blot of <em>Creb<sup>+/+</sup></em> and <em>Creb<sup>+/YF</sup></em> ES cell DNA digested with <em>BstXI</em> and hybridized to a 3â² probe external to the targeting cassette. 8.7-kb (<em>Creb<sup>YF</sup></em>) and 7.4-kb (<em>Creb<sup>+</sup></em>) bands indicated.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s004">https://doi.org/10.1371/journal.pone.0024714.s004</a></p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s005" id="pone.0024714.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s005">Figure S5. </a></h3><a id="article1.body1.sec4.supplementary-material5.caption1.p1" name="article1.body1.sec4.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Quantification of western blot data shown in </strong><a href="#pone-0024714-g003"><strong>Figure 3C</strong></a><strong>.</strong> Average pCREB/total CREB ratio (Â±stdev). <em>n</em>â=â4 mice per genotype.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s005">https://doi.org/10.1371/journal.pone.0024714.s005</a></p><a id="article1.body1.sec4.supplementary-material5.caption1.p2" name="article1.body1.sec4.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s006" id="pone.0024714.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s006">Figure S6. </a></h3><a id="article1.body1.sec4.supplementary-material6.caption1.p1" name="article1.body1.sec4.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Quantification of western blot data shown in </strong><a href="#pone-0024714-g004"><strong>Figure 4D, 4E</strong></a><strong>.</strong> Densitometric ratios for indicated protein/GAPDH control on same blot shown in arbitrary units (A.U.). <em>Creb<sup>+/+</sup></em> (black bars) and <em>Creb<sup>YF/YF</sup></em> (open bars) incubated in differentiation medium (DM) for the indicated times in hours. A) MYF5, B) PAX7, C) p27, D) Myogenin (MYOG), E) MEF2, F) Myosin heavy chain (MHC).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s006">https://doi.org/10.1371/journal.pone.0024714.s006</a></p><a id="article1.body1.sec4.supplementary-material6.caption1.p2" name="article1.body1.sec4.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s007" id="pone.0024714.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s007">Figure S7. </a></h3><a id="article1.body1.sec4.supplementary-material7.caption1.p1" name="article1.body1.sec4.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Average injured area after cardiotoxin treatment.</strong> Average mononucleated area (mm<sup>2</sup>) in cardiotoxin injured muscle 5 days after injury. Average of 5 fields per mouse Â±stdev. <em>n</em>â=â5 mice per genotype.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s007">https://doi.org/10.1371/journal.pone.0024714.s007</a></p><a id="article1.body1.sec4.supplementary-material7.caption1.p2" name="article1.body1.sec4.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s008" id="pone.0024714.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s008">Table S1. </a></h3><a id="article1.body1.sec4.supplementary-material8.caption1.p1" name="article1.body1.sec4.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Summary of offspring statistics from </strong><strong><em>Creb<sup>+/YF</sup></em></strong><strong> intercrosses.</strong> Sex and genotype data from 20 litters (134 pups) of <em>Creb<sup>+/YF</sup></em> heterozygous intercrosses. Two pups of unknown genotype (but known sex) were excluded from genotype summaries. Expected offspring ratios are shown. Chi squared probability tests were applied with 1 (sex) or 2 (genotype) degrees of freedom with continuity correction as appropriate to determine <em>p</em> values.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s008">https://doi.org/10.1371/journal.pone.0024714.s008</a></p><a id="article1.body1.sec4.supplementary-material8.caption1.p2" name="article1.body1.sec4.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s009" id="pone.0024714.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s009">Table S2. </a></h3><a id="article1.body1.sec4.supplementary-material9.caption1.p1" name="article1.body1.sec4.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Oligonucleotide primers utilized for QPCR assays.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s009">https://doi.org/10.1371/journal.pone.0024714.s009</a></p><a id="article1.body1.sec4.supplementary-material9.caption1.p2" name="article1.body1.sec4.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0024714.s010" id="pone.0024714.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0024714.s010">Materials and Methods S1. </a></h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0024714.s010">https://doi.org/10.1371/journal.pone.0024714.s010</a></p><a id="article1.body1.sec4.supplementary-material10.caption1.p1" name="article1.body1.sec4.supplementary-material10.caption1.p1" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The authors thank Drs. Tony Hunter, Kuo-Fen Lee, Yi-Ping Li, and Nicholas Justice for reagents and advice. We thank Olga Chumakova in the University of Texas Health Integrative Biology and Pharmacology Cytodynamic Imaging Facility and Cecilia Ljungberg in the Baylor College of Medicine Intellectual and Developmental Disabilities Research Center (IDDRC) RNA <em>In Situ</em> Hybridization core for assistance with experiments.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: RB RS MM LF. Performed the experiments: RB LF RS. Analyzed the data: RB RS LF MM. Contributed reagents/materials/analysis tools: LF MM. Wrote the paper: RB. Conceived, designed, and created the CREB-YF animal model: LF MM. Conceived, designed and performed the experiments on muscle and myocytes: RB RS.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0024714-Charge1" id="pone.0024714-Charge1" class="link-target"></a>Charge SB, Rudnicki MA (2004) Cellular and molecular regulation of muscle regeneration. Physiol Rev  84: 209â238.  <ul class="reflinks"><li><a href="#" data-author="Charge" data-cit="ChargeSBRudnickiMA2004Cellular%20and%20molecular%20regulation%20of%20muscle%20regeneration.Physiol%20Rev84209238" data-title="Cellular%20and%20molecular%20regulation%20of%20muscle%20regeneration." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cellular+and+molecular+regulation+of+muscle+regeneration.+Charge+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0024714-Chen1" id="pone.0024714-Chen1" class="link-target"></a>Chen AE, Ginty DD, Fan CM (2005) Protein kinase A signalling via CREB controls myogenesis induced by Wnt proteins. Nature  433: 317â322.  <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenAEGintyDDFanCM2005Protein%20kinase%20A%20signalling%20via%20CREB%20controls%20myogenesis%20induced%20by%20Wnt%20proteins.Nature433317322" data-title="Protein%20kinase%20A%20signalling%20via%20CREB%20controls%20myogenesis%20induced%20by%20Wnt%20proteins." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Protein+kinase+A+signalling+via+CREB+controls+myogenesis+induced+by+Wnt+proteins.+Chen+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0024714-Keren1" id="pone.0024714-Keren1" class="link-target"></a>Keren A, Keren-Politansky A, Bengal E (2008) A p38 MAPK-CREB pathway functions to pattern mesoderm in Xenopus. Dev Biol  322: 86â94.  <ul class="reflinks"><li><a href="#" data-author="Keren" data-cit="KerenAKeren-PolitanskyABengalE2008A%20p38%20MAPK-CREB%20pathway%20functions%20to%20pattern%20mesoderm%20in%20Xenopus.Dev%20Biol3228694" data-title="A%20p38%20MAPK-CREB%20pathway%20functions%20to%20pattern%20mesoderm%20in%20Xenopus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+p38+MAPK-CREB+pathway+functions+to+pattern+mesoderm+in+Xenopus.+Keren+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0024714-Zalin1" id="pone.0024714-Zalin1" class="link-target"></a>Zalin RJ, Montague W (1975) Changes in cyclic AMP, adenylate cyclase and protein kinase levels during the development of embryonic chick skeletal muscle. Exp Cell Res  93: 55â62.  <ul class="reflinks"><li><a href="#" data-author="Zalin" data-cit="ZalinRJMontagueW1975Changes%20in%20cyclic%20AMP%2C%20adenylate%20cyclase%20and%20protein%20kinase%20levels%20during%20the%20development%20of%20embryonic%20chick%20skeletal%20muscle.Exp%20Cell%20Res935562" data-title="Changes%20in%20cyclic%20AMP%2C%20adenylate%20cyclase%20and%20protein%20kinase%20levels%20during%20the%20development%20of%20embryonic%20chick%20skeletal%20muscle." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Changes+in+cyclic+AMP%2C+adenylate+cyclase+and+protein+kinase+levels+during+the+development+of+embryonic+chick+skeletal+muscle.+Zalin+1975" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0024714-Lynch1" id="pone.0024714-Lynch1" class="link-target"></a>Lynch GS, Ryall JG (2008) Role of beta-adrenoceptor signaling in skeletal muscle: implications for muscle wasting and disease. Physiol Rev  88: 729â767.  <ul class="reflinks"><li><a href="#" data-author="Lynch" data-cit="LynchGSRyallJG2008Role%20of%20beta-adrenoceptor%20signaling%20in%20skeletal%20muscle%3A%20implications%20for%20muscle%20wasting%20and%20disease.Physiol%20Rev88729767" data-title="Role%20of%20beta-adrenoceptor%20signaling%20in%20skeletal%20muscle%3A%20implications%20for%20muscle%20wasting%20and%20disease." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Role+of+beta-adrenoceptor+signaling+in+skeletal+muscle%3A+implications+for+muscle+wasting+and+disease.+Lynch+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0024714-Mayr1" id="pone.0024714-Mayr1" class="link-target"></a>Mayr B, Montminy M (2001) Transcriptional regulation by the phosphorylation-dependent factor CREB. Nat Rev Mol Cell Biol  2: 599â609.  <ul class="reflinks"><li><a href="#" data-author="Mayr" data-cit="MayrBMontminyM2001Transcriptional%20regulation%20by%20the%20phosphorylation-dependent%20factor%20CREB.Nat%20Rev%20Mol%20Cell%20Biol2599609" data-title="Transcriptional%20regulation%20by%20the%20phosphorylation-dependent%20factor%20CREB." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptional+regulation+by+the+phosphorylation-dependent+factor+CREB.+Mayr+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0024714-Berdeaux1" id="pone.0024714-Berdeaux1" class="link-target"></a>Berdeaux R, Goebel N, Banaszynski L, Takemori H, Wandless T, et al.  (2007) SIK1 is a class II HDAC kinase that promotes survival of skeletal myocytes. Nat Med  13: 597â603.  <ul class="reflinks"><li><a href="#" data-author="Berdeaux" data-cit="BerdeauxRGoebelNBanaszynskiLTakemoriHWandlessT2007SIK1%20is%20a%20class%20II%20HDAC%20kinase%20that%20promotes%20survival%20of%20skeletal%20myocytes.Nat%20Med13597603" data-title="SIK1%20is%20a%20class%20II%20HDAC%20kinase%20that%20promotes%20survival%20of%20skeletal%20myocytes." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=SIK1+is+a+class+II+HDAC+kinase+that+promotes+survival+of+skeletal+myocytes.+Berdeaux+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0024714-Magenta1" id="pone.0024714-Magenta1" class="link-target"></a>Magenta A, Cenciarelli C, De Santa F, Fuschi P, Martelli F, et al.  (2003) MyoD stimulates RB promoter activity via the CREB/p300 nuclear transduction pathway. Mol Cell Biol  23: 2893â2906.  <ul class="reflinks"><li><a href="#" data-author="Magenta" data-cit="MagentaACenciarelliCDe%20SantaFFuschiPMartelliF2003MyoD%20stimulates%20RB%20promoter%20activity%20via%20the%20CREB%2Fp300%20nuclear%20transduction%20pathway.Mol%20Cell%20Biol2328932906" data-title="MyoD%20stimulates%20RB%20promoter%20activity%20via%20the%20CREB%2Fp300%20nuclear%20transduction%20pathway." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MyoD+stimulates+RB+promoter+activity+via+the+CREB%2Fp300+nuclear+transduction+pathway.+Magenta+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0024714-Iezzi1" id="pone.0024714-Iezzi1" class="link-target"></a>Iezzi S, Di Padova M, Serra C, Caretti G, Simone C, et al.  (2004) Deacetylase inhibitors increase muscle cell size by promoting myoblast recruitment and fusion through induction of follistatin. Dev Cell  6: 673â684.  <ul class="reflinks"><li><a href="#" data-author="Iezzi" data-cit="IezziSDi%20PadovaMSerraCCarettiGSimoneC2004Deacetylase%20inhibitors%20increase%20muscle%20cell%20size%20by%20promoting%20myoblast%20recruitment%20and%20fusion%20through%20induction%20of%20follistatin.Dev%20Cell6673684" data-title="Deacetylase%20inhibitors%20increase%20muscle%20cell%20size%20by%20promoting%20myoblast%20recruitment%20and%20fusion%20through%20induction%20of%20follistatin." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Deacetylase+inhibitors+increase+muscle+cell+size+by+promoting+myoblast+recruitment+and+fusion+through+induction+of+follistatin.+Iezzi+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0024714-Griffin1" id="pone.0024714-Griffin1" class="link-target"></a>Griffin CA, Kafadar KA, Pavlath GK (2009) MOR23 promotes muscle regeneration and regulates cell adhesion and migration. Dev Cell  17: 649â661.  <ul class="reflinks"><li><a href="#" data-author="Griffin" data-cit="GriffinCAKafadarKAPavlathGK2009MOR23%20promotes%20muscle%20regeneration%20and%20regulates%20cell%20adhesion%20and%20migration.Dev%20Cell17649661" data-title="MOR23%20promotes%20muscle%20regeneration%20and%20regulates%20cell%20adhesion%20and%20migration." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MOR23+promotes+muscle+regeneration+and+regulates+cell+adhesion+and+migration.+Griffin+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0024714-Shi1" id="pone.0024714-Shi1" class="link-target"></a>Shi X, Garry DJ (2006) Muscle stem cells in development, regeneration, and disease. Genes Dev  20: 1692â1708.  <ul class="reflinks"><li><a href="#" data-author="Shi" data-cit="ShiXGarryDJ2006Muscle%20stem%20cells%20in%20development%2C%20regeneration%2C%20and%20disease.Genes%20Dev2016921708" data-title="Muscle%20stem%20cells%20in%20development%2C%20regeneration%2C%20and%20disease." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Muscle+stem+cells+in+development%2C+regeneration%2C+and+disease.+Shi+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0024714-Tidball1" id="pone.0024714-Tidball1" class="link-target"></a>Tidball JG (1995) Inflammatory cell response to acute muscle injury. Med Sci Sports Exerc  27: 1022â1032.  <ul class="reflinks"><li><a href="#" data-author="Tidball" data-cit="TidballJG1995Inflammatory%20cell%20response%20to%20acute%20muscle%20injury.Med%20Sci%20Sports%20Exerc2710221032" data-title="Inflammatory%20cell%20response%20to%20acute%20muscle%20injury." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Inflammatory+cell+response+to+acute+muscle+injury.+Tidball+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0024714-Zhang1" id="pone.0024714-Zhang1" class="link-target"></a>Zhang X, Odom DT, Koo SH, Conkright MD, Canettieri G, et al.  (2005) Genome-wide analysis of cAMP-response element binding protein occupancy, phosphorylation, and target gene activation in human tissues. Proc Natl Acad Sci U S A  102: 4459â4464.  <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangXOdomDTKooSHConkrightMDCanettieriG2005Genome-wide%20analysis%20of%20cAMP-response%20element%20binding%20protein%20occupancy%2C%20phosphorylation%2C%20and%20target%20gene%20activation%20in%20human%20tissues.Proc%20Natl%20Acad%20Sci%20U%20S%20A10244594464" data-title="Genome-wide%20analysis%20of%20cAMP-response%20element%20binding%20protein%20occupancy%2C%20phosphorylation%2C%20and%20target%20gene%20activation%20in%20human%20tissues." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+analysis+of+cAMP-response+element+binding+protein+occupancy%2C+phosphorylation%2C+and+target+gene+activation+in+human+tissues.+Zhang+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0024714-Ravnskjaer1" id="pone.0024714-Ravnskjaer1" class="link-target"></a>Ravnskjaer K, Kester H, Liu Y, Zhang X, Lee D, et al.  (2007) Cooperative interactions between CBP and TORC2 confer selectivity to CREB target gene expression. EMBO J  26: 2880â2889.  <ul class="reflinks"><li><a href="#" data-author="Ravnskjaer" data-cit="RavnskjaerKKesterHLiuYZhangXLeeD2007Cooperative%20interactions%20between%20CBP%20and%20TORC2%20confer%20selectivity%20to%20CREB%20target%20gene%20expression.EMBO%20J2628802889" data-title="Cooperative%20interactions%20between%20CBP%20and%20TORC2%20confer%20selectivity%20to%20CREB%20target%20gene%20expression." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cooperative+interactions+between+CBP+and+TORC2+confer+selectivity+to+CREB+target+gene+expression.+Ravnskjaer+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0024714-Conkright1" id="pone.0024714-Conkright1" class="link-target"></a>Conkright MD, Guzman E, Flechner L, Su AI, Hogenesch JB, et al.  (2003) Genome-wide analysis of CREB target genes reveals a core promoter requirement for cAMP responsiveness. Mol Cell  11: 1101â1108.  <ul class="reflinks"><li><a href="#" data-author="Conkright" data-cit="ConkrightMDGuzmanEFlechnerLSuAIHogeneschJB2003Genome-wide%20analysis%20of%20CREB%20target%20genes%20reveals%20a%20core%20promoter%20requirement%20for%20cAMP%20responsiveness.Mol%20Cell1111011108" data-title="Genome-wide%20analysis%20of%20CREB%20target%20genes%20reveals%20a%20core%20promoter%20requirement%20for%20cAMP%20responsiveness." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+analysis+of+CREB+target+genes+reveals+a+core+promoter+requirement+for+cAMP+responsiveness.+Conkright+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0024714-Pearen1" id="pone.0024714-Pearen1" class="link-target"></a>Pearen MA, Muscat GE (2010) Minireview: Nuclear hormone receptor 4A signaling: implications for metabolic disease. Mol Endocrinol  24: 1891â1903.  <ul class="reflinks"><li><a href="#" data-author="Pearen" data-cit="PearenMAMuscatGE2010Minireview%3A%20Nuclear%20hormone%20receptor%204A%20signaling%3A%20implications%20for%20metabolic%20disease.Mol%20Endocrinol2418911903" data-title="Minireview%3A%20Nuclear%20hormone%20receptor%204A%20signaling%3A%20implications%20for%20metabolic%20disease." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Minireview%3A+Nuclear+hormone+receptor+4A+signaling%3A+implications+for+metabolic+disease.+Pearen+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0024714-Yan1" id="pone.0024714-Yan1" class="link-target"></a>Yan Z, Choi S, Liu X, Zhang M, Schageman JJ, et al.  (2003) Highly coordinated gene regulation in mouse skeletal muscle regeneration. J Biol Chem  278: 8826â8836.  <ul class="reflinks"><li><a href="#" data-author="Yan" data-cit="YanZChoiSLiuXZhangMSchagemanJJ2003Highly%20coordinated%20gene%20regulation%20in%20mouse%20skeletal%20muscle%20regeneration.J%20Biol%20Chem27888268836" data-title="Highly%20coordinated%20gene%20regulation%20in%20mouse%20skeletal%20muscle%20regeneration." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Highly+coordinated+gene+regulation+in+mouse+skeletal+muscle+regeneration.+Yan+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0024714-Eichele1" id="pone.0024714-Eichele1" class="link-target"></a>Eichele G (2011)  <a href="http://genepaint.org">genepaint.org</a>.  <ul class="find-nolinks"></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0024714-Hoffman1" id="pone.0024714-Hoffman1" class="link-target"></a>Hoffman EP, Brown RH Jr, Kunkel LM (1987) Dystrophin: the protein product of the Duchenne muscular dystrophy locus. Cell  51: 919â928.  <ul class="reflinks"><li><a href="#" data-author="Hoffman" data-cit="HoffmanEPBrownRHJrKunkelLM1987Dystrophin%3A%20the%20protein%20product%20of%20the%20Duchenne%20muscular%20dystrophy%20locus.Cell51919928" data-title="Dystrophin%3A%20the%20protein%20product%20of%20the%20Duchenne%20muscular%20dystrophy%20locus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dystrophin%3A+the+protein+product+of+the+Duchenne+muscular+dystrophy+locus.+Hoffman+1987" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0024714-Hoffman2" id="pone.0024714-Hoffman2" class="link-target"></a>Hoffman EP, Knudson CM, Campbell KP, Kunkel LM (1987) Subcellular fractionation of dystrophin to the triads of skeletal muscle. Nature  330: 754â758.  <ul class="reflinks"><li><a href="#" data-author="Hoffman" data-cit="HoffmanEPKnudsonCMCampbellKPKunkelLM1987Subcellular%20fractionation%20of%20dystrophin%20to%20the%20triads%20of%20skeletal%20muscle.Nature330754758" data-title="Subcellular%20fractionation%20of%20dystrophin%20to%20the%20triads%20of%20skeletal%20muscle." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Subcellular+fractionation+of+dystrophin+to+the+triads+of+skeletal+muscle.+Hoffman+1987" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0024714-Bulfield1" id="pone.0024714-Bulfield1" class="link-target"></a>Bulfield G, Siller WG, Wight PA, Moore KJ (1984) X chromosome-linked muscular dystrophy (mdx) in the mouse. Proc Natl Acad Sci U S A  81: 1189â1192.  <ul class="reflinks"><li><a href="#" data-author="Bulfield" data-cit="BulfieldGSillerWGWightPAMooreKJ1984X%20chromosome-linked%20muscular%20dystrophy%20%28mdx%29%20in%20the%20mouse.Proc%20Natl%20Acad%20Sci%20U%20S%20A8111891192" data-title="X%20chromosome-linked%20muscular%20dystrophy%20%28mdx%29%20in%20the%20mouse." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=X+chromosome-linked+muscular+dystrophy+%28mdx%29+in+the+mouse.+Bulfield+1984" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0024714-Tatsumi1" id="pone.0024714-Tatsumi1" class="link-target"></a>Tatsumi R, Anderson JE, Nevoret CJ, Halevy O, Allen RE (1998) HGF/SF is present in normal adult skeletal muscle and is capable of activating satellite cells. Dev Biol  194: 114â128.  <ul class="reflinks"><li><a href="#" data-author="Tatsumi" data-cit="TatsumiRAndersonJENevoretCJHalevyOAllenRE1998HGF%2FSF%20is%20present%20in%20normal%20adult%20skeletal%20muscle%20and%20is%20capable%20of%20activating%20satellite%20cells.Dev%20Biol194114128" data-title="HGF%2FSF%20is%20present%20in%20normal%20adult%20skeletal%20muscle%20and%20is%20capable%20of%20activating%20satellite%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=HGF%2FSF+is+present+in+normal+adult+skeletal+muscle+and+is+capable+of+activating+satellite+cells.+Tatsumi+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0024714-Chen2" id="pone.0024714-Chen2" class="link-target"></a>Chen G, Quinn LS (1992) Partial characterization of skeletal myoblast mitogens in mouse crushed muscle extract. J Cell Physiol  153: 563â574.  <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenGQuinnLS1992Partial%20characterization%20of%20skeletal%20myoblast%20mitogens%20in%20mouse%20crushed%20muscle%20extract.J%20Cell%20Physiol153563574" data-title="Partial%20characterization%20of%20skeletal%20myoblast%20mitogens%20in%20mouse%20crushed%20muscle%20extract." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Partial+characterization+of+skeletal+myoblast+mitogens+in+mouse+crushed+muscle+extract.+Chen+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0024714-Cox1" id="pone.0024714-Cox1" class="link-target"></a>Cox LJ, Hengst U, Gurskaya NG, Lukyanov KA, Jaffrey SR (2008) Intra-axonal translation and retrograde trafficking of CREB promotes neuronal survival. Nat Cell Biol  10: 149â159.  <ul class="reflinks"><li><a href="#" data-author="Cox" data-cit="CoxLJHengstUGurskayaNGLukyanovKAJaffreySR2008Intra-axonal%20translation%20and%20retrograde%20trafficking%20of%20CREB%20promotes%20neuronal%20survival.Nat%20Cell%20Biol10149159" data-title="Intra-axonal%20translation%20and%20retrograde%20trafficking%20of%20CREB%20promotes%20neuronal%20survival." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Intra-axonal+translation+and+retrograde+trafficking+of+CREB+promotes+neuronal+survival.+Cox+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0024714-Garat1" id="pone.0024714-Garat1" class="link-target"></a>Garat CV, Fankell D, Erickson PF, Reusch JE, Bauer NN, et al.  (2006) Platelet-derived growth factor BB induces nuclear export and proteasomal degradation of CREB via phosphatidylinositol 3-kinase/Akt signaling in pulmonary artery smooth muscle cells. Mol Cell Biol  26: 4934â4948.  <ul class="reflinks"><li><a href="#" data-author="Garat" data-cit="GaratCVFankellDEricksonPFReuschJEBauerNN2006Platelet-derived%20growth%20factor%20BB%20induces%20nuclear%20export%20and%20proteasomal%20degradation%20of%20CREB%20via%20phosphatidylinositol%203-kinase%2FAkt%20signaling%20in%20pulmonary%20artery%20smooth%20muscle%20cells.Mol%20Cell%20Biol2649344948" data-title="Platelet-derived%20growth%20factor%20BB%20induces%20nuclear%20export%20and%20proteasomal%20degradation%20of%20CREB%20via%20phosphatidylinositol%203-kinase%2FAkt%20signaling%20in%20pulmonary%20artery%20smooth%20muscle%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Platelet-derived+growth+factor+BB+induces+nuclear+export+and+proteasomal+degradation+of+CREB+via+phosphatidylinositol+3-kinase%2FAkt+signaling+in+pulmonary+artery+smooth+muscle+cells.+Garat+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0024714-LefrancoisMartinez1" id="pone.0024714-LefrancoisMartinez1" class="link-target"></a>Lefrancois-Martinez AM, Blondet-Trichard A, Binart N, Val P, Chambon C, et al.  (2011) Transcriptional control of adrenal steroidogenesis : novel connection between JAK2 and PKA through stabilization of transcription factor CREB. J Biol Chem.   <ul class="reflinks"><li><a href="#" data-author="Lefrancois-Martinez" data-cit="Lefrancois-MartinezAMBlondet-TrichardABinartNValPChambonC2011Transcriptional%20control%20of%20adrenal%20steroidogenesis%20%3A%20novel%20connection%20between%20JAK2%20and%20PKA%20through%20stabilization%20of%20transcription%20factor%20CREB.J%20Biol%20Chem" data-title="Transcriptional%20control%20of%20adrenal%20steroidogenesis%20%3A%20novel%20connection%20between%20JAK2%20and%20PKA%20through%20stabilization%20of%20transcription%20factor%20CREB." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptional+control+of+adrenal+steroidogenesis+%3A+novel+connection+between+JAK2+and+PKA+through+stabilization+of+transcription+factor+CREB.+Lefrancois-Martinez+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0024714-Mayr2" id="pone.0024714-Mayr2" class="link-target"></a>Mayr BM, Canettieri G, Montminy MR (2001) Distinct effects of cAMP and mitogenic signals on CREB-binding protein recruitment impart specificity to target gene activation via CREB. Proc Natl Acad Sci U S A  98: 10936â10941.  <ul class="reflinks"><li><a href="#" data-author="Mayr" data-cit="MayrBMCanettieriGMontminyMR2001Distinct%20effects%20of%20cAMP%20and%20mitogenic%20signals%20on%20CREB-binding%20protein%20recruitment%20impart%20specificity%20to%20target%20gene%20activation%20via%20CREB.Proc%20Natl%20Acad%20Sci%20U%20S%20A981093610941" data-title="Distinct%20effects%20of%20cAMP%20and%20mitogenic%20signals%20on%20CREB-binding%20protein%20recruitment%20impart%20specificity%20to%20target%20gene%20activation%20via%20CREB." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Distinct+effects+of+cAMP+and+mitogenic+signals+on+CREB-binding+protein+recruitment+impart+specificity+to+target+gene+activation+via+CREB.+Mayr+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0024714-Ahn1" id="pone.0024714-Ahn1" class="link-target"></a>Ahn S, Olive M, Aggarwal S, Krylov D, Ginty DD, et al.  (1998) A dominant-negative inhibitor of CREB reveals that it is a general mediator of stimulus-dependent transcription of c-fos. Mol Cell Biol  18: 967â977.  <ul class="reflinks"><li><a href="#" data-author="Ahn" data-cit="AhnSOliveMAggarwalSKrylovDGintyDD1998A%20dominant-negative%20inhibitor%20of%20CREB%20reveals%20that%20it%20is%20a%20general%20mediator%20of%20stimulus-dependent%20transcription%20of%20c-fos.Mol%20Cell%20Biol18967977" data-title="A%20dominant-negative%20inhibitor%20of%20CREB%20reveals%20that%20it%20is%20a%20general%20mediator%20of%20stimulus-dependent%20transcription%20of%20c-fos." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+dominant-negative+inhibitor+of+CREB+reveals+that+it+is+a+general+mediator+of+stimulus-dependent+transcription+of+c-fos.+Ahn+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0024714-Du1" id="pone.0024714-Du1" class="link-target"></a>Du K, Asahara H, Jhala US, Wagner BL, Montminy M (2000) Characterization of a CREB gain-of-function mutant with constitutive transcriptional activity in vivo. Mol Cell Biol  20: 4320â4327.  <ul class="reflinks"><li><a href="#" data-author="Du" data-cit="DuKAsaharaHJhalaUSWagnerBLMontminyM2000Characterization%20of%20a%20CREB%20gain-of-function%20mutant%20with%20constitutive%20transcriptional%20activity%20in%20vivo.Mol%20Cell%20Biol2043204327" data-title="Characterization%20of%20a%20CREB%20gain-of-function%20mutant%20with%20constitutive%20transcriptional%20activity%20in%20vivo." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+a+CREB+gain-of-function+mutant+with+constitutive+transcriptional+activity+in+vivo.+Du+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0024714-Jagasia1" id="pone.0024714-Jagasia1" class="link-target"></a>Jagasia R, Steib K, Englberger E, Herold S, Faus-Kessler T, et al.  (2009) GABA-cAMP response element-binding protein signaling regulates maturation and survival of newly generated neurons in the adult hippocampus. J Neurosci  29: 7966â7977.  <ul class="reflinks"><li><a href="#" data-author="Jagasia" data-cit="JagasiaRSteibKEnglbergerEHeroldSFaus-KesslerT2009GABA-cAMP%20response%20element-binding%20protein%20signaling%20regulates%20maturation%20and%20survival%20of%20newly%20generated%20neurons%20in%20the%20adult%20hippocampus.J%20Neurosci2979667977" data-title="GABA-cAMP%20response%20element-binding%20protein%20signaling%20regulates%20maturation%20and%20survival%20of%20newly%20generated%20neurons%20in%20the%20adult%20hippocampus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=GABA-cAMP+response+element-binding+protein+signaling+regulates+maturation+and+survival+of+newly+generated+neurons+in+the+adult+hippocampus.+Jagasia+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0024714-Han1" id="pone.0024714-Han1" class="link-target"></a>Han JH, Kushner SA, Yiu AP, Cole CJ, Matynia A, et al.  (2007) Neuronal competition and selection during memory formation. Science  316: 457â460.  <ul class="reflinks"><li><a href="#" data-author="Han" data-cit="HanJHKushnerSAYiuAPColeCJMatyniaA2007Neuronal%20competition%20and%20selection%20during%20memory%20formation.Science316457460" data-title="Neuronal%20competition%20and%20selection%20during%20memory%20formation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Neuronal+competition+and+selection+during+memory+formation.+Han+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0024714-Gopalakrishnan1" id="pone.0024714-Gopalakrishnan1" class="link-target"></a>Gopalakrishnan L, Scarpulla RC (1994) Differential regulation of respiratory chain subunits by a CREB-dependent signal transduction pathway. Role of cyclic AMP in cytochrome c and COXIV gene expression. J Biol Chem  269: 105â113.  <ul class="reflinks"><li><a href="#" data-author="Gopalakrishnan" data-cit="GopalakrishnanLScarpullaRC1994Differential%20regulation%20of%20respiratory%20chain%20subunits%20by%20a%20CREB-dependent%20signal%20transduction%20pathway.%20Role%20of%20cyclic%20AMP%20in%20cytochrome%20c%20and%20COXIV%20gene%20expression.J%20Biol%20Chem269105113" data-title="Differential%20regulation%20of%20respiratory%20chain%20subunits%20by%20a%20CREB-dependent%20signal%20transduction%20pathway.%20Role%20of%20cyclic%20AMP%20in%20cytochrome%20c%20and%20COXIV%20gene%20expression." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+regulation+of+respiratory+chain+subunits+by+a+CREB-dependent+signal+transduction+pathway.+Role+of+cyclic+AMP+in+cytochrome+c+and+COXIV+gene+expression.+Gopalakrishnan+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0024714-Akimoto1" id="pone.0024714-Akimoto1" class="link-target"></a>Akimoto T, Sorg BS, Yan Z (2004) Real-time imaging of peroxisome proliferator-activated receptor-gamma coactivator-1alpha promoter activity in skeletal muscles of living mice. Am J Physiol Cell Physiol  287: C790â796.  <ul class="reflinks"><li><a href="#" data-author="Akimoto" data-cit="AkimotoTSorgBSYanZ2004Real-time%20imaging%20of%20peroxisome%20proliferator-activated%20receptor-gamma%20coactivator-1alpha%20promoter%20activity%20in%20skeletal%20muscles%20of%20living%20mice.Am%20J%20Physiol%20Cell%20Physiol287C790796" data-title="Real-time%20imaging%20of%20peroxisome%20proliferator-activated%20receptor-gamma%20coactivator-1alpha%20promoter%20activity%20in%20skeletal%20muscles%20of%20living%20mice." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Real-time+imaging+of+peroxisome+proliferator-activated+receptor-gamma+coactivator-1alpha+promoter+activity+in+skeletal+muscles+of+living+mice.+Akimoto+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0024714-Wu1" id="pone.0024714-Wu1" class="link-target"></a>Wu Z, Huang X, Feng Y, Handschin C, Gullicksen PS, et al.  (2006) Transducer of regulated CREB-binding proteins (TORCs) induce PGC-1alpha transcription and mitochondrial biogenesis in muscle cells. Proc Natl Acad Sci U S A  103: 14379â14384.  <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuZHuangXFengYHandschinCGullicksenPS2006Transducer%20of%20regulated%20CREB-binding%20proteins%20%28TORCs%29%20induce%20PGC-1alpha%20transcription%20and%20mitochondrial%20biogenesis%20in%20muscle%20cells.Proc%20Natl%20Acad%20Sci%20U%20S%20A1031437914384" data-title="Transducer%20of%20regulated%20CREB-binding%20proteins%20%28TORCs%29%20induce%20PGC-1alpha%20transcription%20and%20mitochondrial%20biogenesis%20in%20muscle%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transducer+of+regulated+CREB-binding+proteins+%28TORCs%29+induce+PGC-1alpha+transcription+and+mitochondrial+biogenesis+in+muscle+cells.+Wu+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0024714-Song1" id="pone.0024714-Song1" class="link-target"></a>Song WJ, Schreiber WE, Zhong E, Liu FF, Kornfeld BD, et al.  (2008) Exendin-4 stimulation of cyclin A2 in beta-cell proliferation. Diabetes  57: 2371â2381.  <ul class="reflinks"><li><a href="#" data-author="Song" data-cit="SongWJSchreiberWEZhongELiuFFKornfeldBD2008Exendin-4%20stimulation%20of%20cyclin%20A2%20in%20beta-cell%20proliferation.Diabetes5723712381" data-title="Exendin-4%20stimulation%20of%20cyclin%20A2%20in%20beta-cell%20proliferation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Exendin-4+stimulation+of+cyclin+A2+in+beta-cell+proliferation.+Song+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0024714-Lepper1" id="pone.0024714-Lepper1" class="link-target"></a>Lepper C, Conway SJ, Fan CM (2009) Adult satellite cells and embryonic muscle progenitors have distinct genetic requirements. Nature  460: 627â631.  <ul class="reflinks"><li><a href="#" data-author="Lepper" data-cit="LepperCConwaySJFanCM2009Adult%20satellite%20cells%20and%20embryonic%20muscle%20progenitors%20have%20distinct%20genetic%20requirements.Nature460627631" data-title="Adult%20satellite%20cells%20and%20embryonic%20muscle%20progenitors%20have%20distinct%20genetic%20requirements." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Adult+satellite+cells+and+embryonic+muscle+progenitors+have+distinct+genetic+requirements.+Lepper+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0024714-Whalen1" id="pone.0024714-Whalen1" class="link-target"></a>Whalen RG, Harris JB, Butler-Browne GS, Sesodia S (1990) Expression of myosin isoforms during notexin-induced regeneration of rat soleus muscles. Dev Biol  141: 24â40.  <ul class="reflinks"><li><a href="#" data-author="Whalen" data-cit="WhalenRGHarrisJBButler-BrowneGSSesodiaS1990Expression%20of%20myosin%20isoforms%20during%20notexin-induced%20regeneration%20of%20rat%20soleus%20muscles.Dev%20Biol1412440" data-title="Expression%20of%20myosin%20isoforms%20during%20notexin-induced%20regeneration%20of%20rat%20soleus%20muscles." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+of+myosin+isoforms+during+notexin-induced+regeneration+of+rat+soleus+muscles.+Whalen+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0024714-Wen1" id="pone.0024714-Wen1" class="link-target"></a>Wen AY, Sakamoto KM, Miller LS (2010) The role of the transcription factor CREB in immune function. J Immunol  185: 6413â6419.  <ul class="reflinks"><li><a href="#" data-author="Wen" data-cit="WenAYSakamotoKMMillerLS2010The%20role%20of%20the%20transcription%20factor%20CREB%20in%20immune%20function.J%20Immunol18564136419" data-title="The%20role%20of%20the%20transcription%20factor%20CREB%20in%20immune%20function." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+role+of+the+transcription+factor+CREB+in+immune+function.+Wen+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0024714-Gao1" id="pone.0024714-Gao1" class="link-target"></a>Gao Y, Deng K, Hou J, Bryson JB, Barco A, et al.  (2004) Activated CREB is sufficient to overcome inhibitors in myelin and promote spinal axon regeneration in vivo. Neuron  44: 609â621.  <ul class="reflinks"><li><a href="#" data-author="Gao" data-cit="GaoYDengKHouJBrysonJBBarcoA2004Activated%20CREB%20is%20sufficient%20to%20overcome%20inhibitors%20in%20myelin%20and%20promote%20spinal%20axon%20regeneration%20in%20vivo.Neuron44609621" data-title="Activated%20CREB%20is%20sufficient%20to%20overcome%20inhibitors%20in%20myelin%20and%20promote%20spinal%20axon%20regeneration%20in%20vivo." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Activated+CREB+is+sufficient+to+overcome+inhibitors+in+myelin+and+promote+spinal+axon+regeneration+in+vivo.+Gao+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0024714-Liu1" id="pone.0024714-Liu1" class="link-target"></a>Liu H, Chen SE, Jin B, Carson JA, Niu A, et al.  (2010) TIMP3: a physiological regulator of adult myogenesis. J Cell Sci  123: 2914â2921.  <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuHChenSEJinBCarsonJANiuA2010TIMP3%3A%20a%20physiological%20regulator%20of%20adult%20myogenesis.J%20Cell%20Sci12329142921" data-title="TIMP3%3A%20a%20physiological%20regulator%20of%20adult%20myogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=TIMP3%3A+a+physiological+regulator+of+adult+myogenesis.+Liu+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0024714-Herzig1" id="pone.0024714-Herzig1" class="link-target"></a>Herzig S, Long F, Jhala US, Hedrick S, Quinn R, et al.  (2001) CREB regulates hepatic gluconeogenesis through the coactivator PGC-1. Nature  413: 179â183.  <ul class="reflinks"><li><a href="#" data-author="Herzig" data-cit="HerzigSLongFJhalaUSHedrickSQuinnR2001CREB%20regulates%20hepatic%20gluconeogenesis%20through%20the%20coactivator%20PGC-1.Nature413179183" data-title="CREB%20regulates%20hepatic%20gluconeogenesis%20through%20the%20coactivator%20PGC-1." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=CREB+regulates+hepatic+gluconeogenesis+through+the+coactivator+PGC-1.+Herzig+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0024714-Bader1" id="pone.0024714-Bader1" class="link-target"></a>Bader D, Masaki T, Fischman DA (1982) Immunochemical analysis of myosin heavy chain during avian myogenesis in vivo and in vitro. J Cell Biol  95: 763â770.  <ul class="reflinks"><li><a href="#" data-author="Bader" data-cit="BaderDMasakiTFischmanDA1982Immunochemical%20analysis%20of%20myosin%20heavy%20chain%20during%20avian%20myogenesis%20in%20vivo%20and%20in%20vitro.J%20Cell%20Biol95763770" data-title="Immunochemical%20analysis%20of%20myosin%20heavy%20chain%20during%20avian%20myogenesis%20in%20vivo%20and%20in%20vitro." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Immunochemical+analysis+of+myosin+heavy+chain+during+avian+myogenesis+in+vivo+and+in+vitro.+Bader+1982" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0024714-Webster1" id="pone.0024714-Webster1" class="link-target"></a>Webster C, Silberstein L, Hays AP, Blau HM (1988) Fast muscle fibers are preferentially affected in Duchenne muscular dystrophy. Cell  52: 503â513.  <ul class="reflinks"><li><a href="#" data-author="Webster" data-cit="WebsterCSilbersteinLHaysAPBlauHM1988Fast%20muscle%20fibers%20are%20preferentially%20affected%20in%20Duchenne%20muscular%20dystrophy.Cell52503513" data-title="Fast%20muscle%20fibers%20are%20preferentially%20affected%20in%20Duchenne%20muscular%20dystrophy." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Fast+muscle+fibers+are+preferentially+affected+in+Duchenne+muscular+dystrophy.+Webster+1988" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0024714-Wright1" id="pone.0024714-Wright1" class="link-target"></a>Wright WE, Binder M, Funk W (1991) Cyclic amplification and selection of targets (CASTing) for the myogenin consensus binding site. Mol Cell Biol  11: 4104â4110.  <ul class="reflinks"><li><a href="#" data-author="Wright" data-cit="WrightWEBinderMFunkW1991Cyclic%20amplification%20and%20selection%20of%20targets%20%28CASTing%29%20for%20the%20myogenin%20consensus%20binding%20site.Mol%20Cell%20Biol1141044110" data-title="Cyclic%20amplification%20and%20selection%20of%20targets%20%28CASTing%29%20for%20the%20myogenin%20consensus%20binding%20site." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cyclic+amplification+and+selection+of+targets+%28CASTing%29+for+the+myogenin+consensus+binding+site.+Wright+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0024714-Ellis1" id="pone.0024714-Ellis1" class="link-target"></a>Ellis JM, Man NT, Morris GE, Ginjaar IB, Moorman AF, et al.  (1990) Specificity of dystrophin analysis improved with monoclonal antibodies. Lancet  336: 881â882.  <ul class="reflinks"><li><a href="#" data-author="Ellis" data-cit="EllisJMManNTMorrisGEGinjaarIBMoormanAF1990Specificity%20of%20dystrophin%20analysis%20improved%20with%20monoclonal%20antibodies.Lancet336881882" data-title="Specificity%20of%20dystrophin%20analysis%20improved%20with%20monoclonal%20antibodies." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Specificity+of+dystrophin+analysis+improved+with+monoclonal+antibodies.+Ellis+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0024714-Karasseva1" id="pone.0024714-Karasseva1" class="link-target"></a>Karasseva N, Tsika G, Ji J, Zhang A, Mao X, et al.  (2003) Transcription enhancer factor 1 binds multiple muscle MEF2 and A/T-rich elements during fast-to-slow skeletal muscle fiber type transitions. Mol Cell Biol  23: 5143â5164.  <ul class="reflinks"><li><a href="#" data-author="Karasseva" data-cit="KarassevaNTsikaGJiJZhangAMaoX2003Transcription%20enhancer%20factor%201%20binds%20multiple%20muscle%20MEF2%20and%20A%2FT-rich%20elements%20during%20fast-to-slow%20skeletal%20muscle%20fiber%20type%20transitions.Mol%20Cell%20Biol2351435164" data-title="Transcription%20enhancer%20factor%201%20binds%20multiple%20muscle%20MEF2%20and%20A%2FT-rich%20elements%20during%20fast-to-slow%20skeletal%20muscle%20fiber%20type%20transitions." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcription+enhancer+factor+1+binds+multiple+muscle+MEF2+and+A%2FT-rich+elements+during+fast-to-slow+skeletal+muscle+fiber+type+transitions.+Karasseva+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0024714-Teng1" id="pone.0024714-Teng1" class="link-target"></a>Teng SC, Chen YY, Su YN, Chou PC, Chiang YC, et al.  (2004) Direct activation of HSP90A transcription by c-Myc contributes to c-Myc-induced transformation. J Biol Chem  279: 14649â14655.  <ul class="reflinks"><li><a href="#" data-author="Teng" data-cit="TengSCChenYYSuYNChouPCChiangYC2004Direct%20activation%20of%20HSP90A%20transcription%20by%20c-Myc%20contributes%20to%20c-Myc-induced%20transformation.J%20Biol%20Chem2791464914655" data-title="Direct%20activation%20of%20HSP90A%20transcription%20by%20c-Myc%20contributes%20to%20c-Myc-induced%20transformation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Direct+activation+of+HSP90A+transcription+by+c-Myc+contributes+to+c-Myc-induced+transformation.+Teng+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0024714-Gratzner1" id="pone.0024714-Gratzner1" class="link-target"></a>Gratzner HG (1982) Monoclonal antibody to 5-bromo- and 5-iododeoxyuridine: A new reagent for detection of DNA replication. Science  218: 474â475.  <ul class="reflinks"><li><a href="#" data-author="Gratzner" data-cit="GratznerHG1982Monoclonal%20antibody%20to%205-bromo-%20and%205-iododeoxyuridine%3A%20A%20new%20reagent%20for%20detection%20of%20DNA%20replication.Science218474475" data-title="Monoclonal%20antibody%20to%205-bromo-%20and%205-iododeoxyuridine%3A%20A%20new%20reagent%20for%20detection%20of%20DNA%20replication." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Monoclonal+antibody+to+5-bromo-+and+5-iododeoxyuridine%3A+A+new+reagent+for+detection+of+DNA+replication.+Gratzner+1982" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0024714-Yaylaoglu1" id="pone.0024714-Yaylaoglu1" class="link-target"></a>Yaylaoglu MB, Titmus A, Visel A, Alvarez-Bolado G, Thaller C, et al.  (2005) Comprehensive expression atlas of fibroblast growth factors and their receptors generated by a novel robotic in situ hybridization platform. Dev Dyn  234: 371â386.  <ul class="reflinks"><li><a href="#" data-author="Yaylaoglu" data-cit="YaylaogluMBTitmusAViselAAlvarez-BoladoGThallerC2005Comprehensive%20expression%20atlas%20of%20fibroblast%20growth%20factors%20and%20their%20receptors%20generated%20by%20a%20novel%20robotic%20in%20situ%20hybridization%20platform.Dev%20Dyn234371386" data-title="Comprehensive%20expression%20atlas%20of%20fibroblast%20growth%20factors%20and%20their%20receptors%20generated%20by%20a%20novel%20robotic%20in%20situ%20hybridization%20platform." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Comprehensive+expression+atlas+of+fibroblast+growth+factors+and+their+receptors+generated+by+a+novel+robotic+in+situ+hybridization+platform.+Yaylaoglu+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0024714-Lein1" id="pone.0024714-Lein1" class="link-target"></a>Lein ES, Hawrylycz MJ, Ao N, Ayres M, Bensinger A, et al.  (2007) Genome-wide atlas of gene expression in the adult mouse brain. Nature  445: 168â176.  <ul class="reflinks"><li><a href="#" data-author="Lein" data-cit="LeinESHawrylyczMJAoNAyresMBensingerA2007Genome-wide%20atlas%20of%20gene%20expression%20in%20the%20adult%20mouse%20brain.Nature445168176" data-title="Genome-wide%20atlas%20of%20gene%20expression%20in%20the%20adult%20mouse%20brain." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+atlas+of+gene+expression+in+the+adult+mouse+brain.+Lein+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

