



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>The biosynthesis of rubber is thought to take place on the surface of rubber particles in laticifers, highly specialized cells that are present in more than 40 plant families. The small rubber particle protein (SRPP) has been supposed to be involved in rubber biosynthesis, and recently five SRPPs (TbSRPP1–5) were identified in the rubber-producing dandelion species <em>Taraxacum brevicorniculatum</em>. Here, we demonstrate by immunogold labeling that TbSRPPs are localized to rubber particles, and that rubber particles mainly consist of TbSRPP3, 4 and 5 as shown by high-resolution two-dimensional gel electrophoresis and mass spectrometric analysis. We also carried out an RNA-interference approach in transgenic plants to address the function of TbSRPPs in rubber biosynthesis as well as rubber particle morphology and stability. TbSRPP-RNAi transgenic <em>T. brevicorniculatum</em> plants showed a 40–50% reduction in the dry rubber content, but neither the rubber weight average molecular mass nor the polydispersity of the rubber were affected. Although no phenotypical differences to wild-type particles could be observed <em>in vivo</em>, rubber particles from the TbSRPP-RNAi transgenic lines were less stable and tend to rapidly aggregate in expelling latex after wounding of laticifers. Our results prove that TbSRPPs are very crucial for rubber production in <em>T. brevicorniculatum</em>, probably by contributing to a most favourable and stable rubber particle architecture for efficient rubber biosynthesis and eventually storage.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Hillebrand A, Post JJ, Wurbs D, Wahler D, Lenders M, Krzyzanek V, et al.  (2012) Down-Regulation of Small Rubber Particle Protein Expression Affects Integrity of Rubber Particles and Rubber Content in <em>Taraxacum brevicorniculatum</em>. PLoS ONE 7(7):
           e41874.
        
        https://doi.org/10.1371/journal.pone.0041874</p><p><strong>Editor: </strong>Luis Herrera-Estrella, Centro de Investigación y de Estudios Avanzados del IPN, Mexico</p><p><strong>Received: </strong>April 16, 2012; <strong>Accepted: </strong>June 26, 2012; <strong>Published: </strong> July 23, 2012</p><p><strong>Copyright: </strong> © 2012 Hillebrand et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by a grant from the European Union Framework Program 7 (project no. 212827, EU-PEARLS), the Deutsche Bundesstiftung Umwelt (AZ13255) and the Ministry of Science and Education of Germany (grant no. FKZ 0315971C). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Approximately 2,500 plants produce natural rubber poly (<em>cis</em>-1,4-isoprene), which is stored in rubber particles dispersed throughout the cytosol of either laticifers or specialized parenchyma cells <a href="#pone.0041874-Cornish1" class="ref-tip">[1]</a>. Natural rubber is synthesized by the successive addition of isopentenyl pyrophosphate (IPP) to an initiating allylic pyrophosphate, e.g. farnesyl pyrophosphate (FPP) <a href="#pone.0041874-Cornish1" class="ref-tip">[1]</a>–<a href="#pone.0041874-Kharel1" class="ref-tip">[5]</a>. This polymerization reaction is thought to take place on the surface of rubber particles, mediated by a particle bound <em>cis</em>-prenyltransferase (CPT) <a href="#pone.0041874-Cornish1" class="ref-tip">[1]</a>, <a href="#pone.0041874-Archer1" class="ref-tip">[6]</a>–<a href="#pone.0041874-Schmidt1" class="ref-tip">[8]</a>. Recently, we identified three rubber particle bound long-chain CPTs in <em>Taraxacum brevicorniculatum</em> (<em>T. brevicorniculatum</em> was erroneously labeled as <em>T. koksaghyz</em> in botanical gardens <a href="#pone.0041874-Kirschner1" class="ref-tip">[9]</a>) that appear to be the key enzymes for rubber biosynthesis in this species <a href="#pone.0041874-Schmidt1" class="ref-tip">[8]</a>, <a href="#pone.0041874-Post1" class="ref-tip">[10]</a>. Detailed analysis of rubber particles in different species revealed a conserved spherical structure with sizes ranging from 0.2–6.5 µm in <em>Ficus carica</em> and <em>Ficus benghalensis</em>, 1–2 µm in <em>Parthenium argentatum,</em> 0.08–2 µm in <em>Hevea brasiliensis</em> and 0.01–1 µm in <em>T. brevicorniculatum</em> <a href="#pone.0041874-Schmidt1" class="ref-tip">[8]</a>, <a href="#pone.0041874-Gomez1" class="ref-tip">[11]</a>–<a href="#pone.0041874-Singh1" class="ref-tip">[13]</a>. Rubber particles comprise a homogenous rubber core surrounded by a monolayer membrane containing lipids, proteins, and several minor compounds <a href="#pone.0041874-Cornish3" class="ref-tip">[12]</a>, <a href="#pone.0041874-Siler2" class="ref-tip">[14]</a>, <a href="#pone.0041874-Wood1" class="ref-tip">[15]</a>. This composite monolayer membrane probably mediates the colloidal stability of rubber particles by providing a net negative surface charge that prevents their coalescence in the cytosol <a href="#pone.0041874-Cornish3" class="ref-tip">[12]</a>, <a href="#pone.0041874-Siler2" class="ref-tip">[14]</a>. In principle, colloidal stability depends on the balance between repulsion and attractive van-der-Waals forces, and electrostatic forces are often responsible for the repulsion of colloids in a suspension <a href="#pone.0041874-Verwey1" class="ref-tip">[16]</a>–<a href="#pone.0041874-Riddick1" class="ref-tip">[18]</a>. The net surface charge of colloids can be estimated by the determination of their zeta (ζ)-potential, which is the electrical potential difference at the interface of the suspending fluid and the stationary fluid attached to the colloid surface <a href="#pone.0041874-Verwey1" class="ref-tip">[16]</a>, <a href="#pone.0041874-Riddick1" class="ref-tip">[18]</a>. In addition to repulsion through a similar (positive or negative) surface charge, repulsion of colloids can also be mediated by steric hindrance of adsorbed large molecules, such as proteins <a href="#pone.0041874-Heller1" class="ref-tip">[17]</a>, <a href="#pone.0041874-Jucker1" class="ref-tip">[19]</a>–<a href="#pone.0041874-Grasso1" class="ref-tip">[22]</a>. Amongst others, two highly homologous proteins are associated with the surface of <em>H. brasiliensis</em> rubber particles, i.e. the small rubber particle protein (SRPP; here designated as HbSRPP) and the rubber elongation factor (REF; here designated as HbREF) <a href="#pone.0041874-Dennis1" class="ref-tip">[23]</a>–<a href="#pone.0041874-Oh1" class="ref-tip">[25]</a>. SRPP homologs were also identified in <em>P. argentatum</em> (guayule homolog of SRPP, GHS <a href="#pone.0041874-Kim1" class="ref-tip">[26]</a>) and <em>T. brevicorniculatum</em> (TbSRPP1–5, previously designated as TkSRPP1–5 <a href="#pone.0041874-Schmidt2" class="ref-tip">[27]</a>). The functions of SRPP and REF are yet unclear. However, there is some evidence indicating a potential role in rubber biosynthesis: <em>In vitro</em>-experiments revealed that the addition of recombinant HbSRPP to <em>H. brasiliensis</em> rubber particles suspended in latex cytosol increased the incorporation of IPP, while addition of antibodies directed against HbSRPP and HbREF inhibited IPP incorporation <a href="#pone.0041874-Dennis2" class="ref-tip">[24]</a>, <a href="#pone.0041874-Oh1" class="ref-tip">[25]</a>. Moreover, the analysis of HbREF gene expression in <em>H. brasiliensis</em> accessions with low to high rubber yields revealed that HbREF expression correlates positively with the amount of rubber <a href="#pone.0041874-Priya1" class="ref-tip">[28]</a>. However, it has been proposed that the rubber-producing species <em>F. carica</em> and <em>F. benghalensis</em> do not possess a SRPP homolog, suggesting that SRPP is not a general and essential protein for rubber biosynthesis <a href="#pone.0041874-Singh1" class="ref-tip">[13]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>The recently identified SRPPs in <em>T. brevicorniculatum</em> (TbSRPP1–5) shared sequence identities of 41–66% to each other and 38–43% with HbSRPP. Except for TbSRPP2, all TbSRPPs are predominately expressed in latex and their spatiotemporal expression pattern correlates with latex and rubber production in <em>T. brevicorniculatum</em> <a href="#pone.0041874-Schmidt2" class="ref-tip">[27]</a>. To analyze the function of TbSRPPs <em>in vivo</em>, we generated transgenic <em>T. brevicorniculatum</em> plants with strongly reduced TbSRPP protein levels, caused by RNA-interference. We revealed that exuded latex as well as freeze-dried roots of transgenic plants contain significantly lower amounts of natural rubber compared to wild-type plants. Rubber particles in expelling latex of transgenic plants appeared to be less stable compared to wild-type, whereas neither the rubber weight average molecular mass nor the polydispersity were affected. Hence, we conclude that TbSRPPs probably support rubber biosynthesis by the stabilization of rubber particles.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="TbSRPPs are the major rubber particle proteins"></a>
<h3>TbSRPPs are the major rubber particle proteins</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>We recently identified and cloned five genes (<em>TbSRPP1–5</em>) encoding small rubber particle proteins from <em>T. brevicorniculatum</em> <a href="#pone.0041874-Schmidt2" class="ref-tip">[27]</a>. As reported for other SRPPs <a href="#pone.0041874-Singh1" class="ref-tip">[13]</a>, <a href="#pone.0041874-Oh1" class="ref-tip">[25]</a>, we first set out to determine whether TbSRPP1–5 are associated with rubber particles by two-dimensional (2-D) gel electrophoresis followed by mass spectrometry. We revealed a cluster of highly abundant protein spots with isoelectric point (pI) values between pH 4 and pH 6 (<a href="#pone-0041874-g001">Fig. 1A</a>). Mass spectrometric (MS) analysis of these protein spots generated representative peptide sequences for TbSRPP2, TbSRPP3, TbSRPP4 and TbSRPP5, whereas no peptides were detected for TbSRPP1 (Tab. S1). The most abundant rubber particle protein was TbSRPP3, followed by TbSRPP4 and TbSRPP5, while TbSRPP2 was present only in small amounts (<a href="#pone-0041874-g001">Fig. 1A</a>, Tab. S1).</p>
<a class="link-target" id="pone-0041874-g001" name="pone-0041874-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0041874.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0041874.g001" data-doi="info:doi/10.1371/journal.pone.0041874" data-uri="info:doi/10.1371/journal.pone.0041874.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0041874.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Association of TbSRPPs with rubber particles. A.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>Fifty micrograms of purified rubber particle proteins were separated by isoelectric focussing (pH 3–10 IPG strips) and subsequent SDS-PAGE. The gel was stained with colloidal Coomassie Brilliant Blue. <strong>B.</strong> Backscattered electron imaging of rubber particles labeled with 10 nm gold particles by immunodetection using the TbSRPP-antibody. <strong>C.</strong> Gold labeling of rubber particles was not appreciably detected in backscattered electron images, when the corresponding pre-immune serum was used as primary antibody. Micrographs are shown as inverted images. Scale bars  = 400 nm.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0041874.g001">
              https://doi.org/10.1371/journal.pone.0041874.g001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>Next, the presence of TbSRPPs on rubber particles was investigated by scanning electron microscopy (SEM) using a polyclonal antibody raised against TbSRPP3. In Western blot experiments, this antibody recognized all TbSRPPs with the same efficiency, most probably due to the high amino acid sequence conservation between members of this protein family <a href="#pone.0041874-Schmidt2" class="ref-tip">[27]</a>, and will therefore be referred to as anti-TbSRPP (Fig. S1, Text S1). Immunogold labeling of rubber particles using anti-TbSRPP and a gold-coupled secondary antibody with subsequent visualization by SEM directly confirmed the presence of TbSRPPs on the rubber particle surface (<a href="#pone-0041874-g001">Fig. 1B</a>). Control experiments with pre-immune serum did not show significant staining of rubber particles (<a href="#pone-0041874-g001">Fig. 1C</a>).</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Depletion of TbSRPP1-5 by RNA-interference"></a>
<h3>Depletion of TbSRPP1-5 by RNA-interference</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>To further address the function of TbSRPPs in rubber biosynthesis we developed transgenic <em>T. brevicorniculatum</em> plants with reduced <em>TbSRPPs</em> expression levels by RNA-interference (RNAi). Therefore, we introduced a <em>TbSRPP3</em>-RNAi construct into the genome of <em>T. brevicorniculatum</em> by <em>Agrobacterium-</em>mediated transformation. Out of ten independently regenerated transgenic plants, six lines displayed reduced transcript levels for all <em>TbSRPP</em> genes and were therefore designated as TbSRPP-RNAi transgenic plants (data not shown). Subsequently, three TbSRPP-RNAi transgenic plants, labeled S3, S76 and S85, were selected for further analysis of <em>TbSRPP1–5</em> expression by quantitative RT-PCR using total RNA from latex. The expression of all <em>TbSRPP</em> genes was severely reduced by the <em>TbSRPP3</em>-RNAi construct when compared to wild-type plants (<a href="#pone-0041874-g002">Fig. 2A</a>). The most severe reduction in <em>TbSRPPs</em> expression was found in S3, where only residual amounts of <em>TbSRPP5</em> are still visible.</p>
<a class="link-target" id="pone-0041874-g002" name="pone-0041874-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0041874.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0041874.g002" data-doi="info:doi/10.1371/journal.pone.0041874" data-uri="info:doi/10.1371/journal.pone.0041874.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0041874.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Analysis of <em>TbSRPP1–5</em> expression in TbSRPP-RNAi transgenic and wild-type plants. A.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p><em>TbSRPP</em> transcription levels in the latex of three independent TbSRPP-RNAi transgenic lines (designated S3, S76 and S85) were quantified by RT-PCR. Relative transcription of <em>TbSRPPs</em> was normalized to the constitutively expressed control gene <em>TbACTIN</em>. The ratios of <em>TbSRPPs</em>/<em>TbACTIN</em> transcription levels in wild-type (wt) served as a reference and were set to the value of one. Values represent mean ± SE (<em>n</em> = 3). <strong>B, C.</strong> Fifty microgram of rubber particle proteins from wild-type (B) and TbSRPP-RNAi lines (C, exemplarily shown for line S3) was separated by isoelectric focussing (pH 3–6 IPG strips) and subsequent SDS-PAGE. Proteins were transferred to nitrocellulose membranes and TbSRPPs were detected using an anti-TbSRPP antibody and a secondary antibody conjugated with horse radish peroxidase. <strong>D, E.</strong> Ponceau-S staining of the nitrocellulose membranes confirmed equal loading of gels prepared for wild-type (D) and line S3 (E), by showing that proteins spots of the C-terminus of the Polyphenoloxidase-1 (TbPPO-1) were present in similar amounts <a href="#pone.0041874-Wahler1" class="ref-tip">[29]</a>, <a href="#pone.0041874-Wahler2" class="ref-tip">[30]</a>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0041874.g002">
              https://doi.org/10.1371/journal.pone.0041874.g002</a></p></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>Next, the effect of the <em>TbSRPP3</em>-RNAi construct was analyzed on protein level by 2-D gel electrophoresis of rubber particle proteins followed by immunodetection using the anti-TbSRPP antibody. Equal loading of the gels was verified after Ponceau-S staining of the nitrocellulose membrane by comparing the intensities of the protein spots representing the C-terminus of the Polyphenoloxidase 1 (TbPPO-1), which is expressed consistently in <em>T. brevicorniculatum</em> and thus suitable as loading control <a href="#pone.0041874-Wahler1" class="ref-tip">[29]</a>, <a href="#pone.0041874-Wahler2" class="ref-tip">[30]</a>. Compared to wild-type plants (<a href="#pone-0041874-g002">Fig. 2B</a>), the levels for TbSRPPs were strongly reduced as exemplarily shown for S3 (<a href="#pone-0041874-g002">Fig. 2C</a>). It should be noted that <em>TbSRPP</em> transcription (<a href="#pone-0041874-g002">Fig. 2A</a>) strongly correlates with the observed protein levels as only residual amounts were identified for TbSRPP5 by 2-D gel electrophoresis (<a href="#pone-0041874-g002">Fig. 2C</a>). The transgenic lines showed regular growth and development under standard greenhouse conditions.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Reduced TbSRPP protein levels had severe impacts on rubber production"></a>
<h3>Reduced TbSRPP protein levels had severe impacts on rubber production</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>To test if depleted TbSRPP levels affect rubber biosynthesis, the dry rubber content in exuding latex and freeze-dried roots was measured. Compared to wild-type, the latex of TbSRPP-RNAi transgenic plants contained significantly lower amounts of rubber (24–37% of wild-type, <a href="#pone-0041874-t001">Tab. 1</a>) and the same was evident for the rubber content of freeze-dried roots (<a href="#pone-0041874-t001">Tab. 1</a>). Additionally, a clear correlation between <em>TbSRPP</em> expression levels and rubber content was indicated (<a href="#pone-0041874-t001">Tab. 1</a>, see also <a href="#pone-0041874-g002">Fig. 2A</a>).</p>
<a class="link-target" id="pone-0041874-t001" name="pone-0041874-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0041874.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0041874.t001" data-doi="info:doi/10.1371/journal.pone.0041874" data-uri="info:doi/10.1371/journal.pone.0041874.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0041874.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Table <strong>1.</strong> Content, molecular mass and polydispersity of rubber from exuded latex and freeze-dried roots and corresponding long-chain prenyltransferase activity.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0041874.t001">
              https://doi.org/10.1371/journal.pone.0041874.t001</a></p></div><a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2" class="link-target"></a><p>Next, we analyzed the weight average molecular mass (Μw) and number average molecular mass (Μn) of rubber obtained from exuding latex. Using size exclusion chromatography (SEC) no significant differences could be detected between rubber of wild-type and TbSRPP-RNAi transgenic plants indicating that polymer length and polydispersity of the residual rubber were not influenced by altered TbSRPPs expression (<a href="#pone-0041874-t001">Tab. 1</a>).</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3" class="link-target"></a><p>To figure out whether reduced TbSRPP levels affected the activity of the long chain prenyltransferase in latex, we carried out <em>in vitro</em> [1-<sup>14</sup>C] IPP incorporation assays using whole latex from TbSRPP-RNAi transgenic and wild-type plants as described by Post <em>et</em> <em>al</em>. <a href="#pone.0041874-Post1" class="ref-tip">[10]</a>. After removal of the short- and medium-chain polyprenol diphosphates by <em>n</em>-butanol extraction, the long-chain products were obtained with a mixture of toluene and hexane and the incorporated radioactivity was measured. This experiment revealed that the activity of the long-chain prenyltransferase in latex of TbSRPP-RNAi transgenic plants was indeed significantly lower under this condition (<a href="#pone-0041874-t001">Tab. 1</a>).</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Silencing of TbSRPPs reduces the stability of rubber particles in exuding latex"></a>
<h3>Silencing of TbSRPPs reduces the stability of rubber particles in exuding latex</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>We set out to analyze if the depleted levels of TbSRPPs affected the architecture of rubber particles by ultrastructural investigations using transmission electron microscopy (TEM). TEM images of ultra-thin cross sections from roots of TbSRPP-RNAi transgenic (<a href="#pone-0041874-g003">Fig. 3A</a>) and wild-type plants (<a href="#pone-0041874-g003">Fig. 3C</a>) showed laticifers with electron dense material, organelle-like structures, and rubber particles. In the analyzed sections, rubber particles appeared spherical and did not show any obvious morphological differences in both, TbSRPP-RNAi transgenic (<a href="#pone-0041874-g003">Fig. 3B</a>) and wild-type (<a href="#pone-0041874-g003">Fig. 3D</a>) plants.</p>
<a class="link-target" id="pone-0041874-g003" name="pone-0041874-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0041874.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0041874.g003" data-doi="info:doi/10.1371/journal.pone.0041874" data-uri="info:doi/10.1371/journal.pone.0041874.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0041874.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> <em>In situ</em> analysis of the ultra-structure of rubber particles from TbSRPP-RNAi transgenic plants and wild-type plants by transmission electron microscopy.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p><strong>A.</strong> Micrograph of ultra-thin cross sections of roots from TbSRPP-RNAi transgenic plant (S3) at low and <strong>B.</strong> higher magnification. <strong>C.</strong> Micrograph of ultra-thin cross sections of roots from a wild-type plant at low and <strong>D.</strong> higher magnification. Enlarged areas are indicated with dashed boxes. Key: RP, rubber particle; V, vacuole; CW, cell wall; PC, parenchyma cell. Scale bars  = 1 µm.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0041874.g003">
              https://doi.org/10.1371/journal.pone.0041874.g003</a></p></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>We further studied the colloidal stability of purified rubber particles from exuding latex of TbSRPP-RNAi transgenic and wild-type plants by measuring their ζ-potential at different pH values. Both types of rubber particles showed a similar ζ-potential ranging from +14 mV at pH 2.2 to −18 mV at pH 7.2 (<a href="#pone-0041874-g004">Fig. 4A</a>). The pI was approximately pH 4.2. In contrast to the ζ-potential, significant differences were obtained for their size. Rubber particles from wild-type latex revealed a z-average size of 295 nm at pH 7.2 (<a href="#pone-0041874-g004">Fig. 4B</a>) and showed an increased diameter at pH-values near their pI (pH 3.2–4.8, <a href="#pone-0041874-g004">Fig. 4B</a>). Rubber particles obtained from TbSRPP-RNAi transgenic plants, however, displayed with 406 nm at pH 7.2 a significantly higher z-average diameter compared to the wild-type control. A similar increase in z-average diameter was also evident at other pH values (<a href="#pone-0041874-g004">Fig. 4B</a>).</p>
<a class="link-target" id="pone-0041874-g004" name="pone-0041874-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0041874.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0041874.g004" data-doi="info:doi/10.1371/journal.pone.0041874" data-uri="info:doi/10.1371/journal.pone.0041874.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0041874.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0041874.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0041874.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0041874.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> <em>In vitro</em> analysis of the colloidal stability of rubber particles from TbSRPP-RNAi transgenic plants and wild-type plants.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig2.caption1.p1" name="article1.body1.sec2.sec4.fig2.caption1.p1" class="link-target"></a><p><strong>A.</strong> Measurement of the zeta (ζ)-potential of rubber particles from wild-type plants (filled circles) and TbSRPP-RNAi transgenic plants (open circles) suspended in modified rubber extraction buffer (REB) at given pH values. Values represent mean ± SD of three independent plant lines (S3, S76 and S85) respectively of six wild-type plants (wt). No significant differences were detected among the plants using Students t-test (P, 0.05). <strong>B.</strong> Measurement of the z-average diameter of rubber particles from wild-type plants (filled circles) and TbSRPP-RNAi transgenic plants (open circles) suspended in modified REB with the indicated pH values. Values represent mean ± SD of three independent plant lines (S3, S76 and S85) respectively of six wild-type plants (wt). Significant differences have been detected among TbSRPP-RNAi lines and wild-type plants using Students t-test (P, 0.01). <strong>C, F.</strong> Confocal laser scanning microscopic analysis of isolated rubber particles from TbSRPP-RNAi transgenic plants (C) and wild-type plants (F) that were stained with nile red. <strong>D, E</strong> <strong>and</strong> <strong>G, H.</strong> Transmission electron microscopic analysis of isolated rubber particles from TbSRPP-RNAi transgenic and wild-type plants. <strong>D.</strong> Micrograph of isolated rubber particles from TbSRPP-RNAi transgenic plant line S3 suspended in modified rubber extraction buffer (REB) pH 7.2 at low and <strong>E.</strong> higher magnification. <strong>G.</strong> Micrograph of rubber particles from wild-type suspended in modified REB pH 7.2 at low and <strong>H.</strong> higher magnification. Enlarged areas are indicated with dashed boxes. Scale bars = 1 µm.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0041874.g004">
              https://doi.org/10.1371/journal.pone.0041874.g004</a></p></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3" class="link-target"></a><p>The increase of the z-average diameter could indicate an aggregation of rubber particles, although such structures have not been observed <em>in situ</em> (<a href="#pone-0041874-g003">Fig. 3</a>). To proof if the differences in z-average diameter point toward the formation of rubber particle aggregates, we stained the rubber particles with the lipophilic fluorescent dye nile red <a href="#pone.0041874-Sando1" class="ref-tip">[31]</a> and analyzed the samples by confocal laser scanning microscopy (CLSM, <a href="#pone-0041874-g004">Fig. 4C, F</a>). While the rubber particles of TbSRPP-RNAi transgenic plants formed large aggregates at pH 7.2, their wild-type counterparts appeared to be evenly distributed in the suspending buffer. For more detailed observations we again carried out TEM-analysis and confirmed that rubber particles fromTbSRPP-RNAi transgenic plants indeed tend to aggregate at pH 7.2 and in some cases are also partially fused (<a href="#pone-0041874-g004">Fig. 4D, E</a>). No similar observations were made for rubber particles of wild-type latex (<a href="#pone-0041874-g004">Fig. 4G, H</a>).</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>The biosynthesis of rubber at the surface of rubber particles is only little understood. Recently, CPTs were identified as one important component of the polymer elongation process in <em>T. brevicorniculatum</em> <a href="#pone.0041874-Post1" class="ref-tip">[10]</a>, however other proteinaceous and non-proteinaceous factors appear to be necessary for efficient rubber production.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>In this study we showed that the three SRPP homologs TbSRPP3, 4 and 5 are the major rubber particle proteins in <em>T. brevicorniculatum</em> and confirmed their association with rubber particles by immunogold labeling and SEM (<a href="#pone-0041874-g001">Fig. 1</a>, Tab. S1). While we were also able to detect small amounts of TbSRPP2 at the rubber particle surface, TbSRPP1 seems to be not or only in non-detectable amounts (by 2D-gel electrophoresis) associated with the rubber particle surface, since no peptide sequences for TbSRPP1 could be detected. This could result from the washing steps during the purification procedure applied to the rubber particles by which more loosely associated proteins might detach from the rubber particle surface (<a href="#pone-0041874-g001">Fig. 1A</a>, Tab. S1). In <em>H. brasiliensis,</em> the closely related proteins HbREF and HbSRPP are also associated with rubber particles, whereby HbREF is the most abundant rubber particle protein followed by HbSRPP <a href="#pone.0041874-Singh1" class="ref-tip">[13]</a>, <a href="#pone.0041874-Dennis2" class="ref-tip">[24]</a>, <a href="#pone.0041874-Oh1" class="ref-tip">[25]</a>. HbREF and HbSRPP as well as their homolog GHS from <em>P. argentatum</em> have been supposed to positively influence rubber biosynthesis: It has been shown that heterologously expressed GST-HbSRPP as well as His-GHS fusion proteins enhance IPP incorporation activity of rubber particles <em>in vitro</em>, while immunoinhibition by appropriate HbREF and HbSRPP antibodies reduces the IPP incorporation <a href="#pone.0041874-Dennis2" class="ref-tip">[24]</a>–<a href="#pone.0041874-Kim1" class="ref-tip">[26]</a>. Therefore, it has been suggested that HbSRPP is part of the rubber transferase complex or that it is involved in the synthesis or recruitment of substrates for rubber biosynthesis initiation and/or rubber chain elongation <a href="#pone.0041874-Oh1" class="ref-tip">[25]</a>. However, Singh and co-workers speculated that SRPP is not crucial for rubber biosynthesis since its detection on <em>F. carica</em> and <em>F. benghalensis</em> rubber particles by immunogold labeling with an HbSRPP antibody failed <a href="#pone.0041874-Singh1" class="ref-tip">[13]</a>. As SRPP homologs have been detected in species that produce rubber of high molecular mass (<em>H. brasiliensis</em> and <em>P. argentatum</em>) but not in species possessing only low molecular mass rubber (<em>F. carica</em> and <em>F. benghalensis</em>), it has been supposed that HbSRPP and GHS might be important factors for determination of the rubber molecular mass by assisting CPT function <a href="#pone.0041874-Kim1" class="ref-tip">[26]</a>. Due to the lack of <em>in vivo</em> evidence, the roles of SRPP, REF, and GHS in rubber biosynthesis have yet not been sufficiently proven.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>To elucidate the role of TbSRPPs in rubber biosynthesis <em>in vivo</em>, we generated transgenic plants expressing an RNAi construct directed against <em>TbSRPPs</em>. Quantitative RT-PCR and 2-D gel electrophoresis with subsequent immunodetection using the anti-TbSRPP antibody confirmed that expression of all TbSRPPs was severely silenced in the transgenic lines (<a href="#pone-0041874-g002">Fig. 2</a>). By determination of the dry rubber content from exuded latex and freeze-dried roots we were able to show that the TbSRPP-RNAi transgenic plants contained significantly lower amounts of natural rubber compared to wild-type controls (<a href="#pone-0041874-t001">Tab. 1</a>) indicating an impaired rubber biosynthesis in the TbSRPP-RNAi transgenic lines. In contrast, SEC-analysis of the rubber samples revealed a wild-type like rubber chain elongation since neither the weight average molecular mass nor the polydispersity was affected by reduced TbSRPP protein levels (<a href="#pone-0041874-t001">Tab. 1</a>). Therefore, we propose that TbSRPPs are not mandatory for the formation of high molecular mass rubber. Instead, the rubber biosynthesis seems to be less effective in TbSRPP-RNAi transgenic plants, probably resulting from an inefficient enzymatic activity of the long-chain prenyltransferase (TbCPT1–3) <a href="#pone.0041874-Post1" class="ref-tip">[10]</a>, as also suggested by the data obtained from the <em>in vitro</em> prenyltransferase assay (<a href="#pone-0041874-t001">Tab. 1</a>).</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>Rubber particles are not only required for the storage of rubber molecules, but also to provide a suitable and stable platform for rubber biosynthesis. Although one could expect that the depletion of major integral rubber particle proteins might have a dramatic effect on the morphology of rubber particles, TEM-analysis of ultra-thin cross sections of laticifers revealed no obvious differences in the architecture of rubber particles from TbSRPP-RNAi transgenic and wild-type plants (<a href="#pone-0041874-g003">Fig. 3</a>). In addition, TbSRPPs appear not to contribute to the surface charge and subsequently to the electrostatic repulsion of rubber particles since both, rubber particles from TbSRPP-RNAi transgenic and wild-type plants showed a nearly identical ζ-potential (<a href="#pone-0041874-g004">Fig. 4A</a>). This is in consistence with results obtained for <em>H. brasiliensis</em> rubber particles, whose negative surface charge is mainly derived from adsorbed long chain fatty acids and only little from proteins <a href="#pone.0041874-Ho1" class="ref-tip">[32]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>Depletion of the rubber particle surface charge by suspending rubber particles in a buffer with a pH-value close to their isoelectric point led to an increase of their z-average diameter, indicating the formation of aggregates (<a href="#pone-0041874-g004">Fig. 4B</a>). This suggests that electrostatic repulsion contributes to a great extent to the colloidal stability of <em>T. brevicorniculatum</em> rubber particles. Nevertheless, even at pH 7.2 the rubber particle ζ-potential of approximately −18 mV remains insufficient for electrostatic force to solely provide particle stabilization, which requires a ζ-potential of at least ±30 mV <a href="#pone.0041874-Riddick1" class="ref-tip">[18]</a>, <a href="#pone.0041874-Genovese1" class="ref-tip">[33]</a>. Therefore, the colloidal stability of <em>T. brevicorniculatum</em> rubber particles is not exclusively mediated by electrostatic repulsion. This is strongly supported by the fact that the depletion of TbSRPPs, which have no impact on the rubber particle surface charge, led to a significant increase of the z-average diameter of rubber particles from TbSRPP-RNAi transgenic plants compared to wild-type (<a href="#pone-0041874-g004">Fig. 4A, B</a>). Microscopic analysis of purified rubber particles using CLSM and TEM confirmed that even at pH 7.2, where the surface charge of the rubber particles is considerably high, rubber particles from TbSRPP-RNAi transgenic plants aggregated intensely, while wild-type particles remained solitary (<a href="#pone-0041874-g004">Fig. 4A, C–H</a>). Therefore, we suppose that in addition to the electrostatic repulsion of rubber particles (most probably provided by their membrane lipids) TbSRPPs contribute to their stabilization by steric hindrance. This is supported by the fact that at low surface charges rubber particles from TbSRPP<em>-</em>RNAi transgenic plants showed an even more intensive aggregation compared to wild-type particles, suggesting an additive effect of electrostatic repulsion and steric repulsion mediated by TbSRPPs as also considered for other colloidal systems (<a href="#pone-0041874-g004">Fig. 4A, B</a>) <a href="#pone.0041874-Jucker1" class="ref-tip">[19]</a>–<a href="#pone.0041874-Grasso1" class="ref-tip">[22]</a>. Accordingly, the monolayer-membrane surrounded seed oil bodies are sterically stabilized by oleosin, their major surface protein <a href="#pone.0041874-Tzen1" class="ref-tip">[34]</a>, <a href="#pone.0041874-Peng1" class="ref-tip">[35]</a>. The assumption that proteins on the surface of rubber particles contribute to their stability is also supported by the fact that <em>H. brasiliensis</em> rubber particles become unstable when treated with proteolytic enzymes <a href="#pone.0041874-Sansatsadeekul1" class="ref-tip">[36]</a>.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6" class="link-target"></a><p>In summary, our data obtained by photon correlation spectroscopy of rubber particles suggest that TbSRPPs are involved in the stabilization of the rubber particle membrane. As a consequence, TbSRPPs-depleted rubber particles may not provide a suitable environment for the CPT and subsequently efficient IPP-polymerization. To gain deeper insight into the functional role of individual SRPPs in rubber biosynthesis, it will be necessary to generate transcript specific knock-down plants and to analyze their respective effect on rubber biosynthesis and rubber particle biogenesis as well as rubber particle architecture and stability.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="Plant material and cultivation conditions"></a>
<h3>Plant material and cultivation conditions</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p><em>T. brevicorniculatum</em> plants and seeds were obtained from the Botanical Gardens Karlsruhe (Germany) and cultivated in a greenhouse at 18°C and a 16 h photoperiod with 20 kilolux light intensity.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="Preparation of T. brevicorniculatum rubber particles"></a>
<h3>Preparation of <em>T. brevicorniculatum</em> rubber particles</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>Latex from <em>T. brevicorniculatum</em> was harvested as described previously <a href="#pone.0041874-Schmidt1" class="ref-tip">[8]</a>. The latex suspension was separated into three phases by centrifugation at 4°C for 5 min and 8,000×g. The rubber phase and the soluble phase were transferred to a new tube and centrifuged at 4°C for 15 min at 10,000xg. The soluble phase was isolated using a syringe and afterwards the remaining rubber particle suspension, if not stated otherwise, was diluted with an appropriate volume rubber extraction buffer (REB) <a href="#pone.0041874-Schmidt1" class="ref-tip">[8]</a>. All experiments described were carried out with freshly prepared rubber particles.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="Two-dimensional gel electrophoresis of rubber particle proteins"></a>
<h3>Two-dimensional gel electrophoresis of rubber particle proteins</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Fifty microliter of rubber particle suspension was used for protein extraction using the ZOOM® 2D Protein Solubilizer 1 according to the manufacturer's recommendations (Invitrogen, Karlsruhe, Germany) with the following modification: Before rehydration, acetone precipitation was carried out to concentrate proteins. After solubilising proteins in rehydration buffer for at least 4 h, 50 µg protein (determined according to Bradford) <a href="#pone.0041874-Bradford1" class="ref-tip">[37]</a> was loaded on pH 3–10 7 cm or pH 3–6 7 cm ReadyStrip<sup>TM</sup> IPG strips (Biorad, Munich, Germany). The strips were rehydrated overnight and isoelectric focussing was carried out at 20°C and a maximum current of 50 µA using the following program: 300 V for 30 min, 1000 V for 30 min, 5000 V until 12 kVh were reached. Proteins were either stained with colloidal Coomassie Brilliant Blue (Fermentas, St. Leon-Rot, Germany) or transferred to nitrocellulose membranes, which were stained with 0.1% Ponceau-S in 5% acetic acid to confirm equal loading of proteins. Membranes were incubated with the anti-TbSRPP antibody (1∶250) for 1 h and a secondary mouse anti-rabbit IgG conjugated to horse radish peroxidase according to manufacturer's recommendation (Sigma, Schnelldorf, Germany). TbSRPPs were visualized on X-ray films by chemiluminescence detection.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="Protein identification by mass spectrometrical analysis"></a>
<h3>Protein identification by mass spectrometrical analysis</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>Protein bands were excised from gels and subjected to in-gel trypsin digestion following the descriptions in Shevchenko <em>et</em> <em>al</em>. <a href="#pone.0041874-Shevchenko1" class="ref-tip">[38]</a>. Subsequent LC-MS/MS was accomplished using an Ultimate 3000 system (Dionex, Sunnyvale, CA, USA) for Nano-LC and a LTQ Orbitrap XL mass spectrometer (Thermo, Bremen, Germany) <a href="#pone.0041874-Naumann1" class="ref-tip">[39]</a>. The five most abundant precursor ions in the first MS were used for MS/MS fragmentations. MS/MS data was searched against a database created from all protein data available at the public NCBI non-redundant (nr) database (<a href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>, October 2009 status) using the OMSSA software (version 2.1.4) <a href="#pone.0041874-Geer1" class="ref-tip">[40]</a>. Mass tolerances were set to 0.02 Da for the precursor ions and to 0.5 Da for the product ions. An estimation of true protein identification hits was achieved by searching a concatenated database of target and decoy peptide mass information and defining an adaptive E-value threshold for all hits originating from one sample in such a way that the estimated false positive rate was ≤1% and by a mass accuracy requirement of 5 ppm.</p>
</div>

<div id="section5" class="section toc-section"><a id="s4e" name="s4e" class="link-target" title="Construction of expression vector for heterologous expression of TbSRPP3 and generation of antibodies against TbSRPPs"></a>
<h3>Construction of expression vector for heterologous expression of TbSRPP3 and generation of antibodies against TbSRPPs</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>The full length cDNA encoding <em>TbSRPP3</em> was amplified using the primer combination srpp3-EcoRI and srpp3-rev_NotI (Tab. S1). The cDNA was subsequently inserted into the pCRII-TOPO® vector (Invitrogen, Karlsruhe, Germany), sequenced and then transferred into the expression vector pGEX4-T1, using the EcoRI and NotI restriction sites (GE Healthcare Europe GmbH, Freiburg, Germany). This allowed the expression of TbSRPP3 in C-terminal fusion to glutathione-S-transferase (GST) for heterologous expression in <em>Escherichia coli</em> BL21-cells and subsequent antibody generation. The <em>TbSRPP3</em> clone in pGEX4-T1 was expressed in 300-ml cultures of <em>E. coli</em> strain BL21, induced with 1 mM isopropyl-beta-D-thiogalactopyranoside (IPTG). The protein band containing heterologously expressed TbSRPP3 was excised from the gel, the eluted protein was sequenced by MALDI-MS and send to EUROGENTEC (Seraing, Belgium) for antibody production. The pre-immune and antibody sera were tested by western blot analysis against recombinant TbSRPP1-5 proteins.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4f" name="s4f" class="link-target" title="Total RNA extraction from T. brevicorniculatum latex and cDNA synthesis"></a>
<h3>Total RNA extraction from <em>T. brevicorniculatum</em> latex and cDNA synthesis</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>Extraction of total RNA from <em>T. brevicorniculatum</em> latex and cDNA synthesis was carried out as previously described <a href="#pone.0041874-Schmidt2" class="ref-tip">[27]</a>, whereas random hexamer primer were used to initiate the reverse transcription reaction (Fermentas, St. Leon-Rot, Germany).</p>
</div>

<div id="section7" class="section toc-section"><a id="s4g" name="s4g" class="link-target" title="Quantitative real time PCR experiments"></a>
<h3>Quantitative real time PCR experiments</h3>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1" class="link-target"></a><p>Quantitative real time PCR was carried out using an iCycler realtime PCR system (Biorad, Munich, Germany). 7.5 µl of serial cDNA dilutions was added to 12.5 µl IQ<sup>TM</sup>SYBRGreen Supermix and 5 µl Primermix (50 pmol each). The primers used for the amplification of the <em>TbSRPP</em> transcripts are listed in table S2. The transcript of the housekeeping gene <em>TbACTIN</em> was used as a reference (Tab. S2). Melting curves were analyzed to exclude false amplifications. After determination of the C<sub>t</sub>-values, the relative expression of the genes was calculated using the REST-MCS software <a href="#pone.0041874-Pfaffl1" class="ref-tip">[41]</a>.</p>
</div>

<div id="section8" class="section toc-section"><a id="s4h" name="s4h" class="link-target" title="Cloning of TbSRPP3-RNAi construct"></a>
<h3>Cloning of <em>TbSRPP3</em>-RNAi construct</h3>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1" class="link-target"></a><p>For cloning of the <em>TbSRPP3</em>-RNAi construct, a 363 bp-fragment specific for all <em>TbSRPP</em> genes was obtained from <em>TbSRPP3</em> cDNA (coordinates 270–632 of cDNA) by PCR using the primers TbSRPP3 RNAi KpnI fw and TbSRPP3 RNAi XhoI rev (Tab. S2). To avoid off-target suppression of genes, we validated that the chosen cDNA fragment gave no hits for other than TbSRPP1–5 homologs in the public available sequences and especially in EST datasets for the closely related plant species <em>T. koksaghyz</em> and <em>T. officinale</em> using the Basic Local Alignment Search Tool (BLAST; <a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>). The cDNA fragment was inserted into the KpnI and XhoI sites of the Gateway® pENTR<sup>TM</sup>4 and transferred into the pFGC5941 (<a href="http://www.chromDB.org">www.chromDB.org</a>) using the Gateway®LR Clonase enzyme mix (Invitrogen, Karlsruhe, Germany). The integrity of the construct was verified by sequencing.</p>
</div>

<div id="section9" class="section toc-section"><a id="s4i" name="s4i" class="link-target" title="Agrobacterium-mediated transformation of T. brevicorniculatum"></a>
<h3><em>Agrobacterium</em>-mediated transformation of <em>T. brevicorniculatum</em></h3>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1" class="link-target"></a><p><em>Agrobacterium</em>-mediated transformation was done as previously described <a href="#pone.0041874-Wahler1" class="ref-tip">[29]</a> using the <em>Agrobacterium tumefaciens</em> strain EHA105 that has been transformed with the binary plasmid.</p>
</div>

<div id="section10" class="section toc-section"><a id="s4j" name="s4j" class="link-target" title="Determination of dry rubber content"></a>
<h3>Determination of dry rubber content</h3>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1" class="link-target"></a><p>To determine the dry rubber content in exuding latex, roots of <em>T. brevicorniculatum</em> plants were dissected with a razor blade and 20 µl of the expelling latex was transferred to 100 µl REB supplemented with 5 mM DTT. The samples were centrifuged at 4°C and 12000×g and overlaid with 100 µl methanol to coagulate rubber particles. The rubber coagulum was air-dried for 24 h and weighted. To determine the dry rubber content of whole roots, the freeze-dried entire roots were ground into a fine powder using a coffee grinder. For extraction, 200 mg root powder and 4 ml toluene were mixed thoroughly and incubated at 70°C for 18 h. The organic phase was transferred to a new glass vial and toluene was evaporated. The remaining material was dissolved in 300 µl toluene for 30 min and then 600 µl methanol was added to precipitate the rubber. After 30 min the samples were centrifuged at RT and 16,000×g for 2 min. The precipitated rubber was washed with water and acetone by vigorous shaking for 30 min at RT to remove proteins and resins. Subsequently the rubber samples were dried and weighted.</p>
</div>

<div id="section11" class="section toc-section"><a id="s4k" name="s4k" class="link-target" title="Size exclusion chromatography of T. brevicorniculatum rubber"></a>
<h3>Size exclusion chromatography of <em>T. brevicorniculatum</em> rubber</h3>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1" class="link-target"></a><p>Toluene was added to air-dried rubber samples to a final concentration of 0.75 mg ml<sup>−1</sup> and dissolved at 60°C for 16 h. The samples were filtered with Cameo 3 N syringe (Nylon, 0.22 micron) filters (Roth, Karlsruhe, Germany) and transferred to new glass vials. Size exclusion chromatography (SEC)-analysis was carried out using a SECcurity GPC system with a column combination including a PSS SDV 20 µ 8.0×50 mm precolumn, a PSS SDV Linear XL 20 µ 8×300 nm analytical column and a PSS SDV 100 Å 20 µ 8×300 mm analytical column (Polymer Standards Service, Mainz, Germany).</p>
</div>

<div id="section12" class="section toc-section"><a id="s4l" name="s4l" class="link-target" title="[1-14C] IPP incorporation assay"></a>
<h3>[1-<sup>14</sup>C] IPP incorporation assay</h3>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1" class="link-target"></a><p>The [1-<sup>14</sup>C] IPP incorporation rate of crude latex was determined as described previously <a href="#pone.0041874-Post1" class="ref-tip">[10]</a>.</p>
</div>

<div id="section13" class="section toc-section"><a id="s4m" name="s4m" class="link-target" title="Determination of rubber particle ζ-potential and z-average diameter"></a>
<h3>Determination of rubber particle ζ-potential and z-average diameter</h3>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1" class="link-target"></a><p>20 µl of rubber particle suspension was diluted in 1180 µl modified REB (350 mM Sorbitol, 5 mM MgCl, 50 mM NaCl, 40 mM NaH<sub>2</sub>PO<sub>4</sub> titrated with 20 mM Citrate to the desired pH) to reach a minimal derived count rate of 250 kcps. One milliliter of the suspension was injected into a Malvern Clear Zeta Potential Cell. The Derjaguin-Landau-Verwey-Overbeek theory allows the quantification of the electrostatic stability of lyophobic colloids by measuring their ζ-potential (Verwey, 1947). The electrophoretic mobility of rubber particles was measured by Laser Doppler Velocimetry and was converted to ζ-potential by the Dispersion Technology Software (DTS) using Henrýs equation and the Smoluchowski approximation. The intensity weighted mean hydrodynamic diameter (z-average diameter) was measured by dynamic light scattering. The ζ-potential and z-average diameter of rubber particles at different pH-values were determined using a Zetasizer Nano ZS and DTS (Malvern Instruments Ltd., Worcestershire, United Kingdom).</p>
</div>

<div id="section14" class="section toc-section"><a id="s4n" name="s4n" class="link-target" title="Scanning electron microscopy"></a>
<h3>Scanning electron microscopy</h3>
<a id="article1.body1.sec4.sec14.p1" name="article1.body1.sec4.sec14.p1" class="link-target"></a><p>Rubber particles were fixed with 2.5% glutaraldehyde in 0.05 M sodium cacodylate buffer for 1 hour at room temperature. After 3 times of washing with 0.05 M sodium cacodylate buffer pH 7.3 (at 5,000×g), they were postfixed with 1% osmium tetroxide for 1 hour. Finally the particles were washed 3 times with 0.05 M sodium cacodylate buffer and subsequently 3 times with PBS (pH 7.4), each for 5 minutes.</p>
<a id="article1.body1.sec4.sec14.p2" name="article1.body1.sec4.sec14.p2" class="link-target"></a><p>For immunogold labeling, 10 µl aliquots of the suspensions were placed on a freshly glow-discharged carbon-layer (thickness ∼10 nm) onto a Pioloform film supported by a Cu-mesh grid, let sediment for 5 minutes, and washed with PBS (pH 7.4) for 10 minutes. The grids were then incubated in PBS buffer containing 5% bovine serum albumin (BSA) (pH 7.4) for 15 minutes, followed by 3 times washing with PBS (pH 7.4) each for 5 minutes. After this, the grids were incubated for 3 hours either with the primary anti-TbSRPP antibody (1∶5 in PBS, pH 7.4) or with the pre-immunserum (1∶5 in PBS, pH 7.4) as a negative control. Subsequently, grids were washed 5 times for 5 minutes with PBS (pH 7.7) and then 5 minutes with Tris-BSA-buffer (pH 8.2). After this the samples were incubated with a secondary goat-anti rabbit IgG antibody coupled to 10 nm colloidal gold particles (Sigma-Aldrich, Schnelldorf, Germany) diluted 1∶20 in Tris-BSA-buffer (pH 8.2) for 2 hours. Finally, samples were washed with Tris-BSA-buffer (pH 8.2) 6 times for 5 minutes and once with distilled water for 5 minutes. After air drying, the sample was covered by 2 nm platinum/carbon. Images were recorded by a field emission SEM S-5000 (Hitachi, Japan) at electron energy of 20 kV; both, secondary and backscattered electron signals were recorded.</p>
</div>

<div id="section15" class="section toc-section"><a id="s4o" name="s4o" class="link-target" title="Transmission electron microscopy"></a>
<h3>Transmission electron microscopy</h3>
<a id="article1.body1.sec4.sec15.p1" name="article1.body1.sec4.sec15.p1" class="link-target"></a><p>TEM of ultra-thin root cross sections was carried out as described previously <a href="#pone.0041874-Post1" class="ref-tip">[10]</a>.</p>
</div>

<div id="section16" class="section toc-section"><a id="s4p" name="s4p" class="link-target" title="Confocal laser scanning microscopy"></a>
<h3>Confocal laser scanning microscopy</h3>
<a id="article1.body1.sec4.sec16.p1" name="article1.body1.sec4.sec16.p1" class="link-target"></a><p>Fifty microliters of freshly prepared rubber particles were supplemented with 1 µl nile red (15 µM solubilised in DMSO) and fluorescence was visualized by CLSM (excitation 514 nm, emission 509–620 nm, Leica DMIRE2, Wetzlar, Germany).</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0041874"></div><div class="supplementary-material"><a name="pone.0041874.s001" id="pone.0041874.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0041874.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Anti-TbSRPP antibody detects heterologously expressed TbSRPP1–5. A.</strong> Protein extracts from pellet and supernatant from <em>E. coli</em> cultures heterologously expressing TbSRPP1–5 proteins were separated by SDS-PAGE and subsequently stained with Coomassie Brilliant Blue. <strong>B.</strong> Protein extracts from pellet and supernatant from <em>E. coli</em> cultures heterologously expressing TbSRPP1–5 proteins were separated by SDS-PAGE and subsequently transferred to a nitrocellulose membrane. TbSRPP1–5 were detected using the anti-TbSRPP antibody and a secondary antibody coupled with alkaline phosphatase (1, TbSRPP1; 2, TbSRPP2; 3, TbSRPP3; 4, TbSRPP4; 5, TbSRPP5; P, pellet; S, supernatant).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0041874.s001">https://doi.org/10.1371/journal.pone.0041874.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0041874.s002" id="pone.0041874.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0041874.s002">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Identification of TbSRPP2, TbSRPP3, TbSRPP4, and TbSRPP5 on </strong>
            <strong>
              <em>T. brevicorniculatum</em>
            </strong>
            <strong> rubber particles by mass spectrometric analysis.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0041874.s002">https://doi.org/10.1371/journal.pone.0041874.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0041874.s003" id="pone.0041874.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0041874.s003">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Oligonucleotide sequences and respective assignments.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0041874.s003">https://doi.org/10.1371/journal.pone.0041874.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0041874.s004" id="pone.0041874.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0041874.s004">Text S1. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Materials and Methods section regarding experiments performed for Figure</strong> <strong>S1.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0041874.s004">https://doi.org/10.1371/journal.pone.0041874.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The technical assistance of Sandra Ponanta and Daniela Ahlert (Institute of Plant Biology and Biotechnology, Münster), Ulrike Keller and Ursula Malkus (Institute for Medical Physics and Biophysics, Münster) as well as Raphael Soeur (Fraunhofer Institute for Molecular Biology and Applied Ecology, Aachen) is kindly acknowledged. We thank Fabrice Brunel and Francisco M. Goycoolea (Institute of Plant Biology and Biotechnology, Münster) for their support concerning dynamic light scattering and size exclusion chromatography experiments.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: CSG DP AH. Performed the experiments: AH JP D. Wurbs D. Wahler ML VK CSG. Analyzed the data: AH CSG D. Wahler. Contributed reagents/materials/analysis tools: VK. Wrote the paper: AH DP CSG.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0041874-Cornish1" id="pone.0041874-Cornish1" class="link-target"></a>Cornish K (2001) Similarities and differences in rubber biochemistry among plant species. Phytochemistry  57: 1123–1134.  <ul class="reflinks"><li><a href="#" data-author="Cornish" data-cit="CornishK2001Similarities%20and%20differences%20in%20rubber%20biochemistry%20among%20plant%20species.Phytochemistry5711231134" data-title="Similarities%20and%20differences%20in%20rubber%20biochemistry%20among%20plant%20species." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Similarities+and+differences+in+rubber+biochemistry+among+plant+species.+Cornish+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0041874-Poulter1" id="pone.0041874-Poulter1" class="link-target"></a>Poulter CD, Rilling HC (1976) Prenyltransferase – mechanism of reaction. Biochemistry  15: 1079–1083.  <ul class="reflinks"><li><a href="#" data-author="Poulter" data-cit="PoulterCDRillingHC1976Prenyltransferase%20%E2%80%93%20mechanism%20of%20reaction.Biochemistry1510791083" data-title="Prenyltransferase%20%E2%80%93%20mechanism%20of%20reaction." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prenyltransferase+%E2%80%93+mechanism+of+reaction.+Poulter+1976" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0041874-Poulter2" id="pone.0041874-Poulter2" class="link-target"></a>Poulter CD, Rilling HC (1978) The prenyl transfer-reaction. Enzymatic and mechanistic studies of 1′-4 coupling reaction in the terpene biosynthetic-pathway. Accounts Chem Res  11: 307–313.  <ul class="reflinks"><li><a href="#" data-author="Poulter" data-cit="PoulterCDRillingHC1978The%20prenyl%20transfer-reaction.%20Enzymatic%20and%20mechanistic%20studies%20of%201%E2%80%B2-4%20coupling%20reaction%20in%20the%20terpene%20biosynthetic-pathway.Accounts%20Chem%20Res11307313" data-title="The%20prenyl%20transfer-reaction.%20Enzymatic%20and%20mechanistic%20studies%20of%201%E2%80%B2-4%20coupling%20reaction%20in%20the%20terpene%20biosynthetic-pathway." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+prenyl+transfer-reaction.+Enzymatic+and+mechanistic+studies+of+1%E2%80%B2-4+coupling+reaction+in+the+terpene+biosynthetic-pathway.+Poulter+1978" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0041874-Cornish2" id="pone.0041874-Cornish2" class="link-target"></a>Cornish K, Backhaus RA (1990) Rubber transferase activity in rubber particles of guayule. Phytochemistry  29: 3809–3813.  <ul class="reflinks"><li><a href="#" data-author="Cornish" data-cit="CornishKBackhausRA1990Rubber%20transferase%20activity%20in%20rubber%20particles%20of%20guayule.Phytochemistry2938093813" data-title="Rubber%20transferase%20activity%20in%20rubber%20particles%20of%20guayule." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Rubber+transferase+activity+in+rubber+particles+of+guayule.+Cornish+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0041874-Kharel1" id="pone.0041874-Kharel1" class="link-target"></a>Kharel Y, Koyama T (2003) Molecular analysis of cis-prenyl chain elongating enzymes. Nat Prod Rep  20: 111–118.  <ul class="reflinks"><li><a href="#" data-author="Kharel" data-cit="KharelYKoyamaT2003Molecular%20analysis%20of%20cis-prenyl%20chain%20elongating%20enzymes.Nat%20Prod%20Rep20111118" data-title="Molecular%20analysis%20of%20cis-prenyl%20chain%20elongating%20enzymes." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+analysis+of+cis-prenyl+chain+elongating+enzymes.+Kharel+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0041874-Archer1" id="pone.0041874-Archer1" class="link-target"></a>Archer BL, Audley BG, Cockbain EG, McSweeny GP (1963) The biosynthesis of rubber – incorporation of mevalonate and isopentenyl pyrophosphate into rubber by <em>Hevea brasiliensis</em> -latex fractions. Biochem J  89: 565–574.  <ul class="reflinks"><li><a href="#" data-author="Archer" data-cit="ArcherBLAudleyBGCockbainEGMcSweenyGP1963The%20biosynthesis%20of%20rubber%20%E2%80%93%20incorporation%20of%20mevalonate%20and%20isopentenyl%20pyrophosphate%20into%20rubber%20by%20Hevea%20brasiliensis%20-latex%20fractions.Biochem%20J89565574" data-title="The%20biosynthesis%20of%20rubber%20%E2%80%93%20incorporation%20of%20mevalonate%20and%20isopentenyl%20pyrophosphate%20into%20rubber%20by%20Hevea%20brasiliensis%20-latex%20fractions." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+biosynthesis+of+rubber+%E2%80%93+incorporation+of+mevalonate+and+isopentenyl+pyrophosphate+into+rubber+by+Hevea+brasiliensis+-latex+fractions.+Archer+1963" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0041874-Siler1" id="pone.0041874-Siler1" class="link-target"></a>Siler DJ, Cornish K (1993) A protein from <em>Ficus elastica</em> rubber particles is related to proteins from <em>Hevea brasiliensis</em> and <em>Parthenium argentatum</em>. Phytochemistry  32: 1097–1102.  <ul class="reflinks"><li><a href="#" data-author="Siler" data-cit="SilerDJCornishK1993A%20protein%20from%20Ficus%20elastica%20rubber%20particles%20is%20related%20to%20proteins%20from%20Hevea%20brasiliensis%20and%20Parthenium%20argentatum.Phytochemistry3210971102" data-title="A%20protein%20from%20Ficus%20elastica%20rubber%20particles%20is%20related%20to%20proteins%20from%20Hevea%20brasiliensis%20and%20Parthenium%20argentatum." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+protein+from+Ficus+elastica+rubber+particles+is+related+to+proteins+from+Hevea+brasiliensis+and+Parthenium+argentatum.+Siler+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0041874-Schmidt1" id="pone.0041874-Schmidt1" class="link-target"></a>Schmidt T, Lenders M, Hillebrand A, van Deenen N, Munt O, et al.  (2010) Characterization of rubber particles and rubber chain elongation in <em>Taraxacum koksaghyz</em>. BMC Biochem  11: 11.  <ul class="reflinks"><li><a href="#" data-author="Schmidt" data-cit="SchmidtTLendersMHillebrandAvan%20DeenenNMuntO2010Characterization%20of%20rubber%20particles%20and%20rubber%20chain%20elongation%20in%20Taraxacum%20koksaghyz.BMC%20Biochem1111" data-title="Characterization%20of%20rubber%20particles%20and%20rubber%20chain%20elongation%20in%20Taraxacum%20koksaghyz." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+rubber+particles+and+rubber+chain+elongation+in+Taraxacum+koksaghyz.+Schmidt+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0041874-Kirschner1" id="pone.0041874-Kirschner1" class="link-target"></a>Kirschner J, Štěpánek J, Černý T, De Heer P, van Dijk PJ (2012) Available <em>ex situ</em> germplasm of the potential rubber crop <em>Taraxacum koksaghyz</em> belongs to a poor rubber producer, <em>T. brevicorniculatum</em> (Compositae-Crepidinae). Genet Resour Crop Evol.  DOI:<a href="https://doi.org/10.1007/s10722-012-9848-0">https://doi.org/10.1007/s10722-012-9848-0</a>.  <ul class="find-nolinks"></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0041874-Post1" id="pone.0041874-Post1" class="link-target"></a>Post J, van Deenen N, Fricke J, Kowalski N, Wurbs D, et al.  (2012) Laticifer-Specific cis-Prenyltransferase Silencing Affects the Rubber, Triterpene, and Inulin Content of <em>Taraxacum brevicorniculatum</em>. Plant Physiol  158: 1406–1417.  <ul class="reflinks"><li><a href="#" data-author="Post" data-cit="PostJvan%20DeenenNFrickeJKowalskiNWurbsD2012Laticifer-Specific%20cis-Prenyltransferase%20Silencing%20Affects%20the%20Rubber%2C%20Triterpene%2C%20and%20Inulin%20Content%20of%20Taraxacum%20brevicorniculatum.Plant%20Physiol15814061417" data-title="Laticifer-Specific%20cis-Prenyltransferase%20Silencing%20Affects%20the%20Rubber%2C%20Triterpene%2C%20and%20Inulin%20Content%20of%20Taraxacum%20brevicorniculatum." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Laticifer-Specific+cis-Prenyltransferase+Silencing+Affects+the+Rubber%2C+Triterpene%2C+and+Inulin+Content+of+Taraxacum+brevicorniculatum.+Post+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0041874-Gomez1" id="pone.0041874-Gomez1" class="link-target"></a>Gomez JB, Hamzah S (1989) Particle size distribution in <em>Hevea</em> latex – some observations on the electron microscopic method. J Nat Rubber Res  4: 204–211.  <ul class="reflinks"><li><a href="#" data-author="Gomez" data-cit="GomezJBHamzahS1989Particle%20size%20distribution%20in%20Hevea%20latex%20%E2%80%93%20some%20observations%20on%20the%20electron%20microscopic%20method.J%20Nat%20Rubber%20Res4204211" data-title="Particle%20size%20distribution%20in%20Hevea%20latex%20%E2%80%93%20some%20observations%20on%20the%20electron%20microscopic%20method." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Particle+size+distribution+in+Hevea+latex+%E2%80%93+some+observations+on+the+electron+microscopic+method.+Gomez+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0041874-Cornish3" id="pone.0041874-Cornish3" class="link-target"></a>Cornish K, Wood DF, Windle JJ (1999) Rubber particles from four different species, examined by transmission electron microscopy and electronparamagnetic-resonance spin labeling, are found to consist of a homogeneous rubber core enclosed by a contiguous, monolayer biomembrane. Planta  210: 85–96.  <ul class="reflinks"><li><a href="#" data-author="Cornish" data-cit="CornishKWoodDFWindleJJ1999Rubber%20particles%20from%20four%20different%20species%2C%20examined%20by%20transmission%20electron%20microscopy%20and%20electronparamagnetic-resonance%20spin%20labeling%2C%20are%20found%20to%20consist%20of%20a%20homogeneous%20rubber%20core%20enclosed%20by%20a%20contiguous%2C%20monolayer%20biomembrane.Planta2108596" data-title="Rubber%20particles%20from%20four%20different%20species%2C%20examined%20by%20transmission%20electron%20microscopy%20and%20electronparamagnetic-resonance%20spin%20labeling%2C%20are%20found%20to%20consist%20of%20a%20homogeneous%20rubber%20core%20enclosed%20by%20a%20contiguous%2C%20monolayer%20biomembrane." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Rubber+particles+from+four+different+species%2C+examined+by+transmission+electron+microscopy+and+electronparamagnetic-resonance+spin+labeling%2C+are+found+to+consist+of+a+homogeneous+rubber+core+enclosed+by+a+contiguous%2C+monolayer+biomembrane.+Cornish+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0041874-Singh1" id="pone.0041874-Singh1" class="link-target"></a>Singh AP, Wi SG, Chung GC, Kim YS, Kang H (2003) The micromorphology and protein characterization of rubber particles in <em>Ficus carica</em>, <em>Ficus benghalensis</em> and <em>Hevea brasiliensis</em>. J Exp Bot  54: 985–992.  <ul class="reflinks"><li><a href="#" data-author="Singh" data-cit="SinghAPWiSGChungGCKimYSKangH2003The%20micromorphology%20and%20protein%20characterization%20of%20rubber%20particles%20in%20Ficus%20carica%2C%20Ficus%20benghalensis%20and%20Hevea%20brasiliensis.J%20Exp%20Bot54985992" data-title="The%20micromorphology%20and%20protein%20characterization%20of%20rubber%20particles%20in%20Ficus%20carica%2C%20Ficus%20benghalensis%20and%20Hevea%20brasiliensis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+micromorphology+and+protein+characterization+of+rubber+particles+in+Ficus+carica%2C+Ficus+benghalensis+and+Hevea+brasiliensis.+Singh+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0041874-Siler2" id="pone.0041874-Siler2" class="link-target"></a>Siler DJ, Goodrich-Tanrikulu M, Cornish K, Stafford AE, McKeon TA (1997) Composition of rubber particles of <em>Hevea brasiliensis</em>, <em>Parthenium argentatum</em>, <em>Ficus elastica</em>, and <em>Euphorbia lactiflua</em> indicates unconventional surface structure. Plant Physiol Bioch  35: 881–889.  <ul class="reflinks"><li><a href="#" data-author="Siler" data-cit="SilerDJGoodrich-TanrikuluMCornishKStaffordAEMcKeonTA1997Composition%20of%20rubber%20particles%20of%20Hevea%20brasiliensis%2C%20Parthenium%20argentatum%2C%20Ficus%20elastica%2C%20and%20Euphorbia%20lactiflua%20indicates%20unconventional%20surface%20structure.Plant%20Physiol%20Bioch35881889" data-title="Composition%20of%20rubber%20particles%20of%20Hevea%20brasiliensis%2C%20Parthenium%20argentatum%2C%20Ficus%20elastica%2C%20and%20Euphorbia%20lactiflua%20indicates%20unconventional%20surface%20structure." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Composition+of+rubber+particles+of+Hevea+brasiliensis%2C+Parthenium+argentatum%2C+Ficus+elastica%2C+and+Euphorbia+lactiflua+indicates+unconventional+surface+structure.+Siler+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0041874-Wood1" id="pone.0041874-Wood1" class="link-target"></a>Wood DF, Cornish K (2000) Microstructure of purified rubber particles. Int J Plant Sci  161: 435–445.  <ul class="reflinks"><li><a href="#" data-author="Wood" data-cit="WoodDFCornishK2000Microstructure%20of%20purified%20rubber%20particles.Int%20J%20Plant%20Sci161435445" data-title="Microstructure%20of%20purified%20rubber%20particles." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Microstructure+of+purified+rubber+particles.+Wood+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0041874-Verwey1" id="pone.0041874-Verwey1" class="link-target"></a>Verwey EJW (1947) Theory of the stability of lyophobic colloids. J Phys Chem  51: 631–636.  <ul class="reflinks"><li><a href="#" data-author="Verwey" data-cit="VerweyEJW1947Theory%20of%20the%20stability%20of%20lyophobic%20colloids.J%20Phys%20Chem51631636" data-title="Theory%20of%20the%20stability%20of%20lyophobic%20colloids." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Theory+of+the+stability+of+lyophobic+colloids.+Verwey+1947" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0041874-Heller1" id="pone.0041874-Heller1" class="link-target"></a>Heller W, Pugh TL (1960) “Steric” Stabilization of Colloidal Solutions by Adsorption of Flexible Macromolecules. J Polym Sci  47: 203–217.  <ul class="reflinks"><li><a href="#" data-author="Heller" data-cit="HellerWPughTL1960%E2%80%9CSteric%E2%80%9D%20Stabilization%20of%20Colloidal%20Solutions%20by%20Adsorption%20of%20Flexible%20Macromolecules.J%20Polym%20Sci47203217" data-title="%E2%80%9CSteric%E2%80%9D%20Stabilization%20of%20Colloidal%20Solutions%20by%20Adsorption%20of%20Flexible%20Macromolecules." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=%E2%80%9CSteric%E2%80%9D+Stabilization+of+Colloidal+Solutions+by+Adsorption+of+Flexible+Macromolecules.+Heller+1960" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0041874-Riddick1" id="pone.0041874-Riddick1" class="link-target"></a>Riddick TM (1968) Zeta-Meter Manual. Zeter Meter inc. New York.   <ul class="find-nolinks"></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0041874-Jucker1" id="pone.0041874-Jucker1" class="link-target"></a>Jucker BA, Zehnder AJB, Harms H (1998) Quantification of Polymer Interactions in Bacterial Adhesion. Environ Sci Technol  32: 2909–2915.  <ul class="reflinks"><li><a href="#" data-author="Jucker" data-cit="JuckerBAZehnderAJBHarmsH1998Quantification%20of%20Polymer%20Interactions%20in%20Bacterial%20Adhesion.Environ%20Sci%20Technol3229092915" data-title="Quantification%20of%20Polymer%20Interactions%20in%20Bacterial%20Adhesion." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Quantification+of+Polymer+Interactions+in+Bacterial+Adhesion.+Jucker+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0041874-Giasson1" id="pone.0041874-Giasson1" class="link-target"></a>Giasson S, Kuhl TL, Israelachvili JN (1998) Adsorption and Interaction Forces of Micellar and Microemulsion Solutions in Ultrathin Films. Langmuir  14: 891–898.  <ul class="reflinks"><li><a href="#" data-author="Giasson" data-cit="GiassonSKuhlTLIsraelachviliJN1998Adsorption%20and%20Interaction%20Forces%20of%20Micellar%20and%20Microemulsion%20Solutions%20in%20Ultrathin%20Films.Langmuir14891898" data-title="Adsorption%20and%20Interaction%20Forces%20of%20Micellar%20and%20Microemulsion%20Solutions%20in%20Ultrathin%20Films." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Adsorption+and+Interaction+Forces+of+Micellar+and+Microemulsion+Solutions+in+Ultrathin+Films.+Giasson+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0041874-Freitas1" id="pone.0041874-Freitas1" class="link-target"></a>Freitas AM, Sharma MM (2001) Detachment of Particles from Surfaces: An AFM Study. J Colloid Interface Sci  233: 73–82.  <ul class="reflinks"><li><a href="#" data-author="Freitas" data-cit="FreitasAMSharmaMM2001Detachment%20of%20Particles%20from%20Surfaces%3A%20An%20AFM%20Study.J%20Colloid%20Interface%20Sci2337382" data-title="Detachment%20of%20Particles%20from%20Surfaces%3A%20An%20AFM%20Study." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Detachment+of+Particles+from+Surfaces%3A+An+AFM+Study.+Freitas+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0041874-Grasso1" id="pone.0041874-Grasso1" class="link-target"></a>Grasso D, Subramaniam K, Butkus M, Strevett K, Bergendahl J (2002) A review of non-DLVO interactions in environmental colloidal systems. Rev Environ Sci Biotechnol  1: 17–38.  <ul class="reflinks"><li><a href="#" data-author="Grasso" data-cit="GrassoDSubramaniamKButkusMStrevettKBergendahlJ2002A%20review%20of%20non-DLVO%20interactions%20in%20environmental%20colloidal%20systems.Rev%20Environ%20Sci%20Biotechnol11738" data-title="A%20review%20of%20non-DLVO%20interactions%20in%20environmental%20colloidal%20systems." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+review+of+non-DLVO+interactions+in+environmental+colloidal+systems.+Grasso+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0041874-Dennis1" id="pone.0041874-Dennis1" class="link-target"></a>Dennis MS, Henzel WJ, Bell J, Kohr W, Light DR (1989) Amino acid sequence of rubber elongation factor protein associated with rubber particles in <em>Hevea</em> Latex. J Biol Chem  264: 18618–18626.  <ul class="reflinks"><li><a href="#" data-author="Dennis" data-cit="DennisMSHenzelWJBellJKohrWLightDR1989Amino%20acid%20sequence%20of%20rubber%20elongation%20factor%20protein%20associated%20with%20rubber%20particles%20in%20Hevea%20Latex.J%20Biol%20Chem2641861818626" data-title="Amino%20acid%20sequence%20of%20rubber%20elongation%20factor%20protein%20associated%20with%20rubber%20particles%20in%20Hevea%20Latex." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Amino+acid+sequence+of+rubber+elongation+factor+protein+associated+with+rubber+particles+in+Hevea+Latex.+Dennis+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0041874-Dennis2" id="pone.0041874-Dennis2" class="link-target"></a>Dennis MS, Light DR (1989) Rubber elongation factor from <em>Hevea brasiliensis</em>. J Biol Chem  264: 18608–1861.  <ul class="reflinks"><li><a href="#" data-author="Dennis" data-cit="DennisMSLightDR1989Rubber%20elongation%20factor%20from%20Hevea%20brasiliensis.J%20Biol%20Chem264186081861" data-title="Rubber%20elongation%20factor%20from%20Hevea%20brasiliensis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Rubber+elongation+factor+from+Hevea+brasiliensis.+Dennis+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0041874-Oh1" id="pone.0041874-Oh1" class="link-target"></a>Oh SK, Kang H, Shin DH, Yang J, Chow K-S, et al.  (1999) Isolation, Characterization, and Functional Analysis of a Novel cDNA Clone Encoding a Small Rubber Particle Protein from <em>Hevea brasiliensis</em>. J Biol Chem  274: 17132–17138.  <ul class="reflinks"><li><a href="#" data-author="Oh" data-cit="OhSKKangHShinDHYangJChowK-S1999Isolation%2C%20Characterization%2C%20and%20Functional%20Analysis%20of%20a%20Novel%20cDNA%20Clone%20Encoding%20a%20Small%20Rubber%20Particle%20Protein%20from%20Hevea%20brasiliensis.J%20Biol%20Chem2741713217138" data-title="Isolation%2C%20Characterization%2C%20and%20Functional%20Analysis%20of%20a%20Novel%20cDNA%20Clone%20Encoding%20a%20Small%20Rubber%20Particle%20Protein%20from%20Hevea%20brasiliensis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Isolation%2C+Characterization%2C+and+Functional+Analysis+of+a+Novel+cDNA+Clone+Encoding+a+Small+Rubber+Particle+Protein+from+Hevea+brasiliensis.+Oh+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0041874-Kim1" id="pone.0041874-Kim1" class="link-target"></a>Kim IJ, Ryu SB, Kwak YS, Kang H (2004) A novel cDNA from <em>Parthenium argentatum</em> Gray enhances the rubber biosynthetic activity <em>in vitro</em>. J Exp Bot  55: 377–385.  <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimIJRyuSBKwakYSKangH2004A%20novel%20cDNA%20from%20Parthenium%20argentatum%20Gray%20enhances%20the%20rubber%20biosynthetic%20activity%20in%20vitro.J%20Exp%20Bot55377385" data-title="A%20novel%20cDNA%20from%20Parthenium%20argentatum%20Gray%20enhances%20the%20rubber%20biosynthetic%20activity%20in%20vitro." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+novel+cDNA+from+Parthenium+argentatum+Gray+enhances+the+rubber+biosynthetic+activity+in+vitro.+Kim+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0041874-Schmidt2" id="pone.0041874-Schmidt2" class="link-target"></a>Schmidt T, Hillebrand A, Wurbs D, Wahler D, Lenders M, et al.  (2010) Molecular cloning and characterization of rubber biosynthetic genes from <em>Taraxacum koksaghyz</em>. Plant Mol Biol Rep  28: 277–284.  <ul class="reflinks"><li><a href="#" data-author="Schmidt" data-cit="SchmidtTHillebrandAWurbsDWahlerDLendersM2010Molecular%20cloning%20and%20characterization%20of%20rubber%20biosynthetic%20genes%20from%20Taraxacum%20koksaghyz.Plant%20Mol%20Biol%20Rep28277284" data-title="Molecular%20cloning%20and%20characterization%20of%20rubber%20biosynthetic%20genes%20from%20Taraxacum%20koksaghyz." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+cloning+and+characterization+of+rubber+biosynthetic+genes+from+Taraxacum+koksaghyz.+Schmidt+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0041874-Priya1" id="pone.0041874-Priya1" class="link-target"></a>Priya P, Venkatachalan P, Thulaseedharan A (2007) Differential expression pattern of rubber elongation factor (REF) mRNA transcripts from high and low yielding clones of rubber tree (<em>Hevea brasiliensis</em> Muell. Arg.). Plant Cell Rep  26: 1833–1838.  <ul class="reflinks"><li><a href="#" data-author="Priya" data-cit="PriyaPVenkatachalanPThulaseedharanA2007Differential%20expression%20pattern%20of%20rubber%20elongation%20factor%20%28REF%29%20mRNA%20transcripts%20from%20high%20and%20low%20yielding%20clones%20of%20rubber%20tree%20%28Hevea%20brasiliensis%20Muell.%20Arg.%29.Plant%20Cell%20Rep2618331838" data-title="Differential%20expression%20pattern%20of%20rubber%20elongation%20factor%20%28REF%29%20mRNA%20transcripts%20from%20high%20and%20low%20yielding%20clones%20of%20rubber%20tree%20%28Hevea%20brasiliensis%20Muell.%20Arg.%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+expression+pattern+of+rubber+elongation+factor+%28REF%29+mRNA+transcripts+from+high+and+low+yielding+clones+of+rubber+tree+%28Hevea+brasiliensis+Muell.+Arg.%29.+Priya+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0041874-Wahler1" id="pone.0041874-Wahler1" class="link-target"></a>Wahler D, Schulze Gronover C, Richter C, Foucu F, Twyman RM, et al.  (2009) Polyphenoloxidase Silencing Affects Latex Coagulation in <em>Taraxacum</em> Species. Plant Physiol  151: 334–346.  <ul class="reflinks"><li><a href="#" data-author="Wahler" data-cit="WahlerDSchulze%20GronoverCRichterCFoucuFTwymanRM2009Polyphenoloxidase%20Silencing%20Affects%20Latex%20Coagulation%20in%20Taraxacum%20Species.Plant%20Physiol151334346" data-title="Polyphenoloxidase%20Silencing%20Affects%20Latex%20Coagulation%20in%20Taraxacum%20Species." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Polyphenoloxidase+Silencing+Affects+Latex+Coagulation+in+Taraxacum+Species.+Wahler+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0041874-Wahler2" id="pone.0041874-Wahler2" class="link-target"></a>Wahler D, Colby T, Kowalski NA, Harzen A, Wotzka SY, et al.  (2012) Proteomic analysis of latex from the rubber-producing plant <em>Taraxacum brevicorniculatum</em>. Proteomics  12: 901–905.  <ul class="reflinks"><li><a href="#" data-author="Wahler" data-cit="WahlerDColbyTKowalskiNAHarzenAWotzkaSY2012Proteomic%20analysis%20of%20latex%20from%20the%20rubber-producing%20plant%20Taraxacum%20brevicorniculatum.Proteomics12901905" data-title="Proteomic%20analysis%20of%20latex%20from%20the%20rubber-producing%20plant%20Taraxacum%20brevicorniculatum." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Proteomic+analysis+of+latex+from+the+rubber-producing+plant+Taraxacum+brevicorniculatum.+Wahler+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0041874-Sando1" id="pone.0041874-Sando1" class="link-target"></a>Sando T, Hayashi T, Takeda T, Akiyama Y, Nakazawa Y, et al.  (2009) Histochemical study of detailed laticifer structure and rubber biosynthesis-related protein localization in <em>Hevea brasiliensis</em> using spectral confocal laser scanning microscopy. Planta  230: 215–225.  <ul class="reflinks"><li><a href="#" data-author="Sando" data-cit="SandoTHayashiTTakedaTAkiyamaYNakazawaY2009Histochemical%20study%20of%20detailed%20laticifer%20structure%20and%20rubber%20biosynthesis-related%20protein%20localization%20in%20Hevea%20brasiliensis%20using%20spectral%20confocal%20laser%20scanning%20microscopy.Planta230215225" data-title="Histochemical%20study%20of%20detailed%20laticifer%20structure%20and%20rubber%20biosynthesis-related%20protein%20localization%20in%20Hevea%20brasiliensis%20using%20spectral%20confocal%20laser%20scanning%20microscopy." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Histochemical+study+of+detailed+laticifer+structure+and+rubber+biosynthesis-related+protein+localization+in+Hevea+brasiliensis+using+spectral+confocal+laser+scanning+microscopy.+Sando+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0041874-Ho1" id="pone.0041874-Ho1" class="link-target"></a>Ho CC (1989) Changes in electrokinetic properties of natural rubber latex after surface chemical modifications. Colloid Polym Sci  267: 643–647.  <ul class="reflinks"><li><a href="#" data-author="Ho" data-cit="HoCC1989Changes%20in%20electrokinetic%20properties%20of%20natural%20rubber%20latex%20after%20surface%20chemical%20modifications.Colloid%20Polym%20Sci267643647" data-title="Changes%20in%20electrokinetic%20properties%20of%20natural%20rubber%20latex%20after%20surface%20chemical%20modifications." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Changes+in+electrokinetic+properties+of+natural+rubber+latex+after+surface+chemical+modifications.+Ho+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0041874-Genovese1" id="pone.0041874-Genovese1" class="link-target"></a>Genovese DB, Lozano JE (2001) The effect of hydrocolloids on the stability and viscosity of cloudy apple juices. Food Hydrocolloid  15: 1–7.  <ul class="reflinks"><li><a href="#" data-author="Genovese" data-cit="GenoveseDBLozanoJE2001The%20effect%20of%20hydrocolloids%20on%20the%20stability%20and%20viscosity%20of%20cloudy%20apple%20juices.Food%20Hydrocolloid1517" data-title="The%20effect%20of%20hydrocolloids%20on%20the%20stability%20and%20viscosity%20of%20cloudy%20apple%20juices." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+effect+of+hydrocolloids+on+the+stability+and+viscosity+of+cloudy+apple+juices.+Genovese+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0041874-Tzen1" id="pone.0041874-Tzen1" class="link-target"></a>Tzen JTC, Huang AHC (1992) Surface Structure and Properties of Plant Seed Oil Bodies. J Cell Biol  117: 327–335.  <ul class="reflinks"><li><a href="#" data-author="Tzen" data-cit="TzenJTCHuangAHC1992Surface%20Structure%20and%20Properties%20of%20Plant%20Seed%20Oil%20Bodies.J%20Cell%20Biol117327335" data-title="Surface%20Structure%20and%20Properties%20of%20Plant%20Seed%20Oil%20Bodies." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Surface+Structure+and+Properties+of+Plant+Seed+Oil+Bodies.+Tzen+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0041874-Peng1" id="pone.0041874-Peng1" class="link-target"></a>Peng C-C, Lin I-P, Lin C-K, Tzen JTC (2003) Size and Stability of Reconstituted Sesame Oil Bodies. Biotechnol  Progr19: 1623–1626.  <ul class="reflinks"><li><a href="#" data-author="Peng" data-cit="PengC-CLinI-PLinC-KTzenJTC2003Size%20and%20Stability%20of%20Reconstituted%20Sesame%20Oil%20Bodies.BiotechnolProgr1916231626" data-title="Size%20and%20Stability%20of%20Reconstituted%20Sesame%20Oil%20Bodies." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Size+and+Stability+of+Reconstituted+Sesame+Oil+Bodies.+Peng+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0041874-Sansatsadeekul1" id="pone.0041874-Sansatsadeekul1" class="link-target"></a>Sansatsadeekul J, Sakdapipanich J, Rojruthai P (2011) Characterization of associated proteins and phospholipids in natural rubber latex. J Biosci Bioeng  111: 628–634.  <ul class="reflinks"><li><a href="#" data-author="Sansatsadeekul" data-cit="SansatsadeekulJSakdapipanichJRojruthaiP2011Characterization%20of%20associated%20proteins%20and%20phospholipids%20in%20natural%20rubber%20latex.J%20Biosci%20Bioeng111628634" data-title="Characterization%20of%20associated%20proteins%20and%20phospholipids%20in%20natural%20rubber%20latex." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+associated+proteins+and+phospholipids+in+natural+rubber+latex.+Sansatsadeekul+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0041874-Bradford1" id="pone.0041874-Bradford1" class="link-target"></a>Bradford MM (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal  Biochem72: 248–254.  <ul class="reflinks"><li><a href="#" data-author="Bradford" data-cit="BradfordMM1976A%20rapid%20and%20sensitive%20method%20for%20the%20quantitation%20of%20microgram%20quantities%20of%20protein%20utilizing%20the%20principle%20of%20protein-dye%20binding.AnalBiochem72248254" data-title="A%20rapid%20and%20sensitive%20method%20for%20the%20quantitation%20of%20microgram%20quantities%20of%20protein%20utilizing%20the%20principle%20of%20protein-dye%20binding." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding.+Bradford+1976" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0041874-Shevchenko1" id="pone.0041874-Shevchenko1" class="link-target"></a>Shevchenko A, Tomas H, Havli J, Olsen JV, Mann M (2007) In-gel digestion for mass spectrometric characterization of proteins and proteomes. Nat Protoc  1: 2856–2860.  <ul class="reflinks"><li><a href="#" data-author="Shevchenko" data-cit="ShevchenkoATomasHHavliJOlsenJVMannM2007In-gel%20digestion%20for%20mass%20spectrometric%20characterization%20of%20proteins%20and%20proteomes.Nat%20Protoc128562860" data-title="In-gel%20digestion%20for%20mass%20spectrometric%20characterization%20of%20proteins%20and%20proteomes." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=In-gel+digestion+for+mass+spectrometric+characterization+of+proteins+and+proteomes.+Shevchenko+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0041874-Naumann1" id="pone.0041874-Naumann1" class="link-target"></a>Naumann B, Stauber EJ, Busch A, Sommer F, Hippler M (2005) N-terminal processing of Lhca3 is a key step in remodeling of the photosystem I-light-harvesting complex under iron deficiency in <em>Chlamydomonas reinhardtii</em>. J Biol Chem  280: 20431–20441.  <ul class="reflinks"><li><a href="#" data-author="Naumann" data-cit="NaumannBStauberEJBuschASommerFHipplerM2005N-terminal%20processing%20of%20Lhca3%20is%20a%20key%20step%20in%20remodeling%20of%20the%20photosystem%20I-light-harvesting%20complex%20under%20iron%20deficiency%20in%20Chlamydomonas%20reinhardtii.J%20Biol%20Chem2802043120441" data-title="N-terminal%20processing%20of%20Lhca3%20is%20a%20key%20step%20in%20remodeling%20of%20the%20photosystem%20I-light-harvesting%20complex%20under%20iron%20deficiency%20in%20Chlamydomonas%20reinhardtii." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=N-terminal+processing+of+Lhca3+is+a+key+step+in+remodeling+of+the+photosystem+I-light-harvesting+complex+under+iron+deficiency+in+Chlamydomonas+reinhardtii.+Naumann+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0041874-Geer1" id="pone.0041874-Geer1" class="link-target"></a>Geer LY, Markey SP, Kowalak J A, Wagner L, Xu M, et al.  (2004) Open mass spectrometry search algorithm. J Proteome Res  3: 958–964.  <ul class="reflinks"><li><a href="#" data-author="Geer" data-cit="GeerLYMarkeySPKowalak%20JAWagnerLXuM2004Open%20mass%20spectrometry%20search%20algorithm.J%20Proteome%20Res3958964" data-title="Open%20mass%20spectrometry%20search%20algorithm." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Open+mass+spectrometry+search+algorithm.+Geer+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0041874-Pfaffl1" id="pone.0041874-Pfaffl1" class="link-target"></a>Pfaffl MW, Horgan GW, Dempfle L (2002) Relative expression software tool (REST<sup>©</sup>) for group-wise comparison and statistical analysis of relative expression results in real-time PCR. Nucleic Acid Res  30: e36.  <ul class="reflinks"><li><a href="#" data-author="Pfaffl" data-cit="PfafflMWHorganGWDempfleL2002Relative%20expression%20software%20tool%20%28REST%C2%A9%29%20for%20group-wise%20comparison%20and%20statistical%20analysis%20of%20relative%20expression%20results%20in%20real-time%20PCR.Nucleic%20Acid%20Res30e36" data-title="Relative%20expression%20software%20tool%20%28REST%C2%A9%29%20for%20group-wise%20comparison%20and%20statistical%20analysis%20of%20relative%20expression%20results%20in%20real-time%20PCR." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Relative+expression+software+tool+%28REST%C2%A9%29+for+group-wise+comparison+and+statistical+analysis+of+relative+expression+results+in+real-time+PCR.+Pfaffl+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

