



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2>
<div id="section1" class="section toc-section"><a id="" name="" class="link-target" title="Background"></a>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1" class="link-target"></a><p>The genus <em>Cronobacter</em> (formerly called <em>Enterobacter sakazakii</em>) is composed of five species; <em>C. sakazakii</em>, <em>C. malonaticus</em>, <em>C. turicensis</em>, <em>C. muytjensii</em>, and <em>C. dublinensis</em>. The genus includes opportunistic human pathogens, and the first three species have been associated with neonatal infections. The most severe diseases are caused in neonates and include fatal necrotizing enterocolitis and meningitis. The genetic basis of the diversity within the genus is unknown, and few virulence traits have been identified.</p>
</div>

<div id="section2" class="section toc-section"><a id="" name="" class="link-target" title="Methodology/Principal Findings"></a>
<h3>Methodology/Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1" class="link-target"></a><p>We report here the first sequence of a member of this genus, <em>C. sakazakii</em> strain BAA-894. The genome of <em>Cronobacter sakazakii</em> strain BAA-894 comprises a 4.4 Mb chromosome (57% GC content) and two plasmids; 31 kb (51% GC) and 131 kb (56% GC). The genome was used to construct a 387,000 probe oligonucleotide tiling DNA microarray covering the whole genome. Comparative genomic hybridization (CGH) was undertaken on five other <em>C. sakazakii</em> strains, and representatives of the four other <em>Cronobacter</em> species. Among 4,382 annotated genes inspected in this study, about 55% of genes were common to all <em>C. sakazakii</em> strains and 43% were common to all <em>Cronobacter</em> strains, with 10–17% absence of genes.</p>
</div>

<div id="section3" class="section toc-section"><a id="" name="" class="link-target" title="Conclusions/Significance"></a>
<h3>Conclusions/Significance</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1" class="link-target"></a><p>CGH highlighted 15 clusters of genes in <em>C. sakazakii</em> BAA-894 that were divergent or absent in more than half of the tested strains; six of these are of probable prophage origin. Putative virulence factors were identified in these prophage and in other variable regions. A number of genes unique to <em>Cronobacter</em> species associated with neonatal infections (<em>C. sakazakii</em>, <em>C. malonaticus</em> and <em>C. turicensis</em>) were identified. These included a copper and silver resistance system known to be linked to invasion of the blood-brain barrier by neonatal meningitic strains of <em>Escherichia coli</em>. In addition, genes encoding for multidrug efflux pumps and adhesins were identified that were unique to <em>C. sakazakii</em> strains from outbreaks in neonatal intensive care units.</p>
</div>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kucerova E, Clifton SW, Xia X-Q, Long F, Porwollik S, Fulton L, et al.  (2010) Genome Sequence of <em>Cronobacter sakazakii</em> BAA-894 and Comparative Genomic Hybridization Analysis with Other <em>Cronobacter</em> Species. PLoS ONE 5(3):
           e9556.
        
        https://doi.org/10.1371/journal.pone.0009556</p><p><strong>Editor: </strong>Niyaz Ahmed, University of Hyderabad, India</p><p><strong>Received: </strong>December 6, 2009; <strong>Accepted: </strong>February 14, 2010; <strong>Published: </strong> March 8, 2010</p><p><strong>Copyright: </strong> © 2010 Kucerova et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by National Institutes of Health (NIH) grants R01AI52237, R01AI075093, and R01AI073971, and by a grant from Nottingham Trent University. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p><em>Cronobacter</em> spp. (formerly <em>Enterobacter sakazakii</em>) are Gram-negative, motile, non-sporeforming, peritrichous rods of the <em>Enterobacteriaceae</em> family. <em>Cronobacter</em> is a ubiquitous organism present in a wide range of environments, including water, soil, and a variety of processed foods and fresh produce <a href="#pone.0009556-Friedemann1" class="ref-tip">[1]</a>. The bacterium has been isolated from factory production lines including powdered infant formula factories and households <a href="#pone.0009556-Kandhai1" class="ref-tip">[2]</a> as well as from a wide range of clinical samples including cerebrospinal fluid, blood, bone marrow, sputum, urine and faeces <a href="#pone.0009556-Gurtler1" class="ref-tip">[3]</a>. The organism is an opportunistic pathogen of humans that can cause infections in all age groups. However, low birth weight neonates are most at risk. In this host group <em>Cronobacter</em> has been associated with outbreaks of necrotizing enterocolitis, meningitis and septicaemia. Infections with these presentations result in exceptionally high mortality rates ranging from 40 to 80 percent <a href="#pone.0009556-Lai1" class="ref-tip">[4]</a>. In recent years, some outbreaks of bacterial infection in neonatal intensive care units (NICU) have been traced to powdered formula contaminated with <em>Cronobacter</em> <a href="#pone.0009556-Biering1" class="ref-tip">[5]</a>–<a href="#pone.0009556-vanAcker1" class="ref-tip">[8]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p><em>Cronobacter</em> was defined as ‘yellow-pigmented <em>Enterobacter cloacae</em>’ until 1980, when it was designated a new species, <em>Enterobacter sakazakii</em>, by Farmer <em>et al</em> <a href="#pone.0009556-Farmer1" class="ref-tip">[9]</a>. Analysis of both partial 16S rDNA and <em>hsp60</em> sequences showed that <em>E. sakazakii</em> isolates formed at least four distinct clusters, and it was proposed that clusters 2, 3, and 4 could be unique species <a href="#pone.0009556-Iversen1" class="ref-tip">[10]</a>. Based on DNA-DNA hybridization and phenotyping, <em>Enterobacter sakazakii</em> was subsequently proposed to be re-classified into a new genus <em>Cronobacter</em>, composed of five distinct species: <em>Cronobacter sakazakii</em>, <em>C. malonaticus, C. turicensis</em>, <em>C. muytjensii</em> and <em>C. dublinensis</em> <a href="#pone.0009556-Iversen2" class="ref-tip">[11]</a>. Due to their close relatedness <em>C. sakazakii</em> and <em>C. malonaticus</em> are difficult to distinguish by 16S rDNA sequence analysis. However, multilocus sequence typing (MLST) differentiates between the two species, and also reveals a strong clonal nature of the organism <a href="#pone.0009556-Baldwin1" class="ref-tip">[12]</a>. Previous studies on ‘<em>E. sakazakii</em>’ will therefore be difficult to interpret unless the strains are re-examined and re-classified according to the current taxonomic structure.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p><em>Cronobacter</em> strains vary in their virulence, as determined by epidemiological studies and in-house mammalian tissue culture <a href="#pone.0009556-CaubillaBarron1" class="ref-tip">[6]</a>, <a href="#pone.0009556-Townsend1" class="ref-tip">[13]</a>, <a href="#pone.0009556-Townsend2" class="ref-tip">[14]</a>, but their virulence mechanisms are unknown. The bacteria can attach to intestinal cells and survive in macrophages <a href="#pone.0009556-Townsend1" class="ref-tip">[13]</a>, but the specific receptors involved remain to be determined. To date, only strains from <em>C. sakazakii</em>, <em>C. malonaticus</em> and <em>C. turicensis</em> have been associated with neonatal infections. Recently it was shown that the disruption of tight junctions significantly enhances association of <em>C. sakazakii</em> with Caco2 cells <a href="#pone.0009556-Kim1" class="ref-tip">[15]</a>. Some reports suggest a similarity between the tropism of <em>Cronobacter</em> and <em>Citrobacter koseri</em> for invasion and infection of the central nervous system <a href="#pone.0009556-Burdette1" class="ref-tip">[16]</a>, <a href="#pone.0009556-Willis1" class="ref-tip">[17]</a>. It was noted that brain abscesses due to <em>Cronobacter</em> and <em>Citrobacter koseri</em> were morphologically similar and may be due to similar virulence mechanisms <a href="#pone.0009556-Kline1" class="ref-tip">[18]</a>. The first putative <em>Cronobacter</em> virulence factors were enterotoxin-like compounds produced by four of eighteen strains <a href="#pone.0009556-Pagotto1" class="ref-tip">[19]</a>. The genes encoding the putative toxin have yet to be identified, however.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Here, we present the genome sequence of <em>C. sakazakii</em> strain BAA-894, isolated from powdered formula associated with a NICU outbreak <a href="#pone.0009556-Himelright1" class="ref-tip">[7]</a>, and use that sequence for comparative genomic hybridization (CGH) analysis of physiological and virulence related traits across the <em>Cronobacter</em> genus. Due to the severity of infant infection, a better understanding of the genomic variation between <em>Cronobacter</em> spp. is needed, and will be of interest to manufacturers of powdered infant formula, regulatory bodies, as well as those studying the evolution and diversity of pathogenicity.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results and Discussion"></a><h2>Results and Discussion</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Cronobacter sakazakii BAA-894 Genome"></a>
<h3><em>Cronobacter sakazakii</em> BAA-894 Genome</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>The complete sequencing of the genome of <em>C. sakazakii</em> BAA-894 revealed that it was composed of 1 chromosome (4.36837 Mb, 57% GC) and 2 plasmids (pESAK2 31 kb, 51% GC, pESAK3 131 kb, 56% GC); (Genbank accessions CP000783-5). A largely automated annotation of the genome resulted in the identification of 4,392 genes, covering 87% of the chromosome, 38 genes covering 83% of pESAK2 and 127 genes, covering 87% of pESAK3. The genome is aligned to many other enterobacterial genomes in the Enterix <a href="#pone.0009556-Florea1" class="ref-tip">[20]</a>, <a href="#pone.0009556-Florea2" class="ref-tip">[21]</a> server (<a href="http://enterix.cbcb.umd.edu/enteric/enteric.html">http://enterix.cbcb.umd.edu/enteric/enteric.html</a>).</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Genome Cluster Analysis"></a>
<h3>Genome Cluster Analysis</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>In order to compare closely related genomes to the sequence of <em>C. sakazakii</em> BAA-894, we designed a set of 384,030 50-mer oligonucleotides that tiled the whole genome in both strands at an average density of about one oligonucleotide every 12 bases. An array was then manufactured by Roche NimbleGen (<a href="http://www.nimblegen.com">www.nimblegen.com</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>Genomic diversity of 10 strains of <em>Cronobacter</em> representing the five different recognized species of this genus (<a href="#pone-0009556-t001">Table 1</a>) was analyzed by CGH on this tiled DNA microarray against the sequenced strain <em>C. sakazakii</em> BAA-894. <em>Cronobacter</em> genes were classified as present, absent or of intermediate status, as defined in the <a href="#s3">Materials and Methods</a> section. The raw data is deposited in GenBank GEO, accession number GSE19308.</p>
<a class="link-target" id="pone-0009556-t001" name="pone-0009556-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0009556.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0009556.t001" data-doi="info:doi/10.1371/journal.pone.0009556" data-uri="info:doi/10.1371/journal.pone.0009556.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0009556.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Bacterial strains used in this study.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0009556.t001">
              https://doi.org/10.1371/journal.pone.0009556.t001</a></p></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3" class="link-target"></a><p>To determine the presence or absence of genes, the median log<sub>2</sub> ratio of the genome relative to the reference strain for all the oligonucleotides in that gene was calculated. Then GACK analysis <a href="#pone.0009556-Kim2" class="ref-tip">[22]</a> was used, which sets a floating threshold for presence and absence of genes for every hybridization (see below). Using Cluster and Treeview softwares <a href="#pone.0009556-Eisen1" class="ref-tip">[23]</a>, <em>Cronobacter</em> strains formed two distinct phylogenetic clusters. All <em>C. sakazakii</em> strains formed one cluster (<a href="#pone-0009556-g001">Figure 1</a>). <em>C. malonaticus, C. turicensis, C. dublinensis</em> and <em>C. malonaticus</em> formed a second, separate cluster. Within <em>C. sakazakii</em>, strains 701 and 767 were the most closely related and clustered together with strain 20. Previously, strains 701 and 767 were shown to belong to the same pulse field gel electrophoresis restriction digestion type <a href="#pone.0009556-CaubillaBarron1" class="ref-tip">[6]</a>. Although the clinical details of the source of <em>C. sakazakii</em> strain 20 are unknown, the strain belongs to MLST sequence type 4 (as do 701 and 767), which is a stable clone of <em>C. sakazakii</em> isolated from both powdered infant formula and clinical sources <a href="#pone.0009556-Baldwin1" class="ref-tip">[12]</a>. <em>C. sakazakii</em> strain ATCC 29544<sup>T</sup> (species type strain) formed a separate branch within the <em>C. sakazakii</em> cluster. The remaining <em>Cronobacter</em> species formed sub-clusters: <em>C. malonaticus</em> clustered with <em>C. turicensis</em> and <em>C. dublinensis</em> grouped with <em>C. muytjensii</em>. The tree remained identical when adjacent genes were collapsed into a single phylogenetic character if they had the same pattern of presence and absence.</p>
<a class="link-target" id="pone-0009556-g001" name="pone-0009556-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0009556.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0009556.g001" data-doi="info:doi/10.1371/journal.pone.0009556" data-uri="info:doi/10.1371/journal.pone.0009556.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0009556.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Whole genome clustering analysis showing the relationship between 10 <em>Cronobacter</em> isolates.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>Genomic regions GR1–GR15 are marked. Clustering analysis was performed using Gene Cluster (EisenSoftware). Hierarchical clustering was performed using the average linkage method on the trinary matrix based on the CGH analysis (1 for presence, 0 for uncertain and −1 for absence/divergence of a gene). For description of strains refer to <a href="#pone-0009556-t001">Table 1</a>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0009556.g001">
              https://doi.org/10.1371/journal.pone.0009556.g001</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Core Genome"></a>
<h3>Core Genome</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Of the 4,382 unique annotated gene sequences represented on the microarray, 54.9% (2404) were common to all <em>C. sakazakii</em> strains and 43.3% (1899) were common to all five <em>Cronobacter</em> species. The vast majority of these shared genes are predicted to encode cellular essential functions such as energy metabolism, biosynthesis, DNA, RNA and protein synthesis, cell division and membrane transport. The proportion of genes absent from test strains compared with <em>C. sakazakii</em> BAA-894 ranged from 10.3% (453) in <em>C. sakazakii</em> strain 20 to 17.1% (751) in <em>C. muytjensii</em> (<a href="#pone-0009556-t002">Table 2</a>). In total, 5.1% (224) of BAA-894 genes were absent in all <em>C. sakazakii</em> strains, and 3.1% (137) genes were absent in all <em>Cronobacter</em> strains (<a href="#pone-0009556-t002">Table 2</a>). Even though <em>C. muytjensii</em> and <em>C. malonaticus</em> are classified as separate species, the proportion of absent genes was only 11.3% and 11.9%, respectively, when compared to <em>C. sakazakii</em> BAA-894. This is in concordance with the previous 16S sequence comparison studies which showed that all <em>Cronobacter</em> strains are closely related (<a href="#pone-0009556-t001">Table 1</a>).</p>
<a class="link-target" id="pone-0009556-t002" name="pone-0009556-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0009556.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0009556.t002" data-doi="info:doi/10.1371/journal.pone.0009556" data-uri="info:doi/10.1371/journal.pone.0009556.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0009556.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Number of absent or divergent and intermediate genes in <em>Cronobacter</em> strains when compared to the sequenced <em>C. sakazakii</em> BAA-894.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0009556.t002">
              https://doi.org/10.1371/journal.pone.0009556.t002</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="C. sakazakii Unique Genes"></a>
<h3><em>C. sakazakii</em> Unique Genes</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>CGH analysis showed there were 21 genes present in all five <em>C. sakazakii</em> strains which were absent in <em>C. malonaticus, C. muytjensii, C. turicensis</em> and <em>C. dublinensis</em>. The genes unique to <em>C. sakazakii</em> strains were in two separate clusters of proteins involved in pilus assembly (ESA_02540–ESA_02542 and ESA_02796–ESA_02799), pilin FimA proteins (ESA_02541, ESA_02542, ESA_02796 and ESA_02799), porin PapC (ESA_02797) and the chaperone PapD (ESA_02798). The genes unique to <em>C. sakazakii</em> also included proteins for the phosphotransferase system (ESA_03303 and ESA_03305), a putative sialic acid transporter (ESA_03611), N-acetylneuraminate lyase (ESA_03612) and RelB from a toxin/antitoxin system (ESA_00257).</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Invasion of Brain Microvascular Endothelial Cells"></a>
<h3>Invasion of Brain Microvascular Endothelial Cells</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Because <em>Cronobacter</em> is associated with often fatal cases of neonatal meningitis, the status of genes identified in other organisms as associated with invasion of brain microvascular endothelial cells (BMEC) (<em>ibeA, ibeB, yijP</em> and <em>ompA</em>) in the sequenced isolate was of particular interest <a href="#pone.0009556-Huang1" class="ref-tip">[24]</a>–<a href="#pone.0009556-Wang1" class="ref-tip">[27]</a>. The gene encoding OmpA was present in all tested strains. This protein is associated with the invasive ability of neonatal meningitic <em>E. coli</em>. While genes <em>ibeA</em> and <em>yijP</em> produced no match in the reference strain <em>C. sakazakii</em> BAA-894, <em>ibeB</em> (synonymous to <em>cusC</em>) was found. <em>CusC</em> belongs to a cluster of genes encoding a copper and silver resistance cation efflux system which allows bacteria to invade BMEC <a href="#pone.0009556-Franke1" class="ref-tip">[28]</a>. The complete cation efflux system c<em>usA</em> (ESA_04242), <em>cusB</em> (ESA_04241), <em>cusC</em> (ESA_04239), <em>cusF</em> (ESA_04240) and its regulatory gene <em>cusR</em> (ESA_04238) was present in strains associated with neonatal infections (<em>C. sakazakii</em> ATCC 29544<sup>T</sup>, 701, 767, 696, <em>C. turicensis</em> and <em>C. malonaticus</em>), and absent in the other tested strains.</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Other Physiological Traits"></a>
<h3>Other Physiological Traits</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>The presence of genes conferring physiological traits commonly associated with <em>Cronobacter</em> spp. was examined. Seventy genes involved in desiccation resistance <a href="#pone.0009556-Osaili1" class="ref-tip">[29]</a>, the metalloprotease <em>zpx</em> which causes rounding of Chinese hamster ovary (CHO) cells <a href="#pone.0009556-Kothary1" class="ref-tip">[30]</a> and yellow pigment production genes <a href="#pone.0009556-Lehner1" class="ref-tip">[31]</a> were present on the arrays. All these genes were present in all 10 <em>Cronobacter</em> strains tested.</p>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="Comparison of C. sakazakii Neonatal Intensive Care Unit (NICU) Outbreak Strains with C. sakazakii Type Strain ATCC 29544T"></a>
<h3>Comparison of <em>C. sakazakii</em> Neonatal Intensive Care Unit (NICU) Outbreak Strains with <em>C. sakazakii</em> Type Strain ATCC 29544<sup>T</sup></h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>The genes that were shared by the three strains associated with <em>C. sakazakii</em> outbreaks in NICUs (BAA-894, 701 &amp; 767) were compared with the <em>C. sakazakii</em> species type strain ATCC 29544<sup>T</sup>, which showed decreased virulence properties compared to strains 701 and 767 in tissue culture studies <a href="#pone.0009556-Townsend2" class="ref-tip">[14]</a>. One hundred and forty-four genes present in the three NICU strains were absent in the type strain, 66 (46%) in clusters of consecutive genes based on the annotation of BAA-894. In most of these clusters, genes encoding proteins associated with resistance to different forms of stress were identified, including multidrug efflux systems, genes involved in resistance to oxidative stress, and those with a putative function in resistance to metals. The complete list of genes present in NICU outbreak strains <em>C. sakazakii</em> BAA-894, 707 and 767 and absent in the <em>C. sakazakii</em> type strain ATCC 29544<sup>T</sup> is in <a href="#pone.0009556.s002">Table S1</a>; genes of interest are listed below.</p>

<div id="section1" class="section toc-section"><a id="s2g1" name="s2g1" class="link-target" title="Category I"></a><h4>Category I.</h4><a id="article1.body1.sec2.sec7.sec1.p1" name="article1.body1.sec2.sec7.sec1.p1" class="link-target"></a><p>Genes encoding proteins associated with resistance to different forms of antibiotics: (i), a transcriptional regulator (ESA_01938) from the TetR family of protein repressors that control the level of susceptibility to hydrophobic antibiotics and detergents; (ii), a homologue of the CpmG protein involved in carbapenem resistance (ESA_pESA3p05435); (iii), a protein conferring resistance to antimicrobial peptides Mig-14 (ESA_pESA3p05439); and (iv), a transcriptional regulator (ESA_pESA3p05448) involved in tetracycline resistance.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2g2" name="s2g2" class="link-target" title="Category II"></a><h4>Category II.</h4><a id="article1.body1.sec2.sec7.sec2.p1" name="article1.body1.sec2.sec7.sec2.p1" class="link-target"></a><p>Genes encoding multidrug efflux systems: (i), a cationic drug transporter (ESA_01940) from the family of proteins that confer resistance to a wide range of toxic compounds; (ii), genes for complete bacterial ABC-transport systems involved in active transport across the cytoplasmic membrane (ESA_01944–ESA_01946); and (iii), a variety of genes encoding multidrug efflux components located on the plasmid pESA3 in BAA-894 (<a href="#pone.0009556.s002">Table S1</a>).</p>
</div>

<div id="section3" class="section toc-section"><a id="s2g3" name="s2g3" class="link-target" title="Category III"></a><h4>Category III.</h4><a id="article1.body1.sec2.sec7.sec3.p1" name="article1.body1.sec2.sec7.sec3.p1" class="link-target"></a><p>Genes involved in resistance to oxidative stress and genes with a putative function in resistance to metals: (i), a redox-sensitive transcriptional activator SoxR (ESA_00115); (ii), a glutathione S-transferase (ESA_00116); (iii), ADP-ribose pyrophosphatase involved in oxidative stress protection (ESA_pESA3p05446); (iv), an arsenate reductase (ESA_pESA3p05485); and (v), a predicted transcriptional regulator involved in mercurium resistance ESA_pESA3p05463.</p>
</div>

<div id="section4" class="section toc-section"><a id="s2g4" name="s2g4" class="link-target" title="Category IV"></a><h4>Category IV.</h4><a id="article1.body1.sec2.sec7.sec4.p1" name="article1.body1.sec2.sec7.sec4.p1" class="link-target"></a><p>Other genes of interest include: (i), putative adhesins which are recognized as virulence factors in enteric bacteria <a href="#pone.0009556-Campellone1" class="ref-tip">[32]</a> (ESA_00983–ESA_00986); (ii), the universal stress protein UspA (ESA_01955) which can enhance the rate of cell survival during prolonged exposure to stress conditions <a href="#pone.0009556-Nystrom1" class="ref-tip">[33]</a>; (iii), a gene encoding a Type VI secretion lysozyme-related protein (ESA_02735); (iv), a gene for a predicted virulence SciE-type protein (ESA_02736) which affects the ability of bacteria to enter eukaryotic cells <a href="#pone.0009556-Folkesson1" class="ref-tip">[34]</a>; and (v), genes involved in pilus assembly (ESA_03515 and ESA_03516).</p>
</div>
</div>

<div id="section8" class="section toc-section"><a id="s2h" name="s2h" class="link-target" title="C. sakazakii Plasmids"></a>
<h3><em>C. sakazakii</em> Plasmids</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>The sequenced strain <em>C. sakazakii</em> BAA-894 contains two plasmids; pESA2 (31 kb) and pESA3 (131 kb). Thirty-eight genes were annotated on pESA2 and 127 genes on pESA3.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2" class="link-target"></a><p>The copy number of the plasmids was estimated from the median hybridization signals of oligonucleotides representing the plasmid compared to the sequenced genome. The ratio was (1∶1.1∶8.6) for the chromosome versus pESA2 versus pESA3. Thus, pESA2 exists as low copy, and pESA3 appears to be a moderate copy number plasmid.</p>
<a id="article1.body1.sec2.sec8.p3" name="article1.body1.sec2.sec8.p3" class="link-target"></a><p>The genes on pESA2 were absent in all other strains tested except <em>C. turicensis</em>, which had 19 (61.3%) genes present, and <em>C. sakazakii</em> 696, which had 4 (12.9%) genes present. The results for genes on pESA3 are summarized in <a href="#pone-0009556-t003">Table 3</a>.</p>
<a class="link-target" id="pone-0009556-t003" name="pone-0009556-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0009556.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0009556.t003" data-doi="info:doi/10.1371/journal.pone.0009556" data-uri="info:doi/10.1371/journal.pone.0009556.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0009556.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> pESA3 genes present in <em>Cronobacter</em> strains when compared to the sequenced strain of <em>C. sakazakii</em> BAA-894.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0009556.t003">
              https://doi.org/10.1371/journal.pone.0009556.t003</a></p></div><a id="article1.body1.sec2.sec8.p4" name="article1.body1.sec2.sec8.p4" class="link-target"></a><p>Note that it is possible that some or all of the detected genes are on the chromosome in other strains. In addition, genes on a multicopy or medium copy plasmid may require a different degree of divergence to be identified as absent or divergent by comparative hybridizations. Plasmid profiling was performed on the Cronobacter strains analyzed by comparative hybridization,. A plasmid of a size similar to pESA2 (31 kb) was detected in <em>C. sakazakii</em> 696 and in <em>C. turicensis</em>, which is in accordance with our CGH results. A large plasmid similar in size to pESA3 (131 kb) was visible in <em>C. sakazakii</em> strains ATCC 29544<sup>T</sup>, ATCC 2868, 20, 696, 701, 767 and <em>C. malonaticus</em> (<a href="#pone-0009556-t003">Table 3</a>).</p>
</div>

<div id="section9" class="section toc-section"><a id="s2i" name="s2i" class="link-target" title="Genomic Regions Absent in Some Strains of Cronobacter"></a>
<h3>Genomic Regions Absent in Some Strains of <em>Cronobacter</em></h3>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1" class="link-target"></a><p>Genes that were absent in more than half of the <em>Cronobacter</em> strains relative to the sequenced strain <em>C. sakazakii</em> BAA-894 were selected for further analysis. These genes form 15 clusters of contiguous genes (based on the annotation of the reference genome). These are shown on <a href="#pone-0009556-g002">Figure 2</a>, where the number of strains in which a particular gene was classified as absent is plotted against the gene locus. The clusters were designated as regions GR1–GR15.</p>
<a class="link-target" id="pone-0009556-g002" name="pone-0009556-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0009556.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0009556.g002" data-doi="info:doi/10.1371/journal.pone.0009556" data-uri="info:doi/10.1371/journal.pone.0009556.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0009556.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Distribution of the 15 variable regions across the <em>C. sakazakii</em> BAA-894 genome.</div><p class="caption_target"><a id="article1.body1.sec2.sec9.fig1.caption1.p1" name="article1.body1.sec2.sec9.fig1.caption1.p1" class="link-target"></a><p>Each column represents a gene classified as absent by CGH analysis in at least one strain. The height of the columns indicate the number of strains (out of 10) in which the gene was found to be absent. The major variable regions (blue) and prophages (red) are indicated in order of their appearance in the genome of <em>C. sakazakii</em> BAA-894.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0009556.g002">
              https://doi.org/10.1371/journal.pone.0009556.g002</a></p></div></div>

<div id="section10" class="section toc-section"><a id="s2j" name="s2j" class="link-target" title="Prophages"></a>
<h3>Prophages</h3>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1" class="link-target"></a><p>Of the 15 clusters, three putative prophage genomes and one prophage fragment were identified by Prophinder <a href="#pone.0009556-LimaMendez1" class="ref-tip">[35]</a>, and two additional regions are probable prophage fragments based on the presence of phage protein homologues identified by BLASTX (<a href="#pone-0009556-t003">Table 3</a>).</p>
<a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2" class="link-target"></a><p>In the three prophage gene clusters (prophages 1, 2 and 3), genes encoding close homologues of known phage genes involved in integration, lysis and termination as well as head and tail structure were identified based on amino acid identity searches in IMG-JGI (<a href="http://img.jgi.doe.gov/cgi-bin/pub/main.cgi">http://img.jgi.doe.gov/cgi-bin/pub/main.cgi</a>). The average GC content of the sequenced <em>C. sakazakii</em> BAA-894 genome is 56%, the GC content of prophages 1, 2 and 3 was 53, 49 and 51%, respectively. The complete list of annotated putative prophage genes is available <a href="#pone.0009556.s003">Table S2</a>. In addition, <a href="#pone.0009556.s001">Figure S1</a>. shows the status of all putative prophage genes in the 10 <em>Cronobacter</em> strains.</p>
<a id="article1.body1.sec2.sec10.p3" name="article1.body1.sec2.sec10.p3" class="link-target"></a><p><strong>Prophage 1</strong> (<strong>GR4</strong>; ESA_00990–ESA_01052). In the 46-kb putative prophage 1 (<a href="#pone-0009556-g003">Figure 3A</a>), 30/63 (48%) hypothetical proteins were similar to known phage proteins. Homologues of phage genes involved in integration, lysis, head morphogenesis, tail assembly and phage regulation were identified. Prophage 1 also contains a gene (ESA_00997) encoding a protein homologous to the <em>eae</em>-like adhesion protein associated with the attaching and effacing phenotype. The <em>eae</em> gene is carried by some other bacteriophages of enteric pathogens: <em>Salmonella</em> phage epsilon34, <em>E. coli</em> O157∶H7 bacteriophage PhiV10, enterobacterial phage P22 and enterobacterial phage epsilon15.</p>
<a class="link-target" id="pone-0009556-g003" name="pone-0009556-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0009556.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0009556.g003" data-doi="info:doi/10.1371/journal.pone.0009556" data-uri="info:doi/10.1371/journal.pone.0009556.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0009556.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> Gene maps of putative prophages.</div><p class="caption_target"><a id="article1.body1.sec2.sec10.fig1.caption1.p1" name="article1.body1.sec2.sec10.fig1.caption1.p1" class="link-target"></a><p>A. Gene map of putative prophage 1. B. Gene map of putative prophage 2. C. Gene map of putative prophage 3. Annotation of the putative prophage genes is available in <a href="#pone.0009556.s003">Table S2</a>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0009556.g003">
              https://doi.org/10.1371/journal.pone.0009556.g003</a></p></div><a id="article1.body1.sec2.sec10.p4" name="article1.body1.sec2.sec10.p4" class="link-target"></a><p><strong>Prophage 2</strong> (<strong>GR6</strong>; ESA_01608–ESA_01644). Putative prophage 2 (<a href="#pone-0009556-g003">Figure 3B</a>) contains 37 genes, out of which 25 (68%) were homologous to known phage proteins. Prophage 2 contains several lambdoid phage genes encoding the following proteins: repressor CII (ESA_01613), replication proteins O and P (ESA_01614 and ESA_01615), the antitermination protein Q (ESA_01622), small and large subunits of the phage terminase (ESA_01632 and ESA_01633) as well as head and tail morphogenesis proteins. Head morphogenesis genes (ESA_01635–ESA_01637) were similar to head proteins of bacteriophage HK97 from the family of lambda phages. Two gene clusters have very low average GC content; 33% (ESA_01616–ESA_01620) and 44% (ESA_01627–ESA_01631). Both these clusters contain hypothetical proteins that showed no similarity with known phage proteins or functions.</p>
<a id="article1.body1.sec2.sec10.p5" name="article1.body1.sec2.sec10.p5" class="link-target"></a><p><strong>Prophage 3</strong> (<strong>GR12</strong>; ESA_03025–ESA_03102) was the largest (47 kb) putative prophage identified (<a href="#pone-0009556-g003">Figure 3C</a>). Thirty-four genes (39%) genes had close homology to known phage proteins or functions. The rest of the annotated genes are conserved proteins of unknown functions or hypothetical proteins. Similarly to prophage 2, several regulatory genes characteristic for lambdoid phages were identified: repressor proteins CI, CII and CIII, early gene regulator protein, replication proteins O and P as well as N independence proteins NinBFGZ. A cluster of three O-antigen conversion genes (ESA_03026–ESA_03028) was found in putative prophage 3 between phage integrase and tail morphogenesis genes (<a href="#pone-0009556-g003">Figure 3C</a>). The putative colicin uptake protein TolA (ESA_03048) may be involved in the internalization of the bacteriophage, as the Tol pathway can be also used for the translocation of phages into the bacterial cell <a href="#pone.0009556-Lazdunski1" class="ref-tip">[36]</a>. CGH showed that the entire genome of prophage 3 or its close relatives are absent from the genomes of all other <em>Cronobacter</em> strains tested except <em>C. turicensis</em>. In this species, 18 prophage genes (mostly annotated as hypothetical proteins) were classified as present (<a href="#pone.0009556.s001">Figure S1</a>). Interestingly, the CIII regulator protein (ESA_03094), the Kil protein (ESA_03095), and both large and small terminase subunits (ESA_03052 and ESA_03053) were present in two <em>C. sakazakii</em> strains (701 and 767), possibly as a part of different bacteriophages. These two strains were isolated from two fatal neonate cases of <em>C. sakazakii</em> infection <a href="#pone.0009556-CaubillaBarron1" class="ref-tip">[6]</a>. A cluster of putative phage tail proteins (ESA_03029–ESA_03034) and a cluster containing phage head morphogenesis genes and a putative colicin uptake gene (ESA_03039–ESA_03051) were homologous to genes of the <em>S. enterica</em> serovar Typhi Vi type II phage E1 which may use virulence-associated capsular antigen as entry. This antigen was present on the surface of clinical Typhi isolates <a href="#pone.0009556-Pickard1" class="ref-tip">[37]</a>. Although most genes of the putative prophage 3 were absent in all 10 <em>Cronobacter</em> strains tested by CGH, the region corresponding to the phage Vi genes (ESA_03041–ESA_03048) was present in <em>C. turicensis</em> and partially present in <em>C. sakazakii</em> strains 20, 701 and 767 (<a href="#pone.0009556.s001">Figure S1</a>). Strains 701 and 767 were both associated with fatal outbreaks, and are in MLST sequence type 4 with strain 20 <a href="#pone.0009556-Baldwin1" class="ref-tip">[12]</a>.</p>
<a id="article1.body1.sec2.sec10.p6" name="article1.body1.sec2.sec10.p6" class="link-target"></a><p><strong>Prophage fragment 1</strong> (<strong>GR3</strong>; ESA_00604–ESA_00630). Eight of 19 genes in this region encode proteins associated with phages: plasmid and phage DNA primase (ESA_00620), a protein from Ash phage family (ESA_00624), the phage transcriptional regulator AlpA (ESA_00625), a putative phage capsid protein (ESA_00626), the phage transcriptional activator Ogr/Delta (ESA_00627) and phage integrase (ESA_00630). ESA_00618 was homologous to ea59 of lambda bacteriophage and ESA_00622 was homologous to a P4 phage protein. This cluster is most probably a phage remnant and may not encode a functional phage due to the absence of homologues of known structural tail proteins. The phage cluster was absent in all other strains. However, a short region, ESA_00609–ESA_00617, was present in <em>C. sakazakii</em> strains 701 and 767. A group of restriction endonucleases belonging to this cluster encoding a restriction-modification methyltransferase subunit (ESA_00614), a restriction endonuclease S subunit (ESA_00615), a hypothetical protein (ESA_00616) and a type I site-specific deoxyribonuclease (ESA_00617) were homologous to genes api49, api50, api51 and api52, respectively, from the <em>Yersinia pseudotuberculosis</em> adhesion pathogenicity island <a href="#pone.0009556-Collyn1" class="ref-tip">[38]</a>.</p>
<a id="article1.body1.sec2.sec10.p7" name="article1.body1.sec2.sec10.p7" class="link-target"></a><p><strong>Prophage fragment 2</strong> (<strong>GR10</strong>). The region ESA_02304–ESA_02339 is likely to represent another prophage remnant. The region mostly contains hypothetical proteins with unknown functions. However, eight genes showed some degree of homology to proteins of phage origin. The cluster is flanked by a gene homologous to phage methyltransferase (ESA_02304) and a gene containing a site-specific recombinase domain that is found in putative integrases/recombinases of mobile genetic elements of diverse bacteria and phages (ESA_02339). It also includes genes homologous to phage lysozyme (ESA_02309), a phage tail component (ESA_02311), a putative phage tape measure protein (ESA_02313), another unspecified phage protein (ESA_02316), a major capsid protein (ESA_02319) and a phage portal protein (ESA_02320). GR10 might be another remnant of a prophage that has previously integrated into <em>C. sakazakii</em> BAA-894 genome. The presence of putative integrase flanking the cluster suggests introduction of this cluster into the genome by horizontal gene transfer. This cluster was absent in all <em>Cronobacter</em> strains except the genes encoding phage lysozyme (ESA_02309) and a hypothetical protein (ESA_02310), present in <em>C. sakazakii</em> strains 2 and 20, probably as a part of different prophages.</p>
<a id="article1.body1.sec2.sec10.p8" name="article1.body1.sec2.sec10.p8" class="link-target"></a><p><strong>Prophage fragment 3</strong> (<strong>GR11</strong>; ESA_02740–ESA_02755). The gene cluster of the 8 kb putative prophage fragment 3 comprises 16 hypothetical proteins, 10 of which (63%) may be associated with phage functions. As it lacks genes for head and tail morphogenesis as well as phage regulatory genes, it is likely to be a non-functional phage remnant.</p>
<a id="article1.body1.sec2.sec10.p9" name="article1.body1.sec2.sec10.p9" class="link-target"></a><p>Two thirds of all gamma-proteobacteria and low GC Gram-negative bacteria harbor prophages <a href="#pone.0009556-Brussow1" class="ref-tip">[39]</a>. There is an increasing body of evidence that phages play a pivotal role in the diversification of bacterial species. Some phages can carry additional cargo genes, which are not required for the phage cycle and are suspected or proven virulence factors <a href="#pone.0009556-Brussow1" class="ref-tip">[39]</a>. Such genes are typically located near prophage ends, downstream of phage tail genes or next to Q or N-like antiterminator genes <a href="#pone.0009556-Boyd1" class="ref-tip">[40]</a>. Putative prophages identified in this study contain genes which are not similar to any other known prophage genes. Moreover, prophage 1 contains a gene encoding a protein homologous to the <em>eae</em>-like adhesion protein, which is a recognized virulence factor in enteropathogenic <em>E</em>. <em>coli</em> associated with the attaching and effacing phenotype <a href="#pone.0009556-Kaper1" class="ref-tip">[41]</a>. It is hypothesized that most prophages are lost from bacterial genomes shortly after their acquisition <a href="#pone.0009556-Brussow1" class="ref-tip">[39]</a>. Hence, some of the cargo genes carried by the prophages remaining in the chromosome of <em>Cronobacter</em> are possible virulence factors or fitness factors that increase the survival of the bacterium in its host. Further research into these putative virulence factors is warranted.</p>
</div>

<div id="section11" class="section toc-section"><a id="s2k" name="s2k" class="link-target" title="Non-Phage Regions That Differ among the Strains"></a>
<h3>Non-Phage Regions That Differ among the Strains</h3>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1" class="link-target"></a><p>The complete list of genes belonging to the variable non-phage regions and their presence in tested strains is available in <a href="#pone.0009556.s003">Table S2</a>.</p>
<a id="article1.body1.sec2.sec11.p2" name="article1.body1.sec2.sec11.p2" class="link-target"></a><p><strong>GR1</strong> (ESA_00140–ESA_00145) is a small cluster of type VI secretion system genes. ESA_00142 shares a conserved region with a family of IcmF-related proteins proposed to be involved in increased <em>Vibrio cholerae</em> adherence to epithelial cells <a href="#pone.0009556-Das1" class="ref-tip">[42]</a>. ESA_00143 is a secretion protein belonging to the VC_A0110 family; mutations in proteins of this family are associated with impaired virulence <a href="#pone.0009556-Pukatzki1" class="ref-tip">[43]</a>. ESA_00145, a secretion lipoprotein from a VC _A0113 family was present in <em>C. sakazakii</em> strains 701 and 767 associated with fatal outbreaks. The rest of GR1 was absent in all tested strains except <em>C. sakazakii</em> 696.</p>
<a id="article1.body1.sec2.sec11.p3" name="article1.body1.sec2.sec11.p3" class="link-target"></a><p><strong>GR2</strong> (ESA_00292–ESA_00310) mostly contains uncharacterized conserved proteins. It also contains the gene encoding a protein from a family of beta-lactamases (ESA_00299).</p>
<a id="article1.body1.sec2.sec11.p4" name="article1.body1.sec2.sec11.p4" class="link-target"></a><p><strong>GR5</strong> (ESA_01179–ESA_01189) contains a cluster of proteins involved in cell wall biogenesis and nucleotide sugar metabolism. GR5 corresponds to the <em>C. sakazakii</em> O-antigen gene locus used to distinguish the two <em>Cronobacter</em> serotypes O∶1 and O∶2 <a href="#pone.0009556-Mullane1" class="ref-tip">[44]</a>. DNA microarray analysis revealed that GR5 is highly divergent; its genes were not sufficiently similar to be detected by microarray hybridization in any other <em>Cronobacter</em> strains. The O-antigen locus contains two more genes (homologues of ESA_01177 and ESA_01178) which were present in all strains except <em>C. sakazakii</em> 696.</p>
<a id="article1.body1.sec2.sec11.p5" name="article1.body1.sec2.sec11.p5" class="link-target"></a><p><strong>GR7</strong> (ESA_01775–ESA_01804). Most genes in GR7 were predicted to be involved in tellurite and stress resistance. It contains homologues of tellurite resistance proteins TerA, TerC, TerD, TerY and TerZ. The cluster contains two putative transposase genes, which suggests that the cluster was acquired by horizontal transfer. GR7 was found to be carried on plasmid pK29 of <em>Klebsiella pneumoniae</em> strain NK29, plasmid pEC-IMPQ of <em>Enterobacter cloacae</em>, plasmid R478 of <em>Serratia marcescens</em> and plasmid pAPEC-O1-R of <em>Escherichia coli</em> APEC O1, which is further evidence of horizontal gene transfer. As the gene cluster was entirely absent from all other <em>Cronobacter</em> strains in our study, the reference strain BAA-894 probably acquired the tellurite resistance cluster recently.</p>
<a id="article1.body1.sec2.sec11.p6" name="article1.body1.sec2.sec11.p6" class="link-target"></a><p><strong>GR8</strong> (ESA_01970–ESA_01976) contains seven genes encoding pilus assembly proteins. Fimbriae (or pili) enable bacteria to colonize the epithelium of specific host organs and are therefore considered major virulence factors <a href="#pone.0009556-Abraham1" class="ref-tip">[45]</a>. This cluster of genes was absent in all <em>Cronobacter</em> strains except <em>C. sakazakii</em> strain 20.</p>
<a id="article1.body1.sec2.sec11.p7" name="article1.body1.sec2.sec11.p7" class="link-target"></a><p><strong>GR9</strong> (ESA_02032–ESA_02041) contains genes encoding hypothetical proteins and four proteins involved in Type VI secretion system (ESA_02037–ESA_02040). These four genes were present in <em>C. sakazakii</em> strains 1 and 20 and absent in all other strains.</p>
<a id="article1.body1.sec2.sec11.p8" name="article1.body1.sec2.sec11.p8" class="link-target"></a><p><strong>GR13</strong> (ESA_03887–ESA_03912) is a cluster of 16 hypothetical proteins without homology to known proteins or functions. This cluster was absent in all other <em>Cronobacter</em> strains.</p>
<a id="article1.body1.sec2.sec11.p9" name="article1.body1.sec2.sec11.p9" class="link-target"></a><p><strong>GR14</strong> (ESA_04248–ESA_04255) is a cluster of genes involved in copper resistance (<a href="#pone-0009556-t004">Table 4</a>). This cluster was found in <em>C. sakazakii</em> strain 1 and 696, as well as <em>C. turicensis</em> and <em>C. malonaticus</em>.</p>
<a class="link-target" id="pone-0009556-t004" name="pone-0009556-t004"></a><div class="figure" data-doi="10.1371/journal.pone.0009556.t004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0009556.t004" data-doi="info:doi/10.1371/journal.pone.0009556" data-uri="info:doi/10.1371/journal.pone.0009556.t004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0009556.t004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0009556.t004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0009556.t004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0009556.t004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 4. </span> Putative prophages identified in the genome of <em>C. sakazakii</em> BAA-894 by Prophinder <a href="#pone.0009556-LimaMendez1" class="ref-tip">[35]</a> or BlastX.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0009556.t004">
              https://doi.org/10.1371/journal.pone.0009556.t004</a></p></div><a id="article1.body1.sec2.sec11.p10" name="article1.body1.sec2.sec11.p10" class="link-target"></a><p><strong>GR15</strong> (ESA_pESA3p05493–ESA_pESA3p05505) involves genes located on <em>C. sakazakii</em> plasmid pESA3 and includes components of type IV and type VI secretion pathways (<a href="#pone.0009556.s003">Table S2</a>) as well as a gene encoding an outer membrane protein from the OmpA family (ESA_pESA3p05495). GR15 was absent from all strains except <em>C. sakazakii</em> strains 1 and 696.</p>
<a id="article1.body1.sec2.sec11.p11" name="article1.body1.sec2.sec11.p11" class="link-target"></a><p>Most of the described regions contain suspected or proven virulence factors. The genes in GR1, GR9 and GR15 are involved in a type VI secretion system, a newly described mechanism for protein transport across the cell envelope of Gram-negative bacteria that can increase adherence to epithelial cells <a href="#pone.0009556-Mougous1" class="ref-tip">[46]</a>, <a href="#pone.0009556-Schlieker1" class="ref-tip">[47]</a>. GR3 contains four genes (ESA_00614–ESA_00617) that are homologous to a restriction-modification gene cluster (api49–api52) in the <em>Yersinia pseudotuberculosis</em> pathogenicity island (YAPI) <a href="#pone.0009556-Collyn1" class="ref-tip">[38]</a>. As these genes were present in strains 701 and 767 isolated from two neonates that died as a result of infection by <em>Cronobacter</em> during an outbreak in France <a href="#pone.0009556-vanAcker1" class="ref-tip">[8]</a>, <a href="#pone.0009556-Willis1" class="ref-tip">[17]</a>, and are absent in all other strains tested by CGH, these genes may be important virulence factors contributing to the pathogenicity of <em>Cronobacter</em>.</p>
<a id="article1.body1.sec2.sec11.p12" name="article1.body1.sec2.sec11.p12" class="link-target"></a><p><em>Cronobacter</em> virulence factors have not been extensively studied, although it is known that <em>Cronobacter</em> species vary in their virulence with respect to invasion of intestinal cells, survival in macrophages and serum resistance <a href="#pone.0009556-Townsend1" class="ref-tip">[13]</a>, <a href="#pone.0009556-Townsend2" class="ref-tip">[14]</a>. GR 5 (ESA_01179–ESA_01189) encodes the lipopolysaccharide (LPS) genes. Characterisation of LPS structure and consequently O-antigen can be important in developing identification schemes based on serotyping, and has a role in virulence and serum resistance of the organism. The LPS is one of the few structural features of <em>Cronobacter</em> which has been investigated and it is known that it varies across the <em>Cronobacter</em> spp. In <em>C. sakazakii</em> and <em>C. malonaticus</em>, the LPS are composed of various branched polymers, whereas they are unbranched in <em>C. muytjensii</em>. In <em>C. sakazakii</em> BAA-894 <a href="#pone.0009556-MacLean1" class="ref-tip">[48]</a> it is a branched polymer of pentasaccharide units composed of 2-acetamido-2-deoxy-D-galactose, 3-(N-acetyl-L-alanylamido)-3-deoxy-D-quinovose, D-glucuronic acid, and D-glucose. <em>C. sakazakii</em> strain 767 is also a branched polymer but of a repeating heptasaccharides composed of 2-acetamido-2-deoxy-D-glucose, and D-galacturonic acid, L-rhamnose, and D-glucose <a href="#pone.0009556-Czerwicka1" class="ref-tip">[49]</a>. <em>C. malonaticus</em> LPS <a href="#pone.0009556-MacLean2" class="ref-tip">[50]</a> is also a branched pentasaccharide unit of 2-amino-2-deoxy-D-glucose, 2-amino-2-deoxy-D-galactose, 3-deoxy-D-manno-oct-2-ulosonic acid, D-galactose and D-glucose residues. Whereas, <em>C. muytjensii</em> LPS <a href="#pone.0009556-MacLean3" class="ref-tip">[51]</a> is a linear unbranched pentasccharide polymer of 2-acetamido-2-deoxy-D-galactose, 2-acetamido-2-deoxy-D-glucose, 2-acetamido-3-deoxy-D-quinovose, L-rhamnose and D-glucuronic acid. These considerable differences correspond with the lack of sequence conservation in GR5 as revealed in the microarray analysis. The individual genes encoding these differences in enyzmology have yet to be assigned.</p>
</div>

<div id="section12" class="section toc-section"><a id="s2l" name="s2l" class="link-target" title="Comparison to Other Enterobacterial Genera"></a>
<h3>Comparison to Other Enterobacterial Genera</h3>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1" class="link-target"></a><p>The sequenced genome <em>C. sakazakii</em> BAA-894 was compared to the genomes of <em>Citrobacter koseri</em> BAA-895, <em>Klebsiella oxytoca</em> VJSK009, <em>E. coli</em> K12 MG1655 and <em>Salmonella enterica</em> Typhimurium strain LT2, representing some of the most closely related genera to <em>Cronobacter.</em> Using a threshold of identity of &gt;85% in a 100 base window, 334 genes were present in all <em>Cronobacter</em> but absent or diverged in the four members of other genera (manuscript in preparation). These genes included a cluster of type VI secretion genes (ESA_03943 - ESA_03948) which might be involved in virulence, and a putative palatinose operon (ESA_02709 - ESA_02715). Alpha-glucosidase activity, which has been linked to palatinose metabolism, is considered as one of the major biochemical traits that distinguish <em>Cronobacter</em> from other related <em>Enterobacteriaceae</em>.</p>
</div>

<div id="section13" class="section toc-section"><a id="s2m" name="s2m" class="link-target" title="Summary and Conclusions"></a>
<h3>Summary and Conclusions</h3>
<a id="article1.body1.sec2.sec13.p1" name="article1.body1.sec2.sec13.p1" class="link-target"></a><p>Using a whole-genome ∼384,000 oligonucleotide tiling microarray, we analyzed the genomic content of isolates representing all five <em>Cronobacter</em> species by CGH. A dynamic determination of cut-offs GACK <a href="#pone.0009556-Kim2" class="ref-tip">[22]</a> was used to minimize the number of incorrectly categorized genes. Among 6 strains of <em>C. sakazakii</em> 2,404 genes (54.9%) represented a core shared genome. Of these genes 1,899 (43.3%) were also in the core genome when compared to four other <em>Cronobacter</em> species. CGH highlighted a copper/silver resistance cluster associated with invasion of BMEC, which were unique to the three <em>Cronobacter</em> species associated with neonatal infections, as well as efflux pumps and adhesins unique to <em>C. sakazakii</em> strains from NICU outbreaks.</p>
<a id="article1.body1.sec2.sec13.p2" name="article1.body1.sec2.sec13.p2" class="link-target"></a><p>The main genetic features that distinguished the <em>Cronobacter</em> strains were putative prophages and several other gene clusters, a pattern of divergence typical among bacteria <a href="#pone.0009556-Porwollik1" class="ref-tip">[52]</a>. A few of the regions present in the sequenced strain and absent in some other <em>Cronobacter</em> strains are found in only a few other <em>Enterobacteriaceae</em>. For example, GR 7 is found in four of the hundreds of <em>Enterobacteriaceae</em> genomes that have been sequenced, which indicates that this region may have been horizontally acquired, possibly from a source outside of the <em>Enterobateriaeciae</em>. We have shown that gene acquisition via integration of phages and other mobile elements and specific gene-loss play a major role in <em>Cronobacter</em> evolution and diversity. Fifteen clusters of genes including three putative prophages and three putative prophage fragments that were absent in more than half tested strains were identified. In most of them, putative virulence genes were identified.</p>
<a id="article1.body1.sec2.sec13.p3" name="article1.body1.sec2.sec13.p3" class="link-target"></a><p>Future studies will focus on the expression of virulence related genes and their role in the pathogenicity of <em>Cronobacter</em> species, particularly the mechanisms of neonatal infection.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="Strains and Culture Conditions"></a>
<h3>Strains and Culture Conditions</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p><em>Cronobacter</em> strains were selected which represented the five recognized species, and included those from reported clinical cases (<a href="#pone-0009556-t001">Table 1</a>). All <em>Cronobacter</em> strains were stored at −80°C in Nutrient Broth (Oxoid, UK) with 10% glycerol, subcultured on Trypticase Soy Agar (Oxoid, UK) and checked for purity. Overnight Trypticase Soy Broth (Oxoid, UK) cultures were used for DNA extraction.</p>
</div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="DNA Extraction"></a>
<h3>DNA Extraction</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>Total genomic DNA was isolated using the QIAGEN Genomic-tip 100/G and Genomic DNA Buffer Set (<a href="http://www.1.qiagen.com">www.1.qiagen.com</a>) with extended cell lysis (1 h) and two additional washes of the precipitated DNA. The DNA samples were checked for fragmentation on 0.8% agarose gels and checked for protein and RNA content by spectrophotometry.</p>
</div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="Sequencing of C. sakazakii ATCC BAA-894 and Assembly"></a>
<h3>Sequencing of <em>C. sakazakii</em> ATCC BAA-894 and Assembly</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>The complete genome of <em>C. sakazakii</em> BAA-894, a strain isolated from a powdered formula used during an NICU outbreak <a href="#pone.0009556-Himelright1" class="ref-tip">[7]</a>, was sequenced using the whole genome shotgun method, supplemented with end sequencing of a fosmid library. Sonicated and size-fractionated DNA was cloned into plasmid vectors (pOTw13). Subclones and fosmids were sequenced using dye-primer and dye-terminator chemistry on ABI 3730 sequencing robots. Using the PCAP assembly software program <a href="#pone.0009556-Huang3" class="ref-tip">[53]</a>, 51,289 sequence reads, representing 6.2 fold coverage, were assembled. Part of the 6.2X coverage included 1.19X fosmids.Under-represented areas, gaps, and ambiguities were then addressed by performing automated sequence improvement <a href="#pone.0009556-Gordon1" class="ref-tip">[54]</a> using directed sequencing from the subclones (plasmids and fosmids). Following the auto-finish process, correcting misassembled regions, resolving ambiguous bases, and filling the remaining gaps by additional directed sequencing and PCR, completed the finishing process. This yielded a product with a final estimated accuracy of 99.99%.</p>
</div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="Annotation"></a>
<h3>Annotation</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>AceDB was the primary annotation database. The identification of protein-coding genes used a combination of GeneMark, Glimmer 2.0 and Glimmer 3.0; an evidence-based approach was used to prioritize genes for inclusion into a final gene set. Genes missed by the two <em>ab initio</em> gene predictors were identified using BlastX.</p>
</div>

<div id="section5" class="section toc-section"><a id="s3e" name="s3e" class="link-target" title="C. sakazakii ATCC BAA-894 Microarray Design and Comparative Genome Hybridization Analysis (CGH)"></a>
<h3><em>C. sakazakii</em> ATCC BAA-894 Microarray Design and Comparative Genome Hybridization Analysis (CGH)</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>A 384,030 probe oligonucleotide tiling DNA microarray was designed which comprised the complete genomic sequence of <em>C. sakazakii</em> ATCC BAA-894. Probes were designed at an average of less than 12 base spacing on alternating strands, leading to an average of over 100 50-mer oligonucleotide probes per annotated gene.</p>
<a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2" class="link-target"></a><p>Every possible 50-base probe from both strands of the ESA genome and two plasmids was tested for the ability to be manufactured by NimbleGen. Those that required too many NimbleGen cycles were shortened. Resulting candidate probes that were less than 35 bases long were thrown out. The remaining 9,061,350 potential probes had an average melting temperature (Tm) of 74 degrees Celcius. Probes that had Tm above the average were shorted, down to a minimum of 35 bases. The probes were selected from the pool of 9,061,350 potential probes by selecting the best probe at 11.375-base increments, alternating between strands each time. The resulting 386,802 candidate probe sequences where analyzed to remove any probes that covered the same region as another probe due to duplications in the genome and the resulting 384,030 unique sequences were chosen for the array. Mappings between the unique probe sequences and their genome/plasmid/gene positions were stored in a separate file.</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3" class="link-target"></a><p>Sample labeling, CGH and data normalization were performed according to the method described at <a href="http://www.nimblegen.com/products/lit/cgh_userguide_v5p1.pdf">http://www.nimblegen.com/products/lit/cgh_userguide_v5p1.pdf</a>.</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4" class="link-target"></a><p>DNA from the sequenced strain, <em>C. sakazakii</em> BAA-894, was used as the internal array control. For within-array normalization, a LOWESS method <a href="#pone.0009556-Cleveland1" class="ref-tip">[55]</a> was used as spatial correction and QSPline <a href="#pone.0009556-Workman1" class="ref-tip">[56]</a> was used to correct for dye bias. The raw data is deposited in GenBank GEO (accession number GSE19308).</p>
</div>

<div id="section6" class="section toc-section"><a id="s3f" name="s3f" class="link-target" title="Data Visualization by WebArrayDB"></a>
<h3>Data Visualization by WebArrayDB</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>The CGH plotter available at WebArray (<a href="http://www.webarraydb.org/webarray/index.html">www.webarraydb.org/webarray/index.html</a>) was used to create CGH plots. This used the log<sub>2</sub> intensity ratio microarray data to calculate the median log<sub>2</sub> intensity ratios for each <em>C. sakazakii</em> BAA-894 gene. WebArrayDB is a database system and online cross-platform analysis suite for analysis of microarray data, which allows storage of the data in the repository and their online analysis <a href="#pone.0009556-Xia1" class="ref-tip">[57]</a>.</p>
</div>

<div id="section7" class="section toc-section"><a id="s3g" name="s3g" class="link-target" title="Dynamic Cut-Off Determination by GACK"></a>
<h3>Dynamic Cut-Off Determination by GACK</h3>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1" class="link-target"></a><p>Each gene was represented by tens or hundreds of separate oligonucleotides. These measurmements were condensed into a single median ratio for each gene. The data were further analyzed using the dynamic cut-off determination tool GACK <a href="#pone.0009556-Kim2" class="ref-tip">[22]</a>. The normal probability density function was calculated from the characteristics of the main peak of the data distribution. This was used to calculate the estimated probability of presence that gives a statistical validation of the gene assignment. Each CGH experiment was attributed a specific set of cut-offs to minimize the number of falsely assigned genes.</p>
<a id="article1.body1.sec3.sec7.p2" name="article1.body1.sec3.sec7.p2" class="link-target"></a><p><em>Cronobacter</em> genes were classified according to the most stringent settings of the trinary output of GACK as present, intermediate or absent. Genes classified as ‘present’ had a log<sub>2</sub> intensity ratio (test/reference) greater than the cut-off value corresponding to the 100% estimated probability of presence (EPP) calculated by GACK. The ‘absent’ genes had a log<sub>2</sub> ratio inferior to the cut-off value corresponding to 0% EPP, and can include genes with sufficient sequence divergence. The ‘intermediate’ category included genes whose status could not be assigned with certainty. The EPP function and cut-offs were determined separately for hybridization data of each strain (<a href="#pone.0009556.s004">Table S3</a>).</p>
</div>

<div id="section8" class="section toc-section"><a id="s3h" name="s3h" class="link-target" title="Prophage Identification"></a>
<h3>Prophage Identification</h3>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1" class="link-target"></a><p>ACLAME Prophinder was used for prophage identification <a href="#pone.0009556-LimaMendez1" class="ref-tip">[35]</a>. This was accessed at <a href="http://aclame.ulb.ac.be/Tools/Prophinder/">http://aclame.ulb.ac.be/Tools/Prophinder/</a></p>
</div>

<div id="section9" class="section toc-section"><a id="s3i" name="s3i" class="link-target" title="16S rDNA Sequence Analysis"></a>
<h3>16S rDNA Sequence Analysis</h3>
<a id="article1.body1.sec3.sec9.p1" name="article1.body1.sec3.sec9.p1" class="link-target"></a><p>Partial 16S rDNA sequence analysis was done as previously described <a href="#pone.0009556-Iversen1" class="ref-tip">[10]</a>.</p>
</div>

<div id="section10" class="section toc-section"><a id="s3j" name="s3j" class="link-target" title="Plasmid Profiling"></a>
<h3>Plasmid Profiling</h3>
<a id="article1.body1.sec3.sec10.p1" name="article1.body1.sec3.sec10.p1" class="link-target"></a><p>Plasmid DNA was isolated according to the method described in <a href="#pone.0009556-Kado1" class="ref-tip">[58]</a>.</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0009556"></div><div class="supplementary-material"><a name="pone.0009556.s001" id="pone.0009556.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0009556.s001">Figure S1. </a></h3><a id="article1.body1.sec4.supplementary-material1.caption1.p1" name="article1.body1.sec4.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">Absence/presence status of prophage genes in <em>Cronobacter</em> strains. Red indicates absence/divergence of a particular gene, orange indicates uncertain status and green indicates presence of a gene.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0009556.s001">https://doi.org/10.1371/journal.pone.0009556.s001</a></p><a id="article1.body1.sec4.supplementary-material1.caption1.p2" name="article1.body1.sec4.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.15 MB TIF)</p>
</div><div class="supplementary-material"><a name="pone.0009556.s002" id="pone.0009556.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0009556.s002">Table S1. </a></h3><a id="article1.body1.sec4.supplementary-material2.caption1.p1" name="article1.body1.sec4.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Selected genomic regions present in NICU outbreak strains <em>C. sakazakii</em> 707 and 767 and absent in <em>C. sakazakii</em> type strain ATCC 29544T.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0009556.s002">https://doi.org/10.1371/journal.pone.0009556.s002</a></p><a id="article1.body1.sec4.supplementary-material2.caption1.p2" name="article1.body1.sec4.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.16 MB DOC)</p>
</div><div class="supplementary-material"><a name="pone.0009556.s003" id="pone.0009556.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0009556.s003">Table S2. </a></h3><a id="article1.body1.sec4.supplementary-material3.caption1.p1" name="article1.body1.sec4.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">Non-phage genomic regions absent in more than half of the tested strains.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0009556.s003">https://doi.org/10.1371/journal.pone.0009556.s003</a></p><a id="article1.body1.sec4.supplementary-material3.caption1.p2" name="article1.body1.sec4.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.40 MB DOC)</p>
</div><div class="supplementary-material"><a name="pone.0009556.s004" id="pone.0009556.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0009556.s004">Table S3. </a></h3><a id="article1.body1.sec4.supplementary-material4.caption1.p1" name="article1.body1.sec4.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">Cut-off values used for the assignment of absent, intermediate or present gene status.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0009556.s004">https://doi.org/10.1371/journal.pone.0009556.s004</a></p><a id="article1.body1.sec4.supplementary-material4.caption1.p2" name="article1.body1.sec4.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank Yollande Pui Cheng for technical assistance and Liliana Florea of Bioinformatics advise.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: SWC RKW MM. Performed the experiments: EK LAF CF PJM KK WW RSF DF NS VB WEN. Analyzed the data: EK SWC XQX AW MM SF. Contributed reagents/materials/analysis tools: SWC XQX FL SP KHP MM. Wrote the paper: EK MM SF.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0009556-Friedemann1" id="pone.0009556-Friedemann1" class="link-target"></a>Friedemann M (2007) <em>Enterobacter sakazakii</em> in food and beverages (other than infant formula and milk powder). Int J Food Microbiol  116: 1–10.  <ul class="reflinks"><li><a href="#" data-author="Friedemann" data-cit="FriedemannM2007Enterobacter%20sakazakii%20in%20food%20and%20beverages%20%28other%20than%20infant%20formula%20and%20milk%20powder%29.Int%20J%20Food%20Microbiol116110" data-title="Enterobacter%20sakazakii%20in%20food%20and%20beverages%20%28other%20than%20infant%20formula%20and%20milk%20powder%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii+in+food+and+beverages+%28other+than+infant+formula+and+milk+powder%29.+Friedemann+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0009556-Kandhai1" id="pone.0009556-Kandhai1" class="link-target"></a>Kandhai MC, Reij MW, Gorris LG, Guillaume-Gentil O, van Schothorst M (2004) Occurrence of <em>Enterobacter sakazakii</em> in food production environments and households. Lancet  363(9402): 39–40.  <ul class="reflinks"><li><a href="#" data-author="Kandhai" data-cit="KandhaiMCReijMWGorrisLGGuillaume-GentilOvan%20SchothorstM2004Occurrence%20of%20Enterobacter%20sakazakii%20in%20food%20production%20environments%20and%20households.Lancet363%289402%293940" data-title="Occurrence%20of%20Enterobacter%20sakazakii%20in%20food%20production%20environments%20and%20households." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Occurrence+of+Enterobacter+sakazakii+in+food+production+environments+and+households.+Kandhai+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0009556-Gurtler1" id="pone.0009556-Gurtler1" class="link-target"></a>Gurtler JB, Kornacki JL, Beuchat LR (2005) <em>Enterobacter sakazakii</em>: a coliform of increased concern to infant health. Int J Food Microbiol  104: 1–34.  <ul class="reflinks"><li><a href="#" data-author="Gurtler" data-cit="GurtlerJBKornackiJLBeuchatLR2005Enterobacter%20sakazakii%3A%20a%20coliform%20of%20increased%20concern%20to%20infant%20health.Int%20J%20Food%20Microbiol104134" data-title="Enterobacter%20sakazakii%3A%20a%20coliform%20of%20increased%20concern%20to%20infant%20health." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii%3A+a+coliform+of+increased+concern+to+infant+health.+Gurtler+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0009556-Lai1" id="pone.0009556-Lai1" class="link-target"></a>Lai KK (2001) <em>Enterobacter sakazakii</em> infections among neonates, infants, children, and adults. Medicine  80: 113–122.  <ul class="reflinks"><li><a href="#" data-author="Lai" data-cit="LaiKK2001Enterobacter%20sakazakii%20infections%20among%20neonates%2C%20infants%2C%20children%2C%20and%20adults.Medicine80113122" data-title="Enterobacter%20sakazakii%20infections%20among%20neonates%2C%20infants%2C%20children%2C%20and%20adults." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii+infections+among+neonates%2C+infants%2C+children%2C+and+adults.+Lai+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0009556-Biering1" id="pone.0009556-Biering1" class="link-target"></a>Biering G, Karlsson S, Clark NC, Jonsdottir KE, Ludvigsson P, et al.  (1989) Three cases of neonatal meningitis caused by <em>Enterobacter sakazakii</em> in powdered milk. J Clin Microbiol  27: 2054–6.  <ul class="reflinks"><li><a href="#" data-author="Biering" data-cit="BieringGKarlssonSClarkNCJonsdottirKELudvigssonP1989Three%20cases%20of%20neonatal%20meningitis%20caused%20by%20Enterobacter%20sakazakii%20in%20powdered%20milk.J%20Clin%20Microbiol2720546" data-title="Three%20cases%20of%20neonatal%20meningitis%20caused%20by%20Enterobacter%20sakazakii%20in%20powdered%20milk." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Three+cases+of+neonatal+meningitis+caused+by+Enterobacter+sakazakii+in+powdered+milk.+Biering+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0009556-CaubillaBarron1" id="pone.0009556-CaubillaBarron1" class="link-target"></a>Caubilla-Barron J, Hurrell E, Townsend S, Cheetham P, Loc-Carrillo C, et al.  (2007) Genotypic and phenotypic analysis of <em>Enterobacter sakazakii</em> strains from an outbreak resulting in fatalities in a neonatal intensive care unit in France. J Clin Microbiol  45: 3979–85.  <ul class="reflinks"><li><a href="#" data-author="Caubilla-Barron" data-cit="Caubilla-BarronJHurrellETownsendSCheethamPLoc-CarrilloC2007Genotypic%20and%20phenotypic%20analysis%20of%20Enterobacter%20sakazakii%20strains%20from%20an%20outbreak%20resulting%20in%20fatalities%20in%20a%20neonatal%20intensive%20care%20unit%20in%20France.J%20Clin%20Microbiol45397985" data-title="Genotypic%20and%20phenotypic%20analysis%20of%20Enterobacter%20sakazakii%20strains%20from%20an%20outbreak%20resulting%20in%20fatalities%20in%20a%20neonatal%20intensive%20care%20unit%20in%20France." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genotypic+and+phenotypic+analysis+of+Enterobacter+sakazakii+strains+from+an+outbreak+resulting+in+fatalities+in+a+neonatal+intensive+care+unit+in+France.+Caubilla-Barron+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0009556-Himelright1" id="pone.0009556-Himelright1" class="link-target"></a>Himelright I, Harris E, Lorch V, Anderson M (2002) <em>Enterobacter sakazakii</em> infections associated with the use of powdered infant formula—Tennessee, 2001. J Am Med Assoc  287: 2204–2205.  <ul class="reflinks"><li><a href="#" data-author="Himelright" data-cit="HimelrightIHarrisELorchVAndersonM2002Enterobacter%20sakazakii%20infections%20associated%20with%20the%20use%20of%20powdered%20infant%20formula%E2%80%94Tennessee%2C%202001.J%20Am%20Med%20Assoc28722042205" data-title="Enterobacter%20sakazakii%20infections%20associated%20with%20the%20use%20of%20powdered%20infant%20formula%E2%80%94Tennessee%2C%202001." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii+infections+associated+with+the+use+of+powdered+infant+formula%E2%80%94Tennessee%2C+2001.+Himelright+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0009556-vanAcker1" id="pone.0009556-vanAcker1" class="link-target"></a>van Acker J, de Smet F, Muyldermans G, Bougatef A, Naessens A, et al.  (2001) Outbreak of necrotizing enterocolitis associated with <em>Enterobacter sakazakii</em> in powdered milk formula. J Clin Microbiol  39: 293–7.  <ul class="reflinks"><li><a href="#" data-author="van%20Acker" data-cit="van%20AckerJde%20SmetFMuyldermansGBougatefANaessensA2001Outbreak%20of%20necrotizing%20enterocolitis%20associated%20with%20Enterobacter%20sakazakii%20in%20powdered%20milk%20formula.J%20Clin%20Microbiol392937" data-title="Outbreak%20of%20necrotizing%20enterocolitis%20associated%20with%20Enterobacter%20sakazakii%20in%20powdered%20milk%20formula." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Outbreak+of+necrotizing+enterocolitis+associated+with+Enterobacter+sakazakii+in+powdered+milk+formula.+van+Acker+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0009556-Farmer1" id="pone.0009556-Farmer1" class="link-target"></a>Farmer JJ, Asbury MA, Hickman FW, Brenner DJ, the Enterobacteriaceae Study Group (USA) (1980) “<em>Enterobacter sakazakii</em>: a new species of <em>“Enterobacteriaceae”</em> isolated from clinical specimens”. Int J Syst Bacteriol  30: 569–84.  <ul class="reflinks"><li><a href="#" data-author="Farmer" data-cit="FarmerJJAsburyMAHickmanFWBrennerDJthe%20Enterobacteriaceae%20Study%20Group%20%28USA%291980%E2%80%9CEnterobacter%20sakazakii%3A%20a%20new%20species%20of%20%E2%80%9CEnterobacteriaceae%E2%80%9D%20isolated%20from%20clinical%20specimens%E2%80%9D.Int%20J%20Syst%20Bacteriol3056984" data-title="%E2%80%9CEnterobacter%20sakazakii%3A%20a%20new%20species%20of%20%E2%80%9CEnterobacteriaceae%E2%80%9D%20isolated%20from%20clinical%20specimens%E2%80%9D." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=%E2%80%9CEnterobacter+sakazakii%3A+a+new+species+of+%E2%80%9CEnterobacteriaceae%E2%80%9D+isolated+from+clinical+specimens%E2%80%9D.+Farmer+1980" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0009556-Iversen1" id="pone.0009556-Iversen1" class="link-target"></a>Iversen C, Waddington M, On SL, Forsythe S (2004) Identification and phylogeny of <em>Enterobacter sakazakii</em> relative to <em>Enterobacter</em> and <em>Citrobacter</em> Species. J Clin Microbiol  42: 5368–70.  <ul class="reflinks"><li><a href="#" data-author="Iversen" data-cit="IversenCWaddingtonMOnSLForsytheS2004Identification%20and%20phylogeny%20of%20Enterobacter%20sakazakii%20relative%20to%20Enterobacter%20and%20Citrobacter%20Species.J%20Clin%20Microbiol42536870" data-title="Identification%20and%20phylogeny%20of%20Enterobacter%20sakazakii%20relative%20to%20Enterobacter%20and%20Citrobacter%20Species." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+phylogeny+of+Enterobacter+sakazakii+relative+to+Enterobacter+and+Citrobacter+Species.+Iversen+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0009556-Iversen2" id="pone.0009556-Iversen2" class="link-target"></a>Iversen C, Mullane N, McCardell B, Tall BD, Lehner A, et al.  (2008) <em>Cronobacter</em> gen. nov., a new genus to accommodate the biogroups of <em>Enterobacter sakazakii</em>, and proposal of <em>Cronobacter sakazakii</em> gen. nov., comb. nov., <em>Cronobacter malonaticus</em> sp. nov., <em>Cronobacter turicensis</em> sp. nov., <em>Cronobacter muytjensii sp.</em> nov., <em>Cronobacter dublinensis</em> sp. nov., <em>Cronobacter</em> genomospecies 1, and of three subspecies, <em>Cronobacter</em> dublinensis subsp. <em>dublinensis</em> subsp. nov., <em>Cronobacter dublinensis</em> subsp. <em>lausannensis</em> subsp. nov. and <em>Cronobacter dublinensis</em> subsp. <em>lactaridi</em> subsp. nov. Int J Syst Evol Microbiol  58(Pt 6): 1442–7.  <ul class="reflinks"><li><a href="#" data-author="Iversen" data-cit="IversenCMullaneNMcCardellBTallBDLehnerA2008Cronobacter%20gen.%20nov.%2C%20a%20new%20genus%20to%20accommodate%20the%20biogroups%20of%20Enterobacter%20sakazakii%2C%20and%20proposal%20of%20Cronobacter%20sakazakii%20gen.%20nov.%2C%20comb.%20nov.%2C%20Cronobacter%20malonaticus%20sp.%20nov.%2C%20Cronobacter%20turicensis%20sp.%20nov.%2C%20Cronobacter%20muytjensii%20sp.%20nov.%2C%20Cronobacter%20dublinensis%20sp.%20nov.%2C%20Cronobacter%20genomospecies%201%2C%20and%20of%20three%20subspecies%2C%20Cronobacter%20dublinensis%20subsp.%20dublinensis%20subsp.%20nov.%2C%20Cronobacter%20dublinensis%20subsp.%20lausannensis%20subsp.%20nov.%20and%20Cronobacter%20dublinensis%20subsp.%20lactaridi%20subsp.%20nov.Int%20J%20Syst%20Evol%20Microbiol58%28Pt%206%2914427" data-title="Cronobacter%20gen.%20nov.%2C%20a%20new%20genus%20to%20accommodate%20the%20biogroups%20of%20Enterobacter%20sakazakii%2C%20and%20proposal%20of%20Cronobacter%20sakazakii%20gen.%20nov.%2C%20comb.%20nov.%2C%20Cronobacter%20malonaticus%20sp.%20nov.%2C%20Cronobacter%20turicensis%20sp.%20nov.%2C%20Cronobacter%20muytjensii%20sp.%20nov.%2C%20Cronobacter%20dublinensis%20sp.%20nov.%2C%20Cronobacter%20genomospecies%201%2C%20and%20of%20three%20subspecies%2C%20Cronobacter%20dublinensis%20subsp.%20dublinensis%20subsp.%20nov.%2C%20Cronobacter%20dublinensis%20subsp.%20lausannensis%20subsp.%20nov.%20and%20Cronobacter%20dublinensis%20subsp.%20lactaridi%20subsp.%20nov." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cronobacter+gen.+nov.%2C+a+new+genus+to+accommodate+the+biogroups+of+Enterobacter+sakazakii%2C+and+proposal+of+Cronobacter+sakazakii+gen.+nov.%2C+comb.+nov.%2C+Cronobacter+malonaticus+sp.+nov.%2C+Cronobacter+turicensis+sp.+nov.%2C+Cronobacter+muytjensii+sp.+nov.%2C+Cronobacter+dublinensis+sp.+nov.%2C+Cronobacter+genomospecies+1%2C+and+of+three+subspecies%2C+Cronobacter+dublinensis+subsp.+dublinensis+subsp.+nov.%2C+Cronobacter+dublinensis+subsp.+lausannensis+subsp.+nov.+and+Cronobacter+dublinensis+subsp.+lactaridi+subsp.+nov.+Iversen+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0009556-Baldwin1" id="pone.0009556-Baldwin1" class="link-target"></a>Baldwin A, Loughlin M, Caubilla-Barron J, Kucerova E, Manning G, et al.  (2009) Multilocus sequence typing of <em>Cronobacter sakazakii</em> and <em>Cronobacter malonaticus</em> reveals stable clonal structures with clinical significance which do not correlate with biotypes. BMC Microbiol  9: 223.  <ul class="reflinks"><li><a href="#" data-author="Baldwin" data-cit="BaldwinALoughlinMCaubilla-BarronJKucerovaEManningG2009Multilocus%20sequence%20typing%20of%20Cronobacter%20sakazakii%20and%20Cronobacter%20malonaticus%20reveals%20stable%20clonal%20structures%20with%20clinical%20significance%20which%20do%20not%20correlate%20with%20biotypes.BMC%20Microbiol9223" data-title="Multilocus%20sequence%20typing%20of%20Cronobacter%20sakazakii%20and%20Cronobacter%20malonaticus%20reveals%20stable%20clonal%20structures%20with%20clinical%20significance%20which%20do%20not%20correlate%20with%20biotypes." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multilocus+sequence+typing+of+Cronobacter+sakazakii+and+Cronobacter+malonaticus+reveals+stable+clonal+structures+with+clinical+significance+which+do+not+correlate+with+biotypes.+Baldwin+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0009556-Townsend1" id="pone.0009556-Townsend1" class="link-target"></a>Townsend S, Hurrell E, Forsythe S (2008) Virulence studies of <em>Enterobacter sakazakii</em> isolates associated with a neonatal intensive care unit outbreak. BMC Microbiol  8: 64.  <ul class="reflinks"><li><a href="#" data-author="Townsend" data-cit="TownsendSHurrellEForsytheS2008Virulence%20studies%20of%20Enterobacter%20sakazakii%20isolates%20associated%20with%20a%20neonatal%20intensive%20care%20unit%20outbreak.BMC%20Microbiol864" data-title="Virulence%20studies%20of%20Enterobacter%20sakazakii%20isolates%20associated%20with%20a%20neonatal%20intensive%20care%20unit%20outbreak." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Virulence+studies+of+Enterobacter+sakazakii+isolates+associated+with+a+neonatal+intensive+care+unit+outbreak.+Townsend+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0009556-Townsend2" id="pone.0009556-Townsend2" class="link-target"></a>Townsend SM, Hurrell E, Gonzalez-Gomez I, Lowe J, Frye JG, et al.  (2007) <em>Enterobacter sakazakii</em> invades brain capillary endothelial cells, persists in human macrophages influencing cytokine secretion and induces severe brain pathology in the neonatal rat. Microbiology  153(Pt 10): 3538–47.  <ul class="reflinks"><li><a href="#" data-author="Townsend" data-cit="TownsendSMHurrellEGonzalez-GomezILoweJFryeJG2007Enterobacter%20sakazakii%20invades%20brain%20capillary%20endothelial%20cells%2C%20persists%20in%20human%20macrophages%20influencing%20cytokine%20secretion%20and%20induces%20severe%20brain%20pathology%20in%20the%20neonatal%20rat.Microbiology153%28Pt%2010%29353847" data-title="Enterobacter%20sakazakii%20invades%20brain%20capillary%20endothelial%20cells%2C%20persists%20in%20human%20macrophages%20influencing%20cytokine%20secretion%20and%20induces%20severe%20brain%20pathology%20in%20the%20neonatal%20rat." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii+invades+brain+capillary+endothelial+cells%2C+persists+in+human+macrophages+influencing+cytokine+secretion+and+induces+severe+brain+pathology+in+the+neonatal+rat.+Townsend+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0009556-Kim1" id="pone.0009556-Kim1" class="link-target"></a>Kim KP, Loessner MJ (2008) <em>Enterobacter sakazakii</em> invasion in human intestinal Caco-2 cells requires the host cell cytoskeleton and is enhanced by disruption of tight junction. Infect Immun  76: 562–70.  <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimKPLoessnerMJ2008Enterobacter%20sakazakii%20invasion%20in%20human%20intestinal%20Caco-2%20cells%20requires%20the%20host%20cell%20cytoskeleton%20and%20is%20enhanced%20by%20disruption%20of%20tight%20junction.Infect%20Immun7656270" data-title="Enterobacter%20sakazakii%20invasion%20in%20human%20intestinal%20Caco-2%20cells%20requires%20the%20host%20cell%20cytoskeleton%20and%20is%20enhanced%20by%20disruption%20of%20tight%20junction." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii+invasion+in+human+intestinal+Caco-2+cells+requires+the+host+cell+cytoskeleton+and+is+enhanced+by+disruption+of+tight+junction.+Kim+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0009556-Burdette1" id="pone.0009556-Burdette1" class="link-target"></a>Burdette JH, Santos C (2000) <em>Enterobacter sakazakii</em> brain abscess in the neonate: the importance of neuroradiologic imaging. Pediatr Radiol  30: 33–4.  <ul class="reflinks"><li><a href="#" data-author="Burdette" data-cit="BurdetteJHSantosC2000Enterobacter%20sakazakii%20brain%20abscess%20in%20the%20neonate%3A%20the%20importance%20of%20neuroradiologic%20imaging.Pediatr%20Radiol30334" data-title="Enterobacter%20sakazakii%20brain%20abscess%20in%20the%20neonate%3A%20the%20importance%20of%20neuroradiologic%20imaging." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii+brain+abscess+in+the+neonate%3A+the+importance+of+neuroradiologic+imaging.+Burdette+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0009556-Willis1" id="pone.0009556-Willis1" class="link-target"></a>Willis J, Robinson JE (1988) <em>Enterobacter sakazakii</em> meningitis in neonates. Pediatr Infect Dis J  7: 196–9.  <ul class="reflinks"><li><a href="#" data-author="Willis" data-cit="WillisJRobinsonJE1988Enterobacter%20sakazakii%20meningitis%20in%20neonates.Pediatr%20Infect%20Dis%20J71969" data-title="Enterobacter%20sakazakii%20meningitis%20in%20neonates." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii+meningitis+in+neonates.+Willis+1988" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0009556-Kline1" id="pone.0009556-Kline1" class="link-target"></a>Kline MW (1988) <em>Citrobacter</em> meningitis and brain abscess in infancy: Epidemiology, pathogenesis, and treatment. The Journal of Pediatrics  113: 430–434.  <ul class="reflinks"><li><a href="#" data-author="Kline" data-cit="KlineMW1988Citrobacter%20meningitis%20and%20brain%20abscess%20in%20infancy%3A%20Epidemiology%2C%20pathogenesis%2C%20and%20treatment.The%20Journal%20of%20Pediatrics113430434" data-title="Citrobacter%20meningitis%20and%20brain%20abscess%20in%20infancy%3A%20Epidemiology%2C%20pathogenesis%2C%20and%20treatment." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Citrobacter+meningitis+and+brain+abscess+in+infancy%3A+Epidemiology%2C+pathogenesis%2C+and+treatment.+Kline+1988" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0009556-Pagotto1" id="pone.0009556-Pagotto1" class="link-target"></a>Pagotto FJ, Nazarowec-White M, Bidawid S, Farber JM (2003) <em>Enterobacter sakazakii</em>: infectivity and enterotoxin production in vitro and in vivo. J Food Prot  66: 370–5.  <ul class="reflinks"><li><a href="#" data-author="Pagotto" data-cit="PagottoFJNazarowec-WhiteMBidawidSFarberJM2003Enterobacter%20sakazakii%3A%20infectivity%20and%20enterotoxin%20production%20in%20vitro%20and%20in%20vivo.J%20Food%20Prot663705" data-title="Enterobacter%20sakazakii%3A%20infectivity%20and%20enterotoxin%20production%20in%20vitro%20and%20in%20vivo." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Enterobacter+sakazakii%3A+infectivity+and+enterotoxin+production+in+vitro+and+in+vivo.+Pagotto+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0009556-Florea1" id="pone.0009556-Florea1" class="link-target"></a>Florea L, McClelland M, Riemer C, Schwartz S, Miller W (2003) EnteriX 2003: Visualization tools for genome alignments of <em>Enterobacteriaceae</em>. Nucleic Acids Res  31: 3527–32.  <ul class="reflinks"><li><a href="#" data-author="Florea" data-cit="FloreaLMcClellandMRiemerCSchwartzSMillerW2003EnteriX%202003%3A%20Visualization%20tools%20for%20genome%20alignments%20of%20Enterobacteriaceae.Nucleic%20Acids%20Res31352732" data-title="EnteriX%202003%3A%20Visualization%20tools%20for%20genome%20alignments%20of%20Enterobacteriaceae." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=EnteriX+2003%3A+Visualization+tools+for+genome+alignments+of+Enterobacteriaceae.+Florea+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0009556-Florea2" id="pone.0009556-Florea2" class="link-target"></a>Florea L, Riemer C, Schwartz S, Zhang Z, Stojanovic N, et al.  (2000) Web-based visualization tools for bacterial genome alignments. Nucleic Acids Res  28: 3486–96.  <ul class="reflinks"><li><a href="#" data-author="Florea" data-cit="FloreaLRiemerCSchwartzSZhangZStojanovicN2000Web-based%20visualization%20tools%20for%20bacterial%20genome%20alignments.Nucleic%20Acids%20Res28348696" data-title="Web-based%20visualization%20tools%20for%20bacterial%20genome%20alignments." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Web-based+visualization+tools+for+bacterial+genome+alignments.+Florea+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0009556-Kim2" id="pone.0009556-Kim2" class="link-target"></a>Kim CC, Joyce EA, Chan K, Falkow S (2002) Improved analytical methods for microarray-based genome-composition analysis. Genome Biol  3: RESEARCH0065.  <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimCCJoyceEAChanKFalkowS2002Improved%20analytical%20methods%20for%20microarray-based%20genome-composition%20analysis.Genome%20Biol3RESEARCH0065" data-title="Improved%20analytical%20methods%20for%20microarray-based%20genome-composition%20analysis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Improved+analytical+methods+for+microarray-based+genome-composition+analysis.+Kim+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0009556-Eisen1" id="pone.0009556-Eisen1" class="link-target"></a>Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and display of genome-wide expression patterns. Proc Natl Acad Sci U S A  95: 14863–8.  <ul class="reflinks"><li><a href="#" data-author="Eisen" data-cit="EisenMBSpellmanPTBrownPOBotsteinD1998Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns.Proc%20Natl%20Acad%20Sci%20U%20S%20A95148638" data-title="Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cluster+analysis+and+display+of+genome-wide+expression+patterns.+Eisen+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0009556-Huang1" id="pone.0009556-Huang1" class="link-target"></a>Huang SH, Wan ZS, Chen YH, Jong AY, Kim KS (2001) Further characterization of <em>Escherichia coli</em> brain microvascular endothelial cell invasion gene ibeA by deletion, complementation, and protein expression. J Infect Dis  183: 1071–8.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangSHWanZSChenYHJongAYKimKS2001Further%20characterization%20of%20Escherichia%20coli%20brain%20microvascular%20endothelial%20cell%20invasion%20gene%20ibeA%20by%20deletion%2C%20complementation%2C%20and%20protein%20expression.J%20Infect%20Dis18310718" data-title="Further%20characterization%20of%20Escherichia%20coli%20brain%20microvascular%20endothelial%20cell%20invasion%20gene%20ibeA%20by%20deletion%2C%20complementation%2C%20and%20protein%20expression." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Further+characterization+of+Escherichia+coli+brain+microvascular+endothelial+cell+invasion+gene+ibeA+by+deletion%2C+complementation%2C+and+protein+expression.+Huang+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0009556-Huang2" id="pone.0009556-Huang2" class="link-target"></a>Huang SH, Chen YH, Fu Q, Stins M, Wang Y, et al.  (1999) Identification and characterization of an <em>Escherichia coli</em> invasion gene locus, ibeB, required for penetration of brain microvascular endothelial cells. Infect Immun  67: 2103–9.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangSHChenYHFuQStinsMWangY1999Identification%20and%20characterization%20of%20an%20Escherichia%20coli%20invasion%20gene%20locus%2C%20ibeB%2C%20required%20for%20penetration%20of%20brain%20microvascular%20endothelial%20cells.Infect%20Immun6721039" data-title="Identification%20and%20characterization%20of%20an%20Escherichia%20coli%20invasion%20gene%20locus%2C%20ibeB%2C%20required%20for%20penetration%20of%20brain%20microvascular%20endothelial%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+characterization+of+an+Escherichia+coli+invasion+gene+locus%2C+ibeB%2C+required+for+penetration+of+brain+microvascular+endothelial+cells.+Huang+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0009556-Prasadarao1" id="pone.0009556-Prasadarao1" class="link-target"></a>Prasadarao NV, Wass CA, Weiser JN, Stins MF, Huang SH, et al.  (1996) Outer membrane protein A of <em>Escherichia coli</em> contributes to invasion of brain microvascular endothelial cells. Infect Immun  64: 146–53.  <ul class="reflinks"><li><a href="#" data-author="Prasadarao" data-cit="PrasadaraoNVWassCAWeiserJNStinsMFHuangSH1996Outer%20membrane%20protein%20A%20of%20Escherichia%20coli%20contributes%20to%20invasion%20of%20brain%20microvascular%20endothelial%20cells.Infect%20Immun6414653" data-title="Outer%20membrane%20protein%20A%20of%20Escherichia%20coli%20contributes%20to%20invasion%20of%20brain%20microvascular%20endothelial%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Outer+membrane+protein+A+of+Escherichia+coli+contributes+to+invasion+of+brain+microvascular+endothelial+cells.+Prasadarao+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0009556-Wang1" id="pone.0009556-Wang1" class="link-target"></a>Wang Y, Huang SH, Wass CA, Stins MF, Kim KS (1999) The gene locus yijP contributes to <em>Escherichia coli</em> K1 invasion of brain microvascular endothelial cells. Infect Immun  67: 4751–6.  <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangYHuangSHWassCAStinsMFKimKS1999The%20gene%20locus%20yijP%20contributes%20to%20Escherichia%20coli%20K1%20invasion%20of%20brain%20microvascular%20endothelial%20cells.Infect%20Immun6747516" data-title="The%20gene%20locus%20yijP%20contributes%20to%20Escherichia%20coli%20K1%20invasion%20of%20brain%20microvascular%20endothelial%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+gene+locus+yijP+contributes+to+Escherichia+coli+K1+invasion+of+brain+microvascular+endothelial+cells.+Wang+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0009556-Franke1" id="pone.0009556-Franke1" class="link-target"></a>Franke S, Grass G, Rensing C, Nies DH (2003) Molecular analysis of the copper-transporting efflux system CusCFBA of <em>Escherichia coli</em>. J Bacteriol  185: 3804–12.  <ul class="reflinks"><li><a href="#" data-author="Franke" data-cit="FrankeSGrassGRensingCNiesDH2003Molecular%20analysis%20of%20the%20copper-transporting%20efflux%20system%20CusCFBA%20of%20Escherichia%20coli.J%20Bacteriol185380412" data-title="Molecular%20analysis%20of%20the%20copper-transporting%20efflux%20system%20CusCFBA%20of%20Escherichia%20coli." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+analysis+of+the+copper-transporting+efflux+system+CusCFBA+of+Escherichia+coli.+Franke+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0009556-Osaili1" id="pone.0009556-Osaili1" class="link-target"></a>Osaili T, Forsythe S (2009) Desiccation resistance and persistence of <em>Cronobacter</em> species in infant formula. Int J Food Microbiol  136: 214–220.  <ul class="reflinks"><li><a href="#" data-author="Osaili" data-cit="OsailiTForsytheS2009Desiccation%20resistance%20and%20persistence%20of%20Cronobacter%20species%20in%20infant%20formula.Int%20J%20Food%20Microbiol136214220" data-title="Desiccation%20resistance%20and%20persistence%20of%20Cronobacter%20species%20in%20infant%20formula." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Desiccation+resistance+and+persistence+of+Cronobacter+species+in+infant+formula.+Osaili+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0009556-Kothary1" id="pone.0009556-Kothary1" class="link-target"></a>Kothary MH, McCardell BA, Frazar CD, Deer D, Tall BD (2007) Characterization of the zinc-containing metalloprotease encoded by zpx and development of a species-specific detection method for <em>Enterobacter sakazakii</em>. Appl Environ Microbiol  73: 4142–51.  <ul class="reflinks"><li><a href="#" data-author="Kothary" data-cit="KotharyMHMcCardellBAFrazarCDDeerDTallBD2007Characterization%20of%20the%20zinc-containing%20metalloprotease%20encoded%20by%20zpx%20and%20development%20of%20a%20species-specific%20detection%20method%20for%20Enterobacter%20sakazakii.Appl%20Environ%20Microbiol73414251" data-title="Characterization%20of%20the%20zinc-containing%20metalloprotease%20encoded%20by%20zpx%20and%20development%20of%20a%20species-specific%20detection%20method%20for%20Enterobacter%20sakazakii." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+the+zinc-containing+metalloprotease+encoded+by+zpx+and+development+of+a+species-specific+detection+method+for+Enterobacter+sakazakii.+Kothary+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0009556-Lehner1" id="pone.0009556-Lehner1" class="link-target"></a>Lehner A, Grimm M, Rattei T, Ruepp A, Frishman D, et al.  (2006) Cloning and characterization of <em>Enterobacter sakazakii</em> pigment genes and in situ spectroscopic analysis of the pigment. FEMS Microbiol Lett  265: 244–8.  <ul class="reflinks"><li><a href="#" data-author="Lehner" data-cit="LehnerAGrimmMRatteiTRueppAFrishmanD2006Cloning%20and%20characterization%20of%20Enterobacter%20sakazakii%20pigment%20genes%20and%20in%20situ%20spectroscopic%20analysis%20of%20the%20pigment.FEMS%20Microbiol%20Lett2652448" data-title="Cloning%20and%20characterization%20of%20Enterobacter%20sakazakii%20pigment%20genes%20and%20in%20situ%20spectroscopic%20analysis%20of%20the%20pigment." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cloning+and+characterization+of+Enterobacter+sakazakii+pigment+genes+and+in+situ+spectroscopic+analysis+of+the+pigment.+Lehner+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0009556-Campellone1" id="pone.0009556-Campellone1" class="link-target"></a>Campellone KG, Leong JM (2003) Tails of two Tirs: actin pedestal formation by enteropathogenic <em>E. coli</em> and enterohemorrhagic <em>E. coli</em> O157:H7. Curr Opin Microbiol  6: 82–90.  <ul class="reflinks"><li><a href="#" data-author="Campellone" data-cit="CampelloneKGLeongJM2003Tails%20of%20two%20Tirs%3A%20actin%20pedestal%20formation%20by%20enteropathogenic%20E.%20coli%20and%20enterohemorrhagic%20E.%20coli%20O157%3AH7.Curr%20Opin%20Microbiol68290" data-title="Tails%20of%20two%20Tirs%3A%20actin%20pedestal%20formation%20by%20enteropathogenic%20E.%20coli%20and%20enterohemorrhagic%20E.%20coli%20O157%3AH7." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tails+of+two+Tirs%3A+actin+pedestal+formation+by+enteropathogenic+E.+coli+and+enterohemorrhagic+E.+coli+O157%3AH7.+Campellone+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0009556-Nystrom1" id="pone.0009556-Nystrom1" class="link-target"></a>Nystrom T, Neidhardt FC (1994) Expression and role of the universal stress protein, UspA, of <em>Escherichia coli</em> during growth arrest. Mol Microbiol  11: 537–44.  <ul class="reflinks"><li><a href="#" data-author="Nystrom" data-cit="NystromTNeidhardtFC1994Expression%20and%20role%20of%20the%20universal%20stress%20protein%2C%20UspA%2C%20of%20Escherichia%20coli%20during%20growth%20arrest.Mol%20Microbiol1153744" data-title="Expression%20and%20role%20of%20the%20universal%20stress%20protein%2C%20UspA%2C%20of%20Escherichia%20coli%20during%20growth%20arrest." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+and+role+of+the+universal+stress+protein%2C+UspA%2C+of+Escherichia+coli+during+growth+arrest.+Nystrom+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0009556-Folkesson1" id="pone.0009556-Folkesson1" class="link-target"></a>Folkesson A, Lofdahl S, Normark S (2002) The <em>Salmonella enterica</em> subspecies I specific centisome 7 genomic island encodes novel protein families present in bacteria living in close contact with eukaryotic cells. Res Microbiol  153: 537–45.  <ul class="reflinks"><li><a href="#" data-author="Folkesson" data-cit="FolkessonALofdahlSNormarkS2002The%20Salmonella%20enterica%20subspecies%20I%20specific%20centisome%207%20genomic%20island%20encodes%20novel%20protein%20families%20present%20in%20bacteria%20living%20in%20close%20contact%20with%20eukaryotic%20cells.Res%20Microbiol15353745" data-title="The%20Salmonella%20enterica%20subspecies%20I%20specific%20centisome%207%20genomic%20island%20encodes%20novel%20protein%20families%20present%20in%20bacteria%20living%20in%20close%20contact%20with%20eukaryotic%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Salmonella+enterica+subspecies+I+specific+centisome+7+genomic+island+encodes+novel+protein+families+present+in+bacteria+living+in+close+contact+with+eukaryotic+cells.+Folkesson+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0009556-LimaMendez1" id="pone.0009556-LimaMendez1" class="link-target"></a>Lima-Mendez G, Van Helden J, Toussaint A, Leplae R (2008) Prophinder: a computational tool for prophage prediction in prokaryotic genomes. Bioinformatics  24: 863–5.  <ul class="reflinks"><li><a href="#" data-author="Lima-Mendez" data-cit="Lima-MendezGVan%20HeldenJToussaintALeplaeR2008Prophinder%3A%20a%20computational%20tool%20for%20prophage%20prediction%20in%20prokaryotic%20genomes.Bioinformatics248635" data-title="Prophinder%3A%20a%20computational%20tool%20for%20prophage%20prediction%20in%20prokaryotic%20genomes." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prophinder%3A+a+computational+tool+for+prophage+prediction+in+prokaryotic+genomes.+Lima-Mendez+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0009556-Lazdunski1" id="pone.0009556-Lazdunski1" class="link-target"></a>Lazdunski CJ (1988) Pore-forming colicins: synthesis, extracellular release, mode of action, immunity. Biochimie  70: 1291–6.  <ul class="reflinks"><li><a href="#" data-author="Lazdunski" data-cit="LazdunskiCJ1988Pore-forming%20colicins%3A%20synthesis%2C%20extracellular%20release%2C%20mode%20of%20action%2C%20immunity.Biochimie7012916" data-title="Pore-forming%20colicins%3A%20synthesis%2C%20extracellular%20release%2C%20mode%20of%20action%2C%20immunity." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Pore-forming+colicins%3A+synthesis%2C+extracellular+release%2C+mode+of+action%2C+immunity.+Lazdunski+1988" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0009556-Pickard1" id="pone.0009556-Pickard1" class="link-target"></a>Pickard D, Thomson NR, Baker S, Wain J, Pardo M, et al.  (2008) Molecular characterization of the <em>Salmonella enterica</em> serovar <em>Typhi</em> Vi-typing bacteriophage E1. J Bacteriol  190: 2580–7.  <ul class="reflinks"><li><a href="#" data-author="Pickard" data-cit="PickardDThomsonNRBakerSWainJPardoM2008Molecular%20characterization%20of%20the%20Salmonella%20enterica%20serovar%20Typhi%20Vi-typing%20bacteriophage%20E1.J%20Bacteriol19025807" data-title="Molecular%20characterization%20of%20the%20Salmonella%20enterica%20serovar%20Typhi%20Vi-typing%20bacteriophage%20E1." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+characterization+of+the+Salmonella+enterica+serovar+Typhi+Vi-typing+bacteriophage+E1.+Pickard+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0009556-Collyn1" id="pone.0009556-Collyn1" class="link-target"></a>Collyn F, Billault A, Mullet C, Simonet M, Marceau M (2004) YAPI, a new <em>Yersinia pseudotuberculosis</em> pathogenicity island. Infect Immun  72: 4784–90.  <ul class="reflinks"><li><a href="#" data-author="Collyn" data-cit="CollynFBillaultAMulletCSimonetMMarceauM2004YAPI%2C%20a%20new%20Yersinia%20pseudotuberculosis%20pathogenicity%20island.Infect%20Immun72478490" data-title="YAPI%2C%20a%20new%20Yersinia%20pseudotuberculosis%20pathogenicity%20island." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=YAPI%2C+a+new+Yersinia+pseudotuberculosis+pathogenicity+island.+Collyn+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0009556-Brussow1" id="pone.0009556-Brussow1" class="link-target"></a>Brussow H, Canchaya C, Hardt WD (2004) Phages and the evolution of bacterial pathogens: from genomic rearrangements to lysogenic conversion. Microbiol Mol Biol Rev  68: 560–602.  <ul class="reflinks"><li><a href="#" data-author="Brussow" data-cit="BrussowHCanchayaCHardtWD2004Phages%20and%20the%20evolution%20of%20bacterial%20pathogens%3A%20from%20genomic%20rearrangements%20to%20lysogenic%20conversion.Microbiol%20Mol%20Biol%20Rev68560602" data-title="Phages%20and%20the%20evolution%20of%20bacterial%20pathogens%3A%20from%20genomic%20rearrangements%20to%20lysogenic%20conversion." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phages+and+the+evolution+of+bacterial+pathogens%3A+from+genomic+rearrangements+to+lysogenic+conversion.+Brussow+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0009556-Boyd1" id="pone.0009556-Boyd1" class="link-target"></a>Boyd EF, Brussow H (2002) Common themes among bacteriophage-encoded virulence factors and diversity among the bacteriophages involved. Trends Microbiol  10: 521–9.  <ul class="reflinks"><li><a href="#" data-author="Boyd" data-cit="BoydEFBrussowH2002Common%20themes%20among%20bacteriophage-encoded%20virulence%20factors%20and%20diversity%20among%20the%20bacteriophages%20involved.Trends%20Microbiol105219" data-title="Common%20themes%20among%20bacteriophage-encoded%20virulence%20factors%20and%20diversity%20among%20the%20bacteriophages%20involved." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Common+themes+among+bacteriophage-encoded+virulence+factors+and+diversity+among+the+bacteriophages+involved.+Boyd+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0009556-Kaper1" id="pone.0009556-Kaper1" class="link-target"></a>Kaper JB (1998) The locus of enterocyte effacement pathogenicity island of Shiga toxin-producing <em>Escherichia coli</em> O157:H7 and other attaching and effacing <em>E. coli</em>. Jpn J Med Sci Biol  51: SupplS101–7.  <ul class="reflinks"><li><a href="#" data-author="Kaper" data-cit="KaperJB1998The%20locus%20of%20enterocyte%20effacement%20pathogenicity%20island%20of%20Shiga%20toxin-producing%20Escherichia%20coli%20O157%3AH7%20and%20other%20attaching%20and%20effacing%20E.%20coli.Jpn%20J%20Med%20Sci%20Biol51SupplS1017" data-title="The%20locus%20of%20enterocyte%20effacement%20pathogenicity%20island%20of%20Shiga%20toxin-producing%20Escherichia%20coli%20O157%3AH7%20and%20other%20attaching%20and%20effacing%20E.%20coli." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+locus+of+enterocyte+effacement+pathogenicity+island+of+Shiga+toxin-producing+Escherichia+coli+O157%3AH7+and+other+attaching+and+effacing+E.+coli.+Kaper+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0009556-Das1" id="pone.0009556-Das1" class="link-target"></a>Das S, Chakrabortty A, Banerjee R, Chaudhuri K (2002) Involvement of in vivo induced icmF gene of <em>Vibrio cholerae</em> in motility, adherence to epithelial cells, and conjugation frequency. Biochem Biophys Res Commun  295: 922–8.  <ul class="reflinks"><li><a href="#" data-author="Das" data-cit="DasSChakraborttyABanerjeeRChaudhuriK2002Involvement%20of%20in%20vivo%20induced%20icmF%20gene%20of%20Vibrio%20cholerae%20in%20motility%2C%20adherence%20to%20epithelial%20cells%2C%20and%20conjugation%20frequency.Biochem%20Biophys%20Res%20Commun2959228" data-title="Involvement%20of%20in%20vivo%20induced%20icmF%20gene%20of%20Vibrio%20cholerae%20in%20motility%2C%20adherence%20to%20epithelial%20cells%2C%20and%20conjugation%20frequency." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Involvement+of+in+vivo+induced+icmF+gene+of+Vibrio+cholerae+in+motility%2C+adherence+to+epithelial+cells%2C+and+conjugation+frequency.+Das+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0009556-Pukatzki1" id="pone.0009556-Pukatzki1" class="link-target"></a>Pukatzki S, Ma AT, Sturtevant D, Krastins B, Sarracino D, et al.  (2006) Identification of a conserved bacterial protein secretion system in <em>Vibrio cholerae</em> using the Dictyostelium host model system. Proc Natl Acad Sci U S A  103: 1528–33.  <ul class="reflinks"><li><a href="#" data-author="Pukatzki" data-cit="PukatzkiSMaATSturtevantDKrastinsBSarracinoD2006Identification%20of%20a%20conserved%20bacterial%20protein%20secretion%20system%20in%20Vibrio%20cholerae%20using%20the%20Dictyostelium%20host%20model%20system.Proc%20Natl%20Acad%20Sci%20U%20S%20A103152833" data-title="Identification%20of%20a%20conserved%20bacterial%20protein%20secretion%20system%20in%20Vibrio%20cholerae%20using%20the%20Dictyostelium%20host%20model%20system." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+a+conserved+bacterial+protein+secretion+system+in+Vibrio+cholerae+using+the+Dictyostelium+host+model+system.+Pukatzki+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0009556-Mullane1" id="pone.0009556-Mullane1" class="link-target"></a>Mullane N, O'Gaora P, Nally JE, Iversen C, Whyte P, et al.  (2008) Molecular analysis of the <em>Enterobacter sakazakii</em> O-antigen gene locus. Appl Environ Microbiol  74: 3783–94.  <ul class="reflinks"><li><a href="#" data-author="Mullane" data-cit="MullaneNO%27GaoraPNallyJEIversenCWhyteP2008Molecular%20analysis%20of%20the%20Enterobacter%20sakazakii%20O-antigen%20gene%20locus.Appl%20Environ%20Microbiol74378394" data-title="Molecular%20analysis%20of%20the%20Enterobacter%20sakazakii%20O-antigen%20gene%20locus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+analysis+of+the+Enterobacter+sakazakii+O-antigen+gene+locus.+Mullane+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0009556-Abraham1" id="pone.0009556-Abraham1" class="link-target"></a>Abraham SN, Jonsson AB, Normark S (1998) Fimbriae-mediated host-pathogen cross-talk. Curr Opin Microbiol  1: 75–81.  <ul class="reflinks"><li><a href="#" data-author="Abraham" data-cit="AbrahamSNJonssonABNormarkS1998Fimbriae-mediated%20host-pathogen%20cross-talk.Curr%20Opin%20Microbiol17581" data-title="Fimbriae-mediated%20host-pathogen%20cross-talk." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Fimbriae-mediated+host-pathogen+cross-talk.+Abraham+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0009556-Mougous1" id="pone.0009556-Mougous1" class="link-target"></a>Mougous JD, Senaratne RH, Petzold CJ, Jain M, Lee DH, et al.  (2006) A sulfated metabolite produced by stf3 negatively regulates the virulence of <em>Mycobacterium tuberculosis</em>. Proc Natl Acad Sci U S A  103: 4258–63.  <ul class="reflinks"><li><a href="#" data-author="Mougous" data-cit="MougousJDSenaratneRHPetzoldCJJainMLeeDH2006A%20sulfated%20metabolite%20produced%20by%20stf3%20negatively%20regulates%20the%20virulence%20of%20Mycobacterium%20tuberculosis.Proc%20Natl%20Acad%20Sci%20U%20S%20A103425863" data-title="A%20sulfated%20metabolite%20produced%20by%20stf3%20negatively%20regulates%20the%20virulence%20of%20Mycobacterium%20tuberculosis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+sulfated+metabolite+produced+by+stf3+negatively+regulates+the+virulence+of+Mycobacterium+tuberculosis.+Mougous+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0009556-Schlieker1" id="pone.0009556-Schlieker1" class="link-target"></a>Schlieker C, Zentgraf H, Dersch P, Mogk A (2005) ClpV, a unique Hsp100/Clp member of pathogenic proteobacteria. Biol Chem  386: 1115–27.  <ul class="reflinks"><li><a href="#" data-author="Schlieker" data-cit="SchliekerCZentgrafHDerschPMogkA2005ClpV%2C%20a%20unique%20Hsp100%2FClp%20member%20of%20pathogenic%20proteobacteria.Biol%20Chem386111527" data-title="ClpV%2C%20a%20unique%20Hsp100%2FClp%20member%20of%20pathogenic%20proteobacteria." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=ClpV%2C+a+unique+Hsp100%2FClp+member+of+pathogenic+proteobacteria.+Schlieker+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0009556-MacLean1" id="pone.0009556-MacLean1" class="link-target"></a>MacLean LL, Pagotto F, Farber JM, Perry MB (2009) Structure of the antigenic repeating pentasaccharide unit of the LPS O-polysaccharide of <em>Cronobacter sakazakii</em> implicated in the Tennessee outbreak. Biochem Cell Biol  87: 459–465.  <ul class="reflinks"><li><a href="#" data-author="MacLean" data-cit="MacLeanLLPagottoFFarberJMPerryMB2009Structure%20of%20the%20antigenic%20repeating%20pentasaccharide%20unit%20of%20the%20LPS%20O-polysaccharide%20of%20Cronobacter%20sakazakii%20implicated%20in%20the%20Tennessee%20outbreak.Biochem%20Cell%20Biol87459465" data-title="Structure%20of%20the%20antigenic%20repeating%20pentasaccharide%20unit%20of%20the%20LPS%20O-polysaccharide%20of%20Cronobacter%20sakazakii%20implicated%20in%20the%20Tennessee%20outbreak." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Structure+of+the+antigenic+repeating+pentasaccharide+unit+of+the+LPS+O-polysaccharide+of+Cronobacter+sakazakii+implicated+in+the+Tennessee+outbreak.+MacLean+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0009556-Czerwicka1" id="pone.0009556-Czerwicka1" class="link-target"></a>Czerwicka M, Forsythe SJ, Bychowska A, Dziadziuszko H, Kunikowska D, et al.  (2010) Chemical structure of the O-polysaccharide isolated from <em>Cronobacter sakazakii</em> 767. Carbohydrate Research. In Press.   <ul class="reflinks"><li><a href="#" data-author="Czerwicka" data-cit="CzerwickaMForsytheSJBychowskaADziadziuszkoHKunikowskaD2010Chemical%20structure%20of%20the%20O-polysaccharide%20isolated%20from%20Cronobacter%20sakazakii%20767.Carbohydrate%20Research.%20In%20Press" data-title="Chemical%20structure%20of%20the%20O-polysaccharide%20isolated%20from%20Cronobacter%20sakazakii%20767." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Chemical+structure+of+the+O-polysaccharide+isolated+from+Cronobacter+sakazakii+767.+Czerwicka+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0009556-MacLean2" id="pone.0009556-MacLean2" class="link-target"></a>MacLean LL, Vinogradov E, Pagotto F, Farber JM, Perry MB (2009) Characterization of the O-antigen in the lipopolysaccharide of <em>Cronobacter (Enterobacter) malonaticus</em> 3267. Biochem Cell Biol  87: 927–932.  <ul class="reflinks"><li><a href="#" data-author="MacLean" data-cit="MacLeanLLVinogradovEPagottoFFarberJMPerryMB2009Characterization%20of%20the%20O-antigen%20in%20the%20lipopolysaccharide%20of%20Cronobacter%20%28Enterobacter%29%20malonaticus%203267.Biochem%20Cell%20Biol87927932" data-title="Characterization%20of%20the%20O-antigen%20in%20the%20lipopolysaccharide%20of%20Cronobacter%20%28Enterobacter%29%20malonaticus%203267." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+the+O-antigen+in+the+lipopolysaccharide+of+Cronobacter+%28Enterobacter%29+malonaticus+3267.+MacLean+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0009556-MacLean3" id="pone.0009556-MacLean3" class="link-target"></a>MacLean LL, Pagotto F, Farber J M, Perry MB (2009) The structure of the O-antigen in the endotoxin of the emerging food pathogen <em>Cronobacter</em> (<em>Enterobacter</em>) <em>muytjensii</em> strain 3270. Carbohydr Res  344: 667–671.  <ul class="reflinks"><li><a href="#" data-author="MacLean" data-cit="MacLeanLLPagottoFFarber%20JMPerryMB2009The%20structure%20of%20the%20O-antigen%20in%20the%20endotoxin%20of%20the%20emerging%20food%20pathogen%20Cronobacter%20%28Enterobacter%29%20muytjensii%20strain%203270.Carbohydr%20Res344667671" data-title="The%20structure%20of%20the%20O-antigen%20in%20the%20endotoxin%20of%20the%20emerging%20food%20pathogen%20Cronobacter%20%28Enterobacter%29%20muytjensii%20strain%203270." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+structure+of+the+O-antigen+in+the+endotoxin+of+the+emerging+food+pathogen+Cronobacter+%28Enterobacter%29+muytjensii+strain+3270.+MacLean+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0009556-Porwollik1" id="pone.0009556-Porwollik1" class="link-target"></a>Porwollik S, Wong RM, McClelland M (2002) Evolutionary genomics of <em>Salmonella</em>: gene acquisitions revealed by microarray analysis. Proc Natl Acad Sci U S A  99: 8956–61.  <ul class="reflinks"><li><a href="#" data-author="Porwollik" data-cit="PorwollikSWongRMMcClellandM2002Evolutionary%20genomics%20of%20Salmonella%3A%20gene%20acquisitions%20revealed%20by%20microarray%20analysis.Proc%20Natl%20Acad%20Sci%20U%20S%20A99895661" data-title="Evolutionary%20genomics%20of%20Salmonella%3A%20gene%20acquisitions%20revealed%20by%20microarray%20analysis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Evolutionary+genomics+of+Salmonella%3A+gene+acquisitions+revealed+by+microarray+analysis.+Porwollik+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0009556-Huang3" id="pone.0009556-Huang3" class="link-target"></a>Huang X, Wang J, Aluru S, Yang SP, Hillier L (2003) PCAP: a whole-genome assembly program. Genome Res  13: 2164–70.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangXWangJAluruSYangSPHillierL2003PCAP%3A%20a%20whole-genome%20assembly%20program.Genome%20Res13216470" data-title="PCAP%3A%20a%20whole-genome%20assembly%20program." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=PCAP%3A+a+whole-genome+assembly+program.+Huang+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0009556-Gordon1" id="pone.0009556-Gordon1" class="link-target"></a>Gordon D, Desmarais C, Green P (2001) Automated finishing with autofinish. Genome Res  11: 614–25.  <ul class="reflinks"><li><a href="#" data-author="Gordon" data-cit="GordonDDesmaraisCGreenP2001Automated%20finishing%20with%20autofinish.Genome%20Res1161425" data-title="Automated%20finishing%20with%20autofinish." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Automated+finishing+with+autofinish.+Gordon+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pone.0009556-Cleveland1" id="pone.0009556-Cleveland1" class="link-target"></a>Cleveland WS (1979) Robust locally weighted regression and smoothing scatterplots. Journal of the American Statistical Association  74: 829–836.  <ul class="reflinks"><li><a href="#" data-author="Cleveland" data-cit="ClevelandWS1979Robust%20locally%20weighted%20regression%20and%20smoothing%20scatterplots.Journal%20of%20the%20American%20Statistical%20Association74829836" data-title="Robust%20locally%20weighted%20regression%20and%20smoothing%20scatterplots." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Robust+locally+weighted+regression+and+smoothing+scatterplots.+Cleveland+1979" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pone.0009556-Workman1" id="pone.0009556-Workman1" class="link-target"></a>Workman C, Jensen LJ, Jarmer H, Berka R, Gautier L, et al.  (2002) A new non-linear normalization method for reducing variability in DNA microarray experiments. Genome Biol vol  3: research0048.1–0048.16.  <ul class="reflinks"><li><a href="#" data-author="Workman" data-cit="WorkmanCJensenLJJarmerHBerkaRGautierL2002A%20new%20non-linear%20normalization%20method%20for%20reducing%20variability%20in%20DNA%20microarray%20experiments.Genome%20Biol%20vol3research0048.10048.16" data-title="A%20new%20non-linear%20normalization%20method%20for%20reducing%20variability%20in%20DNA%20microarray%20experiments." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+new+non-linear+normalization+method+for+reducing+variability+in+DNA+microarray+experiments.+Workman+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pone.0009556-Xia1" id="pone.0009556-Xia1" class="link-target"></a>Xia XQ, McClelland M, Porwollik S, Song W, Cong X, et al.  (2009) WebArrayDB: cross-platform microarray data analysis and public data repository. Bioinformatics  25: 2425–9.  <ul class="reflinks"><li><a href="#" data-author="Xia" data-cit="XiaXQMcClellandMPorwollikSSongWCongX2009WebArrayDB%3A%20cross-platform%20microarray%20data%20analysis%20and%20public%20data%20repository.Bioinformatics2524259" data-title="WebArrayDB%3A%20cross-platform%20microarray%20data%20analysis%20and%20public%20data%20repository." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=WebArrayDB%3A+cross-platform+microarray+data+analysis+and+public+data+repository.+Xia+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pone.0009556-Kado1" id="pone.0009556-Kado1" class="link-target"></a>Kado CI, Liu ST (1981) Rapid procedure for detection and isolation of large and small plasmids. J Bacteriol  145: 1365–1373.  <ul class="reflinks"><li><a href="#" data-author="Kado" data-cit="KadoCILiuST1981Rapid%20procedure%20for%20detection%20and%20isolation%20of%20large%20and%20small%20plasmids.J%20Bacteriol14513651373" data-title="Rapid%20procedure%20for%20detection%20and%20isolation%20of%20large%20and%20small%20plasmids." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Rapid+procedure+for+detection+and+isolation+of+large+and+small+plasmids.+Kado+1981" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

