



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>This work aimed to elucidate the distribution of <em>Chrysoperla externa</em> haplotypes and investigate whether it exhibits structure based on genetic composition as opposed to geographic location. The genetic diversity of <em>C</em>. <em>externa</em>, analyzed by AMOVA using the <em>COI</em> and <em>16S</em> rRNA genes as mitochondrial markers, showed significant haplotype structure arising from genetic differences that was not associated with sampling location. This was reflected in the network grouping. Bayesian inference showed that haplotype distribution may have its origins in <em>C</em>. <em>externa</em> divergence into two distinct clades, which dispersed to various locations, and their subsequent diversification. The evolutionary history of <em>C</em>. <em>externa</em> may include multiple ancestral haplotypes differentiating within the same geographic area to generate the current broad genetic diversity, so that the earlier geographical history has been erased, and now we have highlighted its more recent genetic history.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Morales-Corrêa e Castro AC, Barbosa NCCP (2017) Recent evolutionary history of <em>Chrysoperla externa</em> (Hagen 1861) (Neuroptera: Chrysopidae) in Brazil. PLoS ONE 12(5):
           e0177414.
        
        https://doi.org/10.1371/journal.pone.0177414</p><p><strong>Editor: </strong>Tzen-Yuh Chiang, National Cheng Kung University, TAIWAN</p><p><strong>Received: </strong>November 2, 2016; <strong>Accepted: </strong>April 26, 2017; <strong>Published: </strong> May 16, 2017</p><p><strong>Copyright: </strong> © 2017 Morales-Corrêa e Castro, Barbosa. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>All relevant data are within the paper and its Supporting Information files.</p><p><strong>Funding: </strong>ACMCC received support from Conselho Nacional de Desenvolvimento Científico e Tecnológico—Programa Taxonomia (150150/2007-5). NCCPB received support from Conselho Nacional de Desenvolvimento Científico e Tecnológico—(grant to master degree).</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Green lacewings are insects of the Chrysopidae, a family of 1,413 species and subspecies distributed among 82 genera [<a href="#pone.0177414.ref001" class="ref-tip">1</a>]. <em>Chrysoperla</em> Steinmann 1964 comprises 36 lacewing species, four of which are found in Brazil: <em>Chrysoperla defreitasi</em> Brooks 1994, <em>Chrysoperla raimundoi</em> Freitas &amp; Penny 2001, <em>Chrysoperla genanigra</em> Freitas 2003, and <em>Chrysoperla externa</em> (Hagen 1861). The last named is one of the most common lacewing species in the Americas and can be found from the southern USA to Argentina [<a href="#pone.0177414.ref002" class="ref-tip">2</a>].</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Studies of the biology of <em>C</em>. <em>externa</em> and its use as a biological control agent have been published since the 1970s [<a href="#pone.0177414.ref003" class="ref-tip">3</a>]. The genetic variation and degree of population structure of <em>C</em>. <em>externa</em> began to be studied only in 2007, culminating in publication of a study conducted in different agrosystems and seasons using mitochondrial DNA as a molecular marker in specimens from Jaboticabal, São Paulo (SP), Brazil [<a href="#pone.0177414.ref004" class="ref-tip">4</a>]. The specimens of <em>C</em>. <em>externa</em> showed high haplotype diversity (0.956) with 24 haplotypes in 36 specimens analyzed, and no population structure. The study of population structure of this species in São Paulo State was the main, but not sole, focus of later research. The genetic variability and genetic relationships among populations were estimated by genetic markers <em>ISSR</em> and the <em>COI</em> gene in 90 specimens from 12 populations in SP. High levels of diversity were observed for both markers, with the highest values found in municipalities having the largest areas of native vegetation [<a href="#pone.0177414.ref005" class="ref-tip">5</a>]. A further 12 populations and 114 specimens were studied in SP, revealing high genetic diversity and lack of geographic structure, suggesting that <em>C</em>. <em>externa</em> is a single population, regardless of sampling location [<a href="#pone.0177414.ref006" class="ref-tip">6</a>]. The most comprehensive study of <em>C</em>. <em>externa</em> distribution compared the variability and degree of structure in populations from areas of native vegetation with those from agrosystems, analyzing 122 specimens and 12 previously unstudied locations, primarily Brazilian states. The results showed population expansion in agro-ecosystems, whereas populations in the native environment remained stable over time. Analysis showed significant genetic structure among the sampled groups, which, combined with its within-group absence, corroborated group identities [<a href="#pone.0177414.ref007" class="ref-tip">7</a>].</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Although the studies carried out to date have analyzed a sufficient number of specimens for population studies of <em>C</em>. <em>externa</em>, different groups were sampled, making it difficult to compare parameters and gain a comprehensive understanding of the species evolution. Research using the mitochondrial marker <em>16S rRNA</em> gene, as proposed in this study, provides information on population history on a wider time scale than do the <em>COI</em> gene and <em>ISSR</em> markers previously used [<a href="#pone.0177414.ref004" class="ref-tip">4</a>–<a href="#pone.0177414.ref007" class="ref-tip">7</a>].</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>The goal of this study was to reveal the genetic structure of <em>C</em>. <em>externa</em>, combining, for the first time, data on all specimens analyzed for the <em>COI</em> gene in different locations and data of specimens from locations not previously analyzed, along with analysis of the <em>16S rRNA</em> gene, to investigate whether populations of <em>C</em>. <em>externa</em> exhibits differentiation based on genetic composition or on the geographic source of the specimens.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Materials and methods"></a><h2>Materials and methods</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Biological materials"></a>
<h3>Biological materials</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>This study gathers new specimens of <em>C</em>. <em>externa</em> and analyzed in previous papers by our group [4, 5 and 7]. We sampled 123 localities in Brazil in the states of São Paulo (110), Minas Gerais (7), Paraná (3), Goiás (1), Rio Grande do Norte (1) and Distrito Federal (1) and at a location in Paraguay; all the localities are georeferenced (<a href="#pone.0177414.s004">S1 Table</a>). Collections were performed from 2005 to 2010 with permission issued by the agency responsible for environmental studies in Brazil—ICMBio-IBAMA (Chico Mendes Institute for Biodiversity Conservation of the Brazilian Institute of the Environment). All the specimens were collected in open areas and on agrosystems borders. No collecting activities occurred in protected areas, natural reserves or state/federal parks, nor did it involve endangered or protected species, as shown in the table of localities (<a href="#pone.0177414.s004">S1 Table</a>). The specimens were stored in absolute ethanol and identified based on external morphological characteristics according to Brooks and Barnard [<a href="#pone.0177414.ref008" class="ref-tip">8</a>] by Dr. Sérgio de Freitas. Currently, the vouchers specimens of this work, together with the other individual specimens collected in the expeditions are deposited in the Coleção de Referência Entomológica (Collection of Entomological Reference) do Departamento de Fitossanidade of the Faculdade de Ciências Agrárias e Veterinárias (FCAV-UNESP), as part of the Neuroptera Collection and open for scientific consultation.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="DNA extraction"></a>
<h3>DNA extraction</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>DNA was extracted from the thorax, while the head, wings, and abdomen were stored in absolute ethanol also in the Coleção de Referência Entomológica at the Departamento de Fitossanidade of the FCAV-UNESP (<a href="#pone.0177414.s004">S1 Table</a>). The DNA was extracted using the Wizard<sup>®</sup> Genomic DNA Purification Kit (Promega), following the manufacturer’s protocol and it is stored at the Laboratório de Biologia Evolutiva (LaBE) do Departamento de Biologia Aplicada à Agropecuária—FCAV-UNESP.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Amplification and sequencing"></a>
<h3>Amplification and sequencing</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>The mitochondrial genes <em>COI</em> and <em>16S rRNA</em> were amplified by PCR in a Mastercycler thermal cycler (Eppendorf). The reaction was conducted in a final volume of 25 μL consisting of 12.5 μL of GoTaq<sup>®</sup> Colorless Master Mix (Promega), 0.4 μM of each of the primers, and 2.5 μL total DNA (~40 ng). The <em>COI</em> gene was amplified with primers C1-J-2183 (<span class="monospace">5’CAACATTTATTTTGATTTTTTGG3’</span>) and TL2-N-3014 (<span class="monospace">5’TCCATTGCACTAATCTGCCATATTA3’</span>) [<a href="#pone.0177414.ref009" class="ref-tip">9</a>] under the following conditions: initial denaturation at 94°C for 2 minutes followed by 35 cycles of 40 seconds at 94°C, 50 seconds at 55°C, and 1 minute at 72°C; with a final extension phase of 72°C for 10 minutes. The <em>16S</em> gene was amplified with primers LR-J-12887 (<span class="monospace">5’ CCGGTTTGAACTCAGATCATGT 3’</span>) and SR-N-13398b (<span class="monospace">5’ CRCYTGTTTAWCAAAAACAT 3’</span>) [<a href="#pone.0177414.ref009" class="ref-tip">9</a>] under conditions as follows: initial denaturation at 95°C for 5 minutes followed by 33 cycles of 20 seconds at 93°C, 40 seconds at 50°C, and 2 minutes at 72°C; with a final extension phase of 72°C for 10 minutes. The PCR products were subjected to electrophoresis in 1% agarose gel and stained with ethidium bromide (1ng/mL) to confirm amplification. Each PCR product was purified using the Wizard<sup>®</sup> SV Gel and PCR Clean-Up System (Promega) according to the manufacturer’s protocol. The products were sequenced using the same primers and amplification conditions. The sequencing reactions were conducted in an ABI 3730 XL DNA Analyzer (Applied Biosystems, Foster City, California, CA) automatic sequencer using the BigDye<sup>®</sup> Terminator v.3.1 Cycle Sequencing Kit (Perkin-Elmer Applied Biosystems), and the sequences were deposited in the Laboratório de Biologia Evolutiva (LaBE) database at the Departamento de Biologia Aplicada à Agropecuária, FCAV. The haplotypes were deposited in the GenBank database under the accession numbers KJ586656 to KJ586673 and KX099401 to KX099607 (<a href="#pone.0177414.s005">S2</a> and <a href="#pone.0177414.s006">S3</a> Tables).</p>
</div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Data analysis"></a>
<h3>Data analysis</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>The sequences were read in Chromas Lite v.2.1.1 [<a href="#pone.0177414.ref010" class="ref-tip">10</a>] and aligned with Muscle Tool Web Services [<a href="#pone.0177414.ref011" class="ref-tip">11</a>]. Descriptive analyses were performed using DnaSP v.5.10.01 [<a href="#pone.0177414.ref012" class="ref-tip">12</a>], and the number of polymorphic sites (S), number of haplotypes (h), haplotype diversity (Hd), nucleotide diversity (π), and average number of nucleotide differences (k) were obtained. Nucleotide composition was calculated by MEGA v.6.06 [<a href="#pone.0177414.ref013" class="ref-tip">13</a>]. The Analysis of Molecular Variance (AMOVA) was applied using Arlequin v.3.5.1.3 [<a href="#pone.0177414.ref014" class="ref-tip">14</a>]. A haplotype network was built using TCS v.1.21 [<a href="#pone.0177414.ref015" class="ref-tip">15</a>]. The presence of population structure was tested for the genes separately using a Bayesian approach with Structure v.2.3.4 [<a href="#pone.0177414.ref016" class="ref-tip">16</a>–<a href="#pone.0177414.ref019" class="ref-tip">19</a>]. Several runs were realized without replicates to estimate the probable K value (clusters) based on the ln(Pr(X|K) value. Twenty-five independent runs were carried out for each K value, ranging from 1 to 8 for the <em>COI</em> gene and 1 to 5 for the <em>16S</em> gene. For each run, 10,000 iterations were carried out after a burn-in of 10,000 iterations. The assumed parameters for both genes were "no model admixture" because they were haploid mitochondrial genes, and "all frequencies correlated" since the sequences had high similarity [<a href="#pone.0177414.ref017" class="ref-tip">17</a>]. To detect the number of genetically homogeneous groups (K) that best fit the data, we used Evanno implemented [<a href="#pone.0177414.ref020" class="ref-tip">20</a>] in the Structure Harvester website [<a href="#pone.0177414.ref021" class="ref-tip">21</a>]. A graphic representation of the results obtained by the Structure program was created in CLUMPAK beta version [<a href="#pone.0177414.ref022" class="ref-tip">22</a>]. Tajima’s D [<a href="#pone.0177414.ref023" class="ref-tip">23</a>] and Fu’s Fs [<a href="#pone.0177414.ref024" class="ref-tip">24</a>] neutrality tests were performed with concatenated data by DnaSP v.5.10.01 [<a href="#pone.0177414.ref012" class="ref-tip">12</a>], to determine if populations followed a neutral model of evolution with constant population size over time. The saturation of the data was assessed with the DAMBE v.6.0.48 [<a href="#pone.0177414.ref025" class="ref-tip">25</a>] using the Test of substitution saturation by Xia et al. [<a href="#pone.0177414.ref026" class="ref-tip">26</a>, <a href="#pone.0177414.ref027" class="ref-tip">27</a>]. The phylogenetic reconstruction was performed by Bayesian inference using the package BEAUti and BEAST v.1.8.2 [<a href="#pone.0177414.ref028" class="ref-tip">28</a>] with concatenated data. The substitution model most appropriate for the <em>COI</em> gene was GTR+I+G and for <em>16S</em> gene was HKY+I, obtained using JModelTest v.2.1.3, utilizing the Bayesian Information Criterion [<a href="#pone.0177414.ref029" class="ref-tip">29</a>]. We performed 150 million MCMC simulations and sampled every 1500 steps, under the lognormal relaxed clock model and a constant-size coalescent tree prior. The species of Chrysopidae, <em>Chrysoperla defreitase</em> (<em>COI</em>: KX099404; <em>16S</em>: KX099549), <em>Chrysopodes divisa</em> (<em>COI</em>: KX099403; <em>16S</em>: KX099548), <em>Ceraeochrysa cubana</em> (<em>COI</em>: KX099402; <em>16S</em>: KX099547), <em>Leucochrysa cruentata</em> (<em>COI</em>: KX099401; <em>16S</em>: KX099546), and Hemerobiidae, <em>Neuronema laminatum</em> (NC_028153.1) were used as outgroups. The priors were checked in Tracer v.1.6 [<a href="#pone.0177414.ref030" class="ref-tip">30</a>] and the maximum clade credibility tree was generated using Tree Annotator v.1.8.2 [<a href="#pone.0177414.ref031" class="ref-tip">31</a>], considering the burn-in value to equal 1% of the total trees generated in BEAST. This was viewed and edited in FigTree v.1.4.2 [<a href="#pone.0177414.ref032" class="ref-tip">32</a>].</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec007" name="sec007" data-toc="sec007" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="Descriptive statistics"></a>
<h3>Descriptive statistics</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>We obtained 477 sequences of 730 bp each for the <em>COI</em> gene and 158 haplotypes (<a href="#pone.0177414.s005">S2 Table</a>), and 466 sequences of 488 bp each for the <em>16S</em> gene and 58 haplotypes (<a href="#pone.0177414.s006">S3 Table</a>). The concatenated data generated 387 sequences of 1,218 bp each, which presented an average nucleotide composition of 40.3% thymine (T), 14.5% cytosine (C), 32.8% adenine (A), and 12.4% guanine (G). One hundred fifty-eight polymorphic sites (S) were obtained, resulting in 193 haplotypes (h) (<a href="#pone.0177414.s007">S4 Table</a>) and an average haplotype diversity (Hd) of 0.9627. The average number of nucleotide differences (k) and average nucleotide diversity (π) were 3.321 and 0.00274, respectively.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Population structure"></a>
<h3>Population structure</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>The genetic structure was used to investigate the species <em>C</em>. <em>externa</em> in a geographic and genetic context. Initially, AMOVA was used to test the structure by locality, considering all populations as a single group. For this grouping, the p-value was not significant for either gene (<em>COI</em>: p = 0.69404±0.01436; <em>16S</em>: p = 0.56794±0.01391), showing that there was not sufficient evidence to prove that there is genetic structure among geographic populations.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>The haplotype network for both genes showed well-defined groups. The <em>COI</em> gene comprised four groups (<a href="#pone-0177414-g001">Fig 1</a>), and the 16S gene exhibited five groups (<a href="#pone-0177414-g002">Fig 2</a>). All groups contained specimens from different localities; hence grouping based on geographic distribution was not observed (<a href="#pone.0177414.s004">S1</a>–<a href="#pone.0177414.s006">S3</a> Tables). The distribution of haplotypes was coincident in Group 4, in which the specimens were isolated from the other groups with respect to both genes (<a href="#pone.0177414.s001">S1 Fig</a> and <a href="#pone.0177414.s002">S2 Fig</a>). All specimens in Group 3 of the <em>16S</em> gene were also in Group 1 of the <em>COI</em> gene. The specimens were not coincident in other groups; specimens belonging to a group in the <em>COI</em> gene appeared in different groups in the <em>16S</em> gene, and vice-versa (<a href="#pone.0177414.s001">S1 Fig</a> and <a href="#pone.0177414.s002">S2 Fig</a>).</p>
<a class="link-target" id="pone-0177414-g001" name="pone-0177414-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0177414.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0177414.g001" data-doi="info:doi/10.1371/journal.pone.0177414" data-uri="info:doi/10.1371/journal.pone.0177414.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0177414.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Haplotype network for the <em>COI</em> gene of <em>Chrysoperla externa</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>Square represent the ancestral haplotypes. The size of circle reflects the frequency of the haplotypes in the sample. Solid lines correspond to a mutational change connecting two haplotypes with a probability &gt;95%. Small black dots denote missing intermediate haplotypes.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0177414.g001">
              https://doi.org/10.1371/journal.pone.0177414.g001</a></p></div><a class="link-target" id="pone-0177414-g002" name="pone-0177414-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0177414.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0177414.g002" data-doi="info:doi/10.1371/journal.pone.0177414" data-uri="info:doi/10.1371/journal.pone.0177414.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0177414.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Haplotype network for the <em>16S</em> gene of <em>Chrysoperla externa</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1" class="link-target"></a><p>Square represents the ancestral haplotype. The size of circle reflects the frequency of the haplotypes in the sample. Solid lines correspond to a mutational change connecting two haplotypes with a probability &gt;95%. Small black circles denote missing intermediate haplotypes.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0177414.g002">
              https://doi.org/10.1371/journal.pone.0177414.g002</a></p></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3" class="link-target"></a><p>Posteriorly, AMOVA was used to test the structure based on groups defined by the parsimony analysis in TCS. For this grouping, the p-value was significant for both genes (p = 0.00000 ± 0.00000 for both), and the F<sub>ST</sub> presented high values (<em>COI</em>: F<sub>ST</sub> = 0.55559; <em>16S</em>: F<sub>ST</sub> = 0.75517). The population pairwise F<sub>ST</sub>s also presented high values, demonstrating that Group 4 did not skew this structure (<a href="#pone-0177414-t001">Table 1</a>).</p>
<a class="link-target" id="pone-0177414-t001" name="pone-0177414-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0177414.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0177414.t001" data-doi="info:doi/10.1371/journal.pone.0177414" data-uri="info:doi/10.1371/journal.pone.0177414.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0177414.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Pairwise F<sub>ST</sub>s for grouping based on haplotype network constructed by using the statistical parsimony method of <em>Chrysoperla externa</em>.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0177414.t001">
              https://doi.org/10.1371/journal.pone.0177414.t001</a></p></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4" class="link-target"></a><p>Structure analysis was used to visualize the genetic similarity among individuals and test the presence of population structure. For the <em>COI</em> gene, individuals were grouped into six clusters (Figs <a href="#pone-0177414-g001">1</a> and <a href="#pone-0177414-g003">3</a>), which were interpreted as mutational steps occurring from an ancestral haplotype not present in the analysis. Individuals of the Groups 1 and 3 presented similar proportions of the coloured segments that represent the individual’s probability of belonging to the cluster with that colour. Group 2 presented the same clusters as Groups 1 and 3, but in different probability. Group 4 appeared separately, similar to results of TCS, indicating genetic composition distinct from other groups. The <em>16S</em> gene was separated into two clusters, with Groups 1, 3, and 4 in a cluster and Groups 2 and 5 in another (Figs <a href="#pone-0177414-g002">2</a> and <a href="#pone-0177414-g004">4</a>).</p>
<a class="link-target" id="pone-0177414-g003" name="pone-0177414-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0177414.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0177414.g003" data-doi="info:doi/10.1371/journal.pone.0177414" data-uri="info:doi/10.1371/journal.pone.0177414.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0177414.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Population structure for grouping based on haplotype network constructed by using the statistical parsimony method for <em>COI</em> gene of <em>Chrysoperla externa</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1" class="link-target"></a><p>Each individual is represented by a thin vertical line, which is partitioned into coloured segments that represent the individual’s probability of belonging to the cluster with that colour. Colors represent clusters (K), interpreted as mutational steps from an ancestral haplotype not present in the analysis.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0177414.g003">
              https://doi.org/10.1371/journal.pone.0177414.g003</a></p></div><a class="link-target" id="pone-0177414-g004" name="pone-0177414-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0177414.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0177414.g004" data-doi="info:doi/10.1371/journal.pone.0177414" data-uri="info:doi/10.1371/journal.pone.0177414.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0177414.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Population structure for grouping based on haplotype network constructed by using the statistical parsimony method for <em>16S</em> gene of <em>Chrysoperla externa</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig4.caption1.p1" name="article1.body1.sec3.sec2.fig4.caption1.p1" class="link-target"></a><p>Each individual is represented by a thin vertical line, which is partitioned into coloured segments that represent the individual’s probability of belonging to the cluster with that colour. Colors represent clusters (K), interpreted as mutational steps from an ancestral haplotype not present in the analysis.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0177414.g004">
              https://doi.org/10.1371/journal.pone.0177414.g004</a></p></div></div>

<div id="section3" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Bayesian analysis and neutrality test"></a>
<h3>Bayesian analysis and neutrality test</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>Bayesian inference analysis was performed to obtain the probable relationship between haplotypes of <em>C</em>. <em>externa</em>. The saturation test showed little saturation, indicating that the date was appropriate for phylogenetic analysis (Iss&lt;ISSc; p &lt; 0.05). <em>Chrysoperla externa</em> split into two distinct clades with 100% posterior probability, one comprised of individuals appearing only in the TCS Group 4 and a second clade including individuals of other groups (<a href="#pone-0177414-g005">Fig 5</a>). The clade comprising Groups 1, 2, 3, and 5 showed internal clades that were not monophyletic, while the Group 4 clade was monophyletic and showed more recent divergence.</p>
<a class="link-target" id="pone-0177414-g005" name="pone-0177414-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0177414.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0177414.g005" data-doi="info:doi/10.1371/journal.pone.0177414" data-uri="info:doi/10.1371/journal.pone.0177414.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0177414.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0177414.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0177414.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0177414.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Bayesian inference analysis for the concatenated genes of <em>Chrysoperla externa</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p>The node type represents Bayesian posterior probability.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0177414.g005">
              https://doi.org/10.1371/journal.pone.0177414.g005</a></p></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>Based on the Bayesian inference separation, the neutrality test was conducted for Group 4 separately from the remaining groups. The observed values of Tajima’s D and Fu’s F<sub>S</sub> neutrality tests were -2.27254 (p &lt; 0.01) and -17.508 (p = 0.000), respectively, for Group 4, and -2.68706 (p &lt; 0.01) and -315.397 (p = 0.000), respectively, for other groups.</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="sec011" name="sec011" data-toc="sec011" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>Our results demonstrated that <em>C</em>. <em>externa</em> has significant haplotype structure arising from genetic differences not shown to be influenced by present geography. Almost all the 158 <em>COI</em> haplotypes and 58 <em>16S</em> haplotypes were found in all sampled locations.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>The AMOVA demonstrated that haplotype differences among locations were not significant, showing that the distances between sampling sites, in Brazil and Paraguay, did not act as a barrier, and produced no geographic differentiation. This would be a necessary premise for the phylogeographic inference test, which could assess the haplotype and geographic association [<a href="#pone.0177414.ref033" class="ref-tip">33</a>]. This did not occur in the present or in previous studies of <em>C</em>. <em>externa</em> [<a href="#pone.0177414.ref004" class="ref-tip">4</a>–<a href="#pone.0177414.ref007" class="ref-tip">7</a>]. However, the presence of distinct groups was notable in the haplotype networks (Figs <a href="#pone-0177414-g001">1</a> and <a href="#pone-0177414-g002">2</a>) as a consequence of the level of difference/mutation among them (<a href="#pone.0177414.s003">S1 Supporting Information</a>).</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>Phylogeographic studies use the clades formed in the haplotype network to test the hypotheses of genetic/geographical association and several studies show a correspondence between the geographic distribution and network topology [<a href="#pone.0177414.ref034" class="ref-tip">34</a>–<a href="#pone.0177414.ref043" class="ref-tip">43</a>]. Therefore, we used the groups formed in the network to assess by AMOVA the relationship between the distinct haplotypic groups, and the analysis revealed significant differences in both analyzed genes. Hence, the haplotypes showed genetic differences sufficient to be allocated into groups, and that different haplotypic groups coexisted in a location.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Bayesian inference performed on the concatenated data suggested that there was initially a divergence of <em>C</em>. <em>externa</em> haplotypes into two distinct clades, which diversified posteriorly (<a href="#pone-0177414-g005">Fig 5</a>). This clear separation of haplotypes into clades of <em>C</em>. <em>externa</em> does not correspond to a geographic structure, as confirmed by AMOVA. The clade with the most recent diversification was composed exclusively of individuals from Group 4 in both gene haplotype networks. In both analyzed genes, these 34 individuals showed greater similarity to one another than to the other groups.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>In the structure analysis of the <em>COI</em> gene using Structure (<a href="#pone-0177414-g003">Fig 3</a>), the union of Groups 1, 2, and 3 indicated by the presence of at least one cluster in common, and the segregation of Group 4, reinforced the results of the Bayesian analysis (<a href="#pone-0177414-g005">Fig 5</a>) for the divergence into two clades. For <em>16S</em>, the structure analysis showed the separation into only two clusters (<a href="#pone-0177414-g004">Fig 4</a>) with Group 4 remaining associated with Groups 1 and 3. This can be attributed to the fact that the <em>16S</em> gene is conserved and non-coding, so that the primary difference observed between the two clusters is the insertion of one or two bases at positions 173 and 174 (<a href="#pone.0177414.s003">S1 Supporting Information</a>). Thus, the phylogenetic information that shows the initial divergence may have been diluted in the structure analysis, since it does not include the ancestral state present in the outgroup [<a href="#pone.0177414.ref044" class="ref-tip">44</a>, <a href="#pone.0177414.ref045" class="ref-tip">45</a>].</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6" class="link-target"></a><p>The population expansion followed by diversification of haplotypes can be inferred by the star-like network topology of both genes, as well as by the neutrality tests. The networks displayed two more frequent haplotypes that were considered ancestral (<a href="#pone.0177414.s005">S2 Table</a> and <a href="#pone.0177414.s006">S3 Table</a>) from which many haplotypes were derived, suggesting that most of these haplotypes emerged later [<a href="#pone.0177414.ref046" class="ref-tip">46</a>]. The results of Tajima's D and Fu's F<sub>S</sub> neutrality tests were negative, indicating an excess of rare alleles within the population, which may suggest population expansion [<a href="#pone.0177414.ref047" class="ref-tip">47</a>, <a href="#pone.0177414.ref048" class="ref-tip">48</a>].</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7" class="link-target"></a><p>These results suggest that the evolutionary history of <em>C</em>. <em>externa</em> may include the existence of multiple ancestral haplotypes within a same geographic area. However, it is possible to infer that this pattern found for <em>C</em>. <em>externa</em> can be derived from a strong anthropic action, involving exchanges of plant seedlings with the petiolate eggs between agricultural producers. Thus, this agricultural practice may have promoted homogenization in the populations trough the introduction of haplotypes from other regions with distinct genetic characteristics, covering up a possible regional differentiation. As a result of this introduction, it is currently observed a pattern of co-existence of multiple ancestral haplotypes in the same place, but belonging to groups excessively genetically distinct to be sharing the same area of occurrence in a biogeographic point of view, and thus indicating that these haplotypes have had an allopatric differentiation.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8" class="link-target"></a><p>Therefore, it is possible that the earlier geographical history of <em>C</em>. <em>externa</em> has been erased, and now we have highlighted its more recent genetic history.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec012" name="sec012" data-toc="sec012" class="link-target" title="Supporting information"></a><h2>Supporting information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0177414"></div><div class="supplementary-material"><a name="pone.0177414.s001" id="pone.0177414.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0177414.s001">S1 Fig. </a>Haplotype network for the <em>COI</em> gene of <em>Chrysoperla externa</em>.</h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">Square represents the ancestral haplotype. The size of circles reflects the frequency of the haplotypes in the sample. Haplotype names correspond to names in <a href="#pone.0177414.s004">S1 Table</a>. Solid lines correspond to a mutational change connecting two haplotypes with probability &gt;95%. Small circles denote missing intermediate haplotypes.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0177414.s001">https://doi.org/10.1371/journal.pone.0177414.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0177414.s002" id="pone.0177414.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0177414.s002">S2 Fig. </a>Haplotype network for the <em>16S</em> gene of <em>Chrysoperla externa</em>.</h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Square represents the ancestral haplotype. The size of circles reflects the frequency of the haplotypes in the sample. Haplotype names correspond to names in <a href="#pone.0177414.s004">S1 Table</a>. Solid lines correspond to a mutational change connecting two haplotypes with probability &gt;95%. Small circles denote missing intermediate haplotypes.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0177414.s002">https://doi.org/10.1371/journal.pone.0177414.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0177414.s003" id="pone.0177414.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0177414.s003">S1 Supporting Information. </a>Variable sites of haplotypes in concatenated genes of <em>Chrysoperla externa</em>.</h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">Positions from 1 to 730 are for the <em>COI</em> gene, and from 731 to 1218 are for the <em>16S</em> gene.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0177414.s003">https://doi.org/10.1371/journal.pone.0177414.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(TXT)</p>
</div><div class="supplementary-material"><a name="pone.0177414.s004" id="pone.0177414.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0177414.s004">S1 Table. </a>Sampled localities.</h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">Sampling localities, geographic coordinates, number of specimens analyzed (<em>COI</em>/<em>16S</em>), and <em>Chrysoperla externa</em> voucher number.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0177414.s004">https://doi.org/10.1371/journal.pone.0177414.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0177414.s005" id="pone.0177414.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0177414.s005">S2 Table. </a>Haplotype list of the <em>COI</em> gene.</h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI">Identification code (ID) of haplotypes for <em>COI</em> gene; number of specimens containing each haplotype; GenBank accession number; <em>Chrysoperla externa</em> voucher number.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0177414.s005">https://doi.org/10.1371/journal.pone.0177414.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0177414.s006" id="pone.0177414.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0177414.s006">S3 Table. </a>Haplotype list of the <em>16S</em> gene.</h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI">Identification code (ID) of haplotypes for <em>16S</em> gene; number of specimens containing each haplotype; GenBank accession number; <em>Chrysoperla externa</em> voucher number.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0177414.s006">https://doi.org/10.1371/journal.pone.0177414.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0177414.s007" id="pone.0177414.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0177414.s007">S4 Table. </a>Haplotype list of concatenated genes.</h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI">Identification code (ID) of haplotypes for concatenated data (<em>COI</em> and <em>16S</em> genes); number of specimens containing each haplotype; <em>Chrysoperla externa</em> voucher number.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0177414.s007">https://doi.org/10.1371/journal.pone.0177414.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank to deceased Prof. Dr. Sergio de Freitas, who was supported by Fundação de Amparo à Pesquisa do Estado de São Paulo—(2008/05727-8), by morphological identification of specimens and laboratory support to research, and two reviewers by critical reading of this manuscript.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>

<ol class="simple">

<li><strong>Conceptualization:</strong> ACMCC.</li>

<li><strong>Data curation:</strong> ACMCC NCCPB.</li>

<li><strong>Formal analysis:</strong> NCCPB.</li>

<li><strong>Funding acquisition:</strong> ACMCC NCCPB.</li>

<li><strong>Investigation:</strong> ACMCC NCCPB.</li>

<li><strong>Methodology:</strong> ACMCC NCCPB.</li>

<li><strong>Project administration:</strong> ACMCC NCCPB.</li>

<li><strong>Resources:</strong> ACMCC NCCPB.</li>

<li><strong>Validation:</strong> ACMCC NCCPB.</li>

<li><strong>Visualization:</strong> ACMCC NCCPB.</li>

<li><strong>Writing – original draft:</strong> ACMCC NCCPB.</li>

<li><strong>Writing – review &amp; editing:</strong> ACMCC NCCPB.</li>

</ol>
</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0177414.ref001" id="pone.0177414.ref001" class="link-target"></a>Oswald JD. Neuropterida Species of the World. Version 4.0 [Internet]. College Station (TX): Department of Entomology, Texas A&amp;M University (US); 2015 [updated 2015 July 6; cited 2016 July 13]. Available from: <a href="http://lacewing.tamu.edu/SpeciesCatalogue/">http://lacewing.tamu.edu/SpeciesCatalogue/</a>. <ul class="find-nolinks"></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0177414.ref002" id="pone.0177414.ref002" class="link-target"></a>Brooks SJ. A taxonomic review of the common green lacewing genus <em>Chrysoperla</em> (Neuroptera: Chrysoperla). Bull Nat Hist Mus Entomol ser.1994; 63: 137–210. <ul class="reflinks"><li><a href="#" data-author="Brooks" data-cit="BrooksSJ.%20A%20taxonomic%20review%20of%20the%20common%20green%20lacewing%20genus%20Chrysoperla%20%28Neuroptera%3A%20Chrysoperla%29.%20Bull%20Nat%20Hist%20Mus%20Entomol%20ser.1994%3B%2063%3A%20137%E2%80%93210." data-title="A%20taxonomic%20review%20of%20the%20common%20green%20lacewing%20genus%20Chrysoperla%20%28Neuroptera%3A%20Chrysoperla%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+taxonomic+review+of+the+common+green+lacewing+genus+Chrysoperla+%28Neuroptera%3A+Chrysoperla%29+Brooks+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0177414.ref003" id="pone.0177414.ref003" class="link-target"></a>Albuquerque GS, Tauber CA, Tauber MJ. <em>Chrysoperla externa</em> (Neuroptera: Chrysopidae): life history and potential for biological control in Central and South America. Biol control. 1994; 4: 8–13. <ul class="reflinks"><li><a href="#" data-author="Albuquerque" data-cit="AlbuquerqueGS%2C%20TauberCA%2C%20TauberMJ.%20Chrysoperla%20externa%20%28Neuroptera%3A%20Chrysopidae%29%3A%20life%20history%20and%20potential%20for%20biological%20control%20in%20Central%20and%20South%20America.%20Biol%20control.%201994%3B%204%3A%208%E2%80%9313." data-title="Chrysoperla%20externa%20%28Neuroptera%3A%20Chrysopidae%29%3A%20life%20history%20and%20potential%20for%20biological%20control%20in%20Central%20and%20South%20America" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Chrysoperla+externa+%28Neuroptera%3A+Chrysopidae%29%3A+life+history+and+potential+for+biological+control+in+Central+and+South+America+Albuquerque+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0177414.ref004" id="pone.0177414.ref004" class="link-target"></a>Morales AC, Freitas S. Haplotype characterization of the COI mitochondrial gene in <em>Chrysoperla externa</em> (Neuroptera: Chrysopidae) from different environments in Jaboticabal, state of São Paulo, southeastern Brazil. Braz J Biol. 2010; 70: 1115–1121. pmid:21180923 <ul class="reflinks"><li><a href="#" data-author="Morales" data-cit="MoralesAC%2C%20FreitasS.%20Haplotype%20characterization%20of%20the%20COI%20mitochondrial%20gene%20in%20Chrysoperla%20externa%20%28Neuroptera%3A%20Chrysopidae%29%20from%20different%20environments%20in%20Jaboticabal%2C%20state%20of%20S%C3%A3o%20Paulo%2C%20southeastern%20Brazil.%20Braz%20J%20Biol.%202010%3B%2070%3A%201115%E2%80%931121.%2021180923" data-title="Haplotype%20characterization%20of%20the%20COI%20mitochondrial%20gene%20in%20Chrysoperla%20externa%20%28Neuroptera%3A%20Chrysopidae%29%20from%20different%20environments%20in%20Jaboticabal%2C%20state%20of%20S%C3%A3o%20Paulo%2C%20southeastern%20Brazil" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21180923" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Haplotype+characterization+of+the+COI+mitochondrial+gene+in+Chrysoperla+externa+%28Neuroptera%3A+Chrysopidae%29+from+different+environments+in+Jaboticabal%2C+state+of+S%C3%A3o+Paulo%2C+southeastern+Brazil+Morales+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0177414.ref005" id="pone.0177414.ref005" class="link-target"></a>Barbosa NCCP, Freitas S, Morales AC. Distinct genetic structure in populations of <em>Chrysoperla externa</em> (Hagen) (Neuroptera, Chrysopidae) shown by genetic markers ISSR and COI gene. Rev Bras Entomol. 2014; 58: 203–211. <ul class="reflinks"><li><a href="#" data-author="Barbosa" data-cit="BarbosaNCCP%2C%20FreitasS%2C%20MoralesAC.%20Distinct%20genetic%20structure%20in%20populations%20of%20Chrysoperla%20externa%20%28Hagen%29%20%28Neuroptera%2C%20Chrysopidae%29%20shown%20by%20genetic%20markers%20ISSR%20and%20COI%20gene.%20Rev%20Bras%20Entomol.%202014%3B%2058%3A%20203%E2%80%93211." data-title="Distinct%20genetic%20structure%20in%20populations%20of%20Chrysoperla%20externa%20%28Hagen%29%20%28Neuroptera%2C%20Chrysopidae%29%20shown%20by%20genetic%20markers%20ISSR%20and%20COI%20gene" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Distinct+genetic+structure+in+populations+of+Chrysoperla+externa+%28Hagen%29+%28Neuroptera%2C+Chrysopidae%29+shown+by+genetic+markers+ISSR+and+COI+gene+Barbosa+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0177414.ref006" id="pone.0177414.ref006" class="link-target"></a>Lavagnini TC, Morales AC, Freitas S. Population genetics of <em>Chrysoperla externa</em> (Neuroptera: Chrysopidae) and implications for biological control. Braz. J. Biol. 2015; 75:878–885.  pmid:26675904 <ul class="reflinks"><li><a href="#" data-author="Lavagnini" data-cit="LavagniniTC%2C%20MoralesAC%2C%20FreitasS.%20Population%20genetics%20of%20Chrysoperla%20externa%20%28Neuroptera%3A%20Chrysopidae%29%20and%20implications%20for%20biological%20control.%20Braz.%20J.%20Biol.%202015%3B%2075%3A878%E2%80%93885.%20doi%3A%2010.1590%2F1519-6984.02014%2026675904" data-title="Population%20genetics%20of%20Chrysoperla%20externa%20%28Neuroptera%3A%20Chrysopidae%29%20and%20implications%20for%20biological%20control" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26675904" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Population+genetics+of+Chrysoperla+externa+%28Neuroptera%3A+Chrysopidae%29+and+implications+for+biological+control+Lavagnini+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0177414.ref007" id="pone.0177414.ref007" class="link-target"></a>Morales AC, Lavagnini TC, Freitas S. Loss of genetic variability induced by agroecosystems: <em>Chrysoperla externa</em> (Hagen) (Neuroptera: Chrysopidae) as a case study. Neotrop Entomol. 2013; 42: 32–38.  pmid:23949710 <ul class="reflinks"><li><a href="#" data-author="Morales" data-cit="MoralesAC%2C%20LavagniniTC%2C%20FreitasS.%20Loss%20of%20genetic%20variability%20induced%20by%20agroecosystems%3A%20Chrysoperla%20externa%20%28Hagen%29%20%28Neuroptera%3A%20Chrysopidae%29%20as%20a%20case%20study.%20Neotrop%20Entomol.%202013%3B%2042%3A%2032%E2%80%9338.%20doi%3A%2010.1007%2Fs13744-012-0085-1%2023949710" data-title="Loss%20of%20genetic%20variability%20induced%20by%20agroecosystems%3A%20Chrysoperla%20externa%20%28Hagen%29%20%28Neuroptera%3A%20Chrysopidae%29%20as%20a%20case%20study" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23949710" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Loss+of+genetic+variability+induced+by+agroecosystems%3A+Chrysoperla+externa+%28Hagen%29+%28Neuroptera%3A+Chrysopidae%29+as+a+case+study+Morales+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0177414.ref008" id="pone.0177414.ref008" class="link-target"></a>Brooks SJ, Barnard PC. The green lacewings of the world: a generic review (Neuroptera: Chrysopidae). Bull Nat Hist Mus Entomol ser. 1990; 59: 117–286. <ul class="reflinks"><li><a href="#" data-author="Brooks" data-cit="BrooksSJ%2C%20BarnardPC.%20The%20green%20lacewings%20of%20the%20world%3A%20a%20generic%20review%20%28Neuroptera%3A%20Chrysopidae%29.%20Bull%20Nat%20Hist%20Mus%20Entomol%20ser.%201990%3B%2059%3A%20117%E2%80%93286." data-title="The%20green%20lacewings%20of%20the%20world%3A%20a%20generic%20review%20%28Neuroptera%3A%20Chrysopidae%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+green+lacewings+of+the+world%3A+a+generic+review+%28Neuroptera%3A+Chrysopidae%29+Brooks+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0177414.ref009" id="pone.0177414.ref009" class="link-target"></a>Simon C, Frati F, Beckenbach A, Crespi B, Liu H, Flook P. Evolution, weighting and phylogenetic utility of mitochondrial gene sequences and a compilation of conserved Polymerase Chain Reaction primers. Ann Entomol Soc Am. 1994; 87: 651–701. <ul class="reflinks"><li><a href="#" data-author="Simon" data-cit="SimonC%2C%20FratiF%2C%20BeckenbachA%2C%20CrespiB%2C%20LiuH%2C%20FlookP.%20Evolution%2C%20weighting%20and%20phylogenetic%20utility%20of%20mitochondrial%20gene%20sequences%20and%20a%20compilation%20of%20conserved%20Polymerase%20Chain%20Reaction%20primers.%20Ann%20Entomol%20Soc%20Am.%201994%3B%2087%3A%20651%E2%80%93701." data-title="Evolution%2C%20weighting%20and%20phylogenetic%20utility%20of%20mitochondrial%20gene%20sequences%20and%20a%20compilation%20of%20conserved%20Polymerase%20Chain%20Reaction%20primers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Evolution%2C+weighting+and+phylogenetic+utility+of+mitochondrial+gene+sequences+and+a+compilation+of+conserved+Polymerase+Chain+Reaction+primers+Simon+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0177414.ref010" id="pone.0177414.ref010" class="link-target"></a>Technelysium Ltd. Chromas Lite. Version 2.1.1 [software].1998–2013 [cited 2016 Mai 5]. Available from: <a href="http://www.technelysium.com.au/chromas_lite.html">http://www.technelysium.com.au/chromas_lite.html</a>. <ul class="find-nolinks"></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0177414.ref011" id="pone.0177414.ref011" class="link-target"></a>McWilliam H, Li W, Uludag M, Squizzato S, Park YM, Buso N, et al. Analysis Tool Web Services from the EMBL-EBI. Nucleic Acids Res. 2013; 41 (Web Server issue): W597–600.  pmid:23671338 <ul class="reflinks"><li><a href="#" data-author="McWilliam" data-cit="McWilliamH%2C%20LiW%2C%20UludagM%2C%20SquizzatoS%2C%20ParkYM%2C%20BusoN%2C%20et%20al.%20Analysis%20Tool%20Web%20Services%20from%20the%20EMBL-EBI.%20Nucleic%20Acids%20Res.%202013%3B%2041%20%28Web%20Server%20issue%29%3A%20W597%E2%80%93600.%20doi%3A%2010.1093%2Fnar%2Fgkt376%2023671338" data-title="Analysis%20Tool%20Web%20Services%20from%20the%20EMBL-EBI" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23671338" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+Tool+Web+Services+from+the+EMBL-EBI+McWilliam+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0177414.ref012" id="pone.0177414.ref012" class="link-target"></a>Librado P, Rozas J. DnaSP v5: a software for comprehensive analysis of DNA polymorphism data. Bioinformatics. 2009; 25: 1451–1452.  pmid:19346325 <ul class="reflinks"><li><a href="#" data-author="Librado" data-cit="LibradoP%2C%20RozasJ.%20DnaSP%20v5%3A%20a%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data.%20Bioinformatics.%202009%3B%2025%3A%201451%E2%80%931452.%20doi%3A%2010.1093%2Fbioinformatics%2Fbtp187%2019346325" data-title="DnaSP%20v5%3A%20a%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19346325" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DnaSP+v5%3A+a+software+for+comprehensive+analysis+of+DNA+polymorphism+data+Librado+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0177414.ref013" id="pone.0177414.ref013" class="link-target"></a>Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0. Mol Biol Evol. 2013; 30: 2725–2729.  pmid:24132122 <ul class="reflinks"><li><a href="#" data-author="Tamura" data-cit="TamuraK%2C%20StecherG%2C%20PetersonD%2C%20FilipskiA%2C%20KumarS.%20MEGA6%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%206.0.%20Mol%20Biol%20Evol.%202013%3B%2030%3A%202725%E2%80%932729.%20doi%3A%2010.1093%2Fmolbev%2Fmst197%2024132122" data-title="MEGA6%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%206.0" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MEGA6%3A+Molecular+Evolutionary+Genetics+Analysis+Version+6.0+Tamura+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0177414.ref014" id="pone.0177414.ref014" class="link-target"></a>Excoffier L, Lischer HEL. Arlequin suite ver 3.5: A new series of programs to perform population genetics analyses under Linux and Windows. Mol Ecol Resour. 2010; 10: 564–567.  pmid:21565059 <ul class="reflinks"><li><a href="#" data-author="Excoffier" data-cit="ExcoffierL%2C%20LischerHEL.%20Arlequin%20suite%20ver%203.5%3A%20A%20new%20series%20of%20programs%20to%20perform%20population%20genetics%20analyses%20under%20Linux%20and%20Windows.%20Mol%20Ecol%20Resour.%202010%3B%2010%3A%20564%E2%80%93567.%20doi%3A%2010.1111%2Fj.1755-0998.2010.02847.x%2021565059" data-title="Arlequin%20suite%20ver%203.5%3A%20A%20new%20series%20of%20programs%20to%20perform%20population%20genetics%20analyses%20under%20Linux%20and%20Windows" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21565059" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Arlequin+suite+ver+3.5%3A+A+new+series+of+programs+to+perform+population+genetics+analyses+under+Linux+and+Windows+Excoffier+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0177414.ref015" id="pone.0177414.ref015" class="link-target"></a>Clement M, Posada D, Crandall KA. TCS: a computer program to estimate gene genealogies. Mol Ecol. 2000; 9: 1657–1659. pmid:11050560 <ul class="reflinks"><li><a href="#" data-author="Clement" data-cit="ClementM%2C%20PosadaD%2C%20CrandallKA.%20TCS%3A%20a%20computer%20program%20to%20estimate%20gene%20genealogies.%20Mol%20Ecol.%202000%3B%209%3A%201657%E2%80%931659.%2011050560" data-title="TCS%3A%20a%20computer%20program%20to%20estimate%20gene%20genealogies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11050560" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=TCS%3A+a+computer+program+to+estimate+gene+genealogies+Clement+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0177414.ref016" id="pone.0177414.ref016" class="link-target"></a>Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000; 155: 945–959. pmid:10835412 <ul class="reflinks"><li><a href="#" data-author="Pritchard" data-cit="PritchardJK%2C%20StephensM%2C%20DonnellyP.%20Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data.%20Genetics.%202000%3B%20155%3A%20945%E2%80%93959.%2010835412" data-title="Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inference+of+population+structure+using+multilocus+genotype+data+Pritchard+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0177414.ref017" id="pone.0177414.ref017" class="link-target"></a>Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. Genetics. 2003; 164: 1567–1587. pmid:12930761 <ul class="reflinks"><li><a href="#" data-author="Falush" data-cit="FalushD%2C%20StephensM%2C%20PritchardJK.%20Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20linked%20loci%20and%20correlated%20allele%20frequencies.%20Genetics.%202003%3B%20164%3A%201567%E2%80%931587.%2012930761" data-title="Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20linked%20loci%20and%20correlated%20allele%20frequencies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930761" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inference+of+population+structure+using+multilocus+genotype+data%3A+linked+loci+and+correlated+allele+frequencies+Falush+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0177414.ref018" id="pone.0177414.ref018" class="link-target"></a>Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: dominant markers and null alleles. Mol Ecol Notes. 2007; 7: 574–578.  pmid:18784791 <ul class="reflinks"><li><a href="#" data-author="Falush" data-cit="FalushD%2C%20StephensM%2C%20PritchardJK.%20Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20dominant%20markers%20and%20null%20alleles.%20Mol%20Ecol%20Notes.%202007%3B%207%3A%20574%E2%80%93578.%20doi%3A%2010.1111%2Fj.1471-8286.2007.01758.x%2018784791" data-title="Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20dominant%20markers%20and%20null%20alleles" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18784791" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inference+of+population+structure+using+multilocus+genotype+data%3A+dominant+markers+and+null+alleles+Falush+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0177414.ref019" id="pone.0177414.ref019" class="link-target"></a>Hubisz MJ, Falush D, Stephens M., Pritchard JK. Inferring weak population structure with the assistance of sample group information. Mol Ecol Resour. 2009; 9: 1322–1332.  pmid:21564903 <ul class="reflinks"><li><a href="#" data-author="Hubisz" data-cit="HubiszMJ%2C%20FalushD%2C%20StephensM.%2C%20PritchardJK.%20Inferring%20weak%20population%20structure%20with%20the%20assistance%20of%20sample%20group%20information.%20Mol%20Ecol%20Resour.%202009%3B%209%3A%201322%E2%80%931332.%20doi%3A%2010.1111%2Fj.1755-0998.2009.02591.x%2021564903" data-title="Inferring%20weak%20population%20structure%20with%20the%20assistance%20of%20sample%20group%20information" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21564903" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inferring+weak+population+structure+with+the+assistance+of+sample+group+information+Hubisz+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0177414.ref020" id="pone.0177414.ref020" class="link-target"></a>Evanno G, Regnaut S, Goudet J. Detecting the number of clusters of individuals using the software Structure: a simulation study. Mol Ecol. 2005; 14: 2611–2620.  pmid:15969739 <ul class="reflinks"><li><a href="#" data-author="Evanno" data-cit="EvannoG%2C%20RegnautS%2C%20GoudetJ.%20Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20Structure%3A%20a%20simulation%20study.%20Mol%20Ecol.%202005%3B%2014%3A%202611%E2%80%932620.%20doi%3A%2010.1111%2Fj.1365-294X.2005.02553.x%2015969739" data-title="Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20Structure%3A%20a%20simulation%20study" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Detecting+the+number+of+clusters+of+individuals+using+the+software+Structure%3A+a+simulation+study+Evanno+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0177414.ref021" id="pone.0177414.ref021" class="link-target"></a>Earl DA. Structure Harvester: a website and program for visualizing STRUCTURE output and implementing the Evanno method. Conserv Gene Resour. 2012; 4: 359–361. <ul class="reflinks"><li><a href="#" data-author="Earl" data-cit="EarlDA.%20Structure%20Harvester%3A%20a%20website%20and%20program%20for%20visualizing%20STRUCTURE%20output%20and%20implementing%20the%20Evanno%20method.%20Conserv%20Gene%20Resour.%202012%3B%204%3A%20359%E2%80%93361." data-title="Structure%20Harvester%3A%20a%20website%20and%20program%20for%20visualizing%20STRUCTURE%20output%20and%20implementing%20the%20Evanno%20method" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Structure+Harvester%3A+a+website+and+program+for+visualizing+STRUCTURE+output+and+implementing+the+Evanno+method+Earl+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0177414.ref022" id="pone.0177414.ref022" class="link-target"></a>Kopelman NM, Mayzel J, Jakobsson M, Rosenberg NA, Mayrose I. CLUMPAK: a program for identifying clustering modes and packaging population structure inferences across K. Mol Ecol Resour. 2015; 15:1179–1191.  pmid:25684545 <ul class="reflinks"><li><a href="#" data-author="Kopelman" data-cit="KopelmanNM%2C%20MayzelJ%2C%20JakobssonM%2C%20RosenbergNA%2C%20MayroseI.%20CLUMPAK%3A%20a%20program%20for%20identifying%20clustering%20modes%20and%20packaging%20population%20structure%20inferences%20across%20K.%20Mol%20Ecol%20Resour.%202015%3B%2015%3A1179%E2%80%931191.%20doi%3A%2010.1111%2F1755-0998.12387%2025684545" data-title="CLUMPAK%3A%20a%20program%20for%20identifying%20clustering%20modes%20and%20packaging%20population%20structure%20inferences%20across%20K" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25684545" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=CLUMPAK%3A+a+program+for+identifying+clustering+modes+and+packaging+population+structure+inferences+across+K+Kopelman+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0177414.ref023" id="pone.0177414.ref023" class="link-target"></a>Tajima F. Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. Genetics. 1989; 123: 585–595. pmid:2513255 <ul class="reflinks"><li><a href="#" data-author="Tajima" data-cit="TajimaF.%20Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism.%20Genetics.%201989%3B%20123%3A%20585%E2%80%93595.%202513255" data-title="Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism+Tajima+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0177414.ref024" id="pone.0177414.ref024" class="link-target"></a>Fu Y-X. Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection. Genetics. 1997; 147: 915–925. pmid:9335623 <ul class="reflinks"><li><a href="#" data-author="Fu" data-cit="FuY-X.%20Statistical%20tests%20of%20neutrality%20of%20mutations%20against%20population%20growth%2C%20hitchhiking%20and%20background%20selection.%20Genetics.%201997%3B%20147%3A%20915%E2%80%93925.%209335623" data-title="Statistical%20tests%20of%20neutrality%20of%20mutations%20against%20population%20growth%2C%20hitchhiking%20and%20background%20selection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9335623" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Statistical+tests+of+neutrality+of+mutations+against+population+growth%2C+hitchhiking+and+background+selection+Fu+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0177414.ref025" id="pone.0177414.ref025" class="link-target"></a>Xia X. DAMBE5: A comprehensive software package for data analysis in molecular biology and evolution. Mol Biol Evol. 2013; 30: 1720–1728.  pmid:23564938 <ul class="reflinks"><li><a href="#" data-author="Xia" data-cit="XiaX.%20DAMBE5%3A%20A%20comprehensive%20software%20package%20for%20data%20analysis%20in%20molecular%20biology%20and%20evolution.%20Mol%20Biol%20Evol.%202013%3B%2030%3A%201720%E2%80%931728.%20doi%3A%2010.1093%2Fmolbev%2Fmst064%2023564938" data-title="DAMBE5%3A%20A%20comprehensive%20software%20package%20for%20data%20analysis%20in%20molecular%20biology%20and%20evolution" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23564938" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DAMBE5%3A+A+comprehensive+software+package+for+data+analysis+in+molecular+biology+and+evolution+Xia+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0177414.ref026" id="pone.0177414.ref026" class="link-target"></a>Xia X, Xie Z, Salemi M, Chen L, Wang Y. An index of substitution saturation and its application. Mol Phylogenet Evol. 2003; 26: 1–7. pmid:12470932 <ul class="reflinks"><li><a href="#" data-author="Xia" data-cit="XiaX%2C%20XieZ%2C%20SalemiM%2C%20ChenL%2C%20WangY.%20An%20index%20of%20substitution%20saturation%20and%20its%20application.%20Mol%20Phylogenet%20Evol.%202003%3B%2026%3A%201%E2%80%937.%2012470932" data-title="An%20index%20of%20substitution%20saturation%20and%20its%20application" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12470932" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=An+index+of+substitution+saturation+and+its+application+Xia+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0177414.ref027" id="pone.0177414.ref027" class="link-target"></a>Xia X, Lemey P. Assessing substitution saturation with DAMBE. In Lemey P, Salemi M, Vandamme AM, editors. The Phylogenetic Handbook: a practical approach to DNA and protein phylogeny. Cambridge: Cambridge University Press; 2009. p. 615–630. <ul class="find-nolinks"></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0177414.ref028" id="pone.0177414.ref028" class="link-target"></a>Drummond AJ, Suchard MA, Xie D, Rambaut A. Bayesian phylogenetics with BEAUti and the BEAST 1.7. Mol Biol Evol. 2012; 29: 1969–1973.  pmid:22367748 <ul class="reflinks"><li><a href="#" data-author="Drummond" data-cit="DrummondAJ%2C%20SuchardMA%2C%20XieD%2C%20RambautA.%20Bayesian%20phylogenetics%20with%20BEAUti%20and%20the%20BEAST%201.7.%20Mol%20Biol%20Evol.%202012%3B%2029%3A%201969%E2%80%931973.%20doi%3A%2010.1093%2Fmolbev%2Fmss075%2022367748" data-title="Bayesian%20phylogenetics%20with%20BEAUti%20and%20the%20BEAST%201.7" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22367748" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bayesian+phylogenetics+with+BEAUti+and+the+BEAST+1.7+Drummond+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0177414.ref029" id="pone.0177414.ref029" class="link-target"></a>Darriba D, Taboada GL, Doallo R, Posada D. jModelTest 2: more models, new heuristics and parallel computing. Nat Methods. 2012; 9: 772. <ul class="reflinks"><li><a href="#" data-author="Darriba" data-cit="DarribaD%2C%20TaboadaGL%2C%20DoalloR%2C%20PosadaD.%20jModelTest%202%3A%20more%20models%2C%20new%20heuristics%20and%20parallel%20computing.%20Nat%20Methods.%202012%3B%209%3A%20772." data-title="jModelTest%202%3A%20more%20models%2C%20new%20heuristics%20and%20parallel%20computing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=jModelTest+2%3A+more+models%2C+new+heuristics+and+parallel+computing+Darriba+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0177414.ref030" id="pone.0177414.ref030" class="link-target"></a>Rambaut A, Suchard MA, Xie D, Drummond AJ. Tracer v1.6 [software].2014 [cited 2016 Mai 5]. Available from: <a href="http://beast.bio.ed.ac.uk/Tracer">http://beast.bio.ed.ac.uk/Tracer</a>. <ul class="find-nolinks"></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0177414.ref031" id="pone.0177414.ref031" class="link-target"></a>Rambaut A, Drummond AJ. Tree Annotator v1.8.2 [software]. 2002–2015 [cited 2016 Mai 5]. Available from: <a href="http://beast.bio.ed.ac.uk/TreeAnnotator">http://beast.bio.ed.ac.uk/TreeAnnotator</a>. <ul class="find-nolinks"></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0177414.ref032" id="pone.0177414.ref032" class="link-target"></a>Rambaut A. FigTree v.1.4.2 [software]. 2006–2016 [cited 2016 Mai 5]. Available from: <a href="http://beast.bio.ed.ac.uk/FigTree">http://beast.bio.ed.ac.uk/FigTree</a>. <ul class="find-nolinks"></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0177414.ref033" id="pone.0177414.ref033" class="link-target"></a>Templeton AR. Nested clade analyses of phylogeographic data: testing hypothesis about gene flow and population history. Mol Ecol. 1998; 7: 381–397. pmid:9627999 <ul class="reflinks"><li><a href="#" data-author="Templeton" data-cit="TempletonAR.%20Nested%20clade%20analyses%20of%20phylogeographic%20data%3A%20testing%20hypothesis%20about%20gene%20flow%20and%20population%20history.%20Mol%20Ecol.%201998%3B%207%3A%20381%E2%80%93397.%209627999" data-title="Nested%20clade%20analyses%20of%20phylogeographic%20data%3A%20testing%20hypothesis%20about%20gene%20flow%20and%20population%20history" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9627999" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Nested+clade+analyses+of+phylogeographic+data%3A+testing+hypothesis+about+gene+flow+and+population+history+Templeton+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0177414.ref034" id="pone.0177414.ref034" class="link-target"></a>Milot E, Lisle Gibbs H, Hobson KA. Phylogeography and genetic structure of northern populations of the yellow warbler (<em>Dendroica petechia</em>). Mol Ecol. 2000; 9: 667–681. pmid:10849283 <ul class="reflinks"><li><a href="#" data-author="Milot" data-cit="MilotE%2C%20Lisle%20GibbsH%2C%20HobsonKA.%20Phylogeography%20and%20genetic%20structure%20of%20northern%20populations%20of%20the%20yellow%20warbler%20%28Dendroica%20petechia%29.%20Mol%20Ecol.%202000%3B%209%3A%20667%E2%80%93681.%2010849283" data-title="Phylogeography%20and%20genetic%20structure%20of%20northern%20populations%20of%20the%20yellow%20warbler%20%28Dendroica%20petechia%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10849283" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeography+and+genetic+structure+of+northern+populations+of+the+yellow+warbler+%28Dendroica+petechia%29+Milot+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0177414.ref035" id="pone.0177414.ref035" class="link-target"></a>Muwanika VB, Nyakaana S, Siegismund HR, Arctander P. Phylogeography and population structure of the common warthog (<em>Phacochoerus africanus</em>) inferred from variation in mitochondrial DNA sequences and microsatellite loci. Heredity. 2003; 91: 361–372.  pmid:14512951 <ul class="reflinks"><li><a href="#" data-author="Muwanika" data-cit="MuwanikaVB%2C%20NyakaanaS%2C%20SiegismundHR%2C%20ArctanderP.%20Phylogeography%20and%20population%20structure%20of%20the%20common%20warthog%20%28Phacochoerus%20africanus%29%20inferred%20from%20variation%20in%20mitochondrial%20DNA%20sequences%20and%20microsatellite%20loci.%20Heredity.%202003%3B%2091%3A%20361%E2%80%93372.%20doi%3A%2010.1038%2Fsj.hdy.6800341%2014512951" data-title="Phylogeography%20and%20population%20structure%20of%20the%20common%20warthog%20%28Phacochoerus%20africanus%29%20inferred%20from%20variation%20in%20mitochondrial%20DNA%20sequences%20and%20microsatellite%20loci" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14512951" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeography+and+population+structure+of+the+common+warthog+%28Phacochoerus+africanus%29+inferred+from+variation+in+mitochondrial+DNA+sequences+and+microsatellite+loci+Muwanika+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0177414.ref036" id="pone.0177414.ref036" class="link-target"></a>Godoy JA, Negro JJ, Hiraldo F, Donázar JA. Phylogeography, genetic structure and diversity in the endangered bearded vulture (<em>Gypaetus barbatus</em>, L.) as revealed by mitochondrial DNA. Mol Ecol. 2004; 13: 371–390. pmid:14717893 <ul class="reflinks"><li><a href="#" data-author="Godoy" data-cit="GodoyJA%2C%20NegroJJ%2C%20HiraldoF%2C%20Don%C3%A1zarJA.%20Phylogeography%2C%20genetic%20structure%20and%20diversity%20in%20the%20endangered%20bearded%20vulture%20%28Gypaetus%20barbatus%2C%20L.%29%20as%20revealed%20by%20mitochondrial%20DNA.%20Mol%20Ecol.%202004%3B%2013%3A%20371%E2%80%93390.%2014717893" data-title="Phylogeography%2C%20genetic%20structure%20and%20diversity%20in%20the%20endangered%20bearded%20vulture%20%28Gypaetus%20barbatus%2C%20L.%29%20as%20revealed%20by%20mitochondrial%20DNA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14717893" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeography%2C+genetic+structure+and+diversity+in+the+endangered+bearded+vulture+%28Gypaetus+barbatus%2C+L.%29+as+revealed+by+mitochondrial+DNA+Godoy+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0177414.ref037" id="pone.0177414.ref037" class="link-target"></a>Pellegrino KCM, Rodrigues MT, Waite AN, Morando M, Yassuda YY, Sites JW Jr. Phylogeography and species limits in the <em>Gymnodactylus darwinii</em> complex (Gekkonidae, Squamata): genetic structure coincides with river systems in the Brazilian Atlantic Forest. Biol J Linn Soc Lond. 2005; 85: 13–26. <ul class="reflinks"><li><a href="#" data-author="Pellegrino" data-cit="PellegrinoKCM%2C%20RodriguesMT%2C%20WaiteAN%2C%20MorandoM%2C%20YassudaYY%2C%20SitesJWJr.%20Phylogeography%20and%20species%20limits%20in%20the%20Gymnodactylus%20darwinii%20complex%20%28Gekkonidae%2C%20Squamata%29%3A%20genetic%20structure%20coincides%20with%20river%20systems%20in%20the%20Brazilian%20Atlantic%20Forest.%20Biol%20J%20Linn%20Soc%20Lond.%202005%3B%2085%3A%2013%E2%80%9326." data-title="Phylogeography%20and%20species%20limits%20in%20the%20Gymnodactylus%20darwinii%20complex%20%28Gekkonidae%2C%20Squamata%29%3A%20genetic%20structure%20coincides%20with%20river%20systems%20in%20the%20Brazilian%20Atlantic%20Forest" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeography+and+species+limits+in+the+Gymnodactylus+darwinii+complex+%28Gekkonidae%2C+Squamata%29%3A+genetic+structure+coincides+with+river+systems+in+the+Brazilian+Atlantic+Forest+Pellegrino+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0177414.ref038" id="pone.0177414.ref038" class="link-target"></a>Grazziotin FG, Monzel M, Echeverrigaray S, Bonatto SL. Phylogeography of the <em>Bothrops jararaca</em> complex (Serpentes: Viperidae): past fragmentation and island colonization in the Brazilian Atlantic Forest. Mol Ecol. 2006; 15: 3969–3982.  pmid:17054497 <ul class="reflinks"><li><a href="#" data-author="Grazziotin" data-cit="GrazziotinFG%2C%20MonzelM%2C%20EcheverrigarayS%2C%20BonattoSL.%20Phylogeography%20of%20the%20Bothrops%20jararaca%20complex%20%28Serpentes%3A%20Viperidae%29%3A%20past%20fragmentation%20and%20island%20colonization%20in%20the%20Brazilian%20Atlantic%20Forest.%20Mol%20Ecol.%202006%3B%2015%3A%203969%E2%80%933982.%20doi%3A%2010.1111%2Fj.1365-294X.2006.03057.x%2017054497" data-title="Phylogeography%20of%20the%20Bothrops%20jararaca%20complex%20%28Serpentes%3A%20Viperidae%29%3A%20past%20fragmentation%20and%20island%20colonization%20in%20the%20Brazilian%20Atlantic%20Forest" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17054497" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeography+of+the+Bothrops+jararaca+complex+%28Serpentes%3A+Viperidae%29%3A+past+fragmentation+and+island+colonization+in+the+Brazilian+Atlantic+Forest+Grazziotin+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0177414.ref039" id="pone.0177414.ref039" class="link-target"></a>Horn A, Roux-Morabito G, Lieutier F, Kerdelhue C. Phylogeographic structure and past history of the circum-Mediterranean species <em>Tomicus destruens</em> Woll. (Coleoptera: Scolytinae). Mol Ecol. 2006; 15: 1603–1615.  pmid:16629814 <ul class="reflinks"><li><a href="#" data-author="Horn" data-cit="HornA%2C%20Roux-MorabitoG%2C%20LieutierF%2C%20KerdelhueC.%20Phylogeographic%20structure%20and%20past%20history%20of%20the%20circum-Mediterranean%20species%20Tomicus%20destruens%20Woll.%20%28Coleoptera%3A%20Scolytinae%29.%20Mol%20Ecol.%202006%3B%2015%3A%201603%E2%80%931615.%20doi%3A%2010.1111%2Fj.1365-294X.2006.02872.x%2016629814" data-title="Phylogeographic%20structure%20and%20past%20history%20of%20the%20circum-Mediterranean%20species%20Tomicus%20destruens%20Woll.%20%28Coleoptera%3A%20Scolytinae%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16629814" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeographic+structure+and+past+history+of+the+circum-Mediterranean+species+Tomicus+destruens+Woll.+%28Coleoptera%3A+Scolytinae%29+Horn+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0177414.ref040" id="pone.0177414.ref040" class="link-target"></a>Finn DS, Blouin MS, Lytle DA. Population genetic structure reveals terrestrial affinities for a headwater stream insect. Freshw Biol. 2007; 52: 1881–1897. <ul class="reflinks"><li><a href="#" data-author="Finn" data-cit="FinnDS%2C%20BlouinMS%2C%20LytleDA.%20Population%20genetic%20structure%20reveals%20terrestrial%20affinities%20for%20a%20headwater%20stream%20insect.%20Freshw%20Biol.%202007%3B%2052%3A%201881%E2%80%931897." data-title="Population%20genetic%20structure%20reveals%20terrestrial%20affinities%20for%20a%20headwater%20stream%20insect" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Population+genetic+structure+reveals+terrestrial+affinities+for+a+headwater+stream+insect+Finn+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0177414.ref041" id="pone.0177414.ref041" class="link-target"></a>Turchetto-Zolet AC, Cruz F, Vendramin GG, Simon MF, Salgueiro F, Margis-Pinheiro M, et al. Large-scale phylogeography of the disjunct Neotropical tree species <em>Schizolobium parahyba</em> (Fabaceae-Caesalpinioideae). Mol Phylogenet Evol. 2012; 65: 174–182.  pmid:22750114 <ul class="reflinks"><li><a href="#" data-author="Turchetto-Zolet" data-cit="Turchetto-ZoletAC%2C%20CruzF%2C%20VendraminGG%2C%20SimonMF%2C%20SalgueiroF%2C%20Margis-PinheiroM%2C%20et%20al.%20Large-scale%20phylogeography%20of%20the%20disjunct%20Neotropical%20tree%20species%20Schizolobium%20parahyba%20%28Fabaceae-Caesalpinioideae%29.%20Mol%20Phylogenet%20Evol.%202012%3B%2065%3A%20174%E2%80%93182.%20doi%3A%2010.1016%2Fj.ympev.2012.06.012%2022750114" data-title="Large-scale%20phylogeography%20of%20the%20disjunct%20Neotropical%20tree%20species%20Schizolobium%20parahyba%20%28Fabaceae-Caesalpinioideae%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22750114" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Large-scale+phylogeography+of+the+disjunct+Neotropical+tree+species+Schizolobium+parahyba+%28Fabaceae-Caesalpinioideae%29+Turchetto-Zolet+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0177414.ref042" id="pone.0177414.ref042" class="link-target"></a>Canestrelli D, Bisconti R, Sacco F, Nascetti G. What triggers the rising of an intraspecific biodiversity hotspot? Hints from the agile frog. Sci Rep. 2014; 4: 5042.  pmid:24853644 <ul class="reflinks"><li><a href="#" data-author="Canestrelli" data-cit="CanestrelliD%2C%20BiscontiR%2C%20SaccoF%2C%20NascettiG.%20What%20triggers%20the%20rising%20of%20an%20intraspecific%20biodiversity%20hotspot%3F%20Hints%20from%20the%20agile%20frog.%20Sci%20Rep.%202014%3B%204%3A%205042.%20doi%3A%2010.1038%2Fsrep05042%2024853644" data-title="What%20triggers%20the%20rising%20of%20an%20intraspecific%20biodiversity%20hotspot%3F%20Hints%20from%20the%20agile%20frog" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24853644" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=What+triggers+the+rising+of+an+intraspecific+biodiversity+hotspot%3F+Hints+from+the+agile+frog+Canestrelli+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0177414.ref043" id="pone.0177414.ref043" class="link-target"></a>Viñas MD, Blanco-Bercial L, Bucklinb A, Verheye H, Bersano JGF, Ceballos S. Phylogeography of the copepod <em>Calanoides carinatus</em> s. l. (Krøyer) reveals cryptic species and delimits <em>C</em>. <em>carinatus</em> s. s. distribution in SW Atlantic Ocean. J Exp Mar Bio Ecol. 2015; 468: 97–104. <ul class="reflinks"><li><a href="#" data-author="Vi%C3%B1as" data-cit="Vi%C3%B1asMD%2C%20Blanco-BercialL%2C%20BucklinbA%2C%20VerheyeH%2C%20BersanoJGF%2C%20CeballosS.%20Phylogeography%20of%20the%20copepod%20Calanoides%20carinatus%20s.%20l.%20%28Kr%C3%B8yer%29%20reveals%20cryptic%20species%20and%20delimits%20C.%20carinatus%20s.%20s.%20distribution%20in%20SW%20Atlantic%20Ocean.%20J%20Exp%20Mar%20Bio%20Ecol.%202015%3B%20468%3A%2097%E2%80%93104." data-title="Phylogeography%20of%20the%20copepod%20Calanoides%20carinatus%20s.%20l.%20%28Kr%C3%B8yer%29%20reveals%20cryptic%20species%20and%20delimits%20C.%20carinatus%20s.%20s.%20distribution%20in%20SW%20Atlantic%20Ocean" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeography+of+the+copepod+Calanoides+carinatus+s.+l.+%28Kr%C3%B8yer%29+reveals+cryptic+species+and+delimits+C.+carinatus+s.+s.+distribution+in+SW+Atlantic+Ocean+Vi%C3%B1as+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0177414.ref044" id="pone.0177414.ref044" class="link-target"></a>Ware J, Litman JE, Klass KD, Spearman LA. Relationships among the major lineages of Dictyoptera: the effect of outgroup selection on dictyopteran tree topology. Syst Entomol. 2008; 33: 429–450. <ul class="reflinks"><li><a href="#" data-author="Ware" data-cit="WareJ%2C%20LitmanJE%2C%20KlassKD%2C%20SpearmanLA.%20Relationships%20among%20the%20major%20lineages%20of%20Dictyoptera%3A%20the%20effect%20of%20outgroup%20selection%20on%20dictyopteran%20tree%20topology.%20Syst%20Entomol.%202008%3B%2033%3A%20429%E2%80%93450." data-title="Relationships%20among%20the%20major%20lineages%20of%20Dictyoptera%3A%20the%20effect%20of%20outgroup%20selection%20on%20dictyopteran%20tree%20topology" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Relationships+among+the+major+lineages+of+Dictyoptera%3A+the+effect+of+outgroup+selection+on+dictyopteran+tree+topology+Ware+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0177414.ref045" id="pone.0177414.ref045" class="link-target"></a>Luo AR, Zhang YZ, Qiao HJ, Shi WF, Murphy RW, Zhu CD. Outgroup selection in tree reconstruction: a case study of the family Halictidae (Hymenoptera: Apoidea). Acta Entomol Sin. 2010; 53: 192–201. <ul class="reflinks"><li><a href="#" data-author="Luo" data-cit="LuoAR%2C%20ZhangYZ%2C%20QiaoHJ%2C%20ShiWF%2C%20MurphyRW%2C%20ZhuCD.%20Outgroup%20selection%20in%20tree%20reconstruction%3A%20a%20case%20study%20of%20the%20family%20Halictidae%20%28Hymenoptera%3A%20Apoidea%29.%20Acta%20Entomol%20Sin.%202010%3B%2053%3A%20192%E2%80%93201." data-title="Outgroup%20selection%20in%20tree%20reconstruction%3A%20a%20case%20study%20of%20the%20family%20Halictidae%20%28Hymenoptera%3A%20Apoidea%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Outgroup+selection+in+tree+reconstruction%3A+a+case+study+of+the+family+Halictidae+%28Hymenoptera%3A+Apoidea%29+Luo+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0177414.ref046" id="pone.0177414.ref046" class="link-target"></a>Ferreri M, Qu W, Han B. Phylogenetic networks: A tool to display character conflict and demographic history. Afr J Biotechnol.2011; 10: 12799–12803. <ul class="reflinks"><li><a href="#" data-author="Ferreri" data-cit="FerreriM%2C%20QuW%2C%20HanB.%20Phylogenetic%20networks%3A%20A%20tool%20to%20display%20character%20conflict%20and%20demographic%20history.%20Afr%20J%20Biotechnol.2011%3B%2010%3A%2012799%E2%80%9312803." data-title="Phylogenetic%20networks%3A%20A%20tool%20to%20display%20character%20conflict%20and%20demographic%20history" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogenetic+networks%3A+A+tool+to+display+character+conflict+and+demographic+history+Ferreri+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0177414.ref047" id="pone.0177414.ref047" class="link-target"></a>Ramos-Onsins SE, Rozas J. Statistical properties of new neutrality tests against population growth. Mol Biol Evol. 2002; 19: 2092–2100. pmid:12446801 <ul class="reflinks"><li><a href="#" data-author="Ramos-Onsins" data-cit="Ramos-OnsinsSE%2C%20RozasJ.%20Statistical%20properties%20of%20new%20neutrality%20tests%20against%20population%20growth.%20Mol%20Biol%20Evol.%202002%3B%2019%3A%202092%E2%80%932100.%2012446801" data-title="Statistical%20properties%20of%20new%20neutrality%20tests%20against%20population%20growth" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12446801" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Statistical+properties+of+new+neutrality+tests+against+population+growth+Ramos-Onsins+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0177414.ref048" id="pone.0177414.ref048" class="link-target"></a>Hartl DL, Clark AG. Princípios de genética de populações. 4th ed. Porto Alegre: Artmed; 2010. <ul class="find-nolinks"></ul></li></ol></div>

