



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>The ubiquitously expressed 14-3-3 family of proteins is evolutionarily conserved from yeast to mammals. Their involvement in humoral and cellular immune responses is emerging through studies in drosophila and humans. In humans, a select group of 14-3-3 isoforms are antigenic; however the determinants of their antigenicity are not known. Here, we show that although mammalian 14-3-3 proteins are mostly conserved, subtle differences between their isoforms may give rise to their antigenicity. We observed syntenic relations among all the isoforms of 14-3-3 for mammals, but not with that of birds or amphibians. However, the parasitic 14-3-3 isoforms, which have known antigenic properties, show unique sequence, structure and evolution compared to the human 14-3-3. Moreover we report, for the first time the existence of a bacterial 14-3-3 protein. Contrary to the parasitic isoforms, both bacterial and yeast 14-3-3 exhibited significant homology with mammalian 14-3-3 in protein sequence as well as structure. Furthermore, a human 14-3-3 inhibitor caused significant killing of <em>Candida albicans</em>, which could be due to the inhibition of the structurally similar yeast homologue of 14-3-3, BMH, which is essential for its life cycle. Overall, our bioinformatic analysis combined with the demonstration of a novel antifungal role of a peptide inhibitor of human 14-3-3 indicates that the sequences and structural similarities between the mammalian, bacterial and fungal proteins are likely determinants of the antigenic nature of these proteins. Further, we propose that molecular mimicry triggered by microbial infections with either yeast or bacteria may contribute to the antigenic role of human 14-3-3.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>McGowan JE, Kratch J, Chattopadhyay S, Joe B, Conti HR, Chakravarti R (2017) Bioinformatic analysis reveals new determinants of antigenic 14-3-3 proteins and a novel antifungal strategy. PLoS ONE 12(12):
           e0189503.
        
        https://doi.org/10.1371/journal.pone.0189503</p><p><strong>Editor: </strong>Joy Sturtevant, Louisiana State University, UNITED STATES</p><p><strong>Received: </strong>August 24, 2017; <strong>Accepted: </strong>November 28, 2017; <strong>Published: </strong> December 12, 2017</p><p><strong>Copyright: </strong> © 2017 McGowan et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>All relevant data are within the paper and its Supporting Information file.</p><p><strong>Funding: </strong>This work was supported by the Scientist Development Grant (15SDG2308025)-American Heart Association and the Start Up Funds- University of Toledo. The funders had no role in study design, design, data collection and analysis, decision to publish or preparation of the manuscript.</p><p><strong>Competing interests: </strong> Provisional patent (USPTO) file number: 62/583,697 to Ritu Chakravarti and Heather R. Conti. This does not alter our adherence to PLOS ONE policies on sharing data and materials.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>The 14-3-3 family of proteins is evolutionarily conserved in eukaryotes ranging from yeast to mammals. They are well-studied for their role as adaptor proteins in a multitude of signaling pathways leading to cellular proliferation, differentiation and death [<a href="#pone.0189503.ref001" class="ref-tip">1</a>]. Dysregulation of function or expression of several isoforms has been reported in multiple human diseases [<a href="#pone.0189503.ref002" class="ref-tip">2</a>–<a href="#pone.0189503.ref006" class="ref-tip">6</a>]. Among these, increased expression of 14-3-3 has been observed in body fluids of patients with autoimmune diseases such as rheumatoid arthritis and multiple sclerosis, suggesting an immunological function for 14-3-3 proteins[<a href="#pone.0189503.ref007" class="ref-tip">7</a>, <a href="#pone.0189503.ref008" class="ref-tip">8</a>]. This is also evident in other organisms, e.g. the loss of 14-3-3ε in Drosophila is linked with defects in exocytosis of immune mediators leading to enhanced susceptibility to bacterial infection [<a href="#pone.0189503.ref009" class="ref-tip">9</a>, <a href="#pone.0189503.ref010" class="ref-tip">10</a>], 14-3-3ζ is associated with phagocytosis and resistance against <em>S</em>. <em>aureus</em> in both drosophila and zebrafish [<a href="#pone.0189503.ref011" class="ref-tip">11</a>], and 14-3-3 proteins in plants regulate the pathogenic interactions [<a href="#pone.0189503.ref012" class="ref-tip">12</a>]. These relatively new data are interesting to further investigate how 14-3-3 functions contribute to immune responses. In this regard, it is interesting to note that antigenic properties of 14-3-3 have recently been identified. Antibodies against 14-3-3 are reported in autoimmune, cancer and infectious disease [<a href="#pone.0189503.ref013" class="ref-tip">13</a>–<a href="#pone.0189503.ref018" class="ref-tip">18</a>]; 14-3-3η is antigenic in rheumatoid arthritis [<a href="#pone.0189503.ref015" class="ref-tip">15</a>] and 14-3-3ζ is antigenic in cancer [<a href="#pone.0189503.ref014" class="ref-tip">14</a>, <a href="#pone.0189503.ref018" class="ref-tip">18</a>]. We and others have reported the increased antigenicity of 14-3-3ζ and 14-3-3ε in aneurysmal inflamed thoracic aorta [<a href="#pone.0189503.ref013" class="ref-tip">13</a>, <a href="#pone.0189503.ref019" class="ref-tip">19</a>]. In contrast to these observations, in aneurysmal aortitis, 3 isoforms (beta, tau and gamma) of 14-3-3 family that are found to be upregulated, are not antigenic, suggesting that antigenicity could be an independent function in some diseases or individual isoforms may have different roles in different diseases [<a href="#pone.0189503.ref013" class="ref-tip">13</a>]. Therefore, more information is needed to better understand the factors dictating the antigenic properties of 14-3-3. This in turn may guide the development of specific diagnostic assays for 14-3-3 and its isoforms as clinical markers of autoimmune diseases. We approached this issue with a bioinformatic perspective to compare and contrast the evolutionary structure-function relationships of 14-3-3. There have been several reports of comparative analyses of 14-3-3 family proteins across the species [<a href="#pone.0189503.ref020" class="ref-tip">20</a>–<a href="#pone.0189503.ref024" class="ref-tip">24</a>], but our efforts are directed to specifically understand its new antigenic function.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Our results revealed high degree of structural and phylogenetic similarities between the human, yeast and bacterial 14-3-3 but not with other organisms such as protozoans. Some isoforms of 14-3-3 (including ζ and ε) are better known for their antigenic function [<a href="#pone.0189503.ref013" class="ref-tip">13</a>–<a href="#pone.0189503.ref015" class="ref-tip">15</a>, <a href="#pone.0189503.ref018" class="ref-tip">18</a>, <a href="#pone.0189503.ref019" class="ref-tip">19</a>]; therefore, we performed comparative analyses on these isoforms, primarily ζ. Thus, molecular mimicry triggered by microbial infections may contribute to the antigenic role of human 14-3-3. Furthermore, by assessing the functional similarity in humans and yeast, our study reports a novel antifungal role of a peptide inhibitor of human 14-3-3.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Variability in highly conserved mammalian 14-3-3 proteins"></a>
<h3>Variability in highly conserved mammalian 14-3-3 proteins</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>The seven human isoforms of 14-3-3 (accession number in <a href="#pone.0189503.s006">S1 Table</a>) were aligned to identify regions of conservation and variation. Overall, there was up to 80% homology between the isoforms (<a href="#pone.0189503.s001">S1 Fig</a>). Logo analysis, which shows the highly conserved residues with a greater height [<a href="#pone.0189503.ref025" class="ref-tip">25</a>], also revealed significant conservation among family; however, there were six distinct regions of variation (V1-V6, underlined in <a href="#pone-0189503-g001">Fig 1A</a>). The notable variability defined by ≤50% similarity of residues between proteins was confirmed by the Logo analysis and sequence alignment (<a href="#pone.0189503.s001">S1 Fig</a>). Structurally these variable regions corresponded to the unstructured regions at the ends of helices of the protein structure of 14-3-3 (<a href="#pone-0189503-g001">Fig 1B</a>).</p>
<a class="link-target" id="pone-0189503-g001" name="pone-0189503-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.g001" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Bioinformatic analyses identified six variable regions in human 14-3-3 isoforms.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>Human 14-3-3 isoforms were aligned using the MUSCLE program and analyzed at WebLogo to identify the conservation of protein sequences. Areas of variability are underlined and are numbered as V1 to V6. (B) Structure of 14-3-3 with variable regions in solid blue color is shown. Human 14-3-3ζ (PDB ID 5D2D) was used as a template to overlay the variable domains. N-terminal methionine is indicated in light blue. Variable regions primarily located at the end of helices and in the unstructured regions.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.g001">
              https://doi.org/10.1371/journal.pone.0189503.g001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>In comparison, each human isoform showed more conservation (&gt;99%) when compared to its mammalian orthologs (<a href="#pone-0189503-g002">Fig 2A</a>). Single amino acid substitution was observed in some isoforms such as 14-3-3ζ of rat and mouse in comparison to human; however, this did not impact the structure of protein (<a href="#pone-0189503-g002">Fig 2B</a>). Similarly, a different single amino acid substitution was observed when human 14-3-3ζ compared with 14-3-3 of hamster.</p>
<a class="link-target" id="pone-0189503-g002" name="pone-0189503-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.g002" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> High degree of conservation for mammalian 14-3-3.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig2.caption1.p1" name="article1.body1.sec2.sec1.fig2.caption1.p1" class="link-target"></a><p>(A) 14-3-3ζ protein in mammals (human, rat, mouse, bat, hamster, marmoset, and yak) show 99% conservation of sequence. Only a single amino acid difference (alanine to proline or threonine to alanine) was observed between human and rodent (mouse/rat or hamster) 14-3-3ζ. Accession numbers used for alignment are as follows—human: NP_001129171, rat: NP_037143.2, mouse: NP_035870.1, bat: XP_005879335.1, hamster: XP_003504372.1, marmoset: XP_008981647.1, yak: XP_005887072.1. (B) Structure of rat 14-3-3ζ protein <strong>(</strong>NP_037143.2) was simulated based on the human 14-3-3ζ structure (NP_001129171, Protein Data Bank ID-5WXN). QMEAN calculated was 0.93. Human protein (PDB ID 5D2D) is shown in blue, while rat 14-3-3ζ is shown in white. P112 of rat is shown in red color.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.g002">
              https://doi.org/10.1371/journal.pone.0189503.g002</a></p></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3" class="link-target"></a><p>To understand the basis of antigenicity in case of highly conserved protein sequences, we investigated the local synteny which serves as a tool to survey the extent of conservation of genetic loci during evolution[<a href="#pone.0189503.ref026" class="ref-tip">26</a>]. All seven isoforms of 14-3-3 in human, and so in rat and mouse, albeit, segregating on different chromosomes, show significant conserved synteny in the genomic regions encompassing the gene coding for 14-3-3 across all of their genomes (<a href="#pone-0189503-t001">Table 1</a>, <a href="#pone.0189503.s007">S2 Table</a>). But synteny was not observed when chromosomal regions were compared with Xenopus or chicken. This suggested that the 14-3-3 locus diverged significantly since the evolution of mammals from amphibians and birds.</p>
<a class="link-target" id="pone-0189503-t001" name="pone-0189503-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.t001" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Chromosomal localization of 14-3-3ζ across species.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.t001">
              https://doi.org/10.1371/journal.pone.0189503.t001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Comparisons of parasitic and human 14-3-3"></a>
<h3>Comparisons of parasitic and human 14-3-3</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Because the antigenicity of protozoan 14-3-3 proteins is well studied [<a href="#pone.0189503.ref019" class="ref-tip">19</a>, <a href="#pone.0189503.ref027" class="ref-tip">27</a>, <a href="#pone.0189503.ref028" class="ref-tip">28</a>], we compared protozoan 14-3-3 with its human counterpart. We identified three protein sequences using BLAST searching of the <em>T</em>. <em>gondii</em> genome (GCA_000006565.2); Tg1, Tg2 and Tg3, (TGME49_263090, TGME49_227952, TGME49_269960 respectively) that are similar to the 14-3-3 protein family. All the T. <em>gondii</em> isoforms, Tg1 (323 aa), Tg2 (476 aa) and Tg3 (422 aa) were larger compared to the human 14-3-3ε (255 aa) and human 14-3-3ζ (245 aa). All three isoforms in <em>T</em>. <em>gondii</em>, when aligned with human isoforms (ζ, ε, γ) showed an extra N-terminus sequence that varied between 62–156 amino acids, however, the C-terminus was relatively conserved (<a href="#pone-0189503-g003">Fig 3A</a>). The tail at the C-terminus represented an unstructured region that aligned with the variable region 6 (V6) of the 14-3-3 proteins. The extra 62 amino acid sequence on the N-terminus of Tg1 was found to be overall hydrophilic with notable presence of serine residues (9 out of 62aa). There are four putative phosphorylation sites (S10, S14, T15, T57) with the motifs KKsT and TmaE for CAMP (cAMP and cGMP-dependent protein kinase) and CK2 (Casein Kinase II) respectively. Modeling this sequence in RaptorX with a p-value = 1.62e-07, which is suggestive of highly reliable structure, showed the presence of three beta sheets (<a href="#pone-0189503-g003">Fig 3B</a>). Further, the homology search using <em>T</em>. <em>gondii</em> Tg1 protein (XP_002365409.1) in the non-redundant protein database (blastp 2.2.29) showed several proteins from <em>Neospora caninum</em>, <em>Plasmodium falciparum</em> and <em>Plutella xylostella</em> to be 74–77% homologous. This suggests that the overall 14-3-3 family in the protozoan has been well conserved and is different from mammalian proteins. Overlaying the structure of Tg1 with the human 14-3-3ε indicated that the most significant regions of variations between the protein structures are traceable on both the N and C-termini (<a href="#pone-0189503-g003">Fig 3C</a>).</p>
<a class="link-target" id="pone-0189503-g003" name="pone-0189503-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.g003" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Significant difference lies between human and <em>T</em>. <em>gondii</em> 14-3-3 proteins.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>(A) Sequences of human 14-3-3ζ (HsZ, NP_001129171), 14-3-3ε (HsEP, NP_006752.1), and 14-3-3γ (HsG, NP_036611.2) aligned with <em>T</em>. <em>gondii</em> 14-3-3 proteins (Tg1 = XP_002365409.1, Tg2 = XP_018636600.1 Tg3 = XP_018636082.1) show areas of conservation and significant variation. An extra 62 amino acid at the N-terminal region in Tg1, and even longer N-terminal region in the case of other two family members was identified. (B) <em>T</em>. <em>gondii</em> 14-3-3 (Tg1) structure was simulated by Raptor X, using <em>C</em>. <em>parvum</em> 14-3-3 (PDB ID 2NPM) as a template. P-value of the structure is 1.62e-07. The 62aa N-terminal region is shown in red, and the rest of the protein is shown in grey. (C) Structure alignment of <em>T</em>. <em>gondii</em> 14-3-3 protein (Tg1) and human 14-3-3ε (PDB ID 2BR9). <em>T</em>. <em>gondii</em> protein is shown in white, while human protein is in blue. <em>T</em>. <em>gondii</em> structure uses the same model as in Fig 3B.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.g003">
              https://doi.org/10.1371/journal.pone.0189503.g003</a></p></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>In contrast to our observations with <em>T</em>. <em>gondii</em>, comparisons of the human 14-3-3 with that of parasitic worms such as <em>Schistosoma mansoni</em> showed that all six proteins of the 14-3-3 family in <em>S</em>. <em>mansoni</em> were significantly similar to human 14-3-3 isoforms. Only three isoforms (SMP 3, 4 and 6) had extra sequences at N-termini, whereas two isoforms (SMP2 and 5) were shorter than human isoforms (<a href="#pone.0189503.s002">S2 Fig</a>).</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3" class="link-target"></a><p>Further, because antibodies against 14-3-3 are observed in <em>Angiostrongylus cantonensis</em> infection of the human kidney [<a href="#pone.0189503.ref027" class="ref-tip">27</a>], we performed blast search of the <em>A</em>. <em>cantonensis</em> genome with the nucleotide sequence of human 14-3-3. Interestingly, our analyses revealed a sequence with 80% identity to human 14-3-3ζ isoform. Both proteins showed significant homology to each other with and similarity in structural motifs (<a href="#pone.0189503.s003">S3 Fig</a>). A similar search conducted with the sequences of <em>E</em>. <em>multilocularis</em> also resulted in a 249 amino acid long sequence of <em>E</em>. <em>multilocularis</em> 14-3-3 (CD170865), which, when aligned with the human 14-3-3, showed maximum similarity to the epsilon isoform of 14-3-3 (NP_006752.1, data not shown).</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Comparisons of eukaryotic (yeast) 14-3-3 with human 14-3-3ε"></a>
<h3>Comparisons of eukaryotic (yeast) 14-3-3 with human 14-3-3ε</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Much like in higher eukaryotes, isoforms of 14-3-3 in yeast have been shown to perform wide variety of functions[<a href="#pone.0189503.ref020" class="ref-tip">20</a>–<a href="#pone.0189503.ref022" class="ref-tip">22</a>]. Unlike in human and parasite, the antigenic roles of these isoforms have not been yet reported, but they are implicated in several important functions, including cell survival [<a href="#pone.0189503.ref029" class="ref-tip">29</a>], virulence [<a href="#pone.0189503.ref030" class="ref-tip">30</a>], regulation of gene transcription[<a href="#pone.0189503.ref031" class="ref-tip">31</a>] and sporulation[<a href="#pone.0189503.ref032" class="ref-tip">32</a>]. We compared the two known 14-3-3 isoforms reported in <em>S</em>. <em>cerevisiae</em> (BMH1 and BMH2), in <em>S</em>. <em>pombe</em> (Rad 24 and Rad25) and one isoform reported in <em>C</em>. <em>albicans (</em>BMH), with human 14-3-3. All the yeast homologs closely resembled human 14-3-3ε, as also reported previously [<a href="#pone.0189503.ref022" class="ref-tip">22</a>]. BMH1 of <em>S</em>. <em>cerevisiae</em> exhibited 75% identity and 84% similarity, while BMH2 had 74% identity and 83% similarity with human 14-3-3ε. Similarly, BMH of <em>C</em>. <em>albicans</em> had 75% identity and 83% similarity with human 14-3-3ε (<a href="#pone-0189503-g004">Fig 4A</a>). Further, 3D structural comparison of BMH from <em>C</em>. <em>albicans</em> with human 14-3-3ε revealed an almost completely matched overlay with high QMEAN scores (QMEAN4 = 0.75 and QMEAN6 = 0.73, <a href="#pone-0189503-g004">Fig 4B</a>).</p>
<a class="link-target" id="pone-0189503-g004" name="pone-0189503-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.g004" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> 14-3-3 of <em>Candida albicans</em> and human has similar sequence and structure.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>Sequence alignment of <em>C</em>. <em>albicans</em> 14-3-3 (BMH) with human ζ and ε isoforms revealed 83% similarity with human ε and 79% with ζ protein. <em>Candida</em> BMH protein was simulated using <em>L</em>. <em>thermotolerans</em> 14-3-3 protein (PDB ID 5LX2) as a template. QMEAN was 0.80. <em>Candida</em> protein is shown in white, while human 14-3-3ε protein is shown in blue.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.g004">
              https://doi.org/10.1371/journal.pone.0189503.g004</a></p></div></div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Presence of 14-3-3 in bacteria"></a>
<h3>Presence of 14-3-3 in bacteria</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>Currently, there is no report of the presence of 14-3-3 in bacteria. However, when the sequences of human 14-3-3ε and 14-3-3ζ were blasted against bacterial databases, several hypothetical proteins with up to 50–100% similarities were identified. These proteins were identified in <em>Gammaproteobacteria bacterium</em> (2W06, WP_083217732.1), <em>Acinetobacter baumannii</em> (WP_082181584.1), <em>Streptomyces agglomeratus</em> (WP_069926675.1), and <em>Pseudonocardia sp</em>. (OJY47151.1). Furthermore, the similarity in sequence as well as structure between these bacterial proteins, in particular <em>Gammaproteobacteria</em> and human 14-3-3ζ was striking (<a href="#pone-0189503-g005">Fig 5A and 5B</a>). The QMEAN scores of simulated structure of bacterial 14-3-3 protein were 0.74 and 0.73 for QMEAN4 and QMEAN6 respectively. Our analyses clearly indicate the existence of 14-3-3 protein in bacteria.</p>
<a class="link-target" id="pone-0189503-g005" name="pone-0189503-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.g005" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Sequence and structure of newly identified Gammaproteobacterial 14-3-3 with its human counterpart.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p>(A) Sequence alignment of a hypothetical protein of <em>Gammaproteobacteria</em> (WP_083217732.1) with the human 14-3-3ζ and 14-3-3ε isoforms shows 90 and 82% similarity respectively. (B) Bacterial protein was simulated based upon the human 14-3-3ζ structure (NP_001129171) <strong>(</strong>Protein Data Bank ID 5WXN). QMEAN was 1.04. Structural alignment of a hypothetical Gammaproteobacterial 14-3-3 with the human ζ isoform (PDB ID 5D2D) shows complete overlap. Bacterial protein is shown in white color and the human protein is shown in blue.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.g005">
              https://doi.org/10.1371/journal.pone.0189503.g005</a></p></div></div>

<div id="section5" class="section toc-section"><a id="sec007" name="sec007" class="link-target" title="Phylogenetic analyses reveal parasitic 14-3-3 proteins as the mostly distinct"></a>
<h3>Phylogenetic analyses reveal parasitic 14-3-3 proteins as the mostly distinct</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>There is a significant degree of sequence conservation in the 14-3-3 family from unicellular to multicellular organisms including humans, yet only certain proteins are reported as being antigenic, is intriguing. To gain further insight into the putative antigenicity, we performed a phylogenetic analysis on the 14-3-3 proteins from phylum Chordata including mammals (<em>Homo sapiens</em> and <em>Mus musculus</em>), amphibian (<em>Xenopus tropicalis</em>), fish (<em>Danio rerio</em>), birds (<em>Gallus gallus</em>), Arthropoda (<em>Drosophila melanogaster</em>) and ascidian (<em>Ciona intestinalis</em>), plant (<em>Oryza sativa japonica</em>), several parasites from phylum including Apicomplexa (<em>T</em>. <em>gondii</em>), Platyhelminthes (<em>S</em>. <em>mansoni</em>) etc., Ascomycota (<em>S</em>. <em>cerevisiae</em>, <em>C</em>. <em>albicans</em>), and bacteria (<em>Gammaproteobacteria</em>). The phylogenetic study using PhyML program with bootstrap analysis with a value of 100, suggested that most isoforms of 14-3-3 families in mammals had an independent node. Most isoforms from human, plant, amphibian and bird tended to cluster together, but two of the plant isoforms (OsG and OsH) had a separate node. However, most parasite (<em>T</em>. <em>gondii</em>, <em>S</em>. <em>mansoni</em>), fly (<em>D</em>. <em>melanogaster</em>) and yeast (<em>S</em>. <em>cerevisiae</em>) 14-3-3 proteins maintained a separate node. In accordance to the previous report [<a href="#pone.0189503.ref020" class="ref-tip">20</a>], our results also suggest that the epsilon isoform of 14-3-3 proteins probably separated from the rest of the family during the early phases of evolution, but remained closer to the 14-3-3 of parasites, yeast and ascidians such as <em>C</em>. <em>intestinalis</em>, and, therefore, may be considered as ancestral protein of the family. The most striking observation of this analysis was that most of the isoforms from parasites (particularly SMP6, Tg2 and Tg3) had their own independent node. The functional significance of this separation from 14-3-3 in all other organisms is yet to be investigated.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="Human 14-3-3 inhibitor is a novel antifungal agent"></a>
<h3>Human 14-3-3 inhibitor is a novel antifungal agent</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>All of our above described analyses based on protein sequence, structure and phylogenetic status provided clues for putative, yet undiscovered, structure-function relationships of the 14-3-3 proteins. The phylogenetic analyses, wherein the positioning of the <em>Candida</em> BMH very close to human 14-3-3ε (<a href="#pone-0189503-g006">Fig 6A and 6B</a>), led to our hypothesis that the structural similarity between <em>Candida</em> BMH and human 14-3-3ε may indicate that a known inhibitor of human 14-3-3 would also inhibit <em>Candida</em> BMH. Because <em>Candida</em> BMH is known to be important for the survival of <em>C</em>. <em>albicans</em>, we predicted that such an inhibition would arrest the growth of <em>Candida</em>. Therefore, we tested the effect of R18, a peptide inhibitor of human 14-3-3, on the survival of <em>C</em>. <em>albicans</em>. Incubation of <em>C</em>. <em>albicans</em> with R18 resulted in a dose-dependent increase in the killing of <em>C</em>. <em>albicans</em> (<a href="#pone-0189503-g007">Fig 7A</a>, <a href="#pone.0189503.s004">S4 Fig</a>). Notably, the antifungal effect of R18 on <em>C</em>. <em>albicans</em> was as pronounced as a known inhibitor of <em>C</em>. <em>albicans</em>, β-defensin-3 (BD-3) [<a href="#pone.0189503.ref033" class="ref-tip">33</a>, <a href="#pone.0189503.ref034" class="ref-tip">34</a>]. Similarly, in the infection assays using oral epithelial cells, we observed effective inhibition of <em>C</em>. <em>albicans</em> survival when BV02, another 14-3-3 inhibitor [<a href="#pone.0189503.ref035" class="ref-tip">35</a>], was applied (<a href="#pone-0189503-g007">Fig 7B</a>). Furthermore, we tested the 14-3-3 inhibitor for its effect on the <em>S</em>. <em>cerevisiae</em> which also encodes two isoforms of 14-3-3 (BMH1 and BMH2) that are about 75% identical to human 14-3-3ε. These two isoforms are redundant in their function; however, the loss of BMH1 causes delays in cell cycle [<a href="#pone.0189503.ref036" class="ref-tip">36</a>]. In phylogenetic analysis, we found all yeast 14-3-3 proteins were clustered close to each other and to human 14-3-3ε. Inspite of conserved residues that are needed for binding to R18 peptide to 14-3-3, incubation of <em>S</em>. <em>cerevisiae</em> with R18 did not result in significant killing (<a href="#pone.0189503.s004">S4 Fig</a>).</p>
<a class="link-target" id="pone-0189503-g006" name="pone-0189503-g006"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.g006" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 6. </span> Phylogeny tree of 14-3-3 family suggests parasitic 14-3-3 diverged early on.</div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1" class="link-target"></a><p>Multiple sequence alignment from MUSCLE was used to create phylogeny tree using PhyML program with bootstrap analysis (value of 100) at <a href="http://www.phylogeny.fr">www.phylogeny.fr</a>. (A) A cladogram tree style with branch support value at each node is shown. Common node for <em>Candida</em> BMH and human epsilon proteins is shown by asterisk. (B) Circular tree emphasizes the early separation of parasitic proteins (Tg2, Tg3 and SmP6) from the rest of 14-3-3 proteins that are clustered together.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.g006">
              https://doi.org/10.1371/journal.pone.0189503.g006</a></p></div><a class="link-target" id="pone-0189503-g007" name="pone-0189503-g007"></a><div class="figure" data-doi="10.1371/journal.pone.0189503.g007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0189503.g007" data-doi="info:doi/10.1371/journal.pone.0189503" data-uri="info:doi/10.1371/journal.pone.0189503.g007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0189503.g007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0189503.g007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0189503.g007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0189503.g007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 7. </span> Human 14-3-3 inhibitor kills <em>C</em>. <em>albicans</em>.</div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig2.caption1.p1" name="article1.body1.sec2.sec6.fig2.caption1.p1" class="link-target"></a><p>(A) <em>Candida</em> in exponential growth phase were subjected to 200nM of BD3 (β-defensin-3) or different doses of 14-3-3 inhibitor (R18) for 1hour. Inhibitor was then removed and cell survival was calculated as described in the method section. Significant (* p = 0.0053) cell death was observed in case of R18 treated set. (B) Incubation of TR146 cells with <em>Candida</em> for 1 hour in the presence of 1μM of BV02, another 14-3-3 inhibitor, resulted in statistically significant (*p = 0.012) reduction in colony forming units.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0189503.g007">
              https://doi.org/10.1371/journal.pone.0189503.g007</a></p></div></div>
</div>

<div id="section3" class="section toc-section"><a id="sec009" name="sec009" data-toc="sec009" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>Global bioinformatic approaches to compare and contrast the similarities and differences between proteins are widely applied to investigate the antigenic nature of proteins [<a href="#pone.0189503.ref037" class="ref-tip">37</a>]. Like many other autoantigens, the basis for the antigenicity of 14-3-3 is unknown. To explore a structure-function basis for the antigenic properties of 14-3-3 proteins, we utilized a bioinformatics based approach. In the mammalian isoforms, we identified 6 regions of variability, but these regions were highly conserved among orthologs. Phylogenetic analysis suggested that based on the sequence, the 14-3-3 family may be classified as epsilon-like and non-epsilon like, isoforms [<a href="#pone.0189503.ref020" class="ref-tip">20</a>]. Earlier report on the 14-3-3 protein from <em>G</em>. <em>duodenalis</em>, a parasite from phylum Metamonada, also suggested its closeness to human 14-3-3ε [<a href="#pone.0189503.ref038" class="ref-tip">38</a>]. Among the parasitic proteins, 14-3-3 from <em>T</em>. <em>gondii</em> in particular, showed unique sequences and structures, which were significantly different from that in humans. In addition to significant variation within the overlapping regions, Tg1-3 isoforms possess the additional N-terminus sequences that were strikingly different from that in mammalian 14-3-3. Our report has, for the first time, uncovered the existence of bacterial 14-3-3 in the <em>Gammaproteobacteria</em> genome, with close resemblance to human 14-3-3ζ. Also, 14-3-3 of the yeast origin showed significant similarity with the human epsilon and zeta isoforms [<a href="#pone.0189503.ref021" class="ref-tip">21</a>, <a href="#pone.0189503.ref022" class="ref-tip">22</a>]. To evaluate the significance of our bioinformatic analyses, we hypothesized that structurally similar regions in two different species could be exploited for their functional interactions with known inhibitors. 14-3-3 inhibitors have been studied for promoting cell death in cancerous cells [<a href="#pone.0189503.ref035" class="ref-tip">35</a>, <a href="#pone.0189503.ref039" class="ref-tip">39</a>], we noticed striking similarity between the human and yeast 14-3-3 in the key residues responsible for binding to R18 peptide, that led us to demonstrate a novel re-purposing of R18 as an antifungal agent. This was further aided by the specificity in the inhibition of <em>Candida</em> and not S. <em>cerevisiae</em>, another closely related family of yeast. Although BMH1 and BMH2 are redundant proteins that together are essential for <em>S</em>. <em>cerevisiae</em> life cycle, knockout of either of them results in delayed cell cycle [<a href="#pone.0189503.ref021" class="ref-tip">21</a>, <a href="#pone.0189503.ref022" class="ref-tip">22</a>, <a href="#pone.0189503.ref036" class="ref-tip">36</a>]. It is unknown whether the peptide inhibitor (R18) differentially inhibits their activities. Our strategy of short-term (1 hour) inhibition of yeast 14-3-3 may also have contributed to the observed effect. Future studies will be required to specifically understand these differences.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>In the case of autoimmune aortitis, the trigger of disease remains unknown. In-spite of several attempts, a microbial involvement in autoimmune aortitis remains a question [<a href="#pone.0189503.ref040" class="ref-tip">40</a>, <a href="#pone.0189503.ref041" class="ref-tip">41</a>]. The presence of self-antigens that mimic bacterial pathogens has been suggested in several autoimmune diseases. This includes mycobacterial HSP in Behçet syndrome, HLA-B27 with several gram-negative bacteria (<em>Klebsiella pneumonia</em>, <em>Shigella</em> etc.) in spondyloarthropathies etc.[<a href="#pone.0189503.ref042" class="ref-tip">42</a>]. Specific reports suggest correlated presence of autoantibody with <em>T</em>. <em>gondii</em> infection such as in giant cell arteritis (anti-14-3-3 IgG) or Wegener’s granulomatosis [<a href="#pone.0189503.ref019" class="ref-tip">19</a>, <a href="#pone.0189503.ref043" class="ref-tip">43</a>]. This raised the possibility that parasitic 14-3-3 can serve as an antigen in vasculitides through molecular mimicry. Our bioinformatic studies revealed limited commonalities and significant variations between the human and <em>T</em>. <em>gondii</em> 14-3-3 proteins in both sequence and structure. This knowledge can be utilized to distinguish the antibodies against human and <em>T</em>. <em>gondii</em> 14-3-3 proteins. This leads to our speculation that protozoan (<em>T</em>. <em>gondii</em>) 14-3-3 may not be sufficient to trigger a molecular mimicry-based autoantigenic function in humans. Similarly, we observed that in-spite of lost synteny between mammals and birds, the isoforms were still clustered together in the phylogenetic analyses, thereby suggesting limited use of synteny in understanding antigenic function of 14-3-3. In contrast, the existence of a new bacterial form of 14-3-3, which is similar to the human 14-3-3, leads us to speculate that the antigenic 14-3-3 in autoimmune aortitis may be connected with bacterial infections (<a href="#pone.0189503.s005">S5 Fig</a>).</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>Along these lines, our phylogenetic analysis further showed the relative closeness between the human and <em>C</em>. <em>albicans</em> 14-3-3 [<a href="#pone.0189503.ref021" class="ref-tip">21</a>, <a href="#pone.0189503.ref022" class="ref-tip">22</a>]. Although antigenic 14-3-3 of <em>C</em>. <em>albicans</em> is unknown, many human diseases with antigenic 14-3-3, such as hepatocellular carcinoma and giant cell arteritis, are associated with <em>Candida</em> infection [<a href="#pone.0189503.ref044" class="ref-tip">44</a>–<a href="#pone.0189503.ref046" class="ref-tip">46</a>]. This may be associated with increased susceptibility for fungal infection in the immune-compromised population [<a href="#pone.0189503.ref044" class="ref-tip">44</a>] and infection-induced hyper activation of immune responses[<a href="#pone.0189503.ref046" class="ref-tip">46</a>]. Based on the phylogenetic analyses of human and yeast 14-3-3, and effective inhibition of <em>Candida</em> by human 14-3-3 inhibitor, we speculate that foreign peptides (<em>Candida)</em> may invoke an autoimmune response in giant cell arteritis.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>The ability of 14-3-3 proteins to homo- or hetero-dimerize regulates their functions [<a href="#pone.0189503.ref047" class="ref-tip">47</a>], and may provide additional information related to their antigenicity. This in part is supported by Lalle et al.,[<a href="#pone.0189503.ref038" class="ref-tip">38</a>] wherein they showed that 14-3-3 proteins from <em>D</em>. <em>melanogaster</em> and <em>G</em>. <em>duodenalis</em> are able to heterodimerize, suggesting that low level chronic infection may expose a non-self functional molecule capable of immune activation.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>Beyond determining the antigenic properties of 14-3-3, the identification of specific peptide sequence similarities between the human and yeast forms of 14-3-3, presented us with a unique opportunity to contemplate the repurposing of the 14-3-3 inhibitor such as, R18, as a putative antifungal agent for <em>Candida</em>. The finding that R18 indeed can elicit a dose-dependent antifungal activity against <em>Candida</em> is not only a novel discovery by itself, but points further to evolutionarily conserved shared properties of 14-3-3 functions that may contribute to molecular mimicry type of responses in eukaryotes, resulting in autoimmunity. Thus, by the bioinformatic assessment of 14-3-3 sequences across species, our study serves as a fundamental basis towards the next steps required for identifying and understanding the new antigenic function of 14-3-3, which in turn aids to differentiate between infectious and autoimmune diseases.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec010" name="sec010" data-toc="sec010" class="link-target" title="Methods"></a><h2>Methods</h2>
<div id="section1" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="Bioinformatic analyses"></a>
<h3>Bioinformatic analyses</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>Accession numbers of genes were obtained (<a href="#pone.0189503.s006">S1 Table</a>) and BLAST searches were performed by the protein or nucleotide based search criteria using the basic local alignment search tool (Blast) service provided by the NCBI (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). Alignment was then entered into WebLogo to create the sequence logo to study conservation of sequence [<a href="#pone.0189503.ref025" class="ref-tip">25</a>]. The Ensembl release 89 (<a href="https://www.ensembl.org/index.html">https://www.ensembl.org/index.html</a>) was used for mining genomic datasets. The following assemblies were used: Human (GRCh38.p10), Rat (Rnor_6.0), Mouse (GRCm38.p4), Toxoplasma (GCA_000006565.2), <em>Candida</em> (ASM18296v3), Saccharomyces (R64), Xenopus (GL172639.1), Drosophila (GCF_000001215.4) and Gammaproteobacteria (NCBI assembly- ASM171528v1, GenBank accession number-GCA_001715285.1). The website, <a href="http://www.Phylogeny.fr">www.Phylogeny.fr</a>, was used to build phylogenetic tree of 14-3-3 family of proteins [<a href="#pone.0189503.ref048" class="ref-tip">48</a>]. MUSCLE and Clustal W were used for multiple sequence alignments. SWISS-MODEL was used to simulate rat 14-3-3ζ and Gammaproteobacteria hypothetical protein structure, with PDB ID 5WXN serving as a template for both, while the crystal structure of human 14-3-3ζ was downloaded from Protein Data Bank (PDB ID 5D2D) [<a href="#pone.0189503.ref049" class="ref-tip">49</a>]. Visualization and structure alignment was done using VMD [<a href="#pone.0189503.ref050" class="ref-tip">50</a>]. Raptor X was used to model toxoplasma 14-3-3, with PDB ID 3M50 serving as a template [<a href="#pone.0189503.ref051" class="ref-tip">51</a>].</p>
</div>

<div id="section2" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="C. albicans and S. cerevisiae killing assays"></a>
<h3><em>C</em>. <em>albicans</em> and <em>S</em>. <em>cerevisiae</em> killing assays</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>Yeast cells were grown in YPD broth overnight at 30°C then subcultured to the log phase of growth [<a href="#pone.0189503.ref052" class="ref-tip">52</a>]. The cells were washed in 10mM sodium phosphate buffer and then 10,000 cells were co-incubated with buffer, 500nM of R18 (Enzo Life Sciences, Inc.) or 200nM of BD3 (BioBasics, Inc.) as a positive control. Cells were incubated at 37°C (<em>C</em>. <em>albicans</em> CAF2-1) or 30°C (<em>S</em>. <em>cerevisiae</em> AH106) for one hour in 10mM sodium phosphate buffer, serially diluted and plated on YPD agar plates. Plates were incubated at 30°C until yeast colonies were visible for counting. Percent killing was calculated compared to a negative control containing buffer only and yeast cells.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="Candida killing assay with human oral epithelial cells"></a>
<h3><em>Candida</em> killing assay with human oral epithelial cells</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>TR146 cells (Sigma) were grown to confluence in six well dishes using DMEM+F12 supplemented with 10% FBS and 1% pen/strep. Once confluence was reached, cells were deprived for 24 hours with DMEM supplemented with 1% BSA. After 24h, media was conditioned with 500nM of BV02. A control of unconditioned deprivation media was included as a control. After 24 hours approximately 10,000 <em>C</em>. <em>albicans</em> (CAF2-1) yeast cells that had been cultured to the log-phase of growth were added to each of the wells and incubated at 37°C for one hour. After incubation, TR146 cells and <em>C</em>. <em>albicans</em> were removed from the plate and resuspended in 10mM sodium phosphate buffer, serially diluted and plated on YPD agar plates supplemented with gentamicin and ampicillin. Plates were incubated overnight at 30°C. Plates were then counted for number of <em>Candida</em> colonies and percent killing was calculated compared to a negative control.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec014" name="sec014" class="link-target" title="Statistical analysis"></a>
<h3>Statistical analysis</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>For simulated structures, we used the QMEAN score (SWISS-MODEL) or probability (Raptor X), as provided by the platform utilized for determination. As suggested by Raptor X, p values less than 10<sup>−3</sup> and 10<sup>−4</sup> was considered significant for alpha helix and beta sheet dominant structures, respectively. For QMEAN, scores range from 0–1, with 1 being good [<a href="#pone.0189503.ref053" class="ref-tip">53</a>]. For the survival experiments, we used two tailed unpaired t test with at least 95% confidence to determine the significance of difference.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="sec015" name="sec015" data-toc="sec015" class="link-target" title="Supporting information"></a><h2>Supporting information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0189503"></div><div class="supplementary-material"><a name="pone.0189503.s001" id="pone.0189503.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0189503.s001">S1 Fig. </a>Sequence alignment of human isoforms.</h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">All human 14-3-3 isoforms were aligned using MUSCLE program. Fully conserved residues are marked with an asterisk (*), mostly conserved with a colon (:), weakly conserved with a period (.) and non-conserved are unlabeled. Area of variability are underlined and are numbered as v1 to v6. Labels are hsEP = ε, hsS = σ, hsG = γ, hsET = η, hsT = τ, hsZ = ζ, hsA = α.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0189503.s001">https://doi.org/10.1371/journal.pone.0189503.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0189503.s002" id="pone.0189503.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0189503.s002">S2 Fig. </a>Sequence alignment of S. mansoni vs human isoforms.</h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Alignment of <em>S</em>. <em>mansoni</em> 14-3-3 isoforms (assembly ASM23792v2) with human 14-3-3 isoforms is shown.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0189503.s002">https://doi.org/10.1371/journal.pone.0189503.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0189503.s003" id="pone.0189503.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0189503.s003">S3 Fig. </a>Sequence alignment of Angiostrongylus vs human isoforms.</h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">Alignment of <em>A</em>. <em>cantonensis</em> 14-3-3 with human zeta isoform showed significant homology between the two proteins.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0189503.s003">https://doi.org/10.1371/journal.pone.0189503.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0189503.s004" id="pone.0189503.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0189503.s004">S4 Fig. </a>Effect of 14-3-3 inhibitor on the yeast growth.</h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">(A) Inhibition of 14-3-3 in <em>S</em>. <em>cerevisiae</em> does not decrease cell survivality. Unlike <em>C</em>. <em>albicans</em>, Inhibition of 14-3-3 by peptide inhibitor R18 (500nM) did not result in statistically significant change in the yeast survival in comparison to b-defensin 3 (BD3, 200nM). (B) Representative Images from the <em>Candida</em> survival experiment (<a href="#pone-0189503-g007">Fig 7A</a>) are shown. Number of counted colonies on each plate are listed below.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0189503.s004">https://doi.org/10.1371/journal.pone.0189503.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0189503.s005" id="pone.0189503.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0189503.s005">S5 Fig. </a>Alignment of all 14-3-3 genes investigated in the current study.</h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI">A table showing abbreviations that are used in alignment file, followed by alignment file.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0189503.s005">https://doi.org/10.1371/journal.pone.0189503.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0189503.s006" id="pone.0189503.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0189503.s006">S1 Table. </a>List of 14-3-3 proteins studied.</h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI">Accession Numbers of All 14-3-3 Proteins. obtained from NCBI, are listed.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0189503.s006">https://doi.org/10.1371/journal.pone.0189503.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0189503.s007" id="pone.0189503.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0189503.s007">S2 Table. </a>Chromosomal location of 14-3-3.</h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI">Position of 14-3-3 isoforms In different species is listed.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0189503.s007">https://doi.org/10.1371/journal.pone.0189503.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We acknowledge the support of Dr. Scott Crawley (University of Toledo) for sharing the <em>S</em>. <em>cerevisiae</em> (strain AH106) for our study.</p>
</div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0189503.ref001" id="pone.0189503.ref001" class="link-target"></a>Morrison DK. The 14-3-3 proteins: integrators of diverse signaling cues that impact cell fate and cancer development. Trends Cell Biol. 2009;19(1):16–23. Epub 2008/11/26.  pmid:19027299; PubMed Central PMCID: PMC3073487. <ul class="reflinks"><li><a href="#" data-author="Morrison" data-cit="MorrisonDK.%20The%2014-3-3%20proteins%3A%20integrators%20of%20diverse%20signaling%20cues%20that%20impact%20cell%20fate%20and%20cancer%20development.%20Trends%20Cell%20Biol.%202009%3B19%281%29%3A16%E2%80%9323.%20Epub%202008%2F11%2F26.%20doi%3A%2010.1016%2Fj.tcb.2008.10.003%2019027299%3B%20PubMed%20Central%20PMCID%3A%20PMC3073487." data-title="The%2014-3-3%20proteins%3A%20integrators%20of%20diverse%20signaling%20cues%20that%20impact%20cell%20fate%20and%20cancer%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19027299" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+14-3-3+proteins%3A+integrators+of+diverse+signaling+cues+that+impact+cell+fate+and+cancer+development+Morrison+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0189503.ref002" id="pone.0189503.ref002" class="link-target"></a>Tsai HC, Huang YL, Chen YS, Yen CM, Tsai R, Lee SS, et al. 14-3-3beta protein expression in eosinophilic meningitis caused by Angiostrongylus cantonensis infection. BMC Res Notes. 2014;7:97.  pmid:24555778; PubMed Central PMCID: PMCPMC3932789. <ul class="reflinks"><li><a href="#" data-author="Tsai" data-cit="TsaiHC%2C%20HuangYL%2C%20ChenYS%2C%20YenCM%2C%20TsaiR%2C%20LeeSS%2C%20et%20al.%2014-3-3beta%20protein%20expression%20in%20eosinophilic%20meningitis%20caused%20by%20Angiostrongylus%20cantonensis%20infection.%20BMC%20Res%20Notes.%202014%3B7%3A97.%20doi%3A%2010.1186%2F1756-0500-7-97%2024555778%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3932789." data-title="14-3-3beta%20protein%20expression%20in%20eosinophilic%20meningitis%20caused%20by%20Angiostrongylus%20cantonensis%20infection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24555778" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3beta+protein+expression+in+eosinophilic+meningitis+caused+by+Angiostrongylus+cantonensis+infection+Tsai+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0189503.ref003" id="pone.0189503.ref003" class="link-target"></a>Zhao J, Meyerkord CL, Du Y, Khuri FR, Fu H. 14-3-3 proteins as potential therapeutic targets. Semin Cell Dev Biol. 2011;22(7):705–12. Epub 2011/10/11.  pmid:21983031; PubMed Central PMCID: PMC3207012. <ul class="reflinks"><li><a href="#" data-author="Zhao" data-cit="ZhaoJ%2C%20MeyerkordCL%2C%20DuY%2C%20KhuriFR%2C%20FuH.%2014-3-3%20proteins%20as%20potential%20therapeutic%20targets.%20Semin%20Cell%20Dev%20Biol.%202011%3B22%287%29%3A705%E2%80%9312.%20Epub%202011%2F10%2F11.%20doi%3A%2010.1016%2Fj.semcdb.2011.09.012%2021983031%3B%20PubMed%20Central%20PMCID%3A%20PMC3207012." data-title="14-3-3%20proteins%20as%20potential%20therapeutic%20targets" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21983031" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+proteins+as+potential+therapeutic+targets+Zhao+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0189503.ref004" id="pone.0189503.ref004" class="link-target"></a>Kilani RT, Maksymowych WP, Aitken A, Boire G, St-Pierre Y, Li Y, et al. Detection of high levels of 2 specific isoforms of 14-3-3 proteins in synovial fluid from patients with joint inflammation. The Journal of rheumatology. 2007;34(8):1650–7. Epub 2007/07/06. pmid:17611984. <ul class="reflinks"><li><a href="#" data-author="Kilani" data-cit="KilaniRT%2C%20MaksymowychWP%2C%20AitkenA%2C%20BoireG%2C%20St-PierreY%2C%20LiY%2C%20et%20al.%20Detection%20of%20high%20levels%20of%202%20specific%20isoforms%20of%2014-3-3%20proteins%20in%20synovial%20fluid%20from%20patients%20with%20joint%20inflammation.%20The%20Journal%20of%20rheumatology.%202007%3B34%288%29%3A1650%E2%80%937.%20Epub%202007%2F07%2F06.%2017611984." data-title="Detection%20of%20high%20levels%20of%202%20specific%20isoforms%20of%2014-3-3%20proteins%20in%20synovial%20fluid%20from%20patients%20with%20joint%20inflammation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17611984" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Detection+of+high+levels+of+2+specific+isoforms+of+14-3-3+proteins+in+synovial+fluid+from+patients+with+joint+inflammation+Kilani+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0189503.ref005" id="pone.0189503.ref005" class="link-target"></a>Hatzipetros I, Gocze P, Koszegi T, Jaray A, Szereday L, Polgar B, et al. Investigating the clinical potential for 14-3-3 zeta protein to serve as a biomarker for epithelial ovarian cancer. J Ovarian Res. 2013;6(1):79. Epub 2013/11/19.  pmid:24238270; PubMed Central PMCID: PMC3835546. <ul class="reflinks"><li><a href="#" data-author="Hatzipetros" data-cit="HatzipetrosI%2C%20GoczeP%2C%20KoszegiT%2C%20JarayA%2C%20SzeredayL%2C%20PolgarB%2C%20et%20al.%20Investigating%20the%20clinical%20potential%20for%2014-3-3%20zeta%20protein%20to%20serve%20as%20a%20biomarker%20for%20epithelial%20ovarian%20cancer.%20J%20Ovarian%20Res.%202013%3B6%281%29%3A79.%20Epub%202013%2F11%2F19.%20doi%3A%2010.1186%2F1757-2215-6-79%2024238270%3B%20PubMed%20Central%20PMCID%3A%20PMC3835546." data-title="Investigating%20the%20clinical%20potential%20for%2014-3-3%20zeta%20protein%20to%20serve%20as%20a%20biomarker%20for%20epithelial%20ovarian%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24238270" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Investigating+the+clinical+potential+for+14-3-3+zeta+protein+to+serve+as+a+biomarker+for+epithelial+ovarian+cancer+Hatzipetros+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0189503.ref006" id="pone.0189503.ref006" class="link-target"></a>Foote M, Zhou Y. 14-3-3 proteins in neurological disorders. Int J Biochem Mol Biol. 2012;3(2):152–64. Epub 2012/07/10. pmid:22773956; PubMed Central PMCID: PMC3388734. <ul class="reflinks"><li><a href="#" data-author="Foote" data-cit="FooteM%2C%20ZhouY.%2014-3-3%20proteins%20in%20neurological%20disorders.%20Int%20J%20Biochem%20Mol%20Biol.%202012%3B3%282%29%3A152%E2%80%9364.%20Epub%202012%2F07%2F10.%2022773956%3B%20PubMed%20Central%20PMCID%3A%20PMC3388734." data-title="14-3-3%20proteins%20in%20neurological%20disorders" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22773956" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+proteins+in+neurological+disorders+Foote+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0189503.ref007" id="pone.0189503.ref007" class="link-target"></a>Maksymowych WP, Marotta A. 14-3-3eta: a novel biomarker platform for rheumatoid arthritis. Clinical and experimental rheumatology. 2014;32(5 Suppl 85):S-35-9. pmid:25365087. <ul class="reflinks"><li><a href="#" data-author="Maksymowych" data-cit="MaksymowychWP%2C%20MarottaA.%2014-3-3eta%3A%20a%20novel%20biomarker%20platform%20for%20rheumatoid%20arthritis.%20Clinical%20and%20experimental%20rheumatology.%202014%3B32%285%20Suppl%2085%29%3AS-35-9.%2025365087." data-title="14-3-3eta%3A%20a%20novel%20biomarker%20platform%20for%20rheumatoid%20arthritis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25365087" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3eta%3A+a+novel+biomarker+platform+for+rheumatoid+arthritis+Maksymowych+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0189503.ref008" id="pone.0189503.ref008" class="link-target"></a>Colucci M, Roccatagliata L, Capello E, Narciso E, Latronico N, Tabaton M, et al. The 14-3-3 protein in multiple sclerosis: a marker of disease severity. Mult Scler. 2004;10(5):477–81.  pmid:15471360. <ul class="reflinks"><li><a href="#" data-author="Colucci" data-cit="ColucciM%2C%20RoccatagliataL%2C%20CapelloE%2C%20NarcisoE%2C%20LatronicoN%2C%20TabatonM%2C%20et%20al.%20The%2014-3-3%20protein%20in%20multiple%20sclerosis%3A%20a%20marker%20of%20disease%20severity.%20Mult%20Scler.%202004%3B10%285%29%3A477%E2%80%9381.%20doi%3A%2010.1191%2F1352458504ms1089oa%2015471360." data-title="The%2014-3-3%20protein%20in%20multiple%20sclerosis%3A%20a%20marker%20of%20disease%20severity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15471360" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+14-3-3+protein+in+multiple+sclerosis%3A+a+marker+of+disease+severity+Colucci+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0189503.ref009" id="pone.0189503.ref009" class="link-target"></a>Shandala T, Woodcock JM, Ng Y, Biggs L, Skoulakis EM, Brooks DA, et al. Drosophila 14-3-3epsilon has a crucial role in anti-microbial peptide secretion and innate immunity. J Cell Sci. 2011;124(Pt 13):2165–74. Epub 2011/06/15.  pmid:21670199. <ul class="reflinks"><li><a href="#" data-author="Shandala" data-cit="ShandalaT%2C%20WoodcockJM%2C%20NgY%2C%20BiggsL%2C%20SkoulakisEM%2C%20BrooksDA%2C%20et%20al.%20Drosophila%2014-3-3epsilon%20has%20a%20crucial%20role%20in%20anti-microbial%20peptide%20secretion%20and%20innate%20immunity.%20J%20Cell%20Sci.%202011%3B124%28Pt%2013%29%3A2165%E2%80%9374.%20Epub%202011%2F06%2F15.%20doi%3A%2010.1242%2Fjcs.080598%2021670199." data-title="Drosophila%2014-3-3epsilon%20has%20a%20crucial%20role%20in%20anti-microbial%20peptide%20secretion%20and%20innate%20immunity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21670199" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Drosophila+14-3-3epsilon+has+a+crucial+role+in+anti-microbial+peptide+secretion+and+innate+immunity+Shandala+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0189503.ref010" id="pone.0189503.ref010" class="link-target"></a>Liu HM, Loo YM, Horner SM, Zornetzer GA, Katze MG, Gale M Jr., The mitochondrial targeting chaperone 14-3-3epsilon regulates a RIG-I translocon that mediates membrane association and innate antiviral immunity. Cell Host Microbe. 2012;11(5):528–37. Epub 2012/05/23.  pmid:22607805; PubMed Central PMCID: PMC3358705. <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuHM%2C%20LooYM%2C%20HornerSM%2C%20ZornetzerGA%2C%20KatzeMG%2C%20GaleMJr.%2C%20The%20mitochondrial%20targeting%20chaperone%2014-3-3epsilon%20regulates%20a%20RIG-I%20translocon%20that%20mediates%20membrane%20association%20and%20innate%20antiviral%20immunity.%20Cell%20Host%20Microbe.%202012%3B11%285%29%3A528%E2%80%9337.%20Epub%202012%2F05%2F23.%20doi%3A%2010.1016%2Fj.chom.2012.04.006%2022607805%3B%20PubMed%20Central%20PMCID%3A%20PMC3358705." data-title="The%20mitochondrial%20targeting%20chaperone%2014-3-3epsilon%20regulates%20a%20RIG-I%20translocon%20that%20mediates%20membrane%20association%20and%20innate%20antiviral%20immunity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22607805" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+mitochondrial+targeting+chaperone+14-3-3epsilon+regulates+a+RIG-I+translocon+that+mediates+membrane+association+and+innate+antiviral+immunity+Liu+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0189503.ref011" id="pone.0189503.ref011" class="link-target"></a>Ulvila J, Vanha-aho LM, Kleino A, Vaha-Makila M, Vuoksio M, Eskelinen S, et al. Cofilin regulator 14-3-3zeta is an evolutionarily conserved protein required for phagocytosis and microbial resistance. J Leukoc Biol. 2011;89(5):649–59. Epub 2011/01/07.  pmid:21208897. <ul class="reflinks"><li><a href="#" data-author="Ulvila" data-cit="UlvilaJ%2C%20Vanha-ahoLM%2C%20KleinoA%2C%20Vaha-MakilaM%2C%20VuoksioM%2C%20EskelinenS%2C%20et%20al.%20Cofilin%20regulator%2014-3-3zeta%20is%20an%20evolutionarily%20conserved%20protein%20required%20for%20phagocytosis%20and%20microbial%20resistance.%20J%20Leukoc%20Biol.%202011%3B89%285%29%3A649%E2%80%9359.%20Epub%202011%2F01%2F07.%20doi%3A%2010.1189%2Fjlb.0410195%2021208897." data-title="Cofilin%20regulator%2014-3-3zeta%20is%20an%20evolutionarily%20conserved%20protein%20required%20for%20phagocytosis%20and%20microbial%20resistance" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21208897" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cofilin+regulator+14-3-3zeta+is+an+evolutionarily+conserved+protein+required+for+phagocytosis+and+microbial+resistance+Ulvila+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0189503.ref012" id="pone.0189503.ref012" class="link-target"></a>Lozano-Duran R, Robatzek S. 14-3-3 proteins in plant-pathogen interactions. Mol Plant Microbe Interact. 2015;28(5):511–8.  pmid:25584723. <ul class="reflinks"><li><a href="#" data-author="Lozano-Duran" data-cit="Lozano-DuranR%2C%20RobatzekS.%2014-3-3%20proteins%20in%20plant-pathogen%20interactions.%20Mol%20Plant%20Microbe%20Interact.%202015%3B28%285%29%3A511%E2%80%938.%20doi%3A%2010.1094%2FMPMI-10-14-0322-CR%2025584723." data-title="14-3-3%20proteins%20in%20plant-pathogen%20interactions" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25584723" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+proteins+in+plant-pathogen+interactions+Lozano-Duran+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0189503.ref013" id="pone.0189503.ref013" class="link-target"></a>Chakravarti R, Gupta K, Swain M, Willard B, Scholtz J, Svensson LG, et al. 14-3-3 in Thoracic Aortic Aneurysms: Identification of a Novel Autoantigen in Large Vessel Vasculitis. Arthritis &amp; rheumatology (Hoboken, NJ). 2015;67(7):1913–21. Epub 2015/04/29.  pmid:25917817; PubMed Central PMCID: PMCPmc4624269. <ul class="reflinks"><li><a href="#" data-author="Chakravarti" data-cit="ChakravartiR%2C%20GuptaK%2C%20SwainM%2C%20WillardB%2C%20ScholtzJ%2C%20SvenssonLG%2C%20et%20al.%2014-3-3%20in%20Thoracic%20Aortic%20Aneurysms%3A%20Identification%20of%20a%20Novel%20Autoantigen%20in%20Large%20Vessel%20Vasculitis.%20Arthritis%20%26%20rheumatology%20%28Hoboken%2C%20NJ%29.%202015%3B67%287%29%3A1913%E2%80%9321.%20Epub%202015%2F04%2F29.%20doi%3A%2010.1002%2Fart.39130%2025917817%3B%20PubMed%20Central%20PMCID%3A%20PMCPmc4624269." data-title="14-3-3%20in%20Thoracic%20Aortic%20Aneurysms%3A%20Identification%20of%20a%20Novel%20Autoantigen%20in%20Large%20Vessel%20Vasculitis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25917817" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+in+Thoracic+Aortic+Aneurysms%3A+Identification+of+a+Novel+Autoantigen+in+Large+Vessel+Vasculitis+Chakravarti+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0189503.ref014" id="pone.0189503.ref014" class="link-target"></a>Neal CL, Yu D. 14-3-3zeta as a prognostic marker and therapeutic target for cancer. Expert Opin Ther Targets. 2010;14(12):1343–54.  pmid:21058923; PubMed Central PMCID: PMCPMC3017465. <ul class="reflinks"><li><a href="#" data-author="Neal" data-cit="NealCL%2C%20YuD.%2014-3-3zeta%20as%20a%20prognostic%20marker%20and%20therapeutic%20target%20for%20cancer.%20Expert%20Opin%20Ther%20Targets.%202010%3B14%2812%29%3A1343%E2%80%9354.%20doi%3A%2010.1517%2F14728222.2010.531011%2021058923%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3017465." data-title="14-3-3zeta%20as%20a%20prognostic%20marker%20and%20therapeutic%20target%20for%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21058923" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3zeta+as+a+prognostic+marker+and+therapeutic+target+for+cancer+Neal+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0189503.ref015" id="pone.0189503.ref015" class="link-target"></a>Maksymowych WP, Boire G, van Schaardenburg D, Wichuk S, Turk S, Boers M, et al. 14-3-3eta Autoantibodies: Diagnostic Use in Early Rheumatoid Arthritis. The Journal of rheumatology. 2015;42(9):1587–94.  pmid:26178283. <ul class="reflinks"><li><a href="#" data-author="Maksymowych" data-cit="MaksymowychWP%2C%20BoireG%2C%20van%20SchaardenburgD%2C%20WichukS%2C%20TurkS%2C%20BoersM%2C%20et%20al.%2014-3-3eta%20Autoantibodies%3A%20Diagnostic%20Use%20in%20Early%20Rheumatoid%20Arthritis.%20The%20Journal%20of%20rheumatology.%202015%3B42%289%29%3A1587%E2%80%9394.%20doi%3A%2010.3899%2Fjrheum.141385%2026178283." data-title="14-3-3eta%20Autoantibodies%3A%20Diagnostic%20Use%20in%20Early%20Rheumatoid%20Arthritis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26178283" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3eta+Autoantibodies%3A+Diagnostic+Use+in+Early+Rheumatoid+Arthritis+Maksymowych+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0189503.ref016" id="pone.0189503.ref016" class="link-target"></a>Siles-Lucas M, Merli M, Gottstein B. 14-3-3 proteins in Echinococcus: their role and potential as protective antigens. Exp Parasitol. 2008;119(4):516–23.  pmid:18316081. <ul class="reflinks"><li><a href="#" data-author="Siles-Lucas" data-cit="Siles-LucasM%2C%20MerliM%2C%20GottsteinB.%2014-3-3%20proteins%20in%20Echinococcus%3A%20their%20role%20and%20potential%20as%20protective%20antigens.%20Exp%20Parasitol.%202008%3B119%284%29%3A516%E2%80%9323.%20doi%3A%2010.1016%2Fj.exppara.2008.01.009%2018316081." data-title="14-3-3%20proteins%20in%20Echinococcus%3A%20their%20role%20and%20potential%20as%20protective%20antigens" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18316081" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+proteins+in+Echinococcus%3A+their+role+and+potential+as+protective+antigens+Siles-Lucas+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0189503.ref017" id="pone.0189503.ref017" class="link-target"></a>Assossou O, Besson F, Rouault JP, Persat F, Ferrandiz J, Mayencon M, et al. Characterization of an excreted/secreted antigen form of 14-3-3 protein in Toxoplasma gondii tachyzoites. FEMS Microbiol Lett. 2004;234(1):19–25.  pmid:15109715. <ul class="reflinks"><li><a href="#" data-author="Assossou" data-cit="AssossouO%2C%20BessonF%2C%20RouaultJP%2C%20PersatF%2C%20FerrandizJ%2C%20MayenconM%2C%20et%20al.%20Characterization%20of%20an%20excreted%2Fsecreted%20antigen%20form%20of%2014-3-3%20protein%20in%20Toxoplasma%20gondii%20tachyzoites.%20FEMS%20Microbiol%20Lett.%202004%3B234%281%29%3A19%E2%80%9325.%20doi%3A%2010.1016%2Fj.femsle.2004.02.024%2015109715." data-title="Characterization%20of%20an%20excreted%2Fsecreted%20antigen%20form%20of%2014-3-3%20protein%20in%20Toxoplasma%20gondii%20tachyzoites" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15109715" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+an+excreted%2Fsecreted+antigen+form+of+14-3-3+protein+in+Toxoplasma+gondii+tachyzoites+Assossou+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0189503.ref018" id="pone.0189503.ref018" class="link-target"></a>Liu M, Liu X, Ren P, Li J, Chai Y, Zheng SJ, et al. A cancer-related protein 14-3-3zeta is a potential tumor-associated antigen in immunodiagnosis of hepatocellular carcinoma. Tumour Biol. 2014;35(5):4247–56.  pmid:24390614; PubMed Central PMCID: PMCPMC4096569. <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuM%2C%20LiuX%2C%20RenP%2C%20LiJ%2C%20ChaiY%2C%20ZhengSJ%2C%20et%20al.%20A%20cancer-related%20protein%2014-3-3zeta%20is%20a%20potential%20tumor-associated%20antigen%20in%20immunodiagnosis%20of%20hepatocellular%20carcinoma.%20Tumour%20Biol.%202014%3B35%285%29%3A4247%E2%80%9356.%20doi%3A%2010.1007%2Fs13277-013-1555-8%2024390614%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC4096569." data-title="A%20cancer-related%20protein%2014-3-3zeta%20is%20a%20potential%20tumor-associated%20antigen%20in%20immunodiagnosis%20of%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24390614" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+cancer-related+protein+14-3-3zeta+is+a+potential+tumor-associated+antigen+in+immunodiagnosis+of+hepatocellular+carcinoma+Liu+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0189503.ref019" id="pone.0189503.ref019" class="link-target"></a>Kistner A, Bigler MB, Glatz K, Egli SB, Baldin FS, Marquardsen FA, et al. Characteristics of autoantibodies targeting 14-3-3 proteins and their association with clinical features in newly diagnosed giant cell arteritis. Rheumatology (Oxford, England). 2017. Epub 2017/01/09.  pmid:28064210. <ul class="reflinks"><li><a href="#" data-author="Kistner" data-cit="KistnerA%2C%20BiglerMB%2C%20GlatzK%2C%20EgliSB%2C%20BaldinFS%2C%20MarquardsenFA%2C%20et%20al.%20Characteristics%20of%20autoantibodies%20targeting%2014-3-3%20proteins%20and%20their%20association%20with%20clinical%20features%20in%20newly%20diagnosed%20giant%20cell%20arteritis.%20Rheumatology%20%28Oxford%2C%20England%29.%202017.%20Epub%202017%2F01%2F09.%20doi%3A%2010.1093%2Frheumatology%2Fkew469%2028064210." data-title="Characteristics%20of%20autoantibodies%20targeting%2014-3-3%20proteins%20and%20their%20association%20with%20clinical%20features%20in%20newly%20diagnosed%20giant%20cell%20arteritis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/28064210" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Characteristics+of+autoantibodies+targeting+14-3-3+proteins+and+their+association+with+clinical+features+in+newly+diagnosed+giant+cell+arteritis+Kistner+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0189503.ref020" id="pone.0189503.ref020" class="link-target"></a>Wang W, Shakes DC. Molecular evolution of the 14-3-3 protein family. J Mol Evol. 1996;43(4):384–98. pmid:8798343. <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangW%2C%20ShakesDC.%20Molecular%20evolution%20of%20the%2014-3-3%20protein%20family.%20J%20Mol%20Evol.%201996%3B43%284%29%3A384%E2%80%9398.%208798343." data-title="Molecular%20evolution%20of%20the%2014-3-3%20protein%20family" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8798343" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+evolution+of+the+14-3-3+protein+family+Wang+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0189503.ref021" id="pone.0189503.ref021" class="link-target"></a>van Heusden GP, Steensma HY. Yeast 14-3-3 proteins. Yeast. 2006;23(3):159–71.  pmid:16498703. <ul class="reflinks"><li><a href="#" data-author="van%20Heusden" data-cit="van%20HeusdenGP%2C%20SteensmaHY.%20Yeast%2014-3-3%20proteins.%20Yeast.%202006%3B23%283%29%3A159%E2%80%9371.%20doi%3A%2010.1002%2Fyea.1338%2016498703." data-title="Yeast%2014-3-3%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16498703" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Yeast+14-3-3+proteins+van+Heusden+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0189503.ref022" id="pone.0189503.ref022" class="link-target"></a>Kumar R. An account of fungal 14-3-3 proteins. Eur J Cell Biol. 2017;96(2):206–17.  pmid:28258766. <ul class="reflinks"><li><a href="#" data-author="Kumar" data-cit="KumarR.%20An%20account%20of%20fungal%2014-3-3%20proteins.%20Eur%20J%20Cell%20Biol.%202017%3B96%282%29%3A206%E2%80%9317.%20doi%3A%2010.1016%2Fj.ejcb.2017.02.006%2028258766." data-title="An%20account%20of%20fungal%2014-3-3%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/28258766" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=An+account+of+fungal+14-3-3+proteins+Kumar+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0189503.ref023" id="pone.0189503.ref023" class="link-target"></a>Tzivion G, Shen YH, Zhu J. 14-3-3 proteins; bringing new definitions to scaffolding. Oncogene. 2001;20(44):6331–8.  pmid:11607836. <ul class="reflinks"><li><a href="#" data-author="Tzivion" data-cit="TzivionG%2C%20ShenYH%2C%20ZhuJ.%2014-3-3%20proteins%3B%20bringing%20new%20definitions%20to%20scaffolding.%20Oncogene.%202001%3B20%2844%29%3A6331%E2%80%938.%20doi%3A%2010.1038%2Fsj.onc.1204777%2011607836." data-title="14-3-3%20proteins%3B%20bringing%20new%20definitions%20to%20scaffolding" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11607836" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+proteins%3B+bringing+new+definitions+to+scaffolding+Tzivion+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0189503.ref024" id="pone.0189503.ref024" class="link-target"></a>Rosenquist M, Sehnke P, Ferl RJ, Sommarin M, Larsson C. Evolution of the 14-3-3 protein family: does the large number of isoforms in multicellular organisms reflect functional specificity? J Mol Evol. 2000;51(5):446–58. pmid:11080367. <ul class="reflinks"><li><a href="#" data-author="Rosenquist" data-cit="RosenquistM%2C%20SehnkeP%2C%20FerlRJ%2C%20SommarinM%2C%20LarssonC.%20Evolution%20of%20the%2014-3-3%20protein%20family%3A%20does%20the%20large%20number%20of%20isoforms%20in%20multicellular%20organisms%20reflect%20functional%20specificity%3F%20J%20Mol%20Evol.%202000%3B51%285%29%3A446%E2%80%9358.%2011080367." data-title="Evolution%20of%20the%2014-3-3%20protein%20family%3A%20does%20the%20large%20number%20of%20isoforms%20in%20multicellular%20organisms%20reflect%20functional%20specificity%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11080367" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Evolution+of+the+14-3-3+protein+family%3A+does+the+large+number+of+isoforms+in+multicellular+organisms+reflect+functional+specificity%3F+Rosenquist+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0189503.ref025" id="pone.0189503.ref025" class="link-target"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE. WebLogo: a sequence logo generator. Genome Res. 2004;14(6):1188–90.  pmid:15173120; PubMed Central PMCID: PMCPMC419797. <ul class="reflinks"><li><a href="#" data-author="Crooks" data-cit="CrooksGE%2C%20HonG%2C%20ChandoniaJM%2C%20BrennerSE.%20WebLogo%3A%20a%20sequence%20logo%20generator.%20Genome%20Res.%202004%3B14%286%29%3A1188%E2%80%9390.%20doi%3A%2010.1101%2Fgr.849004%2015173120%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC419797." data-title="WebLogo%3A%20a%20sequence%20logo%20generator" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15173120" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=WebLogo%3A+a+sequence+logo+generator+Crooks+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0189503.ref026" id="pone.0189503.ref026" class="link-target"></a>Chakravarti R, Adams JC. Comparative genomics of the syndecans defines an ancestral genomic context associated with matrilins in vertebrates. BMC Genomics. 2006;7:83.  pmid:16620374; PubMed Central PMCID: PMCPMC1464127. <ul class="reflinks"><li><a href="#" data-author="Chakravarti" data-cit="ChakravartiR%2C%20AdamsJC.%20Comparative%20genomics%20of%20the%20syndecans%20defines%20an%20ancestral%20genomic%20context%20associated%20with%20matrilins%20in%20vertebrates.%20BMC%20Genomics.%202006%3B7%3A83.%20doi%3A%2010.1186%2F1471-2164-7-83%2016620374%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC1464127." data-title="Comparative%20genomics%20of%20the%20syndecans%20defines%20an%20ancestral%20genomic%20context%20associated%20with%20matrilins%20in%20vertebrates" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16620374" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Comparative+genomics+of+the+syndecans+defines+an+ancestral+genomic+context+associated+with+matrilins+in+vertebrates+Chakravarti+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0189503.ref027" id="pone.0189503.ref027" class="link-target"></a>Morassutti AL, Levert K, Perelygin A, da Silva AJ, Wilkins P, Graeff-Teixeira C. The 31-kDa antigen of Angiostrongylus cantonensis comprises distinct antigenic glycoproteins. Vector Borne Zoonotic Dis. 2012;12(11):961–8.  pmid:22925026; PubMed Central PMCID: PMCPMC3491624. <ul class="reflinks"><li><a href="#" data-author="Morassutti" data-cit="MorassuttiAL%2C%20LevertK%2C%20PerelyginA%2C%20da%20SilvaAJ%2C%20WilkinsP%2C%20Graeff-TeixeiraC.%20The%2031-kDa%20antigen%20of%20Angiostrongylus%20cantonensis%20comprises%20distinct%20antigenic%20glycoproteins.%20Vector%20Borne%20Zoonotic%20Dis.%202012%3B12%2811%29%3A961%E2%80%938.%20doi%3A%2010.1089%2Fvbz.2011.0957%2022925026%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3491624." data-title="The%2031-kDa%20antigen%20of%20Angiostrongylus%20cantonensis%20comprises%20distinct%20antigenic%20glycoproteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22925026" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+31-kDa+antigen+of+Angiostrongylus+cantonensis+comprises+distinct+antigenic+glycoproteins+Morassutti+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0189503.ref028" id="pone.0189503.ref028" class="link-target"></a>Duarte J, Herbert F, Guiyedi V, Franetich JF, Roland J, Cazenave PA, et al. High levels of immunoglobulin E autoantibody to 14-3-3 epsilon protein correlate with protection against severe Plasmodium falciparum malaria. J Infect Dis. 2012;206(11):1781–9.  pmid:22984113. <ul class="reflinks"><li><a href="#" data-author="Duarte" data-cit="DuarteJ%2C%20HerbertF%2C%20GuiyediV%2C%20FranetichJF%2C%20RolandJ%2C%20CazenavePA%2C%20et%20al.%20High%20levels%20of%20immunoglobulin%20E%20autoantibody%20to%2014-3-3%20epsilon%20protein%20correlate%20with%20protection%20against%20severe%20Plasmodium%20falciparum%20malaria.%20J%20Infect%20Dis.%202012%3B206%2811%29%3A1781%E2%80%939.%20doi%3A%2010.1093%2Finfdis%2Fjis595%2022984113." data-title="High%20levels%20of%20immunoglobulin%20E%20autoantibody%20to%2014-3-3%20epsilon%20protein%20correlate%20with%20protection%20against%20severe%20Plasmodium%20falciparum%20malaria" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22984113" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=High+levels+of+immunoglobulin+E+autoantibody+to+14-3-3+epsilon+protein+correlate+with+protection+against+severe+Plasmodium+falciparum+malaria+Duarte+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0189503.ref029" id="pone.0189503.ref029" class="link-target"></a>Cognetti D, Davis D, Sturtevant J. The Candida albicans 14-3-3 gene, BMH1, is essential for growth. Yeast. 2002;19(1):55–67.  pmid:11754483. <ul class="reflinks"><li><a href="#" data-author="Cognetti" data-cit="CognettiD%2C%20DavisD%2C%20SturtevantJ.%20The%20Candida%20albicans%2014-3-3%20gene%2C%20BMH1%2C%20is%20essential%20for%20growth.%20Yeast.%202002%3B19%281%29%3A55%E2%80%9367.%20doi%3A%2010.1002%2Fyea.804%2011754483." data-title="The%20Candida%20albicans%2014-3-3%20gene%2C%20BMH1%2C%20is%20essential%20for%20growth" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11754483" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+Candida+albicans+14-3-3+gene%2C+BMH1%2C+is+essential+for+growth+Cognetti+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0189503.ref030" id="pone.0189503.ref030" class="link-target"></a>Kelly MN, Johnston DA, Peel BA, Morgan TW, Palmer GE, Sturtevant JE. Bmh1p (14-3-3) mediates pathways associated with virulence in Candida albicans. Microbiology. 2009;155(Pt 5):1536–46.  pmid:19372164; PubMed Central PMCID: PMCPMC2772093. <ul class="reflinks"><li><a href="#" data-author="Kelly" data-cit="KellyMN%2C%20JohnstonDA%2C%20PeelBA%2C%20MorganTW%2C%20PalmerGE%2C%20SturtevantJE.%20Bmh1p%20%2814-3-3%29%20mediates%20pathways%20associated%20with%20virulence%20in%20Candida%20albicans.%20Microbiology.%202009%3B155%28Pt%205%29%3A1536%E2%80%9346.%20doi%3A%2010.1099%2Fmic.0.027532-0%2019372164%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC2772093." data-title="Bmh1p%20%2814-3-3%29%20mediates%20pathways%20associated%20with%20virulence%20in%20Candida%20albicans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19372164" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bmh1p+%2814-3-3%29+mediates+pathways+associated+with+virulence+in+Candida+albicans+Kelly+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0189503.ref031" id="pone.0189503.ref031" class="link-target"></a>Parua PK, Ratnakumar S, Braun KA, Dombek KM, Arms E, Ryan PM, et al. 14-3-3 (Bmh) proteins inhibit transcription activation by Adr1 through direct binding to its regulatory domain. Mol Cell Biol. 2010;30(22):5273–83.  pmid:20855531; PubMed Central PMCID: PMCPMC2976377. <ul class="reflinks"><li><a href="#" data-author="Parua" data-cit="ParuaPK%2C%20RatnakumarS%2C%20BraunKA%2C%20DombekKM%2C%20ArmsE%2C%20RyanPM%2C%20et%20al.%2014-3-3%20%28Bmh%29%20proteins%20inhibit%20transcription%20activation%20by%20Adr1%20through%20direct%20binding%20to%20its%20regulatory%20domain.%20Mol%20Cell%20Biol.%202010%3B30%2822%29%3A5273%E2%80%9383.%20doi%3A%2010.1128%2FMCB.00715-10%2020855531%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC2976377." data-title="14-3-3%20%28Bmh%29%20proteins%20inhibit%20transcription%20activation%20by%20Adr1%20through%20direct%20binding%20to%20its%20regulatory%20domain" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20855531" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+%28Bmh%29+proteins+inhibit+transcription+activation+by+Adr1+through+direct+binding+to+its+regulatory+domain+Parua+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0189503.ref032" id="pone.0189503.ref032" class="link-target"></a>Slubowski CJ, Paulissen SM, Huang LS. The GCKIII kinase Sps1 and the 14-3-3 isoforms, Bmh1 and Bmh2, cooperate to ensure proper sporulation in Saccharomyces cerevisiae. PLoS One. 2014;9(11):e113528.  pmid:25409301; PubMed Central PMCID: PMCPMC4237420. <ul class="reflinks"><li><a href="#" data-author="Slubowski" data-cit="SlubowskiCJ%2C%20PaulissenSM%2C%20HuangLS.%20The%20GCKIII%20kinase%20Sps1%20and%20the%2014-3-3%20isoforms%2C%20Bmh1%20and%20Bmh2%2C%20cooperate%20to%20ensure%20proper%20sporulation%20in%20Saccharomyces%20cerevisiae.%20PLoS%20One.%202014%3B9%2811%29%3Ae113528.%20doi%3A%2010.1371%2Fjournal.pone.0113528%2025409301%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC4237420." data-title="The%20GCKIII%20kinase%20Sps1%20and%20the%2014-3-3%20isoforms%2C%20Bmh1%20and%20Bmh2%2C%20cooperate%20to%20ensure%20proper%20sporulation%20in%20Saccharomyces%20cerevisiae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409301" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+GCKIII+kinase+Sps1+and+the+14-3-3+isoforms%2C+Bmh1+and+Bmh2%2C+cooperate+to+ensure+proper+sporulation+in+Saccharomyces+cerevisiae+Slubowski+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0189503.ref033" id="pone.0189503.ref033" class="link-target"></a>Feng Z, Jiang B, Chandra J, Ghannoum M, Nelson S, Weinberg A. Human beta-defensins: differential activity against candidal species and regulation by Candida albicans. J Dent Res. 2005;84(5):445–50.  pmid:15840781. <ul class="reflinks"><li><a href="#" data-author="Feng" data-cit="FengZ%2C%20JiangB%2C%20ChandraJ%2C%20GhannoumM%2C%20NelsonS%2C%20WeinbergA.%20Human%20beta-defensins%3A%20differential%20activity%20against%20candidal%20species%20and%20regulation%20by%20Candida%20albicans.%20J%20Dent%20Res.%202005%3B84%285%29%3A445%E2%80%9350.%20doi%3A%2010.1177%2F154405910508400509%2015840781." data-title="Human%20beta-defensins%3A%20differential%20activity%20against%20candidal%20species%20and%20regulation%20by%20Candida%20albicans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15840781" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Human+beta-defensins%3A+differential+activity+against+candidal+species+and+regulation+by+Candida+albicans+Feng+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0189503.ref034" id="pone.0189503.ref034" class="link-target"></a>Conti HR, Shen F, Nayyar N, Stocum E, Sun JN, Lindemann MJ, et al. Th17 cells and IL-17 receptor signaling are essential for mucosal host defense against oral candidiasis. J Exp Med. 2009;206(2):299–311.  pmid:19204111; PubMed Central PMCID: PMCPMC2646568. <ul class="reflinks"><li><a href="#" data-author="Conti" data-cit="ContiHR%2C%20ShenF%2C%20NayyarN%2C%20StocumE%2C%20SunJN%2C%20LindemannMJ%2C%20et%20al.%20Th17%20cells%20and%20IL-17%20receptor%20signaling%20are%20essential%20for%20mucosal%20host%20defense%20against%20oral%20candidiasis.%20J%20Exp%20Med.%202009%3B206%282%29%3A299%E2%80%93311.%20doi%3A%2010.1084%2Fjem.20081463%2019204111%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC2646568." data-title="Th17%20cells%20and%20IL-17%20receptor%20signaling%20are%20essential%20for%20mucosal%20host%20defense%20against%20oral%20candidiasis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19204111" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Th17+cells+and+IL-17+receptor+signaling+are+essential+for+mucosal+host+defense+against+oral+candidiasis+Conti+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0189503.ref035" id="pone.0189503.ref035" class="link-target"></a>Mancini M, Corradi V, Petta S, Barbieri E, Manetti F, Botta M, et al. A new nonpeptidic inhibitor of 14-3-3 induces apoptotic cell death in chronic myeloid leukemia sensitive or resistant to imatinib. J Pharmacol Exp Ther. 2011;336(3):596–604.  pmid:21041536. <ul class="reflinks"><li><a href="#" data-author="Mancini" data-cit="ManciniM%2C%20CorradiV%2C%20PettaS%2C%20BarbieriE%2C%20ManettiF%2C%20BottaM%2C%20et%20al.%20A%20new%20nonpeptidic%20inhibitor%20of%2014-3-3%20induces%20apoptotic%20cell%20death%20in%20chronic%20myeloid%20leukemia%20sensitive%20or%20resistant%20to%20imatinib.%20J%20Pharmacol%20Exp%20Ther.%202011%3B336%283%29%3A596%E2%80%93604.%20doi%3A%2010.1124%2Fjpet.110.172536%2021041536." data-title="A%20new%20nonpeptidic%20inhibitor%20of%2014-3-3%20induces%20apoptotic%20cell%20death%20in%20chronic%20myeloid%20leukemia%20sensitive%20or%20resistant%20to%20imatinib" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21041536" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+new+nonpeptidic+inhibitor+of+14-3-3+induces+apoptotic+cell+death+in+chronic+myeloid+leukemia+sensitive+or+resistant+to+imatinib+Mancini+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0189503.ref036" id="pone.0189503.ref036" class="link-target"></a>Lottersberger F, Rubert F, Baldo V, Lucchini G, Longhese MP. Functions of Saccharomyces cerevisiae 14-3-3 proteins in response to DNA damage and to DNA replication stress. Genetics. 2003;165(4):1717–32. pmid:14704161; PubMed Central PMCID: PMCPMC1462906. <ul class="reflinks"><li><a href="#" data-author="Lottersberger" data-cit="LottersbergerF%2C%20RubertF%2C%20BaldoV%2C%20LucchiniG%2C%20LongheseMP.%20Functions%20of%20Saccharomyces%20cerevisiae%2014-3-3%20proteins%20in%20response%20to%20DNA%20damage%20and%20to%20DNA%20replication%20stress.%20Genetics.%202003%3B165%284%29%3A1717%E2%80%9332.%2014704161%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC1462906." data-title="Functions%20of%20Saccharomyces%20cerevisiae%2014-3-3%20proteins%20in%20response%20to%20DNA%20damage%20and%20to%20DNA%20replication%20stress" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704161" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Functions+of+Saccharomyces+cerevisiae+14-3-3+proteins+in+response+to+DNA+damage+and+to+DNA+replication+stress+Lottersberger+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0189503.ref037" id="pone.0189503.ref037" class="link-target"></a>Lu G, Villalba M, Coscia MR, Hoffman DR, King TP. Sequence analysis and antigenic cross-reactivity of a venom allergen, antigen 5, from hornets, wasps, and yellow jackets. J Immunol. 1993;150(7):2823–30. pmid:8454859. <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuG%2C%20VillalbaM%2C%20CosciaMR%2C%20HoffmanDR%2C%20KingTP.%20Sequence%20analysis%20and%20antigenic%20cross-reactivity%20of%20a%20venom%20allergen%2C%20antigen%205%2C%20from%20hornets%2C%20wasps%2C%20and%20yellow%20jackets.%20J%20Immunol.%201993%3B150%287%29%3A2823%E2%80%9330.%208454859." data-title="Sequence%20analysis%20and%20antigenic%20cross-reactivity%20of%20a%20venom%20allergen%2C%20antigen%205%2C%20from%20hornets%2C%20wasps%2C%20and%20yellow%20jackets" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8454859" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence+analysis+and+antigenic+cross-reactivity+of+a+venom+allergen%2C+antigen+5%2C+from+hornets%2C+wasps%2C+and+yellow+jackets+Lu+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0189503.ref038" id="pone.0189503.ref038" class="link-target"></a>Lalle M, Leptourgidou F, Camerini S, Pozio E, Skoulakis EM. Interkingdom complementation reveals structural conservation and functional divergence of 14-3-3 proteins. PLoS One. 2013;8(10):e78090.  pmid:24147113; PubMed Central PMCID: PMCPMC3795638. <ul class="reflinks"><li><a href="#" data-author="Lalle" data-cit="LalleM%2C%20LeptourgidouF%2C%20CameriniS%2C%20PozioE%2C%20SkoulakisEM.%20Interkingdom%20complementation%20reveals%20structural%20conservation%20and%20functional%20divergence%20of%2014-3-3%20proteins.%20PLoS%20One.%202013%3B8%2810%29%3Ae78090.%20doi%3A%2010.1371%2Fjournal.pone.0078090%2024147113%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3795638." data-title="Interkingdom%20complementation%20reveals%20structural%20conservation%20and%20functional%20divergence%20of%2014-3-3%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24147113" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Interkingdom+complementation+reveals+structural+conservation+and+functional+divergence+of+14-3-3+proteins+Lalle+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0189503.ref039" id="pone.0189503.ref039" class="link-target"></a>Masters SC, Fu H. 14-3-3 proteins mediate an essential anti-apoptotic signal. J Biol Chem. 2001;276(48):45193–200.  pmid:11577088. <ul class="reflinks"><li><a href="#" data-author="Masters" data-cit="MastersSC%2C%20FuH.%2014-3-3%20proteins%20mediate%20an%20essential%20anti-apoptotic%20signal.%20J%20Biol%20Chem.%202001%3B276%2848%29%3A45193%E2%80%93200.%20doi%3A%2010.1074%2Fjbc.M105971200%2011577088." data-title="14-3-3%20proteins%20mediate%20an%20essential%20anti-apoptotic%20signal" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11577088" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=14-3-3+proteins+mediate+an+essential+anti-apoptotic+signal+Masters+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0189503.ref040" id="pone.0189503.ref040" class="link-target"></a>Gilden D, White T, Khmeleva N, Katz BJ, Nagel MA. Blinded search for varicella zoster virus in giant cell arteritis (GCA)-positive and GCA-negative temporal arteries. J Neurol Sci. 2016;364:141–3.  pmid:27084233; PubMed Central PMCID: PMCPMC4834150. <ul class="reflinks"><li><a href="#" data-author="Gilden" data-cit="GildenD%2C%20WhiteT%2C%20KhmelevaN%2C%20KatzBJ%2C%20NagelMA.%20Blinded%20search%20for%20varicella%20zoster%20virus%20in%20giant%20cell%20arteritis%20%28GCA%29-positive%20and%20GCA-negative%20temporal%20arteries.%20J%20Neurol%20Sci.%202016%3B364%3A141%E2%80%933.%20doi%3A%2010.1016%2Fj.jns.2016.03.020%2027084233%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC4834150." data-title="Blinded%20search%20for%20varicella%20zoster%20virus%20in%20giant%20cell%20arteritis%20%28GCA%29-positive%20and%20GCA-negative%20temporal%20arteries" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27084233" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Blinded+search+for+varicella+zoster+virus+in+giant+cell+arteritis+%28GCA%29-positive+and+GCA-negative+temporal+arteries+Gilden+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0189503.ref041" id="pone.0189503.ref041" class="link-target"></a>Sammel AM, Rosettenstein K, Post JJ, Bertouch JV. Limited evidence of active varicella zoster virus (VZV) infection in a cohort of acute giant cell arteritis (GCA) patients. Annals of the Rheumatic Diseases. 2016;75:567. <ul class="reflinks"><li><a href="#" data-author="Sammel" data-cit="SammelAM%2C%20RosettensteinK%2C%20PostJJ%2C%20BertouchJV.%20Limited%20evidence%20of%20active%20varicella%20zoster%20virus%20%28VZV%29%20infection%20in%20a%20cohort%20of%20acute%20giant%20cell%20arteritis%20%28GCA%29%20patients.%20Annals%20of%20the%20Rheumatic%20Diseases.%202016%3B75%3A567." data-title="Limited%20evidence%20of%20active%20varicella%20zoster%20virus%20%28VZV%29%20infection%20in%20a%20cohort%20of%20acute%20giant%20cell%20arteritis%20%28GCA%29%20patients" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Limited+evidence+of+active+varicella+zoster+virus+%28VZV%29+infection+in+a+cohort+of+acute+giant+cell+arteritis+%28GCA%29+patients+Sammel+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0189503.ref042" id="pone.0189503.ref042" class="link-target"></a>Cusick MF, Libbey JE, Fujinami RS. Molecular mimicry as a mechanism of autoimmune disease. Clin Rev Allergy Immunol. 2012;42(1):102–11.  pmid:22095454; PubMed Central PMCID: PMCPMC3266166. <ul class="reflinks"><li><a href="#" data-author="Cusick" data-cit="CusickMF%2C%20LibbeyJE%2C%20FujinamiRS.%20Molecular%20mimicry%20as%20a%20mechanism%20of%20autoimmune%20disease.%20Clin%20Rev%20Allergy%20Immunol.%202012%3B42%281%29%3A102%E2%80%9311.%20doi%3A%2010.1007%2Fs12016-011-8293-8%2022095454%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3266166." data-title="Molecular%20mimicry%20as%20a%20mechanism%20of%20autoimmune%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22095454" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+mimicry+as+a+mechanism+of+autoimmune+disease+Cusick+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0189503.ref043" id="pone.0189503.ref043" class="link-target"></a>Lidar M, Lipschitz N, Langevitz P, Barzilai O, Ram M, Porat-Katz BS, et al. Infectious serologies and autoantibodies in wegener's granulomatosis and other vasculitides: Novel associations disclosed using the rad BioPlex 2200. 2009. p. 649–57. <ul class="find-nolinks"></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0189503.ref044" id="pone.0189503.ref044" class="link-target"></a>Mohd Bakri M, Mohd Hussaini H, Rachel Holmes A, David Cannon R, Mary Rich A. Revisiting the association between candidal infection and carcinoma, particularly oral squamous cell carcinoma. J Oral Microbiol. 2010;2.  pmid:21523221; PubMed Central PMCID: PMCPMC3084579. <ul class="reflinks"><li><a href="#" data-author="Mohd%20Bakri" data-cit="Mohd%20BakriM%2C%20Mohd%20HussainiH%2C%20Rachel%20HolmesA%2C%20David%20CannonR%2C%20Mary%20RichA.%20Revisiting%20the%20association%20between%20candidal%20infection%20and%20carcinoma%2C%20particularly%20oral%20squamous%20cell%20carcinoma.%20J%20Oral%20Microbiol.%202010%3B2.%20doi%3A%2010.3402%2Fjom.v2i0.5780%2021523221%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3084579." data-title="Revisiting%20the%20association%20between%20candidal%20infection%20and%20carcinoma%2C%20particularly%20oral%20squamous%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21523221" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Revisiting+the+association+between+candidal+infection+and+carcinoma%2C+particularly+oral+squamous+cell+carcinoma+Mohd+Bakri+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0189503.ref045" id="pone.0189503.ref045" class="link-target"></a>Grignetti M, Carraro M, Faccini L. [An atypical presentation of a case of Horton's giant-cell arteritis]. Minerva Med. 1995;86(12):551–3. pmid:8684682. <ul class="reflinks"><li><a href="#" data-author="Grignetti" data-cit="GrignettiM%2C%20CarraroM%2C%20FacciniL.%20%5BAn%20atypical%20presentation%20of%20a%20case%20of%20Horton%27s%20giant-cell%20arteritis%5D.%20Minerva%20Med.%201995%3B86%2812%29%3A551%E2%80%933.%208684682." data-title="An%20atypical%20presentation%20of%20a%20case%20of%20Horton%27s%20giant-cell%20arteritis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8684682" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=An+atypical+presentation+of+a+case+of+Horton%27s+giant-cell+arteritis+Grignetti+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0189503.ref046" id="pone.0189503.ref046" class="link-target"></a>Kumamoto CA. Inflammation and gastrointestinal Candida colonization. Curr Opin Microbiol. 2011;14(4):386–91.  pmid:21802979; PubMed Central PMCID: PMCPMC3163673. <ul class="reflinks"><li><a href="#" data-author="Kumamoto" data-cit="KumamotoCA.%20Inflammation%20and%20gastrointestinal%20Candida%20colonization.%20Curr%20Opin%20Microbiol.%202011%3B14%284%29%3A386%E2%80%9391.%20doi%3A%2010.1016%2Fj.mib.2011.07.015%2021802979%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3163673." data-title="Inflammation%20and%20gastrointestinal%20Candida%20colonization" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21802979" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inflammation+and+gastrointestinal+Candida+colonization+Kumamoto+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0189503.ref047" id="pone.0189503.ref047" class="link-target"></a>Yang X, Lee WH, Sobott F, Papagrigoriou E, Robinson CV, Grossmann JG, et al. Structural basis for protein-protein interactions in the 14-3-3 protein family. Proc Natl Acad Sci U S A. 2006;103(46):17237–42.  pmid:17085597; PubMed Central PMCID: PMCPMC1859916. <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangX%2C%20LeeWH%2C%20SobottF%2C%20PapagrigoriouE%2C%20RobinsonCV%2C%20GrossmannJG%2C%20et%20al.%20Structural%20basis%20for%20protein-protein%20interactions%20in%20the%2014-3-3%20protein%20family.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202006%3B103%2846%29%3A17237%E2%80%9342.%20doi%3A%2010.1073%2Fpnas.0605779103%2017085597%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC1859916." data-title="Structural%20basis%20for%20protein-protein%20interactions%20in%20the%2014-3-3%20protein%20family" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17085597" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Structural+basis+for+protein-protein+interactions+in+the+14-3-3+protein+family+Yang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0189503.ref048" id="pone.0189503.ref048" class="link-target"></a>Dereeper A, Guignon V, Blanc G, Audic S, Buffet S, Chevenet F, et al. Phylogeny.fr: robust phylogenetic analysis for the non-specialist. Nucleic Acids Res. 2008;36(Web Server issue):W465–9.  pmid:18424797; PubMed Central PMCID: PMCPMC2447785. <ul class="reflinks"><li><a href="#" data-author="Dereeper" data-cit="DereeperA%2C%20GuignonV%2C%20BlancG%2C%20AudicS%2C%20BuffetS%2C%20ChevenetF%2C%20et%20al.%20Phylogeny.fr%3A%20robust%20phylogenetic%20analysis%20for%20the%20non-specialist.%20Nucleic%20Acids%20Res.%202008%3B36%28Web%20Server%20issue%29%3AW465%E2%80%939.%20doi%3A%2010.1093%2Fnar%2Fgkn180%2018424797%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC2447785." data-title="Phylogeny.fr%3A%20robust%20phylogenetic%20analysis%20for%20the%20non-specialist" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18424797" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeny.fr%3A+robust+phylogenetic+analysis+for+the+non-specialist+Dereeper+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0189503.ref049" id="pone.0189503.ref049" class="link-target"></a>Guex N, Peitsch MC, Schwede T. Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: a historical perspective. Electrophoresis. 2009;30 Suppl 1:S162–73.  pmid:19517507. <ul class="reflinks"><li><a href="#" data-author="Guex" data-cit="GuexN%2C%20PeitschMC%2C%20SchwedeT.%20Automated%20comparative%20protein%20structure%20modeling%20with%20SWISS-MODEL%20and%20Swiss-PdbViewer%3A%20a%20historical%20perspective.%20Electrophoresis.%202009%3B30%20Suppl%201%3AS162%E2%80%9373.%20doi%3A%2010.1002%2Felps.200900140%2019517507." data-title="Automated%20comparative%20protein%20structure%20modeling%20with%20SWISS-MODEL%20and%20Swiss-PdbViewer%3A%20a%20historical%20perspective" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19517507" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Automated+comparative+protein+structure+modeling+with+SWISS-MODEL+and+Swiss-PdbViewer%3A+a+historical+perspective+Guex+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0189503.ref050" id="pone.0189503.ref050" class="link-target"></a>Humphrey W, Dalke A, Schulten K. VMD: visual molecular dynamics. J Mol Graph. 1996;14(1):33–8, 27–8. pmid:8744570. <ul class="reflinks"><li><a href="#" data-author="Humphrey" data-cit="HumphreyW%2C%20DalkeA%2C%20SchultenK.%20VMD%3A%20visual%20molecular%20dynamics.%20J%20Mol%20Graph.%201996%3B14%281%29%3A33%E2%80%938%2C%2027%E2%80%938.%208744570." data-title="VMD%3A%20visual%20molecular%20dynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8744570" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=VMD%3A+visual+molecular+dynamics+Humphrey+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0189503.ref051" id="pone.0189503.ref051" class="link-target"></a>Kallberg M, Wang H, Wang S, Peng J, Wang Z, Lu H, et al. Template-based protein structure modeling using the RaptorX web server. Nat Protoc. 2012;7(8):1511–22.  pmid:22814390; PubMed Central PMCID: PMCPMC4730388. <ul class="reflinks"><li><a href="#" data-author="Kallberg" data-cit="KallbergM%2C%20WangH%2C%20WangS%2C%20PengJ%2C%20WangZ%2C%20LuH%2C%20et%20al.%20Template-based%20protein%20structure%20modeling%20using%20the%20RaptorX%20web%20server.%20Nat%20Protoc.%202012%3B7%288%29%3A1511%E2%80%9322.%20doi%3A%2010.1038%2Fnprot.2012.085%2022814390%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC4730388." data-title="Template-based%20protein%20structure%20modeling%20using%20the%20RaptorX%20web%20server" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22814390" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Template-based+protein+structure+modeling+using+the+RaptorX+web+server+Kallberg+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0189503.ref052" id="pone.0189503.ref052" class="link-target"></a>Conti HR, Baker O, Freeman AF, Jang WS, Holland SM, Li RA, et al. New mechanism of oral immunity to mucosal candidiasis in hyper-IgE syndrome. Mucosal Immunol. 2011;4(4):448–55.  pmid:21346738; PubMed Central PMCID: PMCPMC3119375. <ul class="reflinks"><li><a href="#" data-author="Conti" data-cit="ContiHR%2C%20BakerO%2C%20FreemanAF%2C%20JangWS%2C%20HollandSM%2C%20LiRA%2C%20et%20al.%20New%20mechanism%20of%20oral%20immunity%20to%20mucosal%20candidiasis%20in%20hyper-IgE%20syndrome.%20Mucosal%20Immunol.%202011%3B4%284%29%3A448%E2%80%9355.%20doi%3A%2010.1038%2Fmi.2011.5%2021346738%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC3119375." data-title="New%20mechanism%20of%20oral%20immunity%20to%20mucosal%20candidiasis%20in%20hyper-IgE%20syndrome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21346738" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=New+mechanism+of+oral+immunity+to+mucosal+candidiasis+in+hyper-IgE+syndrome+Conti+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0189503.ref053" id="pone.0189503.ref053" class="link-target"></a>Benkert P, Tosatto SC, Schomburg D. QMEAN: A comprehensive scoring function for model quality assessment. Proteins. 2008;71(1):261–77.  pmid:17932912. <ul class="reflinks"><li><a href="#" data-author="Benkert" data-cit="BenkertP%2C%20TosattoSC%2C%20SchomburgD.%20QMEAN%3A%20A%20comprehensive%20scoring%20function%20for%20model%20quality%20assessment.%20Proteins.%202008%3B71%281%29%3A261%E2%80%9377.%20doi%3A%2010.1002%2Fprot.21715%2017932912." data-title="QMEAN%3A%20A%20comprehensive%20scoring%20function%20for%20model%20quality%20assessment" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932912" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=QMEAN%3A+A+comprehensive+scoring+function+for+model+quality+assessment+Benkert+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

