



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Familial hypercholesterolemia is an autosomal dominant inherited disease characterized by elevated plasma low-density lipoprotein cholesterol (LDL-C). It is mainly caused by mutations of the low-density lipoprotein receptor (<em>LDLR</em>) gene. Currently, the methods of whole genome sequencing or whole exome sequencing for screening mutations in familial hypercholesterolemia are not applicable in China due to high cost. We performed targeted exome sequencing of 167 genes implicated in the homozygous phenotype of a proband pedigree to identify candidate mutations, validated them in the family of the proband, studied the functions of the mutant protein, and followed up serum lipid levels after treatment. We discovered that exon 9 c.1268 T&gt;C and exon 8 c.1129 T&gt;G compound heterozygous mutations in the <em>LDLR</em> gene in the proband derived from the mother and father, respectively, in which the mutation of c.1129 T&gt;G has not been reported previously. The mutant LDL-R protein had 57% and 52% binding and internalization functions, respectively, compared with that of the wild type. After 6 months of therapy, the LDL-C level of the proband decreased by more than 50% and the LDL-C of the other family members with heterozygous mutation also reduced to normal. Targeted exome sequencing is an effective method for screening mutation genes in familial hypercholesterolemia. The exon 8 and 9 mutations of the <em>LDLR</em> gene were pedigree mutations. The functions of the mutant LDL-R protein were decreased significantly compared with that of the wild type. Simvastatin plus ezetimibe was proven safe and effective in this preschool-age child.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Wu W-F, Sun L-Y, Pan X-D, Yang S-W, Wang L-Y (2014) Use of Targeted Exome Sequencing in Genetic Diagnosis of Chinese Familial Hypercholesterolemia. PLoS ONE 9(4):
           e94697.
        
        https://doi.org/10.1371/journal.pone.0094697</p><p><strong>Editor: </strong>Reiner Albert Veitia, Institut Jacques Monod, France</p><p><strong>Received: </strong>January 3, 2014; <strong>Accepted: </strong>March 18, 2014; <strong>Published: </strong> April 10, 2014</p><p><strong>Copyright: </strong> © 2014 Wu et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by grants from the Natural Science Foundation of Beijing (No. 7112040) and National Natural Science Foundation of China (No. 81170810 and 81271923). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Familial hypercholesterolemia (FH; MIM #143890) is a serious single-gene, dominant genetic disease. The main clinical manifestations are significantly elevated plasma low-density lipoprotein cholesterol (LDL-C) levels. The pathological basis of FH is LDL receptor (<em>LDLR</em>; MIM #606945) gene mutations that cause cholesterol metabolism dysfunction. Clinically, FH is divided into homozygous and heterozygous phenotypes. The latter is relatively common and leads to premature coronary heart disease; it occurs in 1∶500 people in most countries. The former is rare, occurring in 1∶1,000,000 births; however, its symptoms are severe <a href="#pone.0094697-Kim1" class="ref-tip">[1]</a>, <a href="#pone.0094697-Kastelein1" class="ref-tip">[2]</a>. However, as the most populous country in the world, China may bear a heavy burden of this genetic disorder. However, there is no applicable genetic screening method in clinical practice in China.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>To date, 3 genes <em>LDLR</em>, apolipoprotein B (APOB), and proprotein convertase subtilisin/kexin 9 (PCSK9) have been accepted <a href="#pone.0094697-Chiou1" class="ref-tip">[3]</a>. Reported mutations of FH widely spread over the coding regions of these causative genes. Genetic screening through traditional approaches, such as direct sequencing is therefore difficult. A high-throughput and cost-effective method to detect the genetic defects is needed. Whole exome sequencing has been proved to be a powerful tool to discover novel disease-related genes or genetic mutations in large genomic regions <a href="#pone.0094697-Futema1" class="ref-tip">[4]</a>–<a href="#pone.0094697-Motazacker1" class="ref-tip">[6]</a>. With the progresses on next-generation sequencing (NGS) and bioinformatics, it has been demonstrated to have higher efficiency but lower cost comparing with whole gene sequencing. However, the astronomical information amount, subsequent arduous data processing and large cost limit its widely application in practice in China. In this study, we utilized targeted exome sequencing (TES) to study genetic defects in one family, in order to establish a strategy feasible to genetic diagnosis of FH patients. We also studied the binding and internalization functions of the mutant protein and followed up the effect of lipid-lowering therapy in the pedigree.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Ethics Statement"></a>
<h3>Ethics Statement</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>The study was reviewed and approved by the Ethical Committee of Beijing Anzhen Hospital, and all participants signed an informed consent form. For those aged &lt;18 years, written consent was obtained from either parent.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Study Population and Data Collection"></a>
<h3>Study Population and Data Collection</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>This study was part of a large study aimed at screening FH mutations using TES in the Han Chinese population. An FH family was recruited at Beijing Anzhen Hospital in February 2013 (<a href="#pone-0094697-g001">Figure 1</a>). The proband (III-1) was a 5-year-old boy with distinct clinical features of skin xanthoma near the elbow, lap, and hips (<a href="#pone-0094697-g002">Figure 2</a>); his parents had no apparent clinical features but had high TC and LDL-C upon physical examination. Homozygous FH was diagnosed based on the following criteria: (1) plasma or serum LDL-C&gt;10 mmol/L; (2) presence of tendon and cutaneous xanthomas at an early age; (3) autosomal inheritance of hypercholesterolemia in relatives; (4) presence of primary hypercholesterolemia in the parents of the proband <a href="#pone.0094697-Bertolini1" class="ref-tip">[7]</a>. Secondary causes of hypercholesterolemia were excluded. The clinical features of members in the FH family were investigated (<a href="#pone-0094697-t001">Table 1</a>).</p>
<a class="link-target" id="pone-0094697-g001" name="pone-0094697-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.g001" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Pedigree and mutations of a hypercholesterolemic family.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>Arrow denotes the proband. Shaded and partially shaded shapes denote compound heterozygotes and heterozygotes with different mutations, respectively. The exon 8 c.1129 G&gt;T and exon 9 c.1268 T&gt;C mutations are depicted on the right.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.g001">
              https://doi.org/10.1371/journal.pone.0094697.g001</a></p></div><a class="link-target" id="pone-0094697-g002" name="pone-0094697-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.g002" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Clinical manifestations of the proband.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1" class="link-target"></a><p>(A) Achilles tendon thickening. (B) Cutaneous xanthomas on the elbow region. (C) Arcus corneae.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.g002">
              https://doi.org/10.1371/journal.pone.0094697.g002</a></p></div><a class="link-target" id="pone-0094697-t001" name="pone-0094697-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.t001" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Clinical Data of the Family.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.t001">
              https://doi.org/10.1371/journal.pone.0094697.t001</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Blood Sampling and Measurements"></a>
<h3>Blood Sampling and Measurements</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Blood samples were obtained from participants after a 12-h fast. TC, triglycerides (TG), LDL-C, and high-density lipoprotein cholesterol (HDL-C) were measured using routine commercial kits (Beckman Coulter, Brea, CA) and a Beckman AU 4500 automated biochemical analyzer (Beckman Coulter).</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Illumina Library Construction"></a>
<h3>Illumina Library Construction</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>Proband genomic DNA was extracted from whole blood using a DNA Extraction Kit (TianGen, Beijing, China) according to the manufacturer's instructions. DNA quantification was performed using a NanoDrop 2000 unit (Thermo Fisher Scientific, Wilmington, DE). DNA (minimum 3 μg) was used for the indexed Illumina libraries as per the manufacturer's protocol. A final library size of 350–400 bp, including adapter sequences, was selected.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Enrichment and Sequencing of Disease Genes"></a>
<h3>Enrichment and Sequencing of Disease Genes</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>In total, 167 disease genes, including the genes relevant with blood lipids <a href="#pone.0094697-Teslovich1" class="ref-tip">[8]</a> and cholic acid metabolism (<a href="#pone.0094697.s001">Table S1</a>) were selected by a gene capture strategy using a GenCap Custom Enrichment Kit (MyGenostics, Beijing, China) according to previously described technologies <a href="#pone.0094697-He1" class="ref-tip">[9]</a>, <a href="#pone.0094697-Wu1" class="ref-tip">[10]</a>. The capture experiment was conducted according to the manufacturer's protocol. Briefly, 1 μg DNA library was mixed with Buffer BL and a GenCap hypercholesterolemia probe (MyGenostics) and heated in a polymerase chain reaction (PCR) machine at 95°C for 7 min and 65°C for 2 min. 23 μL of the 65°C pre-warmed Buffer HY (MyGenostics) was added; the mixture was held at 65°C with the PCR lid heat on for 22 h for hybridization. 50 μL MyOne beads (Life Technology, Carlsbad, CA) were washed in 500 μL 1× binding buffer thrice and re-suspended in 80 μL 1× binding buffer. 64 μL 2× binding buffer was added, the mixture transferred into a tube containing 80 μL MyOne beads, and spun for 1 h on a rotator. We washed the beads once with WB1 buffer at room temperature for 15 min and WB3 buffer thrice at 65°C for 15 min. Elution buffer was used to elute the bound DNA, which was amplified as follows: 98°C for 30 s; 98°C for 25 s, 65°C for 30 s, 72°C for 30 s (15 cycles); 72°C for 5 min. We purified the PCR product using SPRI beads (Beckman Coulter) using the manufacturer's protocol. Enrichment libraries were sequenced on an Illumina HiSeq 2000 sequencer (Illumina, San Diego, CA) for 100-bp paired reads.</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Bioinformatics Analysis"></a>
<h3>Bioinformatics Analysis</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>After sequencing, we retrieved high-quality reads from raw reads by filtering out low-quality reads and adaptor sequences using the Solexa QA package <a href="#pone.0094697-Cox1" class="ref-tip">[11]</a> and cutadapt program (<a href="http://code.google.com/p/cutadapt/">http://code.google.com/p/cutadapt/</a>), respectively. We used the SOAPaligner program <a href="#pone.0094697-Li1" class="ref-tip">[12]</a> to align the clean read sequences to the human reference genome (hg19).</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2" class="link-target"></a><p>To detect exon duplication and deletions, the coverage of each position was plotted by the base positions. Higher coverage of a region denoted duplication; uncovered regions indicated deletions. After removing duplicates with Picard software <a href="#pone.0094697-Li2" class="ref-tip">[13]</a>, single-nucleotide polymorphisms (SNPs) were identified using the SOAPsnp program <a href="#pone.0094697-Li1" class="ref-tip">[12]</a> (<a href="http://soap.genomics.org.cn/soapsnp.html">http://soap.genomics.org.cn/soapsnp.html</a>). Subsequently, reads were realigned to the reference genome using the Burrows–Wheeler alignment program <a href="#pone.0094697-Li3" class="ref-tip">[14]</a>, and insertions or deletions (InDels) were identified with the Genome Analysis Toolkit <a href="#pone.0094697-DePristo1" class="ref-tip">[15]</a> (<a href="http://www.broadinstitute.org/gsa/wiki/index.php/Home_Page">http://www.broadinstitute.org/gsa/wiki/index.php/Home_Page</a>). We annotated identified SNPs and InDels using the Exome-assistant program (<a href="http://122.228.158.106/exomeassistant">http://122.228.158.106/exomeassistant</a>). Short read alignment and candidate SNP and InDel validation was performed using MagicViewer <a href="#pone.0094697-Hou1" class="ref-tip">[16]</a>. We used the PolyPhen, SIFT, PANTHER and PMUT algorithms to evaluate non-synonymous variants to determine pathogenicity <a href="#pone.0094697-Jin1" class="ref-tip">[17]</a>. Sequencing data were deposited in NIH Short Read Archive (SRR1169449).</p>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="Expanded Validation"></a>
<h3>Expanded Validation</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>DNA samples from all family members were obtained for Sanger sequencing. We also PCR-amplified the coding regions of the mutations identified as described for conventional direct sequencing. We used an ABI 3500 Genetic Analyzer (Applied Biosystems, Foster City, CA) to cycle-sequence purified PCR products. We analyzed the Sanger sequencing results using Mutation Surveyor (Softgenetics, State College, PA), reconfirming them using the same procedure.</p>
</div>

<div id="section8" class="section toc-section"><a id="s2h" name="s2h" class="link-target" title="Construction of Mutant"></a>
<h3>Construction of Mutant</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>The C356G LDL-R mutant was constructed according to the protocol previously described <a href="#pone.0094697-Wang1" class="ref-tip">[18]</a>.</p>
</div>

<div id="section9" class="section toc-section"><a id="s2i" name="s2i" class="link-target" title="Functional Studies of the Mutant"></a>
<h3>Functional Studies of the Mutant</h3>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1" class="link-target"></a><p>LDL-R binding and internalization functions were measured as previously describe <a href="#pone.0094697-Wang1" class="ref-tip">[18]</a>.</p>
</div>

<div id="section10" class="section toc-section"><a id="s2j" name="s2j" class="link-target" title="Lipid-lowering Therapy, Blood Lipid Target, and Follow-up"></a>
<h3>Lipid-lowering Therapy, Blood Lipid Target, and Follow-up</h3>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1" class="link-target"></a><p>According to whether the proband reached the target ≥50% LDL-C reduction from baseline <a href="#pone.0094697-European1" class="ref-tip">[19]</a>, we treated him with 10 and 20 mg simvastatin daily for 2 months, and then 20 mg simvastatin plus 10 mg ezetimibe daily for the next 2 months (<a href="#pone-0094697-g003">Figure 3</a>). Heterozygous patients in the family received 20 mg simvastatin daily.</p>
<a class="link-target" id="pone-0094697-g003" name="pone-0094697-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.g003" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> Serum lipid changes after lipid-lowering therapy.</div><p class="caption_target"><a id="article1.body1.sec2.sec10.fig1.caption1.p1" name="article1.body1.sec2.sec10.fig1.caption1.p1" class="link-target"></a><p>Plasma concentrations of TC (square), TG (triangle), LDL-C (diamond), and HDL-C (circle) in the proband. S, simvastatin; E, ezetimibe.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.g003">
              https://doi.org/10.1371/journal.pone.0094697.g003</a></p></div><a id="article1.body1.sec2.sec10.p2" name="article1.body1.sec2.sec10.p2" class="link-target"></a><p>Blood lipids were re-examined every month until 6 months later. The subjects were followed for serious adverse reactions.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="General Clinical Data"></a>
<h3>General Clinical Data</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>The general clinical data of the family are listed in <a href="#pone-0094697-t001">Table 1</a>. TC and LDL-C levels in the proband (III-1) were extremely high before treatment and he exhibited typical clinical features. The maternal grandfather (I-2), paternal grandmother (I-3), mother (II-1), father (II-2), uncle (II-3), and cousin (III-3) had a clinical diagnosis of heterozygous FH and high TC and LDL-C levels when recruited; only II-3 had cutaneous xanthoma. Other family members had normal serum lipid levels. All subjects had normal HDL-C levels, no tendon xanthoma, and we confirmed that there was no evidence of coronary heart disease.</p>
</div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="TES Identification of Candidate Mutations"></a>
<h3>TES Identification of Candidate Mutations</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>We performed TES of 167 genes implicated in FH. The average sequencing depths on the targeted regions of samples that underwent TES was 640.32 (<a href="#pone.0094697.s002">Table S2</a>). The sample had 96.9% coverage of the targeted regions. Meanwhile, there was 92.3% and 89.3% coverage of targeted exons for 10 and 20 reads, respectively. Via SOAPsnp <a href="#pone.0094697-Li2" class="ref-tip">[13]</a>, an average 650 variants were identified in the sample. Among them were 386 non-synonymous variants, missense, nonsense, and splicing variants. This was narrowed down to 46 by excluding variants reported in HapMap 28 and the SNP release of the 1000 Genome Project with minor allele frequency&gt;0.05. For missense variants, computational prediction by PolyPhen, SIFT, PANTHER, PMUT and consistency of genetic transmission mode further narrowed the number of candidate mutations to &lt;8. For the 7 coding InDels initially identified in the sample using the Genome Analysis Toolkit program <a href="#pone.0094697-DePristo1" class="ref-tip">[15]</a>, we identified 2 compound heterozygous variants(I402T and C356G) on LDLR gene that correlated with the disease phenotype (<a href="#pone-0094697-g004">Figure 4</a>).</p>
<a class="link-target" id="pone-0094697-g004" name="pone-0094697-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.g004" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Flow chart of the data analysis.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.g004">
              https://doi.org/10.1371/journal.pone.0094697.g004</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="Expanded Familial Validation and Sanger Sequencing Confirmation"></a>
<h3>Expanded Familial Validation and Sanger Sequencing Confirmation</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>We validated TES results from the proband to the entire family using Sanger sequencing. Sanger sequencing of the coding region confirmed that the I402T and C356G mutations were transmitted from the paternal grandmother (I-3) and the maternal grandfather (I-2), respectively. And, the heterozygous <em>LDLR</em> mutation I402T was carried by the father (II-2), uncle (II-3), and cousin (III-3), while the heterozygous <em>LDLR</em> mutation C356G was carried by the mother (II-1). But the sibling (III-2) carried no <em>LDLR</em> mutation. So the proband (III-1) carried 2 compound heterozygous <em>LDLR</em> mutations (C356G and I402T) (<a href="#pone-0094697-g001">Figure 1</a>). Furthermore, we performed co-segregation analysis to confirm the extracted mutations in the pedigree. All mutations were confirmed to co-segregate well with the disease in the family.</p>
</div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="Effect of C356G LDLR Mutation on LDL-R Function"></a>
<h3>Effect of C356G <em>LDLR</em> Mutation on LDL-R Function</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>The effects of the C356G mutation on LDL-R function was investigated by transient expression in HEK-293 cells. To measure LDL-R binding and internalization functions, HEK-293 cells were harvested 48 h after transfection and analyzed by flow cytometry. Analyses of antibody-labeled cells and measurement of activity revealed that receptors with the C356G mutation had 57% binding and 52% internalization activity compared with that of the wild-type receptor. The majority of EGFP-positive cells could not bind and internalize LDL effectively (<a href="#pone-0094697-g005">Figure 5</a>, <a href="#pone-0094697-t002">Table 2</a>).</p>
<a class="link-target" id="pone-0094697-g005" name="pone-0094697-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.g005" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> Characterization of C356G mutation LDL-R binding and internalization function.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1" class="link-target"></a><p>(A, D) Wild-type LDL-R binding and internalization function. (B, E) C356G LDL-R binding and internalization function. (C, F) Histogram of wild-type and C356G mutant LDL-R binding and internalization function.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.g005">
              https://doi.org/10.1371/journal.pone.0094697.g005</a></p></div><a class="link-target" id="pone-0094697-t002" name="pone-0094697-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.t002" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> LDL-R Function Comparisons.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.t002">
              https://doi.org/10.1371/journal.pone.0094697.t002</a></p></div></div>

<div id="section5" class="section toc-section"><a id="s3e" name="s3e" class="link-target" title="Family Lipid Level Follow-up"></a>
<h3>Family Lipid Level Follow-up</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>Lipid-lowering therapy was started upon the proband being diagnosed with FH. He was initially given 10 mg simvastatin daily to treat hyperlipidemia, but after 2 months, his LDL-C decreased by only 25% from baseline. We increased the dose to 20 mg daily; 2 months later, his LDL-C levels had decreased by about 35% from baseline, meaning the target level still had not been achieved. Therefore, we combined 20 mg simvastatin with 10 mg ezetimibe daily. At the end of the fifth month, his LDL-C levels had declined by 50% as compared to baseline and remained at about 7.4 mmol/L (<a href="#pone-0094697-g003">Figure 3</a>). After 6 months, lipid levels in heterozygous patients in the family had decreased to normal levels (<a href="#pone-0094697-t003">Table 3</a>). All subjects experienced no adverse reaction during treatment.</p>
<a class="link-target" id="pone-0094697-t003" name="pone-0094697-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0094697.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0094697.t003" data-doi="info:doi/10.1371/journal.pone.0094697" data-uri="info:doi/10.1371/journal.pone.0094697.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0094697.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0094697.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0094697.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0094697.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Lipid Levels of Heterozygous Family Members after 6-month 20 mg Simvastatin Treatment.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0094697.t003">
              https://doi.org/10.1371/journal.pone.0094697.t003</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>Estimated with the prevalence of heterozygous 1/500, there should be about 2.6 million potential FH patients in China, but the definitions of patients and potential patients are inconsistent. The first reason is that both doctors and patients have poor understanding of FH, and serum lipid levels are not examined during routine physical examination of children. On the other hand, only a few laboratories in China have the capability for detecting gene(s) mutations of FH. Therefore, a significant number of FH patients are not diagnosed and treated in a timely manner. Foreign research has shown that early diagnosis and intervention can delay the occurrence of complications of atherosclerosis and coronary heart disease in FH patients <a href="#pone.0094697-EspinheiraMdo1" class="ref-tip">[20]</a>–<a href="#pone.0094697-Elis1" class="ref-tip">[22]</a>. Traditional screening for FH mutations is also infeasible because distribution of the mutation sites is scattered. Whole exome sequencing has been proved to be a powerful tool to discover novel disease-related genes or genetic mutations in large genomic regions <a href="#pone.0094697-Futema1" class="ref-tip">[4]</a>–<a href="#pone.0094697-Motazacker1" class="ref-tip">[6]</a>, but clinical application in China is rarely accepted due to high cost. In the present study, we carried out TES for 167 genes implicated in FH and found that the average sequencing depth of the target genes was &gt;640, coverage of the target region was about 97%. The data from sequencing was about 650 Mb one sample (<a href="#pone.0094697.s002">Table S2</a>), equal to about 10% of whole exome sequencing. Therefore, we have proved that deep exome sequencing of target 167 genes is a fast and efficient method to assist the diagnosis. The cost was saved at least 50% compared with whole exome sequencing, even less the amount of work saved. Notably, only a single sample from the proband was sufficient for identifying the causative mutation in the family; finalization could be enhanced using intra-familial mutation validation and co-segregation analysis. Therefore, there are 3 advantages of TES: 1) the genetic screening range can be reduced greatly; 2) data analysis can be performed with high efficiency; 3) it is much less expensive than whole exome sequencing.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>The latest version of the LDL-R FH database of the University College London (<a href="http://www.ucl.ac.uk/ldlr">www.ucl.ac.uk/ldlr</a>, accessed Jan 1, 2014) states that &gt;1125 <em>LDLR</em> gene variants worldwide, including different point mutations, small and large fragment deletions, insertions, rearrangements, and many new mutations, were recently discovered. The I402T mutation, transmitted from the paternal grandmother in the present study, has been reported in Switzerland, France, and Malaysia <a href="#pone.0094697-Ekstrm1" class="ref-tip">[23]</a>–<a href="#pone.0094697-Amsellem1" class="ref-tip">[25]</a>, with binding and internalization functions of the mutant LDL-R protein decreasing to 59% and 54%, respectively, compared with the normal protein <a href="#pone.0094697-Chang1" class="ref-tip">[26]</a>. The C356Y mutation (nucleotide change to c.1130G&gt;A) was first reported by Ekstrom in 1998 <a href="#pone.0094697-Ekstrom1" class="ref-tip">[27]</a>, but the C356G mutation (nucleotide change to c.1129T&gt;G) has not been reported previously. In this study, we concentrated on the effect of the novel C356G mutation on LDL-R function. Flow cytometry revealed that receptors with the C356G mutation had 57% binding activity compared with that of the wild-type receptor. The mutant receptor also displayed 52% internalization function compared to that of the wild-type. In the LDL-R protein, the C356G mutation is located in the epidermal growth factor precursor homology domain B region, a disulfide- and cysteine-rich region. As this sequence is highly conserved, it may play an important functional role. It has been proven that the cysteine-to-glycine mutation has a marked effect on protein folding, impairing receptor function <a href="#pone.0094697-Chang1" class="ref-tip">[26]</a>. The epidermal growth factor precursor homology domain has been implicated in low-pH receptor recycling and lipoprotein release <a href="#pone.0094697-Jeon1" class="ref-tip">[28]</a>; therefore, mutations in this region will disrupt the natural primary function of the uptake of cholesterol-carrying particles into cells, thus lipids remain in the blood, resulting in hyperlipidemia.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>The proband was diagnosed with compound heterozygous FH and was administrated with cholesterol synthesis inhibitor, simvastatin (dose titrated from 10 to 20 mg daily), accompany with diet therapy, though evidence of simvastatin use in child is limited. Four months later, the LDL-C decreased just by less than 50%, did not reach the target. We assumed that such a reduction amount may arise from either possible statin escape <a href="#pone.0094697-Wu2" class="ref-tip">[29]</a> or the sole blocking function of cholesterol synthesis but not cholesterol absorption. We combined simvastatin with the cholesterol absorption inhibitor ezetimibe (10 mg daily); after 1 month, his LDL-C levels were reduced by 50% from baseline, consistent with the report by Catapano et al <a href="#pone.0094697-Catapano1" class="ref-tip">[30]</a>. The therapy failed to lower LDL-C down to normal range. Nevertheless, the target (reduction ≥ 50%) recommended by the guideline was eventually met <a href="#pone.0094697-European1" class="ref-tip">[19]</a>. During treatment, the skin xanthoma partly lightened and no subject developed the adverse reactions of liver function damage and creatine phosphokinase elevation; the growth in terms of height and weight of the children was unaffected as well. This lipid-lowering therapy not only is suitable for adult patients with hyperlipidemia <a href="#pone.0094697-Ballantyne1" class="ref-tip">[31]</a>, but also is safe for this preschool-age (&lt;6 years) compound heterozygous child.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Diagnosis of FH in China largely depends on clinical features, but patients with heterozygous mutations, such as the parents of the proband, often do not exhibit detectable clinical features. FH is detected in such patients only by measuring their cholesterol levels and by genetic diagnosis. In such patients, preventive treatment followed by early diagnosis is of great importance to avoid damage stemming from long-term elevation of cholesterol levels.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>In summary, we successfully diagnosed FH genetically using next-generation sequencing methods and proved that it is a rapid, high-throughput, and efficient screening strategy. Briefly, TES of the 167 genes was sufficient and clinically useful for revealing genetic defects in patients with genetic dyslipidemia disease comprehensively.</p>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0094697"></div><div class="supplementary-material"><a name="pone.0094697.s001" id="pone.0094697.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0094697.s001">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">List of the genes captured in the present study.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0094697.s001">https://doi.org/10.1371/journal.pone.0094697.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0094697.s002" id="pone.0094697.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0094697.s002">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Data summary of the targeted exome sequencing.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0094697.s002">https://doi.org/10.1371/journal.pone.0094697.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank all family members for their participation in this study.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: LYW SWY. Performed the experiments: WFW XDP. Analyzed the data: LYS WFW. Contributed reagents/materials/analysis tools: XDP. Wrote the paper: WFW.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0094697-Kim1" id="pone.0094697-Kim1" class="link-target"></a>Kim YR, Han KH (2013) Familial hypercholesterolemia and the atherosclerotic disease. <em>Korean Circ J</em> 43: 363–367. <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimYR%2C%20HanKH%20%282013%29%20Familial%20hypercholesterolemia%20and%20the%20atherosclerotic%20disease.%20Korean%20Circ%20J%2043%3A%20363%E2%80%93367." data-title="Familial%20hypercholesterolemia%20and%20the%20atherosclerotic%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Familial+hypercholesterolemia+and+the+atherosclerotic+disease+Kim+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0094697-Kastelein1" id="pone.0094697-Kastelein1" class="link-target"></a>Kastelein JJ, van Leuven SI, Burgess L, Evans GW, Kuivenhoven JA, et al. (2007) Effect of torcetrapib on carotid atherosclerosis in familial hypercholesterolemia. <em>N Engl J Med</em> 356: 1620–1630. <ul class="reflinks"><li><a href="#" data-author="Kastelein" data-cit="KasteleinJJ%2C%20van%20LeuvenSI%2C%20BurgessL%2C%20EvansGW%2C%20KuivenhovenJA%2C%20et%20al.%20%282007%29%20Effect%20of%20torcetrapib%20on%20carotid%20atherosclerosis%20in%20familial%20hypercholesterolemia.%20N%20Engl%20J%20Med%20356%3A%201620%E2%80%931630." data-title="Effect%20of%20torcetrapib%20on%20carotid%20atherosclerosis%20in%20familial%20hypercholesterolemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Effect+of+torcetrapib+on+carotid+atherosclerosis+in+familial+hypercholesterolemia+Kastelein+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0094697-Chiou1" id="pone.0094697-Chiou1" class="link-target"></a>Chiou KR, Charng MJ (2010) Detection of mutations and large rearrangements of the low-density lipoprotein receptor gene in Taiwanese patients with familial hypercholesterolemia. <em>Am J Cardiol</em> 105: 1752–1758. <ul class="reflinks"><li><a href="#" data-author="Chiou" data-cit="ChiouKR%2C%20CharngMJ%20%282010%29%20Detection%20of%20mutations%20and%20large%20rearrangements%20of%20the%20low-density%20lipoprotein%20receptor%20gene%20in%20Taiwanese%20patients%20with%20familial%20hypercholesterolemia.%20Am%20J%20Cardiol%20105%3A%201752%E2%80%931758." data-title="Detection%20of%20mutations%20and%20large%20rearrangements%20of%20the%20low-density%20lipoprotein%20receptor%20gene%20in%20Taiwanese%20patients%20with%20familial%20hypercholesterolemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Detection+of+mutations+and+large+rearrangements+of+the+low-density+lipoprotein+receptor+gene+in+Taiwanese+patients+with+familial+hypercholesterolemia+Chiou+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0094697-Futema1" id="pone.0094697-Futema1" class="link-target"></a>Futema M, Plagnol V, Whittall RA, Neil HA (2012) Simon Broome Register Group, et al (2012) Use of targeted exome sequencing as a diagnostic tool for Familial Hypercholesterolaemia. J Med Genet 49: 644–649. <ul class="reflinks"><li><a href="#" data-author="Futema" data-cit="FutemaM%2C%20PlagnolV%2C%20WhittallRA%2C%20NeilHA%20%282012%29%20Simon%20Broome%20Register%20Group%2C%20et%20al%20%282012%29%20Use%20of%20targeted%20exome%20sequencing%20as%20a%20diagnostic%20tool%20for%20Familial%20Hypercholesterolaemia.%20J%20Med%20Genet%2049%3A%20644%E2%80%93649." data-title="Use%20of%20targeted%20exome%20sequencing%20as%20a%20diagnostic%20tool%20for%20Familial%20Hypercholesterolaemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Use+of+targeted+exome+sequencing+as+a+diagnostic+tool+for+Familial+Hypercholesterolaemia+Futema+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0094697-Vandrovcova1" id="pone.0094697-Vandrovcova1" class="link-target"></a>Vandrovcova J, Thomas ER, Atanur SS, Norsworthy PJ, Neuwirth C, et al. (2013) The use of next-generation sequencing in clinical diagnosis of familial hypercholesterolemia. Genet Med 15: 948–957. <ul class="reflinks"><li><a href="#" data-author="Vandrovcova" data-cit="VandrovcovaJ%2C%20ThomasER%2C%20AtanurSS%2C%20NorsworthyPJ%2C%20NeuwirthC%2C%20et%20al.%20%282013%29%20The%20use%20of%20next-generation%20sequencing%20in%20clinical%20diagnosis%20of%20familial%20hypercholesterolemia.%20Genet%20Med%2015%3A%20948%E2%80%93957." data-title="The%20use%20of%20next-generation%20sequencing%20in%20clinical%20diagnosis%20of%20familial%20hypercholesterolemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+use+of+next-generation+sequencing+in+clinical+diagnosis+of+familial+hypercholesterolemia+Vandrovcova+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0094697-Motazacker1" id="pone.0094697-Motazacker1" class="link-target"></a>Motazacker MM, Pirruccello J, Huijgen R, Do R, Gabriel S, et al. (2012) Advances in genetics show the need for extending screening strategies for autosomal dominant hypercholesterolaemia. Eur Heart J 33: 1360–1366. <ul class="reflinks"><li><a href="#" data-author="Motazacker" data-cit="MotazackerMM%2C%20PirruccelloJ%2C%20HuijgenR%2C%20DoR%2C%20GabrielS%2C%20et%20al.%20%282012%29%20Advances%20in%20genetics%20show%20the%20need%20for%20extending%20screening%20strategies%20for%20autosomal%20dominant%20hypercholesterolaemia.%20Eur%20Heart%20J%2033%3A%201360%E2%80%931366." data-title="Advances%20in%20genetics%20show%20the%20need%20for%20extending%20screening%20strategies%20for%20autosomal%20dominant%20hypercholesterolaemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Advances+in+genetics+show+the+need+for+extending+screening+strategies+for+autosomal+dominant+hypercholesterolaemia+Motazacker+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0094697-Bertolini1" id="pone.0094697-Bertolini1" class="link-target"></a>Bertolini S, Cassanelli S, Garuti R, Ghisellini M, Simone ML, et al. (1999) Analysis of LDL receptor gene mutations in Italian patients with homozygous familial hypercholesterolemia. Arterioscler Thromb Vasc Biol 19: 408–418. <ul class="reflinks"><li><a href="#" data-author="Bertolini" data-cit="BertoliniS%2C%20CassanelliS%2C%20GarutiR%2C%20GhiselliniM%2C%20SimoneML%2C%20et%20al.%20%281999%29%20Analysis%20of%20LDL%20receptor%20gene%20mutations%20in%20Italian%20patients%20with%20homozygous%20familial%20hypercholesterolemia.%20Arterioscler%20Thromb%20Vasc%20Biol%2019%3A%20408%E2%80%93418." data-title="Analysis%20of%20LDL%20receptor%20gene%20mutations%20in%20Italian%20patients%20with%20homozygous%20familial%20hypercholesterolemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+of+LDL+receptor+gene+mutations+in+Italian+patients+with+homozygous+familial+hypercholesterolemia+Bertolini+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0094697-Teslovich1" id="pone.0094697-Teslovich1" class="link-target"></a>Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM, et al. (2010) Biological, clinical and population relevance of 95 loci for blood lipids. Nature 466: 707–713. <ul class="reflinks"><li><a href="#" data-author="Teslovich" data-cit="TeslovichTM%2C%20MusunuruK%2C%20SmithAV%2C%20EdmondsonAC%2C%20StylianouIM%2C%20et%20al.%20%282010%29%20Biological%2C%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids.%20Nature%20466%3A%20707%E2%80%93713." data-title="Biological%2C%20clinical%20and%20population%20relevance%20of%2095%20loci%20for%20blood%20lipids" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Biological%2C+clinical+and+population+relevance+of+95+loci+for+blood+lipids+Teslovich+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0094697-He1" id="pone.0094697-He1" class="link-target"></a>He J, Wu J, Jiao Y, Wagner-Johnston N, Ambinder RF, et al. (2011) IgH gene rearrangements as plasma biomarkers in Non-Hodgkin's lymphoma patients. Oncotarget 2: 178–185. <ul class="reflinks"><li><a href="#" data-author="He" data-cit="HeJ%2C%20WuJ%2C%20JiaoY%2C%20Wagner-JohnstonN%2C%20AmbinderRF%2C%20et%20al.%20%282011%29%20IgH%20gene%20rearrangements%20as%20plasma%20biomarkers%20in%20Non-Hodgkin%27s%20lymphoma%20patients.%20Oncotarget%202%3A%20178%E2%80%93185." data-title="IgH%20gene%20rearrangements%20as%20plasma%20biomarkers%20in%20Non-Hodgkin%27s%20lymphoma%20patients" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=IgH+gene+rearrangements+as+plasma+biomarkers+in+Non-Hodgkin%27s+lymphoma+patients+He+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0094697-Wu1" id="pone.0094697-Wu1" class="link-target"></a>Wu J, Matthaei H, Maitra A, Dal Molin M, Wood LD, et al. (2011) Recurrent GNAS mutations define an unexpected pathway for pancreatic cyst development. Sci Transl Med 3: 92ra66. <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuJ%2C%20MatthaeiH%2C%20MaitraA%2C%20Dal%20MolinM%2C%20WoodLD%2C%20et%20al.%20%282011%29%20Recurrent%20GNAS%20mutations%20define%20an%20unexpected%20pathway%20for%20pancreatic%20cyst%20development.%20Sci%20Transl%20Med%203%3A%2092ra66." data-title="Recurrent%20GNAS%20mutations%20define%20an%20unexpected%20pathway%20for%20pancreatic%20cyst%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Recurrent+GNAS+mutations+define+an+unexpected+pathway+for+pancreatic+cyst+development+Wu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0094697-Cox1" id="pone.0094697-Cox1" class="link-target"></a>Cox MP, Peterson DA, Biggs PJ (2010) SolexaQA: At-a-glance quality assessment of Illumina second-generation sequencing data. BMC Bioinformatics 11: 485. <ul class="reflinks"><li><a href="#" data-author="Cox" data-cit="CoxMP%2C%20PetersonDA%2C%20BiggsPJ%20%282010%29%20SolexaQA%3A%20At-a-glance%20quality%20assessment%20of%20Illumina%20second-generation%20sequencing%20data.%20BMC%20Bioinformatics%2011%3A%20485." data-title="SolexaQA%3A%20At-a-glance%20quality%20assessment%20of%20Illumina%20second-generation%20sequencing%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=SolexaQA%3A+At-a-glance+quality+assessment+of+Illumina+second-generation+sequencing+data+Cox+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0094697-Li1" id="pone.0094697-Li1" class="link-target"></a>Li R, Yu C, Li Y, Lam TW, Yiu SM, et al. (2009) SOAP2: an improved ultrafast tool for short read alignment. Bioinformatics 25: 1966–1967. <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiR%2C%20YuC%2C%20LiY%2C%20LamTW%2C%20YiuSM%2C%20et%20al.%20%282009%29%20SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment.%20Bioinformatics%2025%3A%201966%E2%80%931967." data-title="SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=SOAP2%3A+an+improved+ultrafast+tool+for+short+read+alignment+Li+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0094697-Li2" id="pone.0094697-Li2" class="link-target"></a>Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, et al. (2009) The Sequence Alignment/Map format and SAMtools. Bioinformatics 25: 2078–2079. <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiH%2C%20HandsakerB%2C%20WysokerA%2C%20FennellT%2C%20RuanJ%2C%20et%20al.%20%282009%29%20The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools.%20Bioinformatics%2025%3A%202078%E2%80%932079." data-title="The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Sequence+Alignment%2FMap+format+and+SAMtools+Li+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0094697-Li3" id="pone.0094697-Li3" class="link-target"></a>Li H, Durbin R (2009) Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25: 1754–1760. <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiH%2C%20DurbinR%20%282009%29%20Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform.%20Bioinformatics%2025%3A%201754%E2%80%931760." data-title="Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Fast+and+accurate+short+read+alignment+with+Burrows-Wheeler+transform+Li+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0094697-DePristo1" id="pone.0094697-DePristo1" class="link-target"></a>DePristo MA, Banks E, Poplin R, Garimella KV, Maguire JR, et al. (2011) A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet 43: 491–498. <ul class="reflinks"><li><a href="#" data-author="DePristo" data-cit="DePristoMA%2C%20BanksE%2C%20PoplinR%2C%20GarimellaKV%2C%20MaguireJR%2C%20et%20al.%20%282011%29%20A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data.%20Nat%20Genet%2043%3A%20491%E2%80%93498." data-title="A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+framework+for+variation+discovery+and+genotyping+using+next-generation+DNA+sequencing+data+DePristo+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0094697-Hou1" id="pone.0094697-Hou1" class="link-target"></a>Hou H, Zhao F, Zhou L, Zhu E, Teng H, et al.. (2010) MagicViewer: integrated solution for next-generation sequencing data visualization and genetic variation detection and annotation. Nucleic Acids Res 38(Web Server issue): : W732–736. <ul class="find-nolinks"></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0094697-Jin1" id="pone.0094697-Jin1" class="link-target"></a>Jin ZB, Mandai M, Yokota T, Higuchi K, Ohmori K, et al. (2008) Identifying pathogenic genetic background of simplex or multiplex retinitis pigmentosa patients: a large scale mutation screening study. J Med Genet 45: 465–472. <ul class="reflinks"><li><a href="#" data-author="Jin" data-cit="JinZB%2C%20MandaiM%2C%20YokotaT%2C%20HiguchiK%2C%20OhmoriK%2C%20et%20al.%20%282008%29%20Identifying%20pathogenic%20genetic%20background%20of%20simplex%20or%20multiplex%20retinitis%20pigmentosa%20patients%3A%20a%20large%20scale%20mutation%20screening%20study.%20J%20Med%20Genet%2045%3A%20465%E2%80%93472." data-title="Identifying%20pathogenic%20genetic%20background%20of%20simplex%20or%20multiplex%20retinitis%20pigmentosa%20patients%3A%20a%20large%20scale%20mutation%20screening%20study" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identifying+pathogenic+genetic+background+of+simplex+or+multiplex+retinitis+pigmentosa+patients%3A+a+large+scale+mutation+screening+study+Jin+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0094697-Wang1" id="pone.0094697-Wang1" class="link-target"></a>Wang L, Lin J, Liu S, Cao S, Liu J, et al. (2009) Mutations in the LDL receptor gene in four Chinese homozygous familial hypercholesterolemia phenotype patients. Nutr Metab Cardiovasc Dis 19: 391–400. <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangL%2C%20LinJ%2C%20LiuS%2C%20CaoS%2C%20LiuJ%2C%20et%20al.%20%282009%29%20Mutations%20in%20the%20LDL%20receptor%20gene%20in%20four%20Chinese%20homozygous%20familial%20hypercholesterolemia%20phenotype%20patients.%20Nutr%20Metab%20Cardiovasc%20Dis%2019%3A%20391%E2%80%93400." data-title="Mutations%20in%20the%20LDL%20receptor%20gene%20in%20four%20Chinese%20homozygous%20familial%20hypercholesterolemia%20phenotype%20patients" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mutations+in+the+LDL+receptor+gene+in+four+Chinese+homozygous+familial+hypercholesterolemia+phenotype+patients+Wang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0094697-European1" id="pone.0094697-European1" class="link-target"></a>European Association for Cardiovascular Prevention &amp; Rehabilitation (2011) Reiner Z, Catapano AL, De Backer G, Graham I, et al. (2011) ESC/EAS Guidelines for the management of dyslipidaemias: the Task Force for the management of dyslipidaemias of the European Society of Cardiology (ESC) and the European Atherosclerosis Society (EAS). Eur Heart J 32: 1769–1818. <ul class="reflinks"><li><a href="#" data-author="Reiner" data-cit="European%20Association%20for%20Cardiovascular%20Prevention%20%26%20Rehabilitation%20%282011%29%20ReinerZ%2C%20CatapanoAL%2C%20De%20BackerG%2C%20GrahamI%2C%20et%20al.%20%282011%29%20ESC%2FEAS%20Guidelines%20for%20the%20management%20of%20dyslipidaemias%3A%20the%20Task%20Force%20for%20the%20management%20of%20dyslipidaemias%20of%20the%20European%20Society%20of%20Cardiology%20%28ESC%29%20and%20the%20European%20Atherosclerosis%20Society%20%28EAS%29.%20Eur%20Heart%20J%2032%3A%201769%E2%80%931818." data-title="ESC%2FEAS%20Guidelines%20for%20the%20management%20of%20dyslipidaemias%3A%20the%20Task%20Force%20for%20the%20management%20of%20dyslipidaemias%20of%20the%20European%20Society%20of%20Cardiology%20%28ESC%29%20and%20the%20European%20Atherosclerosis%20Society%20%28EAS%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=ESC%2FEAS+Guidelines+for+the+management+of+dyslipidaemias%3A+the+Task+Force+for+the+management+of+dyslipidaemias+of+the+European+Society+of+Cardiology+%28ESC%29+and+the+European+Atherosclerosis+Society+%28EAS%29+Reiner+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0094697-EspinheiraMdo1" id="pone.0094697-EspinheiraMdo1" class="link-target"></a>Espinheira Mdo C, Vasconcelos C, Medeiros AM, Alves AC, Bourbon M, et al. (2013) Hypercholesterolemia–a disease with expression from childhood. Rev Port Cardiol 32: 379–386. <ul class="reflinks"><li><a href="#" data-author="Espinheira%20Mdo" data-cit="Espinheira%20MdoC%2C%20VasconcelosC%2C%20MedeirosAM%2C%20AlvesAC%2C%20BourbonM%2C%20et%20al.%20%282013%29%20Hypercholesterolemia%E2%80%93a%20disease%20with%20expression%20from%20childhood.%20Rev%20Port%20Cardiol%2032%3A%20379%E2%80%93386." data-title="Hypercholesterolemia%E2%80%93a%20disease%20with%20expression%20from%20childhood" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hypercholesterolemia%E2%80%93a+disease+with+expression+from+childhood+Espinheira+Mdo+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0094697-Alonso1" id="pone.0094697-Alonso1" class="link-target"></a>Alonso R, Mata P, Zambón D, Mata N, Fuentes-Jiménez F (2013) Early diagnosis and treatment of familial hypercholesterolemia: improving patient outcomes. Expert Rev Cardiovasc Ther 11: 327–342. <ul class="reflinks"><li><a href="#" data-author="Alonso" data-cit="AlonsoR%2C%20MataP%2C%20Zamb%C3%B3nD%2C%20MataN%2C%20Fuentes-Jim%C3%A9nezF%20%282013%29%20Early%20diagnosis%20and%20treatment%20of%20familial%20hypercholesterolemia%3A%20improving%20patient%20outcomes.%20Expert%20Rev%20Cardiovasc%20Ther%2011%3A%20327%E2%80%93342." data-title="Early%20diagnosis%20and%20treatment%20of%20familial%20hypercholesterolemia%3A%20improving%20patient%20outcomes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Early+diagnosis+and+treatment+of+familial+hypercholesterolemia%3A+improving+patient+outcomes+Alonso+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0094697-Elis1" id="pone.0094697-Elis1" class="link-target"></a>Elis A, Zhou R, Stein EA (2013) Treatment of familial hypercholesterolaemia in children and adolescents in the last three decades. Cardiol Young 10: 1–5. <ul class="reflinks"><li><a href="#" data-author="Elis" data-cit="ElisA%2C%20ZhouR%2C%20SteinEA%20%282013%29%20Treatment%20of%20familial%20hypercholesterolaemia%20in%20children%20and%20adolescents%20in%20the%20last%20three%20decades.%20Cardiol%20Young%2010%3A%201%E2%80%935." data-title="Treatment%20of%20familial%20hypercholesterolaemia%20in%20children%20and%20adolescents%20in%20the%20last%20three%20decades" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Treatment+of+familial+hypercholesterolaemia+in+children+and+adolescents+in+the+last+three+decades+Elis+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0094697-Ekstrm1" id="pone.0094697-Ekstrm1" class="link-target"></a>Ekström U, Abrahamson M, Sveger T, Lombardi P, Nilsson-Ehle P (1995) An efficient screening procedure detecting six novel mutations in the LDL receptor gene in Swedish children with hypercholesterolemia. Hum Genet 96: 147–150. <ul class="reflinks"><li><a href="#" data-author="Ekstr%C3%B6m" data-cit="Ekstr%C3%B6mU%2C%20AbrahamsonM%2C%20SvegerT%2C%20LombardiP%2C%20Nilsson-EhleP%20%281995%29%20An%20efficient%20screening%20procedure%20detecting%20six%20novel%20mutations%20in%20the%20LDL%20receptor%20gene%20in%20Swedish%20children%20with%20hypercholesterolemia.%20Hum%20Genet%2096%3A%20147%E2%80%93150." data-title="An%20efficient%20screening%20procedure%20detecting%20six%20novel%20mutations%20in%20the%20LDL%20receptor%20gene%20in%20Swedish%20children%20with%20hypercholesterolemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+efficient+screening+procedure+detecting+six+novel+mutations+in+the+LDL+receptor+gene+in+Swedish+children+with+hypercholesterolemia+Ekstr%C3%B6m+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0094697-Khoo1" id="pone.0094697-Khoo1" class="link-target"></a>Khoo KL, van Acker P, Defesche JC, Tan H, van de Kerkhof L, et al. (2000) Low-density lipoprotein receptor gene mutations in a Southeast Asian population with familial hypercholesterolemia. Clin Genet 58: 98–105. <ul class="reflinks"><li><a href="#" data-author="Khoo" data-cit="KhooKL%2C%20van%20AckerP%2C%20DefescheJC%2C%20TanH%2C%20van%20de%20KerkhofL%2C%20et%20al.%20%282000%29%20Low-density%20lipoprotein%20receptor%20gene%20mutations%20in%20a%20Southeast%20Asian%20population%20with%20familial%20hypercholesterolemia.%20Clin%20Genet%2058%3A%2098%E2%80%93105." data-title="Low-density%20lipoprotein%20receptor%20gene%20mutations%20in%20a%20Southeast%20Asian%20population%20with%20familial%20hypercholesterolemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Low-density+lipoprotein+receptor+gene+mutations+in+a+Southeast+Asian+population+with+familial+hypercholesterolemia+Khoo+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0094697-Amsellem1" id="pone.0094697-Amsellem1" class="link-target"></a>Amsellem S, Briffaut D, Carrié A, Rabès JP, Girardet JP, et al. (2002) Intronic mutations outside of Alu-repeat-rich domains of the LDL receptor gene are a cause of familial hypercholesterolemia. Hum Genet 111: 501–510. <ul class="reflinks"><li><a href="#" data-author="Amsellem" data-cit="AmsellemS%2C%20BriffautD%2C%20Carri%C3%A9A%2C%20Rab%C3%A8sJP%2C%20GirardetJP%2C%20et%20al.%20%282002%29%20Intronic%20mutations%20outside%20of%20Alu-repeat-rich%20domains%20of%20the%20LDL%20receptor%20gene%20are%20a%20cause%20of%20familial%20hypercholesterolemia.%20Hum%20Genet%20111%3A%20501%E2%80%93510." data-title="Intronic%20mutations%20outside%20of%20Alu-repeat-rich%20domains%20of%20the%20LDL%20receptor%20gene%20are%20a%20cause%20of%20familial%20hypercholesterolemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Intronic+mutations+outside+of+Alu-repeat-rich+domains+of+the+LDL+receptor+gene+are+a+cause+of+familial+hypercholesterolemia+Amsellem+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0094697-Chang1" id="pone.0094697-Chang1" class="link-target"></a>Chang JH, Pan JP, Tai DY, Huang AC, Li PH, et al. (2003) Identification and characterization of LDL receptor gene mutations in hyperlipidemic Chinese. J Lipid Res 44: 1850–1858. <ul class="reflinks"><li><a href="#" data-author="Chang" data-cit="ChangJH%2C%20PanJP%2C%20TaiDY%2C%20HuangAC%2C%20LiPH%2C%20et%20al.%20%282003%29%20Identification%20and%20characterization%20of%20LDL%20receptor%20gene%20mutations%20in%20hyperlipidemic%20Chinese.%20J%20Lipid%20Res%2044%3A%201850%E2%80%931858." data-title="Identification%20and%20characterization%20of%20LDL%20receptor%20gene%20mutations%20in%20hyperlipidemic%20Chinese" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+characterization+of+LDL+receptor+gene+mutations+in+hyperlipidemic+Chinese+Chang+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0094697-Ekstrom1" id="pone.0094697-Ekstrom1" class="link-target"></a>Ekstrom U, Abrahamson M, Wallmark A, Florén CH, Nilsson-Ehle P (1998) Mutations in the low-density lipoprotein receptor gene in Swedish familial hypercholesterolaemia patients: clinical expression and treatment response. Eur J Clin Invest 28: 740–747. <ul class="reflinks"><li><a href="#" data-author="Ekstrom" data-cit="EkstromU%2C%20AbrahamsonM%2C%20WallmarkA%2C%20Flor%C3%A9nCH%2C%20Nilsson-EhleP%20%281998%29%20Mutations%20in%20the%20low-density%20lipoprotein%20receptor%20gene%20in%20Swedish%20familial%20hypercholesterolaemia%20patients%3A%20clinical%20expression%20and%20treatment%20response.%20Eur%20J%20Clin%20Invest%2028%3A%20740%E2%80%93747." data-title="Mutations%20in%20the%20low-density%20lipoprotein%20receptor%20gene%20in%20Swedish%20familial%20hypercholesterolaemia%20patients%3A%20clinical%20expression%20and%20treatment%20response" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mutations+in+the+low-density+lipoprotein+receptor+gene+in+Swedish+familial+hypercholesterolaemia+patients%3A+clinical+expression+and+treatment+response+Ekstrom+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0094697-Jeon1" id="pone.0094697-Jeon1" class="link-target"></a>Jeon H, Meng W, Takagi J, Eck MJ, Springer TA, et al. (2001) Implications for familial hypercholesterolemia from the structure of the LDL receptor YWTD-EGF domain pair. Nat Struct Biol 8: 499–504. <ul class="reflinks"><li><a href="#" data-author="Jeon" data-cit="JeonH%2C%20MengW%2C%20TakagiJ%2C%20EckMJ%2C%20SpringerTA%2C%20et%20al.%20%282001%29%20Implications%20for%20familial%20hypercholesterolemia%20from%20the%20structure%20of%20the%20LDL%20receptor%20YWTD-EGF%20domain%20pair.%20Nat%20Struct%20Biol%208%3A%20499%E2%80%93504." data-title="Implications%20for%20familial%20hypercholesterolemia%20from%20the%20structure%20of%20the%20LDL%20receptor%20YWTD-EGF%20domain%20pair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Implications+for+familial+hypercholesterolemia+from+the+structure+of+the+LDL+receptor+YWTD-EGF+domain+pair+Jeon+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0094697-Wu2" id="pone.0094697-Wu2" class="link-target"></a>Wu WF, Wang QH, Zhang T, Mi SH, Liu Y, et al. (2013) Gas chromatography analysis of serum cholesterol synthesis and absorption markers used to predict the efficacy of simvastatin in patients with coronary heart disease. Clin Biochem 46: 993–998. <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuWF%2C%20WangQH%2C%20ZhangT%2C%20MiSH%2C%20LiuY%2C%20et%20al.%20%282013%29%20Gas%20chromatography%20analysis%20of%20serum%20cholesterol%20synthesis%20and%20absorption%20markers%20used%20to%20predict%20the%20efficacy%20of%20simvastatin%20in%20patients%20with%20coronary%20heart%20disease.%20Clin%20Biochem%2046%3A%20993%E2%80%93998." data-title="Gas%20chromatography%20analysis%20of%20serum%20cholesterol%20synthesis%20and%20absorption%20markers%20used%20to%20predict%20the%20efficacy%20of%20simvastatin%20in%20patients%20with%20coronary%20heart%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Gas+chromatography+analysis+of+serum+cholesterol+synthesis+and+absorption+markers+used+to+predict+the+efficacy+of+simvastatin+in+patients+with+coronary+heart+disease+Wu+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0094697-Catapano1" id="pone.0094697-Catapano1" class="link-target"></a>Catapano AL, Davidson MH, Ballantyne CM, Brady WE, Gazzara RA, et al. (2006) Lipid-altering efficacy of the ezetimibe/simvastatin single tablet versus rosuvastatin in hypercholesterolemic patients. Curr Med Res Opin 22: 2041–2053. <ul class="reflinks"><li><a href="#" data-author="Catapano" data-cit="CatapanoAL%2C%20DavidsonMH%2C%20BallantyneCM%2C%20BradyWE%2C%20GazzaraRA%2C%20et%20al.%20%282006%29%20Lipid-altering%20efficacy%20of%20the%20ezetimibe%2Fsimvastatin%20single%20tablet%20versus%20rosuvastatin%20in%20hypercholesterolemic%20patients.%20Curr%20Med%20Res%20Opin%2022%3A%202041%E2%80%932053." data-title="Lipid-altering%20efficacy%20of%20the%20ezetimibe%2Fsimvastatin%20single%20tablet%20versus%20rosuvastatin%20in%20hypercholesterolemic%20patients" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Lipid-altering+efficacy+of+the+ezetimibe%2Fsimvastatin+single+tablet+versus+rosuvastatin+in+hypercholesterolemic+patients+Catapano+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0094697-Ballantyne1" id="pone.0094697-Ballantyne1" class="link-target"></a>Ballantyne CM, Abate N, Yuan Z, King TR, Palmisano J (2005) Dose-comparison study of the combination of ezetimibe and simvastatin (Vytorin) versus atorvastatin in patients with hypercholesterolemia: the Vytorin Versus Atorvastatin (VYVA) study. Am Heart J 149: 464–73. <ul class="reflinks"><li><a href="#" data-author="Ballantyne" data-cit="BallantyneCM%2C%20AbateN%2C%20YuanZ%2C%20KingTR%2C%20PalmisanoJ%20%282005%29%20Dose-comparison%20study%20of%20the%20combination%20of%20ezetimibe%20and%20simvastatin%20%28Vytorin%29%20versus%20atorvastatin%20in%20patients%20with%20hypercholesterolemia%3A%20the%20Vytorin%20Versus%20Atorvastatin%20%28VYVA%29%20study.%20Am%20Heart%20J%20149%3A%20464%E2%80%9373." data-title="Dose-comparison%20study%20of%20the%20combination%20of%20ezetimibe%20and%20simvastatin%20%28Vytorin%29%20versus%20atorvastatin%20in%20patients%20with%20hypercholesterolemia%3A%20the%20Vytorin%20Versus%20Atorvastatin%20%28VYVA%29%20study" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dose-comparison+study+of+the+combination+of+ezetimibe+and+simvastatin+%28Vytorin%29+versus+atorvastatin+in+patients+with+hypercholesterolemia%3A+the+Vytorin+Versus+Atorvastatin+%28VYVA%29+study+Ballantyne+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

