



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>The relationship between the stem cells and the bone turnover in uremic bone disease due to chronic renal failure (CRF) is not described. The aim of this study was to investigate the effect of bone turnover status on stem cell properties. To search for the presence of such link and shed some light on stem-cell relevant mechanisms of bone turnover, we carried out a study with mesenchymal stem cells. Tissue biopsies were taken from the abdominal subcutaneous adipose tissue of a CRF patient with secondary hyperparathyroidism with the high turnover bone disease. This patient underwent parathyroidectomy operation (PTX) and another sample was taken from this patient after PTX. A CRF patient with adynamic bone disease with low turnover and a healthy control were also included. Mesenchymal stem cells isolated from the subjects were analyzed using proteomic and molecular approaches. Except ALP activity, the bone turnover status did not affect common stem cell properties. However, detailed proteome analysis revealed the presence of regulated protein spots. A total of 32 protein spots were identified following 2D gel electrophoresis and MALDI-TOF/TOF analyzes. The identified proteins were classified into seven distinct groups and their potential relationship to bone turnover were discussed. Distinct protein expression patterns emerged in relation to the bone turnover status indicate a possible link between the stem cells and bone turnover in uremic bone disease due to CRF.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Kasap M, Yeğenağa I, Akpinar G, Tuncay M, Aksoy A, Karaoz E (2015) Comparative Proteome Analysis of hAT-MSCs Isolated from Chronic Renal Failure Patients with Differences in Their Bone Turnover Status. PLoS ONE 10(11):
           e0142934.
        
        https://doi.org/10.1371/journal.pone.0142934</p><p><strong>Editor: </strong>Gianpaolo Papaccio, Second University of Naples, ITALY</p><p><strong>Received: </strong>September 2, 2015; <strong>Accepted: </strong>October 28, 2015; <strong>Published: </strong> November 17, 2015</p><p><strong>Copyright: </strong> © 2015 Kasap et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited</p><p><strong>Data Availability: </strong>All relevant data are within the paper and its Supporting Information files.</p><p><strong>Funding: </strong>The author(s) received specific funding for this work from Kocaeli University BAP.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Uremic bone disease (UBD) is a destructive complication of CRF. UBD has recently been redefined as the bone and mineral disease in CRF [<a href="#pone.0142934.ref001" class="ref-tip">1</a>]. Severity of UBD affects the rate of morbidity, mortality and quality of life in CRF patients. UBD appears mostly in complicated clinical presentations. While some patients suffer from secondary hyperthyroidism (SHPT) (<span class="underline">H</span>igh <span class="underline">T</span>urnover <span class="underline">B</span>one <span class="underline">D</span>isease, HTBD) and associated bone abnormalities, others might have adynamic bone disease (ADBD) (<span class="underline">L</span>ow <span class="underline">T</span>urnover <span class="underline">B</span>one <span class="underline">D</span>isease-LTBD) and osteoporosis [<a href="#pone.0142934.ref001" class="ref-tip">1</a>–<a href="#pone.0142934.ref003" class="ref-tip">3</a>]. Furthermore, extra osseous calcification occurs often in CRF patients with LTBD [<a href="#pone.0142934.ref004" class="ref-tip">4</a>–<a href="#pone.0142934.ref006" class="ref-tip">6</a>]. Some etiologic factors affecting the type of bone disorder development in CRF were reported recently, but there are some contradictions in this field [<a href="#pone.0142934.ref002" class="ref-tip">2</a>]. There's still a need for understanding the molecular basis that will help understanding of the etiology of the type of bone disorders in CRF.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Stem cells are undifferentiated cells that can divide, differentiate and self-renew to produce new stem cells in multicellular organisms [<a href="#pone.0142934.ref007" class="ref-tip">7</a>]. They can be used in biomedical research, drug discovery, and toxicity testing, as a model in understanding diseases and most importantly for therapeutic purposes in regenerative medicine [<a href="#pone.0142934.ref008" class="ref-tip">8</a>, <a href="#pone.0142934.ref009" class="ref-tip">9</a>]. It was demonstrated that systemic application of adipose-tissue-derived stem cells induced expression of the proteins to stimulate osteoblastic and osteoclastic functions in C57BL/6J mice and treated osteoporosis [<a href="#pone.0142934.ref010" class="ref-tip">10</a>]. Adipose tissue derived stem cells (hAT-MSCs) is one of the most practical postnatal stem cells to use in research because of the ease to obtain without any ethical concern. In addition to these aspects, hAT- MSCs may be preferred in bone disease research due to several other reasons; 1) In hypercalcemic status, adipose tissue may get calcified [<a href="#pone.0142934.ref011" class="ref-tip">11</a>–<a href="#pone.0142934.ref014" class="ref-tip">14</a>], 2) During the follow up of CRF patients, soft tissue calcifications occur [<a href="#pone.0142934.ref004" class="ref-tip">4</a>], 3) Adipose tissue is an easily accessible tissue and may be used to study molecular mechanisms of bone diseases [<a href="#pone.0142934.ref011" class="ref-tip">11</a>].</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>The hypothesis of this study was based on the possibility that adipose tissue calcification in CRF patients might originate from differentiation of the stem cells present in the adipose tissue. If the stem cells isolated from adipose tissue of different degrees of parathormon function were studied by proteomic analysis; then some important clues about the molecular mechanisms behind the soft tissue calcification may be obtained. Therefore, we performed a study in which hAT-MSCs isolated from two patients with different types of bone status, namely HTBD with CRF and LTBD with CRF were compared both at the cellular and molecular levels. Furthermore, the patient with HTBD underwent PTX surgery, which allowed us to make further comparisons of hAT-MSCs isolated before and after operation. hAT-MSCs from a healthy control were also included for comparisons.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>The method of choice in this study was 2DE-based proteomics, because investigation of changes in global proteome level may provide significant information about changes at the protein level, post translational modifications (PTMs) and molecular pathways [<a href="#pone.0142934.ref015" class="ref-tip">15</a>]. Comparison of the protein expression profiles of stem cells isolated from hAT revealed the presence of a conserved protein pattern that included 70 spots, of which 32 were reliably identified by MALDI-TOF/TOF and shown to be regulated. In addition, the cells were compared on the basis of cellular morphology, expression of common stem cell markers and osteoblastic differentiation potentials as well as their ALP and telomerase activities. This is the first study presenting a list of proteins that may have importance in understanding of the differences between HTBD and LTBD in CRF.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Material and Methods"></a><h2>Material and Methods</h2><a id="article1.body1.sec2.p1" name="article1.body1.sec2.p1" class="link-target"></a><p>This study was approved by Ethical Committee of Kocaeli University. Written informed consent from the donor or the next of kin was obtained to use these samples in research.</p>

<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Isolation and culture of hAT-MSCs"></a>
<h3>Isolation and culture of hAT-MSCs</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>Tissue biopsies were taken from the abdominal subcutaneous adipose tissue of the following individuals: (1) a CRF patient with HTBD. This patient underwent PTX and one week later another sample was taken from this patient after PTX (2) a CRF patient with LTBD (3) a healthy control (<a href="#pone-0142934-t001">Table 1</a>).</p>
<a class="link-target" id="pone-0142934-t001" name="pone-0142934-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.t001" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Clinical and main laboratory findings of the patients.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.t001">
              https://doi.org/10.1371/journal.pone.0142934.t001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>Tissue samples were washed several times with Hanks’ balanced salt solution (HBSS) with 5% antibiocantimycotic solution without calcium and magnesium to remove blood (LifeTech, USA). A single cell suspension of adipose tissue cells was obtained by using enzymatic digestion and mechanical means. Hat-MSCs were then isolated as described by Ertas, G. et al. [<a href="#pone.0142934.ref016" class="ref-tip">16</a>]. After cells reached 80–90% confluency, they were treated with 0.025% trypsin-EDTA for 3 min. The released cells were collected, centrifuged, and replaced at a rate of 1:3 and 1:4 for subculture.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Characterization of MSCs by Flow Cytometry"></a>
<h3>Characterization of MSCs by Flow Cytometry</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>A FACS Calibur (BD Biosciences, San Diego, USA) flow cytometry instrument was used in characterization experiments. The forward and side scatter profiles gated out debris and dead cells were analyzed using Cell Quest software (BD Biosciences). Antibodies (Becton Dickinson) against CD3, CD4, CD5, CD7, CD8, CD10, CD11b, CD13, CD14, CD15, CD17, CD19, CD29, CD33, CD34, CD44, CD45, CD71, CD73, CD90, CD106, CD123, CD133, CD138, CD146, CD166, HLA-ABC, HLA-G and HLA-DR were used for immunophenotyping of MSCs [<a href="#pone.0142934.ref017" class="ref-tip">17</a>].</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="In vitro adipogenic differentiation"></a>
<h3>In vitro adipogenic differentiation</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Cells from passage three (0,35x10<sup>5</sup> cells/cm<sup>2</sup>) were seeded onto coated type I collagen coverslips in 6-well plates for adipogenic differentiation. The adipogenic medium, LG-DMEM, contained 10% FBS, 0.5 mM isobutyl-methylxanthine (IBMX-Sigma-Aldrich), 10<sup>−6</sup> M dexamethasone (Sigma-Aldrich, Fluka Chemie AG, Buchs, Switzerland), 10μg/ml insulin (Invitrogen/GIBCO), 200M indomethacin (Sigma-Aldrich), and 1% penicilin/streptomycin (Invitrogen/GIBCO). The differentiation lasted for four weeks and the medium was replaced twice a week. Intracellular lipid droplets, an indicator of the adipogenic differentiation were confirmed by Oil Red O staining.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="In vitro osteoblastic differentiation"></a>
<h3>In vitro osteoblastic differentiation</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>For osteoblastic differentiation, cells from passage three (0,3x10<sup>5</sup> cells/cm<sup>2</sup>) was grown in LG-DMEM supplemented with 100 nM dexamethasone (Sigma-Aldrich), 0.05M ascorbate-2-phosphate (Wako Chemicals), 10 mM β-glycerophosphate (Sigma-Aldrich), 1% penicilin/streptomycin and 10% FBS. The differentiation was accomplished within four weeks and the medium was replaced twice a week during the incubation. Alizarin Red staining was performed to demonstrate that osteoblastic differentiation was achieved.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec007" name="sec007" class="link-target" title="Alizarin red S staining"></a>
<h3>Alizarin red S staining</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>For staining, the cells were fixed for 5 min in ice-cold 70% ethanol and then allowed to dry. Cells were stained with 2% alizarin red S (pH 4.1–4.3) solution for 30 s to 1 min, then washed with distilled water (20 dips). The stained cells were dehydrated in acetone (20 dips), fixed in acetone-xylene (1:1) solution (20 dips), cleared with xylene (20 dips), dried and mounted in mounting medium.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="ALP activity"></a>
<h3>ALP activity</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>ALP activity assay was performed at days 1, 4, 7, 14, and 21. Cells were washed with PBS for three times and protein extraction was performed using Mammalian Protein Extract Reagent (M-PER) (Thermo Scientific). p-Nitrophenyl Phosphate (Sigma Aldrich, N7653) was used as the liquid substrate to detect ALP activity and the incubation of the cells with the liquid substrate lasted for 30 min. A VersaMax Microplate Reader (Molecular Devices) was used to measure the optical density at 405 nm. Bicinchoninic Acid Protein Determination method (SigmaAldrich) was used to measure total protein concentration of the cell lysate. ALP activity was expressed as OD 405 nm/mg of protein. Three technical replicates of each sample were performed.</p>
</div>

<div id="section7" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Telomerase activity"></a>
<h3>Telomerase activity</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>Telomerase activity was determined by the conventional telomeric repeat amplification protocol (TRAP). TRAP TeloTAGGG PCR enzyme-linked immunosorbent assay (ELISA) kit (Roche, Mannheim, Germany) was used and the relative telomerase activity was calculated as the ratio of the absorbance value of the sample to that of the control.</p>
</div>

<div id="section8" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Statistical analysis"></a>
<h3>Statistical analysis</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>Flow cytometry, stem cell differentiation and telomerase activity measurements were performed on passage three cells. Three experimental replications, both technical and biological were performed. Statistical analysis of the data was carried out using one-way analysis of variance (ANOVA), followed by Tukey's test for multiple comparisons to determine the values that were significantly different (SPSS Version 10, IL, Chicago). Differences were considered statistically significant at p &lt; 0.05.</p>
</div>

<div id="section9" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="Protein extraction and 2D-gel electrophoresis"></a>
<h3>Protein extraction and 2D-gel electrophoresis</h3>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1" class="link-target"></a><p>For analysis of proteome profiles, the cells were grown in defined media (Stem Cell Technologies, USA). Equal number of cells (3x10<sup>5</sup>) was seeded into T-75 flasks in triplicates. When 70% confluency was reached, the cells were washed with ample amount of ice-cold PBS for three times and removed from the plates by scraping. After 10 min centrifugation at 4°C at 2000xg, excess PBS was decanted and 250 μl of cell lysis buffer (2D-rehydration buffer: 8M urea, 2M thiourea, 4% CHAPS, 30 mM Tris pH 8.5, 1x protease inhibitor cocktail) was added onto each cell pellet. To achieve complete lysis, the cells were vigorously vortexed for 1 min in lysis buffer and the supernatant containing the soluble protein fraction was obtained by centrifugation at 20000xg for 30 min at 4°C and stored in Lo-bind tubes (Eppendorf, USA) at -80°C after snap-frozen in liquid nitrogen. Protein concentration was determined by using modified Lowry assay with the BSA standard (BioRad, USA). 80μg of protein was loaded onto immobilized pH gradient strips (IPG) (11cm, pH 5–8) (BioRad, USA) via passive rehydration. Separation based on isoelectric points was performed by using Protean isoelectric focusing cell (BioRad, USA). The strips were run through a stepwise incremental voltage program (250V for 20 min (linear), 4000 V for 2hr (linear) and 10000 V/hr (rapid)). The strips were then subjected to a two-step equilibration in equilibration buffers containing 6M urea, 2% SDS, 0.375M Tris-HCl pH 8.8, 20% glycerol and 2% DTT for the first step and the same buffer without DTT but with iodoacetamide (2.5%) for the second step. Following isoelectric focusing, the strips were subjected to SDS-PAGE using 12% in-house made gels. The gels were stained with Colloidal Coomassie stain (KeraFast, USA) and visualized with VersaDoc4000 MP (BioRad, USA). PDQuest Advance (BioRad, USA) 2DE-analysis software was used for comparison of protein spot profiles. For automated spot detection, parameters used were sensitivity (13.8), spot size scale (3) and minimum peak intensity (258).</p>
</div>

<div id="section10" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="Image analysis and spot cutting"></a>
<h3>Image analysis and spot cutting</h3>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1" class="link-target"></a><p>An automated crop tool was used to prepare the images for analysis using PDQuest Advance (BioRad, USA). Automated analyses were performed to detect total spot numbers and volumes within the normalized area. A manual editing tool was used to inspect the determined protein spots detected by the software. Spots that were prone to variation were excluded if they were hard to identify by visual inspection. Spots were cut by using automated spot cutting tool, ExQuest spot cutter (BioRad,USA), and disposed into 96 well plates for protein identification.</p>
</div>

<div id="section11" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="Identification of proteins"></a>
<h3>Identification of proteins</h3>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1" class="link-target"></a><p>Protein identification experiments were performed at Kocaeli University DEKART proteomics laboratory (<a href="http://kabiproteomics.kocaeli.edu.tr/">http://kabiproteomics.kocaeli.edu.tr/</a>) by using ABSCIEX MALDI-TOF/TOF 5800 system. In-gel tryptic digestion of the proteins was performed by using an in-gel digestion kit following the recommended protocol of the manufacturer (Pierce, USA). Zip-Tip cleaning was performed for each digested sample following the recommended protocol (Millipore, USA) before deposition onto a MALDI plate. Peak data were analyzed with MASCOT by using a streamline software, Protein Pilot (ABSCIEX,USA). The parameters for searching were; enzyme of trypsin, 1 missed cleavage, fixed modifications of carbamidomethyl (C), variable modifications of oxidation (M), peptide mass tolerance: 50ppm, fragment mass tolerance: ±0.4 Da, peptide charge of 1+ and monoisotopic. Only significant hits, as defined by the MASCOT probability analysis (<em>p</em> &lt; 0.05) were accepted. Classification of the proteins was performed by using a freely available classification system, PANTHER (<a href="http://www.pantherdb.org/">http://www.pantherdb.org/</a>) [<a href="#pone.0142934.ref018" class="ref-tip">18</a>].</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec014" name="sec014" data-toc="sec014" class="link-target" title="Results"></a><h2>Results</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>Adipose tissues were obtained from the abdominal subcutaneous layer of CRF patients with HTBD before and a week after PTX (50 year-old male) and also from a LTBD) patient (70 year-old male) and from a healthy control (40 year-old male). MSCs were isolated from these adipose tissues and characterized to some extent. The stem cells displayed similar morphological properties, which were large, flattened and had fibroblast like shape. No significant morphological changes were detected throughout the study and they displayed similar growth characteristics. Some MSC specific cell surface markers were used to define the cultured cells. The data indicated that the cells expressed some commonly accepted MSCs markers (CD13, CD44, CD90, CD166, CD73, HLA ABC ve CD29) (<a href="#pone-0142934-t002">Table 2</a> and <a href="#pone.0142934.s001">S1 Fig</a>). These findings were similar to the immunophenotypic MSC characteristics of undifferentiated states of hAT-MSCs reported earlier [<a href="#pone.0142934.ref019" class="ref-tip">19</a>].</p>
<a class="link-target" id="pone-0142934-t002" name="pone-0142934-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.t002" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Percentages determined by flow cytometry analysis for markers used to identify hAT-MSCs.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.t002">
              https://doi.org/10.1371/journal.pone.0142934.t002</a></p></div><a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>To assess multipotential characteristics of the stem cells, measurement of differentiation potential is required. Each cell line was subjected to adipogenic and osteoblastic differentiation and differentiations were characterized by using immunohistochemical and fluorescence techniques. At the beginning of adipogenic differentiation, the presence of lipid droplets was evident and towards the end of differentiation, the lipid droplets enlarged and invaded the entire cytoplasm (<a href="#pone-0142934-g001">Fig 1A</a>). During osteoblastic differentiation, cellular aggregates were observed in the culture plates and were gradually increased until the end of the experiment. These aggregates were characterized by the presence of amorphous material deposits (<a href="#pone-0142934-g001">Fig 1B</a>). Alizarin red S stained these nodular aggregates in osteoblastic cultures, demonstrating that the amorphous deposits observed under the microscope were in fact calcium deposits (<a href="#pone-0142934-g001">Fig 1C</a>). At day 30, positive alizarin red aggregates were larger and stained intensively indicating the presence of a more extensive calcium deposition. Control cultures showed only minimal background staining.</p>
<a class="link-target" id="pone-0142934-g001" name="pone-0142934-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.g001" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Differentiation of hAT-MSCs from PTX-hAT-MSCs, HTBD-hAT-MSCs and LTBD-hAT-MSCs.</div><p class="caption_target"><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1" class="link-target"></a><p>The cells were subjected to (A and B) adipogenic and (C) osteogenic differentiations as described in material and methods section. (A) Inverted microscope image of the cells subjected to adipogenic differentiation. (B) Oil Red-O staining of adipogenic cells differentiated from hAT-MSCs. (C) Alizarin Red staining of osteogenic cells differentiated from hAT-MSCs.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.g001">
              https://doi.org/10.1371/journal.pone.0142934.g001</a></p></div><a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>ALP activity, another osteoblastic differentiation marker, gradually decreased in all cell lines except HTBD-hAT-MSCs, which reached to its peak activity at day 4 (<a href="#pone-0142934-g002">Fig 2</a>). In overall, the ALP activity was higher in cell cultures generated from the stem cells isolated from the patient tissue with HTBD than from the stem cells of the same patient after PTX operation (PTX-hAT-MSCs). ALP activity in the cell cultures generated from the stem cells isolated from the patient with LTBD was almost undetectable.</p>
<a class="link-target" id="pone-0142934-g002" name="pone-0142934-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.g002" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Alkaline phosphatase activity of hAT-MSCs.</div><p class="caption_target"><a id="article1.body1.sec3.fig2.caption1.p1" name="article1.body1.sec3.fig2.caption1.p1" class="link-target"></a><p>PTX: Stem cells isolated from human adipose tissue after parathyroidectomy operation. HTBD: Stem cells isolated from human adipose tissue obtained from a patient with secondary hyperparathyroidism (High turnover bone disease). LTBD: Stem cells isolated from human adipose tissue obtained from a patient with adynamic bone disease (Low turnover bone disease).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.g002">
              https://doi.org/10.1371/journal.pone.0142934.g002</a></p></div><a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>Telomere shortening is an indicator of stem cell aging and can be monitored by measuring the telomerase activity. Telomerase activities of PTX-hAT-MSCs, HTBD-hAT-MSCs and LTBD-hAT-MSCs were measured at P3. The telomerase activity of HTBD-hAT-MSCs was 6.47±2.69 amol/μg total protein, PTX-hAT-MSCs was 4.09±1.18 amol/μg total protein and LTBD-hAT-MSCs was 0.03±0.02 amol/μg total protein (<a href="#pone-0142934-g003">Fig 3</a>).</p>
<a class="link-target" id="pone-0142934-g003" name="pone-0142934-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.g003" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Telomerase activities of hAT-MSCs obtained from HTBD-hAT-MSCs, PTX-hAT-MSCs and LTBD-hAT-MSCs.</div><p class="caption_target"><a id="article1.body1.sec3.fig3.caption1.p1" name="article1.body1.sec3.fig3.caption1.p1" class="link-target"></a><p>HTBD: Stem cells isolated from human adipose tissue obtained from a patient with secondary hyperparathyroidism (High turnover bone disease). LTBD: Stem cells isolated from human adipose tissue obtained from a patient with adynamic bone disease (Low turnover bone disease). The activities were measured by conventional telomeric repeat amplification protocol (TRAP) and relative telomerase activities (RTA) were calculated with respect to the control template equivalent to 0.001 mol/<em>μ</em>L DNA.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.g003">
              https://doi.org/10.1371/journal.pone.0142934.g003</a></p></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>To understand the complexity observed in these cells at the molecular level, a comparative 2DE-based proteomic study was performed. Well-resolved and reproducible 2DE gel maps were produced as shown in <a href="#pone-0142934-g004">Fig 4</a>. An average of 410±30 spots per analytical gel was detected when the gels were subjected to automated spot detection and analysis. However, the number of spots that reliably matched to every member was 71 with an overall mean coefficient of 89.55 percent. The rest of the spots were either not matched or were not expressed by all three stem cell lines. By using PDQuest advance gel analysis software, changes in spot intensities among these 71 matching spots were compared. Spots that were up or down regulated more than 2-fold was considered to be subject to regulation. Ratios for up and down regulated protein spots were given in <a href="#pone-0142934-t003">Table 3</a>. To identify the conserved protein spots, the spots were cut from a preparative gel with an automated spot cutting instrument and subjected to in-gel tryptic digestion followed by MALDI-TOF/TOF analysis. A total of 32 protein spots were identified. The identified proteins were subjected analysis based on their molecular function and their involvement in biological processes (<a href="#pone-0142934-g005">Fig 5</a>). Proteins were classified into seven distinct groups, namely (1) structural proteins, (2) proteins involved in protein folding and stress, (3) redox metabolism, (4) protein biosynthesis and degradation, (5) proteins involved in transcription, (6) energy and (7) amino acid metabolisms. There were five proteins which could not be classified with the others and were evaluated separately.</p>
<a class="link-target" id="pone-0142934-g004" name="pone-0142934-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.g004" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Comparative proteome analysis of hAT-MSCs obtained from HTBD-hAT-MSCs, PTX-hAT-MSCs and LTBD-hAT-MSCs.</div><p class="caption_target"><a id="article1.body1.sec3.fig4.caption1.p1" name="article1.body1.sec3.fig4.caption1.p1" class="link-target"></a><p>HTBD: Stem cells isolated from human adipose tissue obtained from a patient with secondary hyperparathyroidism (High turnover bone disease). LTBD: Stem cells isolated from human adipose tissue obtained from a patient with adynamic bone disease (Low turnover bone disease). (A) MSCs were subjected to protein isolation and proteins were loaded onto pH 5 to pH 8 IPG strips for the first dimension and precast SDS-PAGE gels for the second dimension separation and stained with Colloidal Coomassie Blue for 24 hr after 24 hours of fixation. The gels are representative of three gels from each MSC type. (B) The 2DE gel showing the protein spots that were cut from the gels and were subjected to MALDI-TOF/TOF analysis for identification.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.g004">
              https://doi.org/10.1371/journal.pone.0142934.g004</a></p></div><a class="link-target" id="pone-0142934-g005" name="pone-0142934-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.g005" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Classification of identified proteins based on their molecular function and their involvement in biological processes.</div><p class="caption_target"><a id="article1.body1.sec3.fig5.caption1.p1" name="article1.body1.sec3.fig5.caption1.p1" class="link-target"></a><p>Pie chart representing the distribution of the identified proteins based on their molecular function and biological processes. Assignments were made on the basis of information from PANTHER analysis (<a href="http://www.pantherdb.org/">http://www.pantherdb.org/</a>) as well as NCBI (<a href="http://www.ncbi.nlm.nih.gov/pubmed">http://www.ncbi.nlm.nih.gov/pubmed</a>) and Swiss-Prot/TrEMBL annotations (<a href="http://www.expasy.org/">http://www.expasy.org/</a>).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.g005">
              https://doi.org/10.1371/journal.pone.0142934.g005</a></p></div><a class="link-target" id="pone-0142934-t003" name="pone-0142934-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0142934.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0142934.t003" data-doi="info:doi/10.1371/journal.pone.0142934" data-uri="info:doi/10.1371/journal.pone.0142934.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0142934.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0142934.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0142934.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0142934.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Proteins identified by MALDI-TOF/TOF analysis and classification according to their function.</div><p class="caption_target"><a id="article1.body1.sec3.table-wrap2.caption1.p1" name="article1.body1.sec3.table-wrap2.caption1.p1" class="link-target"></a><p>The last three columns represent the values for the regulation rations of the proteins in comparison to the control.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0142934.t003">
              https://doi.org/10.1371/journal.pone.0142934.t003</a></p></div></div>

<div id="section4" class="section toc-section"><a id="sec015" name="sec015" data-toc="sec015" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>There is a possibility that stem cells will be used to cure some of the incurable human diseases. However, for this to happen in the near future, novel stem cell therapies which rely upon an understanding of the biology of these interesting cells have to be developed. Stem cells exist in a specialized environment in the body known as “the niche”. The niche is known to control the behavior of the stem cells and ensures their survival. Most human stem cell niches are deep in the body and so difficult to study. Therefore, the question of how changes in stem cell niches of the host affect the stem cell properties is among the most challenging questions to be answered [<a href="#pone.0142934.ref020" class="ref-tip">20</a>, <a href="#pone.0142934.ref021" class="ref-tip">21</a>]. One approach to answer this type of question is perhaps to isolate stem cells from the same niche of the same individual under two different conditions for comparison such as before and after the treatment of a disease that may affect both the niche and the stem cell properties. In this study, the stem cells from an easily accessible stem cell niche, subcutaneous adipose tissue, were isolated from an individual with CRF with HTBD who underwent PTX operation were used. This allowed us to compare the same type of stem cells isolated from the same type of niche under two different host conditions to answer the question of how changes in the stem cell environment of the host affected stem cell properties. Furthermore, we studied stem cells from an aged-match healthy individual as the control and the stem cells from a patient with CRF accompanied by LTBD. Comparisons were made among the stem cells by measuring ALP activities, telomerase activities, adipogenic and osteoblastic differentiation potentials and changes at the proteome level.</p>

<div id="section1" class="section toc-section"><a id="sec016" name="sec016" class="link-target" title="Comparison of ALP activities revealed the presence of predefined molecular mechanisms in stem cells"></a>
<h3>Comparison of ALP activities revealed the presence of predefined molecular mechanisms in stem cells</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>ALP activity is known to reflect the rate of bone turnover in adult individuals [<a href="#pone.0142934.ref022" class="ref-tip">22</a>]. However, to the best of our knowledge, in most studies, although ALP activity during the osteoblastic differentiation was measured in MSCs, the results were not evaluated in terms of their relevance to the rate of bone turnover. In this study, hAT-MSCs of the patient with HTBD displayed significantly higher ALP activity during osteoblastic differentiation, then the hAT-MSCs of the same patient after PTX operation. In addition, hAT-MSCs of the CRF patient with LTBD had the lowest ALP activity (<a href="#pone-0142934-g002">Fig 2</a>). These observations implied that ALP activity of hAT-MSCs during osteoblastic differentiation may be correlated with the bone turnover status. The stem cells should be expected not to have predefined molecular mechanisms leading them to differentiation. However, our findings indicated and supported, that there are predefined molecular mechanisms to underline the features of the observed host diseases in stem cells.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec017" name="sec017" class="link-target" title="Telomerase activity in stem cells was not affected by bone turnover status of the host"></a>
<h3>Telomerase activity in stem cells was not affected by bone turnover status of the host</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>Age-related telomerase shortening due to loss of telomerase activity in AT-MSCs might be an indicator of host aging from whom the stem cells were obtained [<a href="#pone.0142934.ref023" class="ref-tip">23</a>]. We observed a donor-age associated decrease in telomerase activities. The least telomerase activity was observed in the elderly patient with LTBD (<a href="#pone-0142934-g003">Fig 3</a>). The patient who underwent PTX displayed low bone turnover in a week after the operation, although the stem cells isolated from this patient after PTX still had relatively high telomerase activity indicating that the turnover status of the bone might not really affect the level of telomerase activity.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec018" name="sec018" class="link-target" title="Changes in the host status cause changes in global protein expression profile in the hAT-MSCs"></a>
<h3>Changes in the host status cause changes in global protein expression profile in the hAT-MSCs</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Most of the identified proteins were classified into seven categories (<a href="#pone-0142934-t002">Table 2</a> and <a href="#pone-0142934-g005">Fig 5</a>). Some of these play critical roles in cellular architecture. Also, proteins that were part of the protein folding machinery were described. Some of those were inducible chaperons that were expressed under stress conditions. Transcription, protein biosynthesis and degradation related proteins that we detected indicated the presence of active cell growth. Redox metabolism-related scavenger proteins that were detected may be an indicator of active cell growth as well. The presence of apoptosis, transcription, protein biosynthesis/degradation and energy metabolism-related proteins indicated cellular self-renewal and proliferation. Some of the proteins identified in this study were similar to the proteins reported by the earlier studies [<a href="#pone.0142934.ref024" class="ref-tip">24</a>, <a href="#pone.0142934.ref025" class="ref-tip">25</a>].</p>
</div>

<div id="section4" class="section toc-section"><a id="sec019" name="sec019" class="link-target" title="The abundant proteins present in the high turnover state, but absent in the low turnover state require attention"></a>
<h3>The abundant proteins present in the high turnover state, but absent in the low turnover state require attention</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>There were two structural proteins that required our attention since their abundances were high before PTX in comparison to the control. These proteins were F-acting-capping protein and keratin, type 1cytosceleton 10. These two proteins were not regulated in AT-MSCs of the patient with LTBD in comparison to the control, although there was a high level of regulation in AT-MSCs of the patient with HTBD in comparison to the control. This finding indicated that these proteins may play pivotal roles in determining the status of the bone turnover. This interpretation might make sense since the actin cytoskeleton controls cell shape, mobility, division and intracellular transports. Although the functions of these two proteins in stem cells are not well established, their roles in cancer cells are known in detail [<a href="#pone.0142934.ref026" class="ref-tip">26</a>]. They are involved in cell growth, proliferation, stiffness, movement of invasiveness in cancer cells [<a href="#pone.0142934.ref027" class="ref-tip">27</a>]. The F-acting capping protein which is an actin regulator and constructed as a heterodimer consists of alpha and beta subunits and functions as a tumor suppressor protein. F-actin-capping protein subunit β plays a role for actin binding and regulates cell morphology and cytoskeleton organization under normal circumstances [<a href="#pone.0142934.ref026" class="ref-tip">26</a>–<a href="#pone.0142934.ref028" class="ref-tip">28</a>].</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2" class="link-target"></a><p>There is no study establishing a relationship between Keratin type 1 cytoskeleton 10 protein and bone turnover. However, the protein content of blood in 25 patients with bone fracture was reported [<a href="#pone.0142934.ref029" class="ref-tip">29</a>]. Twelve of the 213 identified proteins were found to be related to the bone and cartilage metabolism. Keratin, type 1cytoskeloton 10 were present among them, and it was claimed that this protein should have a connection with the bone formation [<a href="#pone.0142934.ref029" class="ref-tip">29</a>]. Our study is the first study showing a change in expression of keratin type 1 cytoskeleton 10 protein in response to changes in bone turnover. However,a study reported a correlation between organization of MSCs cytoskeleton elements and, changes in cell morphology and expression of some markers [<a href="#pone.0142934.ref030" class="ref-tip">30</a>]. In that study, the researchers found out that cell shape of MSCs and their cytoskeleton organization have changed during osteoblastic differentiation [<a href="#pone.0142934.ref030" class="ref-tip">30</a>]. This reorganization of the cytoskeleton may occur as an early response to changes in some hormones such as dexamethosone, growth hormone, parathyroid hormone. This early report supported our findings about changing the expression of some structural proteins in response to bone turnover, which is affected by the PTH level [<a href="#pone.0142934.ref031" class="ref-tip">31</a>].</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3" class="link-target"></a><p>Redox regulation is an essential physiological process in the survival of all cell types [<a href="#pone.0142934.ref032" class="ref-tip">32</a>]. Imbalance in a redox system creates oxidative stress in the cells, resulting in an impairment of cellular function, lipid peroxidation, protein degradation and nucleic acid breakage. There was a high level of peroxiredoxin expression in AT-MSCs of the patient with HTBD but not after the PTX operation and not with the patients with LTBD.</p>
<a id="article1.body1.sec4.sec4.p4" name="article1.body1.sec4.sec4.p4" class="link-target"></a><p>Peroxiredoxin is proposed to play a role in cell signaling by regulating radical oxygen species [<a href="#pone.0142934.ref033" class="ref-tip">33</a>]. In general, the increase in peroxiredoxin levels can be interpreted as an indicator of active cell growth, which produces radical oxygen species (ROS). However, there is a report about the relationship between osteoclastogenesis and peroxiredoxin-2 through the pathway induced by RANKL (receptor activator of NF-κB ligand), which is a well known osteoclastic biomarker. RANKL induces multiple signaling pathways causing long lasting ROS increase and Ca<sup>2+</sup> oscillations in bone morrow-derived monocytes, which result in differentiation into osteoclasts [<a href="#pone.0142934.ref034" class="ref-tip">34</a>]. The physiological consequence of RANKL signaling activation pathway is imitated in peroxiredoxin-2 knocked out mice and the results demonstrated a clear relationship among ROS, ROS scavenger proteins and the decrease in bone density. This study indirectly support our findings about the relation between the increases in peroxiredoxin levels and HTBD. In addition, there is a report about the usefulness of peroxiredoxin-2 as a biomarker to predict chemotherapy response in osteosarcome treatment, although the molecular mechanism behind the increase in peroxiredoxin-2 levels in osteosarcomes with poor response is not explained [<a href="#pone.0142934.ref035" class="ref-tip">35</a>].</p>
<a id="article1.body1.sec4.sec4.p5" name="article1.body1.sec4.sec4.p5" class="link-target"></a><p>Heterogeneous nuclear ribonucleoproteins (hnRNPs) constitute a set of polypeptides that bind heterogeneous nuclear RNA, the transcripts produced by RNA polymerase II [<a href="#pone.0142934.ref036" class="ref-tip">36</a>]. There is no report about the effect of hnRNP expression on bone turnover. However, hnRNP is a multifunctional protein that is involved not only in processing heterogeneous nuclear RNAs (hnRNAs) into mature mRNAs, but also acting as <em>trans</em>-factors in regulating gene expression [<a href="#pone.0142934.ref037" class="ref-tip">37</a>]. It is our prediction that the change in abundance of hnRNP might be the consequence of its trans-acting function.</p>
<a id="article1.body1.sec4.sec4.p6" name="article1.body1.sec4.sec4.p6" class="link-target"></a><p>Two proteins were identified in stem cells isolated from high bone turnover state related to the metabolic acidosis. This is a relevant finding because the patient we studied suffered from CRF and chronic metabolic acidosis. The first one was serine dehydratase protein (SDH), which catalyzes pyridoxal phosphate (PLP) dependent deamination of serine and threonine to produce pyruvate and α-ketobutyrate, respectively. This is one of the few enzymes that directly release ammonia from amino acids. This enzyme is reported to play an important role in adaptation to metabolic acidosis [<a href="#pone.0142934.ref038" class="ref-tip">38</a>].</p>
<a id="article1.body1.sec4.sec4.p7" name="article1.body1.sec4.sec4.p7" class="link-target"></a><p>Glutaminase (GA) kidney isoform-1 is a well studied enzyme with well-defined catalytic properties. This enzyme is known to catalyze the reaction that produces ammonium from glutamine (GLN) in renal tubular cells; so it plays an important role in maintaining acid-base homeostasis [<a href="#pone.0142934.ref039" class="ref-tip">39</a>]. Surprisingly, in this study, GA was also detected in AT-MSCs of the patient with SHPT (HTBD) before the PTX operation. GA has three different isoforms: GA Isoforms 1, 2 and 3. Isoform 1 is highly expressed in the brain and the kidney. There is no report stating expression of GA isoform-1, 2 or 3 in adipose tissue. However, since the stem cells isolated from the hAT displayed a higher abundance of this enzyme, the enzyme should either be expressed in AT at a low level or the AT has the potential to express GA isoform 1, when it is necessary. The reason why we observed such a high level GA isoform-1 expression in stem cells in HTBD patient might be due the fact that hyperparathyroidism is a possible cause for metabolic acidosis [<a href="#pone.0142934.ref040" class="ref-tip">40</a>].</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="sec020" name="sec020" data-toc="sec020" class="link-target" title="Conclusion"></a><h2>Conclusion</h2><a id="article1.body1.sec5.p1" name="article1.body1.sec5.p1" class="link-target"></a><p>This is the first study providing proteomic details of hAT-MSCs isolated from the same host under different conditions. Some of the identified and regulated proteins may be linked to the disease status of the host. However, some proteins that were identified did not reveal any association between the host’s disease status and the metabolic changes of the stem cells. We are aware of the fact that the study was undertaken <em>in vitro</em> and changes that reflect <em>in vivo</em> conditions may be missing or may not reflect the reality. However, any observation <em>in vitro</em> is worthy of investigation in <em>in vivo</em> and may provide valuable clues about the metabolic events. The list of the candidate proteins generated in this study needs further attention to establish firm, conclusive connections between disease-associated changes in stem cell metabolism.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec021" name="sec021" data-toc="sec021" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0142934"></div><div class="supplementary-material"><a name="pone.0142934.s001" id="pone.0142934.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0142934.s001">S1 Fig. </a>Flow cytometry analysis of of hAT-MSCs obtained from HTBD-hAT-MSCs, PTX-hAT-MSCs, LTBD-hAT-MSCs and the control.</h3><a id="article1.body1.sec6.supplementary-material1.caption1.p1" name="article1.body1.sec6.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">HTBD: Stem cells isolated from human adipose tissue obtained from a patient with secondary hyperparathyroidism (High turnover bone disease). LTBD: Stem cells isolated from human adipose tissue obtained from a patient with adynamic bone disease (Low turnover bone disease).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0142934.s001">https://doi.org/10.1371/journal.pone.0142934.s001</a></p><a id="article1.body1.sec6.supplementary-material1.caption1.p2" name="article1.body1.sec6.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The authors would like to thank Dr. Gulcin Gacar for her technical assistance during flow cytometry analysis. The authors received specific funding for this work from Kocaeli University BAP Project number: 2011/041.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: IY MK GA. Performed the experiments: IY MK GA AA EK. Analyzed the data: IY MK GA EK. Contributed reagents/materials/analysis tools: MK GA EK MT. Wrote the paper: IY MK.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0142934.ref001" id="pone.0142934.ref001" class="link-target"></a>Ott SM. Bone disease in CKD. Curr Opin Nephrol Hypertens. 21(4):376–81. Epub 2012.  pmid:22531161. <ul class="reflinks"><li><a href="#" data-author="Ott" data-cit="OttSM.%20Bone%20disease%20in%20CKD.%20Curr%20Opin%20Nephrol%20Hypertens.%2021%284%29%3A376%E2%80%9381.%20Epub%202012.%20doi%3A%2010.1097%2FMNH.0b013e32835421f6%2022531161." data-title="Bone%20disease%20in%20CKD" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22531161" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bone+disease+in+CKD+Ott+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0142934.ref002" id="pone.0142934.ref002" class="link-target"></a>Komaba H, Kakuta T, Fukagawa M. Diseases of the parathyroid gland in chronic kidney disease. Clin Exp Nephrol. 15(6):797–809. Epub 2011/08/06.  pmid:21818548. <ul class="reflinks"><li><a href="#" data-author="Komaba" data-cit="KomabaH%2C%20KakutaT%2C%20FukagawaM.%20Diseases%20of%20the%20parathyroid%20gland%20in%20chronic%20kidney%20disease.%20Clin%20Exp%20Nephrol.%2015%286%29%3A797%E2%80%93809.%20Epub%202011%2F08%2F06.%20doi%3A%2010.1007%2Fs10157-011-0502-5%2021818548." data-title="Diseases%20of%20the%20parathyroid%20gland%20in%20chronic%20kidney%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21818548" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Diseases+of+the+parathyroid+gland+in+chronic+kidney+disease+Komaba+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0142934.ref003" id="pone.0142934.ref003" class="link-target"></a>Moorthi RN, Moe SM. Recent advances in the noninvasive diagnosis of renal osteodystrophy. Kidney Int. 84(5):886–94. Epub 2013/06/27. ki2013254 [pii]  pmid:23802194. <ul class="reflinks"><li><a href="#" data-author="Moorthi" data-cit="MoorthiRN%2C%20MoeSM.%20Recent%20advances%20in%20the%20noninvasive%20diagnosis%20of%20renal%20osteodystrophy.%20Kidney%20Int.%2084%285%29%3A886%E2%80%9394.%20Epub%202013%2F06%2F27.%20ki2013254%20%5Bpii%5D%20doi%3A%2010.1038%2Fki.2013.254%2023802194." data-title="Recent%20advances%20in%20the%20noninvasive%20diagnosis%20of%20renal%20osteodystrophy" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23802194" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Recent+advances+in+the+noninvasive+diagnosis+of+renal+osteodystrophy+Moorthi+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0142934.ref004" id="pone.0142934.ref004" class="link-target"></a>Cozzolino M, Brancaccio D, Gallieni M, Slatopolsky E. Pathogenesis of vascular calcification in chronic kidney disease. Kidney Int. 2005;68(2):429–36. Epub 2005/07/15. KID421 [pii]  pmid:16014020. <ul class="reflinks"><li><a href="#" data-author="Cozzolino" data-cit="CozzolinoM%2C%20BrancaccioD%2C%20GallieniM%2C%20SlatopolskyE.%20Pathogenesis%20of%20vascular%20calcification%20in%20chronic%20kidney%20disease.%20Kidney%20Int.%202005%3B68%282%29%3A429%E2%80%9336.%20Epub%202005%2F07%2F15.%20KID421%20%5Bpii%5D%20doi%3A%2010.1111%2Fj.1523-1755.2005.00421.x%2016014020." data-title="Pathogenesis%20of%20vascular%20calcification%20in%20chronic%20kidney%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16014020" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Pathogenesis+of+vascular+calcification+in+chronic+kidney+disease+Cozzolino+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0142934.ref005" id="pone.0142934.ref005" class="link-target"></a>Bover J, Urena P, Brandenburg V, Goldsmith D, Ruiz C, DaSilva I, et al. Adynamic bone disease: from bone to vessels in chronic kidney disease. Semin Nephrol. 34(6):626–40. Epub 2014/12/17. S0270-9295(14)00148-X [pii]  pmid:25498381. <ul class="reflinks"><li><a href="#" data-author="Bover" data-cit="BoverJ%2C%20UrenaP%2C%20BrandenburgV%2C%20GoldsmithD%2C%20RuizC%2C%20DaSilvaI%2C%20et%20al.%20Adynamic%20bone%20disease%3A%20from%20bone%20to%20vessels%20in%20chronic%20kidney%20disease.%20Semin%20Nephrol.%2034%286%29%3A626%E2%80%9340.%20Epub%202014%2F12%2F17.%20S0270-9295%2814%2900148-X%20%5Bpii%5D%20doi%3A%2010.1016%2Fj.semnephrol.2014.09.008%2025498381." data-title="Adynamic%20bone%20disease%3A%20from%20bone%20to%20vessels%20in%20chronic%20kidney%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25498381" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Adynamic+bone+disease%3A+from+bone+to+vessels+in+chronic+kidney+disease+Bover+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0142934.ref006" id="pone.0142934.ref006" class="link-target"></a>Drueke TB. Foreword: extraskeletal calcifications in patients with chronic renal failure. Nephrol Dial Transplant. 2002;17(2):330–1. Epub 2002/01/29. pmid:11812900. <ul class="reflinks"><li><a href="#" data-author="Drueke" data-cit="DruekeTB.%20Foreword%3A%20extraskeletal%20calcifications%20in%20patients%20with%20chronic%20renal%20failure.%20Nephrol%20Dial%20Transplant.%202002%3B17%282%29%3A330%E2%80%931.%20Epub%202002%2F01%2F29.%2011812900." data-title="Foreword%3A%20extraskeletal%20calcifications%20in%20patients%20with%20chronic%20renal%20failure" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11812900" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Foreword%3A+extraskeletal+calcifications+in+patients+with+chronic+renal+failure+Drueke+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0142934.ref007" id="pone.0142934.ref007" class="link-target"></a>Cookson C. Mother of all cells. Scientists hope for huge benefits from the surge of research on embryonic stem cells. Sci Am. 2005;293(1):A6–11. Epub 2005/07/13. pmid:16008293. <ul class="reflinks"><li><a href="#" data-author="Cookson" data-cit="CooksonC.%20Mother%20of%20all%20cells.%20Scientists%20hope%20for%20huge%20benefits%20from%20the%20surge%20of%20research%20on%20embryonic%20stem%20cells.%20Sci%20Am.%202005%3B293%281%29%3AA6%E2%80%9311.%20Epub%202005%2F07%2F13.%2016008293." data-title="Mother%20of%20all%20cells.%20Scientists%20hope%20for%20huge%20benefits%20from%20the%20surge%20of%20research%20on%20embryonic%20stem%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16008293" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mother+of+all+cells.+Scientists+hope+for+huge+benefits+from+the+surge+of+research+on+embryonic+stem+cells+Cookson+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0142934.ref008" id="pone.0142934.ref008" class="link-target"></a>Mimeault M, Hauke R, Batra SK. Recent advances on the molecular mechanisms involved in the drug resistance of cancer cells and novel targeting therapies. Clin Pharmacol Ther. 2008;83(5):673–91. Epub 2007/09/06. 6100296 [pii]  pmid:17786164. <ul class="reflinks"><li><a href="#" data-author="Mimeault" data-cit="MimeaultM%2C%20HaukeR%2C%20BatraSK.%20Recent%20advances%20on%20the%20molecular%20mechanisms%20involved%20in%20the%20drug%20resistance%20of%20cancer%20cells%20and%20novel%20targeting%20therapies.%20Clin%20Pharmacol%20Ther.%202008%3B83%285%29%3A673%E2%80%9391.%20Epub%202007%2F09%2F06.%206100296%20%5Bpii%5D%20doi%3A%2010.1038%2Fsj.clpt.6100296%2017786164." data-title="Recent%20advances%20on%20the%20molecular%20mechanisms%20involved%20in%20the%20drug%20resistance%20of%20cancer%20cells%20and%20novel%20targeting%20therapies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17786164" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Recent+advances+on+the+molecular+mechanisms+involved+in+the+drug+resistance+of+cancer+cells+and+novel+targeting+therapies+Mimeault+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0142934.ref009" id="pone.0142934.ref009" class="link-target"></a>Gimble JM. Adipose tissue-derived therapeutics. Expert Opin Biol Ther. 2003;3(5):705–13. Epub 2003/07/26.  pmid:12880371. <ul class="reflinks"><li><a href="#" data-author="Gimble" data-cit="GimbleJM.%20Adipose%20tissue-derived%20therapeutics.%20Expert%20Opin%20Biol%20Ther.%202003%3B3%285%29%3A705%E2%80%9313.%20Epub%202003%2F07%2F26.%20doi%3A%2010.1517%2F14712598.3.5.705%2012880371." data-title="Adipose%20tissue-derived%20therapeutics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12880371" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Adipose+tissue-derived+therapeutics+Gimble+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0142934.ref010" id="pone.0142934.ref010" class="link-target"></a>Lee K, Kim H, Kim JM, Kim JR, Kim KJ, Kim YJ, et al. Systemic transplantation of human adipose-derived stem cells stimulates bone repair by promoting osteoblast and osteoclast function. J Cell Mol Med. 15(10):2082–94. Epub 2010/12/17.  pmid:21159123. <ul class="reflinks"><li><a href="#" data-author="Lee" data-cit="LeeK%2C%20KimH%2C%20KimJM%2C%20KimJR%2C%20KimKJ%2C%20KimYJ%2C%20et%20al.%20Systemic%20transplantation%20of%20human%20adipose-derived%20stem%20cells%20stimulates%20bone%20repair%20by%20promoting%20osteoblast%20and%20osteoclast%20function.%20J%20Cell%20Mol%20Med.%2015%2810%29%3A2082%E2%80%9394.%20Epub%202010%2F12%2F17.%20doi%3A%2010.1111%2Fj.1582-4934.2010.01230.x%2021159123." data-title="Systemic%20transplantation%20of%20human%20adipose-derived%20stem%20cells%20stimulates%20bone%20repair%20by%20promoting%20osteoblast%20and%20osteoclast%20function" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21159123" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Systemic+transplantation+of+human+adipose-derived+stem+cells+stimulates+bone+repair+by+promoting+osteoblast+and+osteoclast+function+Lee+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0142934.ref011" id="pone.0142934.ref011" class="link-target"></a>Zuk PA, Zhu M, Ashjian P, De Ugarte DA, Huang JI, Mizuno H, et al. Human adipose tissue is a source of multipotent stem cells. Mol Biol Cell. 2002;13(12):4279–95. Epub 2002/12/12.  pmid:12475952. <ul class="reflinks"><li><a href="#" data-author="Zuk" data-cit="ZukPA%2C%20ZhuM%2C%20AshjianP%2C%20De%20UgarteDA%2C%20HuangJI%2C%20MizunoH%2C%20et%20al.%20Human%20adipose%20tissue%20is%20a%20source%20of%20multipotent%20stem%20cells.%20Mol%20Biol%20Cell.%202002%3B13%2812%29%3A4279%E2%80%9395.%20Epub%202002%2F12%2F12.%20doi%3A%2010.1091%2Fmbc.E02-02-0105%2012475952." data-title="Human%20adipose%20tissue%20is%20a%20source%20of%20multipotent%20stem%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12475952" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Human+adipose+tissue+is+a+source+of+multipotent+stem+cells+Zuk+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0142934.ref012" id="pone.0142934.ref012" class="link-target"></a>Clarke PR, Williams HI. Ossification in extradural fat in Paget's disease of the spine. Br J Surg. 1975;62(7):571–2. Epub 1975/07/01. pmid:809080. <ul class="reflinks"><li><a href="#" data-author="Clarke" data-cit="ClarkePR%2C%20WilliamsHI.%20Ossification%20in%20extradural%20fat%20in%20Paget%27s%20disease%20of%20the%20spine.%20Br%20J%20Surg.%201975%3B62%287%29%3A571%E2%80%932.%20Epub%201975%2F07%2F01.%20809080." data-title="Ossification%20in%20extradural%20fat%20in%20Paget%27s%20disease%20of%20the%20spine" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/809080" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Ossification+in+extradural+fat+in+Paget%27s+disease+of+the+spine+Clarke+1975" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0142934.ref013" id="pone.0142934.ref013" class="link-target"></a>Shackelfold GDBL, McAlister WH. Calcified subcutaneous fat necrosis in infancy. J Can Assoc Radio. 1975;26(3):1107–10. <ul class="reflinks"><li><a href="#" data-author="Shackelfold" data-cit="ShackelfoldGDBL%2C%20McAlisterWH.%20Calcified%20subcutaneous%20fat%20necrosis%20in%20infancy.%20J%20Can%20Assoc%20Radio.%201975%3B26%283%29%3A1107%E2%80%9310." data-title="Calcified%20subcutaneous%20fat%20necrosis%20in%20infancy" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Calcified+subcutaneous+fat+necrosis+in+infancy+Shackelfold+1975" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0142934.ref014" id="pone.0142934.ref014" class="link-target"></a>Mitra S, Dove J, Somisetty SK. Subcutaneous fat necrosis in newborn-an unusual case and review of literature. Eur J Pediatr. 170(9):1107–10. Epub 2011/02/15.  pmid:21318229. <ul class="reflinks"><li><a href="#" data-author="Mitra" data-cit="MitraS%2C%20DoveJ%2C%20SomisettySK.%20Subcutaneous%20fat%20necrosis%20in%20newborn-an%20unusual%20case%20and%20review%20of%20literature.%20Eur%20J%20Pediatr.%20170%289%29%3A1107%E2%80%9310.%20Epub%202011%2F02%2F15.%20doi%3A%2010.1007%2Fs00431-011-1405-x%2021318229." data-title="Subcutaneous%20fat%20necrosis%20in%20newborn-an%20unusual%20case%20and%20review%20of%20literature" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21318229" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Subcutaneous+fat+necrosis+in+newborn-an+unusual+case+and+review+of+literature+Mitra+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0142934.ref015" id="pone.0142934.ref015" class="link-target"></a>van Hoof D, Krijgsveld J, Mummery C. Proteomic analysis of stem cell differentiation and early development. Cold Spring Harb Perspect Biol. 4(3). Epub 2012/02/10. cshperspect.a008177 [pii]  pmid:22317846. <ul class="reflinks"><li><a href="#" data-author="van%20Hoof" data-cit="van%20HoofD%2C%20KrijgsveldJ%2C%20MummeryC.%20Proteomic%20analysis%20of%20stem%20cell%20differentiation%20and%20early%20development.%20Cold%20Spring%20Harb%20Perspect%20Biol.%204%283%29.%20Epub%202012%2F02%2F10.%20cshperspect.a008177%20%5Bpii%5D%20doi%3A%2010.1101%2Fcshperspect.a008177%2022317846." data-title="Proteomic%20analysis%20of%20stem%20cell%20differentiation%20and%20early%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22317846" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Proteomic+analysis+of+stem+cell+differentiation+and+early+development+van+Hoof+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0142934.ref016" id="pone.0142934.ref016" class="link-target"></a>Ertas G, Ural E, Ural D, Aksoy A, Kozdag G, Gacar G, et al. Comparative analysis of apoptotic resistance of mesenchymal stem cells isolated from human bone marrow and adipose tissue. ScientificWorldJournal. 2012:105698. Epub 2012/05/24.  pmid:22619573. <ul class="reflinks"><li><a href="#" data-author="Ertas" data-cit="ErtasG%2C%20UralE%2C%20UralD%2C%20AksoyA%2C%20KozdagG%2C%20GacarG%2C%20et%20al.%20Comparative%20analysis%20of%20apoptotic%20resistance%20of%20mesenchymal%20stem%20cells%20isolated%20from%20human%20bone%20marrow%20and%20adipose%20tissue.%20ScientificWorldJournal.%202012%3A105698.%20Epub%202012%2F05%2F24.%20doi%3A%2010.1100%2F2012%2F105698%2022619573." data-title="Comparative%20analysis%20of%20apoptotic%20resistance%20of%20mesenchymal%20stem%20cells%20isolated%20from%20human%20bone%20marrow%20and%20adipose%20tissue" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22619573" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Comparative+analysis+of+apoptotic+resistance+of+mesenchymal+stem+cells+isolated+from+human+bone+marrow+and+adipose+tissue+Ertas+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0142934.ref017" id="pone.0142934.ref017" class="link-target"></a>Karaoz E, Aksoy A, Ayhan S, Sariboyaci AE, Kaymaz F, Kasap M. Characterization of mesenchymal stem cells from rat bone marrow: ultrastructural properties, differentiation potential and immunophenotypic markers. Histochem Cell Biol. 2009;132(5):533–46. Epub 2009/08/19.  pmid:19688349. <ul class="reflinks"><li><a href="#" data-author="Karaoz" data-cit="KaraozE%2C%20AksoyA%2C%20AyhanS%2C%20SariboyaciAE%2C%20KaymazF%2C%20KasapM.%20Characterization%20of%20mesenchymal%20stem%20cells%20from%20rat%20bone%20marrow%3A%20ultrastructural%20properties%2C%20differentiation%20potential%20and%20immunophenotypic%20markers.%20Histochem%20Cell%20Biol.%202009%3B132%285%29%3A533%E2%80%9346.%20Epub%202009%2F08%2F19.%20doi%3A%2010.1007%2Fs00418-009-0629-6%2019688349." data-title="Characterization%20of%20mesenchymal%20stem%20cells%20from%20rat%20bone%20marrow%3A%20ultrastructural%20properties%2C%20differentiation%20potential%20and%20immunophenotypic%20markers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19688349" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+mesenchymal+stem+cells+from+rat+bone+marrow%3A+ultrastructural+properties%2C+differentiation+potential+and+immunophenotypic+markers+Karaoz+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0142934.ref018" id="pone.0142934.ref018" class="link-target"></a>Mi H, Muruganujan A, Casagrande JT, Thomas PD. Large-scale gene function analysis with the PANTHER classification system. Nat Protoc. 8(8):1551–66. Epub 2013/07/23. nprot.2013.092 [pii]  pmid:23868073. <ul class="reflinks"><li><a href="#" data-author="Mi" data-cit="MiH%2C%20MuruganujanA%2C%20CasagrandeJT%2C%20ThomasPD.%20Large-scale%20gene%20function%20analysis%20with%20the%20PANTHER%20classification%20system.%20Nat%20Protoc.%208%288%29%3A1551%E2%80%9366.%20Epub%202013%2F07%2F23.%20nprot.2013.092%20%5Bpii%5D%20doi%3A%2010.1038%2Fnprot.2013.092%2023868073." data-title="Large-scale%20gene%20function%20analysis%20with%20the%20PANTHER%20classification%20system" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23868073" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Large-scale+gene+function+analysis+with+the+PANTHER+classification+system+Mi+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0142934.ref019" id="pone.0142934.ref019" class="link-target"></a>Akpinar G, Kasap M, Aksoy A, Duruksu G, Gacar G, Karaoz E. Phenotypic and proteomic characteristics of human dental pulp derived mesenchymal stem cells from a natal, an exfoliated deciduous, and an impacted third molar tooth. Stem Cells Int. 2014:457059. Epub 2014/11/08.  pmid:25379041. <ul class="reflinks"><li><a href="#" data-author="Akpinar" data-cit="AkpinarG%2C%20KasapM%2C%20AksoyA%2C%20DuruksuG%2C%20GacarG%2C%20KaraozE.%20Phenotypic%20and%20proteomic%20characteristics%20of%20human%20dental%20pulp%20derived%20mesenchymal%20stem%20cells%20from%20a%20natal%2C%20an%20exfoliated%20deciduous%2C%20and%20an%20impacted%20third%20molar%20tooth.%20Stem%20Cells%20Int.%202014%3A457059.%20Epub%202014%2F11%2F08.%20doi%3A%2010.1155%2F2014%2F457059%2025379041." data-title="Phenotypic%20and%20proteomic%20characteristics%20of%20human%20dental%20pulp%20derived%20mesenchymal%20stem%20cells%20from%20a%20natal%2C%20an%20exfoliated%20deciduous%2C%20and%20an%20impacted%20third%20molar%20tooth" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25379041" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phenotypic+and+proteomic+characteristics+of+human+dental+pulp+derived+mesenchymal+stem+cells+from+a+natal%2C+an+exfoliated+deciduous%2C+and+an+impacted+third+molar+tooth+Akpinar+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0142934.ref020" id="pone.0142934.ref020" class="link-target"></a>Rojas-Rios P, Gonzalez-Reyes A. Concise review: The plasticity of stem cell niches: a general property behind tissue homeostasis and repair. Stem Cells. 32(4):852–9. Epub 2013/12/21.  pmid:24356972. <ul class="reflinks"><li><a href="#" data-author="Rojas-Rios" data-cit="Rojas-RiosP%2C%20Gonzalez-ReyesA.%20Concise%20review%3A%20The%20plasticity%20of%20stem%20cell%20niches%3A%20a%20general%20property%20behind%20tissue%20homeostasis%20and%20repair.%20Stem%20Cells.%2032%284%29%3A852%E2%80%939.%20Epub%202013%2F12%2F21.%20doi%3A%2010.1002%2Fstem.1621%2024356972." data-title="Concise%20review%3A%20The%20plasticity%20of%20stem%20cell%20niches%3A%20a%20general%20property%20behind%20tissue%20homeostasis%20and%20repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24356972" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Concise+review%3A+The+plasticity+of+stem+cell+niches%3A+a+general+property+behind+tissue+homeostasis+and+repair+Rojas-Rios+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0142934.ref021" id="pone.0142934.ref021" class="link-target"></a>Yamashita YM, Fuller MT, Jones DL. Signaling in stem cell niches: lessons from the Drosophila germline. J Cell Sci. 2005;118(Pt 4):665–72. Epub 2005/02/11. 118/4/665 [pii]  pmid:15701923. <ul class="reflinks"><li><a href="#" data-author="Yamashita" data-cit="YamashitaYM%2C%20FullerMT%2C%20JonesDL.%20Signaling%20in%20stem%20cell%20niches%3A%20lessons%20from%20the%20Drosophila%20germline.%20J%20Cell%20Sci.%202005%3B118%28Pt%204%29%3A665%E2%80%9372.%20Epub%202005%2F02%2F11.%20118%2F4%2F665%20%5Bpii%5D%20doi%3A%2010.1242%2Fjcs.01680%2015701923." data-title="Signaling%20in%20stem%20cell%20niches%3A%20lessons%20from%20the%20Drosophila%20germline" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15701923" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Signaling+in+stem+cell+niches%3A+lessons+from+the+Drosophila+germline+Yamashita+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0142934.ref022" id="pone.0142934.ref022" class="link-target"></a>Ferreira A, Drueke TB. Biological markers in the diagnosis of the different forms of renal osteodystrophy. Am J Med Sci. 2000;320(2):85–9. Epub 2000/09/12. pmid:10981481. <ul class="reflinks"><li><a href="#" data-author="Ferreira" data-cit="FerreiraA%2C%20DruekeTB.%20Biological%20markers%20in%20the%20diagnosis%20of%20the%20different%20forms%20of%20renal%20osteodystrophy.%20Am%20J%20Med%20Sci.%202000%3B320%282%29%3A85%E2%80%939.%20Epub%202000%2F09%2F12.%2010981481." data-title="Biological%20markers%20in%20the%20diagnosis%20of%20the%20different%20forms%20of%20renal%20osteodystrophy" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10981481" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Biological+markers+in+the+diagnosis+of+the+different+forms+of+renal+osteodystrophy+Ferreira+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0142934.ref023" id="pone.0142934.ref023" class="link-target"></a>Passos JF, von Zglinicki T. Mitochondria, telomeres and cell senescence. Exp Gerontol. 2005;40(6):466–72. Epub 2005/06/21. S0531-5565(05)00081-1 [pii]  pmid:15963673. <ul class="reflinks"><li><a href="#" data-author="Passos" data-cit="PassosJF%2C%20von%20ZglinickiT.%20Mitochondria%2C%20telomeres%20and%20cell%20senescence.%20Exp%20Gerontol.%202005%3B40%286%29%3A466%E2%80%9372.%20Epub%202005%2F06%2F21.%20S0531-5565%2805%2900081-1%20%5Bpii%5D%20doi%3A%2010.1016%2Fj.exger.2005.04.006%2015963673." data-title="Mitochondria%2C%20telomeres%20and%20cell%20senescence" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15963673" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mitochondria%2C+telomeres+and+cell+senescence+Passos+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0142934.ref024" id="pone.0142934.ref024" class="link-target"></a>DeLany JP, Floyd ZE, Zvonic S, Smith A, Gravois A, Reiners E, et al. Proteomic analysis of primary cultures of human adipose-derived stem cells: modulation by Adipogenesis. Mol Cell Proteomics. 2005;4(6):731–40. Epub 2005/03/09. M400198-MCP200 [pii]  pmid:15753122. <ul class="reflinks"><li><a href="#" data-author="DeLany" data-cit="DeLanyJP%2C%20FloydZE%2C%20ZvonicS%2C%20SmithA%2C%20GravoisA%2C%20ReinersE%2C%20et%20al.%20Proteomic%20analysis%20of%20primary%20cultures%20of%20human%20adipose-derived%20stem%20cells%3A%20modulation%20by%20Adipogenesis.%20Mol%20Cell%20Proteomics.%202005%3B4%286%29%3A731%E2%80%9340.%20Epub%202005%2F03%2F09.%20M400198-MCP200%20%5Bpii%5D%20doi%3A%2010.1074%2Fmcp.M400198-MCP200%2015753122." data-title="Proteomic%20analysis%20of%20primary%20cultures%20of%20human%20adipose-derived%20stem%20cells%3A%20modulation%20by%20Adipogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15753122" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Proteomic+analysis+of+primary+cultures+of+human+adipose-derived+stem+cells%3A+modulation+by+Adipogenesis+DeLany+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0142934.ref025" id="pone.0142934.ref025" class="link-target"></a>Park HW, Shin JS, Kim CW. Proteome of mesenchymal stem cells. Proteomics. 2007;7(16):2881–94. Epub 2007/07/27.  pmid:17654458. <ul class="reflinks"><li><a href="#" data-author="Park" data-cit="ParkHW%2C%20ShinJS%2C%20KimCW.%20Proteome%20of%20mesenchymal%20stem%20cells.%20Proteomics.%202007%3B7%2816%29%3A2881%E2%80%9394.%20Epub%202007%2F07%2F27.%20doi%3A%2010.1002%2Fpmic.200700089%2017654458." data-title="Proteome%20of%20mesenchymal%20stem%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17654458" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Proteome+of+mesenchymal+stem+cells+Park+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0142934.ref026" id="pone.0142934.ref026" class="link-target"></a>Amandio AR, Gaspar P, Whited JL, Janody F. Subunits of the Drosophila actin-capping protein heterodimer regulate each other at multiple levels. PLoS One. 9(5):e96326. Epub 2014/05/03.  PONE-D-14-03157 [pii]. pmid:24788460. <ul class="reflinks"><li><a href="#" data-author="Amandio" data-cit="AmandioAR%2C%20GasparP%2C%20WhitedJL%2C%20JanodyF.%20Subunits%20of%20the%20Drosophila%20actin-capping%20protein%20heterodimer%20regulate%20each%20other%20at%20multiple%20levels.%20PLoS%20One.%209%285%29%3Ae96326.%20Epub%202014%2F05%2F03.%20doi%3A%2010.1371%2Fjournal.pone.0096326%20PONE-D-14-03157%20%5Bpii%5D.%2024788460." data-title="Subunits%20of%20the%20Drosophila%20actin-capping%20protein%20heterodimer%20regulate%20each%20other%20at%20multiple%20levels" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24788460" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Subunits+of+the+Drosophila+actin-capping+protein+heterodimer+regulate+each+other+at+multiple+levels+Amandio+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0142934.ref027" id="pone.0142934.ref027" class="link-target"></a>Stevenson RP, Veltman D, Machesky LM. Actin-bundling proteins in cancer progression at a glance. J Cell Sci. 125(Pt 5):1073–9. Epub 2012/04/12. 125/5/1073 [pii]  pmid:22492983. <ul class="reflinks"><li><a href="#" data-author="Stevenson" data-cit="StevensonRP%2C%20VeltmanD%2C%20MacheskyLM.%20Actin-bundling%20proteins%20in%20cancer%20progression%20at%20a%20glance.%20J%20Cell%20Sci.%20125%28Pt%205%29%3A1073%E2%80%939.%20Epub%202012%2F04%2F12.%20125%2F5%2F1073%20%5Bpii%5D%20doi%3A%2010.1242%2Fjcs.093799%2022492983." data-title="Actin-bundling%20proteins%20in%20cancer%20progression%20at%20a%20glance" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22492983" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Actin-bundling+proteins+in+cancer+progression+at+a+glance+Stevenson+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0142934.ref028" id="pone.0142934.ref028" class="link-target"></a>Fernandez BG, Gaspar P, Bras-Pereira C, Jezowska B, Rebelo SR, Janody F. Actin-Capping Protein and the Hippo pathway regulate F-actin and tissue growth in Drosophila. Development. 138(11):2337–46. Epub 2011/04/29. dev.063545 [pii]  pmid:21525075. <ul class="reflinks"><li><a href="#" data-author="Fernandez" data-cit="FernandezBG%2C%20GasparP%2C%20Bras-PereiraC%2C%20JezowskaB%2C%20RebeloSR%2C%20JanodyF.%20Actin-Capping%20Protein%20and%20the%20Hippo%20pathway%20regulate%20F-actin%20and%20tissue%20growth%20in%20Drosophila.%20Development.%20138%2811%29%3A2337%E2%80%9346.%20Epub%202011%2F04%2F29.%20dev.063545%20%5Bpii%5D%20doi%3A%2010.1242%2Fdev.063545%2021525075." data-title="Actin-Capping%20Protein%20and%20the%20Hippo%20pathway%20regulate%20F-actin%20and%20tissue%20growth%20in%20Drosophila" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21525075" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Actin-Capping+Protein+and+the+Hippo+pathway+regulate+F-actin+and+tissue+growth+in+Drosophila+Fernandez+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0142934.ref029" id="pone.0142934.ref029" class="link-target"></a>Grgurevic L, Macek B, Durdevic D, Vukicevic S. Detection of bone and cartilage-related proteins in plasma of patients with a bone fracture using liquid chromatography-mass spectrometry. Int Orthop. 2007;31(6):743–51. Epub 2007/07/03.  pmid:17602227. <ul class="reflinks"><li><a href="#" data-author="Grgurevic" data-cit="GrgurevicL%2C%20MacekB%2C%20DurdevicD%2C%20VukicevicS.%20Detection%20of%20bone%20and%20cartilage-related%20proteins%20in%20plasma%20of%20patients%20with%20a%20bone%20fracture%20using%20liquid%20chromatography-mass%20spectrometry.%20Int%20Orthop.%202007%3B31%286%29%3A743%E2%80%9351.%20Epub%202007%2F07%2F03.%20doi%3A%2010.1007%2Fs00264-007-0404-z%2017602227." data-title="Detection%20of%20bone%20and%20cartilage-related%20proteins%20in%20plasma%20of%20patients%20with%20a%20bone%20fracture%20using%20liquid%20chromatography-mass%20spectrometry" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17602227" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Detection+of+bone+and+cartilage-related+proteins+in+plasma+of+patients+with+a+bone+fracture+using+liquid+chromatography-mass+spectrometry+Grgurevic+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0142934.ref030" id="pone.0142934.ref030" class="link-target"></a>Rodriguez JP, Gonzalez M, Rios S, Cambiazo V. Cytoskeletal organization of human mesenchymal stem cells (MSC) changes during their osteogenic differentiation. J Cell Biochem. 2004;93(4):721–31. Epub 2005/01/22.  pmid:15660416. <ul class="reflinks"><li><a href="#" data-author="Rodriguez" data-cit="RodriguezJP%2C%20GonzalezM%2C%20RiosS%2C%20CambiazoV.%20Cytoskeletal%20organization%20of%20human%20mesenchymal%20stem%20cells%20%28MSC%29%20changes%20during%20their%20osteogenic%20differentiation.%20J%20Cell%20Biochem.%202004%3B93%284%29%3A721%E2%80%9331.%20Epub%202005%2F01%2F22.%20doi%3A%2010.1002%2Fjcb.20234%2015660416." data-title="Cytoskeletal%20organization%20of%20human%20mesenchymal%20stem%20cells%20%28MSC%29%20changes%20during%20their%20osteogenic%20differentiation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15660416" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cytoskeletal+organization+of+human+mesenchymal+stem+cells+%28MSC%29+changes+during+their+osteogenic+differentiation+Rodriguez+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0142934.ref031" id="pone.0142934.ref031" class="link-target"></a>Rodriguez JP, Garat S, Gajardo H, Pino AM, Seitz G. Abnormal osteogenesis in osteoporotic patients is reflected by altered mesenchymal stem cells dynamics. J Cell Biochem. 1999;75(3):414–23. Epub 1999/10/28.  [pii]. pmid:10536365. <ul class="reflinks"><li><a href="#" data-author="Rodriguez" data-cit="RodriguezJP%2C%20GaratS%2C%20GajardoH%2C%20PinoAM%2C%20SeitzG.%20Abnormal%20osteogenesis%20in%20osteoporotic%20patients%20is%20reflected%20by%20altered%20mesenchymal%20stem%20cells%20dynamics.%20J%20Cell%20Biochem.%201999%3B75%283%29%3A414%E2%80%9323.%20Epub%201999%2F10%2F28.%20doi%3A%2010.1002%2F%28SICI%291097-4644%2819991201%2975%3A3%3C414%3A%3AAID-JCB7%3E3.0.CO%3B2-C%20%5Bpii%5D.%2010536365." data-title="Abnormal%20osteogenesis%20in%20osteoporotic%20patients%20is%20reflected%20by%20altered%20mesenchymal%20stem%20cells%20dynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10536365" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Abnormal+osteogenesis+in+osteoporotic+patients+is+reflected+by+altered+mesenchymal+stem+cells+dynamics+Rodriguez+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0142934.ref032" id="pone.0142934.ref032" class="link-target"></a>Ahsan MK, Lekli I, Ray D, Yodoi J, Das DK. Redox regulation of cell survival by the thioredoxin superfamily: an implication of redox gene therapy in the heart. Antioxid Redox Signal. 2009;11(11):2741–58. Epub 2009/07/09.  pmid:19583492. <ul class="reflinks"><li><a href="#" data-author="Ahsan" data-cit="AhsanMK%2C%20LekliI%2C%20RayD%2C%20YodoiJ%2C%20DasDK.%20Redox%20regulation%20of%20cell%20survival%20by%20the%20thioredoxin%20superfamily%3A%20an%20implication%20of%20redox%20gene%20therapy%20in%20the%20heart.%20Antioxid%20Redox%20Signal.%202009%3B11%2811%29%3A2741%E2%80%9358.%20Epub%202009%2F07%2F09.%20doi%3A%2010.1089%2FARS.2009.2683%2019583492." data-title="Redox%20regulation%20of%20cell%20survival%20by%20the%20thioredoxin%20superfamily%3A%20an%20implication%20of%20redox%20gene%20therapy%20in%20the%20heart" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19583492" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Redox+regulation+of+cell+survival+by+the+thioredoxin+superfamily%3A+an+implication+of+redox+gene+therapy+in+the+heart+Ahsan+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0142934.ref033" id="pone.0142934.ref033" class="link-target"></a>Rhee SG, Chae HZ, Kim K. Peroxiredoxins: a historical overview and speculative preview of novel mechanisms and emerging concepts in cell signaling. Free Radic Biol Med. 2005;38(12):1543–52. Epub 2005/05/27. S0891-5849(05)00098-5 [pii]  pmid:15917183. <ul class="reflinks"><li><a href="#" data-author="Rhee" data-cit="RheeSG%2C%20ChaeHZ%2C%20KimK.%20Peroxiredoxins%3A%20a%20historical%20overview%20and%20speculative%20preview%20of%20novel%20mechanisms%20and%20emerging%20concepts%20in%20cell%20signaling.%20Free%20Radic%20Biol%20Med.%202005%3B38%2812%29%3A1543%E2%80%9352.%20Epub%202005%2F05%2F27.%20S0891-5849%2805%2900098-5%20%5Bpii%5D%20doi%3A%2010.1016%2Fj.freeradbiomed.2005.02.026%2015917183." data-title="Peroxiredoxins%3A%20a%20historical%20overview%20and%20speculative%20preview%20of%20novel%20mechanisms%20and%20emerging%20concepts%20in%20cell%20signaling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15917183" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Peroxiredoxins%3A+a+historical+overview+and+speculative+preview+of+novel+mechanisms+and+emerging+concepts+in+cell+signaling+Rhee+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0142934.ref034" id="pone.0142934.ref034" class="link-target"></a>Kim MS, Yang YM, Son A, Tian YS, Lee SI, Kang SW, et al. RANKL-mediated reactive oxygen species pathway that induces long lasting Ca2+ oscillations essential for osteoclastogenesis. J Biol Chem. 285(10):6913–21. Epub 2010/01/06. M109.051557 [pii]  pmid:20048168. <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimMS%2C%20YangYM%2C%20SonA%2C%20TianYS%2C%20LeeSI%2C%20KangSW%2C%20et%20al.%20RANKL-mediated%20reactive%20oxygen%20species%20pathway%20that%20induces%20long%20lasting%20Ca2%2B%20oscillations%20essential%20for%20osteoclastogenesis.%20J%20Biol%20Chem.%20285%2810%29%3A6913%E2%80%9321.%20Epub%202010%2F01%2F06.%20M109.051557%20%5Bpii%5D%20doi%3A%2010.1074%2Fjbc.M109.051557%2020048168." data-title="RANKL-mediated%20reactive%20oxygen%20species%20pathway%20that%20induces%20long%20lasting%20Ca2%2B%20oscillations%20essential%20for%20osteoclastogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20048168" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=RANKL-mediated+reactive+oxygen+species+pathway+that+induces+long+lasting+Ca2%2B+oscillations+essential+for+osteoclastogenesis+Kim+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0142934.ref035" id="pone.0142934.ref035" class="link-target"></a>Kikuta K, Tochigi N, Saito S, Shimoda T, Morioka H, Toyama Y, et al. Peroxiredoxin 2 as a chemotherapy responsiveness biomarker candidate in osteosarcoma revealed by proteomics. Proteomics Clin Appl. 4(5):560–7. Epub 2010/12/08.  pmid:21137073. <ul class="reflinks"><li><a href="#" data-author="Kikuta" data-cit="KikutaK%2C%20TochigiN%2C%20SaitoS%2C%20ShimodaT%2C%20MoriokaH%2C%20ToyamaY%2C%20et%20al.%20Peroxiredoxin%202%20as%20a%20chemotherapy%20responsiveness%20biomarker%20candidate%20in%20osteosarcoma%20revealed%20by%20proteomics.%20Proteomics%20Clin%20Appl.%204%285%29%3A560%E2%80%937.%20Epub%202010%2F12%2F08.%20doi%3A%2010.1002%2Fprca.200900172%2021137073." data-title="Peroxiredoxin%202%20as%20a%20chemotherapy%20responsiveness%20biomarker%20candidate%20in%20osteosarcoma%20revealed%20by%20proteomics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21137073" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Peroxiredoxin+2+as+a+chemotherapy+responsiveness+biomarker+candidate+in+osteosarcoma+revealed+by+proteomics+Kikuta+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0142934.ref036" id="pone.0142934.ref036" class="link-target"></a>Honore B, Rasmussen HH, Vorum H, Dejgaard K, Liu X, Gromov P, et al. Heterogeneous nuclear ribonucleoproteins H, H', and F are members of a ubiquitously expressed subfamily of related but distinct proteins encoded by genes mapping to different chromosomes. J Biol Chem. 1995;270(48):28780–9. Epub 1995/12/01. pmid:7499401. <ul class="reflinks"><li><a href="#" data-author="Honore" data-cit="HonoreB%2C%20RasmussenHH%2C%20VorumH%2C%20DejgaardK%2C%20LiuX%2C%20GromovP%2C%20et%20al.%20Heterogeneous%20nuclear%20ribonucleoproteins%20H%2C%20H%27%2C%20and%20F%20are%20members%20of%20a%20ubiquitously%20expressed%20subfamily%20of%20related%20but%20distinct%20proteins%20encoded%20by%20genes%20mapping%20to%20different%20chromosomes.%20J%20Biol%20Chem.%201995%3B270%2848%29%3A28780%E2%80%939.%20Epub%201995%2F12%2F01.%207499401." data-title="Heterogeneous%20nuclear%20ribonucleoproteins%20H%2C%20H%27%2C%20and%20F%20are%20members%20of%20a%20ubiquitously%20expressed%20subfamily%20of%20related%20but%20distinct%20proteins%20encoded%20by%20genes%20mapping%20to%20different%20chromosomes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7499401" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Heterogeneous+nuclear+ribonucleoproteins+H%2C+H%27%2C+and+F+are+members+of+a+ubiquitously+expressed+subfamily+of+related+but+distinct+proteins+encoded+by+genes+mapping+to+different+chromosomes+Honore+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0142934.ref037" id="pone.0142934.ref037" class="link-target"></a>Chaudhury A, Chander P, Howe PH. Heterogeneous nuclear ribonucleoproteins (hnRNPs) in cellular processes: Focus on hnRNP E1's multifunctional regulatory roles. RNA. 16(8):1449–62. Epub 2010/06/30. rna.2254110 [pii]  pmid:20584894. <ul class="reflinks"><li><a href="#" data-author="Chaudhury" data-cit="ChaudhuryA%2C%20ChanderP%2C%20HowePH.%20Heterogeneous%20nuclear%20ribonucleoproteins%20%28hnRNPs%29%20in%20cellular%20processes%3A%20Focus%20on%20hnRNP%20E1%27s%20multifunctional%20regulatory%20roles.%20RNA.%2016%288%29%3A1449%E2%80%9362.%20Epub%202010%2F06%2F30.%20rna.2254110%20%5Bpii%5D%20doi%3A%2010.1261%2Frna.2254110%2020584894." data-title="Heterogeneous%20nuclear%20ribonucleoproteins%20%28hnRNPs%29%20in%20cellular%20processes%3A%20Focus%20on%20hnRNP%20E1%27s%20multifunctional%20regulatory%20roles" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20584894" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Heterogeneous+nuclear+ribonucleoproteins+%28hnRNPs%29+in+cellular+processes%3A+Focus+on+hnRNP+E1%27s+multifunctional+regulatory+roles+Chaudhury+" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0142934.ref038" id="pone.0142934.ref038" class="link-target"></a>Lopez-Flores I, Peragon J, Valderrama R, Esteban FJ, Luque F, Peinado MA, et al. Downregulation in the expression of the serine dehydratase in the rat liver during chronic metabolic acidosis. Am J Physiol Regul Integr Comp Physiol. 2006;291(5):R1295–302. Epub 2006/06/24. 00095.2006 [pii]  pmid:16793941. <ul class="reflinks"><li><a href="#" data-author="Lopez-Flores" data-cit="Lopez-FloresI%2C%20PeragonJ%2C%20ValderramaR%2C%20EstebanFJ%2C%20LuqueF%2C%20PeinadoMA%2C%20et%20al.%20Downregulation%20in%20the%20expression%20of%20the%20serine%20dehydratase%20in%20the%20rat%20liver%20during%20chronic%20metabolic%20acidosis.%20Am%20J%20Physiol%20Regul%20Integr%20Comp%20Physiol.%202006%3B291%285%29%3AR1295%E2%80%93302.%20Epub%202006%2F06%2F24.%2000095.2006%20%5Bpii%5D%20doi%3A%2010.1152%2Fajpregu.00095.2006%2016793941." data-title="Downregulation%20in%20the%20expression%20of%20the%20serine%20dehydratase%20in%20the%20rat%20liver%20during%20chronic%20metabolic%20acidosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16793941" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Downregulation+in+the+expression+of+the+serine+dehydratase+in+the+rat+liver+during+chronic+metabolic+acidosis+Lopez-Flores+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0142934.ref039" id="pone.0142934.ref039" class="link-target"></a>Curthoys NP, Lowry OH. The distribution of glutaminase isoenzymes in the various structures of the nephron in normal, acidotic, and alkalotic rat kidney. J Biol Chem. 1973;248(1):162–8. Epub 1973/01/10. pmid:4692829. <ul class="reflinks"><li><a href="#" data-author="Curthoys" data-cit="CurthoysNP%2C%20LowryOH.%20The%20distribution%20of%20glutaminase%20isoenzymes%20in%20the%20various%20structures%20of%20the%20nephron%20in%20normal%2C%20acidotic%2C%20and%20alkalotic%20rat%20kidney.%20J%20Biol%20Chem.%201973%3B248%281%29%3A162%E2%80%938.%20Epub%201973%2F01%2F10.%204692829." data-title="The%20distribution%20of%20glutaminase%20isoenzymes%20in%20the%20various%20structures%20of%20the%20nephron%20in%20normal%2C%20acidotic%2C%20and%20alkalotic%20rat%20kidney" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/4692829" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+distribution+of+glutaminase+isoenzymes+in+the+various+structures+of+the+nephron+in+normal%2C+acidotic%2C+and+alkalotic+rat+kidney+Curthoys+1973" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0142934.ref040" id="pone.0142934.ref040" class="link-target"></a>Ibrahim H, Lee YJ, Curthoys NP. Renal response to metabolic acidosis: role of mRNA stabilization. Kidney Int. 2008;73(1):11–8. Epub 2007/10/05. 5002581 [pii]  pmid:17914349. <ul class="reflinks"><li><a href="#" data-author="Ibrahim" data-cit="IbrahimH%2C%20LeeYJ%2C%20CurthoysNP.%20Renal%20response%20to%20metabolic%20acidosis%3A%20role%20of%20mRNA%20stabilization.%20Kidney%20Int.%202008%3B73%281%29%3A11%E2%80%938.%20Epub%202007%2F10%2F05.%205002581%20%5Bpii%5D%20doi%3A%2010.1038%2Fsj.ki.5002581%2017914349." data-title="Renal%20response%20to%20metabolic%20acidosis%3A%20role%20of%20mRNA%20stabilization" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17914349" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Renal+response+to+metabolic+acidosis%3A+role+of+mRNA+stabilization+Ibrahim+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

