



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>MicroRNAs (miRNAs) are small, single-stranded, non-coding RNAs that play pivotal roles in human cancer development and progression, such as tumor metastasis. Here, we identified the miRNAs that regulate hepatocellular carcinoma (HCC) cell migration by a high-throughput screening method using the classical wound-healing assay with time-lapse video microscopy and validation with a transwell migration assay. Eleven miRNAs (miR-134, -146b-3p, -188-3p, -525-3p, -661, -767-5p, -891a, -891b, -1244, -1247 and miR-1471) were found to promote or inhibit HCC cell migration. Further investigation revealed that miR-134 suppressed the invasion and metastasis of HCC cells <em>in vitro</em> and <em>in vivo</em>, and integrin beta 1 (ITGB1) was a direct and functional target gene of miR-134. Moreover, miR-134 inhibited the phosphorylation of focal adhesion kinase (FAK) and the activation of RhoA downstream of the ITGB1 pathway, thereby decreasing stress fiber formation and cell adhesion in HCC cells. In conclusion, we demonstrated that miR-134 is a novel metastasis suppressor in HCC and could be a potential therapeutic target for the treatment of HCC.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zha R, Guo W, Zhang Z, Qiu Z, Wang Q, Ding J, et al.  (2014) Genome-Wide Screening Identified That miR-134 Acts as a Metastasis Suppressor by Targeting Integrin β1 in Hepatocellular Carcinoma. PLoS ONE 9(2):
           e87665.
        
        https://doi.org/10.1371/journal.pone.0087665</p><p><strong>Editor: </strong>William B. Coleman, University of North Carolina School of Medicine, United States of America</p><p><strong>Received: </strong>August 22, 2013; <strong>Accepted: </strong>December 28, 2013; <strong>Published: </strong> February 3, 2014</p><p><strong>Copyright: </strong> © 2014 Zha et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the National 973 Key Basic Research Program (2013CB910500); the National Natural Science Foundation of China (81125016, 81071637, and 91029728); The Key Specialized Project for the Infectious Diseases (2012ZX10002-009013); Shanghai Science and Technology Commission (11XD1404500); Shanghai Municipal Education Commission and Shanghai Municipal Health Bureau (11SG18, XBR2011039, and 20114Y159). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Worldwide, hepatocellular carcinoma (HCC) is the fifth most common malignancy in men, the seventh most common malignancy in women, and the third leading cause of cancer deaths, affecting over 500,000 people <a href="#pone.0087665-ElSerag1" class="ref-tip">[1]</a>. HCC usually develops as a consequence of underlying liver disease and is most often associated with cirrhosis. Additionally, metastasis or recurrence commonly causes mortality in patients <a href="#pone.0087665-Aravalli1" class="ref-tip">[2]</a>. Metastasis is a complex and multistep biological process that includes local invasion, intravasation, survival in the circulation, extravasation, arrest at a distant organ site, micrometastasis formation and metastatic colonization <a href="#pone.0087665-Valastyan1" class="ref-tip">[3]</a>, <a href="#pone.0087665-Steeg1" class="ref-tip">[4]</a>. Each step requires the coordinated spatial and temporal expression of genes and proteins. Many molecules contribute to this process, and some of these molecules are involved in the mechanical aspects, whereas others modulate regulatory pathways <a href="#pone.0087665-Valastyan1" class="ref-tip">[3]</a>. However, the molecular mechanism of metastasis has not been comprehensively elucidated.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>MicroRNAs (miRNAs) are small, single-stranded, non-coding RNAs that post- transcriptionally regulate the expression of target genes, usually by binding to the 3′-UTR of the respective mRNAs <a href="#pone.0087665-Ebert1" class="ref-tip">[5]</a>. They are involved in many physiological and pathological processes, such as embryonic development, cell differentiation, tumorigenesis and cancer metastasis <a href="#pone.0087665-Pauli1" class="ref-tip">[6]</a>, <a href="#pone.0087665-Garofalo1" class="ref-tip">[7]</a>, <a href="#pone.0087665-Li1" class="ref-tip">[8]</a>, <a href="#pone.0087665-Huang1" class="ref-tip">[9]</a>. Many previous studies have demonstrated that miRNAs play important roles in cell migration and the metastasis of several tumor types, such as breast cancer, glioblastoma, colon cancer and hepatocellular carcinoma <a href="#pone.0087665-Baranwal1" class="ref-tip">[10]</a>, <a href="#pone.0087665-Le1" class="ref-tip">[11]</a>, <a href="#pone.0087665-Huang2" class="ref-tip">[12]</a>, <a href="#pone.0087665-Ding1" class="ref-tip">[13]</a>, <a href="#pone.0087665-Ying1" class="ref-tip">[14]</a>, <a href="#pone.0087665-Yao1" class="ref-tip">[15]</a>, <a href="#pone.0087665-Tian1" class="ref-tip">[16]</a>. However, the function and mechanisms of miRNA-mediated regulation of HCC metastasis are largely unknown.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Here, we performed a large-scale screening to identify the miRNAs involved in HCC cell migration by classical wound-healing assays with time-lapse video microscopy. After independent validation with transwell migration assays, we found that 11 miRNAs regulated the migration of HCC cells <em>in vitro</em>. MicroRNA-134, the candidate tumor metastasis suppressor, is often downregulated in HCC <a href="#pone.0087665-Li2" class="ref-tip">[17]</a> and significantly inhibited invasion and metastasis by targeting the ITGB1 pathway in HCC.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Ethics statement"></a>
<h3>Ethics statement</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>All human materials were obtained with written informed consent, and the protocols were approved by the Ethical Review Committee of the World Health Organization of the Collaborating Center for Research in Human Production and authorized by the Shanghai Municipal Government. The samples used in this study also have been described in a previous study <a href="#pone.0087665-Ding2" class="ref-tip">[18]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>All experiments involving animals were performed in accordance with protocols approved by the Shanghai Medical Experimental Animal Care Commission.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Oligonucleotides"></a>
<h3>Oligonucleotides</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>All human miRNA mimics were obtained from GenePharma (Shanghai, China), and detailed information is listed in <a href="#pone.0087665.s001">Table S1</a>. The inhibitor (2′-<em>O</em>-methyl modification) of miR-134 was synthesized by Ribobio (Guangzhou, China).</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Cell culture and transfection"></a>
<h3>Cell culture and transfection</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>SK-HEP-1 and Huh-7 cells were purchased from ATCC, and the MHCC-LM3 cell line was a gift from Zhongshan Hospital, Shanghai, China. SK-HEP-1, Huh-7, and MHCC-LM3 cells were cultured in DMEM supplemented with 10% fetal bovine serum (Life Technologies, CA, USA), 100 IU/ml penicillin and 100 µg/ml streptomycin under a 5% CO<sub>2</sub> atmosphere at 37°C. MicroRNA mimics were transfected using Lipofectamine RNAiMAX (Life Technologies) at a final concentration of 50 nM, and plasmid transfection was performed using Lipofectamine 2000 according to the manufacturer's instructions.</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Wound-healing assay combined with time-lapse video microscopy"></a>
<h3>Wound-healing assay combined with time-lapse video microscopy</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>For wound-healing assays, SK-HEP-1 cells were transfected with miRNA mimics for 48 hours and then plated at 2×10<sup>5</sup> cells per well in 24-well plates with CytoSelect™ inserts (Cell Biolabs, INC. CA, USA). After 24 hours of incubation, the inserts were carefully removed from the wells, and the cells were washed once with growth medium and then imaged using a 10× objective on an Olympus IX81 motorized inverted microscope at 37°C under 5% CO<sub>2</sub>. Images of the wound-healing assays were acquired at 30-minute intervals for 24 hours. The percent closure of the cells into the wound field was measured using Image-Plus Pro software. The index of the inhibition of cell movement was calculated using the following equation: cell migration length in test miRNA condition/cell migration length in negative control condition×100%.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="In vitro migration and invasion assays"></a>
<h3><em>In vitro</em> migration and invasion assays</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>For the migration assay, 3×10<sup>4</sup> SK-HEP-1 cells or 5×10<sup>4</sup> Huh-7 cells were seeded in the top chamber of the insert (BD Biosciences, NJ, USA). For the invasion assay, 3×10<sup>4</sup> SK-HEP-1 cells or 1×10<sup>5</sup> Huh-7 cells were added to the upper chamber of the insert, which had previously been coated with 40 µl of Matrigel (BD Biosciences). After several hours of incubation at 37°C, cells that had migrated or invaded through the insert were fixed and stained in a dye solution containing 0.2% crystal violet and 20% methanol. The cells that had migrated or invaded through the insert were counted and imaged using an IX71 inverted microscope (Olympus Corp., Tokyo, Japan).</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Plasmid construction and lentiviral production"></a>
<h3>Plasmid construction and lentiviral production</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>The sequence of pre-miR-134 and the open reading frame sequence of ITGB1 were amplified and cloned into the pWPXL lentiviral vector (a gift from Dr. Didier Trono) and the pLVX lentiviral vector (TaKaRa, Tokyo, Japan), respectively. The wild-type or mutant 3′-UTR of ITGB1 was inserted (Xho I+Not I) downstream of the stop codon of Renilla luciferase in the psiCHECK2 vector (Promega). Lentiviral production was performed according to instructions supplied by Addgene (<a href="http://www.addgene.org">www.addgene.org</a>). SK-HEP-1, Huh-7 and LM3 cells were infected with recombinant lentivirus transducing units in the presence of 6 µg/ml polybrene (Sigma, MA, USA). The sequences of the primers used for PCR are listed in <a href="#pone.0087665.s002">Table S2</a>.</p>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="RNA extraction and quantitative real-time PCR"></a>
<h3>RNA extraction and quantitative real-time PCR</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>Total RNA was extracted using TRIzol reagent (Invitrogen, CA, USA). Reverse-transcribed complementary DNA was synthesized using the PrimeScript RT reagent kit (TaKaRa). Real-time polymerase chain reaction (PCR) was performed using SYBR Premix ExTaq (TaKaRa). For miRNA detection, mature miR-134 was reverse-transcribed with specific stem-loop primers and quantified using a TaqMan probe. Normalization was performed with U6 or RNU6B small nuclear (Life Technologies). The sequences of the primers used for PCR are listed in <a href="#pone.0087665.s002">Table S2</a>.</p>
</div>

<div id="section8" class="section toc-section"><a id="s2h" name="s2h" class="link-target" title="Human tissues"></a>
<h3>Human tissues</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>HCC liver tissues were collected from the surgical specimen cohort of the Qidong Liver Cancer Institute of Jiangsu Province and the first hospital of Zhejiang Province in China. HCC clinical information was collected from patient records, and the details are listed in <a href="#pone.0087665.s003">Table S3</a>.</p>
</div>

<div id="section9" class="section toc-section"><a id="s2i" name="s2i" class="link-target" title="Cell proliferation and colony formation assays"></a>
<h3>Cell proliferation and colony formation assays</h3>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1" class="link-target"></a><p>Cell proliferation was measured by the Cell Counting Kit-8 (CCK-8) assay (Dojindo Corp., Kumamoto, Japan) according to the manufacturer's protocol. For colony formation assays, 5×10<sup>2</sup> cells were plated into each well of a six-well plate and incubated at 37°C for 1 week. The cells were fixed and stained with 0.05% crystal violet in 20% methanol. Colonies were imaged and counted using an IX71 inverted microscope (Olympus Corp.). All samples were measured in triplicate, and three independent experiments were performed.</p>
</div>

<div id="section10" class="section toc-section"><a id="s2j" name="s2j" class="link-target" title="Animal study and histology"></a>
<h3>Animal study and histology</h3>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1" class="link-target"></a><p>For the <em>in vivo</em> orthotopic transplantation assay, 2×10<sup>6</sup> MHCC-LM3 cells stably expressing miR-134 or infected with the control vector were suspended in 40 µl of DMEM/Matrigel (1∶1) for each mouse. Through an 8-mm transverse incision in the upper abdomen, each nude mouse (10 per group, male BALB/c-nude mice) was inoculated orthotopically in the left hepatic lobe using a microsyringe. After 8 weeks, mice were euthanized, and the livers were dissected and fixed in 10% formalin solution for standard histological examination.</p>
</div>

<div id="section11" class="section toc-section"><a id="s2k" name="s2k" class="link-target" title="Microarray analysis"></a>
<h3>Microarray analysis</h3>
<a id="article1.body1.sec2.sec11.p1" name="article1.body1.sec2.sec11.p1" class="link-target"></a><p>SK-HEP-1 cells were transfected with the miR-134 mimic or negative control for 48 hours. RNA was extracted for gene expression profiling on an Affymetrix Human U133 Plus 2.0 chip. Microarray analysis of mRNA expression profile was performed as previously described <a href="#pone.0087665-Wu1" class="ref-tip">[19]</a>. Genes with 0.5-fold change were designated as downregulated in miR-134-overexpressing cells compared with the negative control cells. The data were submitted into ArrayExpress (accession: E-MTAB-2018).</p>
</div>

<div id="section12" class="section toc-section"><a id="s2l" name="s2l" class="link-target" title="Luciferase assay"></a>
<h3>Luciferase assay</h3>
<a id="article1.body1.sec2.sec12.p1" name="article1.body1.sec2.sec12.p1" class="link-target"></a><p>SK-HEP-1 and Huh-7 cells were cultured in 96-well plates and cotransfected with 20 ng of psiCHECK2 reporter and 5 pmol miR-134 mimic or negative control. After 48 hours, Renilla luciferase activity was measured using the dual luciferase reporter assay system (Promega). Renilla luminescence units were normalized to <em>Firefly</em> luciferase luminescence units,</p>
</div>

<div id="section13" class="section toc-section"><a id="s2m" name="s2m" class="link-target" title="Immunoblotting and Rho GTPase activation assays"></a>
<h3>Immunoblotting and Rho GTPase activation assays</h3>
<a id="article1.body1.sec2.sec13.p1" name="article1.body1.sec2.sec13.p1" class="link-target"></a><p>Proteins were separated by SDS-PAGE and transferred to a nitrocellulose membrane (Bio-Rad, CA, USA). The membrane was blocked with 5% non-fat milk and incubated with a rabbit anti-ITGB1 antibody (mAb) (Cell Signaling Technology, MA, USA) (1∶1,000), rabbit anti-FAK antibody (mAb) (1∶1,000), rabbit anti-phospho-FAK antibody (Tyr397) (1∶1,000) or mouse anti-GAPDH antibody (mAb) (1∶2,000) (Thermo Fisher Scientific, MA, USA). The proteins were detected using enhanced chemiluminescence reagents (Thermo Fisher Scientific). Activation of Rho-family GTPase was detected using the Rho and Rac1/Cdc42 activation assay kit (Millipore Corp., MA, USA) according to the manufacturer's instructions.</p>
</div>

<div id="section14" class="section toc-section"><a id="s2n" name="s2n" class="link-target" title="Immunofluorescence assay"></a>
<h3>Immunofluorescence assay</h3>
<a id="article1.body1.sec2.sec14.p1" name="article1.body1.sec2.sec14.p1" class="link-target"></a><p>SK-HEP-1 and Huh-7 cells were plated on glass slides and cultured for 48 hours. Cells were then fixed with 4% paraformaldehyde and permeabilized with 0.1% Triton-X 100, blocked in PBS containing 10% FBS for 1 hour at room temperature and incubated with a mouse monoclonal anti-α-tubulin (Sigma-Aldrich, MA, USA) or anti-vimentin (Dako, Glostrup, Denmark) antibodies overnight at 4°C. After three PBS washes, the slides were exposed to an Alexa Fluor® 488-conjugated secondary antibody (Life Technologies). F-actin and nuclei were stained with Alexa Fluor® 594 phalloidin and 4′, 6-diamidino-2-phenylindole dihydrochloride (DAPI), respectively. Cytoskeletal alterations were observed and documented using an Olympus FV1000 confocal microscope.</p>
</div>

<div id="section15" class="section toc-section"><a id="s2o" name="s2o" class="link-target" title="Cell-matrix adhesion assay"></a>
<h3>Cell-matrix adhesion assay</h3>
<a id="article1.body1.sec2.sec15.p1" name="article1.body1.sec2.sec15.p1" class="link-target"></a><p>Ninety-six-well plates were precoated with 20 mg/ml fibronectin (Sigma-Aldrich). SK-HEP-1 and Huh-7 cells were trypsinized and incubated in DMEM without serum for 45 or 30 minutes, respectively. The unattached cells were washed, and the attached cells were fixed with 70% ethanol and stained with 0.1% crystal violet in 20% ethanol. Then, the stained crystal violet was dissolved in 10% acetic acid, and the absorbance value (OD) was measured at 597 nm. Wells coated with bovine serum albumin served as a negative control, whereas cells that were incubated in complete culture medium for 8 hours served as a positive control. The cell matrix adhesion (CMA) index was calculated as the OD value (test-negative control)/OD value (positive control-negative control). All adhesion experiments were performed in triplicate wells and repeated at least three times.</p>
</div>

<div id="section16" class="section toc-section"><a id="s2p" name="s2p" class="link-target" title="Statistical analysis"></a>
<h3>Statistical analysis</h3>
<a id="article1.body1.sec2.sec16.p1" name="article1.body1.sec2.sec16.p1" class="link-target"></a><p>All results are presented as the mean ± SEM. The grouped data were analyzed by one-way ANOVA tests, and the differences between groups were assessed by the Bonferroni post-test following one-way ANOVA. The Mann-Whitney test was used for testing the difference between two group values. For tissue samples, differences between variables were assessed by the chi-square test or Fisher's exact test. P&lt;0.05 was considered statistically significant.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="Large-scale screening for miRNAs that regulate cancer cell migration"></a>
<h3>Large-scale screening for miRNAs that regulate cancer cell migration</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>To screen for the miRNAs regulating cell motility, we utilized time-lapse video microscopy in the wound-healing assay (<a href="#pone-0087665-g001">Figure 1A</a>). Representative videos of the wound region and cell morphology are shown in the Supplementary Videos (<a href="#pone.0087665.s006">Video S1</a>, <a href="#pone.0087665.s007">Video S2</a> and <a href="#pone.0087665.s008">Video S3</a>). Cells transfected with miR-891a migrated more quickly into the wound region than the negative control cells. In total, 646 miRNAs (miRBase 15.0, <a href="#pone.0087665.s001">Table S1</a>) were investigated. The primary screening indicated that 48 miRNAs regulated cell migration with an index value under −50% or over 40% (<a href="#pone-0087665-g001">Figure 1B</a> and <a href="#pone.0087665.s004">Table S4</a>).</p>
<a class="link-target" id="pone-0087665-g001" name="pone-0087665-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0087665.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0087665.g001" data-doi="info:doi/10.1371/journal.pone.0087665" data-uri="info:doi/10.1371/journal.pone.0087665.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0087665.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Large-scale screening for miRNAs that regulate cancer cell migration.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p><strong>A.</strong> Schematic diagram of the screening strategy. <strong>B.</strong> The results of the primary wound-healing screening. Each dot represents the inhibition index of an individual miRNA. The cutoff value was &gt;40% (inhibition) or &lt;−50% (promotion). <strong>C.</strong> The results of the second round of screening with administration of the proliferation inhibitor 2′, 3′- dideoxythymidine (DDT). <strong>D.</strong> Transwell migration assays for selected miRNAs (*p&lt;0.05). SK-HEP-1 cells were plated into each well of a 12-well plate and transfected with individual miRNA mimics. The number of cells per field was counted after cells were placed into the top chamber of each insert for 5.5 hours. The results were normalized against data from cells transfected with the negative control. Differences between two groups were analyzed by the Mann-Whitney test.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0087665.g001">
              https://doi.org/10.1371/journal.pone.0087665.g001</a></p></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>To eliminate the impact of cell multiplication, we repeated the wound-healing assay with the selected 48 miRNAs in the presence of the proliferation inhibitor 2′, 3′- dideoxythymidine (DDT); the movement induced by transfection with 22 miRNAs in the presence of DDT was similar to that in the primary screening (<a href="#pone-0087665-g001">Figure 1C</a> and <a href="#pone.0087665.s004">Table S4</a>).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>To further confirm this result, we performed transwell migration assays in SK-HEP-1 cells transfected with the 22 miRNAs for 48 hours. Finally, 11 miRNAs (miR-134, -146b-3p, -188-3p, -525-3p, -661, -767-5p, -891a, -891b, -1244, -1247 and miR-1471) of the 22 miRNAs demonstrated consistent regulatory behavior. (<a href="#pone-0087665-g001">Figure 1D</a> and <a href="#pone.0087665.s004">Table S4</a>).</p>
</div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="miR-134 is inversely correlated with HCC lymphatic invasion and vascular invasion"></a>
<h3>miR-134 is inversely correlated with HCC lymphatic invasion and vascular invasion</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>Our previous work revealed that miR-134 is often downregulated in HCC tissues <a href="#pone.0087665-Li2" class="ref-tip">[17]</a>. Furthermore, we investigated the correlation between the miR-134 expression level and the clinical or pathological variables in two groups (cutoff value of median), and found that miR-134 expression is inversely correlated with lymphatic invasion (p = 0.04) and vascular invasion (p = 0.04) in HCC (<a href="#pone-0087665-t001">Table 1</a>). However, no relationship was found between miR-134 expression and tumor size or multinodularity.</p>
<a class="link-target" id="pone-0087665-t001" name="pone-0087665-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0087665.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0087665.t001" data-doi="info:doi/10.1371/journal.pone.0087665" data-uri="info:doi/10.1371/journal.pone.0087665.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0087665.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Correlation of clinicopathological features with tumor miR-134 expression in HCC.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0087665.t001">
              https://doi.org/10.1371/journal.pone.0087665.t001</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="miR-134 inhibits HCC cell invasion and metastasis in vitro and in vivo"></a>
<h3>miR-134 inhibits HCC cell invasion and metastasis <em>in vitro</em> and <em>in vivo</em></h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>To determine whether the enhanced expression of miR-134 inhibits HCC cell invasion and metastasis, we first measured the expression of miR-134 in various HCC cell lines and constructed three miR-134-overexpressing stable cell lines by lentiviral transduction (Figure S1A in <a href="#pone.0087665.s005">File S1</a>). Cell proliferation and colony formation assays revealed that the ectopic expression of miR-134 had no obvious effects on HCC cell proliferation <em>in vitro</em> (Figure S1B–D in <a href="#pone.0087665.s005">File S1</a>), while transwell experiments showed that the cells with stable miR-134-overexpression, which were derived from SK-HEP-1, MHCC-LM3 and Huh-7 cells, exhibited a lower migratory and invasive potential than the control cells (<a href="#pone-0087665-g002">Figure 2A, 2B</a> and Figure S2 in <a href="#pone.0087665.s005">File S1</a>). In contrast, the suppression of endogenous miR-134 by its specific inhibitor enhanced the migratory and invasive abilities in Huh-7 cells (<a href="#pone-0087665-g002">Figure 2C</a>). To further explore the role of miR-134 in tumor invasion and metastasis <em>in vivo</em>, MHCC-LM3 cells, which possess a relatively strong <em>in vivo</em> metastatic capacity, were orthotopically transplanted into the livers of nude mice. The number of intrahepatic nodules in the liver was dramatically decreased in the miR-134 groups compared to the vector controls after <em>in situ</em> growth for 8 weeks (<a href="#pone-0087665-g002">Figure 2D</a>). Together, these findings suggest that miR-134 is a negative regulator of metastasis in HCC.</p>
<a class="link-target" id="pone-0087665-g002" name="pone-0087665-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0087665.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0087665.g002" data-doi="info:doi/10.1371/journal.pone.0087665" data-uri="info:doi/10.1371/journal.pone.0087665.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0087665.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> miR-134 inhibits HCC cell migration and invasion <em>in vitro</em> and <em>in vivo</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p><strong>A</strong> and <strong>B.</strong> Transwell migration (<strong>A</strong>) and invasion (<strong>B</strong>) assays for SK-HEP-1 and Huh-7 cells stably overexpressing miR-134 or negative control (lenti-vector). Representative images are shown with the quantification of five randomly selected fields. <strong>C.</strong> Transwell migration and invasion assays of Huh-7 cells were performed after transfection with the miR-134 inhibitor or negative control. Differences between two groups were analyzed by the Mann-Whitney test. <strong>D.</strong> H &amp; E staining of liver isolated from mice that received orthotopic injections of miR-134-transduced or vector-infected LM3 cells at 8 weeks after transplantation. Intrahepatic nodules were counted and analyzed using the Mann-Whitney test (n = 10 mice per group).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0087665.g002">
              https://doi.org/10.1371/journal.pone.0087665.g002</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="miR-134 impacts ITGB1 expression by directly binding to its 3′-UTR"></a>
<h3>miR-134 impacts ITGB1 expression by directly binding to its 3′-UTR</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Generally, miRNAs exert their function by binding to the 3′-UTR of their target genes, blocking subsequent protein expression. Integrin beta 1 (ITGB1) and cadherin 9 (CDH9) were predicted to be the target genes of miR-134 by combining three target-predicting algorithms (PicTar, <a href="http://www.pictar.org">www.pictar.org</a>, TargetScan, <a href="http://www.targetscan.org">www.targetscan.org</a> and miRanda, <a href="http://www.microrna.org">www.microrna.org</a>). In addition, we performed an RNA microarray assay in SK-HEP-1 cells, and the results showed that the ITGB1 mRNA expression level decreased by 66% in miR-134 overexpressing cells compared with control cells (the expression ratio was 0.341524), whereas CDH9 mRNA expression levels remained relatively stable. These results indicate that ITGB1 is a putative target gene of miR-134 (<a href="#pone-0087665-g003">Figure 3A</a>).</p>
<a class="link-target" id="pone-0087665-g003" name="pone-0087665-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0087665.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0087665.g003" data-doi="info:doi/10.1371/journal.pone.0087665" data-uri="info:doi/10.1371/journal.pone.0087665.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0087665.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> miR-134 downregulates ITGB1 by directly targeting its 3′UTR.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1" class="link-target"></a><p><strong>A.</strong> Schema of the candidate genes identified by four independent approaches. <strong>B.</strong> Putative binding sequences of ITGB1 for miR-134 in the human (Hsa), chimpanzee (Ptr), rhesus (Mml) and mouse (Mmu). The ITGB1 seed sequences are highlighted by ruby italic letters. <strong>C.</strong> Putative miR-134-binding sites in the wild-type ITGB1 3′-UTR (highlighted by ruby letters) or 3′-UTR segments containing mutant (highlighted by blue letters) binding sites are shown. <strong>D.</strong> Relative luciferase activity assays for constructs containing luciferase and wild-type or mutant ITGB1 3′-UTRs were performed after cotransfection with a miR-134 mimic or negative control. The <em>Renilla</em> luciferase vector was cotransfected as an internal control. Data were analyzed by one-way ANOVA (p&lt;0.01), and differences between two groups were assessed by the Bonferroni post-test. <strong>E.</strong> The mRNA expression levels of ITGB1 in SK-HEP-1 and Huh-7 cells overexpressing miR-134 or negative control. Differences between two groups were analyzed by the Mann-Whitney test. <strong>F.</strong> The protein levels of ITGB1 in SK-HEP-1 and Huh-7 cells infected with miR-134 or control lentivirus. GAPDH served as an internal control. <strong>G.</strong> Protein levels of ITGB1 in Huh-7 cells after transfection with the miR-134 inhibitor or inhibitor control. GAPDH served as an internal control.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0087665.g003">
              https://doi.org/10.1371/journal.pone.0087665.g003</a></p></div><a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2" class="link-target"></a><p>Next, analysis of the 3′-UTR sequence of ITGB1 using TargetScan revealed two possible binding sites of miR-134, both of which are highly conserved in humans, chimpanzees, rhesus monkeys and mice (<a href="#pone-0087665-g003">Figure 3B</a>). To further determine the direct binding of miR-134 to the 3′-UTR, the fragments of the ITPG1 3′-UTR (either wild-type or mutant, <a href="#pone-0087665-g003">Figure 3C</a>) were constructed and inserted downstream of the stop codon of Renilla luciferase in the psiCHECK2 vector. MiR-134 was cotransfected with different 3′-UTR constructs into SK-HEP-1 cells. Compared to the control, the relative luciferase activity of wild-type 3′-UTR of ITGB1 was sharply decreased by miR-134, whereas the relative luciferase activity of the 3′-UTR construct containing two mutant binding sites was not altered. In addition, our results indicate that the two binding sites act synergically in miR-134 regulation because single mutation of these two binding sites only partially restored the luciferase activity (<a href="#pone-0087665-g003">Figure 3D</a>). Similar results were observed when different 3′-UTRs constructed with psiCHECK2 were transfected into Huh-7 cells with the miR-134 mimic (Figure S3 in <a href="#pone.0087665.s005">File S1</a>). Furthermore, overexpression of miR-134 remarkably decreased the endogenous mRNA and protein levels of ITGB1 (<a href="#pone-0087665-g003">Figure 3E, 3F</a>). Conversely, inhibition of miR-134 increased the expression of ITGB1 in Huh-7 cells (<a href="#pone-0087665-g003">Figure 3G</a>). Collectively, these data provide further evidence that ITGB1 is a direct target of miR-134.</p>
</div>

<div id="section5" class="section toc-section"><a id="s3e" name="s3e" class="link-target" title="Restoration of ITGB1 abrogates miR-134-repressed HCC cell migration and invasion"></a>
<h3>Restoration of ITGB1 abrogates miR-134-repressed HCC cell migration and invasion</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>Previous studies showed that the mRNA level of ITGB1 was upregulated in HCC samples compared to the adjacent normal liver tissue <a href="#pone.0087665-Liu1" class="ref-tip">[20]</a>. To explore whether ITGB1 contributes to the suppressive effect of miR-134, the complete ORF of ITGB1 without its 3′-UTR, which resulted in the constitutive expression of ITGB1 resistant to miR-134 inhibition, was introduced into SK-HEP-1 and Huh-7 cells via lentiviral transduction. Remarkably, overexpression of ITGB1 strongly increased SK-HEP-1 and Huh-7 cell migration and invasion (<a href="#pone-0087665-g004">Figure 4A and 4B</a>). We further observed that ectopic ITGB1-ORF without 3′UTR in miR-134-transduced SK-HEP-1 and Hun-7 cells abrogated the inhibitory effect of miR-134 on HCC migration and invasion (<a href="#pone-0087665-g004">Figure 4C and 4D</a>). Together, these findings provide evidence that miR-134 suppresses HCC cell invasion and metastasis by directly targeting ITGB1.</p>
<a class="link-target" id="pone-0087665-g004" name="pone-0087665-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0087665.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0087665.g004" data-doi="info:doi/10.1371/journal.pone.0087665" data-uri="info:doi/10.1371/journal.pone.0087665.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0087665.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Restoration of ITGB1 expression rescues miR-134-induced suppression of HCC migration and invasion.</div><p class="caption_target"><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1" class="link-target"></a><p><strong>A</strong> and <strong>B.</strong> Transwell migration assays of (A) SK-HEP-1 and (B) Huh-7 cells stably overexpressing ITGB1. Representative images are shown with the quantification of five randomly selected fields. The bars represent the SEM of three independent experiments. Differences between two groups were analyzed by the Mann-Whitney test. Western blot analysis revealed that the protein expression of ITGB1 increased after infection with the PLVX-ITGB1 (ORF) virus. <strong>C.</strong> Transwell migration and invasion assays for SK-HEP-1-vector or SK-HEP-1- miR-134 cells were performed after infection with ITGB1 (ORF) or the vector control lentivirus. Transwell migration and invasion data were analyzed by a one-way ANOVA (p&lt;0.01, p&lt;0.01 respectively); differences between two groups were assessed by the Bonferroni post-test. ITGB1 protein levels were determined by Western blot analysis. <strong>D.</strong> Transwell migration and invasion assays for Huh-7-vector or Huh-7-miR-134 cells were performed after infection with ITGB1 (ORF) or vector control lentivirus. ITGB1 protein levels were determined by Western blot analysis. Transwell migration and invasion data were analyzed by a one-way ANOVA (p&lt;0.01, p&lt;0.01 respectively); differences between two groups were assessed by the Bonferroni post-test.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0087665.g004">
              https://doi.org/10.1371/journal.pone.0087665.g004</a></p></div></div>

<div id="section6" class="section toc-section"><a id="s3f" name="s3f" class="link-target" title="Overexpression of miR-134 represses HCC cell stress fiber formation and cell adhesion"></a>
<h3>Overexpression of miR-134 represses HCC cell stress fiber formation and cell adhesion</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>To further study the role of miR-134 in cell motility, we examined the status of stress fiber formation and polymerized actin in SK-HEP-1 and Huh-7 cells with miR-134 overexpression. Using phalloidin staining, we found that the maturation of stress fibers was suppressed in cells transfected with the miR-134 mimic (<a href="#pone-0087665-g005">Figure 5A and 5C</a>). However, miR-134 did not affect the consistency of vimentin or tubulin, which implies that miR-134 does not interfere the formation of intermediate filaments (Figure S4 in <a href="#pone.0087665.s005">File S1</a>). Previous work has shown that phosphorylated focal adhesion kinase (FAK) activates RhoA and drives the formation of stress fibers and the maturation of cell adhesion <a href="#pone.0087665-Huveneers1" class="ref-tip">[21]</a>; thus, we determined the status of the FAK signaling pathway, including the phosphorylation of FAK and downstream activation of Rho, Rac1 and Cdc42. Western blotting showed that miR-134 attenuated FAK phosphorylation and suppressed the activation of RhoA in SK-HEP-1 and Huh-7 cells (<a href="#pone-0087665-g005">Figure 5B and 5D</a>). Moreover, cell adhesion assays also revealed that miR-134 inhibited cell-matrix adhesion in SK-HEP-1 and Huh-7 cells (<a href="#pone-0087665-g005">Figure 5E</a>).</p>
<a class="link-target" id="pone-0087665-g005" name="pone-0087665-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0087665.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0087665.g005" data-doi="info:doi/10.1371/journal.pone.0087665" data-uri="info:doi/10.1371/journal.pone.0087665.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0087665.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> Ectopic expression of miR-134 impairs the stress fiber network in HCC cells.</div><p class="caption_target"><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1" class="link-target"></a><p><strong>A.</strong> and <strong>C.</strong> Ectopic expression of miR-134 disrupted the stress fiber network in SK-HEP-1 (<strong>A</strong>) and Huh-7 cells (<strong>C</strong>). Stress fibers (polymerized actin) and actin filaments were detected by phalloidin staining. DAPI staining was used to detect nuclei. Scale bar: 20 µm. <strong>B.</strong> and <strong>D.</strong> The protein levels of FAK phosphorylation and activity of Rho/Rac1/Cdc42 GTPases in SK-HEP-1 (<strong>B</strong>) and Huh-7 (<strong>D</strong>) cells. Samples for the Western blot were analyzed on a single gel. <strong>E.</strong> Cellular adhesion assays were performed in SK-HEP-1 and Huh-7 cells. The results are representative of at least three independent experiments. Differences between two groups were analyzed by the Mann-Whitney test. NC: negative control, FN: fibronectin.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0087665.g005">
              https://doi.org/10.1371/journal.pone.0087665.g005</a></p></div><a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2" class="link-target"></a><p>In addition, ectopic ITGB1 abrogated miR-134-suppressed cell stress fiber formation. Nevertheless, this abrogation was reversed by siRNA knockdown of RhoA (<a href="#pone-0087665-g006">Figure 6A, 6B</a>, Figure S5A and S5B in <a href="#pone.0087665.s005">File S1</a>). Similarly, reintroduction of ITGB1 rescued miR-134-suppressed cell adhesion, and this effect was abrogated by knockdown of RhoA (<a href="#pone-0087665-g006">Figure 6C</a> and Figure S5C in <a href="#pone.0087665.s005">File S1</a>). Taken together, these data suggest that ITGB1 plays a pivotal role in miR-134-mediated cell stress fiber formation and cell adhesion.</p>
<a class="link-target" id="pone-0087665-g006" name="pone-0087665-g006"></a><div class="figure" data-doi="10.1371/journal.pone.0087665.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0087665.g006" data-doi="info:doi/10.1371/journal.pone.0087665" data-uri="info:doi/10.1371/journal.pone.0087665.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0087665.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0087665.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0087665.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0087665.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 6. </span> ITGB1 is involved in the miR-134-induced inhibition of cell stress fiber formation and cell adhesion.</div><p class="caption_target"><a id="article1.body1.sec3.sec6.fig2.caption1.p1" name="article1.body1.sec3.sec6.fig2.caption1.p1" class="link-target"></a><p><strong>A.</strong> Ectopic expression of miR-134 disrupted the stress fiber network in SK-HEP-1 cells, and this effect was reversed by ITGB1 transfection. Stress fiber formation was repressed by transfecting siRhoA into SK-HEP-1-ITGB1 cells. F-actin was used as the stress fiber marker. DAPI staining was used to detect nuclei. Scale bar: 20 µm. <strong>B.</strong> FAK protein phosphorylation and the protein levels of the Rho GTPases in SK-HEP-1-ITGB1 cells were analyzed after transfection with miR-134 and siRhoA. <strong>C.</strong> The cellular adhesion assay was performed in SK-HEP-1 cells. The results are representative of at least three independent experiments, and data were analyzed by a one-way ANOVA (p&lt;0.05); differences between two groups were assessed by the Bonferroni post-test. NC: negative control, FN: fibronectin.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0087665.g006">
              https://doi.org/10.1371/journal.pone.0087665.g006</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>Previous studies have shown that time-lapse imaging of living cells is versatile for investigating cell mobility. This technique allowed us to investigate many phenotypes and morphologies associated with cell movement and mobility <a href="#pone.0087665-Simpson1" class="ref-tip">[22]</a>, <a href="#pone.0087665-Fotos1" class="ref-tip">[23]</a>. In this study, we used time-lapse video microscopy to observe cell movement in the scratch-wound assay, validated the results using the transwell migration assay, and identified the migration-suppressing miRNA (miR-134) and migration-facilitating miRNAs (miR-1247, miR-1244, miR-146b-3p, miR-1471, miR-188-3p, miR-661, miR-891a, miR-891b and miR-767-5P) in SK-HEP-1 cells.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>miR-134 is located on chromosome 14q32.31, which is frequently lost in gastrointestinal stromal tumors (GISTs) <a href="#pone.0087665-Haller1" class="ref-tip">[24]</a>, and its expression is often decreased in GIST, osteosarcoma, glioma and HCC <a href="#pone.0087665-Li2" class="ref-tip">[17]</a>, <a href="#pone.0087665-Haller1" class="ref-tip">[24]</a>, <a href="#pone.0087665-Sarver1" class="ref-tip">[25]</a>, <a href="#pone.0087665-Lages1" class="ref-tip">[26]</a>. Boominathan reported that TA-p73 may regulate the post-transcriptional processing of miR-134 by increasing Ago1/2 activity, thereby repressing cancer stem cell proliferation <a href="#pone.0087665-Boominathan1" class="ref-tip">[27]</a>. However, the regulation of miR-134 expression in HCC remains to be elucidated. Moreover, low expression of miR-134 is significantly associated with shorter disease-free survival or tumor progression in GIST. Additionally, miR-134 inhibits cell migration and EMT in A549 cells <a href="#pone.0087665-Zhang1" class="ref-tip">[28]</a>, <a href="#pone.0087665-Li3" class="ref-tip">[29]</a>. Our previous work also demonstrated that miR-134 is often downregulated in HCC tissues, and its expression level is reversely correlated with HCC lymphatic invasion and vascular invasion. We also verified that ectopic expression of miR-134 suppresses HCC cell invasion and metastasis both <em>in vitro</em> and <em>in vivo</em>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>The present work shows that miR-134 participates in the integrin-signaling pathway by directly regulating ITGB1 expression. ITGB1, which is a subunit of a family of heterodimeric transmembrane receptors, can combine with different integrin α subunits to form heterodimers and bind a large number of extracellular ligands. For example, α1β1, α2β1, α10β1 and α11β1 are collagen-binding integrins; α3β1, α6β1 and α7β1 bind to laminin and α4β1, α5β1, α8β1 and αvβ1 bind to fibronectin <a href="#pone.0087665-Brakebusch1" class="ref-tip">[30]</a>. In many cancer types, integrin beta 1 promotes cancer cell proliferation and survival and regulates cell focal adhesion and tumor metastasis <a href="#pone.0087665-FeldingHabermann1" class="ref-tip">[31]</a>, <a href="#pone.0087665-Caccavari1" class="ref-tip">[32]</a>, <a href="#pone.0087665-Allen1" class="ref-tip">[33]</a>, <a href="#pone.0087665-Grzesiak1" class="ref-tip">[34]</a>. Masumoto reported that a monoclonal antibody against ITGB1 blocked HCC cell invasion <a href="#pone.0087665-Masumoto1" class="ref-tip">[35]</a>, and Mizuno found that overexpression of α3β1 integrins enhanced HepG2 cell migration and invasion <a href="#pone.0087665-Mizuno1" class="ref-tip">[36]</a>, while inhibition of α3β1 integrins suppressed SMMC-7721 cell migration <a href="#pone.0087665-Fu1" class="ref-tip">[37]</a>. Our study showed that ITGB1 promotes HCC cell migration and invasion and rescues miR-134-induced repression of cell migration and invasion. In addition, ITGB1 mRNA was upregulated in HCC tissues compared with the adjacent normal liver tissue <a href="#pone.0087665-Liu1" class="ref-tip">[20]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>When extracellular ligands bind to integrins, their cytoplasmic domains connect to intracellular signaling molecules to activate an immediate signal transduction pathway or crosstalk with growth factor receptors. Then, FAK, the key molecule of the integrin-signaling pathway, is recruited to the integrin C-terminal domain and autophosphorylated at tyrosine397, thereby creating a binding site for the Src-homology 2 (SH2) domain of Src. The active FAK-Src complex then promotes the activation of the small Rho family members, Rac1, Cdc42 and RhoA <a href="#pone.0087665-Huveneers1" class="ref-tip">[21]</a>. RhoA is a crucial molecule in cell cytoskeleton remodeling and actin reorganization because it activates myosin II through Rho-kinase (ROCK) and contributes to actin polymerization in the formation of stress fibers <a href="#pone.0087665-Rottner1" class="ref-tip">[38]</a>. In addition to the inhibitory effect of miR-134 on ITGB1 expression, our data demonstrated that miR-134 affects FAK phosphorylation and RhoA activation in HCC cells. GTP-Rho is decreased and the formation of stress fibers is inhibited by overexpression of miR-134. Overexpression of ITGB1 could reverse the inhibitory effect of miR-134 by increasing of GTP-Rho. When we knockdown RhoA, GTP-Rho decreases and the formation of stress fibers is suppressed.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>Cell migration is frequently composed of the cycles consisting of protrusion at the front, the formation and stabilization of adhesions at the front and then the retraction and destabilization of adhesions at the rear <a href="#pone.0087665-Rottner1" class="ref-tip">[38]</a>. Protrusion may be mediated by a variety of distinct structures, including lamellipodia, filopodia and stress fibers. Rac1 stimulates lamellipodium formation, Cdc42 induces filopodium formation and RhoA induces the formation of stress fibers <a href="#pone.0087665-Kaibuchi1" class="ref-tip">[39]</a>. As miR-134 could reduce ITGB1 expression and the consequent FAK phosphorylation and RhoA activation, it is not surprising to find that miR-134 inhibits cell adhesion and migration in HCC.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6" class="link-target"></a><p>In conclusion, we identified potential tumor metastasis-regulating miRNAs through a cell migration assay. Our results show that miR-134 significantly inhibits HCC cell invasion and metastasis <em>in vitro</em> and <em>in vivo</em> and further highlights the critical role of ITGB1 and its downstream pathway in miR-134-mediated inhibition of metastasis in HCC. This work suggests that miR-134 could be a novel therapeutic target for the treatment of HCC.</p>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0087665"></div><div class="supplementary-material"><a name="pone.0087665.s001" id="pone.0087665.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s001">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>List of human miRNAs tested in the study.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s001">https://doi.org/10.1371/journal.pone.0087665.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0087665.s002" id="pone.0087665.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s002">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Primer sequences for real time PCR and constructs.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s002">https://doi.org/10.1371/journal.pone.0087665.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0087665.s003" id="pone.0087665.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s003">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Clinical information of human hepatocellular carcinoma samples used in the study.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s003">https://doi.org/10.1371/journal.pone.0087665.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0087665.s004" id="pone.0087665.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s004">Table S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>List of miRNAs capable of regulating HCC cell migration.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s004">https://doi.org/10.1371/journal.pone.0087665.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLS)</p>
</div><div class="supplementary-material"><a name="pone.0087665.s005" id="pone.0087665.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s005">File S1. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Supporting information on results obtained, containing Figure S1, S2, S3, S4 and S5.</strong> Figure S1, Overexpression of miR-134 has no effect on HCC cell growth. <strong>A.</strong> the relative expression levels of miR-134 in HCC cells. The expression levels of mature miR-134 are normalized by U6 small nuclear RNA. The CCK-8 assays of SK-HEP-1 (<strong>B</strong>) and Huh-7 (<strong>C</strong>) cells were performed after infected with lentivirus expressing miR-134 or the control vector. <strong>D.</strong> The colony formation assay for SK-HEP-1 cells infected with lentivirus expressing miR-134 or the control vector. A total of 500 cells per well were seeded and cultivated for 2 weeks. The colonies were fixed and stained in a dye solution containing 0.1% crystal violet and 20% methanol. Figure S2, Overexpression of miR-134 inhibits MHCC-LM3 cell migration and invasion. Transwell migration assays of MHCC-LM3 cells were performed after infection with lentivirus expressing miR-134 or the control vector. For the migration assay, 10<sup>5</sup> cells were placed into the top chamber of the insert and cultured for 28 hours. For the invasion assay, 2*10<sup>5</sup> cells were added to the upper chamber of the insert which had previously been coated with 40 µl of Matrigel and cultured for 48 hours. Cells that had migrated through the pore were fixed and stained with a mixture of 20% methanol and 0.1% crystal violet for 0.5 hour. Differences between two groups were analyzed by the Mann-Whitney test. Figure S3, miR-134 directly targets 3′-UTR of ITGB1 The psiCHECK2 vector, psiCHECK2-wild-ITGB1-3′UTR or psiCHECK2-mutant- ITGB1-3′UTR was transfected into Huh-7 cells with miR-134 mimic or negative control. Data were analyzed by one-way ANOVA (p&lt;0.01), and differences between two groups were assessed by the Bonferroni post-test. Figure S4, Overexpression of miR-134 has no effect on microtubule and intermediate filament in HCC cells. Microtubules were demonstrated by α-tubulin staining using α-tubulin mAb in SK-HEP-1 (<strong>A</strong>) and Huh-7 (<strong>B</strong>) cells. DAPI staining is used to detect nuclei. Scale bar: 20 µm. Intermediate filaments were demonstrated by Vimentin using Vimentin mAb in SK-HEP-1 (<strong>C</strong>) and Huh-7 (<strong>D</strong>) cells. DAPI staining is used to detect nuclei. Scale bar: 20 µm. Figure S5, ITGB1 contribute to the inhibition of cell stress fiber formation and cell adhesion by miR-134. <strong>A.</strong> Ectopic expression of miR-134 disrupted the stress fiber network in Huh-7 cells, and this effect was reversed by ITGB1 transfection. Stress fiber formation was repressed by transfecting siRhoA into Huh-7-ITGB1 cells. F-actin was used as the stress fiber marker. DAPI staining was used to detect nuclei. <strong>B.</strong> FAK protein phosphorylation and the protein levels of the Rho GTPases in Huh-7-ITGB1 cells were analyzed after transfection with miR-134 and siRhoA. <strong>C.</strong> The cellular adhesion assay was performed in Huh-7 cells. The results were representative of at least three independent experiments, and data were analyzed by a one-way ANOVA (p&lt;0.01); differences between two groups were assessed by the Bonferroni post-test.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s005">https://doi.org/10.1371/journal.pone.0087665.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0087665.s006" id="pone.0087665.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s006">Video S1. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Cell migration after treatment with negative control.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s006">https://doi.org/10.1371/journal.pone.0087665.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(MP4)</p>
</div><div class="supplementary-material"><a name="pone.0087665.s007" id="pone.0087665.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s007">Video S2. </a></h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Cell migration after treatment with miR-891a.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s007">https://doi.org/10.1371/journal.pone.0087665.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(MP4)</p>
</div><div class="supplementary-material"><a name="pone.0087665.s008" id="pone.0087665.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0087665.s008">Video S3. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Cell migration after treatment with miR-134.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0087665.s008">https://doi.org/10.1371/journal.pone.0087665.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(MP4)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank Professor Didier Trono for kindly providing the pWPXL, psPAX2 and pMD2.G lentivirus plasmids.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: XH RZ. Performed the experiments: RZ. Analyzed the data: RZ WG ZZ. Contributed reagents/materials/analysis tools: ZQ QW JD SH TC JG MY. Wrote the paper: RZ.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0087665-ElSerag1" id="pone.0087665-ElSerag1" class="link-target"></a>El-Serag HB (2011) Hepatocellular Carcinoma. New England Journal of Medicine 365: 1118–1127. <ul class="reflinks"><li><a href="#" data-author="El-Serag" data-cit="El-SeragHB%20%282011%29%20Hepatocellular%20Carcinoma.%20New%20England%20Journal%20of%20Medicine%20365%3A%201118%E2%80%931127." data-title="Hepatocellular%20Carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hepatocellular+Carcinoma+El-Serag+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0087665-Aravalli1" id="pone.0087665-Aravalli1" class="link-target"></a>Aravalli RN, Steer CJ, Cressman EN (2008) Molecular mechanisms of hepatocellular carcinoma. Hepatology 48: 2047–2063. <ul class="reflinks"><li><a href="#" data-author="Aravalli" data-cit="AravalliRN%2C%20SteerCJ%2C%20CressmanEN%20%282008%29%20Molecular%20mechanisms%20of%20hepatocellular%20carcinoma.%20Hepatology%2048%3A%202047%E2%80%932063." data-title="Molecular%20mechanisms%20of%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+mechanisms+of+hepatocellular+carcinoma+Aravalli+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0087665-Valastyan1" id="pone.0087665-Valastyan1" class="link-target"></a>Valastyan S, Weinberg RA (2011) Tumor metastasis: molecular insights and evolving paradigms. Cell 147: 275–292. <ul class="reflinks"><li><a href="#" data-author="Valastyan" data-cit="ValastyanS%2C%20WeinbergRA%20%282011%29%20Tumor%20metastasis%3A%20molecular%20insights%20and%20evolving%20paradigms.%20Cell%20147%3A%20275%E2%80%93292." data-title="Tumor%20metastasis%3A%20molecular%20insights%20and%20evolving%20paradigms" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor+metastasis%3A+molecular+insights+and+evolving+paradigms+Valastyan+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0087665-Steeg1" id="pone.0087665-Steeg1" class="link-target"></a>Steeg PS (2006) Tumor metastasis: mechanistic insights and clinical challenges. Nat Med 12: 895–904. <ul class="reflinks"><li><a href="#" data-author="Steeg" data-cit="SteegPS%20%282006%29%20Tumor%20metastasis%3A%20mechanistic%20insights%20and%20clinical%20challenges.%20Nat%20Med%2012%3A%20895%E2%80%93904." data-title="Tumor%20metastasis%3A%20mechanistic%20insights%20and%20clinical%20challenges" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor+metastasis%3A+mechanistic+insights+and+clinical+challenges+Steeg+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0087665-Ebert1" id="pone.0087665-Ebert1" class="link-target"></a>Ebert MS, Sharp PA (2012) Roles for microRNAs in conferring robustness to biological processes. Cell 149: 515–524. <ul class="reflinks"><li><a href="#" data-author="Ebert" data-cit="EbertMS%2C%20SharpPA%20%282012%29%20Roles%20for%20microRNAs%20in%20conferring%20robustness%20to%20biological%20processes.%20Cell%20149%3A%20515%E2%80%93524." data-title="Roles%20for%20microRNAs%20in%20conferring%20robustness%20to%20biological%20processes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Roles+for+microRNAs+in+conferring+robustness+to+biological+processes+Ebert+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0087665-Pauli1" id="pone.0087665-Pauli1" class="link-target"></a>Pauli A, Rinn JL, Schier AF (2011) Non-coding RNAs as regulators of embryogenesis. Nat Rev Genet 12: 136–149. <ul class="reflinks"><li><a href="#" data-author="Pauli" data-cit="PauliA%2C%20RinnJL%2C%20SchierAF%20%282011%29%20Non-coding%20RNAs%20as%20regulators%20of%20embryogenesis.%20Nat%20Rev%20Genet%2012%3A%20136%E2%80%93149." data-title="Non-coding%20RNAs%20as%20regulators%20of%20embryogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Non-coding+RNAs+as+regulators+of+embryogenesis+Pauli+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0087665-Garofalo1" id="pone.0087665-Garofalo1" class="link-target"></a>Garofalo M, Romano G, Di Leva G, Nuovo G, Jeon YJ, et al. (2012) EGFR and MET receptor tyrosine kinase-altered microRNA expression induces tumorigenesis and gefitinib resistance in lung cancers. Nat Med 18: 74–82. <ul class="reflinks"><li><a href="#" data-author="Garofalo" data-cit="GarofaloM%2C%20RomanoG%2C%20Di%20LevaG%2C%20NuovoG%2C%20JeonYJ%2C%20et%20al.%20%282012%29%20EGFR%20and%20MET%20receptor%20tyrosine%20kinase-altered%20microRNA%20expression%20induces%20tumorigenesis%20and%20gefitinib%20resistance%20in%20lung%20cancers.%20Nat%20Med%2018%3A%2074%E2%80%9382." data-title="EGFR%20and%20MET%20receptor%20tyrosine%20kinase-altered%20microRNA%20expression%20induces%20tumorigenesis%20and%20gefitinib%20resistance%20in%20lung%20cancers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=EGFR+and+MET+receptor+tyrosine+kinase-altered+microRNA+expression+induces+tumorigenesis+and+gefitinib+resistance+in+lung+cancers+Garofalo+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0087665-Li1" id="pone.0087665-Li1" class="link-target"></a>Li Y, Kowdley KV (2012) Cellular microRNA and the tumorigenesis of hepatocellular carcinoma. Ann Hepatol 11: 272–274. <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiY%2C%20KowdleyKV%20%282012%29%20Cellular%20microRNA%20and%20the%20tumorigenesis%20of%20hepatocellular%20carcinoma.%20Ann%20Hepatol%2011%3A%20272%E2%80%93274." data-title="Cellular%20microRNA%20and%20the%20tumorigenesis%20of%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cellular+microRNA+and+the+tumorigenesis+of+hepatocellular+carcinoma+Li+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0087665-Huang1" id="pone.0087665-Huang1" class="link-target"></a>Huang S, He X (2010) microRNAs: tiny RNA molecules, huge driving forces to move the cell. Protein Cell 1: 916–926. <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangS%2C%20HeX%20%282010%29%20microRNAs%3A%20tiny%20RNA%20molecules%2C%20huge%20driving%20forces%20to%20move%20the%20cell.%20Protein%20Cell%201%3A%20916%E2%80%93926." data-title="microRNAs%3A%20tiny%20RNA%20molecules%2C%20huge%20driving%20forces%20to%20move%20the%20cell" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=microRNAs%3A+tiny+RNA+molecules%2C+huge+driving+forces+to+move+the+cell+Huang+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0087665-Baranwal1" id="pone.0087665-Baranwal1" class="link-target"></a>Baranwal S, Alahari SK (2010) miRNA control of tumor cell invasion and metastasis. Int J Cancer 126: 1283–1290. <ul class="reflinks"><li><a href="#" data-author="Baranwal" data-cit="BaranwalS%2C%20AlahariSK%20%282010%29%20miRNA%20control%20of%20tumor%20cell%20invasion%20and%20metastasis.%20Int%20J%20Cancer%20126%3A%201283%E2%80%931290." data-title="miRNA%20control%20of%20tumor%20cell%20invasion%20and%20metastasis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miRNA+control+of+tumor+cell+invasion+and+metastasis+Baranwal+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0087665-Le1" id="pone.0087665-Le1" class="link-target"></a>Le XF, Merchant O, Bast RC, Calin GA (2010) The Roles of MicroRNAs in the Cancer Invasion-Metastasis Cascade. Cancer Microenviron 3: 137–147. <ul class="reflinks"><li><a href="#" data-author="Le" data-cit="LeXF%2C%20MerchantO%2C%20BastRC%2C%20CalinGA%20%282010%29%20The%20Roles%20of%20MicroRNAs%20in%20the%20Cancer%20Invasion-Metastasis%20Cascade.%20Cancer%20Microenviron%203%3A%20137%E2%80%93147." data-title="The%20Roles%20of%20MicroRNAs%20in%20the%20Cancer%20Invasion-Metastasis%20Cascade" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Roles+of+MicroRNAs+in+the+Cancer+Invasion-Metastasis+Cascade+Le+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0087665-Huang2" id="pone.0087665-Huang2" class="link-target"></a>Huang S, He X (2011) The role of microRNAs in liver cancer progression. Br J Cancer 104: 235–240. <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangS%2C%20HeX%20%282011%29%20The%20role%20of%20microRNAs%20in%20liver%20cancer%20progression.%20Br%20J%20Cancer%20104%3A%20235%E2%80%93240." data-title="The%20role%20of%20microRNAs%20in%20liver%20cancer%20progression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+role+of+microRNAs+in+liver+cancer+progression+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0087665-Ding1" id="pone.0087665-Ding1" class="link-target"></a>Ding J, Huang S, Wu S, Zhao Y, Liang L, et al. (2010) Gain of miR-151 on chromosome 8q24.3 facilitates tumour cell migration and spreading through downregulating RhoGDIA. Nat Cell Biol 12: 390–399. <ul class="reflinks"><li><a href="#" data-author="Ding" data-cit="DingJ%2C%20HuangS%2C%20WuS%2C%20ZhaoY%2C%20LiangL%2C%20et%20al.%20%282010%29%20Gain%20of%20miR-151%20on%20chromosome%208q24.3%20facilitates%20tumour%20cell%20migration%20and%20spreading%20through%20downregulating%20RhoGDIA.%20Nat%20Cell%20Biol%2012%3A%20390%E2%80%93399." data-title="Gain%20of%20miR-151%20on%20chromosome%208q24.3%20facilitates%20tumour%20cell%20migration%20and%20spreading%20through%20downregulating%20RhoGDIA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Gain+of+miR-151+on+chromosome+8q24.3+facilitates+tumour+cell+migration+and+spreading+through+downregulating+RhoGDIA+Ding+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0087665-Ying1" id="pone.0087665-Ying1" class="link-target"></a>Ying Q, Liang L, Guo W, Zha R, Tian Q, et al. (2011) Hypoxia-inducible microRNA-210 augments the metastatic potential of tumor cells by targeting vacuole membrane protein 1 in hepatocellular carcinoma. Hepatology 54: 2064–2075. <ul class="reflinks"><li><a href="#" data-author="Ying" data-cit="YingQ%2C%20LiangL%2C%20GuoW%2C%20ZhaR%2C%20TianQ%2C%20et%20al.%20%282011%29%20Hypoxia-inducible%20microRNA-210%20augments%20the%20metastatic%20potential%20of%20tumor%20cells%20by%20targeting%20vacuole%20membrane%20protein%201%20in%20hepatocellular%20carcinoma.%20Hepatology%2054%3A%202064%E2%80%932075." data-title="Hypoxia-inducible%20microRNA-210%20augments%20the%20metastatic%20potential%20of%20tumor%20cells%20by%20targeting%20vacuole%20membrane%20protein%201%20in%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hypoxia-inducible+microRNA-210+augments+the+metastatic+potential+of+tumor+cells+by+targeting+vacuole+membrane+protein+1+in+hepatocellular+carcinoma+Ying+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0087665-Yao1" id="pone.0087665-Yao1" class="link-target"></a>Yao J, Liang L, Huang S, Ding J, Tan N, et al. (2010) MicroRNA-30d promotes tumor invasion and metastasis by targeting Galphai2 in hepatocellular carcinoma. Hepatology 51: 846–856. <ul class="reflinks"><li><a href="#" data-author="Yao" data-cit="YaoJ%2C%20LiangL%2C%20HuangS%2C%20DingJ%2C%20TanN%2C%20et%20al.%20%282010%29%20MicroRNA-30d%20promotes%20tumor%20invasion%20and%20metastasis%20by%20targeting%20Galphai2%20in%20hepatocellular%20carcinoma.%20Hepatology%2051%3A%20846%E2%80%93856." data-title="MicroRNA-30d%20promotes%20tumor%20invasion%20and%20metastasis%20by%20targeting%20Galphai2%20in%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-30d+promotes+tumor+invasion+and+metastasis+by+targeting+Galphai2+in+hepatocellular+carcinoma+Yao+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0087665-Tian1" id="pone.0087665-Tian1" class="link-target"></a>Tian Q, Liang L, Ding J, Zha R, Shi H, et al. (2012) MicroRNA-550a acts as a pro-metastatic gene and directly targets cytoplasmic polyadenylation element-binding protein 4 in hepatocellular carcinoma. PLoS One 7: e48958. <ul class="reflinks"><li><a href="#" data-author="Tian" data-cit="TianQ%2C%20LiangL%2C%20DingJ%2C%20ZhaR%2C%20ShiH%2C%20et%20al.%20%282012%29%20MicroRNA-550a%20acts%20as%20a%20pro-metastatic%20gene%20and%20directly%20targets%20cytoplasmic%20polyadenylation%20element-binding%20protein%204%20in%20hepatocellular%20carcinoma.%20PLoS%20One%207%3A%20e48958." data-title="MicroRNA-550a%20acts%20as%20a%20pro-metastatic%20gene%20and%20directly%20targets%20cytoplasmic%20polyadenylation%20element-binding%20protein%204%20in%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-550a+acts+as+a+pro-metastatic+gene+and+directly+targets+cytoplasmic+polyadenylation+element-binding+protein+4+in+hepatocellular+carcinoma+Tian+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0087665-Li2" id="pone.0087665-Li2" class="link-target"></a>Li W, Xie L, He X, Li J, Tu K, et al. (2008) Diagnostic and prognostic implications of microRNAs in human hepatocellular carcinoma. Int J Cancer 123: 1616–1622. <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiW%2C%20XieL%2C%20HeX%2C%20LiJ%2C%20TuK%2C%20et%20al.%20%282008%29%20Diagnostic%20and%20prognostic%20implications%20of%20microRNAs%20in%20human%20hepatocellular%20carcinoma.%20Int%20J%20Cancer%20123%3A%201616%E2%80%931622." data-title="Diagnostic%20and%20prognostic%20implications%20of%20microRNAs%20in%20human%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Diagnostic+and+prognostic+implications+of+microRNAs+in+human+hepatocellular+carcinoma+Li+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0087665-Ding2" id="pone.0087665-Ding2" class="link-target"></a>Ding J, Huang S, Wang Y, Tian Q, Zha R, et al. (2013) Genome-wide screening reveals that miR-195 targets the TNF-alpha/NF-kappaB pathway by down-regulating IkappaB kinase alpha and TAB3 in hepatocellular carcinoma. Hepatology 58: 654–666. <ul class="reflinks"><li><a href="#" data-author="Ding" data-cit="DingJ%2C%20HuangS%2C%20WangY%2C%20TianQ%2C%20ZhaR%2C%20et%20al.%20%282013%29%20Genome-wide%20screening%20reveals%20that%20miR-195%20targets%20the%20TNF-alpha%2FNF-kappaB%20pathway%20by%20down-regulating%20IkappaB%20kinase%20alpha%20and%20TAB3%20in%20hepatocellular%20carcinoma.%20Hepatology%2058%3A%20654%E2%80%93666." data-title="Genome-wide%20screening%20reveals%20that%20miR-195%20targets%20the%20TNF-alpha%2FNF-kappaB%20pathway%20by%20down-regulating%20IkappaB%20kinase%20alpha%20and%20TAB3%20in%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+screening+reveals+that+miR-195+targets+the+TNF-alpha%2FNF-kappaB+pathway+by+down-regulating+IkappaB+kinase+alpha+and+TAB3+in+hepatocellular+carcinoma+Ding+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0087665-Wu1" id="pone.0087665-Wu1" class="link-target"></a>Wu XM, Shao XQ, Meng XX, Zhang XN, Zhu L, et al. (2011) Genome-wide analysis of microRNA and mRNA expression signatures in hydroxycamptothecin-resistant gastric cancer cells. Acta Pharmacol Sin 32: 259–269. <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuXM%2C%20ShaoXQ%2C%20MengXX%2C%20ZhangXN%2C%20ZhuL%2C%20et%20al.%20%282011%29%20Genome-wide%20analysis%20of%20microRNA%20and%20mRNA%20expression%20signatures%20in%20hydroxycamptothecin-resistant%20gastric%20cancer%20cells.%20Acta%20Pharmacol%20Sin%2032%3A%20259%E2%80%93269." data-title="Genome-wide%20analysis%20of%20microRNA%20and%20mRNA%20expression%20signatures%20in%20hydroxycamptothecin-resistant%20gastric%20cancer%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+analysis+of+microRNA+and+mRNA+expression+signatures+in+hydroxycamptothecin-resistant+gastric+cancer+cells+Wu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0087665-Liu1" id="pone.0087665-Liu1" class="link-target"></a>Liu LX, Jiang HC, Liu ZH, Zhou J, Zhang WH, et al. (2002) Integrin gene expression profiles of human hepatocellular carcinoma. World J Gastroenterol 8: 631–637. <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuLX%2C%20JiangHC%2C%20LiuZH%2C%20ZhouJ%2C%20ZhangWH%2C%20et%20al.%20%282002%29%20Integrin%20gene%20expression%20profiles%20of%20human%20hepatocellular%20carcinoma.%20World%20J%20Gastroenterol%208%3A%20631%E2%80%93637." data-title="Integrin%20gene%20expression%20profiles%20of%20human%20hepatocellular%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Integrin+gene+expression+profiles+of+human+hepatocellular+carcinoma+Liu+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0087665-Huveneers1" id="pone.0087665-Huveneers1" class="link-target"></a>Huveneers S, Danen EH (2009) Adhesion signaling - crosstalk between integrins, Src and Rho. J Cell Sci 122: 1059–1069. <ul class="reflinks"><li><a href="#" data-author="Huveneers" data-cit="HuveneersS%2C%20DanenEH%20%282009%29%20Adhesion%20signaling%20-%20crosstalk%20between%20integrins%2C%20Src%20and%20Rho.%20J%20Cell%20Sci%20122%3A%201059%E2%80%931069." data-title="Adhesion%20signaling%20-%20crosstalk%20between%20integrins%2C%20Src%20and%20Rho" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Adhesion+signaling+-+crosstalk+between+integrins%2C+Src+and+Rho+Huveneers+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0087665-Simpson1" id="pone.0087665-Simpson1" class="link-target"></a>Simpson KJ, Selfors LM, Bui J, Reynolds A, Leake D, et al. (2008) Identification of genes that regulate epithelial cell migration using an siRNA screening approach. Nat Cell Biol 10: 1027–1038. <ul class="reflinks"><li><a href="#" data-author="Simpson" data-cit="SimpsonKJ%2C%20SelforsLM%2C%20BuiJ%2C%20ReynoldsA%2C%20LeakeD%2C%20et%20al.%20%282008%29%20Identification%20of%20genes%20that%20regulate%20epithelial%20cell%20migration%20using%20an%20siRNA%20screening%20approach.%20Nat%20Cell%20Biol%2010%3A%201027%E2%80%931038." data-title="Identification%20of%20genes%20that%20regulate%20epithelial%20cell%20migration%20using%20an%20siRNA%20screening%20approach" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+genes+that+regulate+epithelial+cell+migration+using+an+siRNA+screening+approach+Simpson+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0087665-Fotos1" id="pone.0087665-Fotos1" class="link-target"></a>Fotos JS, Patel VP, Karin NJ, Temburni MK, Koh JT, et al. (2006) Automated time-lapse microscopy and high-resolution tracking of cell migration. Cytotechnology 51: 7–19. <ul class="reflinks"><li><a href="#" data-author="Fotos" data-cit="FotosJS%2C%20PatelVP%2C%20KarinNJ%2C%20TemburniMK%2C%20KohJT%2C%20et%20al.%20%282006%29%20Automated%20time-lapse%20microscopy%20and%20high-resolution%20tracking%20of%20cell%20migration.%20Cytotechnology%2051%3A%207%E2%80%9319." data-title="Automated%20time-lapse%20microscopy%20and%20high-resolution%20tracking%20of%20cell%20migration" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Automated+time-lapse+microscopy+and+high-resolution+tracking+of+cell+migration+Fotos+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0087665-Haller1" id="pone.0087665-Haller1" class="link-target"></a>Haller F, von Heydebreck A, Zhang JD, Gunawan B, Langer C, et al. (2010) Localization- and mutation-dependent microRNA (miRNA) expression signatures in gastrointestinal stromal tumours (GISTs), with a cluster of co-expressed miRNAs located at 14q32.31. J Pathol 220: 71–86. <ul class="reflinks"><li><a href="#" data-author="Haller" data-cit="HallerF%2C%20von%20HeydebreckA%2C%20ZhangJD%2C%20GunawanB%2C%20LangerC%2C%20et%20al.%20%282010%29%20Localization-%20and%20mutation-dependent%20microRNA%20%28miRNA%29%20expression%20signatures%20in%20gastrointestinal%20stromal%20tumours%20%28GISTs%29%2C%20with%20a%20cluster%20of%20co-expressed%20miRNAs%20located%20at%2014q32.31.%20J%20Pathol%20220%3A%2071%E2%80%9386." data-title="Localization-%20and%20mutation-dependent%20microRNA%20%28miRNA%29%20expression%20signatures%20in%20gastrointestinal%20stromal%20tumours%20%28GISTs%29%2C%20with%20a%20cluster%20of%20co-expressed%20miRNAs%20located%20at%2014q32.31" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Localization-+and+mutation-dependent+microRNA+%28miRNA%29+expression+signatures+in+gastrointestinal+stromal+tumours+%28GISTs%29%2C+with+a+cluster+of+co-expressed+miRNAs+located+at+14q32.31+Haller+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0087665-Sarver1" id="pone.0087665-Sarver1" class="link-target"></a>Sarver AL, Thayanithy V, Scott MC, Cleton-Jansen AM, Hogendoorn PC, et al. (2013) MicroRNAs at the human 14q32 locus have prognostic significance in osteosarcoma. Orphanet J Rare Dis 8: 7. <ul class="reflinks"><li><a href="#" data-author="Sarver" data-cit="SarverAL%2C%20ThayanithyV%2C%20ScottMC%2C%20Cleton-JansenAM%2C%20HogendoornPC%2C%20et%20al.%20%282013%29%20MicroRNAs%20at%20the%20human%2014q32%20locus%20have%20prognostic%20significance%20in%20osteosarcoma.%20Orphanet%20J%20Rare%20Dis%208%3A%207." data-title="MicroRNAs%20at%20the%20human%2014q32%20locus%20have%20prognostic%20significance%20in%20osteosarcoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+at+the+human+14q32+locus+have+prognostic+significance+in+osteosarcoma+Sarver+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0087665-Lages1" id="pone.0087665-Lages1" class="link-target"></a>Lages E, Guttin A, El Atifi M, Ramus C, Ipas H, et al. (2011) MicroRNA and target protein patterns reveal physiopathological features of glioma subtypes. PLoS One 6: e20600. <ul class="reflinks"><li><a href="#" data-author="Lages" data-cit="LagesE%2C%20GuttinA%2C%20El%20AtifiM%2C%20RamusC%2C%20IpasH%2C%20et%20al.%20%282011%29%20MicroRNA%20and%20target%20protein%20patterns%20reveal%20physiopathological%20features%20of%20glioma%20subtypes.%20PLoS%20One%206%3A%20e20600." data-title="MicroRNA%20and%20target%20protein%20patterns%20reveal%20physiopathological%20features%20of%20glioma%20subtypes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+and+target+protein+patterns+reveal+physiopathological+features+of+glioma+subtypes+Lages+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0087665-Boominathan1" id="pone.0087665-Boominathan1" class="link-target"></a>Boominathan L (2010) The tumor suppressors p53, p63, and p73 are regulators of microRNA processing complex. PLoS One 5: e10615. <ul class="reflinks"><li><a href="#" data-author="Boominathan" data-cit="BoominathanL%20%282010%29%20The%20tumor%20suppressors%20p53%2C%20p63%2C%20and%20p73%20are%20regulators%20of%20microRNA%20processing%20complex.%20PLoS%20One%205%3A%20e10615." data-title="The%20tumor%20suppressors%20p53%2C%20p63%2C%20and%20p73%20are%20regulators%20of%20microRNA%20processing%20complex" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+tumor+suppressors+p53%2C+p63%2C+and+p73+are+regulators+of+microRNA+processing+complex+Boominathan+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0087665-Zhang1" id="pone.0087665-Zhang1" class="link-target"></a>Zhang X, Wang H, Zhang S, Song J, Zhang Y, et al. (2012) MiR-134 functions as a regulator of cell proliferation, apoptosis, and migration involving lung septation. <em>In Vitro</em> Cell Dev Biol Anim 48: 131–136. <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangX%2C%20WangH%2C%20ZhangS%2C%20SongJ%2C%20ZhangY%2C%20et%20al.%20%282012%29%20MiR-134%20functions%20as%20a%20regulator%20of%20cell%20proliferation%2C%20apoptosis%2C%20and%20migration%20involving%20lung%20septation.%20In%20Vitro%20Cell%20Dev%20Biol%20Anim%2048%3A%20131%E2%80%93136." data-title="MiR-134%20functions%20as%20a%20regulator%20of%20cell%20proliferation%2C%20apoptosis%2C%20and%20migration%20involving%20lung%20septation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MiR-134+functions+as+a+regulator+of+cell+proliferation%2C+apoptosis%2C+and+migration+involving+lung+septation+Zhang+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0087665-Li3" id="pone.0087665-Li3" class="link-target"></a>Li J, Wang Y, Luo J, Fu Z, Ying J, et al. (2012) miR-134 inhibits epithelial to mesenchymal transition by targeting FOXM1 in non-small cell lung cancer cells. FEBS Lett 586: 3761–3765. <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiJ%2C%20WangY%2C%20LuoJ%2C%20FuZ%2C%20YingJ%2C%20et%20al.%20%282012%29%20miR-134%20inhibits%20epithelial%20to%20mesenchymal%20transition%20by%20targeting%20FOXM1%20in%20non-small%20cell%20lung%20cancer%20cells.%20FEBS%20Lett%20586%3A%203761%E2%80%933765." data-title="miR-134%20inhibits%20epithelial%20to%20mesenchymal%20transition%20by%20targeting%20FOXM1%20in%20non-small%20cell%20lung%20cancer%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miR-134+inhibits+epithelial+to+mesenchymal+transition+by+targeting+FOXM1+in+non-small+cell+lung+cancer+cells+Li+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0087665-Brakebusch1" id="pone.0087665-Brakebusch1" class="link-target"></a>Brakebusch C, Fassler R (2005) beta 1 integrin function <em>in vivo</em>: adhesion, migration and more. Cancer Metastasis Rev 24: 403–411. <ul class="reflinks"><li><a href="#" data-author="Brakebusch" data-cit="BrakebuschC%2C%20FasslerR%20%282005%29%20beta%201%20integrin%20function%20in%20vivo%3A%20adhesion%2C%20migration%20and%20more.%20Cancer%20Metastasis%20Rev%2024%3A%20403%E2%80%93411." data-title="beta%201%20integrin%20function%20in%20vivo%3A%20adhesion%2C%20migration%20and%20more" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=beta+1+integrin+function+in+vivo%3A+adhesion%2C+migration+and+more+Brakebusch+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0087665-FeldingHabermann1" id="pone.0087665-FeldingHabermann1" class="link-target"></a>Felding-Habermann B (2003) Integrin adhesion receptors in tumor metastasis. Clin Exp Metastasis 20: 203–213. <ul class="reflinks"><li><a href="#" data-author="Felding-Habermann" data-cit="Felding-HabermannB%20%282003%29%20Integrin%20adhesion%20receptors%20in%20tumor%20metastasis.%20Clin%20Exp%20Metastasis%2020%3A%20203%E2%80%93213." data-title="Integrin%20adhesion%20receptors%20in%20tumor%20metastasis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Integrin+adhesion+receptors+in+tumor+metastasis+Felding-Habermann+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0087665-Caccavari1" id="pone.0087665-Caccavari1" class="link-target"></a>Caccavari F, Valdembri D, Sandri C, Bussolino F, Serini G (2010) Integrin signaling and lung cancer. Cell Adh Migr 4: 124–129. <ul class="reflinks"><li><a href="#" data-author="Caccavari" data-cit="CaccavariF%2C%20ValdembriD%2C%20SandriC%2C%20BussolinoF%2C%20SeriniG%20%282010%29%20Integrin%20signaling%20and%20lung%20cancer.%20Cell%20Adh%20Migr%204%3A%20124%E2%80%93129." data-title="Integrin%20signaling%20and%20lung%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Integrin+signaling+and+lung+cancer+Caccavari+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0087665-Allen1" id="pone.0087665-Allen1" class="link-target"></a>Allen MD, Vaziri R, Green M, Chelala C, Brentnall AR, et al. (2011) Clinical and functional significance of alpha9beta1 integrin expression in breast cancer: a novel cell-surface marker of the basal phenotype that promotes tumour cell invasion. J Pathol 223: 646–658. <ul class="reflinks"><li><a href="#" data-author="Allen" data-cit="AllenMD%2C%20VaziriR%2C%20GreenM%2C%20ChelalaC%2C%20BrentnallAR%2C%20et%20al.%20%282011%29%20Clinical%20and%20functional%20significance%20of%20alpha9beta1%20integrin%20expression%20in%20breast%20cancer%3A%20a%20novel%20cell-surface%20marker%20of%20the%20basal%20phenotype%20that%20promotes%20tumour%20cell%20invasion.%20J%20Pathol%20223%3A%20646%E2%80%93658." data-title="Clinical%20and%20functional%20significance%20of%20alpha9beta1%20integrin%20expression%20in%20breast%20cancer%3A%20a%20novel%20cell-surface%20marker%20of%20the%20basal%20phenotype%20that%20promotes%20tumour%20cell%20invasion" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Clinical+and+functional+significance+of+alpha9beta1+integrin+expression+in+breast+cancer%3A+a+novel+cell-surface+marker+of+the+basal+phenotype+that+promotes+tumour+cell+invasion+Allen+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0087665-Grzesiak1" id="pone.0087665-Grzesiak1" class="link-target"></a>Grzesiak JJ, Tran Cao HS, Burton DW, Kaushal S, Vargas F, et al. (2011) Knockdown of the beta(1) integrin subunit reduces primary tumor growth and inhibits pancreatic cancer metastasis. Int J Cancer 129: 2905–2915. <ul class="reflinks"><li><a href="#" data-author="Grzesiak" data-cit="GrzesiakJJ%2C%20Tran%20CaoHS%2C%20BurtonDW%2C%20KaushalS%2C%20VargasF%2C%20et%20al.%20%282011%29%20Knockdown%20of%20the%20beta%281%29%20integrin%20subunit%20reduces%20primary%20tumor%20growth%20and%20inhibits%20pancreatic%20cancer%20metastasis.%20Int%20J%20Cancer%20129%3A%202905%E2%80%932915." data-title="Knockdown%20of%20the%20beta%281%29%20integrin%20subunit%20reduces%20primary%20tumor%20growth%20and%20inhibits%20pancreatic%20cancer%20metastasis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Knockdown+of+the+beta%281%29+integrin+subunit+reduces+primary+tumor+growth+and+inhibits+pancreatic+cancer+metastasis+Grzesiak+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0087665-Masumoto1" id="pone.0087665-Masumoto1" class="link-target"></a>Masumoto A, Arao S, Otsuki M (1999) Role of beta1 integrins in adhesion and invasion of hepatocellular carcinoma cells. Hepatology 29: 68–74. <ul class="reflinks"><li><a href="#" data-author="Masumoto" data-cit="MasumotoA%2C%20AraoS%2C%20OtsukiM%20%281999%29%20Role%20of%20beta1%20integrins%20in%20adhesion%20and%20invasion%20of%20hepatocellular%20carcinoma%20cells.%20Hepatology%2029%3A%2068%E2%80%9374." data-title="Role%20of%20beta1%20integrins%20in%20adhesion%20and%20invasion%20of%20hepatocellular%20carcinoma%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Role+of+beta1+integrins+in+adhesion+and+invasion+of+hepatocellular+carcinoma+cells+Masumoto+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0087665-Mizuno1" id="pone.0087665-Mizuno1" class="link-target"></a>Mizuno H, Ogura M, Saito Y, Sekine W, Sano R, et al. (2008) Changes in adhesive and migratory characteristics of hepatocellular carcinoma (HCC) cells induced by expression of alpha3beta1 integrin. Biochim Biophys Acta 1780: 564–570. <ul class="reflinks"><li><a href="#" data-author="Mizuno" data-cit="MizunoH%2C%20OguraM%2C%20SaitoY%2C%20SekineW%2C%20SanoR%2C%20et%20al.%20%282008%29%20Changes%20in%20adhesive%20and%20migratory%20characteristics%20of%20hepatocellular%20carcinoma%20%28HCC%29%20cells%20induced%20by%20expression%20of%20alpha3beta1%20integrin.%20Biochim%20Biophys%20Acta%201780%3A%20564%E2%80%93570." data-title="Changes%20in%20adhesive%20and%20migratory%20characteristics%20of%20hepatocellular%20carcinoma%20%28HCC%29%20cells%20induced%20by%20expression%20of%20alpha3beta1%20integrin" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Changes+in+adhesive+and+migratory+characteristics+of+hepatocellular+carcinoma+%28HCC%29+cells+induced+by+expression+of+alpha3beta1+integrin+Mizuno+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0087665-Fu1" id="pone.0087665-Fu1" class="link-target"></a>Fu BH, Wu ZZ, Qin J (2011) Effects of integrin alpha6beta1 on migration of hepatocellular carcinoma cells. Mol Biol Rep 38: 3271–3276. <ul class="reflinks"><li><a href="#" data-author="Fu" data-cit="FuBH%2C%20WuZZ%2C%20QinJ%20%282011%29%20Effects%20of%20integrin%20alpha6beta1%20on%20migration%20of%20hepatocellular%20carcinoma%20cells.%20Mol%20Biol%20Rep%2038%3A%203271%E2%80%933276." data-title="Effects%20of%20integrin%20alpha6beta1%20on%20migration%20of%20hepatocellular%20carcinoma%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Effects+of+integrin+alpha6beta1+on+migration+of+hepatocellular+carcinoma+cells+Fu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0087665-Rottner1" id="pone.0087665-Rottner1" class="link-target"></a>Rottner K, Stradal TE (2011) Actin dynamics and turnover in cell motility. Curr Opin Cell Biol 23: 569–578. <ul class="reflinks"><li><a href="#" data-author="Rottner" data-cit="RottnerK%2C%20StradalTE%20%282011%29%20Actin%20dynamics%20and%20turnover%20in%20cell%20motility.%20Curr%20Opin%20Cell%20Biol%2023%3A%20569%E2%80%93578." data-title="Actin%20dynamics%20and%20turnover%20in%20cell%20motility" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Actin+dynamics+and+turnover+in+cell+motility+Rottner+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0087665-Kaibuchi1" id="pone.0087665-Kaibuchi1" class="link-target"></a>Kaibuchi K, Kuroda S, Amano M (1999) Regulation of the cytoskeleton and cell adhesion by the Rho family GTPases in mammalian cells. Annu Rev Biochem 68: 459–486. <ul class="reflinks"><li><a href="#" data-author="Kaibuchi" data-cit="KaibuchiK%2C%20KurodaS%2C%20AmanoM%20%281999%29%20Regulation%20of%20the%20cytoskeleton%20and%20cell%20adhesion%20by%20the%20Rho%20family%20GTPases%20in%20mammalian%20cells.%20Annu%20Rev%20Biochem%2068%3A%20459%E2%80%93486." data-title="Regulation%20of%20the%20cytoskeleton%20and%20cell%20adhesion%20by%20the%20Rho%20family%20GTPases%20in%20mammalian%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+the+cytoskeleton+and+cell+adhesion+by+the+Rho+family+GTPases+in+mammalian+cells+Kaibuchi+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

