



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Mutations in human genes can be responsible for inherited genetic disorders and cancer. Mutations can arise due to environmental factors or spontaneously. It has been shown that certain DNA sequences are more prone to mutate. These sites are termed hotspots and exhibit a higher mutation frequency than expected by chance. In contrast, DNA sequences with lower mutation frequencies than expected by chance are termed coldspots. Mutation hotspots are usually derived from a mutation spectrum, which reflects particular population where an effect of a common ancestor plays a role. To detect coldspots/hotspots unaffected by population bias, we analysed the presence of germline mutations obtained from HGMD database in the 5-nucleotide segments repeatedly occurring in genes associated with common inherited disorders, in particular, the <em>PAH</em>, <em>LDLR</em>, <em>CFTR</em>, <em>F8</em>, and <em>F9</em> genes. Statistically significant sequences (mutational motifs) rarely associated with mutations (coldspots) and frequently associated with mutations (hotspots) exhibited characteristic sequence patterns, e.g. coldspots contained purine tract while hotspots showed alternating purine-pyrimidine bases, often with the presence of <span class="monospace">CpG</span> dinucleotide. Using molecular dynamics simulations and free energy calculations, we analysed the global bending properties of two selected coldspots and two hotspots with a <span class="monospace">G/T</span> mismatch. We observed that the coldspots were inherently more flexible than the hotspots. We assume that this property might be critical for effective mismatch repair as DNA with a mutation recognized by MutSα protein is noticeably bent.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Růžička M, Kulhánek P, Radová L, Čechová A, Špačková N, Fajkusová L, et al.  (2017) DNA mutation motifs in the genes associated with inherited diseases. PLoS ONE 12(8):
           e0182377.
        
        https://doi.org/10.1371/journal.pone.0182377</p><p><strong>Editor: </strong>Tamar Schlick, New York University, UNITED STATES</p><p><strong>Received: </strong>March 27, 2017; <strong>Accepted: </strong>July 17, 2017; <strong>Published: </strong> August 2, 2017</p><p><strong>Copyright: </strong> © 2017 Růžička et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>All relevant data are within the paper and its Supporting Information files.</p><p><strong>Funding: </strong>This research was financially supported by Grant Agency of the Czech Republic (GA16-11619S/2016) and by the Ministry of Education, Youth and Sports of the Czech Republic under the project CEITEC 2020 (LQ1601). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Computational resources were provided by the CESNET LM2015042 and the CERIT Scientific Cloud LM2015085, provided under the programme "Projects of Large Research, Development, and Innovations Infrastructures". This work was supported by The Ministry of Education, Youth and Sports from the Large Infrastructures for Research, Experimental Development and Innovations project „IT4Innovations National Supercomputing Center – LM2015070“.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Genomic integrity and stability is important for any living organism. Cells evolved various repair pathways to maintain the correct transmission of genetic information to the next generation, e.g. nucleotide excision repair (NER), base excision repair (BER), mismatch repair (MMR), homologous recombination repair and post-replication repair [<a href="#pone.0182377.ref001" class="ref-tip">1</a>]. Mutations arising in parent’s germ cells are termed <em>de novo</em> mutations and can cause various inherited disorders. Errors in DNA can arise due to environmental factors or spontaneously, e.g. during DNA replication or due to deamination of 5-methylcytosine. With a fully functional repair system, the frequency of spontaneous error is ca. 1 per 10<sup>9</sup>–10<sup>10</sup> base pairs per replication [<a href="#pone.0182377.ref002" class="ref-tip">2</a>]. In human genes, a mutational strand asymmetry was observed as a consequence of both transcription [<a href="#pone.0182377.ref003" class="ref-tip">3</a>–<a href="#pone.0182377.ref005" class="ref-tip">5</a>] and replication processes [<a href="#pone.0182377.ref006" class="ref-tip">6</a>–<a href="#pone.0182377.ref008" class="ref-tip">8</a>]. Detecting DNA damage on a transcribed DNA strand by RNA polymerase initiates transcription coupled repair (TCR), a sub-pathway of NER. This leads to strand asymmetry that can be expressed using <span class="monospace">GC</span> and <span class="monospace">TA</span> skew profiles. Strand asymmetry arising during replication is related to different synthesis and proofreading mechanisms of the leading strand (replicated continuously from the origin) and the lagging strand (replicated in discrete steps towards the origin). Previous studies have shown that certain DNA sequences are more prone to mutate [<a href="#pone.0182377.ref009" class="ref-tip">9</a>,<a href="#pone.0182377.ref010" class="ref-tip">10</a>]. These sites are termed hotspots and exhibit a higher mutation frequency than expected by chance. DNA sequences with lower frequencies than expected by chance are termed coldspots. Perhaps, the most well-known hotspot is the <span class="monospace">CpG</span> dinucleotide associated with the <span class="monospace">C&gt;T</span> mutation, resulting in <span class="monospace">TpG</span> or <span class="monospace">CpA</span> (on the other strand) transitions [<a href="#pone.0182377.ref011" class="ref-tip">11</a>]. During this change, cytosine is methylated and subsequently deaminated [<a href="#pone.0182377.ref012" class="ref-tip">12</a>]. In spite of the effective repair pathways, BER and MMR, this mutation is very frequent [<a href="#pone.0182377.ref013" class="ref-tip">13</a>–<a href="#pone.0182377.ref015" class="ref-tip">15</a>]. It was proposed that due to high <span class="monospace">CpG</span> site mutability, these dinucleotides occur less frequently in the genome than would be expected by chance [<a href="#pone.0182377.ref016" class="ref-tip">16</a>]. Cytosine methylation is also one of the most frequent DNA modifications used to control gene expression in the cell, hence it represents a naturally occurring base modification [<a href="#pone.0182377.ref017" class="ref-tip">17</a>,<a href="#pone.0182377.ref018" class="ref-tip">18</a>]. Apart from the <span class="monospace">CpG</span> sites, there are other sequences with higher mutation rates, e.g. <span class="monospace">CpHpG</span> trinucleotide (where <span class="monospace">H</span> stands for <span class="monospace">A</span>, <span class="monospace">C</span> or <span class="monospace">T</span>) [<a href="#pone.0182377.ref019" class="ref-tip">19</a>] or <span class="monospace">GTAAGT</span> motif [<a href="#pone.0182377.ref020" class="ref-tip">20</a>]. It was observed that a sequence of ±2 nucleotides (nt) around a mismatch has an influence on the relative rates of single nucleotide variations (SNVs) causing human inherited disorders [<a href="#pone.0182377.ref015" class="ref-tip">15</a>,<a href="#pone.0182377.ref021" class="ref-tip">21</a>]. In addition, all mismatch types are not repaired with the same efficiency [<a href="#pone.0182377.ref022" class="ref-tip">22</a>]. In the cell, mismatches and small insertion/deletion loops (IDLs) are primarily targeted by MutSα protein within the MMR pathway [<a href="#pone.0182377.ref002" class="ref-tip">2</a>]. DNA with a mismatch is bent by ca. 60° in the MutSα/DNA complex (so the angle between the arms is 120°) [<a href="#pone.0182377.ref023" class="ref-tip">23</a>–<a href="#pone.0182377.ref026" class="ref-tip">26</a>]. A model where MutSα slides along DNA and scans for flexible regions corresponding to mismatches has been proposed [<a href="#pone.0182377.ref027" class="ref-tip">27</a>]. It seems that DNA flexibility, which play a role in various cell processes [<a href="#pone.0182377.ref028" class="ref-tip">28</a>–<a href="#pone.0182377.ref030" class="ref-tip">30</a>], might also be a critical factor to detect mismatches in DNA. In the MutSα/DNA complex, a mismatch is further recognized by phenylalanine 432 and glutamine 434 from conserved Phe-X-Glu motif [<a href="#pone.0182377.ref026" class="ref-tip">26</a>,<a href="#pone.0182377.ref031" class="ref-tip">31</a>]. The interaction of MutSα with DNA is also coupled with its ATPase activity. There are two ATP non-equivalent hydrolytic sites, each located at MSH2 and MSH6 units of MutSα [<a href="#pone.0182377.ref026" class="ref-tip">26</a>]. The ATP sites are part of the ATPase domains that belong to ABC-transporter superfamily [<a href="#pone.0182377.ref024" class="ref-tip">24</a>,<a href="#pone.0182377.ref032" class="ref-tip">32</a>]. Despite ATP and DNA binding sites being separated by a distance of about 70 Å, allosteric signalling between them exists [<a href="#pone.0182377.ref033" class="ref-tip">33</a>]. The MutSα protein with ADP bound can diffuse freely on DNA and search for mismatches [<a href="#pone.0182377.ref034" class="ref-tip">34</a>]. During this stage, MutSα can bind ATP, which is hydrolysed in one subunit faster than in the second one [<a href="#pone.0182377.ref035" class="ref-tip">35</a>]. The ATP binding initiates the formation of a clamp composed of MSH2 and MSH6 DNA binding domains around the DNA. This clamp is relaxed after ATP hydrolysis so there is an open/close state of MutSα. After MutSα recognizes a mismatch, ATP hydrolysis is suppressed and MSH2 and MSH6 DNA binding domains form a closed clamp. The Phe-X-Glu motif contacts the mismatch and DNA is bent [<a href="#pone.0182377.ref035" class="ref-tip">35</a>]. This event initiates interaction with the MulL protein and the subsequent repair process [<a href="#pone.0182377.ref036" class="ref-tip">36</a>]. These conformational processes are probably associated with the release of energy from ATP hydrolysis, but the precise mechanism is not yet known.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>To better understand the emergence of mutations in DNA genes, we performed a two-step study. Firstly, we analysed DNA sequences of five genes associated with common inherited disorders where large numbers of different SNVs were reported in the Human Gene Mutation Database (HGMD, <a href="http://www.hgmd.cf.ac.uk/ac/index.php">http://www.hgmd.cf.ac.uk/ac/index.php</a>). This database contains known gene mutations responsible for human inherited diseases, plus disease-associated functional polymorphisms. Mutation spectra for various populations are not provided there, i.e. each mutation appears just once in the database with its first reference in literature. We focused on mutations in the <em>PAH</em> gene (associated with hyperphenylalanineamia), <em>LDLR</em> gene (associated with hypercholesterolemia), <em>CFTR</em> gene (associated with cystic fibrosis), and <em>F8</em> and <em>F9</em> genes (associated with hemophilia A and B, respectively). In these genes, we identified repeatedly occurring 5-nucleotide (5-nt) sequences that are: i) rarely associated with mutations (coldspots) and ii) frequently associated with mutations (hotspots). Secondly, we investigated the bending properties of two hotspots and two coldspots using advanced computational techniques. Although the parameters characterizing trinucleotide bending with respect to nucleosome and DNase I have been derived [<a href="#pone.0182377.ref037" class="ref-tip">37</a>,<a href="#pone.0182377.ref038" class="ref-tip">38</a>], their utilization for our purpose is limited as we focus on specific DNA deformation with a mismatch base pair induced by the MutSα protein. As shown in a previous study focused on DNA A-tracts, one sequence can behave differently with respect to different deformations [<a href="#pone.0182377.ref029" class="ref-tip">29</a>]. In particular, we employed Molecular Dynamics (MD) simulations implemented in the AMBER program package [<a href="#pone.0182377.ref039" class="ref-tip">39</a>] and free energy calculations using the adaptive biasing method (ABF) [<a href="#pone.0182377.ref040" class="ref-tip">40</a>] enhanced by the multiple walker approach (MWA) [<a href="#pone.0182377.ref041" class="ref-tip">41</a>]. Based on our calculations, we were able to derive the free energy change needed to bend a straight DNA duplex with a coldspot or hotspot towards the bent geometry of DNA found in the MutSα/DNA complex.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Materials and methods"></a><h2>Materials and methods</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Analysis of mutations in DNA sequences"></a>
<h3>Analysis of mutations in DNA sequences</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>We analysed germinal mutations in five genes: <em>PAH</em>, <em>LDLR</em>, <em>CFTR</em>, <em>F8</em> and <em>F9</em> based on the HGMD. We focused primarily on mutations in exons. To increase the number of mutations for our analysis, we also considered the intron sequences between the exons where the mutations occurred. Because genes are sequenced in introns to different extents, number of mutations in these parts differs. We included intron segments of 7 nt long for <em>PAH</em>, 4 nt long for <em>LDLR</em>, 9 nt long for <em>CFTR</em>, 6 nt long for <em>F8</em> and 5 nt long for <em>F9</em> on both sides of the exons. In addition, 2 nt before the first codon (5’- untranslated region (5’-UTR)) and 2 nt after the last codon (3’- untranslated region (3’-UTR)) were also included for each gene, so that the first and last coding nucleotide occur in the middle of the segments included in the analysis (see below and <strong><a href="#pone-0182377-g001">Fig 1</a></strong>). The total lengths of analysed DNA sequences including the introns and 5’-UTR/3’-UTR are indicated in <strong><a href="#pone-0182377-t001">Table 1</a></strong>.</p>
<a class="link-target" id="pone-0182377-g001" name="pone-0182377-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.g001" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Scheme of the beginning of the <em>PAH</em> gene shows segmentation of DNA into 5-nt segments.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>Middle position of each segment is highlighted by a red letter. Coding sequence is in uppercase letters, first two nucleotides (5’-UTR) are in lowercase letters.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.g001">
              https://doi.org/10.1371/journal.pone.0182377.g001</a></p></div><a class="link-target" id="pone-0182377-t001" name="pone-0182377-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.t001" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Survey of analysed genes, lengths of DNA sequences, number of identified 5-nt segments and number of germline mutations found in HGMD database.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.t001">
              https://doi.org/10.1371/journal.pone.0182377.t001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>The DNA sequence of each gene was divided into 5-nt segments (<strong><a href="#pone-0182377-g001">Fig 1</a></strong>). Subsequently, we analysed if the middle positions of the obtained segments (or their complement sequences) contained mutations, e.g. in the case of the <span class="monospace">AAAAT</span> segment (written from 5’ to 3’ end throughout the text), we also looked for mutations in the complement sequence <span class="monospace">ATTTT</span>. We created program in Java for this purpose. Four nucleobases combined in five positions allow 1024 unique 5-nt segments to form. Since we considered the segments together with their complements, the total number of unique 5-nt segments (motifs) is only 512. <strong><a href="#pone-0182377-t001">Table 1</a> </strong>shows the number of different 5-nt motifs detected in the analysed DNA sequences. Missense/nonsense and splicing mutations were taken from the HGMD. We started this analysis in 2014 and used HGMD mutation data from this year. To verify our approach, we repeated the analysis with new HGMD mutation data from 2016, which contains about 8% more mutations than in 2014 in the 5 analysed genes (see <strong><a href="#pone-0182377-t001">Table 1</a></strong>). For the 2014 and 2016 HGMD dataset, we carried out the following analysis. Various substitutions in the middle position (e.g. <span class="monospace">C&gt;T</span>, <span class="monospace">A</span>, <span class="monospace">G</span>) were recorded as one mutation. Our idea was to find out if a particular 5-nt segment is associated with mutations or not. These data were statistically evaluated to select representative 5-nt segments frequently containing mutations in the middle position (hotspots), and segments where the middle position rarely contained mutations (coldspots). Statistical evaluation was performed by R package (<a href="http://www.r-project.org/">www.r-project.org</a>). The proportional test was applied with an expected 75% probability for a hotspot or 25% for a coldspot, respectively. Individual p-values for a 5-nt segment and its complement in each gene were further combined into a single p-value by the Fisher method. We considered statistically significant segments with Fisher combined p-value &lt; 0.1. Obtained coldspots and hotspots from the 2014 and 2016 dataset were compared. Further, for the 2016 mutation dataset we performed additional statistical analysis where we considered all detected substitutions in the middle position. Each position in this approach can possess three substitutions and one original nucleotide. We again used a proportional test with an expected 75% probability for a hotspot or 25% for a coldspot, respectively. Individual p-values for a 5-nt segment and its complement were combined into a single p-value by the Fisher method.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3" class="link-target"></a><p>A nucleosome positioning was analyzed using a novel algorithm that combines a statistical mechanics model and knowledge of periodically occurring dinucleotides in histone octamers [<a href="#pone.0182377.ref042" class="ref-tip">42</a>].</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="MD simulations and free energy calculations"></a>
<h3>MD simulations and free energy calculations</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>To determine the bending properties of DNA with a mismatch base pair, we performed a conformational transition between the straight DNA duplex built by the nab module of AMBER 14 [<a href="#pone.0182377.ref039" class="ref-tip">39</a>] as a right handed B-DNA and the bent DNA conformation found in the complex with MutSα [<a href="#pone.0182377.ref026" class="ref-tip">26</a>] (PDB ID: 2O8B). Due to the complexity of the problem, the bending was performed in absence of MutSα. DNA with a mismatch <span class="monospace">G/T</span> pair built by the nab module was solvated with TIP3P water molecules [<a href="#pone.0182377.ref043" class="ref-tip">43</a>] in a truncated octahedral periodic box (with minimal distance 10 Å to the walls) and neutralized with sodium counterions [<a href="#pone.0182377.ref044" class="ref-tip">44</a>] using the xleap module of AMBER, a new version of the DNA force field parmbsc1 [<a href="#pone.0182377.ref045" class="ref-tip">45</a>] was used. Firstly, the system was equilibrated in three steps. Minimization of the system was performed in 3000 steps, then the system was heated to 300K during 100 ps at a constant volume and finally, a 500 ps long simulation was run at 300 K and 100 kPa at the NpT conditions. After that we ran production dynamics and the system was simulated for 150 ns at the NpT conditions. During the equilibration and production phases, dynamics of terminal base pairs was limited by the wall distance restraints imposed on hydrogen bonds between terminal bases to avoid the formation of flanking bases that would influence the dynamics of the remaining DNA structure. Five restart files from the production MD simulation (each taken after 30 ns) were used as the starting coordinates for the subsequent parallel ABF method accelerated by the MWA [<a href="#pone.0182377.ref040" class="ref-tip">40</a>,<a href="#pone.0182377.ref041" class="ref-tip">41</a>,<a href="#pone.0182377.ref046" class="ref-tip">46</a>]. This ensured independence of the starting configurations for ABF simulations. All ABF simulations were performed in the modified PMEMD program from AMBER, connected with PMFLib [<a href="#pone.0182377.ref047" class="ref-tip">47</a>] implementing both ABF and MWA methods. The total sampling time of one ABF simulation was 200 ns, which provides converged free energy profiles (we also tested 400 ns long simulations that provided close to identical results, see <strong><a href="#pone.0182377.s006">S1 Fig</a></strong> for comparison). The collective variable used in the free energy calculations was the mass weighted root-mean square distance (RMSD) to a target structure, which was derived from the bent X-ray DNA structure. We tested two coldspots and two hotspots that differ in a sequence of the central 5 nt-segment (<strong><a href="#pone-0182377-g002">Fig 2</a></strong>). Therefore, bases in the central 5-nt long segments of the bent X-ray DNA structure were mutated <em>in silico</em>. Possible clashes introduced by changes in the sequence were removed by careful optimization while the overall shape was maintained by positional restraints towards the bent X-ray DNA structure. To avoid possible bias which could be introduced by this <em>in silico</em> procedure, three types of RMSD collective variable were tested. They differ in the definition of atoms, which were employed in the structure superimposition and calculation of RMSD. Set A included all of the entire DNA structure’s heavy atoms corresponding to residues 1 to 30. Set B included all of the heavy atoms from terminal 5-nt long segments and the heavy atoms of the backbone from the central 5-nt long segments corresponding to the AMBER mask notation ((:1–5,11–20,26–30) | (:6–10,21–25@P,OP1,OP2,O3',O5',C3',C4',C5')) and, finally, set C included the DNA backbone heavy atoms corresponding to the AMBER mask notation (:1–30@P,OP1,OP2,O3',O5',C3',C4',C5'). In the bent X-ray DNA structure, the central <span class="monospace">G/T</span> pair is modestly opened towards minor groove, showing a shear parameter of about 5 Å (this parameter is one of the six helical parameters describing the orientation of bases in a base pair) [<a href="#pone.0182377.ref048" class="ref-tip">48</a>], while in the relaxed DNA conformations where it is positioned inside the duplex this parameter has a value of about -2.2 Å. Therefore, we imposed weak wall restraints to keep the shear parameter in the range from -10 Å to 0 Å so that the <span class="monospace">G/T</span> stayed stable inside the DNA duplex without changing the hydrogen pairing during DNA bending. As observed in test simulations, opening of the <span class="monospace">G/T</span> pair represents irreversible events which influence the calculated free energy profiles. The free energy profiles were calculated for RMSD in the interval from 1.45 to 5.45 Å. The relaxed DNA is shown as a minimum on the free energy profile, representing a stable thermodynamic state. On the contrary, the bent structure is not thermodynamically stable because of the absence of MutSα in our model. Therefore, we had to select a RMSD value which would represent the bend state and serve to compare coldspots and hotspots. To guess a fair estimate, we have analysed behaviour of the relaxed DNA state observed in the unrestrained production dynamics. RMSD towards an average DNA structure was calculated over the entire production dynamics. Depending on the system and the atom set employed in calculating RMSD, the RMSD value fluctuated from 1 to 4 Å, with maximum occurrence at about 1.55 Å for set A and B (<strong><a href="#pone.0182377.s007">S2 Fig</a>)</strong>. Set C exhibits a slightly higher value of about 1.80 Å. We assumed the similar effect of thermal fluctuations on RMSD deviation from the target bent DNA structures and used a value of 1.55 Å as an RMSD threshold representing the bent DNA.</p>
<a class="link-target" id="pone-0182377-g002" name="pone-0182377-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.g002" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> 2D structures of the studied DNA systems with a central <span class="monospace">G/T</span> pair using the ABF method.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>The sequence corresponds to DNA complexed with MutSα except the central 5-nt segment (highlighted by the colored box) which is either a coldspot (yellow box) or hotspot (pink box).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.g002">
              https://doi.org/10.1371/journal.pone.0182377.g002</a></p></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>We also analyzed bending of DNA duplex with <span class="monospace">G/T</span> using another collective variable used in the work of Sharma, et al [<a href="#pone.0182377.ref049" class="ref-tip">49</a>]. This collective variable is represented by the angle between helical arms of the DNA duplex defined by three centers of masses defined by nucleotide residues (2–5, 26–29), (6–10, 21–25), and (11–14, 17–20). This collective variable was tested in a range from 90° to 170°. Bend structures, however, did not resemble X-ray conformation of DNA in complex with MutSα, but rather showed smooth deformations along the helical axis of DNA (see <strong><a href="#pone.0182377.s008">S3 Fig</a></strong> for comparison with the bent X-ray DNA and a selected MD structure from the ABF calculation, where the collective variable was RMSD). Therefore, the bending angle was not used as a collective variable in our study. The MD trajectories were processed using the ptraj module of AMBER and visualized using the VMD program [<a href="#pone.0182377.ref050" class="ref-tip">50</a>].</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" data-toc="sec005" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Selected mutation motifs (coldspots and hotspots) and their signatures"></a>
<h3>Selected mutation motifs (coldspots and hotspots) and their signatures</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>DNA 5-nt segments with their complements were extracted from five studied genes. The number of identified unique segments in each gene is shown in <strong><a href="#pone-0182377-t001">Table 1</a></strong>. It can be seen that not all combinations (512) were found in these genes. Even <em>F8</em> and <em>CFTR</em> genes with the longest DNA sequences do not contain all possible combinations. There are several motifs which occur especially rarely, e.g. <span class="monospace">CCGCG/CGCGG, CGACG/CGTCG, ATACG/CGTAT, CGCGA/TCGCG, ACGCG/CGCGT, CGCGC/GCGCG</span>. These motifs were detected just in 1 out of 5 genes and are rich in <span class="monospace">CpG</span> dinucleotide. Their occurrence is most likely suppressed in DNA sequences due to the reasons discussed in the Introduction. <strong><a href="#pone.0182377.s001">S1</a> and <a href="#pone.0182377.s002">S2</a> Tables</strong> show all segments detected in <em>PAH</em>, <em>LDLR</em>, <em>F8</em>, <em>F9</em> and <em>CFTR</em> genes and a number of mutations found in the middle position in the 2014 and 2016 mutation datasets, respectively. With the use of proportional tests and the Fisher method, we calculated the combined p-value for each segment, which indicates its significance to be a coldspot or hotspot. <strong><a href="#pone-0182377-t002">Table 2</a> </strong>shows the top 20 coldspots with the lowest Fisher combined p-values in the 2014 and 2016 datasets and <strong><a href="#pone-0182377-t003">Table 3</a> </strong>shows the top 20 hotspots with the lowest Fisher combined p-values in the 2014 and 2016 datasets.</p>
<a class="link-target" id="pone-0182377-t002" name="pone-0182377-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.t002" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Top twenty coldspots in 2014 and 2016 datasets.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.t002">
              https://doi.org/10.1371/journal.pone.0182377.t002</a></p></div><a class="link-target" id="pone-0182377-t003" name="pone-0182377-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.t003" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Top twenty hotspots in 2014 and 2016 datasets.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.t003">
              https://doi.org/10.1371/journal.pone.0182377.t003</a></p></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>Coldspots derived from the 2014 dataset are almost identical to coldspots derived from the 2016 dataset. In particular, 18 out of 20 coldspots can be found in both datasets. Similarly, 19 hotspots from the 2014 dataset can be found in the 2016 dataset. Thus, although the number of detected mutations increased in the genes during the two years, it did not significantly affect the final selection of coldspots and hotspots.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>Herein, we discuss features of coldspots/hotspots found with the 2016 dataset. The most apparent feature of the detected coldspots is the presence of consecutive purines (purine tract) (<strong><a href="#pone-0182377-t002">Table 2</a></strong>). In particular, 18 out of 20 coldspots contain four or five purine tracts. Such a pattern is not seen in the hotspot sequences, where just one motif out of 20 contains a four purine tract (<span class="monospace">TGGAA</span>). The identified hotspots frequently show <span class="monospace">CpG</span> dinucleotide in the middle position (detected in 8 out of 20 motifs). In the top 20 coldspot sequences, this dinucleotide was not detected. Further, we used a sequence logo tool [<a href="#pone.0182377.ref051" class="ref-tip">51</a>] to visualize the sequence pattern for the 20 top coldspots and hotspots. Since each motif consists of forward and reverse part, only sequences (either forward or reverse) containing a purine base in the middle position were analysed. It revealed that adenine is the prevailing middle base in coldspots and it is surrounded by either adenines or guanines (<strong><a href="#pone-0182377-g003">Fig 3</a></strong>). In the case of hotspots, the prevailing central purine base is guanine and other positions are highly variable (<strong><a href="#pone-0182377-g003">Fig 3</a></strong>).</p>
<a class="link-target" id="pone-0182377-g003" name="pone-0182377-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.g003" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Sequence logo for top 20 coldspots (top) and top 20 hotspots (bottom).</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.g003">
              https://doi.org/10.1371/journal.pone.0182377.g003</a></p></div><a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4" class="link-target"></a><p>We also used another strategy to detect coldspots and hotspots. In particular, we considered all substitutions in the 5-nt motifs' middle position. The top 20 detected coldspots basically agree with our first analysis, where more substitutions in the middle position were counted as one mutation. In particular, 17 out of 20 coldspots with the lowest p-value were also found based on our first analysis (<strong><a href="#pone-0182377-t002">Table 2</a> and <a href="#pone.0182377.s003">S3 Table</a></strong>). Considering hotspots, the top 20 motifs show rather high combined p-values (<strong><a href="#pone.0182377.s003">S3 Table</a></strong>). Based on this second statistical approach, significant hotspots should contain many different substitutions on all of their occurrences in the genes. We observed that this is a rather rare event. As in our first approach, we considered statistically significant motifs with a combined p-value &lt;0.1. There are only five hotspots which satisfy this criterion (first three motifs out of these five were also found with our previous approach (<strong><a href="#pone-0182377-t003">Table 3</a> and <a href="#pone.0182377.s003">S3 Table</a></strong>). Thus, this second statistical approach might be more useful when more data (substitutions) are available in the HGMD.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5" class="link-target"></a><p>In <strong><a href="#pone-0182377-g004">Fig 4</a></strong>, we show a number of mutations (different substitutions counted as one), the number of all substitutions, and occurrences of the top 20 coldspots/hotspots detected using our first statistical approach (results in <strong>Tables <a href="#pone-0182377-t002">2</a> and <a href="#pone-0182377-t003">3</a></strong>) with the HGMD 2016 dataset. It can be seen that coldspots occur more frequently than hotspots and contain significantly less mutations, particularly, coldspots occur approximately two times more frequently and contain two times less mutations than hotspots. The figure also shows that hotspots more often possess different substitutions than coldspots (see red and blue columns in the <strong><a href="#pone-0182377-g004">Fig 4</a></strong>). In addition, for the top 20 coldspots/hotspots we extracted the type of base substitutions. In the case of coldspots where adenine occurs predominantly in the middle position, the most frequent substitution is <span class="monospace">A→G</span>, while in the hotspots where guanine occurs predominantly in the middle position we observed that <span class="monospace">G→A</span> is the most frequent substitution (<strong><a href="#pone.0182377.s009">S4 Fig</a></strong>). This corresponds to the general assumption that transitions are more frequent than transversions [<a href="#pone.0182377.ref052" class="ref-tip">52</a>].</p>
<a class="link-target" id="pone-0182377-g004" name="pone-0182377-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.g004" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Occurrences of top 20 coldspots (top) and top 20 hotspots (bottom) in the five studied genes visualized with the number of detected mutations and substitutions in middle position.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.g004">
              https://doi.org/10.1371/journal.pone.0182377.g004</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec007" name="sec007" class="link-target" title="Coldspots and hotspots in the TP53 gene"></a>
<h3>Coldspots and hotspots in the <em>TP53</em> gene</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>We analyzed the behaviour of the top twenty coldspots and hotspots selected based on the 5 genes in the <em>TP53</em> gene, which also contains many different germline mutations along the nucleotide sequence. Germline TP53 mutations were taken from the HGMD database (germline TP53 mutations are also presented in the IARC database <a href="http://p53.iarc.fr/">http://p53.iarc.fr/</a> together with TP53 somatic mutations). We analyzed mutations in the exons and included 4-nt long intron segments plus 2 nt before the first codon and 2 nt after the last codon as for analysis of the five genes. The total TP53 length was 1255 nt and there were 302 mutations, see <strong><a href="#pone.0182377.s004">S4 Table</a></strong>. We observed that only 1% of coldspot sequences are associated with mutations while in the case of hotspots it is 21% (<strong><a href="#pone.0182377.s005">S5 Table</a></strong>). Regarding sequences which are neither hotspots nor coldspots (they have a combined p-value &gt; 0.1 in both coldspot and hotspot datasets), 12% of them are associated with mutations. This is higher than for coldspots but lower than for hotspots (if we consider all motifs with a combined p-value &gt; 0.1 in both datasets, we also get that 12% of them are associated with mutations). These findings indicate our motif selection can also be applied to other genes.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="Bending analysis of coldspots and hotspots"></a>
<h3>Bending analysis of coldspots and hotspots</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>We investigated the bending properties of two coldspots, <span class="monospace">AAGAA</span> and <span class="monospace">CAGTG</span>, and two hotspots, <span class="monospace">AGGTA</span> and <span class="monospace">TGGAA</span>. The selected coldspots were detected in the statistical analysis of the HGMD 2014 dataset (<strong><a href="#pone-0182377-t002">Table 2</a></strong>). The first coldspot was also identified within the first 20 coldspots in the 2016 dataset, the second one occurs at the 23<sup>rd</sup> position in the HGMD 2016 dataset, but its combined Fisher p-value is still very low (9.22E-05). The selected hotspots were both detected in the 2014 and 2016 datasets within the top 20 hotspots (<strong><a href="#pone-0182377-t003">Table 3</a></strong>). We intentionally selected hotspots that do not contain <span class="monospace">CpG</span> dinucleotide in the middle position. The presence of such hotspots among our top 20 hotspots could be due to high <span class="monospace">CpG</span> site mutability rather than due to intrinsic flexibility, which was tested in our calculations.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>The bending was quantified by the free energy change (<strong><a href="#pone-0182377-g005">Fig 5</a></strong>) that is necessary to bring a relaxed straight DNA to a conformation observed in the bent X-ray DNA structure found in the MutSα/DNA complex. As a collective variable describing the necessary geometrical change, we employed RMSD towards the target bent DNA structure. The relaxed DNA, which is thermodynamically stable state, is exhibited as a minimum on the free energy profiles around 4.5 Å, while the bend DNA lies in the region around 1.55 Å, which is a typical deviation of RMSD from the target structure due to thermal atom fluctuations. The free energy profiles show that the two hotspots with <span class="monospace">G/T</span> pairs are stiffer than the two coldspots (<strong><a href="#pone-0182377-g005">Fig 5</a></strong>). For instance, the free energy necessary to bend a straight DNA is 16.3 for <span class="monospace">AGGTA</span> and 16.1 for <span class="monospace">TGGAA</span> while for <span class="monospace">AAGAA</span> and <span class="monospace">CAGTG</span> it is only 14.2 and 14.3 kcal mol<sup>-1</sup>, respectively.</p>
<a class="link-target" id="pone-0182377-g005" name="pone-0182377-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0182377.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0182377.g005" data-doi="info:doi/10.1371/journal.pone.0182377" data-uri="info:doi/10.1371/journal.pone.0182377.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0182377.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0182377.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0182377.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0182377.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Free energy change for two coldspots and two hotspots with <span class="monospace">G/T</span> pair with respect to DNA bending described by RMSD calculated over the set A atoms.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0182377.g005">
              https://doi.org/10.1371/journal.pone.0182377.g005</a></p></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3" class="link-target"></a><p>Due to the different composition of central 5-nt long segments, the target bent DNA structures were derived from the experimental bent X-ray structure by <em>in silico</em> mutagenesis. To examine, if this procedure could negatively influence the observed DNA bending properties, we employed three different sets of atoms in the RMSD calculation. The first set (set A) contains all the DNA’s heavy atom (<strong><a href="#pone-0182377-g005">Fig 5</a></strong>), while the atoms of modified nucleobases in the central regions were excluded in the second set (set B). The free energy profiles calculated for set B show nearly the same results (<strong><a href="#pone.0182377.s010">S5 Fig</a></strong>) as observed for set A. It indicates that the preparation of the target structure does not alter the bending properties and observed hotspots and coldspot flexibility, hence the flexibility is indeed an intrinsic property of the nucleotide sequence. We also tested the third set (set C), where only the DNA’s backbone atoms were considered (<strong><a href="#pone.0182377.s011">S6 Fig</a></strong>). Even here, where nucleobase sequences cannot alter calculation of RMSD and thus the free energy profiles, the difference between coldspots and hotspots is visible. However, the discrimination of coldspots and hotspots is significantly smaller mainly because the bending of DNA was not as sharp as in the two other atom sets.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Nucleosome positioning"></a>
<h3>Nucleosome positioning</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Repair of DNA sequences in linker regions is more efficient than in the nucleosomes [<a href="#pone.0182377.ref053" class="ref-tip">53</a>], therefore we analyzed the putative positioning of nucleosomes on two coldspots (<span class="monospace">AAGAA</span> and <span class="monospace">CAGTG</span>), and two hotspots (<span class="monospace">AGGTA</span> and <span class="monospace">TGGAA</span>) in exons of the <em>PAH</em> gene (the same motifs employed in the bending analysis). <strong><a href="#pone.0182377.s012">S7 Fig</a></strong> shows the localization of these motifs in the PAH exons mapped onto the predicted nucleosome positions. We observed a roughly equal distribution of both coldspots and hotspots in the nucleosomes and in the linker regions. In particular, the predicted localization of the selected coldspots and hotspots inside/outside the nucleosome is the following: <span class="monospace">AAGAA</span> (4/7), <span class="monospace">CAGTG</span> (6/5), <span class="monospace">AGGTA</span> (1/3), <span class="monospace">TGGAA</span> (4/6). These findings indicate no association between the repair process of these motifs and their localization in a chromatin.</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="sec010" name="sec010" data-toc="sec010" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>In the five genes (<em>PAH</em>, <em>LDLR</em>, <em>CFTR</em>, <em>F8</em>, and <em>F9</em>) leading to common inherited disorders we detected sequences (mutational motifs) rarely associated with SNVs (coldspots) and frequently associated with SNVs (hotspots). Our approach is based on the analysis of mutations obtained from the HGMD database in the 5-nt segments which repeatedly occur in DNA sequences. It contrasts with the common strategy, where hotspots are derived from a mutation spectrum, which is a distribution of frequencies of every type of mutation along nucleotide sequences of a target gene [<a href="#pone.0182377.ref010" class="ref-tip">10</a>,<a href="#pone.0182377.ref054" class="ref-tip">54</a>]. The occurrence of mutations in the mutation spectra, however, often reflects the particular population where an effect of a common ancestor plays a role, i.e. frequent mutations originate from a single mutation event (founder effect) and are spread throughout the population as observed in our [<a href="#pone.0182377.ref055" class="ref-tip">55</a>] and other studies [<a href="#pone.0182377.ref056" class="ref-tip">56</a>,<a href="#pone.0182377.ref057" class="ref-tip">57</a>]. Utilizing mutation spectra to detect coldspots is not convenient [<a href="#pone.0182377.ref010" class="ref-tip">10</a>].</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>Based on our approach, we observed that the majority of the top 20 coldspots (18 out of 20) contain purine tract, i.e. four to five nt long purine sequence. These sites were associated with a minimum of SNVs. In contrast, the purines tracts were not seen in the top 20 hotspots. These sequences often contain <em>alternating purine</em>-pyrimidine bases. Nine out of the top 20 hotspots showed <span class="monospace">CpG</span> dinucleotide in the middle position (<strong><a href="#pone-0182377-t003">Table 3</a></strong>), which can explain their higher mutation rate.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>We performed MD simulations and free energy calculations to better understand the bending properties of DNA with mutations. We analyzed <span class="monospace">AAGAA</span> and <span class="monospace">CAGTG</span> coldspots and <span class="monospace">AGGTA</span> and <span class="monospace">TGGAA</span> hotspots, which does not contain <span class="monospace">CpG</span> (<strong><a href="#pone-0182377-g002">Fig 2</a></strong>). We hypothesized that higher mutability of hotspots could be due to their higher stiffness. Indeed, we observed that the selected coldspots with mismatch <span class="monospace">G/T</span> are about 2 kcal mol<sup>-1</sup> more flexible than the selected hotspots with <span class="monospace">G/T</span> (<strong><a href="#pone-0182377-g005">Fig 5</a></strong>). This supports the idea that flexible sequences could be more effectively repaired by MMR. We did not analyse bending properties of the DNA duplex with a central canonical pair as there is no experimentally bent structure with this pair that could serve as a target conformation for our calculations.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>As mentioned above, the selected coldspots consist of purine tracts where mononucleotide stretches frequently occur, e.g. <span class="monospace">AAAGA</span>, <span class="monospace">AAAAA</span>, <span class="monospace">AGAAA</span>, etc (<strong><a href="#pone-0182377-t002">Table 2</a></strong>). It is known that such sequences tend to cause DNA polymerase slippage during DNA replication, which results in IDLs [<a href="#pone.0182377.ref058" class="ref-tip">58</a>]. With the use of HGMD we can find a number of IDLs in our coldspots. It is known that the mutation rate of IDLs is lower than that of substitutions, however, homonucleotide tracts represent an exception [<a href="#pone.0182377.ref059" class="ref-tip">59</a>]. We assume that associating coldspots with IDLs is due to the increased mutation rate of these sites for this error type, even though mismatches in these sequences might be effectively repaired. The rate of mutations in IDLs can be further modulated by the length of a repetitive tract and base composition [<a href="#pone.0182377.ref060" class="ref-tip">60</a>–<a href="#pone.0182377.ref062" class="ref-tip">62</a>].</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>In summary, our study detected DNA mutation motifs rarely associated with germinal SNVs (coldspots) and motifs which are very frequently associated with SNVs (hotspots). For two selected coldspots it was shown that they are inherently more flexible than hotspots. To conclude that coldspots are generally more flexible than hotspots, so they could be more effectively repaired by MMR, the bending properties of more systems with various mismatches have to be investigated. We are going to analyse other sequences in the next study. We would also like to focus on hotspots with <span class="monospace">CpG</span> dinucleotide. It is possible that certain sequences with <span class="monospace">CpG</span> will be flexible while others can be very stiff, which would make them super-hotspots in combination with the high mutability of <span class="monospace">CpG</span>. We assume that knowledge of DNA motifs, which are extremely ineffectively repaired, can help to identify potentially causal mutations in introns. For these mutations, analysis of transcripts or minigene assays could be done to reveal their impact [<a href="#pone.0182377.ref063" class="ref-tip">63</a>,<a href="#pone.0182377.ref064" class="ref-tip">64</a>]. More importantly, this is essential in cases when no mutations for patients with a clear clinical phenotype are detected.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec011" name="sec011" data-toc="sec011" class="link-target" title="Supporting information"></a><h2>Supporting information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0182377"></div><div class="supplementary-material"><a name="pone.0182377.s001" id="pone.0182377.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s001">S1 Table. </a>DNA motifs (5-nt segments and their complements) detected in 5 genes and a number of mutations associated with middle positions in HGMD 2014 dataset.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s001">https://doi.org/10.1371/journal.pone.0182377.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s002" id="pone.0182377.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s002">S2 Table. </a>DNA motifs (5-nt segments and their complements) detected in 5 genes and number of mutations associated with middle positions in HGMD 2016 dataset.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s002">https://doi.org/10.1371/journal.pone.0182377.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s003" id="pone.0182377.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s003">S3 Table. </a>Top twenty coldspots and hotspots in 2016 dataset where all substitutions were taken into account.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s003">https://doi.org/10.1371/journal.pone.0182377.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s004" id="pone.0182377.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s004">S4 Table. </a>Updated <a href="#pone-0182377-t001">Table 1</a> about <em>TP53</em> gene.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s004">https://doi.org/10.1371/journal.pone.0182377.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s005" id="pone.0182377.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s005">S5 Table. </a>Number of occurrences and mutations of top 20 coldspots and top 20 hotspots in the <em>TP53</em> gene.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s005">https://doi.org/10.1371/journal.pone.0182377.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s006" id="pone.0182377.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s006">S1 Fig. </a>Comparison of free energy profiles of 200 ns and 400 ns long ABF calculations run for two coldspots and two hotspots with <span class="monospace">G/T</span> mismatch where we used set A for calculation of RMSD.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s006">https://doi.org/10.1371/journal.pone.0182377.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s007" id="pone.0182377.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s007">S2 Fig. </a>Distribution of RMSD values observed for a relaxed DNA in the unrestrained production dynamics of tested coldspots and hotspots.</h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI">The distribution is calculated with respect to an average DNA structure. Set A (red) and B (blue) show a maximum at 1.55 Å while set C (green) exhibits a maximum shifted to 1.8 Å.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s007">https://doi.org/10.1371/journal.pone.0182377.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s008" id="pone.0182377.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s008">S3 Fig. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI">A) X-ray DNA structure from DNA/MutSα complex. B) Snapshot MD structure from ABF simulation where we used set B for calculation of RMSD, actual RMSD value of the snapshot is 1.55 Å. C) Snapshot MD structure from ABF simulation where collective variable was angle among three mass centres, actual angle value of the snapshot is 98°.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s008">https://doi.org/10.1371/journal.pone.0182377.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s009" id="pone.0182377.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s009">S4 Fig. </a>Nucleotide substitutions detected in the middle position in top 20 coldspots (left) and hotspots (right).</h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI">Each column shows individual substitutions in the motif analyzed in the 5 genes. Nucleotides with percentage indicate total sum of particular base substitution.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s009">https://doi.org/10.1371/journal.pone.0182377.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s010" id="pone.0182377.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s010">S5 Fig. </a>Free energy profiles for two coldspots and two hotspots with <span class="monospace">G/T</span> pair where we used set B for calculation of RMSD.</h3><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1" class="link-target"></a><p class="preSiDOI">At 1.55 Å the free energy change for <span class="monospace">AGGTA</span>, <span class="monospace">TGGAA</span>, <span class="monospace">AAGAA</span> and <span class="monospace">CAGTG</span> is 14.1, 14.2, 12.8, and 11.8 kcal mol<sup>-1</sup>, respectively.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s010">https://doi.org/10.1371/journal.pone.0182377.s010</a></p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s011" id="pone.0182377.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s011">S6 Fig. </a>Free energy profiles for two coldspots and two hotspots with <span class="monospace">G/T</span> pair where we used set C for calculation of RMSD.</h3><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1" class="link-target"></a><p class="preSiDOI">At 1.55 Å the free energy for <span class="monospace">AGGTA</span>, <span class="monospace">TGGAA</span>, <span class="monospace">AAGAA</span> and <span class="monospace">CAGTG</span> is 19.5, 18.7, 17.9, and 17.5 kcal mol<sup>-1</sup>, respectively.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s011">https://doi.org/10.1371/journal.pone.0182377.s011</a></p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0182377.s012" id="pone.0182377.s012" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0182377.s012">S7 Fig. </a>Predicted positioning of nucleosomes on two coldspots (C), <span class="monospace">AAGAA</span> and <span class="monospace">CAGTG</span>, and two hotspots (H), <span class="monospace">AGGTA</span> and <span class="monospace">TGGAA</span>, in the exons of the <em>PAH</em> gene.</h3><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1" class="link-target"></a><p class="preSiDOI">Nucleosome positions are visualized by yellow peaks, exon positions are colored by grey bars. Each plot shows 1000-nt long fragment. Motifs were considered on nucleosome with occupancy &gt; 0.005.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0182377.s012">https://doi.org/10.1371/journal.pone.0182377.s012</a></p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div></div>



<div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0182377.ref001" id="pone.0182377.ref001" class="link-target"></a>Baarends WM, van der Laan R, Grootegoed JA, DNA repair mechanisms and gametogenesis. Reproduction. 2001;121: 31–39. pmid:11226027 <ul class="reflinks"><li><a href="#" data-author="Baarends" data-cit="BaarendsWM%2C%20van%20der%20LaanR%2C%20GrootegoedJA%2C%20DNA%20repair%20mechanisms%20and%20gametogenesis.%20Reproduction.%202001%3B121%3A%2031%E2%80%9339.%2011226027" data-title="DNA%20repair%20mechanisms%20and%20gametogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11226027" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+repair+mechanisms+and+gametogenesis+Baarends+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0182377.ref002" id="pone.0182377.ref002" class="link-target"></a>Iyer RR, Pluciennik A, V B, Modrich PL, DNA mismatch repair: Functions and mechanisms. Chem Rev. 2006;106: 302–323.  pmid:16464007 <ul class="reflinks"><li><a href="#" data-author="Iyer" data-cit="IyerRR%2C%20PluciennikA%2C%20VB%2C%20ModrichPL%2C%20DNA%20mismatch%20repair%3A%20Functions%20and%20mechanisms.%20Chem%20Rev.%202006%3B106%3A%20302%E2%80%93323.%20doi%3A%2010.1021%2Fcr0404794%2016464007" data-title="DNA%20mismatch%20repair%3A%20Functions%20and%20mechanisms" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16464007" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+mismatch+repair%3A+Functions+and+mechanisms+Iyer+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0182377.ref003" id="pone.0182377.ref003" class="link-target"></a>Hanawalt PC, Spivak G, Transcription-coupled DNA repair: two decades of progress and surprises. Nat Rev Mol Cell Biol. 2008;9: 958–970.  pmid:19023283 <ul class="reflinks"><li><a href="#" data-author="Hanawalt" data-cit="HanawaltPC%2C%20SpivakG%2C%20Transcription-coupled%20DNA%20repair%3A%20two%20decades%20of%20progress%20and%20surprises.%20Nat%20Rev%20Mol%20Cell%20Biol.%202008%3B9%3A%20958%E2%80%93970.%20doi%3A%2010.1038%2Fnrm2549%2019023283" data-title="Transcription-coupled%20DNA%20repair%3A%20two%20decades%20of%20progress%20and%20surprises" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19023283" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Transcription-coupled+DNA+repair%3A+two+decades+of+progress+and+surprises+Hanawalt+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0182377.ref004" id="pone.0182377.ref004" class="link-target"></a>Touchon M, Arneodo A, d'Aubenton-Carafa Y, Thermes C, Transcription-coupled and splicing-coupled strand asymmetries in eukaryotic genomes. Nucleic Acids Res. 2004;32: 4969–4978.  pmid:15388799 <ul class="reflinks"><li><a href="#" data-author="Touchon" data-cit="TouchonM%2C%20ArneodoA%2C%20d%27Aubenton-CarafaY%2C%20ThermesC%2C%20Transcription-coupled%20and%20splicing-coupled%20strand%20asymmetries%20in%20eukaryotic%20genomes.%20Nucleic%20Acids%20Res.%202004%3B32%3A%204969%E2%80%934978.%20doi%3A%2010.1093%2Fnar%2Fgkh823%2015388799" data-title="Transcription-coupled%20and%20splicing-coupled%20strand%20asymmetries%20in%20eukaryotic%20genomes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15388799" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Transcription-coupled+and+splicing-coupled+strand+asymmetries+in+eukaryotic+genomes+Touchon+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0182377.ref005" id="pone.0182377.ref005" class="link-target"></a>Touchon M, Nicolay S, Arneodo A, d'Aubenton-Carafa Y, Thermes C, Transcription-coupled TA and GC strand asymmetries in the human genome. Febs Lett. 2003;555: 579–582. pmid:14675777 <ul class="reflinks"><li><a href="#" data-author="Touchon" data-cit="TouchonM%2C%20NicolayS%2C%20ArneodoA%2C%20d%27Aubenton-CarafaY%2C%20ThermesC%2C%20Transcription-coupled%20TA%20and%20GC%20strand%20asymmetries%20in%20the%20human%20genome.%20Febs%20Lett.%202003%3B555%3A%20579%E2%80%93582.%2014675777" data-title="Transcription-coupled%20TA%20and%20GC%20strand%20asymmetries%20in%20the%20human%20genome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14675777" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Transcription-coupled+TA+and+GC+strand+asymmetries+in+the+human+genome+Touchon+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0182377.ref006" id="pone.0182377.ref006" class="link-target"></a>Brodie of Brodie EB, Nicolay S, Touchon M, Audit B, d'Aubenton-Carafa Y, Thermes C, et al., From DNA sequence analysis to modeling replication in the human genome. Phys Rev Lett. 2005;94. <ul class="reflinks"><li><a href="#" data-author="Brodie%20of%20Brodie" data-cit="Brodie%20of%20BrodieEB%2C%20NicolayS%2C%20TouchonM%2C%20AuditB%2C%20d%27Aubenton-CarafaY%2C%20ThermesC%2C%20et%20al.%2C%20From%20DNA%20sequence%20analysis%20to%20modeling%20replication%20in%20the%20human%20genome.%20Phys%20Rev%20Lett.%202005%3B94." data-title="From%20DNA%20sequence%20analysis%20to%20modeling%20replication%20in%20the%20human%20genome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=From+DNA+sequence+analysis+to+modeling+replication+in+the+human+genome+Brodie+of+Brodie+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0182377.ref007" id="pone.0182377.ref007" class="link-target"></a>Chen CL, Duquenne L, Audit B, Guilbaud G, Rappailles A, Baker A, et al., Replication-Associated Mutational Asymmetry in the Human Genome. Mol Biol Evol. 2011;28: 2327–2337.  pmid:21368316 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenCL%2C%20DuquenneL%2C%20AuditB%2C%20GuilbaudG%2C%20RappaillesA%2C%20BakerA%2C%20et%20al.%2C%20Replication-Associated%20Mutational%20Asymmetry%20in%20the%20Human%20Genome.%20Mol%20Biol%20Evol.%202011%3B28%3A%202327%E2%80%932337.%20doi%3A%2010.1093%2Fmolbev%2Fmsr056%2021368316" data-title="Replication-Associated%20Mutational%20Asymmetry%20in%20the%20Human%20Genome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21368316" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Replication-Associated+Mutational+Asymmetry+in+the+Human+Genome+Chen+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0182377.ref008" id="pone.0182377.ref008" class="link-target"></a>Touchon M, Nicolay S, Audit B, Brodie Brodie EB, d'Aubenton-Carafa Y, Arneodo A, et al., Replication-associated strand asymmetries in mammalian genomes: Toward detection of replication origins. Proc Natl Acad Sci U S A. 2005;102: 9836–9841.  pmid:15985556 <ul class="reflinks"><li><a href="#" data-author="Touchon" data-cit="TouchonM%2C%20NicolayS%2C%20AuditB%2C%20Brodie%20BrodieEB%2C%20d%27Aubenton-CarafaY%2C%20ArneodoA%2C%20et%20al.%2C%20Replication-associated%20strand%20asymmetries%20in%20mammalian%20genomes%3A%20Toward%20detection%20of%20replication%20origins.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202005%3B102%3A%209836%E2%80%939841.%20doi%3A%2010.1073%2Fpnas.0500577102%2015985556" data-title="Replication-associated%20strand%20asymmetries%20in%20mammalian%20genomes%3A%20Toward%20detection%20of%20replication%20origins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15985556" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Replication-associated+strand+asymmetries+in+mammalian+genomes%3A+Toward+detection+of+replication+origins+Touchon+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0182377.ref009" id="pone.0182377.ref009" class="link-target"></a>Cooper DN, Bacolla A, Ferec C, Vasquez KM, Kehrer-Sawatzki H, Chen JM, On the Sequence-Directed Nature of Human Gene Mutation: The Role of Genomic Architecture and the Local DNA Sequence Environment in Mediating Gene Mutations Underlying Human Inherited Disease. Human Mut. 2011;32: 1075–1099. <ul class="reflinks"><li><a href="#" data-author="Cooper" data-cit="CooperDN%2C%20BacollaA%2C%20FerecC%2C%20VasquezKM%2C%20Kehrer-SawatzkiH%2C%20ChenJM%2C%20On%20the%20Sequence-Directed%20Nature%20of%20Human%20Gene%20Mutation%3A%20The%20Role%20of%20Genomic%20Architecture%20and%20the%20Local%20DNA%20Sequence%20Environment%20in%20Mediating%20Gene%20Mutations%20Underlying%20Human%20Inherited%20Disease.%20Human%20Mut.%202011%3B32%3A%201075%E2%80%931099." data-title="On%20the%20Sequence-Directed%20Nature%20of%20Human%20Gene%20Mutation%3A%20The%20Role%20of%20Genomic%20Architecture%20and%20the%20Local%20DNA%20Sequence%20Environment%20in%20Mediating%20Gene%20Mutations%20Underlying%20Human%20Inherited%20Disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=On+the+Sequence-Directed+Nature+of+Human+Gene+Mutation%3A+The+Role+of+Genomic+Architecture+and+the+Local+DNA+Sequence+Environment+in+Mediating+Gene+Mutations+Underlying+Human+Inherited+Disease+Cooper+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0182377.ref010" id="pone.0182377.ref010" class="link-target"></a>Rogozin IB, Pavlov YI, Theoretical analysis of mutation hotspots and their DNA sequence context specificity. Mutat Res-Rev Mutat Res. 2003;544: 65–85. <ul class="reflinks"><li><a href="#" data-author="Rogozin" data-cit="RogozinIB%2C%20PavlovYI%2C%20Theoretical%20analysis%20of%20mutation%20hotspots%20and%20their%20DNA%20sequence%20context%20specificity.%20Mutat%20Res-Rev%20Mutat%20Res.%202003%3B544%3A%2065%E2%80%9385." data-title="Theoretical%20analysis%20of%20mutation%20hotspots%20and%20their%20DNA%20sequence%20context%20specificity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Theoretical+analysis+of+mutation+hotspots+and+their+DNA+sequence+context+specificity+Rogozin+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0182377.ref011" id="pone.0182377.ref011" class="link-target"></a>Youssoufian H, Kazazian HH, Phillips DG, Aronis S, Tsiftis G, Brown VA, et al., Recurrent Mutations in Hemophilia-a Give Evidence for CpG Mutation Hotspots. Nature. 1986;324: 380–382.  pmid:3097553 <ul class="reflinks"><li><a href="#" data-author="Youssoufian" data-cit="YoussoufianH%2C%20KazazianHH%2C%20PhillipsDG%2C%20AronisS%2C%20TsiftisG%2C%20BrownVA%2C%20et%20al.%2C%20Recurrent%20Mutations%20in%20Hemophilia-a%20Give%20Evidence%20for%20CpG%20Mutation%20Hotspots.%20Nature.%201986%3B324%3A%20380%E2%80%93382.%20doi%3A%2010.1038%2F324380a0%203097553" data-title="Recurrent%20Mutations%20in%20Hemophilia-a%20Give%20Evidence%20for%20CpG%20Mutation%20Hotspots" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/3097553" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Recurrent+Mutations+in+Hemophilia-a+Give+Evidence+for+CpG+Mutation+Hotspots+Youssoufian+1986" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0182377.ref012" id="pone.0182377.ref012" class="link-target"></a>Coulondre C, Miller JH, Farabaugh PJ, Gilbert W, Molecular-Basis of Base Substitution Hotspots in Escherichia-Coli. Nature. 1978;274: 775–780. pmid:355893 <ul class="reflinks"><li><a href="#" data-author="Coulondre" data-cit="CoulondreC%2C%20MillerJH%2C%20FarabaughPJ%2C%20GilbertW%2C%20Molecular-Basis%20of%20Base%20Substitution%20Hotspots%20in%20Escherichia-Coli.%20Nature.%201978%3B274%3A%20775%E2%80%93780.%20355893" data-title="Molecular-Basis%20of%20Base%20Substitution%20Hotspots%20in%20Escherichia-Coli" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/355893" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular-Basis+of+Base+Substitution+Hotspots+in+Escherichia-Coli+Coulondre+1978" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0182377.ref013" id="pone.0182377.ref013" class="link-target"></a>Cooper DN, Krawczak M (1993) Human Gene Mutation. Oxford: BIOS Scientific Publishers. <ul class="find-nolinks"></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0182377.ref014" id="pone.0182377.ref014" class="link-target"></a>Neuberger MS, Harris RS, Di Noia J, Petersen-Mahrt SK, Immunity through DNA deamination. Trends Biochem Sci. 2003;28: 305–312.  pmid:12826402 <ul class="reflinks"><li><a href="#" data-author="Neuberger" data-cit="NeubergerMS%2C%20HarrisRS%2C%20Di%20NoiaJ%2C%20Petersen-MahrtSK%2C%20Immunity%20through%20DNA%20deamination.%20Trends%20Biochem%20Sci.%202003%3B28%3A%20305%E2%80%93312.%20doi%3A%2010.1016%2FS0968-0004%2803%2900111-7%2012826402" data-title="Immunity%20through%20DNA%20deamination" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12826402" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Immunity+through+DNA+deamination+Neuberger+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0182377.ref015" id="pone.0182377.ref015" class="link-target"></a>Ollila J, Lappalainen I, Vihinen M, Sequence specificity in CpG mutation hotspots. FEBS Lett. 1996;396: 119–122. pmid:8914970 <ul class="reflinks"><li><a href="#" data-author="Ollila" data-cit="OllilaJ%2C%20LappalainenI%2C%20VihinenM%2C%20Sequence%20specificity%20in%20CpG%20mutation%20hotspots.%20FEBS%20Lett.%201996%3B396%3A%20119%E2%80%93122.%208914970" data-title="Sequence%20specificity%20in%20CpG%20mutation%20hotspots" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8914970" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence+specificity+in+CpG+mutation+hotspots+Ollila+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0182377.ref016" id="pone.0182377.ref016" class="link-target"></a>Bird AP DNA Methylation and the Frequency of CpG in Animal DNA. Nucleic Acids Res. 1980;8: 1499–1504. <ul class="reflinks"><li><a href="#" data-author="Bird%20AP" data-cit="Bird%20APDNA%20Methylation%20and%20the%20Frequency%20of%20CpG%20in%20Animal%20DNA.%20Nucleic%20Acids%20Res.%201980%3B8%3A%201499%E2%80%931504." data-title="Methylation%20and%20the%20Frequency%20of%20CpG%20in%20Animal%20DNA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Methylation+and+the+Frequency+of+CpG+in+Animal+DNA+Bird+AP+1980" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0182377.ref017" id="pone.0182377.ref017" class="link-target"></a>Bird A, The Essentials of DNA Methylation. Cell. 1992;70: 5–8. pmid:1377983 <ul class="reflinks"><li><a href="#" data-author="Bird" data-cit="BirdA%2C%20The%20Essentials%20of%20DNA%20Methylation.%20Cell.%201992%3B70%3A%205%E2%80%938.%201377983" data-title="The%20Essentials%20of%20DNA%20Methylation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1377983" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+Essentials+of+DNA+Methylation+Bird+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0182377.ref018" id="pone.0182377.ref018" class="link-target"></a>Kass SU, Landsberger N, Wolffe AP, DNA methylation directs a time dependent repression of transcription initiation. Curr Biol. 1997;7: 157–165. pmid:9395433 <ul class="reflinks"><li><a href="#" data-author="Kass" data-cit="KassSU%2C%20LandsbergerN%2C%20WolffeAP%2C%20DNA%20methylation%20directs%20a%20time%20dependent%20repression%20of%20transcription%20initiation.%20Curr%20Biol.%201997%3B7%3A%20157%E2%80%93165.%209395433" data-title="DNA%20methylation%20directs%20a%20time%20dependent%20repression%20of%20transcription%20initiation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9395433" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+methylation+directs+a+time+dependent+repression+of+transcription+initiation+Kass+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0182377.ref019" id="pone.0182377.ref019" class="link-target"></a>Cooper DN, Mort M, Stenson PD, Ball EV, Chuzhanova NA, Methylation-mediated deamination of 5-methylcytosine appears to give rise to mutations causing human inherited disease in CpNpG trinucleotides, as well as in CpG dinucleotides. Hum Genomics. 2010;4: 406–410.  pmid:20846930 <ul class="reflinks"><li><a href="#" data-author="Cooper" data-cit="CooperDN%2C%20MortM%2C%20StensonPD%2C%20BallEV%2C%20ChuzhanovaNA%2C%20Methylation-mediated%20deamination%20of%205-methylcytosine%20appears%20to%20give%20rise%20to%20mutations%20causing%20human%20inherited%20disease%20in%20CpNpG%20trinucleotides%2C%20as%20well%20as%20in%20CpG%20dinucleotides.%20Hum%20Genomics.%202010%3B4%3A%20406%E2%80%93410.%20doi%3A%2010.1186%2F1479-7364-4-6-406%2020846930" data-title="Methylation-mediated%20deamination%20of%205-methylcytosine%20appears%20to%20give%20rise%20to%20mutations%20causing%20human%20inherited%20disease%20in%20CpNpG%20trinucleotides%2C%20as%20well%20as%20in%20CpG%20dinucleotides" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20846930" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Methylation-mediated+deamination+of+5-methylcytosine+appears+to+give+rise+to+mutations+causing+human+inherited+disease+in+CpNpG+trinucleotides%2C+as+well+as+in+CpG+dinucleotides+Cooper+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0182377.ref020" id="pone.0182377.ref020" class="link-target"></a>Chuzhanova NA, Anassis EJ, Ball EV, Krawczak M, Cooper DN, Meta-analysis of indels causing human genetic disease: Mechanisms of mutagenesis and the role of local DNA sequence complexity. Human Mut. 2003;21: 28–44. <ul class="reflinks"><li><a href="#" data-author="Chuzhanova" data-cit="ChuzhanovaNA%2C%20AnassisEJ%2C%20BallEV%2C%20KrawczakM%2C%20CooperDN%2C%20Meta-analysis%20of%20indels%20causing%20human%20genetic%20disease%3A%20Mechanisms%20of%20mutagenesis%20and%20the%20role%20of%20local%20DNA%20sequence%20complexity.%20Human%20Mut.%202003%3B21%3A%2028%E2%80%9344." data-title="Meta-analysis%20of%20indels%20causing%20human%20genetic%20disease%3A%20Mechanisms%20of%20mutagenesis%20and%20the%20role%20of%20local%20DNA%20sequence%20complexity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Meta-analysis+of+indels+causing+human+genetic+disease%3A+Mechanisms+of+mutagenesis+and+the+role+of+local+DNA+sequence+complexity+Chuzhanova+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0182377.ref021" id="pone.0182377.ref021" class="link-target"></a>Krawczak M, Ball EV, Cooper DN, Neighboring-nucleotide effects on the rates of germ-line single-base-pair substitution in human genes. Am J Hum Genet. 1998;63: 474–488.  pmid:9683596 <ul class="reflinks"><li><a href="#" data-author="Krawczak" data-cit="KrawczakM%2C%20BallEV%2C%20CooperDN%2C%20Neighboring-nucleotide%20effects%20on%20the%20rates%20of%20germ-line%20single-base-pair%20substitution%20in%20human%20genes.%20Am%20J%20Hum%20Genet.%201998%3B63%3A%20474%E2%80%93488.%20doi%3A%2010.1086%2F301965%209683596" data-title="Neighboring-nucleotide%20effects%20on%20the%20rates%20of%20germ-line%20single-base-pair%20substitution%20in%20human%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9683596" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Neighboring-nucleotide+effects+on+the+rates+of+germ-line+single-base-pair+substitution+in+human+genes+Krawczak+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0182377.ref022" id="pone.0182377.ref022" class="link-target"></a>Modrich P, Mechanisms and Biological Effects of Mismatch Repair. Annu Rev Genet. 1991;25: 229–253.  pmid:1812808 <ul class="reflinks"><li><a href="#" data-author="Modrich" data-cit="ModrichP%2C%20Mechanisms%20and%20Biological%20Effects%20of%20Mismatch%20Repair.%20Annu%20Rev%20Genet.%201991%3B25%3A%20229%E2%80%93253.%20doi%3A%2010.1146%2Fannurev.ge.25.120191.001305%201812808" data-title="Mechanisms%20and%20Biological%20Effects%20of%20Mismatch%20Repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1812808" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mechanisms+and+Biological+Effects+of+Mismatch+Repair+Modrich+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0182377.ref023" id="pone.0182377.ref023" class="link-target"></a>Lamers MH, Perrakis A, Enzlin JH, Winterwerp HHK, de Wind N, Sixma TK, The crystal structure of DNA mismatch repair protein MutS binding to a G center dot T mismatch. Nature. 2000;407: 711–717.  pmid:11048711 <ul class="reflinks"><li><a href="#" data-author="Lamers" data-cit="LamersMH%2C%20PerrakisA%2C%20EnzlinJH%2C%20WinterwerpHHK%2C%20de%20WindN%2C%20SixmaTK%2C%20The%20crystal%20structure%20of%20DNA%20mismatch%20repair%20protein%20MutS%20binding%20to%20a%20G%20center%20dot%20T%20mismatch.%20Nature.%202000%3B407%3A%20711%E2%80%93717.%20doi%3A%2010.1038%2F35037523%2011048711" data-title="The%20crystal%20structure%20of%20DNA%20mismatch%20repair%20protein%20MutS%20binding%20to%20a%20G%20center%20dot%20T%20mismatch" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11048711" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+crystal+structure+of+DNA+mismatch+repair+protein+MutS+binding+to+a+G+center+dot+T+mismatch+Lamers+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0182377.ref024" id="pone.0182377.ref024" class="link-target"></a>Obmolova G, Ban C, Hsieh P, Yang W, Crystal structures of mismatch repair protein MutS and its complex with a substrate DNA. Nature. 2000;407: 703–710.  pmid:11048710 <ul class="reflinks"><li><a href="#" data-author="Obmolova" data-cit="ObmolovaG%2C%20BanC%2C%20HsiehP%2C%20YangW%2C%20Crystal%20structures%20of%20mismatch%20repair%20protein%20MutS%20and%20its%20complex%20with%20a%20substrate%20DNA.%20Nature.%202000%3B407%3A%20703%E2%80%93710.%20doi%3A%2010.1038%2F35037509%2011048710" data-title="Crystal%20structures%20of%20mismatch%20repair%20protein%20MutS%20and%20its%20complex%20with%20a%20substrate%20DNA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11048710" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Crystal+structures+of+mismatch+repair+protein+MutS+and+its+complex+with+a+substrate+DNA+Obmolova+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0182377.ref025" id="pone.0182377.ref025" class="link-target"></a>Wang H, Yang Y, Schofield MJ, Du CW, Fridman Y, Lee SD, et al., DNA bending and unbending by MutS govern mismatch recognition and specificity. Proc Natl Acad Sci U S A. 2003;100: 14822–14827.  pmid:14634210 <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangH%2C%20YangY%2C%20SchofieldMJ%2C%20DuCW%2C%20FridmanY%2C%20LeeSD%2C%20et%20al.%2C%20DNA%20bending%20and%20unbending%20by%20MutS%20govern%20mismatch%20recognition%20and%20specificity.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202003%3B100%3A%2014822%E2%80%9314827.%20doi%3A%2010.1073%2Fpnas.2433654100%2014634210" data-title="DNA%20bending%20and%20unbending%20by%20MutS%20govern%20mismatch%20recognition%20and%20specificity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14634210" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+bending+and+unbending+by+MutS+govern+mismatch+recognition+and+specificity+Wang+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0182377.ref026" id="pone.0182377.ref026" class="link-target"></a>Warren JJ, Pohlhaus TJ, Changela A, Iyer RR, Modrich PL, Beese LS, Structure of the human MutS alpha DNA lesion recognition complex. Mol Cell. 2007;26: 579–592.  pmid:17531815 <ul class="reflinks"><li><a href="#" data-author="Warren" data-cit="WarrenJJ%2C%20PohlhausTJ%2C%20ChangelaA%2C%20IyerRR%2C%20ModrichPL%2C%20BeeseLS%2C%20Structure%20of%20the%20human%20MutS%20alpha%20DNA%20lesion%20recognition%20complex.%20Mol%20Cell.%202007%3B26%3A%20579%E2%80%93592.%20doi%3A%2010.1016%2Fj.molcel.2007.04.018%2017531815" data-title="Structure%20of%20the%20human%20MutS%20alpha%20DNA%20lesion%20recognition%20complex" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17531815" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Structure+of+the+human+MutS+alpha+DNA+lesion+recognition+complex+Warren+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0182377.ref027" id="pone.0182377.ref027" class="link-target"></a>Gorman J, Chowdhury A, Surtees JA, Shimada J, Reichman DR, Alani E, et al., Dynamic basis for one-dimensional DNA scanning by the mismatch repair complex Msh2-Msh6. Mol Cell. 2007;28: 359–370.  pmid:17996701 <ul class="reflinks"><li><a href="#" data-author="Gorman" data-cit="GormanJ%2C%20ChowdhuryA%2C%20SurteesJA%2C%20ShimadaJ%2C%20ReichmanDR%2C%20AlaniE%2C%20et%20al.%2C%20Dynamic%20basis%20for%20one-dimensional%20DNA%20scanning%20by%20the%20mismatch%20repair%20complex%20Msh2-Msh6.%20Mol%20Cell.%202007%3B28%3A%20359%E2%80%93370.%20doi%3A%2010.1016%2Fj.molcel.2007.09.008%2017996701" data-title="Dynamic%20basis%20for%20one-dimensional%20DNA%20scanning%20by%20the%20mismatch%20repair%20complex%20Msh2-Msh6" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996701" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Dynamic+basis+for+one-dimensional+DNA+scanning+by+the+mismatch+repair+complex+Msh2-Msh6+Gorman+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0182377.ref028" id="pone.0182377.ref028" class="link-target"></a>Drsata T, Lankas F (2013) Theoretical models of DNA flexibility. Wiley Interdisciplinary Reviews-Computational Molecular Science. pp. 355–363. <ul class="find-nolinks"></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0182377.ref029" id="pone.0182377.ref029" class="link-target"></a>Drsata T, Spackova N, Jurecka P, Zgarbova M, Sponer J, Lankas F, Mechanical properties of symmetric and asymmetric DNA A-tracts: implications for looping and nucleosome positioning. Nucleic Acids Res. 2014;42: 7383–7394.  pmid:24829460 <ul class="reflinks"><li><a href="#" data-author="Drsata" data-cit="DrsataT%2C%20SpackovaN%2C%20JureckaP%2C%20ZgarbovaM%2C%20SponerJ%2C%20LankasF%2C%20Mechanical%20properties%20of%20symmetric%20and%20asymmetric%20DNA%20A-tracts%3A%20implications%20for%20looping%20and%20nucleosome%20positioning.%20Nucleic%20Acids%20Res.%202014%3B42%3A%207383%E2%80%937394.%20doi%3A%2010.1093%2Fnar%2Fgku338%2024829460" data-title="Mechanical%20properties%20of%20symmetric%20and%20asymmetric%20DNA%20A-tracts%3A%20implications%20for%20looping%20and%20nucleosome%20positioning" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24829460" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mechanical+properties+of+symmetric+and+asymmetric+DNA+A-tracts%3A+implications+for+looping+and+nucleosome+positioning+Drsata+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0182377.ref030" id="pone.0182377.ref030" class="link-target"></a>Ruscio JZ, Onufriev A, A computational study of nucleosomal DNA flexibility. Biophys J. 2006;91: 4121–4132.  pmid:16891359 <ul class="reflinks"><li><a href="#" data-author="Ruscio" data-cit="RuscioJZ%2C%20OnufrievA%2C%20A%20computational%20study%20of%20nucleosomal%20DNA%20flexibility.%20Biophys%20J.%202006%3B91%3A%204121%E2%80%934132.%20doi%3A%2010.1529%2Fbiophysj.106.082099%2016891359" data-title="A%20computational%20study%20of%20nucleosomal%20DNA%20flexibility" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16891359" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+computational+study+of+nucleosomal+DNA+flexibility+Ruscio+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0182377.ref031" id="pone.0182377.ref031" class="link-target"></a>Jiricny J, Postreplicative Mismatch Repair. Cold Spring Harbor Perspectives in Biology. 2013;5. <ul class="reflinks"><li><a href="#" data-author="Jiricny" data-cit="JiricnyJ%2C%20Postreplicative%20Mismatch%20Repair.%20Cold%20Spring%20Harbor%20Perspectives%20in%20Biology.%202013%3B5." data-title="Postreplicative%20Mismatch%20Repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Postreplicative+Mismatch+Repair+Jiricny+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0182377.ref032" id="pone.0182377.ref032" class="link-target"></a>Locher KP, Structure and mechanism of ABC transporters. Curr Opin Struct Biol 2004;14: 426–431.  pmid:15313236 <ul class="reflinks"><li><a href="#" data-author="Locher" data-cit="LocherKP%2C%20Structure%20and%20mechanism%20of%20ABC%20transporters.%20Curr%20Opin%20Struct%20Biol%202004%3B14%3A%20426%E2%80%93431.%20doi%3A%2010.1016%2Fj.sbi.2004.06.005%2015313236" data-title="Structure%20and%20mechanism%20of%20ABC%20transporters" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15313236" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Structure+and+mechanism+of+ABC+transporters+Locher+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0182377.ref033" id="pone.0182377.ref033" class="link-target"></a>Sharma A, Doucette C, Biro FN, Hingorani MM, Slow Conformational Changes in MutS and DNA Direct Ordered Transitions between Mismatch Search, Recognition and Signaling of DNA Repair. J Mol Biol. 2013;425: 4192–4205.  pmid:23973435 <ul class="reflinks"><li><a href="#" data-author="Sharma" data-cit="SharmaA%2C%20DoucetteC%2C%20BiroFN%2C%20HingoraniMM%2C%20Slow%20Conformational%20Changes%20in%20MutS%20and%20DNA%20Direct%20Ordered%20Transitions%20between%20Mismatch%20Search%2C%20Recognition%20and%20Signaling%20of%20DNA%20Repair.%20J%20Mol%20Biol.%202013%3B425%3A%204192%E2%80%934205.%20doi%3A%2010.1016%2Fj.jmb.2013.08.011%2023973435" data-title="Slow%20Conformational%20Changes%20in%20MutS%20and%20DNA%20Direct%20Ordered%20Transitions%20between%20Mismatch%20Search%2C%20Recognition%20and%20Signaling%20of%20DNA%20Repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23973435" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Slow+Conformational+Changes+in+MutS+and+DNA+Direct+Ordered+Transitions+between+Mismatch+Search%2C+Recognition+and+Signaling+of+DNA+Repair+Sharma+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0182377.ref034" id="pone.0182377.ref034" class="link-target"></a>Qiu RY, DeRocco VC, Harris C, Sharma A, Hingorani MM, Erie DA, et al., Large conformational changes in MutS during DNA scanning, mismatch recognition and repair signalling. Embo J. 2012;31: 2528–2540.  pmid:22505031 <ul class="reflinks"><li><a href="#" data-author="Qiu" data-cit="QiuRY%2C%20DeRoccoVC%2C%20HarrisC%2C%20SharmaA%2C%20HingoraniMM%2C%20ErieDA%2C%20et%20al.%2C%20Large%20conformational%20changes%20in%20MutS%20during%20DNA%20scanning%2C%20mismatch%20recognition%20and%20repair%20signalling.%20Embo%20J.%202012%3B31%3A%202528%E2%80%932540.%20doi%3A%2010.1038%2Femboj.2012.95%2022505031" data-title="Large%20conformational%20changes%20in%20MutS%20during%20DNA%20scanning%2C%20mismatch%20recognition%20and%20repair%20signalling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22505031" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Large+conformational+changes+in+MutS+during+DNA+scanning%2C+mismatch+recognition+and+repair+signalling+Qiu+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0182377.ref035" id="pone.0182377.ref035" class="link-target"></a>Hingorani MM, Mismatch binding, ADP-ATP exchange and intramolecular signaling during mismatch repair. DNA Repair. 2016;38: 24–31.  pmid:26704427 <ul class="reflinks"><li><a href="#" data-author="Hingorani" data-cit="HingoraniMM%2C%20Mismatch%20binding%2C%20ADP-ATP%20exchange%20and%20intramolecular%20signaling%20during%20mismatch%20repair.%20DNA%20Repair.%202016%3B38%3A%2024%E2%80%9331.%20doi%3A%2010.1016%2Fj.dnarep.2015.11.017%2026704427" data-title="Mismatch%20binding%2C%20ADP-ATP%20exchange%20and%20intramolecular%20signaling%20during%20mismatch%20repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26704427" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mismatch+binding%2C+ADP-ATP+exchange+and+intramolecular+signaling+during+mismatch+repair+Hingorani+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0182377.ref036" id="pone.0182377.ref036" class="link-target"></a>Groothuizen FS, Winkler I, Cristovao M, Fish A, Winterwerp HHK, Reumer A, et al., MutS/MutL crystal structure reveals that the MutS sliding clamp loads MutL onto DNA. Elife. 2015;4. <ul class="reflinks"><li><a href="#" data-author="Groothuizen" data-cit="GroothuizenFS%2C%20WinklerI%2C%20CristovaoM%2C%20FishA%2C%20WinterwerpHHK%2C%20ReumerA%2C%20et%20al.%2C%20MutS%2FMutL%20crystal%20structure%20reveals%20that%20the%20MutS%20sliding%20clamp%20loads%20MutL%20onto%20DNA.%20Elife.%202015%3B4." data-title="MutS%2FMutL%20crystal%20structure%20reveals%20that%20the%20MutS%20sliding%20clamp%20loads%20MutL%20onto%20DNA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MutS%2FMutL+crystal+structure+reveals+that+the+MutS+sliding+clamp+loads+MutL+onto+DNA+Groothuizen+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0182377.ref037" id="pone.0182377.ref037" class="link-target"></a>Brukner I, Sanchez R, Suck D, Pongor S, Sequence-Dependent Bending Propensity of DNA as Revealed by Dnase-I—Parameters for Trinucleotides. Embo J. 1995;14: 1812–1818. pmid:7737131 <ul class="reflinks"><li><a href="#" data-author="Brukner" data-cit="BruknerI%2C%20SanchezR%2C%20SuckD%2C%20PongorS%2C%20Sequence-Dependent%20Bending%20Propensity%20of%20DNA%20as%20Revealed%20by%20Dnase-I%E2%80%94Parameters%20for%20Trinucleotides.%20Embo%20J.%201995%3B14%3A%201812%E2%80%931818.%207737131" data-title="Sequence-Dependent%20Bending%20Propensity%20of%20DNA%20as%20Revealed%20by%20Dnase-I%E2%80%94Parameters%20for%20Trinucleotides" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7737131" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence-Dependent+Bending+Propensity+of+DNA+as+Revealed+by+Dnase-I%E2%80%94Parameters+for+Trinucleotides+Brukner+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0182377.ref038" id="pone.0182377.ref038" class="link-target"></a>Gabrielian A, Pongor S, Correlation of intrinsic DNA curvature with DNA property periodicity. Febs Lett. 1996;393: 65–68. pmid:8804425 <ul class="reflinks"><li><a href="#" data-author="Gabrielian" data-cit="GabrielianA%2C%20PongorS%2C%20Correlation%20of%20intrinsic%20DNA%20curvature%20with%20DNA%20property%20periodicity.%20Febs%20Lett.%201996%3B393%3A%2065%E2%80%9368.%208804425" data-title="Correlation%20of%20intrinsic%20DNA%20curvature%20with%20DNA%20property%20periodicity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8804425" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Correlation+of+intrinsic+DNA+curvature+with+DNA+property+periodicity+Gabrielian+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0182377.ref039" id="pone.0182377.ref039" class="link-target"></a>Case DA, Babin V, Berryman JT, Betz RM, Cai Q, Cerutti DS, et al., AMBER 14. University of California: San Francisco. 2014. <ul class="find-nolinks"></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0182377.ref040" id="pone.0182377.ref040" class="link-target"></a>Darve E, Rodríguez-Gómez D, Pohorille A, Adaptive Biasing Force Method for Scalar and Vector Free Energy Calculations. J Chem Phys. 2008;128: 144120.  pmid:18412436 <ul class="reflinks"><li><a href="#" data-author="Darve" data-cit="DarveE%2C%20Rodr%C3%ADguez-G%C3%B3mezD%2C%20PohorilleA%2C%20Adaptive%20Biasing%20Force%20Method%20for%20Scalar%20and%20Vector%20Free%20Energy%20Calculations.%20J%20Chem%20Phys.%202008%3B128%3A%20144120.%20doi%3A%2010.1063%2F1.2829861%2018412436" data-title="Adaptive%20Biasing%20Force%20Method%20for%20Scalar%20and%20Vector%20Free%20Energy%20Calculations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18412436" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Adaptive+Biasing+Force+Method+for+Scalar+and+Vector+Free+Energy+Calculations+Darve+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0182377.ref041" id="pone.0182377.ref041" class="link-target"></a>Raiteri P, Laio A, Gervasio F, Micheletti C, Parrinello M, Efficient reconstruction of complex free energy landscapes by multiple walkers metadynamics. J Phys Chem B. 2006;110: 3533–3539.  pmid:16494409 <ul class="reflinks"><li><a href="#" data-author="Raiteri" data-cit="RaiteriP%2C%20LaioA%2C%20GervasioF%2C%20MichelettiC%2C%20ParrinelloM%2C%20Efficient%20reconstruction%20of%20complex%20free%20energy%20landscapes%20by%20multiple%20walkers%20metadynamics.%20J%20Phys%20Chem%20B.%202006%3B110%3A%203533%E2%80%933539.%20doi%3A%2010.1021%2Fjp054359r%2016494409" data-title="Efficient%20reconstruction%20of%20complex%20free%20energy%20landscapes%20by%20multiple%20walkers%20metadynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16494409" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Efficient+reconstruction+of+complex+free+energy+landscapes+by+multiple+walkers+metadynamics+Raiteri+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0182377.ref042" id="pone.0182377.ref042" class="link-target"></a>van der Heijden T, van Vugt JJFA, Logie C, van Noort J, Sequence-based prediction of single nucleosome positioning and genome-wide nucleosome occupancy (vol 109, pg E2514, 2012). Proceedings of the National Academy of Sciences of the United States of America. 2013;110: 6240–6240. <ul class="reflinks"><li><a href="#" data-author="van%20der%20Heijden" data-cit="van%20der%20HeijdenT%2C%20van%20VugtJJFA%2C%20LogieC%2C%20van%20NoortJ%2C%20Sequence-based%20prediction%20of%20single%20nucleosome%20positioning%20and%20genome-wide%20nucleosome%20occupancy%20%28vol%20109%2C%20pg%20E2514%2C%202012%29.%20Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America.%202013%3B110%3A%206240%E2%80%936240." data-title="Sequence-based%20prediction%20of%20single%20nucleosome%20positioning%20and%20genome-wide%20nucleosome%20occupancy%20%28vol%20109%2C%20pg%20E2514%2C%202012%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence-based+prediction+of+single+nucleosome+positioning+and+genome-wide+nucleosome+occupancy+%28vol+109%2C+pg+E2514%2C+2012%29+van+der+Heijden+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0182377.ref043" id="pone.0182377.ref043" class="link-target"></a>Jorgensen W, Chandrasekhar J, Madura J, Impey R, Klein M, Comparison of Simple Potential Functions for Simulating Liquid Water. J Chem Phys. 1983;79: 926–935. <ul class="reflinks"><li><a href="#" data-author="Jorgensen" data-cit="JorgensenW%2C%20ChandrasekharJ%2C%20MaduraJ%2C%20ImpeyR%2C%20KleinM%2C%20Comparison%20of%20Simple%20Potential%20Functions%20for%20Simulating%20Liquid%20Water.%20J%20Chem%20Phys.%201983%3B79%3A%20926%E2%80%93935." data-title="Comparison%20of%20Simple%20Potential%20Functions%20for%20Simulating%20Liquid%20Water" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Comparison+of+Simple+Potential+Functions+for+Simulating+Liquid+Water+Jorgensen+1983" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0182377.ref044" id="pone.0182377.ref044" class="link-target"></a>Joung IS, Cheatham TE, Determination of alkali and halide monovalent ion parameters for use in explicitly solvated biomolecular simulations. J Phys Chem B. 2008;112: 9020–9041.  pmid:18593145 <ul class="reflinks"><li><a href="#" data-author="Joung" data-cit="JoungIS%2C%20CheathamTE%2C%20Determination%20of%20alkali%20and%20halide%20monovalent%20ion%20parameters%20for%20use%20in%20explicitly%20solvated%20biomolecular%20simulations.%20J%20Phys%20Chem%20B.%202008%3B112%3A%209020%E2%80%939041.%20doi%3A%2010.1021%2Fjp8001614%2018593145" data-title="Determination%20of%20alkali%20and%20halide%20monovalent%20ion%20parameters%20for%20use%20in%20explicitly%20solvated%20biomolecular%20simulations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18593145" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Determination+of+alkali+and+halide+monovalent+ion+parameters+for+use+in+explicitly+solvated+biomolecular+simulations+Joung+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0182377.ref045" id="pone.0182377.ref045" class="link-target"></a>Ivani I, Dans PD, Noy A, Perez A, Faustino I, Hospital A, et al., Parmbsc1: a refined force field for DNA simulations. Nat Methods. 2016;13: 55-+.  pmid:26569599 <ul class="reflinks"><li><a href="#" data-author="Ivani" data-cit="IvaniI%2C%20DansPD%2C%20NoyA%2C%20PerezA%2C%20FaustinoI%2C%20HospitalA%2C%20et%20al.%2C%20Parmbsc1%3A%20a%20refined%20force%20field%20for%20DNA%20simulations.%20Nat%20Methods.%202016%3B13%3A%2055-%2B.%20doi%3A%2010.1038%2Fnmeth.3658%2026569599" data-title="Parmbsc1%3A%20a%20refined%20force%20field%20for%20DNA%20simulations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26569599" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Parmbsc1%3A+a+refined+force+field+for+DNA+simulations+Ivani+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0182377.ref046" id="pone.0182377.ref046" class="link-target"></a>Lelievre T, Rousset M, Stoltz G, Long-Time Convergence of an Adaptive Biasing Force Method. Nonlinearity. 2008;21: 1155–1181. <ul class="reflinks"><li><a href="#" data-author="Lelievre" data-cit="LelievreT%2C%20RoussetM%2C%20StoltzG%2C%20Long-Time%20Convergence%20of%20an%20Adaptive%20Biasing%20Force%20Method.%20Nonlinearity.%202008%3B21%3A%201155%E2%80%931181." data-title="Long-Time%20Convergence%20of%20an%20Adaptive%20Biasing%20Force%20Method" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Long-Time+Convergence+of+an+Adaptive+Biasing+Force+Method+Lelievre+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0182377.ref047" id="pone.0182377.ref047" class="link-target"></a>Kulhanek P, Stepan J, Fuxreiter M, Mones L, Strelcova Z, Petrek M, PMFLib—A Toolkit for Free Energy Calculations; <a href="https://lcc.ncbr.muni.cz/whitezone/development/pmflib/index.html">https://lcc.ncbr.muni.cz/whitezone/development/pmflib/index.html</a>. 2013. <ul class="find-nolinks"></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0182377.ref048" id="pone.0182377.ref048" class="link-target"></a>Lu XJ, Olson WK, 3DNA: a software package for the analysis, rebuilding and visualization of three-dimensional nucleic acid structures. Nucleic Acids Res. 2003;31: 5108–5121.  pmid:12930962 <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuXJ%2C%20OlsonWK%2C%203DNA%3A%20a%20software%20package%20for%20the%20analysis%2C%20rebuilding%20and%20visualization%20of%20three-dimensional%20nucleic%20acid%20structures.%20Nucleic%20Acids%20Res.%202003%3B31%3A%205108%E2%80%935121.%20doi%3A%2010.1093%2Fnar%2Fgkg680%2012930962" data-title="3DNA%3A%20a%20software%20package%20for%20the%20analysis%2C%20rebuilding%20and%20visualization%20of%20three-dimensional%20nucleic%20acid%20structures" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930962" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=3DNA%3A+a+software+package+for+the+analysis%2C+rebuilding+and+visualization+of+three-dimensional+nucleic+acid+structures+Lu+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0182377.ref049" id="pone.0182377.ref049" class="link-target"></a>Sharma M, Predeus AV, Mukherjee S, Feig M, DNA Bending Propensity in the Presence of Base Mismatches: Implications for DNA Repair. J Phys Chem B. 2013;117: 6194–6205.  pmid:23621762 <ul class="reflinks"><li><a href="#" data-author="Sharma" data-cit="SharmaM%2C%20PredeusAV%2C%20MukherjeeS%2C%20FeigM%2C%20DNA%20Bending%20Propensity%20in%20the%20Presence%20of%20Base%20Mismatches%3A%20Implications%20for%20DNA%20Repair.%20J%20Phys%20Chem%20B.%202013%3B117%3A%206194%E2%80%936205.%20doi%3A%2010.1021%2Fjp403127a%2023621762" data-title="DNA%20Bending%20Propensity%20in%20the%20Presence%20of%20Base%20Mismatches%3A%20Implications%20for%20DNA%20Repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23621762" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+Bending+Propensity+in+the+Presence+of+Base+Mismatches%3A+Implications+for+DNA+Repair+Sharma+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0182377.ref050" id="pone.0182377.ref050" class="link-target"></a>Humphrey W, Dalke A, Schulten K, VMD—Visual Molecular Dynamics. J. Mol. Graph. Model. 1996;14: 33–38. <ul class="reflinks"><li><a href="#" data-author="Humphrey" data-cit="HumphreyW%2C%20DalkeA%2C%20SchultenK%2C%20VMD%E2%80%94Visual%20Molecular%20Dynamics.%20J.%20Mol.%20Graph.%20Model.%201996%3B14%3A%2033%E2%80%9338." data-title="VMD%E2%80%94Visual%20Molecular%20Dynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=VMD%E2%80%94Visual+Molecular+Dynamics+Humphrey+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0182377.ref051" id="pone.0182377.ref051" class="link-target"></a>Crooks GE, Hon G, Chandonia JM, Brenner SE, WebLogo: A sequence logo generator. Genome Res. 2004;14: 1188–1190.  pmid:15173120 <ul class="reflinks"><li><a href="#" data-author="Crooks" data-cit="CrooksGE%2C%20HonG%2C%20ChandoniaJM%2C%20BrennerSE%2C%20WebLogo%3A%20A%20sequence%20logo%20generator.%20Genome%20Res.%202004%3B14%3A%201188%E2%80%931190.%20doi%3A%2010.1101%2Fgr.849004%2015173120" data-title="WebLogo%3A%20A%20sequence%20logo%20generator" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15173120" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=WebLogo%3A+A+sequence+logo+generator+Crooks+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0182377.ref052" id="pone.0182377.ref052" class="link-target"></a>Graur D, Li WH (2000) Fundamentals of molecular evolution. Sunderland, MA: Sinauer Associates. Sunderland, MA. <ul class="find-nolinks"></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0182377.ref053" id="pone.0182377.ref053" class="link-target"></a>Thoma F, Light and dark in chromatin repair: repair of UV-induced DNA lesions by photolyase and nucleotide excision repair. Embo Journal. 1999;18: 6585–6598.  pmid:10581233 <ul class="reflinks"><li><a href="#" data-author="Thoma" data-cit="ThomaF%2C%20Light%20and%20dark%20in%20chromatin%20repair%3A%20repair%20of%20UV-induced%20DNA%20lesions%20by%20photolyase%20and%20nucleotide%20excision%20repair.%20Embo%20Journal.%201999%3B18%3A%206585%E2%80%936598.%20doi%3A%2010.1093%2Femboj%2F18.23.6585%2010581233" data-title="Light%20and%20dark%20in%20chromatin%20repair%3A%20repair%20of%20UV-induced%20DNA%20lesions%20by%20photolyase%20and%20nucleotide%20excision%20repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10581233" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Light+and+dark+in+chromatin+repair%3A+repair+of+UV-induced+DNA+lesions+by+photolyase+and+nucleotide+excision+repair+Thoma+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0182377.ref054" id="pone.0182377.ref054" class="link-target"></a>Rogozin IB, Babenko VN, Milanesi L, P Y.I., Computational analysis of mutation spectra. Brief Bioinform. 2003;4: 210–227. pmid:14582516 <ul class="reflinks"><li><a href="#" data-author="Rogozin" data-cit="RogozinIB%2C%20BabenkoVN%2C%20MilanesiL%2C%20PY.I.%2C%20Computational%20analysis%20of%20mutation%20spectra.%20Brief%20Bioinform.%202003%3B4%3A%20210%E2%80%93227.%2014582516" data-title="Computational%20analysis%20of%20mutation%20spectra" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14582516" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Computational+analysis+of+mutation+spectra+Rogozin+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pone.0182377.ref055" id="pone.0182377.ref055" class="link-target"></a>Buckova H, Noskova H, Borska R, Reblova K, Pinkova B, Zapletalova E, et al., Autosomal recessive congenital ichthyoses in the Czech Republic. Brit J Dermatol. 2016;174: 405–407. <ul class="reflinks"><li><a href="#" data-author="Buckova" data-cit="BuckovaH%2C%20NoskovaH%2C%20BorskaR%2C%20ReblovaK%2C%20PinkovaB%2C%20ZapletalovaE%2C%20et%20al.%2C%20Autosomal%20recessive%20congenital%20ichthyoses%20in%20the%20Czech%20Republic.%20Brit%20J%20Dermatol.%202016%3B174%3A%20405%E2%80%93407." data-title="Autosomal%20recessive%20congenital%20ichthyoses%20in%20the%20Czech%20Republic" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Autosomal+recessive+congenital+ichthyoses+in+the+Czech+Republic+Buckova+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pone.0182377.ref056" id="pone.0182377.ref056" class="link-target"></a>Chien YH, Chiang SC, Huang A, Chou SP, Tseng SS, Huang YT, et al., Mutation spectrum in Taiwanese patients with phenylalanine hydroxylase deficiency and a founder effect for the R241C mutation. Human Mut. 2004;23: 206. <ul class="reflinks"><li><a href="#" data-author="Chien" data-cit="ChienYH%2C%20ChiangSC%2C%20HuangA%2C%20ChouSP%2C%20TsengSS%2C%20HuangYT%2C%20et%20al.%2C%20Mutation%20spectrum%20in%20Taiwanese%20patients%20with%20phenylalanine%20hydroxylase%20deficiency%20and%20a%20founder%20effect%20for%20the%20R241C%20mutation.%20Human%20Mut.%202004%3B23%3A%20206." data-title="Mutation%20spectrum%20in%20Taiwanese%20patients%20with%20phenylalanine%20hydroxylase%20deficiency%20and%20a%20founder%20effect%20for%20the%20R241C%20mutation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mutation+spectrum+in+Taiwanese+patients+with+phenylalanine+hydroxylase+deficiency+and+a+founder+effect+for+the+R241C+mutation+Chien+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pone.0182377.ref057" id="pone.0182377.ref057" class="link-target"></a>Chiu YH, Chang YC, Chang YH, Niu DM, Yang YL, Ye J, et al., Mutation spectrum of and founder effects affecting the PTS gene in East Asian populations. J Hum Gen. 2012;57: 145–152. <ul class="reflinks"><li><a href="#" data-author="Chiu" data-cit="ChiuYH%2C%20ChangYC%2C%20ChangYH%2C%20NiuDM%2C%20YangYL%2C%20YeJ%2C%20et%20al.%2C%20Mutation%20spectrum%20of%20and%20founder%20effects%20affecting%20the%20PTS%20gene%20in%20East%20Asian%20populations.%20J%20Hum%20Gen.%202012%3B57%3A%20145%E2%80%93152." data-title="Mutation%20spectrum%20of%20and%20founder%20effects%20affecting%20the%20PTS%20gene%20in%20East%20Asian%20populations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mutation+spectrum+of+and+founder+effects+affecting+the+PTS+gene+in+East+Asian+populations+Chiu+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pone.0182377.ref058" id="pone.0182377.ref058" class="link-target"></a>Streisin G, Okada Y, Emrich J, Newton J, Tsugita A, Terzaghi E, et al., Frameshift Mutations and Genetic Code. Cold Spring Harb Symp Quant Biol. 1966;31: 77–84. pmid:5237214 <ul class="reflinks"><li><a href="#" data-author="Streisin" data-cit="StreisinG%2C%20OkadaY%2C%20EmrichJ%2C%20NewtonJ%2C%20TsugitaA%2C%20TerzaghiE%2C%20et%20al.%2C%20Frameshift%20Mutations%20and%20Genetic%20Code.%20Cold%20Spring%20Harb%20Symp%20Quant%20Biol.%201966%3B31%3A%2077%E2%80%9384.%205237214" data-title="Frameshift%20Mutations%20and%20Genetic%20Code" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/5237214" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Frameshift+Mutations+and+Genetic+Code+Streisin+1966" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pone.0182377.ref059" id="pone.0182377.ref059" class="link-target"></a>Chen JQ, Wu Y, Yang HW, Bergelson J, Kreitman M, Tian DC, Variation in the Ratio of Nucleotide Substitution and Indel Rates across Genomes in Mammals and Bacteria. Mol Biol Evol. 2009;26: 1523–1531.  pmid:19329651 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenJQ%2C%20WuY%2C%20YangHW%2C%20BergelsonJ%2C%20KreitmanM%2C%20TianDC%2C%20Variation%20in%20the%20Ratio%20of%20Nucleotide%20Substitution%20and%20Indel%20Rates%20across%20Genomes%20in%20Mammals%20and%20Bacteria.%20Mol%20Biol%20Evol.%202009%3B26%3A%201523%E2%80%931531.%20doi%3A%2010.1093%2Fmolbev%2Fmsp063%2019329651" data-title="Variation%20in%20the%20Ratio%20of%20Nucleotide%20Substitution%20and%20Indel%20Rates%20across%20Genomes%20in%20Mammals%20and%20Bacteria" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19329651" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Variation+in+the+Ratio+of+Nucleotide+Substitution+and+Indel+Rates+across+Genomes+in+Mammals+and+Bacteria+Chen+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span class="order">60.
            </span><a name="pone.0182377.ref060" id="pone.0182377.ref060" class="link-target"></a>Gragg H, Harfe BD, Jinks-Robertson S, Base composition of mononucleotide runs affects DNA polymerase slippage and removal of frameshift intermediates by mismatch repair in Saccharomyces cerevisiae. Mol Cell Biol. 2002;22: 8756–8762.  pmid:12446792 <ul class="reflinks"><li><a href="#" data-author="Gragg" data-cit="GraggH%2C%20HarfeBD%2C%20Jinks-RobertsonS%2C%20Base%20composition%20of%20mononucleotide%20runs%20affects%20DNA%20polymerase%20slippage%20and%20removal%20of%20frameshift%20intermediates%20by%20mismatch%20repair%20in%20Saccharomyces%20cerevisiae.%20Mol%20Cell%20Biol.%202002%3B22%3A%208756%E2%80%938762.%20doi%3A%2010.1128%2FMCB.22.24.8756-8762.2002%2012446792" data-title="Base%20composition%20of%20mononucleotide%20runs%20affects%20DNA%20polymerase%20slippage%20and%20removal%20of%20frameshift%20intermediates%20by%20mismatch%20repair%20in%20Saccharomyces%20cerevisiae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12446792" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Base+composition+of+mononucleotide+runs+affects+DNA+polymerase+slippage+and+removal+of+frameshift+intermediates+by+mismatch+repair+in+Saccharomyces+cerevisiae+Gragg+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span class="order">61.
            </span><a name="pone.0182377.ref061" id="pone.0182377.ref061" class="link-target"></a>Kroutil LC, Register K, Bebenek K, Kunkel TA, Exonucleolytic proofreading during replication of repetitive DNA. Biochemistry. 1996;35: 1046–1053.  pmid:8547240 <ul class="reflinks"><li><a href="#" data-author="Kroutil" data-cit="KroutilLC%2C%20RegisterK%2C%20BebenekK%2C%20KunkelTA%2C%20Exonucleolytic%20proofreading%20during%20replication%20of%20repetitive%20DNA.%20Biochemistry.%201996%3B35%3A%201046%E2%80%931053.%20doi%3A%2010.1021%2Fbi952178h%208547240" data-title="Exonucleolytic%20proofreading%20during%20replication%20of%20repetitive%20DNA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8547240" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Exonucleolytic+proofreading+during+replication+of+repetitive+DNA+Kroutil+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span class="order">62.
            </span><a name="pone.0182377.ref062" id="pone.0182377.ref062" class="link-target"></a>Tran HT, Keen JD, Kricker M, Resnick MA, Gordenin DA, Hypermutability of homonucleotide runs in mismatch repair and DNA polymerase proofreading yeast mutants. Mol Cell Biol. 1997;17: 2859–2865. pmid:9111358 <ul class="reflinks"><li><a href="#" data-author="Tran" data-cit="TranHT%2C%20KeenJD%2C%20KrickerM%2C%20ResnickMA%2C%20GordeninDA%2C%20Hypermutability%20of%20homonucleotide%20runs%20in%20mismatch%20repair%20and%20DNA%20polymerase%20proofreading%20yeast%20mutants.%20Mol%20Cell%20Biol.%201997%3B17%3A%202859%E2%80%932865.%209111358" data-title="Hypermutability%20of%20homonucleotide%20runs%20in%20mismatch%20repair%20and%20DNA%20polymerase%20proofreading%20yeast%20mutants" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9111358" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Hypermutability+of+homonucleotide+runs+in+mismatch+repair+and+DNA+polymerase+proofreading+yeast+mutants+Tran+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref63"><span class="order">63.
            </span><a name="pone.0182377.ref063" id="pone.0182377.ref063" class="link-target"></a>Grodecka L, Kramarek M, Lockerova P, Kovacova T, Ravcukova B, Richterova R, et al., No Major Effect of the CDH1 c.2440-6C &gt; G Mutation on Splicing Detected in Last Exon-Specific Splicing Minigene Assay. Gene Chromosomes Cancer. 2014;53: 798–801. <ul class="reflinks"><li><a href="#" data-author="Grodecka" data-cit="GrodeckaL%2C%20KramarekM%2C%20LockerovaP%2C%20KovacovaT%2C%20RavcukovaB%2C%20RichterovaR%2C%20et%20al.%2C%20No%20Major%20Effect%20of%20the%20CDH1%20c.2440-6C%20%3E%20G%20Mutation%20on%20Splicing%20Detected%20in%20Last%20Exon-Specific%20Splicing%20Minigene%20Assay.%20Gene%20Chromosomes%20Cancer.%202014%3B53%3A%20798%E2%80%93801." data-title="No%20Major%20Effect%20of%20the%20CDH1%20c.2440-6C%20%3E%20G%20Mutation%20on%20Splicing%20Detected%20in%20Last%20Exon-Specific%20Splicing%20Minigene%20Assay" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=No+Major+Effect+of+the+CDH1+c.2440-6C+%3E+G+Mutation+on+Splicing+Detected+in+Last+Exon-Specific+Splicing+Minigene+Assay+Grodecka+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref64"><span class="order">64.
            </span><a name="pone.0182377.ref064" id="pone.0182377.ref064" class="link-target"></a>Grodecka L, Lockerova P, Ravcukova B, Buratti E, Baralle FE, Dusek L, et al., Exon First Nucleotide Mutations in Splicing: Evaluation of In Silico Prediction Tools. Plos One. 2014;9. <ul class="reflinks"><li><a href="#" data-author="Grodecka" data-cit="GrodeckaL%2C%20LockerovaP%2C%20RavcukovaB%2C%20BurattiE%2C%20BaralleFE%2C%20DusekL%2C%20et%20al.%2C%20Exon%20First%20Nucleotide%20Mutations%20in%20Splicing%3A%20Evaluation%20of%20In%20Silico%20Prediction%20Tools.%20Plos%20One.%202014%3B9." data-title="Exon%20First%20Nucleotide%20Mutations%20in%20Splicing%3A%20Evaluation%20of%20In%20Silico%20Prediction%20Tools" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Exon+First+Nucleotide+Mutations+in+Splicing%3A+Evaluation+of+In+Silico+Prediction+Tools+Grodecka+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

