



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Cancer is a significant public health problem worldwide. Complete identification of genes related to one type of cancer facilitates earlier diagnosis and effective treatments. In this study, two widely used algorithms, the random walk with restart algorithm and the shortest path algorithm, were adopted to construct two parameterized computational methods, namely, an RWR-based method and an SP-based method; based on these methods, an integrated method was constructed for identifying novel disease genes. To validate the utility of the integrated method, data for oral cancer were used, on which the RWR-based and SP-based methods were trained, thereby building two optimal methods. The integrated method combining these optimal methods was further adopted to identify the novel genes of oral cancer. As a result, 85 novel genes were inferred, among which eleven genes (<em>e</em>.<em>g</em>., MYD88, FGFR2, NF-κBIA) were identified by both the RWR-based and SP-based methods, 70 genes (<em>e</em>.<em>g</em>., BMP4, IFNG, KITLG) were discovered only by the RWR-based method and four genes (L1R1, MCM6, NOG and CXCR3) were predicted only by the SP-based method. Extensive analyses indicate that several novel genes have strong associations with cancers, indicating the effectiveness of the integrated method for identifying disease genes.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Chen L, Yang J, Xing Z, Yuan F, Shu Y, Zhang Y, et al.  (2017) An integrated method for the identification of novel genes related to oral cancer. PLoS ONE 12(4):
           e0175185.
        
        https://doi.org/10.1371/journal.pone.0175185</p><p><strong>Editor: </strong>Quan Zou, Tianjin University, CHINA</p><p><strong>Received: </strong>January 5, 2017; <strong>Accepted: </strong>March 21, 2017; <strong>Published: </strong> April 6, 2017</p><p><strong>Copyright: </strong> © 2017 Chen et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>All relevant data are within the paper and its Supporting Information files.</p><p><strong>Funding: </strong>This study was supported by the National Natural Science Foundation of China (31371335), Natural Science Foundation of Shanghai (17ZR1412500), Shanghai Sailing Program and The Youth Innovation Promotion Association of Chinese Academy of Sciences (CAS) (2016245).</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Cancer is a significant public health problem worldwide. Oral cancer (OC) is a subgroup of head and neck cancer; it develops on the lips, tongue, salivary glands, gingiva, oropharynx, and on buccal surfaces [<a href="#pone.0175185.ref001" class="ref-tip">1</a>]. Oral squamous cell carcinoma (OSCC) accounts for more than 90% of all OC [<a href="#pone.0175185.ref002" class="ref-tip">2</a>]. OC is estimated by the WHO (World Health Organization) to be the eleventh most common cancer in the world, accounting for 300,000 new cases and 145,000 deaths in 2012 [<a href="#pone.0175185.ref003" class="ref-tip">3</a>, <a href="#pone.0175185.ref004" class="ref-tip">4</a>]. The incidence of OC exhibits significant local variation and continues to be high in India, East Asia, Eastern Europe, and parts of South America [<a href="#pone.0175185.ref005" class="ref-tip">5</a>]. Tobacco and alcohol are the most important risk factors for OC [<a href="#pone.0175185.ref006" class="ref-tip">6</a>]. Poor nutrition, genetic factors and viral infection may be potential risk factors for OC [<a href="#pone.0175185.ref007" class="ref-tip">7</a>, <a href="#pone.0175185.ref008" class="ref-tip">8</a>].</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Carcinogenesis is a multi-step process, and a variety of alterations accumulate, driving and gradually increasing tumorigenesis [<a href="#pone.0175185.ref009" class="ref-tip">9</a>]. During the past decade, a large variety of genomic variations has been implicated in OC. EGFR (epidermal growth factor receptor) amplification and over-expression were found in a large proportion of oral tumors [<a href="#pone.0175185.ref010" class="ref-tip">10</a>–<a href="#pone.0175185.ref012" class="ref-tip">12</a>]. ErbB2 amplification and overexpression appear to occur frequently in OC specimens, and high levels of ErbB2 may be related to a worse prognosis of patients [<a href="#pone.0175185.ref012" class="ref-tip">12</a>]. A strong correlation has been detected between c-erbB-2 overexpression and overall survival of patients with oral squamous cell carcinoma [<a href="#pone.0175185.ref013" class="ref-tip">13</a>, <a href="#pone.0175185.ref014" class="ref-tip">14</a>]. The cyclin family plays a critical role in cell cycle progression. Aberrant up-expression of cyclin D accounts for 36–66% of OC. Amplification and an SNP of cyclin D may be associated with a worse prognosis and susceptibility to OC, respectively [<a href="#pone.0175185.ref015" class="ref-tip">15</a>, <a href="#pone.0175185.ref016" class="ref-tip">16</a>]. Several lines of evidence suggest that the p53 tumor suppression network is altered in OC [<a href="#pone.0175185.ref017" class="ref-tip">17</a>, <a href="#pone.0175185.ref018" class="ref-tip">18</a>]. In addition, many other target genes have been reported, such as ras, VEGF (vascular endothelial growth factor), and the MMP (matrix metalloproteinases) family [<a href="#pone.0175185.ref019" class="ref-tip">19</a>–<a href="#pone.0175185.ref021" class="ref-tip">21</a>]. To the best of our knowledge, the mechanism that underlies OC is still unclear. A search for new genes related to OC may facilitate earlier diagnosis and effective treatment.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>In general, many standard methods have been used for detection of virulence genes. In the hospital and laboratory, cancer samples are sliced into pathological sections and stained to determine the disease pathology type. Quantitative reverse transcription-PCR is typically used to detect the mRNA level of genes in cancer. IHC (immunohistochemical) and western blotting measure the expression of related proteins in tissues and cells, respectively. However, it is difficult to analyze genes synthetically and comprehensively. In addition, some large-scale experiments have been exploited to screen virulence genes, such as microarray, GWAS (genome-wide association study) and NGS (next-generation sequencing). The most common techniques are time-consuming and costly, and thus new methods must be explored to identify tumor genes. In recent years, with the development of computer techniques [<a href="#pone.0175185.ref022" class="ref-tip">22</a>–<a href="#pone.0175185.ref028" class="ref-tip">28</a>], some of them can be applied to tackle this problem. Up to now, several computational methods have been proposed to identify disease genes. Many of them are based on guilt-by-association [<a href="#pone.0175185.ref029" class="ref-tip">29</a>], i.e., the assumption that genes are similar to their neighbors in a gene network. Thus, the neighbors of the disease genes are more likely to be disease genes. However, these types of methods are local methods that use only part of the network. Thus, they do not always yield good performance. Many other methods employ the Random Walk with Restart (RWR) algorithm to identify disease genes [<a href="#pone.0175185.ref030" class="ref-tip">30</a>–<a href="#pone.0175185.ref032" class="ref-tip">32</a>]. This algorithm simulates a walker starting from a seed node or a set of seed nodes that represent disease genes and random walking on the network. The probability of a node being a disease gene is updated until the probabilities of all nodes become stable. The genes corresponding to nodes with high probabilities are selected as novel candidate disease genes. Recently, computational methods have adopted the shortest path (SP) algorithm to address the problem [<a href="#pone.0175185.ref033" class="ref-tip">33</a>–<a href="#pone.0175185.ref039" class="ref-tip">39</a>]. This algorithm assumes that genes lying in the shortest paths connecting any two disease genes may also be disease genes. Clearly, methods based on RWR or SP algorithms take full advantage of the network compared with those based on guilt-by-association. Thus, they can yield clues for the discovery of novel disease genes.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>In this study, we used the RWR and SP algorithms to construct two novel computational methods, the RWR-based method and the SP-based method, respectively, for the identification of novel disease genes. Additionally, an integrated method was constructed by combining these two methods. To indicate the effectiveness of the integrated method, disease genes of OC were employed. It has been reported in some studies that using only the RWR algorithm and SP algorithm consistently produces several false discoveries [<a href="#pone.0175185.ref033" class="ref-tip">33</a>, <a href="#pone.0175185.ref040" class="ref-tip">40</a>], which may be caused by the structure of the network or lack of consideration of the essential properties of genes. Thus, further rules, produced by the permutation test, associations between candidates and the validation of genes according to their properties, were integrated into the RWR-based and SP-based methods. These methods were executed using the corresponding large network that was built using protein-protein interaction (PPI) information and that was trained on validated OC genes to determine the optimal parameters. The obtained optimal methods were used to infer novel genes related to OC, and the integrated method combined the predicted genes using these two optimal methods to yield 85 novel genes. Among them, eleven genes were obtained by both methods, 70 genes were obtained only by the RWR-based method and four genes were obtained only by the SP-based method. According to the analyses, several genes show stimulative or suppressive effects on cancers by experiments or have a certain relationship with cancers reported in published papers, indicating the utility of the integrated method. It is also clear that the integrated method can provide more comprehensive analysis of various diseases because it is capable of producing more possible disease genes than the RWR-based method or the SP-based method.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Materials and methods"></a><h2>Materials and methods</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Genes related to oral cancer"></a>
<h3>Genes related to oral cancer</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>The OC-related genes were collected from the following three sources: (1) 44 genes were retrieved from UniProt (<a href="http://www.uniprot.org/">http://www.uniprot.org/</a>, accessed in July, 2015) [<a href="#pone.0175185.ref041" class="ref-tip">41</a>] after ‘human oral cancer reviewed’ was input as a keyword; (2) seven genes were chosen from the catalogue of oral cancer from the TSGene (<a href="https://bioinfo.uth.edu/TSGene/">https://bioinfo.uth.edu/TSGene/</a>, accessed in July, 2015) database; and (3) 156 genes were retrieved from the NCI (National Cancer Institute, <a href="https://gforge.nci.nih.gov">https://gforge.nci.nih.gov</a>, accessed in July, 2015) database using ‘Homo sapiens’ as a keyword. After combining the OC-related genes mentioned above, 202 OC-related genes were finally obtained; these genes are provided in <a href="#pone.0175185.s002">S1 Table</a>. Because our methods are based on the network constructed from the PPI information retrieved from Search Tool for the Retrieval of Interacting Genes/Proteins (STRING) [<a href="#pone.0175185.ref042" class="ref-tip">42</a>], these 202 OC-related genes were mapped into their Ensembl IDs, and those not occurring in the network were discarded. One hundred eighty-four Ensembl IDs of OC-related genes were accessed and composed the set <em>S</em><sub>oc</sub>.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Protein-protein interactions"></a>
<h3>Protein-protein interactions</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>PPIs play important roles in several intracellular and intercellular biochemical processes. The known PPI information has been widely used to investigate several protein-related problems, such as protein function prediction [<a href="#pone.0175185.ref043" class="ref-tip">43</a>, <a href="#pone.0175185.ref044" class="ref-tip">44</a>] and disease gene identification [<a href="#pone.0175185.ref033" class="ref-tip">33</a>, <a href="#pone.0175185.ref034" class="ref-tip">34</a>, <a href="#pone.0175185.ref036" class="ref-tip">36</a>, <a href="#pone.0175185.ref039" class="ref-tip">39</a>]. Based on the results reported in these previous studies, it can be concluded that two proteins that can interact with each other always have a functional relationship. In this study, we attempted to discover novel candidate OC-related genes using validated OC-related genes. Thus, we can utilize PPIs to search for proteins that have a functional relationship with proteins encoded by validated OC-related genes, thereby mining novel OC-related genes.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>The PPI information was retrieved from STRING (<a href="http://string-db.org/">http://string-db.org/</a>, version 9.1) [<a href="#pone.0175185.ref042" class="ref-tip">42</a>], a well-known online public database that contains known and predicted protein interactions. The interactions reported in STRING are derived from the following four types of sources: (I) Genomic Context; (II) High-throughput Experiments; (III) (Conserved) Coexpression; (IV) Previous Knowledge, which include direct (physical) and indirect (functional) associations between proteins and thus offer more chances to mine hidden protein information. From the database, a file called ‘<a href="http://string91.embl.de/newstring_download/protein.links.v9.1.txt.gz">protein.links.v9.1.txt.gz</a>‘ was retrieved, which contained 2,425,314 human PPIs. Each PPI was represented by two Ensembl IDs and a score ranging between 150 and 999, which indicates the strength of the interaction; i.e., proteins in interactions with higher scores are regarded as more likely to interact with each other. Let us denote the score of a PPI between proteins <em>p</em><sub>1</sub> and <em>p</em><sub>2</sub> by <em>S</em>(<em>p</em><sub>1</sub>, <em>p</em><sub>2</sub>).</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="RWR-based method"></a>
<h3>RWR-based method</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>RWR is a ranking algorithm [<a href="#pone.0175185.ref030" class="ref-tip">30</a>]. In a network, this algorithm simulates a walker starting from a seed node or a set of seed nodes and random walking on the network. In each step, the probability of the walker walking to each node is updated, and it stabilizes after several steps. Nodes in the network are ranked by the final probabilities assigned to them. Based on the validated disease genes (used as seed nodes), the RWR algorithm can be used to discover novel disease genes by investigating novel genes with high ranks. This algorithm has therefore been used to mine novel disease genes in recent years [<a href="#pone.0175185.ref030" class="ref-tip">30</a>–<a href="#pone.0175185.ref032" class="ref-tip">32</a>]. However, the RWR algorithm consistently provides false discoveries [<a href="#pone.0175185.ref040" class="ref-tip">40</a>], supporting the need for additional screening rules. In this study, we used the RWR algorithm as the basic algorithm and added some rules to build the present RWR-based learning method for the identification of novel OC-related genes. Furthermore, to obtain a better RWR-based method, some parameters were employed that are determined by training the method.</p>

<div id="section1" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Network construction for RWR algorithm"></a><h4>Network construction for RWR algorithm.</h4><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1" class="link-target"></a><p>An accurate network is important for the identification of novel OC-related genes. Here, we adopted the PPI information mentioned in the Section “Protein-protein interactions” to build the network. The constructed network defined the 20,770 proteins occurring in the 2,425,314 human PPIs as nodes, and two nodes were adjacent if and only if the corresponding proteins could interact with each other. It can be observed that each edge represented a PPI. To employ the interaction score in the network, each edge was assigned a weight that was defined as the score of the corresponding PPI. For convenience, let us denote the constructed network by <em>N</em><sub>RWR</sub>.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec007" name="sec007" class="link-target" title="Searching for new candidate genes using the RWR algorithm"></a><h4>Searching for new candidate genes using the RWR algorithm.</h4><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1" class="link-target"></a><p>As mentioned in the Section “Genes related to oral cancer”, a set <em>S</em><sub>oc,</sub> consisting of 184 Ensembl IDs of OC-related genes was used in this study. The RWR algorithm simulated a random walker starting from <em>S</em><sub>oc</sub>. Before executing the RWR algorithm on <em>N</em><sub>RWR</sub>, each node in <em>N</em><sub>RWR</sub> was assigned a probability, with 1/184 set for the Ensembl IDs in <em>S</em><sub>oc</sub> and 0 set for the remaining nodes. The initial probability of each node constituted the probability vector <em>P</em><sub>0</sub>. The RWR algorithm updated the probability vector in each step. Let <em>P</em><sub><em>t</em></sub> denote the probability vector after performing the <em>t</em>-th step, which can be updated by
<a name="pone.0175185.e001" id="pone.0175185.e001" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e001" class="inline-graphic"><span class="note">(1)</span></span>
where <em>r</em> is set to 0.8 and <em>A</em> is the column-wise normalized adjacency matrix of <em>N</em><sub>RWR</sub>. The update procedure was repeated until the change between P<sub><em>t</em></sub> and <em>P</em><sub><em>t</em>+1</sub>, measured by <em>L</em><sub>1</sub> norm, was less than 1e-6. Ensembl IDs with high probabilities were considered to be encoded by OC-related genes. Because we did not know which probability was a suitable threshold for selecting candidate genes, we established a parameter, <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e002" class="inline-graphic"></span>, for the threshold, which can be determined by training the RWR-based method.</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="SP-based method"></a>
<h3>SP-based method</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>The SP algorithm is a classic graph algorithm. In recent years, some investigators have applied the SP algorithm for the identification of disease genes [<a href="#pone.0175185.ref033" class="ref-tip">33</a>–<a href="#pone.0175185.ref039" class="ref-tip">39</a>]. The new candidate genes were extracted from the shortest paths connecting any two validated disease genes. Here, we built an SP-based machine learning method to identify novel OC-related genes. Like the RWR-based method, further screening rules were established to discard non-essential candidate genes and select important ones.</p>

<div id="section1" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Network construction for SP algorithm"></a><h4>Network construction for SP algorithm.</h4><a id="article1.body1.sec2.sec4.sec1.p1" name="article1.body1.sec2.sec4.sec1.p1" class="link-target"></a><p>Similar to <em>N</em><sub>RWR</sub> for the RWR-based method, we also constructed the network <em>N</em><sub>SP</sub>. As mentioned in the Section “Network construction for RWR-based method”, each edge in <em>N</em><sub>RWR</sub> represented a PPI. In addition, each PPI had an interaction score as described in the Section “Protein-protein interactions”. This score was used to define the weight of the edge. The range of the interaction score was between 150 and 999, and an edge with a low weight indicated strong correlations between its endpoints in the SP-based model. Thus, for an edge <em>e</em> with endpoints <em>n</em><sub>1</sub> and <em>n</em><sub>2</sub>, weight was defined in the following manner:
<a name="pone.0175185.e003" id="pone.0175185.e003" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e003" class="inline-graphic"><span class="note">(2)</span></span>
where <em>p</em><sub>1</sub> and <em>p</em><sub>2</sub> were two corresponding proteins of nodes <em>n</em><sub>1</sub> and <em>n</em><sub>2</sub>. The constructed network <em>N</em><sub>SP</sub> consisted of 20,770 nodes and 2,425,314 edges.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Searching for new candidate genes using the SP algorithm"></a><h4>Searching for new candidate genes using the SP algorithm.</h4><a id="article1.body1.sec2.sec4.sec2.p1" name="article1.body1.sec2.sec4.sec2.p1" class="link-target"></a><p>The SP algorithm, Dijkstra’s algorithm [<a href="#pone.0175185.ref045" class="ref-tip">45</a>], was applied to the network <em>N</em><sub>SP</sub> to search for all of the shortest paths connecting any two Ensembl IDs in <em>S</em><sub>oc</sub>. From the obtained shortest paths, we extracted the inner nodes that did not represent validated OC-related genes. Genes corresponding to the extracted nodes were considered to be related to OC and denoted candidate genes. In addition, each candidate gene was assigned a measurement, called betweenness, which was defined as the number of shortest paths within which it was contained.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="Screening rules"></a>
<h3>Screening rules</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>The RWR and SP algorithms can produce a number of candidate genes for OC when the probability threshold is provided for the RWR algorithm. However, several false discoveries are inevitable, as mentioned in previous studies [<a href="#pone.0175185.ref033" class="ref-tip">33</a>, <a href="#pone.0175185.ref040" class="ref-tip">40</a>]. To screen out these false discoveries, a series of screening rules were added to this section and integrated into the RWR-based and SP-based methods. Some parameters were employed to build the rules. Their optimal values in RWR-based and SP-based methods are determined by training these two methods, respectively.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>The probability that a candidate gene produced by the RWR algorithm was influenced by the structure of the network <em>N</em><sub>RWR</sub> and the betweenness for a candidate gene produced by the SP algorithm was also clearly influenced by the structure of the <em>N</em><sub>SP</sub> network. Some candidate genes receiving high probabilities (betweenness) were not specific to OC. Thus, a permutation test was designed that first randomly constructed 1,000 sets of Ensembl IDs such that each set had the same size of <em>S</em><sub>oc</sub>, denoted by <em>S</em><sub>1,</sub> <em>S</em><sub>2</sub>,⋯, <em>S</em><sub>1000</sub>. For each set, the RWR algorithm and SP algorithm were executed on <em>N</em><sub>RWR</sub> and <em>N</em><sub>SP</sub>, respectively, by setting the Ensembl IDs in the set as the input, yielding a probability for each candidate gene produced by the RWR algorithm and a betweenness for each candidate gene produced by the SP algorithm. Finally, for each candidate gene, there was one probability (betweenness) for <em>S</em><sub>oc</sub> and 1,000 probabilities (betweenness) for <em>S</em><sub>1</sub>, <em>S</em><sub>2</sub>,⋯,<em>S</em><sub>1000</sub>. If the candidate gene was specific to OC, its probability (betweenness) for <em>S</em><sub>oc</sub> should clearly be larger than most probabilities (betweenness) for <em>S</em><sub>1</sub>, <em>S</em><sub>2</sub>,⋯,<em>S</em><sub>1000</sub>. Thus, we calculated the p-value for each candidate gene <em>g</em>, which was defined by
<a name="pone.0175185.e004" id="pone.0175185.e004" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e004" class="inline-graphic"><span class="note">(3)</span></span>
where <em>δ</em><sub><em>i</em></sub> = 1 if the probability (betweenness) for <em>S</em><sub><em>i</em></sub> was larger than that of <em>S</em><sub>oc</sub>; <em>δ</em><sub><em>i</em></sub> = 0 otherwise. Because 0.05 was always used as an important cut-off for the significance level of the test, it was set to be the threshold of the p-value; <em>i</em>.<em>e</em>., candidate genes with p-values less than 0.05 were selected.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3" class="link-target"></a><p>The network-based method is useful for complicated problems. However, this type of method seldom considers the essential properties of nodes in the network, leading to several false discoveries. Rules utilizing the essential properties of candidate and validated genes are needed to exclude false discoveries and select important ones.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4" class="link-target"></a><p>As mentioned in the Section “Protein-protein interactions”, two proteins that can interact with each other always have a functional relationship. Furthermore, considering the interaction scores, proteins in an interaction with a high score are more likely to have a strong functional relationship than those in an interaction with a low score. Thus, the interaction score can be used to measure the associations between candidate genes and OC-related genes. For each candidate gene <em>g</em>, we calculated the maximum interaction score (<em>MIS</em>), which was defined by
<a name="pone.0175185.e005" id="pone.0175185.e005" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e005" class="inline-graphic"><span class="note">(4)</span></span></p>
<a id="article1.body1.sec2.sec5.p5" name="article1.body1.sec2.sec5.p5" class="link-target"></a><p>Clearly, a candidate gene with a high <em>MIS</em> is more likely to be a novel OC-related gene and thus should be selected. However, the threshold of <em>MIS</em> was not easy to determine. Thus, it was set as a parameter, <em>p</em><sub><em>MIS</em></sub>. In the RWR-based method, the optimal value of <em>p</em><sub><em>MIS</em></sub> is determined by training this method. In addition, the optimal value of <em>p</em><sub><em>MIS</em></sub> in the SP-based method is also accessed by training the SP-based method.</p>
<a id="article1.body1.sec2.sec5.p6" name="article1.body1.sec2.sec5.p6" class="link-target"></a><p>The <em>MIS</em> measures the associations between candidate genes and OC-related genes. The following measurement evaluates the associations between them in another way. It is known that OC-related genes must be highly related to some gene ontology (GO) terms or biological pathways. Thus, we can use the annotated GO term and KEGG pathway information for candidate genes and OC-related genes to evaluate their correlations. To achieve this goal, each candidate gene or OC-related gene was encoded by its GO enrichment scores and KEGG enrichment scores. For a candidate gene <em>g</em> and an OC-related gene <em>g</em>′, their associations can be measured by
<a name="pone.0175185.e006" id="pone.0175185.e006" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e006" class="inline-graphic"><span class="note">(5)</span></span>
where <em>ES</em>(<em>g</em>) (<em>ES</em>(<em>g</em>′), respectively) is a vector consisting of the GO enrichment scores and KEGG enrichment scores of <em>g</em> (<em>g</em>′, respectively). A high value for <a href="#pone.0175185.e006">Eq 5</a> indicated a strong association. Similar to the definition of <em>MIS</em>, we calculated the maximum function score (<em>MFS</em>) of each candidate gene <em>g</em> by
<a name="pone.0175185.e007" id="pone.0175185.e007" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e007" class="inline-graphic"><span class="note">(6)</span></span></p>
<a id="article1.body1.sec2.sec5.p7" name="article1.body1.sec2.sec5.p7" class="link-target"></a><p>Similarly, a candidate gene with a high <em>MFS</em> might be a novel OC-related gene with a high probability and should be selected. Additionally, it was difficult to determine the threshold of <em>MFS</em>. The parameter <em>p</em><sub><em>MFS</em></sub> was also established for this threshold. Similar to parameter <em>p</em><sub><em>MIS</em></sub> mentioned above, the optimal value of <em>p</em><sub><em>MFS</em></sub> in the RWR-based and SP-based methods is determined by training these two methods, respectively.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="Integrated method"></a>
<h3>Integrated method</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>The RWR-based and SP-based methods, together with the screen rules mentioned in the Section “Screening rules”, were constructed. The pseudo-codes of these two methods are listed in Tables <a href="#pone-0175185-t001">1</a> and <a href="#pone-0175185-t002">2</a>, respectively. The integrated method encompassed these two methods by combining their results.</p>
<a class="link-target" id="pone-0175185-t001" name="pone-0175185-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.t001" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> The pseudo-code of the RWR-based method.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.t001">
              https://doi.org/10.1371/journal.pone.0175185.t001</a></p></div><a class="link-target" id="pone-0175185-t002" name="pone-0175185-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.t002" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> The pseudo-code of the SP-based method.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.t002">
              https://doi.org/10.1371/journal.pone.0175185.t002</a></p></div></div>

<div id="section7" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="Evaluation methods"></a>
<h3>Evaluation methods</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>The RWR-based and SP-based methods were applied to identify novel OC-related genes. However, there are some parameters in these two methods that must be determined before they are used to identify novel OC-related genes. Thus, these two methods were trained on the validated OC-related gene set <em>S</em><sub>oc</sub>, through which the optimal parameters can be determined. We used the jackknife test [<a href="#pone.0175185.ref046" class="ref-tip">46</a>, <a href="#pone.0175185.ref047" class="ref-tip">47</a>], which is one of the classic cross-validation methods [<a href="#pone.0175185.ref048" class="ref-tip">48</a>, <a href="#pone.0175185.ref049" class="ref-tip">49</a>], to evaluate the performance of these two methods, <em>i</em>.<em>e</em>., each OC-related gene in <em>S</em><sub>oc</sub> was singled out sequentially, and the remaining genes in <em>S</em><sub>oc</sub> were used to generate predictions under various combinations of parameters. When training the methods, we supposed that all genes in the network other than the validated OC-related genes were negative; <em>i</em>.<em>e</em>., they were not OC-related genes. The performance of the method can be measured according to the following two features: (1) whether the selected OC-related gene can be recovered by executing the method on the remaining OC-related genes; (2) the predicted genes other than the selected OC-related gene should be as low as possible. Thus, we considered the following two measurements: precision and recall, which are always used to evaluate the performance of the methods on a binary classification problem in the fields of <a href="https://en.wikipedia.org/wiki/Pattern_recognition">pattern recognition</a> and information retrieval. Recall is defined as the proportion of retrieved OC-related genes among all OC-related genes, and precision represents the proportion of retrieved OC-related genes among all retrieved genes. Furthermore, another measurement, the F1-measure, is often used to evaluate overall performance, and it can be calculated by
<a name="pone.0175185.e009" id="pone.0175185.e009" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e009" class="inline-graphic"><span class="note">(7)</span></span></p>
<a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2" class="link-target"></a><p><a href="#pone.0175185.e009">Eq 7</a> shows that recall and precision have the same role. However, in this study, we considered recall to be more important than precision because the method with low recall and high precision could not reliably produce the predicted results. Thus, we simply revised <a href="#pone.0175185.e009">Eq 7</a> and defined a new measurement, namely, F1-measure-R, which can be computed by
<a name="pone.0175185.e010" id="pone.0175185.e010" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e010" class="inline-graphic"><span class="note">(8)</span></span></p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3" class="link-target"></a><p>Because the retrieved genes differ when the selected OC-related gene was not the same, the recall, precision and F1-measure-R must be considered each time for the predicted results. Thus, under a combination of parameters, the RWR-based and SP-based methods can produce a series of recall, precision and F1-measure-R values. We calculated the average values to indicate the performance of the method under this combination of parameters. For convenience, the recall, precision and F1-measure-R presented in the rest of this paper represent the average values.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec014" name="sec014" data-toc="sec014" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec015" name="sec015" class="link-target" title="Optimized parameters for the RWR-based and SP-based methods"></a>
<h3>Optimized parameters for the RWR-based and SP-based methods</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>As mentioned in the Section “RWR-based method”, “SP-based method” and “Screening rule”, some parameters should be optimized for the RWR-based method and the SP-base method. To extract an optimal combination of parameters for each method, the RWR-based method and the SP-base method were trained on <em>S</em><sub>oc</sub>, and their performance was evaluated by the Jackknife test.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>For the RWR-based method, three parameters, the threshold of probability <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e011" class="inline-graphic"></span>, the threshold of MIS <em>p</em><sub><em>MIS</em></sub>, and the threshold of MFS <em>p</em><sub><em>MFS</em></sub>, should be optimized. For <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e012" class="inline-graphic"></span>, we tried various values ranging from 1E-05 to 1E-04; for <em>p</em><sub><em>MIS</em></sub>, we tried three values, 400, 700, 900, which are reported in STRING for thresholds of medium confidence, high confidence and highest confidence, respectively; for <em>p</em><sub><em>MFS</em></sub>, we tried various values ranging from 0 to 0.9. The measurements mentioned in the Section “Evaluation methods” for the RWR-based method with different combinations of parameters are listed in <a href="#pone.0175185.s003">S2 Table</a>. For ease of observation, the values of F1-measure-R obtained using the RWR-based method with different parameters are illustrated in <a href="#pone-0175185-g001">Fig 1</a>; we can see the same level of performance when the parameters <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e013" class="inline-graphic"></span> and <em>p</em><sub><em>MFS</em></sub> are equivalent. The maximum F1-measure-R was 1.677E-03 when the parameters were set to be <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e014" class="inline-graphic"></span>, <em>p</em><sub><em>MFS</em></sub> = 0.8 and <em>p</em><sub><em>MIS</em></sub> = 400 or 700. Because this combination of parameters yielded the best performance, they were used to build the optimal RWR-based method, which is adopted to identify novel OC-related genes.</p>
<a class="link-target" id="pone-0175185-g001" name="pone-0175185-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.g001" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> The performance of the RWR-based method under different combinations of parameters.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p>(a) The performance of the RWR-based method setting <em>p</em><sub><em>MIS</em></sub> = 400. (b) The performance of the RWR-based method setting <em>p</em><sub><em>MIS</em></sub> = 700. (c) The performance of the RWR-based method setting <em>p</em><sub><em>MIS</em></sub> = 900.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.g001">
              https://doi.org/10.1371/journal.pone.0175185.g001</a></p></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>The SP-based method was also trained to extract the optimal parameters for <em>p</em><sub><em>MIS</em></sub> and <em>p</em><sub><em>MFS</em></sub>. We tested three values for <em>p</em><sub><em>MIS</em></sub>, as mentioned in the above paragraph, and various values ranging from 0 to 0.9 for <em>p</em><sub><em>MFS</em></sub>. For the results obtained using the SP-based method with different combinations of parameters, the measurements mentioned in the Section “Evaluation methods” were counted and are provided in <a href="#pone.0175185.s004">S3 Table</a>. Additionally, three curves are plotted in <a href="#pone-0175185-g002">Fig 2</a> to show the values of F1-measure-R obtained using the SP-based method with different values of <em>p</em><sub><em>MFS</em></sub> and a fixed value of <em>p</em><sub><em>MIS</em></sub>. When the values of <em>p</em><sub><em>MFS</em></sub> were small, the values of F1-measure-R were proportional to the value of <em>p</em><sub><em>MIS</em></sub>. However, the values of F1-measure-R were almost the same when <em>p</em><sub><em>MFS</em></sub> were large. The maximum F1-measure-R was 2.693E-04 when the parameters were set to be <em>p</em><sub><em>MFS</em></sub> = 0.8 and <em>p</em><sub><em>MIS</em></sub> = 400 or 700 or 900. Similarly, we used these values to build the optimal SP-based method for the identification of novel OC-related genes.</p>
<a class="link-target" id="pone-0175185-g002" name="pone-0175185-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.g002" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> The performance of the SP-based method under different combinations of parameters.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1" class="link-target"></a><p>There are three lines in this figure, which represent the performance of the SP-based method with different thresholds of maximum interaction score. In detail, the full line represents the performance of the SP-based method with the threshold of maximum interaction score 900, the dot line represents the performance of the SP-based method with the threshold of maximum interaction score 700, the dash line represents the performance of the SP-based method with the threshold of maximum interaction score 400.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.g002">
              https://doi.org/10.1371/journal.pone.0175185.g002</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec016" name="sec016" class="link-target" title="Inferred results of the RWR-based method"></a>
<h3>Inferred results of the RWR-based method</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>As mentioned in the Section “Optimized parameters for the RWR-based and SP-based methods”, the optimal RWR-based method was built using <span class="inline-formula"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0175185.e015" class="inline-graphic"></span>, <em>p</em><sub><em>MFS</em></sub> = 0.8 and <em>p</em><sub><em>MIS</em></sub> = 400 or 700. This method was further used to identify novel OC-related genes based on all validated OC-related genes mentioned in the Section “Genes related to oral cancer”. Because the optimal RWR-based method contains two options for the parameter <em>p</em><sub><em>MIS</em></sub>, this method was executed twice. First, <em>p</em><sub><em>MIS</em></sub> was set to 400, while in the second iteration, it was set to 700. The genes identified using this method with a different parameter, <em>p</em><sub><em>MIS</em></sub>, are listed in <a href="#pone.0175185.s005">S4 Table</a>. A careful review of the results showed that the identified genes produced using these two optimal RWR-based methods were equivalent. They all identified 81 novel genes. Because all of these genes were viewed as high probability by the RWR method and had strong associations with validated OC-related genes, we believe that they are highly related to OC, and they are termed putative OC-related genes.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec017" name="sec017" class="link-target" title="Inferred results of the SP-based method"></a>
<h3>Inferred results of the SP-based method</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>The optimal SP-based method was built in the Section “Optimized parameters for the RWR-based and SP-based methods”, in which <em>p</em><sub><em>MFS</em></sub> was set to 0.8 and <em>p</em><sub><em>MIS</em></sub> was set to 400, 700 or 900. This method was also adopted to identify novel OC-related genes. By setting <em>p</em><sub><em>MIS</em></sub> to 400, 700 or 900, we can build three optimal SP-based methods. They were executed sequentially to identify novel OC-related genes. Similarly, they all yielded fifteen identical genes, which are listed in <a href="#pone-0175185-t003">Table 3</a>. It is believed that these genes are closely related to OC, and they are termed putative OC-related genes.</p>
<a class="link-target" id="pone-0175185-t003" name="pone-0175185-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.t003" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Genes identified by the optimal SP-based method.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.t003">
              https://doi.org/10.1371/journal.pone.0175185.t003</a></p></div></div>

<div id="section4" class="section toc-section"><a id="sec018" name="sec018" class="link-target" title="Inferred results of the integrated method"></a>
<h3>Inferred results of the integrated method</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>As mentioned in the Sections “Inferred results of the RWR-based method” and “Inferred results of the SP-based method”, the RWR-based method yielded 81 putative genes, and the SP-based method yielded fifteen putative genes. The union of these two putative gene sets provided the results of the integrated method, in which 85 putative genes were obtained. Their distribution is illustrated in <a href="#pone-0175185-g003">Fig 3</a>; we can see that eleven putative genes were identified by both the RWR-based and the SP-based methods. Among the remaining putative genes, 70 were identified using the RWR-based method, and four putative genes were identified using the SP-based method. Because the principles of the RWR-based and SP-based methods are very different, the identified novel OC-related genes were not the same. By considering the identified genes produced by either of them, we can obtain more putative genes and have an opportunity to extensively study disease genes in OC. To indicate the obtained 85 putative genes are highly related to OC, we extracted a sub-network containing these putative genes and OC-related genes from <em>N</em><sub>RWR</sub> and <em>N</em><sub>SP</sub> as shown in <a href="#pone-0175185-g004">Fig 4</a>.</p>
<a class="link-target" id="pone-0175185-g003" name="pone-0175185-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.g003" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> The distribution of the 85 putative OC-related genes obtained in this study.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1" class="link-target"></a><p>The blue part represents the set consisting of 70 putative genes obtained using only the RWR-based method. The red part represents the set consisting of 11 putative genes obtained using both the RWR-based and SP-based methods. The green part represents the set consisting of 4 putative genes obtained using only the SP-based method.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.g003">
              https://doi.org/10.1371/journal.pone.0175185.g003</a></p></div><a class="link-target" id="pone-0175185-g004" name="pone-0175185-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.g004" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> The sub-network containing the putative genes and OC-related genes that was extracted from the network for RWR-based and SP-based methods.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig2.caption1.p1" name="article1.body1.sec3.sec4.fig2.caption1.p1" class="link-target"></a><p>The blue nodes represent putative genes and red nodes represent OC-related genes.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.g004">
              https://doi.org/10.1371/journal.pone.0175185.g004</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="sec019" name="sec019" data-toc="sec019" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>Using the RWR-based method, 81 genes were obtained. Using the SP-based method, fifteen genes were accessed. All of these genes were deemed to be significantly associated with OC. Furthermore, eleven genes were identified by both methods, which may be more important than the others. In this section, some important putative genes (listed in Tables <a href="#pone-0175185-t004">4</a>–<a href="#pone-0175185-t006">6</a>) were extensively analyzed to confirm their associations with OC.</p>
<a class="link-target" id="pone-0175185-t004" name="pone-0175185-t004"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.t004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.t004" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.t004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.t004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 4. </span> Eleven putative genes identified using both RWR-based and SP-based methods.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.t004">
              https://doi.org/10.1371/journal.pone.0175185.t004</a></p></div><a class="link-target" id="pone-0175185-t005" name="pone-0175185-t005"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.t005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.t005" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.t005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.t005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 5. </span> Important genes among the seventy putative genes identified using the RWR-based method.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.t005">
              https://doi.org/10.1371/journal.pone.0175185.t005</a></p></div><a class="link-target" id="pone-0175185-t006" name="pone-0175185-t006"></a><div class="figure" data-doi="10.1371/journal.pone.0175185.t006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0175185.t006" data-doi="info:doi/10.1371/journal.pone.0175185" data-uri="info:doi/10.1371/journal.pone.0175185.t006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0175185.t006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0175185.t006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0175185.t006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0175185.t006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 6. </span> Four putative genes identified using the SP-based method.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0175185.t006">
              https://doi.org/10.1371/journal.pone.0175185.t006</a></p></div>
<div id="section1" class="section toc-section"><a id="sec020" name="sec020" class="link-target" title="Putative genes identified using both RWR-based and SP-based methods"></a>
<h3>Putative genes identified using both RWR-based and SP-based methods</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>The following eleven proteins were identified using both RWR-based and SP-based methods: CYP3A43, FGF4, NFKBIA, THBS1, IL1B, CXCL13, CYP2A6, SELE, STAT1, MYD88 and FGFR2, as listed in <a href="#pone-0175185-t004">Table 4</a>.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2" class="link-target"></a><p>MYD88 (myeloid differentiation primary response 88) encodes a cytosolic adapter protein that functions as a key signal transducer in the interleukin-1 and Toll-like receptor (TLRs) signaling pathways. TLRs and their ligands play a crucial role in inflammation and host defense [<a href="#pone.0175185.ref050" class="ref-tip">50</a>]. In a broad variety of tumor tissues and cell lines, aberrant expression of TLRs plays a role in tumor immune escape or resistance to apoptosis [<a href="#pone.0175185.ref051" class="ref-tip">51</a>, <a href="#pone.0175185.ref052" class="ref-tip">52</a>]. The expression of TLR4 and MyD88 is also aberrant and affects the down-stream signaling pathway [<a href="#pone.0175185.ref053" class="ref-tip">53</a>]. In this study, MyD88 was predicted using both RWR-based and SP-based methods, and thus it may be a potential target for OC.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3" class="link-target"></a><p>FGFR2 (fibroblast growth factor receptor 2) belongs to the FGFR tyrosine kinase family, which is one of the most frequently altered kinase families in some types of cancer [<a href="#pone.0175185.ref054" class="ref-tip">54</a>]. Point mutations have been observed in 12% of endometrial carcinomas [<a href="#pone.0175185.ref055" class="ref-tip">55</a>]. These studies suggest that FGFR2 may act together with the regulation of the PI3K/AKT/mTOR pathway to drive endometrial cancer growth in a subset of patients [<a href="#pone.0175185.ref056" class="ref-tip">56</a>, <a href="#pone.0175185.ref057" class="ref-tip">57</a>]. FGFR2 mutations also occur in oral squamous cell carcinoma, and a patient with OSCC was found to respond to pazopanib, a multiple tyrosine kinase inhibitor [<a href="#pone.0175185.ref058" class="ref-tip">58</a>]. These findings support the potential diagnostic and therapeutic value of FGFR2 as an effective strategy for the treatment of patients with OC with defined molecular characteristics. In addition, our results showed that FGF4 had a significant probability in both methods. Some large-scale experiments have suggested that aberrant amplification of FGF4 occurs in several types of cancer, including lymph node metastasis and urinary bladder cancer [<a href="#pone.0175185.ref059" class="ref-tip">59</a>, <a href="#pone.0175185.ref060" class="ref-tip">60</a>].</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4" class="link-target"></a><p>NF-κBIA (nuclear factor-kappa B inhibitor alpha) inhibits NF-κB, which is involved in inflammatory responses and is a hallmark linking inflammation to tumor development and progression [<a href="#pone.0175185.ref061" class="ref-tip">61</a>, <a href="#pone.0175185.ref062" class="ref-tip">62</a>]. The polymorphic variations in NFKBIA were associated with the risk of various cancers, including gastric cancer, prostate cancer and melanoma [<a href="#pone.0175185.ref063" class="ref-tip">63</a>–<a href="#pone.0175185.ref065" class="ref-tip">65</a>]. NFKBIA polymorphisms have significant associations with OSCC [<a href="#pone.0175185.ref066" class="ref-tip">66</a>]. In our study, NFKBIA had a close relationship with OC using both methods.</p>
<a id="article1.body1.sec4.sec1.p5" name="article1.body1.sec4.sec1.p5" class="link-target"></a><p>SELE (selectin E), which was found in cytokine-stimulated endothelial cells, functions in the accumulation of blood leukocytes at sites of inflammation [<a href="#pone.0175185.ref067" class="ref-tip">67</a>]. The level of SELE-mediated adhesion of colon cancer and head and neck squamous cell cancer cells to the endothelium has been implied to be involved in metastasis [<a href="#pone.0175185.ref068" class="ref-tip">68</a>, <a href="#pone.0175185.ref069" class="ref-tip">69</a>]. Our analysis revealed that SELE might be a putative marker for tumorigenesis in OC. A study in Taiwan supports the idea that SELE-related inflammation plays a crucial role in the pathogenesis process of OSCC [<a href="#pone.0175185.ref070" class="ref-tip">70</a>]. Future investigations of the function of SELE may clarify the mechanism of tumorigenesis and metastasis.</p>
<a id="article1.body1.sec4.sec1.p6" name="article1.body1.sec4.sec1.p6" class="link-target"></a><p>THBS1 (thrombospondin 1) is a subunit of a disulfide-linked homotrimeric protein, which mediates cell-cell and cell-matrix interactions [<a href="#pone.0175185.ref071" class="ref-tip">71</a>]. The significant relationship between THBS1 and OC was detected using both RWR-based and SP-based methods. Several studies have indicated that THBS1 has crucial functions in oral tumorigenesis [<a href="#pone.0175185.ref072" class="ref-tip">72</a>, <a href="#pone.0175185.ref073" class="ref-tip">73</a>]. THBS1 might be a potential diagnostic and therapeutic target for OC.</p>
<a id="article1.body1.sec4.sec1.p7" name="article1.body1.sec4.sec1.p7" class="link-target"></a><p>STAT1 (signal transducer and activator of transcription 1), which belongs to the STAT protein family, mediates the expression of a variety of genes and cell viability in response to stimuli and pathogens [<a href="#pone.0175185.ref074" class="ref-tip">74</a>]. Aberrant activation of STAT1 has frequently been found in various cancers, such as head and neck cancer [<a href="#pone.0175185.ref075" class="ref-tip">75</a>]. A similar STAT1 activation status was detected in patients with OSCC [<a href="#pone.0175185.ref076" class="ref-tip">76</a>]. In this study, STAT1 displayed a significant association with OC using both the RWR-based and SP-based methods, and this molecular marker could help in guiding diagnostic and therapeutic decisions in patients with OC.</p>
<a id="article1.body1.sec4.sec1.p8" name="article1.body1.sec4.sec1.p8" class="link-target"></a><p>Among these eleven putative genes, including genes such as MYD88, FGFR2 and THBS1, stimulative or suppressive effects on cancers have been shown by the experiments. We speculate that these three genes have certain functions in OC. In addition, it has been reported that other genes, including NFKBIA, SELE, STAT1, IL1B, CYP2A6, CYP3A43, FGF4 and CXCL13 [<a href="#pone.0175185.ref077" class="ref-tip">77</a>–<a href="#pone.0175185.ref080" class="ref-tip">80</a>], have a certain relationship with various forms cancer. However, the mechanism has not been thoroughly studied. In our analysis, they have a significant relationship with OC, and the mechanism should be explored.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec021" name="sec021" class="link-target" title="Putative genes identified only by the RWR-based method"></a>
<h3>Putative genes identified only by the RWR-based method</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>Seventy proteins were identified only by the RWR-based method. The important genes are BMP4, CCL2, CCL20, CCL5, CCR7, IFNG, KITLG, TNFRSF1A, PIK3CB, PIK3CD, PIK3CG, CYP2B6, CYP2C19, CYP2J2, CYP3A4, CYP4X1 and RAC3; these genes are listed in <a href="#pone-0175185-t005">Table 5</a>. Some of them participate in tumorigenesis by regulating cell growth or apoptosis, such as BMP4 and RAC3. Some genes are important factors in the immune system of cancer patients, such as CCL2, CCL20, CCL5, CCR7, IFNG and P450 family. Detailed function analyses of candidate genes are shown in <a href="#pone.0175185.s001">S1 File</a>.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2" class="link-target"></a><p>We identified several novel putative genes using only the RWR-based method, which have strong associations with the tumorigenesis of OC. The functions of some genes have been explored in experiments, such as BMP4, IFNG, KITLG, CCL5, CCR5, CCR7, CYP2J2, PIK3CB, PIK3CG and RAC3; others revealed mutations or aberrant expression in cancers, but the mechanism is not clear. Future research is required to replicate and validate the effects of these genes.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec022" name="sec022" class="link-target" title="Putative genes identified only by the SP-based method"></a>
<h3>Putative genes identified only by the SP-based method</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Four proteins were predicted to be closely associated with OC using the SP-based method but not the RWR-based method. These genes were IL1R1, MCM6, NOG and CXCR3; they are listed in <a href="#pone-0175185-t006">Table 6</a>. It has been reported that CXCR3 can promote metastasis in some types of cancer [<a href="#pone.0175185.ref081" class="ref-tip">81</a>–<a href="#pone.0175185.ref086" class="ref-tip">86</a>]. Other genes showed aberrant expression or mutations in cancers, which merits attention. Detailed function analyses of candidate genes are shown in <a href="#pone.0175185.s001">S1 File</a>.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2" class="link-target"></a><p>Based on the analyses of the novel genes obtained by the integrated method, we found that several genes have shown stimulative or suppressive effects on cancers by experiments or aberrant expression or mutations in cancers reported in published papers, implying the effectiveness of the integrated method. In addition, the integrated method can provide more potential disease genes for investigating OC because it combines the results yielded by the RWR-based and SP-based methods; i.e., simultaneous usage of the RWR-based and SP-based method can help us mine more information about disease genes in oral cancer. We believe that it is helpful to investigate various diseases using both the RWR-based and the SP-based methods.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec023" name="sec023" class="link-target" title="GO-term and pathway function enrichment analysis of putative genes using DAVID"></a>
<h3>GO-term and pathway function enrichment analysis of putative genes using DAVID</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>By the integrated method, 85 putative genes were obtained. To uncover the biological meaning behind these genes, the functional annotation tool, the Database for Annotation, Visualization and Integrated Discovery (DAVID) [<a href="#pone.0175185.ref087" class="ref-tip">87</a>], was adopted to analyze them. The obtained results are provided in <a href="#pone.0175185.s006">S5 Table</a>.</p>
<a id="article1.body1.sec4.sec4.p2" name="article1.body1.sec4.sec4.p2" class="link-target"></a><p>For the GO enrichment results yielded by DAVID (see <a href="#pone.0175185.s006">S5 Table</a>), we can see that seven-two biological process (BP) GO terms, fifteen molecular function (MF) GO terms and nine cellular component (CC) GO terms are with statistically significance (FDR&lt;0.05). The biological process mostly focused on cell proliferation, apoptosis, transcription and signal transduction. Malignant proliferation and anti-apoptosis are the major characteristics of many cancers. In this study, it has been found many proliferation related genes such as EGF, IGF1 and CDKN1B. EGF (epidermal growth factor) and its receptor EGFR affect cellular processes like proliferation, motility and adhesion. EGF has a higher expression and is correlated with the progression of cancer such as breast cancer [<a href="#pone.0175185.ref088" class="ref-tip">88</a>]. The circulating IGF1 (insulin-like growth factor-1) level is associated with the risk to develop breast cancer [<a href="#pone.0175185.ref089" class="ref-tip">89</a>, <a href="#pone.0175185.ref090" class="ref-tip">90</a>]. CDKN1B functions as a key cell cycle gatekeeper to prevent or slow down cell division [<a href="#pone.0175185.ref091" class="ref-tip">91</a>, <a href="#pone.0175185.ref092" class="ref-tip">92</a>]. Aberrant downregulating CDKN1B may promote the proliferation of multiple myeloma cells [<a href="#pone.0175185.ref093" class="ref-tip">93</a>]. Un-controlled expanded number of cancers is determined not only by cell proliferation but also by the cell evading apoptosis. In our study, many genes are enriched in apoptosis process such as TP53, CASP3 and cmyc. The mutations of TP53 are the most common alteration in cancer and are related to cell apoptosis, cell cycle and malignancy [<a href="#pone.0175185.ref094" class="ref-tip">94</a>–<a href="#pone.0175185.ref097" class="ref-tip">97</a>]. TP53 could arrest cell cycle and induced cell apoptosis in response to DNA damage [<a href="#pone.0175185.ref098" class="ref-tip">98</a>]. It has been reported that over-expression of c-Myc drives the level of BAX and other apoptosis-related genes and is involved in cell apoptosis [<a href="#pone.0175185.ref099" class="ref-tip">99</a>–<a href="#pone.0175185.ref103" class="ref-tip">103</a>]. CASPs is a kind of cysteine-dependent aspartate-specific proteases, and is associated with the initiation and execution of apoptosis. As an activate effector, CASP3 receives the apoptotic signals to perform the cell death process [<a href="#pone.0175185.ref104" class="ref-tip">104</a>]. In our result, a series of genes were enriched in the GO term related apoptosis, which proved process of apoptosis is a key process in oral cancer. In addition, several other biological process GO terms were shown such as transcription and signal transduction. In proliferation, apoptosis and other processes, functional genes were selectively transcribed and expressed. For example, c-Myc is a transcription factor which could mediate a series of down-stream genes express to drive cellular proliferation and apoptosis [<a href="#pone.0175185.ref105" class="ref-tip">105</a>]. It can be seen from <a href="#pone.0175185.s006">S5 Table</a> that putative genes were enriched in several CC GO terms and MF GO terms such as growth factor activity (MF), cytokine activity (MF), enzyme or protein binding (MF), extracellular region (CC), extracellular space (CC) and plasma membrane (CC). These results suggest that the functional activity and localization of the protein are directly or indirectly related to oral cancer.</p>
<a id="article1.body1.sec4.sec4.p3" name="article1.body1.sec4.sec4.p3" class="link-target"></a><p>The DAVID also produced the KEGG pathway enrichment on putative genes (see <a href="#pone.0175185.s006">S5 Table</a>), several pathways were highlighted such as TNF signaling pathway, PI3K-Akt pathway, Pap1 pathway, MAPK pathway, TLR pathway, NF-kB pathway, JaK-STAT, Ras pathway and cytokine-cytokine receptor interaction. TNF pathway could be activated by various signals to affect the immunity, cell growth, apoptosis and other biological behaviors of tumor cells [<a href="#pone.0175185.ref106" class="ref-tip">106</a>–<a href="#pone.0175185.ref110" class="ref-tip">110</a>]. Protein-protein interaction and pathways is a cross-talk network. TNF can also activates the NF-kB pathway. The NF-kB pathway is activated in various cancers [<a href="#pone.0175185.ref111" class="ref-tip">111</a>, <a href="#pone.0175185.ref112" class="ref-tip">112</a>]. Activation of NF-kB can be inhibited by the blockade of PI3K/Akt and ERK pathway to suppress tumor metastasis [<a href="#pone.0175185.ref113" class="ref-tip">113</a>]. It was reported that PI3K/AKT-NF-kB is an axis which promotes bone metastasis in prostate cancer [<a href="#pone.0175185.ref114" class="ref-tip">114</a>]. The Jak/STAT pathway is critical in normal tissues and tumors and the Jak kinase family includes JAK1, JAK2, JAK3 and TYK2 [<a href="#pone.0175185.ref115" class="ref-tip">115</a>]. Jak mediated STAT phosphorylation leads to their nuclear translocation. STAT molecules bind specific promoter of genes and result in the transcription in nucleus, which regulate the cell proliferation, differentiation and apoptosis [<a href="#pone.0175185.ref115" class="ref-tip">115</a>, <a href="#pone.0175185.ref116" class="ref-tip">116</a>]. It was reported that over-activation of the JAK/STAT pathway is related to subsets of patients with certain solid tumors and chronic myeloid leukemia [<a href="#pone.0175185.ref117" class="ref-tip">117</a>, <a href="#pone.0175185.ref118" class="ref-tip">118</a>]. MAPK pathway is induced by activation of TLR (toll like receptors) and NOD (nucleotide-binding oligomerization domain receptors), which initiates of inflammation and are involved in cancer proliferation and control [<a href="#pone.0175185.ref119" class="ref-tip">119</a>]. Ras pathway is a highly conserved pathway in cell function, including cell proliferation, differentiation and signaling transduction. This pathway is commonly deregulated in cancers, making the components in the pathway as targets for therapeutic interventions [<a href="#pone.0175185.ref120" class="ref-tip">120</a>]. In these pathways, several putative genes were enriched, which indicates these pathway and gene need more attention in tumorigenesis of oral cancer.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="sec024" name="sec024" data-toc="sec024" class="link-target" title="Conclusions"></a><h2>Conclusions</h2><a id="article1.body1.sec5.p1" name="article1.body1.sec5.p1" class="link-target"></a><p>In this study, we investigated genes related to oral cancer. Two popular algorithms, the random walk with a restart algorithm and the shortest path algorithm, which are often used to identify novel disease genes, were integrated with some further rules to build two computational methods. An integrated method was further built by combining these two methods. To access an optimal prediction method for the identification of genes related to oral cancer using the integrated method, these two methods were trained on validated genes. The optimal prediction method was further adopted to identify novel genes related to oral cancer. The results indicated the following facts: (1) The integrated method is effective for identification of disease genes of oral cancer; (2) Candidate genes produced by the integrated method provide an opportunity to achieve a more extensive investigation of oral cancer. We hope that the integrated method can be useful for identifying novel disease genes. In view of the utility of the RWR-based and SP-based methods for identification of disease genes, it is hopeful that these methods can be applied to investigate other problems, such as DNA-binding protein prediction [<a href="#pone.0175185.ref121" class="ref-tip">121</a>], protein fold recognition [<a href="#pone.0175185.ref122" class="ref-tip">122</a>, <a href="#pone.0175185.ref123" class="ref-tip">123</a>], detection of tubule boundary [<a href="#pone.0175185.ref124" class="ref-tip">124</a>], <em>etc</em>.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec025" name="sec025" data-toc="sec025" class="link-target" title="Supporting information"></a><h2>Supporting information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0175185"></div><div class="supplementary-material"><a name="pone.0175185.s001" id="pone.0175185.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0175185.s001">S1 File. </a>Detailed analysis of the putative genes obtained by RWR-based method or SP-based method.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0175185.s001">https://doi.org/10.1371/journal.pone.0175185.s001</a></p><a id="article1.body1.sec6.supplementary-material1.caption1.p1" name="article1.body1.sec6.supplementary-material1.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0175185.s002" id="pone.0175185.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0175185.s002">S1 Table. </a>202 genes related to oral cancer and their sources.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0175185.s002">https://doi.org/10.1371/journal.pone.0175185.s002</a></p><a id="article1.body1.sec6.supplementary-material2.caption1.p1" name="article1.body1.sec6.supplementary-material2.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0175185.s003" id="pone.0175185.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0175185.s003">S2 Table. </a>The performance of the RWR-based method with different combinations of parameters.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0175185.s003">https://doi.org/10.1371/journal.pone.0175185.s003</a></p><a id="article1.body1.sec6.supplementary-material3.caption1.p1" name="article1.body1.sec6.supplementary-material3.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0175185.s004" id="pone.0175185.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0175185.s004">S3 Table. </a>The performance of the SP-based method with different combinations of parameters.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0175185.s004">https://doi.org/10.1371/journal.pone.0175185.s004</a></p><a id="article1.body1.sec6.supplementary-material4.caption1.p1" name="article1.body1.sec6.supplementary-material4.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0175185.s005" id="pone.0175185.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0175185.s005">S4 Table. </a>Genes identified by the optimal RWR-based method.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0175185.s005">https://doi.org/10.1371/journal.pone.0175185.s005</a></p><a id="article1.body1.sec6.supplementary-material5.caption1.p1" name="article1.body1.sec6.supplementary-material5.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0175185.s006" id="pone.0175185.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0175185.s006">S5 Table. </a>The enrichment analysis of putative genes yielded by DAVID.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0175185.s006">https://doi.org/10.1371/journal.pone.0175185.s006</a></p><a id="article1.body1.sec6.supplementary-material6.caption1.p1" name="article1.body1.sec6.supplementary-material6.caption1.p1" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>This study was supported by the National Natural Science Foundation of China (31371335), Natural Science Foundation of Shanghai (17ZR1412500), Shanghai Sailing Program and The Youth Innovation Promotion Association of Chinese Academy of Sciences (CAS) (2016245).</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>

<ol class="simple">

<li><strong>Conceptualization:</strong> TH HL YDC.</li>

<li><strong>Data curation:</strong> ZX FY YS.</li>

<li><strong>Formal analysis:</strong> LC JY YZ XK.</li>

<li><strong>Funding acquisition:</strong> TH YDC.</li>

<li><strong>Methodology:</strong> LC YDC.</li>

<li><strong>Supervision:</strong> LC YDC.</li>

<li><strong>Validation:</strong> JY ZX FY TH.</li>

<li><strong>Writing – original draft:</strong> LC JY.</li>

<li><strong>Writing – review &amp; editing:</strong> TH YDC.</li>

</ol>
</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0175185.ref001" id="pone.0175185.ref001" class="link-target"></a>Tsantoulis PK, Kastrinakis NG, Tourvas AD, Laskaris G, Gorgoulis VG. Advances in the biology of oral cancer. Oral Oncol. 2007;43(6):523–34.  pmid:17258495 <ul class="reflinks"><li><a href="#" data-author="Tsantoulis" data-cit="TsantoulisPK%2C%20KastrinakisNG%2C%20TourvasAD%2C%20LaskarisG%2C%20GorgoulisVG.%20Advances%20in%20the%20biology%20of%20oral%20cancer.%20Oral%20Oncol.%202007%3B43%286%29%3A523%E2%80%9334.%20doi%3A%2010.1016%2Fj.oraloncology.2006.11.010%2017258495" data-title="Advances%20in%20the%20biology%20of%20oral%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17258495" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Advances+in+the+biology+of+oral+cancer+Tsantoulis+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0175185.ref002" id="pone.0175185.ref002" class="link-target"></a>Shridhar K, Walia GK, Aggarwal A, Gulati S, Geetha AV, Prabhakaran D, et al. DNA methylation markers for oral pre-cancer progression: A critical review. Oral Oncol. 2016;53:1–9.  pmid:26690652 <ul class="reflinks"><li><a href="#" data-author="Shridhar" data-cit="ShridharK%2C%20WaliaGK%2C%20AggarwalA%2C%20GulatiS%2C%20GeethaAV%2C%20PrabhakaranD%2C%20et%20al.%20DNA%20methylation%20markers%20for%20oral%20pre-cancer%20progression%3A%20A%20critical%20review.%20Oral%20Oncol.%202016%3B53%3A1%E2%80%939.%20doi%3A%2010.1016%2Fj.oraloncology.2015.11.012%2026690652" data-title="DNA%20methylation%20markers%20for%20oral%20pre-cancer%20progression%3A%20A%20critical%20review" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26690652" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+methylation+markers+for+oral+pre-cancer+progression%3A+A+critical+review+Shridhar+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0175185.ref003" id="pone.0175185.ref003" class="link-target"></a>Bray F, Ren JS, Masuyer E, Ferlay J. Global estimates of cancer prevalence for 27 sites in the adult population in 2008. International journal of cancer Journal international du cancer. 2013;132(5):1133–45.  pmid:22752881 <ul class="reflinks"><li><a href="#" data-author="Bray" data-cit="BrayF%2C%20RenJS%2C%20MasuyerE%2C%20FerlayJ.%20Global%20estimates%20of%20cancer%20prevalence%20for%2027%20sites%20in%20the%20adult%20population%20in%202008.%20International%20journal%20of%20cancer%20Journal%20international%20du%20cancer.%202013%3B132%285%29%3A1133%E2%80%9345.%20doi%3A%2010.1002%2Fijc.27711%2022752881" data-title="Global%20estimates%20of%20cancer%20prevalence%20for%2027%20sites%20in%20the%20adult%20population%20in%202008" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22752881" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Global+estimates+of+cancer+prevalence+for+27+sites+in+the+adult+population+in+2008+Bray+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0175185.ref004" id="pone.0175185.ref004" class="link-target"></a>Noda M. International Agency for Research on Cancer (<a href="http://www.iarc.fr/">http://www.iarc.fr/</a>). Japanese journal of clinical oncology. 1999;29(11):592. pmid:10678568 <ul class="reflinks"><li><a href="#" data-author="Noda" data-cit="NodaM.%20International%20Agency%20for%20Research%20on%20Cancer%20%28http%3A%2F%2Fwww.iarc.fr%2F%29.%20Japanese%20journal%20of%20clinical%20oncology.%201999%3B29%2811%29%3A592.%2010678568" data-title="International%20Agency%20for%20Research%20on%20Cancer%20%28http%3A%2F%2Fwww.iarc.fr%2F%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10678568" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=International+Agency+for+Research+on+Cancer+%28http%3A%2F%2Fwww.iarc.fr%2F%29+Noda+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0175185.ref005" id="pone.0175185.ref005" class="link-target"></a>Cancer incidence in five continents. Volume IX. IARC Sci Publ. 2008;(160):1–837. pmid:19388204 <ul class="reflinks"><li><a href="#" data-author="" data-cit="Cancer%20incidence%20in%20five%20continents.%20Volume%20IX.%20IARC%20Sci%20Publ.%202008%3B%28160%29%3A1%E2%80%93837.%2019388204" data-title="Cancer%20incidence%20in%20five%20continents.%20Volume%20IX" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19388204" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cancer+incidence+in+five+continents.+Volume+IX++2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0175185.ref006" id="pone.0175185.ref006" class="link-target"></a>Warnakulasuriya S, Sutherland G, Scully C. Tobacco, oral cancer, and treatment of dependence. Oral Oncol. 2005;41(3):244–60.  pmid:15743687 <ul class="reflinks"><li><a href="#" data-author="Warnakulasuriya" data-cit="WarnakulasuriyaS%2C%20SutherlandG%2C%20ScullyC.%20Tobacco%2C%20oral%20cancer%2C%20and%20treatment%20of%20dependence.%20Oral%20Oncol.%202005%3B41%283%29%3A244%E2%80%9360.%20doi%3A%2010.1016%2Fj.oraloncology.2004.08.010%2015743687" data-title="Tobacco%2C%20oral%20cancer%2C%20and%20treatment%20of%20dependence" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15743687" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tobacco%2C+oral+cancer%2C+and+treatment+of+dependence+Warnakulasuriya+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0175185.ref007" id="pone.0175185.ref007" class="link-target"></a>Lucenteforte E, Garavello W, Bosetti C, La Vecchia C. Dietary factors and oral and pharyngeal cancer risk. Oral Oncol. 2009;45(6):461–7.  pmid:18990606 <ul class="reflinks"><li><a href="#" data-author="Lucenteforte" data-cit="LucenteforteE%2C%20GaravelloW%2C%20BosettiC%2C%20La%20VecchiaC.%20Dietary%20factors%20and%20oral%20and%20pharyngeal%20cancer%20risk.%20Oral%20Oncol.%202009%3B45%286%29%3A461%E2%80%937.%20doi%3A%2010.1016%2Fj.oraloncology.2008.09.002%2018990606" data-title="Dietary%20factors%20and%20oral%20and%20pharyngeal%20cancer%20risk" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18990606" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Dietary+factors+and+oral+and+pharyngeal+cancer+risk+Lucenteforte+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0175185.ref008" id="pone.0175185.ref008" class="link-target"></a>Pavia M, Pileggi C, Nobile CG, Angelillo IF. Association between fruit and vegetable consumption and oral cancer: a meta-analysis of observational studies. The American journal of clinical nutrition. 2006;83(5):1126–34. pmid:16685056 <ul class="reflinks"><li><a href="#" data-author="Pavia" data-cit="PaviaM%2C%20PileggiC%2C%20NobileCG%2C%20AngelilloIF.%20Association%20between%20fruit%20and%20vegetable%20consumption%20and%20oral%20cancer%3A%20a%20meta-analysis%20of%20observational%20studies.%20The%20American%20journal%20of%20clinical%20nutrition.%202006%3B83%285%29%3A1126%E2%80%9334.%2016685056" data-title="Association%20between%20fruit%20and%20vegetable%20consumption%20and%20oral%20cancer%3A%20a%20meta-analysis%20of%20observational%20studies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16685056" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Association+between+fruit+and+vegetable+consumption+and+oral+cancer%3A+a+meta-analysis+of+observational+studies+Pavia+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0175185.ref009" id="pone.0175185.ref009" class="link-target"></a>Fearon ER, Vogelstein B. A genetic model for colorectal tumorigenesis. Cell. 1990;61(5):759–67. pmid:2188735 <ul class="reflinks"><li><a href="#" data-author="Fearon" data-cit="FearonER%2C%20VogelsteinB.%20A%20genetic%20model%20for%20colorectal%20tumorigenesis.%20Cell.%201990%3B61%285%29%3A759%E2%80%9367.%202188735" data-title="A%20genetic%20model%20for%20colorectal%20tumorigenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2188735" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+genetic+model+for+colorectal+tumorigenesis+Fearon+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0175185.ref010" id="pone.0175185.ref010" class="link-target"></a>Nagatsuka H, Ishiwari Y, Tsujigiwa H, Nakano K, Nagai N. Quantitation of epidermal growth factor receptor gene amplification by competitive polymerase chain reaction in pre-malignant and malignant oral epithelial lesions. Oral Oncol. 2001;37(7):599–604. pmid:11564582 <ul class="reflinks"><li><a href="#" data-author="Nagatsuka" data-cit="NagatsukaH%2C%20IshiwariY%2C%20TsujigiwaH%2C%20NakanoK%2C%20NagaiN.%20Quantitation%20of%20epidermal%20growth%20factor%20receptor%20gene%20amplification%20by%20competitive%20polymerase%20chain%20reaction%20in%20pre-malignant%20and%20malignant%20oral%20epithelial%20lesions.%20Oral%20Oncol.%202001%3B37%287%29%3A599%E2%80%93604.%2011564582" data-title="Quantitation%20of%20epidermal%20growth%20factor%20receptor%20gene%20amplification%20by%20competitive%20polymerase%20chain%20reaction%20in%20pre-malignant%20and%20malignant%20oral%20epithelial%20lesions" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11564582" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Quantitation+of+epidermal+growth+factor+receptor+gene+amplification+by+competitive+polymerase+chain+reaction+in+pre-malignant+and+malignant+oral+epithelial+lesions+Nagatsuka+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0175185.ref011" id="pone.0175185.ref011" class="link-target"></a>Ishitoya J, Toriyama M, Oguchi N, Kitamura K, Ohshima M, Asano K, et al. Gene amplification and overexpression of EGF receptor in squamous cell carcinomas of the head and neck. British journal of cancer. 1989;59(4):559–62. pmid:2713242 <ul class="reflinks"><li><a href="#" data-author="Ishitoya" data-cit="IshitoyaJ%2C%20ToriyamaM%2C%20OguchiN%2C%20KitamuraK%2C%20OhshimaM%2C%20AsanoK%2C%20et%20al.%20Gene%20amplification%20and%20overexpression%20of%20EGF%20receptor%20in%20squamous%20cell%20carcinomas%20of%20the%20head%20and%20neck.%20British%20journal%20of%20cancer.%201989%3B59%284%29%3A559%E2%80%9362.%202713242" data-title="Gene%20amplification%20and%20overexpression%20of%20EGF%20receptor%20in%20squamous%20cell%20carcinomas%20of%20the%20head%20and%20neck" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2713242" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Gene+amplification+and+overexpression+of+EGF+receptor+in+squamous+cell+carcinomas+of+the+head+and+neck+Ishitoya+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0175185.ref012" id="pone.0175185.ref012" class="link-target"></a>Shiraki M, Odajima T, Ikeda T, Sasaki A, Satoh M, Yamaguchi A, et al. Combined expression of p53, cyclin D1 and epidermal growth factor receptor improves estimation of prognosis in curatively resected oral cancer. Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc. 2005;18(11):1482–9. <ul class="reflinks"><li><a href="#" data-author="Shiraki" data-cit="ShirakiM%2C%20OdajimaT%2C%20IkedaT%2C%20SasakiA%2C%20SatohM%2C%20YamaguchiA%2C%20et%20al.%20Combined%20expression%20of%20p53%2C%20cyclin%20D1%20and%20epidermal%20growth%20factor%20receptor%20improves%20estimation%20of%20prognosis%20in%20curatively%20resected%20oral%20cancer.%20Modern%20pathology%3A%20an%20official%20journal%20of%20the%20United%20States%20and%20Canadian%20Academy%20of%20Pathology%2C%20Inc.%202005%3B18%2811%29%3A1482%E2%80%939." data-title="Combined%20expression%20of%20p53%2C%20cyclin%20D1%20and%20epidermal%20growth%20factor%20receptor%20improves%20estimation%20of%20prognosis%20in%20curatively%20resected%20oral%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Combined+expression+of+p53%2C+cyclin+D1+and+epidermal+growth+factor+receptor+improves+estimation+of+prognosis+in+curatively+resected+oral+cancer+Shiraki+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0175185.ref013" id="pone.0175185.ref013" class="link-target"></a>Xia W, Lau YK, Zhang HZ, Liu AR, Li L, Kiyokawa N, et al. Strong correlation between c-erbB-2 overexpression and overall survival of patients with oral squamous cell carcinoma. Clin Cancer Res. 1997;3(1):3–9. pmid:9815530 <ul class="reflinks"><li><a href="#" data-author="Xia" data-cit="XiaW%2C%20LauYK%2C%20ZhangHZ%2C%20LiuAR%2C%20LiL%2C%20KiyokawaN%2C%20et%20al.%20Strong%20correlation%20between%20c-erbB-2%20overexpression%20and%20overall%20survival%20of%20patients%20with%20oral%20squamous%20cell%20carcinoma.%20Clin%20Cancer%20Res.%201997%3B3%281%29%3A3%E2%80%939.%209815530" data-title="Strong%20correlation%20between%20c-erbB-2%20overexpression%20and%20overall%20survival%20of%20patients%20with%20oral%20squamous%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9815530" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Strong+correlation+between+c-erbB-2+overexpression+and+overall+survival+of+patients+with+oral+squamous+cell+carcinoma+Xia+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0175185.ref014" id="pone.0175185.ref014" class="link-target"></a>Werkmeister R, Brandt B, Joos U. Clinical relevance of erbB-1 and -2 oncogenes in oral carcinomas. Oral Oncol. 2000;36(1):100–5. pmid:10889928 <ul class="reflinks"><li><a href="#" data-author="Werkmeister" data-cit="WerkmeisterR%2C%20BrandtB%2C%20JoosU.%20Clinical%20relevance%20of%20erbB-1%20and%20-2%20oncogenes%20in%20oral%20carcinomas.%20Oral%20Oncol.%202000%3B36%281%29%3A100%E2%80%935.%2010889928" data-title="Clinical%20relevance%20of%20erbB-1%20and%20-2%20oncogenes%20in%20oral%20carcinomas" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10889928" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Clinical+relevance+of+erbB-1+and+-2+oncogenes+in+oral+carcinomas+Werkmeister+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0175185.ref015" id="pone.0175185.ref015" class="link-target"></a>Miyamoto R, Uzawa N, Nagaoka S, Hirata Y, Amagasa T. Prognostic significance of cyclin D1 amplification and overexpression in oral squamous cell carcinomas. Oral Oncol. 2003;39(6):610–8. pmid:12798405 <ul class="reflinks"><li><a href="#" data-author="Miyamoto" data-cit="MiyamotoR%2C%20UzawaN%2C%20NagaokaS%2C%20HirataY%2C%20AmagasaT.%20Prognostic%20significance%20of%20cyclin%20D1%20amplification%20and%20overexpression%20in%20oral%20squamous%20cell%20carcinomas.%20Oral%20Oncol.%202003%3B39%286%29%3A610%E2%80%938.%2012798405" data-title="Prognostic%20significance%20of%20cyclin%20D1%20amplification%20and%20overexpression%20in%20oral%20squamous%20cell%20carcinomas" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12798405" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Prognostic+significance+of+cyclin+D1+amplification+and+overexpression+in+oral+squamous+cell+carcinomas+Miyamoto+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0175185.ref016" id="pone.0175185.ref016" class="link-target"></a>Koontongkaew S, Chareonkitkajorn L, Chanvitan A, Leelakriangsak M, Amornphimoltham P. Alterations of p53, pRb, cyclin D(1) and cdk4 in human oral and pharyngeal squamous cell carcinomas. Oral Oncol. 2000;36(4):334–9. pmid:10899671 <ul class="reflinks"><li><a href="#" data-author="Koontongkaew" data-cit="KoontongkaewS%2C%20ChareonkitkajornL%2C%20ChanvitanA%2C%20LeelakriangsakM%2C%20AmornphimolthamP.%20Alterations%20of%20p53%2C%20pRb%2C%20cyclin%20D%281%29%20and%20cdk4%20in%20human%20oral%20and%20pharyngeal%20squamous%20cell%20carcinomas.%20Oral%20Oncol.%202000%3B36%284%29%3A334%E2%80%939.%2010899671" data-title="Alterations%20of%20p53%2C%20pRb%2C%20cyclin%20D%281%29%20and%20cdk4%20in%20human%20oral%20and%20pharyngeal%20squamous%20cell%20carcinomas" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10899671" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Alterations+of+p53%2C+pRb%2C+cyclin+D%281%29+and+cdk4+in+human+oral+and+pharyngeal+squamous+cell+carcinomas+Koontongkaew+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0175185.ref017" id="pone.0175185.ref017" class="link-target"></a>Huang MF, Chang YC, Liao PS, Huang TH, Tsay CH, Chou MY. Loss of heterozygosity of p53 gene of oral cancer detected by exfoliative cytology. Oral Oncol. 1999;35(3):296–301. pmid:10621851 <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangMF%2C%20ChangYC%2C%20LiaoPS%2C%20HuangTH%2C%20TsayCH%2C%20ChouMY.%20Loss%20of%20heterozygosity%20of%20p53%20gene%20of%20oral%20cancer%20detected%20by%20exfoliative%20cytology.%20Oral%20Oncol.%201999%3B35%283%29%3A296%E2%80%93301.%2010621851" data-title="Loss%20of%20heterozygosity%20of%20p53%20gene%20of%20oral%20cancer%20detected%20by%20exfoliative%20cytology" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10621851" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Loss+of+heterozygosity+of+p53+gene+of+oral+cancer+detected+by+exfoliative+cytology+Huang+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0175185.ref018" id="pone.0175185.ref018" class="link-target"></a>Largey JS, Meltzer SJ, Yin J, Norris K, Sauk JJ, Archibald DW. Loss of heterozygosity of p53 in oral cancers demonstrated by the polymerase chain reaction. Cancer. 1993;71(6):1933–7. pmid:8443743 <ul class="reflinks"><li><a href="#" data-author="Largey" data-cit="LargeyJS%2C%20MeltzerSJ%2C%20YinJ%2C%20NorrisK%2C%20SaukJJ%2C%20ArchibaldDW.%20Loss%20of%20heterozygosity%20of%20p53%20in%20oral%20cancers%20demonstrated%20by%20the%20polymerase%20chain%20reaction.%20Cancer.%201993%3B71%286%29%3A1933%E2%80%937.%208443743" data-title="Loss%20of%20heterozygosity%20of%20p53%20in%20oral%20cancers%20demonstrated%20by%20the%20polymerase%20chain%20reaction" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8443743" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Loss+of+heterozygosity+of+p53+in+oral+cancers+demonstrated+by+the+polymerase+chain+reaction+Largey+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0175185.ref019" id="pone.0175185.ref019" class="link-target"></a>Sakata K. Alterations of tumor suppressor genes and the H-ras oncogene in oral squamous cell carcinoma. Journal of oral pathology &amp; medicine: official publication of the International Association of Oral Pathologists and the American Academy of Oral Pathology. 1996;25(6):302–7. <ul class="reflinks"><li><a href="#" data-author="Sakata" data-cit="SakataK.%20Alterations%20of%20tumor%20suppressor%20genes%20and%20the%20H-ras%20oncogene%20in%20oral%20squamous%20cell%20carcinoma.%20Journal%20of%20oral%20pathology%20%26%20medicine%3A%20official%20publication%20of%20the%20International%20Association%20of%20Oral%20Pathologists%20and%20the%20American%20Academy%20of%20Oral%20Pathology.%201996%3B25%286%29%3A302%E2%80%937." data-title="Alterations%20of%20tumor%20suppressor%20genes%20and%20the%20H-ras%20oncogene%20in%20oral%20squamous%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Alterations+of+tumor+suppressor+genes+and+the+H-ras+oncogene+in+oral+squamous+cell+carcinoma+Sakata+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0175185.ref020" id="pone.0175185.ref020" class="link-target"></a>Shang ZJ, Li ZB, Li JR. VEGF is up-regulated by hypoxic stimulation and related to tumour angiogenesis and severity of disease in oral squamous cell carcinoma: in vitro and in vivo studies. Int J Oral Maxillofac Surg. 2006;35(6):533–8.  pmid:16388929 <ul class="reflinks"><li><a href="#" data-author="Shang" data-cit="ShangZJ%2C%20LiZB%2C%20LiJR.%20VEGF%20is%20up-regulated%20by%20hypoxic%20stimulation%20and%20related%20to%20tumour%20angiogenesis%20and%20severity%20of%20disease%20in%20oral%20squamous%20cell%20carcinoma%3A%20in%20vitro%20and%20in%20vivo%20studies.%20Int%20J%20Oral%20Maxillofac%20Surg.%202006%3B35%286%29%3A533%E2%80%938.%20doi%3A%2010.1016%2Fj.ijom.2005.09.006%2016388929" data-title="VEGF%20is%20up-regulated%20by%20hypoxic%20stimulation%20and%20related%20to%20tumour%20angiogenesis%20and%20severity%20of%20disease%20in%20oral%20squamous%20cell%20carcinoma%3A%20in%20vitro%20and%20in%20vivo%20studies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16388929" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=VEGF+is+up-regulated+by+hypoxic+stimulation+and+related+to+tumour+angiogenesis+and+severity+of+disease+in+oral+squamous+cell+carcinoma%3A+in+vitro+and+in+vivo+studies+Shang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0175185.ref021" id="pone.0175185.ref021" class="link-target"></a>Patel BP, Shah PM, Rawal UM, Desai AA, Shah SV, Rawal RM, et al. Activation of MMP-2 and MMP-9 in patients with oral squamous cell carcinoma. J Surg Oncol. 2005;90(2):81–8.  pmid:15844188 <ul class="reflinks"><li><a href="#" data-author="Patel" data-cit="PatelBP%2C%20ShahPM%2C%20RawalUM%2C%20DesaiAA%2C%20ShahSV%2C%20RawalRM%2C%20et%20al.%20Activation%20of%20MMP-2%20and%20MMP-9%20in%20patients%20with%20oral%20squamous%20cell%20carcinoma.%20J%20Surg%20Oncol.%202005%3B90%282%29%3A81%E2%80%938.%20doi%3A%2010.1002%2Fjso.20240%2015844188" data-title="Activation%20of%20MMP-2%20and%20MMP-9%20in%20patients%20with%20oral%20squamous%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15844188" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Activation+of+MMP-2+and+MMP-9+in+patients+with+oral+squamous+cell+carcinoma+Patel+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0175185.ref022" id="pone.0175185.ref022" class="link-target"></a>Zeng X, Liao Y, Liu Y, Zou Q. Prediction and validation of disease genes using HeteSim Scores. IEEE/ACM Trans Comput Biol Bioinform. 2016. <ul class="reflinks"><li><a href="#" data-author="Zeng" data-cit="ZengX%2C%20LiaoY%2C%20LiuY%2C%20ZouQ.%20Prediction%20and%20validation%20of%20disease%20genes%20using%20HeteSim%20Scores.%20IEEE%2FACM%20Trans%20Comput%20Biol%20Bioinform.%202016." data-title="Prediction%20and%20validation%20of%20disease%20genes%20using%20HeteSim%20Scores" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+and+validation+of+disease+genes+using+HeteSim+Scores+Zeng+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0175185.ref023" id="pone.0175185.ref023" class="link-target"></a>Chen X, Perez-Jimenez MJ, Valencia-Cabrera L, Wang BZ, Zeng XX. Computing with viruses. Theoretical Computer Science. 2016;623:146–59. <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenX%2C%20Perez-JimenezMJ%2C%20Valencia-CabreraL%2C%20WangBZ%2C%20ZengXX.%20Computing%20with%20viruses.%20Theoretical%20Computer%20Science.%202016%3B623%3A146%E2%80%9359." data-title="Computing%20with%20viruses" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Computing+with+viruses+Chen+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0175185.ref024" id="pone.0175185.ref024" class="link-target"></a>Zou Q, Wan S, Ju Y, Tang J, Zeng X. Pretata: predicting TATA binding proteins with novel features and dimensionality reduction strategy. BMC Syst Biol. 2016;10(Suppl 4):114.  pmid:28155714 <ul class="reflinks"><li><a href="#" data-author="Zou" data-cit="ZouQ%2C%20WanS%2C%20JuY%2C%20TangJ%2C%20ZengX.%20Pretata%3A%20predicting%20TATA%20binding%20proteins%20with%20novel%20features%20and%20dimensionality%20reduction%20strategy.%20BMC%20Syst%20Biol.%202016%3B10%28Suppl%204%29%3A114.%20doi%3A%2010.1186%2Fs12918-016-0353-5%2028155714" data-title="Pretata%3A%20predicting%20TATA%20binding%20proteins%20with%20novel%20features%20and%20dimensionality%20reduction%20strategy" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/28155714" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Pretata%3A+predicting+TATA+binding+proteins+with+novel+features+and+dimensionality+reduction+strategy+Zou+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0175185.ref025" id="pone.0175185.ref025" class="link-target"></a>Gu B, Sheng VS, Tay KY, Romano W, Li S. Incremental Support Vector Learning for Ordinal Regression. Ieee T Neur Net Lear. 2015;26(7):1403–16. <ul class="reflinks"><li><a href="#" data-author="Gu" data-cit="GuB%2C%20ShengVS%2C%20TayKY%2C%20RomanoW%2C%20LiS.%20Incremental%20Support%20Vector%20Learning%20for%20Ordinal%20Regression.%20Ieee%20T%20Neur%20Net%20Lear.%202015%3B26%287%29%3A1403%E2%80%9316." data-title="Incremental%20Support%20Vector%20Learning%20for%20Ordinal%20Regression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Incremental+Support+Vector+Learning+for+Ordinal+Regression+Gu+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0175185.ref026" id="pone.0175185.ref026" class="link-target"></a>Gu B, Sheng VS, Li S. Bi-parameter space partition for cost-sensitive SVM. Proceedings of the 24th International Conference on Artificial Intelligence; Buenos Aires, Argentina. 2832741: AAAI Press; 2015. p. 3532–9. <ul class="find-nolinks"></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0175185.ref027" id="pone.0175185.ref027" class="link-target"></a>Tang W, Liao Z, Zou Q. Which statistical significance test best detects oncomiRNAs in cancer tissues? An exploratory analysis. Oncotarget. 2016;7(51):85613–23.  pmid:27784000 <ul class="reflinks"><li><a href="#" data-author="Tang" data-cit="TangW%2C%20LiaoZ%2C%20ZouQ.%20Which%20statistical%20significance%20test%20best%20detects%20oncomiRNAs%20in%20cancer%20tissues%3F%20An%20exploratory%20analysis.%20Oncotarget.%202016%3B7%2851%29%3A85613%E2%80%9323.%20doi%3A%2010.18632%2Foncotarget.12828%2027784000" data-title="Which%20statistical%20significance%20test%20best%20detects%20oncomiRNAs%20in%20cancer%20tissues%3F%20An%20exploratory%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27784000" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Which+statistical+significance+test+best+detects+oncomiRNAs+in+cancer+tissues%3F+An+exploratory+analysis+Tang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0175185.ref028" id="pone.0175185.ref028" class="link-target"></a>Zhang JR, Ju Y, Lu HJ, Xuan P, Zou Q. Accurate Identification of Cancerlectins through Hybrid Machine Learning Technology. Int J Genomics. 2016. <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangJR%2C%20JuY%2C%20LuHJ%2C%20XuanP%2C%20ZouQ.%20Accurate%20Identification%20of%20Cancerlectins%20through%20Hybrid%20Machine%20Learning%20Technology.%20Int%20J%20Genomics.%202016." data-title="Accurate%20Identification%20of%20Cancerlectins%20through%20Hybrid%20Machine%20Learning%20Technology" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Accurate+Identification+of+Cancerlectins+through+Hybrid+Machine+Learning+Technology+Zhang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0175185.ref029" id="pone.0175185.ref029" class="link-target"></a>Oliver S. Guilt-by-association goes global. Nature. 2000;403(6770):601–3.  pmid:10688178 <ul class="reflinks"><li><a href="#" data-author="Oliver" data-cit="OliverS.%20Guilt-by-association%20goes%20global.%20Nature.%202000%3B403%286770%29%3A601%E2%80%933.%20doi%3A%2010.1038%2F35001165%2010688178" data-title="Guilt-by-association%20goes%20global" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10688178" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Guilt-by-association+goes+global+Oliver+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0175185.ref030" id="pone.0175185.ref030" class="link-target"></a>Kohler S, Bauer S, Horn D, Robinson PN. Walking the interactome for prioritization of candidate disease genes. The Amerian Journal of Human Genetics. 2008;82(4):949–58. <ul class="reflinks"><li><a href="#" data-author="Kohler" data-cit="KohlerS%2C%20BauerS%2C%20HornD%2C%20RobinsonPN.%20Walking%20the%20interactome%20for%20prioritization%20of%20candidate%20disease%20genes.%20The%20Amerian%20Journal%20of%20Human%20Genetics.%202008%3B82%284%29%3A949%E2%80%9358." data-title="Walking%20the%20interactome%20for%20prioritization%20of%20candidate%20disease%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Walking+the+interactome+for+prioritization+of+candidate+disease+genes+Kohler+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0175185.ref031" id="pone.0175185.ref031" class="link-target"></a>Navlakha S, Kingsford C. The power of protein interaction networks for associating genes with diseases. Bioinformatics. 2010;26(8):1057–63.  pmid:20185403 <ul class="reflinks"><li><a href="#" data-author="Navlakha" data-cit="NavlakhaS%2C%20KingsfordC.%20The%20power%20of%20protein%20interaction%20networks%20for%20associating%20genes%20with%20diseases.%20Bioinformatics.%202010%3B26%288%29%3A1057%E2%80%9363.%20doi%3A%2010.1093%2Fbioinformatics%2Fbtq076%2020185403" data-title="The%20power%20of%20protein%20interaction%20networks%20for%20associating%20genes%20with%20diseases" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20185403" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+power+of+protein+interaction+networks+for+associating+genes+with+diseases+Navlakha+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0175185.ref032" id="pone.0175185.ref032" class="link-target"></a>Care MA, Bradford JR, Needham CJ, Bulpitt AJ, Westhead DR. Combining the interactome and deleterious SNP predictions to improve disease gene identification. Hum Mutat. 2009;30(3):485–92.  pmid:19156842 <ul class="reflinks"><li><a href="#" data-author="Care" data-cit="CareMA%2C%20BradfordJR%2C%20NeedhamCJ%2C%20BulpittAJ%2C%20WestheadDR.%20Combining%20the%20interactome%20and%20deleterious%20SNP%20predictions%20to%20improve%20disease%20gene%20identification.%20Hum%20Mutat.%202009%3B30%283%29%3A485%E2%80%9392.%20doi%3A%2010.1002%2Fhumu.20917%2019156842" data-title="Combining%20the%20interactome%20and%20deleterious%20SNP%20predictions%20to%20improve%20disease%20gene%20identification" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19156842" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Combining+the+interactome+and+deleterious+SNP+predictions+to+improve+disease+gene+identification+Care+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0175185.ref033" id="pone.0175185.ref033" class="link-target"></a>Chen L, Xing Z, Huang T, Shu Y, Huang G, Li H-P. Application of the shortest path algorithm for the discovery of breast cancer related genes. Current Bioinformatics. 2016;11(1):51–8. <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenL%2C%20XingZ%2C%20HuangT%2C%20ShuY%2C%20HuangG%2C%20LiH-P.%20Application%20of%20the%20shortest%20path%20algorithm%20for%20the%20discovery%20of%20breast%20cancer%20related%20genes.%20Current%20Bioinformatics.%202016%3B11%281%29%3A51%E2%80%938." data-title="Application%20of%20the%20shortest%20path%20algorithm%20for%20the%20discovery%20of%20breast%20cancer%20related%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Application+of+the+shortest+path+algorithm+for+the+discovery+of+breast+cancer+related+genes+Chen+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0175185.ref034" id="pone.0175185.ref034" class="link-target"></a>Gao Y-F, Shu Y, Yang L, He Y-C, Li L-P, Huang G, et al. A Graphic Method for Identification of Novel Glioma Related Genes. BioMed Research International. 2014;2014:891945.  pmid:25050377 <ul class="reflinks"><li><a href="#" data-author="Gao" data-cit="GaoY-F%2C%20ShuY%2C%20YangL%2C%20HeY-C%2C%20LiL-P%2C%20HuangG%2C%20et%20al.%20A%20Graphic%20Method%20for%20Identification%20of%20Novel%20Glioma%20Related%20Genes.%20BioMed%20Research%20International.%202014%3B2014%3A891945.%20doi%3A%2010.1155%2F2014%2F891945%2025050377" data-title="A%20Graphic%20Method%20for%20Identification%20of%20Novel%20Glioma%20Related%20Genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25050377" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+Graphic+Method+for+Identification+of+Novel+Glioma+Related+Genes+Gao+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0175185.ref035" id="pone.0175185.ref035" class="link-target"></a>Zhang J, Yang J, Huang T, Shu Y, Chen L. Identification of novel proliferative diabetic retinopathy related genes on protein-protein interaction network. Neurocomputing. 2016;217:63–72. <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangJ%2C%20YangJ%2C%20HuangT%2C%20ShuY%2C%20ChenL.%20Identification%20of%20novel%20proliferative%20diabetic%20retinopathy%20related%20genes%20on%20protein-protein%20interaction%20network.%20Neurocomputing.%202016%3B217%3A63%E2%80%9372." data-title="Identification%20of%20novel%20proliferative%20diabetic%20retinopathy%20related%20genes%20on%20protein-protein%20interaction%20network" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+novel+proliferative+diabetic+retinopathy+related+genes+on+protein-protein+interaction+network+Zhang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0175185.ref036" id="pone.0175185.ref036" class="link-target"></a>Chen L, Wang B, Wang S, Yang J, Hu J, Xie Z, et al. OPMSP: A computational method integrating protein interaction and sequence information for the identification of novel putative oncogenes. Protein Pept Lett. 2016;23(12):1081–94. pmid:27774893 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenL%2C%20WangB%2C%20WangS%2C%20YangJ%2C%20HuJ%2C%20XieZ%2C%20et%20al.%20OPMSP%3A%20A%20computational%20method%20integrating%20protein%20interaction%20and%20sequence%20information%20for%20the%20identification%20of%20novel%20putative%20oncogenes.%20Protein%20Pept%20Lett.%202016%3B23%2812%29%3A1081%E2%80%9394.%2027774893" data-title="OPMSP%3A%20A%20computational%20method%20integrating%20protein%20interaction%20and%20sequence%20information%20for%20the%20identification%20of%20novel%20putative%20oncogenes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27774893" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=OPMSP%3A+A+computational+method+integrating+protein+interaction+and+sequence+information+for+the+identification+of+novel+putative+oncogenes+Chen+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0175185.ref037" id="pone.0175185.ref037" class="link-target"></a>Chen L, Huang T, Zhang YH, Jiang Y, Zheng M, Cai YD. Identification of novel candidate drivers connecting different dysfunctional levels for lung adenocarcinoma using protein-protein interactions and a shortest path approach. Sci Rep. 2016;6:29849. Epub 2016/07/15.  pmid:27412431 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenL%2C%20HuangT%2C%20ZhangYH%2C%20JiangY%2C%20ZhengM%2C%20CaiYD.%20Identification%20of%20novel%20candidate%20drivers%20connecting%20different%20dysfunctional%20levels%20for%20lung%20adenocarcinoma%20using%20protein-protein%20interactions%20and%20a%20shortest%20path%20approach.%20Sci%20Rep.%202016%3B6%3A29849.%20Epub%202016%2F07%2F15.%20doi%3A%2010.1038%2Fsrep29849%2027412431" data-title="Identification%20of%20novel%20candidate%20drivers%20connecting%20different%20dysfunctional%20levels%20for%20lung%20adenocarcinoma%20using%20protein-protein%20interactions%20and%20a%20shortest%20path%20approach" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27412431" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+novel+candidate+drivers+connecting+different+dysfunctional+levels+for+lung+adenocarcinoma+using+protein-protein+interactions+and+a+shortest+path+approach+Chen+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0175185.ref038" id="pone.0175185.ref038" class="link-target"></a>Chen L, Yang J, Huang T, Kong X, Lu L, Cai Y-D. Mining for novel tumor suppressor genes using a shortest path approach. Journal of Biomolecular Structure and Dynamics. 2016;34(3):664–75.  pmid:26209080 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenL%2C%20YangJ%2C%20HuangT%2C%20KongX%2C%20LuL%2C%20CaiY-D.%20Mining%20for%20novel%20tumor%20suppressor%20genes%20using%20a%20shortest%20path%20approach.%20Journal%20of%20Biomolecular%20Structure%20and%20Dynamics.%202016%3B34%283%29%3A664%E2%80%9375.%20doi%3A%2010.1080%2F07391102.2015.1042915%2026209080" data-title="Mining%20for%20novel%20tumor%20suppressor%20genes%20using%20a%20shortest%20path%20approach" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26209080" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mining+for+novel+tumor+suppressor+genes+using+a+shortest+path+approach+Chen+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0175185.ref039" id="pone.0175185.ref039" class="link-target"></a>Cai Y-D, Zhang Q, Zhang Y-H, Chen L, Huang T. Identification of genes associated with breast cancer metastasis to bone on a protein-protein interaction network with a shortest path algorithm. J Proteome Res. 2017;16(2):1027–38.  pmid:28076954 <ul class="reflinks"><li><a href="#" data-author="Cai" data-cit="CaiY-D%2C%20ZhangQ%2C%20ZhangY-H%2C%20ChenL%2C%20HuangT.%20Identification%20of%20genes%20associated%20with%20breast%20cancer%20metastasis%20to%20bone%20on%20a%20protein-protein%20interaction%20network%20with%20a%20shortest%20path%20algorithm.%20J%20Proteome%20Res.%202017%3B16%282%29%3A1027%E2%80%9338.%20doi%3A%2010.1021%2Facs.jproteome.6b00950%2028076954" data-title="Identification%20of%20genes%20associated%20with%20breast%20cancer%20metastasis%20to%20bone%20on%20a%20protein-protein%20interaction%20network%20with%20a%20shortest%20path%20algorithm" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/28076954" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+genes+associated+with+breast+cancer+metastasis+to+bone+on+a+protein-protein+interaction+network+with+a+shortest+path+algorithm+Cai+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0175185.ref040" id="pone.0175185.ref040" class="link-target"></a>Chen L, Zhang YH, Huang T, Cai YD. Identifying novel protein phenotype annotations by hybridizing protein-protein interactions and protein sequence similarities. Molecular Genetics and Genomics. 2016;291(2):913–34.  pmid:26728152 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenL%2C%20ZhangYH%2C%20HuangT%2C%20CaiYD.%20Identifying%20novel%20protein%20phenotype%20annotations%20by%20hybridizing%20protein-protein%20interactions%20and%20protein%20sequence%20similarities.%20Molecular%20Genetics%20and%20Genomics.%202016%3B291%282%29%3A913%E2%80%9334.%20doi%3A%2010.1007%2Fs00438-015-1157-9%2026728152" data-title="Identifying%20novel%20protein%20phenotype%20annotations%20by%20hybridizing%20protein-protein%20interactions%20and%20protein%20sequence%20similarities" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26728152" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identifying+novel+protein+phenotype+annotations+by+hybridizing+protein-protein+interactions+and+protein+sequence+similarities+Chen+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0175185.ref041" id="pone.0175185.ref041" class="link-target"></a>The UniProt Consortium. Update on activities at the Universal Protein Resource (UniProt) in 2013. Nucleic Acids Res. 2013;41(Database issue):D43–7. Epub 2012/11/20.  pmid:23161681 <ul class="reflinks"><li><a href="#" data-author="" data-cit="The%20UniProt%20Consortium.%20Update%20on%20activities%20at%20the%20Universal%20Protein%20Resource%20%28UniProt%29%20in%202013.%20Nucleic%20Acids%20Res.%202013%3B41%28Database%20issue%29%3AD43%E2%80%937.%20Epub%202012%2F11%2F20.%20doi%3A%2010.1093%2Fnar%2Fgks1068%2023161681" data-title="Update%20on%20activities%20at%20the%20Universal%20Protein%20Resource%20%28UniProt%29%20in%202013" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161681" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Update+on+activities+at+the+Universal+Protein+Resource+%28UniProt%29+in+2013++2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0175185.ref042" id="pone.0175185.ref042" class="link-target"></a>Franceschini A, Szklarczyk D, Frankild S, Kuhn M, Simonovic M, Roth A, et al. STRING v9. 1: protein-protein interaction networks, with increased coverage and integration. Nucleic acids research. 2013;41(D1):D808–D15. <ul class="reflinks"><li><a href="#" data-author="Franceschini" data-cit="FranceschiniA%2C%20SzklarczykD%2C%20FrankildS%2C%20KuhnM%2C%20SimonovicM%2C%20RothA%2C%20et%20al.%20STRING%20v9.%201%3A%20protein-protein%20interaction%20networks%2C%20with%20increased%20coverage%20and%20integration.%20Nucleic%20acids%20research.%202013%3B41%28D1%29%3AD808%E2%80%93D15." data-title="STRING%20v9.%201%3A%20protein-protein%20interaction%20networks%2C%20with%20increased%20coverage%20and%20integration" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=STRING+v9.+1%3A+protein-protein+interaction+networks%2C+with+increased+coverage+and+integration+Franceschini+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0175185.ref043" id="pone.0175185.ref043" class="link-target"></a>Ng KL, Ciou JS, Huang CH. Prediction of protein functions based on function-function correlation relations. Comput Biol Med. 2010;40(3):300–5.  pmid:20089249 <ul class="reflinks"><li><a href="#" data-author="Ng" data-cit="NgKL%2C%20CiouJS%2C%20HuangCH.%20Prediction%20of%20protein%20functions%20based%20on%20function-function%20correlation%20relations.%20Comput%20Biol%20Med.%202010%3B40%283%29%3A300%E2%80%935.%20doi%3A%2010.1016%2Fj.compbiomed.2010.01.001%2020089249" data-title="Prediction%20of%20protein%20functions%20based%20on%20function-function%20correlation%20relations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20089249" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Prediction+of+protein+functions+based+on+function-function+correlation+relations+Ng+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0175185.ref044" id="pone.0175185.ref044" class="link-target"></a>Hu LL, Huang T, Shi X, Lu WC, Cai YD, Chou KC. Predicting functions of proteins in mouse based on weighted protein-protein interaction network and protein hybrid properties. PLoS ONE. 2011;6(1):e14556.  pmid:21283518 <ul class="reflinks"><li><a href="#" data-author="Hu" data-cit="HuLL%2C%20HuangT%2C%20ShiX%2C%20LuWC%2C%20CaiYD%2C%20ChouKC.%20Predicting%20functions%20of%20proteins%20in%20mouse%20based%20on%20weighted%20protein-protein%20interaction%20network%20and%20protein%20hybrid%20properties.%20PLoS%20ONE.%202011%3B6%281%29%3Ae14556.%20doi%3A%2010.1371%2Fjournal.pone.0014556%2021283518" data-title="Predicting%20functions%20of%20proteins%20in%20mouse%20based%20on%20weighted%20protein-protein%20interaction%20network%20and%20protein%20hybrid%20properties" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21283518" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Predicting+functions+of+proteins+in+mouse+based+on+weighted+protein-protein+interaction+network+and+protein+hybrid+properties+Hu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0175185.ref045" id="pone.0175185.ref045" class="link-target"></a>Gormen TH, Leiserson CE, Rivest RL, Stein C, editors. Introduction to algorithms: MIT press Cambridge, MA; 1990. <ul class="find-nolinks"></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0175185.ref046" id="pone.0175185.ref046" class="link-target"></a>Chen L, Zeng W-M, Cai Y-D, Feng K-Y, Chou K-C. Predicting Anatomical Therapeutic Chemical (ATC) Classification of Drugs by Integrating Chemical-Chemical Interactions and Similarities. PLoS ONE. 2012;7(4):e35254.  pmid:22514724 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenL%2C%20ZengW-M%2C%20CaiY-D%2C%20FengK-Y%2C%20ChouK-C.%20Predicting%20Anatomical%20Therapeutic%20Chemical%20%28ATC%29%20Classification%20of%20Drugs%20by%20Integrating%20Chemical-Chemical%20Interactions%20and%20Similarities.%20PLoS%20ONE.%202012%3B7%284%29%3Ae35254.%20doi%3A%2010.1371%2Fjournal.pone.0035254%2022514724" data-title="Predicting%20Anatomical%20Therapeutic%20Chemical%20%28ATC%29%20Classification%20of%20Drugs%20by%20Integrating%20Chemical-Chemical%20Interactions%20and%20Similarities" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22514724" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Predicting+Anatomical+Therapeutic+Chemical+%28ATC%29+Classification+of+Drugs+by+Integrating+Chemical-Chemical+Interactions+and+Similarities+Chen+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0175185.ref047" id="pone.0175185.ref047" class="link-target"></a>Wei L, Xing P, Shi G, Ji ZL, Zou Q. Fast prediction of protein methylation sites using a sequence-based feature selection technique. IEEE/ACM Trans Comput Biol Bioinform. 2017. <ul class="reflinks"><li><a href="#" data-author="Wei" data-cit="WeiL%2C%20XingP%2C%20ShiG%2C%20JiZL%2C%20ZouQ.%20Fast%20prediction%20of%20protein%20methylation%20sites%20using%20a%20sequence-based%20feature%20selection%20technique.%20IEEE%2FACM%20Trans%20Comput%20Biol%20Bioinform.%202017." data-title="Fast%20prediction%20of%20protein%20methylation%20sites%20using%20a%20sequence-based%20feature%20selection%20technique" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Fast+prediction+of+protein+methylation+sites+using+a+sequence-based+feature+selection+technique+Wei+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0175185.ref048" id="pone.0175185.ref048" class="link-target"></a>Wei L, Xing P, Tang J, Zou Q. PhosPred-RF: a novel sequence-based predictor for phosphorylation sites using sequential information only. IEEE Trans Nanobioscience. 2017. <ul class="reflinks"><li><a href="#" data-author="Wei" data-cit="WeiL%2C%20XingP%2C%20TangJ%2C%20ZouQ.%20PhosPred-RF%3A%20a%20novel%20sequence-based%20predictor%20for%20phosphorylation%20sites%20using%20sequential%20information%20only.%20IEEE%20Trans%20Nanobioscience.%202017." data-title="PhosPred-RF%3A%20a%20novel%20sequence-based%20predictor%20for%20phosphorylation%20sites%20using%20sequential%20information%20only" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=PhosPred-RF%3A+a+novel+sequence-based+predictor+for+phosphorylation+sites+using+sequential+information+only+Wei+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0175185.ref049" id="pone.0175185.ref049" class="link-target"></a>Kohavi R, editor A study of cross-validation and bootstrap for accuracy estimation and model selection. International joint Conference on artificial intelligence; 1995: Lawrence Erlbaum Associates Ltd. <ul class="find-nolinks"></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0175185.ref050" id="pone.0175185.ref050" class="link-target"></a>Akira S, Uematsu S, Takeuchi O. Pathogen recognition and innate immunity. Cell. 2006;124(4):783–801.  pmid:16497588 <ul class="reflinks"><li><a href="#" data-author="Akira" data-cit="AkiraS%2C%20UematsuS%2C%20TakeuchiO.%20Pathogen%20recognition%20and%20innate%20immunity.%20Cell.%202006%3B124%284%29%3A783%E2%80%93801.%20doi%3A%2010.1016%2Fj.cell.2006.02.015%2016497588" data-title="Pathogen%20recognition%20and%20innate%20immunity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16497588" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Pathogen+recognition+and+innate+immunity+Akira+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0175185.ref051" id="pone.0175185.ref051" class="link-target"></a>Szajnik M, Szczepanski MJ, Czystowska M, Elishaev E, Mandapathil M, Nowak-Markwitz E, et al. TLR4 signaling induced by lipopolysaccharide or paclitaxel regulates tumor survival and chemoresistance in ovarian cancer. Oncogene. 2009;28(49):4353–63.  pmid:19826413 <ul class="reflinks"><li><a href="#" data-author="Szajnik" data-cit="SzajnikM%2C%20SzczepanskiMJ%2C%20CzystowskaM%2C%20ElishaevE%2C%20MandapathilM%2C%20Nowak-MarkwitzE%2C%20et%20al.%20TLR4%20signaling%20induced%20by%20lipopolysaccharide%20or%20paclitaxel%20regulates%20tumor%20survival%20and%20chemoresistance%20in%20ovarian%20cancer.%20Oncogene.%202009%3B28%2849%29%3A4353%E2%80%9363.%20doi%3A%2010.1038%2Fonc.2009.289%2019826413" data-title="TLR4%20signaling%20induced%20by%20lipopolysaccharide%20or%20paclitaxel%20regulates%20tumor%20survival%20and%20chemoresistance%20in%20ovarian%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19826413" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=TLR4+signaling+induced+by+lipopolysaccharide+or+paclitaxel+regulates+tumor+survival+and+chemoresistance+in+ovarian+cancer+Szajnik+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0175185.ref052" id="pone.0175185.ref052" class="link-target"></a>Szczepanski MJ, Czystowska M, Szajnik M, Harasymczuk M, Boyiadzis M, Kruk-Zagajewska A, et al. Triggering of Toll-like receptor 4 expressed on human head and neck squamous cell carcinoma promotes tumor development and protects the tumor from immune attack. Cancer Res. 2009;69(7):3105–13.  pmid:19318560 <ul class="reflinks"><li><a href="#" data-author="Szczepanski" data-cit="SzczepanskiMJ%2C%20CzystowskaM%2C%20SzajnikM%2C%20HarasymczukM%2C%20BoyiadzisM%2C%20Kruk-ZagajewskaA%2C%20et%20al.%20Triggering%20of%20Toll-like%20receptor%204%20expressed%20on%20human%20head%20and%20neck%20squamous%20cell%20carcinoma%20promotes%20tumor%20development%20and%20protects%20the%20tumor%20from%20immune%20attack.%20Cancer%20Res.%202009%3B69%287%29%3A3105%E2%80%9313.%20doi%3A%2010.1158%2F0008-5472.CAN-08-3838%2019318560" data-title="Triggering%20of%20Toll-like%20receptor%204%20expressed%20on%20human%20head%20and%20neck%20squamous%20cell%20carcinoma%20promotes%20tumor%20development%20and%20protects%20the%20tumor%20from%20immune%20attack" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19318560" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Triggering+of+Toll-like+receptor+4+expressed+on+human+head+and+neck+squamous+cell+carcinoma+promotes+tumor+development+and+protects+the+tumor+from+immune+attack+Szczepanski+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0175185.ref053" id="pone.0175185.ref053" class="link-target"></a>Tucci M, Stucci S, Passarelli A, Giudice G, Dammacco F, Silvestris F. The immune escape in melanoma: role of the impaired dendritic cell function. Expert review of clinical immunology. 2014;10(10):1395–404.  pmid:25225774 <ul class="reflinks"><li><a href="#" data-author="Tucci" data-cit="TucciM%2C%20StucciS%2C%20PassarelliA%2C%20GiudiceG%2C%20DammaccoF%2C%20SilvestrisF.%20The%20immune%20escape%20in%20melanoma%3A%20role%20of%20the%20impaired%20dendritic%20cell%20function.%20Expert%20review%20of%20clinical%20immunology.%202014%3B10%2810%29%3A1395%E2%80%93404.%20doi%3A%2010.1586%2F1744666X.2014.955851%2025225774" data-title="The%20immune%20escape%20in%20melanoma%3A%20role%20of%20the%20impaired%20dendritic%20cell%20function" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25225774" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+immune+escape+in+melanoma%3A+role+of+the+impaired+dendritic+cell+function+Tucci+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0175185.ref054" id="pone.0175185.ref054" class="link-target"></a>Cancer Genome Atlas Research N. Comprehensive genomic characterization of squamous cell lung cancers. Nature. 2012;489(7417):519–25.  pmid:22960745 <ul class="reflinks"><li><a href="#" data-author="" data-cit="Cancer%20Genome%20Atlas%20Research%20N.%20Comprehensive%20genomic%20characterization%20of%20squamous%20cell%20lung%20cancers.%20Nature.%202012%3B489%287417%29%3A519%E2%80%9325.%20doi%3A%2010.1038%2Fnature11404%2022960745" data-title="Comprehensive%20genomic%20characterization%20of%20squamous%20cell%20lung%20cancers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22960745" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Comprehensive+genomic+characterization+of+squamous+cell+lung+cancers++2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pone.0175185.ref055" id="pone.0175185.ref055" class="link-target"></a>Dutt A, Salvesen HB, Chen TH, Ramos AH, Onofrio RC, Hatton C, et al. Drug-sensitive FGFR2 mutations in endometrial carcinoma. Proc Natl Acad Sci U S A. 2008;105(25):8713–7.  pmid:18552176 <ul class="reflinks"><li><a href="#" data-author="Dutt" data-cit="DuttA%2C%20SalvesenHB%2C%20ChenTH%2C%20RamosAH%2C%20OnofrioRC%2C%20HattonC%2C%20et%20al.%20Drug-sensitive%20FGFR2%20mutations%20in%20endometrial%20carcinoma.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202008%3B105%2825%29%3A8713%E2%80%937.%20doi%3A%2010.1073%2Fpnas.0803379105%2018552176" data-title="Drug-sensitive%20FGFR2%20mutations%20in%20endometrial%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18552176" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Drug-sensitive+FGFR2+mutations+in+endometrial+carcinoma+Dutt+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pone.0175185.ref056" id="pone.0175185.ref056" class="link-target"></a>Byron SA, Gartside MG, Wellens CL, Mallon MA, Keenan JB, Powell MA, et al. Inhibition of activated fibroblast growth factor receptor 2 in endometrial cancer cells induces cell death despite PTEN abrogation. Cancer Res. 2008;68(17):6902–7.  pmid:18757403 <ul class="reflinks"><li><a href="#" data-author="Byron" data-cit="ByronSA%2C%20GartsideMG%2C%20WellensCL%2C%20MallonMA%2C%20KeenanJB%2C%20PowellMA%2C%20et%20al.%20Inhibition%20of%20activated%20fibroblast%20growth%20factor%20receptor%202%20in%20endometrial%20cancer%20cells%20induces%20cell%20death%20despite%20PTEN%20abrogation.%20Cancer%20Res.%202008%3B68%2817%29%3A6902%E2%80%937.%20doi%3A%2010.1158%2F0008-5472.CAN-08-0770%2018757403" data-title="Inhibition%20of%20activated%20fibroblast%20growth%20factor%20receptor%202%20in%20endometrial%20cancer%20cells%20induces%20cell%20death%20despite%20PTEN%20abrogation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18757403" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inhibition+of+activated+fibroblast+growth+factor+receptor+2+in+endometrial+cancer+cells+induces+cell+death+despite+PTEN+abrogation+Byron+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pone.0175185.ref057" id="pone.0175185.ref057" class="link-target"></a>Gozgit JM, Squillace RM, Wongchenko MJ, Miller D, Wardwell S, Mohemmad Q, et al. Combined targeting of FGFR2 and mTOR by ponatinib and ridaforolimus results in synergistic antitumor activity in FGFR2 mutant endometrial cancer models. Cancer Chemother Pharmacol. 2013;71(5):1315–23.  pmid:23468082 <ul class="reflinks"><li><a href="#" data-author="Gozgit" data-cit="GozgitJM%2C%20SquillaceRM%2C%20WongchenkoMJ%2C%20MillerD%2C%20WardwellS%2C%20MohemmadQ%2C%20et%20al.%20Combined%20targeting%20of%20FGFR2%20and%20mTOR%20by%20ponatinib%20and%20ridaforolimus%20results%20in%20synergistic%20antitumor%20activity%20in%20FGFR2%20mutant%20endometrial%20cancer%20models.%20Cancer%20Chemother%20Pharmacol.%202013%3B71%285%29%3A1315%E2%80%9323.%20doi%3A%2010.1007%2Fs00280-013-2131-z%2023468082" data-title="Combined%20targeting%20of%20FGFR2%20and%20mTOR%20by%20ponatinib%20and%20ridaforolimus%20results%20in%20synergistic%20antitumor%20activity%20in%20FGFR2%20mutant%20endometrial%20cancer%20models" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23468082" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Combined+targeting+of+FGFR2+and+mTOR+by+ponatinib+and+ridaforolimus+results+in+synergistic+antitumor+activity+in+FGFR2+mutant+endometrial+cancer+models+Gozgit+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pone.0175185.ref058" id="pone.0175185.ref058" class="link-target"></a>Liao RG, Jung J, Tchaicha J, Wilkerson MD, Sivachenko A, Beauchamp EM, et al. Inhibitor-sensitive FGFR2 and FGFR3 mutations in lung squamous cell carcinoma. Cancer Res. 2013;73(16):5195–205.  pmid:23786770 <ul class="reflinks"><li><a href="#" data-author="Liao" data-cit="LiaoRG%2C%20JungJ%2C%20TchaichaJ%2C%20WilkersonMD%2C%20SivachenkoA%2C%20BeauchampEM%2C%20et%20al.%20Inhibitor-sensitive%20FGFR2%20and%20FGFR3%20mutations%20in%20lung%20squamous%20cell%20carcinoma.%20Cancer%20Res.%202013%3B73%2816%29%3A5195%E2%80%93205.%20doi%3A%2010.1158%2F0008-5472.CAN-12-3950%2023786770" data-title="Inhibitor-sensitive%20FGFR2%20and%20FGFR3%20mutations%20in%20lung%20squamous%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23786770" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inhibitor-sensitive+FGFR2+and+FGFR3+mutations+in+lung+squamous+cell+carcinoma+Liao+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pone.0175185.ref059" id="pone.0175185.ref059" class="link-target"></a>Zaharieva BM, Simon R, Diener PA, Ackermann D, Maurer R, Alund G, et al. High-throughput tissue microarray analysis of 11q13 gene amplification (CCND1, FGF3, FGF4, EMS1) in urinary bladder cancer. The Journal of pathology. 2003;201(4):603–8.  pmid:14648664 <ul class="reflinks"><li><a href="#" data-author="Zaharieva" data-cit="ZaharievaBM%2C%20SimonR%2C%20DienerPA%2C%20AckermannD%2C%20MaurerR%2C%20AlundG%2C%20et%20al.%20High-throughput%20tissue%20microarray%20analysis%20of%2011q13%20gene%20amplification%20%28CCND1%2C%20FGF3%2C%20FGF4%2C%20EMS1%29%20in%20urinary%20bladder%20cancer.%20The%20Journal%20of%20pathology.%202003%3B201%284%29%3A603%E2%80%938.%20doi%3A%2010.1002%2Fpath.1481%2014648664" data-title="High-throughput%20tissue%20microarray%20analysis%20of%2011q13%20gene%20amplification%20%28CCND1%2C%20FGF3%2C%20FGF4%2C%20EMS1%29%20in%20urinary%20bladder%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14648664" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=High-throughput+tissue+microarray+analysis+of+11q13+gene+amplification+%28CCND1%2C+FGF3%2C+FGF4%2C+EMS1%29+in+urinary+bladder+cancer+Zaharieva+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span class="order">60.
            </span><a name="pone.0175185.ref060" id="pone.0175185.ref060" class="link-target"></a>Sugahara K, Michikawa Y, Ishikawa K, Shoji Y, Iwakawa M, Shibahara T, et al. Combination effects of distinct cores in 11q13 amplification region on cervical lymph node metastasis of oral squamous cell carcinoma. International journal of oncology. 2011;39(4):761–9.  pmid:21701773 <ul class="reflinks"><li><a href="#" data-author="Sugahara" data-cit="SugaharaK%2C%20MichikawaY%2C%20IshikawaK%2C%20ShojiY%2C%20IwakawaM%2C%20ShibaharaT%2C%20et%20al.%20Combination%20effects%20of%20distinct%20cores%20in%2011q13%20amplification%20region%20on%20cervical%20lymph%20node%20metastasis%20of%20oral%20squamous%20cell%20carcinoma.%20International%20journal%20of%20oncology.%202011%3B39%284%29%3A761%E2%80%939.%20doi%3A%2010.3892%2Fijo.2011.1094%2021701773" data-title="Combination%20effects%20of%20distinct%20cores%20in%2011q13%20amplification%20region%20on%20cervical%20lymph%20node%20metastasis%20of%20oral%20squamous%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21701773" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Combination+effects+of+distinct+cores+in+11q13+amplification+region+on+cervical+lymph+node+metastasis+of+oral+squamous+cell+carcinoma+Sugahara+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span class="order">61.
            </span><a name="pone.0175185.ref061" id="pone.0175185.ref061" class="link-target"></a>Karin M, Greten FR. NF-kappaB: linking inflammation and immunity to cancer development and progression. Nat Rev Immunol. 2005;5(10):749–59. Epub 2005/09/22.  pmid:16175180 <ul class="reflinks"><li><a href="#" data-author="Karin" data-cit="KarinM%2C%20GretenFR.%20NF-kappaB%3A%20linking%20inflammation%20and%20immunity%20to%20cancer%20development%20and%20progression.%20Nat%20Rev%20Immunol.%202005%3B5%2810%29%3A749%E2%80%9359.%20Epub%202005%2F09%2F22.%20doi%3A%2010.1038%2Fnri1703%2016175180" data-title="NF-kappaB%3A%20linking%20inflammation%20and%20immunity%20to%20cancer%20development%20and%20progression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16175180" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=NF-kappaB%3A+linking+inflammation+and+immunity+to+cancer+development+and+progression+Karin+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span class="order">62.
            </span><a name="pone.0175185.ref062" id="pone.0175185.ref062" class="link-target"></a>Pikarsky E, Porat RM, Stein I, Abramovitch R, Amit S, Kasem S, et al. NF-kappaB functions as a tumour promoter in inflammation-associated cancer. Nature. 2004;431(7007):461–6.  pmid:15329734 <ul class="reflinks"><li><a href="#" data-author="Pikarsky" data-cit="PikarskyE%2C%20PoratRM%2C%20SteinI%2C%20AbramovitchR%2C%20AmitS%2C%20KasemS%2C%20et%20al.%20NF-kappaB%20functions%20as%20a%20tumour%20promoter%20in%20inflammation-associated%20cancer.%20Nature.%202004%3B431%287007%29%3A461%E2%80%936.%20doi%3A%2010.1038%2Fnature02924%2015329734" data-title="NF-kappaB%20functions%20as%20a%20tumour%20promoter%20in%20inflammation-associated%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15329734" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=NF-kappaB+functions+as+a+tumour+promoter+in+inflammation-associated+cancer+Pikarsky+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref63"><span class="order">63.
            </span><a name="pone.0175185.ref063" id="pone.0175185.ref063" class="link-target"></a>Bu H, Rosdahl I, Sun XF, Zhang H. Importance of polymorphisms in NF-kappaB1 and NF-kappaBIalpha genes for melanoma risk, clinicopathological features and tumor progression in Swedish melanoma patients. J Cancer Res Clin Oncol. 2007;133(11):859–66.  pmid:17492467 <ul class="reflinks"><li><a href="#" data-author="Bu" data-cit="BuH%2C%20RosdahlI%2C%20SunXF%2C%20ZhangH.%20Importance%20of%20polymorphisms%20in%20NF-kappaB1%20and%20NF-kappaBIalpha%20genes%20for%20melanoma%20risk%2C%20clinicopathological%20features%20and%20tumor%20progression%20in%20Swedish%20melanoma%20patients.%20J%20Cancer%20Res%20Clin%20Oncol.%202007%3B133%2811%29%3A859%E2%80%9366.%20doi%3A%2010.1007%2Fs00432-007-0228-7%2017492467" data-title="Importance%20of%20polymorphisms%20in%20NF-kappaB1%20and%20NF-kappaBIalpha%20genes%20for%20melanoma%20risk%2C%20clinicopathological%20features%20and%20tumor%20progression%20in%20Swedish%20melanoma%20patients" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17492467" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Importance+of+polymorphisms+in+NF-kappaB1+and+NF-kappaBIalpha+genes+for+melanoma+risk%2C+clinicopathological+features+and+tumor+progression+in+Swedish+melanoma+patients+Bu+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref64"><span class="order">64.
            </span><a name="pone.0175185.ref064" id="pone.0175185.ref064" class="link-target"></a>Zhang P, Wei Q, Li X, Wang K, Zeng H, Bu H, et al. A functional insertion/deletion polymorphism in the promoter region of the NFKB1 gene increases susceptibility for prostate cancer. Cancer Genet Cytogenet. 2009;191(2):73–7.  pmid:19446741 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangP%2C%20WeiQ%2C%20LiX%2C%20WangK%2C%20ZengH%2C%20BuH%2C%20et%20al.%20A%20functional%20insertion%2Fdeletion%20polymorphism%20in%20the%20promoter%20region%20of%20the%20NFKB1%20gene%20increases%20susceptibility%20for%20prostate%20cancer.%20Cancer%20Genet%20Cytogenet.%202009%3B191%282%29%3A73%E2%80%937.%20doi%3A%2010.1016%2Fj.cancergencyto.2009.01.017%2019446741" data-title="A%20functional%20insertion%2Fdeletion%20polymorphism%20in%20the%20promoter%20region%20of%20the%20NFKB1%20gene%20increases%20susceptibility%20for%20prostate%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19446741" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+functional+insertion%2Fdeletion+polymorphism+in+the+promoter+region+of+the+NFKB1+gene+increases+susceptibility+for+prostate+cancer+Zhang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref65"><span class="order">65.
            </span><a name="pone.0175185.ref065" id="pone.0175185.ref065" class="link-target"></a>Lo SS, Chen JH, Wu CW, Lui WY. Functional polymorphism of NFKB1 promoter may correlate to the susceptibility of gastric cancer in aged patients. Surgery. 2009;145(3):280–5.  pmid:19231580 <ul class="reflinks"><li><a href="#" data-author="Lo" data-cit="LoSS%2C%20ChenJH%2C%20WuCW%2C%20LuiWY.%20Functional%20polymorphism%20of%20NFKB1%20promoter%20may%20correlate%20to%20the%20susceptibility%20of%20gastric%20cancer%20in%20aged%20patients.%20Surgery.%202009%3B145%283%29%3A280%E2%80%935.%20doi%3A%2010.1016%2Fj.surg.2008.11.001%2019231580" data-title="Functional%20polymorphism%20of%20NFKB1%20promoter%20may%20correlate%20to%20the%20susceptibility%20of%20gastric%20cancer%20in%20aged%20patients" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19231580" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Functional+polymorphism+of+NFKB1+promoter+may+correlate+to+the+susceptibility+of+gastric+cancer+in+aged+patients+Lo+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref66"><span class="order">66.
            </span><a name="pone.0175185.ref066" id="pone.0175185.ref066" class="link-target"></a>Lin CW, Hsieh YS, Hsin CH, Su CW, Lin CH, Wei LH, et al. Effects of NFKB1 and NFKBIA gene polymorphisms on susceptibility to environmental factors and the clinicopathologic development of oral cancer. PLoS One. 2012;7(4):e35078.  pmid:22509384 <ul class="reflinks"><li><a href="#" data-author="Lin" data-cit="LinCW%2C%20HsiehYS%2C%20HsinCH%2C%20SuCW%2C%20LinCH%2C%20WeiLH%2C%20et%20al.%20Effects%20of%20NFKB1%20and%20NFKBIA%20gene%20polymorphisms%20on%20susceptibility%20to%20environmental%20factors%20and%20the%20clinicopathologic%20development%20of%20oral%20cancer.%20PLoS%20One.%202012%3B7%284%29%3Ae35078.%20doi%3A%2010.1371%2Fjournal.pone.0035078%2022509384" data-title="Effects%20of%20NFKB1%20and%20NFKBIA%20gene%20polymorphisms%20on%20susceptibility%20to%20environmental%20factors%20and%20the%20clinicopathologic%20development%20of%20oral%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22509384" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Effects+of+NFKB1+and+NFKBIA+gene+polymorphisms+on+susceptibility+to+environmental+factors+and+the+clinicopathologic+development+of+oral+cancer+Lin+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref67"><span class="order">67.
            </span><a name="pone.0175185.ref067" id="pone.0175185.ref067" class="link-target"></a>Bevilacqua MP, Nelson RM. Selectins. J Clin Invest. 1993;91(2):379–87.  pmid:7679406 <ul class="reflinks"><li><a href="#" data-author="Bevilacqua" data-cit="BevilacquaMP%2C%20NelsonRM.%20Selectins.%20J%20Clin%20Invest.%201993%3B91%282%29%3A379%E2%80%9387.%20doi%3A%2010.1172%2FJCI116210%207679406" data-title="Selectins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7679406" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Selectins+Bevilacqua+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref68"><span class="order">68.
            </span><a name="pone.0175185.ref068" id="pone.0175185.ref068" class="link-target"></a>Lauri D, Needham L, Martin-Padura I, Dejana E. Tumor cell adhesion to endothelial cells: endothelial leukocyte adhesion molecule-1 as an inducible adhesive receptor specific for colon carcinoma cells. Journal of the National Cancer Institute. 1991;83(18):1321–4. pmid:1715924 <ul class="reflinks"><li><a href="#" data-author="Lauri" data-cit="LauriD%2C%20NeedhamL%2C%20Martin-PaduraI%2C%20DejanaE.%20Tumor%20cell%20adhesion%20to%20endothelial%20cells%3A%20endothelial%20leukocyte%20adhesion%20molecule-1%20as%20an%20inducible%20adhesive%20receptor%20specific%20for%20colon%20carcinoma%20cells.%20Journal%20of%20the%20National%20Cancer%20Institute.%201991%3B83%2818%29%3A1321%E2%80%934.%201715924" data-title="Tumor%20cell%20adhesion%20to%20endothelial%20cells%3A%20endothelial%20leukocyte%20adhesion%20molecule-1%20as%20an%20inducible%20adhesive%20receptor%20specific%20for%20colon%20carcinoma%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1715924" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor+cell+adhesion+to+endothelial+cells%3A+endothelial+leukocyte+adhesion+molecule-1+as+an+inducible+adhesive+receptor+specific+for+colon+carcinoma+cells+Lauri+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref69"><span class="order">69.
            </span><a name="pone.0175185.ref069" id="pone.0175185.ref069" class="link-target"></a>Wenzel CT, Scher RL, Richtsmeier WJ. Adhesion of head and neck squamous cell carcinoma to endothelial cells. The missing links. Archives of otolaryngology—head &amp; neck surgery. 1995;121(11):1279–86. <ul class="reflinks"><li><a href="#" data-author="Wenzel" data-cit="WenzelCT%2C%20ScherRL%2C%20RichtsmeierWJ.%20Adhesion%20of%20head%20and%20neck%20squamous%20cell%20carcinoma%20to%20endothelial%20cells.%20The%20missing%20links.%20Archives%20of%20otolaryngology%E2%80%94head%20%26%20neck%20surgery.%201995%3B121%2811%29%3A1279%E2%80%9386." data-title="Adhesion%20of%20head%20and%20neck%20squamous%20cell%20carcinoma%20to%20endothelial%20cells.%20The%20missing%20links" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Adhesion+of+head+and+neck+squamous+cell+carcinoma+to+endothelial+cells.+The+missing+links+Wenzel+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref70"><span class="order">70.
            </span><a name="pone.0175185.ref070" id="pone.0175185.ref070" class="link-target"></a>Chang PY, Kuo YB, Wu TL, Liao CT, Sun YC, Yen TC, et al. Association and prognostic value of serum inflammation markers in patients with leukoplakia and oral cavity cancer. Clinical chemistry and laboratory medicine: CCLM / FESCC. 2013;51(6):1291–300. <ul class="reflinks"><li><a href="#" data-author="Chang" data-cit="ChangPY%2C%20KuoYB%2C%20WuTL%2C%20LiaoCT%2C%20SunYC%2C%20YenTC%2C%20et%20al.%20Association%20and%20prognostic%20value%20of%20serum%20inflammation%20markers%20in%20patients%20with%20leukoplakia%20and%20oral%20cavity%20cancer.%20Clinical%20chemistry%20and%20laboratory%20medicine%3A%20CCLM%20%2F%20FESCC.%202013%3B51%286%29%3A1291%E2%80%93300." data-title="Association%20and%20prognostic%20value%20of%20serum%20inflammation%20markers%20in%20patients%20with%20leukoplakia%20and%20oral%20cavity%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Association+and+prognostic+value+of+serum+inflammation+markers+in+patients+with+leukoplakia+and+oral+cavity+cancer+Chang+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref71"><span class="order">71.
            </span><a name="pone.0175185.ref071" id="pone.0175185.ref071" class="link-target"></a>Lahav J. The functions of thrombospondin and its involvement in physiology and pathophysiology. Biochim Biophys Acta. 1993;1182(1):1–14. pmid:8347680 <ul class="reflinks"><li><a href="#" data-author="Lahav" data-cit="LahavJ.%20The%20functions%20of%20thrombospondin%20and%20its%20involvement%20in%20physiology%20and%20pathophysiology.%20Biochim%20Biophys%20Acta.%201993%3B1182%281%29%3A1%E2%80%9314.%208347680" data-title="The%20functions%20of%20thrombospondin%20and%20its%20involvement%20in%20physiology%20and%20pathophysiology" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8347680" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+functions+of+thrombospondin+and+its+involvement+in+physiology+and+pathophysiology+Lahav+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref72"><span class="order">72.
            </span><a name="pone.0175185.ref072" id="pone.0175185.ref072" class="link-target"></a>Yao L, Zhao YL, Itoh S, Wada S, Yue L, Furuta I. Thrombospondin-1 expression in oral squamous cell carcinomas: correlations with tumor vascularity, clinicopathological features and survival. Oral Oncol. 2000;36(6):539–44. pmid:11036248 <ul class="reflinks"><li><a href="#" data-author="Yao" data-cit="YaoL%2C%20ZhaoYL%2C%20ItohS%2C%20WadaS%2C%20YueL%2C%20FurutaI.%20Thrombospondin-1%20expression%20in%20oral%20squamous%20cell%20carcinomas%3A%20correlations%20with%20tumor%20vascularity%2C%20clinicopathological%20features%20and%20survival.%20Oral%20Oncol.%202000%3B36%286%29%3A539%E2%80%9344.%2011036248" data-title="Thrombospondin-1%20expression%20in%20oral%20squamous%20cell%20carcinomas%3A%20correlations%20with%20tumor%20vascularity%2C%20clinicopathological%20features%20and%20survival" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11036248" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Thrombospondin-1+expression+in+oral+squamous+cell+carcinomas%3A+correlations+with+tumor+vascularity%2C+clinicopathological+features+and+survival+Yao+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref73"><span class="order">73.
            </span><a name="pone.0175185.ref073" id="pone.0175185.ref073" class="link-target"></a>Pal SK, Nguyen CT, Morita KI, Miki Y, Kayamori K, Yamaguchi A, et al. THBS1 is induced by TGFB1 in the cancer stroma and promotes invasion of oral squamous cell carcinoma. Journal of oral pathology &amp; medicine: official publication of the International Association of Oral Pathologists and the American Academy of Oral Pathology. 2016. <ul class="reflinks"><li><a href="#" data-author="Pal" data-cit="PalSK%2C%20NguyenCT%2C%20MoritaKI%2C%20MikiY%2C%20KayamoriK%2C%20YamaguchiA%2C%20et%20al.%20THBS1%20is%20induced%20by%20TGFB1%20in%20the%20cancer%20stroma%20and%20promotes%20invasion%20of%20oral%20squamous%20cell%20carcinoma.%20Journal%20of%20oral%20pathology%20%26%20medicine%3A%20official%20publication%20of%20the%20International%20Association%20of%20Oral%20Pathologists%20and%20the%20American%20Academy%20of%20Oral%20Pathology.%202016." data-title="THBS1%20is%20induced%20by%20TGFB1%20in%20the%20cancer%20stroma%20and%20promotes%20invasion%20of%20oral%20squamous%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=THBS1+is+induced+by+TGFB1+in+the+cancer+stroma+and+promotes+invasion+of+oral+squamous+cell+carcinoma+Pal+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref74"><span class="order">74.
            </span><a name="pone.0175185.ref074" id="pone.0175185.ref074" class="link-target"></a>Durbin JE, Hackenmiller R, Simon MC, Levy DE. Targeted disruption of the mouse Stat1 gene results in compromised innate immunity to viral disease. Cell. 1996;84(3):443–50. pmid:8608598 <ul class="reflinks"><li><a href="#" data-author="Durbin" data-cit="DurbinJE%2C%20HackenmillerR%2C%20SimonMC%2C%20LevyDE.%20Targeted%20disruption%20of%20the%20mouse%20Stat1%20gene%20results%20in%20compromised%20innate%20immunity%20to%20viral%20disease.%20Cell.%201996%3B84%283%29%3A443%E2%80%9350.%208608598" data-title="Targeted%20disruption%20of%20the%20mouse%20Stat1%20gene%20results%20in%20compromised%20innate%20immunity%20to%20viral%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8608598" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Targeted+disruption+of+the+mouse+Stat1+gene+results+in+compromised+innate+immunity+to+viral+disease+Durbin+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref75"><span class="order">75.
            </span><a name="pone.0175185.ref075" id="pone.0175185.ref075" class="link-target"></a>Xi S, Dyer KF, Kimak M, Zhang Q, Gooding WE, Chaillet JR, et al. Decreased STAT1 expression by promoter methylation in squamous cell carcinogenesis. Journal of the National Cancer Institute. 2006;98(3):181–9.  pmid:16449678 <ul class="reflinks"><li><a href="#" data-author="Xi" data-cit="XiS%2C%20DyerKF%2C%20KimakM%2C%20ZhangQ%2C%20GoodingWE%2C%20ChailletJR%2C%20et%20al.%20Decreased%20STAT1%20expression%20by%20promoter%20methylation%20in%20squamous%20cell%20carcinogenesis.%20Journal%20of%20the%20National%20Cancer%20Institute.%202006%3B98%283%29%3A181%E2%80%939.%20doi%3A%2010.1093%2Fjnci%2Fdjj020%2016449678" data-title="Decreased%20STAT1%20expression%20by%20promoter%20methylation%20in%20squamous%20cell%20carcinogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16449678" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Decreased+STAT1+expression+by+promoter+methylation+in+squamous+cell+carcinogenesis+Xi+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref76"><span class="order">76.
            </span><a name="pone.0175185.ref076" id="pone.0175185.ref076" class="link-target"></a>Laimer K, Spizzo G, Obrist P, Gastl G, Brunhuber T, Schafer G, et al. STAT1 activation in squamous cell cancer of the oral cavity: a potential predictive marker of response to adjuvant chemotherapy. Cancer. 2007;110(2):326–33.  pmid:17559122 <ul class="reflinks"><li><a href="#" data-author="Laimer" data-cit="LaimerK%2C%20SpizzoG%2C%20ObristP%2C%20GastlG%2C%20BrunhuberT%2C%20SchaferG%2C%20et%20al.%20STAT1%20activation%20in%20squamous%20cell%20cancer%20of%20the%20oral%20cavity%3A%20a%20potential%20predictive%20marker%20of%20response%20to%20adjuvant%20chemotherapy.%20Cancer.%202007%3B110%282%29%3A326%E2%80%9333.%20doi%3A%2010.1002%2Fcncr.22813%2017559122" data-title="STAT1%20activation%20in%20squamous%20cell%20cancer%20of%20the%20oral%20cavity%3A%20a%20potential%20predictive%20marker%20of%20response%20to%20adjuvant%20chemotherapy" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17559122" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=STAT1+activation+in+squamous+cell+cancer+of+the+oral+cavity%3A+a+potential+predictive+marker+of+response+to+adjuvant+chemotherapy+Laimer+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref77"><span class="order">77.
            </span><a name="pone.0175185.ref077" id="pone.0175185.ref077" class="link-target"></a>Haroun F, Al-Shaar L, Habib RH, El-Saghir N, Tfayli A, Bazarbachi A, et al. Effects of CYP2B6 genetic polymorphisms in patients receiving cyclophosphamide combination chemotherapy for breast cancer. Cancer Chemother Pharmacol. 2015;75(1):207–14.  pmid:25428516 <ul class="reflinks"><li><a href="#" data-author="Haroun" data-cit="HarounF%2C%20Al-ShaarL%2C%20HabibRH%2C%20El-SaghirN%2C%20TfayliA%2C%20BazarbachiA%2C%20et%20al.%20Effects%20of%20CYP2B6%20genetic%20polymorphisms%20in%20patients%20receiving%20cyclophosphamide%20combination%20chemotherapy%20for%20breast%20cancer.%20Cancer%20Chemother%20Pharmacol.%202015%3B75%281%29%3A207%E2%80%9314.%20doi%3A%2010.1007%2Fs00280-014-2632-4%2025428516" data-title="Effects%20of%20CYP2B6%20genetic%20polymorphisms%20in%20patients%20receiving%20cyclophosphamide%20combination%20chemotherapy%20for%20breast%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25428516" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Effects+of+CYP2B6+genetic+polymorphisms+in+patients+receiving+cyclophosphamide+combination+chemotherapy+for+breast+cancer+Haroun+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref78"><span class="order">78.
            </span><a name="pone.0175185.ref078" id="pone.0175185.ref078" class="link-target"></a>Li Y, Zhao W, Zhao Z, Wu J, Chen L, Ma Y, et al. IL1B gene polymorphisms, age and the risk of non-small cell lung cancer in a Chinese population. Lung Cancer. 2015;89(3):232–7.  pmid:26141218 <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiY%2C%20ZhaoW%2C%20ZhaoZ%2C%20WuJ%2C%20ChenL%2C%20MaY%2C%20et%20al.%20IL1B%20gene%20polymorphisms%2C%20age%20and%20the%20risk%20of%20non-small%20cell%20lung%20cancer%20in%20a%20Chinese%20population.%20Lung%20Cancer.%202015%3B89%283%29%3A232%E2%80%937.%20doi%3A%2010.1016%2Fj.lungcan.2015.06.009%2026141218" data-title="IL1B%20gene%20polymorphisms%2C%20age%20and%20the%20risk%20of%20non-small%20cell%20lung%20cancer%20in%20a%20Chinese%20population" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26141218" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=IL1B+gene+polymorphisms%2C+age+and+the+risk+of+non-small+cell+lung+cancer+in+a+Chinese+population+Li+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref79"><span class="order">79.
            </span><a name="pone.0175185.ref079" id="pone.0175185.ref079" class="link-target"></a>Zeigler-Johnson C, Friebel T, Walker AH, Wang Y, Spangler E, Panossian S, et al. CYP3A4, CYP3A5, and CYP3A43 genotypes and haplotypes in the etiology and severity of prostate cancer. Cancer Res. 2004;64(22):8461–7.  pmid:15548719 <ul class="reflinks"><li><a href="#" data-author="Zeigler-Johnson" data-cit="Zeigler-JohnsonC%2C%20FriebelT%2C%20WalkerAH%2C%20WangY%2C%20SpanglerE%2C%20PanossianS%2C%20et%20al.%20CYP3A4%2C%20CYP3A5%2C%20and%20CYP3A43%20genotypes%20and%20haplotypes%20in%20the%20etiology%20and%20severity%20of%20prostate%20cancer.%20Cancer%20Res.%202004%3B64%2822%29%3A8461%E2%80%937.%20doi%3A%2010.1158%2F0008-5472.CAN-04-1651%2015548719" data-title="CYP3A4%2C%20CYP3A5%2C%20and%20CYP3A43%20genotypes%20and%20haplotypes%20in%20the%20etiology%20and%20severity%20of%20prostate%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15548719" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=CYP3A4%2C+CYP3A5%2C+and+CYP3A43+genotypes+and+haplotypes+in+the+etiology+and+severity+of+prostate+cancer+Zeigler-Johnson+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref80"><span class="order">80.
            </span><a name="pone.0175185.ref080" id="pone.0175185.ref080" class="link-target"></a>Chen L, Huang Z, Yao G, Lyu X, Li J, Hu X, et al. The expression of CXCL13 and its relation to unfavorable clinical characteristics in young breast cancer. J Transl Med. 2015;13:168.  pmid:25990390 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenL%2C%20HuangZ%2C%20YaoG%2C%20LyuX%2C%20LiJ%2C%20HuX%2C%20et%20al.%20The%20expression%20of%20CXCL13%20and%20its%20relation%20to%20unfavorable%20clinical%20characteristics%20in%20young%20breast%20cancer.%20J%20Transl%20Med.%202015%3B13%3A168.%20doi%3A%2010.1186%2Fs12967-015-0521-1%2025990390" data-title="The%20expression%20of%20CXCL13%20and%20its%20relation%20to%20unfavorable%20clinical%20characteristics%20in%20young%20breast%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25990390" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+expression+of+CXCL13+and+its+relation+to+unfavorable+clinical+characteristics+in+young+breast+cancer+Chen+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref81"><span class="order">81.
            </span><a name="pone.0175185.ref081" id="pone.0175185.ref081" class="link-target"></a>Walser TC, Ma X, Kundu N, Dorsey R, Goloubeva O, Fulton AM. Immune-mediated modulation of breast cancer growth and metastasis by the chemokine Mig (CXCL9) in a murine model. Journal of immunotherapy. 2007;30(5):490–8.  pmid:17589289 <ul class="reflinks"><li><a href="#" data-author="Walser" data-cit="WalserTC%2C%20MaX%2C%20KunduN%2C%20DorseyR%2C%20GoloubevaO%2C%20FultonAM.%20Immune-mediated%20modulation%20of%20breast%20cancer%20growth%20and%20metastasis%20by%20the%20chemokine%20Mig%20%28CXCL9%29%20in%20a%20murine%20model.%20Journal%20of%20immunotherapy.%202007%3B30%285%29%3A490%E2%80%938.%20doi%3A%2010.1097%2FCJI.0b013e318031b551%2017589289" data-title="Immune-mediated%20modulation%20of%20breast%20cancer%20growth%20and%20metastasis%20by%20the%20chemokine%20Mig%20%28CXCL9%29%20in%20a%20murine%20model" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17589289" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Immune-mediated+modulation+of+breast+cancer+growth+and+metastasis+by+the+chemokine+Mig+%28CXCL9%29+in+a+murine+model+Walser+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref82"><span class="order">82.
            </span><a name="pone.0175185.ref082" id="pone.0175185.ref082" class="link-target"></a>Walser TC, Rifat S, Ma X, Kundu N, Ward C, Goloubeva O, et al. Antagonism of CXCR3 inhibits lung metastasis in a murine model of metastatic breast cancer. Cancer Res. 2006;66(15):7701–7.  pmid:16885372 <ul class="reflinks"><li><a href="#" data-author="Walser" data-cit="WalserTC%2C%20RifatS%2C%20MaX%2C%20KunduN%2C%20WardC%2C%20GoloubevaO%2C%20et%20al.%20Antagonism%20of%20CXCR3%20inhibits%20lung%20metastasis%20in%20a%20murine%20model%20of%20metastatic%20breast%20cancer.%20Cancer%20Res.%202006%3B66%2815%29%3A7701%E2%80%937.%20doi%3A%2010.1158%2F0008-5472.CAN-06-0709%2016885372" data-title="Antagonism%20of%20CXCR3%20inhibits%20lung%20metastasis%20in%20a%20murine%20model%20of%20metastatic%20breast%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16885372" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Antagonism+of+CXCR3+inhibits+lung+metastasis+in+a+murine+model+of+metastatic+breast+cancer+Walser+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref83"><span class="order">83.
            </span><a name="pone.0175185.ref083" id="pone.0175185.ref083" class="link-target"></a>Kawada K, Hosogi H, Sonoshita M, Sakashita H, Manabe T, Shimahara Y, et al. Chemokine receptor CXCR3 promotes colon cancer metastasis to lymph nodes. Oncogene. 2007;26(32):4679–88.  pmid:17297455 <ul class="reflinks"><li><a href="#" data-author="Kawada" data-cit="KawadaK%2C%20HosogiH%2C%20SonoshitaM%2C%20SakashitaH%2C%20ManabeT%2C%20ShimaharaY%2C%20et%20al.%20Chemokine%20receptor%20CXCR3%20promotes%20colon%20cancer%20metastasis%20to%20lymph%20nodes.%20Oncogene.%202007%3B26%2832%29%3A4679%E2%80%9388.%20doi%3A%2010.1038%2Fsj.onc.1210267%2017297455" data-title="Chemokine%20receptor%20CXCR3%20promotes%20colon%20cancer%20metastasis%20to%20lymph%20nodes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17297455" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chemokine+receptor+CXCR3+promotes+colon+cancer+metastasis+to+lymph+nodes+Kawada+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref84"><span class="order">84.
            </span><a name="pone.0175185.ref084" id="pone.0175185.ref084" class="link-target"></a>Ma X, Norsworthy K, Kundu N, Rodgers WH, Gimotty PA, Goloubeva O, et al. CXCR3 expression is associated with poor survival in breast cancer and promotes metastasis in a murine model. Mol Cancer Ther. 2009;8(3):490–8.  pmid:19276169 <ul class="reflinks"><li><a href="#" data-author="Ma" data-cit="MaX%2C%20NorsworthyK%2C%20KunduN%2C%20RodgersWH%2C%20GimottyPA%2C%20GoloubevaO%2C%20et%20al.%20CXCR3%20expression%20is%20associated%20with%20poor%20survival%20in%20breast%20cancer%20and%20promotes%20metastasis%20in%20a%20murine%20model.%20Mol%20Cancer%20Ther.%202009%3B8%283%29%3A490%E2%80%938.%20doi%3A%2010.1158%2F1535-7163.MCT-08-0485%2019276169" data-title="CXCR3%20expression%20is%20associated%20with%20poor%20survival%20in%20breast%20cancer%20and%20promotes%20metastasis%20in%20a%20murine%20model" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19276169" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=CXCR3+expression+is+associated+with+poor+survival+in+breast+cancer+and+promotes+metastasis+in+a+murine+model+Ma+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref85"><span class="order">85.
            </span><a name="pone.0175185.ref085" id="pone.0175185.ref085" class="link-target"></a>Cambien B, Karimdjee BF, Richard-Fiardo P, Bziouech H, Barthel R, Millet MA, et al. Organ-specific inhibition of metastatic colon carcinoma by CXCR3 antagonism. British journal of cancer. 2009;100(11):1755–64.  pmid:19436305 <ul class="reflinks"><li><a href="#" data-author="Cambien" data-cit="CambienB%2C%20KarimdjeeBF%2C%20Richard-FiardoP%2C%20BziouechH%2C%20BarthelR%2C%20MilletMA%2C%20et%20al.%20Organ-specific%20inhibition%20of%20metastatic%20colon%20carcinoma%20by%20CXCR3%20antagonism.%20British%20journal%20of%20cancer.%202009%3B100%2811%29%3A1755%E2%80%9364.%20doi%3A%2010.1038%2Fsj.bjc.6605078%2019436305" data-title="Organ-specific%20inhibition%20of%20metastatic%20colon%20carcinoma%20by%20CXCR3%20antagonism" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19436305" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Organ-specific+inhibition+of+metastatic+colon+carcinoma+by+CXCR3+antagonism+Cambien+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref86"><span class="order">86.
            </span><a name="pone.0175185.ref086" id="pone.0175185.ref086" class="link-target"></a>Pradelli E, Karimdjee-Soilihi B, Michiels JF, Ricci JE, Millet MA, Vandenbos F, et al. Antagonism of chemokine receptor CXCR3 inhibits osteosarcoma metastasis to lungs. International journal of cancer Journal international du cancer. 2009;125(11):2586–94.  pmid:19544560 <ul class="reflinks"><li><a href="#" data-author="Pradelli" data-cit="PradelliE%2C%20Karimdjee-SoilihiB%2C%20MichielsJF%2C%20RicciJE%2C%20MilletMA%2C%20VandenbosF%2C%20et%20al.%20Antagonism%20of%20chemokine%20receptor%20CXCR3%20inhibits%20osteosarcoma%20metastasis%20to%20lungs.%20International%20journal%20of%20cancer%20Journal%20international%20du%20cancer.%202009%3B125%2811%29%3A2586%E2%80%9394.%20doi%3A%2010.1002%2Fijc.24665%2019544560" data-title="Antagonism%20of%20chemokine%20receptor%20CXCR3%20inhibits%20osteosarcoma%20metastasis%20to%20lungs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19544560" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Antagonism+of+chemokine+receptor+CXCR3+inhibits+osteosarcoma+metastasis+to+lungs+Pradelli+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref87"><span class="order">87.
            </span><a name="pone.0175185.ref087" id="pone.0175185.ref087" class="link-target"></a>Huang da W, Sherman BT, Lempicki RA. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. Nat Protoc. 2009;4(1):44–57. Epub 2009/01/10.  pmid:19131956 <ul class="reflinks"><li><a href="#" data-author="Huang%20da" data-cit="Huang%20daW%2C%20ShermanBT%2C%20LempickiRA.%20Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources.%20Nat%20Protoc.%202009%3B4%281%29%3A44%E2%80%9357.%20Epub%202009%2F01%2F10.%20doi%3A%2010.1038%2Fnprot.2008.211%2019131956" data-title="Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131956" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Systematic+and+integrative+analysis+of+large+gene+lists+using+DAVID+bioinformatics+resources+Huang+da+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref88"><span class="order">88.
            </span><a name="pone.0175185.ref088" id="pone.0175185.ref088" class="link-target"></a>Kim BK, Lee JW, Park PJ, Shin YS, Lee WY, Lee KA, et al. The multiplex bead array approach to identifying serum biomarkers associated with breast cancer. Breast cancer research: BCR. 2009;11(2):R22.  pmid:19400944 <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimBK%2C%20LeeJW%2C%20ParkPJ%2C%20ShinYS%2C%20LeeWY%2C%20LeeKA%2C%20et%20al.%20The%20multiplex%20bead%20array%20approach%20to%20identifying%20serum%20biomarkers%20associated%20with%20breast%20cancer.%20Breast%20cancer%20research%3A%20BCR.%202009%3B11%282%29%3AR22.%20doi%3A%2010.1186%2Fbcr2247%2019400944" data-title="The%20multiplex%20bead%20array%20approach%20to%20identifying%20serum%20biomarkers%20associated%20with%20breast%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19400944" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+multiplex+bead+array+approach+to+identifying+serum+biomarkers+associated+with+breast+cancer+Kim+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref89"><span class="order">89.
            </span><a name="pone.0175185.ref089" id="pone.0175185.ref089" class="link-target"></a>Key TJ, Appleby PN, Reeves GK, Roddam AW, Breast TEH. Insulin-like growth factor 1 (IGF1), IGF binding protein 3 (IGFBP3), and breast cancer risk: pooled individual data analysis of 17 prospective studies. Lancet Oncology. 2010;11(6):530–42.  pmid:20472501 <ul class="reflinks"><li><a href="#" data-author="Key" data-cit="KeyTJ%2C%20ApplebyPN%2C%20ReevesGK%2C%20RoddamAW%2C%20BreastTEH.%20Insulin-like%20growth%20factor%201%20%28IGF1%29%2C%20IGF%20binding%20protein%203%20%28IGFBP3%29%2C%20and%20breast%20cancer%20risk%3A%20pooled%20individual%20data%20analysis%20of%2017%20prospective%20studies.%20Lancet%20Oncology.%202010%3B11%286%29%3A530%E2%80%9342.%20doi%3A%2010.1016%2FS1470-2045%2810%2970095-4%2020472501" data-title="Insulin-like%20growth%20factor%201%20%28IGF1%29%2C%20IGF%20binding%20protein%203%20%28IGFBP3%29%2C%20and%20breast%20cancer%20risk%3A%20pooled%20individual%20data%20analysis%20of%2017%20prospective%20studies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20472501" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Insulin-like+growth+factor+1+%28IGF1%29%2C+IGF+binding+protein+3+%28IGFBP3%29%2C+and+breast+cancer+risk%3A+pooled+individual+data+analysis+of+17+prospective+studies+Key+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref90"><span class="order">90.
            </span><a name="pone.0175185.ref090" id="pone.0175185.ref090" class="link-target"></a>Karamouzis MV, Papavassiliou AG. Targeting insulin-like growth factor in breast cancer therapeutics. Crit Rev Oncol Hematol. 2012;84(1):8–17.  pmid:22424863 <ul class="reflinks"><li><a href="#" data-author="Karamouzis" data-cit="KaramouzisMV%2C%20PapavassiliouAG.%20Targeting%20insulin-like%20growth%20factor%20in%20breast%20cancer%20therapeutics.%20Crit%20Rev%20Oncol%20Hematol.%202012%3B84%281%29%3A8%E2%80%9317.%20doi%3A%2010.1016%2Fj.critrevonc.2012.02.010%2022424863" data-title="Targeting%20insulin-like%20growth%20factor%20in%20breast%20cancer%20therapeutics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22424863" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Targeting+insulin-like+growth+factor+in+breast+cancer+therapeutics+Karamouzis+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref91"><span class="order">91.
            </span><a name="pone.0175185.ref091" id="pone.0175185.ref091" class="link-target"></a>Polyak K, Lee MH, Erdjumentbromage H, Koff A, Roberts JM, Tempst P, et al. Cloning of P27(Kip1), a Cyclin-Dependent Kinase Inhibitor and a Potential Mediator of Extracellular Antimitogenic Signals. Cell. 1994;78(1):59–66. pmid:8033212 <ul class="reflinks"><li><a href="#" data-author="Polyak" data-cit="PolyakK%2C%20LeeMH%2C%20ErdjumentbromageH%2C%20KoffA%2C%20RobertsJM%2C%20TempstP%2C%20et%20al.%20Cloning%20of%20P27%28Kip1%29%2C%20a%20Cyclin-Dependent%20Kinase%20Inhibitor%20and%20a%20Potential%20Mediator%20of%20Extracellular%20Antimitogenic%20Signals.%20Cell.%201994%3B78%281%29%3A59%E2%80%9366.%208033212" data-title="Cloning%20of%20P27%28Kip1%29%2C%20a%20Cyclin-Dependent%20Kinase%20Inhibitor%20and%20a%20Potential%20Mediator%20of%20Extracellular%20Antimitogenic%20Signals" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8033212" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cloning+of+P27%28Kip1%29%2C+a+Cyclin-Dependent+Kinase+Inhibitor+and+a+Potential+Mediator+of+Extracellular+Antimitogenic+Signals+Polyak+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref92"><span class="order">92.
            </span><a name="pone.0175185.ref092" id="pone.0175185.ref092" class="link-target"></a>Chang BL, Zheng SQL, Isaacs SD, Wiley KE, Turner A, Li G, et al. A polymorphism in the CDKN1B gene is associated with increased risk of hereditary prostate cancer. Cancer Research. 2004;64(6):1997–9. pmid:15026335 <ul class="reflinks"><li><a href="#" data-author="Chang" data-cit="ChangBL%2C%20ZhengSQL%2C%20IsaacsSD%2C%20WileyKE%2C%20TurnerA%2C%20LiG%2C%20et%20al.%20A%20polymorphism%20in%20the%20CDKN1B%20gene%20is%20associated%20with%20increased%20risk%20of%20hereditary%20prostate%20cancer.%20Cancer%20Research.%202004%3B64%286%29%3A1997%E2%80%939.%2015026335" data-title="A%20polymorphism%20in%20the%20CDKN1B%20gene%20is%20associated%20with%20increased%20risk%20of%20hereditary%20prostate%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15026335" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+polymorphism+in+the+CDKN1B+gene+is+associated+with+increased+risk+of+hereditary+prostate+cancer+Chang+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref93"><span class="order">93.
            </span><a name="pone.0175185.ref093" id="pone.0175185.ref093" class="link-target"></a>Lang T, Nie YL. MiR-148a participates in the growth of RPMI8226 multiple myeloma cells by regulating CDKN1B. Biomedicine &amp; Pharmacotherapy. 2016;84:1967–71. <ul class="reflinks"><li><a href="#" data-author="Lang" data-cit="LangT%2C%20NieYL.%20MiR-148a%20participates%20in%20the%20growth%20of%20RPMI8226%20multiple%20myeloma%20cells%20by%20regulating%20CDKN1B.%20Biomedicine%20%26%20Pharmacotherapy.%202016%3B84%3A1967%E2%80%9371." data-title="MiR-148a%20participates%20in%20the%20growth%20of%20RPMI8226%20multiple%20myeloma%20cells%20by%20regulating%20CDKN1B" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MiR-148a+participates+in+the+growth+of+RPMI8226+multiple+myeloma+cells+by+regulating+CDKN1B+Lang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref94"><span class="order">94.
            </span><a name="pone.0175185.ref094" id="pone.0175185.ref094" class="link-target"></a>Nanda MS, Shah ZA, Sameer AS, Syeed N, Murtaza I, Siddiqi MA, et al. TP53-Molecular Soldier's Mutations in Bladder Cancer in the Kashmiri Population. Asian Pacific Journal of Cancer Prevention. 2011;12(1):67–72. pmid:21517233 <ul class="reflinks"><li><a href="#" data-author="Nanda" data-cit="NandaMS%2C%20ShahZA%2C%20SameerAS%2C%20SyeedN%2C%20MurtazaI%2C%20SiddiqiMA%2C%20et%20al.%20TP53-Molecular%20Soldier%27s%20Mutations%20in%20Bladder%20Cancer%20in%20the%20Kashmiri%20Population.%20Asian%20Pacific%20Journal%20of%20Cancer%20Prevention.%202011%3B12%281%29%3A67%E2%80%9372.%2021517233" data-title="TP53-Molecular%20Soldier%27s%20Mutations%20in%20Bladder%20Cancer%20in%20the%20Kashmiri%20Population" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21517233" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=TP53-Molecular+Soldier%27s+Mutations+in+Bladder+Cancer+in+the+Kashmiri+Population+Nanda+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref95"><span class="order">95.
            </span><a name="pone.0175185.ref095" id="pone.0175185.ref095" class="link-target"></a>Cheng LA, Zhang SB, MacLennan GT, Williamson SR, Lopez-Beltran A, Montironi R. Bladder cancer: translating molecular genetic insights into clinical practice. Human Pathology. 2011;42(4):455–81.  pmid:21106220 <ul class="reflinks"><li><a href="#" data-author="Cheng" data-cit="ChengLA%2C%20ZhangSB%2C%20MacLennanGT%2C%20WilliamsonSR%2C%20Lopez-BeltranA%2C%20MontironiR.%20Bladder%20cancer%3A%20translating%20molecular%20genetic%20insights%20into%20clinical%20practice.%20Human%20Pathology.%202011%3B42%284%29%3A455%E2%80%9381.%20doi%3A%2010.1016%2Fj.humpath.2010.07.007%2021106220" data-title="Bladder%20cancer%3A%20translating%20molecular%20genetic%20insights%20into%20clinical%20practice" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21106220" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bladder+cancer%3A+translating+molecular+genetic+insights+into+clinical+practice+Cheng+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref96"><span class="order">96.
            </span><a name="pone.0175185.ref096" id="pone.0175185.ref096" class="link-target"></a>Levine AJ, Oren M. The first 30 years of p53: growing ever more complex. Nat Rev Cancer. 2009;9(10):749–58.  pmid:19776744 <ul class="reflinks"><li><a href="#" data-author="Levine" data-cit="LevineAJ%2C%20OrenM.%20The%20first%2030%20years%20of%20p53%3A%20growing%20ever%20more%20complex.%20Nat%20Rev%20Cancer.%202009%3B9%2810%29%3A749%E2%80%9358.%20doi%3A%2010.1038%2Fnrc2723%2019776744" data-title="The%20first%2030%20years%20of%20p53%3A%20growing%20ever%20more%20complex" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19776744" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+first+30+years+of+p53%3A+growing+ever+more+complex+Levine+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref97"><span class="order">97.
            </span><a name="pone.0175185.ref097" id="pone.0175185.ref097" class="link-target"></a>Nishiyama H, Watanabe J, Ogawa O. p53 and chemosensitivity in bladder cancer. Int J Clin Oncol. 2008;13(4):282–6.  pmid:18704627 <ul class="reflinks"><li><a href="#" data-author="Nishiyama" data-cit="NishiyamaH%2C%20WatanabeJ%2C%20OgawaO.%20p53%20and%20chemosensitivity%20in%20bladder%20cancer.%20Int%20J%20Clin%20Oncol.%202008%3B13%284%29%3A282%E2%80%936.%20doi%3A%2010.1007%2Fs10147-008-0815-x%2018704627" data-title="p53%20and%20chemosensitivity%20in%20bladder%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18704627" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=p53+and+chemosensitivity+in+bladder+cancer+Nishiyama+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref98"><span class="order">98.
            </span><a name="pone.0175185.ref098" id="pone.0175185.ref098" class="link-target"></a>Wang SJ, Gu W. To be, or not to be: functional dilemma of p53 metabolic regulation. Curr Opin Oncol. 2014;26(1):78–85.  pmid:24240177 <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangSJ%2C%20GuW.%20To%20be%2C%20or%20not%20to%20be%3A%20functional%20dilemma%20of%20p53%20metabolic%20regulation.%20Curr%20Opin%20Oncol.%202014%3B26%281%29%3A78%E2%80%9385.%20doi%3A%2010.1097%2FCCO.0000000000000024%2024240177" data-title="To%20be%2C%20or%20not%20to%20be%3A%20functional%20dilemma%20of%20p53%20metabolic%20regulation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24240177" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=To+be%2C+or+not+to+be%3A+functional+dilemma+of+p53+metabolic+regulation+Wang+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref99"><span class="order">99.
            </span><a name="pone.0175185.ref099" id="pone.0175185.ref099" class="link-target"></a>Evan GI, Wyllie AH, Gilbert CS, Littlewood TD, Land H, Brooks M, et al. Induction of Apoptosis in Fibroblasts by C-Myc Protein. Cell. 1992;69(1):119–28. pmid:1555236 <ul class="reflinks"><li><a href="#" data-author="Evan" data-cit="EvanGI%2C%20WyllieAH%2C%20GilbertCS%2C%20LittlewoodTD%2C%20LandH%2C%20BrooksM%2C%20et%20al.%20Induction%20of%20Apoptosis%20in%20Fibroblasts%20by%20C-Myc%20Protein.%20Cell.%201992%3B69%281%29%3A119%E2%80%9328.%201555236" data-title="Induction%20of%20Apoptosis%20in%20Fibroblasts%20by%20C-Myc%20Protein" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1555236" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Induction+of+Apoptosis+in+Fibroblasts+by+C-Myc+Protein+Evan+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref100"><span class="order">100.
            </span><a name="pone.0175185.ref100" id="pone.0175185.ref100" class="link-target"></a>Bissonnette RP, Echeverri F, Mahboubi A, Green DR. Apoptotic cell death induced by c-myc is inhibited by bcl-2. Nature. 1992;359(6395):552–4.  pmid:1406975 <ul class="reflinks"><li><a href="#" data-author="Bissonnette" data-cit="BissonnetteRP%2C%20EcheverriF%2C%20MahboubiA%2C%20GreenDR.%20Apoptotic%20cell%20death%20induced%20by%20c-myc%20is%20inhibited%20by%20bcl-2.%20Nature.%201992%3B359%286395%29%3A552%E2%80%934.%20doi%3A%2010.1038%2F359552a0%201406975" data-title="Apoptotic%20cell%20death%20induced%20by%20c-myc%20is%20inhibited%20by%20bcl-2" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1406975" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Apoptotic+cell+death+induced+by+c-myc+is+inhibited+by+bcl-2+Bissonnette+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref101"><span class="order">101.
            </span><a name="pone.0175185.ref101" id="pone.0175185.ref101" class="link-target"></a>Egle A, Harris AW, Bouillet P, Cory S. Bim is a suppressor of Myc-induced mouse B cell leukemia. Proceedings of the National Academy of Sciences of the United States of America. 2004;101(16):6164–9.  pmid:15079075 <ul class="reflinks"><li><a href="#" data-author="Egle" data-cit="EgleA%2C%20HarrisAW%2C%20BouilletP%2C%20CoryS.%20Bim%20is%20a%20suppressor%20of%20Myc-induced%20mouse%20B%20cell%20leukemia.%20Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America.%202004%3B101%2816%29%3A6164%E2%80%939.%20doi%3A%2010.1073%2Fpnas.0401471101%2015079075" data-title="Bim%20is%20a%20suppressor%20of%20Myc-induced%20mouse%20B%20cell%20leukemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15079075" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bim+is+a+suppressor+of+Myc-induced+mouse+B+cell+leukemia+Egle+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref102"><span class="order">102.
            </span><a name="pone.0175185.ref102" id="pone.0175185.ref102" class="link-target"></a>Murphy DJ, Junttila MR, Pouyet L, Karnezis A, Shchors K, Bui DA, et al. Distinct thresholds govern Myc's biological output in vivo. Cancer Cell. 2008;14(6):447–57.  pmid:19061836 <ul class="reflinks"><li><a href="#" data-author="Murphy" data-cit="MurphyDJ%2C%20JunttilaMR%2C%20PouyetL%2C%20KarnezisA%2C%20ShchorsK%2C%20BuiDA%2C%20et%20al.%20Distinct%20thresholds%20govern%20Myc%27s%20biological%20output%20in%20vivo.%20Cancer%20Cell.%202008%3B14%286%29%3A447%E2%80%9357.%20doi%3A%2010.1016%2Fj.ccr.2008.10.018%2019061836" data-title="Distinct%20thresholds%20govern%20Myc%27s%20biological%20output%20in%20vivo" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19061836" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Distinct+thresholds+govern+Myc%27s+biological+output+in+vivo+Murphy+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref103"><span class="order">103.
            </span><a name="pone.0175185.ref103" id="pone.0175185.ref103" class="link-target"></a>Mitchell KO, Ricci MS, Miyashita T, Dicker DT, Jin ZY, Reed JC, et al. Bax is a transcriptional target and mediator of c-Myc-induced apoptosis. Cancer Research. 2000;60(22):6318–25. pmid:11103792 <ul class="reflinks"><li><a href="#" data-author="Mitchell" data-cit="MitchellKO%2C%20RicciMS%2C%20MiyashitaT%2C%20DickerDT%2C%20JinZY%2C%20ReedJC%2C%20et%20al.%20Bax%20is%20a%20transcriptional%20target%20and%20mediator%20of%20c-Myc-induced%20apoptosis.%20Cancer%20Research.%202000%3B60%2822%29%3A6318%E2%80%9325.%2011103792" data-title="Bax%20is%20a%20transcriptional%20target%20and%20mediator%20of%20c-Myc-induced%20apoptosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11103792" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bax+is+a+transcriptional+target+and+mediator+of+c-Myc-induced+apoptosis+Mitchell+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref104"><span class="order">104.
            </span><a name="pone.0175185.ref104" id="pone.0175185.ref104" class="link-target"></a>Nicholson DW, Thornberry NA. Caspases: killer proteases. Trends Biochem Sci. 1997;22(8):299–306. pmid:9270303 <ul class="reflinks"><li><a href="#" data-author="Nicholson" data-cit="NicholsonDW%2C%20ThornberryNA.%20Caspases%3A%20killer%20proteases.%20Trends%20Biochem%20Sci.%201997%3B22%288%29%3A299%E2%80%93306.%209270303" data-title="Caspases%3A%20killer%20proteases" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9270303" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Caspases%3A+killer+proteases+Nicholson+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref105"><span class="order">105.
            </span><a name="pone.0175185.ref105" id="pone.0175185.ref105" class="link-target"></a>Dang CV, O'Donnell KA, Zeller KI, Nguyen T, Osthus RC, Li F. The c-Myc target gene network. Seminars in cancer biology. 2006;16(4):253–64.  pmid:16904903 <ul class="reflinks"><li><a href="#" data-author="Dang" data-cit="DangCV%2C%20O%27DonnellKA%2C%20ZellerKI%2C%20NguyenT%2C%20OsthusRC%2C%20LiF.%20The%20c-Myc%20target%20gene%20network.%20Seminars%20in%20cancer%20biology.%202006%3B16%284%29%3A253%E2%80%9364.%20doi%3A%2010.1016%2Fj.semcancer.2006.07.014%2016904903" data-title="The%20c-Myc%20target%20gene%20network" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16904903" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+c-Myc+target+gene+network+Dang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref106"><span class="order">106.
            </span><a name="pone.0175185.ref106" id="pone.0175185.ref106" class="link-target"></a>Zhao X, Mohaupt M, Jiang J, Liu S, Li B, Qin Z. Tumor necrosis factor receptor 2-mediated tumor suppression is nitric oxide dependent and involves angiostasis. Cancer Res. 2007;67(9):4443–50.  pmid:17483359 <ul class="reflinks"><li><a href="#" data-author="Zhao" data-cit="ZhaoX%2C%20MohauptM%2C%20JiangJ%2C%20LiuS%2C%20LiB%2C%20QinZ.%20Tumor%20necrosis%20factor%20receptor%202-mediated%20tumor%20suppression%20is%20nitric%20oxide%20dependent%20and%20involves%20angiostasis.%20Cancer%20Res.%202007%3B67%289%29%3A4443%E2%80%9350.%20doi%3A%2010.1158%2F0008-5472.CAN-07-0185%2017483359" data-title="Tumor%20necrosis%20factor%20receptor%202-mediated%20tumor%20suppression%20is%20nitric%20oxide%20dependent%20and%20involves%20angiostasis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483359" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor+necrosis+factor+receptor+2-mediated+tumor+suppression+is+nitric+oxide+dependent+and+involves+angiostasis+Zhao+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref107"><span class="order">107.
            </span><a name="pone.0175185.ref107" id="pone.0175185.ref107" class="link-target"></a>Qin Z, Kruger-Krasagakes S, Kunzendorf U, Hock H, Diamantstein T, Blankenstein T. Expression of tumor necrosis factor by different tumor cell lines results either in tumor suppression or augmented metastasis. J Exp Med. 1993;178(1):355–60. pmid:8315391 <ul class="reflinks"><li><a href="#" data-author="Qin" data-cit="QinZ%2C%20Kruger-KrasagakesS%2C%20KunzendorfU%2C%20HockH%2C%20DiamantsteinT%2C%20BlankensteinT.%20Expression%20of%20tumor%20necrosis%20factor%20by%20different%20tumor%20cell%20lines%20results%20either%20in%20tumor%20suppression%20or%20augmented%20metastasis.%20J%20Exp%20Med.%201993%3B178%281%29%3A355%E2%80%9360.%208315391" data-title="Expression%20of%20tumor%20necrosis%20factor%20by%20different%20tumor%20cell%20lines%20results%20either%20in%20tumor%20suppression%20or%20augmented%20metastasis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8315391" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+of+tumor+necrosis+factor+by+different+tumor+cell+lines+results+either+in+tumor+suppression+or+augmented+metastasis+Qin+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref108"><span class="order">108.
            </span><a name="pone.0175185.ref108" id="pone.0175185.ref108" class="link-target"></a>Blankenstein T, Qin ZH, Uberla K, Muller W, Rosen H, Volk HD, et al. Tumor Suppression after Tumor-Cell Targeted Tumor-Necrosis-Factor-Alpha Gene-Transfer. Journal of Experimental Medicine. 1991;173(5):1047–52. pmid:2022919 <ul class="reflinks"><li><a href="#" data-author="Blankenstein" data-cit="BlankensteinT%2C%20QinZH%2C%20UberlaK%2C%20MullerW%2C%20RosenH%2C%20VolkHD%2C%20et%20al.%20Tumor%20Suppression%20after%20Tumor-Cell%20Targeted%20Tumor-Necrosis-Factor-Alpha%20Gene-Transfer.%20Journal%20of%20Experimental%20Medicine.%201991%3B173%285%29%3A1047%E2%80%9352.%202022919" data-title="Tumor%20Suppression%20after%20Tumor-Cell%20Targeted%20Tumor-Necrosis-Factor-Alpha%20Gene-Transfer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2022919" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor+Suppression+after+Tumor-Cell+Targeted+Tumor-Necrosis-Factor-Alpha+Gene-Transfer+Blankenstein+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref109"><span class="order">109.
            </span><a name="pone.0175185.ref109" id="pone.0175185.ref109" class="link-target"></a>Oliff A, Defeo-Jones D, Boyer M, Martinez D, Kiefer D, Vuocolo G, et al. Tumors secreting human TNF/cachectin induce cachexia in mice. Cell. 1987;50(4):555–63. pmid:3607879 <ul class="reflinks"><li><a href="#" data-author="Oliff" data-cit="OliffA%2C%20Defeo-JonesD%2C%20BoyerM%2C%20MartinezD%2C%20KieferD%2C%20VuocoloG%2C%20et%20al.%20Tumors%20secreting%20human%20TNF%2Fcachectin%20induce%20cachexia%20in%20mice.%20Cell.%201987%3B50%284%29%3A555%E2%80%9363.%203607879" data-title="Tumors%20secreting%20human%20TNF%2Fcachectin%20induce%20cachexia%20in%20mice" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/3607879" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tumors+secreting+human+TNF%2Fcachectin+induce+cachexia+in+mice+Oliff+1987" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref110"><span class="order">110.
            </span><a name="pone.0175185.ref110" id="pone.0175185.ref110" class="link-target"></a>Kulbe H, Thompson R, Wilson JL, Robinson S, Hagemann T, Fatah R, et al. The inflammatory cytokine tumor necrosis factor-alpha generates an autocrine tumor-promoting network in epithelial ovarian cancer cells. Cancer Res. 2007;67(2):585–92.  pmid:17234767 <ul class="reflinks"><li><a href="#" data-author="Kulbe" data-cit="KulbeH%2C%20ThompsonR%2C%20WilsonJL%2C%20RobinsonS%2C%20HagemannT%2C%20FatahR%2C%20et%20al.%20The%20inflammatory%20cytokine%20tumor%20necrosis%20factor-alpha%20generates%20an%20autocrine%20tumor-promoting%20network%20in%20epithelial%20ovarian%20cancer%20cells.%20Cancer%20Res.%202007%3B67%282%29%3A585%E2%80%9392.%20doi%3A%2010.1158%2F0008-5472.CAN-06-2941%2017234767" data-title="The%20inflammatory%20cytokine%20tumor%20necrosis%20factor-alpha%20generates%20an%20autocrine%20tumor-promoting%20network%20in%20epithelial%20ovarian%20cancer%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17234767" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+inflammatory+cytokine+tumor+necrosis+factor-alpha+generates+an+autocrine+tumor-promoting+network+in+epithelial+ovarian+cancer+cells+Kulbe+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref111"><span class="order">111.
            </span><a name="pone.0175185.ref111" id="pone.0175185.ref111" class="link-target"></a>Ben-Neriah Y, Karin M. Inflammation meets cancer, with NF-kappaB as the matchmaker. Nat Immunol. 2011;12(8):715–23.  pmid:21772280 <ul class="reflinks"><li><a href="#" data-author="Ben-Neriah" data-cit="Ben-NeriahY%2C%20KarinM.%20Inflammation%20meets%20cancer%2C%20with%20NF-kappaB%20as%20the%20matchmaker.%20Nat%20Immunol.%202011%3B12%288%29%3A715%E2%80%9323.%20doi%3A%2010.1038%2Fni.2060%2021772280" data-title="Inflammation%20meets%20cancer%2C%20with%20NF-kappaB%20as%20the%20matchmaker" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21772280" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inflammation+meets+cancer%2C+with+NF-kappaB+as+the+matchmaker+Ben-Neriah+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref112"><span class="order">112.
            </span><a name="pone.0175185.ref112" id="pone.0175185.ref112" class="link-target"></a>DiDonato JA, Mercurio F, Karin M. NF-kappa B and the link between inflammation and cancer. Immunol Rev. 2012;246:379–400.  pmid:22435567 <ul class="reflinks"><li><a href="#" data-author="DiDonato" data-cit="DiDonatoJA%2C%20MercurioF%2C%20KarinM.%20NF-kappa%20B%20and%20the%20link%20between%20inflammation%20and%20cancer.%20Immunol%20Rev.%202012%3B246%3A379%E2%80%93400.%20doi%3A%2010.1111%2Fj.1600-065X.2012.01099.x%2022435567" data-title="NF-kappa%20B%20and%20the%20link%20between%20inflammation%20and%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22435567" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=NF-kappa+B+and+the+link+between+inflammation+and+cancer+DiDonato+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref113"><span class="order">113.
            </span><a name="pone.0175185.ref113" id="pone.0175185.ref113" class="link-target"></a>Kang MH, Oh SC, Lee HJ, Kang HN, Kim JL, Kim JS, et al. Metastatic function of BMP-2 in gastric cancer cells: the role of PI3K/AKT, MAPK, the NF-kappaB pathway, and MMP-9 expression. Exp Cell Res. 2011;317(12):1746–62.  pmid:21570392 <ul class="reflinks"><li><a href="#" data-author="Kang" data-cit="KangMH%2C%20OhSC%2C%20LeeHJ%2C%20KangHN%2C%20KimJL%2C%20KimJS%2C%20et%20al.%20Metastatic%20function%20of%20BMP-2%20in%20gastric%20cancer%20cells%3A%20the%20role%20of%20PI3K%2FAKT%2C%20MAPK%2C%20the%20NF-kappaB%20pathway%2C%20and%20MMP-9%20expression.%20Exp%20Cell%20Res.%202011%3B317%2812%29%3A1746%E2%80%9362.%20doi%3A%2010.1016%2Fj.yexcr.2011.04.006%2021570392" data-title="Metastatic%20function%20of%20BMP-2%20in%20gastric%20cancer%20cells%3A%20the%20role%20of%20PI3K%2FAKT%2C%20MAPK%2C%20the%20NF-kappaB%20pathway%2C%20and%20MMP-9%20expression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21570392" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Metastatic+function+of+BMP-2+in+gastric+cancer+cells%3A+the+role+of+PI3K%2FAKT%2C+MAPK%2C+the+NF-kappaB+pathway%2C+and+MMP-9+expression+Kang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref114"><span class="order">114.
            </span><a name="pone.0175185.ref114" id="pone.0175185.ref114" class="link-target"></a>Graham TR, Odero-Marah VA, Chung LW, Agrawal KC, Davis R, Abdel-Mageed AB. PI3K/Akt-dependent transcriptional regulation and activation of BMP-2-Smad signaling by NF-kappaB in metastatic prostate cancer cells. The Prostate. 2009;69(2):168–80.  pmid:18942118 <ul class="reflinks"><li><a href="#" data-author="Graham" data-cit="GrahamTR%2C%20Odero-MarahVA%2C%20ChungLW%2C%20AgrawalKC%2C%20DavisR%2C%20Abdel-MageedAB.%20PI3K%2FAkt-dependent%20transcriptional%20regulation%20and%20activation%20of%20BMP-2-Smad%20signaling%20by%20NF-kappaB%20in%20metastatic%20prostate%20cancer%20cells.%20The%20Prostate.%202009%3B69%282%29%3A168%E2%80%9380.%20doi%3A%2010.1002%2Fpros.20870%2018942118" data-title="PI3K%2FAkt-dependent%20transcriptional%20regulation%20and%20activation%20of%20BMP-2-Smad%20signaling%20by%20NF-kappaB%20in%20metastatic%20prostate%20cancer%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18942118" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=PI3K%2FAkt-dependent+transcriptional+regulation+and+activation+of+BMP-2-Smad+signaling+by+NF-kappaB+in+metastatic+prostate+cancer+cells+Graham+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref115"><span class="order">115.
            </span><a name="pone.0175185.ref115" id="pone.0175185.ref115" class="link-target"></a>Levine RL, Pardanani A, Tefferi A, Gilliland DG. Role of JAK2 in the pathogenesis and therapy of myeloproliferative disorders. Nat Rev Cancer. 2007;7(9):673–83.  pmid:17721432 <ul class="reflinks"><li><a href="#" data-author="Levine" data-cit="LevineRL%2C%20PardananiA%2C%20TefferiA%2C%20GillilandDG.%20Role%20of%20JAK2%20in%20the%20pathogenesis%20and%20therapy%20of%20myeloproliferative%20disorders.%20Nat%20Rev%20Cancer.%202007%3B7%289%29%3A673%E2%80%9383.%20doi%3A%2010.1038%2Fnrc2210%2017721432" data-title="Role%20of%20JAK2%20in%20the%20pathogenesis%20and%20therapy%20of%20myeloproliferative%20disorders" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721432" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Role+of+JAK2+in+the+pathogenesis+and+therapy+of+myeloproliferative+disorders+Levine+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref116"><span class="order">116.
            </span><a name="pone.0175185.ref116" id="pone.0175185.ref116" class="link-target"></a>Silva M, Richard C, Benito A, Sanz C, Olalla I, Fernandez-Luna JL. Expression of Bcl-x in erythroid precursors from patients with polycythemia vera. N Engl J Med. 1998;338(9):564–71.  pmid:9475763 <ul class="reflinks"><li><a href="#" data-author="Silva" data-cit="SilvaM%2C%20RichardC%2C%20BenitoA%2C%20SanzC%2C%20OlallaI%2C%20Fernandez-LunaJL.%20Expression%20of%20Bcl-x%20in%20erythroid%20precursors%20from%20patients%20with%20polycythemia%20vera.%20N%20Engl%20J%20Med.%201998%3B338%289%29%3A564%E2%80%9371.%20doi%3A%2010.1056%2FNEJM199802263380902%209475763" data-title="Expression%20of%20Bcl-x%20in%20erythroid%20precursors%20from%20patients%20with%20polycythemia%20vera" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9475763" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+of+Bcl-x+in+erythroid+precursors+from+patients+with+polycythemia+vera+Silva+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref117"><span class="order">117.
            </span><a name="pone.0175185.ref117" id="pone.0175185.ref117" class="link-target"></a>Sansone P, Bromberg J. Targeting the Interleukin-6/Jak/Stat Pathway in Human Malignancies. Journal of Clinical Oncology. 2012;30(9):1005–14.  pmid:22355058 <ul class="reflinks"><li><a href="#" data-author="Sansone" data-cit="SansoneP%2C%20BrombergJ.%20Targeting%20the%20Interleukin-6%2FJak%2FStat%20Pathway%20in%20Human%20Malignancies.%20Journal%20of%20Clinical%20Oncology.%202012%3B30%289%29%3A1005%E2%80%9314.%20doi%3A%2010.1200%2FJCO.2010.31.8907%2022355058" data-title="Targeting%20the%20Interleukin-6%2FJak%2FStat%20Pathway%20in%20Human%20Malignancies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22355058" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Targeting+the+Interleukin-6%2FJak%2FStat+Pathway+in+Human+Malignancies+Sansone+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref118"><span class="order">118.
            </span><a name="pone.0175185.ref118" id="pone.0175185.ref118" class="link-target"></a>Traer E, MacKenzie R, Snead J, Agarwal A, Eiring AM, O'Hare T, et al. Blockade of JAK2-mediated extrinsic survival signals restores sensitivity of CML cells to ABL inhibitors. Leukemia. 2012;26(5):1140–3.  pmid:22094585 <ul class="reflinks"><li><a href="#" data-author="Traer" data-cit="TraerE%2C%20MacKenzieR%2C%20SneadJ%2C%20AgarwalA%2C%20EiringAM%2C%20O%27HareT%2C%20et%20al.%20Blockade%20of%20JAK2-mediated%20extrinsic%20survival%20signals%20restores%20sensitivity%20of%20CML%20cells%20to%20ABL%20inhibitors.%20Leukemia.%202012%3B26%285%29%3A1140%E2%80%933.%20doi%3A%2010.1038%2Fleu.2011.325%2022094585" data-title="Blockade%20of%20JAK2-mediated%20extrinsic%20survival%20signals%20restores%20sensitivity%20of%20CML%20cells%20to%20ABL%20inhibitors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22094585" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Blockade+of+JAK2-mediated+extrinsic+survival+signals+restores+sensitivity+of+CML+cells+to+ABL+inhibitors+Traer+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref119"><span class="order">119.
            </span><a name="pone.0175185.ref119" id="pone.0175185.ref119" class="link-target"></a>Groeger S, Jarzina F, Domann E, Meyle J. Porphyromonas gingivalis activates NFkappaB and MAPK pathways in human oral epithelial cells. BMC Immunol. 2017;18(1):1.  pmid:28056810 <ul class="reflinks"><li><a href="#" data-author="Groeger" data-cit="GroegerS%2C%20JarzinaF%2C%20DomannE%2C%20MeyleJ.%20Porphyromonas%20gingivalis%20activates%20NFkappaB%20and%20MAPK%20pathways%20in%20human%20oral%20epithelial%20cells.%20BMC%20Immunol.%202017%3B18%281%29%3A1.%20doi%3A%2010.1186%2Fs12865-016-0185-5%2028056810" data-title="Porphyromonas%20gingivalis%20activates%20NFkappaB%20and%20MAPK%20pathways%20in%20human%20oral%20epithelial%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/28056810" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Porphyromonas+gingivalis+activates+NFkappaB+and+MAPK+pathways+in+human+oral+epithelial+cells+Groeger+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref120"><span class="order">120.
            </span><a name="pone.0175185.ref120" id="pone.0175185.ref120" class="link-target"></a>McCubrey JA, Steelman LS, Chappell WH, Abrams SL, Franklin RA, Montalto G, et al. Ras/Raf/MEK/ERK and PI3K/PTEN/Akt/mTOR Cascade Inhibitors: How Mutations Can Result in Therapy Resistance and How to Overcome Resistance. Oncotarget. 2012;3(10):1068–111.  pmid:23085539 <ul class="reflinks"><li><a href="#" data-author="McCubrey" data-cit="McCubreyJA%2C%20SteelmanLS%2C%20ChappellWH%2C%20AbramsSL%2C%20FranklinRA%2C%20MontaltoG%2C%20et%20al.%20Ras%2FRaf%2FMEK%2FERK%20and%20PI3K%2FPTEN%2FAkt%2FmTOR%20Cascade%20Inhibitors%3A%20How%20Mutations%20Can%20Result%20in%20Therapy%20Resistance%20and%20How%20to%20Overcome%20Resistance.%20Oncotarget.%202012%3B3%2810%29%3A1068%E2%80%93111.%20doi%3A%2010.18632%2Foncotarget.659%2023085539" data-title="Ras%2FRaf%2FMEK%2FERK%20and%20PI3K%2FPTEN%2FAkt%2FmTOR%20Cascade%20Inhibitors%3A%20How%20Mutations%20Can%20Result%20in%20Therapy%20Resistance%20and%20How%20to%20Overcome%20Resistance" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23085539" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Ras%2FRaf%2FMEK%2FERK+and+PI3K%2FPTEN%2FAkt%2FmTOR+Cascade+Inhibitors%3A+How+Mutations+Can+Result+in+Therapy+Resistance+and+How+to+Overcome+Resistance+McCubrey+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref121"><span class="order">121.
            </span><a name="pone.0175185.ref121" id="pone.0175185.ref121" class="link-target"></a>Wei L, Tang J, Zou Q. Local-DPP: An improved DNA-binding protein prediction method by exploring local evolutionary information. Inform Sciences. 2017;384:135–44. <ul class="reflinks"><li><a href="#" data-author="Wei" data-cit="WeiL%2C%20TangJ%2C%20ZouQ.%20Local-DPP%3A%20An%20improved%20DNA-binding%20protein%20prediction%20method%20by%20exploring%20local%20evolutionary%20information.%20Inform%20Sciences.%202017%3B384%3A135%E2%80%9344." data-title="Local-DPP%3A%20An%20improved%20DNA-binding%20protein%20prediction%20method%20by%20exploring%20local%20evolutionary%20information" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Local-DPP%3A+An+improved+DNA-binding+protein+prediction+method+by+exploring+local+evolutionary+information+Wei+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref122"><span class="order">122.
            </span><a name="pone.0175185.ref122" id="pone.0175185.ref122" class="link-target"></a>Wei L, Zou Q. Recent Progress in Machine Learning-Based Methods for Protein Fold Recognition. International journal of molecular sciences. 2016;17(12). <ul class="reflinks"><li><a href="#" data-author="Wei" data-cit="WeiL%2C%20ZouQ.%20Recent%20Progress%20in%20Machine%20Learning-Based%20Methods%20for%20Protein%20Fold%20Recognition.%20International%20journal%20of%20molecular%20sciences.%202016%3B17%2812%29." data-title="Recent%20Progress%20in%20Machine%20Learning-Based%20Methods%20for%20Protein%20Fold%20Recognition" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Recent+Progress+in+Machine+Learning-Based+Methods+for+Protein+Fold+Recognition+Wei+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref123"><span class="order">123.
            </span><a name="pone.0175185.ref123" id="pone.0175185.ref123" class="link-target"></a>Wei L, Liao M, Gao X, Zou Q. Enhanced Protein Fold Prediction Method Through a Novel Feature Extraction Technique. IEEE Trans Nanobioscience. 2015;14(6):649–59.  pmid:26335556 <ul class="reflinks"><li><a href="#" data-author="Wei" data-cit="WeiL%2C%20LiaoM%2C%20GaoX%2C%20ZouQ.%20Enhanced%20Protein%20Fold%20Prediction%20Method%20Through%20a%20Novel%20Feature%20Extraction%20Technique.%20IEEE%20Trans%20Nanobioscience.%202015%3B14%286%29%3A649%E2%80%9359.%20doi%3A%2010.1109%2FTNB.2015.2450233%2026335556" data-title="Enhanced%20Protein%20Fold%20Prediction%20Method%20Through%20a%20Novel%20Feature%20Extraction%20Technique" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26335556" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Enhanced+Protein+Fold+Prediction+Method+Through+a+Novel+Feature+Extraction+Technique+Wei+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref124"><span class="order">124.
            </span><a name="pone.0175185.ref124" id="pone.0175185.ref124" class="link-target"></a>Su R, Zhang C, Pham TD, Davey R, Bischof L, Vallotton P, et al. Detection of tubule boundaries based on circular shortest path and polar-transformation of arbitrary shapes. Journal of microscopy. 2016;264(2):127–42.  pmid:27172164 <ul class="reflinks"><li><a href="#" data-author="Su" data-cit="SuR%2C%20ZhangC%2C%20PhamTD%2C%20DaveyR%2C%20BischofL%2C%20VallottonP%2C%20et%20al.%20Detection%20of%20tubule%20boundaries%20based%20on%20circular%20shortest%20path%20and%20polar-transformation%20of%20arbitrary%20shapes.%20Journal%20of%20microscopy.%202016%3B264%282%29%3A127%E2%80%9342.%20doi%3A%2010.1111%2Fjmi.12421%2027172164" data-title="Detection%20of%20tubule%20boundaries%20based%20on%20circular%20shortest%20path%20and%20polar-transformation%20of%20arbitrary%20shapes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27172164" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Detection+of+tubule+boundaries+based+on+circular+shortest+path+and+polar-transformation+of+arbitrary+shapes+Su+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref125"><span class="order">125.
            </span><a name="pone.0175185.ref125" id="pone.0175185.ref125" class="link-target"></a>Bromberg J. Stat proteins and oncogenesis. J Clin Invest. 2002;109(9):1139–42.  pmid:11994401 <ul class="reflinks"><li><a href="#" data-author="Bromberg" data-cit="BrombergJ.%20Stat%20proteins%20and%20oncogenesis.%20J%20Clin%20Invest.%202002%3B109%289%29%3A1139%E2%80%9342.%20doi%3A%2010.1172%2FJCI15617%2011994401" data-title="Stat%20proteins%20and%20oncogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11994401" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Stat+proteins+and+oncogenesis+Bromberg+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref126"><span class="order">126.
            </span><a name="pone.0175185.ref126" id="pone.0175185.ref126" class="link-target"></a>Wu YW, Tang W, Zuo JP. Toll-like receptors: potential targets for lupus treatment. Acta pharmacologica Sinica. 2015;36(12):1395–407.  pmid:26592511 <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuYW%2C%20TangW%2C%20ZuoJP.%20Toll-like%20receptors%3A%20potential%20targets%20for%20lupus%20treatment.%20Acta%20pharmacologica%20Sinica.%202015%3B36%2812%29%3A1395%E2%80%93407.%20doi%3A%2010.1038%2Faps.2015.91%2026592511" data-title="Toll-like%20receptors%3A%20potential%20targets%20for%20lupus%20treatment" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26592511" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Toll-like+receptors%3A+potential+targets+for+lupus+treatment+Wu+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref127"><span class="order">127.
            </span><a name="pone.0175185.ref127" id="pone.0175185.ref127" class="link-target"></a>Xiang W, Chao ZY, Feng DY. Role of Toll-like receptor/MYD88 signaling in neurodegenerative diseases. Reviews in the neurosciences. 2015;26(4):407–14.  pmid:25870959 <ul class="reflinks"><li><a href="#" data-author="Xiang" data-cit="XiangW%2C%20ChaoZY%2C%20FengDY.%20Role%20of%20Toll-like%20receptor%2FMYD88%20signaling%20in%20neurodegenerative%20diseases.%20Reviews%20in%20the%20neurosciences.%202015%3B26%284%29%3A407%E2%80%9314.%20doi%3A%2010.1515%2Frevneuro-2014-0067%2025870959" data-title="Role%20of%20Toll-like%20receptor%2FMYD88%20signaling%20in%20neurodegenerative%20diseases" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25870959" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Role+of+Toll-like+receptor%2FMYD88+signaling+in+neurodegenerative+diseases+Xiang+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref128"><span class="order">128.
            </span><a name="pone.0175185.ref128" id="pone.0175185.ref128" class="link-target"></a>Muller M, Herrath J, Malmstrom V. IL-1R1 is expressed on both Helios(+) and Helios(-)FoxP3(+)CD4(+) T cells in the rheumatic joint. Clinical and Experimental Immunology. 2015;182(1):90–100.  pmid:26076982 <ul class="reflinks"><li><a href="#" data-author="Muller" data-cit="MullerM%2C%20HerrathJ%2C%20MalmstromV.%20IL-1R1%20is%20expressed%20on%20both%20Helios%28%2B%29%20and%20Helios%28-%29FoxP3%28%2B%29CD4%28%2B%29%20T%20cells%20in%20the%20rheumatic%20joint.%20Clinical%20and%20Experimental%20Immunology.%202015%3B182%281%29%3A90%E2%80%93100.%20doi%3A%2010.1111%2Fcei.12668%2026076982" data-title="IL-1R1%20is%20expressed%20on%20both%20Helios%28%2B%29%20and%20Helios%28-%29FoxP3%28%2B%29CD4%28%2B%29%20T%20cells%20in%20the%20rheumatic%20joint" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26076982" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=IL-1R1+is+expressed+on+both+Helios%28%2B%29+and+Helios%28-%29FoxP3%28%2B%29CD4%28%2B%29+T+cells+in+the+rheumatic+joint+Muller+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref129"><span class="order">129.
            </span><a name="pone.0175185.ref129" id="pone.0175185.ref129" class="link-target"></a>Danielson PB. The cytochrome P450 superfamily: biochemistry, evolution and drug metabolism in humans. Curr Drug Metab. 2002;3(6):561–97. pmid:12369887 <ul class="reflinks"><li><a href="#" data-author="Danielson" data-cit="DanielsonPB.%20The%20cytochrome%20P450%20superfamily%3A%20biochemistry%2C%20evolution%20and%20drug%20metabolism%20in%20humans.%20Curr%20Drug%20Metab.%202002%3B3%286%29%3A561%E2%80%9397.%2012369887" data-title="The%20cytochrome%20P450%20superfamily%3A%20biochemistry%2C%20evolution%20and%20drug%20metabolism%20in%20humans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12369887" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+cytochrome+P450+superfamily%3A+biochemistry%2C+evolution+and+drug+metabolism+in+humans+Danielson+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref130"><span class="order">130.
            </span><a name="pone.0175185.ref130" id="pone.0175185.ref130" class="link-target"></a>Haudenschild DR, Palmer SM, Moseley TA, You Z, Reddi AH. Bone morphogenetic protein (BMP)-6 signaling and BMP antagonist noggin in prostate cancer. Cancer Res. 2004;64(22):8276–84.  pmid:15548695 <ul class="reflinks"><li><a href="#" data-author="Haudenschild" data-cit="HaudenschildDR%2C%20PalmerSM%2C%20MoseleyTA%2C%20YouZ%2C%20ReddiAH.%20Bone%20morphogenetic%20protein%20%28BMP%29-6%20signaling%20and%20BMP%20antagonist%20noggin%20in%20prostate%20cancer.%20Cancer%20Res.%202004%3B64%2822%29%3A8276%E2%80%9384.%20doi%3A%2010.1158%2F0008-5472.CAN-04-2251%2015548695" data-title="Bone%20morphogenetic%20protein%20%28BMP%29-6%20signaling%20and%20BMP%20antagonist%20noggin%20in%20prostate%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15548695" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bone+morphogenetic+protein+%28BMP%29-6+signaling+and+BMP+antagonist+noggin+in+prostate+cancer+Haudenschild+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref131"><span class="order">131.
            </span><a name="pone.0175185.ref131" id="pone.0175185.ref131" class="link-target"></a>Zhang X, Ibrahimi OA, Olsen SK, Umemori H, Mohammadi M, Ornitz DM. Receptor specificity of the fibroblast growth factor family. The complete mammalian FGF family. J Biol Chem. 2006;281(23):15694–700.  pmid:16597617 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangX%2C%20IbrahimiOA%2C%20OlsenSK%2C%20UmemoriH%2C%20MohammadiM%2C%20OrnitzDM.%20Receptor%20specificity%20of%20the%20fibroblast%20growth%20factor%20family.%20The%20complete%20mammalian%20FGF%20family.%20J%20Biol%20Chem.%202006%3B281%2823%29%3A15694%E2%80%93700.%20doi%3A%2010.1074%2Fjbc.M601252200%2016597617" data-title="Receptor%20specificity%20of%20the%20fibroblast%20growth%20factor%20family.%20The%20complete%20mammalian%20FGF%20family" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16597617" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Receptor+specificity+of+the+fibroblast+growth+factor+family.+The+complete+mammalian+FGF+family+Zhang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref132"><span class="order">132.
            </span><a name="pone.0175185.ref132" id="pone.0175185.ref132" class="link-target"></a>Zhou H, Wu J, Wang T, Zhang X, Liu D. CXCL10/CXCR3 axis promotes the invasion of gastric cancer via PI3K/AKT pathway-dependent MMPs production. Biomedicine &amp; pharmacotherapy = Biomedecine &amp; pharmacotherapie. 2016;82:479–88. <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouH%2C%20WuJ%2C%20WangT%2C%20ZhangX%2C%20LiuD.%20CXCL10%2FCXCR3%20axis%20promotes%20the%20invasion%20of%20gastric%20cancer%20via%20PI3K%2FAKT%20pathway-dependent%20MMPs%20production.%20Biomedicine%20%26%20pharmacotherapy%20%3D%20Biomedecine%20%26%20pharmacotherapie.%202016%3B82%3A479%E2%80%9388." data-title="CXCL10%2FCXCR3%20axis%20promotes%20the%20invasion%20of%20gastric%20cancer%20via%20PI3K%2FAKT%20pathway-dependent%20MMPs%20production" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=CXCL10%2FCXCR3+axis+promotes+the+invasion+of+gastric+cancer+via+PI3K%2FAKT+pathway-dependent+MMPs+production+Zhou+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref133"><span class="order">133.
            </span><a name="pone.0175185.ref133" id="pone.0175185.ref133" class="link-target"></a>Pranzatelli MR, Tate ED, McGee NR, Travelstead AL, Verhulst SJ, Ransohoff RM. Expression of CXCR3 and its ligands CXCL9, -10 and -11 in paediatric opsoclonus-myoclonus syndrome. Clin Exp Immunol. 2013;172(3):427–36.  pmid:23600831 <ul class="reflinks"><li><a href="#" data-author="Pranzatelli" data-cit="PranzatelliMR%2C%20TateED%2C%20McGeeNR%2C%20TravelsteadAL%2C%20VerhulstSJ%2C%20RansohoffRM.%20Expression%20of%20CXCR3%20and%20its%20ligands%20CXCL9%2C%20-10%20and%20-11%20in%20paediatric%20opsoclonus-myoclonus%20syndrome.%20Clin%20Exp%20Immunol.%202013%3B172%283%29%3A427%E2%80%9336.%20doi%3A%2010.1111%2Fcei.12065%2023600831" data-title="Expression%20of%20CXCR3%20and%20its%20ligands%20CXCL9%2C%20-10%20and%20-11%20in%20paediatric%20opsoclonus-myoclonus%20syndrome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23600831" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+of+CXCR3+and+its+ligands+CXCL9%2C+-10+and+-11+in+paediatric+opsoclonus-myoclonus+syndrome+Pranzatelli+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref134"><span class="order">134.
            </span><a name="pone.0175185.ref134" id="pone.0175185.ref134" class="link-target"></a>Baenziger NL, Brodie GN, Majerus PW. Isolation and properties of a thrombin-sensitive protein of human platelets. J Biol Chem. 1972;247(9):2723–31. pmid:4260214 <ul class="reflinks"><li><a href="#" data-author="Baenziger" data-cit="BaenzigerNL%2C%20BrodieGN%2C%20MajerusPW.%20Isolation%20and%20properties%20of%20a%20thrombin-sensitive%20protein%20of%20human%20platelets.%20J%20Biol%20Chem.%201972%3B247%289%29%3A2723%E2%80%9331.%204260214" data-title="Isolation%20and%20properties%20of%20a%20thrombin-sensitive%20protein%20of%20human%20platelets" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/4260214" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Isolation+and+properties+of+a+thrombin-sensitive+protein+of+human+platelets+Baenziger+1972" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref135"><span class="order">135.
            </span><a name="pone.0175185.ref135" id="pone.0175185.ref135" class="link-target"></a>Bornstein P. Diversity of function is inherent in matricellular proteins: an appraisal of thrombospondin 1. The Journal of cell biology. 1995;130(3):503–6. pmid:7542656 <ul class="reflinks"><li><a href="#" data-author="Bornstein" data-cit="BornsteinP.%20Diversity%20of%20function%20is%20inherent%20in%20matricellular%20proteins%3A%20an%20appraisal%20of%20thrombospondin%201.%20The%20Journal%20of%20cell%20biology.%201995%3B130%283%29%3A503%E2%80%936.%207542656" data-title="Diversity%20of%20function%20is%20inherent%20in%20matricellular%20proteins%3A%20an%20appraisal%20of%20thrombospondin%201" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7542656" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Diversity+of+function+is+inherent+in+matricellular+proteins%3A+an+appraisal+of+thrombospondin+1+Bornstein+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref136"><span class="order">136.
            </span><a name="pone.0175185.ref136" id="pone.0175185.ref136" class="link-target"></a>Labib K, Tercero JA, Diffley JF. Uninterrupted MCM2-7 function required for DNA replication fork progression. Science. 2000;288(5471):1643–7. pmid:10834843 <ul class="reflinks"><li><a href="#" data-author="Labib" data-cit="LabibK%2C%20TerceroJA%2C%20DiffleyJF.%20Uninterrupted%20MCM2-7%20function%20required%20for%20DNA%20replication%20fork%20progression.%20Science.%202000%3B288%285471%29%3A1643%E2%80%937.%2010834843" data-title="Uninterrupted%20MCM2-7%20function%20required%20for%20DNA%20replication%20fork%20progression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10834843" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Uninterrupted+MCM2-7+function+required+for+DNA+replication+fork+progression+Labib+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref137"><span class="order">137.
            </span><a name="pone.0175185.ref137" id="pone.0175185.ref137" class="link-target"></a>Elcin AE, Parmaksiz M, Dogan A, Seker S, Durkut S, Dalva K, et al. Differential gene expression profiling of human adipose stem cells differentiating into smooth muscle-like cells by TGFbeta1/BMP4. Exp Cell Res. 2017. <ul class="reflinks"><li><a href="#" data-author="Elcin" data-cit="ElcinAE%2C%20ParmaksizM%2C%20DoganA%2C%20SekerS%2C%20DurkutS%2C%20DalvaK%2C%20et%20al.%20Differential%20gene%20expression%20profiling%20of%20human%20adipose%20stem%20cells%20differentiating%20into%20smooth%20muscle-like%20cells%20by%20TGFbeta1%2FBMP4.%20Exp%20Cell%20Res.%202017." data-title="Differential%20gene%20expression%20profiling%20of%20human%20adipose%20stem%20cells%20differentiating%20into%20smooth%20muscle-like%20cells%20by%20TGFbeta1%2FBMP4" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+gene+expression+profiling+of+human+adipose+stem+cells+differentiating+into+smooth+muscle-like+cells+by+TGFbeta1%2FBMP4+Elcin+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref138"><span class="order">138.
            </span><a name="pone.0175185.ref138" id="pone.0175185.ref138" class="link-target"></a>Vallinoto AC, Graca ES, Araujo MS, Azevedo VN, Cayres-Vallinoto I, Machado LF, et al. IFNG +874T/A polymorphism and cytokine plasma levels are associated with susceptibility to Mycobacterium tuberculosis infection and clinical manifestation of tuberculosis. Human immunology. 2010;71(7):692–6.  pmid:20353805 <ul class="reflinks"><li><a href="#" data-author="Vallinoto" data-cit="VallinotoAC%2C%20GracaES%2C%20AraujoMS%2C%20AzevedoVN%2C%20Cayres-VallinotoI%2C%20MachadoLF%2C%20et%20al.%20IFNG%20%2B874T%2FA%20polymorphism%20and%20cytokine%20plasma%20levels%20are%20associated%20with%20susceptibility%20to%20Mycobacterium%20tuberculosis%20infection%20and%20clinical%20manifestation%20of%20tuberculosis.%20Human%20immunology.%202010%3B71%287%29%3A692%E2%80%936.%20doi%3A%2010.1016%2Fj.humimm.2010.03.008%2020353805" data-title="IFNG%20%2B874T%2FA%20polymorphism%20and%20cytokine%20plasma%20levels%20are%20associated%20with%20susceptibility%20to%20Mycobacterium%20tuberculosis%20infection%20and%20clinical%20manifestation%20of%20tuberculosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20353805" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=IFNG+%2B874T%2FA+polymorphism+and+cytokine+plasma+levels+are+associated+with+susceptibility+to+Mycobacterium+tuberculosis+infection+and+clinical+manifestation+of+tuberculosis+Vallinoto+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref139"><span class="order">139.
            </span><a name="pone.0175185.ref139" id="pone.0175185.ref139" class="link-target"></a>Elgizouli M, Lowe DM, Speckmann C, Schubert D, Hulsdunker J, Eskandarian Z, et al. Activating PI3Kdelta mutations in a cohort of 669 patients with primary immunodeficiency. Clin Exp Immunol. 2016;183(2):221–9.  pmid:26437962 <ul class="reflinks"><li><a href="#" data-author="Elgizouli" data-cit="ElgizouliM%2C%20LoweDM%2C%20SpeckmannC%2C%20SchubertD%2C%20HulsdunkerJ%2C%20EskandarianZ%2C%20et%20al.%20Activating%20PI3Kdelta%20mutations%20in%20a%20cohort%20of%20669%20patients%20with%20primary%20immunodeficiency.%20Clin%20Exp%20Immunol.%202016%3B183%282%29%3A221%E2%80%939.%20doi%3A%2010.1111%2Fcei.12706%2026437962" data-title="Activating%20PI3Kdelta%20mutations%20in%20a%20cohort%20of%20669%20patients%20with%20primary%20immunodeficiency" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26437962" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Activating+PI3Kdelta+mutations+in+a+cohort+of+669+patients+with+primary+immunodeficiency+Elgizouli+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref140"><span class="order">140.
            </span><a name="pone.0175185.ref140" id="pone.0175185.ref140" class="link-target"></a>Wang G, Wang H, Zhang C, Liu T, Li Q, Lin X, et al. Rac3 regulates cell proliferation through cell cycle pathway and predicts prognosis in lung adenocarcinoma. Tumour Biol. 2016;37(9):12597–607.  pmid:27402308 <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangG%2C%20WangH%2C%20ZhangC%2C%20LiuT%2C%20LiQ%2C%20LinX%2C%20et%20al.%20Rac3%20regulates%20cell%20proliferation%20through%20cell%20cycle%20pathway%20and%20predicts%20prognosis%20in%20lung%20adenocarcinoma.%20Tumour%20Biol.%202016%3B37%289%29%3A12597%E2%80%93607.%20doi%3A%2010.1007%2Fs13277-016-5126-7%2027402308" data-title="Rac3%20regulates%20cell%20proliferation%20through%20cell%20cycle%20pathway%20and%20predicts%20prognosis%20in%20lung%20adenocarcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27402308" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Rac3+regulates+cell+proliferation+through+cell+cycle+pathway+and+predicts+prognosis+in+lung+adenocarcinoma+Wang+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref141"><span class="order">141.
            </span><a name="pone.0175185.ref141" id="pone.0175185.ref141" class="link-target"></a>Haataja L, Kaartinen V, Groffen J, Heisterkamp N. The small GTPase Rac3 interacts with the integrin-binding protein CIB and promotes integrin alpha(IIb)beta(3)-mediated adhesion and spreading. J Biol Chem. 2002;277(10):8321–8.  pmid:11756406 <ul class="reflinks"><li><a href="#" data-author="Haataja" data-cit="HaatajaL%2C%20KaartinenV%2C%20GroffenJ%2C%20HeisterkampN.%20The%20small%20GTPase%20Rac3%20interacts%20with%20the%20integrin-binding%20protein%20CIB%20and%20promotes%20integrin%20alpha%28IIb%29beta%283%29-mediated%20adhesion%20and%20spreading.%20J%20Biol%20Chem.%202002%3B277%2810%29%3A8321%E2%80%938.%20doi%3A%2010.1074%2Fjbc.M105363200%2011756406" data-title="The%20small%20GTPase%20Rac3%20interacts%20with%20the%20integrin-binding%20protein%20CIB%20and%20promotes%20integrin%20alpha%28IIb%29beta%283%29-mediated%20adhesion%20and%20spreading" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11756406" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+small+GTPase+Rac3+interacts+with+the+integrin-binding+protein+CIB+and+promotes+integrin+alpha%28IIb%29beta%283%29-mediated+adhesion+and+spreading+Haataja+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref142"><span class="order">142.
            </span><a name="pone.0175185.ref142" id="pone.0175185.ref142" class="link-target"></a>Mira JP, Benard V, Groffen J, Sanders LC, Knaus UG. Endogenous, hyperactive Rac3 controls proliferation of breast cancer cells by a p21-activated kinase-dependent pathway. Proc Natl Acad Sci U S A. 2000;97(1):185–9. pmid:10618392 <ul class="reflinks"><li><a href="#" data-author="Mira" data-cit="MiraJP%2C%20BenardV%2C%20GroffenJ%2C%20SandersLC%2C%20KnausUG.%20Endogenous%2C%20hyperactive%20Rac3%20controls%20proliferation%20of%20breast%20cancer%20cells%20by%20a%20p21-activated%20kinase-dependent%20pathway.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202000%3B97%281%29%3A185%E2%80%939.%2010618392" data-title="Endogenous%2C%20hyperactive%20Rac3%20controls%20proliferation%20of%20breast%20cancer%20cells%20by%20a%20p21-activated%20kinase-dependent%20pathway" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10618392" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Endogenous%2C+hyperactive+Rac3+controls+proliferation+of+breast+cancer+cells+by+a+p21-activated+kinase-dependent+pathway+Mira+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

