



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Metazoans protect themselves from environmental toxins and virulent pathogens through detoxification and immune responses. We previously identified a small molecule xenobiotic toxin that extends survival of <em>Caenorhabditis elegans</em> infected with human bacterial pathogens by activating the conserved p38 MAP kinase PMK-1 host defense pathway. Here we investigate the cellular mechanisms that couple activation of a detoxification response to innate immunity. From an RNAi screen of 1,420 genes expressed in the <em>C. elegans</em> intestine, we identified the conserved Mediator subunit MDT-15/MED15 and 28 other gene inactivations that abrogate the induction of PMK-1-dependent immune effectors by this small molecule. We demonstrate that MDT-15/MED15 is required for the xenobiotic-induced expression of p38 MAP kinase PMK-1-dependent immune genes and protection from <em>Pseudomonas aeruginosa</em> infection. We also show that MDT-15 controls the induction of detoxification genes and functions to protect the host from bacteria-derived phenazine toxins. These data define a central role for MDT-15/MED15 in the coordination of xenobiotic detoxification and innate immune responses.</p>
</div><div class="abstract toc-section"><a id="abstract1" name="abstract1" class="link-target" data-toc="abstract1" title="Author Summary"></a>
<h2>Author Summary</h2>
<a id="article1.front1.article-meta1.abstract2.p1" name="article1.front1.article-meta1.abstract2.p1" class="link-target"></a><p>Metazoans respond to environmental threats in part through conserved pathways that coordinate protective transcriptional responses. During infection with an invasive pathogen, for example, innate immune pathways regulate the secretion of antimicrobial immune effectors. Likewise, exposure to toxic molecules leads to the induction of detoxification mechanisms that protect the host from the deleterious effects of these compounds. Here we find that a conserved transcriptional regulator MDT-15/MED15 links xenobiotic detoxification and immune responses in a manner that is important for protection during bacterial infection. We also show that MDT-15/MED15 is necessary for the host to resist the lethal effects of secreted toxins produced by pathogenic bacteria. Rapid coordination of these protective host responses through MDT-15/MED15 may therefore be part of a conserved survival strategy in the wild.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Pukkila-Worley R, Feinbaum RL, McEwan DL, Conery AL, Ausubel FM (2014) The Evolutionarily Conserved Mediator Subunit MDT-15/MED15 Links Protective Innate Immune Responses and Xenobiotic Detoxification. PLoS Pathog 10(5):
           e1004143.
        
        https://doi.org/10.1371/journal.ppat.1004143</p><p><strong>Editor: </strong>Robin Charles May, University of Birmingham, United Kingdom</p><p><strong>Received: </strong>November 13, 2013; <strong>Accepted: </strong>April 11, 2014; <strong>Published: </strong> May 29, 2014</p><p><strong>Copyright: </strong> © 2014 Pukkila-Worley et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by the Cancer Research Institute-Irvington Institute Fellowship Program (to RPW) and by National Institutes of Health awards K08 AI081747 (to RPW), F32 AI098307 (to DLM), R01 AI085581 (to FMA), and P01 AI083214 (to FMA). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>In nature, organisms encounter environmental insults, such as chemical toxins, secreted microbial virulence factors and invasive pathogens, that threaten their ability to survive and reproduce. As a result, metazoans have evolved protective pathways to counter these challenges. For example, gene families such as cytochrome P450s (CYPs), glutathione-s-transferases (GSTs), and UDP-glucuronosyltransferases (UDPs) detoxify xenobiotic small molecule toxins and are conserved from nematodes to humans <a href="#ppat.1004143-Lindblom1" class="ref-tip">[1]</a>. Likewise, innate immune defenses provide protection from invasive pathogens <a href="#ppat.1004143-PukkilaWorley1" class="ref-tip">[2]</a>. Recent publications have suggested that recognition of xenobiotic toxins is involved in the activation of immune response pathways <a href="#ppat.1004143-Melo1" class="ref-tip">[3]</a>, <a href="#ppat.1004143-Runkel1" class="ref-tip">[4]</a>. From an evolutionary perspective, it is logical that hosts respond to threats encountered in the wild at least in part through surveillance pathways that monitor the integrity of core cellular machinery, which are often the targets of xenobiotic small molecules or microbe-generated toxins. These studies predict that organisms may integrate detoxification and immune responses as a means to respond rapidly to such challenges, but the mechanisms underlying this coordinated host response have not been reported.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Our research group and others use bacterial and fungal pathogenesis assays in the nematode <em>Caenorhabditis elegans</em> to investigate mechanisms of immune pathway activation in intestinal epithelial cells <a href="#ppat.1004143-PukkilaWorley1" class="ref-tip">[2]</a>. Genetic analyses of <em>C. elegans</em> that are hypersusceptible to bacterial infection have revealed that the nematode mounts defense responses through evolutionarily conserved innate immune pathways. For example, the <em>C. elegans</em> NSY-1/SEK-1/PMK-1 Mitogen Activated Protein (MAP) kinase pathway, orthologous to the ASK1 (MAP kinase kinase kinase)/MKK3/6 (MAP kinase kinase)/p38 (MAP kinase) pathway in mammals, is required for protection against pathogens <a href="#ppat.1004143-Kim1" class="ref-tip">[5]</a>. <em>C. elegans</em> animals carrying loss-of-function mutations in this pathway have defects in the basal and pathogen-induced expression of immune effectors and are hypersusceptible to killing by bacterial and fungal pathogens <a href="#ppat.1004143-Kim1" class="ref-tip">[5]</a>–<a href="#ppat.1004143-PukkilaWorley2" class="ref-tip">[7]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>We previously used a <em>C. elegans</em> pathogenesis assay as a means to identify small molecules that protect the host during bacterial infection <a href="#ppat.1004143-Moy1" class="ref-tip">[8]</a>. One of the compounds identified in this screen, a small molecule called RPW-24, extended the survival of nematodes infected with the human bacterial pathogen <em>Pseudomonas aeruginosa</em> by stimulating the host immune response via the p38 MAP kinase PMK-1 pathway <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. A genome-wide microarray analysis of animals exposed to RPW-24 revealed that, in addition to inducing the transcription of putative immune effectors, this molecule also strongly upregulated Phase I and Phase II detoxification enzymes (CYPs, GSTs and UDPs), suggesting that RPW-24 is a xenobiotic toxin to <em>C. elegans</em>. Consistent with this hypothesis, RPW-24 caused a dose dependent reduction of nematode lifespan on nonpathogenic food and delayed development of animals that were exposed starting at the first larval stage.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Here we sought to use RPW-24 as a tool to characterize mechanisms of p38 MAP kinase PMK-1 pathway activation in <em>C. elegans</em>. We found that activation of PMK-1-regulated pathogen response genes is genetically linked to the induction of genes involved in the detoxification of small molecule toxins. We show that the evolutionarily conserved Mediator subunit MDT-15/MED15 is required for the induction of the p38 MAP kinase PMK-1-mediated immune effectors as well as non-PMK-1-dependent detoxification genes by RPW-24. These data demonstrate that the host response to a xenobiotic involves coordination of detoxification and innate immune responses via the Mediator subunit MDT-15. Moreover, loss of MDT-15 function has important physiological effects on the ability of an animal to mount protective immune responses, resist bacterial infection and survive challenge from lethal bacterial toxins.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="RNAi Screen Identifies Regulators of p38 MAP Kinase PMK-1-Dependent Genes"></a>
<h3>RNAi Screen Identifies Regulators of p38 MAP Kinase PMK-1-Dependent Genes</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>To investigate mechanisms of immune activation in <em>C. elegans</em>, we generated a GFP transcriptional reporter for the immune response gene <em>F08G5.6</em>. <em>F08G5.6</em> is a putative immune effector that contains a CUB-like domain <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a> and is transcriptionally induced by exposure of <em>C. elegans</em> to several bacterial pathogens, including <em>P. aeruginosa</em> <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a>, <a href="#ppat.1004143-Bolz1" class="ref-tip">[10]</a>. We chose <em>F08G5.6</em> for these studies because it is upregulated more than 100-fold by RPW-24 in a manner that requires the p38 MAP kinase PMK-1 <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p><em>pF08G5.6::GFP</em> was induced in the <em>C. elegans</em> intestine during <em>P. aeruginosa</em> infection and GFP expression was also robustly upregulated in <em>pF08G5.6::GFP</em> animals following exposure to RPW-24 when animals were feeding on nonpathogenic <em>E. coli</em> (<a href="#ppat-1004143-g001">Figure 1A</a>). When three components of the p38 MAP kinase PMK-1 signaling cassette were individually knocked down by RNAi [<em>tir-1</em> <a href="#ppat.1004143-Liberati1" class="ref-tip">[11]</a>, <em>pmk-1</em> <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a> and <em>atf-7</em> <a href="#ppat.1004143-Shivers1" class="ref-tip">[12]</a>], <em>pF08G5.6::GFP</em> induction by RPW-24 was entirely abrogated (<a href="#ppat-1004143-g001">Figure 1A</a>).</p>
<a class="link-target" id="ppat-1004143-g001" name="ppat-1004143-g001"></a><div class="figure" data-doi="10.1371/journal.ppat.1004143.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.1004143.g001" data-doi="info:doi/10.1371/journal.ppat.1004143" data-uri="info:doi/10.1371/journal.ppat.1004143.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.1004143.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> RNAi screen identifies a role for the <em>C. elegans</em> Mediator subunit MDT-15 in regulating the induction of the p38 MAP kinase PMK-1-dependent immune reporter <em>pF08G5.6::GFP</em>.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) <em>C. elegans</em> carrying the <em>pF08G5.6::GFP</em> immune reporter were exposed to the vector control (L4440) or the indicated RNAi strain, and then transferred at the L4 stage to either <em>E. coli</em> OP50 food, <em>P. aeruginosa</em> or <em>E. coli</em> OP50 supplemented with RPW-24 for 18 hours. Photographs were acquired using the same imaging conditions. (B) <em>C. elegans mdt-15(RNAi)</em> and <em>mdt-15(tm2182)</em> animals carrying the <em>pF08G5.6::GFP</em> immune reporter with or without <em>agEx114</em> (<em>pmdt-15::mdt-15</em>) were exposed to RPW-24 as described above. In all animals shown in this figure, red pharyngeal expression is the <em>pmyo-2::mCherry</em> co-injection marker, which confirms the presence of the <em>pF08G5.6::GFP</em> transgene. In the <em>mdt-15(tm2182)</em> animals carrying the <em>agEx114</em> array, the presence of the <em>pmdt-15::mdt-15</em> transgene is confirmed by the <em>pmyo-3::mCherry</em> co-injection marker which is expressed in the body wall muscle.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.1004143.g001">
              https://doi.org/10.1371/journal.ppat.1004143.g001</a></p></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3" class="link-target"></a><p>The level of <em>F08G5.6</em> induction during bacterial infection is dependent upon the virulence of the invading pathogen (<a href="#ppat.1004143.s001">Figure S1</a>). We exposed <em>C. elegans</em> to several <em>P. aeruginosa</em> strains, each of which was previously shown to have a different pathogenic potential toward nematodes <a href="#ppat.1004143-Lee1" class="ref-tip">[13]</a> and used qRT-PCR to determine the expression levels of <em>F08G5.6</em> in these animals. In general, more pathogenic <em>P. aeruginosa</em> strains caused significantly greater induction of <em>F08G5.6</em>, suggesting that some aspect of <em>P. aeruginosa</em> virulence, rather than a structural feature of the bacteria itself, causes the activation of <em>F08G5.6</em>. Consistent with these data, it was previously shown that <em>C. elegans</em> primarily responds to virulence-related cues to mount its innate immune defenses towards <em>P. aeruginosa</em> <a href="#ppat.1004143-Estes1" class="ref-tip">[14]</a>–<a href="#ppat.1004143-Dunbar1" class="ref-tip">[16]</a>.</p>
<a id="article1.body1.sec2.sec1.p4" name="article1.body1.sec2.sec1.p4" class="link-target"></a><p>To identify genes that regulate the p38 MAP kinase PMK-1 pathway in response to RPW-24, we screened a library of RNAi clones corresponding to 1,420 genes expressed in <em>C. elegans</em> intestinal epithelium (approximately 9% of the genome, <a href="#ppat.1004143.s007">Table S1B</a>) for gene inactivations that abrogated the RPW-24-mediated induction of <em>pF08G5.6::GFP</em>. We specifically focused on intestinally expressed genes because of the recognized role for intestinal cells in coordinating the host response to ingested pathogens <a href="#ppat.1004143-PukkilaWorley1" class="ref-tip">[2]</a>, <a href="#ppat.1004143-Shivers2" class="ref-tip">[17]</a>, and because <em>P. aeruginosa</em> and RPW-24 induce <em>F08G5.6</em> expression in the intestine (<a href="#ppat-1004143-g001">Figure 1A</a>). Our initial screening effort identified 153 genes that, when inactivated by RNAi, diminished or abrogated the induction of <em>pF08G5.6::GFP</em> by RPW-24.</p>
<a id="article1.body1.sec2.sec1.p5" name="article1.body1.sec2.sec1.p5" class="link-target"></a><p>We took several steps to identify specific regulators of p38 MAP kinase PMK-1-dependent immune effectors among these 153 gene inactivations. First, we noticed that knockdown of many of these genes markedly slowed nematode growth. To eliminate genes that simply reduced GFP reporter expression as a consequence of pleiotropic effects on worm growth and development, we determined if these 153 gene inactivations also affected induction of the <em>C. elegans</em> immune reporter <em>irg-1::GFP</em> <a href="#ppat.1004143-Estes1" class="ref-tip">[14]</a>. <em>irg-1</em> is strongly upregulated in intestinal epithelial cells during <em>P. aeruginosa</em> infection or by an <em>E. coli</em> strain that expresses the bacterial virulence factor Exotoxin A (ToxA), but via a pathway independent of p38 MAP kinase PMK-1 signaling <a href="#ppat.1004143-Estes1" class="ref-tip">[14]</a>–<a href="#ppat.1004143-Dunbar1" class="ref-tip">[16]</a>. Moreover, RPW-24 does not cause the induction of <em>irg-1::GFP</em> and mutation of the <em>zip-2</em> gene, which encodes the transcription factor that regulates <em>irg-1</em> expression, does not affect the RPW-24-mediated induction of <em>F08G5.6</em> (data not shown) or alter the ability of RPW-24 to extend the survival time of nematodes infected with <em>P. aeruginosa</em> <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. We therefore discarded the genes that, when inactivated, reduced the induction of <em>irg-1::GFP</em> by <em>E. coli</em> expressing ToxA, reasoning that they were unlikely to be specific regulators of the p38 MAP kinase PMK-1-dependent pathogen response genes. Using this approach, we selected 56 of the 153 genes for further study.</p>
<a id="article1.body1.sec2.sec1.p6" name="article1.body1.sec2.sec1.p6" class="link-target"></a><p>In a tertiary screen, we determined the effects of these 56 gene inactivations on <em>pF35E12.5::GFP</em>, a second immune reporter that is also strongly induced in the intestine by RPW-24 in a PMK-1-dependent manner <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>, <a href="#ppat.1004143-Bolz1" class="ref-tip">[10]</a>. 29 of the 56 RNAi clones reduced or eliminated the induction of both the <em>pF35E12.5::GFP</em> and <em>pF08G5.6::GFP</em> reporters by RPW-24 (<a href="#ppat.1004143.s007">Table S1A</a>). Validating the screen, the 29 clones we identified as putative regulators of the p38 MAP kinase PMK-1-dependent genes included the three known components of the p38 MAP kinase PMK-1 pathway that were present in the screening library, which suggested that the screen could identify additional, unrecognized components of this signaling pathway.</p>
<a id="article1.body1.sec2.sec1.p7" name="article1.body1.sec2.sec1.p7" class="link-target"></a><p>To confirm further the results of the screen, we used RNAi to knockdown the expression of a representative sample of the 29 genes identified, and tested the induction levels of <em>F08G5.6</em> and <em>F35E12.5</em> by RPW-24 with qRT-PCR (<a href="#ppat.1004143.s007">Table S1A</a>). For all six genes tested, we verified that inactivation of the gene by RNAi dramatically reduced the induction levels of the p38 MAP kinase PMK-1-regulated genes <em>F08G5.6</em> and <em>F35E12.5</em>.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Mediator Subunit MDT-15/MED15 Regulates p38 MAP Kinase PMK-1 Pathway-Dependent Genes"></a>
<h3>Mediator Subunit MDT-15/MED15 Regulates p38 MAP Kinase PMK-1 Pathway-Dependent Genes</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>One of the strongest hits from our RNAi screen was <em>mdt-15</em>, which encodes a subunit of the Mediator complex homologous to mammalian MED15 (78% sequence identity) <a href="#ppat.1004143-Taubert1" class="ref-tip">[18]</a>–<a href="#ppat.1004143-Yang1" class="ref-tip">[20]</a>. Knockdown of <em>mdt-15</em> eliminated all visible expression of the <em>pF08G5.6::GFP</em> and <em>pF35E12.5::GFP</em> reporters, and reduced expression of these genes by at least two orders of magnitude in response to RPW-24 (<a href="#ppat-1004143-g001">Figure 1B</a> and <a href="#ppat.1004143.s007">Table S1</a>). MDT-15 was previously found to regulate the transcription of detoxification genes, including cytochrome P450s, glutathione-s-transferases, and UDP-glucuronosyltransferases <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>, gene classes that are strongly induced by RPW-24 <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. To study the role of <em>mdt-15</em> in the regulation of p38 MAP kinase PMK-1 gene activation, we crossed the <em>pF08G5.6::GFP</em> reporter into the hypomorphic <em>mdt-15(tm2182)</em> allele, which was previously shown to recapitulate many of the phenotypes observed in <em>mdt-15(RNAi)</em> animals <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>, <a href="#ppat.1004143-Goh1" class="ref-tip">[21]</a>. As in <em>mdt-15(RNAi)</em> animals, we observed no induction of GFP when <em>mdt-15(tm2182); pF08G5.6::GFP</em> animals were exposed to RPW-24 (<a href="#ppat-1004143-g001">Figure 1B</a>). An extrachromosomal array containing wild-type <em>mdt-15</em> under its own promoter partially restored RPW-24-induced GFP expression in <em>mdt-15(tm2182);pF08G5.6::GFP</em> animals (<a href="#ppat-1004143-g001">Figure 1B</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>To determine if MDT-15 is required for the induction of other RPW-24-induced genes, we used NanoString nCounter gene expression analysis to generate transcription profiles of 118 <em>C. elegans</em> genes with known involvement in immune, stress and detoxification responses (<a href="#ppat.1004143.s008">Table S2</a>). As in our microarray analysis <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>, we found that RPW-24 caused robust transcriptional changes in wild-type nematodes. 40 of the 118 genes in the NanoString codeset were induced at least 4-fold or greater. 25 of these 40 genes are putative immune effectors upregulated during pathogen infection (shown in <a href="#ppat-1004143-g002">Figure 2</a>) and 13 are genes putatively involved in the detoxification of small molecule toxins (discussed below). Of note, we had previously observed that 31 of these 40 genes, including 28 of the 31 most strongly induced, were also upregulated in whole genome Affymetrix GeneChip microarray analysis of wild-type animals exposed to RPW-24 versus DMSO <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. Of the 25 pathogen-induced genes upregulated by RPW-24, 21 have been shown to be induced during <em>P. aeruginosa</em> infection <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a>, <a href="#ppat.1004143-PukkilaWorley2" class="ref-tip">[7]</a>, <a href="#ppat.1004143-Wong1" class="ref-tip">[22]</a>, <a href="#ppat.1004143-ORourke1" class="ref-tip">[23]</a>. The RPW-24-induced expression of 13 of the 25 putative immune effectors was significantly reduced in <em>pmk-1(km25)</em> loss-of-function mutants compared to wild-type controls (<a href="#ppat-1004143-g002">Figure 2</a> top panel) in accord with the previously determined role for p38 MAP kinase PMK-1 in regulating both pathogen-induced <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a> and RPW-24-induced <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a> expression of putative immune effector genes.</p>
<a class="link-target" id="ppat-1004143-g002" name="ppat-1004143-g002"></a><div class="figure" data-doi="10.1371/journal.ppat.1004143.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.1004143.g002" data-doi="info:doi/10.1371/journal.ppat.1004143" data-uri="info:doi/10.1371/journal.ppat.1004143.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.1004143.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> The Mediator subunit MDT-15 regulates p38 MAP kinase PMK-1-dependent and independent immune genes in response to RPW-24.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>The expression of 118 <em>C. elegans</em> genes was analyzed using NanoString nCounter gene expression analysis in wild-type N2 and <em>pmk-1(km25)</em> animals (top) and in vector control (L4440) and <em>mdt-15(RNAi)</em> animals (bottom) exposed to either 70 µM RPW-24 or the solvent control DMSO. The 40 genes that were induced 4-fold or greater in wild-type N2 animals by RPW-24 are presented in <a href="#ppat-1004143-g002">Figures 2</a> and <a href="#ppat-1004143-g006">6</a>. The 13 genes that were dependent on the p38 MAP kinase PMK-1 for their induction in the top panel are grouped and indicated in the bottom panel. Data are the average of two replicates each of which was normalized to three control genes with error bars representing standard deviation and are presented as the value relative to the average expression from the replicates of the indicated gene in the baseline condition [N2 animals (top) or vector control (L4440) animals (bottom) exposed to DMSO]. We confirmed that <em>mdt-15</em> expression was significantly knocked down by <em>mdt-15(RNAi)</em> in each of these experiments (<em>p</em>&lt;0.001)(see <a href="#ppat.1004143.s002">Figure S2A</a>). * <em>p</em>&lt;0.05 for the comparison of the RPW-24-induced conditions.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.1004143.g002">
              https://doi.org/10.1371/journal.ppat.1004143.g002</a></p></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3" class="link-target"></a><p>Consistent with a role for MDT-15 in the expression of PMK-1 activated genes, the NanoString analysis revealed that the RPW-24-dependent induction of 10 of the 13 PMK-1-dependent genes was abrogated in <em>mdt-15(RNAi)</em> animals compared to controls (<a href="#ppat-1004143-g002">Figure 2</a> bottom panel). We used qRT-PCR to show that <em>mdt-15</em> was knocked down by RNAi in this experiment and to confirm that <em>mdt-15</em> depletion caused a dramatic reduction in the RPW-24-induced expression of three <em>pmk-1</em>-dependent immune genes (<a href="#ppat.1004143.s002">Figure S2A</a>). Further, we found that the RPW-24-mediated induction levels of these three immune genes was reduced in the <em>mdt-15(tm2182)</em> mutant, which recapitulated our findings in <em>mdt-15(RNAi)</em> animals (<a href="#ppat.1004143.s002">Figure S2B</a>).</p>
<a id="article1.body1.sec2.sec2.p4" name="article1.body1.sec2.sec2.p4" class="link-target"></a><p>Knockdown of <em>mdt-15</em> reduced the expression of the top five most strongly upregulated p38 MAP kinase PMK-1-dependent immune effectors by several orders of magnitude (<em>C32H11.1</em>, <em>F35E12.5</em>, <em>F08G5.6</em>, <em>F49F1.7</em>, and <em>F49F1.1</em>) (<a href="#ppat-1004143-g002">Figure 2</a> top panel). These five genes were still induced by RPW-24 in <em>pmk-1(km25)</em> loss-of-function animals, albeit to levels markedly lower than in wild-type animals (<a href="#ppat-1004143-g002">Figure 2</a> top panel), but their induction was entirely abrogated by knockdown of <em>mdt-15</em> (<a href="#ppat-1004143-g002">Figure 2</a> bottom panel). These data suggest that MDT-15 coordinates inputs from PMK-1 and other immune signaling pathway(s) to modulate the expression of these p38 MAP kinase PMK-1-dependent putative immune effectors. We also identified a requirement for <em>mdt-15</em> in the induction of five putative immune effectors (<em>clec-206</em>, <em>C29F7.2</em>, <em>K05B2.4</em>, <em>cdr-1</em>, <em>T28C12.4</em>) that are not transcriptional targets of the PMK-1 pathway (<a href="#ppat-1004143-g002">Figure 2</a>, compare top and bottom panels). Thus, MDT-15 is required for the induction of p38 MAP kinase PMK-1-dependent immune genes and a second group of defense effectors that are independent of the p38 MAP kinase PMK-1 signaling pathway.</p>
<a id="article1.body1.sec2.sec2.p5" name="article1.body1.sec2.sec2.p5" class="link-target"></a><p>The p38 MAP kinase PMK-1 pathway plays an important role in the regulation of putative immune effectors during <em>P. aeruginosa</em> infection <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a>. To determine whether MDT-15/MED15 is also involved in the regulation of these genes during bacterial infection, we infected <em>C. elegans</em> with <em>P. aeruginosa</em> PA14 for 8 hours and used qRT-PCR to compare the induction levels of several immune response genes in <em>mdt-15(RNAi)</em> and control animals. The basal and pathogen-induced expression of three p38 MAP kinase PMK-1-dependent immune effectors (<em>C32H11.1</em>, <em>F08G5.6</em> and <em>F35E12.5</em>) was reduced in <em>mdt-15(RNAi)</em> animals by one to three orders of magnitude (<a href="#ppat-1004143-g003">Figure 3</a>). We also tested the induction levels of three genes whose transcription is activated during <em>P. aeruginosa</em> infection in a manner independent of PMK-1 (<em>irg-1, irg-2</em> and <em>F01D5.5</em>) and found that PMK-1-independent immune effectors were induced in <em>mdt-15(RNAi)</em> animals during <em>P. aeruginosa</em> infection to levels comparable to that observed in wild-type animals (<a href="#ppat-1004143-g003">Figure 3</a>). The expression levels of <em>irg-1</em> and <em>irg-2</em> were higher under basal conditions in <em>mdt-15(RNAi)</em> animals compared to L4440 controls (<a href="#ppat-1004143-g003">Figure 3</a>). Together, these data support the observations from our NanoString experiments that MDT-15 is required for the expression of some, but not all, immune genes that are activated in response to an environmental insult.</p>
<a class="link-target" id="ppat-1004143-g003" name="ppat-1004143-g003"></a><div class="figure" data-doi="10.1371/journal.ppat.1004143.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.1004143.g003" data-doi="info:doi/10.1371/journal.ppat.1004143" data-uri="info:doi/10.1371/journal.ppat.1004143.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.1004143.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> The Mediator subunit MDT-15 regulates the induction of some, but not all, immune genes during <em>P. aeruginosa</em> infection.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1" class="link-target"></a><p>The expression of putative <em>C. elegans</em> immune effectors was analyzed by qRT-PCR in vector control (L4440) and <em>mdt-15(RNAi)</em> animals exposed to <em>P. aeruginosa</em> and the negative control <em>E. coli</em> OP50 for 8 hours. Data are the average of three replicates each normalized to a control gene with error bars representing SEM and are presented as the value relative to the average expression from all three replicates of the indicated gene in the baseline condition (L4440 animals exposed to <em>E. coli</em>). <em>mdt-15</em> expression was significantly knocked down by <em>mdt-15(RNAi)</em> in these experiments (<em>p</em>&lt;0.001).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.1004143.g003">
              https://doi.org/10.1371/journal.ppat.1004143.g003</a></p></div><a id="article1.body1.sec2.sec2.p6" name="article1.body1.sec2.sec2.p6" class="link-target"></a><p>One possible explanation for these observations is that the Mediator subunit MDT-15 is a general regulator of transcription that non-specifically affects the expression of a large number of genes. In the NanoString experiment, however, we identified 10 genes that were induced by RPW-24 in <em>mdt-15(RNAi)</em> animals to levels similar to the wild-type controls (<a href="#ppat-1004143-g002">Figure 2</a> bottom panel). Also of note, our secondary screen demonstrated that <em>mdt-15(RNAi)</em> had no effect on the induction of <em>irg-1::GFP</em> by ToxA. We further wondered if the defects in expression of the PMK-1 targets in <em>mdt-15(RNAi)</em> animals were due to direct transcriptional regulation of the p38 MAP kinase PMK-1 pathway components by MDT-15. However, we found that the mRNA levels of <em>pmk-1</em>, <em>tir-1</em> and <em>sek-1</em> in <em>mdt-15(RNAi)</em> animals were not different from the wild-type control (<a href="#ppat.1004143.s002">Figure S2C</a>).</p>
<a id="article1.body1.sec2.sec2.p7" name="article1.body1.sec2.sec2.p7" class="link-target"></a><p>Taken together, the data in this section indicate that MDT-15 is required for the transcriptional activation of immune effectors controlled by the p38 MAP kinase PMK-1, and at least one other pathway. However, MDT-15 is not required for the induction of all defense-related genes.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="The Mediator Subunit MDT-15 Acts Downstream of the p38 MAP Kinase PMK-1 to Regulate the Induction of F08G5.6 and F35E12.5"></a>
<h3>The Mediator Subunit MDT-15 Acts Downstream of the p38 MAP Kinase PMK-1 to Regulate the Induction of <em>F08G5.6</em> and <em>F35E12.5</em></h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>To determine if MDT-15 acts genetically upstream or downstream of the p38 MAP kinase PMK-1 in coordinating the induction of immune effectors during <em>P. aeruginosa</em> infection, we utilized the MAP kinase phosphatase VHP-1, which is a negative regulator of the p38 MAP kinase PMK-1 <a href="#ppat.1004143-Kim2" class="ref-tip">[24]</a>. Knockdown of <em>vhp-1</em> in animals carrying the <em>F08G5.6::GFP</em> p38 MAP kinase PMK-1 immune reporter caused increased induction of GFP expression during <em>P. aeruginosa</em> infection in a manner dependent on <em>pmk-1</em> expression (<a href="#ppat-1004143-g004">Figure 4A</a>). This induction of <em>F08G5.6::GFP</em> by <em>vhp-1(RNAi)</em> was entirely suppressed in the <em>mdt-15(tm2182)</em> partial loss-of-function allele (<a href="#ppat-1004143-g004">Figure 4A</a>). We used qRT-PCR to confirm this observation (<a href="#ppat-1004143-g004">Figure 4B</a>). We found that RNAi knockdown of <em>vhp-1</em> in wild-type <em>C. elegans</em> caused constitutive activation of <em>F08G5.6</em>, <em>F35E12.5</em> and <em>C32H11.1</em> in a manner that required <em>mdt-15</em> when nematodes were growing on their normal food source, <em>E. coli</em> OP50 (<a href="#ppat-1004143-g004">Figure 4B</a>). During <em>P. aeruginosa</em> infection, we also observed a requirement for MDT-15 in the <em>vhp-1(RNAi)</em>-mediated induction of <em>F08G5.6</em> and <em>F35E12.5</em>, but not <em>C32H11.1</em>. The significance of this discrepancy is unclear, but may be explained by the observations of Taubert et al. who found that only 52% of <em>mdt-15</em> targets were misregulated in the hypomorphic <em>mdt-15(tm2182)</em> allele, which encodes a truncated form of the MDT-15 protein <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>, <a href="#ppat.1004143-Goh1" class="ref-tip">[21]</a>. Indeed, we also observed that <em>mdt-15(RNAi)</em> was more effective than <em>mdt-15(tm2182)</em> at reducing the expression of the six <em>mdt-15</em> targets we studied following exposure to RPW-24, including <em>C32H11.1</em> (<a href="#ppat.1004143.s002">Figures S2A, S2B and S2D</a>). In any case, these data suggest that MDT-15 acts downstream of the p38 MAP kinase PMK-1 to coordinate the induction of at least some immune effector genes.</p>
<a class="link-target" id="ppat-1004143-g004" name="ppat-1004143-g004"></a><div class="figure" data-doi="10.1371/journal.ppat.1004143.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.1004143.g004" data-doi="info:doi/10.1371/journal.ppat.1004143" data-uri="info:doi/10.1371/journal.ppat.1004143.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.1004143.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> The Mediator subunit MDT-15 acts downstream of the p38 MAP kinase PMK-1 to regulate the induction of <em>F08G5.6</em> and <em>F35E12.5</em>.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>(A) Wild-type or <em>mdt-15(tm2182)</em> mutant synchronized L1 animals containing the <em>pF08G5.6::GFP</em> immune reporter were grown on vector control (L4440), <em>vhp-1</em>(RNAi) or a combination of <em>vhp-1(RNAi)</em> and <em>pmk-1(RNAi)</em> bacteria and then transferred as L4 animals to PA14 for 18 hours. Animals were photographed under the same imaging conditions. (B) qRT-PCR was used to examine the expression levels of <em>F08G5.6</em>, <em>F35E12.5</em> and <em>C32H11.1</em> in wild-type N2 and <em>mdt-15(tm2182)</em> mutant animals exposed to <em>vhp-1(RNAi)</em> or the vector control (L4440) under basal conditions (as described above) and 8 hours after exposure to <em>P. aeruginosa</em>. Knockdown of <em>vhp-1</em> caused significant induction of <em>F08G5.6</em> and <em>F35E12.5</em> in wild-type N2 animals (<em>p</em>&lt;0.001), but not in <em>mdt-15(tm2182)</em> animals (<em>p</em>&gt;0.05), under baseline (<em>E. coli</em>) and pathogen-induced conditions. The expression of <em>C32H11.1</em> was significantly induced by <em>vhp-1(RNAi)</em> (<em>p</em>&lt;0.001) in an <em>mdt-15</em>-dependent manner under baseline conditions (<em>p</em>&lt;0.001), but not following exposure to <em>P. aeruginosa</em>. Data are the average of two biological replicates each normalized to a control gene with error bars representing SEM and are presented as the value relative to the average expression of the indicated gene in the baseline condition (L4440 animals exposed to <em>E. coli</em>).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.1004143.g004">
              https://doi.org/10.1371/journal.ppat.1004143.g004</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Mediator Subunit MDT-15/MED15 Is Required for Defense against P. aeruginosa Infection"></a>
<h3>Mediator Subunit MDT-15/MED15 Is Required for Defense against <em>P. aeruginosa</em> Infection</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>The experiments in the preceding sections show that the Mediator subunit MDT-15/MED15 is necessary for the induction of the p38 MAP kinase PMK-1-regulated genes, both in response to RPW-24 and during <em>P. aeruginosa</em> infection. We therefore reasoned that mutation or RNAi-mediated knockdown of <em>mdt-15</em> would result in enhanced susceptibility to <em>P. aeruginosa</em> infection. Initial <em>P. aeruginosa</em> pathogenesis assays showed a modest, but significant and reproducible, enhanced susceptibility to infection in <em>mdt-15(RNAi)</em> (<a href="#ppat.1004143.s003">Figure S3A</a>) and <em>mdt-15(tm2182)</em> (<a href="#ppat.1004143.s003">Figure S3B</a>) animals compared to controls. However, both <em>mdt-15(tm2182)</em> and <em>mdt-15(RNAi)</em> animals have reduced brood sizes and varying degrees of sterility. Sterile animals are more resistant to <em>P. aeruginosa</em> infection than wild-type animals, due in part to <em>daf-16</em>-dependent induction of stress response genes <a href="#ppat.1004143-Miyata1" class="ref-tip">[25]</a>. To eliminate this potentially confounding effect, we made all animals in the <em>P. aeruginosa</em> pathogenesis assay sterile by knocking down <em>cdc-25.1</em>, a technique that has been used previously in <em>C. elegans</em> bacterial pathogenesis assays <a href="#ppat.1004143-Irazoqui1" class="ref-tip">[26]</a>. Under these conditions, we found that <em>mdt-15(tm2182)</em> animals were markedly hypersusceptible to <em>P. aeruginosa</em> infection compared to control animals (<a href="#ppat-1004143-g005">Figure 5</a>). Moreover, injection of the <em>mdt-15</em> gene under control of its own promoter partially rescued the enhanced susceptibility to <em>P. aeruginosa</em> phenotype of <em>mdt-15(tm2182)</em> animals (<a href="#ppat-1004143-g005">Figure 5</a>). Knockdown of <em>mdt-15</em> by RNAi also caused a hypersusceptibility to <em>P. aeruginosa</em> phenotype in <em>C. elegans fer-15(b26);fem-1(hc17)</em> sterile animals <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a>, <a href="#ppat.1004143-Feinbaum1" class="ref-tip">[27]</a>, and as predicted, to a greater degree than wild-type animals that were not made sterile by <em>cdc-25.1(RNAi)</em> (<a href="#ppat.1004143.s003">Figure S3C</a>).</p>
<a class="link-target" id="ppat-1004143-g005" name="ppat-1004143-g005"></a><div class="figure" data-doi="10.1371/journal.ppat.1004143.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.1004143.g005" data-doi="info:doi/10.1371/journal.ppat.1004143" data-uri="info:doi/10.1371/journal.ppat.1004143.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.1004143.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> MDT-15 is required for defense against <em>P. aeruginosa</em> infection.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p>A <em>P. aeruginosa</em> pathogenesis assay with wild-type N2, <em>mdt-15(tm2182)</em> mutant worms and <em>mdt-15(tm2182)</em> animals carrying <em>pmdt-15::mdt-15</em> (three independent lines <em>agEx116</em>, <em>agEx117</em> and <em>agEx118</em>) is shown. The difference in <em>P. aeruginosa</em> susceptibility between <em>mdt-15(tm2182)</em> animals and each of the three transgenic lines carrying <em>pmdt-15::mdt-15</em> is significant, as is the survival difference between N2 and <em>mdt-15(tm2182)</em> animals (<em>p</em>&lt;0.001). For sample sizes, see <a href="#ppat.1004143.s009">Table S3</a>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.1004143.g005">
              https://doi.org/10.1371/journal.ppat.1004143.g005</a></p></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>We also found that the ability of RPW-24 to extend survival of <em>P. aeruginosa</em>-infected wild-type and sterile nematodes was significantly attenuated in <em>mdt-15(RNAi)</em> animals compared to controls, suggesting that MDT-15 is required for the immunostimulatory activity of RPW-24 (<a href="#ppat.1004143.s003">Figure S3A and S3C</a>). The degree of lifespan extension by RPW-24 during <em>P. aeruginosa</em> infection was reduced in <em>mdt-15(tm2182)</em> compared to controls, but this difference did not reach statistical significance (<em>p</em> = 0.09)(<a href="#ppat.1004143.s003">Figure S3B</a>). As discussed above, the gene expression defects of <em>mdt-15</em> targets were more severe in <em>mdt-15(RNAi)</em> animals than in the hypomorphic <em>mdt-15(tm2182)</em> allele <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>, <a href="#ppat.1004143-Goh1" class="ref-tip">[21]</a>, which may account for this observation.</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3" class="link-target"></a><p>One caveat concerning the observation that <em>mdt-15</em> depleted animals are hypersusceptible to <em>P. aeruginosa</em> infection is that <em>mdt-15(RNAi)</em> and <em>mdt-15(tm2182)</em> animals have a reduced lifespan when grown on the normal laboratory food source <em>E. coli</em> OP50 compared to wild-type controls <a href="#ppat.1004143-Taubert1" class="ref-tip">[18]</a>, <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>. Several observations indicate, however, that MDT-15 is an important modulator of nematode survival during bacterial infection. First, we have shown above that <em>mdt-15(RNAi)</em> animals fail to upregulate p38 MAP kinase PMK-1-dependent immune effectors both in response to a xenobiotic toxin and during <em>P. aeruginosa</em> infection, but retain the ability to induce other immune genes in response to pathogens and following exposure to the bacterial toxin ToxA. In addition, <em>mdt-15(RNAi)</em> animals do not respond to the immunostimulatory effects of RPW-24. We have shown previously that <em>C. elegans</em> with mutations in the ZIP-1 and FSHR-1 immune pathways, which act in parallel to the p38 MAP kinase PMK-1 cassette, are hypersusceptible to <em>P. aeruginosa</em> infection, but retain the ability to respond to RPW-24 <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. That <em>mdt-15(RNAi)</em> animals are blind to the immunostimulatory effects of RPW-24 suggests a specific role of MDT-15 in regulating p38 MAP kinase PMK-1 pathway activity. It is also important to note that despite their reduced lifespan, <em>mdt-15(RNAi)</em> animals are not sensitive to all environmental insults. For example, animals deficient in <em>mdt-15</em> are sensitive to the toxin fluoranthene, but not β-naphthoflavone, and are not more sensitive to high temperatures than wild-type animals <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Mediator Subunit MDT-15/MED15 Regulates the Induction of p38 MAP Kinase PMK-1-Independent Detoxification Genes and Is Required to Resist the Toxic Effects of the Xenobiotic RPW-24"></a>
<h3>Mediator Subunit MDT-15/MED15 Regulates the Induction of p38 MAP Kinase PMK-1-Independent Detoxification Genes and Is Required to Resist the Toxic Effects of the Xenobiotic RPW-24</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>We previously demonstrated that RPW-24 is a xenobiotic toxin <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. MDT-15 is known to coordinate protection from the toxin fluoranthene and regulate the transcriptional induction of CYPs <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>. We found that the RPW-24-mediated induction of the 13 detoxification genes in the NanoString codeset was nearly entirely abrogated by RNAi knockdown of <em>mdt-15</em> (<a href="#ppat-1004143-g006">Figure 6A</a> top panel and <a href="#ppat.1004143.s008">Table S2</a>). This result was confirmed for three detoxification genes by qRT-PCR (<a href="#ppat.1004143.s002">Figures S2B and S2D</a>). To determine if MDT-15 is required to protect <em>C. elegans</em> from the toxic effects of RPW-24, we studied the development of wild-type, <em>pmk-1(RNAi)</em> and <em>mdt-15(RNAi)</em> in the presence of the xenobiotic RPW-24, an assay that has been used previously to assess the toxicity of small molecules <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>. We found that RPW-24 slowed the development of control and <em>pmk-1(RNAi)</em> animals to similar degree, and that <em>mdt-15(RNAi)</em> animals were markedly delayed in the presence of RPW-24 (<a href="#ppat-1004143-g006">Figures 6B</a> and <a href="#ppat.1004143.s004">S4</a>). These data show that MDT-15 controls the induction of detoxification genes following exposure to RPW-24 and is required to resist the toxic effects of this xenobiotic.</p>
<a class="link-target" id="ppat-1004143-g006" name="ppat-1004143-g006"></a><div class="figure" data-doi="10.1371/journal.ppat.1004143.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.1004143.g006" data-doi="info:doi/10.1371/journal.ppat.1004143" data-uri="info:doi/10.1371/journal.ppat.1004143.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.1004143.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 6. </span> Protection from the toxic effects of the xenobiotic RPW-24 requires MDT-15, but not PMK-1.</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>(A) The thirteen xenobiotic detoxification genes that were induced 4-fold or greater by RPW-24 in the NanoString nCounter gene expression analysis are presented. The top panel compares the RPW-24-mediated induction of these genes in vector control (L4440) and <em>mdt-15(RNAi)</em> animals, and the bottom panel shows these data for wild-type N2 versus <em>pmk-1(km25)</em> animals, as described in the legend for <a href="#ppat-1004143-g002">Figure 2</a>. * <em>p</em>&lt;0.05 for the comparison of the RPW-24-induced conditions. (B) Vector control (L4440), <em>mdt-15(RNAi)</em> and <em>pmk-1(RNAi)</em> animals were exposed to 70 µM RPW-24 or the solvent control DMSO from the L1 stage and photographed after 70 hours of development at 20°C. See <a href="#ppat.1004143.s004">Figure S4</a> for the quantification data from this experiment.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.1004143.g006">
              https://doi.org/10.1371/journal.ppat.1004143.g006</a></p></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>Given that MDT-15 is required for the regulation of detoxification genes and the p38 MAP kinase PMK-1 pathway, we wondered if the detoxification machinery in <em>C. elegans</em> is regulated by the p38 MAP kinase PMK-1. We found, however, that 12 of the 13 RPW-24-induced detoxification genes were upregulated in <em>pmk-1(km25)</em> null mutants to levels comparable to that in wild-type animals exposed to RPW-24 (<a href="#ppat-1004143-g006">Figure 6A</a> bottom panel). We used qRT-PCR to confirm this observation for three cytochrome P450 genes (<a href="#ppat.1004143.s002">Figure S2D</a>). Thus, the MDT-15-dependent xenobiotic detoxification program is induced in a manner independent of the p38 MAP kinase PMK-1.</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="MDT-15/MED15 Is Not Required for the Avoidance Behavior Induced by RPW-24"></a>
<h3>MDT-15/MED15 Is Not Required for the Avoidance Behavior Induced by RPW-24</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>Many xenobiotic toxins, including RPW-24, induce an avoidance response wherein <em>C. elegans</em> leave a lawn of bacterial food, to which they are otherwise attracted, if it contains a toxic compound <a href="#ppat.1004143-Melo1" class="ref-tip">[3]</a>, <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. We therefore wondered if MDT-15 is required for the avoidance behavior induced by RPW-24. However, <em>pmk-1(km25)</em> and <em>mdt-15(tm2182)</em> animals left the lawn of <em>E. coli</em> containing RPW-24 as readily as wild-type animals (<a href="#ppat.1004143.s005">Figure S5</a>). We also observed a similar phenotype when we knocked down the expression of <em>pmk-1</em> and <em>mdt-15</em> in the neuronally-sensitive RNAi strain TU3311 (data not shown). Thus, animals lacking the function of MDT-15 and PMK-1 are still able to recognize RPW-24 as a toxin.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2" class="link-target"></a><p>In summary, the data in this and the preceding section suggest that MDT-15, but not PMK-1, controls the induction of genes that are required to resist the toxic effects of RPW-24, and that neither are required for avoidance of RPW-24.</p>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="Mediator Subunit MDT-15/MED15 Promotes Resistance to Secreted P. aeruginosa Phenazine Toxins"></a>
<h3>Mediator Subunit MDT-15/MED15 Promotes Resistance to Secreted <em>P. aeruginosa</em> Phenazine Toxins</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>We have shown that MDT-15 regulates the <em>C. elegans</em> detoxification response to the xenobiotic RPW-24. We therefore hypothesized that MDT-15 might also be required for protection from lethal secreted toxins produced by pathogenic bacteria. To address this question, we used an assay that allows the specific study of <em>C. elegans</em> killing by secreted low molecular weight toxins of <em>P. aeruginosa</em>. When <em>P. aeruginosa</em> is grown on high osmolarity media, phenazine toxins produced by the bacteria are lethal to nematodes <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a>, <a href="#ppat.1004143-MahajanMiklos1" class="ref-tip">[29]</a>. In contrast to the “slow killing” infection assay, which was used in the assays described above, wild-type nematodes exposed to these “fast killing” conditions die within a few hours via a process that does not require live bacteria <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a>, <a href="#ppat.1004143-MahajanMiklos1" class="ref-tip">[29]</a>. Wild-type <em>C. elegans</em> at the fourth larval stage of development, but not young adult animals, are exquisitely sensitive to the phenazine toxins produced by <em>P. aeruginosa</em> in this assay and are killed within 6 hours of exposure. We took advantage of the inherent resistance of young adult animals to test whether <em>mdt-15(tm2182)</em> animals are hypersusceptible to the phenazine toxins produced by <em>P. aeruginosa</em>. As expected, both wild-type and <em>mdt-15(tm2182)</em> L4 animals were rapidly killed by <em>P. aeruginosa</em> in the “fast kill” assay (<a href="#ppat.1004143.s006">Figure S6A</a>). <em>P. aeruginosa</em> carrying deletions of both phenazine biosynthetic operons (<em>Δphz</em>) does not make these toxins and accordingly had reduced ability to kill both wild-type and <em>mdt-15(tm2182)</em> animals (<a href="#ppat.1004143.s006">Figure S6A</a>). In contrast to L4 animals, we observed that almost no young adult, wild-type animals were killed after six hours of exposure to the phenazine toxins compared with 98% death of L4 staged animals (<a href="#ppat-1004143-g007">Figures 7A</a> and <a href="#ppat.1004143.s006">S6A</a>), which reproduces the findings of others <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a>, <a href="#ppat.1004143-MahajanMiklos1" class="ref-tip">[29]</a>. In contrast to wild-type animals, young adult <em>mdt-15(tm2182)</em> animals were dramatically susceptible to <em>P. aeruginosa</em> in this assay and this pathogenesis required the secretion of phenazine toxins, as <em>P. aeruginosa Δphz</em> was markedly less pathogenic toward <em>mdt-15(tm2182)</em> young adults (<a href="#ppat-1004143-g007">Figure 7A</a>). Moreover, <em>mdt-15(tm2182)</em> animals were not simply hypersusceptible to the high osmolarity conditions of this assay because we observed no mortality over the course of the assay in <em>mdt-15(tm2182)</em> mutants exposed to “fast kill” media containing the normal nematode food source <em>E. coli</em> OP50 (<a href="#ppat-1004143-g007">Figure 7A</a>).</p>
<a class="link-target" id="ppat-1004143-g007" name="ppat-1004143-g007"></a><div class="figure" data-doi="10.1371/journal.ppat.1004143.g007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.1004143.g007" data-doi="info:doi/10.1371/journal.ppat.1004143" data-uri="info:doi/10.1371/journal.ppat.1004143.g007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.1004143.g007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.1004143.g007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.1004143.g007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.1004143.g007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 7. </span> Resistance to <em>P. aeruginosa</em> phenazine toxins requires MDT-15.</div><p class="caption_target"><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1" class="link-target"></a><p>(A) Wild-type N2, <em>mdt-15(tm2182)</em> and <em>pmk-1(km25)</em> animals were tested in the toxic phenazine <em>P. aeruginosa</em> pathogenesis assay (also called “fast killing” assay). <em>C. elegans</em> young adult animals were exposed to wild-type (WT) <em>P. aeruginosa</em>, <em>P. aeruginosa</em> carrying a deletion in both of the phenazine biosynthetic operons (<em>Δphz</em>) or <em>E. coli</em> OP50. The difference between <em>mdt-15(tm2182)</em> and N2 animals exposed to <em>P. aeruginosa</em> WT is significant (<em>p</em>&lt;0.001), as is the difference in <em>mdt-15(tm2182)</em> animals exposed to <em>P. aeruginosa</em> WT and <em>P. aeruginosa Δphz (p</em>&lt;0.001). There is no significant difference in <em>pmk-1(km25)</em> animals exposed to <em>P. aeruginosa</em> WT and <em>P. aeruginosa Δphz</em> (<em>p</em>&gt;0.05). This assay is representative of two independent experiments. <a href="#ppat.1004143.s006">Figure S6A</a> presents the control condition for this experiment showing that both <em>C. elegans</em> N2 and <em>mdt-15(tm2182)</em> at the L4 stage are sensitive to toxic phenazine-mediated killing (<em>p</em>&lt;0.001). (B) Vector control (L4440), <em>mdt-15(RNAi)</em> and <em>pmk-1(RNAi)</em> animals were exposed to high osmolarity “fast kill” media (pH 5) with <em>E. coli</em> as the food source in the presence or absence of phenazine-1-carboxylic acid and 1-hydroxyphenazine (labeled “toxic phenazines” above) at the approximate concentrations produced by <em>P. aeruginosa</em> under standard assay conditions. The difference in killing between <em>mdt-15(RNAi)</em> animals exposed to toxic phenazines and the other conditions is significant (<em>p</em>&lt;0.001). See also <a href="#ppat.1004143.s006">Figure S6</a>. For sample sizes, see <a href="#ppat.1004143.s009">Table S3</a>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.1004143.g007">
              https://doi.org/10.1371/journal.ppat.1004143.g007</a></p></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2" class="link-target"></a><p>We confirmed that phenazine toxins are lethal to <em>mdt-15</em>-depleted animals by supplementing “fast kill” growth media with both phenazine-1-carboxylic acid and 1-hydroxyphenazine in the absence of pathogen <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a>. These two particular phenazines are toxic to wild-type nematodes <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a> and, as expected, the mixture of phenazine-1-carboxylic acid and 1-hydroxyphenazine rapidly killed L4 animals (<a href="#ppat.1004143.s006">Figure S6B</a>). Young adult wild-type animals were resistant to the lethal effects of these molecules (<a href="#ppat-1004143-g007">Figure 7B</a>). However, <em>mdt-15(RNAi)</em> young adult animals were dramatically susceptible to phenazine-mediated killing (<a href="#ppat-1004143-g007">Figure 7B</a>).</p>
<a id="article1.body1.sec2.sec7.p3" name="article1.body1.sec2.sec7.p3" class="link-target"></a><p>We also found that <em>pmk-1(km25)</em> loss-of-function mutants were more susceptible to <em>P. aeruginosa</em> in the “fast killing” assay than wild-type animals, although were less susceptible than <em>mdt-15(tm2182)</em> animals. This lethality, however, was mediated by factors other than phenazine toxins, since there was no difference in pathogenicity of the wild-type and <em>Δphz P. aeruginosa</em> toward <em>pmk-1(km25)</em> mutants (<a href="#ppat-1004143-g007">Figure 7A</a>). Likewise, <em>pmk-1(RNAi)</em> animals were not more susceptible to phenazines-1-carboxylic acid and 1-hydroxyphenazine than L4440 RNAi control animals (<a href="#ppat-1004143-g007">Figure 7B</a>).</p>
<a id="article1.body1.sec2.sec7.p4" name="article1.body1.sec2.sec7.p4" class="link-target"></a><p>Together, these data define a role for MDT-15, but not PMK-1, in the protection from bacterial-derived phenazine toxins.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>Detecting and countering environmental threats is central to the ability of organisms to survive and reproduce in the wild. We examined the <em>C. elegans</em> response to the xenobiotic RPW-24, which is able to induce a host immune response that is protective for animals infected with the lethal bacterial pathogen <em>P. aeruginosa</em> <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. In an RNAi screen with RPW-24, we identified a number of genes, including <em>mdt-15/MED15</em>, which are required for induction of p38 MAP kinase PMK-1-dependent immune effectors. <em>mdt-15</em> encodes a subunit of the highly conserved Mediator complex that controls the activation of a variety of genes involved in the response to external stress. We demonstrate that: <em>(i)</em> MDT-15 is required for the induction of p38 MAP kinase PMK-1-dependent immune effectors following exposure to a xenobiotic toxin, as well as during infection with <em>P. aeruginosa</em>, <em>(ii)</em> MDT-15 controls the expression of some p38 MAP kinase PMK-1-independent immune effectors, but not all defense genes, (<em>iii</em>), MDT-15 functions downstream of the p38 MAP kinase PMK-1 cascade to control the induction of at least two immune effectors, <em>(iv)</em> the induction of xenobiotic detoxification genes and protection from the toxic effects of RPW-24 requires MDT-15, but not the p38 MAP kinase PMK-1, and (<em>iv</em>) MDT-15 is necessary for protection during <em>P. aeruginosa</em> infection and from phenazine toxins secreted by this organism.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>The Mediator complex is strongly conserved from yeasts to humans, and is required for transcription by physically interacting with and directing the activity of RNA polymerase II <a href="#ppat.1004143-Conaway1" class="ref-tip">[30]</a>, <a href="#ppat.1004143-Malik1" class="ref-tip">[31]</a>. Although initial studies from yeast described the Mediator complex as a general regulator of transcription <a href="#ppat.1004143-Holstege1" class="ref-tip">[32]</a>, it is now becoming clear that the individual subunits of this complex, of which there are at least 26, play important roles in translating inputs from cell signaling pathways to specific outputs <a href="#ppat.1004143-Conaway1" class="ref-tip">[30]</a>, <a href="#ppat.1004143-Malik1" class="ref-tip">[31]</a>, <a href="#ppat.1004143-Huang1" class="ref-tip">[33]</a>. For example, the regulation of chemotherapeutic resistance in cancer cells requires MED12 <a href="#ppat.1004143-Huang1" class="ref-tip">[33]</a>, MED23 channels MAP kinase signaling activity to coordinate cell growth <a href="#ppat.1004143-Boyer1" class="ref-tip">[34]</a>, and MED15 serves as a regulatory node in lipid metabolism by directing the activity of the transcriptional activator SREBP (sterol regulatory element binding protein) <a href="#ppat.1004143-Yang1" class="ref-tip">[20]</a>. Interestingly, MED15's role as a lipid sensor is strongly conserved. In <em>C. elegans</em>, the MED15 homolog MDT-15 interacts with SBP-1, the SREBP homolog, and serves as an important regulator of lipid metabolism <a href="#ppat.1004143-Taubert1" class="ref-tip">[18]</a>–<a href="#ppat.1004143-Yang1" class="ref-tip">[20]</a>. Likewise, the MED15 homolog in yeast (called GAL11p) serves a similar function <a href="#ppat.1004143-Thakur1" class="ref-tip">[35]</a>.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>Taubert et al. showed in a <em>C. elegans</em> genome-wide microarray analysis that MDT-15 coordinates the transcription of Phase I and Phase II detoxification genes, and is required to resist the lethal effects of a xenobiotic toxin <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a> and agents that induce oxidative stress <a href="#ppat.1004143-Goh1" class="ref-tip">[21]</a>. Consistent with these observations, we found that MDT-15 is necessary to mount a detoxification response toward and provide protection from RPW-24, a xenobiotic toxin. Our studies extend the known functions of this conserved Mediator subunit to innate immunity in demonstrating that MDT-15 plays a critical role in controlling immune and detoxification pathway activation in a manner that is important for protection from bacterial infection and from the secreted phenazine toxins of <em>P. aeruginosa</em>.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>We propose that the Mediator subunit MDT-15 links innate immune activation to xenobiotic detoxification responses as part of a general strategy to ensure survival in harsh environments. In their natural habits, nematodes encounter numerous threats from ingested pathogens and xenobiotic toxins <a href="#ppat.1004143-Lindblom1" class="ref-tip">[1]</a>. Thus, coupling xenobiotic detoxification to innate immune activation may have been selected for by pathogens that secrete soluble toxins during infection. Whether the protection from phenazine toxins mediated by MDT-15 requires detoxification genes or occurs via another mechanism is not known, but it is interesting to note that MDT-15's function as a regulator of host protection may be evolutionarily conserved. In yeasts, the MDT-15 homolog Gal11p coordinates a protective cellular response following xenobiotic exposure, which is manifest as the upregulation of drug efflux pumps <a href="#ppat.1004143-Thakur2" class="ref-tip">[36]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>The data presented in this study imply the existence of surveillance mechanisms that monitor for xenobiotic toxins (or their effects) and signal through MDT-15/MED15 to simultaneously activate protective detoxification and innate immune responses. Whether such activation occurs in the context of xenobiotic-induced organelle dysfunction is not known. McEwan et al. <a href="#ppat.1004143-McEwan1" class="ref-tip">[15]</a> and Dunbar et al. <a href="#ppat.1004143-Dunbar1" class="ref-tip">[16]</a> found that <em>C. elegans</em> monitor the integrity of its translational machinery as a means to detect pathogen invasion. Inhibition of translation by the secreted bacterial toxin ToxA leads to an antibacterial response via known immune pathways involving the transcription factor ZIP-2 and the p38 MAP kinase PMK-1. Several observations suggest, however, that MDT-15-mediated regulation of immune activation and detoxification gene induction by RPW-24 does not occur via a mechanism involving translation inhibition. First, we previously found that a loss-of-function mutation in the <em>zip-2</em> gene did not affect the ability of RPW-24 to extend the survival of animals infected with <em>P. aeruginosa</em> and genome-wide transcriptional analysis of animals exposed to RPW-24 did not suggest that this compound is an inhibitor of translation <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. Second, the transcriptome analysis of animals exposed to ToxA did not show an abundance of Phase I and II detoxification genes <a href="#ppat.1004143-McEwan1" class="ref-tip">[15]</a>. Finally, we show here that the ToxA-responsive, ZIP-2-regulated genes, <em>irg-1</em> and <em>irg-2</em>, are induced in <em>mdt-15(RNAi)</em> animals during <em>P. aeruginosa</em> infection to levels comparable to that observed in wild-type animals.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6" class="link-target"></a><p>Melo et al. recently studied the behavioral response of <em>C. elegans</em> to xenobiotic toxins known to disrupt the function of the mitochondria, the ribosome, and the endoplasmic reticulum <a href="#ppat.1004143-Melo1" class="ref-tip">[3]</a>. Inhibiting these essential processes by the action of small molecules or through targeted gene disruptions triggered an avoidance response, which required serotonergic and JNK signaling pathways. They proposed that organisms monitor disruption of core metabolic processes as a means to detect pathogen invasion and challenges from xenobiotic toxins. Based on the data reported here, however, it is not clear, whether immune signaling is an integral part of xenobiotic-elicited avoidance behavior, at least with respect to RPW-24. We found that the function of neither MDT-15 nor PMK-1 was required for the avoidance of RPW-24, indicating that the behavioral component of this protective response occurs upstream of MDT-15, or via separate mechanism altogether.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7" class="link-target"></a><p>It will be interesting to determine the mechanism by which MDT-15 activates immune and detoxification responses in <em>C. elegans</em>. In our genetic screen for regulators of the p38 MAP kinase PMK-1 pathway, we identified a number of genes that are involved in fatty acid biosynthesis, including <em>mdt-15, fat-6, fat-7, elo-5, acs-19</em> and <em>C25A1.5</em>. Indeed, MDT-15 is known to control the expression of <em>fat-6</em> and <em>fat-7</em> <a href="#ppat.1004143-Taubert1" class="ref-tip">[18]</a>–<a href="#ppat.1004143-Yang1" class="ref-tip">[20]</a>. It is therefore possible that a fatty acid signaling molecule or membrane component is required for p38 MAP kinase PMK-1 activity. We found, however, in epistasis analyses with the MAP kinase phosphatase VHP-1 that MDT-15 functions downstream of PMK-1 to coordinate the expression of <em>F08G5.6</em> and <em>F35E12.5</em>. Thus, for at least a subset of immune genes, MDT-15 likely also physically interacts with sequence-specific regulators, such as ATF-7, a transcription factor that is the downstream signaling target of the p38 MAP kinase PMK-1 pathway <a href="#ppat.1004143-Shivers1" class="ref-tip">[12]</a>, to coordinate protective host responses mounted following exposure to xenobiotic toxins.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="C. elegans and Bacterial Strains"></a>
<h3><em>C. elegans</em> and Bacterial Strains</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p><em>C. elegans</em> were grown on standard NGM plates with <em>E. coli</em> OP50 <a href="#ppat.1004143-Brenner1" class="ref-tip">[37]</a> unless otherwise noted. The previously published <em>C. elegans</em> strains used in this study were: N2 Bristol <a href="#ppat.1004143-Brenner1" class="ref-tip">[37]</a>, <em>pmk-1(km25)</em> <a href="#ppat.1004143-Kim1" class="ref-tip">[5]</a>, AY101 [<em>acIs101</em>[<em>pDB09.1(pF35E12.5::GFP</em>); pRF4(<em>rol-6(su1006))</em>] <a href="#ppat.1004143-Bolz1" class="ref-tip">[10]</a>, XA7702 <em>mdt-15(tm2182)</em> <a href="#ppat.1004143-Taubert2" class="ref-tip">[19]</a>, <a href="#ppat.1004143-Goh1" class="ref-tip">[21]</a>, CF512 <em>fer-15(b26);fem-1(hc17)</em> <a href="#ppat.1004143-Garigan1" class="ref-tip">[38]</a>, and AU0133 [<em>agIs17</em>(<em>pirg-1::GFP</em>; <em>pmyo-2::mCherry)</em>] <a href="#ppat.1004143-Estes1" class="ref-tip">[14]</a>. The <em>C. elegans</em> strains created for this study were: AU0307 [<em>agIs44(pF08G5.6::GFP::unc-54-3′UTR; pmyo-2::mCherry)</em>], AU0316 [<em>mdt-15(tm2182); agIs44</em>], AU0325 [<em>mdt-15(tm2182); agEx116 (mdt-15;pmyo-3::mCherry</em>)], AU0326 [<em>mdt-15(tm2182); agEx117 (mdt-15;pmyo-3::mCherry)</em>], AU0327 [<em>mdt-15(tm2182); agEx118 (mdt-15;pmyo-3::mCherry)</em>] and AU0323 [<em>mdt-15(tm2182); agIs44; agEx114 (mdt-15;pmyo-3::mCherry)</em>].</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2" class="link-target"></a><p>The strain carrying <em>agIs44</em> was constructed by PCR amplification from N2 genomic DNA of an 851 bp region upstream of the start codon of the <em>F08G5.6</em> gene (primers <span class="gene">GACTTGTCAAATGAACAATTTTATCAAATCTCA</span> and <span class="gene">CGCCTAGGTGTCAATTGATAATGAATA</span>) and ligated to the GFP coding region and <em>unc-54-3′UTR</em> sequences amplified from pPD95.75 using published primers, and a previously described protocol <a href="#ppat.1004143-McEwan2" class="ref-tip">[39]</a>. The <em>agIs44</em> construct was transformed into N2 animals with the co-injection marker <em>pmyo-2::mCherry</em> using established methods <a href="#ppat.1004143-Mello1" class="ref-tip">[40]</a>. A strain carrying the <em>pF08G5.6::GFP::unc-54-3′UTR</em> and <em>pmyo-2::mCherry</em> transgenes in an extrachromosomal array was irradiated, and strains carrying the integrated array <em>agIs44</em> were isolated. AU0307 was backcrossed to N2 five times.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3" class="link-target"></a><p>The <em>mdt-15</em> rescuing arrays agEx116, agEx117 and agEx118 contain a 4.8 kb <em>mdt-15</em> genomic fragment, which includes 707 bp upstream and 1075 bp downstream of the <em>mdt-15</em> coding region, amplified from N2 genomic DNA (primers <span class="gene">GGAGTATCAGAAGCTCACGATGCTC</span> and <span class="gene">CCAAATAATACTAACCACCACATATCTTCCATT</span>). This <em>mdt-15</em> genomic fragment was transformed into N2 animals or AU0316 with the co-injection marker <em>pmyo-3::mCherry</em> using established methods.</p>
<a id="article1.body1.sec4.sec1.p4" name="article1.body1.sec4.sec1.p4" class="link-target"></a><p>RNAi clones presented in this study were from the Ahringer <a href="#ppat.1004143-Kamath1" class="ref-tip">[41]</a> or Vidal <a href="#ppat.1004143-Rual1" class="ref-tip">[42]</a> RNAi libraries unless otherwise stated. The <em>atf-7</em> <a href="#ppat.1004143-Shivers1" class="ref-tip">[12]</a> and the <em>pmk-1</em> <a href="#ppat.1004143-Kim1" class="ref-tip">[5]</a> RNAi clones have been previously reported. All RNAi clones presented in this study have been confirmed by sequencing. The <em>P. aeruginosa</em> strain PA14 were used for all studies, unless otherwise indicated. The <em>P. aeruginosa</em> strains used in <a href="#ppat.1004143.s001">Figure S1</a> have been previously described <a href="#ppat.1004143-Lee1" class="ref-tip">[13]</a> and were (in order of descending virulence toward <em>C. elegans</em>): CF18, PA14, MSH10, S54485, PA01, PAK, 19660, and E2. The <em>P. aeruginosa</em> PA14 phenazine null mutant (<em>Δphz</em>) lacks both the <em>phzA1-G1</em> and <em>phzA2-G2</em> operons and has been previously described <a href="#ppat.1004143-Dietrich1" class="ref-tip">[43]</a>. The BL21 <em>E. coli</em> strain that expresses the bacterial toxin Exotoxin A (ToxA) has been previously described <a href="#ppat.1004143-McEwan1" class="ref-tip">[15]</a>.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="Feeding RNAi Screen"></a>
<h3>Feeding RNAi Screen</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>1,420 RNAi clones that correspond to genes expressed in the <em>C. elegans</em> intestine based on their annotation in Wormbase (<a href="http://www.wormbase.org">www.wormbase.org</a>) in April, 2008 were selected from the Ahringer <a href="#ppat.1004143-Kamath1" class="ref-tip">[41]</a> or Vidal <a href="#ppat.1004143-Rual1" class="ref-tip">[42]</a> RNAi libraries (see <a href="#ppat.1004143.s007">Table S1B</a>). RNAi clones were pinned into 1.2 ml of LB plus 100 µg/ml carbenicillin in 96-well culture blocks (Corning Incorporated) and grown overnight at 37°C with shaking at 950 RPM in a Multitron II Shaking Incubator (Appropriate Technical Resources). 40 µL of the 10× concentrated overnight culture were added to each well of a 24-well plate containing RNAi agar medium and grown overnight at room temperature. The following day, 50–100 L1 staged AU0307 animals, which carry the <em>agIs44</em> transgene, were added to each well and allowed to grow until they were at the L4 or young adult stage. Worms were then transferred to new 24-well screening plates containing 1 mL of “slow kill” media supplemented with 70 µM RPW-24 and seeded with <em>E. coli</em> OP50 food. Animals were dried on the screening plates for several hours at room temperature and then incubated overnight at 20C. The L4440 vector and <em>pmk-1</em> RNAi clones were included on each of the screening plates as the negative and positive controls, respectively. Animals were scored for GFP expression and rated on a subjective scale from 0 (no GFP expression in response to RPW-24) to 3 (RPW-24-mediated induction of GFP expression equivalent to L4440). Exposure of the <em>C. elegans</em> transcriptional reporter <em>irg-1::GFP</em> to an <em>E. coli</em> strain that expresses ToxA was performed as previously described <a href="#ppat.1004143-McEwan1" class="ref-tip">[15]</a>.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="C. elegans Bacterial Infection and Other Assays"></a>
<h3><em>C. elegans</em> Bacterial Infection and Other Assays</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>“Slow killing” <em>P. aeruginosa</em> infection were performed as previously described <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>, <a href="#ppat.1004143-Tan1" class="ref-tip">[44]</a>. In all of these assays, the final concentration of DMSO was 1% and RPW-24 was used at a concentration of 70 µM, unless otherwise indicated. The propensity of wild-type <em>C. elegans</em> to leave a lawn of bacteria supplemented with RPW-24 was assayed using a previously described protocol <a href="#ppat.1004143-Melo1" class="ref-tip">[3]</a>, <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a> with minor modifications. Rather than adding the toxin on top of the small lawn of food, 20 µg of RPW-24 was mixed with <em>E. coli</em> OP50, which was spotted onto NGM plates. To assess the toxicity of RPW-24, we assayed the development of animals exposed to vector control (L4440), <em>pmk-1(RNAi)</em> and <em>mdt-15(RNAi)</em> in the presence of 70 µM RPW-24, as previously described <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. <em>P. aeruginosa</em> “fast kill” pathogenesis assays were conducted with late L4 and early young adult animals (picked 1–3 hours after the L4 molt) obtained from timed egg lays as described <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a>, <a href="#ppat.1004143-MahajanMiklos1" class="ref-tip">[29]</a>. For the killing assay using toxic phenazines, 50 µg/ml phenazine-1-carboxylic acid (PCA) and 5 µg/ml 1-hydroxyphenazine in DMSO were added to modified “fast kill” media (1% bacto-peptone, 1% glucose, 1% NaCl, 150 mM sorbitol, 1.7% bacto agar, 5 µg/ml cholesterol and 50 mM sodium citrate, pH 5) <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a>. These phenazine concentrations correspond to the amount of PCA and 1-hydroxy-phenazine that are produced under “fast kill” conditions <a href="#ppat.1004143-Cezairliyan1" class="ref-tip">[28]</a>. <em>E. coli</em> OP50 was used as the food source. The modified “fast kill” media pH 5.0 plus 1% DMSO was used as the control condition. These assays were incubated at 21–23°C.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="Quantitative RT-PCR (qRT-PCR) and NanoString nCounter Gene Expression Analyses"></a>
<h3>Quantitative RT-PCR (qRT-PCR) and NanoString nCounter Gene Expression Analyses</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>Synchronized L1 staged <em>C. elegans</em> N2 animals were grown to L4/young adult stage on the indicated RNAi strain, transferred to assay plates and incubated at 25°C for 24 hours. To prepare the assay plates, 70 µM RPW-24 or DMSO was added to 20 mL “slow killing” media <a href="#ppat.1004143-Tan1" class="ref-tip">[44]</a> in 10 cm petri dishes seeded with <em>E. coli</em> OP50. N2 and <em>pmk-1(km25)</em> animals were raised on <em>E. coli</em> OP50 and exposed to the above conditions for 18 hours at 20°C. For qRT-PCR studies of nematodes infected with <em>P. aeruginosa</em> PA14 or the indicated strain of <em>P. aeruginosa</em>, 20 mL of “slow killing” media containing either DMSO or 70 µM RPW-24 was added to 10 cm petri dishes. Plates were seeded with either 75 µL of <em>E. coli</em> OP50 or <em>P. aeruginosa</em>, each from cultures grown for 15 hours at 37°C. The plates were incubated for 24 hours at 37°C and 24 hours at 25°C. L4/young adult animals were added to the assay plates and incubated at 25°C for eight hours. RNA was isolated using TriReagent (Molecular Research Center, Inc.) and analyzed by NanoString nCounter Gene Expression Analysis (NanoString Technologies) using a “codeset” designed by NanoString that contained probes for 118 <em>C. elegans</em> genes (<a href="#ppat.1004143.s008">Table S2</a>). Probe hybridization, data acquisition and analysis were performed according to instructions from NanoString with each RNA sample normalized to the control genes <em>snb-1</em>, <em>ama-1</em> and <em>act-1</em>. For the qRT-PCR studes, RNA was reverse transcribed to cDNA using the Retroscript kit (Life Technologies) and analyzed using a CFX1000 machine (Bio-Rad) with previously published primers <a href="#ppat.1004143-Troemel1" class="ref-tip">[6]</a>, <a href="#ppat.1004143-Taubert1" class="ref-tip">[18]</a>. The qRT-PCR primers for the <em>mdt-15</em>, <em>pmk-1</em>, <em>tir-1</em> and <em>sek-1</em> genes were designed for this study and are available upon request. All values were normalized against the control gene <em>snb-1</em>. Fold change was calculated using the Pfaffl method <a href="#ppat.1004143-Pfaffl1" class="ref-tip">[45]</a>.</p>
</div>

<div id="section5" class="section toc-section"><a id="s4e" name="s4e" class="link-target" title="Microscopy"></a>
<h3>Microscopy</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>Nematodes were mounted onto agar pads, paralyzed with 10 mM levamisole (Sigma) and photographed using a Zeiss AXIO Imager Z1 microscope with a Zeiss AxioCam HRm camera and Axiovision 4.6 (Zeiss) software. For comparisons of GFP expression in the <em>F08G5.6::GFP</em> transgenic animals, photographs were acquired using the same imaging conditions.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4f" name="s4f" class="link-target" title="Statistical Analyses"></a>
<h3>Statistical Analyses</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>Differences in survival of <em>C. elegans</em> animals in the <em>P. aeruginosa</em> pathogenesis assays were determined with the log-rank test in each of two biological replicates. Differences were considered significant only if the <em>p</em> value was less than 0.05 for both replicates. In the manuscript, data from one experiement that is representative of both replicates is shown and the sample sizes for these experiments are given in <a href="#ppat.1004143.s009">Table S3</a>. To determine if the increase in survival conferred by RPW-24 treatment was different in one population compared to another, we examined the difference in the effect of RPW-24 treatment on the hazard in each group using a Cox proportional hazard model (Stata13, Stata, College Station, TX) from two biological replicates, as previously described <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. Fold changes in the qRT-PCR analyses were compared using unpaired, two-tailed student <em>t</em>-tests.</p>
</div>

<div id="section7" class="section toc-section"><a id="s4g" name="s4g" class="link-target" title="Accession Numbers"></a>
<h3>Accession Numbers</h3>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1" class="link-target"></a><p>Accession numbers for genes and gene products are given for the publically available database Wormbase (<a href="http://www.wormbase.org">http://www.wormbase.org</a>). The accession numbers for the principal genes mentioned in this paper are: <em>atf-7 (C07G2.2), C32H11.1, cyp-35A1 (C03G6.14), cyp-35B2 (K07C6.3), cyp-35C1 (C06B3.3), F35E12.5, F08G5.6, F01D5.5, fshr-1 (C50H2.1), irg-1 (C07G3.2), irg-2 (C49G7.5), mdt-15 (R12B2.5), nsy-1 (F59A6.1), pmk-1 (B0218.3), sek-1 (R03G5.2), skn-1 (T19E7.2), tir-1(F13B10.1)</em>, and <em>zip-2 (K02F3.4)</em>. Other accession numbers are given in <a href="#ppat-1004143-g002">Figure 2</a>, <a href="#ppat-1004143-g006">Figure 6</a>, <a href="#ppat.1004143.s007">Table S1</a> and <a href="#ppat.1004143.s008">Table S2</a>.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.ppat.1004143"></div><div class="supplementary-material"><a name="ppat.1004143.s001" id="ppat.1004143.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong><em>P. aeruginosa</em></strong><strong> strains with greater virulence towards </strong><strong><em>C. elegans</em></strong><strong> cause more robust induction of the immune response gene </strong><strong><em>F08G5.6</em></strong><strong>.</strong> (A) Wild-type N2 <em>C. elegans</em> were exposed for eight hours to eight <em>P. aeruginosa</em> strains, each with a different virulence potential toward <em>C. elegans</em> <a href="#ppat.1004143-Lee1" class="ref-tip">[13]</a>. Data are the average of two biological replicates, each normalized to a control gene with error bars representing SEM and are presented as the fold induction of <em>F08G5.6</em> compared to its expression in <em>C. elegans</em> exposed to <em>E. coli</em> OP50. *<em>p</em>&lt;0.05 for the difference from the <em>F08G5.6</em> induction in the most virulent <em>P. aeruginosa</em> strain (CF18). (B) Worms carrying the <em>pF08G5.6::GFP</em> reporter were exposed to <em>P. aeruginosa</em> PA14 and PAK for 18 hours, and photographed under the same conditions.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s001">https://doi.org/10.1371/journal.ppat.1004143.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s002" id="ppat.1004143.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s002">Figure S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>The Mediator subunit MDT-15 regulates immune and detoxification genes in response to RPW-24.</strong> (A) The expression of putative <em>C. elegans</em> immune effector genes was analyzed by qRT-PCR in vector control (L4440) and <em>mdt-15(RNAi)</em> animals exposed to either 70 µM RPW-24 or the solvent control DMSO, as indicated. The difference in expression of each of the genes in both <em>mdt-15(RNAi)</em> conditions was significantly different from vector control (L4440) (<em>p</em>&lt;0.001). (B) The RPW-24-induced expression levels for the indicated genes were compared in <em>mdt-15(tm2182)</em> mutant animals and wild-type controls. Data are presented as the average expression value of the indicated gene following RPW-24 exposure in <em>mdt-15(tm2812)</em> animals versus wild-type animals. (C) The expression levels of three components of the p38 MAP kinase PMK-1 pathway were compared in animals exposed to 70 µM RPW-24 or DMSO. (D) RPW-24-mediated induction levels of <em>cyp-35B1</em>, <em>cyp-35B2</em> and <em>cyp-35C1</em> were assessed in <em>pmk-1(km25)</em> and wild-type N2 animals as described above. There was no significant difference in the induction of these genes (<em>p</em>&gt;0.05). All data in this figure are the average of two or three replicates, each normalized to a control gene with error bars representing SEM. For the analyses in A and C, data are presented as the value relative to the average expression from all replicates of the indicated gene in the baseline condition [L4440 (A) or N2 (C) animals exposed to DMSO].</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s002">https://doi.org/10.1371/journal.ppat.1004143.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s003" id="ppat.1004143.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s003">Figure S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>The Mediator subunit MDT-15 is required for defense against </strong><strong><em>P. aeruginosa</em></strong><strong> infection.</strong> <em>C. elegans</em> wild-type N2 (A) and <em>fer-15(b26);fem-1(hc17)</em> sterile (C) animals were exposed to vector control (L4440) or <em>mdt-15(RNAi)</em> and infected with <em>P. aeruginosa</em> in the presence of the solvent control DMSO or 70 µM RPW-24. There is a significant difference in susceptibility to <em>P. aeruginosa</em> infection between the L4440/DMSO and <em>mdt-15(RNAi)</em>/DMSO conditions in A and C (<em>p</em>&lt;0.001). In addition, the magnitude of lifespan prolongation conferred by RPW-24 during <em>P. aeruginosa</em> infection was significantly reduced in <em>mdt-15(RNAi)</em> animals compared to control animals for both assays (<em>p</em>&lt;0.001). (B) <em>C. elegans</em> wild-type N2 or <em>mdt-15(tm2182)</em> animals were infected with <em>P. aeruginosa</em> in the presence of the solvent control DMSO or 70 µM RPW-24. There is a significant difference between the N2/DMSO and <em>mdt-15(tm2182)</em>/DMSO conditions, which was also significant in a biological replicate of this experiment (<em>p</em>&lt;0.001). The degree of lifespan extension conferred by RPW-24 exposure during <em>P. aeruginosa</em> infection was reduced in <em>mdt-15(tm2182)</em> animals compared to wild-type animals, but the difference did not reach statistical significance (<em>p</em> = 0.09). In A, B and C, data at each time point are the average of three plates per strain, each with approximately 30–40 animals per plate (see <a href="#ppat.1004143.s009">Table S3</a> for sample sizes). Each assay presented in this figure is representative of two independent experiments.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s003">https://doi.org/10.1371/journal.ppat.1004143.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s004" id="ppat.1004143.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s004">Figure S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Protection from the toxic effects of the xenobiotic RPW-24 requires MDT-15, but not PMK-1.</strong> The percentage of the population of animals that were at the indicated development stage after 70 hours of incubation at 20°C is presented. Adult animals all had a fully formed vulva and were divided into three categories based on the number of eggs carried in the animal. Larval stages were determined based on microscopic examination of the gonad. See <a href="#ppat.1004143.s009">Table S3</a> for the sample sizes.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s004">https://doi.org/10.1371/journal.ppat.1004143.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s005" id="ppat.1004143.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s005">Figure S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Neither MDT-15 nor PMK-1 are required for the behavioral avoidance response to RPW-24.</strong> (A) The percentage of wild-type N2, <em>pmk-1(km25)</em> and <em>mdt-15(tm2182)</em> animals that were off a lawn of <em>E. coli</em> OP50 supplemented with either DMSO or 20 µg RPW-24 (average of three replicates) 16 hours after exposure is presented with error bars representing standard deviation. There is no significant difference in the percentage of animals off a lawn containing RPW-24 among the conditions in this experiment (<em>p</em>&gt;0.05). See <a href="#ppat.1004143.s009">Table S3</a> for the sample sizes. (B) Representative photographs of each condition are shown.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s005">https://doi.org/10.1371/journal.ppat.1004143.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s006" id="ppat.1004143.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s006">Figure S6. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Resistance to </strong><strong><em>P. aeruginosa</em></strong><strong> phenazine toxins requires MDT-15.</strong> The controls for the experiment in <a href="#ppat-1004143-g007">Figure 7</a> are presented. (A) <em>C. elegans</em> wild-type N2 and <em>mdt-15(tm2182)</em> animals at the L4 stage are sensitive to killing by <em>P. aeruginosa</em> in the “fast killing” toxic phenazine assay. (B) Likewise, L4 staged nematodes are rapidly killed by toxic phenazines added to assay media in the absence of pathogen. These experiments were conducted in parallel with and as described for the data presented in <a href="#ppat-1004143-g007">Figure 7</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s006">https://doi.org/10.1371/journal.ppat.1004143.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s007" id="ppat.1004143.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s007">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Genes identified in an RNAi screen for regulators of the p38 MAP kinase PMK-1 pathway.</strong> (A) Presented are the 29 genes that were identified in a screen of 1,420 RNAi clones, the complete list of which is given (B), because they reduce or abrogate the induction of the p38 MAP kinase PMK-1-dependent immune reporters <em>F08G5.6::GFP</em> and <em>F35E12.5::GFP</em>. Knockdown of these genes does not affect the induction of <em>irg-1::GFP</em> by an <em>E. coli</em> strain engineered to express ToxA. qRT-PCR of three replicates was used to confirm that the RNAi-mediated knockdown of the indicated gene in wild-type animals reduced the induction of <em>F08G5.6</em> and <em>F35E12.5</em> by RPW-24. We previously showed that <em>atf-7</em> and <em>pmk-1</em> are required for the RPW-24-mediated induction of <em>F08G5.6</em> and <em>F35E12.5</em> <a href="#ppat.1004143-PukkilaWorley3" class="ref-tip">[9]</a>. These 29 genes were identified in an RNAi screen of 1,420 clones, which are presented in (B). Of note, the 29 genes presented in (A) were sequenced confirmed.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s007">https://doi.org/10.1371/journal.ppat.1004143.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s008" id="ppat.1004143.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s008">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Relative expression of the 118 genes in the nCounter Gene Expression Analysis.</strong> Nanostring nCoutner Gene Expression Analysis data for (A) N2 wild-type and <em>pmk-(km25)</em>, and (B) Vector control (L4440) and <em>mdt-15(RNAi)</em>, each exposed to DMSO or RPW-24, are presented. Expression values are the average of two independent replicates each of which was normalized to three control genes and are presented as the value relative to the average expression of the indicated gene in the baseline condition [N2 animals (A) or vector control (L4440) animals (B) exposed to DMSO]. <em>p</em> values for the comparison of the indicated conditions are presented.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s008">https://doi.org/10.1371/journal.ppat.1004143.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="ppat.1004143.s009" id="ppat.1004143.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.ppat.1004143.s009">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Samples sizes for the assays.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.ppat.1004143.s009">https://doi.org/10.1371/journal.ppat.1004143.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We are grateful to Alexander A. Soukas and the members of his laboratory for many helpful discussions, and to Justine A. Melo and Gary Ruvkun for constructing the RNAi feeding library used in these studies. We also thank Gary Ruvkun for his comments on the manuscript, Robert Dowen for technical advice, and the <em>Caenorhabititis</em> Genetics Center for sending us strains used in this study.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: RPW RLF FMA. Performed the experiments: RPW RLF DLM ALC. Analyzed the data: RPW RLF. Contributed reagents/materials/analysis tools: RPW RLF DLM ALC. Wrote the paper: RPW RLF FMA.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="ppat.1004143-Lindblom1" id="ppat.1004143-Lindblom1" class="link-target"></a>Lindblom TH, Dodd AK (2006) Xenobiotic detoxification in the nematode <em>Caenorhabditis elegans</em>. J Exp Zoolog Part A Comp Exp Biol 305: 720–730. <ul class="reflinks"><li><a href="#" data-author="Lindblom" data-cit="LindblomTH%2C%20DoddAK%20%282006%29%20Xenobiotic%20detoxification%20in%20the%20nematode%20Caenorhabditis%20elegans.%20J%20Exp%20Zoolog%20Part%20A%20Comp%20Exp%20Biol%20305%3A%20720%E2%80%93730." data-title="Xenobiotic%20detoxification%20in%20the%20nematode%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Xenobiotic+detoxification+in+the+nematode+Caenorhabditis+elegans+Lindblom+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="ppat.1004143-PukkilaWorley1" id="ppat.1004143-PukkilaWorley1" class="link-target"></a>Pukkila-Worley R, Ausubel FM (2012) Immune defense mechanisms in the <em>Caenorhabditis elegans</em> intestinal epithelium. Curr Opin Immunol 24: 3–9. <ul class="reflinks"><li><a href="#" data-author="Pukkila-Worley" data-cit="Pukkila-WorleyR%2C%20AusubelFM%20%282012%29%20Immune%20defense%20mechanisms%20in%20the%20Caenorhabditis%20elegans%20intestinal%20epithelium.%20Curr%20Opin%20Immunol%2024%3A%203%E2%80%939." data-title="Immune%20defense%20mechanisms%20in%20the%20Caenorhabditis%20elegans%20intestinal%20epithelium" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Immune+defense+mechanisms+in+the+Caenorhabditis+elegans+intestinal+epithelium+Pukkila-Worley+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="ppat.1004143-Melo1" id="ppat.1004143-Melo1" class="link-target"></a>Melo JA, Ruvkun G (2012) Inactivation of conserved <em>C. elegans</em> genes engages pathogen- and xenobiotic-associated defenses. Cell 149: 452–466. <ul class="reflinks"><li><a href="#" data-author="Melo" data-cit="MeloJA%2C%20RuvkunG%20%282012%29%20Inactivation%20of%20conserved%20C.%20elegans%20genes%20engages%20pathogen-%20and%20xenobiotic-associated%20defenses.%20Cell%20149%3A%20452%E2%80%93466." data-title="Inactivation%20of%20conserved%20C.%20elegans%20genes%20engages%20pathogen-%20and%20xenobiotic-associated%20defenses" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Inactivation+of+conserved+C.+elegans+genes+engages+pathogen-+and+xenobiotic-associated+defenses+Melo+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="ppat.1004143-Runkel1" id="ppat.1004143-Runkel1" class="link-target"></a>Runkel ED, Liu S, Baumeister R, Schulze E (2013) Surveillance-activated defenses block the ROS-induced mitochondrial unfolded protein response. PLoS Genet 9: e1003346. <ul class="reflinks"><li><a href="#" data-author="Runkel" data-cit="RunkelED%2C%20LiuS%2C%20BaumeisterR%2C%20SchulzeE%20%282013%29%20Surveillance-activated%20defenses%20block%20the%20ROS-induced%20mitochondrial%20unfolded%20protein%20response.%20PLoS%20Genet%209%3A%20e1003346." data-title="Surveillance-activated%20defenses%20block%20the%20ROS-induced%20mitochondrial%20unfolded%20protein%20response" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Surveillance-activated+defenses+block+the+ROS-induced+mitochondrial+unfolded+protein+response+Runkel+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="ppat.1004143-Kim1" id="ppat.1004143-Kim1" class="link-target"></a>Kim DH, Feinbaum R, Alloing G, Emerson FE, Garsin DA, et al. (2002) A conserved p38 MAP kinase pathway in <em>Caenorhabditis elegans</em> innate immunity. Science 297: 623–626. <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimDH%2C%20FeinbaumR%2C%20AlloingG%2C%20EmersonFE%2C%20GarsinDA%2C%20et%20al.%20%282002%29%20A%20conserved%20p38%20MAP%20kinase%20pathway%20in%20Caenorhabditis%20elegans%20innate%20immunity.%20Science%20297%3A%20623%E2%80%93626." data-title="A%20conserved%20p38%20MAP%20kinase%20pathway%20in%20Caenorhabditis%20elegans%20innate%20immunity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+conserved+p38+MAP+kinase+pathway+in+Caenorhabditis+elegans+innate+immunity+Kim+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="ppat.1004143-Troemel1" id="ppat.1004143-Troemel1" class="link-target"></a>Troemel ER, Chu SW, Reinke V, Lee SS, Ausubel FM, et al. (2006) p38 MAPK Regulates Expression of Immune Response Genes and Contributes to Longevity in <em>C. elegans</em>. PLoS Genet 2: e183. <ul class="reflinks"><li><a href="#" data-author="Troemel" data-cit="TroemelER%2C%20ChuSW%2C%20ReinkeV%2C%20LeeSS%2C%20AusubelFM%2C%20et%20al.%20%282006%29%20p38%20MAPK%20Regulates%20Expression%20of%20Immune%20Response%20Genes%20and%20Contributes%20to%20Longevity%20in%20C.%20elegans.%20PLoS%20Genet%202%3A%20e183." data-title="p38%20MAPK%20Regulates%20Expression%20of%20Immune%20Response%20Genes%20and%20Contributes%20to%20Longevity%20in%20C.%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=p38+MAPK+Regulates+Expression+of+Immune+Response+Genes+and+Contributes+to+Longevity+in+C.+elegans+Troemel+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="ppat.1004143-PukkilaWorley2" id="ppat.1004143-PukkilaWorley2" class="link-target"></a>Pukkila-Worley R, Ausubel FM, Mylonakis E (2011) <em>Candida albicans</em> Infection of <em>Caenorhabditis elegans</em> Induces Antifungal Immune Defenses. PLoS Pathog 7: e1002074. <ul class="reflinks"><li><a href="#" data-author="Pukkila-Worley" data-cit="Pukkila-WorleyR%2C%20AusubelFM%2C%20MylonakisE%20%282011%29%20Candida%20albicans%20Infection%20of%20Caenorhabditis%20elegans%20Induces%20Antifungal%20Immune%20Defenses.%20PLoS%20Pathog%207%3A%20e1002074." data-title="Candida%20albicans%20Infection%20of%20Caenorhabditis%20elegans%20Induces%20Antifungal%20Immune%20Defenses" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Candida+albicans+Infection+of+Caenorhabditis+elegans+Induces+Antifungal+Immune+Defenses+Pukkila-Worley+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="ppat.1004143-Moy1" id="ppat.1004143-Moy1" class="link-target"></a>Moy TI, Conery AL, Larkins-Ford J, Wu G, Mazitschek R, et al. (2009) High-throughput screen for novel antimicrobials using a whole animal infection model. ACS Chem Biol 4: 527–533. <ul class="reflinks"><li><a href="#" data-author="Moy" data-cit="MoyTI%2C%20ConeryAL%2C%20Larkins-FordJ%2C%20WuG%2C%20MazitschekR%2C%20et%20al.%20%282009%29%20High-throughput%20screen%20for%20novel%20antimicrobials%20using%20a%20whole%20animal%20infection%20model.%20ACS%20Chem%20Biol%204%3A%20527%E2%80%93533." data-title="High-throughput%20screen%20for%20novel%20antimicrobials%20using%20a%20whole%20animal%20infection%20model" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=High-throughput+screen+for+novel+antimicrobials+using+a+whole+animal+infection+model+Moy+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="ppat.1004143-PukkilaWorley3" id="ppat.1004143-PukkilaWorley3" class="link-target"></a>Pukkila-Worley R, Feinbaum R, Kirienko NV, Larkins-Ford J, Conery AL, et al. (2012) Stimulation of Host Immune Defenses by a Small Molecule Protects <em>C. elegans</em> from Bacterial Infection. PLoS Genet 8: e1002733. <ul class="reflinks"><li><a href="#" data-author="Pukkila-Worley" data-cit="Pukkila-WorleyR%2C%20FeinbaumR%2C%20KirienkoNV%2C%20Larkins-FordJ%2C%20ConeryAL%2C%20et%20al.%20%282012%29%20Stimulation%20of%20Host%20Immune%20Defenses%20by%20a%20Small%20Molecule%20Protects%20C.%20elegans%20from%20Bacterial%20Infection.%20PLoS%20Genet%208%3A%20e1002733." data-title="Stimulation%20of%20Host%20Immune%20Defenses%20by%20a%20Small%20Molecule%20Protects%20C.%20elegans%20from%20Bacterial%20Infection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Stimulation+of+Host+Immune+Defenses+by+a+Small+Molecule+Protects+C.+elegans+from+Bacterial+Infection+Pukkila-Worley+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="ppat.1004143-Bolz1" id="ppat.1004143-Bolz1" class="link-target"></a>Bolz DD, Tenor JL, Aballay A (2010) A conserved PMK-1/p38 MAPK is required in <em>Caenorhabditis elegans</em> tissue-specific immune response to <em>Yersinia pestis</em> infection. J Biol Chem 285: 10832–10840. <ul class="reflinks"><li><a href="#" data-author="Bolz" data-cit="BolzDD%2C%20TenorJL%2C%20AballayA%20%282010%29%20A%20conserved%20PMK-1%2Fp38%20MAPK%20is%20required%20in%20Caenorhabditis%20elegans%20tissue-specific%20immune%20response%20to%20Yersinia%20pestis%20infection.%20J%20Biol%20Chem%20285%3A%2010832%E2%80%9310840." data-title="A%20conserved%20PMK-1%2Fp38%20MAPK%20is%20required%20in%20Caenorhabditis%20elegans%20tissue-specific%20immune%20response%20to%20Yersinia%20pestis%20infection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+conserved+PMK-1%2Fp38+MAPK+is+required+in+Caenorhabditis+elegans+tissue-specific+immune+response+to+Yersinia+pestis+infection+Bolz+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="ppat.1004143-Liberati1" id="ppat.1004143-Liberati1" class="link-target"></a>Liberati NT, Fitzgerald KA, Kim DH, Feinbaum R, Golenbock DT, et al. (2004) Requirement for a conserved Toll/interleukin-1 resistance domain protein in the <em>Caenorhabditis elegans</em> immune response. Proc Natl Acad Sci USA 101: 6593–6598. <ul class="reflinks"><li><a href="#" data-author="Liberati" data-cit="LiberatiNT%2C%20FitzgeraldKA%2C%20KimDH%2C%20FeinbaumR%2C%20GolenbockDT%2C%20et%20al.%20%282004%29%20Requirement%20for%20a%20conserved%20Toll%2Finterleukin-1%20resistance%20domain%20protein%20in%20the%20Caenorhabditis%20elegans%20immune%20response.%20Proc%20Natl%20Acad%20Sci%20USA%20101%3A%206593%E2%80%936598." data-title="Requirement%20for%20a%20conserved%20Toll%2Finterleukin-1%20resistance%20domain%20protein%20in%20the%20Caenorhabditis%20elegans%20immune%20response" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Requirement+for+a+conserved+Toll%2Finterleukin-1+resistance+domain+protein+in+the+Caenorhabditis+elegans+immune+response+Liberati+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="ppat.1004143-Shivers1" id="ppat.1004143-Shivers1" class="link-target"></a>Shivers RP, Pagano DJ, Kooistra T, Richardson CE, Reddy KC, et al. (2010) Phosphorylation of the Conserved Transcription Factor ATF-7 by PMK-1 p38 MAPK Regulates Innate Immunity in <em>Caenorhabditis elegans</em>. PLoS Genet 6: e1000892. <ul class="reflinks"><li><a href="#" data-author="Shivers" data-cit="ShiversRP%2C%20PaganoDJ%2C%20KooistraT%2C%20RichardsonCE%2C%20ReddyKC%2C%20et%20al.%20%282010%29%20Phosphorylation%20of%20the%20Conserved%20Transcription%20Factor%20ATF-7%20by%20PMK-1%20p38%20MAPK%20Regulates%20Innate%20Immunity%20in%20Caenorhabditis%20elegans.%20PLoS%20Genet%206%3A%20e1000892." data-title="Phosphorylation%20of%20the%20Conserved%20Transcription%20Factor%20ATF-7%20by%20PMK-1%20p38%20MAPK%20Regulates%20Innate%20Immunity%20in%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phosphorylation+of+the+Conserved+Transcription+Factor+ATF-7+by+PMK-1+p38+MAPK+Regulates+Innate+Immunity+in+Caenorhabditis+elegans+Shivers+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="ppat.1004143-Lee1" id="ppat.1004143-Lee1" class="link-target"></a>Lee DG, Urbach JM, Wu G, Liberati NT, Feinbaum RL, et al. (2006) Genomic analysis reveals that <em>Pseudomonas aeruginosa</em> virulence is combinatorial. Genome Biol 7: R90. <ul class="reflinks"><li><a href="#" data-author="Lee" data-cit="LeeDG%2C%20UrbachJM%2C%20WuG%2C%20LiberatiNT%2C%20FeinbaumRL%2C%20et%20al.%20%282006%29%20Genomic%20analysis%20reveals%20that%20Pseudomonas%20aeruginosa%20virulence%20is%20combinatorial.%20Genome%20Biol%207%3A%20R90." data-title="Genomic%20analysis%20reveals%20that%20Pseudomonas%20aeruginosa%20virulence%20is%20combinatorial" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genomic+analysis+reveals+that+Pseudomonas+aeruginosa+virulence+is+combinatorial+Lee+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="ppat.1004143-Estes1" id="ppat.1004143-Estes1" class="link-target"></a>Estes KA, Dunbar TL, Powell JR, Ausubel FM, Troemel ER (2010) bZIP transcription factor <em>zip-2</em> mediates an early response to <em>Pseudomonas aeruginosa</em> infection in <em>Caenorhabditis elegans</em>. Proc Natl Acad Sci USA 107: 2153–2158. <ul class="reflinks"><li><a href="#" data-author="Estes" data-cit="EstesKA%2C%20DunbarTL%2C%20PowellJR%2C%20AusubelFM%2C%20TroemelER%20%282010%29%20bZIP%20transcription%20factor%20zip-2%20mediates%20an%20early%20response%20to%20Pseudomonas%20aeruginosa%20infection%20in%20Caenorhabditis%20elegans.%20Proc%20Natl%20Acad%20Sci%20USA%20107%3A%202153%E2%80%932158." data-title="bZIP%20transcription%20factor%20zip-2%20mediates%20an%20early%20response%20to%20Pseudomonas%20aeruginosa%20infection%20in%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=bZIP+transcription+factor+zip-2+mediates+an+early+response+to+Pseudomonas+aeruginosa+infection+in+Caenorhabditis+elegans+Estes+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="ppat.1004143-McEwan1" id="ppat.1004143-McEwan1" class="link-target"></a>McEwan DL, Kirienko NV, Ausubel FM (2012) Host Translational Inhibition by <em>Pseudomonas aeruginosa</em> Exotoxin A Triggers an Immune Response in <em>Caenorhabditis elegans</em>. Cell Host Microbe 11: 364–374. <ul class="reflinks"><li><a href="#" data-author="McEwan" data-cit="McEwanDL%2C%20KirienkoNV%2C%20AusubelFM%20%282012%29%20Host%20Translational%20Inhibition%20by%20Pseudomonas%20aeruginosa%20Exotoxin%20A%20Triggers%20an%20Immune%20Response%20in%20Caenorhabditis%20elegans.%20Cell%20Host%20Microbe%2011%3A%20364%E2%80%93374." data-title="Host%20Translational%20Inhibition%20by%20Pseudomonas%20aeruginosa%20Exotoxin%20A%20Triggers%20an%20Immune%20Response%20in%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Host+Translational+Inhibition+by+Pseudomonas+aeruginosa+Exotoxin+A+Triggers+an+Immune+Response+in+Caenorhabditis+elegans+McEwan+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="ppat.1004143-Dunbar1" id="ppat.1004143-Dunbar1" class="link-target"></a>Dunbar TL, Yan Z, Balla KM, Smelkinson MG, Troemel ER (2012) <em>C. elegans</em> Detects Pathogen-Induced Translational Inhibition to Activate Immune Signaling. Cell Host Microbe 11: 375–386. <ul class="reflinks"><li><a href="#" data-author="Dunbar" data-cit="DunbarTL%2C%20YanZ%2C%20BallaKM%2C%20SmelkinsonMG%2C%20TroemelER%20%282012%29%20C.%20elegans%20Detects%20Pathogen-Induced%20Translational%20Inhibition%20to%20Activate%20Immune%20Signaling.%20Cell%20Host%20Microbe%2011%3A%20375%E2%80%93386." data-title="C.%20elegans%20Detects%20Pathogen-Induced%20Translational%20Inhibition%20to%20Activate%20Immune%20Signaling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=C.+elegans+Detects+Pathogen-Induced+Translational+Inhibition+to+Activate+Immune+Signaling+Dunbar+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="ppat.1004143-Shivers2" id="ppat.1004143-Shivers2" class="link-target"></a>Shivers RP, Kooistra T, Chu SW, Pagano DJ, Kim DH (2009) Tissue-specific activities of an immune signaling module regulate physiological responses to pathogenic and nutritional bacteria in <em>C. elegans</em>. Cell Host Microbe 6: 321–330. <ul class="reflinks"><li><a href="#" data-author="Shivers" data-cit="ShiversRP%2C%20KooistraT%2C%20ChuSW%2C%20PaganoDJ%2C%20KimDH%20%282009%29%20Tissue-specific%20activities%20of%20an%20immune%20signaling%20module%20regulate%20physiological%20responses%20to%20pathogenic%20and%20nutritional%20bacteria%20in%20C.%20elegans.%20Cell%20Host%20Microbe%206%3A%20321%E2%80%93330." data-title="Tissue-specific%20activities%20of%20an%20immune%20signaling%20module%20regulate%20physiological%20responses%20to%20pathogenic%20and%20nutritional%20bacteria%20in%20C.%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tissue-specific+activities+of+an+immune+signaling+module+regulate+physiological+responses+to+pathogenic+and+nutritional+bacteria+in+C.+elegans+Shivers+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="ppat.1004143-Taubert1" id="ppat.1004143-Taubert1" class="link-target"></a>Taubert S, Van Gilst MR, Hansen M, Yamamoto KR (2006) A Mediator subunit, MDT-15, integrates regulation of fatty acid metabolism by NHR-49-dependent and -independent pathways in <em>C. elegans</em>. Genes Dev 20: 1137–1149. <ul class="reflinks"><li><a href="#" data-author="Taubert" data-cit="TaubertS%2C%20Van%20GilstMR%2C%20HansenM%2C%20YamamotoKR%20%282006%29%20A%20Mediator%20subunit%2C%20MDT-15%2C%20integrates%20regulation%20of%20fatty%20acid%20metabolism%20by%20NHR-49-dependent%20and%20-independent%20pathways%20in%20C.%20elegans.%20Genes%20Dev%2020%3A%201137%E2%80%931149." data-title="A%20Mediator%20subunit%2C%20MDT-15%2C%20integrates%20regulation%20of%20fatty%20acid%20metabolism%20by%20NHR-49-dependent%20and%20-independent%20pathways%20in%20C.%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+Mediator+subunit%2C+MDT-15%2C+integrates+regulation+of+fatty+acid+metabolism+by+NHR-49-dependent+and+-independent+pathways+in+C.+elegans+Taubert+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="ppat.1004143-Taubert2" id="ppat.1004143-Taubert2" class="link-target"></a>Taubert S, Hansen M, Van Gilst MR, Cooper SB, Yamamoto KR (2008) The Mediator subunit MDT-15 confers metabolic adaptation to ingested material. PLoS Genet 4: e1000021. <ul class="reflinks"><li><a href="#" data-author="Taubert" data-cit="TaubertS%2C%20HansenM%2C%20Van%20GilstMR%2C%20CooperSB%2C%20YamamotoKR%20%282008%29%20The%20Mediator%20subunit%20MDT-15%20confers%20metabolic%20adaptation%20to%20ingested%20material.%20PLoS%20Genet%204%3A%20e1000021." data-title="The%20Mediator%20subunit%20MDT-15%20confers%20metabolic%20adaptation%20to%20ingested%20material" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Mediator+subunit+MDT-15+confers+metabolic+adaptation+to+ingested+material+Taubert+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="ppat.1004143-Yang1" id="ppat.1004143-Yang1" class="link-target"></a>Yang F, Vought BW, Satterlee JS, Walker AK, Jim Sun Z-Y, et al. (2006) An ARC/Mediator subunit required for SREBP control of cholesterol and lipid homeostasis. Nature 442: 700–704. <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangF%2C%20VoughtBW%2C%20SatterleeJS%2C%20WalkerAK%2C%20Jim%20SunZ-Y%2C%20et%20al.%20%282006%29%20An%20ARC%2FMediator%20subunit%20required%20for%20SREBP%20control%20of%20cholesterol%20and%20lipid%20homeostasis.%20Nature%20442%3A%20700%E2%80%93704." data-title="An%20ARC%2FMediator%20subunit%20required%20for%20SREBP%20control%20of%20cholesterol%20and%20lipid%20homeostasis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+ARC%2FMediator+subunit+required+for+SREBP+control+of+cholesterol+and+lipid+homeostasis+Yang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="ppat.1004143-Goh1" id="ppat.1004143-Goh1" class="link-target"></a>Goh GYS, Martelli KL, Parhar KS, Kwong AWL, Wong MA, et al. (2013) The conserved Mediator subunit MDT-15 is required for oxidative stress responses in <em>C. elegans</em>. Aging Cell 13(1): 70–9. <ul class="reflinks"><li><a href="#" data-author="Goh" data-cit="GohGYS%2C%20MartelliKL%2C%20ParharKS%2C%20KwongAWL%2C%20WongMA%2C%20et%20al.%20%282013%29%20The%20conserved%20Mediator%20subunit%20MDT-15%20is%20required%20for%20oxidative%20stress%20responses%20in%20C.%20elegans.%20Aging%20Cell%2013%281%29%3A%2070%E2%80%939." data-title="The%20conserved%20Mediator%20subunit%20MDT-15%20is%20required%20for%20oxidative%20stress%20responses%20in%20C.%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+conserved+Mediator+subunit+MDT-15+is+required+for+oxidative+stress+responses+in+C.+elegans+Goh+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="ppat.1004143-Wong1" id="ppat.1004143-Wong1" class="link-target"></a>Wong D, Bazopoulou D, Pujol N, Tavernarakis N, Ewbank JJ (2007) Genome-wide investigation reveals pathogen-specific and shared signatures in the response of <em>Caenorhabditis elegans</em> to infection. Genome Biol 8: R194. <ul class="reflinks"><li><a href="#" data-author="Wong" data-cit="WongD%2C%20BazopoulouD%2C%20PujolN%2C%20TavernarakisN%2C%20EwbankJJ%20%282007%29%20Genome-wide%20investigation%20reveals%20pathogen-specific%20and%20shared%20signatures%20in%20the%20response%20of%20Caenorhabditis%20elegans%20to%20infection.%20Genome%20Biol%208%3A%20R194." data-title="Genome-wide%20investigation%20reveals%20pathogen-specific%20and%20shared%20signatures%20in%20the%20response%20of%20Caenorhabditis%20elegans%20to%20infection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+investigation+reveals+pathogen-specific+and+shared+signatures+in+the+response+of+Caenorhabditis+elegans+to+infection+Wong+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="ppat.1004143-ORourke1" id="ppat.1004143-ORourke1" class="link-target"></a>O'Rourke D, Baban D, Demidova M, Mott R, Hodgkin J (2006) Genomic clusters, putative pathogen recognition molecules, and antimicrobial genes are induced by infection of <em>C. elegans</em> with <em>M. nematophilum</em>. Genome Res 16: 1005–1016. <ul class="reflinks"><li><a href="#" data-author="O%27Rourke" data-cit="O%27RourkeD%2C%20BabanD%2C%20DemidovaM%2C%20MottR%2C%20HodgkinJ%20%282006%29%20Genomic%20clusters%2C%20putative%20pathogen%20recognition%20molecules%2C%20and%20antimicrobial%20genes%20are%20induced%20by%20infection%20of%20C.%20elegans%20with%20M.%20nematophilum.%20Genome%20Res%2016%3A%201005%E2%80%931016." data-title="Genomic%20clusters%2C%20putative%20pathogen%20recognition%20molecules%2C%20and%20antimicrobial%20genes%20are%20induced%20by%20infection%20of%20C.%20elegans%20with%20M.%20nematophilum" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genomic+clusters%2C+putative+pathogen+recognition+molecules%2C+and+antimicrobial+genes+are+induced+by+infection+of+C.+elegans+with+M.+nematophilum+O%27Rourke+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="ppat.1004143-Kim2" id="ppat.1004143-Kim2" class="link-target"></a>Kim DH, Liberati NT, Mizuno T, Inoue H, Hisamoto N, et al. (2004) Integration of <em>Caenorhabditis elegans</em> MAPK pathways mediating immunity and stress resistance by MEK-1 MAPK kinase and VHP-1 MAPK phosphatase. Proc Natl Acad Sci USA 101: 10990–10994. <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimDH%2C%20LiberatiNT%2C%20MizunoT%2C%20InoueH%2C%20HisamotoN%2C%20et%20al.%20%282004%29%20Integration%20of%20Caenorhabditis%20elegans%20MAPK%20pathways%20mediating%20immunity%20and%20stress%20resistance%20by%20MEK-1%20MAPK%20kinase%20and%20VHP-1%20MAPK%20phosphatase.%20Proc%20Natl%20Acad%20Sci%20USA%20101%3A%2010990%E2%80%9310994." data-title="Integration%20of%20Caenorhabditis%20elegans%20MAPK%20pathways%20mediating%20immunity%20and%20stress%20resistance%20by%20MEK-1%20MAPK%20kinase%20and%20VHP-1%20MAPK%20phosphatase" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Integration+of+Caenorhabditis+elegans+MAPK+pathways+mediating+immunity+and+stress+resistance+by+MEK-1+MAPK+kinase+and+VHP-1+MAPK+phosphatase+Kim+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="ppat.1004143-Miyata1" id="ppat.1004143-Miyata1" class="link-target"></a>Miyata S, Begun J, Troemel ER, Ausubel FM (2008) DAF-16-dependent suppression of immunity during reproduction in <em>Caenorhabditis elegan</em>s. Genetics 178: 903–918. <ul class="reflinks"><li><a href="#" data-author="Miyata" data-cit="MiyataS%2C%20BegunJ%2C%20TroemelER%2C%20AusubelFM%20%282008%29%20DAF-16-dependent%20suppression%20of%20immunity%20during%20reproduction%20in%20Caenorhabditis%20elegans.%20Genetics%20178%3A%20903%E2%80%93918." data-title="DAF-16-dependent%20suppression%20of%20immunity%20during%20reproduction%20in%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=DAF-16-dependent+suppression+of+immunity+during+reproduction+in+Caenorhabditis+elegans+Miyata+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="ppat.1004143-Irazoqui1" id="ppat.1004143-Irazoqui1" class="link-target"></a>Irazoqui JE, Troemel ER, Feinbaum RL, Luhachack LG, Cezairliyan BO, et al. (2010) Distinct pathogenesis and host responses during infection of <em>C. elegans</em> by <em>P. aeruginosa</em> and <em>S. aureus</em>. PLoS Pathog 6: e1000982. <ul class="reflinks"><li><a href="#" data-author="Irazoqui" data-cit="IrazoquiJE%2C%20TroemelER%2C%20FeinbaumRL%2C%20LuhachackLG%2C%20CezairliyanBO%2C%20et%20al.%20%282010%29%20Distinct%20pathogenesis%20and%20host%20responses%20during%20infection%20of%20C.%20elegans%20by%20P.%20aeruginosa%20and%20S.%20aureus.%20PLoS%20Pathog%206%3A%20e1000982." data-title="Distinct%20pathogenesis%20and%20host%20responses%20during%20infection%20of%20C.%20elegans%20by%20P.%20aeruginosa%20and%20S.%20aureus" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Distinct+pathogenesis+and+host+responses+during+infection+of+C.+elegans+by+P.+aeruginosa+and+S.+aureus+Irazoqui+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="ppat.1004143-Feinbaum1" id="ppat.1004143-Feinbaum1" class="link-target"></a>Feinbaum RL, Urbach JM, Liberati NT, Djonovic S, Adonizio A, et al. (2012) Genome-wide identification of <em>Pseudomonas aeruginosa</em> virulence-related genes using a <em>Caenorhabditis elegans</em> infection model. PLoS Pathog 8: e1002813. <ul class="reflinks"><li><a href="#" data-author="Feinbaum" data-cit="FeinbaumRL%2C%20UrbachJM%2C%20LiberatiNT%2C%20DjonovicS%2C%20AdonizioA%2C%20et%20al.%20%282012%29%20Genome-wide%20identification%20of%20Pseudomonas%20aeruginosa%20virulence-related%20genes%20using%20a%20Caenorhabditis%20elegans%20infection%20model.%20PLoS%20Pathog%208%3A%20e1002813." data-title="Genome-wide%20identification%20of%20Pseudomonas%20aeruginosa%20virulence-related%20genes%20using%20a%20Caenorhabditis%20elegans%20infection%20model" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+identification+of+Pseudomonas+aeruginosa+virulence-related+genes+using+a+Caenorhabditis+elegans+infection+model+Feinbaum+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="ppat.1004143-Cezairliyan1" id="ppat.1004143-Cezairliyan1" class="link-target"></a>Cezairliyan B, Vinayavekhin N, Grenfell-Lee D, Yuen GJ, Saghatelian A, et al. (2013) Identification of <em>Pseudomonas aeruginosa</em> phenazines that kill <em>Caenorhabditis elegans</em>. PLoS Pathog 9: e1003101. <ul class="reflinks"><li><a href="#" data-author="Cezairliyan" data-cit="CezairliyanB%2C%20VinayavekhinN%2C%20Grenfell-LeeD%2C%20YuenGJ%2C%20SaghatelianA%2C%20et%20al.%20%282013%29%20Identification%20of%20Pseudomonas%20aeruginosa%20phenazines%20that%20kill%20Caenorhabditis%20elegans.%20PLoS%20Pathog%209%3A%20e1003101." data-title="Identification%20of%20Pseudomonas%20aeruginosa%20phenazines%20that%20kill%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+Pseudomonas+aeruginosa+phenazines+that+kill+Caenorhabditis+elegans+Cezairliyan+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="ppat.1004143-MahajanMiklos1" id="ppat.1004143-MahajanMiklos1" class="link-target"></a>Mahajan-Miklos S, Tan MW, Rahme LG, Ausubel FM (1999) Molecular mechanisms of bacterial virulence elucidated using a <em>Pseudomonas aeruginosa</em>-<em>Caenorhabditis elegans</em> pathogenesis model. Cell 96: 47–56. <ul class="reflinks"><li><a href="#" data-author="Mahajan-Miklos" data-cit="Mahajan-MiklosS%2C%20TanMW%2C%20RahmeLG%2C%20AusubelFM%20%281999%29%20Molecular%20mechanisms%20of%20bacterial%20virulence%20elucidated%20using%20a%20Pseudomonas%20aeruginosa-Caenorhabditis%20elegans%20pathogenesis%20model.%20Cell%2096%3A%2047%E2%80%9356." data-title="Molecular%20mechanisms%20of%20bacterial%20virulence%20elucidated%20using%20a%20Pseudomonas%20aeruginosa-Caenorhabditis%20elegans%20pathogenesis%20model" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+mechanisms+of+bacterial+virulence+elucidated+using+a+Pseudomonas+aeruginosa-Caenorhabditis+elegans+pathogenesis+model+Mahajan-Miklos+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="ppat.1004143-Conaway1" id="ppat.1004143-Conaway1" class="link-target"></a>Conaway RC, Conaway JW (2011) Function and regulation of the Mediator complex. Curr Opin Genet Dev 21: 225–230. <ul class="reflinks"><li><a href="#" data-author="Conaway" data-cit="ConawayRC%2C%20ConawayJW%20%282011%29%20Function%20and%20regulation%20of%20the%20Mediator%20complex.%20Curr%20Opin%20Genet%20Dev%2021%3A%20225%E2%80%93230." data-title="Function%20and%20regulation%20of%20the%20Mediator%20complex" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Function+and+regulation+of+the+Mediator+complex+Conaway+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="ppat.1004143-Malik1" id="ppat.1004143-Malik1" class="link-target"></a>Malik S, Roeder RG (2010) The metazoan Mediator co-activator complex as an integrative hub for transcriptional regulation. Nat Rev Genet 11: 761–772. <ul class="reflinks"><li><a href="#" data-author="Malik" data-cit="MalikS%2C%20RoederRG%20%282010%29%20The%20metazoan%20Mediator%20co-activator%20complex%20as%20an%20integrative%20hub%20for%20transcriptional%20regulation.%20Nat%20Rev%20Genet%2011%3A%20761%E2%80%93772." data-title="The%20metazoan%20Mediator%20co-activator%20complex%20as%20an%20integrative%20hub%20for%20transcriptional%20regulation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+metazoan+Mediator+co-activator+complex+as+an+integrative+hub+for+transcriptional+regulation+Malik+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="ppat.1004143-Holstege1" id="ppat.1004143-Holstege1" class="link-target"></a>Holstege FC, Jennings EG, Wyrick JJ, Lee TI, Hengartner CJ, et al. (1998) Dissecting the regulatory circuitry of a eukaryotic genome. Cell 95: 717–728. <ul class="reflinks"><li><a href="#" data-author="Holstege" data-cit="HolstegeFC%2C%20JenningsEG%2C%20WyrickJJ%2C%20LeeTI%2C%20HengartnerCJ%2C%20et%20al.%20%281998%29%20Dissecting%20the%20regulatory%20circuitry%20of%20a%20eukaryotic%20genome.%20Cell%2095%3A%20717%E2%80%93728." data-title="Dissecting%20the%20regulatory%20circuitry%20of%20a%20eukaryotic%20genome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dissecting+the+regulatory+circuitry+of+a+eukaryotic+genome+Holstege+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="ppat.1004143-Huang1" id="ppat.1004143-Huang1" class="link-target"></a>Huang S, Hölzel M, Knijnenburg T, Schlicker A, Roepman P, et al. (2012) MED12 controls the response to multiple cancer drugs through regulation of TGF-β receptor signaling. Cell 151: 937–950. <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangS%2C%20H%C3%B6lzelM%2C%20KnijnenburgT%2C%20SchlickerA%2C%20RoepmanP%2C%20et%20al.%20%282012%29%20MED12%20controls%20the%20response%20to%20multiple%20cancer%20drugs%20through%20regulation%20of%20TGF-%CE%B2%20receptor%20signaling.%20Cell%20151%3A%20937%E2%80%93950." data-title="MED12%20controls%20the%20response%20to%20multiple%20cancer%20drugs%20through%20regulation%20of%20TGF-%CE%B2%20receptor%20signaling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MED12+controls+the+response+to+multiple+cancer+drugs+through+regulation+of+TGF-%CE%B2+receptor+signaling+Huang+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="ppat.1004143-Boyer1" id="ppat.1004143-Boyer1" class="link-target"></a>Boyer TG, Martin ME, Lees E, Ricciardi RP, Berk AJ (1999) Mammalian Srb/Mediator complex is targeted by adenovirus E1A protein. Nature 399: 276–279. <ul class="reflinks"><li><a href="#" data-author="Boyer" data-cit="BoyerTG%2C%20MartinME%2C%20LeesE%2C%20RicciardiRP%2C%20BerkAJ%20%281999%29%20Mammalian%20Srb%2FMediator%20complex%20is%20targeted%20by%20adenovirus%20E1A%20protein.%20Nature%20399%3A%20276%E2%80%93279." data-title="Mammalian%20Srb%2FMediator%20complex%20is%20targeted%20by%20adenovirus%20E1A%20protein" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mammalian+Srb%2FMediator+complex+is+targeted+by+adenovirus+E1A+protein+Boyer+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="ppat.1004143-Thakur1" id="ppat.1004143-Thakur1" class="link-target"></a>Thakur JK, Arthanari H, Yang F, Chau KH, Wagner G, et al. (2009) Mediator subunit Gal11p/MED15 is required for fatty acid-dependent gene activation by yeast transcription factor Oaf1p. J Biol Chem 284: 4422–4428. <ul class="reflinks"><li><a href="#" data-author="Thakur" data-cit="ThakurJK%2C%20ArthanariH%2C%20YangF%2C%20ChauKH%2C%20WagnerG%2C%20et%20al.%20%282009%29%20Mediator%20subunit%20Gal11p%2FMED15%20is%20required%20for%20fatty%20acid-dependent%20gene%20activation%20by%20yeast%20transcription%20factor%20Oaf1p.%20J%20Biol%20Chem%20284%3A%204422%E2%80%934428." data-title="Mediator%20subunit%20Gal11p%2FMED15%20is%20required%20for%20fatty%20acid-dependent%20gene%20activation%20by%20yeast%20transcription%20factor%20Oaf1p" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mediator+subunit+Gal11p%2FMED15+is+required+for+fatty+acid-dependent+gene+activation+by+yeast+transcription+factor+Oaf1p+Thakur+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="ppat.1004143-Thakur2" id="ppat.1004143-Thakur2" class="link-target"></a>Thakur JK, Arthanari H, Yang F, Pan S-J, Fan X, et al. (2008) A nuclear receptor-like pathway regulating multidrug resistance in fungi. Nature 452: 604–609. <ul class="reflinks"><li><a href="#" data-author="Thakur" data-cit="ThakurJK%2C%20ArthanariH%2C%20YangF%2C%20PanS-J%2C%20FanX%2C%20et%20al.%20%282008%29%20A%20nuclear%20receptor-like%20pathway%20regulating%20multidrug%20resistance%20in%20fungi.%20Nature%20452%3A%20604%E2%80%93609." data-title="A%20nuclear%20receptor-like%20pathway%20regulating%20multidrug%20resistance%20in%20fungi" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+nuclear+receptor-like+pathway+regulating+multidrug+resistance+in+fungi+Thakur+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="ppat.1004143-Brenner1" id="ppat.1004143-Brenner1" class="link-target"></a>Brenner S (1974) The genetics of <em>Caenorhabditis elegans</em>. Genetics 77: 71–94. <ul class="reflinks"><li><a href="#" data-author="Brenner" data-cit="BrennerS%20%281974%29%20The%20genetics%20of%20Caenorhabditis%20elegans.%20Genetics%2077%3A%2071%E2%80%9394." data-title="The%20genetics%20of%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+genetics+of+Caenorhabditis+elegans+Brenner+1974" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="ppat.1004143-Garigan1" id="ppat.1004143-Garigan1" class="link-target"></a>Garigan D, Hsu A-L, Fraser AG, Kamath RS, Ahringer J, et al. (2002) Genetic analysis of tissue aging in <em>Caenorhabditis elegans</em>: a role for heat-shock factor and bacterial proliferation. Genetics 161: 1101–1112. <ul class="reflinks"><li><a href="#" data-author="Garigan" data-cit="GariganD%2C%20HsuA-L%2C%20FraserAG%2C%20KamathRS%2C%20AhringerJ%2C%20et%20al.%20%282002%29%20Genetic%20analysis%20of%20tissue%20aging%20in%20Caenorhabditis%20elegans%3A%20a%20role%20for%20heat-shock%20factor%20and%20bacterial%20proliferation.%20Genetics%20161%3A%201101%E2%80%931112." data-title="Genetic%20analysis%20of%20tissue%20aging%20in%20Caenorhabditis%20elegans%3A%20a%20role%20for%20heat-shock%20factor%20and%20bacterial%20proliferation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+analysis+of+tissue+aging+in+Caenorhabditis+elegans%3A+a+role+for+heat-shock+factor+and+bacterial+proliferation+Garigan+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="ppat.1004143-McEwan2" id="ppat.1004143-McEwan2" class="link-target"></a>McEwan DL, Weisman AS, Hunter CP (2012) Uptake of extracellular double-stranded RNA by SID-2. Mol Cell 47: 746–754. <ul class="reflinks"><li><a href="#" data-author="McEwan" data-cit="McEwanDL%2C%20WeismanAS%2C%20HunterCP%20%282012%29%20Uptake%20of%20extracellular%20double-stranded%20RNA%20by%20SID-2.%20Mol%20Cell%2047%3A%20746%E2%80%93754." data-title="Uptake%20of%20extracellular%20double-stranded%20RNA%20by%20SID-2" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Uptake+of+extracellular+double-stranded+RNA+by+SID-2+McEwan+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="ppat.1004143-Mello1" id="ppat.1004143-Mello1" class="link-target"></a>Mello CC, Kramer JM, Stinchcomb D, Ambros V (1991) Efficient gene transfer in <em>C. elegans</em>: extrachromosomal maintenance and integration of transforming sequences. EMBO J 10: 3959–3970. <ul class="reflinks"><li><a href="#" data-author="Mello" data-cit="MelloCC%2C%20KramerJM%2C%20StinchcombD%2C%20AmbrosV%20%281991%29%20Efficient%20gene%20transfer%20in%20C.%20elegans%3A%20extrachromosomal%20maintenance%20and%20integration%20of%20transforming%20sequences.%20EMBO%20J%2010%3A%203959%E2%80%933970." data-title="Efficient%20gene%20transfer%20in%20C.%20elegans%3A%20extrachromosomal%20maintenance%20and%20integration%20of%20transforming%20sequences" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Efficient+gene+transfer+in+C.+elegans%3A+extrachromosomal+maintenance+and+integration+of+transforming+sequences+Mello+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="ppat.1004143-Kamath1" id="ppat.1004143-Kamath1" class="link-target"></a>Kamath RS, Ahringer J (2003) Genome-wide RNAi screening in <em>Caenorhabditis elegans</em>. Methods 30: 313–321. <ul class="reflinks"><li><a href="#" data-author="Kamath" data-cit="KamathRS%2C%20AhringerJ%20%282003%29%20Genome-wide%20RNAi%20screening%20in%20Caenorhabditis%20elegans.%20Methods%2030%3A%20313%E2%80%93321." data-title="Genome-wide%20RNAi%20screening%20in%20Caenorhabditis%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+RNAi+screening+in+Caenorhabditis+elegans+Kamath+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="ppat.1004143-Rual1" id="ppat.1004143-Rual1" class="link-target"></a>Rual J-F, Ceron J, Koreth J, Hao T, Nicot A-S, et al. (2004) Toward improving <em>Caenorhabditis elegans</em> phenome mapping with an ORFeome-based RNAi library. Genome Res 14: 2162–2168. <ul class="reflinks"><li><a href="#" data-author="Rual" data-cit="RualJ-F%2C%20CeronJ%2C%20KorethJ%2C%20HaoT%2C%20NicotA-S%2C%20et%20al.%20%282004%29%20Toward%20improving%20Caenorhabditis%20elegans%20phenome%20mapping%20with%20an%20ORFeome-based%20RNAi%20library.%20Genome%20Res%2014%3A%202162%E2%80%932168." data-title="Toward%20improving%20Caenorhabditis%20elegans%20phenome%20mapping%20with%20an%20ORFeome-based%20RNAi%20library" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Toward+improving+Caenorhabditis+elegans+phenome+mapping+with+an+ORFeome-based+RNAi+library+Rual+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="ppat.1004143-Dietrich1" id="ppat.1004143-Dietrich1" class="link-target"></a>Dietrich LEP, Price-Whelan A, Petersen A, Whiteley M, Newman DK (2006) The phenazine pyocyanin is a terminal signalling factor in the quorum sensing network of <em>Pseudomonas aeruginosa</em>. Mol Microbiol 61: 1308–1321. <ul class="reflinks"><li><a href="#" data-author="Dietrich" data-cit="DietrichLEP%2C%20Price-WhelanA%2C%20PetersenA%2C%20WhiteleyM%2C%20NewmanDK%20%282006%29%20The%20phenazine%20pyocyanin%20is%20a%20terminal%20signalling%20factor%20in%20the%20quorum%20sensing%20network%20of%20Pseudomonas%20aeruginosa.%20Mol%20Microbiol%2061%3A%201308%E2%80%931321." data-title="The%20phenazine%20pyocyanin%20is%20a%20terminal%20signalling%20factor%20in%20the%20quorum%20sensing%20network%20of%20Pseudomonas%20aeruginosa" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+phenazine+pyocyanin+is+a+terminal+signalling+factor+in+the+quorum+sensing+network+of+Pseudomonas+aeruginosa+Dietrich+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="ppat.1004143-Tan1" id="ppat.1004143-Tan1" class="link-target"></a>Tan MW, Mahajan-Miklos S, Ausubel FM (1999) Killing of <em>Caenorhabditis elegans</em> by <em>Pseudomonas aeruginosa</em> used to model mammalian bacterial pathogenesis. Proc Natl Acad Sci USA 96: 715–720. <ul class="reflinks"><li><a href="#" data-author="Tan" data-cit="TanMW%2C%20Mahajan-MiklosS%2C%20AusubelFM%20%281999%29%20Killing%20of%20Caenorhabditis%20elegans%20by%20Pseudomonas%20aeruginosa%20used%20to%20model%20mammalian%20bacterial%20pathogenesis.%20Proc%20Natl%20Acad%20Sci%20USA%2096%3A%20715%E2%80%93720." data-title="Killing%20of%20Caenorhabditis%20elegans%20by%20Pseudomonas%20aeruginosa%20used%20to%20model%20mammalian%20bacterial%20pathogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Killing+of+Caenorhabditis+elegans+by+Pseudomonas+aeruginosa+used+to+model+mammalian+bacterial+pathogenesis+Tan+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="ppat.1004143-Pfaffl1" id="ppat.1004143-Pfaffl1" class="link-target"></a>Pfaffl MW (2001) A new mathematical model for relative quantification in real-time RT-PCR. Nucleic Acids Res 29: e45. <ul class="reflinks"><li><a href="#" data-author="Pfaffl" data-cit="PfafflMW%20%282001%29%20A%20new%20mathematical%20model%20for%20relative%20quantification%20in%20real-time%20RT-PCR.%20Nucleic%20Acids%20Res%2029%3A%20e45." data-title="A%20new%20mathematical%20model%20for%20relative%20quantification%20in%20real-time%20RT-PCR" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+new+mathematical+model+for+relative+quantification+in+real-time+RT-PCR+Pfaffl+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

