



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>MicroRNAs play key roles in tumor metastasis. Here, we describe the regulation and function of miR-218 in gastric cancer (GC) metastasis. miR-218 expression is decreased along with the expression of one of its host genes, Slit3 in metastatic GC. However, Robo1, one of several Slit receptors, is negatively regulated by miR-218, thus establishing a negative feedback loop. Decreased miR-218 levels eliminate Robo1 repression, which activates the Slit-Robo1 pathway through the interaction between Robo1 and Slit2, thus triggering tumor metastasis. The restoration of miR-218 suppresses Robo1 expression and inhibits tumor cell invasion and metastasis <em>in vitro</em> and <em>in vivo</em>. Taken together, our results describe a Slit-miR-218-Robo1 regulatory circuit whose disruption may contribute to GC metastasis. Targeting miR-218 may provide a strategy for blocking tumor metastasis.</p>
</div><div class="abstract toc-section"><a id="abstract1" name="abstract1" class="link-target" data-toc="abstract1" title="Author Summary"></a>
<h2>Author Summary</h2>
<a id="article1.front1.article-meta1.abstract2.p1" name="article1.front1.article-meta1.abstract2.p1" class="link-target"></a><p>MicroRNAs have been identified as playing important roles in tumor metastasis, but their impact on GC metastasis has been poorly explored. We have discovered miR-218, which functions as a suppressor of tumor metastasis and is correlated with clinical stage, lymph node metastasis, and prognosis in patients with GC. Our results show that miR-218 is part of a regulatory circuit involving the Slit-Robo1 pathway. In metastatic tumor cells, miR-218 was suppressed along with Slit3, one of its host genes. Meanwhile, Robo1, one of several Slit receptors, is upregulated in response to the decrease in miR-218, which in turn induced a reactive upregulation of the Slit-Robo1 pathway through an interaction with Slit2, thus facilitating tumor cell migration and invasion. Such findings not only provide new insights into the metastatic mechanisms in GC but also provide evidence for a novel miRNA–mediated regulatory mode of receptor signaling.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Tie J, Pan Y, Zhao L, Wu K, Liu J, Sun S, et al.  (2010) MiR-218 Inhibits Invasion and Metastasis of Gastric Cancer by Targeting the Robo1 Receptor. PLoS Genet 6(3):
           e1000879.
        
        https://doi.org/10.1371/journal.pgen.1000879</p><p><strong>Editor: </strong>Bruce E. Clurman, Fred Hutchinson Cancer Research Center, United States of America</p><p><strong>Received: </strong>July 15, 2009; <strong>Accepted: </strong>February 10, 2010; <strong>Published: </strong> March 12, 2010</p><p><strong>Copyright: </strong> © 2010 Tie et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This study was supported by the National Natural Science Foundation of China (No. 30670970, No. 30873026, No. 30672369, No. 30530780) and the National Basic Research Program of China (No.2010CB529300). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Advances in diagnostic and therapeutic approaches have led to excellent expectations for long-term survival for early gastric cancer (GC). However, the prognosis for advanced GC with extensive invasion and metastasis remains poor <a href="#pgen.1000879-Oue1" class="ref-tip">[1]</a>. In order to metastasize, tumor cells must pass through a series of sequential and selective events, including detachment, migration, local invasion, angiogenesis, intravasation, survival in the circulatory system, extravasation, and regrowth in different organs. In the metastatic cascade, invasion of GC into the surrounding tissue is a crucial early step <a href="#pgen.1000879-Gupta1" class="ref-tip">[2]</a>–<a href="#pgen.1000879-Steeg1" class="ref-tip">[4]</a><strong>.</strong> However, the mechanisms of invasion have not yet been fully elucidated.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>A large number of microribonucleic acids (microRNAs or miRNAs) have been recently implicated in cancer metastasis <a href="#pgen.1000879-Nicoloso1" class="ref-tip">[5]</a>, including miR-10b, miR-21, miR-126, miR-335, miR-373, miR-146, miR-520c, and miR-205 in breast cancer <a href="#pgen.1000879-Ma1" class="ref-tip">[6]</a>–<a href="#pgen.1000879-Hurst1" class="ref-tip">[11]</a>; miR-224 and miR-21 in prostate cancer <a href="#pgen.1000879-Prueitt1" class="ref-tip">[12]</a>,<a href="#pgen.1000879-Li1" class="ref-tip">[13]</a>; miR-29c in nasopharyngeal carcinomas <a href="#pgen.1000879-Sengupta1" class="ref-tip">[14]</a>; miR-10a, miR-222, miR-125b, miR-7, and miR-452 in urothelial carcinomas <a href="#pgen.1000879-Veerla1" class="ref-tip">[15]</a>; miR-182 in melanoma <a href="#pgen.1000879-Segura1" class="ref-tip">[16]</a>; miR-92b and miR-9/9* in brain tumors <a href="#pgen.1000879-Nass1" class="ref-tip">[17]</a>; and miR-21 in colorectal cancer <a href="#pgen.1000879-Asangani1" class="ref-tip">[18]</a>. However, very few miRNAs known to be involved in GC metastasis have been reported. miRNAs are naturally occurring, short, non-coding RNA molecules that negatively regulate gene expression <a href="#pgen.1000879-Lewis1" class="ref-tip">[19]</a>. In mammals, mature miRNAs are generated from pri-miRNAs and pre-miRNAs via sequential processing by Drosha and Dicer and are found in many organisms. They consist of 21–24 nucleotides, integrate into RNA-inducing silencing complexes, and pair with the 3′ untranslated regions (3′-UTR) of specific target messenger RNAs (mRNAs) to suppress translation or induce degradation of the target mRNAs <a href="#pgen.1000879-Lai1" class="ref-tip">[20]</a>. Emerging evidence has revealed that miRNAs play key roles in various biological processes, including cell differentiation, proliferation, apoptosis, stress resistance, fat metabolism, tumorigenesis, and metastasis <a href="#pgen.1000879-Kloosterman1" class="ref-tip">[21]</a>–<a href="#pgen.1000879-Lu1" class="ref-tip">[23]</a>. A better understanding of the changes in miRNA expression during GC invasion may lead to a better understanding of GC development, as well as possible improvements in the diagnosis and treatment of advanced GC.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>In the present study, we established high (MKN28-M and SGC7901-M) and low invasive cell sublines (MKN28-NM and SGC7901-NM) using a repetitive transwell assay <em>in vitro</em>. We then examined the global miRNA expression profile in each cell subline using a miRNA microarray to identify differentially expressed miRNAs related to human GC invasion. In total, 45 miRNAs were shown to be differentially expressed in invasive vs. non-invasive GC cells. Among these, miR-218, a significantly downregulated miRNA in highly invasive cells, was shown to be closely correlated with GC tumorigenesis and metastasis in patients. More recently, a decrease in miR-218 has been reported in several kinds of solid tumors, including prostate cancer, GC, lung cancer, and cervical carcinoma <a href="#pgen.1000879-Volinia1" class="ref-tip">[24]</a>–<a href="#pgen.1000879-Yanaihara1" class="ref-tip">[27]</a>, but this decrease in miRNA-218 was simply screened out as being one of the dozens of potential miRNAs of interest in the cancers described above. No further studies have been performed to assess the significance of miR-218 in tumor metastasis. Here, we have found that decreased miR-218 expression was correlated with advanced clinical stage, lymph node metastasis, and poor prognosis in patients, and re-expression of miR-218 in metastatic cells was able to inhibit migration, invasion, and metastasis formation both <em>in vitro</em> and <em>in vivo</em>. Using a bioinformatics search for miR-218 targets, we pinpointed the receptor Robo1 as miR-218's functional target, and we confirmed that the interaction between miR-218 and Robo1 was crucial to GC cell motility by demonstrating that there was an inverse correlation between miR-218 and Robo1 in GC cell lines as well as in GC patients. Furthermore, we discovered an intriguing negative feedback loop involving Slit, miR-218, and Robo1, in which miR-218 can be derived from either of two genes located in the introns of two distinct members of the Slit protein family. In addition, members of this family are ligands of the Robo1 receptor. We demonstrated that expression of the two miRNA precursor genes (miR-218-1 and miR-218-2) correlated with expression of the host genes (Slit2 and Slit3, respectively) and that the mature miR-218 was mainly derived from the miR-218-2 precursor, with a concomitant reduction of host Slit3 but not of Slit2 in metastatic GC cells. Thus, upregulation of Robo1 in response to the decrease in miR-218 induced a reactive upregulation of the Slit-Robo1 pathway through its interaction with Slit2, thus facilitating tumor cell invasion and metastasis. Our findings not only provide new insights into the metastatic mechanisms in GC, but they also revealed a novel regulatory mechanism of receptor signaling.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Establishment and characterization of cell sublines with different invasive and metastatic potentials"></a>
<h3>Establishment and characterization of cell sublines with different invasive and metastatic potentials</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>To establish the GC metastasis models, we created invasive and non-invasive cell sublines from the human GC cell lines SGC7901 and MKN28 using the repeated transwell approach (<a href="#pgen-1000879-g001">Figure 1A</a>, see <a href="#s4">Materials and Methods</a>). Briefly, a repetitive invasion assay was performed, and those cells that failed to invade the membranes and cells that had the ability to migrate through the collagen-coated membrane in all selection rounds were separated. After ten rounds of selection, we obtained invasive (MKN28-M and SGC7901-M) and non-invasive cell sublines (MKN28-NM and SGC7901-NM). The metastatic properties of each cell subline were then characterized <em>in vitro</em> and <em>in vivo</em>. As shown in <a href="#pgen-1000879-g001">Figure 1B and 1C</a>, migration ability of MKN28-M cells was approximately 4-fold greater than that of MKN28-NM cells. Likewise, the invasive potential was about 5-fold greater for MKN28-M cells as compared to MKN28-NM cells. In the <em>in vivo</em> studies, tumor cell metastasis was observed in nude mice. As shown in <a href="#pgen-1000879-g001">Figure 1D and 1E</a>, almost no metastatic GC cells were detected in the lungs or livers of nude mice at 10 weeks after injection of MKN28-NM cells, whereas most of the mice injected with MKN28-M cells displayed obvious lung or liver metastases. Similar results were observed for SGC7901-M and SGC7901-NM cells (data not shown). No significant differences in cell proliferation or cell-cycle distribution were observed among these cell sublines (<a href="#pgen.1000879.s009">Text S1</a>, <a href="#pgen.1000879.s001">Figures S1</a> and <a href="#pgen.1000879.s002">S2</a>).</p>
<a class="link-target" id="pgen-1000879-g001" name="pgen-1000879-g001"></a><div class="figure" data-doi="10.1371/journal.pgen.1000879.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1000879.g001" data-doi="info:doi/10.1371/journal.pgen.1000879" data-uri="info:doi/10.1371/journal.pgen.1000879.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1000879.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> The metastatic characteristics of each cell subline.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) General scheme of the establishment of invasive and non-invasive cell sublines derived from human GC cell lines. (B,C) <em>In vitro</em> migration and invasion activity of each cell subline<strong>.</strong> Migration and invasion activities were measured <em>in vitro</em> with transwell chambers, as described in <a href="#s4">Materials and Methods</a>. Photos are representative fields of invasive cells on the membrane. Magnification, 100x. Bar graphs represent the average number of cells on the underside of the membrane ± SE. ** <em>P</em>&lt;0.01 as compared with non-invasive cells, analyzed by <em>t</em>-test. (D and E) Metastasis potential of each cell subline <em>in vivo</em> (n = 10). (D) The incidence of metastasis in mice that received intravenous tail injections of each selected cell subline. (E) H&amp;E staining of lungs and livers isolated from mice that received intravenous tail injections of MKN28-NM and MKN28-M cells, respectively. Magnification, 100x.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1000879.g001">
              https://doi.org/10.1371/journal.pgen.1000879.g001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Identification of metastasis-related miRNAs by array-based hybridization"></a>
<h3>Identification of metastasis-related miRNAs by array-based hybridization</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>To identify miRNAs potentially involved in GC invasion, we examined global miRNA expression in each cell subline using the microRNA array (v.10.0, Exiqon, Vedbaek, Denmark), which consists of 847 capture probes for mature human miRNAs. The microarray results revealed that the expression of 124 miRNAs significantly differed between the highly invasive variant MKN28-M and the non-invasive cell subline MKN28-NM. Of these, 83 were upregulated and 41 were downregulated. Compared with SGC7901-NM, 62 miRNAs were differentially expressed in the SGC7901-M cell subline, including 47 downregulated and 15 upregulated miRNAs. In total, 11 miRNAs were found to be upregulated and 34 miRNAs were downregulated in both MKN28-M and SGC7901-M cells compared with those in the corresponding non-invasive sublines (<a href="#pgen.1000879.s006">Table S1</a>).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>Of the 45 differentially regulated miRNAs, miR-218 was one of those that displayed significantly differential expression. miR-218 has been reported to be downregulated in cervical cancer <a href="#pgen.1000879-Martinez1" class="ref-tip">[25]</a>, GC <a href="#pgen.1000879-Petrocca1" class="ref-tip">[26]</a>, lung cancer <a href="#pgen.1000879-Yanaihara1" class="ref-tip">[27]</a> and prostate cancer <a href="#pgen.1000879-Leite1" class="ref-tip">[28]</a>, indicating possible involvement in both oncogenic transformation and tumor metastasis. However, miRNA-218 was only one of the many potential miRNAs of interest in cancers. In this work, miR-218 has been investigated in much greater detail. To validate the microarray results, we assessed miR-218 expression in the GC cell sublines previously mentioned and in the immortal gastric epithelial cell line GES using qRT–PCR <a href="#pgen.1000879-He1" class="ref-tip">[29]</a>. miR-218 expression was significantly decreased in MKN28-M and SGC7901-M cells and was lower in all four GC cell sublines compared to immortalized human gastric epithelial GES cells (<a href="#pgen-1000879-g002">Figure 2A</a>). Furthermore, we compared miR-218 expression in the primary GC tumor vs. the metastatic lymph nodes in 10 patients with stage III/IV GC using qRT–PCR. As shown in <a href="#pgen-1000879-g002">Figure 2B</a>, mature miR-218 levels were significantly decreased in 7 out of 10 metastatic lymph nodes, indicating that miR-218 may play a causal role in GC metastasis.</p>
<a class="link-target" id="pgen-1000879-g002" name="pgen-1000879-g002"></a><div class="figure" data-doi="10.1371/journal.pgen.1000879.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1000879.g002" data-doi="info:doi/10.1371/journal.pgen.1000879" data-uri="info:doi/10.1371/journal.pgen.1000879.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1000879.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Validation of miR-218 expression in metastatic GC cells.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>The expression of miR-218 was investigated by qRT-PCR. (A) Each bar represents the relative fold change compared to GES cell lines. (B) Bars represent relative fold changes between primary GC and metastatic lymph nodes from the same patient. Each sample was analyzed in triplicate and was normalized to U6. Fold change was calculated by 2<sup>−ΔΔCt</sup>. The results were consistent with the microarray data.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1000879.g002">
              https://doi.org/10.1371/journal.pgen.1000879.g002</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Decreased miR-218 expression in GC was associated with advanced clinical stage, lymph node metastasis, and poor patient prognosis"></a>
<h3>Decreased miR-218 expression in GC was associated with advanced clinical stage, lymph node metastasis, and poor patient prognosis</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>To determine the potential clinicopathological implications of altered miR-218 expression, we investigated the expression levels of miR-218 in 40 GC tissues (T) and non-tumor mucosa (N) b<strong>y</strong> qRT–PCR. The term -ΔCt was used to describe the expression level of miR-218. Consistent with the above data, the results verified that the miR-218 expression level in GC (-13.81±0.15, mean ± SE) tissues was significantly lower than that in non-neoplastic mucosa (-11.62±0.15, mean ± SE) (<em>P</em>&lt;0.0001, t = 10.62, paired <em>t</em>-test) (<a href="#pgen-1000879-g003">Figure 3A</a>). Correlations between the miR-218 expression level and clinicopathologic characteristics of GC are summarized in <a href="#pgen-1000879-t001">Table 1</a>. Statistically significant associations between the miR-218 expression level and clinical stage and between the miR-218 expression level and GC metastasis were observed in this study. The median expression of miR-218 was −14.25±0.17 in the 22 cases with advanced stage (stage III and IV) disease, whereas the median expression was −13.27±0.20 (<em>P</em> = 0.0010, Mann-Whitney test) in the 18 cases with early-stage (stages I and II) disease. In the 29 cases of GC with lymph node metastasis, the median expression of miR-218 was −14.09±0.16, which was significantly lower than the median expression (−13.07±0.24) in the 11 non-metastatic GC cases (<em>P</em> = 0.0036). The expression of miR-218 in GC patients did not correlate with age, gender, tumor size, or cell differentiation. Moreover, we examined whether the level of miR-218 expression was associated with survival in patients with GC. Patients were subsequently divided into low expression (n = 20) and high expression groups (n = 20) based on miR-218 levels greater or less than the mean (−13.81) (<a href="#pgen-1000879-g003">Figure 3B</a>). Kaplan–Meier survival analyses revealed that patients whose primary tumors displayed low expression of miR-218 had a shorter median survival time. The three-year survival rate of patients with low miR-218 expression was 30%, which was significantly lower than the survival rate in patients with high miR-218 expression (65%; <em>P</em> = 0.0012, log-rank test; <a href="#pgen-1000879-g003">Figure 3C</a>).</p>
<a class="link-target" id="pgen-1000879-g003" name="pgen-1000879-g003"></a><div class="figure" data-doi="10.1371/journal.pgen.1000879.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1000879.g003" data-doi="info:doi/10.1371/journal.pgen.1000879" data-uri="info:doi/10.1371/journal.pgen.1000879.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1000879.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> miR-218 expression in clinical GC specimens.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>(A) miR-218 was differentially expressed between GC and the corresponding non-neoplastic mucosa (N). The term -ΔCt was used to describe the expression level of miR-218 (−ΔCt = CtU6−CtmiR-218). A significant difference was detected in the mean value of miR-218 expression between these two groups (<em>P</em>&lt;0.0001, t = 10.62, paired <em>t</em>-test). (B) The same GC samples as in (A) were divided into two groups according to the mean expression of miR-218 (mean, −13.81). Cases with levels of miR-218 below the mean were miR-218 low expressers (<em>n</em> = 20), and those with levels of miR-218 above the mean were miR-218 high expressers (<em>n</em> = 20). (C) Kaplan–Meier survival curve and log-rank test for GC patients between high and low miR-218 expressers. miR-218 expression demonstrated a significant relationship with patient survival (log-rank, <em>P</em> = 0.0012).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1000879.g003">
              https://doi.org/10.1371/journal.pgen.1000879.g003</a></p></div><a class="link-target" id="pgen-1000879-t001" name="pgen-1000879-t001"></a><div class="figure" data-doi="10.1371/journal.pgen.1000879.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1000879.t001" data-doi="info:doi/10.1371/journal.pgen.1000879" data-uri="info:doi/10.1371/journal.pgen.1000879.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1000879.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> The relationship between clinicopathological parameters and miR-218 expression in primary gastric adenocarcinoma.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1000879.t001">
              https://doi.org/10.1371/journal.pgen.1000879.t001</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Ectopic expression of miR-218 inhibited tumor cell invasion and metastasis in vitro and in vivo"></a>
<h3>Ectopic expression of miR-218 inhibited tumor cell invasion and metastasis <em>in vitro</em> and <em>in vivo</em></h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>To study the role of miR-218 in GC metastasis, MKN28-M cells were transfected with pGenesil-1-miR-218 or a control vector expressing a nonspecific miRNA, cel-miR-67, using Lipofectamine 2000 (Invitrogen, Carlsbad, CA, USA). The cells were then selected with 400 mg/L G418 to generate MKN28-M-miR-218 and MKN28-M-miR-control stable cells. We found that ectopic expression of miR-218 resulted in an approximately three-fold reduction in migration and invasiveness. To determine whether the loss of miR-218 would promote the migration or invasion of cancer cells, we silenced miR-218 with an antisense oligonucleotide inhibitor in the MKN28-NM cell line, resulting in a three- to four-fold increase in cell migration and invasiveness (<a href="#pgen-1000879-g004">Figure 4A and 4B</a>). To test if inhibition of tumor invasion by miR-218 is caused by impairing the invasive ability of tumor cells, we excluded the effect of miR-218 on the proliferation and cell cycle distribution of gastric cancer cells. Over-expression of miR-218 did not affect the proliferation and the cell cycle distribution of MKN28-M cells <em>in vitro</em> (<a href="#pgen.1000879.s005">Figure S5A and S5D</a>). To further investigate the inhibition of <em>in vivo</em> tumor metastasis by miR-218, we implanted MKN28-M-miR-218 cells that were stably expressing miR-218 or control cells into nude mice through the lateral tail vein. Lung and liver metastasis of GC was apparent in mice injected with MKN28-M-miR-control cells. In contrast, few metastatic tumors were detected in mice injected with MKN28-M-miR-218 cells (<a href="#pgen-1000879-g004">Figure 4C</a>). Furthermore, we simultaneously observed the growth of the primary tumors and the incidence of distant metastasis in the nude mice injected subcutaneously with MKN28-M-miR-218 cells or control cells. The results showed lung or liver metastasis was apparent in 3 out of 10 mice injected with MKN28-M-miR-control cells; in stark contrast, no metastasis were found in mice injected with MKN28-M-miR-218 cells (<a href="#pgen.1000879.s005">Figure S5E</a>). These results are consistent with data obtained from tail vein assays that assess cancer metastasis and indicate that miR-218 has the ability to suppress metastasis without affecting cell proliferation.</p>
<a class="link-target" id="pgen-1000879-g004" name="pgen-1000879-g004"></a><div class="figure" data-doi="10.1371/journal.pgen.1000879.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1000879.g004" data-doi="info:doi/10.1371/journal.pgen.1000879" data-uri="info:doi/10.1371/journal.pgen.1000879.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1000879.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> miR-218 suppressed tumor cell invasion and metastasis.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p>(A) qRT–PCR analysis of miR-218 in MKN28-M cells transfected with the miR-218-expression vector or the miR-control vector and MKN28-NM cells transfected with anti-miR-218 or a negative control. (B) Cell invasion assay. (B1) Average number of invasive cells from three independent experiments ± SE. * <em>P</em>&lt;0.05. (B2) Representative fields of invasive cells on the membrane. (C) <em>In vivo</em> metastasis assay. MKN28-M cells were transfected with the miR-218-expression vector or the miR-control vector and injected into nude mice via the tail vein, as described in <a href="#s4">Materials and Methods</a>. Animals were killed 10 weeks after injection. (C1) Incidence of metastasis in mice. (C2) Representative H&amp;E staining of lungs and livers isolated from mice that received injections of MKN28-M-miR-control or MKN28-M-miR-218 cells.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1000879.g004">
              https://doi.org/10.1371/journal.pgen.1000879.g004</a></p></div></div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Robo1 was a direct functional target of miR-218 in GC metastasis"></a>
<h3>Robo1 was a direct functional target of miR-218 in GC metastasis</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>To assess how a low level of miR-218 expression contributes to the invasion and metastasis of GC, we searched for the potential regulatory targets of miR-218 using prediction tools, including miRanda, PicTar, and TargetScan. Although hundreds of different targets were predicted, those genes involved in migration or invasion may be the relevant targets with respect to the biological functions of miR-218. We then performed a functional classification of the predicted targets using the DAVID program (<a href="http://david.abcc.ncifcrf.gov/">http://david.abcc.ncifcrf.gov/</a>). Of these genes, Robo1 is regarded as a proto-oncogene and harbors migration-promoting activity <a href="#pgen.1000879-Wang1" class="ref-tip">[30]</a>–<a href="#pgen.1000879-Legg1" class="ref-tip">[35]</a>. Mertsch <em>et al.</em> demonstrated that Robo1 facilitates glioma cell migration mediated by Slit2 <a href="#pgen.1000879-Mertsch1" class="ref-tip">[36]</a>. Schmid <em>et al.</em> found that breast tumor cell migration is induced by the Slit2-Robo1 interaction <em>in vitro</em> <a href="#pgen.1000879-Schmid1" class="ref-tip">[37]</a>. These findings suggest that Robo1 may be a target for miR-218. To further test our hypothesis, we analyzed the expression of miR-218 and Robo1 in GES and in non-invasive (MKN28-NM and SGC7901-NM) and invasive (MKN28-M and SGC7901-M) GC cells. The results showed a negative correlation between the levels of miR-218 and Robo1 mRNA in these cells (<a href="#pgen.1000879.s003">Figure S3A</a>). Furthermore, we observed that Robo1 mRNA (<a href="#pgen.1000879.s003">Figure S3B</a>) and protein (<a href="#pgen-1000879-g005">Figure 5B2</a>) levels were decreased when miR-218 was expressed by pGenesil-1-miR-218 in MKN28-M cells (<a href="#pgen-1000879-g005">Figure 5B1</a>). The reverse was observed for Robo1 expression when miR-218 was knocked down in MKN28-NM cells (<a href="#pgen-1000879-g005">Figure 5C1</a> and <a href="#pgen-1000879-g005">Figure 5C2</a>). The inverse relationship between miR-218 and Robo1 expression was further confirmed by immunohistochemistry (<a href="#pgen.1000879.s009">Text S1</a>) in 40 cases of gastric cancer, in matched adjacent normal tissues that were also used in clinicopathological studies, and in 29 matched metastases. The results show that Robo1 was upregulated in GC, especially in metastatic GC (<a href="#pgen.1000879.s004">Figure S4</a>), in which miR-218 has a relatively low expression.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>To obtain further direct evidence that Robo1 is a target of miR-218, we investigated the binding site of miR-218 in the 3′-UTR of Robo1 mRNA (<a href="#pgen-1000879-g005">Figure 5A</a>). We constructed a luciferase reporter (Luc-Robo1) in which the nucleotides of the Robo1 3′-UTR complementary to miR-218 (nt 971–978) were inserted into the pMIR-REPORT miRNA expression reporter vector <a href="#pgen.1000879-Gramantieri1" class="ref-tip">[38]</a>. Correspondingly, we also generated both a mutant reporter (Luc-Robo1-mu), in which the first six nucleotides in the miR-218 seed-region complementary sites were deleted, and a control reporter, which contained a non-related fragment of cDNA (Luc-Ctrl). miR-218-expression plasmids were co-transfected with Luc-Robo1, Luc-Robo1-mu, or Luc-Ctrl into MKN28-M cells. The assays showed that the luciferase activity in the Luc-Robo1-transfected cells was significantly decreased compared to the luciferase activity in the mutant and negative control cells (<em>P</em>&lt;0.05), suggesting that miR-218 reduced the luciferase activity of Luc-Robo1 but had no effect on Luc-Robo1-mu (<a href="#pgen-1000879-g005">Figure 5D</a>). Therefore, we concluded that the inserted fragment of Robo1 (nt 971<em>–</em>978) was the target of miR-218.</p>
<a class="link-target" id="pgen-1000879-g005" name="pgen-1000879-g005"></a><div class="figure" data-doi="10.1371/journal.pgen.1000879.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1000879.g005" data-doi="info:doi/10.1371/journal.pgen.1000879" data-uri="info:doi/10.1371/journal.pgen.1000879.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1000879.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> miR-218 targeted Robo1 by binding to its 3′-UTR.</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>(A) The Robo1 3′-UTR was a potential target of miR-218. (B and C) miR-218 and Robo1 levels were analyzed by qRT–PCR and western blot, respectively. Robo1 levels decreased when miR-218 was upregulated in response to the miR-218-expression vector in MKN28-M cells, whereas the reverse was observed for Robo1 expression when miR-218 was knocked down in MKN28-NM cells. (D) MKN28-M cells were co-transfected with miR-218 and a luciferase reporter (Luc-Robo1) containing a fragment of the Robo1 3′-UTR harboring either the miR-218 binding site or a mutant (Luc-Robo1<em>-</em>mu) in which the first six nucleotides of the miR-218 binding site were deleted. A luciferase reporter construct engineered with a non-related fragment of cDNA was used as a negative control (Luc-control). The assays showed that luciferase activity in the Luc-Robo1 group was significantly decreased compared to the luciferase activity of the mutant and negative control groups. (E) MKN28-M-miR-218 cells, which stably over-expressed miR-218, were transiently transfected with a Robo1 expression construct or a Robo1 mutant construct lacking the miR-218 binding site. MKN28-M cells were transfected with Robo1 siRNA or a negative control siRNA. Western blot analysis for Robo1 showed that co-transfection of miR-218 and the Robo1 mutant construct produced higher levels of Robo1 protein than co-transfection of miR-218 and the Robo1 construct. Robo1 siRNA effectively reduced the amount Robo1 protein observed. (F) The cell invasion assay indicated that Robo1 mutant constructs could reverse the effect of miR-218-mediated suppression of cell invasion. Knockdown of Robo1 by siRNA in MKN28-M cells inhibited cell invasion. * <em>P</em>&lt;0.05.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1000879.g005">
              https://doi.org/10.1371/journal.pgen.1000879.g005</a></p></div><a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3" class="link-target"></a><p>Robo1 has been shown to be over-expressed in cancer cells and is known to promote tumor angiogenesis and metastasis via an interaction with Slit <a href="#pgen.1000879-Seth1" class="ref-tip">[39]</a>,<a href="#pgen.1000879-Abdollahi1" class="ref-tip">[40]</a>. To test whether Robo1 is functionally regulated by miR-218, we generated a Robo1 expression construct containing only a fragment of the predicted miR-218 binding site and Robo1 mutant expression vector entirely lacking the 3′-UTR. We also made the Robo1 siRNA. MKN28-M-miR-218 cells, which stably expressed miR-218 ectopically, were transiently transfected with the Robo1 construct or the mutant construct (with no miR-218 binding site), and MKN28-M cells were transfected with Robo1 siRNA or a negative control siRNA. MKN28-M-miR-218 cells transfected with the Robo1 mutant construct showed a 3.8-fold increase in invasion ability compared to cells transfected with the Robo1 construct. These results indicate that introduction of mutant Robo1 cDNA that lacked the miR-218 binding site into the miR-218-overexpressing cells reversed the effect of miR-218-mediated suppression of cell invasion. However, the effect of Robo1 was repressed by miR-218 in the presence of the Robo1 3′-UTR containing the miR-218 binding sites. Knockdown of Robo1 by siRNA in MKN28-M cells inhibited cell invasion, which fell to levels similar to those observed after transfection with the miR-218-expressing vector (<a href="#pgen-1000879-g005">Figure 5E and 5F</a>). These observations suggest that miR-218 directly suppresses Robo1-mediated cell invasion.</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Slit2, but not Slit3, can interact with Robo1, enriched by the absence of miR-218, to promote GC invasion"></a>
<h3>Slit2, but not Slit3, can interact with Robo1, enriched by the absence of miR-218, to promote GC invasion</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>Two types of miRNAs exist: intergenic and intronic. The former are located in non-coding regions between genes, and their corresponding pri-miRNAs are generally transcribed from their own promoters by RNA polymerase II. The latter are located within the introns of host genes, and their biogenesis is controlled by the host gene promoters <a href="#pgen.1000879-Shi1" class="ref-tip">[41]</a>,<a href="#pgen.1000879-Lin1" class="ref-tip">[42]</a>. miR-218 is an intronic miRNA. Two genes code for mature miR-218, miR-218-1 and miR-218-2, which are located within intron 15 of Slit2 and intron 14 of Slit3, respectively (<a href="#pgen-1000879-g006">Figure 6A</a>). The intronic location of the two miR-218 genes prompted us to ask whether miR-218-1 and miR-218-2 are transcribed together with their host gene mRNAs. To test this hypothesis, we used qRT-PCR to examine the expression of the miR-218-1 precursor, the miR-218-2 precursor, mature miR-218, Slit2 mRNA, and Slit3 mRNA in the GC tissues used in the survival analysis. Statistical analysis of the correlation coefficient of the qRT-PCR results revealed a significant positive correlation between the levels of Slit2 mRNA and miR-218-1 and between the levels of Slit3 mRNA and miR-218-2 (<a href="#pgen-1000879-g006">Figure 6B and 6C</a>). These results indicate that the miR-218 coding genes, miR-218-1 and miR-218-2, are transcribed together with their host genes, Slit2 and Slit3, respectively. A significant positive correlation between the levels of miR-218 and miR-218-2 (<a href="#pgen-1000879-g006">Figure 6D</a>) was seen in GC; however, no such correlation was seen between the levels of miR-218 and miR-218-1 (<a href="#pgen-1000879-g006">Figure 6E</a>). These results indicate that downregulation of miR-218 in GC is promoted by a decrease in miR-218-2, but not in miR-218-1. Consistent with this conclusion, Slit3 expression was significantly reduced in GC (−22.43±0.21, mean ± SE) compared to normal gastric tissue (−20.79±0.23, mean ± SE), (<em>P</em>&lt;0.0001, t = 7.67, paired <em>t</em>-test) (<a href="#pgen-1000879-g006">Figure 6F</a>), whereas Slit2 expression was not significantly different (<em>P</em> = 0.0772, paired <em>t</em>-test) (<a href="#pgen-1000879-g006">Figure 6G</a>). In summary, our experimental results suggest that significant upregulation of the Robo1 gene in response to removal of miR-218 may induce a subsequent upregulation of the Slit-Robo1 pathway through its interaction with Slit2, facilitating tumor cell migration and invasion.</p>
<a class="link-target" id="pgen-1000879-g006" name="pgen-1000879-g006"></a><div class="figure" data-doi="10.1371/journal.pgen.1000879.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1000879.g006" data-doi="info:doi/10.1371/journal.pgen.1000879" data-uri="info:doi/10.1371/journal.pgen.1000879.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1000879.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1000879.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1000879.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1000879.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 6. </span> Results of the expression analysis of miR-218, miR-218-1, miR-218-2, Slit2, and Slit3 in 40 matched GC tumors and corresponding normal tissues via qRT–PCR.</div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1" class="link-target"></a><p>(A) Schematic representation of the miR-218 genomic locus hosted in the intron of Slit. Expression patterns of Slit2 with miR-218-1 (B) and Slit3 with miR-218-2 (C) exhibited a significant positive correlation, as did mature miR-218 with the miR-218-2 precursor (D), but not with miR-218-1 (E), in GC. A significant differential gene expression pattern was detected between normal and tumor samples with regard to Slit3 (<em>P</em>&lt;0.0001, paired Student's <em>t</em>-test, <a href="#pgen-1000879-g006">Figure 6F</a>), but not Slit2 (<em>P</em> = 0.0772, <a href="#pgen-1000879-g006">Figure 6G</a>). Using relative quantification methods, the results were expressed as –ΔCt. The left and right lines of (F,G) represent the mean values for the normal and tumor groups, respectively.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1000879.g006">
              https://doi.org/10.1371/journal.pgen.1000879.g006</a></p></div></div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>To study a disease, it is vital to construct an ideal model. In the present study, we isolated invasive and non-invasive cell subpopulations from established human GC cell lines using the repeated transwell approach, which has been successfully applied in many studies investigating tumor metastasis <a href="#pgen.1000879-Chu1" class="ref-tip">[43]</a>–<a href="#pgen.1000879-Yu1" class="ref-tip">[48]</a>. The results of metastatic examination <em>in vitro</em> and <em>in vivo</em> showed that the established cell sublines had distinct invasive and metastatic capabilities. Here, we screened not only cell sublines derived from GC cell lines with high-invasive potential, but also those with low-invasive potential. With the exception of their metastatic abilities, the selected cell sublines were both quite similar, since they share the same genetic background. Since the major difference between the two types of sublines is metastatic capability, the genes that differ between them should correlate well with metastasis. Moreover, our method is able to distinguish invasion stages from metastasis and enables the study of specific steps in metastasis, which cannot be assessed in the live-animal model.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>Recently, miRNAs have been reported to promote <a href="#pgen.1000879-Zhu1" class="ref-tip">[49]</a>,<a href="#pgen.1000879-Hiyoshi1" class="ref-tip">[50]</a> or suppress <a href="#pgen.1000879-Crawford1" class="ref-tip">[51]</a>–<a href="#pgen.1000879-Gebeshuber1" class="ref-tip">[54]</a> tumor metastasis, providing a new perspective on the metastatic process. Nonetheless, the role of miRNAs in GC metastasis is lacking. In this report, we explored and obtained for the first time 45 metastasis-related miRNAs in GC based on a well-established metastasis cell model. The finding that miR-218 was downregulated in metastatic GC is intriguing, as decreased miR-218 levels have been reported in several types of solid tumors <a href="#pgen.1000879-Volinia1" class="ref-tip">[24]</a>–<a href="#pgen.1000879-Yanaihara1" class="ref-tip">[27]</a>,<a href="#pgen.1000879-Dalmay1" class="ref-tip">[55]</a>, indicating that the loss of miR-218 may be a common event in tumorigenesis. In the present study, we focused on the effect of miR-218 on GC metastasis and demonstrated that miR-218 acts as a tumor suppressor in GC metastasis. Restoration of miR-218 reduced cell migration and invasion <em>in vitro</em> and tumor metastasis <em>in vivo</em>. To obtain stable cell lines that over-expressed miR-218, we transfected MKN28-M cells with miR-218 plasmids and screened by G418. We selected twelve cell colonies in the miR-218-transfected group and found 10 out of 12 colonies exhibited remarkably uniform, stable and high-level expression of miR-218. Furthermore, three randomly chosen monoclonal cell lines exhibited similar reduction in invasive ablility. However, plasmid transfection strategies often result in lower integration efficiency compared to viral expression leading to the possibility of stochastic selection of rare functionally heterogeneous variants from the initial bulk population. Therefore the future use of viral expression systems should create a more unbiased starting population to test our hypothesis.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>As part of our research on how the loss of miR-218 affects GC metastasis, we demonstrated that Robo1 was a critical downstream target of miR-218. It is known that Robo is an axon guidance receptor for Slit and is conserved in animals ranging from fruit flies to mammals. In mammals, three Slit (Slit1–3) and four Robo (Robo1–4) genes have been described <a href="#pgen.1000879-Huminiecki1" class="ref-tip">[56]</a>,<a href="#pgen.1000879-Brose1" class="ref-tip">[57]</a>. The Slit–Robo interactions convey signals mediating repulsive cues on axons and growth cones during neural development and participate in T cell and monocyte chemotaxis <a href="#pgen.1000879-Kidd1" class="ref-tip">[58]</a>–<a href="#pgen.1000879-Prasad1" class="ref-tip">[64]</a>. As for other developmental pathways, aberrant expression of the Slit<em>-</em>Robo genes has been observed in a variety of tumor types <a href="#pgen.1000879-Rossi1" class="ref-tip">[65]</a>–<a href="#pgen.1000879-Prasad2" class="ref-tip">[68]</a>. For instance, in breast carcinoma tissue samples, Robo1 has been shown to be over-expressed, and it has been demonstrated to induce migration of breast cancer cell lines <a href="#pgen.1000879-Schmid1" class="ref-tip">[37]</a>. Slit2-Robo1 signaling facilitates glioma cell migration <a href="#pgen.1000879-Mertsch1" class="ref-tip">[36]</a> and is involved in angiogenesis by increasing microvessel density and tumor mass in a tumor xenograft model <a href="#pgen.1000879-Wang1" class="ref-tip">[30]</a>. Wang <em>et al.</em> demonstrated that the over-expression of Robo1 in new blood vessels in tumors induces cancer neovascularization and growth via an interaction between Robo1 and its ligand, Slit2. They also identified phosphoinositide-3-kinase (PI-3K) as a downstream effector of Slit2/Robo1 signaling. This suggests that there exists a Slit–Robo1–PI-3K cascade that could lead to the generation of phosphoinositol-3,4,5-triphosphate and the subsequent activation of small GTPases that mediate cell movement and remodeling of the actin cytoskeleton <a href="#pgen.1000879-Wang1" class="ref-tip">[30]</a>,<a href="#pgen.1000879-Legg1" class="ref-tip">[35]</a>. In contrast, other studies have argued that the downregulation of Robo1 caused by deletions or epigenetic modifications may play a role in tumor progression <a href="#pgen.1000879-Dallol1" class="ref-tip">[69]</a>–<a href="#pgen.1000879-Jin1" class="ref-tip">[73]</a>. We propose that a tissue-specific expression pattern exists for the Slit-Robo genes.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>In the current study, we found that Robo1 was often expressed at high levels in invasive cells and at low levels in non-invasive cells, whereas miR-218 displayed the opposite expression pattern. When we transfected the miR-218-expression vector and inhibiting oligonucleotides into MKN28-M and MKN28-NM cells, respectively, an inverse expression pattern was observed between miR-218 and Robo1, that is, if miR-218 expression was high, Robo1 expression was low and vice versa. This result was further confirmed in clinical samples and in luciferase activity assays. We also noticed that induction of expression of Robo1 by the Robo1 mutant construct without the miR-218 binding site could reverse miR-218-mediated suppression of tumor cell invasion. In contrast, knockdown of Robo1 gene expression by RNAi had an effect on reducing tumor cell invasion similar to that of the restoration of miR-218, although Robo1 knockdown alone demonstrated a weak effect. This could be because Robo1 is not the only target of miR-218 that is relevant to tumor metastasis. These findings indicate that the invasion suppression effect of miR-218 is at least partly mediated through a decrease in Robo1 expression. This is also the first study to show that the tumor-associated gene Robo1 is negatively regulated by miR-218 via a specific target site (nt 971–978) within the 3′-UTR. The Robo1 receptor is crucial for the response to extracellular signals and cellular phenotypic changes; therefore, tight regulation of the Robo1 receptor by miR-218 may facilitate more robust signal transduction. Anti-Robo1 monoclonal antibody has been reported to be an effective treatment for Robo1-expressing cancers <a href="#pgen.1000879-Wang1" class="ref-tip">[30]</a>,<a href="#pgen.1000879-Ito1" class="ref-tip">[33]</a>. In the present study, we found a new inhibitor of Robo1, miR-218, that may potentially be used to treat some types of cancer.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>As mentioned above, Slit1, Slit2, and Slit3 comprise the Slit family of proteins. Although the genes overlap, their expression patterns and functions are distinct. The former two proteins are known to be involved in axon guidance and cell migration <a href="#pgen.1000879-Schmid1" class="ref-tip">[37]</a>,<a href="#pgen.1000879-Andrews1" class="ref-tip">[74]</a>, while Slit3 is involved in the development of organs and organ systems, including the diaphragm and the kidney <a href="#pgen.1000879-Liu1" class="ref-tip">[75]</a>. In agreement with these data, we found that Slit2, but not Slit3, interacted with Robo1 to promote GC invasion.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6" class="link-target"></a><p>In addition, we demonstrated that the miR-218 coding genes were located in and transcribed together with Slit genes, which were Robo1 ligands, thus creating a negative feedback loop that regulates Slit/Robo1 signaling. Sailen Barik demonstrated that an intronic miRNA, miR-338, silenced genes that are functionally antagonistic to its host gene product, thus creating a positive feedback loop that assists in the physiological role of the host gene <a href="#pgen.1000879-Barik1" class="ref-tip">[76]</a>. However, a miRNA hosted in a ligand gene that simultaneously targets its corresponding receptor gene has never been reported. We have identified a negative regulatory loop involving the ligand (host gene), the intronic miRNA, and the receptor, in which the miRNA is co-transcribed with the ligand while receptor expression is repressed. The associations among ligand/receptor and intronic miRNA indicate that the early steps in the information flow may have built-in controls to limit excess signal propagation, which include a negative feedback loop to preserve homeostasis. This regulatory model, which is based on intronic miRNAs, is a novel mechanism of regulation in receptor signaling systems. Even though the full regulatory circuitry of miR-218 has yet to be completely elucidated, our study revealed a potential negative feedback pattern in which a miRNA was co-transcribed with the Slits ligand and repressed Robo1 receptor expression. This study not only provides new insights into the metastatic mechanism of GC but also generates a set of testable hypotheses that are helpful for understanding the miRNA-mediated regulation of cellular ligand/receptor interactions.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7" class="link-target"></a><p>In conclusion, we have identified miRNAs that are aberrantly expressed in invasive GC cells compared with non-invasive GC cells. Here we have showed that when significantly downregulated, miR-218 promoted GC cell invasion and metastasis, at least in part via induction of Robo1. This result indicates that restoration of miR-218 may be a rational therapeutic strategy for the treatment of GC in the future. It remains to be investigated whether the other differentially expressed miRNAs found in this study also participate in GC metastasis. Importantly, our findings have implications for describing new mechanisms for miRNA-mediated regulation of receptor signaling.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="Ethics statement"></a>
<h3>Ethics statement</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>All experimental procedures were approved by the Institutional Review Board of the Fourth Military Medical University. Written informed consent was obtained for all patient samples. Animal experiments were performed with the approval of the Institutional Committee for Animal Research and in conformity with national guidelines for the care and use of laboratory animals.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="Cell culture"></a>
<h3>Cell culture</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>The human GC cell lines MKN28 and SGC7901 were routinely maintained in RPMI-1640 medium (GIBCO, Carlsbad, CA, USA) supplemented with 10% fetal bovine serum, 100 U/ml of penicillin sodium, and 100 µg/ml of streptomycin sulfate at 37°C in a humidified air atmosphere containing 5% carbon dioxide. Throughout the experiment, cells were used in the logarithmic phase of growth.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="Isolation of invasive and non-invasive cell sublines using transwell chambers"></a>
<h3>Isolation of invasive and non-invasive cell sublines using transwell chambers</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Six-well polycarbonate transwell membrane inserts with 8-µm pores (Corning, USA) were used to isolate cell sublines with different levels of invasiveness from the cultured MKN28 cell line. First, cells that were serum-starved for 24 h were suspended in serum-free RPMI-1640 to a final cell density of 5×10<sup>5</sup> cells/mL. A 1 mL cell suspension was seeded into the top chamber, which was coated with 200 mg/mL of Matrigel (BD Biosciences, San Jose, CA, USA), and the lower well beneath the polycarbonate membranes was filled with 2.5 mL RPMI-1640 medium supplemented with 20% bovine serum to create a chemotactic gradient to stimulate penetration of the cells. Following incubation for 24 h at 37°C, the invasive cells on the underside of the membrane and the non-invasive cells on the top of the membrane were harvested aseptically and were expanded for selection. Via ten-round selection, the cell subline that failed to invade through the membranes in all selection rounds was designated as MKN28-NM, and the subline that was able to migrate through the membranes was designated as MKN28-M. We also obtained two cell sublines, SGC7901-M and SGC7901-NM, derived from the SGC7901 GC cell line, using the same method (<a href="#pgen-1000879-g001">Figure 1</a>).</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="In vitro migration and invasion assays"></a>
<h3><em>In vitro</em> migration and invasion assays</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>A 24-well transwell plate (8-µm pore size, Corning, USA) was used to measure each cell line's migratory and invasive ability. For transwell migration assays, 2.5×10<sup>4</sup> cells were plated in the top chamber lined with a non-coated membrane. For invasion assays, chamber inserts were coated with 200 mg/mL of Matrigel and dried overnight under sterile conditions. Then, 5×10<sup>4</sup> cells were plated in the top chamber. In both assays, cells were suspended in medium without serum or growth factors, and medium supplemented with serum was used as a chemoattractant in the lower chamber. After incubation at 37°C for 24 h, the top chambers were wiped with cotton wool to remove the non-migratory or non-invasive cells. The invading cells on the underside of the membrane were fixed in 100% methanol for 10 min, air-dried, stained in 0.1% crystal violet, and counted under a microscope. The mean of triplicate assays for each experimental condition was used.</p>
</div>

<div id="section5" class="section toc-section"><a id="s4e" name="s4e" class="link-target" title="Experimental metastasis"></a>
<h3>Experimental metastasis</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>To produce experimental metastasis, cells were washed and resuspended in PBS. Five-week-old BALB/C-nu/nu nude mice obtained from the Shanghai Laboratory Animal Center of China were injected into the lateral tail vein, and the animals were maintained in a sterile animal facility. Each tumor cell subline was injected into ten mice. After ten weeks, the mice were killed, and the lungs and liver were examined for metastases. Tumor tissues derived from various organs were dissected and examined histologically. The experiments were repeated two to three times.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4f" name="s4f" class="link-target" title="miRNA microarrays"></a>
<h3>miRNA microarrays</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>Total RNA was extracted from each cell subline using the miRVana miRNA Isolation Kit (Ambion Inc., Austin, TX, USA) according to the manufacturer's instructions. The purity and quantity of the isolated RNAs were assessed using 1% formaldehyde-agarose gel electrophoresis and spectrophotometry (Bio-Rad, Hercules, CA, USA). We then submitted the samples to KangChen-Biotech (Shanghai, China) for array hybridization on a miRCURY LNA microRNA array (v.10.0, Exiqon, Vedbaek, Denmark). Each microarray chip was hybridized with a single sample labeled with either Cy3 or Cy5. Background subtraction and normalization were performed. We selected miRNAs whose expression levels between invasive cell sublines and non-invasive cell sublines differed by at least 1.5-fold.</p>
</div>

<div id="section7" class="section toc-section"><a id="s4g" name="s4g" class="link-target" title="Clinical samples"></a>
<h3>Clinical samples</h3>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1" class="link-target"></a><p>Forty patients (30 males and 10 females) who had undergone gastrectomy with lymph node dissection for gastric carcinoma at Xijing Hospital between March and September of 2003 were included in the study (<a href="#pgen.1000879.s007">Table S2</a>). The patients ranged in age from 26 to 77 years (median 54.13 years). None of the patients received preoperative chemotherapy. The resected specimens were histologically examined by H&amp;E staining. The primary tumor tissues and corresponding non-tumor mucosa and lymph nodes were collected from each patient immediately after surgical removal and snap-frozen in liquid nitrogen until further use. Total RNA from the frozen tissues was isolated with Trizol (Invitrogen, Carlsbad, CA, USA) according to the manufacturer's instructions.</p>
</div>

<div id="section8" class="section toc-section"><a id="s4h" name="s4h" class="link-target" title="Quantitative real-time RT–PCR (qRT–PCR)"></a>
<h3>Quantitative real-time RT–PCR (qRT–PCR)</h3>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1" class="link-target"></a><p>Total RNA was extracted with suitable reagents. The TaqMan stem-loop RT-PCR method was used to assess the expression of miRNAs with kits from Applied Biosystems (Foster City, CA, USA). SYBR green real-time RT-PCR was performed to detect Slit2, Slit3, and Robo1. All RT-PCR experiments were performed on a Chromo4 Real-Time PCR Detection System (Bio-Rad, Hercules, CA, USA). The primers for miR-218 and its precursors were obtained from Applied Biosystems(Foster City, California, USA)and Eurogentec North America, Inc (Flintkote Avenue, San Diego, California, USA), respectively. The primers for Slit2, Slit3, and Robo1 were designed to produce amplicons that were 76–150 bp in length and with an annealing temperature of approximately 60°C using Primer Premier v5.0 Software. Data are presented as fold differences relative to either 18S for Slit2, Slit3, and Robo1 or U6 for miRNA based on calculations of 2<sup>−ΔΔCt</sup>. All primer sequences in this study are listed in <a href="#pgen.1000879.s008">Table S3</a>.</p>
</div>

<div id="section9" class="section toc-section"><a id="s4i" name="s4i" class="link-target" title="Construct design and cell transfections"></a>
<h3>Construct design and cell transfections</h3>

<div id="section1" class="section toc-section"><a id="s4i1" name="s4i1" class="link-target" title="miR-218-expressing vector"></a><h4>miR-218-expressing vector.</h4><a id="article1.body1.sec4.sec9.sec1.p1" name="article1.body1.sec4.sec9.sec1.p1" class="link-target"></a><p>The precursor sequence of miR-218 (110 bp, MI0000295) generated by annealing and primer extension with miR-218-precursor-F and miR-218-precursor-R (<a href="#pgen.1000879.s008">Table S3</a>) was digested with BamHI and HindIII and cloned into the BamHI-HindIII fragment of the pGenesil-1 vector. A construct including the nonspecific miRNA cel-miR-67 (99 bp, MI0000038) was used as a negative control.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4i2" name="s4i2" class="link-target" title="Luc-Robo1 vector"></a><h4>Luc-Robo1 vector.</h4><a id="article1.body1.sec4.sec9.sec2.p1" name="article1.body1.sec4.sec9.sec2.p1" class="link-target"></a><p>The Robo1 3′-UTR containing the predicted miR-218 binding site was amplified by RT-PCR from the total RNA of cultured MKN28 cells and was cloned into the pCR2.1-TOPO vector (Invitrogen, Carlsbad, CA, USA). The pCR2.1-TOPO-Robo1 3′-UTR construct was digested with SpeI and HindIII. The resulting fragment was subcloned into the SpeI and HindIII sites of the pMIR-REPORT miRNA expression reporter vector (Applied Biosystems). The first six nucleotides complementary to the miR-218 seed-region were deleted from the mutant constructs using the QuikChange Site-Directed Mutagenesis Kit (Stratagene) according to the manufacturer's protocol.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4i3" name="s4i3" class="link-target" title="Robo1-expressing vector with or without miR-218 binding sites"></a><h4>Robo1-expressing vector with or without miR-218 binding sites.</h4><a id="article1.body1.sec4.sec9.sec3.p1" name="article1.body1.sec4.sec9.sec3.p1" class="link-target"></a><p>Full-length Robo1 cDNA that entirely lacks the 3′-UTR (Clone ID: 9057080) was purchased from Open Biosystems (USA) and was subcloned into the eukaryotic expression vector pcDNA3.1(+) to generate the Robo1 mutant expression vector. A Robo1-expressing vector was constructed by inserting the fragment of the predicted miR-218 binding site into the Robo1 mutant expression vector. Robo1 siRNA and negative control oligonucleotides were purchased from GenePharma (Shanghai, China).</p>
</div>

<div id="section4" class="section toc-section"><a id="s4i4" name="s4i4" class="link-target" title="Cell transfections"></a><h4>Cell transfections.</h4><a id="article1.body1.sec4.sec9.sec4.p1" name="article1.body1.sec4.sec9.sec4.p1" class="link-target"></a><p>MKN28-M cells were transfected with the miR-218-expressing vector or the control vector expressing a nonspecific miRNA, cel-miR-67, using Lipofectamine 2000 (Invitrogen), and were selected with 400 mg/L G418 to generate two stable monoclonal cell lines (a miR-218 stable cell line, MKN28-M-miR-218, and a control stable cell line, MKN28-M-miR-control). The oligonucleotides comprising the miR-218 inhibitor and the mismatched sequence negative control were purchased from Ambion Inc. and were transfected into MKN28-NM cells using Oligofectamine (Invitrogen). The Robo1-expressing vector with or without the miR-218 binding sites was transfected into MKN28-M-miR-218 cells, and Robo1 siRNA was transfected into MKN28-M cells<strong>.</strong></p>
</div>

<div id="section5" class="section toc-section"><a id="s4i5" name="s4i5" class="link-target" title="Luciferase assay"></a><h4>Luciferase assay.</h4><a id="article1.body1.sec4.sec9.sec5.p1" name="article1.body1.sec4.sec9.sec5.p1" class="link-target"></a><p>The pMIR-REPORT β-galactosidase control vector and Luc-Robo1, Luc-Robo1-mu, or Luc-control were co-transfected into MKN28-M-miR-218 cells. Lysates were prepared at 48 h post-transfection. Luciferase activity was measured using the Dual-Light luminescent reporter gene assay (Applied Biosystems). All measurements were normalized to β-galactosidase activities to correct for variations in transfection efficiencies and for non-miR-218-specific effects of miRNA transfection on enzymatic activity.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4i6" name="s4i6" class="link-target" title="Western blot"></a><h4>Western blot.</h4><a id="article1.body1.sec4.sec9.sec6.p1" name="article1.body1.sec4.sec9.sec6.p1" class="link-target"></a><p>Cellular proteins were extracted and separated in SDS-PAGE gels, and western blot analyses were performed according to standard procedures. Western blotting of β-actin on the same membrane was used as a loading control. The antibodies used were anti-Robo1 (SC-25672) and anti-β-actin (sc-47778), both from Santa Cruz Biotechnology (CA, USA).</p>
</div>

<div id="section7" class="section toc-section"><a id="s4i7" name="s4i7" class="link-target" title="Statistical analyses"></a><h4>Statistical analyses.</h4><a id="article1.body1.sec4.sec9.sec7.p1" name="article1.body1.sec4.sec9.sec7.p1" class="link-target"></a><p>All data are presented as means ± SE and were analyzed using Prism 5.0 software (GraphPad). The significance of the observed differences was determined with the Student's <em>t</em>-test or the χ2 test. The relationships among the miR-218-1 precursor, the miR-218-2 precursor, mature miR-218, Slit2 mRNA, and Slit3 mRNA were analyzed by correlation coefficients and linear regression analysis. <em>P</em>&lt;0.05 was considered statistically significant. * <em>P</em>&lt;0.05; ** <em>P</em>&lt;0.01.</p>
</div>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pgen.1000879"></div><div class="supplementary-material"><a name="pgen.1000879.s001" id="pgen.1000879.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">No significant difference in the proliferation rate was observed in the three cell sublines. (A) Proliferation rates of the cell sublines were detected by the MTT assay. (B) Tumor volume growth curves for each cell subline are shown. Tumor sizes were measured using calipers. Tumor volume was calculated using the formula (length × width<sup>2</sup>)/2. (<em>n</em> = 5, paired Wilcoxon test, <em>P</em>&gt;0.05). (C) On day 37, all tumors were collected to measure tumor weights (<em>P</em>&gt;0.05; <em>n</em> = 5). (D) Photos of tumors 37 days after injection with MKN28, MKN28-NM, or MKN28-M cells.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s001">https://doi.org/10.1371/journal.pgen.1000879.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(6.14 MB TIF)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s002" id="pgen.1000879.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s002">Figure S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Cell-cycle analysis of three established cell sublines. (A) Representative flow cytometry results for each cell subline. (B) Cell-cycle distribution (<em>P</em>&gt;0.05; <em>n</em> = 3).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s002">https://doi.org/10.1371/journal.pgen.1000879.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(2.82 MB TIF)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s003" id="pgen.1000879.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s003">Figure S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">qRT-PCR analysis of the relative expression of miR-218 and Robo1. (A) Expression of miR-218 and Robo1 were reversed in invasive (MKN28-M and SGC7901-M) and non-invasive GC cells (MKN28-NM and SGC7901-NM) compared with GES cells. (B) Robo1 mRNA levels decreased when miR-218 was upregulated in response to transfection of MKN28-M cells with miR-218-expressing vector.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s003">https://doi.org/10.1371/journal.pgen.1000879.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(2.56 MB TIF)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s004" id="pgen.1000879.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s004">Figure S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">Immunohistochemical analysis of Robo1. (A-F) represent normal and tumor tissues taken from the same patient and processed in the same way using paraffin sectioning. (A,C,E) H&amp;E staining of normal gastric mucosa, primary gastric cancer, and the ovarian metastasis from gastric cancer. Magnification, 100×. (B,D,F) Robo1 in normal gastric mucosa, primary gastric cancer, and ovarian metastasis from gastric cancer (serial section adjacent to the H&amp;E-stained specimen). Magnification, 100×. Robo1 was expressed at low levels in the gastric epithelial cells of normal tissues and was expressed at increased levels in gastric cancer tissues, especially in metastatic tumor tissues.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s004">https://doi.org/10.1371/journal.pgen.1000879.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(9.91 MB TIF)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s005" id="pgen.1000879.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s005">Figure S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI">miR-218 has the ability to specifically suppress metastasis without affecting cell proliferation. (A) MTT assay of the effects of miR-218 on proliferation of MKN28-M cells. No significant difference in the proliferation rate was found between MKN28-M-miR-218 cells stably over-expressing miR-218 and control cells. (B) MKN28-M-miR-218 and control cells were subcutaneously injected into nude mice. Growth curves of primary gastric cancers formed by MKN28-M-miR-218 cells or control cells are shown. Tumor sizes were measured using calipers. Tumor volume was calculated using the formula (length × width<sup>2</sup>)/2. Each data point represents the mean ± standard error (<em>n</em> = 10; <em>P</em>&gt;0.05). (C) Median tumor weight at day 72. Data are presented as mean ± standard error (<em>n</em> = 10; <em>P</em>&gt;0.05). (D) Cell cycle distribution (<em>P</em>&gt;0.05; <em>n</em> = 3). (E) Representative H&amp;E staining of lungs and livers isolated from mice that received injections of MKN28-M-miR-control or MKN28-M-miR-218 cells. Magnification, 200×.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s005">https://doi.org/10.1371/journal.pgen.1000879.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(3.55 MB TIF)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s006" id="pgen.1000879.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s006">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI">Differentially expressed miRNAs in highly invasive GC cells versus non-invasive GC cells.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s006">https://doi.org/10.1371/journal.pgen.1000879.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.10 MB DOC)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s007" id="pgen.1000879.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s007">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI">Clinicopathologic features in 40 tumor samples.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s007">https://doi.org/10.1371/journal.pgen.1000879.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.25 MB DOC)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s008" id="pgen.1000879.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s008">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI">Primer sequences used in the study.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s008">https://doi.org/10.1371/journal.pgen.1000879.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.04 MB DOC)</p>
</div><div class="supplementary-material"><a name="pgen.1000879.s009" id="pgen.1000879.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1000879.s009">Text S1. </a></h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI">Supplementary methods.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1000879.s009">https://doi.org/10.1371/journal.pgen.1000879.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.05 MB DOC)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank Zhao Yiling, Department of Pathology, Xijing Hospital, for histological assessment; Liang Shuhui for help with the animal experiment; Li Zeng, Cheng Ying, and Shen Xin for collecting the clinical samples; and He Biyun for assistance with statistical data analysis. We also thank Derek M. Erion for revising the manuscript.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: JT YP KW JL YN DF. Performed the experiments: JT YP LZ SS BW YG YZ TQ. Analyzed the data: JT QL DF. Contributed reagents/materials/analysis tools: XG QZ DF. Wrote the paper: JT.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pgen.1000879-Oue1" id="pgen.1000879-Oue1" class="link-target"></a>Oue N, Aung PP, Mitani Y, Kuniyasu H, Nakayama H, et al.  (2005) Genes involved in invasion and metastasis of gastric cancer identified by array-based hybridization and serial analysis of gene expression. Oncology  69: Suppl 117–22.  <ul class="reflinks"><li><a href="#" data-author="Oue" data-cit="OueNAungPPMitaniYKuniyasuHNakayamaH2005Genes%20involved%20in%20invasion%20and%20metastasis%20of%20gastric%20cancer%20identified%20by%20array-based%20hybridization%20and%20serial%20analysis%20of%20gene%20expression.Oncology69Suppl%2011722" data-title="Genes%20involved%20in%20invasion%20and%20metastasis%20of%20gastric%20cancer%20identified%20by%20array-based%20hybridization%20and%20serial%20analysis%20of%20gene%20expression." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genes+involved+in+invasion+and+metastasis+of+gastric+cancer+identified+by+array-based+hybridization+and+serial+analysis+of+gene+expression.+Oue+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pgen.1000879-Gupta1" id="pgen.1000879-Gupta1" class="link-target"></a>Gupta GP, Massague J (2006) Cancer metastasis: building a framework. Cell  127: 679–695.  <ul class="reflinks"><li><a href="#" data-author="Gupta" data-cit="GuptaGPMassagueJ2006Cancer%20metastasis%3A%20building%20a%20framework.Cell127679695" data-title="Cancer%20metastasis%3A%20building%20a%20framework." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cancer+metastasis%3A+building+a+framework.+Gupta+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pgen.1000879-Klein1" id="pgen.1000879-Klein1" class="link-target"></a>Klein CA (2008) Cancer. The metastasis cascade. Science  321: 1785–1787.  <ul class="reflinks"><li><a href="#" data-author="Klein" data-cit="KleinCA2008Cancer.%20The%20metastasis%20cascade.Science32117851787" data-title="Cancer.%20The%20metastasis%20cascade." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cancer.+The+metastasis+cascade.+Klein+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pgen.1000879-Steeg1" id="pgen.1000879-Steeg1" class="link-target"></a>Steeg PS (2006) Tumor metastasis: mechanistic insights and clinical challenges. Nat Med  12: 895–904.  <ul class="reflinks"><li><a href="#" data-author="Steeg" data-cit="SteegPS2006Tumor%20metastasis%3A%20mechanistic%20insights%20and%20clinical%20challenges.Nat%20Med12895904" data-title="Tumor%20metastasis%3A%20mechanistic%20insights%20and%20clinical%20challenges." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor+metastasis%3A+mechanistic+insights+and+clinical+challenges.+Steeg+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pgen.1000879-Nicoloso1" id="pgen.1000879-Nicoloso1" class="link-target"></a>Nicoloso MS, Spizzo R, Shimizu M, Rossi S, Calin GA (2009) MicroRNAs - the micro steering wheel of tumour metastases Nat Rev Cancer.   <ul class="reflinks"><li><a href="#" data-author="Nicoloso" data-cit="NicolosoMSSpizzoRShimizuMRossiSCalinGA2009MicroRNAs%20-%20the%20micro%20steering%20wheel%20of%20tumour%20metastasesNat%20Rev%20Cancer" data-title="MicroRNAs%20-%20the%20micro%20steering%20wheel%20of%20tumour%20metastases" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+-+the+micro+steering+wheel+of+tumour+metastases+Nicoloso+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pgen.1000879-Ma1" id="pgen.1000879-Ma1" class="link-target"></a>Ma L, Teruya-Feldstein J, Weinberg RA (2007) Tumour invasion and metastasis initiated by microRNA-10b in breast cancer. Nature  449: 682–688.  <ul class="reflinks"><li><a href="#" data-author="Ma" data-cit="MaLTeruya-FeldsteinJWeinbergRA2007Tumour%20invasion%20and%20metastasis%20initiated%20by%20microRNA-10b%20in%20breast%20cancer.Nature449682688" data-title="Tumour%20invasion%20and%20metastasis%20initiated%20by%20microRNA-10b%20in%20breast%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tumour+invasion+and+metastasis+initiated+by+microRNA-10b+in+breast+cancer.+Ma+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pgen.1000879-Huang1" id="pgen.1000879-Huang1" class="link-target"></a>Huang Q, Gumireddy K, Schrier M, le Sage C, Nagel R, et al.  (2008) The microRNAs miR-373 and miR-520c promote tumour invasion and metastasis. Nat Cell Biol  10: 202–210.  <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangQGumireddyKSchrierMle%20SageCNagelR2008The%20microRNAs%20miR-373%20and%20miR-520c%20promote%20tumour%20invasion%20and%20metastasis.Nat%20Cell%20Biol10202210" data-title="The%20microRNAs%20miR-373%20and%20miR-520c%20promote%20tumour%20invasion%20and%20metastasis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+microRNAs+miR-373+and+miR-520c+promote+tumour+invasion+and+metastasis.+Huang+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pgen.1000879-Lee1" id="pgen.1000879-Lee1" class="link-target"></a>Lee DY, Deng Z, Wang CH, Yang BB (2007) MicroRNA-378 promotes cell survival, tumor growth, and angiogenesis by targeting SuFu and Fus-1 expression. Proc Natl Acad Sci U S A  104: 20350–20355.  <ul class="reflinks"><li><a href="#" data-author="Lee" data-cit="LeeDYDengZWangCHYangBB2007MicroRNA-378%20promotes%20cell%20survival%2C%20tumor%20growth%2C%20and%20angiogenesis%20by%20targeting%20SuFu%20and%20Fus-1%20expression.Proc%20Natl%20Acad%20Sci%20U%20S%20A1042035020355" data-title="MicroRNA-378%20promotes%20cell%20survival%2C%20tumor%20growth%2C%20and%20angiogenesis%20by%20targeting%20SuFu%20and%20Fus-1%20expression." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-378+promotes+cell+survival%2C+tumor+growth%2C+and+angiogenesis+by+targeting+SuFu+and+Fus-1+expression.+Lee+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pgen.1000879-Gregory1" id="pgen.1000879-Gregory1" class="link-target"></a>Gregory PA, Bert AG, Paterson EL, Barry SC, Tsykin A, et al.  (2008) The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1. Nat Cell Biol  10: 593–601.  <ul class="reflinks"><li><a href="#" data-author="Gregory" data-cit="GregoryPABertAGPatersonELBarrySCTsykinA2008The%20miR-200%20family%20and%20miR-205%20regulate%20epithelial%20to%20mesenchymal%20transition%20by%20targeting%20ZEB1%20and%20SIP1.Nat%20Cell%20Biol10593601" data-title="The%20miR-200%20family%20and%20miR-205%20regulate%20epithelial%20to%20mesenchymal%20transition%20by%20targeting%20ZEB1%20and%20SIP1." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+miR-200+family+and+miR-205+regulate+epithelial+to+mesenchymal+transition+by+targeting+ZEB1+and+SIP1.+Gregory+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pgen.1000879-Wu1" id="pgen.1000879-Wu1" class="link-target"></a>Wu H, Zhu S, Mo YY (2009) Suppression of cell growth and invasion by miR-205 in breast cancer. Cell Res.   <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuHZhuSMoYY2009Suppression%20of%20cell%20growth%20and%20invasion%20by%20miR-205%20in%20breast%20cancer.Cell%20Res" data-title="Suppression%20of%20cell%20growth%20and%20invasion%20by%20miR-205%20in%20breast%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Suppression+of+cell+growth+and+invasion+by+miR-205+in+breast+cancer.+Wu+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pgen.1000879-Hurst1" id="pgen.1000879-Hurst1" class="link-target"></a>Hurst DR, Edmonds MD, Scott GK, Benz CC, Vaidya KS, et al.  (2009) Breast cancer metastasis suppressor 1 up-regulates miR-146, which suppresses breast cancer metastasis. Cancer Res  69: 1279–1283.  <ul class="reflinks"><li><a href="#" data-author="Hurst" data-cit="HurstDREdmondsMDScottGKBenzCCVaidyaKS2009Breast%20cancer%20metastasis%20suppressor%201%20up-regulates%20miR-146%2C%20which%20suppresses%20breast%20cancer%20metastasis.Cancer%20Res6912791283" data-title="Breast%20cancer%20metastasis%20suppressor%201%20up-regulates%20miR-146%2C%20which%20suppresses%20breast%20cancer%20metastasis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Breast+cancer+metastasis+suppressor+1+up-regulates+miR-146%2C+which+suppresses+breast+cancer+metastasis.+Hurst+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pgen.1000879-Prueitt1" id="pgen.1000879-Prueitt1" class="link-target"></a>Prueitt RL, Yi M, Hudson RS, Wallace TA, Howe TM, et al.  (2008) Expression of microRNAs and protein-coding genes associated with perineural invasion in prostate cancer. Prostate  68: 1152–1164.  <ul class="reflinks"><li><a href="#" data-author="Prueitt" data-cit="PrueittRLYiMHudsonRSWallaceTAHoweTM2008Expression%20of%20microRNAs%20and%20protein-coding%20genes%20associated%20with%20perineural%20invasion%20in%20prostate%20cancer.Prostate6811521164" data-title="Expression%20of%20microRNAs%20and%20protein-coding%20genes%20associated%20with%20perineural%20invasion%20in%20prostate%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+of+microRNAs+and+protein-coding+genes+associated+with+perineural+invasion+in+prostate+cancer.+Prueitt+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pgen.1000879-Li1" id="pgen.1000879-Li1" class="link-target"></a>Li T, Li D, Sha J, Sun P, Huang Y (2009) MicroRNA-21 directly targets MARCKS and promotes apoptosis resistance and invasion in prostate cancer cells. Biochem Biophys Res Commun.   <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiTLiDShaJSunPHuangY2009MicroRNA-21%20directly%20targets%20MARCKS%20and%20promotes%20apoptosis%20resistance%20and%20invasion%20in%20prostate%20cancer%20cells.Biochem%20Biophys%20Res%20Commun" data-title="MicroRNA-21%20directly%20targets%20MARCKS%20and%20promotes%20apoptosis%20resistance%20and%20invasion%20in%20prostate%20cancer%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-21+directly+targets+MARCKS+and+promotes+apoptosis+resistance+and+invasion+in+prostate+cancer+cells.+Li+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pgen.1000879-Sengupta1" id="pgen.1000879-Sengupta1" class="link-target"></a>Sengupta S, den Boon JA, Chen IH, Newton MA, Stanhope SA, et al.  (2008) MicroRNA 29c is down-regulated in nasopharyngeal carcinomas, up-regulating mRNAs encoding extracellular matrix proteins. Proc Natl Acad Sci U S A  105: 5874–5878.  <ul class="reflinks"><li><a href="#" data-author="Sengupta" data-cit="SenguptaSden%20BoonJAChenIHNewtonMAStanhopeSA2008MicroRNA%2029c%20is%20down-regulated%20in%20nasopharyngeal%20carcinomas%2C%20up-regulating%20mRNAs%20encoding%20extracellular%20matrix%20proteins.Proc%20Natl%20Acad%20Sci%20U%20S%20A10558745878" data-title="MicroRNA%2029c%20is%20down-regulated%20in%20nasopharyngeal%20carcinomas%2C%20up-regulating%20mRNAs%20encoding%20extracellular%20matrix%20proteins." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+29c+is+down-regulated+in+nasopharyngeal+carcinomas%2C+up-regulating+mRNAs+encoding+extracellular+matrix+proteins.+Sengupta+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pgen.1000879-Veerla1" id="pgen.1000879-Veerla1" class="link-target"></a>Veerla S, Lindgren D, Kvist A, Frigyesi A, Staaf J, et al.  (2009) MiRNA expression in urothelial carcinomas: important roles of miR-10a, miR-222, miR-125b, miR-7 and miR-452 for tumor stage and metastasis, and frequent homozygous losses of miR-31. Int J Cancer  124: 2236–2242.  <ul class="reflinks"><li><a href="#" data-author="Veerla" data-cit="VeerlaSLindgrenDKvistAFrigyesiAStaafJ2009MiRNA%20expression%20in%20urothelial%20carcinomas%3A%20important%20roles%20of%20miR-10a%2C%20miR-222%2C%20miR-125b%2C%20miR-7%20and%20miR-452%20for%20tumor%20stage%20and%20metastasis%2C%20and%20frequent%20homozygous%20losses%20of%20miR-31.Int%20J%20Cancer12422362242" data-title="MiRNA%20expression%20in%20urothelial%20carcinomas%3A%20important%20roles%20of%20miR-10a%2C%20miR-222%2C%20miR-125b%2C%20miR-7%20and%20miR-452%20for%20tumor%20stage%20and%20metastasis%2C%20and%20frequent%20homozygous%20losses%20of%20miR-31." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MiRNA+expression+in+urothelial+carcinomas%3A+important+roles+of+miR-10a%2C+miR-222%2C+miR-125b%2C+miR-7+and+miR-452+for+tumor+stage+and+metastasis%2C+and+frequent+homozygous+losses+of+miR-31.+Veerla+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pgen.1000879-Segura1" id="pgen.1000879-Segura1" class="link-target"></a>Segura MF, Hanniford D, Menendez S, Reavie L, Zou X, et al.  (2009) Aberrant miR-182 expression promotes melanoma metastasis by repressing FOXO3 and microphthalmia-associated transcription factor. Proc Natl Acad Sci U S A  106: 1814–1819.  <ul class="reflinks"><li><a href="#" data-author="Segura" data-cit="SeguraMFHannifordDMenendezSReavieLZouX2009Aberrant%20miR-182%20expression%20promotes%20melanoma%20metastasis%20by%20repressing%20FOXO3%20and%20microphthalmia-associated%20transcription%20factor.Proc%20Natl%20Acad%20Sci%20U%20S%20A10618141819" data-title="Aberrant%20miR-182%20expression%20promotes%20melanoma%20metastasis%20by%20repressing%20FOXO3%20and%20microphthalmia-associated%20transcription%20factor." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Aberrant+miR-182+expression+promotes+melanoma+metastasis+by+repressing+FOXO3+and+microphthalmia-associated+transcription+factor.+Segura+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pgen.1000879-Nass1" id="pgen.1000879-Nass1" class="link-target"></a>Nass D, Rosenwald S, Meiri E, Gilad S, Tabibian-Keissar H, et al.  (2008) MiR-92b and miR-9/9* Are Specifically Expressed in Brain Primary Tumors and Can Be Used to Differentiate Primary from Metastatic Brain Tumors. Brain Pathol.   <ul class="reflinks"><li><a href="#" data-author="Nass" data-cit="NassDRosenwaldSMeiriEGiladSTabibian-KeissarH2008MiR-92b%20and%20miR-9%2F9%2A%20Are%20Specifically%20Expressed%20in%20Brain%20Primary%20Tumors%20and%20Can%20Be%20Used%20to%20Differentiate%20Primary%20from%20Metastatic%20Brain%20Tumors.Brain%20Pathol" data-title="MiR-92b%20and%20miR-9%2F9%2A%20Are%20Specifically%20Expressed%20in%20Brain%20Primary%20Tumors%20and%20Can%20Be%20Used%20to%20Differentiate%20Primary%20from%20Metastatic%20Brain%20Tumors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MiR-92b+and+miR-9%2F9%2A+Are+Specifically+Expressed+in+Brain+Primary+Tumors+and+Can+Be+Used+to+Differentiate+Primary+from+Metastatic+Brain+Tumors.+Nass+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pgen.1000879-Asangani1" id="pgen.1000879-Asangani1" class="link-target"></a>Asangani IA, Rasheed SA, Nikolova DA, Leupold JH, Colburn NH, et al.  (2008) MicroRNA-21 (miR-21) post-transcriptionally downregulates tumor suppressor Pdcd4 and stimulates invasion, intravasation and metastasis in colorectal cancer. Oncogene  27: 2128–2136.  <ul class="reflinks"><li><a href="#" data-author="Asangani" data-cit="AsanganiIARasheedSANikolovaDALeupoldJHColburnNH2008MicroRNA-21%20%28miR-21%29%20post-transcriptionally%20downregulates%20tumor%20suppressor%20Pdcd4%20and%20stimulates%20invasion%2C%20intravasation%20and%20metastasis%20in%20colorectal%20cancer.Oncogene2721282136" data-title="MicroRNA-21%20%28miR-21%29%20post-transcriptionally%20downregulates%20tumor%20suppressor%20Pdcd4%20and%20stimulates%20invasion%2C%20intravasation%20and%20metastasis%20in%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-21+%28miR-21%29+post-transcriptionally+downregulates+tumor+suppressor+Pdcd4+and+stimulates+invasion%2C+intravasation+and+metastasis+in+colorectal+cancer.+Asangani+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pgen.1000879-Lewis1" id="pgen.1000879-Lewis1" class="link-target"></a>Lewis BP, Burge CB, Bartel DP (2005) Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. Cell  120: 15–20.  <ul class="reflinks"><li><a href="#" data-author="Lewis" data-cit="LewisBPBurgeCBBartelDP2005Conserved%20seed%20pairing%2C%20often%20flanked%20by%20adenosines%2C%20indicates%20that%20thousands%20of%20human%20genes%20are%20microRNA%20targets.Cell1201520" data-title="Conserved%20seed%20pairing%2C%20often%20flanked%20by%20adenosines%2C%20indicates%20that%20thousands%20of%20human%20genes%20are%20microRNA%20targets." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Conserved+seed+pairing%2C+often+flanked+by+adenosines%2C+indicates+that+thousands+of+human+genes+are+microRNA+targets.+Lewis+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pgen.1000879-Lai1" id="pgen.1000879-Lai1" class="link-target"></a>Lai EC (2002) Micro RNAs are complementary to 3′ UTR sequence motifs that mediate negative post-transcriptional regulation. Nat Genet  30: 363–364.  <ul class="reflinks"><li><a href="#" data-author="Lai" data-cit="LaiEC2002Micro%20RNAs%20are%20complementary%20to%203%E2%80%B2%20UTR%20sequence%20motifs%20that%20mediate%20negative%20post-transcriptional%20regulation.Nat%20Genet30363364" data-title="Micro%20RNAs%20are%20complementary%20to%203%E2%80%B2%20UTR%20sequence%20motifs%20that%20mediate%20negative%20post-transcriptional%20regulation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Micro+RNAs+are+complementary+to+3%E2%80%B2+UTR+sequence+motifs+that+mediate+negative+post-transcriptional+regulation.+Lai+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pgen.1000879-Kloosterman1" id="pgen.1000879-Kloosterman1" class="link-target"></a>Kloosterman WP, Plasterk RH (2006) The diverse functions of microRNAs in animal development and disease. Dev Cell  11: 441–450.  <ul class="reflinks"><li><a href="#" data-author="Kloosterman" data-cit="KloostermanWPPlasterkRH2006The%20diverse%20functions%20of%20microRNAs%20in%20animal%20development%20and%20disease.Dev%20Cell11441450" data-title="The%20diverse%20functions%20of%20microRNAs%20in%20animal%20development%20and%20disease." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+diverse+functions+of+microRNAs+in+animal+development+and+disease.+Kloosterman+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pgen.1000879-EsquelaKerscher1" id="pgen.1000879-EsquelaKerscher1" class="link-target"></a>Esquela-Kerscher A, Slack FJ (2006) Oncomirs - microRNAs with a role in cancer. Nat Rev Cancer  6: 259–269.  <ul class="reflinks"><li><a href="#" data-author="Esquela-Kerscher" data-cit="Esquela-KerscherASlackFJ2006Oncomirs%20-%20microRNAs%20with%20a%20role%20in%20cancer.Nat%20Rev%20Cancer6259269" data-title="Oncomirs%20-%20microRNAs%20with%20a%20role%20in%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Oncomirs+-+microRNAs+with+a+role+in+cancer.+Esquela-Kerscher+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pgen.1000879-Lu1" id="pgen.1000879-Lu1" class="link-target"></a>Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al.  (2005) MicroRNA expression profiles classify human cancers. Nature  435: 834–838.  <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuJGetzGMiskaEAAlvarez-SaavedraELambJ2005MicroRNA%20expression%20profiles%20classify%20human%20cancers.Nature435834838" data-title="MicroRNA%20expression%20profiles%20classify%20human%20cancers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+expression+profiles+classify+human+cancers.+Lu+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pgen.1000879-Volinia1" id="pgen.1000879-Volinia1" class="link-target"></a>Volinia S, Calin GA, Liu CG, Ambs S, Cimmino A, et al.  (2006) A microRNA expression signature of human solid tumors defines cancer gene targets. Proc Natl Acad Sci U S A  103: 2257–2261.  <ul class="reflinks"><li><a href="#" data-author="Volinia" data-cit="VoliniaSCalinGALiuCGAmbsSCimminoA2006A%20microRNA%20expression%20signature%20of%20human%20solid%20tumors%20defines%20cancer%20gene%20targets.Proc%20Natl%20Acad%20Sci%20U%20S%20A10322572261" data-title="A%20microRNA%20expression%20signature%20of%20human%20solid%20tumors%20defines%20cancer%20gene%20targets." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+microRNA+expression+signature+of+human+solid+tumors+defines+cancer+gene+targets.+Volinia+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pgen.1000879-Martinez1" id="pgen.1000879-Martinez1" class="link-target"></a>Martinez I, Gardiner AS, Board KF, Monzon FA, Edwards RP, et al.  (2008) Human papillomavirus type 16 reduces the expression of microRNA-218 in cervical carcinoma cells. Oncogene  27: 2575–2582.  <ul class="reflinks"><li><a href="#" data-author="Martinez" data-cit="MartinezIGardinerASBoardKFMonzonFAEdwardsRP2008Human%20papillomavirus%20type%2016%20reduces%20the%20expression%20of%20microRNA-218%20in%20cervical%20carcinoma%20cells.Oncogene2725752582" data-title="Human%20papillomavirus%20type%2016%20reduces%20the%20expression%20of%20microRNA-218%20in%20cervical%20carcinoma%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+papillomavirus+type+16+reduces+the+expression+of+microRNA-218+in+cervical+carcinoma+cells.+Martinez+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pgen.1000879-Petrocca1" id="pgen.1000879-Petrocca1" class="link-target"></a>Petrocca F, Visone R, Onelli MR, Shah MH, Nicoloso MS, et al.  (2008) E2F1-regulated microRNAs impair TGFbeta-dependent cell-cycle arrest and apoptosis in gastric cancer. Cancer Cell  13: 272–286.  <ul class="reflinks"><li><a href="#" data-author="Petrocca" data-cit="PetroccaFVisoneROnelliMRShahMHNicolosoMS2008E2F1-regulated%20microRNAs%20impair%20TGFbeta-dependent%20cell-cycle%20arrest%20and%20apoptosis%20in%20gastric%20cancer.Cancer%20Cell13272286" data-title="E2F1-regulated%20microRNAs%20impair%20TGFbeta-dependent%20cell-cycle%20arrest%20and%20apoptosis%20in%20gastric%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=E2F1-regulated+microRNAs+impair+TGFbeta-dependent+cell-cycle+arrest+and+apoptosis+in+gastric+cancer.+Petrocca+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pgen.1000879-Yanaihara1" id="pgen.1000879-Yanaihara1" class="link-target"></a>Yanaihara N, Caplen N, Bowman E, Seike M, Kumamoto K, et al.  (2006) Unique microRNA molecular profiles in lung cancer diagnosis and prognosis. Cancer Cell  9: 189–198.  <ul class="reflinks"><li><a href="#" data-author="Yanaihara" data-cit="YanaiharaNCaplenNBowmanESeikeMKumamotoK2006Unique%20microRNA%20molecular%20profiles%20in%20lung%20cancer%20diagnosis%20and%20prognosis.Cancer%20Cell9189198" data-title="Unique%20microRNA%20molecular%20profiles%20in%20lung%20cancer%20diagnosis%20and%20prognosis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Unique+microRNA+molecular+profiles+in+lung+cancer+diagnosis+and+prognosis.+Yanaihara+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pgen.1000879-Leite1" id="pgen.1000879-Leite1" class="link-target"></a>Leite KR, Sousa-Canavez JM, Reis ST, Tomiyama AH, Camara-Lopes LH, et al.  (2009) Change in expression of miR-let7c, miR-100, and miR-218 from high grade localized prostate cancer to metastasis. Urol Oncol.   <ul class="reflinks"><li><a href="#" data-author="Leite" data-cit="LeiteKRSousa-CanavezJMReisSTTomiyamaAHCamara-LopesLH2009Change%20in%20expression%20of%20miR-let7c%2C%20miR-100%2C%20and%20miR-218%20from%20high%20grade%20localized%20prostate%20cancer%20to%20metastasis.Urol%20Oncol" data-title="Change%20in%20expression%20of%20miR-let7c%2C%20miR-100%2C%20and%20miR-218%20from%20high%20grade%20localized%20prostate%20cancer%20to%20metastasis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Change+in+expression+of+miR-let7c%2C+miR-100%2C+and+miR-218+from+high+grade+localized+prostate+cancer+to+metastasis.+Leite+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pgen.1000879-He1" id="pgen.1000879-He1" class="link-target"></a>He XX, Yang J, Ding YW, Liu W, Shen QY, et al.  (2006) Increased epithelial and serum expression of macrophage migration inhibitory factor (MIF) in gastric cancer: potential role of MIF in gastric carcinogenesis. Gut  55: 797–802.  <ul class="reflinks"><li><a href="#" data-author="He" data-cit="HeXXYangJDingYWLiuWShenQY2006Increased%20epithelial%20and%20serum%20expression%20of%20macrophage%20migration%20inhibitory%20factor%20%28MIF%29%20in%20gastric%20cancer%3A%20potential%20role%20of%20MIF%20in%20gastric%20carcinogenesis.Gut55797802" data-title="Increased%20epithelial%20and%20serum%20expression%20of%20macrophage%20migration%20inhibitory%20factor%20%28MIF%29%20in%20gastric%20cancer%3A%20potential%20role%20of%20MIF%20in%20gastric%20carcinogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Increased+epithelial+and+serum+expression+of+macrophage+migration+inhibitory+factor+%28MIF%29+in+gastric+cancer%3A+potential+role+of+MIF+in+gastric+carcinogenesis.+He+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pgen.1000879-Wang1" id="pgen.1000879-Wang1" class="link-target"></a>Wang B, Xiao Y, Ding B-B, Zhang N, Yuan X-b, et al.  (2003) Induction of tumor angiogenesis by Slit-Robo signaling and inhibition of cancer growth by blocking Robo activity. Cancer Cell  4: 19–29.  <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangBXiaoYDingB-BZhangNYuanX-b2003Induction%20of%20tumor%20angiogenesis%20by%20Slit-Robo%20signaling%20and%20inhibition%20of%20cancer%20growth%20by%20blocking%20Robo%20activity.Cancer%20Cell41929" data-title="Induction%20of%20tumor%20angiogenesis%20by%20Slit-Robo%20signaling%20and%20inhibition%20of%20cancer%20growth%20by%20blocking%20Robo%20activity." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Induction+of+tumor+angiogenesis+by+Slit-Robo+signaling+and+inhibition+of+cancer+growth+by+blocking+Robo+activity.+Wang+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pgen.1000879-Suchting1" id="pgen.1000879-Suchting1" class="link-target"></a>Suchting S, Heal P, Tahtis K, Stewart LM, Bicknell R (2005) Soluble Robo4 receptor inhibits in vivo angiogenesis and endothelial cell migration. FASEB J  19: 121–123.  <ul class="reflinks"><li><a href="#" data-author="Suchting" data-cit="SuchtingSHealPTahtisKStewartLMBicknellR2005Soluble%20Robo4%20receptor%20inhibits%20in%20vivo%20angiogenesis%20and%20endothelial%20cell%20migration.FASEB%20J19121123" data-title="Soluble%20Robo4%20receptor%20inhibits%20in%20vivo%20angiogenesis%20and%20endothelial%20cell%20migration." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Soluble+Robo4+receptor+inhibits+in+vivo+angiogenesis+and+endothelial+cell+migration.+Suchting+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pgen.1000879-Grone1" id="pgen.1000879-Grone1" class="link-target"></a>Grone J, Doebler O, Loddenkemper C, Hotz B, Buhr HJ, et al.  (2006) Robo1/Robo4: differential expression of angiogenic markers in colorectal cancer. Oncol Rep  15: 1437–1443.  <ul class="reflinks"><li><a href="#" data-author="Grone" data-cit="GroneJDoeblerOLoddenkemperCHotzBBuhrHJ2006Robo1%2FRobo4%3A%20differential%20expression%20of%20angiogenic%20markers%20in%20colorectal%20cancer.Oncol%20Rep1514371443" data-title="Robo1%2FRobo4%3A%20differential%20expression%20of%20angiogenic%20markers%20in%20colorectal%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Robo1%2FRobo4%3A+differential+expression+of+angiogenic+markers+in+colorectal+cancer.+Grone+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pgen.1000879-Ito1" id="pgen.1000879-Ito1" class="link-target"></a>Ito H, Funahashi S, Yamauchi N, Shibahara J, Midorikawa Y, et al.  (2006) Identification of ROBO1 as a novel hepatocellular carcinoma antigen and a potential therapeutic and diagnostic target. Clin Cancer Res  12: 3257–3264.  <ul class="reflinks"><li><a href="#" data-author="Ito" data-cit="ItoHFunahashiSYamauchiNShibaharaJMidorikawaY2006Identification%20of%20ROBO1%20as%20a%20novel%20hepatocellular%20carcinoma%20antigen%20and%20a%20potential%20therapeutic%20and%20diagnostic%20target.Clin%20Cancer%20Res1232573264" data-title="Identification%20of%20ROBO1%20as%20a%20novel%20hepatocellular%20carcinoma%20antigen%20and%20a%20potential%20therapeutic%20and%20diagnostic%20target." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+ROBO1+as+a+novel+hepatocellular+carcinoma+antigen+and+a+potential+therapeutic+and+diagnostic+target.+Ito+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pgen.1000879-Lee2" id="pgen.1000879-Lee2" class="link-target"></a>Lee CH, Fang CY, Sheu JJ, Chang Y, Takada K, et al.  (2008) Amplicons on chromosome 3 contain oncogenes induced by recurrent exposure to 12-O-tetradecanoylphorbol-13-acetate and sodium n-butyrate and Epstein-Barr virus reactivation in a nasopharyngeal carcinoma cell line. Cancer Genet Cytogenet  185: 1–10.  <ul class="reflinks"><li><a href="#" data-author="Lee" data-cit="LeeCHFangCYSheuJJChangYTakadaK2008Amplicons%20on%20chromosome%203%20contain%20oncogenes%20induced%20by%20recurrent%20exposure%20to%2012-O-tetradecanoylphorbol-13-acetate%20and%20sodium%20n-butyrate%20and%20Epstein-Barr%20virus%20reactivation%20in%20a%20nasopharyngeal%20carcinoma%20cell%20line.Cancer%20Genet%20Cytogenet185110" data-title="Amplicons%20on%20chromosome%203%20contain%20oncogenes%20induced%20by%20recurrent%20exposure%20to%2012-O-tetradecanoylphorbol-13-acetate%20and%20sodium%20n-butyrate%20and%20Epstein-Barr%20virus%20reactivation%20in%20a%20nasopharyngeal%20carcinoma%20cell%20line." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Amplicons+on+chromosome+3+contain+oncogenes+induced+by+recurrent+exposure+to+12-O-tetradecanoylphorbol-13-acetate+and+sodium+n-butyrate+and+Epstein-Barr+virus+reactivation+in+a+nasopharyngeal+carcinoma+cell+line.+Lee+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pgen.1000879-Legg1" id="pgen.1000879-Legg1" class="link-target"></a>Legg JA, Herbert JM, Clissold P, Bicknell R (2008) Slits and Roundabouts in cancer, tumour angiogenesis and endothelial cell migration. Angiogenesis  11: 13–21.  <ul class="reflinks"><li><a href="#" data-author="Legg" data-cit="LeggJAHerbertJMClissoldPBicknellR2008Slits%20and%20Roundabouts%20in%20cancer%2C%20tumour%20angiogenesis%20and%20endothelial%20cell%20migration.Angiogenesis111321" data-title="Slits%20and%20Roundabouts%20in%20cancer%2C%20tumour%20angiogenesis%20and%20endothelial%20cell%20migration." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Slits+and+Roundabouts+in+cancer%2C+tumour+angiogenesis+and+endothelial+cell+migration.+Legg+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pgen.1000879-Mertsch1" id="pgen.1000879-Mertsch1" class="link-target"></a>Mertsch S, Schmitz N, Jeibmann A, Geng JG, Paulus W, et al.  (2008) Slit2 involvement in glioma cell migration is mediated by Robo1 receptor. J Neurooncol  87: 1–7.  <ul class="reflinks"><li><a href="#" data-author="Mertsch" data-cit="MertschSSchmitzNJeibmannAGengJGPaulusW2008Slit2%20involvement%20in%20glioma%20cell%20migration%20is%20mediated%20by%20Robo1%20receptor.J%20Neurooncol8717" data-title="Slit2%20involvement%20in%20glioma%20cell%20migration%20is%20mediated%20by%20Robo1%20receptor." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Slit2+involvement+in+glioma+cell+migration+is+mediated+by+Robo1+receptor.+Mertsch+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pgen.1000879-Schmid1" id="pgen.1000879-Schmid1" class="link-target"></a>Schmid BC, Rezniczek GA, Fabjani G, Yoneda T, Leodolter S, et al.  (2007) The neuronal guidance cue Slit2 induces targeted migration and may play a role in brain metastasis of breast cancer cells. Breast Cancer Res Treat  106: 333–342.  <ul class="reflinks"><li><a href="#" data-author="Schmid" data-cit="SchmidBCRezniczekGAFabjaniGYonedaTLeodolterS2007The%20neuronal%20guidance%20cue%20Slit2%20induces%20targeted%20migration%20and%20may%20play%20a%20role%20in%20brain%20metastasis%20of%20breast%20cancer%20cells.Breast%20Cancer%20Res%20Treat106333342" data-title="The%20neuronal%20guidance%20cue%20Slit2%20induces%20targeted%20migration%20and%20may%20play%20a%20role%20in%20brain%20metastasis%20of%20breast%20cancer%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+neuronal+guidance+cue+Slit2+induces+targeted+migration+and+may+play+a+role+in+brain+metastasis+of+breast+cancer+cells.+Schmid+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pgen.1000879-Gramantieri1" id="pgen.1000879-Gramantieri1" class="link-target"></a>Gramantieri L, Ferracin M, Fornari F, Veronese A, Sabbioni S, et al.  (2007) Cyclin G1 is a target of miR-122a, a microRNA frequently down-regulated in human hepatocellular carcinoma. Cancer Res  67: 6092–6099.  <ul class="reflinks"><li><a href="#" data-author="Gramantieri" data-cit="GramantieriLFerracinMFornariFVeroneseASabbioniS2007Cyclin%20G1%20is%20a%20target%20of%20miR-122a%2C%20a%20microRNA%20frequently%20down-regulated%20in%20human%20hepatocellular%20carcinoma.Cancer%20Res6760926099" data-title="Cyclin%20G1%20is%20a%20target%20of%20miR-122a%2C%20a%20microRNA%20frequently%20down-regulated%20in%20human%20hepatocellular%20carcinoma." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cyclin+G1+is+a+target+of+miR-122a%2C+a+microRNA+frequently+down-regulated+in+human+hepatocellular+carcinoma.+Gramantieri+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pgen.1000879-Seth1" id="pgen.1000879-Seth1" class="link-target"></a>Seth P, Lin Y, Hanai J, Shivalingappa V, Duyao MP, et al.  (2005) Magic roundabout, a tumor endothelial marker: expression and signaling. Biochem Biophys Res Commun  332: 533–541.  <ul class="reflinks"><li><a href="#" data-author="Seth" data-cit="SethPLinYHanaiJShivalingappaVDuyaoMP2005Magic%20roundabout%2C%20a%20tumor%20endothelial%20marker%3A%20expression%20and%20signaling.Biochem%20Biophys%20Res%20Commun332533541" data-title="Magic%20roundabout%2C%20a%20tumor%20endothelial%20marker%3A%20expression%20and%20signaling." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Magic+roundabout%2C+a+tumor+endothelial+marker%3A+expression+and+signaling.+Seth+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pgen.1000879-Abdollahi1" id="pgen.1000879-Abdollahi1" class="link-target"></a>Abdollahi A, Schwager C, Kleeff J, Esposito I, Domhan S, et al.  (2007) Transcriptional network governing the angiogenic switch in human pancreatic cancer. Proc Natl Acad Sci U S A  104: 12890–12895.  <ul class="reflinks"><li><a href="#" data-author="Abdollahi" data-cit="AbdollahiASchwagerCKleeffJEspositoIDomhanS2007Transcriptional%20network%20governing%20the%20angiogenic%20switch%20in%20human%20pancreatic%20cancer.Proc%20Natl%20Acad%20Sci%20U%20S%20A1041289012895" data-title="Transcriptional%20network%20governing%20the%20angiogenic%20switch%20in%20human%20pancreatic%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptional+network+governing+the+angiogenic+switch+in+human+pancreatic+cancer.+Abdollahi+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pgen.1000879-Shi1" id="pgen.1000879-Shi1" class="link-target"></a>Shi XB, Tepper CG, W RW (2008) Cancerous miRNAs and their regulation. Cell Cycle  7: 1529–1538.  <ul class="reflinks"><li><a href="#" data-author="Shi" data-cit="ShiXBTepperCGWRW2008Cancerous%20miRNAs%20and%20their%20regulation.Cell%20Cycle715291538" data-title="Cancerous%20miRNAs%20and%20their%20regulation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cancerous+miRNAs+and+their+regulation.+Shi+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pgen.1000879-Lin1" id="pgen.1000879-Lin1" class="link-target"></a>Lin SL, Miller JD, Ying SY (2006) Intronic MicroRNA (miRNA). J Biomed Biotechnol 2006: 26818. 26818.  <ul class="reflinks"><li><a href="#" data-author="Lin" data-cit="LinSLMillerJDYingSY2006Intronic%20MicroRNA%20%28miRNA%29.J%20Biomed%20Biotechnol%202006%3A%2026818.26818" data-title="Intronic%20MicroRNA%20%28miRNA%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Intronic+MicroRNA+%28miRNA%29.+Lin+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pgen.1000879-Chu1" id="pgen.1000879-Chu1" class="link-target"></a>Chu YW, Yang PC, Yang SC, Shyu YC, Hendrix MJ, et al.  (1997) Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line. Am J Respir Cell Mol Biol  17: 353–360.  <ul class="reflinks"><li><a href="#" data-author="Chu" data-cit="ChuYWYangPCYangSCShyuYCHendrixMJ1997Selection%20of%20invasive%20and%20metastatic%20subpopulations%20from%20a%20human%20lung%20adenocarcinoma%20cell%20line.Am%20J%20Respir%20Cell%20Mol%20Biol17353360" data-title="Selection%20of%20invasive%20and%20metastatic%20subpopulations%20from%20a%20human%20lung%20adenocarcinoma%20cell%20line." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Selection+of+invasive+and+metastatic+subpopulations+from+a+human+lung+adenocarcinoma+cell+line.+Chu+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pgen.1000879-Wang2" id="pgen.1000879-Wang2" class="link-target"></a>Wang CC, Tsai MF, Dai TH, Hong TM, Chan WK, et al.  (2007) Synergistic activation of the tumor suppressor, HLJ1, by the transcription factors YY1 and activator protein 1. Cancer Res  67: 4816–4826.  <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangCCTsaiMFDaiTHHongTMChanWK2007Synergistic%20activation%20of%20the%20tumor%20suppressor%2C%20HLJ1%2C%20by%20the%20transcription%20factors%20YY1%20and%20activator%20protein%201.Cancer%20Res6748164826" data-title="Synergistic%20activation%20of%20the%20tumor%20suppressor%2C%20HLJ1%2C%20by%20the%20transcription%20factors%20YY1%20and%20activator%20protein%201." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Synergistic+activation+of+the+tumor+suppressor%2C+HLJ1%2C+by+the+transcription+factors+YY1+and+activator+protein+1.+Wang+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pgen.1000879-Tseng1" id="pgen.1000879-Tseng1" class="link-target"></a>Tseng RC, Lin RK, Wen CK, Tseng C, Hsu HS, et al.  (2008) Epigenetic silencing of AXIN2/betaTrCP and deregulation of p53-mediated control lead to wild-type beta-catenin nuclear accumulation in lung tumorigenesis. Oncogene  27: 4488–4496.  <ul class="reflinks"><li><a href="#" data-author="Tseng" data-cit="TsengRCLinRKWenCKTsengCHsuHS2008Epigenetic%20silencing%20of%20AXIN2%2FbetaTrCP%20and%20deregulation%20of%20p53-mediated%20control%20lead%20to%20wild-type%20beta-catenin%20nuclear%20accumulation%20in%20lung%20tumorigenesis.Oncogene2744884496" data-title="Epigenetic%20silencing%20of%20AXIN2%2FbetaTrCP%20and%20deregulation%20of%20p53-mediated%20control%20lead%20to%20wild-type%20beta-catenin%20nuclear%20accumulation%20in%20lung%20tumorigenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Epigenetic+silencing+of+AXIN2%2FbetaTrCP+and+deregulation+of+p53-mediated+control+lead+to+wild-type+beta-catenin+nuclear+accumulation+in+lung+tumorigenesis.+Tseng+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pgen.1000879-Chang1" id="pgen.1000879-Chang1" class="link-target"></a>Chang DK, Lin CT, Wu CH, Wu HC (2009) A novel peptide enhances therapeutic efficacy of liposomal anti-cancer drugs in mice models of human lung cancer. PLoS ONE  4: e4171.  <ul class="reflinks"><li><a href="#" data-author="Chang" data-cit="ChangDKLinCTWuCHWuHC2009A%20novel%20peptide%20enhances%20therapeutic%20efficacy%20of%20liposomal%20anti-cancer%20drugs%20in%20mice%20models%20of%20human%20lung%20cancer.PLoS%20ONE4e4171doi%3A10.1371%2Fjournal.pone.0004171" data-title="A%20novel%20peptide%20enhances%20therapeutic%20efficacy%20of%20liposomal%20anti-cancer%20drugs%20in%20mice%20models%20of%20human%20lung%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+novel+peptide+enhances+therapeutic+efficacy+of+liposomal+anti-cancer+drugs+in+mice+models+of+human+lung+cancer.+Chang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pgen.1000879-Torng1" id="pgen.1000879-Torng1" class="link-target"></a>Torng PL, Lee YC, Huang CY, Ye JH, Lin YS, et al.  (2008) Insulin-like growth factor binding protein-3 (IGFBP-3) acts as an invasion-metastasis suppressor in ovarian endometrioid carcinoma. Oncogene  27: 2137–2147.  <ul class="reflinks"><li><a href="#" data-author="Torng" data-cit="TorngPLLeeYCHuangCYYeJHLinYS2008Insulin-like%20growth%20factor%20binding%20protein-3%20%28IGFBP-3%29%20acts%20as%20an%20invasion-metastasis%20suppressor%20in%20ovarian%20endometrioid%20carcinoma.Oncogene2721372147" data-title="Insulin-like%20growth%20factor%20binding%20protein-3%20%28IGFBP-3%29%20acts%20as%20an%20invasion-metastasis%20suppressor%20in%20ovarian%20endometrioid%20carcinoma." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Insulin-like+growth+factor+binding+protein-3+%28IGFBP-3%29+acts+as+an+invasion-metastasis+suppressor+in+ovarian+endometrioid+carcinoma.+Torng+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pgen.1000879-Yu1" id="pgen.1000879-Yu1" class="link-target"></a>Yu SL, Chen HY, Chang GC, Chen CY, Chen HW, et al.  (2008) MicroRNA signature predicts survival and relapse in lung cancer. Cancer Cell  13: 48–57.  <ul class="reflinks"><li><a href="#" data-author="Yu" data-cit="YuSLChenHYChangGCChenCYChenHW2008MicroRNA%20signature%20predicts%20survival%20and%20relapse%20in%20lung%20cancer.Cancer%20Cell134857" data-title="MicroRNA%20signature%20predicts%20survival%20and%20relapse%20in%20lung%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+signature+predicts+survival+and+relapse+in+lung+cancer.+Yu+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pgen.1000879-Zhu1" id="pgen.1000879-Zhu1" class="link-target"></a>Zhu S, Wu H, Wu F, Nie D, Sheng S, et al.  (2008) MicroRNA-21 targets tumor suppressor genes in invasion and metastasis. Cell Res  18: 350–359.  <ul class="reflinks"><li><a href="#" data-author="Zhu" data-cit="ZhuSWuHWuFNieDShengS2008MicroRNA-21%20targets%20tumor%20suppressor%20genes%20in%20invasion%20and%20metastasis.Cell%20Res18350359" data-title="MicroRNA-21%20targets%20tumor%20suppressor%20genes%20in%20invasion%20and%20metastasis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-21+targets+tumor+suppressor+genes+in+invasion+and+metastasis.+Zhu+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pgen.1000879-Hiyoshi1" id="pgen.1000879-Hiyoshi1" class="link-target"></a>Hiyoshi Y, Kamohara H, Karashima R, Sato N, Imamura Y, et al.  (2009) MicroRNA-21 Regulates the Proliferation and Invasion in Esophageal Squamous Cell Carcinoma. Clin Cancer Res.   <ul class="reflinks"><li><a href="#" data-author="Hiyoshi" data-cit="HiyoshiYKamoharaHKarashimaRSatoNImamuraY2009MicroRNA-21%20Regulates%20the%20Proliferation%20and%20Invasion%20in%20Esophageal%20Squamous%20Cell%20Carcinoma.Clin%20Cancer%20Res" data-title="MicroRNA-21%20Regulates%20the%20Proliferation%20and%20Invasion%20in%20Esophageal%20Squamous%20Cell%20Carcinoma." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-21+Regulates+the+Proliferation+and+Invasion+in+Esophageal+Squamous+Cell+Carcinoma.+Hiyoshi+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pgen.1000879-Crawford1" id="pgen.1000879-Crawford1" class="link-target"></a>Crawford M, Brawner E, Batte K, Yu L, Hunter MG, et al.  (2008) MicroRNA-126 inhibits invasion in non-small cell lung carcinoma cell lines. Biochem Biophys Res Commun  373: 607–612.  <ul class="reflinks"><li><a href="#" data-author="Crawford" data-cit="CrawfordMBrawnerEBatteKYuLHunterMG2008MicroRNA-126%20inhibits%20invasion%20in%20non-small%20cell%20lung%20carcinoma%20cell%20lines.Biochem%20Biophys%20Res%20Commun373607612" data-title="MicroRNA-126%20inhibits%20invasion%20in%20non-small%20cell%20lung%20carcinoma%20cell%20lines." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA-126+inhibits+invasion+in+non-small+cell+lung+carcinoma+cell+lines.+Crawford+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pgen.1000879-Tavazoie1" id="pgen.1000879-Tavazoie1" class="link-target"></a>Tavazoie SF, Alarcon C, Oskarsson T, Padua D, Wang Q, et al.  (2008) Endogenous human microRNAs that suppress breast cancer metastasis. Nature  451: 147–152.  <ul class="reflinks"><li><a href="#" data-author="Tavazoie" data-cit="TavazoieSFAlarconCOskarssonTPaduaDWangQ2008Endogenous%20human%20microRNAs%20that%20suppress%20breast%20cancer%20metastasis.Nature451147152" data-title="Endogenous%20human%20microRNAs%20that%20suppress%20breast%20cancer%20metastasis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Endogenous+human+microRNAs+that+suppress+breast+cancer+metastasis.+Tavazoie+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pgen.1000879-Li2" id="pgen.1000879-Li2" class="link-target"></a>Li N, Fu H, Tie Y, Hu Z, Kong W, et al.  (2009) miR-34a inhibits migration and invasion by down-regulation of c-Met expression in human hepatocellular carcinoma cells. Cancer Lett  275: 44–53.  <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiNFuHTieYHuZKongW2009miR-34a%20inhibits%20migration%20and%20invasion%20by%20down-regulation%20of%20c-Met%20expression%20in%20human%20hepatocellular%20carcinoma%20cells.Cancer%20Lett2754453" data-title="miR-34a%20inhibits%20migration%20and%20invasion%20by%20down-regulation%20of%20c-Met%20expression%20in%20human%20hepatocellular%20carcinoma%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miR-34a+inhibits+migration+and+invasion+by+down-regulation+of+c-Met+expression+in+human+hepatocellular+carcinoma+cells.+Li+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pgen.1000879-Gebeshuber1" id="pgen.1000879-Gebeshuber1" class="link-target"></a>Gebeshuber CA, Zatloukal K, Martinez J (2009) miR-29a suppresses tristetraprolin, which is a regulator of epithelial polarity and metastasis. EMBO Rep.   <ul class="reflinks"><li><a href="#" data-author="Gebeshuber" data-cit="GebeshuberCAZatloukalKMartinezJ2009miR-29a%20suppresses%20tristetraprolin%2C%20which%20is%20a%20regulator%20of%20epithelial%20polarity%20and%20metastasis.EMBO%20Rep" data-title="miR-29a%20suppresses%20tristetraprolin%2C%20which%20is%20a%20regulator%20of%20epithelial%20polarity%20and%20metastasis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miR-29a+suppresses+tristetraprolin%2C+which+is+a+regulator+of+epithelial+polarity+and+metastasis.+Gebeshuber+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pgen.1000879-Dalmay1" id="pgen.1000879-Dalmay1" class="link-target"></a>Dalmay T, Edwards DR (2006) MicroRNAs and the hallmarks of cancer. Oncogene  25: 6170–6175.  <ul class="reflinks"><li><a href="#" data-author="Dalmay" data-cit="DalmayTEdwardsDR2006MicroRNAs%20and%20the%20hallmarks%20of%20cancer.Oncogene2561706175" data-title="MicroRNAs%20and%20the%20hallmarks%20of%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+and+the+hallmarks+of+cancer.+Dalmay+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pgen.1000879-Huminiecki1" id="pgen.1000879-Huminiecki1" class="link-target"></a>Huminiecki L, Gorn M, Suchting S, Poulsom R, Bicknell R (2002) Magic Roundabout Is a New Member of the Roundabout Receptor Family That Is Endothelial Specific and Expressed at Sites of Active Angiogenesis. Genomics  79: 547–552.  <ul class="reflinks"><li><a href="#" data-author="Huminiecki" data-cit="HuminieckiLGornMSuchtingSPoulsomRBicknellR2002Magic%20Roundabout%20Is%20a%20New%20Member%20of%20the%20Roundabout%20Receptor%20Family%20That%20Is%20Endothelial%20Specific%20and%20Expressed%20at%20Sites%20of%20Active%20Angiogenesis.Genomics79547552" data-title="Magic%20Roundabout%20Is%20a%20New%20Member%20of%20the%20Roundabout%20Receptor%20Family%20That%20Is%20Endothelial%20Specific%20and%20Expressed%20at%20Sites%20of%20Active%20Angiogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Magic+Roundabout+Is+a+New+Member+of+the+Roundabout+Receptor+Family+That+Is+Endothelial+Specific+and+Expressed+at+Sites+of+Active+Angiogenesis.+Huminiecki+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pgen.1000879-Brose1" id="pgen.1000879-Brose1" class="link-target"></a>Brose K, Tessier-Lavigne M (2000) Slit proteins: key regulators of axon guidance, axonal branching, and cell migration. Curr Opin Neurobiol  10: 95–102.  <ul class="reflinks"><li><a href="#" data-author="Brose" data-cit="BroseKTessier-LavigneM2000Slit%20proteins%3A%20key%20regulators%20of%20axon%20guidance%2C%20axonal%20branching%2C%20and%20cell%20migration.Curr%20Opin%20Neurobiol1095102" data-title="Slit%20proteins%3A%20key%20regulators%20of%20axon%20guidance%2C%20axonal%20branching%2C%20and%20cell%20migration." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Slit+proteins%3A+key+regulators+of+axon+guidance%2C+axonal+branching%2C+and+cell+migration.+Brose+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pgen.1000879-Kidd1" id="pgen.1000879-Kidd1" class="link-target"></a>Kidd T, Brose K, Mitchell KJ, Fetter RD, Tessier-Lavigne M, et al.  (1998) Roundabout controls axon crossing of the CNS midline and defines a novel subfamily of evolutionarily conserved guidance receptors. Cell  92: 205–215.  <ul class="reflinks"><li><a href="#" data-author="Kidd" data-cit="KiddTBroseKMitchellKJFetterRDTessier-LavigneM1998Roundabout%20controls%20axon%20crossing%20of%20the%20CNS%20midline%20and%20defines%20a%20novel%20subfamily%20of%20evolutionarily%20conserved%20guidance%20receptors.Cell92205215" data-title="Roundabout%20controls%20axon%20crossing%20of%20the%20CNS%20midline%20and%20defines%20a%20novel%20subfamily%20of%20evolutionarily%20conserved%20guidance%20receptors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Roundabout+controls+axon+crossing+of+the+CNS+midline+and+defines+a+novel+subfamily+of+evolutionarily+conserved+guidance+receptors.+Kidd+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pgen.1000879-Zallen1" id="pgen.1000879-Zallen1" class="link-target"></a>Zallen JA, Yi BA, Bargmann CI (1998) The conserved immunoglobulin superfamily member SAX-3/Robo directs multiple aspects of axon guidance in C. elegans. Cell  92: 217–227.  <ul class="reflinks"><li><a href="#" data-author="Zallen" data-cit="ZallenJAYiBABargmannCI1998The%20conserved%20immunoglobulin%20superfamily%20member%20SAX-3%2FRobo%20directs%20multiple%20aspects%20of%20axon%20guidance%20in%20C.%20elegans.Cell92217227" data-title="The%20conserved%20immunoglobulin%20superfamily%20member%20SAX-3%2FRobo%20directs%20multiple%20aspects%20of%20axon%20guidance%20in%20C.%20elegans." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+conserved+immunoglobulin+superfamily+member+SAX-3%2FRobo+directs+multiple+aspects+of+axon+guidance+in+C.+elegans.+Zallen+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span class="order">60.
            </span><a name="pgen.1000879-Dickson1" id="pgen.1000879-Dickson1" class="link-target"></a>Dickson BJ, Gilestro GF (2006) Regulation of commissural axon pathfinding by slit and its Robo receptors. Annu Rev Cell Dev Biol  22: 651–675.  <ul class="reflinks"><li><a href="#" data-author="Dickson" data-cit="DicksonBJGilestroGF2006Regulation%20of%20commissural%20axon%20pathfinding%20by%20slit%20and%20its%20Robo%20receptors.Annu%20Rev%20Cell%20Dev%20Biol22651675" data-title="Regulation%20of%20commissural%20axon%20pathfinding%20by%20slit%20and%20its%20Robo%20receptors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+commissural+axon+pathfinding+by+slit+and+its+Robo+receptors.+Dickson+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span class="order">61.
            </span><a name="pgen.1000879-Bhat1" id="pgen.1000879-Bhat1" class="link-target"></a>Bhat KM, Gaziova I, Krishnan S (2007) Regulation of axon guidance by slit and netrin signaling in the Drosophila ventral nerve cord. Genetics  176: 2235–2246.  <ul class="reflinks"><li><a href="#" data-author="Bhat" data-cit="BhatKMGaziovaIKrishnanS2007Regulation%20of%20axon%20guidance%20by%20slit%20and%20netrin%20signaling%20in%20the%20Drosophila%20ventral%20nerve%20cord.Genetics17622352246" data-title="Regulation%20of%20axon%20guidance%20by%20slit%20and%20netrin%20signaling%20in%20the%20Drosophila%20ventral%20nerve%20cord." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+axon+guidance+by+slit+and+netrin+signaling+in+the+Drosophila+ventral+nerve+cord.+Bhat+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span class="order">62.
            </span><a name="pgen.1000879-Wu2" id="pgen.1000879-Wu2" class="link-target"></a>Wu JY, Feng L, Park HT, Havlioglu N, Wen L, et al.  (2001) The neuronal repellent Slit inhibits leukocyte chemotaxis induced by chemotactic factors. Nature  410: 948–952.  <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuJYFengLParkHTHavliogluNWenL2001The%20neuronal%20repellent%20Slit%20inhibits%20leukocyte%20chemotaxis%20induced%20by%20chemotactic%20factors.Nature410948952" data-title="The%20neuronal%20repellent%20Slit%20inhibits%20leukocyte%20chemotaxis%20induced%20by%20chemotactic%20factors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+neuronal+repellent+Slit+inhibits+leukocyte+chemotaxis+induced+by+chemotactic+factors.+Wu+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref63"><span class="order">63.
            </span><a name="pgen.1000879-Guan1" id="pgen.1000879-Guan1" class="link-target"></a>Guan H, Zu G, Xie Y, Tang H, Johnson M, et al.  (2003) Neuronal repellent Slit2 inhibits dendritic cell migration and the development of immune responses. J Immunol  171: 6519–6526.  <ul class="reflinks"><li><a href="#" data-author="Guan" data-cit="GuanHZuGXieYTangHJohnsonM2003Neuronal%20repellent%20Slit2%20inhibits%20dendritic%20cell%20migration%20and%20the%20development%20of%20immune%20responses.J%20Immunol17165196526" data-title="Neuronal%20repellent%20Slit2%20inhibits%20dendritic%20cell%20migration%20and%20the%20development%20of%20immune%20responses." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Neuronal+repellent+Slit2+inhibits+dendritic+cell+migration+and+the+development+of+immune+responses.+Guan+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref64"><span class="order">64.
            </span><a name="pgen.1000879-Prasad1" id="pgen.1000879-Prasad1" class="link-target"></a>Prasad A, Qamri Z, Wu J, Ganju RK (2007) Slit-2/Robo-1 modulates the CXCL12/CXCR4-induced chemotaxis of T cells. J Leukoc Biol  82: 465–476.  <ul class="reflinks"><li><a href="#" data-author="Prasad" data-cit="PrasadAQamriZWuJGanjuRK2007Slit-2%2FRobo-1%20modulates%20the%20CXCL12%2FCXCR4-induced%20chemotaxis%20of%20T%20cells.J%20Leukoc%20Biol82465476" data-title="Slit-2%2FRobo-1%20modulates%20the%20CXCL12%2FCXCR4-induced%20chemotaxis%20of%20T%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Slit-2%2FRobo-1+modulates+the+CXCL12%2FCXCR4-induced+chemotaxis+of+T+cells.+Prasad+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref65"><span class="order">65.
            </span><a name="pgen.1000879-Rossi1" id="pgen.1000879-Rossi1" class="link-target"></a>Rossi MR, Huntoon K, Cowell JK (2005) Differential expression of the LGI and SLIT families of genes in human cancer cells. Gene  356: 85–90.  <ul class="reflinks"><li><a href="#" data-author="Rossi" data-cit="RossiMRHuntoonKCowellJK2005Differential%20expression%20of%20the%20LGI%20and%20SLIT%20families%20of%20genes%20in%20human%20cancer%20cells.Gene3568590" data-title="Differential%20expression%20of%20the%20LGI%20and%20SLIT%20families%20of%20genes%20in%20human%20cancer%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+expression+of+the+LGI+and+SLIT+families+of+genes+in+human+cancer+cells.+Rossi+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref66"><span class="order">66.
            </span><a name="pgen.1000879-Narayan1" id="pgen.1000879-Narayan1" class="link-target"></a>Narayan G, Goparaju C, Arias-Pulido H, Kaufmann AM, Schneider A, et al.  (2006) Promoter hypermethylation-mediated inactivation of multiple Slit-Robo pathway genes in cervical cancer progression. Mol Cancer  5: 16.  <ul class="reflinks"><li><a href="#" data-author="Narayan" data-cit="NarayanGGoparajuCArias-PulidoHKaufmannAMSchneiderA2006Promoter%20hypermethylation-mediated%20inactivation%20of%20multiple%20Slit-Robo%20pathway%20genes%20in%20cervical%20cancer%20progression.Mol%20Cancer516" data-title="Promoter%20hypermethylation-mediated%20inactivation%20of%20multiple%20Slit-Robo%20pathway%20genes%20in%20cervical%20cancer%20progression." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Promoter+hypermethylation-mediated+inactivation+of+multiple+Slit-Robo+pathway+genes+in+cervical+cancer+progression.+Narayan+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref67"><span class="order">67.
            </span><a name="pgen.1000879-Singh1" id="pgen.1000879-Singh1" class="link-target"></a>Singh RK, Indra D, Mitra S, Mondal RK, Basu PS, et al.  (2007) Deletions in chromosome 4 differentially associated with the development of cervical cancer: evidence of slit2 as a candidate tumor suppressor gene. Hum Genet  122: 71–81.  <ul class="reflinks"><li><a href="#" data-author="Singh" data-cit="SinghRKIndraDMitraSMondalRKBasuPS2007Deletions%20in%20chromosome%204%20differentially%20associated%20with%20the%20development%20of%20cervical%20cancer%3A%20evidence%20of%20slit2%20as%20a%20candidate%20tumor%20suppressor%20gene.Hum%20Genet1227181" data-title="Deletions%20in%20chromosome%204%20differentially%20associated%20with%20the%20development%20of%20cervical%20cancer%3A%20evidence%20of%20slit2%20as%20a%20candidate%20tumor%20suppressor%20gene." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Deletions+in+chromosome+4+differentially+associated+with+the+development+of+cervical+cancer%3A+evidence+of+slit2+as+a+candidate+tumor+suppressor+gene.+Singh+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref68"><span class="order">68.
            </span><a name="pgen.1000879-Prasad2" id="pgen.1000879-Prasad2" class="link-target"></a>Prasad A, Paruchuri V, Preet A, Latif F, Ganju RK (2008) Slit-2 induces a tumor-suppressive effect by regulating beta-catenin in breast cancer cells. J Biol Chem  283: 26624–26633.  <ul class="reflinks"><li><a href="#" data-author="Prasad" data-cit="PrasadAParuchuriVPreetALatifFGanjuRK2008Slit-2%20induces%20a%20tumor-suppressive%20effect%20by%20regulating%20beta-catenin%20in%20breast%20cancer%20cells.J%20Biol%20Chem2832662426633" data-title="Slit-2%20induces%20a%20tumor-suppressive%20effect%20by%20regulating%20beta-catenin%20in%20breast%20cancer%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Slit-2+induces+a+tumor-suppressive+effect+by+regulating+beta-catenin+in+breast+cancer+cells.+Prasad+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref69"><span class="order">69.
            </span><a name="pgen.1000879-Dallol1" id="pgen.1000879-Dallol1" class="link-target"></a>Dallol A, Forgacs E, Martinez A, Sekido Y, Walker R, et al.  (2002) Tumour specific promoter region methylation of the human homologue of the Drosophila Roundabout gene DUTT1 (ROBO1) in human cancers. Oncogene  21: 3020–3028.  <ul class="reflinks"><li><a href="#" data-author="Dallol" data-cit="DallolAForgacsEMartinezASekidoYWalkerR2002Tumour%20specific%20promoter%20region%20methylation%20of%20the%20human%20homologue%20of%20the%20Drosophila%20Roundabout%20gene%20DUTT1%20%28ROBO1%29%20in%20human%20cancers.Oncogene2130203028" data-title="Tumour%20specific%20promoter%20region%20methylation%20of%20the%20human%20homologue%20of%20the%20Drosophila%20Roundabout%20gene%20DUTT1%20%28ROBO1%29%20in%20human%20cancers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tumour+specific+promoter+region+methylation+of+the+human+homologue+of+the+Drosophila+Roundabout+gene+DUTT1+%28ROBO1%29+in+human+cancers.+Dallol+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref70"><span class="order">70.
            </span><a name="pgen.1000879-Xian1" id="pgen.1000879-Xian1" class="link-target"></a>Xian J, Clark KJ, Fordham R, Pannell R, Rabbitts TH, et al.  (2001) Inadequate lung development and bronchial hyperplasia in mice with a targeted deletion in the Dutt1/Robo1 gene. Proc Natl Acad Sci U S A  98: 15062–15066.  <ul class="reflinks"><li><a href="#" data-author="Xian" data-cit="XianJClarkKJFordhamRPannellRRabbittsTH2001Inadequate%20lung%20development%20and%20bronchial%20hyperplasia%20in%20mice%20with%20a%20targeted%20deletion%20in%20the%20Dutt1%2FRobo1%20gene.Proc%20Natl%20Acad%20Sci%20U%20S%20A981506215066" data-title="Inadequate%20lung%20development%20and%20bronchial%20hyperplasia%20in%20mice%20with%20a%20targeted%20deletion%20in%20the%20Dutt1%2FRobo1%20gene." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Inadequate+lung+development+and+bronchial+hyperplasia+in+mice+with+a+targeted+deletion+in+the+Dutt1%2FRobo1+gene.+Xian+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref71"><span class="order">71.
            </span><a name="pgen.1000879-Latil1" id="pgen.1000879-Latil1" class="link-target"></a>Latil A, Chene L, Cochant-Priollet B, Mangin P, Fournier G, et al.  (2003) Quantification of expression of netrins, slits and their receptors in human prostate tumors. Int J Cancer  103: 306–315.  <ul class="reflinks"><li><a href="#" data-author="Latil" data-cit="LatilACheneLCochant-PriolletBManginPFournierG2003Quantification%20of%20expression%20of%20netrins%2C%20slits%20and%20their%20receptors%20in%20human%20prostate%20tumors.Int%20J%20Cancer103306315" data-title="Quantification%20of%20expression%20of%20netrins%2C%20slits%20and%20their%20receptors%20in%20human%20prostate%20tumors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Quantification+of+expression+of+netrins%2C+slits+and+their+receptors+in+human+prostate+tumors.+Latil+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref72"><span class="order">72.
            </span><a name="pgen.1000879-Kim1" id="pgen.1000879-Kim1" class="link-target"></a>Kim HK, Zhang H, Li H, Wu TT, Swisher S, et al.  (2008) Slit2 inhibits growth and metastasis of fibrosarcoma and squamous cell carcinoma. Neoplasia  10: 1411–1420.  <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimHKZhangHLiHWuTTSwisherS2008Slit2%20inhibits%20growth%20and%20metastasis%20of%20fibrosarcoma%20and%20squamous%20cell%20carcinoma.Neoplasia1014111420" data-title="Slit2%20inhibits%20growth%20and%20metastasis%20of%20fibrosarcoma%20and%20squamous%20cell%20carcinoma." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Slit2+inhibits+growth+and+metastasis+of+fibrosarcoma+and+squamous+cell+carcinoma.+Kim+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref73"><span class="order">73.
            </span><a name="pgen.1000879-Jin1" id="pgen.1000879-Jin1" class="link-target"></a>Jin J, You H, Yu B, Deng Y, Tang N, et al.  (2009) Epigenetic inactivation of SLIT2 in human hepatocellular carcinomas. Biochem Biophys Res Commun  379: 86–91.  <ul class="reflinks"><li><a href="#" data-author="Jin" data-cit="JinJYouHYuBDengYTangN2009Epigenetic%20inactivation%20of%20SLIT2%20in%20human%20hepatocellular%20carcinomas.Biochem%20Biophys%20Res%20Commun3798691" data-title="Epigenetic%20inactivation%20of%20SLIT2%20in%20human%20hepatocellular%20carcinomas." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Epigenetic+inactivation+of+SLIT2+in+human+hepatocellular+carcinomas.+Jin+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref74"><span class="order">74.
            </span><a name="pgen.1000879-Andrews1" id="pgen.1000879-Andrews1" class="link-target"></a>Andrews W, Liapi A, Plachez C, Camurri L, Zhang J, et al.  (2006) Robo1 regulates the development of major axon tracts and interneuron migration in the forebrain. Development  133: 2243–2252.  <ul class="reflinks"><li><a href="#" data-author="Andrews" data-cit="AndrewsWLiapiAPlachezCCamurriLZhangJ2006Robo1%20regulates%20the%20development%20of%20major%20axon%20tracts%20and%20interneuron%20migration%20in%20the%20forebrain.Development13322432252" data-title="Robo1%20regulates%20the%20development%20of%20major%20axon%20tracts%20and%20interneuron%20migration%20in%20the%20forebrain." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Robo1+regulates+the+development+of+major+axon+tracts+and+interneuron+migration+in+the+forebrain.+Andrews+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref75"><span class="order">75.
            </span><a name="pgen.1000879-Liu1" id="pgen.1000879-Liu1" class="link-target"></a>Liu J, Zhang L, Wang D, Shen H, Jiang M, et al.  (2003) Congenital diaphragmatic hernia, kidney agenesis and cardiac defects associated with Slit3-deficiency in mice. Mech Dev  120: 1059–1070.  <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuJZhangLWangDShenHJiangM2003Congenital%20diaphragmatic%20hernia%2C%20kidney%20agenesis%20and%20cardiac%20defects%20associated%20with%20Slit3-deficiency%20in%20mice.Mech%20Dev12010591070" data-title="Congenital%20diaphragmatic%20hernia%2C%20kidney%20agenesis%20and%20cardiac%20defects%20associated%20with%20Slit3-deficiency%20in%20mice." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Congenital+diaphragmatic+hernia%2C+kidney+agenesis+and+cardiac+defects+associated+with+Slit3-deficiency+in+mice.+Liu+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref76"><span class="order">76.
            </span><a name="pgen.1000879-Barik1" id="pgen.1000879-Barik1" class="link-target"></a>Barik S (2008) An intronic microRNA silences genes that are functionally antagonistic to its host gene. Nucleic Acids Res  36: 5232–5241.  <ul class="reflinks"><li><a href="#" data-author="Barik" data-cit="BarikS2008An%20intronic%20microRNA%20silences%20genes%20that%20are%20functionally%20antagonistic%20to%20its%20host%20gene.Nucleic%20Acids%20Res3652325241" data-title="An%20intronic%20microRNA%20silences%20genes%20that%20are%20functionally%20antagonistic%20to%20its%20host%20gene." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+intronic+microRNA+silences+genes+that+are+functionally+antagonistic+to+its+host+gene.+Barik+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

