



<div class="abstract toc-section"><a id="abstract1" name="abstract1" data-toc="abstract1" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract2.p1" name="article1.front1.article-meta1.abstract2.p1" class="link-target"></a><p>Pathogens that rely upon multiple hosts to complete their life cycles often modify behavior and development of these hosts to coerce them into improving pathogen fitness. However, few studies describe mechanisms underlying host coercion. In this study, we elucidate the mechanism by which an insect-transmitted pathogen of plants alters floral development to convert flowers into vegetative tissues. We find that phytoplasma produce a novel effector protein (SAP54) that interacts with members of the MADS-domain transcription factor (MTF) family, including key regulators SEPALLATA3 and APETALA1, that occupy central positions in the regulation of floral development. SAP54 mediates degradation of MTFs by interacting with proteins of the RADIATION SENSITIVE23 (RAD23) family, eukaryotic proteins that shuttle substrates to the proteasome. Arabidopsis <em>rad23</em> mutants do not show conversion of flowers into leaf-like tissues in the presence of SAP54 and during phytoplasma infection, emphasizing the importance of RAD23 to the activity of SAP54. Remarkably, plants with SAP54-induced leaf-like flowers are more attractive for colonization by phytoplasma leafhopper vectors and this colonization preference is dependent on RAD23. An effector that targets and suppresses flowering while simultaneously promoting insect herbivore colonization is unprecedented. Moreover, RAD23 proteins have, to our knowledge, no known roles in flower development, nor plant defence mechanisms against insects. Thus SAP54 generates a short circuit between two key pathways of the host to alter development, resulting in sterile plants, and promotes attractiveness of these plants to leafhopper vectors helping the obligate phytoplasmas reproduce and propagate (zombie plants).</p>
</div><div class="abstract toc-section"><a id="abstract2" name="abstract2" class="link-target" data-toc="abstract2" title="Author Summary"></a>
<h2>Author Summary</h2>
<a id="article1.front1.article-meta1.abstract3.p1" name="article1.front1.article-meta1.abstract3.p1" class="link-target"></a><p>Parasites that colonize multiple hosts often coerce these hosts into improving their own survival and reproduction rates. However, how parasites do this is largely unknown. Phytoplasmas are bacterial plant parasites that require sap-feeding insect vectors—leafhoppers—for their propagation and dispersal. It has been known for a long time that phytoplasmas stimulate dramatic developmental changes in a broad range of plant species, such as the conversion of flowers into leaves known as phyllody and the proliferation of stems known as “witches' broom.” Here we report how and why phytoplasmas cause these dramatic developmental changes. We identified a phytoplasma virulence protein, SAP54, which transforms flowers into leaves and converts plants into more attractive hosts for the egg-laying and reproduction of their leafhopper vectors. We show that SAP54 exerts its effect by promoting the degradation of proteins that regulate important developmental processes in flowering plants. These proteins are highly conserved transcription factors of the MADS-box family, and reducing their activity through SAP54–mediated degradation curtails flower development, generating sterile plants. This degradation process requires RAD23, a protein that recruits the transcription factors to the protein degradation machinery. The resulting sterile plants, which form leaves in place of flowers, are more attractive to leafhoppers, arguably making phytoplasmas master manipulators of the parasite world.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>MacLean AM, Orlovskis Z, Kowitwanich K, Zdziarska AM, Angenent GC, Immink RGH, et al.  (2014) Phytoplasma Effector SAP54 Hijacks Plant Reproduction by Degrading MADS-box Proteins and Promotes Insect Colonization in a RAD23-Dependent Manner. PLoS Biol 12(4):
           e1001835.
        
        https://doi.org/10.1371/journal.pbio.1001835</p><p><strong>Academic Editor: </strong>Doris Wagner, University of Pennsylvania, United States of America</p><p><strong>Received: </strong>September 25, 2013; <strong>Accepted: </strong>February 28, 2014; <strong>Published: </strong> April 8, 2014</p><p><strong>Copyright: </strong> © 2014 MacLean et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by Biotechnology and Biological Sciences Research Council (BBSRC) Grants BB/J004553/1 and BB/K002848/1, The John Innes Centre (JIC), and The Gatsby Charitable Foundation. The JIC is grant-aided by the BBSRC. A.M.M. was funded by a Marie Curie International Incoming Fellowship FP7-PEOPLE-2010-IIF Grant Agreement Number 274444, and A.M.Z was supported by the EU-LLP Erasmus programme. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p><p><strong>Abbreviations:
          </strong>AG,
            AGAMOUS; AGL,
            AGAMOUS-LIKE; AP1,
            APETALA1; AY-WB,
            Aster Yellows phytoplasma strain Witches' Broom; Col-0,
            Columbia; FUL,
            FRUITFULL; GFP,
            Green fluorescent protein; K-domain,
            Keratin-like domain; MAF1,
            MADS AFFECTING FLOWERING1; MTFs,
            MADS-domain transcription factors; MIKC<sup>C</sup>,
            canonical MADS-domain proteins; MIKC*,
            noncanonical MADS-domain proteins; RAD23,
            RADIATION SENSITIVE23; RFP,
            red fluorescent protein; SEP,
            SEPALLATA; SOC1,
            SUPPRESSOR OF OVEREXPRESSION OF CONSTANS1; UPS,
            ubiquitin/26S proteasome system</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Microorganisms that inhabit eukaryotic hosts must adapt themselves to these living environments, or manipulate their hosts to permit colonization. A classical example of the latter is the release of proteineaceous effectors (i.e., virulence proteins) by microbial pathogens to modulate processes of their eukaryotic hosts, with most effectors acting either directly or indirectly to suppress host defence responses. Prominent examples of these include type III effectors of Gram-negative bacteria such as <em>Pseudomonas</em> and <em>Salmonella</em> <a href="#pbio.1001835-Dean1" class="ref-tip">[1]</a>, RXLR and Crinkler proteins secreted by oomycetes <a href="#pbio.1001835-Bozkurt1" class="ref-tip">[2]</a>, and the TAL effectors of <em>Xanthomonas</em>, which bind to host promoters and misregulate gene expression <a href="#pbio.1001835-Romer1" class="ref-tip">[3]</a>. By these means, the pathogen impairs the host's ability to defend itself, thereby promoting host susceptibility to the invading microorganism.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Pathogens and parasites may also manipulate the behavior and development of their hosts. The protozoan <em>Toxoplasma gondii</em> modifies the behaviour of rats in response to the scent of cat urine, reprogramming the rat's behavioural responses to increase its likelihood of predation <a href="#pbio.1001835-Vyas1" class="ref-tip">[4]</a>. Similarly, the lancet liver fluke infects the brain of ants and compels the insect to climb to the top of a blade of grass and remain motionless until ingested by a grazing ruminant. Thus, these parasites coerce host behavior to improve their opportunities for transmission to a new host. Parasites may also influence host development, with known examples including a pathogenic fungus (<em>Puccinia monoica</em>) that stimulates the growth of pseudo-flowers from infected plant hosts to attract insects that subsequently “pollinate” the fungus <a href="#pbio.1001835-Roy1" class="ref-tip">[5]</a> and bacterial pathogens that alter the profile of organic volatiles released from infected plants as a means of attracting insect vectors <a href="#pbio.1001835-Mayer1" class="ref-tip">[6]</a>. However, molecular mechanisms by which pathogens alter either host behaviour or development are largely unknown.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Host coercion is particularly important for obligate pathogens that are completely dependent on their hosts. Phytoplasma are bacterial plant pathogens that have a dual host life cycle that is dependent on sap-feeding insects for transmission to plants <a href="#pbio.1001835-Sugio1" class="ref-tip">[7]</a>. Insect vectors (planthoppers, leafhoppers, and psyllids) acquire phytoplasma by ingesting the phloem of infected plants. The insect vectors become competent to transmit the bacteria to healthy plants following the colonization of salivary glands by phytoplasma, which are subsequently released into the phloem with saliva during insect feeding. Aster Yellows phytoplasma strain Witches' Broom (AY-WB) can infect a broad range of plants, eliciting symptoms such as phyllody (conversion of flowers into leaf-like structures), virescence (greening of floral organs such as petals and stamens), and witches' brooms (increased proliferation of stems) reflecting perturbations in host development that are beneficial to AY-WB or its insect vector <a href="#pbio.1001835-Sugio1" class="ref-tip">[7]</a>,<a href="#pbio.1001835-Sugio2" class="ref-tip">[8]</a>. We have previously identified an AY-WB effector protein (called SAP54) that transforms flowers into leaf-like vegetative tissues when expressed in <em>Arabidopsis thaliana</em> (hereafter Arabidopsis) <a href="#pbio.1001835-MacLean1" class="ref-tip">[9]</a>. Healthy Arabidopsis flowers are determinate structures with APETALA1 (AP1) regulating gene expression programs during the establishment of floral meristems <a href="#pbio.1001835-Kaufmann1" class="ref-tip">[10]</a>. Flowers of AY-WB–infected plants and SAP54-expressing transgenic Arabidopsis lines exhibit a loss of floral determinacy, reflected by vegetative shoots arising from the center of the flower and from the axil of the first whorl organs. We herein reveal the mechanism by which AY-WB phytoplasma coerces the plant host into suppressing its floral development to the benefit of this pathogen and its insect vector, but at the expense of the plant host reproductive success.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Phytoplasma Effector SAP54 Interacts with MADS-Domain Transcription Factors"></a>
<h3>Phytoplasma Effector SAP54 Interacts with MADS-Domain Transcription Factors</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>We wished to investigate how AY-WB phytoplasma alters flower architecture in infected plants. A yeast two-hybrid screen against an Arabidopsis seedling library (with SAP54 as bait) identified the Type II MADS-domain transcription factors (MTFs) AGAMOUS-LIKE 12 (AGL12), MADS AFFECTING FLOWERING1 (MAF1), and SEPALLATA3 (SEP3) as SAP54 interactors (<a href="#pbio.1001835.s013">Table S1</a>). To investigate the breadth of interactions between SAP54 and MTFs, we examined SAP54 interaction with 106 Arabidopsis MTFs in a matrix-based yeast two-hybrid screen <a href="#pbio.1001835-deFolter1" class="ref-tip">[11]</a>. This confirmed SAP54 interaction with AGL12, MAF1, and SEP3, and furthermore identified 12 additional interacting partners (<a href="#pbio-1001835-g001">Figure 1A</a> and <a href="#pbio.1001835.s014">Table S2</a>), including the well-characterized floral meristem identity and homeotic proteins AP1 <a href="#pbio.1001835-Kaufmann1" class="ref-tip">[10]</a>, and SEP3 paralogues SEP1, SEP2, and SEP4. SAP54 interacts solely with Type II MTFs and not with Type I MTFs in the two-hybrid experiments, indicating that SAP54 primarily targets the MTFs involved in floral transition and floral organ development.</p>
<a class="link-target" id="pbio-1001835-g001" name="pbio-1001835-g001"></a><div class="figure" data-doi="10.1371/journal.pbio.1001835.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pbio.1001835.g001" data-doi="info:doi/10.1371/journal.pbio.1001835" data-uri="info:doi/10.1371/journal.pbio.1001835.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pbio.1001835.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Phytoplasma SAP54 interacts specifically with the Keratin-like (K) domain of selected Type II MADS-box transcription factors (MTFs).</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) A comprehensive yeast two-hybrid screen of 106 Arabidopsis MTFs reveals that SAP54 interacts with members of the Type II subfamily of MTFs (proteins that interact with SAP54 are indicated in red font). For simplicity, not all MTFs are included in the phylogenetic tree. (B) SAP54 interacts primarily with the K domain of AP1. AD, GAL4-activation domain; BD, GAL4-DNA binding domain; EV, empty vector control. (C) Flowers produced from healthy (left) and AY-WB–infected (right) Arabidopsis lines approximately 4 wk postinoculation. (D) SAP54 (indicated by an arrow) co-immunoprecipitates with SEP3–GFP but not FUL–GFP or AG–GFP. Flowers for immunoprecipitation experiments were harvested from transgenic lines pictured in panel C at an early point of infection (approximately 2 wk postinoculation) to minimize MTF loss due to destabilization. Equal loading of samples was confirmed via Bradford assays to quantify protein concentration.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pbio.1001835.g001">
              https://doi.org/10.1371/journal.pbio.1001835.g001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>Type II MTFs are modular proteins consisting of four domains. We found that SAP54 interacts primarily with the Keratin-like (K) domain of AP1 (<a href="#pbio-1001835-g001">Figure 1B</a>) and not with the more highly conserved MADS domain that is present in both plant and animal MTFs. The K domain contributes to the formation of MTF protein–protein interactions when these regulators associate as dimers and quartets <a href="#pbio.1001835-Yang1" class="ref-tip">[12]</a>,<a href="#pbio.1001835-Davies1" class="ref-tip">[13]</a>, and a classical K domain is specific to plant Type II MTFs <a href="#pbio.1001835-AlvarezBuylla1" class="ref-tip">[14]</a>. Therefore SAP54 may have evolved to selectively target plant MTFs (that contain a K domain) and not those of insects (that lack this domain), an important characteristic given that phytoplasmas effectively colonize many organs of their insect vectors <a href="#pbio.1001835-Sugio1" class="ref-tip">[7]</a>.</p>
<a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3" class="link-target"></a><p>To assess whether SAP54 interacts with MTFs in AY-WB–infected plants, we made use of Arabidopsis transgenic lines that express translational fusions of SEP3–GFP, FUL–GFP, or AG–GFP under control of their native promoters <a href="#pbio.1001835-Urbanus1" class="ref-tip">[15]</a> to conduct co-immunoprecipitation experiments. The transgenic lines produce leaf-like flowers that are indistinguishable from those of wild-type Arabidopsis plants when infected with phytoplasma (<a href="#pbio-1001835-g001">Figure 1C</a>), indicating that these lines are suitable to the study of SAP54–MTF interaction. To minimize MTF loss due to SAP54 destabilization (described below), we harvested flowers of transgenic lines shortly after infection with AY-WB (2 wk postinoculation) at a stage when flowers exhibit a normal (non-leaf-like) appearance, yet SAP54 is present in infected plants. SAP54 co-immunoprecipitated with SEP3–GFP in samples of AY-WB–infected plants, but no corresponding protein was detected in healthy controls (<a href="#pbio-1001835-g001">Figure 1D</a> and <a href="#pbio.1001835.s015">Table S3</a>). In contrast, SAP54 did not co-immunoprecipate with FUL–GFP or AG–GFP (<a href="#pbio-1001835-g001">Figure 1D</a>). This experiment confirms that AY-WB produces SAP54 during infection of Arabidopsis and that SAP54 interacts with SEP3 in AY-WB–infected plants. As well, while SAP54 interacts with 15 MTFs in a yeast two-hybrid system (<a href="#pbio-1001835-g001">Figure 1A</a>), it is possible that only a subset of these interactions occur <em>in planta</em>. To address this, we conducted an immunoprecipitation experiment of GFP–SAP54 from <em>35S:GFP</em>–<em>SAP54</em> transgenic Arabidopsis followed by a mass spectrometry analysis to identify interacting proteins. Peptides associated with Type II MTFs MAF1, SUPPRESSOR OF OVEREXPRESSION OF CONSTANS1 (SOC1), SEP1, SEP2, and AP1 were recovered in samples immunoprecipitated with GFP–SAP54 but not with GFP alone (<a href="#pbio.1001835.s016">Table S4</a>). Thus, we conclude that SAP54 indeed interacts with MTFs in infected plants, and that many of the MTFs identified as SAP54 interactors in the yeast two-hybrid screens also interact with SAP54 <em>in planta</em>.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="SAP54 Destabilizes MTFs in an Ubiquitin/26S Proteasome-Dependent Manner"></a>
<h3>SAP54 Destabilizes MTFs in an Ubiquitin/26S Proteasome-Dependent Manner</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>The floral architecture of SAP54-expressing and AY-WB–infected Arabidopsis resembles that of higher order <em>sep</em> mutants (i.e., loss of floral determinacy and conversion of floral organs into leaf-like structures <a href="#pbio.1001835-Ditta1" class="ref-tip">[16]</a>,<a href="#pbio.1001835-Pelaz1" class="ref-tip">[17]</a>), and we hypothesized that SAP54 may act to perturb MTF function. Western blots performed using flowers collected from healthy and AY-WB–infected Arabidopsis lines expressing AP1–GFP or SEP3–GFP revealed that these MTFs appear to be less abundant in infected leaf-like flowers harvested at a late stage of infection (4+ wk postinoculation) (<a href="#pbio-1001835-g002">Figure 2A</a> and <a href="#pbio.1001835.s017">Table S5</a>), suggesting that phytoplasma may act to destabilize these transcription factors. Phytoplasma are obligate biotrophs that are genetically intractable, and thus we were unable to generate an AY-WB <em>SAP54</em> mutant. Therefore, we examined the interaction between SAP54–MTFs more closely by co-expression assays in <em>Nicotiana benthamiana</em> using pTRBO-based vectors, which are <em>Tobacco mosaic virus</em>–based expression vectors that allow for higher levels of <em>in planta</em> protein production compared with 35S constructs <a href="#pbio.1001835-Sugio2" class="ref-tip">[8]</a>,<a href="#pbio.1001835-Lindbo1" class="ref-tip">[18]</a>. We observed that whereas Type II MTFs were detected on immunoblots when transiently co-expressed with a control protein (pTRBO::<em>Flag-RFP</em>), the accumulation of AP1, SEP3, and SOC1 was much reduced or undetectable when co-expressed as 10xmyc-tagged proteins with pTRBO::<em>Flag-SAP54</em> (<a href="#pbio-1001835-g002">Figure 2B</a> and <a href="#pbio.1001835.s018">Table S6</a>). In contrast, the accumulations of non-SAP54 interacting Type I MTFs AGL50, AGL62, and AGL80 in the presence of SAP54 were not or only weakly reduced (<a href="#pbio.1001835.s001">Figure S1A</a>). SAP54 interacted with members of the Type II canonical MADS-box proteins (MIKC<sup>C</sup>) in yeast two-hybrid screens, but not with MIKC* proteins, which have a distinct Keratin-like domain <a href="#pbio.1001835-Verelst1" class="ref-tip">[19]</a>. Consistent with this is the observation that the Type II MIKC* type AGL66 was not destabilized in the presence of SAP54 in <em>N. benthamiana</em> (<a href="#pbio.1001835.s001">Figure S1B</a>). Thus, the phytoplasma effector SAP54 appears to selectively destabilize Type II MIKC<sup>C</sup> MTFs that are the key regulators of floral organ formation in flowering plants.</p>
<a class="link-target" id="pbio-1001835-g002" name="pbio-1001835-g002"></a><div class="figure" data-doi="10.1371/journal.pbio.1001835.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pbio.1001835.g002" data-doi="info:doi/10.1371/journal.pbio.1001835" data-uri="info:doi/10.1371/journal.pbio.1001835.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pbio.1001835.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Phytoplasma SAP54 interacts with and destabilizes MADS-box transcription factors in plants.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>(A) MTFs AP1 and SEP3 are destabilized in AY-WB–infected Arabidopsis lines. Flowers from healthy and phytoplasma-infected plants were harvested approximately 4 wk postinoculation. (B) MTFs are destabilized when expressed in the presence of SAP54. 10xmyc-tagged MTFs were transiently co-expressed with flag-tagged SAP54 or an RFP control in agroinfiltrated <em>N. benthamiana</em> leaves. (C) SAP54-mediated destabilization of AP1 is inhibited by epoxomicin. Infiltrated tissues were treated with 50 µM epoxomicin (dissolved in DMSO) 8 h prior to harvest. (D) MTFs AP1, SEP3, and SOC1 co-immunoprecipitate with GFP-tagged SAP54. Co-immunoprecipitation experiments of these Type II MTFs were performed alongside Type I MTF AGL50, which was not detected. Proteins were transiently expressed in <em>N. benthamiana</em> in the presence or absence of 50 µM epoxomicin to stabilize MTFs.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pbio.1001835.g002">
              https://doi.org/10.1371/journal.pbio.1001835.g002</a></p></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>SAP54 may act directly as a protease to catalyze the proteolysis of select MTFs, or alternatively, this effector may exploit a host mechanism, such as the ubiquitin/26S proteasome system (UPS) to degrade MTFs. Treatment of infiltrated samples with a protease inhibitor cocktail did not affect SAP54 activity (<a href="#pbio.1001835.s001">Figure S1C</a>), however treatment with epoxomicin, a potent inhibitor of the UPS <a href="#pbio.1001835-Meng1" class="ref-tip">[20]</a>, prevented the SAP54-mediated destabilization of AP1 (<a href="#pbio-1001835-g002">Figure 2C</a>, <a href="#pbio.1001835.s019">Table S7</a>, and <a href="#pbio.1001835.s001">Figure S1C</a>). Thus, SAP54 is likely to degrade MTFs via the host UPS. All eukaryotes have a UPS, including yeast. Nonetheless, SAP54 did not degrade the MTFs in yeast two-hybrid experiments in which SAP54 was fused to the GAL4 DNA binding domain (GAL4BD–SAP54) and Type II MIKC<sup>C</sup> MTFs to the GAL4 activation domain (GAL4AD–AP1, GAL4AD–SEP3, and GAL4AD–SOC1) (<a href="#pbio.1001835.s001">Figure S1D</a>). Therefore, the SAP54-mediated degradation of MTFs may require a helper protein(s) from the plant host.</p>
<a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3" class="link-target"></a><p>We lastly employed the transient expression system in <em>N. benthamiana</em> to confirm the interactions between SAP54 and MTFs <em>in planta</em>. 10xmyc-tagged AP1, SEP3, and SOC1 readily co-immunoprecipitate with GFP-tagged SAP54 (but not GFP), whereas Type I AGL50 (Type I MTF) was not pulled down with SAP54 (<a href="#pbio-1001835-g002">Figure 2D</a>, <a href="#pbio.1001835.s020">Tables S8</a>, <a href="#pbio.1001835.s021">S9</a>).</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="SAP54 Interacts with Ubiquitin Binding Proteins RAD23C and RAD23D"></a>
<h3>SAP54 Interacts with Ubiquitin Binding Proteins RAD23C and RAD23D</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>In addition to the MTFs, the yeast two-hybrid screen against the Arabidopsis seedling library revealed that SAP54 interacts with RADIATION SENSITIVE23 (RAD23) family isoforms RAD23C and RAD23D (<a href="#pbio.1001835.s013">Table S1</a>). RAD23 proteins have been proposed to act as shuttle proteins to deliver ubiquitinated substrates to the UPS for degradation in eukaryotes <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a>. Arabidopsis encodes four RAD23 isoforms, however the yeast two-hybrid data indicate that SAP54 interacts specifically with RAD23C and RAD23D but not RAD23A or RAD23B (<a href="#pbio-1001835-g003">Figure 3A</a>). This is consistent with the observation that RAD23C and RAD23D co-immunoprecipitate in the presence of GFP–SAP54 (but not GFP alone) (<a href="#pbio-1001835-g003">Figure 3B</a>) in Arabidopsis. RAD23 proteins did not co-immunoprecipitate with GFP–SAP54 in the <em>rad23CD</em> mutant (<a href="#pbio-1001835-g003">Figure 3B</a>), indicating that SAP54 prefers to interact with RAD23C and RAD23D as opposed to RAD23A and RAD23B <em>in planta</em>.</p>
<a class="link-target" id="pbio-1001835-g003" name="pbio-1001835-g003"></a><div class="figure" data-doi="10.1371/journal.pbio.1001835.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pbio.1001835.g003" data-doi="info:doi/10.1371/journal.pbio.1001835" data-uri="info:doi/10.1371/journal.pbio.1001835.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pbio.1001835.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> Phytoplasma SAP54 interacts with Arabidopsis RAD23 proteins.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>(A) SAP54 interacts with Arabidopsis RAD23C and RAD23D but not RAD23A or RAD23B isoforms in yeast two-hybrid assays. (B) RAD23C (44 kDa) and RAD23D (40 kDa) co-immunoprecipitate with GFP–SAP54 in samples obtained from transgenic Arabidopsis expressing <em>35S:GFP–SAP54</em>. We did not detect RAD23 following immunoprecipitation of GFP (in transgenic Arabidopsis expressing <em>35S:GFP</em>), nor did we detect an interaction with RAD23A or RAD23B in an Arabidopsis <em>rad23CD</em> double mutant. Equal loading of samples was verified via Bradford assays to confirm protein concentration. (C) Flowers produced from transgenic lines expressing <em>35S:GFP–SAP54</em> in wild-type (Col-0) and <em>rad23</em> mutant Arabidopsis lines indicate that SAP54-induced phyllody requires RAD23C and RAD23D. (D) Western blot analysis reveals GFP–SAP54 expression levels in rosette leaves harvested from plants in panel C. GFP–SAP54 is indicated by an arrow. AD, GAL4-activation domain; BD, GAL4–DNA binding domain; EV, empty vector control.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pbio.1001835.g003">
              https://doi.org/10.1371/journal.pbio.1001835.g003</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="RAD23 Proteins Are Essential for Phytoplasma-Induced Phyllody"></a>
<h3>RAD23 Proteins Are Essential for Phytoplasma-Induced Phyllody</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>With the consideration that SAP54 degrades MTFs and that RAD23 proteins act to shuttle poly-ubiquitinated substrates to the UPS, we hypothesized that the Arabidopsis RAD23 proteins may be required for SAP54-mediated degradation. Although the RAD23 proteins are essential, the proteins have largely redundant functions and <em>rad23</em> single mutants and the majority of <em>rad23</em> double mutants do not exhibit any obvious developmental defects (<a href="#pbio.1001835.s002">Figure S2</a>), although higher order <em>rad23B</em> mutants do demonstrate various pleiotropic phenotypes (i.e., shorter stature, smaller siliques, and reduced seed production) <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a>.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>To investigate if RAD23 proteins contribute to SAP54-induced leaf-like flowers, homozygous Arabidopsis <em>rad23</em> T-DNA mutant lines <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a> were transformed with <em>35S:GFP–SAP54</em> (<a href="#pbio-1001835-g003">Figure 3C</a>). In a wild-type Arabidopsis background, expression of <em>GFP–SAP54</em> induces a strong degree of phyllody (growth of leaf-like flowers), virescence (greening of floral organs), and a frequent loss of floral determinacy as evidenced by the outgrowth of stems from the centre of the flower. However, approximately one half of <em>35S:GFP–SAP54–</em>expressing transformants obtained from <em>rad23C</em> (48 of 109 transgenic plants) and <em>rad23D</em> (8 of 22 transgenic plants) mutant lines exhibited a milder degree of phyllody, with loss of determinacy typically restricted to the early onset flowers (<a href="#pbio-1001835-t001">Table 1</a> and <a href="#pbio.1001835.s003">Figure S3</a>). Moreover, the majority of transformants originating from <em>rad23CD</em> double mutants (124 of 138 transgenic plants) and <em>rad23BCD</em> triple mutants (50 of 64 transgenic plants) produced flowers that displayed no signs of phyllody or virescence (<a href="#pbio-1001835-g003">Figure 3C</a>). Western blot analysis confirmed the expression of the GFP-tagged SAP54 in the transgenic lines (<a href="#pbio-1001835-g003">Figure 3D</a>) and revealed that the mild phyllody observed in a minority of <em>rad23CD</em> transgenic lines (14 of 138 transgenic plants) was likely due to a very high level of SAP54 expression (<a href="#pbio.1001835.s004">Figure S4</a>). In contrast, <em>rad23AC</em>, <em>rad23AD</em>, <em>rad23BC</em>, <em>rad23BD</em>, and <em>rad23ABD</em> mutant transgenic lines produced flowers comparable to those observed in a wild-type (Col-0) background (<a href="#pbio-1001835-g003">Figure 3C</a>, <a href="#pbio-1001835-t001">Table 1</a>). Thus, the SAP54-mediated degradation of MTFs is dependent predominantly on RAD23C and RAD23D, whereas other RAD23 isoforms may be involved depending on SAP54 abundance.</p>
<a class="link-target" id="pbio-1001835-t001" name="pbio-1001835-t001"></a><div class="figure" data-doi="10.1371/journal.pbio.1001835.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pbio.1001835.t001" data-doi="info:doi/10.1371/journal.pbio.1001835" data-uri="info:doi/10.1371/journal.pbio.1001835.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pbio.1001835.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Phenotype scoring of <em>35S:GFP–SAP54</em> transgenic lines.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pbio.1001835.t001">
              https://doi.org/10.1371/journal.pbio.1001835.t001</a></p></div><a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3" class="link-target"></a><p>To assess whether the Arabidopsis RAD23 proteins are also required for SAP54-mediated MTF destabilization during AY-WB infection, the various <em>rad23</em> single, double, and triple T-DNA insertion mutants were infected with AY-WB phytoplasma. AY-WB–infected <em>rad23BCD</em> triple mutants produced determinate flower-like organs that resemble those of healthy wild-type Col-0 plants (<a href="#pbio-1001835-g004">Figure 4</a>). The degradation of MTF SEP3 was lost in the phytoplasma-infected <em>rad23BCD</em> mutant, whereas degradation of this MTF was observed in <em>rad23BD</em> mutant (with leaf-like flowers) (<a href="#pbio-1001835-g004">Figure 4</a>). Remarkably these plants still showed other symptoms of infection (<a href="#pbio.1001835.s005">Figure S5</a>), such as the witches' brooms that are typically observed in AY-WB–infected Arabidopsis plants <a href="#pbio.1001835-Sugio1" class="ref-tip">[7]</a>. In contrast, the <em>rad23</em> single and double mutants, including <em>rad23BD</em> and <em>rad23CD</em> mutant lines, and the <em>rad23ABD</em> triple mutants produced leaf-like indeterminate flowers that resemble those of AY-WB–infected wild-type Col-0 plants (<a href="#pbio-1001835-g004">Figure 4</a>, <a href="#pbio.1001835.s006">Figure S6</a>). We conclude from these results that the SAP54-mediated degradation of MTFs requires predominantly RAD23C and RAD23D, but that RAD23B may also facilitate this process in infected hosts.</p>
<a class="link-target" id="pbio-1001835-g004" name="pbio-1001835-g004"></a><div class="figure" data-doi="10.1371/journal.pbio.1001835.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pbio.1001835.g004" data-doi="info:doi/10.1371/journal.pbio.1001835" data-uri="info:doi/10.1371/journal.pbio.1001835.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pbio.1001835.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Arabidopsis <em>rad23BCD</em> triple mutants do not exhibit symptoms of virescence or phyllody when infected with AY-WB.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p>(A) Flowers produced from AY-WB–infected <em>rad23BD</em> mutants produce leaf-like flowers, whereas infected <em>rad23BCD</em> mutants grow flowers that resemble those of healthy plants. (B) Western blot analysis reveals that SEP3 is destabilized in <em>rad23BD</em> leaf-like flowers but not in <em>rad23BCD</em> flowers. SAP54 was detected in flowers harvested from AY-WB–infected <em>rad23</em> mutants but not healthy Arabidopsis plants. (C) The infection status of plants in panel A was confirmed using primers specific for AY-WB.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pbio.1001835.g004">
              https://doi.org/10.1371/journal.pbio.1001835.g004</a></p></div></div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="SAP54 Enhances Phytoplasma Insect Vector Colonization in a RAD23-Dependent Manner"></a>
<h3>SAP54 Enhances Phytoplasma Insect Vector Colonization in a RAD23-Dependent Manner</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>We hypothesized that AY-WB may induce leaf-like flowers as a means of attracting its insect vector, which feeds from the phloem of vegetative tissues. We conducted choice experiments in which <em>M. quadrilineatus</em> adults (10 males and 10 females) were released in the middle of a confined space (<a href="#pbio.1001835.s007">Figure S7</a>) and were allowed free access to AY-WB–infected <em>rad23BD</em> (leaf-like flowers) and <em>rad23BCD</em> (non-leaf-like flowers) plants (<a href="#pbio-1001835-g005">Figure 5A</a>). Insect preference was then assessed by counting the number of nymphs produced from eggs oviposited on individual plants. Insects produced more progeny on infected <em>rad23BD</em> plants versus infected <em>rad23BCD</em> plants (<em>t</em><sub>(5)</sub> = 4.7; <em>p</em> = 0.042; <a href="#pbio-1001835-g005">Figure 5A</a>), supporting a hypothesis that plants with leaf-like flowers may be more attractive hosts for leafhopper ovipositing. No differences in leafhopper progeny numbers were observed between healthy <em>rad23BD</em> and <em>rad23BCD</em> mutants (<em>t</em><sub>(5)</sub> = 0.45; <em>p</em> = 0.694; <a href="#pbio-1001835-g005">Figure 5A</a>), indicating that insects do not exhibit a preference for either T-DNA mutant line in the absence of AY-WB infection.</p>
<a class="link-target" id="pbio-1001835-g005" name="pbio-1001835-g005"></a><div class="figure" data-doi="10.1371/journal.pbio.1001835.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pbio.1001835.g005" data-doi="info:doi/10.1371/journal.pbio.1001835" data-uri="info:doi/10.1371/journal.pbio.1001835.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pbio.1001835.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pbio.1001835.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pbio.1001835.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pbio.1001835.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> Aster leafhopper <em>Macrosteles quadrilineatus</em> demonstrates oviposition preference for plants with leaf-like flowers.</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>(A) <em>M. quadrilineatus</em> produces significantly more progeny on AY-WB–infected <em>rad23BD</em> mutants (leaf-like flower phenotype) compared to <em>rad23BCD</em> mutant plants (non-leaf-like flower phenotype) (<em>t</em><sub>(5)</sub> = 4.7; <em>p</em> = 0.042). Insects do not exhibit a preference between healthy <em>rad23BD</em> and <em>rad23BCD</em> plants (<em>t</em><sub>(5)</sub> = 0.45; <em>p</em> = 0.694). (B) <em>M. quadrilineatus</em> adults produce more nymphs on transgenic Arabidopsis expressing GFP-tagged SAP54 (leaf-like flowers) compared to GFP control plants (wild-type flowers) (<em>t</em><sub>(7)</sub> = 6.45; <em>p</em> = 0.008). In these experiments, 10 male and 10 female <em>M. quadrilineatus</em> adults were released in a choice cage containing two test plants for the period of 5 d. After removal of adult insects, plants were bagged individually and incubated for 14 d to allow nymph emergence. The graphs in panel A and B represent the percentage of <em>M. quadrilineatus</em> nymphs found on each test plant within a single choice cage.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pbio.1001835.g005">
              https://doi.org/10.1371/journal.pbio.1001835.g005</a></p></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>We likewise performed leafhopper choice assays with transgenic Arabidopsis lines expressing <em>35S:GFP</em> (wild-type flowers) or <em>35S:GFP–SAP54</em> (leaf-like flowers) and determined that <em>M. quadrilineatus</em> females preferentially oviposit on transgenic lines that express GFP–SAP54 (<em>t</em><sub>(7)</sub> = 6.45; <em>p</em> = 0.008; <a href="#pbio-1001835-g005">Figure 5B</a>). As the transgenic lines were not infected with AY-WB, these results clearly indicate that SAP54 has the ability to modulate plant–insect interactions, even in the absence of any additional phytoplasma proteins, thereby promoting AY-WB fitness by enhancing leafhopper vector colonization and oviposition.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3" class="link-target"></a><p>Lastly, we wished to assess the contribution of green leaf-like flowers to insect preference in the absence of both AY-WB and SAP54. We thus performed leafhopper choice experiments in which insects were allowed access to Col-0 (wild-type) plants and <em>ap1</em> mutants that produce leaf-like flowers with similarity to mild SAP54-expressing transgenic lines (<a href="#pbio.1001835.s008">Figure S8</a>). Interestingly, the insects did not exhibit a preference, producing equivalent progeny on Arabidopsis wild-type plants and <em>ap1</em> mutants (<em>t</em><sub>(11)</sub> = 0.22; <em>p</em> = 0.835; <a href="#pbio.1001835.s009">Figure S9</a>). The developmental defects in <em>ap1</em> mutant flowers (which lack petals but are nonetheless fertile flowers that produce seed) may not be severe enough to enhance the attractiveness to the insects. Most likely, SAP54-mediated destabilization of additional MTFs (for example, the SEP paralogues and SOC1) is required for the conversion of all floral whorls into leaf-like organs with increased vegetative biomass, which may be important to modulating plant–insect interactions, or SAP54 may promote insect colonization via a mechanism that extends beyond the growth of leaf-like flowers (for example, modulation of phytohormones). Taken together, our data indicate a role of phytoplasma effector protein SAP54 and Arabidopsis RAD23 in flower development and plant defense to insects.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>MTFs comprise a key family of eukaryotic transcription factors that occupy central positions in floral development, dictating both the transition to flowering and the formation of floral organs in all angiosperms <a href="#pbio.1001835-Smaczniak1" class="ref-tip">[22]</a>. The function of MTFs has been the focus of much study, and these proteins have no known role in plant defence. Similarly, RAD23 proteins are broadly conserved among eukaryotes and play essential roles in mediating the turnover of plant proteins, as evidenced by the lethality of the quadruple <em>rad23ABCD</em> Arabidopsis mutant <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a>. Microbial effectors functionally characterized so far target specific processes within defence-related pathways, whereas the phytoplasma effector SAP54 has uniquely evolved to link two key pathways involved in plant reproduction and basic protein regulatory processes to alter host development and shape, and thus represents a highly unusual protein. The advantage of targeting conserved host proteins may be particularly relevant for AY-WB, which has a broad plant host range and where the selection of a host is made by the insect vector and not by the phytoplasma. By employing effectors that target conserved (sub)families of plant proteins, AY-WB increases the likelihood that it can modulate developmental processes in the plant species chosen by the insect.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>Green leaf-like flowers are a hallmark trait of phytoplasma-infected plants, and we propose that this characteristic trademark is the result of an effector-mediated destabilization of conserved MTFs (i.e., posttranslational regulation via proteolysis). AP1 (with CAL and FUL) and the four SEP proteins play prominent roles in the establishment of a floral meristem and in regulating floral organ identity in the majority of flowering plants <a href="#pbio.1001835-Ditta1" class="ref-tip">[16]</a>,<a href="#pbio.1001835-Ferrandiz1" class="ref-tip">[23]</a>. Moreover, SEP3 occupies a central position in mediating the higher order protein interactions necessary to form MTF regulatory complexes <a href="#pbio.1001835-Immink1" class="ref-tip">[24]</a>. Destabilization of SEP3 (and AP1) by SAP54 is expected to significantly impair the establishment of the MTF protein complexes that are necessary to regulate floral development, especially if accompanied by the destabilization of SEP1, SEP2, and SEP4. Whereas there is much evidence that the expression of various MTFs is mis-regulated in phytoplasma-infected plants <a href="#pbio.1001835-Pracros1" class="ref-tip">[25]</a>–<a href="#pbio.1001835-Himeno1" class="ref-tip">[27]</a>, it is unlikely that phytoplasma are directly targeting gene expression. Indeed, the complex auto-regulatory and cross-regulatory network by which MTF gene expression is regulated dictates that the destabilization of several MTFs within this network will result in aberrant gene expression. We have identified several putative SAP54 homologues in other phytoplasma strains (<a href="#pbio.1001835.s010">Figure S10</a>), thus indicating that SAP54 may be a member of a larger effector family that modulates floral development.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>Animals such as insects also encode MTFs, although a classical Keratin-like domain is absent in these proteins <a href="#pbio.1001835-AlvarezBuylla1" class="ref-tip">[14]</a>. As an insect-vectored pathogen of plants, AY-WB infects and colonizes both plants and animals. However, the relationship that exists between the phytoplasma and these two hosts is not the same. Phytoplasma are reliant upon their insect vectors for dispersal in the environment, and the association between the phytoplasma and the insects can be beneficial to both organisms <a href="#pbio.1001835-Sugio1" class="ref-tip">[7]</a>. In contrast, phytoplasma are aggressive pathogens of plants, and exposure of a susceptible plant to phytoplasma eventually leads to the death of the plant. Accordingly, SAP54 may have evolved to bind to the plant specific Keratin-like domain of Type II MTFs as a means of selectively targeting plant MTFs and not insect MTFs, which might have a deleterious effect upon AY-WB's vector. Consistent with this hypothesis is the observation that SAP54 expression is up-regulated in plants compared to insects, and many other genes encoding candidate AY-WB effectors show a host-specific expression <a href="#pbio.1001835-MacLean1" class="ref-tip">[9]</a>. Thus, phytoplasmas are likely to produce insect- and plant-specific effectors, in line with their life cycle involving alternate hosts.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>SAP54 may escape degradation via the host UPS, as evidenced by the lack of an increase in SAP54 protein levels in epoxomicin-treated tissues (<a href="#pbio-1001835-g002">Figure 2C</a>). RAD23 resists proteasomal degradation and is released from the UPS to bind other cargo <a href="#pbio.1001835-Dantuma1" class="ref-tip">[28]</a>. It is possible that SAP54 resists degradation by associating with RAD23. SAP54 may simultaneously interact with MTFs and RAD23 upon which the MTFs are delivered to the UPS for degradation, whereas SAP54–RAD23 complexes are released to bind more MTFs (<a href="#pbio.1001835.s011">Figure S11A</a>). Although we did not observe ubiquitylation of SAP54 and MTFs in the proteomics analyses, we cannot exclude the possibility that RAD23 interacts with ubiquitin groups linked via lysine (K) residue(s) on SAP54 (<a href="#pbio.1001835.s011">Figure S11B</a>). A third possibility is that another pathway is involved in transportation of SAP54–MTF complexes to the host UPS, whereupon SAP54 interacts with RAD23 to evade degradation (<a href="#pbio.1001835.s011">Figure S11C</a>). Future research is focused on dissecting these possibilities and the characterization of other plant proteins that are involved in the SAP54-mediated degradation of MTFs in a RAD23-dependent manner. To our knowledge, this is the first report of a microbial effector that recruits RAD23 proteins to enable the inappropriate degradation of a host protein. Nevertheless, exploitation of the UPS by microbial effectors is rapidly emerging as a common theme in plant–pathogen interactions <a href="#pbio.1001835-Birch1" class="ref-tip">[29]</a>–<a href="#pbio.1001835-Angot1" class="ref-tip">[31]</a>. Pathogen effectors that suppress, stabilize, or mimic the activity of host ubiquitin ligases have been described <a href="#pbio.1001835-Zhang1" class="ref-tip">[32]</a>–<a href="#pbio.1001835-Singer1" class="ref-tip">[36]</a>, whereas other effectors act as deubiquitylases that catalyze the removal of ubiquitin tags from host proteins <a href="#pbio.1001835-LeNegrate1" class="ref-tip">[37]</a>,<a href="#pbio.1001835-LeNegrate2" class="ref-tip">[38]</a>. Pathogens also exploit the host UPS to regulate the degradation, localization, and activity of their own effectors via ubiquitination of these proteins, or even by using ubiquitin as a cofactor <a href="#pbio.1001835-Kubori1" class="ref-tip">[39]</a>–<a href="#pbio.1001835-Patel1" class="ref-tip">[41]</a>. Along this line, RAD23 has been shown to interact with a <em>Pseudomonas syringae</em> effector, HopM1 <a href="#pbio.1001835-Nomura1" class="ref-tip">[42]</a>, suggesting that RAD23 may have a role in mediating defense-related processes in other host–microbe interactions as well. Taken together, our study extends these earlier observations to reveal that pathogens also exploit host machinery (i.e., shuttle proteins) that is accessory to the UPS.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>Insect choice assays involving both AY-WB–infected <em>rad23</em> mutants and GFP(-SAP54) lines yield data in support of a hypothesis that leaf-like vegetative flowers are attractive to the phytoplasma leafhopper vector. Our study further demonstrates that SAP54 promotes the colonization of leafhoppers in a RAD23-dependent manner, however it is likely that the RAD23 proteins only play a supporting role in modulating leafhopper behaviour, as leafhoppers do not appear to discriminate between healthy <em>rad23BD</em> and <em>rad23BCD</em> mutants. On the other hand, experiments performed with Arabidopsis <em>ap1</em> mutants suggest that the mere presence of green flowers may not be sufficient to promote leafhopper colonization. Whereas flowers of <em>ap1</em> mutants to some extent resemble flowers of AY-WB–infected and SAP54-expressing plants (an observation that is consistent with AP1 destabilization via SAP54), our data strongly indicate that SAP54 targets multiple MTFs for destabilization, including SEP3 and its paralogues. It is thus conceivable that the loss of these MTFs in addition to AP1 is necessary to strongly promote leafhopper colonization. An intriguing (and not mutually exclusive) possibility is that SAP54-mediated promotion of leafhopper colonization extends beyond the recruitment of leaf-like flowers as a means of attracting insects. Whereas leafhoppers may be visually attracted to the bushy, leaf-like appearance exhibited by <em>35S:GFP–SAP54</em> plants (<a href="#pbio.1001835.s012">Figure S12</a>), we note that AY-WB–infected <em>rad23BD</em> mutants do not appear bushier than infected <em>rad23BCD</em> mutants, suggesting that “bushiness” is also unlikely to be the sole factor for the observed leafhopper colonization preference. Notwithstanding the above, our study demonstrates convincingly that SAP54 degrades MTFs in a RAD23-dependent manner, resulting in the production of leaf-like flowers and that SAP54 also promotes attractiveness of plants for leafhopper colonization in RAD23-dependent manner. Thus, AY-WB employs at least two protein effectors (SAP54 and SAP11 <a href="#pbio.1001835-Sugio2" class="ref-tip">[8]</a>) that make the plant more suitable for colonization by the insect vector. This is important, because phytoplasmas depend on leafhoppers for transmission and more leafhopper vector progeny will likely result in higher phytoplasma transmission and dispersal rates. Phytoplasma-infected plants are often sterile, as leaf-like flowers do not yield seed, and thus exposed plants become converted into hosts that only serve to help the phytoplasmas reproduce and propagate (zombie plants).</p>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="Yeast Two-Hybrid Analyses"></a>
<h3>Yeast Two-Hybrid Analyses</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>Hybrigenics Services SAS (Paris, France) performed the initial yeast two-hybrid screen, using SAP54 (amino acids 34–124; lacking the signal peptide) cloned into pB27 bait plasmid, as a C-terminal fusion to LexA (N-LexA-SAP54-C). Preliminary testing revealed that SAP54 was not toxic to yeast and did not autoactivate the system. Two screens were performed against a random-primed <em>Arabidopsis thaliana</em> seedlings cDNA library constructed into pP6 prey plasmid. A total of 71.7 million clones (7-fold library coverage) were screened following a mating approach with Y187 (matα) and L40 Gal4 (mata) yeast strains as previously described <a href="#pbio.1001835-FromontRacine1" class="ref-tip">[43]</a>. Of the proteins identified in the Hybrigenics two-hybrid screen and listed in <a href="#pbio.1001835.s013">Table S1</a>, we independently confirmed the interaction of each protein with SAP54 using yeast strain MaV203 (Invitrogen) with plasmids pDEST32 (GAL4–DNA-binding domain) and pDEST22 (GAL4-activation domain) as follows. MaV203 was transformed according to <a href="#pbio.1001835-Gietz1" class="ref-tip">[44]</a>, and transformants were selected and maintained by growth on minimal medium lacking leucine (to select for pDEST32) and tryptophan (to select for pDEST22). To examine protein–protein interactions, freshly transformed yeast colonies were resuspended in 1 mL sterile deionized water, and 10 µL aliquots were spotted upon medium lacking leucine and tryptophan (−L/−W) and medium lacking leucine, tryptophan, histidine, supplemented with 60 mM 3-Amino-1,2,4-triazole (3-AT; Sigma Aldrich) (−L/−W/−H). Growth was scored after 5 to 7 d of incubation at 28°C.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2" class="link-target"></a><p>For the comprehensive MTF yeast two-hybrid assay, a matrix-based approach was followed as described previously <a href="#pbio.1001835-deFolter2" class="ref-tip">[45]</a>. The originally described GAL4-AD and GAL4-BD MTF collection <a href="#pbio.1001835-deFolter1" class="ref-tip">[11]</a> was extended with a number of known MTF splicing variants <a href="#pbio.1001835-Severing1" class="ref-tip">[46]</a>,<a href="#pbio.1001835-vanDijk1" class="ref-tip">[47]</a>, making a total of 106 MTF proteins expressed from the pDEST22 and pDEST32 vector. The above-described pDEST32–SAP54 and a pDEST22–SAP54 construct were used as bait, respectively, in the pair-wise screening. Growth of yeast, and hence interaction events, was scored after 5 d of incubation at 20°C on synthetic dropout (SD) medium lacking leucine, tryptophan, histidine, supplemented with 1 mM 3-AT (−L/−W/−H). All identified positives were rescreened in a second experiment, in which the yeast was spotted onto selective medium lacking leucine, tryptophan, and adenine (−L/−W/−A).</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3" class="link-target"></a><p>For Western blot analysis, we followed a protocol established by Kushnirov <a href="#pbio.1001835-Kushnirov1" class="ref-tip">[48]</a>. Yeast strains were grown in liquid growth medium lacking leucine and tryptophan at 28°C overnight, and 2.5 OD<sub>600</sub> of yeast cells were harvested for each experiment.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="Co-Expression and Co-Immunoprecipitation Assays"></a>
<h3>Co-Expression and Co-Immunoprecipitation Assays</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p><em>SAP54</em> was amplified using a forward primer that encodes a Flag-tag (<a href="#pbio.1001835.s022">Table S10</a>), thus enabling the expression of SAP54 (amino acids 34–122) with an N-terminal Flag-tag in place of its signal peptide. The PCR product was cloned into pTRBO <a href="#pbio.1001835-Lindbo1" class="ref-tip">[18]</a> via restriction enzyme sites AvrII and NotI using standard molecular techniques. The construction of Flag-tagged RFP has already been described <a href="#pbio.1001835-Sugio2" class="ref-tip">[8]</a>. Type I and Type II MTFs were initially cloned into pDONR207 using BP Clonase (Invitrogen), and the genes were then transferred into pGWB21 <a href="#pbio.1001835-Nakagawa1" class="ref-tip">[49]</a> via LR Clonase (Invitrogen) to generate 35S:10xmyc–MTF. Primers used to amplify these genes are included in <a href="#pbio.1001835.s022">Table S10</a>. All genes cloned into pDONR207 as a result of this study were sequenced prior to use to ensure the absence of mutations within the gene. Furthermore, we also sequenced the gene insert within the 10xmyc-tagged MTF plasmids following completion of our co-expression assays to ensure the validity of our plasmids.</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2" class="link-target"></a><p>Co-expression assays were performed using constructs transiently expressed in <em>Nicotiana benthamiana</em> leaves of 3- to 4-wk-old plants that were grown in a controlled growth room in 16/8-h light/dark at 22°C with 55% humidity. 35S:10xmyc-tagged MTFs and 35S:Flag-tagged RFP or SAP54 were expressed in parallel following agroinfiltration using a needless syringe, using a 1∶1 mixture of <em>Agrobacterium tumefaciens</em> (strain GV3101) cultures adjusted to a final OD<sub>600</sub> of 0.1–0.2 (RFP and SAP54) and OD<sub>600</sub> of 0.5 (MTFs). For each co-expression assay, one leaf per plant (two plants per assay) was agroinfiltrated, with the Flag-RFP + Myc-MTF mixture infiltrated on the left side and Flag-SAP54 + Myc-MTF mixture infiltrated on the right side of the same leaf. After 3 d, one leaf disk (11 mm diameter) per infiltrated area was harvested and these were frozen immediately in liquid nitrogen. For Western blot analysis, two frozen leaf disks per sample were ground using a mortar and pestle in liquid nitrogen. We added 30 µL of 4× NuPage LDS sample buffer (Invitrogen) to the powdered tissue, and samples were boiled for 10 min. In a typical co-expression assay, 3 µL (anti-Flag blot), 5 µL (anti-SAP54 blot), and 15 µL (anti-Myc blot) aliquots were loaded into the SDS-PAGE gel.</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3" class="link-target"></a><p>Epoxomicin-treated samples were agroinfiltrated as described above, however 50 µM epoxomicin (Merck Chemicals LTD) was infiltrated using a needless syringe into the relevant area 3 d after agroinfiltration. The epoxomicin was prepared immediately prior to use by adding 2.78 µL of a 18 mM stock (dissolved in 100% DMSO) into 1 mL of sterile water. An equal volume of 100% DMSO (Sigma Aldrich) was added to sterile water to comprise the DMSO-only control, which was infiltrated alongside the epoxomicin treatment. Leaf disks from epoxomicin-treated (and DMSO-treated) samples were harvested after 8 h, and leaf disks were frozen in liquid nitrogen.</p>
<a id="article1.body1.sec4.sec2.p4" name="article1.body1.sec4.sec2.p4" class="link-target"></a><p>For co-immunoprecipitation assays performed using protein transiently expressed in <em>N. benthamiana</em> leaves, agroinfiltration was performed as describe above, however two entire leaves were infiltrated (per construct) to provide sufficient material (typically yielding about 1–2 grams of tissue). Leaf disks were removed from each leaf prior to freezing in liquid nitrogen, and Western blots were performed using these disks to confirm adequate protein expression levels prior to co-immunoprecipitation experiments. Following this verification, the remaining sample was ground using a mortar and pestle in liquid nitrogen, and added to cold extraction buffer (150 mM Tris-HCl, pH 7.5, 150 mM NaCl, 10% glycerol, 10 mM EDTA, 20 mM sodium fluoride, 10 mM DTT, 0.5% (wt/v) polyvinylpolypyrrolidone, 0.1% Triton-X, protease cocktail inhibitor (Sigma Chemical)) on ice. Samples were centrifuged at 3,200× <em>g</em> at 4°C for 15 min, and the supernatant was filtered through a 0.45 µm filter (Sartorius Stedim UK Limited) using a needleless syringe. We added 2 mL filtered extract to 20 µL equilibrated GFP-binding affinity resin (GFP-Trap_M; Chromotek GMBH), and samples were incubated at 4°C overnight upon a rotating wheel. Samples were initially pelleted by centrifugation at 2,700× <em>g</em> for 2 minutes and pellets were washed with 1 mL TBS buffer (10 mM Tris-HCl, pH 7.5, 150 mM NaCl, 0.5 mM EDTA, 0.1% Tween-20). Subsequent wash steps were performed using a magnetic stand to pellet GFP-binding resin. Samples were washed a minimum of three times, and all steps were performed at 4°C using ice-cold buffer. Following the final wash, all buffer was carefully removed using a syringe fitted with a 27G needle, and the resin was resuspended in 20 to 30 µL 4× NuPage LDS sample buffer (Invitrogen) and boiled for 10 min prior to loading on SDS/PAGE gels. In a typical co-immunoprecipitation experiment, 1 to 3 µL (anti-GFP) and 10 to 15 µL (anti-myc) aliquots were loaded onto an SDS-PAGE gel.</p>
<a id="article1.body1.sec4.sec2.p5" name="article1.body1.sec4.sec2.p5" class="link-target"></a><p>Co-immunoprecipitation assays performed using Arabidopsis transgenic lines were performed as described above with a few modifications. Transgenic lines expressing GFP-tagged AG, FUL, and SEP3 under control of their native promoters are described in <a href="#pbio.1001835-Urbanus1" class="ref-tip">[15]</a>,<a href="#pbio.1001835-deFolter3" class="ref-tip">[50]</a>. To minimize loss of MTFs due to SAP54-mediated destabilization, flowers were collected at an early stage of infection approximately 2–3 wk following exposure to noncarrier (for healthy flowers) and AY-WB–carrier (for infected flowers) <em>Macrosteles quadrilineatus</em>. At this point, plants are only beginning to produce flowers and early flowers appear normal or exhibit a mild degree of phyllody. Flowers produced at a later stage of infection (4 wk following inoculation) exhibit a strong degree of phyllody and loss of determinacy (samples of these flowers were harvested for Western blot analysis in <a href="#pbio-1001835-g002">Figure 2A</a>). For mass spectrometry, samples were collected from stably (T1) transformed Arabidopsis lines expressing either <em>35S:GFP–SAP54</em> or <em>35S:GFP</em> as a control. Primary and secondary inflorescences were harvested comprising all stages of developing floral buds from plants grown in a long day photoperiod (16/8-h light/dark). We used 0.85 to 0.90 g of plant tissues per pull-down experiment, and immunoprecipitation was performed using 50 µL equilibrated GFP-binding affinity resin. The GFP-binding resin was resuspended in 45 µL 4× NuPage LDS sample buffer prior to boiling. Western blots (anti-GFP) were performed to confirm the successful immunoprecipitation of GFP–SAP54 (or GFP) using 2 µL aliquots of each sample prior to further analysis via mass spectrometry. The remaining sample (approximately 40 µL) was resolved upon a 1.5 mm NuPAGE 4–12% Bis-Tris gel (Invitrogen) using a MOPS SDS running buffer. Proteins were visualized using SimplyBlue SafeStain (Invitrogen) and protein bands were cut out and collected using a new razor blade. In areas of the lane with no visible protein, 10 mm×10 mm gel slices were collected. All gel slices were destained in 30% ethanol (3×30 min washes at 65°C) prior to mass spectrometry analysis.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="Western Blots"></a>
<h3>Western Blots</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Proteins were separated on 12.5% (wt/v) SDS-PAGE gels and transferred to 0.45 µm Protran BA85 nitrocellulose membranes (Whatman) using the BioRad minigel and blotting systems following standard protocols. Blotted membranes were incubated in blocking buffer (5% (wt/v) milk powder in 1× phosphate buffered saline and 0.1% (v/v) Tween-20) with primary antibody at 4°C overnight. Peroxidase-conjugated anti-rabbit or anti-mouse secondary antibody (Sigma Aldrich) was added to washed blots and incubated at room temperature for 4 h. Bound antibodies were detected using Immobilon Western Chemiluminescent HRP Substrate (Millipore). Protein loading was visualized using Ponceau S solution (0.1% (wt/v) in 5% acetic acid; Sigma Aldrich). Signal intensity was quantified by generating density histograms for each protein band and then determining the area within the corresponding peak using ImageJ.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2" class="link-target"></a><p>Anti-SAP54 antibodies were raised in rabbits (Genscript) injected with purified 6×His-tagged SAP54. A 1∶2,000 dilution of anti-SAP54 was used in Western blots, and is sufficient to detect &lt;10 ng purified SAP54. Monoclonal anti-Myc, anti-GFP, anti-Flag, and anti-GAL4AD antibodies (all from Sigma Aldrich) were used at a 1∶10,000 dilution. Anti-RAD23 antibodies (used at a 1∶10,000 dilution) were provided by R. D. Vierstra and were raised in rabbit <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a>. Anti-SEP3 antibodies (used at a 1∶1,000 dilution) were provided by Cezary Smaczniak and were raised in rabbit.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="Generation of AY-WB–Infected Plants"></a>
<h3>Generation of AY-WB–Infected Plants</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>Healthy and AY-WB–carrier <em>Macrosteles quadrilineatus</em> Forbes (Hemiptera: Cicadellidae) stocks were maintained as previously described <a href="#pbio.1001835-Sugio2" class="ref-tip">[8]</a>. To generate infected Arabidopsis, three male AY-WB–carrier leafhoppers were added per plant to 4-wk-old plants that were previously grown in a short day photoperiod (10/14-h light/dark). Insects and plants were left for 4 to 5 d to allow inoculation of the phytoplasma in a controlled growth chamber set to a long day photoperiod to stimulate flowering (16/8-h light/dark). Insects were then removed and plants were returned to the growth chamber. Leaf-like flowers are typically produced 3 wk following initial exposure to the infected leafhoppers. Healthy controls were included in each experiment and comprised plants that were exposed to an equivalent number of noncarrier <em>Macrosteles quadrilineatus</em> leafhoppers as infected plants, and insects were removed on the same day as infected plants. The infection status of plants was determined via PCR analysis of DNA extracted from rosette leaves using AY-WB–specific primers BR and BF (<a href="#pbio.1001835.s022">Table S10</a>) <a href="#pbio.1001835-Zhang2" class="ref-tip">[51]</a>.</p>
</div>

<div id="section5" class="section toc-section"><a id="s4e" name="s4e" class="link-target" title="Insect Choice Assays"></a>
<h3>Insect Choice Assays</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p><em>rad23BD</em> and <em>rad23BCD</em> lines are described in <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a>, whereas seed for the <em>ap1</em> mutant was obtained from NASC (ID: N6232, allele <em>ap1-12</em>). Plants were sown on insecticide-free F2 compost soil (Levington) and grown at 22°C in a growth chamber adjusted to a short day photoperiod (8/16-h light/dark). Five-week-old plants were infected with AY-WB by adding five male AY-WB–carrier <em>Macrosteles quadrilineatus</em> leafhoppers to each plant in a transparent perspex tube (10 cm high, diameter 4 cm) for 5 d. Test plants were transplanted in 10 cm×10 cm square pots (F2 soil) and grown for an additional 2 wk at 22°C in a growth chamber adjusted to a long day photoperiod (16/8-h light/dark) to stimulate flowering. Prior to choice experiments, three rosette leaves were collected for extraction of genomic DNA to confirm the genotype of all plants (using primers as previously described <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a>) and AY-WB infection status (using AY-WB–specific primers BF and BR in <a href="#pbio.1001835.s022">Table S10</a>).</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2" class="link-target"></a><p>All insect choice experiments were performed in transparent polycarbonate cages 620 mm×300 mm×410 mm (height×width×length). Two opposite sides of the cage were fitted with white nylon mesh held in place by magnetic strips to enclose the cage. Two test plants (21 dai with AY-WB) were randomly placed diagonally opposite each other in the corners of a cage (<a href="#pbio.1001835.s007">Figure S7</a>). Ten male and 10 female healthy adult <em>M. quadrilineatus</em> leafhoppers were released from a transparent perspex tube (9 cm high, diameter 3 cm) in the centre of the cage equidistant from each test plant. Adult insects were removed 5 d after addition to the cage. At that time, plants were removed from the choice cage and enclosed individually in transparent perforated plastic bags, and returned to the growth chamber. Nymphs were counted on each test plant 14 d after removal of adult insects from the cages. Data were expressed as proportion of total number of nymphs found on the test plants within each choice cage.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4f" name="s4f" class="link-target" title="Statistical Analysis"></a>
<h3>Statistical Analysis</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>Statistical analysis was performed in Minitab16. Insect oviposition choice data were analysed using paired <em>t</em> test. Assumptions of the statistical tests (normal distribution and equal variance) were checked with the Anderson-Darling and the Levene's tests, respectively.</p>
</div>

<div id="section7" class="section toc-section"><a id="s4g" name="s4g" class="link-target" title="Generation of 35S:GFP–SAP54 and 35S:GFP Transgenic Arabidopsis Lines"></a>
<h3>Generation of <em>35S:GFP–SAP54</em> and <em>35S:GFP</em> Transgenic Arabidopsis Lines</h3>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1" class="link-target"></a><p>The gene encoding SAP54 (lacking the signal peptide; amino acids 34 to 124) was PCR-amplified using primers attB1SAP54 and attB2SAP54 (<a href="#pbio.1001835.s022">Table S10</a>) and cloned in pDONR207 (Invitrogen) using Clonase BP according to the manufacturer's instructions. For expression in Arabidopsis, <em>SAP54</em> was transferred into Gateway vector pB7WGF2 using Clonase LR as per the manufacturer's instructions. pB7WGF2 encodes an N-terminal GFP fragment under control of the CaMV 35S promoter <a href="#pbio.1001835-Karimi1" class="ref-tip">[52]</a>, thus generating <em>35S:GFP–SAP54</em>. For the construction of a <em>35S:GFP</em> transgenic Arabidopsis line, the gene encoding eGFP was amplified using pB7WGF2 as a template and primers attB1foreGFP and attB2reveGFP (<a href="#pbio.1001835.s022">Table S10</a>). <em>eGFP</em> was then cloned into pDONR207 using Clonase BP, and the gene was transferred into pB7WG2 <a href="#pbio.1001835-Karimi1" class="ref-tip">[52]</a> to create <em>35S:GFP</em>. Arabidopsis plants were transformed via floral dip <a href="#pbio.1001835-Clough1" class="ref-tip">[53]</a> with <em>Agrobacterium tumefaciens</em> strain GV3101. Seedlings of transformed plants were selected by the herbicide glufosinate (BASTA). Transgenic plants expressing GFP–SAP54 are sterile (with the exception of transformants obtained in <em>rad23CD</em> and <em>rad23BCD</em> backgrounds); thus, experiments and phenotypic analyses were performed upon T1 lines. Prior to the assessment of GFP–SAP54–induced phenotypes in <em>rad23</em> T-DNA mutant lines, a minimum of 10 randomly selected plants from each transformation group were examined via PCR analysis, using primers specific to each of the four <em>RAD23</em> genes as described in <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a> to confirm the genotype of the plants. <em>rad23</em> single mutants are the result of T-DNA insertions (SALK lines 066603, 075940, 068091, and 014137) and higher order mutants were kindly provided by Richard Vierstra and are described in <a href="#pbio.1001835-Farmer1" class="ref-tip">[21]</a>.</p>
</div>

<div id="section8" class="section toc-section"><a id="s4h" name="s4h" class="link-target" title="Mass Spectrometry Analysis"></a>
<h3>Mass Spectrometry Analysis</h3>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1" class="link-target"></a><p>Gel slices cut from the SDS-PAGE gel were washed, reduced and alkylated, and treated with trypsin according to standard procedures <a href="#pbio.1001835-Shevchenko1" class="ref-tip">[54]</a>. Peptides were extracted with 5% formic acid/50% acetonitrile, dried down, and re-dissolved in 0.1% TFA. For LC-MS/MS analysis, a sample aliquot was applied via a nanoAcquityTM (Waters, Manchester, UK) UPLCTM-system running at a flow rate of 250 nL min-1 to an LTQ-Orbitrap mass spectrometer (Thermo Fisher, Waltham, MA). Peptides were trapped using a pre-column (Symmetry C18, 5 µm, 180 µm×20 mm, Waters) that was then switched in-line to an analytical column (BEH C18, 1.7 µm, 75 µm×250 mm, Waters) for separation. Peptides were eluted with a gradient of 3–38% acetonitrile in water/0.1% formic acid at a rate of 0.67% min-1. The column was connected to a 10 µm SilicaTip nanospray emitter (New Objective, Woburn, MA, USA) attached to a nanospray interface (Proxeon, Odense, Denmark) for infusion into the mass spectrometer. The mass spectrometer was operated in positive ion mode at a capillary temperature of 200°C. The source voltage and focusing voltages were tuned for the transmission of MRFA peptide (m/z 524) (Sigma Aldrich, St. Louis, MO). Data-dependent analysis was carried out in oribtrap-IT parallel mode using CID fragmentation on the five most abundant ions in each cycle. The orbitrap was run with a resolution of 30,000 over the MS range from m/z 350 to m/z 1800 and an MS target of 106 and 1 s maximum scan time. Collision energy was 35, and an isolation width of 2 was used. Only mono-isotopic 2+ and 3+ charged precursors were selected for MS2. The MS2 was triggered by a minimal signal of 1,000 with an AGC target of 3×104 ions and 150 ms scan time using the chromatography function for peak apex detection. Dynamic exclusion was set to 1 count and 30 s exclusion with an exclusion mass window of ±20 ppm. MS scans were saved in profile mode, whereas MSMS scans were saved in centroid mode.</p>
<a id="article1.body1.sec4.sec8.p2" name="article1.body1.sec4.sec8.p2" class="link-target"></a><p>Raw files were processed with MaxQuant version 1.3.0.5 (<a href="#pbio.1001835-Cox1" class="ref-tip">[55]</a>; <a href="http://maxquant.org">http://maxquant.org</a>) to generate re-calibrated peaklist-files which were used for a database search using an in-house Mascot 2.4 Server (Matrix Science Limited, London, UK). Mascot-mgf files were generated from MaxQuant apl-files using a suitable perl script. Mascot searches were performed on the TAIR_10_pep_20101214.fasta database (<a href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>) using trypsin/P with 2 missed cleavages, 6 ppm precursor tolerance, 0.6 Da fragment tolerance, carbamidomethylation (C) as fixed, and oxidation (M) and acetylation (protein N-terminus) as variable modifications. Mascot search results were imported and evaluated in Scaffold 4.0.4 (proteomsoftware.com, Portland, OR, USA) resulting in a protein false discovery rate of 0.9%.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pbio.1001835"></div><div class="supplementary-material"><a name="pbio.1001835.s001" id="pbio.1001835.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Analysis of SAP54 interactions with MADS-domain proteins.</strong> (A) Type I MTFs AGL50, AGL62, and AGL80 are partially destabilized when transiently co-expressed in the presence of the phytoplasma effector SAP54. (B) Noncanonical Type II MIKC* protein AGL66 is stable in the presence of SAP54. (C) SAP54-mediated destabilization of Type II MIKC<sup>C</sup> protein AP1 is inhibited following treatment with 50 µM epoxomicin, whereas AP1 is destabilized in samples treated with a protease inhibitor cocktail. (D) AP1, SEP3, and SOC1 are not destabilized by SAP54 in yeast.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s001">https://doi.org/10.1371/journal.pbio.1001835.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s002" id="pbio.1001835.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s002">Figure S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Phenotype of various </strong><strong><em>rad23</em></strong><strong> mutants.</strong> (A) Four-week-old Arabidopsis wild-type (Col-0) and <em>rad23</em> mutant lines. Note the reduced stature of <em>rad23BC</em>, <em>rad23BD</em>, <em>rad23ABD</em>, and <em>rad23BCD</em> plants. Scale bar, 5 cm. (B) <em>rad23</em> mutants produce wild-type flowers, with the exception of the <em>rad23BCD</em> triple mutant that frequently produces flowers with five or six petals (lateral and frontal view as shown).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s002">https://doi.org/10.1371/journal.pbio.1001835.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s003" id="pbio.1001835.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s003">Figure S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Scoring of phenotypes exhibited by </strong><strong><em>35S:GFP–SAP54</em></strong><strong> transgenic lines.</strong> Plants were scored as follows, with representative flowers depicted. Absent, flowers are indistinguishable from wild-type based upon visual examination. Mild, enlarged sepals, mild to moderate virescence of petals, stamens produce pollen, and occasional loss of determinacy observed in early arising flowers. Strong, leaf-like sepals, strong virescence of petals, stamens are virescent and do not produce pollen, frequent loss of determinacy throughout the plant.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s003">https://doi.org/10.1371/journal.pbio.1001835.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s004" id="pbio.1001835.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s004">Figure S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Characterization of transgenic Arabidopsis lines expressing </strong><strong><em>35S:GFP–SAP54</em></strong><strong>.</strong> (A) Expression of <em>GFP–SAP54</em> induces phyllody and loss of determinacy in wild-type Arabidopsis Col-0, but the majority of transformants obtained in <em>rad23CD</em> double mutants produce normal flowers. <em>35S:GFP–SAP54 rad23CD</em> line 8 represents a minority of transgenic lines exhibiting a mild degree of virescence and loss of determinacy. (B) Western blot analysis reveals protein levels of GFP–SAP54 (indicated by an arrow) in transgenic plants picture in panel A.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s004">https://doi.org/10.1371/journal.pbio.1001835.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s005" id="pbio.1001835.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s005">Figure S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>AY-WB phytoplasma induces witches' broom but not phyllody in infected </strong><strong><em>rad23BCD</em></strong><strong> triple mutants.</strong> (A) An image of a healthy (wild-type) Arabidopsis plant. (B) Wild-type (Col-0) and <em>rad23</em> triple mutants following infection with AY-WB phytoplasma. Note the occurrence of witches' broom (increased proliferation of stems) in all plants. (C) Wild-type (Col-0) and <em>rad23ABD</em> produce leaf-like flowers when infected with AY-WB, whereas the <em>rad23BCD</em> mutant produces normal flowers.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s005">https://doi.org/10.1371/journal.pbio.1001835.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s006" id="pbio.1001835.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s006">Figure S6. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Arabidopsis </strong><strong><em>rad23</em></strong><strong> mutants produce leaf-like flowers following infection with phytoplasma AY-WB.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s006">https://doi.org/10.1371/journal.pbio.1001835.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s007" id="pbio.1001835.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s007">Figure S7. </a></h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Experimental set-up for insect oviposition choice experiments.</strong> Photograph (top) illustrates the actual arrangement of the test plants (A, D) in a choice cage. Several other choice cages are visible in the background with alternative positioning of the test plants. Diagramme (bottom) depicts all available positions for the test plants in the cage (A, B, C, D). Only two positions are occupied in any given cage, resulting from randomly placing the test plants in two out of the four available corners. Insects are introduced in the center of the cage (equidistant from both plants) and released from a transparent plastic tube (E). Arrows indicate the physical dimensions of the cage.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s007">https://doi.org/10.1371/journal.pbio.1001835.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s008" id="pbio.1001835.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s008">Figure S8. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Arabidopsis </strong><strong><em>ap1-12</em></strong><strong> mutants produce green leaf-like flowers that lack petals.</strong> (A) Images of flowers from healthy and AY-WB–infected Arabidopsis wild-type (Col-0) are compared to a GFP–SAP54–expressing transgenic line and <em>ap1-12</em> mutant. (B) Images of plants representative of healthy and AY-WB–infected Arabidopsis, GFP–SAP54–expressing transgenic lines, and <em>ap1-12</em> mutants. Scale bars, 5 cm. Se, sepal; Pe, petal; St, stamen; Ca, carpel.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s008">https://doi.org/10.1371/journal.pbio.1001835.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s009" id="pbio.1001835.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s009">Figure S9. </a></h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Aster leafhopper </strong><strong><em>Macrosteles quadrilineatus</em></strong><strong> produces a similar number of nymphs on wild-type (wt) and </strong><strong><em>ap1</em></strong><strong> mutant Arabidopsis plants (</strong><strong><em>t</em></strong><strong><sub>(11)</sub> = 0.22; </strong><strong><em>p</em></strong><strong> = 0.835).</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s009">https://doi.org/10.1371/journal.pbio.1001835.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s010" id="pbio.1001835.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s010">Figures S10. </a></h3><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Alignment of amino acid sequences of SAP54 homologues identified in other phytoplasma strains.</strong> M-AY, Maryland aster yellows phytoplasma (ABH11652); Spiraea, Spiraea stunt phytoplasma (ABU55747); OY-M, Onion yellows phytoplasma strain OY-M (PAM_049); PnWB, Peanut Witches' Broom phytoplasma (ZP_23918844).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s010">https://doi.org/10.1371/journal.pbio.1001835.s010</a></p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s011" id="pbio.1001835.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s011">Figure S11. </a></h3><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Models of SAP54-mediated degradation of MTFs.</strong> (A) SAP54 binds directly to both MTFs and RAD23. The latter takes the SAP54–MTF complex to the plant UPS where the MTFs are degraded. SAP54 may remain associated with RAD23 to prevent being degraded. (B) RAD23 and SAP54 do not interact directly, but via one or more ubiquitin moieties linked via lysine (K) residue(s) on SAP54. RAD23 takes the SAP54–MTF complex to the plant UPS (as in A). (C) An unknown pathway is involved in transportation of SAP54–MTF complexes to the host UPS, whereupon SAP54 interacts with RAD23 to evade degradation. RAD23 and SAP54 may interact directly (as in A) or via ubiquitin (as in B).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s011">https://doi.org/10.1371/journal.pbio.1001835.s011</a></p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s012" id="pbio.1001835.s012" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s012">Figure S12. </a></h3><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Phytoplasma effector SAP54 alters host development to promote vegetative growth.</strong> Shown are 7-wk-old (A) and 10-wk-old (B) transgenic Arabidopsis lines expressing <em>35S:GFP</em> (control) and <em>35S:GFP–SAP54</em>. Scale bars, 5 cm.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s012">https://doi.org/10.1371/journal.pbio.1001835.s012</a></p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s013" id="pbio.1001835.s013" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s013">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Clones identified as SAP54 interactors in Hybrigenics screen.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s013">https://doi.org/10.1371/journal.pbio.1001835.s013</a></p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s014" id="pbio.1001835.s014" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s014">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Yeast two-hybrid analysis of SAP54 interactions with MTFs.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s014">https://doi.org/10.1371/journal.pbio.1001835.s014</a></p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s015" id="pbio.1001835.s015" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s015">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Quantification of signal intensity levels (ImageJ) of bands in </strong><a href="#pbio-1001835-g001"><strong>Figure 1D</strong></a><strong>.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s015">https://doi.org/10.1371/journal.pbio.1001835.s015</a></p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s016" id="pbio.1001835.s016" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s016">Table S4. </a></h3><a id="article1.body1.sec5.supplementary-material16.caption1.p1" name="article1.body1.sec5.supplementary-material16.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Mass spectrometry analysis of MTFs that interact with GFP–SAP54.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s016">https://doi.org/10.1371/journal.pbio.1001835.s016</a></p><a id="article1.body1.sec5.supplementary-material16.caption1.p2" name="article1.body1.sec5.supplementary-material16.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s017" id="pbio.1001835.s017" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s017">Table S5. </a></h3><a id="article1.body1.sec5.supplementary-material17.caption1.p1" name="article1.body1.sec5.supplementary-material17.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Signal intensity levels (ImageJ) of bands in </strong><a href="#pbio-1001835-g002"><strong>Figure 2A</strong></a><strong>.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s017">https://doi.org/10.1371/journal.pbio.1001835.s017</a></p><a id="article1.body1.sec5.supplementary-material17.caption1.p2" name="article1.body1.sec5.supplementary-material17.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s018" id="pbio.1001835.s018" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s018">Table S6. </a></h3><a id="article1.body1.sec5.supplementary-material18.caption1.p1" name="article1.body1.sec5.supplementary-material18.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Signal intensity levels (ImageJ) of bands in </strong><a href="#pbio-1001835-g002"><strong>Figure 2B</strong></a><strong>.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s018">https://doi.org/10.1371/journal.pbio.1001835.s018</a></p><a id="article1.body1.sec5.supplementary-material18.caption1.p2" name="article1.body1.sec5.supplementary-material18.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s019" id="pbio.1001835.s019" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s019">Table S7. </a></h3><a id="article1.body1.sec5.supplementary-material19.caption1.p1" name="article1.body1.sec5.supplementary-material19.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Signal intensity levels (ImageJ) of bands in </strong><a href="#pbio-1001835-g002"><strong>Figure 2C</strong></a><strong>.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s019">https://doi.org/10.1371/journal.pbio.1001835.s019</a></p><a id="article1.body1.sec5.supplementary-material19.caption1.p2" name="article1.body1.sec5.supplementary-material19.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s020" id="pbio.1001835.s020" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s020">Table S8. </a></h3><a id="article1.body1.sec5.supplementary-material20.caption1.p1" name="article1.body1.sec5.supplementary-material20.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Signal intensity levels (ImageJ) of IP bands in </strong><a href="#pbio-1001835-g002"><strong>Figure 2D</strong></a><strong>.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s020">https://doi.org/10.1371/journal.pbio.1001835.s020</a></p><a id="article1.body1.sec5.supplementary-material20.caption1.p2" name="article1.body1.sec5.supplementary-material20.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s021" id="pbio.1001835.s021" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s021">Table S9. </a></h3><a id="article1.body1.sec5.supplementary-material21.caption1.p1" name="article1.body1.sec5.supplementary-material21.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Signal intensity levels (ImageJ) of input bands in </strong><a href="#pbio-1001835-g002"><strong>Figure 2D</strong></a><strong>.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s021">https://doi.org/10.1371/journal.pbio.1001835.s021</a></p><a id="article1.body1.sec5.supplementary-material21.caption1.p2" name="article1.body1.sec5.supplementary-material21.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pbio.1001835.s022" id="pbio.1001835.s022" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pbio.1001835.s022">Table S10. </a></h3><a id="article1.body1.sec5.supplementary-material22.caption1.p1" name="article1.body1.sec5.supplementary-material22.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>List of primers used in study.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pbio.1001835.s022">https://doi.org/10.1371/journal.pbio.1001835.s022</a></p><a id="article1.body1.sec5.supplementary-material22.caption1.p2" name="article1.body1.sec5.supplementary-material22.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We are grateful to Professor Richard D. Vierstra (Department of Genetics, University of Wisconsin–Madison) for providing RAD23 antibody and seed for the <em>rad23</em> mutants. Antibodies against SEP3 were kindly provided by Cezary Smaczniak (Laboratory of Molecular Biology, Wageningen University) and Robert Sablowski (Cell &amp; Developmental Biology, JIC). We wish to thank Dr. Chan-Pin Lin (Department of Plant Pathology and Microbiology, National Taiwan University) and Chih-Horng Kuo (Institute of Plant and Microbial Biology, Academia Sinica) for providing the sequence of the Peanut Witches' Broom SAP54 homologue prior to publication of the genome sequence. We gratefully acknowledge Gerhard Saalbach (JIC Proteomics Facility) for performing mass spectrometry analyses, Sam Mugford and Jacqueline Busscher-Lange for technical support, Andrew Davis for providing photographic services, and Jitender Cheema for the construction of the MADS-box protein phylogeny. We are grateful to Mark Banfield (Biological Chemistry, JIC) for help with the purification of SAP54 for antibody production and to Ian Bedford, Anna Jordan, and Gavin Hatt (JIC Insectary) for rearing and care of leafhopper and phytoplasma stocks and the John Innes Horticultural Services for growing the plants used in this study. The mass spectrometry proteomics data have been deposited in the ProteomeXchange Consortium (<a href="http://proteomecentral.proteomexchange.org">http://proteomecentral.proteomexchange.org</a>) via the PRIDE partner repository (26) with the dataset identifier PXD000223 and DOI 10.6019/PXD000223.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>The author(s) have made the following declarations about their contributions: Conceived and designed the experiments: AMM GCA RGHI SAH. Performed the experiments: AMM ZO KK AMZ. Analyzed the data: AMM ZO KK AMZ GCA RGHI SAH. Contributed reagents/materials/analysis tools: AMM GCA RGHI SAH. Wrote the paper: AMM SAH. Edited the manuscript: AMM ZO KK AMZ GCA RGHI SAH.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pbio.1001835-Dean1" id="pbio.1001835-Dean1" class="link-target"></a>Dean P (2011) Functional domains and motifs of bacterial type III effector proteins and their roles in infection. FEMS Microbiol Rev 35: 1100–1125. <ul class="reflinks"><li><a href="#" data-author="Dean" data-cit="DeanP%20%282011%29%20Functional%20domains%20and%20motifs%20of%20bacterial%20type%20III%20effector%20proteins%20and%20their%20roles%20in%20infection.%20FEMS%20Microbiol%20Rev%2035%3A%201100%E2%80%931125." data-title="Functional%20domains%20and%20motifs%20of%20bacterial%20type%20III%20effector%20proteins%20and%20their%20roles%20in%20infection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Functional+domains+and+motifs+of+bacterial+type+III+effector+proteins+and+their+roles+in+infection+Dean+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pbio.1001835-Bozkurt1" id="pbio.1001835-Bozkurt1" class="link-target"></a>Bozkurt TO, Schornack S, Banfield MJ, Kamoun S (2012) Oomycetes, effectors, and all that jazz. Curr Opin Plant Biol 15: 483–492. <ul class="reflinks"><li><a href="#" data-author="Bozkurt" data-cit="BozkurtTO%2C%20SchornackS%2C%20BanfieldMJ%2C%20KamounS%20%282012%29%20Oomycetes%2C%20effectors%2C%20and%20all%20that%20jazz.%20Curr%20Opin%20Plant%20Biol%2015%3A%20483%E2%80%93492." data-title="Oomycetes%2C%20effectors%2C%20and%20all%20that%20jazz" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Oomycetes%2C+effectors%2C+and+all+that+jazz+Bozkurt+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pbio.1001835-Romer1" id="pbio.1001835-Romer1" class="link-target"></a>Romer P, Recht S, Strauss T, Elsaesser J, Schornack S, et al. (2010) Promoter elements of rice susceptibility genes are bound and activated by specific TAL effectors from the bacterial blight pathogen, Xanthomonas oryzae pv. oryzae. New Phytol 187: 1048–1057. <ul class="reflinks"><li><a href="#" data-author="Romer" data-cit="RomerP%2C%20RechtS%2C%20StraussT%2C%20ElsaesserJ%2C%20SchornackS%2C%20et%20al.%20%282010%29%20Promoter%20elements%20of%20rice%20susceptibility%20genes%20are%20bound%20and%20activated%20by%20specific%20TAL%20effectors%20from%20the%20bacterial%20blight%20pathogen%2C%20Xanthomonas%20oryzae%20pv.%20oryzae.%20New%20Phytol%20187%3A%201048%E2%80%931057." data-title="Promoter%20elements%20of%20rice%20susceptibility%20genes%20are%20bound%20and%20activated%20by%20specific%20TAL%20effectors%20from%20the%20bacterial%20blight%20pathogen%2C%20Xanthomonas%20oryzae%20pv.%20oryzae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Promoter+elements+of+rice+susceptibility+genes+are+bound+and+activated+by+specific+TAL+effectors+from+the+bacterial+blight+pathogen%2C+Xanthomonas+oryzae+pv.+oryzae+Romer+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pbio.1001835-Vyas1" id="pbio.1001835-Vyas1" class="link-target"></a>Vyas A, Kim SK, Giacomini N, Boothroyd JC, Sapolsky RM (2007) Behavioral changes induced by Toxoplasma infection of rodents are highly specific to aversion of cat odors. Proc Natl Acad Sci U S A 104: 6442–6447. <ul class="reflinks"><li><a href="#" data-author="Vyas" data-cit="VyasA%2C%20KimSK%2C%20GiacominiN%2C%20BoothroydJC%2C%20SapolskyRM%20%282007%29%20Behavioral%20changes%20induced%20by%20Toxoplasma%20infection%20of%20rodents%20are%20highly%20specific%20to%20aversion%20of%20cat%20odors.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20104%3A%206442%E2%80%936447." data-title="Behavioral%20changes%20induced%20by%20Toxoplasma%20infection%20of%20rodents%20are%20highly%20specific%20to%20aversion%20of%20cat%20odors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Behavioral+changes+induced+by+Toxoplasma+infection+of+rodents+are+highly+specific+to+aversion+of+cat+odors+Vyas+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pbio.1001835-Roy1" id="pbio.1001835-Roy1" class="link-target"></a>Roy BA (1993) Floral mimicry by a plant pathogen. Nature 362: 56–58. <ul class="reflinks"><li><a href="#" data-author="Roy" data-cit="RoyBA%20%281993%29%20Floral%20mimicry%20by%20a%20plant%20pathogen.%20Nature%20362%3A%2056%E2%80%9358." data-title="Floral%20mimicry%20by%20a%20plant%20pathogen" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Floral+mimicry+by+a+plant+pathogen+Roy+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pbio.1001835-Mayer1" id="pbio.1001835-Mayer1" class="link-target"></a>Mayer CJ, Vilcinskas A, Gross J (2008) Phytopathogen lures its insect vector by altering host plant odor. J Chem Ecol 34: 1045–1049. <ul class="reflinks"><li><a href="#" data-author="Mayer" data-cit="MayerCJ%2C%20VilcinskasA%2C%20GrossJ%20%282008%29%20Phytopathogen%20lures%20its%20insect%20vector%20by%20altering%20host%20plant%20odor.%20J%20Chem%20Ecol%2034%3A%201045%E2%80%931049." data-title="Phytopathogen%20lures%20its%20insect%20vector%20by%20altering%20host%20plant%20odor" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phytopathogen+lures+its+insect+vector+by+altering+host+plant+odor+Mayer+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pbio.1001835-Sugio1" id="pbio.1001835-Sugio1" class="link-target"></a>Sugio A, MacLean AM, Kingdom HN, Grieve VM, Manimekalai R, et al. (2011) Diverse targets of phytoplasma effectors: from plant development to defense against insects. Annu Rev Phytopathol 49: 175–195. <ul class="reflinks"><li><a href="#" data-author="Sugio" data-cit="SugioA%2C%20MacLeanAM%2C%20KingdomHN%2C%20GrieveVM%2C%20ManimekalaiR%2C%20et%20al.%20%282011%29%20Diverse%20targets%20of%20phytoplasma%20effectors%3A%20from%20plant%20development%20to%20defense%20against%20insects.%20Annu%20Rev%20Phytopathol%2049%3A%20175%E2%80%93195." data-title="Diverse%20targets%20of%20phytoplasma%20effectors%3A%20from%20plant%20development%20to%20defense%20against%20insects" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Diverse+targets+of+phytoplasma+effectors%3A+from+plant+development+to+defense+against+insects+Sugio+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pbio.1001835-Sugio2" id="pbio.1001835-Sugio2" class="link-target"></a>Sugio A, Kingdom HN, MacLean AM, Grieve VM, Hogenhout SA (2011) Phytoplasma protein effector SAP11 enhances insect vector reproduction by manipulating plant development and defense hormone biosynthesis. Proc Natl Acad Sci U S A 108: E1254–E1263. <ul class="reflinks"><li><a href="#" data-author="Sugio" data-cit="SugioA%2C%20KingdomHN%2C%20MacLeanAM%2C%20GrieveVM%2C%20HogenhoutSA%20%282011%29%20Phytoplasma%20protein%20effector%20SAP11%20enhances%20insect%20vector%20reproduction%20by%20manipulating%20plant%20development%20and%20defense%20hormone%20biosynthesis.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20108%3A%20E1254%E2%80%93E1263." data-title="Phytoplasma%20protein%20effector%20SAP11%20enhances%20insect%20vector%20reproduction%20by%20manipulating%20plant%20development%20and%20defense%20hormone%20biosynthesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phytoplasma+protein+effector+SAP11+enhances+insect+vector+reproduction+by+manipulating+plant+development+and+defense+hormone+biosynthesis+Sugio+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pbio.1001835-MacLean1" id="pbio.1001835-MacLean1" class="link-target"></a>MacLean AM, Sugio A, Makarova OV, Findlay KC, Grieve VM, et al. (2011) Phytoplasma effector SAP54 induces indeterminate leaf-like flower development in arabidopsis plants. Plant Physiol 157: 831–841. <ul class="reflinks"><li><a href="#" data-author="MacLean" data-cit="MacLeanAM%2C%20SugioA%2C%20MakarovaOV%2C%20FindlayKC%2C%20GrieveVM%2C%20et%20al.%20%282011%29%20Phytoplasma%20effector%20SAP54%20induces%20indeterminate%20leaf-like%20flower%20development%20in%20arabidopsis%20plants.%20Plant%20Physiol%20157%3A%20831%E2%80%93841." data-title="Phytoplasma%20effector%20SAP54%20induces%20indeterminate%20leaf-like%20flower%20development%20in%20arabidopsis%20plants" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phytoplasma+effector+SAP54+induces+indeterminate+leaf-like+flower+development+in+arabidopsis+plants+MacLean+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pbio.1001835-Kaufmann1" id="pbio.1001835-Kaufmann1" class="link-target"></a>Kaufmann K, Wellmer F, Muino JM, Ferrier T, Wuest SE, et al. (2010) Orchestration of floral initiation by APETALA1. Science 328: 85–89. <ul class="reflinks"><li><a href="#" data-author="Kaufmann" data-cit="KaufmannK%2C%20WellmerF%2C%20MuinoJM%2C%20FerrierT%2C%20WuestSE%2C%20et%20al.%20%282010%29%20Orchestration%20of%20floral%20initiation%20by%20APETALA1.%20Science%20328%3A%2085%E2%80%9389." data-title="Orchestration%20of%20floral%20initiation%20by%20APETALA1" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Orchestration+of+floral+initiation+by+APETALA1+Kaufmann+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pbio.1001835-deFolter1" id="pbio.1001835-deFolter1" class="link-target"></a>de Folter S, Immink RGH, Kieffer M, Parenicova L, Henz SR, et al. (2005) Comprehensive interaction map of the Arabidopsis MADS box transcription factors. Plant Cell 17: 1424–1433. <ul class="reflinks"><li><a href="#" data-author="de%20Folter" data-cit="de%20FolterS%2C%20ImminkRGH%2C%20KiefferM%2C%20ParenicovaL%2C%20HenzSR%2C%20et%20al.%20%282005%29%20Comprehensive%20interaction%20map%20of%20the%20Arabidopsis%20MADS%20box%20transcription%20factors.%20Plant%20Cell%2017%3A%201424%E2%80%931433." data-title="Comprehensive%20interaction%20map%20of%20the%20Arabidopsis%20MADS%20box%20transcription%20factors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Comprehensive+interaction+map+of+the+Arabidopsis+MADS+box+transcription+factors+de+Folter+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pbio.1001835-Yang1" id="pbio.1001835-Yang1" class="link-target"></a>Yang YZ, Jack T (2004) Defining subdomains of the K domain important for protein-protein interactions of plant MADS proteins. Plant Mol Biol 55: 45–59. <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangYZ%2C%20JackT%20%282004%29%20Defining%20subdomains%20of%20the%20K%20domain%20important%20for%20protein-protein%20interactions%20of%20plant%20MADS%20proteins.%20Plant%20Mol%20Biol%2055%3A%2045%E2%80%9359." data-title="Defining%20subdomains%20of%20the%20K%20domain%20important%20for%20protein-protein%20interactions%20of%20plant%20MADS%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Defining+subdomains+of+the+K+domain+important+for+protein-protein+interactions+of+plant+MADS+proteins+Yang+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pbio.1001835-Davies1" id="pbio.1001835-Davies1" class="link-target"></a>Davies B, EgeaCortines M, Silva ED, Saedler H, Sommer H (1996) Multiple interactions amongst floral homeotic MADS box proteins. Embo J 15: 4330–4343. <ul class="reflinks"><li><a href="#" data-author="Davies" data-cit="DaviesB%2C%20EgeaCortinesM%2C%20SilvaED%2C%20SaedlerH%2C%20SommerH%20%281996%29%20Multiple%20interactions%20amongst%20floral%20homeotic%20MADS%20box%20proteins.%20Embo%20J%2015%3A%204330%E2%80%934343." data-title="Multiple%20interactions%20amongst%20floral%20homeotic%20MADS%20box%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multiple+interactions+amongst+floral+homeotic+MADS+box+proteins+Davies+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pbio.1001835-AlvarezBuylla1" id="pbio.1001835-AlvarezBuylla1" class="link-target"></a>Alvarez-Buylla ER, Pelaz S, Liljegren SJ, Gold SE, Burgeff C, et al. (2000) An ancestral MADS-box gene duplication occurred before the divergence of plants and animals. Proc Natl Acad Sci U S A 97: 5328–5333. <ul class="reflinks"><li><a href="#" data-author="Alvarez-Buylla" data-cit="Alvarez-BuyllaER%2C%20PelazS%2C%20LiljegrenSJ%2C%20GoldSE%2C%20BurgeffC%2C%20et%20al.%20%282000%29%20An%20ancestral%20MADS-box%20gene%20duplication%20occurred%20before%20the%20divergence%20of%20plants%20and%20animals.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%2097%3A%205328%E2%80%935333." data-title="An%20ancestral%20MADS-box%20gene%20duplication%20occurred%20before%20the%20divergence%20of%20plants%20and%20animals" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+ancestral+MADS-box+gene+duplication+occurred+before+the+divergence+of+plants+and+animals+Alvarez-Buylla+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pbio.1001835-Urbanus1" id="pbio.1001835-Urbanus1" class="link-target"></a>Urbanus SL, de Folter S, Shchennikova AV, Kaufmann K, Immink RGH, et al. (2009) In planta localisation patterns of MADS domain proteins during floral development in Arabidopsis thaliana. BMC Plant Biol 9: 5. <ul class="reflinks"><li><a href="#" data-author="Urbanus" data-cit="UrbanusSL%2C%20de%20FolterS%2C%20ShchennikovaAV%2C%20KaufmannK%2C%20ImminkRGH%2C%20et%20al.%20%282009%29%20In%20planta%20localisation%20patterns%20of%20MADS%20domain%20proteins%20during%20floral%20development%20in%20Arabidopsis%20thaliana.%20BMC%20Plant%20Biol%209%3A%205." data-title="In%20planta%20localisation%20patterns%20of%20MADS%20domain%20proteins%20during%20floral%20development%20in%20Arabidopsis%20thaliana" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=In+planta+localisation+patterns+of+MADS+domain+proteins+during+floral+development+in+Arabidopsis+thaliana+Urbanus+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pbio.1001835-Ditta1" id="pbio.1001835-Ditta1" class="link-target"></a>Ditta G, Pinyopich A, Robles P, Pelaz S, Yanofsky MF (2004) The SEP4 gene of Arabidopsis thaliana functions in floral organ and meristem identity. Curr Biol 14: 1935–1940. <ul class="reflinks"><li><a href="#" data-author="Ditta" data-cit="DittaG%2C%20PinyopichA%2C%20RoblesP%2C%20PelazS%2C%20YanofskyMF%20%282004%29%20The%20SEP4%20gene%20of%20Arabidopsis%20thaliana%20functions%20in%20floral%20organ%20and%20meristem%20identity.%20Curr%20Biol%2014%3A%201935%E2%80%931940." data-title="The%20SEP4%20gene%20of%20Arabidopsis%20thaliana%20functions%20in%20floral%20organ%20and%20meristem%20identity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+SEP4+gene+of+Arabidopsis+thaliana+functions+in+floral+organ+and+meristem+identity+Ditta+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pbio.1001835-Pelaz1" id="pbio.1001835-Pelaz1" class="link-target"></a>Pelaz S, Ditta GS, Baumann E, Wisman E, Yanofsky MF (2000) B and C floral organ identity functions require SEPALLATA MADS-box genes. Nature 405: 200–203. <ul class="reflinks"><li><a href="#" data-author="Pelaz" data-cit="PelazS%2C%20DittaGS%2C%20BaumannE%2C%20WismanE%2C%20YanofskyMF%20%282000%29%20B%20and%20C%20floral%20organ%20identity%20functions%20require%20SEPALLATA%20MADS-box%20genes.%20Nature%20405%3A%20200%E2%80%93203." data-title="B%20and%20C%20floral%20organ%20identity%20functions%20require%20SEPALLATA%20MADS-box%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=B+and+C+floral+organ+identity+functions+require+SEPALLATA+MADS-box+genes+Pelaz+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pbio.1001835-Lindbo1" id="pbio.1001835-Lindbo1" class="link-target"></a>Lindbo JA (2007) TRBO: A high-efficiency tobacco mosaic virus RNA-based overexpression vector. Plant Physiol 145: 1232–1240. <ul class="reflinks"><li><a href="#" data-author="Lindbo" data-cit="LindboJA%20%282007%29%20TRBO%3A%20A%20high-efficiency%20tobacco%20mosaic%20virus%20RNA-based%20overexpression%20vector.%20Plant%20Physiol%20145%3A%201232%E2%80%931240." data-title="TRBO%3A%20A%20high-efficiency%20tobacco%20mosaic%20virus%20RNA-based%20overexpression%20vector" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=TRBO%3A+A+high-efficiency+tobacco+mosaic+virus+RNA-based+overexpression+vector+Lindbo+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pbio.1001835-Verelst1" id="pbio.1001835-Verelst1" class="link-target"></a>Verelst W, Twell D, de Folter S, Immink R, Saedler H, et al. (2007) MADS-complexes regulate transcriptome dynamics during pollen maturation. Genome Biol 8 ((11)) R249. <ul class="reflinks"><li><a href="#" data-author="Verelst" data-cit="VerelstW%2C%20TwellD%2C%20de%20FolterS%2C%20ImminkR%2C%20SaedlerH%2C%20et%20al.%20%282007%29%20MADS-complexes%20regulate%20transcriptome%20dynamics%20during%20pollen%20maturation.%20Genome%20Biol%208%20%28%2811%29%29%20R249." data-title="MADS-complexes%20regulate%20transcriptome%20dynamics%20during%20pollen%20maturation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MADS-complexes+regulate+transcriptome+dynamics+during+pollen+maturation+Verelst+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pbio.1001835-Meng1" id="pbio.1001835-Meng1" class="link-target"></a>Meng LH, Mohan R, Kwok BHB, Elofsson M, Sin N, et al. (1999) Epoxomicin, a potent and selective proteasome inhibitor, exhibits in vivo antiinflammatory activity. Proc Natl Acad Sci U S A 96: 10403–10408. <ul class="reflinks"><li><a href="#" data-author="Meng" data-cit="MengLH%2C%20MohanR%2C%20KwokBHB%2C%20ElofssonM%2C%20SinN%2C%20et%20al.%20%281999%29%20Epoxomicin%2C%20a%20potent%20and%20selective%20proteasome%20inhibitor%2C%20exhibits%20in%20vivo%20antiinflammatory%20activity.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%2096%3A%2010403%E2%80%9310408." data-title="Epoxomicin%2C%20a%20potent%20and%20selective%20proteasome%20inhibitor%2C%20exhibits%20in%20vivo%20antiinflammatory%20activity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Epoxomicin%2C+a+potent+and+selective+proteasome+inhibitor%2C+exhibits+in+vivo+antiinflammatory+activity+Meng+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pbio.1001835-Farmer1" id="pbio.1001835-Farmer1" class="link-target"></a>Farmer LM, Book AJ, Lee KH, Lin YL, Fu HY, et al. (2010) The RAD23 family provides an essential connection between the 26S proteasome and ubiquitylated proteins in Arabidopsis. Plant Cell 22: 124–142. <ul class="reflinks"><li><a href="#" data-author="Farmer" data-cit="FarmerLM%2C%20BookAJ%2C%20LeeKH%2C%20LinYL%2C%20FuHY%2C%20et%20al.%20%282010%29%20The%20RAD23%20family%20provides%20an%20essential%20connection%20between%20the%2026S%20proteasome%20and%20ubiquitylated%20proteins%20in%20Arabidopsis.%20Plant%20Cell%2022%3A%20124%E2%80%93142." data-title="The%20RAD23%20family%20provides%20an%20essential%20connection%20between%20the%2026S%20proteasome%20and%20ubiquitylated%20proteins%20in%20Arabidopsis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+RAD23+family+provides+an+essential+connection+between+the+26S+proteasome+and+ubiquitylated+proteins+in+Arabidopsis+Farmer+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pbio.1001835-Smaczniak1" id="pbio.1001835-Smaczniak1" class="link-target"></a>Smaczniak C, Immink RGH, Angenent GC, Kaufmann K (2012) Developmental and evolutionary diversity of plant MADS-domain factors: insights from recent studies. Development 139: 3081–3098. <ul class="reflinks"><li><a href="#" data-author="Smaczniak" data-cit="SmaczniakC%2C%20ImminkRGH%2C%20AngenentGC%2C%20KaufmannK%20%282012%29%20Developmental%20and%20evolutionary%20diversity%20of%20plant%20MADS-domain%20factors%3A%20insights%20from%20recent%20studies.%20Development%20139%3A%203081%E2%80%933098." data-title="Developmental%20and%20evolutionary%20diversity%20of%20plant%20MADS-domain%20factors%3A%20insights%20from%20recent%20studies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Developmental+and+evolutionary+diversity+of+plant+MADS-domain+factors%3A+insights+from+recent+studies+Smaczniak+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pbio.1001835-Ferrandiz1" id="pbio.1001835-Ferrandiz1" class="link-target"></a>Ferrandiz C, Gu Q, Martienssen R, Yanofsky MF (2000) Redundant regulation of meristem identity and plant architecture by FRUITFULL, APETALA1 and CAULIFLOWER. Development 127: 725–734. <ul class="reflinks"><li><a href="#" data-author="Ferrandiz" data-cit="FerrandizC%2C%20GuQ%2C%20MartienssenR%2C%20YanofskyMF%20%282000%29%20Redundant%20regulation%20of%20meristem%20identity%20and%20plant%20architecture%20by%20FRUITFULL%2C%20APETALA1%20and%20CAULIFLOWER.%20Development%20127%3A%20725%E2%80%93734." data-title="Redundant%20regulation%20of%20meristem%20identity%20and%20plant%20architecture%20by%20FRUITFULL%2C%20APETALA1%20and%20CAULIFLOWER" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Redundant+regulation+of+meristem+identity+and+plant+architecture+by+FRUITFULL%2C+APETALA1+and+CAULIFLOWER+Ferrandiz+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pbio.1001835-Immink1" id="pbio.1001835-Immink1" class="link-target"></a>Immink RGH, Tonaco IAN, de Folter S, Shchennikova A, van Dijk ADJ, et al. (2009) SEPALLATA3: the ‘glue’ for MADS box transcription factor complex formation. Genome Biol 10 ((2)) R24. <ul class="reflinks"><li><a href="#" data-author="Immink" data-cit="ImminkRGH%2C%20TonacoIAN%2C%20de%20FolterS%2C%20ShchennikovaA%2C%20van%20DijkADJ%2C%20et%20al.%20%282009%29%20SEPALLATA3%3A%20the%20%E2%80%98glue%E2%80%99%20for%20MADS%20box%20transcription%20factor%20complex%20formation.%20Genome%20Biol%2010%20%28%282%29%29%20R24." data-title="SEPALLATA3%3A%20the%20%E2%80%98glue%E2%80%99%20for%20MADS%20box%20transcription%20factor%20complex%20formation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=SEPALLATA3%3A+the+%E2%80%98glue%E2%80%99+for+MADS+box+transcription+factor+complex+formation+Immink+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pbio.1001835-Pracros1" id="pbio.1001835-Pracros1" class="link-target"></a>Pracros P, Renaudin J, Eveillard S, Mouras A, Hernould M (2006) Tomato flower abnormalities induced by stolbur phytoplasma infection are associated with changes of expression of floral development genes. Mol Plant Microbe Interact 19: 62–68. <ul class="reflinks"><li><a href="#" data-author="Pracros" data-cit="PracrosP%2C%20RenaudinJ%2C%20EveillardS%2C%20MourasA%2C%20HernouldM%20%282006%29%20Tomato%20flower%20abnormalities%20induced%20by%20stolbur%20phytoplasma%20infection%20are%20associated%20with%20changes%20of%20expression%20of%20floral%20development%20genes.%20Mol%20Plant%20Microbe%20Interact%2019%3A%2062%E2%80%9368." data-title="Tomato%20flower%20abnormalities%20induced%20by%20stolbur%20phytoplasma%20infection%20are%20associated%20with%20changes%20of%20expression%20of%20floral%20development%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tomato+flower+abnormalities+induced+by+stolbur+phytoplasma+infection+are+associated+with+changes+of+expression+of+floral+development+genes+Pracros+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pbio.1001835-Su1" id="pbio.1001835-Su1" class="link-target"></a>Su YT, Chen JC, Lin CP (2011) Phytoplasma-induced floral abnormalities in catharanthus roseus are associated with phytoplasma accumulation and transcript repression of floral organ identity genes. Mol Plant Microbe Interact 24: 1502–1512. <ul class="reflinks"><li><a href="#" data-author="Su" data-cit="SuYT%2C%20ChenJC%2C%20LinCP%20%282011%29%20Phytoplasma-induced%20floral%20abnormalities%20in%20catharanthus%20roseus%20are%20associated%20with%20phytoplasma%20accumulation%20and%20transcript%20repression%20of%20floral%20organ%20identity%20genes.%20Mol%20Plant%20Microbe%20Interact%2024%3A%201502%E2%80%931512." data-title="Phytoplasma-induced%20floral%20abnormalities%20in%20catharanthus%20roseus%20are%20associated%20with%20phytoplasma%20accumulation%20and%20transcript%20repression%20of%20floral%20organ%20identity%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phytoplasma-induced+floral+abnormalities+in+catharanthus+roseus+are+associated+with+phytoplasma+accumulation+and+transcript+repression+of+floral+organ+identity+genes+Su+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pbio.1001835-Himeno1" id="pbio.1001835-Himeno1" class="link-target"></a>Himeno M, Neriya Y, Minato N, Miura C, Sugawara K, et al. (2011) Unique morphological changes in plant pathogenic phytoplasma-infected petunia flowers are related to transcriptional regulation of floral homeotic genes in an organ-specific manner. Plant J 67: 971–979. <ul class="reflinks"><li><a href="#" data-author="Himeno" data-cit="HimenoM%2C%20NeriyaY%2C%20MinatoN%2C%20MiuraC%2C%20SugawaraK%2C%20et%20al.%20%282011%29%20Unique%20morphological%20changes%20in%20plant%20pathogenic%20phytoplasma-infected%20petunia%20flowers%20are%20related%20to%20transcriptional%20regulation%20of%20floral%20homeotic%20genes%20in%20an%20organ-specific%20manner.%20Plant%20J%2067%3A%20971%E2%80%93979." data-title="Unique%20morphological%20changes%20in%20plant%20pathogenic%20phytoplasma-infected%20petunia%20flowers%20are%20related%20to%20transcriptional%20regulation%20of%20floral%20homeotic%20genes%20in%20an%20organ-specific%20manner" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Unique+morphological+changes+in+plant+pathogenic+phytoplasma-infected+petunia+flowers+are+related+to+transcriptional+regulation+of+floral+homeotic+genes+in+an+organ-specific+manner+Himeno+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pbio.1001835-Dantuma1" id="pbio.1001835-Dantuma1" class="link-target"></a>Dantuma NP, Heinen C, Hoogstraten D (2009) The ubiquitin receptor Rad23: At the crossroads of nucleotide excision repair and proteasomal degradation. DNA Repair 8: 449–460. <ul class="reflinks"><li><a href="#" data-author="Dantuma" data-cit="DantumaNP%2C%20HeinenC%2C%20HoogstratenD%20%282009%29%20The%20ubiquitin%20receptor%20Rad23%3A%20At%20the%20crossroads%20of%20nucleotide%20excision%20repair%20and%20proteasomal%20degradation.%20DNA%20Repair%208%3A%20449%E2%80%93460." data-title="The%20ubiquitin%20receptor%20Rad23%3A%20At%20the%20crossroads%20of%20nucleotide%20excision%20repair%20and%20proteasomal%20degradation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+ubiquitin+receptor+Rad23%3A+At+the+crossroads+of+nucleotide+excision+repair+and+proteasomal+degradation+Dantuma+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pbio.1001835-Birch1" id="pbio.1001835-Birch1" class="link-target"></a>Birch PRJ, Armstrong M, Bos J, Boevink P, Gilroy EM, et al. (2009) Towards understanding the virulence functions of RXLR effectors of the oomycete plant pathogen Phytophthora infestans. J Exp Bot 60: 1133–1140. <ul class="reflinks"><li><a href="#" data-author="Birch" data-cit="BirchPRJ%2C%20ArmstrongM%2C%20BosJ%2C%20BoevinkP%2C%20GilroyEM%2C%20et%20al.%20%282009%29%20Towards%20understanding%20the%20virulence%20functions%20of%20RXLR%20effectors%20of%20the%20oomycete%20plant%20pathogen%20Phytophthora%20infestans.%20J%20Exp%20Bot%2060%3A%201133%E2%80%931140." data-title="Towards%20understanding%20the%20virulence%20functions%20of%20RXLR%20effectors%20of%20the%20oomycete%20plant%20pathogen%20Phytophthora%20infestans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Towards+understanding+the+virulence+functions+of+RXLR+effectors+of+the+oomycete+plant+pathogen+Phytophthora+infestans+Birch+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pbio.1001835-Anderson1" id="pbio.1001835-Anderson1" class="link-target"></a>Anderson DM, Frank DW (2012) Five mechanisms of manipulation by bacterial effectors: a ubiquitous theme. PLoS Pathog 8 ((8)) e1002823. <ul class="reflinks"><li><a href="#" data-author="Anderson" data-cit="AndersonDM%2C%20FrankDW%20%282012%29%20Five%20mechanisms%20of%20manipulation%20by%20bacterial%20effectors%3A%20a%20ubiquitous%20theme.%20PLoS%20Pathog%208%20%28%288%29%29%20e1002823." data-title="Five%20mechanisms%20of%20manipulation%20by%20bacterial%20effectors%3A%20a%20ubiquitous%20theme" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Five+mechanisms+of+manipulation+by+bacterial+effectors%3A+a+ubiquitous+theme+Anderson+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pbio.1001835-Angot1" id="pbio.1001835-Angot1" class="link-target"></a>Angot A, Vergunst A, Genin S, Peeters N (2007) Exploitation of eukaryotic ubiquitin signaling pathways by effectors translocated by bacterial type III and type IV secretion systems. PLoS Pathog 3: 1–13. <ul class="reflinks"><li><a href="#" data-author="Angot" data-cit="AngotA%2C%20VergunstA%2C%20GeninS%2C%20PeetersN%20%282007%29%20Exploitation%20of%20eukaryotic%20ubiquitin%20signaling%20pathways%20by%20effectors%20translocated%20by%20bacterial%20type%20III%20and%20type%20IV%20secretion%20systems.%20PLoS%20Pathog%203%3A%201%E2%80%9313." data-title="Exploitation%20of%20eukaryotic%20ubiquitin%20signaling%20pathways%20by%20effectors%20translocated%20by%20bacterial%20type%20III%20and%20type%20IV%20secretion%20systems" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Exploitation+of+eukaryotic+ubiquitin+signaling+pathways+by+effectors+translocated+by+bacterial+type+III+and+type+IV+secretion+systems+Angot+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pbio.1001835-Zhang1" id="pbio.1001835-Zhang1" class="link-target"></a>Zhang Y, Higashide WM, McCormick BA, Chen J, Zhou DG (2006) The inflammation-associated Salmonella SopA is a HECT-like E3 ubiquitin ligase. Mol Microbiol 62: 786–793. <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangY%2C%20HigashideWM%2C%20McCormickBA%2C%20ChenJ%2C%20ZhouDG%20%282006%29%20The%20inflammation-associated%20Salmonella%20SopA%20is%20a%20HECT-like%20E3%20ubiquitin%20ligase.%20Mol%20Microbiol%2062%3A%20786%E2%80%93793." data-title="The%20inflammation-associated%20Salmonella%20SopA%20is%20a%20HECT-like%20E3%20ubiquitin%20ligase" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+inflammation-associated+Salmonella+SopA+is+a+HECT-like+E3+ubiquitin+ligase+Zhang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pbio.1001835-Abramovitch1" id="pbio.1001835-Abramovitch1" class="link-target"></a>Abramovitch RB, Janjusevic R, Stebbins CE, Martin GB (2006) Type III effector AvrPtoB requires intrinsic E3 ubiquitin ligase activity to suppress plant cell death and immunity. Proc Natl Acad Sci U S A 103: 2851–2856. <ul class="reflinks"><li><a href="#" data-author="Abramovitch" data-cit="AbramovitchRB%2C%20JanjusevicR%2C%20StebbinsCE%2C%20MartinGB%20%282006%29%20Type%20III%20effector%20AvrPtoB%20requires%20intrinsic%20E3%20ubiquitin%20ligase%20activity%20to%20suppress%20plant%20cell%20death%20and%20immunity.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20103%3A%202851%E2%80%932856." data-title="Type%20III%20effector%20AvrPtoB%20requires%20intrinsic%20E3%20ubiquitin%20ligase%20activity%20to%20suppress%20plant%20cell%20death%20and%20immunity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Type+III+effector+AvrPtoB+requires+intrinsic+E3+ubiquitin+ligase+activity+to+suppress+plant+cell+death+and+immunity+Abramovitch+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pbio.1001835-Bos1" id="pbio.1001835-Bos1" class="link-target"></a>Bos JIB, Armstrong MR, Gilroy EM, Boevink PC, Hein I, et al. (2010) Phytophthora infestans effector AVR3a is essential for virulence and manipulates plant immunity by stabilizing host E3 ligase CMPG1. Proc Natl Acad Sci U S A 107: 9909–9914. <ul class="reflinks"><li><a href="#" data-author="Bos" data-cit="BosJIB%2C%20ArmstrongMR%2C%20GilroyEM%2C%20BoevinkPC%2C%20HeinI%2C%20et%20al.%20%282010%29%20Phytophthora%20infestans%20effector%20AVR3a%20is%20essential%20for%20virulence%20and%20manipulates%20plant%20immunity%20by%20stabilizing%20host%20E3%20ligase%20CMPG1.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20107%3A%209909%E2%80%939914." data-title="Phytophthora%20infestans%20effector%20AVR3a%20is%20essential%20for%20virulence%20and%20manipulates%20plant%20immunity%20by%20stabilizing%20host%20E3%20ligase%20CMPG1" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phytophthora+infestans+effector+AVR3a+is+essential+for+virulence+and+manipulates+plant+immunity+by+stabilizing+host+E3+ligase+CMPG1+Bos+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pbio.1001835-Park1" id="pbio.1001835-Park1" class="link-target"></a>Park CH, Chen SB, Shirsekar G, Zhou B, Khang CH, et al. (2012) The magnaporthe oryzae effector AvrPiz-t targets the RING E3 ubiquitin ligase APIP6 to suppress pathogen-associated molecular pattern-triggered immunity in rice. Plant Cell 24: 4748–4762. <ul class="reflinks"><li><a href="#" data-author="Park" data-cit="ParkCH%2C%20ChenSB%2C%20ShirsekarG%2C%20ZhouB%2C%20KhangCH%2C%20et%20al.%20%282012%29%20The%20magnaporthe%20oryzae%20effector%20AvrPiz-t%20targets%20the%20RING%20E3%20ubiquitin%20ligase%20APIP6%20to%20suppress%20pathogen-associated%20molecular%20pattern-triggered%20immunity%20in%20rice.%20Plant%20Cell%2024%3A%204748%E2%80%934762." data-title="The%20magnaporthe%20oryzae%20effector%20AvrPiz-t%20targets%20the%20RING%20E3%20ubiquitin%20ligase%20APIP6%20to%20suppress%20pathogen-associated%20molecular%20pattern-triggered%20immunity%20in%20rice" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+magnaporthe+oryzae+effector+AvrPiz-t+targets+the+RING+E3+ubiquitin+ligase+APIP6+to+suppress+pathogen-associated+molecular+pattern-triggered+immunity+in+rice+Park+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pbio.1001835-Singer1" id="pbio.1001835-Singer1" class="link-target"></a>Singer AU, Schulze S, Skarina T, Xu XH, Cui H, et al. (2013) A pathogen type III effector with a novel E3 ubiquitin ligase architecture. PLoS Pathog 9 ((1)) e1003121. <ul class="reflinks"><li><a href="#" data-author="Singer" data-cit="SingerAU%2C%20SchulzeS%2C%20SkarinaT%2C%20XuXH%2C%20CuiH%2C%20et%20al.%20%282013%29%20A%20pathogen%20type%20III%20effector%20with%20a%20novel%20E3%20ubiquitin%20ligase%20architecture.%20PLoS%20Pathog%209%20%28%281%29%29%20e1003121." data-title="A%20pathogen%20type%20III%20effector%20with%20a%20novel%20E3%20ubiquitin%20ligase%20architecture" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+pathogen+type+III+effector+with+a+novel+E3+ubiquitin+ligase+architecture+Singer+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pbio.1001835-LeNegrate1" id="pbio.1001835-LeNegrate1" class="link-target"></a>Le Negrate G, Krieg A, Faustin B, Loeffler M, Godzik A, et al. (2008) ChlaDub1 of Chlamydia trachomatis suppresses NF-kappa B activation and inhibits I kappa B alpha ubiquitination and degradation. Cellular Microbiol 10: 1879–1892. <ul class="reflinks"><li><a href="#" data-author="Le%20Negrate" data-cit="Le%20NegrateG%2C%20KriegA%2C%20FaustinB%2C%20LoefflerM%2C%20GodzikA%2C%20et%20al.%20%282008%29%20ChlaDub1%20of%20Chlamydia%20trachomatis%20suppresses%20NF-kappa%20B%20activation%20and%20inhibits%20I%20kappa%20B%20alpha%20ubiquitination%20and%20degradation.%20Cellular%20Microbiol%2010%3A%201879%E2%80%931892." data-title="ChlaDub1%20of%20Chlamydia%20trachomatis%20suppresses%20NF-kappa%20B%20activation%20and%20inhibits%20I%20kappa%20B%20alpha%20ubiquitination%20and%20degradation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=ChlaDub1+of+Chlamydia+trachomatis+suppresses+NF-kappa+B+activation+and+inhibits+I+kappa+B+alpha+ubiquitination+and+degradation+Le+Negrate+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pbio.1001835-LeNegrate2" id="pbio.1001835-LeNegrate2" class="link-target"></a>Le Negrate G, Faustin B, Welsh K, Loeffler M, Krajewska M, et al. (2008) Salmonella secreted factor L deubiquitinase of Salmonella typhimurium inhibits NF-kappa B, suppresses I kappa B alpha ubiquitination and modulates innate immune responses. J Immunol 180: 5045–5056. <ul class="reflinks"><li><a href="#" data-author="Le%20Negrate" data-cit="Le%20NegrateG%2C%20FaustinB%2C%20WelshK%2C%20LoefflerM%2C%20KrajewskaM%2C%20et%20al.%20%282008%29%20Salmonella%20secreted%20factor%20L%20deubiquitinase%20of%20Salmonella%20typhimurium%20inhibits%20NF-kappa%20B%2C%20suppresses%20I%20kappa%20B%20alpha%20ubiquitination%20and%20modulates%20innate%20immune%20responses.%20J%20Immunol%20180%3A%205045%E2%80%935056." data-title="Salmonella%20secreted%20factor%20L%20deubiquitinase%20of%20Salmonella%20typhimurium%20inhibits%20NF-kappa%20B%2C%20suppresses%20I%20kappa%20B%20alpha%20ubiquitination%20and%20modulates%20innate%20immune%20responses" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Salmonella+secreted+factor+L+deubiquitinase+of+Salmonella+typhimurium+inhibits+NF-kappa+B%2C+suppresses+I+kappa+B+alpha+ubiquitination+and+modulates+innate+immune+responses+Le+Negrate+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pbio.1001835-Kubori1" id="pbio.1001835-Kubori1" class="link-target"></a>Kubori T, Galan JE (2003) Temporal regulation of Salmonella virulence effector function by proteasome-dependent protein degradation. Cell 115: 333–342. <ul class="reflinks"><li><a href="#" data-author="Kubori" data-cit="KuboriT%2C%20GalanJE%20%282003%29%20Temporal%20regulation%20of%20Salmonella%20virulence%20effector%20function%20by%20proteasome-dependent%20protein%20degradation.%20Cell%20115%3A%20333%E2%80%93342." data-title="Temporal%20regulation%20of%20Salmonella%20virulence%20effector%20function%20by%20proteasome-dependent%20protein%20degradation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Temporal+regulation+of+Salmonella+virulence+effector+function+by+proteasome-dependent+protein+degradation+Kubori+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pbio.1001835-Anderson2" id="pbio.1001835-Anderson2" class="link-target"></a>Anderson DM, Schmalzer KM, Sato H, Casey M, Terhune SS, et al. (2011) Ubiquitin and ubiquitin-modified proteins activate the Pseudomonas aeruginosa T3SS cytotoxin, ExoU. Mol Microbiol 82: 1454–1467. <ul class="reflinks"><li><a href="#" data-author="Anderson" data-cit="AndersonDM%2C%20SchmalzerKM%2C%20SatoH%2C%20CaseyM%2C%20TerhuneSS%2C%20et%20al.%20%282011%29%20Ubiquitin%20and%20ubiquitin-modified%20proteins%20activate%20the%20Pseudomonas%20aeruginosa%20T3SS%20cytotoxin%2C%20ExoU.%20Mol%20Microbiol%2082%3A%201454%E2%80%931467." data-title="Ubiquitin%20and%20ubiquitin-modified%20proteins%20activate%20the%20Pseudomonas%20aeruginosa%20T3SS%20cytotoxin%2C%20ExoU" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Ubiquitin+and+ubiquitin-modified+proteins+activate+the+Pseudomonas+aeruginosa+T3SS+cytotoxin%2C+ExoU+Anderson+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pbio.1001835-Patel1" id="pbio.1001835-Patel1" class="link-target"></a>Patel JC, Hueffer K, Lam TT, Galan JE (2009) Diversification of a salmonella virulence protein function by ubiquitin-dependent differential localization. Cell 137: 283–294. <ul class="reflinks"><li><a href="#" data-author="Patel" data-cit="PatelJC%2C%20HuefferK%2C%20LamTT%2C%20GalanJE%20%282009%29%20Diversification%20of%20a%20salmonella%20virulence%20protein%20function%20by%20ubiquitin-dependent%20differential%20localization.%20Cell%20137%3A%20283%E2%80%93294." data-title="Diversification%20of%20a%20salmonella%20virulence%20protein%20function%20by%20ubiquitin-dependent%20differential%20localization" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Diversification+of+a+salmonella+virulence+protein+function+by+ubiquitin-dependent+differential+localization+Patel+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pbio.1001835-Nomura1" id="pbio.1001835-Nomura1" class="link-target"></a>Nomura K, Debroy S, Lee YH, Pumplin N, Jones J, et al. (2006) A bacterial virulence protein suppresses host innate immunity to cause plant disease. Science 313: 220–223. <ul class="reflinks"><li><a href="#" data-author="Nomura" data-cit="NomuraK%2C%20DebroyS%2C%20LeeYH%2C%20PumplinN%2C%20JonesJ%2C%20et%20al.%20%282006%29%20A%20bacterial%20virulence%20protein%20suppresses%20host%20innate%20immunity%20to%20cause%20plant%20disease.%20Science%20313%3A%20220%E2%80%93223." data-title="A%20bacterial%20virulence%20protein%20suppresses%20host%20innate%20immunity%20to%20cause%20plant%20disease" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+bacterial+virulence+protein+suppresses+host+innate+immunity+to+cause+plant+disease+Nomura+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pbio.1001835-FromontRacine1" id="pbio.1001835-FromontRacine1" class="link-target"></a>Fromont-Racine M, Rain JC, Legrain P (1997) Toward a functional analysis of the yeast genome through exhaustive two-hybrid screens. Nat Genet 16: 277–282. <ul class="reflinks"><li><a href="#" data-author="Fromont-Racine" data-cit="Fromont-RacineM%2C%20RainJC%2C%20LegrainP%20%281997%29%20Toward%20a%20functional%20analysis%20of%20the%20yeast%20genome%20through%20exhaustive%20two-hybrid%20screens.%20Nat%20Genet%2016%3A%20277%E2%80%93282." data-title="Toward%20a%20functional%20analysis%20of%20the%20yeast%20genome%20through%20exhaustive%20two-hybrid%20screens" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Toward+a+functional+analysis+of+the+yeast+genome+through+exhaustive+two-hybrid+screens+Fromont-Racine+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pbio.1001835-Gietz1" id="pbio.1001835-Gietz1" class="link-target"></a>Gietz RD, Schiestl RH (2007) High-efficiency yeast transformation using the LiAc/SS carrier DNA/PEG method. Nat Protoc 2: 31–34. <ul class="reflinks"><li><a href="#" data-author="Gietz" data-cit="GietzRD%2C%20SchiestlRH%20%282007%29%20High-efficiency%20yeast%20transformation%20using%20the%20LiAc%2FSS%20carrier%20DNA%2FPEG%20method.%20Nat%20Protoc%202%3A%2031%E2%80%9334." data-title="High-efficiency%20yeast%20transformation%20using%20the%20LiAc%2FSS%20carrier%20DNA%2FPEG%20method" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=High-efficiency+yeast+transformation+using+the+LiAc%2FSS+carrier+DNA%2FPEG+method+Gietz+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pbio.1001835-deFolter2" id="pbio.1001835-deFolter2" class="link-target"></a>de Folter S, Immink RG (2011) Yeast protein-protein interaction assays and screens. Methods Mol Biol 754: 145–165. <ul class="reflinks"><li><a href="#" data-author="de%20Folter" data-cit="de%20FolterS%2C%20ImminkRG%20%282011%29%20Yeast%20protein-protein%20interaction%20assays%20and%20screens.%20Methods%20Mol%20Biol%20754%3A%20145%E2%80%93165." data-title="Yeast%20protein-protein%20interaction%20assays%20and%20screens" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Yeast+protein-protein+interaction+assays+and+screens+de+Folter+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pbio.1001835-Severing1" id="pbio.1001835-Severing1" class="link-target"></a>Severing EI, van Dijk ADJ, Morabito G, Busscher-Lange J, Immink RGH, et al. (2012) Predicting the impact of alternative splicing on plant MADS domain protein function. PLoS ONE 7 ((1)) e30524. <ul class="reflinks"><li><a href="#" data-author="Severing" data-cit="SeveringEI%2C%20van%20DijkADJ%2C%20MorabitoG%2C%20Busscher-LangeJ%2C%20ImminkRGH%2C%20et%20al.%20%282012%29%20Predicting%20the%20impact%20of%20alternative%20splicing%20on%20plant%20MADS%20domain%20protein%20function.%20PLoS%20ONE%207%20%28%281%29%29%20e30524." data-title="Predicting%20the%20impact%20of%20alternative%20splicing%20on%20plant%20MADS%20domain%20protein%20function" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Predicting+the+impact+of+alternative+splicing+on+plant+MADS+domain+protein+function+Severing+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pbio.1001835-vanDijk1" id="pbio.1001835-vanDijk1" class="link-target"></a>van Dijk ADJ, Morabito G, Fiers M, van Ham RCHJ, Angenent GC, et al. (2010) Sequence motifs in MADS transcription factors responsible for specificity and diversification of protein-protein interaction. PLoS Comp Biol 6 ((11)) e1001017. <ul class="reflinks"><li><a href="#" data-author="van%20Dijk" data-cit="van%20DijkADJ%2C%20MorabitoG%2C%20FiersM%2C%20van%20HamRCHJ%2C%20AngenentGC%2C%20et%20al.%20%282010%29%20Sequence%20motifs%20in%20MADS%20transcription%20factors%20responsible%20for%20specificity%20and%20diversification%20of%20protein-protein%20interaction.%20PLoS%20Comp%20Biol%206%20%28%2811%29%29%20e1001017." data-title="Sequence%20motifs%20in%20MADS%20transcription%20factors%20responsible%20for%20specificity%20and%20diversification%20of%20protein-protein%20interaction" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence+motifs+in+MADS+transcription+factors+responsible+for+specificity+and+diversification+of+protein-protein+interaction+van+Dijk+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pbio.1001835-Kushnirov1" id="pbio.1001835-Kushnirov1" class="link-target"></a>Kushnirov VV (2000) Rapid and reliable protein extraction from yeast. Yeast 16: 857–860. <ul class="reflinks"><li><a href="#" data-author="Kushnirov" data-cit="KushnirovVV%20%282000%29%20Rapid%20and%20reliable%20protein%20extraction%20from%20yeast.%20Yeast%2016%3A%20857%E2%80%93860." data-title="Rapid%20and%20reliable%20protein%20extraction%20from%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Rapid+and+reliable+protein+extraction+from+yeast+Kushnirov+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pbio.1001835-Nakagawa1" id="pbio.1001835-Nakagawa1" class="link-target"></a>Nakagawa T, Kurose T, Hino T, Tanaka K, Kawamukai M, et al. (2007) Development of series of gateway binary vectors, pGWBs, for realizing efficient construction of fusion genes for plant transformation. J Biosci Bioeng 104: 34–41. <ul class="reflinks"><li><a href="#" data-author="Nakagawa" data-cit="NakagawaT%2C%20KuroseT%2C%20HinoT%2C%20TanakaK%2C%20KawamukaiM%2C%20et%20al.%20%282007%29%20Development%20of%20series%20of%20gateway%20binary%20vectors%2C%20pGWBs%2C%20for%20realizing%20efficient%20construction%20of%20fusion%20genes%20for%20plant%20transformation.%20J%20Biosci%20Bioeng%20104%3A%2034%E2%80%9341." data-title="Development%20of%20series%20of%20gateway%20binary%20vectors%2C%20pGWBs%2C%20for%20realizing%20efficient%20construction%20of%20fusion%20genes%20for%20plant%20transformation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Development+of+series+of+gateway+binary+vectors%2C+pGWBs%2C+for+realizing+efficient+construction+of+fusion+genes+for+plant+transformation+Nakagawa+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pbio.1001835-deFolter3" id="pbio.1001835-deFolter3" class="link-target"></a>de Folter S, Urbanus SL, van Zuijlen LG, Kaufmann K, Angenent GC (2007) Tagging of MADS domain proteins for chromatin immunoprecipitation. BMC Plant Biol 7: 47. <ul class="reflinks"><li><a href="#" data-author="de%20Folter" data-cit="de%20FolterS%2C%20UrbanusSL%2C%20van%20ZuijlenLG%2C%20KaufmannK%2C%20AngenentGC%20%282007%29%20Tagging%20of%20MADS%20domain%20proteins%20for%20chromatin%20immunoprecipitation.%20BMC%20Plant%20Biol%207%3A%2047." data-title="Tagging%20of%20MADS%20domain%20proteins%20for%20chromatin%20immunoprecipitation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tagging+of+MADS+domain+proteins+for+chromatin+immunoprecipitation+de+Folter+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pbio.1001835-Zhang2" id="pbio.1001835-Zhang2" class="link-target"></a>Zhang JH, Hogenhout SA, Nault LR, Hoy CW, Miller SA (2004) Molecular and symptom analyses of phytoplasma strains from lettuce reveal a diverse population. Phytopathology 94: 842–849. <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangJH%2C%20HogenhoutSA%2C%20NaultLR%2C%20HoyCW%2C%20MillerSA%20%282004%29%20Molecular%20and%20symptom%20analyses%20of%20phytoplasma%20strains%20from%20lettuce%20reveal%20a%20diverse%20population.%20Phytopathology%2094%3A%20842%E2%80%93849." data-title="Molecular%20and%20symptom%20analyses%20of%20phytoplasma%20strains%20from%20lettuce%20reveal%20a%20diverse%20population" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+and+symptom+analyses+of+phytoplasma+strains+from+lettuce+reveal+a+diverse+population+Zhang+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pbio.1001835-Karimi1" id="pbio.1001835-Karimi1" class="link-target"></a>Karimi M, Inze D, Depicker A (2002) GATEWAY((TM)) vectors for Agrobacterium-mediated plant transformation. Trends Plant Sci 7: 193–195. <ul class="reflinks"><li><a href="#" data-author="Karimi" data-cit="KarimiM%2C%20InzeD%2C%20DepickerA%20%282002%29%20GATEWAY%28%28TM%29%29%20vectors%20for%20Agrobacterium-mediated%20plant%20transformation.%20Trends%20Plant%20Sci%207%3A%20193%E2%80%93195." data-title="GATEWAY%28%28TM%29%29%20vectors%20for%20Agrobacterium-mediated%20plant%20transformation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=GATEWAY%28%28TM%29%29+vectors+for+Agrobacterium-mediated+plant+transformation+Karimi+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pbio.1001835-Clough1" id="pbio.1001835-Clough1" class="link-target"></a>Clough SJ, Bent AF (1998) Floral dip: a simplified method for Agrobacterium-mediated transformation of Arabidopsis thaliana. Plant J 16: 735–743. <ul class="reflinks"><li><a href="#" data-author="Clough" data-cit="CloughSJ%2C%20BentAF%20%281998%29%20Floral%20dip%3A%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana.%20Plant%20J%2016%3A%20735%E2%80%93743." data-title="Floral%20dip%3A%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Floral+dip%3A+a+simplified+method+for+Agrobacterium-mediated+transformation+of+Arabidopsis+thaliana+Clough+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pbio.1001835-Shevchenko1" id="pbio.1001835-Shevchenko1" class="link-target"></a>Shevchenko A, Tomas H, Havlis J, Olsen JV, Mann M (2006) In-gel digestion for mass spectrometric characterization of proteins and proteomes. Nat Protoc 1: 2856–2860. <ul class="reflinks"><li><a href="#" data-author="Shevchenko" data-cit="ShevchenkoA%2C%20TomasH%2C%20HavlisJ%2C%20OlsenJV%2C%20MannM%20%282006%29%20In-gel%20digestion%20for%20mass%20spectrometric%20characterization%20of%20proteins%20and%20proteomes.%20Nat%20Protoc%201%3A%202856%E2%80%932860." data-title="In-gel%20digestion%20for%20mass%20spectrometric%20characterization%20of%20proteins%20and%20proteomes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=In-gel+digestion+for+mass+spectrometric+characterization+of+proteins+and+proteomes+Shevchenko+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pbio.1001835-Cox1" id="pbio.1001835-Cox1" class="link-target"></a>Cox J, Mann M (2008) MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat Biotechnol 26: 1367–1372. <ul class="reflinks"><li><a href="#" data-author="Cox" data-cit="CoxJ%2C%20MannM%20%282008%29%20MaxQuant%20enables%20high%20peptide%20identification%20rates%2C%20individualized%20p.p.b.-range%20mass%20accuracies%20and%20proteome-wide%20protein%20quantification.%20Nat%20Biotechnol%2026%3A%201367%E2%80%931372." data-title="MaxQuant%20enables%20high%20peptide%20identification%20rates%2C%20individualized%20p.p.b.-range%20mass%20accuracies%20and%20proteome-wide%20protein%20quantification" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MaxQuant+enables+high+peptide+identification+rates%2C+individualized+p.p.b.-range+mass+accuracies+and+proteome-wide+protein+quantification+Cox+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

