



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2>
<div id="section1" class="section toc-section"><a id="" name="" class="link-target" title="Background"></a>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1" class="link-target"></a><p>Upper tract urothelial carcinomas (UT-UC) can invade the pelvicalyceal system making differential diagnosis of the various histologically distinct renal cell carcinoma (RCC) subtypes and UT-UC, difficult. Correct diagnosis is critical for determining appropriate surgery and post-surgical treatments. We aimed to identify microRNA (miRNA) signatures that can accurately distinguish the most prevalent RCC subtypes and UT-UC form the normal kidney.</p>
</div>

<div id="section2" class="section toc-section"><a id="" name="" class="link-target" title="Methods and Findings"></a>
<h3>Methods and Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1" class="link-target"></a><p>miRNA profiling was performed on FFPE tissue sections from RCC and UT-UC and normal kidney and 434 miRNAs were significantly deregulated in cancerous vs. the normal tissue. Hierarchical clustering distinguished UT-UCs from RCCs and classified the various RCC subtypes among them. qRT-PCR validated the deregulated expression profile for the majority of the miRNAs and ROC analysis revealed their capability to discriminate between tumour and normal kidney. An independent cohort of freshly frozen RCC and UT-UC samples was used to validate the deregulated miRNAs with the best discriminatory ability (AUC&gt;0.8, p&lt;0.001). Many of them were located within cytogenetic regions that were previously reported to be significantly aberrated. miRNA targets were predicted using the miRWalk algorithm and ingenuity pathway analysis identified the canonical pathways and curated networks of the deregulated miRNAs. Using the miRWalk algorithm, we further identified the top anti-correlated mRNA/miRNA pairs, between the deregulated miRNAs from our study and the top co-deregulated mRNAs among 5 independent ccRCC GEO datasets. The AB8/13 undifferentiated podocyte cells were used for functional assays using luciferase reporter constructs and the developmental transcription factor TFCP2L1 was proved to be a true target of miR-489, which was the second most upregulated miRNA in ccRCC.</p>
</div>

<div id="section3" class="section toc-section"><a id="" name="" class="link-target" title="Conclusions"></a>
<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1" class="link-target"></a><p>We identified novel miRNAs specific for each RCC subtype and UT-UC, we investigated their putative targets, the networks and pathways in which they participate and we functionally verified the true targets of the top deregulated miRNAs.</p>
</div>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Zaravinos A, Lambrou GI, Mourmouras N, Katafygiotis P, Papagregoriou G, Giannikou K, et al.  (2014) New miRNA Profiles Accurately Distinguish Renal Cell Carcinomas and Upper Tract Urothelial Carcinomas from the Normal Kidney. PLoS ONE 9(3):
           e91646.
        
        https://doi.org/10.1371/journal.pone.0091646</p><p><strong>Editor: </strong>Kin Mang Lau, The Chinese University of Hong Kong, Hong Kong</p><p><strong>Received: </strong>December 5, 2013; <strong>Accepted: </strong>February 11, 2014; <strong>Published: </strong> March 12, 2014</p><p><strong>Copyright: </strong> © 2014 Zaravinos et al. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This project was funded by the Cyprus Research Promotion Foundation, New Infrastructure/ΣΤΡΑΤΗ/0308/24 (co-funding by the EU Structural Funds) to CD. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Renal cell carcinoma (RCC) represents 2–3% of all cancers and accounts for approximately 90% of all kidney malignancies. Apart from surgery, it is both chemotherapy and radiotherapy resistant and it is composed of various morphologically and cytogenetically distinct subtypes. The most prevalent subtypes are clear cell RCC (ccRCC, 75–80%), papillary RCC (papRCC, 10–15%) and chromophobe RCC (chRCC, 5%) <a href="#pone.0091646-MeloniEhrig1" class="ref-tip">[1]</a>. Distinguishing RCC subtypes is of clinical importance because they have different prognoses and consequently different management schemes <a href="#pone.0091646-Ficarra1" class="ref-tip">[2]</a>. However, morphology-based distinction is not always conclusive since some subtypes may have overlapping or related morphologic features. For the purpose of targeted therapy it is especially important to classify the different subtypes of RCC. The histological types arise from different cells of origin in the kidney, different constellations of genetic alterations <a href="#pone.0091646-LopezBeltran1" class="ref-tip">[3]</a>, and expression or mutation in different oncogenic pathways. Therefore, different subtypes offer different molecular candidates for targeted therapy, such as Tyrosine Kinase Inhibitors, Sorafenib and Sunitinib, mTOR inhibitors, Everolimus and Temsirolimus, etc. There is growing evidence that variability in response rates may be linked to sub-classification <a href="#pone.0091646-Taziel1" class="ref-tip">[4]</a>. Therefore, new biomarkers are needed in order to improve the identification and diagnosis of renal tumor subtypes. Recent data suggest that RCC classification through microRNA (miRNA) expression profiles is highly accurate <a href="#pone.0091646-Youssef1" class="ref-tip">[5]</a>, <a href="#pone.0091646-Fridman1" class="ref-tip">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Upper tract urothelial carcinoma (UT-UC) is a relatively uncommon form of kidney cancer arising from the urothelial lining of the renal pelvis and calyces. UT-UC accounts for the majority of bladder cancer; however, it only accounts for about 7% of renal neoplasms <a href="#pone.0091646-Higgins1" class="ref-tip">[7]</a>. UC of the renal pelvis is an aggressive tumour, which may invade the renal parenchyma, mimicking primary renal cell carcinoma. Similarly, advanced RCC can invade the pelvicalyceal system. This can make differential diagnosis of RCC and urothelial carcinoma of the renal pelvis difficult. Correct diagnosis is critical for determining appropriate surgery and post-surgical treatments. For instance, UT-UC including renal pelvis, calyces and ureters will require radical nephrectomy with ureterectomy and bladder cuff resection. However, RCC will require only partial or radical nephrectomy without extensive ureter resection. Correct diagnosis is critical for determining appropriate surgery and post-surgical treatments. Therefore, it is of major importance to identify biomarkers that can accurately distinguish UT-UC from RCC <a href="#pone.0091646-Roupret1" class="ref-tip">[8]</a>, <a href="#pone.0091646-Kirkali1" class="ref-tip">[9]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>MicroRNAs (miRNAs) are small non-coding RNAs of approximately 19–23 nt size, shown to regulate gene expression at the post-transcriptional level, by binding through partial sequence homology to the 3′ UTR of mammalian target mRNAs and causing translational inhibition and/or mRNA degradation <a href="#pone.0091646-Garzon1" class="ref-tip">[10]</a>. miRNAs have been identified as key regulators in many biological processes including development, differentiation, apoptosis and proliferation. They are aberrantly expressed or mutated in cancers, suggesting that they may play a role as a novel class of oncogenes or tumour suppressor genes <a href="#pone.0091646-EsquelaKerscher1" class="ref-tip">[11]</a>. Deregulated miRNAs have been found in many tumours, including kidney cancer and carcinomas of the urinary tract <a href="#pone.0091646-Youssef1" class="ref-tip">[5]</a>, <a href="#pone.0091646-Chow1" class="ref-tip">[12]</a>, <a href="#pone.0091646-White1" class="ref-tip">[13]</a>, <a href="#pone.0091646-Zaravinos1" class="ref-tip">[14]</a>, <a href="#pone.0091646-Chow2" class="ref-tip">[15]</a>, <a href="#pone.0091646-Fridman2" class="ref-tip">[16]</a>, <a href="#pone.0091646-Jung1" class="ref-tip">[17]</a>, <a href="#pone.0091646-Nakada1" class="ref-tip">[18]</a>, <a href="#pone.0091646-Osanto1" class="ref-tip">[19]</a>, <a href="#pone.0091646-Petillo1" class="ref-tip">[20]</a>, <a href="#pone.0091646-Radojicic1" class="ref-tip">[21]</a>; however, not much is known regarding their expression profile in upper tract urothelial carcinomas. Furthermore, in RCC the results are not always consistent <a href="#pone.0091646-Chow1" class="ref-tip">[12]</a>, <a href="#pone.0091646-White1" class="ref-tip">[13]</a>, <a href="#pone.0091646-Jung1" class="ref-tip">[17]</a>, <a href="#pone.0091646-Nakada1" class="ref-tip">[18]</a>, <a href="#pone.0091646-Gottardo1" class="ref-tip">[22]</a>, <a href="#pone.0091646-Huang1" class="ref-tip">[23]</a>, <a href="#pone.0091646-Juan1" class="ref-tip">[24]</a>, <a href="#pone.0091646-Yi1" class="ref-tip">[25]</a>. Additionally, many deregulated miRNAs are located in chromosomal regions that are frequently altered in cancerous situations <a href="#pone.0091646-Sun1" class="ref-tip">[26]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>To our knowledge there are no accurate biomarkers for early detection of both RCC and UT-UC, thereby accounting for late diagnosis and subsequent poor prognosis. Better understanding of RCC and UT-UC pathogenesis is required for identifying biomarkers and developing novel targeted therapies. In the present study, we examined the miRNA expression profiles mainly of ccRCC, as well as in a smaller cohort of papRCC, chRCC and UT-UC samples. We identified novel miRNAs that are specific for each tumour entity, we investigated their putative targets, and we explored the networks and pathways in which these miRNAs participate. Finally, we functionally verified the target gene of the top deregulated miRNA in ccRCC.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Patient samples"></a>
<h3>Patient samples</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>Formalin-fixed and paraffin embedded (FFPE) tissue samples from patients with kidney cancer and upper tract urothelial carcinoma were retrieved from the archives of the Asklipieio General Hospital, Athens, following research ethics board approval from the same Hospital. Paraffin sections from each specimen were reviewed by two independent pathologists and histologically classified according to the WHO classification and staged according to the UICC-TNM classification (2009). Nuclear grade was scored according to the Fuhrman classification system. Kidney specimens were collected between 2007 and 2011. Written informed consent was obtained from all patients included in this study. The study protocol was approved by the Ethics Committees of the Asklipieio General Hospital, Athens and the University of Cyprus. Twenty-four ccRCC, 5 papRCC, 3 chRCC, 5 UT-UC patients and 40 normal kidney tissue (parenchyma/cortex) samples which were used as controls, were enrolled in the present study (<a href="#pone-0091646-t001"><strong>Table 1</strong></a>). None of the patients had a history of previous bladder cancer or concomitant bladder cancer at UT-UC diagnosis. The criteria to select normal kidney tissues were that the normal kidney cortex tissue was from the same individual, from the same kidney, from the same surgical specimen, and collected under the same standard operation procedure conditions: if possible the sampling of normal cortex was done leaving ∼2 cm of macroscopically healthy cortex adjacent to the tumor untouched. ccRCC are macroscopically characterized by a sharp demarcation (fibrous pseudocapsule) between tumor and surrounding non-tumoral tissue. This is also reflected microscopically, as RCC are sharply demarcated from the surrounding tissue. The non-tumoral tissue was therefore obtained in macroscopically clear-cut benign tissue. Haematoxylin and eosin (H&amp;E) staining and immunohistochemistry (IHC) for several markers (AE1/AE3 keratins, CK7, CK8, CK18, CK19, EMA, CD10, Vimentin, EGFR, Ki67, NSE, p53 and S-100) was performed for all RCC samples. Anti-vimentin staining was also performed for UT-UC samples (<a href="#pone-0091646-g001"><strong>Figure 1</strong></a>).</p>
<a class="link-target" id="pone-0091646-g001" name="pone-0091646-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g001" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Haematoxylin and eosin (H&amp;E) staining and immunohistochemistry (IHC).</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p><em>Upper pannel:</em> Representative H&amp;E staining from ccRCC, papRCC, chRCC and normal kidney tissue. About 5–10 serial tissue sections of 10 μm were cut from each FFPE block, deparaffinized with xylene, hydrated and stained with H&amp;E before microscopic examination. When the proportion of tumour cells was &gt;70% the FFPE block was subjected to total RNA extraction. <em>Lower pannel:</em> IHC of FFPE tissue sections using anti-vimentin as primary antibody. Vimentin was predominantly seen in ccRCC and papRCC (∼70% and ∼50%, respectively), but only rarely in chRCC (4%) and absent in the normal kidney. Vimentin was also down-regulated in the majority of UT-UC cases.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g001">
              https://doi.org/10.1371/journal.pone.0091646.g001</a></p></div><a class="link-target" id="pone-0091646-t001" name="pone-0091646-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.t001" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Clinicopathological information of the patients.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.t001">
              https://doi.org/10.1371/journal.pone.0091646.t001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="RNA extraction and microRNA microarray analysis"></a>
<h3>RNA extraction and microRNA microarray analysis</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Total RNA containing small RNAs was isolated from 18 ccRCC, 5 papRCC, 3 chRCC, 4 UT-UC and 20 normal tissue samples using the High Pure FFPE RNA Micro Kit (Roche Applied Science, IN). The quality of the total RNA was verified by an Agilent 2100 Bioanalyzer (Agilent Technologies, Santa Clara, CA) and the concentration was measured spectrophotometrically (Nanodrop technologies, Montchanin, DE). Tissue sections were deparaffinized with xylene and ethanol washes, treated with protease and total RNA containing small RNAs was isolated using the High Pure FFPE RNA Micro Kit (Roche Applied Science). Total RNA (0.5 μg) from each sample and reference was labeled with Hy3 fluorescent label, using the miRCURY LNA microRNA Hi-Power Labeling Kit (Exiqon, Vedbaek, Denmark), according to the manufacturer protocol. The Hy3-labeled samples were hybridized to the miRCURY LNA microRNA Array, 6<sup>th</sup> gen (Exiqon, Vedbaek, Denmark), containing capture probes that target all miRNAs for all species registered in the miRBase v18.0. The hybridization was performed using an Agilent hybridization SureHyb chamber and gasket slide kits. After hybridization, the microarray slides were scanned at 10 μm using the High-Resolution Microarray Scanner (Agilent Technologies) and stored in an ozone free environment. The image analysis was carried out using the ImaGene 8.0 software (BioDiscovery, Inc., USA). Filtering was performed based on the signal intensity. Spots with no signal above the background (flags 1 and 2) were detected and removed. Background correction was performed by subtracting the median global background from the median local background from the signal intensity. Normalization was performed using the quantile normalization algorithm. Normalized data were further extracted, pre-processed and sorted with Microsoft Excel®. For further data analysis the Matlab® (The Mathworks Inc.) computing environment was used. MicroRNAs were considered to be significantly differentially expressed (DE) if they obtained a p-value&lt;0.05 and a FDR≤0.05. Two-way average-linkage hierarchical clustering (HCl) with Euclidian distance was performed with Matlab® and Genesis 1.7.2 (Technische Universitaet-Graz, Austria) software. Our MIAME compliant microarray data were deposited in the Gene Expression Omnibus (GEO; <a href="http://www.ncbi.nlm.nih.gov/geo">www.ncbi.nlm.nih.gov/geo</a>) under accession number GSE48008.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Quantitative real time polymerase chain reaction (qRT-PCR) validation in FFPE samples and an independent cohort of freshly frozen tissue samples"></a>
<h3>Quantitative real time polymerase chain reaction (qRT-PCR) validation in FFPE samples and an independent cohort of freshly frozen tissue samples</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>qRT-PCR for the top DE miRNAs was performed for all FFPE samples for which total RNA was initially extracted (24 ccRCCs, 5 papRCCs, 3 chRCCs, 5 UT-UCs and 40 normal kidney tissue samples) including those used for the microarray analysis.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2" class="link-target"></a><p>Additionally, a blinded independent validation set of 40 freshly frozen kidney tissue samples (10 ccRCCs, 3 papRCCs, 4 chRCCs, 3 UT-UCs and 20 normal kidney cortex samples) was used to study the assay performance. The validation set included samples that met the same selection criteria with the initial FFPE cohort, such as &gt;70% tumor cells in each sample and concordance in diagnosis between two independent pathologists. Samples were collected at the Asklipieio General Hospital, Athens. Only samples for which both pathologists agreed in the diagnosis were included in the validation cohort.</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3" class="link-target"></a><p>Total RNA was extracted by homogenizing approximately 0.2 g of frozen tissue with a tissue homogenizer, followed by isolation using the mirVANA miRNA Isolation Kit (Ambion, Austin, TX) and the concentration was measured spectrophotometrically (Nanodrop technologies, Montchanin, DE). The microRNA in the total RNA sample (80 ng/ul) was converted to cDNA by reverse transcription using the miRCURY LNA Universal RT cDNA synthesis kit (Exiqon, Vedbaek, Denmark). qRT-PCR using appropriate endogenous controls (SNORA66, U6snRNA, RNU1A1) was performed in order to validate the microarray results of the top up-regulated miRNAs: miR-3648, miR-489, miR-638, miR-3656, miR-3687, miR-663b, miR-3126-5p, miR-25-5p, miR-1908, miR-204-3p, miR-711, miR-516b-5p, miR-498 and miR-612; as well as the top down-regulated ones: miR-145-3p, miR-551b-5p, miR-155-3p, miR-106b-3p, miR-140-5p, miR-4284, miR-520g, miR-520e, miR-944, miR-656 and miR-141-3p. The cDNA template was diluted 80x and amplified using SYBR® Green master mix (Exiqon, Vedbaek, Denmark) and LNA microRNA-specific primers on a ViiA 7 Real-Time PCR System (Applied Biosystems). Polymerase activation/DNA denaturation was at 95°C for 10 mins and qPCR amplification was performed for 45 cycles at 95°C for 10 sec and 60°C for 1 min with a ramp-rate 1.6°C/sec, followed by melt curve analysis. All reactions were performed in triplicates. Before proceeding with data normalization, a technical quality assessment was performed based on results of the melting curve, serial dilution curve and no-template-controls. The stability and ranking of the endogenous controls was calculated with the NormFinder algorithim (<a href="http://www.mdl.dk/publicationsnormfinder.htm">www.mdl.dk/publicationsnormfinder.htm</a>). For every endogenous control gene, the pair-wise variation with all other endogenous controls was determined as a gene stability measurement M. Genes with expression stability M&gt;1.5 were considered unstable across the samples and unsuitable for endogenous controls in this experiment. Relative expression was performed using the ΔΔCt method as previously described <a href="#pone.0091646-Radojicic1" class="ref-tip">[21]</a>, <a href="#pone.0091646-Zaravinos2" class="ref-tip">[27]</a>. Data were standardized by log<sub>2</sub> transformation.</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Locked nucleic acids-in situ hybridization (LNA-ISH)"></a>
<h3>Locked nucleic acids-in situ hybridization (LNA-ISH)</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>For LNA-ISH we used the probe sequences: miR-25-5p, 5′-DIG/CAATTGCCCAAGTCTCCGCCT/DIG-3′; miR-3687, 5′-DIG/ACGTCGCACGAACGCCTGT/DIG-3′; LNA U6 snRNA control probe, 5′-DIG/CACGAATTTGCGTGTCATCCTT-3′; LNA scrambled microRNA control probe, 5′-DIG/GTGTAACACGTCTATACGCCCA/DIG-3′. Briefly, 6-µm sections of FFPE tumour and control tissue samples were prepared. Paraffin was melted at 60°C on the day before conducting the ISH experiment and slides were stored overnight at 4°C in an RNase-free environment. The following day, slides were deparaffinized in xylene and ethanol washes, treated with proteinase K (15 μg/ml) for 30 minutes at 37°C, washed in PBS and dehydrated in ethanol. Hybridization mix (1 nM LNA U6 snRNA, 50 nM double-DIG LNA miR-25-5p and miR-3687 probes or 50 nM double-DIG LNA scrambled microRNA probe) (Exiqon, Vedbaek, Denmark) was applied for 2 hours at ∼30°C below the RNA Tm of the corresponding LNA probe, in a StatSpin ThermoBrite System (Abbott Laboratories Ltd). Three different reactions were performed per slide. This was achieved by cutting with scissors the polypropylene coverslips to a size slightly larger than the tissue section. Three serial sections were placed per glass slide, in order to perform the test, the negative and positive controls on the same glass slide. This not only allowed for more efficient work, but permitted the U6 snRNA and scrambled sequence controls to be done under the same exact conditions as miR-25-5p or miR-3687. Slides were washed in SSC buffers prior to incubation with blocking solution. Subsequently, sheep anti-DIG-AP reagent was applied for 1 hour, slides were washed in PBS-T, incubated firstly with AP substrate (NBT/BCIP containing levamisol) for 2 hours at 30°C in a humidifying chamber protected from light, and then in KTBT buffer to stop the reaction. Nuclei were counter stained with Nuclear Fast Red. Slides were carefully rinsed in tap water and dehydrated in ethanol solutions. Finally, slides were mounted, left overnight to allow settlement of the precipitate and analyzed under a light microscope (Nikon eclipse TE2000-U) the subsequent day.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Computational analysis of the deregulated microRNAs"></a>
<h3>Computational analysis of the deregulated microRNAs</h3>

<div id="section1" class="section toc-section"><a id="s2e1" name="s2e1" class="link-target" title="Chromosomal location and phylogenetic analysis of DE miRNAs"></a><h4>Chromosomal location and phylogenetic analysis of DE miRNAs.</h4><a id="article1.body1.sec2.sec5.sec1.p1" name="article1.body1.sec2.sec5.sec1.p1" class="link-target"></a><p>Using miRBase v19 (<a href="http://www.mirbase.org/">www.mirbase.org/</a>), the chromosomal distribution of the DE miRNAs was compared with reported genomic alterations in the different RCC subtypes and UT-UC. The unaligned fasta format of the mature sequences of miRNA gene families was fetched using miRBase v19 and multiple sequence alignment was subsequently performed with the ClustalW algorithm using T-Coffee (<a href="http://www.ebi.ac.uk/Tools/msa/tcoffee/">www.ebi.ac.uk/Tools/msa/tcoffee/</a>). Finally, a phylogenetic tree for the miRNA sequences was built using bayesian reconstruction.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2e2" name="s2e2" class="link-target" title="miRNA target prediction"></a><h4>miRNA target prediction.</h4><a id="article1.body1.sec2.sec5.sec2.p1" name="article1.body1.sec2.sec5.sec2.p1" class="link-target"></a><p>The DE microRNAs were imported into the miRWalk algorithm (<a href="http://www.ma.uni-heidelberg.de/apps/zmf/mirwalk">www.ma.uni-heidelberg.de/apps/zmf/mirwalk</a>) and prediction of miRNA target sequences on the 3′UTR of their target genes was performed using 7 nucleotides as the minimum seed number. A multiple comparison using 4 additional algorithms was performed for filtering purposes, each one working based on different sets of properties regarding mRNA-miRNA targeting; TargetScan, miRanda, miRDB and Diana-microT. Filtering was performed for pairs of miRNA-mRNA targets that were predicted by all five algorithms and gave a p-value&lt;0.05, according to the miRWalk algorithm. This p-value depicts the probability of a miRNA 5′-end sequence to be randomly paired with a given 3′UTR mRNA sequence. The validated gene targets of the DE miRNAs in each RCC subtype and UT-UC were also investigated.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2e3" name="s2e3" class="link-target" title="Ingenuity pathway analysis for the significantly deregulated miRNAs"></a><h4>Ingenuity pathway analysis for the significantly deregulated miRNAs.</h4><a id="article1.body1.sec2.sec5.sec3.p1" name="article1.body1.sec2.sec5.sec3.p1" class="link-target"></a><p>Ingenuity Pathway Analysis, version 7 (IPA; Ingenuity Systems, USA; <a href="http://www.analysis.ingenuity.com">www.analysis.ingenuity.com</a>) was performed to identify the molecular pathways and functional groupings based on published literature for the significantly DE miRNAs. miRNAs were uploaded into IPA and overlaid onto a global molecular network developed from information contained in the application. Networks of these genes were generated by IPA based on their connectivity, each ranked by a score. This score is based on the hypergeometric distribution, calculated with the right-tailed Fisher’s Exact Test, and corresponds to the negative log of this p-value. Functional analysis in IPA identified the published biological functions that were most significantly associated with the genes in the network. Genes or gene products are represented as nodes, where shape indicates functional groups, and the biological relationship between two nodes is represented as an edge (line). All lines are supported by at least one reference in literature, textbook, or from canonical information stored in the Ingenuity Pathways knowledge database.</p>
</div>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Functional assays"></a>
<h3>Functional assays</h3>

<div id="section1" class="section toc-section"><a id="s2f1" name="s2f1" class="link-target" title="Identification of anti-correlated miRNA/mRNA pairs"></a><h4>Identification of anti-correlated miRNA/mRNA pairs.</h4><a id="article1.body1.sec2.sec6.sec1.p1" name="article1.body1.sec2.sec6.sec1.p1" class="link-target"></a><p>Using miRWalk, we searched for gene targets of the top deregulated miRNAs as described above. We also selected the top co-deregulated (top co-up- and top co-down-regulated) mRNA transcripts from 5 ccRCC GEO datasets (accession numbers: GSE6344, GSE4125, GSE15641, GSE781 and GSE6280). For functional analysis we chose the miRNA/mRNA pairs that exhibited anti-correlated profiles between our miRNA profiling and the co-deregulated mRNA transcripts from the aforementioned 5 GEO datasets.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2f2" name="s2f2" class="link-target" title="Bacterial transformation of ligated products and DNA sequencing"></a><h4>Bacterial transformation of ligated products and DNA sequencing.</h4><a id="article1.body1.sec2.sec6.sec2.p1" name="article1.body1.sec2.sec6.sec2.p1" class="link-target"></a><p>Primers were designed to introduce a SpeI and a HindIII restriction enzyme sites to be cloned into the pMiR-REPORT luciferase vector (<strong><a href="#pone.0091646.s006">Table S1</a></strong>). DH5a E. coli cells (Takara, Japan) were gown from glycerol stocks and plasmid DNA was isolated and linearized using SpeI and a HindIII restriction enzymes. PCR-generated inserts were gel purified and ligated into the pMiR-REPORT luciferase vector. Ligation products were transformed into competent DH5a cells and the next day clones were picked and plasmid DNA was isolated. Insert verification included a restriction reaction with SpeI and HindIII. Also, 100 ng of plasmid DNA containing the insert was sequenced to confirm the absence of unwanted mutations, using the forward sequencing primer: <span class="gene">5'-AGGCGATTAAGTTGGGTA-3′</span>.</p>
<a id="article1.body1.sec2.sec6.sec2.p2" name="article1.body1.sec2.sec6.sec2.p2" class="link-target"></a><p>DNA sequencing of predicted target regions was performed using BigDye V3.1 chemistry on an ABI Prism Genetic Analyzer (Applied Biosystems, CA). Sequencing primers were designed to flank the target region but also included an additional 300 bp on average on each side. Sequence electropherograms were obtained from the ABI Sequencing Analysis V5.2 software (Applied Biosystems, CA) and sequences were imported into MEGA4 software to be aligned against a reference sequence with ClustalW algorithm.</p>
</div>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="Real-Time PCR for endogenous miRNA detection"></a>
<h3>Real-Time PCR for endogenous miRNA detection</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>To examine the expression levels of the endogenous miR-23c, mR-204-3p, miR-520a-5p, miR-489, miR-874 and miR-513a-3p in undifferentiated AB8/13 cells, total RNA enriched in small RNAs was isolated from the cells using TRIzol® reagent (Ambion). miRNA specific reverse transcription was performed with the miRCURY LNA Universal RT cDNA synthesis kit (Exiqon, Vedbaek, Denmark). The cDNA template was diluted 80x and amplified using SYBR® Green master mix (Exiqon, Vedbaek, Denmark) and LNA microRNA-specific primers on a ViiA 7 Real-Time PCR System (Applied Biosystems). All reactions were performed in triplicates. SNORA66, U6 snRNA and RNU1A1 were used as reference.</p>
</div>

<div id="section8" class="section toc-section"><a id="s2h" name="s2h" class="link-target" title="Transfection of AB8/13 podocytes and luciferase assays"></a>
<h3>Transfection of AB8/13 podocytes and luciferase assays</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>To evaluate the binding efficiency of the top deregulated miRNAs onto their predicted targets, the pMiR-REPORT miRNA Expression Reporter Vector System (AMBION, Texas) was used. The AB8/13 undifferentiated podocyte cells were incubated at 33°C at 5% CO<sub>2</sub> and cultured in RPMI medium, supplemented with 10% Fetal Bovine Serum (FBS) (Invitrogen, CA), 1% of 100 units/ml Penicillin/Streptomycin (Invitrogen, CA) and 1% Insulin-Transferrin-Selenium (Invitrogen, CA). For the luciferase reporter system experiments, AB8/13 cells were transfected with equal amounts of the pMIR-REPORT luciferase and β-gal vectors (100 ng each), as well as with 25 nM of miRNA inhibitors, mimics, or the AllStars Negative Control scrambled sequence LNA (Qiagen, West Sussex, UK), using lipofectamine 2000 (Invitrogen, CA). The β-gal vector was used for normalization. Every experiment was performed in quadruplicates in 12-well cell culture plates with the appropriate controls. Cells were harvested 16 hours after transfection. The Dual-Light Assay Kit (Applied Biosystems, CA) was used for the quantification of both luciferase and β-gal in an automated luminometer (Sirius, Berthold Detection Systems, Pforzheim, Germany).</p>
</div>

<div id="section9" class="section toc-section"><a id="s2i" name="s2i" class="link-target" title="Western blotting"></a>
<h3>Western blotting</h3>
<a id="article1.body1.sec2.sec9.p1" name="article1.body1.sec2.sec9.p1" class="link-target"></a><p>AB8/13 cells were lysed in equal volumes of pre-heated 2xSDS loading buffer (Sodium Dodecyl Sulphate–125 mM Tris-HCl pH 6.8, 20% Glycerol, 2% SDS, 2% β-mercaptoethanol and bromophenol blue) and homogenized using a 2 ml syringe. Whole cell lysates were subsequently electrophoresed in a 12% SDS-Polyacrylamide gel. Gel transfer was held in a wet transfer system on Hybond Polyvinylidene Fluoride (PVDF–Millipore, MA) membranes. Membranes were blocked with 5% non-fat dry milk in PBS/0.01% Tween20 for 1 hour at room temperature. Primary antibody was diluted in milk and added to the membrane for one hour. TFCP2L1 protein was detected with the goat primary polyclonal antibody C-20 (SantaCruz Biotechnology, CA) at around 48 kDa. β-Tubulin was used as loading control by using the T-4026 primary antibody (Sigma, Taufkirchen, Germany). As secondary antibody we used the rabbit anti-mouse antibody (SantaCruz Biotechnology, CA), conjugated with Horseradish Peroxidase (HRP). Proteins were detected using the Enhanced ChemiLuminescence (ECL) Plus Blotting Detection system (Amersham Biosciences, Buckinghamshire, UK) and were visualized on a ChemiDoc MP imaging system using the Image Lab software (Bio-Rad Laboratories).</p>
</div>

<div id="section10" class="section toc-section"><a id="s2j" name="s2j" class="link-target" title="Statistical analyses"></a>
<h3>Statistical analyses</h3>
<a id="article1.body1.sec2.sec10.p1" name="article1.body1.sec2.sec10.p1" class="link-target"></a><p>Microarray data analysis was performed using Matlab®. The two tailed student t-test was used to test the mean differences between two groups. Continuous variables are expressed as median ± standard deviation unless indicated differently. Receiver operating characteristic (ROC) curves were established to evaluate the diagnostic value of deregulated miRNAs for differentiating between RCC subtypes and controls. Pearson's correlation coefficient (CC) was used to identify positive or negative linear correlation between variables. Linear regression analysis using miRNA expression levels as dependent variables and patient age, sex, tumour stage, grade, size and nodal status as independent variables, was also performed to identify miRNAs associated with the aforementioned characteristics. Linear regression analysis along with ANOVA were also performed to develop models that discriminate RCC/UT-UC from the normal kidney samples. The median expression values of all the DE miRNAs were used to perform linear regression analysis. Similar analysis was performed based on tumor grade. All tests were performed 2-sided and a p≤0.05 was considered as statistically significant. Luciferase expression levels were analyzed using one-way ANOVA followed by Tukey post-testing, after being normalized against β-gal expression levels. One-way ANOVA was also used to test densitometry results from western blot analysis, followed by Tukey post-testing. All statistical analyses were performed using GraphPad Prism v.5 and Matlab®.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="microRNA profiling"></a>
<h3>microRNA profiling</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>microRNA profiling was performed retrospectively in a total of 27 RCCs (3 chRCC, 5 papRCC and 19 ccRCC), 4 UT-UCs and 20 normal kidney tissues. We identified 434 deregulated (DE) miRNAs (p&lt;0.05 and FDR&lt;0.05) in all RCC subtypes and UT-UC versus the normal kidney. The majority of them was down-regulated in RCCs but over-expressed in UT-UCs. Overall, 126 miRNAs (29%) showed increased expression and 303 miRNAs (69.8%) had decreased expression in RCCs and UT-UCs vs. the normal kidneys (<a href="#pone-0091646-g002"><strong>Figure 2</strong></a>). Specifically, 374, 420, 421 and 409 miRNAs were 90% consistently down-regulated in ccRCC, papRCC, chRCC and UT-UC, respectively. Unsupervised two-way hierarchical clustering (HCl) with Euclidian distance accurately discriminated between RCC and UT-UC. Apart from one chRCC sample that was clustered with ccRCCs, HCl also managed to successfully classify the 3 RCC subtypes among them. Furthermore, it showed that ccRCC is more closely related to papRCC and that both are distinct from chRCC or UT-UC (<a href="#pone-0091646-g003"><strong>Figure 3</strong></a>). HCl pair-tissue-wise also discriminated perfectly ccRCC from papRCC, chRCC from ccRCC, chRCC from papRCC, as well as ccRCC from UT-UC, papRCC from UT-UC and chRCC from UT-UC (<strong><a href="#pone.0091646.s001">Figure S1</a></strong>). <strong><a href="#pone.0091646.s007">Table S2</a></strong> lists the top deregulated miRNAs for each RCC subtype and UT-UC. The overlapping relationship of the deregulated miRNAs among the three RCC subtypes, as well as among all tumour types is depicted in <a href="#pone-0091646-g004"><strong>Figure 4</strong></a><strong>.</strong> Tissue-specific miRNAs, including those that can distinguish RCCs from UT-UC, were identified. We found 16 miRNAs that were specifically down-regulated (miR-3144-5p, miR-193b-3p, miR-587, miR-3117-3p, miR-769-5p, miR-617, miR-3615, miR-148b-5p, miR-24-2-5p, miR-142-5p, miR-3164, miR-514b-5p, miR-375, miR-520c-3p, miR-191-5p and miR-520b/miR-520c-3p) and 5 miRNAs that were specifically up-regulated in UT-UC (miR-885-5p, miR-1912, miR-378a-5p, miR-24-3p and miR-26a-2-3p) <strong><a href="#pone.0091646.s008">Table S3</a></strong>.</p>
<a class="link-target" id="pone-0091646-g002" name="pone-0091646-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g002" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> microRNA profiling.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p>Four-hundred and thirty-four miRNAs were statistically significantly deregulated in all RCC subtypes and UT-UC versus the normal kidney. (A) Q-Q (quantile-quantile) plot. Red circles indicate the significantly deregulated miRNAs. (B) Frequencies of the t-scores and p-values. The deregulated miRNAs had a p&lt;0.05. (C) The volcano-plot depicts the 434 statistically significantly deregulated miRNAs in ccRCC, papRCC, chRCC and UT-UC versus the normal kidney, of which the majority was significantly down-regulated in the cancerous tissue compared to the latter. (D) FDR diagram depicting the percentage of FDR with respect to p-value along with a plot of the estimated a priori probability that the null hypothesis π(0), is true versus the tuning parameter, lambda, λ, with a cubic polynomial fitting curve.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g002">
              https://doi.org/10.1371/journal.pone.0091646.g002</a></p></div><a class="link-target" id="pone-0091646-g003" name="pone-0091646-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g003" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> Hierarchical Clustering (HCl).</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig2.caption1.p1" name="article1.body1.sec3.sec1.fig2.caption1.p1" class="link-target"></a><p>The unsupervised two-way HCl with Euclidian distance depicts differential miRNA expression in ccRCC, papRCC, chRCC and UT-UC. The log<sub>2</sub> fold change in each RCC subtype and UT-UC versus the normal kidney tissue was used to construct the heat map. miRNA profiling accurately discriminated between RCC and UT-UC, as well as among ccRCC, papRCC and chRCC. ccRCC, clear cell renal cell carcinoma; papRCC, papillary renal cell carcinoma; chRCC, chromophobe renal cell carcinoma; UT-UC, upper tract urothelial carcinoma. Red and blue colours show significant up- or down-regulation of each miRNA in the tumour versus the normal kidney, respectively.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g003">
              https://doi.org/10.1371/journal.pone.0091646.g003</a></p></div><a class="link-target" id="pone-0091646-g004" name="pone-0091646-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g004" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Overlapping relationship of the deregulated miRNAs.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig3.caption1.p1" name="article1.body1.sec3.sec1.fig3.caption1.p1" class="link-target"></a><p>Venn diagrams illustrate the overlapping relationship of the number of up-regulated miRNAs among RCC subtypes (A), down-regulated miRNAs among RCC subtypes (B), up-regulated miRNAs among RCC subtypes and UT-UCs (C), down-regulated miRNAs between RCC subtypes and UT-UCs (D). Ninety-four miRNAs were co-upregulated among ccRCC, papRCC and chRCC; and 11, 44 and 24 miRNAs were specifically up-regulated in each one of the three RCC subtypes (ccRCC, chRCC and papRCC), respectively. On the other hand, 222 miRNAs were co-down-regulated in the three RCC subtypes, whereas 16, 18 and 5 miRNAs were specifically down-regulated in ccRCC, chRCC and papRCC, respectively. When the DE miRNAs in each RCC subtype were combined with those in UT-UC, we identified 89 and 206 miRNAs that were up- and down-regulated, respectively in all tumor types.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g004">
              https://doi.org/10.1371/journal.pone.0091646.g004</a></p></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>ROC analysis of the top 30 up-regulated miRNAs in each tissue type, using the microarray expression data, evaluated the extent to which they could separate each tumour entity from the normal kidney tissue. Of them, the miRNAs with a p&lt;0.01 and an AUC&gt;0.84 were selected as successful distinguishing markers between each tumor subtype and the normal kidney tissue (<a href="#pone-0091646-g005"><strong>Figure 5</strong></a>). The ROC statistics of the miRNAs that exhibited the best discriminatory ability are depicted in <a href="#pone-0091646-t002"><strong>Table 2</strong></a>.</p>
<a class="link-target" id="pone-0091646-g005" name="pone-0091646-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g005" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> ROC analysis using microarray data.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig4.caption1.p1" name="article1.body1.sec3.sec1.fig4.caption1.p1" class="link-target"></a><p>ROC curves of the top 30 up-regulated miRNAs in each tissue type using the microarray expression data. Of them, the miRNAs with a p&lt;0.01 and an AUC&gt;0.8 were selected as successful distinguishing markers between cancerous and normal tissues. The median area under the curve (AUC) for ccRCC was 0.85 (A); for papRCC, median AUC = 0.94 (B); for chRCC, median AUC = 0.84 (C) and for UT-UC, median AUC = 0.94 (D).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g005">
              https://doi.org/10.1371/journal.pone.0091646.g005</a></p></div><a class="link-target" id="pone-0091646-t002" name="pone-0091646-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.t002" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> ROC statistics of the miRNAs that exhibited the best discriminatory ability in ccRCC (A), papRCC (B), chRCC (C) and UT-UC (D) vs. the normal kidney tissue.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.t002">
              https://doi.org/10.1371/journal.pone.0091646.t002</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="Validation of deregulated miRNA expression by qRT-PCR"></a>
<h3>Validation of deregulated miRNA expression by qRT-PCR</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>We verified by qRT-PCR the expression levels of the top 25 deregulated miRNAs as acquired by miRNA profiling (14 most up-regulated and 11 most down-regulated miRNAs). These included: miR-3648, miR-489, miR-638, miR-3656, miR-3687, miR-663b, miR-3126-5p, miR-25-5p, miR-1908, miR-204-3p, miR-711, miR-516b-5p, miR-498 and miR-612 (top up-regulated miRNAs); as well as miR-145-3p, miR-551b-5p, miR-155-3p, miR-106b-3p, miR-140-5p, miR-4284, miR-520g, miR-520e, miR-944, miR-656 and miR-141-3p (top down-regulated miRNAs). The validated significantly deregulated miRNAs in ccRCC, papRCC, chRCC and UT-UC are depicted in <a href="#pone-0091646-g006"><strong>Figure 6</strong></a>.</p>
<a class="link-target" id="pone-0091646-g006" name="pone-0091646-g006"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g006" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 6. </span> qRT-PCR validation of the top 25 deregulated miRNAs.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>The Volcano-plots depict the significantly deregulated miRNAs in ccRCC, papRCC, chRCC and UT-UC. Eight miRNAs were significantly up-regulated in ccRCC, 5 in papRCC, 3 in chRCC and 4 in UT-UC. On the other hand, miR-656 was significantly down-regulated in papRCC; miR-155-3p, miR-106b-3p, miR-140-5p and miR-656 were significantly down-regulated in chRCC; and miR-520g was significantly down-regulated in UT-UC. The threshold of statistically significant difference was set at p&lt;0.05 and log<sub>2</sub> fold change&gt;2.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g006">
              https://doi.org/10.1371/journal.pone.0091646.g006</a></p></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>The frequencies of differential expression were mostly consistent across the same subtypes. In keeping with the microarray results, some miRNAs showed a similar expression pattern between the subtypes. For example, miR-3687 exhibited a similar up-regulation pattern in all RCC subtypes and in UT-UC; miR-3656, miR-638 and miR-3648 were similarly up-regulated in ccRCC, papRCC and UT-UC and miR-25-5p was similarly up-regulated in ccRCC and papRCC. Also, miR-656 was equally down-regulated in papRCC and chRCC.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3" class="link-target"></a><p>In order to further validate the capability of the DE miRNAs to discriminate between tumour and normal kidney tissue, ROC curves were further constructed for the deregulated miRNAs in each RCC subtype and in UT-UC, using the normalized qRT-PCR data. miRNAs with a p&lt;0.01 and an AUC&gt;0.8 were then selected as successful distinguishing markers between cancerous and normal kidney tissue (<a href="#pone-0091646-g007"><strong>Figure 7</strong></a>). miR-3648, miR-638, miR-3656 and miR-3687 were excellent discriminators between ccRCC and normal tissue (median AUC&gt;0.8, p&lt;0.0001); miR-3687 and miR-25-5p were very good discriminators between papRCC and normal tissue (median AUC&gt;0.8, p&lt;0.05); miR-3687, miR-4284 and miR-141-3p were very good discriminators between chRCC and normal tissue (median AUC&gt;0.8, p&lt;0.05); and miR-3648, miR-3656 and miR-3687 discriminated perfectly UT-UC from the normal tissue (median AUC&gt;0.94, p&lt;0.001). ROC curve data are shown in <a href="#pone-0091646-t003"><strong>Table 3</strong></a>. The qRT-PCR results revealed similar expression patterns with the microarray results (<a href="#pone-0091646-g008"><strong>Figure 8</strong></a>).</p>
<a class="link-target" id="pone-0091646-g007" name="pone-0091646-g007"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g007" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 7. </span> ROC analysis using qRT-PCR data.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1" class="link-target"></a><p>ROC curves of the significantly deregulated miRNAs, using the expression data from qRT-PCR analysis. Of them, the miRNAs with a p&lt;0.01 and an AUC&gt;0.8 were selected as successful distinguishing markers between cancerous and normal kidney tissues. The median area under the curve (AUC) for ccRCC was 0.802 (A); for papRCC, median AUC = 0.756 (B); for chRCC, median AUC = 0.926 (C) and for UT-UC, median AUC = 0.955 (D).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g007">
              https://doi.org/10.1371/journal.pone.0091646.g007</a></p></div><a class="link-target" id="pone-0091646-g008" name="pone-0091646-g008"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g008"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g008" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g008"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g008" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g008">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g008">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g008">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g008">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g008">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g008">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 8. </span> Correlation between microarrays and qRT-PCR.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1" class="link-target"></a><p>Median log<sub>2</sub> fold change expression levels of the 14 most up-regulated and 11 most down-regulated miRNAs between ccRCC, papRCC, chRCC and UT-UC and the normal kidney tissue, as determined by both qRT-PCR and microarray analysis. As shown in the figure, qRT-PCR and microarray results were highly compatible. The most identical results between the two techniques were those for ccRCC, which was expected due to the high sample number (Pearson’s CC = 0.778, p&lt;0.001). The qRT-PCR results for papRCC, chRCC and UT-UC also revealed similar deregulation patterns with those of the microarray experimentation, however the correlation coefficients were lower, apparently due to small sample number (in papRCC, CC = 0.596, p = 0.002; in chRCC CC = 0.570, p = 0.003; in UT-UC, CC = 0.517, p = 0.009).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g008">
              https://doi.org/10.1371/journal.pone.0091646.g008</a></p></div><a class="link-target" id="pone-0091646-t003" name="pone-0091646-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.t003" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> ROC statistics of the miRNAs that exhibited the best discriminatory ability in ccRCC (A), papRCC (B), chRCC (C), and UT-UC (D) vs. the normal kidney tissue, respectively.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.t003">
              https://doi.org/10.1371/journal.pone.0091646.t003</a></p></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4" class="link-target"></a><p>A blinded independent validation set composed of 40 freshly frozen ccRCC, papRCC, chRCC, UT-UC and normal kidney samples was also used to validate the discriminatory ability of the qRT-PCR verified deregulated miRNAs. The results were highly repeatable between the two cohorts, recapitulating the specificity of these miRNAs in each tumour group (in ccRCC, Pearson's CC = 0.987, p&lt;0.001; in papRCC, Pearson's CC = 0.917, p = 0.001; in chRCC, Pearson's CC = 0.952, p&lt;0.001; in UT-UC, Pearson's CC = 0.926, p = 0.001) (<a href="#pone-0091646-g009"><strong>Figure 9</strong></a>).</p>
<a class="link-target" id="pone-0091646-g009" name="pone-0091646-g009"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g009"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g009" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g009"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g009" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g009">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g009">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g009">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g009">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g009">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g009">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 9. </span> Validation in a blinded independent cohort.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig4.caption1.p1" name="article1.body1.sec3.sec2.fig4.caption1.p1" class="link-target"></a><p>A blinded independent validation cohort composed of 40 freshly frozen ccRCC, papRCC, chRCC, UT-UC and normal kidney samples was also used to validate the discriminatory ability of the qRT-PCR verified deregulated miRNAs. The results were highly repeatable between the two cohorts, recapitulating the specificity of these miRNAs in each tumour group.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g009">
              https://doi.org/10.1371/journal.pone.0091646.g009</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="miRNA expression and clinical characteristics of the patients"></a>
<h3>miRNA expression and clinical characteristics of the patients</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>Only a few correlations between the expression levels of the DE miRNAs (verified by qRT-PCR) with the tumour stage, grade and size were identified. The expression levels of the significantly deregulated miRNAs that were verified by qRT-PCR, were correlated with the tumour stage, grade and size. Grade 4 ccRCCs exhibited higher miR-3656 and miR-3687 levels vs. those of grade 3 (p = 0.028 and p = 0.0497, respectively; unpaired t-test). Furthermore, grade 4 ccRCCs exhibited higher miR-21-5p levels vs. grade 2 ccRCCs (p = 0.0449, unpaired t-test). We also found a significant association between miR-21-5p expression and ccRCC size (Pearson's CC = 0.442, p = 0.031). The levels of miR-25-5p were significantly associated with papRCC tumour size (Pearson's CC = 0.935, p = 0.019). Grade II UT-UCs expressed higher levels of miR-3648 and miR-638 vs. grade III ones (p = 0.0153 and p = 0.0057, respectively). Finally, miR-638 was correlated with tumour size (Pearson's CC = 0.878, p = 0.050).</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>Furthermore, miRNA expression was validated using one-way ANOVA along with linear regression. Of note, miR-204-3p, miR-25-5p, miR-3648, miR-3687, miR-489, miR-516b-5p, miR-638 and miR-663b exhibited a linear expression pattern among the 3 RCC subtypes, UT-UC and the normal kidney samples (<em>R<sup>2</sup></em> range = 0.93–0.98). Also, linear regression could classify between RCC and UT-UC, with the latter expressing higher levels of the pre-mentioned miRNAs (<strong><a href="#pone.0091646.s002">Figure S2</a></strong>).</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3" class="link-target"></a><p>Similarly, high linearity was achieved when we grouped together ccRCC, papRCC and chRCC samples, and we compared them to the UT-UC and the normal kidney cases (miR-204-3p, <em>R<sup>2</sup>&gt;</em>0.99; miR-25-5p, <em>R<sup>2</sup>&gt;</em>0.83; miR-3648, <em>R<sup>2</sup>&gt;</em>0.98; miR-3687, <em>R<sup>2</sup>&gt;</em>0.95; miR-489, <em>R<sup>2</sup>&gt;</em>0.99; and miR-638, <em>R<sup>2</sup>&gt;</em>0.98) (<strong><a href="#pone.0091646.s003">Figure S3</a></strong>).</p>
<a id="article1.body1.sec3.sec3.p4" name="article1.body1.sec3.sec3.p4" class="link-target"></a><p>Linear regression of RCC grades with respect to miRNA expression, revealed that miR-148b-5p, miR-1910 and miR-200b-5p, miR-210 and miR-3656, presented linear behavior in a descending order of the RCC grade; whereas miR-3687 and miR-654-5p manifested linear behavior in ascending order of the RCC grade (<strong><a href="#pone.0091646.s004">Figure S4</a></strong>).</p>
<a id="article1.body1.sec3.sec3.p5" name="article1.body1.sec3.sec3.p5" class="link-target"></a><p>Furthermore, miR-148a-5p and miR-940 manifested excellent linear behavior in an descending order of age (<em>R<sup>2</sup></em> = 0.96 and 0.99, respectively), pointing out that both might be age-specific. Also, regression analysis showed that miR-145-3p and miR-1973 manifested increasing expression levels with ascending tumor size (&lt;5cm, 5-10cm and 10-15 cm) (<em>R<sup>2</sup></em> = 0.95 and 0.92, respectively).</p>
<a id="article1.body1.sec3.sec3.p6" name="article1.body1.sec3.sec3.p6" class="link-target"></a><p>Other varieties, such as a patient’s sex, age or tumor stage, were not associated with any other miRNA expression levels.</p>
</div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="Pair-wise miRNA correlations"></a>
<h3>Pair-wise miRNA correlations</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Several DE microRNAs were correlated among each other. In ccRCC, the statistically significant Spearman’s correlation coefficients ranged from r<sub>s</sub> = –0.585 (miR-106b-3p to miR-612; p = 0.002) to r<sub>s</sub> = 0.843 (miR-638 to miR-3687; p&lt;0.0001). In papRCC, the statistically significant Spearman’s correlation coefficients ranged from r<sub>s</sub> = –0.90 (miR-145-3p to miR-711; and miR-21-5p to miR-141-3p; both p = 0.083) to r<sub>s</sub> =  1.0 (miR-638 to miR-3648; miR-663b to miR-3656; miR-516b-5p to miR-489; miR-498 to miR-489; miR-155-3p to miR-489, among others; all p = 0.017). Furthermore, in UT-UC the statistically significant Spearman’s correlation coefficients ranged from r<sub>s</sub> =  –0.9 (miR-551b-5p to miR-3126-5p; miR-520g to miR-3126-5p; miR-141-3p to miR-1908, among others; all p = 0.083) to r<sub>s</sub> =  1.0 (miR-489 to miR-3648; miR-3656 to miR-3648; miR-3656 to miR-489; miR-155-3p to miR-638; miR-122-5p to miR-638, among others; all p = 0.017). Finally, in the normal kidney the statistically significant Spearman’s correlation coefficients ranged from r<sub>s</sub> = –0.325 (miR-551b-5p to miR-155-3p; p = 0.041) to r<sub>s</sub> = 0.885 (miR-3648 to miR-3656; p&lt;0.0001).</p>
</div>

<div id="section5" class="section toc-section"><a id="s3e" name="s3e" class="link-target" title="Validation of the pattern of expression of miR-25-5p and miR-3687 by LNA-ISH"></a>
<h3>Validation of the pattern of expression of miR-25-5p and miR-3687 by LNA-ISH</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>In situ hybridization using digoxigenin labeled LNA probes permitted the spatial localization of two of the most deregulated miRNAs: a) miR-25-5p which was significantly up-regulated in ccRCC and papRCC; and b) miR-3687 which was significantly up-regulated in all 3 RCC subtypes as well as in UT-UC. The detection of both miRNAs was confined to the cytoplasm. The nuclear expression of U6 snRNA was confirmed in all patient samples, whereas the scrambled oligonucleotide was negative in all samples. Each RCC or UT-UC section was compared against its corresponding normal kidney section. The elevated expression of miR-25-5p was confirmed in all ccRCC sections. Furthermore, ccRCCs of high stage and grade stained stronger miR-25-5p compared to lower stage and grade tumours. Also, papRCCs of type II stained stronger for miR-25-5p compared to type I papRCCs (<a href="#pone-0091646-g010"><strong>Figure 10</strong></a>). Likewise, the elevated expression of miR-3687 was confirmed in the majority of both RCC and UT-UC sections.</p>
<a class="link-target" id="pone-0091646-g010" name="pone-0091646-g010"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g010"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g010" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g010"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g010" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g010">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g010">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g010">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g010">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g010">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g010">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 10. </span> Locked Nucleic Acids-In Situ Hybridization (LNA-ISH).</div><p class="caption_target"><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1" class="link-target"></a><p>LNA-ISH for miR-25-5p in ccRCC (A), papRCC (B), chRCC (C) and UT-UC (D). miR-25-5p was confined to the cytoplasm both in normal and tumour sections. The nuclear expression of U6 snRNA (positive control) was confirmed in all the patient samples, whereas the scrambled oligonucleotide was negative in all samples. miR-25-5p high expression was confirmed in all ccRCC sections by LNA-ISH. ccRCCs of high stage and grade stained stronger miR-25-5p compared to lower stage and grade ccRCCs. Each ccRCC section was compared against its corresponding normal kidney section (A). papRCCs of type II stained stronger for miR-25-5p compared to type I papRCCs (B). Validating the qRT-PCR results, miR-25-5p did not stain stronger in chRCC sections vs. the normal tissue ones. This was also confirmed for chRCCs with focal sarcomatoid differentiation, suggesting that miR-25-5p does not play any role in the metastatic behavior of the tumour (C). Verifying the qRT-PCR results, miR-25-5p was not significantly stronger in UT-UC vs. the normal tissue sections (D).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g010">
              https://doi.org/10.1371/journal.pone.0091646.g010</a></p></div></div>

<div id="section6" class="section toc-section"><a id="s3f" name="s3f" class="link-target" title="Correlation of miRNA deregulation and chromosomal aberrations in RCC and UT-UC"></a>
<h3>Correlation of miRNA deregulation and chromosomal aberrations in RCC and UT-UC</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>The chromosomal locations of the DE miRNAs were compared to the most commonly reported chromosomal aberrations in ccRCC, chRCC, papRCC <a href="#pone.0091646-Ljungberg1" class="ref-tip">[28]</a> or UT-UC <a href="#pone.0091646-Roupret2" class="ref-tip">[29]</a>. In general, chromosomes 14q, 19q, Xq exhibited the highest number of deregulated miRNAs. In each RCC subtype, miRNAs had deregulated patterns that agreed with some of the previously reported chromosomal gains and losses; i.e., up-regulated miRNAs were located in regions of chromosomal gains and vice versa. <strong><a href="#pone.0091646.s009">Table S4</a></strong> lists a summary of the reported chromosomal abnormalities associated with the three RCC subtypes and UT-UC, as well as some of the well-studied genes located within those aberrated regions that are involved in the pathogenesis of the tumour subtypes. Many of the significantly deregulated miRNAs are located within cytogenetic regions that are significantly aberrated in the different subtypes. Chromosomal mapping for the deregulated miRNAs in each RCC subtype and in UT-UC is depicted in <a href="#pone-0091646-g011"><strong>Figure 11</strong></a>.</p>
<a class="link-target" id="pone-0091646-g011" name="pone-0091646-g011"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g011"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g011" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g011"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g011" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g011">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g011">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g011">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g011">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g011">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g011">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 11. </span> Correlation of miRNA deregulation and chromosomal aberrations.</div><p class="caption_target"><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1" class="link-target"></a><p>Chromosomal mapping for the deregulated miRNAs in ccRCC (A), papRCC (B), chRCC (C) and UT-UC (D). In ccRCC, the up-regulated miRNAs were mainly mapped on chromosomes Xq, 17q, 5q, 14q, 19q, 11q, 12q, 3p and 10p (in descending order); whereas the down-regulated ones were mainly mapped on chromosomes 14q, 19q, Xq, 8q, 1p, 17q, 7q, 3q, 10q, 12q and 2q (A). In papRCC, the up-regulated miRNAs were mainly mapped on chromosomes 19q, 14q, Xq, 5q, 3p, 16p, 11q, 9q and 15q; whereas the down-regulated miRNAs were mainly mapped on chromosomes 14q, 19q, Xq, 1p, 7q, 8q, 12q, 10q, 17q, 3q, 19p, Xp, 2q, 5, 6q, 4q, 11q, 15q and 20q (B). In chRCC, the up-regulated miRNAs were mainly mapped on chromosomes 14q, Xq, 19q, 17q, 5q, 7q, 11q, 15q, 2q and 12q; whereas the down-regulated ones were mainly mapped on chromosomes 14q, 19q, Xp, 3p, 8q, 17q, 3q, 10q, 12q, 4q, 7q, 3p, 5q, 19p and 20q (C). In UT-UC, the up-regulated miRNAs were mostly mapped on chromosomes 17q, 19q, 11q, 16p, 5q, 14q, 15q, Xq, 1p and 3p; whereas the down-regulated miRNAs were mainly mapped on chromosomes 14q, 19q, Xq, 8q, 1p, 7q, 12q, 3q, 10q, 5q, 17p, 19p, Xp, 2q, 4q, 3p and 8p (D).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g011">
              https://doi.org/10.1371/journal.pone.0091646.g011</a></p></div></div>

<div id="section7" class="section toc-section"><a id="s3g" name="s3g" class="link-target" title="Phylogenetic analysis of the DE miRNAs in RCC"></a>
<h3>Phylogenetic analysis of the DE miRNAs in RCC</h3>
<a id="article1.body1.sec3.sec7.p1" name="article1.body1.sec3.sec7.p1" class="link-target"></a><p>Phylogenetic analysis showed that 340 of the 434 (78.3%) DE miRNAs were highly conserved among species. Specifically, 70.68% of the up-regulated miRNAs and 81.22% of the down-regulated miRNAs were phylogenetically conserved among species, respectively. <strong><a href="#pone.0091646.s005">Figure S5</a></strong> shows representative multiple sequence alignments and trees of miR-489 and miR-25-5p.</p>
<a id="article1.body1.sec3.sec7.p2" name="article1.body1.sec3.sec7.p2" class="link-target"></a><p>IPA Core Analysis was performed for the significantly DE miRNAs, in each RCC subtype and in UT-UC. The log<sub>2</sub> ratio was set at 0.3 (range –1.9155 to 1.1864) and the DE miRNAs were filtered down to 61 for ccRCC, 48 for papRCC, 44 for chRCC and 77 for UT-UC. The top biological functions and the major miRNA networks are shown in <a href="#pone-0091646-g012"><strong>Figure 12</strong></a>.</p>
<a class="link-target" id="pone-0091646-g012" name="pone-0091646-g012"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g012"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g012" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g012"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g012" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g012">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g012">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g012">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g012">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g012">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g012">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 12. </span> Ingenuity pathway analysis (IPA).</div><p class="caption_target"><a id="article1.body1.sec3.sec7.fig1.caption1.p1" name="article1.body1.sec3.sec7.fig1.caption1.p1" class="link-target"></a><p>A. In ccRCC, the most important biological functions of the top deregulated miRNAs were: 1) Cancer (p = 8.71E-11-4.91E-02); 2) Renal and Urological Disease (p = 8.71E-11-2.12E-09); 3) Inflammatory Disease (p = 2.12E-09-2.14E-03); 4) Inflammatory Response (p = 2.12E-09-2.4E-04) and 5) Reproductive System Disease (p = 3.37E-05-2.58E-02). The associated functions of the major miRNA network (score = 30) were: Inflammatory disease, inflammatory response, renal inflammation. Argonaute RISC catalytic component 2 (EIF2C2), tumour protein p53 (TP53), v-myc myelocytomatosis viral oncogene homolog (MYC) and EPH receptor B6 (EPHB6) constituted some of the major central nodes in this network. B. In papRCC, the most important biological functions of the top deregulated miRNAs were: 1) Cancer (p = 9.4E-09-4.15E-02); 2) Endocrine System Disorders (p = 9.4E-09-4.54E-02); 3) Reproductive System Disease (p = 9.4E-09-5.11E-03); 4) Inflammatory Disease (p = 2.17E-06-2.17E-06) and 5) Inflammatory Response (p = 2.17E-06-1.65E-03). The associated functions of the major miRNA network (score = 22) were: Endocrine System Disorders, Reproductive System Disease, Cellular Development. Tumour protein p53 (TP53) and EPH receptor B6 (EPHB6) constituted some of the major central nodes in this network. C. In chRCC, the most important biological functions of the top deregulated miRNAs were: 1) Cancer (p = 8.99E-07-4.73E-02); 2) Inflammatory Disease (p = 1.64E-06-1.64E-06); 3) Inflammatory Response (p = 1.64E-06-1.19E-03); 4) Renal and Urological Disease (p = 1.64E-06-7.96E-03) and 5) Reproductive System Disease (p = 3.02E-04-2.39E-02). The associated functions of the major miRNA network (score = 22) were: Hereditary Disorder, Skeletal and Muscular Disorders, Developmental Disorder. Tumour protein p53 (TP53), B-cell CLL/lymphoma 2 (BCL2) vascular endothelial growth factor A (VEGFA) and v-myc myelocytomatosis viral oncogene homolog (MYC) constituted some of the major central nodes in this network. D. In UT-UC, the most important biological functions of the top deregulated miRNAs were: 1) Inflammatory Disease (p = 2.7E-12-3.42E-02); 2) Inflammatory Response (p = 2.7E-12-1.75E-05); 3) Renal and Urological Disease (p = 2.7E-12-3.35E-09); 4) Cancer (p = 4.67E-10-4.53E-02) and 5) Reproductive System Disease (p = 3.82E-06-4.96E-02). The associated functions of the 2 major miRNA networks were: 1) Connective Tissue Disorders, Inflammatory Disease, Inflammatory Response (score = 25); and 2) Cancer, Reproductive System Disease, Renal and Urological Disease (score = 24). Insulin, hydrogen peroxide and ribosomal protein S15 (RPS15) constituted some of the major central nodes in the first network; whereas tumour protein p53 (TP53) constituted a central node in the second network.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g012">
              https://doi.org/10.1371/journal.pone.0091646.g012</a></p></div></div>

<div id="section8" class="section toc-section"><a id="s3h" name="s3h" class="link-target" title="Target prediction and functional analysis of anti-correlated mRNA/miRNA pairs"></a>
<h3>Target prediction and functional analysis of anti-correlated mRNA/miRNA pairs</h3>
<a id="article1.body1.sec3.sec8.p1" name="article1.body1.sec3.sec8.p1" class="link-target"></a><p>From the GEO datasets with accession numbers GSE6344, GSE4125, GSE15641, GSE781 and GSE6280 we isolated the top co-up- and down-regulated genes in ccRCC. We also isolated the top co-upregulated and top co-downregulated microRNAs according to our microarray analysis in ccRCC. Using the miRWalk algorithm (<a href="http://www.ma.uni-heidelberg.de/apps/zmf/mirwalk">http://www.ma.uni-heidelberg.de/apps/zmf/mirwalk</a>) and four other prediction algorithms (TargetScan, miRanda, miRDB and Diana-microT) we identified the top anti-correlated mRNA/miRNA pairs, with a seed length (SL) of 10 nucleotides or more and a p&lt;0.001. Using this kind of analysis we concluded to the following anti-correlated mRNA/miRNA pairs: ABCA1/miR-23c (SL = 10; p = 0.0032), ATP2B4/miR-520a-5p (SL = 10; p = 0.004) and NR3C1/miR-513a-3p (SL = 11; p = 0.0009) (up-regulated mRNA/down-regulated miRNA); and TFCP2L1/miR-489 (SL = 10; p = 0.0074), TRIM10/miR-874 (SL = 10; p = 0.002), SLCA7A8/miR-210 (SL = 10; p = 0.004) (down-regulated mRNA/up-regulated miRNA).</p>
<a id="article1.body1.sec3.sec8.p2" name="article1.body1.sec3.sec8.p2" class="link-target"></a><p>The AB8/13 undifferentiated podocyte cells were incubated at 33°C at 5% CO2 and cultured in RPMI medium, supplemented with 10% Fetal Bovine Serum (FBS), 1% of 100 units/ml Penicillin/Streptomycin and 1% Insulin-Transferrin-Selenium (ITS). Endogenous expression of miR-23c, miR-520a-5p, miR-513a-3p, miR-489, miR-874 and miR-210 in undifferentiated AB8/13 podocyte cells was verified by qRT-PCR. A segment of the 3′UTR of the top co-deregulated genes was cloned into the 3′UTR of the luciferase gene in pMIR-REPORT plasmid (pMIR-REPORT-Gene of interest). Luciferase reporter plasmids and β-gal reference plasmid were co-transfected in AB8/13 podocyte cell line with the corresponding miRNA mimic, miRNA inhibitor or negative control mimics for 16 hours followed by luciferase and β-gal measurement. Co-transfection of pMIR-REPORT-TFCP2L1 with either miR-489 mimic or inhibitor resulted in significant reduction or increase in luciferase expression versus the negative control, respectively, indicating that miR-489 binds directly on the 3′UTR of TFCP2L1 (p&lt;0.001; ANOVA). In agreement, transfection of miR-489 mimics in AB8/13 cells significantly reduced the endogenous levels of TFCP2L1 protein ∼21% of total expression; while miR-489 inhibitors augmented TFCP2L1 levels by reducing the endogenously expressed miRNA level by ∼17% (p&lt;0.001; ANOVA) (<a href="#pone-0091646-g013"><strong>Figure 13</strong></a>).</p>
<a class="link-target" id="pone-0091646-g013" name="pone-0091646-g013"></a><div class="figure" data-doi="10.1371/journal.pone.0091646.g013"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0091646.g013" data-doi="info:doi/10.1371/journal.pone.0091646" data-uri="info:doi/10.1371/journal.pone.0091646.g013"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0091646.g013" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g013">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0091646.g013">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g013">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0091646.g013">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g013">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0091646.g013">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 13. </span> Western blotting.</div><p class="caption_target"><a id="article1.body1.sec3.sec8.fig1.caption1.p1" name="article1.body1.sec3.sec8.fig1.caption1.p1" class="link-target"></a><p>A. Normalized luciferase relative light units (RLUs) in AB8/13 cell lysates after transfection with sensor constructs. Co-transfection of pMIR-REPORT-TFCP2L1 with either miR-489 mimic or inhibitor resulted in significant reduction or increase in luciferase expression versus the negative control, respectively, indicating that miR-489 binds directly on the 3′UTR of TFCP2L1 (<sup>***</sup>, p&lt;0.001; <sup>**</sup>p&lt;0.05, ANOVA). B. Western blot of TFCP2L1 from AB8/13 cells after transient transfection with miR-489 miRNA LNA mimics, Inhibitors and the AllStars™ Negative Control scrambled sequence LNA. This is a representative of three experiments.The statistical analysis of western blot densitometry results, normalized against the Negative Control is also depicted. Values represent the mean ± SEM. Results illustrate the reduction of TFCP2L1 protein levels at the presence of miR-489 mimics (<sup>*</sup>, p = 0.026), while miRNA Inhibitors significantly increased TFCP2L1 levels (<sup>*</sup>, p = 0.039).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0091646.g013">
              https://doi.org/10.1371/journal.pone.0091646.g013</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>RCC subtype classification based on gene expression largely reiterates the one based on the histological appearance. However, miRNA expression profiles have shown that microRNAs are better indicators for distinguishing cancer from normal tissues, and can successfully classify even poorly differentiated tumours <a href="#pone.0091646-Lu1" class="ref-tip">[30]</a>. In the present study we analyzed the global microRNA expression of the three most common RCC subtypes along with UT-UC and we compared their miRNA profiles with the corresponding profiles from normal kidney tissue. The majority of the deregulated miRNAs was down-regulated in all RCC subtypes and UT-UC, a pattern consistent with miRNA expression profiles in human cancers <a href="#pone.0091646-Lu1" class="ref-tip">[30]</a>. Our hierarchical clustering also showed that RCC and UT-UC tumors could be distinguished perfectly from the normal kidney tissue. Given that RCCs and UT-UCs are distinct tumour entities, it was not surprising that they formed clearly separated clusters. Apart from one chRCC sample which was more heterogeneous and shared equal miRNA expression pattern with several ccRCC samples, each RCC subtype exhibited a distinct miRNA profile (<a href="#pone-0091646-g003"><strong>Figure 3</strong></a>). Furthermore, in agreement with previous reports <a href="#pone.0091646-Youssef1" class="ref-tip">[5]</a>, <a href="#pone.0091646-Fridman2" class="ref-tip">[16]</a>, <a href="#pone.0091646-Petillo1" class="ref-tip">[20]</a>, ccRCC and papRCC subtypes were more closely related when compared with chRCC, reflecting their common origin in the proximal tubule of the renal cortex.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>Using a strict filtering approach in the ROC tests, following qRT-PCR validation for the top deregulated miRNAs in each tumor entity, we concluded in a group of miRNAs that were also validated in an independent patient cohort. Therefore, we suggest that these miRNAs could be successfully used as distinguishing markers for each RCC subtype or UT-UC. We showed that miR-3648, miR-489, miR-638, miR-3656, miR-3687, miR-25-5p, miR-21-5p and miR-663b have significant potential diagnostic value for ccRCC. Moreover, miR-3648, miR-638, miR-3656, miR-3687, miR-25-5p and miR-656 were the best markers for papRCC distinction; whereas miR-3687, miR-4284 and miR-141-3p can accurately discriminate chRCC over the normal kidney. These results were also confirmed using linear regression analysis, where those miRNAs manifested both differential expression and linear behavior between RCC, UT-UC and the normal kidney. However, despite the strict filtering approach that we followed, a validation in a larger papRCC, chRCC and UT-UC sample number would strengthen the diagnostic power of the latter miRNAs. It is particularly important to note that several validated miRNAs were suggested as discriminatory markers in more than one tumour entities and especially miR-3687 could discriminate evenly all RCC subtypes and UT-UC vs. the normal kidney, as also shown from our regression analysis. This finding denotes that deregulation of miR-3687 seems to be a common manifestation during the renal carcinogenic process. Similar miRNA panels were recently used by others in order to distinguish various RCC subtypes from the normal kidney; however the results are not always consistent among them <a href="#pone.0091646-Chow1" class="ref-tip">[12]</a>, <a href="#pone.0091646-White1" class="ref-tip">[13]</a>, <a href="#pone.0091646-Fridman2" class="ref-tip">[16]</a>, <a href="#pone.0091646-Jung1" class="ref-tip">[17]</a>, <a href="#pone.0091646-Osanto1" class="ref-tip">[19]</a>, <a href="#pone.0091646-Petillo1" class="ref-tip">[20]</a>, <a href="#pone.0091646-Huang1" class="ref-tip">[23]</a>, <a href="#pone.0091646-Juan1" class="ref-tip">[24]</a>, <a href="#pone.0091646-Nakada2" class="ref-tip">[31]</a>, <a href="#pone.0091646-Weng1" class="ref-tip">[32]</a>. This can be attributed to various factors, such as different microarray platforms used, different normalization methods followed, different sample number, etc. Spector et al. just recently developed a similar miRNA-based diagnostic assay for the classification of ccRCC, papRCC, chRCC and benign oncocytoma. Following a similar approach, they validated a classifier set of 24 miRNAs to an independent cohort and they distinguished miR-210, miR-195, miR-182, miR-31-3p and miR-551b, among others <a href="#pone.0091646-Spector1" class="ref-tip">[33]</a>. Interestingly enough, these miRNAs were also included among the top 30 deregulated miRNAs in our study (<strong><a href="#pone.0091646.s007">Table S2</a></strong>). Apart from the recent study of Izquierdo et al. <a href="#pone.0091646-Izquierdo1" class="ref-tip">[34]</a>, where the authors investigated miRNA expression in progressing and non-progressing UT-UC patients, this is the first report to our knowledge suggesting a group of miRNAs able to discriminate UT-UC from the normal kidney. Despite the small UT-UC sample number examined, our data show that miR-3648, miR-638, miR-3656 and miR-3687 were the best molecular markers for distinction of UT-UC over the normal kidney (<a href="#pone-0091646-t003"><strong>Table 3</strong></a>). We also compared the data between UT-UC and RCC subtypes, and identified miRNAs that were specific for each tumour entity. Among them 16 miRNAs were specifically down-regulated and 5 miRNAs were specifically up-regulated in UT-UC (<strong><a href="#pone.0091646.s008">Table S3</a></strong>). Approximately 50% of all annotated human miRNA genes are located at fragile sites or areas of the genome that are associated with cancer which are prone to breakage and rearrangement in cancer cells <a href="#pone.0091646-Calin1" class="ref-tip">[35]</a>, <a href="#pone.0091646-Girgis1" class="ref-tip">[36]</a>, <a href="#pone.0091646-Bratslavsky1" class="ref-tip">[37]</a>, <a href="#pone.0091646-Klatte1" class="ref-tip">[38]</a>, <a href="#pone.0091646-Kovacs1" class="ref-tip">[39]</a>, <a href="#pone.0091646-Speicher1" class="ref-tip">[40]</a>, <a href="#pone.0091646-Storkel1" class="ref-tip">[41]</a>, <a href="#pone.0091646-vandenBerg1" class="ref-tip">[42]</a>, <a href="#pone.0091646-Velickovic1" class="ref-tip">[43]</a>, <a href="#pone.0091646-Yoshimoto1" class="ref-tip">[44]</a>. Our data showed a partial matching pattern between the reported cytogenetic changes and miRNA deregulation in RCC and UT-UC. This is in agreement with the recent findings of Chow et al. who demonstrated that chromosomal aberrations in ccRCC can control the deregulation of miRNAs <a href="#pone.0091646-Chow1" class="ref-tip">[12]</a>.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>It was not surprising that IPA revealed that the most significant biological functions of the deregulated miRNAs in RCC involved cancer and renal and urological disease. Furthermore, the systemic inflammatory response has been revealed to contribute to tumorigenesis in the last two decades. Key inflammatory pathways, such as the NF-κB <a href="#pone.0091646-ThapaRJ1" class="ref-tip">[45]</a>, AP-1 <a href="#pone.0091646-Sato1" class="ref-tip">[46]</a> and STAT3 <a href="#pone.0091646-Zaravinos3" class="ref-tip">[47]</a> pathways, are implicated in the proliferation, transformation, survival, invasion, angiogenesis, metastasis, chemoresistance and radioresistance of cancer <a href="#pone.0091646-Aggarwal1" class="ref-tip">[48]</a>. Systemic inflammatory response, represented by elevated C-reactive protein (CRP), platelet count (PC) and erythrocyte sedimentation rate (ESR) was recently shown to predict poor survival in patients with RCC <a href="#pone.0091646-Wu1" class="ref-tip">[49]</a>. The tumor suppressor p53 (TP53) and v-myc avian myelocytomatosis viral oncogene homolog (MYC) constituted central nodes in most miRNA networks. TP53 plays a central role in tumor prevention. As a transcription factor, it mainly exerts its function through transcription regulation of its target genes to initiate various cellular responses. Recent studies have demonstrated that TP53 cross-talks with miRNAs at multiple levels; from inducing their transcription expression and promoting their maturation, to being negatively regulated by them <a href="#pone.0091646-Feng1" class="ref-tip">[50]</a>. MYC expression was recently shown to be significantly upregulated in RCC <a href="#pone.0091646-Yamada1" class="ref-tip">[51]</a> and its implication with miRNAs in RCC was also recently reported <a href="#pone.0091646-Wach1" class="ref-tip">[52]</a>. As a transcription factor, it was recently shown to regulate genes involved in cell growth, including those regulating ribosome and mitochondrial biogenesis and intermediary metabolism; and glycolytic genes, enhancing glycolysis, glutamine uptake and metabolism, and lactate production <a href="#pone.0091646-Teicher1" class="ref-tip">[53]</a>.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Finally, we isolated the top co-upregulated and top co-downregulated mRNA transcripts from 5 independent GEO ccRCC datasets, and combining our miRNA results by a computational approach, we concluded to the six most anti-correlated mRNA/miRNA pairs the association of which was functionally investigated. We showed that the developmental transcription factor TFCP2L1 (transcription factor CP2-like 1) whose loss was recently suggested to play a role in the loss of epithelial differentiation and function in ccRCC, is a true target of miR-489, which was among the ccRCC-specific miRNAs in our study <a href="#pone.0091646-Tun1" class="ref-tip">[54]</a>.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>The current study adds a number of novel deregulated miRNAs accompanied by the molecular pathways in which they are involved mainly in ccRCC. These miRNAs accurately discriminate ccRCC from the normal kidney and can provide the basis for a specific targeted therapy. Due to the small sample number of the papRCC, chRCC and UT-UC samples that were used in the present study, the respective miRNAs should be carefully taken into consideration.</p>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0091646"></div><div class="supplementary-material"><a name="pone.0091646.s001" id="pone.0091646.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Pair-wise Hierarchical Clistering (HCl).</strong> Pair-tissue-wise unsupervised two-way hierarchical clustering with Euclidian distance. The log<sub>2</sub> fold change in each RCC subtype and UT-UC versus the normal kidney tissue was used to construct the heat map. miRNA profiling accurately discriminated between ccRCC and papRCC (A), ccRCC and chRCC (B), chRCC and papRCC (C), ccRCC and UT-UC (D), papRCC and UT-UC (E), and chRCC and UT-UC (F). ccRCC, clear cell renal cell carcinoma; papRCC, papillary renal cell carcinoma; chRCC, chromophobe renal cell carcinoma; UT-UC, upper tract urothelial carcinoma. Red and blue colours show significant up- or down-regulation of each miRNA in the tumour versus the normal kidney, respectively.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s001">https://doi.org/10.1371/journal.pone.0091646.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s002" id="pone.0091646.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s002">Figure S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Regression analysis.</strong> Regression analysis of top up-regulated (A) and top down-regulated (B) miRNAs. miRNAs were tested with one-way ANOVA and further linear regression was used to fit their expression into a linear model. Left columns represent the ANOVA results. Right columns represent the linear regression results after sorting of the median values of the respective miRNAs in ascending order.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s002">https://doi.org/10.1371/journal.pone.0091646.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s003" id="pone.0091646.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s003">Figure S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Regression analysis.</strong> Regression analysis of top up-regulated (A) and top down-regulated (B) miRNAs. miRNAs were tested with one-way ANOVA and further linear regression was used to fit their expression into a linear model. Left columns represent the ANOVA results. Right columns represent the linear regression results after sorting of the median values of the respective miRNAs in ascending order.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s003">https://doi.org/10.1371/journal.pone.0091646.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s004" id="pone.0091646.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s004">Figure S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Regression analysis.</strong> Regression analysis of the top deregulated miRNAs. One-way ANOVA (left columns) along with linear regression (right columns) of the RCC grades with respect to miRNA expression was used to fit miRNA expression into a linear model. Each regression consists of two curves: one of the form y = ax+b (red) and one of the form y = ax3+bx2+cx+d (blue). All results presented obtained a p&lt;0.05. The median expression values of the miRNAs were used. Interestingly, miR-148b-5p, miR-1910 and miR-200b-5p, miR-210 and miR-3656, presented linear behavior in a descending order of the RCC grade; whereas miR-3687 and miR-654-5p manifested linear behavior in ascending order of the RCC grade.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s004">https://doi.org/10.1371/journal.pone.0091646.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s005" id="pone.0091646.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s005">Figure S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Multiple sequence alignments (MSA).</strong> Representative MSA and phylogenetic trees of miR-489 (A) and miR-25-5p (B). The majority of the DE miRNAs (78.45%) were highly conserved among species. Specifically, 70.68% of the up-regulated and 81.22% of the down-regulated miRNAs were phylogenetically conserved among species.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s005">https://doi.org/10.1371/journal.pone.0091646.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s006" id="pone.0091646.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s006">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>The table lists the primer sequences that were designed to introduce a SpeI and a HindIII restriction enzyme sites to be cloned into the pMiR-REPORT luciferase vector.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s006">https://doi.org/10.1371/journal.pone.0091646.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s007" id="pone.0091646.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s007">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Top deregulated miRNAs in ccRCC, papRCC, chRCC and UT-UC vs. the normal kidney tissue.</strong></p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s007">https://doi.org/10.1371/journal.pone.0091646.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s008" id="pone.0091646.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s008">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Co-deregulated and tissue specific miRNAs in ccRCC, papRCC, chRCC and UT-UC.</strong> Co-upregulated and tissue specific miRNAs in RCC subtypes (A). Co-downregulated and tissue specific miRNAs in RCC subtypes (B). Co-upregulated and tissue specific miRNAs in RCC and UT-UC (C). Co-downregulated and tissue specific miRNAs in RCC and UT-UC (D).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s008">https://doi.org/10.1371/journal.pone.0091646.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0091646.s009" id="pone.0091646.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0091646.s009">Table S4. </a></h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Reported chromosomal abnormalities associated with ccRCC, papRCC, chRCC and UT-UC, as well as some of the well-studied genes located within those aberrated regions that are involved in the pathogenesis of the tumour subtypes.</strong> Many of the significantly deregulated miRNAs are located within cytogenetic regions that are significantly aberrated in the different subtypes. In the reported chromosomal aberrations the minus sign denotes a loss and the plus sign denotes a gain in chromosomal material. Red colour (↑) denotes elevated miRNA expression and blue colour (↓) denotes reduced miRNA expression.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0091646.s009">https://doi.org/10.1371/journal.pone.0091646.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The authors would like to thank Revekka Papacharalambous for providing technical assistance. Part of this work was presented orally during the FEBS 13<sup>th</sup> Young Scientists' Forum (YSF), 3–6 July 2013, St. Petersburg, Russia, and as a poster during the FEBS Advanced Lecture Course, Molecular Mechanisms in Signal Transduction and Cancer, August 16–24, 2013 Spetses, Greece.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: AZ. Performed the experiments: AZ GL. Analyzed the data: AZ GL. Contributed reagents/materials/analysis tools: AZ GL NM PK GP KG DD CD. Wrote the paper: AZ. Read and approved the manuscript: GL DD CD.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0091646-MeloniEhrig1" id="pone.0091646-MeloniEhrig1" class="link-target"></a>Meloni-Ehrig AM (2002) Renal cancer: cytogenetic and molecular genetic aspects. Am J Med Genet 115: 164–172. <ul class="reflinks"><li><a href="#" data-author="Meloni-Ehrig" data-cit="Meloni-EhrigAM%20%282002%29%20Renal%20cancer%3A%20cytogenetic%20and%20molecular%20genetic%20aspects.%20Am%20J%20Med%20Genet%20115%3A%20164%E2%80%93172." data-title="Renal%20cancer%3A%20cytogenetic%20and%20molecular%20genetic%20aspects" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Renal+cancer%3A+cytogenetic+and+molecular+genetic+aspects+Meloni-Ehrig+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0091646-Ficarra1" id="pone.0091646-Ficarra1" class="link-target"></a>Ficarra V, Martignoni G, Galfano A, Novara G, Gobbo S, et al. (2006) Prognostic role of the histologic subtypes of renal cell carcinoma after slide revision. Eur Urol 50: 786–793. <ul class="reflinks"><li><a href="#" data-author="Ficarra" data-cit="FicarraV%2C%20MartignoniG%2C%20GalfanoA%2C%20NovaraG%2C%20GobboS%2C%20et%20al.%20%282006%29%20Prognostic%20role%20of%20the%20histologic%20subtypes%20of%20renal%20cell%20carcinoma%20after%20slide%20revision.%20Eur%20Urol%2050%3A%20786%E2%80%93793." data-title="Prognostic%20role%20of%20the%20histologic%20subtypes%20of%20renal%20cell%20carcinoma%20after%20slide%20revision" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prognostic+role+of+the+histologic+subtypes+of+renal+cell+carcinoma+after+slide+revision+Ficarra+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0091646-LopezBeltran1" id="pone.0091646-LopezBeltran1" class="link-target"></a>Lopez-Beltran A, Kirkali Z, Cheng L, Egevad L, Regueiro JC, et al. (2008) Targeted therapies and biological modifiers in urologic tumors: pathobiology and clinical implications. Semin Diagn Pathol 25: 232–244. <ul class="reflinks"><li><a href="#" data-author="Lopez-Beltran" data-cit="Lopez-BeltranA%2C%20KirkaliZ%2C%20ChengL%2C%20EgevadL%2C%20RegueiroJC%2C%20et%20al.%20%282008%29%20Targeted%20therapies%20and%20biological%20modifiers%20in%20urologic%20tumors%3A%20pathobiology%20and%20clinical%20implications.%20Semin%20Diagn%20Pathol%2025%3A%20232%E2%80%93244." data-title="Targeted%20therapies%20and%20biological%20modifiers%20in%20urologic%20tumors%3A%20pathobiology%20and%20clinical%20implications" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Targeted+therapies+and+biological+modifiers+in+urologic+tumors%3A+pathobiology+and+clinical+implications+Lopez-Beltran+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0091646-Taziel1" id="pone.0091646-Taziel1" class="link-target"></a>Tazi el M, Essadi I, Tazi MF, Ahellal Y, M'Rabti H, et al. (2011) Advanced treatments in non-clear renal cell carcinoma. Urol J 8: 1–11. <ul class="reflinks"><li><a href="#" data-author="Tazi%20el" data-cit="Tazi%20elM%2C%20EssadiI%2C%20TaziMF%2C%20AhellalY%2C%20M%27RabtiH%2C%20et%20al.%20%282011%29%20Advanced%20treatments%20in%20non-clear%20renal%20cell%20carcinoma.%20Urol%20J%208%3A%201%E2%80%9311." data-title="Advanced%20treatments%20in%20non-clear%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Advanced+treatments+in+non-clear+renal+cell+carcinoma+Tazi+el+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0091646-Youssef1" id="pone.0091646-Youssef1" class="link-target"></a>Youssef YM, White NMA, Grigull J, Krizova A, Samy C, et al. (2011) Accurate molecular classification of kidney cancer subtypes using microRNA signature. Eur Urol 59: 721–730. <ul class="reflinks"><li><a href="#" data-author="Youssef" data-cit="YoussefYM%2C%20WhiteNMA%2C%20GrigullJ%2C%20KrizovaA%2C%20SamyC%2C%20et%20al.%20%282011%29%20Accurate%20molecular%20classification%20of%20kidney%20cancer%20subtypes%20using%20microRNA%20signature.%20Eur%20Urol%2059%3A%20721%E2%80%93730." data-title="Accurate%20molecular%20classification%20of%20kidney%20cancer%20subtypes%20using%20microRNA%20signature" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Accurate+molecular+classification+of+kidney+cancer+subtypes+using+microRNA+signature+Youssef+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0091646-Fridman1" id="pone.0091646-Fridman1" class="link-target"></a>Fridman E, Dotan Z, Barshack I, David MB, Dov A, et al. (2010) Accurate molecular classification of renal tumors using microRNA expression. J Mol Diagn 12: 687–696. <ul class="reflinks"><li><a href="#" data-author="Fridman" data-cit="FridmanE%2C%20DotanZ%2C%20BarshackI%2C%20DavidMB%2C%20DovA%2C%20et%20al.%20%282010%29%20Accurate%20molecular%20classification%20of%20renal%20tumors%20using%20microRNA%20expression.%20J%20Mol%20Diagn%2012%3A%20687%E2%80%93696." data-title="Accurate%20molecular%20classification%20of%20renal%20tumors%20using%20microRNA%20expression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Accurate+molecular+classification+of+renal+tumors+using+microRNA+expression+Fridman+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0091646-Higgins1" id="pone.0091646-Higgins1" class="link-target"></a>Higgins JP, Kaygusuz G, Wang L, Montgomery K, Mason V, et al. (2007) Placental S100 (S100P) and GATA3: markers for transitional epithelium and urothelial carcinoma discovered by complementary DNA microarray. Am J Surg Pathol 31: 673–680. <ul class="reflinks"><li><a href="#" data-author="Higgins" data-cit="HigginsJP%2C%20KaygusuzG%2C%20WangL%2C%20MontgomeryK%2C%20MasonV%2C%20et%20al.%20%282007%29%20Placental%20S100%20%28S100P%29%20and%20GATA3%3A%20markers%20for%20transitional%20epithelium%20and%20urothelial%20carcinoma%20discovered%20by%20complementary%20DNA%20microarray.%20Am%20J%20Surg%20Pathol%2031%3A%20673%E2%80%93680." data-title="Placental%20S100%20%28S100P%29%20and%20GATA3%3A%20markers%20for%20transitional%20epithelium%20and%20urothelial%20carcinoma%20discovered%20by%20complementary%20DNA%20microarray" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Placental+S100+%28S100P%29+and+GATA3%3A+markers+for+transitional+epithelium+and+urothelial+carcinoma+discovered+by+complementary+DNA+microarray+Higgins+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0091646-Roupret1" id="pone.0091646-Roupret1" class="link-target"></a>Roupret M, Zigeuner R, Palou J, Boehle A, Kaasinen E, et al. (2011) European guidelines for the diagnosis and management of upper urinary tract urothelial cell carcinomas: 2011 update. Eur Urol 59: 584–594. <ul class="reflinks"><li><a href="#" data-author="Roupret" data-cit="RoupretM%2C%20ZigeunerR%2C%20PalouJ%2C%20BoehleA%2C%20KaasinenE%2C%20et%20al.%20%282011%29%20European%20guidelines%20for%20the%20diagnosis%20and%20management%20of%20upper%20urinary%20tract%20urothelial%20cell%20carcinomas%3A%202011%20update.%20Eur%20Urol%2059%3A%20584%E2%80%93594." data-title="European%20guidelines%20for%20the%20diagnosis%20and%20management%20of%20upper%20urinary%20tract%20urothelial%20cell%20carcinomas%3A%202011%20update" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=European+guidelines+for+the+diagnosis+and+management+of+upper+urinary+tract+urothelial+cell+carcinomas%3A+2011+update+Roupret+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0091646-Kirkali1" id="pone.0091646-Kirkali1" class="link-target"></a>Kirkali Z, Tuzel E (2003) Transitional cell carcinoma of the ureter and renal pelvis. Crit Rev Oncol Hematol 47: 155–169. <ul class="reflinks"><li><a href="#" data-author="Kirkali" data-cit="KirkaliZ%2C%20TuzelE%20%282003%29%20Transitional%20cell%20carcinoma%20of%20the%20ureter%20and%20renal%20pelvis.%20Crit%20Rev%20Oncol%20Hematol%2047%3A%20155%E2%80%93169." data-title="Transitional%20cell%20carcinoma%20of%20the%20ureter%20and%20renal%20pelvis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transitional+cell+carcinoma+of+the+ureter+and+renal+pelvis+Kirkali+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0091646-Garzon1" id="pone.0091646-Garzon1" class="link-target"></a>Garzon R, Pichiorri F, Palumbo T, Visentini M, Aqeilan R, et al. (2007) MicroRNA gene expression during retinoic acid-induced differentiation of human acute promyelocytic leukemia. Oncogene 26: 4148–4157. <ul class="reflinks"><li><a href="#" data-author="Garzon" data-cit="GarzonR%2C%20PichiorriF%2C%20PalumboT%2C%20VisentiniM%2C%20AqeilanR%2C%20et%20al.%20%282007%29%20MicroRNA%20gene%20expression%20during%20retinoic%20acid-induced%20differentiation%20of%20human%20acute%20promyelocytic%20leukemia.%20Oncogene%2026%3A%204148%E2%80%934157." data-title="MicroRNA%20gene%20expression%20during%20retinoic%20acid-induced%20differentiation%20of%20human%20acute%20promyelocytic%20leukemia" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+gene+expression+during+retinoic+acid-induced+differentiation+of+human+acute+promyelocytic+leukemia+Garzon+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0091646-EsquelaKerscher1" id="pone.0091646-EsquelaKerscher1" class="link-target"></a>Esquela-Kerscher A, Slack FJ (2006) Oncomirs - microRNAs with a role in cancer. Nat Rev Cancer 6: 259–269. <ul class="reflinks"><li><a href="#" data-author="Esquela-Kerscher" data-cit="Esquela-KerscherA%2C%20SlackFJ%20%282006%29%20Oncomirs%20-%20microRNAs%20with%20a%20role%20in%20cancer.%20Nat%20Rev%20Cancer%206%3A%20259%E2%80%93269." data-title="Oncomirs%20-%20microRNAs%20with%20a%20role%20in%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Oncomirs+-+microRNAs+with+a+role+in+cancer+Esquela-Kerscher+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0091646-Chow1" id="pone.0091646-Chow1" class="link-target"></a>Chow TF, Youssef YM, Lianidou E, Romaschin AD, Honey RJ, et al. (2010) Differential expression profiling of microRNAs and their potential involvement in renal cell carcinoma pathogenesis. Clin Biochem 43: 150–158. <ul class="reflinks"><li><a href="#" data-author="Chow" data-cit="ChowTF%2C%20YoussefYM%2C%20LianidouE%2C%20RomaschinAD%2C%20HoneyRJ%2C%20et%20al.%20%282010%29%20Differential%20expression%20profiling%20of%20microRNAs%20and%20their%20potential%20involvement%20in%20renal%20cell%20carcinoma%20pathogenesis.%20Clin%20Biochem%2043%3A%20150%E2%80%93158." data-title="Differential%20expression%20profiling%20of%20microRNAs%20and%20their%20potential%20involvement%20in%20renal%20cell%20carcinoma%20pathogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+expression+profiling+of+microRNAs+and+their+potential+involvement+in+renal+cell+carcinoma+pathogenesis+Chow+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0091646-White1" id="pone.0091646-White1" class="link-target"></a>White NM, Bao TT, Grigull J, Youssef YM, Girgis A, et al. (2011) miRNA profiling for clear cell renal cell carcinoma: biomarker discovery and identification of potential controls and consequences of miRNA dysregulation. J Urol 186: 1077–1083. <ul class="reflinks"><li><a href="#" data-author="White" data-cit="WhiteNM%2C%20BaoTT%2C%20GrigullJ%2C%20YoussefYM%2C%20GirgisA%2C%20et%20al.%20%282011%29%20miRNA%20profiling%20for%20clear%20cell%20renal%20cell%20carcinoma%3A%20biomarker%20discovery%20and%20identification%20of%20potential%20controls%20and%20consequences%20of%20miRNA%20dysregulation.%20J%20Urol%20186%3A%201077%E2%80%931083." data-title="miRNA%20profiling%20for%20clear%20cell%20renal%20cell%20carcinoma%3A%20biomarker%20discovery%20and%20identification%20of%20potential%20controls%20and%20consequences%20of%20miRNA%20dysregulation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miRNA+profiling+for+clear+cell+renal+cell+carcinoma%3A+biomarker+discovery+and+identification+of+potential+controls+and+consequences+of+miRNA+dysregulation+White+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0091646-Zaravinos1" id="pone.0091646-Zaravinos1" class="link-target"></a>Zaravinos A, Radojicic J, Lambrou GI, Volanis D, Delakas D, et al. (2012) Expression of miRNAs involved in angiogenesis, tumor cell proliferation, tumor suppressor inhibition, epithelial-mesenchymal transition and activation of metastasis in bladder cancer. J Urol 188: 615–623. <ul class="reflinks"><li><a href="#" data-author="Zaravinos" data-cit="ZaravinosA%2C%20RadojicicJ%2C%20LambrouGI%2C%20VolanisD%2C%20DelakasD%2C%20et%20al.%20%282012%29%20Expression%20of%20miRNAs%20involved%20in%20angiogenesis%2C%20tumor%20cell%20proliferation%2C%20tumor%20suppressor%20inhibition%2C%20epithelial-mesenchymal%20transition%20and%20activation%20of%20metastasis%20in%20bladder%20cancer.%20J%20Urol%20188%3A%20615%E2%80%93623." data-title="Expression%20of%20miRNAs%20involved%20in%20angiogenesis%2C%20tumor%20cell%20proliferation%2C%20tumor%20suppressor%20inhibition%2C%20epithelial-mesenchymal%20transition%20and%20activation%20of%20metastasis%20in%20bladder%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+of+miRNAs+involved+in+angiogenesis%2C+tumor+cell+proliferation%2C+tumor+suppressor+inhibition%2C+epithelial-mesenchymal+transition+and+activation+of+metastasis+in+bladder+cancer+Zaravinos+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0091646-Chow2" id="pone.0091646-Chow2" class="link-target"></a>Chow TF, Youssef YM, Lianidou E, Romaschin AD, Honey RJ, et al. (2010) Differential expression profiling of microRNAs and their potential involvement in renal cell carcinoma pathogenesis. Clin Biochem 43: 150–158. <ul class="reflinks"><li><a href="#" data-author="Chow" data-cit="ChowTF%2C%20YoussefYM%2C%20LianidouE%2C%20RomaschinAD%2C%20HoneyRJ%2C%20et%20al.%20%282010%29%20Differential%20expression%20profiling%20of%20microRNAs%20and%20their%20potential%20involvement%20in%20renal%20cell%20carcinoma%20pathogenesis.%20Clin%20Biochem%2043%3A%20150%E2%80%93158." data-title="Differential%20expression%20profiling%20of%20microRNAs%20and%20their%20potential%20involvement%20in%20renal%20cell%20carcinoma%20pathogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+expression+profiling+of+microRNAs+and+their+potential+involvement+in+renal+cell+carcinoma+pathogenesis+Chow+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0091646-Fridman2" id="pone.0091646-Fridman2" class="link-target"></a>Fridman E, Dotan Z, Barshack I, Ben David M, Dov A, et al. (2010) Accurate molecular classification of renal tumors using microRNA expression. J Mol Diagn 12: 687–696. <ul class="reflinks"><li><a href="#" data-author="Fridman" data-cit="FridmanE%2C%20DotanZ%2C%20BarshackI%2C%20Ben%20DavidM%2C%20DovA%2C%20et%20al.%20%282010%29%20Accurate%20molecular%20classification%20of%20renal%20tumors%20using%20microRNA%20expression.%20J%20Mol%20Diagn%2012%3A%20687%E2%80%93696." data-title="Accurate%20molecular%20classification%20of%20renal%20tumors%20using%20microRNA%20expression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Accurate+molecular+classification+of+renal+tumors+using+microRNA+expression+Fridman+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0091646-Jung1" id="pone.0091646-Jung1" class="link-target"></a>Jung M, Mollenkopf HJ, Grimm C, Wagner I, Albrecht M, et al. (2009) MicroRNA profiling of clear cell renal cell cancer identifies a robust signature to define renal malignancy. J Cell Mol Med 13: 3918–3928. <ul class="reflinks"><li><a href="#" data-author="Jung" data-cit="JungM%2C%20MollenkopfHJ%2C%20GrimmC%2C%20WagnerI%2C%20AlbrechtM%2C%20et%20al.%20%282009%29%20MicroRNA%20profiling%20of%20clear%20cell%20renal%20cell%20cancer%20identifies%20a%20robust%20signature%20to%20define%20renal%20malignancy.%20J%20Cell%20Mol%20Med%2013%3A%203918%E2%80%933928." data-title="MicroRNA%20profiling%20of%20clear%20cell%20renal%20cell%20cancer%20identifies%20a%20robust%20signature%20to%20define%20renal%20malignancy" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+profiling+of+clear+cell+renal+cell+cancer+identifies+a+robust+signature+to+define+renal+malignancy+Jung+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0091646-Nakada1" id="pone.0091646-Nakada1" class="link-target"></a>Nakada C, Matsuura K, Tsukamoto Y, Tanigawa M, Yoshimoto T, et al. (2008) Genome-wide microRNA expression profiling in renal cell carcinoma: significant down-regulation of miR-141 and miR-200c. J Pathol 216: 418–427. <ul class="reflinks"><li><a href="#" data-author="Nakada" data-cit="NakadaC%2C%20MatsuuraK%2C%20TsukamotoY%2C%20TanigawaM%2C%20YoshimotoT%2C%20et%20al.%20%282008%29%20Genome-wide%20microRNA%20expression%20profiling%20in%20renal%20cell%20carcinoma%3A%20significant%20down-regulation%20of%20miR-141%20and%20miR-200c.%20J%20Pathol%20216%3A%20418%E2%80%93427." data-title="Genome-wide%20microRNA%20expression%20profiling%20in%20renal%20cell%20carcinoma%3A%20significant%20down-regulation%20of%20miR-141%20and%20miR-200c" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+microRNA+expression+profiling+in+renal+cell+carcinoma%3A+significant+down-regulation+of+miR-141+and+miR-200c+Nakada+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0091646-Osanto1" id="pone.0091646-Osanto1" class="link-target"></a>Osanto S, Qin Y, Buermans HP, Berkers J, Lerut E, et al. (2012) Genome-wide microRNA expression analysis of clear cell renal cell carcinoma by next generation deep sequencing. PLoS One 7: e38298. <ul class="reflinks"><li><a href="#" data-author="Osanto" data-cit="OsantoS%2C%20QinY%2C%20BuermansHP%2C%20BerkersJ%2C%20LerutE%2C%20et%20al.%20%282012%29%20Genome-wide%20microRNA%20expression%20analysis%20of%20clear%20cell%20renal%20cell%20carcinoma%20by%20next%20generation%20deep%20sequencing.%20PLoS%20One%207%3A%20e38298." data-title="Genome-wide%20microRNA%20expression%20analysis%20of%20clear%20cell%20renal%20cell%20carcinoma%20by%20next%20generation%20deep%20sequencing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+microRNA+expression+analysis+of+clear+cell+renal+cell+carcinoma+by+next+generation+deep+sequencing+Osanto+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0091646-Petillo1" id="pone.0091646-Petillo1" class="link-target"></a>Petillo D, Kort EJ, Anema J, Furge KA, Yang XJ, et al. (2009) MicroRNA profiling of human kidney cancer subtypes. Int J Oncol 35: 109–114. <ul class="reflinks"><li><a href="#" data-author="Petillo" data-cit="PetilloD%2C%20KortEJ%2C%20AnemaJ%2C%20FurgeKA%2C%20YangXJ%2C%20et%20al.%20%282009%29%20MicroRNA%20profiling%20of%20human%20kidney%20cancer%20subtypes.%20Int%20J%20Oncol%2035%3A%20109%E2%80%93114." data-title="MicroRNA%20profiling%20of%20human%20kidney%20cancer%20subtypes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+profiling+of+human+kidney+cancer+subtypes+Petillo+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0091646-Radojicic1" id="pone.0091646-Radojicic1" class="link-target"></a>Radojicic J, Zaravinos A, Vrekoussis T, Kafousi M, Spandidos DA, et al. (2011) MicroRNA expression analysis in triple-negative (ER, PR and Her2/neu) breast cancer. Cell Cycle 10: 507–517. <ul class="reflinks"><li><a href="#" data-author="Radojicic" data-cit="RadojicicJ%2C%20ZaravinosA%2C%20VrekoussisT%2C%20KafousiM%2C%20SpandidosDA%2C%20et%20al.%20%282011%29%20MicroRNA%20expression%20analysis%20in%20triple-negative%20%28ER%2C%20PR%20and%20Her2%2Fneu%29%20breast%20cancer.%20Cell%20Cycle%2010%3A%20507%E2%80%93517." data-title="MicroRNA%20expression%20analysis%20in%20triple-negative%20%28ER%2C%20PR%20and%20Her2%2Fneu%29%20breast%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+expression+analysis+in+triple-negative+%28ER%2C+PR+and+Her2%2Fneu%29+breast+cancer+Radojicic+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0091646-Gottardo1" id="pone.0091646-Gottardo1" class="link-target"></a>Gottardo F, Liu CG, Ferracin M, Calin GA, Fassan M, et al. (2007) Micro-RNA profiling in kidney and bladder cancers. Urol Oncol 25: 387–392. <ul class="reflinks"><li><a href="#" data-author="Gottardo" data-cit="GottardoF%2C%20LiuCG%2C%20FerracinM%2C%20CalinGA%2C%20FassanM%2C%20et%20al.%20%282007%29%20Micro-RNA%20profiling%20in%20kidney%20and%20bladder%20cancers.%20Urol%20Oncol%2025%3A%20387%E2%80%93392." data-title="Micro-RNA%20profiling%20in%20kidney%20and%20bladder%20cancers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Micro-RNA+profiling+in+kidney+and+bladder+cancers+Gottardo+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0091646-Huang1" id="pone.0091646-Huang1" class="link-target"></a>Huang Y, Dai Y, Yang J, Chen T, Yin Y, et al. (2009) Microarray analysis of microRNA expression in renal clear cell carcinoma. Eur J Surg Oncol 35: 1119–1123. <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangY%2C%20DaiY%2C%20YangJ%2C%20ChenT%2C%20YinY%2C%20et%20al.%20%282009%29%20Microarray%20analysis%20of%20microRNA%20expression%20in%20renal%20clear%20cell%20carcinoma.%20Eur%20J%20Surg%20Oncol%2035%3A%201119%E2%80%931123." data-title="Microarray%20analysis%20of%20microRNA%20expression%20in%20renal%20clear%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Microarray+analysis+of+microRNA+expression+in+renal+clear+cell+carcinoma+Huang+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0091646-Juan1" id="pone.0091646-Juan1" class="link-target"></a>Juan D, Alexe G, Antes T, Liu H, Madabhushi A, et al. (2010) Identification of a microRNA panel for clear-cell kidney cancer. Urology 75: 835–841. <ul class="reflinks"><li><a href="#" data-author="Juan" data-cit="JuanD%2C%20AlexeG%2C%20AntesT%2C%20LiuH%2C%20MadabhushiA%2C%20et%20al.%20%282010%29%20Identification%20of%20a%20microRNA%20panel%20for%20clear-cell%20kidney%20cancer.%20Urology%2075%3A%20835%E2%80%93841." data-title="Identification%20of%20a%20microRNA%20panel%20for%20clear-cell%20kidney%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+a+microRNA+panel+for+clear-cell+kidney+cancer+Juan+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0091646-Yi1" id="pone.0091646-Yi1" class="link-target"></a>Yi Z, Fu Y, Zhao S, Zhang X, Ma C (2010) Differential expression of miRNA patterns in renal cell carcinoma and nontumorous tissues. J Cancer Res Clin Oncol 136: 855–862. <ul class="reflinks"><li><a href="#" data-author="Yi" data-cit="YiZ%2C%20FuY%2C%20ZhaoS%2C%20ZhangX%2C%20MaC%20%282010%29%20Differential%20expression%20of%20miRNA%20patterns%20in%20renal%20cell%20carcinoma%20and%20nontumorous%20tissues.%20J%20Cancer%20Res%20Clin%20Oncol%20136%3A%20855%E2%80%93862." data-title="Differential%20expression%20of%20miRNA%20patterns%20in%20renal%20cell%20carcinoma%20and%20nontumorous%20tissues" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+expression+of+miRNA+patterns+in+renal+cell+carcinoma+and+nontumorous+tissues+Yi+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0091646-Sun1" id="pone.0091646-Sun1" class="link-target"></a>Sun Y, Koo S, White N, Peralta E, Esau C, et al. (2004) Development of a micro-array to detect human and mouse microRNAs and characterization of expression in human organs. Nucleic Acids Res 32: e188. <ul class="reflinks"><li><a href="#" data-author="Sun" data-cit="SunY%2C%20KooS%2C%20WhiteN%2C%20PeraltaE%2C%20EsauC%2C%20et%20al.%20%282004%29%20Development%20of%20a%20micro-array%20to%20detect%20human%20and%20mouse%20microRNAs%20and%20characterization%20of%20expression%20in%20human%20organs.%20Nucleic%20Acids%20Res%2032%3A%20e188." data-title="Development%20of%20a%20micro-array%20to%20detect%20human%20and%20mouse%20microRNAs%20and%20characterization%20of%20expression%20in%20human%20organs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Development+of+a+micro-array+to+detect+human+and+mouse+microRNAs+and+characterization+of+expression+in+human+organs+Sun+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0091646-Zaravinos2" id="pone.0091646-Zaravinos2" class="link-target"></a>Zaravinos A, Bizakis J, Soufla G, Sourvinos G, Spandidos DA (2007) Mutations and differential expression of the ras family genes in human nasal polyposis. Int J Oncol 31: 1051–1059. <ul class="reflinks"><li><a href="#" data-author="Zaravinos" data-cit="ZaravinosA%2C%20BizakisJ%2C%20SouflaG%2C%20SourvinosG%2C%20SpandidosDA%20%282007%29%20Mutations%20and%20differential%20expression%20of%20the%20ras%20family%20genes%20in%20human%20nasal%20polyposis.%20Int%20J%20Oncol%2031%3A%201051%E2%80%931059." data-title="Mutations%20and%20differential%20expression%20of%20the%20ras%20family%20genes%20in%20human%20nasal%20polyposis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mutations+and+differential+expression+of+the+ras+family+genes+in+human+nasal+polyposis+Zaravinos+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0091646-Ljungberg1" id="pone.0091646-Ljungberg1" class="link-target"></a>Ljungberg B, Cowan NC, Hanbury DC, Hora M, Kuczyk MA, et al. (2010) EAU guidelines on renal cell carcinoma: the 2010 update. Eur Urol 58: 398–406. <ul class="reflinks"><li><a href="#" data-author="Ljungberg" data-cit="LjungbergB%2C%20CowanNC%2C%20HanburyDC%2C%20HoraM%2C%20KuczykMA%2C%20et%20al.%20%282010%29%20EAU%20guidelines%20on%20renal%20cell%20carcinoma%3A%20the%202010%20update.%20Eur%20Urol%2058%3A%20398%E2%80%93406." data-title="EAU%20guidelines%20on%20renal%20cell%20carcinoma%3A%20the%202010%20update" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=EAU+guidelines+on+renal+cell+carcinoma%3A+the+2010+update+Ljungberg+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0091646-Roupret2" id="pone.0091646-Roupret2" class="link-target"></a>Roupret M, Zigeuner R, Palou J, Boehle A, Kaasinen E, et al. (2012) [European guidelines for the diagnosis and management of upper urinary tract urothelial cell carcinomas: 2011 update. European Association of Urology Guideline Group for urothelial cell carcinoma of the upper urinary tract]. Actas Urol Esp 36: 2–14. <ul class="reflinks"><li><a href="#" data-author="Roupret" data-cit="RoupretM%2C%20ZigeunerR%2C%20PalouJ%2C%20BoehleA%2C%20KaasinenE%2C%20et%20al.%20%282012%29%20%5BEuropean%20guidelines%20for%20the%20diagnosis%20and%20management%20of%20upper%20urinary%20tract%20urothelial%20cell%20carcinomas%3A%202011%20update.%20European%20Association%20of%20Urology%20Guideline%20Group%20for%20urothelial%20cell%20carcinoma%20of%20the%20upper%20urinary%20tract%5D.%20Actas%20Urol%20Esp%2036%3A%202%E2%80%9314." data-title="%5BEuropean%20guidelines%20for%20the%20diagnosis%20and%20management%20of%20upper%20urinary%20tract%20urothelial%20cell%20carcinomas%3A%202011%20update.%20European%20Association%20of%20Urology%20Guideline%20Group%20for%20urothelial%20cell%20carcinoma%20of%20the%20upper%20urinary%20tract%5D" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=%5BEuropean+guidelines+for+the+diagnosis+and+management+of+upper+urinary+tract+urothelial+cell+carcinomas%3A+2011+update.+European+Association+of+Urology+Guideline+Group+for+urothelial+cell+carcinoma+of+the+upper+urinary+tract%5D+Roupret+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0091646-Lu1" id="pone.0091646-Lu1" class="link-target"></a>Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al. (2005) MicroRNA expression profiles classify human cancers. Nature 435: 834–838. <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuJ%2C%20GetzG%2C%20MiskaEA%2C%20Alvarez-SaavedraE%2C%20LambJ%2C%20et%20al.%20%282005%29%20MicroRNA%20expression%20profiles%20classify%20human%20cancers.%20Nature%20435%3A%20834%E2%80%93838." data-title="MicroRNA%20expression%20profiles%20classify%20human%20cancers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+expression+profiles+classify+human+cancers+Lu+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0091646-Nakada2" id="pone.0091646-Nakada2" class="link-target"></a>Nakada C, Matsuura K, Tsukamoto Y, Tanigawa M, Yoshimoto T, et al. (2008) Genome-wide microRNA expression profiling in renal cell carcinoma: significant down-regulation of miR-141 and miR-200c. J Pathol 216: 418–427. <ul class="reflinks"><li><a href="#" data-author="Nakada" data-cit="NakadaC%2C%20MatsuuraK%2C%20TsukamotoY%2C%20TanigawaM%2C%20YoshimotoT%2C%20et%20al.%20%282008%29%20Genome-wide%20microRNA%20expression%20profiling%20in%20renal%20cell%20carcinoma%3A%20significant%20down-regulation%20of%20miR-141%20and%20miR-200c.%20J%20Pathol%20216%3A%20418%E2%80%93427." data-title="Genome-wide%20microRNA%20expression%20profiling%20in%20renal%20cell%20carcinoma%3A%20significant%20down-regulation%20of%20miR-141%20and%20miR-200c" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+microRNA+expression+profiling+in+renal+cell+carcinoma%3A+significant+down-regulation+of+miR-141+and+miR-200c+Nakada+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0091646-Weng1" id="pone.0091646-Weng1" class="link-target"></a>Weng L, Wu X, Gao H, Mu B, Li X, et al. (2010) MicroRNA profiling of clear cell renal cell carcinoma by whole-genome small RNA deep sequencing of paired frozen and formalin-fixed, paraffin-embedded tissue specimens. J Pathol 222: 41–51. <ul class="reflinks"><li><a href="#" data-author="Weng" data-cit="WengL%2C%20WuX%2C%20GaoH%2C%20MuB%2C%20LiX%2C%20et%20al.%20%282010%29%20MicroRNA%20profiling%20of%20clear%20cell%20renal%20cell%20carcinoma%20by%20whole-genome%20small%20RNA%20deep%20sequencing%20of%20paired%20frozen%20and%20formalin-fixed%2C%20paraffin-embedded%20tissue%20specimens.%20J%20Pathol%20222%3A%2041%E2%80%9351." data-title="MicroRNA%20profiling%20of%20clear%20cell%20renal%20cell%20carcinoma%20by%20whole-genome%20small%20RNA%20deep%20sequencing%20of%20paired%20frozen%20and%20formalin-fixed%2C%20paraffin-embedded%20tissue%20specimens" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+profiling+of+clear+cell+renal+cell+carcinoma+by+whole-genome+small+RNA+deep+sequencing+of+paired+frozen+and+formalin-fixed%2C+paraffin-embedded+tissue+specimens+Weng+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0091646-Spector1" id="pone.0091646-Spector1" class="link-target"></a>Spector Y, Fridman E, Rosenwald S, Zilber S, Huang Y, et al. (2013) Development and validation of a microRNA-based diagnostic assay for classification of renal cell carcinomas. Mol Oncol 7: 732–738. <ul class="reflinks"><li><a href="#" data-author="Spector" data-cit="SpectorY%2C%20FridmanE%2C%20RosenwaldS%2C%20ZilberS%2C%20HuangY%2C%20et%20al.%20%282013%29%20Development%20and%20validation%20of%20a%20microRNA-based%20diagnostic%20assay%20for%20classification%20of%20renal%20cell%20carcinomas.%20Mol%20Oncol%207%3A%20732%E2%80%93738." data-title="Development%20and%20validation%20of%20a%20microRNA-based%20diagnostic%20assay%20for%20classification%20of%20renal%20cell%20carcinomas" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Development+and+validation+of+a+microRNA-based+diagnostic+assay+for+classification+of+renal+cell+carcinomas+Spector+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0091646-Izquierdo1" id="pone.0091646-Izquierdo1" class="link-target"></a>Izquierdo L, Ingelmo-Torres M, Mallofre C, Lozano JJ, Verhasselt-Crinquette M, et al.. (2013) Prognostic value of microRNA expression pattern in upper tract urothelial carcinoma. BJU Int. <ul class="find-nolinks"></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0091646-Calin1" id="pone.0091646-Calin1" class="link-target"></a>Calin GA, Sevignani C, Dumitru CD, Hyslop T, Noch E, et al. (2004) Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. Proc Natl Acad Sci U S A 101: 2999–3004. <ul class="reflinks"><li><a href="#" data-author="Calin" data-cit="CalinGA%2C%20SevignaniC%2C%20DumitruCD%2C%20HyslopT%2C%20NochE%2C%20et%20al.%20%282004%29%20Human%20microRNA%20genes%20are%20frequently%20located%20at%20fragile%20sites%20and%20genomic%20regions%20involved%20in%20cancers.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20101%3A%202999%E2%80%933004." data-title="Human%20microRNA%20genes%20are%20frequently%20located%20at%20fragile%20sites%20and%20genomic%20regions%20involved%20in%20cancers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+microRNA+genes+are+frequently+located+at+fragile+sites+and+genomic+regions+involved+in+cancers+Calin+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0091646-Girgis1" id="pone.0091646-Girgis1" class="link-target"></a>Girgis AH, Iakovlev VV, Beheshti B, Bayani J, Squire JA, et al. (2012) Multilevel whole-genome analysis reveals candidate biomarkers in clear cell renal cell carcinoma. Cancer Res 72: 5273–5284. <ul class="reflinks"><li><a href="#" data-author="Girgis" data-cit="GirgisAH%2C%20IakovlevVV%2C%20BeheshtiB%2C%20BayaniJ%2C%20SquireJA%2C%20et%20al.%20%282012%29%20Multilevel%20whole-genome%20analysis%20reveals%20candidate%20biomarkers%20in%20clear%20cell%20renal%20cell%20carcinoma.%20Cancer%20Res%2072%3A%205273%E2%80%935284." data-title="Multilevel%20whole-genome%20analysis%20reveals%20candidate%20biomarkers%20in%20clear%20cell%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multilevel+whole-genome+analysis+reveals+candidate+biomarkers+in+clear+cell+renal+cell+carcinoma+Girgis+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0091646-Bratslavsky1" id="pone.0091646-Bratslavsky1" class="link-target"></a>Bratslavsky G, Sudarshan S, Neckers L, Linehan WM (2007) Pseudohypoxic pathways in renal cell carcinoma. Clin Cancer Res 13: 4667–4671. <ul class="reflinks"><li><a href="#" data-author="Bratslavsky" data-cit="BratslavskyG%2C%20SudarshanS%2C%20NeckersL%2C%20LinehanWM%20%282007%29%20Pseudohypoxic%20pathways%20in%20renal%20cell%20carcinoma.%20Clin%20Cancer%20Res%2013%3A%204667%E2%80%934671." data-title="Pseudohypoxic%20pathways%20in%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Pseudohypoxic+pathways+in+renal+cell+carcinoma+Bratslavsky+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0091646-Klatte1" id="pone.0091646-Klatte1" class="link-target"></a>Klatte T, Rao PN, de Martino M, LaRochelle J, Shuch B, et al. (2009) Cytogenetic profile predicts prognosis of patients with clear cell renal cell carcinoma. J Clin Oncol 27: 746–753. <ul class="reflinks"><li><a href="#" data-author="Klatte" data-cit="KlatteT%2C%20RaoPN%2C%20de%20MartinoM%2C%20LaRochelleJ%2C%20ShuchB%2C%20et%20al.%20%282009%29%20Cytogenetic%20profile%20predicts%20prognosis%20of%20patients%20with%20clear%20cell%20renal%20cell%20carcinoma.%20J%20Clin%20Oncol%2027%3A%20746%E2%80%93753." data-title="Cytogenetic%20profile%20predicts%20prognosis%20of%20patients%20with%20clear%20cell%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cytogenetic+profile+predicts+prognosis+of+patients+with+clear+cell+renal+cell+carcinoma+Klatte+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0091646-Kovacs1" id="pone.0091646-Kovacs1" class="link-target"></a>Kovacs G, Fuzesi L, Emanual A, Kung HF (1991) Cytogenetics of papillary renal cell tumors. Genes Chromosomes Cancer 3: 249–255. <ul class="reflinks"><li><a href="#" data-author="Kovacs" data-cit="KovacsG%2C%20FuzesiL%2C%20EmanualA%2C%20KungHF%20%281991%29%20Cytogenetics%20of%20papillary%20renal%20cell%20tumors.%20Genes%20Chromosomes%20Cancer%203%3A%20249%E2%80%93255." data-title="Cytogenetics%20of%20papillary%20renal%20cell%20tumors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cytogenetics+of+papillary+renal+cell+tumors+Kovacs+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0091646-Speicher1" id="pone.0091646-Speicher1" class="link-target"></a>Speicher MR, Schoell B, du Manoir S, Schrock E, Ried T, et al. (1994) Specific loss of chromosomes 1, 2, 6, 10, 13, 17, and 21 in chromophobe renal cell carcinomas revealed by comparative genomic hybridization. Am J Pathol 145: 356–364. <ul class="reflinks"><li><a href="#" data-author="Speicher" data-cit="SpeicherMR%2C%20SchoellB%2C%20du%20ManoirS%2C%20SchrockE%2C%20RiedT%2C%20et%20al.%20%281994%29%20Specific%20loss%20of%20chromosomes%201%2C%202%2C%206%2C%2010%2C%2013%2C%2017%2C%20and%2021%20in%20chromophobe%20renal%20cell%20carcinomas%20revealed%20by%20comparative%20genomic%20hybridization.%20Am%20J%20Pathol%20145%3A%20356%E2%80%93364." data-title="Specific%20loss%20of%20chromosomes%201%2C%202%2C%206%2C%2010%2C%2013%2C%2017%2C%20and%2021%20in%20chromophobe%20renal%20cell%20carcinomas%20revealed%20by%20comparative%20genomic%20hybridization" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Specific+loss+of+chromosomes+1%2C+2%2C+6%2C+10%2C+13%2C+17%2C+and+21+in+chromophobe+renal+cell+carcinomas+revealed+by+comparative+genomic+hybridization+Speicher+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0091646-Storkel1" id="pone.0091646-Storkel1" class="link-target"></a>Storkel S, Eble JN, Adlakha K, Amin M, Blute ML, et al. (1997) Classification of renal cell carcinoma: Workgroup No. 1. Union Internationale Contre le Cancer (UICC) and the American Joint Committee on Cancer (AJCC). Cancer 80: 987–989. <ul class="reflinks"><li><a href="#" data-author="Storkel" data-cit="StorkelS%2C%20EbleJN%2C%20AdlakhaK%2C%20AminM%2C%20BluteML%2C%20et%20al.%20%281997%29%20Classification%20of%20renal%20cell%20carcinoma%3A%20Workgroup%20No.%201.%20Union%20Internationale%20Contre%20le%20Cancer%20%28UICC%29%20and%20the%20American%20Joint%20Committee%20on%20Cancer%20%28AJCC%29.%20Cancer%2080%3A%20987%E2%80%93989." data-title="Classification%20of%20renal%20cell%20carcinoma%3A%20Workgroup%20No.%201.%20Union%20Internationale%20Contre%20le%20Cancer%20%28UICC%29%20and%20the%20American%20Joint%20Committee%20on%20Cancer%20%28AJCC%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Classification+of+renal+cell+carcinoma%3A+Workgroup+No.+1.+Union+Internationale+Contre+le+Cancer+%28UICC%29+and+the+American+Joint+Committee+on+Cancer+%28AJCC%29+Storkel+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0091646-vandenBerg1" id="pone.0091646-vandenBerg1" class="link-target"></a>van den Berg E, Dijkhuizen T, Oosterhuis JW, Geurts van Kessel A, de Jong B, et al. (1997) Cytogenetic classification of renal cell cancer. Cancer Genet Cytogenet 95: 103–107. <ul class="reflinks"><li><a href="#" data-author="van%20den%20Berg" data-cit="van%20den%20BergE%2C%20DijkhuizenT%2C%20OosterhuisJW%2C%20Geurts%20van%20KesselA%2C%20de%20JongB%2C%20et%20al.%20%281997%29%20Cytogenetic%20classification%20of%20renal%20cell%20cancer.%20Cancer%20Genet%20Cytogenet%2095%3A%20103%E2%80%93107." data-title="Cytogenetic%20classification%20of%20renal%20cell%20cancer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cytogenetic+classification+of+renal+cell+cancer+van+den+Berg+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0091646-Velickovic1" id="pone.0091646-Velickovic1" class="link-target"></a>Velickovic M, Delahunt B, Storkel S, Grebem SK (2001) VHL and FHIT locus loss of heterozygosity is common in all renal cancer morphotypes but differs in pattern and prognostic significance. Cancer Res 61: 4815–4819. <ul class="reflinks"><li><a href="#" data-author="Velickovic" data-cit="VelickovicM%2C%20DelahuntB%2C%20StorkelS%2C%20GrebemSK%20%282001%29%20VHL%20and%20FHIT%20locus%20loss%20of%20heterozygosity%20is%20common%20in%20all%20renal%20cancer%20morphotypes%20but%20differs%20in%20pattern%20and%20prognostic%20significance.%20Cancer%20Res%2061%3A%204815%E2%80%934819." data-title="VHL%20and%20FHIT%20locus%20loss%20of%20heterozygosity%20is%20common%20in%20all%20renal%20cancer%20morphotypes%20but%20differs%20in%20pattern%20and%20prognostic%20significance" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=VHL+and+FHIT+locus+loss+of+heterozygosity+is+common+in+all+renal+cancer+morphotypes+but+differs+in+pattern+and+prognostic+significance+Velickovic+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0091646-Yoshimoto1" id="pone.0091646-Yoshimoto1" class="link-target"></a>Yoshimoto T, Matsuura K, Karnan S, Tagawa H, Nakada C, et al. (2007) High-resolution analysis of DNA copy number alterations and gene expression in renal clear cell carcinoma. J Pathol 213: 392–401. <ul class="reflinks"><li><a href="#" data-author="Yoshimoto" data-cit="YoshimotoT%2C%20MatsuuraK%2C%20KarnanS%2C%20TagawaH%2C%20NakadaC%2C%20et%20al.%20%282007%29%20High-resolution%20analysis%20of%20DNA%20copy%20number%20alterations%20and%20gene%20expression%20in%20renal%20clear%20cell%20carcinoma.%20J%20Pathol%20213%3A%20392%E2%80%93401." data-title="High-resolution%20analysis%20of%20DNA%20copy%20number%20alterations%20and%20gene%20expression%20in%20renal%20clear%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=High-resolution+analysis+of+DNA+copy+number+alterations+and+gene+expression+in+renal+clear+cell+carcinoma+Yoshimoto+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0091646-ThapaRJ1" id="pone.0091646-ThapaRJ1" class="link-target"></a>Thapa RJ C, Cheung M, Nogusa S, Pei J, Peri S, et al. (2013) NF-κB inhibition by bortezomib permits IFN-γ-activated RIP1 kinase-dependent necrosis in renal cell carcinoma. Mol Cancer Ther 12: 1568–1578. <ul class="reflinks"><li><a href="#" data-author="Thapa%20RJ" data-cit="Thapa%20RJC%2C%20CheungM%2C%20NogusaS%2C%20PeiJ%2C%20PeriS%2C%20et%20al.%20%282013%29%20NF-%CE%BAB%20inhibition%20by%20bortezomib%20permits%20IFN-%CE%B3-activated%20RIP1%20kinase-dependent%20necrosis%20in%20renal%20cell%20carcinoma.%20Mol%20Cancer%20Ther%2012%3A%201568%E2%80%931578." data-title="NF-%CE%BAB%20inhibition%20by%20bortezomib%20permits%20IFN-%CE%B3-activated%20RIP1%20kinase-dependent%20necrosis%20in%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=NF-%CE%BAB+inhibition+by+bortezomib+permits+IFN-%CE%B3-activated+RIP1+kinase-dependent+necrosis+in+renal+cell+carcinoma+Thapa+RJ+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0091646-Sato1" id="pone.0091646-Sato1" class="link-target"></a>Sato A, Nagase H, Obinata D, Fujiwara K, Fukuda N, et al. (2013) Inhibition of MMP-9 using a pyrrole-imidazole polyamide reduces cell invasion in renal cell carcinoma. Int J Oncol 43: 1441–1446. <ul class="reflinks"><li><a href="#" data-author="Sato" data-cit="SatoA%2C%20NagaseH%2C%20ObinataD%2C%20FujiwaraK%2C%20FukudaN%2C%20et%20al.%20%282013%29%20Inhibition%20of%20MMP-9%20using%20a%20pyrrole-imidazole%20polyamide%20reduces%20cell%20invasion%20in%20renal%20cell%20carcinoma.%20Int%20J%20Oncol%2043%3A%201441%E2%80%931446." data-title="Inhibition%20of%20MMP-9%20using%20a%20pyrrole-imidazole%20polyamide%20reduces%20cell%20invasion%20in%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Inhibition+of+MMP-9+using+a+pyrrole-imidazole+polyamide+reduces+cell+invasion+in+renal+cell+carcinoma+Sato+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0091646-Zaravinos3" id="pone.0091646-Zaravinos3" class="link-target"></a>Zaravinos A, Pieri M, Mourmouras N, Anastasiadou N, Zouvani I, et al.. (2014) Altered metabolic pathways in clear cell renal cell carcinoma: A meta-analysis and validation study focused on the deregulated genes and their associated networks. Oncoscience, Advance Publications. <ul class="find-nolinks"></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0091646-Aggarwal1" id="pone.0091646-Aggarwal1" class="link-target"></a>Aggarwal BB, Gehlot P (2009) Inflammation and cancer: how friendly is the relationship for cancer patients? Curr Opin Pharmacol 9: 351–369. <ul class="reflinks"><li><a href="#" data-author="Aggarwal" data-cit="AggarwalBB%2C%20GehlotP%20%282009%29%20Inflammation%20and%20cancer%3A%20how%20friendly%20is%20the%20relationship%20for%20cancer%20patients%3F%20Curr%20Opin%20Pharmacol%209%3A%20351%E2%80%93369." data-title="Inflammation%20and%20cancer%3A%20how%20friendly%20is%20the%20relationship%20for%20cancer%20patients%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Inflammation+and+cancer%3A+how+friendly+is+the+relationship+for+cancer+patients%3F+Aggarwal+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0091646-Wu1" id="pone.0091646-Wu1" class="link-target"></a>Wu Y, Fu X, Zhu X, He X, Zou C, et al. (2011) Prognostic role of systemic inflammatory response in renal cell carcinoma: a sytemic review and meta-analysis. J Cancer Res Clin Oncol 137: 887–896. <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuY%2C%20FuX%2C%20ZhuX%2C%20HeX%2C%20ZouC%2C%20et%20al.%20%282011%29%20Prognostic%20role%20of%20systemic%20inflammatory%20response%20in%20renal%20cell%20carcinoma%3A%20a%20sytemic%20review%20and%20meta-analysis.%20J%20Cancer%20Res%20Clin%20Oncol%20137%3A%20887%E2%80%93896." data-title="Prognostic%20role%20of%20systemic%20inflammatory%20response%20in%20renal%20cell%20carcinoma%3A%20a%20sytemic%20review%20and%20meta-analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Prognostic+role+of+systemic+inflammatory+response+in+renal+cell+carcinoma%3A+a+sytemic+review+and+meta-analysis+Wu+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pone.0091646-Feng1" id="pone.0091646-Feng1" class="link-target"></a>Feng Z, Zhang C, Wu R, Hu W (2011) Tumor suppressor p53 meets microRNAs. J Mol Cell Biol 3: 44–50. <ul class="reflinks"><li><a href="#" data-author="Feng" data-cit="FengZ%2C%20ZhangC%2C%20WuR%2C%20HuW%20%282011%29%20Tumor%20suppressor%20p53%20meets%20microRNAs.%20J%20Mol%20Cell%20Biol%203%3A%2044%E2%80%9350." data-title="Tumor%20suppressor%20p53%20meets%20microRNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor+suppressor+p53+meets+microRNAs+Feng+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pone.0091646-Yamada1" id="pone.0091646-Yamada1" class="link-target"></a>Yamada Y, Hidaka H, Seki N, Yoshino H, Yamasaki T, et al. (2013) Tumor-suppressive microRNA-135a inhibits cancer cell proliferation by targeting the c-MYC oncogene in renal cell carcinoma. Cancer Sci 104: 304–312. <ul class="reflinks"><li><a href="#" data-author="Yamada" data-cit="YamadaY%2C%20HidakaH%2C%20SekiN%2C%20YoshinoH%2C%20YamasakiT%2C%20et%20al.%20%282013%29%20Tumor-suppressive%20microRNA-135a%20inhibits%20cancer%20cell%20proliferation%20by%20targeting%20the%20c-MYC%20oncogene%20in%20renal%20cell%20carcinoma.%20Cancer%20Sci%20104%3A%20304%E2%80%93312." data-title="Tumor-suppressive%20microRNA-135a%20inhibits%20cancer%20cell%20proliferation%20by%20targeting%20the%20c-MYC%20oncogene%20in%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tumor-suppressive+microRNA-135a+inhibits+cancer+cell+proliferation+by+targeting+the+c-MYC+oncogene+in+renal+cell+carcinoma+Yamada+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pone.0091646-Wach1" id="pone.0091646-Wach1" class="link-target"></a>Wach S, Nolte E, Theil A, Stöhr C, T Rau T, et al. (2013) MicroRNA profiles classify papillary renal cell carcinoma subtypes. Br J Cancer 109: 714–722. <ul class="reflinks"><li><a href="#" data-author="Wach" data-cit="WachS%2C%20NolteE%2C%20TheilA%2C%20St%C3%B6hrC%2C%20T%20RauT%2C%20et%20al.%20%282013%29%20MicroRNA%20profiles%20classify%20papillary%20renal%20cell%20carcinoma%20subtypes.%20Br%20J%20Cancer%20109%3A%20714%E2%80%93722." data-title="MicroRNA%20profiles%20classify%20papillary%20renal%20cell%20carcinoma%20subtypes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+profiles+classify+papillary+renal+cell+carcinoma+subtypes+Wach+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pone.0091646-Teicher1" id="pone.0091646-Teicher1" class="link-target"></a>Teicher BA, Linehan WM, Helman LJ (2012) Targeting cancer metabolism. Clin Cancer Res 18: 5537–5545. <ul class="reflinks"><li><a href="#" data-author="Teicher" data-cit="TeicherBA%2C%20LinehanWM%2C%20HelmanLJ%20%282012%29%20Targeting%20cancer%20metabolism.%20Clin%20Cancer%20Res%2018%3A%205537%E2%80%935545." data-title="Targeting%20cancer%20metabolism" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Targeting+cancer+metabolism+Teicher+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pone.0091646-Tun1" id="pone.0091646-Tun1" class="link-target"></a>Tun HW, Marlow LA, von Roemeling CA, Cooper SJ, Kreinest P, et al. (2010) Pathway signature and cellular differentiation in clear cell renal cell carcinoma. PLoS One 5: e10696. <ul class="reflinks"><li><a href="#" data-author="Tun" data-cit="TunHW%2C%20MarlowLA%2C%20von%20RoemelingCA%2C%20CooperSJ%2C%20KreinestP%2C%20et%20al.%20%282010%29%20Pathway%20signature%20and%20cellular%20differentiation%20in%20clear%20cell%20renal%20cell%20carcinoma.%20PLoS%20One%205%3A%20e10696." data-title="Pathway%20signature%20and%20cellular%20differentiation%20in%20clear%20cell%20renal%20cell%20carcinoma" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Pathway+signature+and+cellular+differentiation+in+clear+cell+renal+cell+carcinoma+Tun+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

