



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p><em>Schizosaccharomyces pombe</em> is an outstanding model organism for cell biological investigations, yet the range of useful and well-characterized fluorescent proteins (XFPs) is limited. We generated and characterized three recoded fluorescent proteins for 3-color analysis in <em>S</em>.<em>pombe</em>, Super-folder GFP, monomeric Kusabira Orange 2 and E2Crimson. Upon optimization and expression in <em>S</em>. <em>pombe</em>, the three proteins enabled sensitive simultaneous 3-color detection capability. Furthermore, we describe a strategy that combines a pulse-chase approach and mathematical modeling to quantify the maturation kinetics of these proteins <em>in vivo</em>. We observed maturation kinetics in <em>S</em>. <em>pombe</em> that are expected from those described for these proteins <em>in vitro</em> and/or in other cell types, but also unpredicted behaviors. Our studies provide a kinetically-characterized, integrated three-color XFP toolbox for <em>S</em>. <em>pombe</em>.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Al-Sady B, Greenstein RA, El-Samad HJ, Braun S, Madhani HD (2016) Sensitive and Quantitative Three-Color Protein Imaging in Fission Yeast Using Spectrally Diverse, Recoded Fluorescent Proteins with Experimentally-Characterized <em>In Vivo</em> Maturation Kinetics. PLoS ONE 11(8):
           e0159292.
        
        https://doi.org/10.1371/journal.pone.0159292</p><p><strong>Editor: </strong>Juan Mata, University of Cambridge, UNITED KINGDOM</p><p><strong>Received: </strong>February 27, 2016; <strong>Accepted: </strong>June 30, 2016; <strong>Published: </strong> August 1, 2016</p><p><strong>Copyright: </strong> © 2016 Al-Sady et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>Plasmids containing the recoded XFPs as C-terminal tagging constructs (see <a href="#pone-0159292-g001">Fig 1</a>) have been deposited to Addgene (ID numbers 74080-74087).</p><p><strong>Funding: </strong>This study was supported by a grant from the National Institutes of Health (<a href="https://www.nigms.nih.gov/Pages/default.aspx">https://www.nigms.nih.gov/Pages/default.aspx</a>) number GM71801 to HDM and a grant from the European Union Network of Excellence EpiGeneSys (<a href="http://www.epigenesys.eu/en/">http://www.epigenesys.eu/en/</a>) number HEALTH-2010-257082 to SB.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Visualizing the relationships between cellular structures in vivo requires the ability to tag and image multiple proteins at once. This is generally accomplished using engineered fluorescent proteins (XFPs)[<a href="#pone.0159292.ref001" class="ref-tip">1</a>]. The fission yeast <em>S</em>. <em>pombe</em> is a premier system for investigations of cell biology. Surprisingly, only a few XFPs have been fully validated for dynamic analysis in <em>S</em>.<em>pombe</em>. In addition, the choice for characterized and optimized fluorescent proteins that are suitable for multicolor analysis is limited compared to the arsenal available for <em>Saccharomyces cerevisiae</em> [<a href="#pone.0159292.ref002" class="ref-tip">2</a>]. Most studies in <em>S</em>.<em>pombe</em> focus on protein fusions to either GFP or YFP. There are few reports of red/orange (mostly mCherry [<a href="#pone.0159292.ref003" class="ref-tip">3</a>–<a href="#pone.0159292.ref008" class="ref-tip">8</a>]) and very few reports of use of blue [<a href="#pone.0159292.ref009" class="ref-tip">9</a>] or far-red fluorescent proteins [<a href="#pone.0159292.ref003" class="ref-tip">3</a>]. The commonly used fluors allow little room for the spectral separation required for robust and specific 3-color detection. The most common combination for imaging in <em>S</em>.<em>pombe</em>, GFP and mCherry, would require a blue protein as a third color, which commonly displays limited brightness. In addition to blue, green and red proteins, a typical combination for spectrally separable 3-color detection is green, orange and far-red. To date, there is only one report to our knowledge of the use of orange and far-red proteins in <em>S</em>.<em>pombe</em>, primarily as fusion proteins for cytoskeletal analysis[<a href="#pone.0159292.ref003" class="ref-tip">3</a>].</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>To address this unmet need, we sought to develop a set of XFPs in <em>S</em>.<em>pombe</em> that combines the following characteristics: 1) good spectral separation to allow flow cytometric and microscopic analysis of three colors, 2) low limits of detection even in three color tracking settings that may not be fully optimized for any one color. Given the slow maturation kinetics and low photo stability of some of the initial XFP isolates [<a href="#pone.0159292.ref010" class="ref-tip">10</a>, <a href="#pone.0159292.ref011" class="ref-tip">11</a>], much of the protein engineering effort has gone to optimizing those parameters [<a href="#pone.0159292.ref012" class="ref-tip">12</a>–<a href="#pone.0159292.ref015" class="ref-tip">15</a>]. Maturation kinetics are of particular importance in the context of imaging dynamic processes. Long maturation times make challenging the investigation of the dynamics of a biological process that occur on relatively short timescales. Maturation times are rarely measured for the system of study [<a href="#pone.0159292.ref016" class="ref-tip">16</a>], but rather typically inferred from <em>E</em>.<em>coli</em> expression or <em>in vitro</em> refolding measurements [<a href="#pone.0159292.ref013" class="ref-tip">13</a>, <a href="#pone.0159292.ref017" class="ref-tip">17</a>–<a href="#pone.0159292.ref019" class="ref-tip">19</a>]. However, it is becoming increasingly evident that the biological environment influences fluorophore maturation, with differences documented even between strains of the same organism [<a href="#pone.0159292.ref020" class="ref-tip">20</a>]. This may not be surprising given the known dependence of maturation on pH and oxygen environment, temperature [<a href="#pone.0159292.ref021" class="ref-tip">21</a>–<a href="#pone.0159292.ref024" class="ref-tip">24</a>] and probably others.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>While determining the maturing rate <em>in vivo</em> is important, it is not trivial. Most previous approaches to extract maturation times rely on measuring <em>in vitro</em> or cell-free refolding rates [<a href="#pone.0159292.ref013" class="ref-tip">13</a>, <a href="#pone.0159292.ref015" class="ref-tip">15</a>, <a href="#pone.0159292.ref018" class="ref-tip">18</a>], which may not apply <em>in vivo</em>, or approaches which require administration of an efficient translation block under constitutive expression. The latter method, while compatible with <em>in vivo</em> settings, requires the ability to measure small increases in fluorescence [<a href="#pone.0159292.ref020" class="ref-tip">20</a>, <a href="#pone.0159292.ref025" class="ref-tip">25</a>]. <em>E</em>.<em>coli</em> is the primary system where transcriptional initiation is combined with a chloramphenicol translation block to measure maturation time [<a href="#pone.0159292.ref012" class="ref-tip">12</a>], relying on IPTG as the inducer. While practical, this system is not available for eukaryotes. Further, since such studies only measure fluorescence, they do not take into account the time lag between induction and translation, yielding a compound rate. An alternative method to estimate maturation time, given the requirement for molecular oxygen for the fluor-forming chemistry, involves re-oxygenation after hypoxia [<a href="#pone.0159292.ref026" class="ref-tip">26</a>, <a href="#pone.0159292.ref027" class="ref-tip">27</a>]. However, this method only addresses the very last step of maturation, fluor cyclization, and may represent an underestimate of maturation kinetics.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>We report here a recoded, optimized set of XFPs for <em>S</em>. <em>pombe</em>, that offers sensitive detection, simultaneous three-color detection, and for which we have determined the <em>in vivo</em> folding kinetics using a novel approach. This new XFP toolbox will enable quantitative and dynamic studies of <em>S</em>. <em>pombe</em> biology. We standardized our optimized XFP toolbox as C-terminal tagging vectors (<a href="#pone-0159292-g001">Fig 1</a>) and deposited relevant plasmids to Addgene for easy community access.</p>
<a class="link-target" id="pone-0159292-g001" name="pone-0159292-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.g001" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> C-terminal tagging vectors for <em>Schizosaccharomyces pombe</em> optimized XFPs.</div><p class="caption_target"><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1" class="link-target"></a><p>SF-GFP, mKO2 and E2C open reading frames were optimized for expression in <em>S</em>.<em>pombe</em>. The ORFs are preceded by a Gly rich linker (<em>S</em>.<em>pombe</em> codon frequency of linker amino acids below) and a PacI restriction site. The XFP is followed by the <em>ura4</em> terminator and a hyg, nat or G418 MX resistance cassette, followed by a PmeI restriction site. For easy of cloning, a common forward (F) and reverse (R) can be used to amplify any of the 9 XFP:ura4t:MX constructs.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.g001">
              https://doi.org/10.1371/journal.pone.0159292.g001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Optimization of SF-GFP mKO2 and E2C for use in S.pombe"></a>
<h3>Optimization of SF-GFP mKO2 and E2C for use in <em>S</em>.<em>pombe</em></h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>In order to assess the usefulness in <em>S</em>.<em>pombe</em> of the XFPs we were testing, we expressed them under control of the <em>bona fide</em> weak promoter of the <em>S</em>.<em>pombe ade6</em> gene and assessed their activity initially by flow cytometry [<a href="#pone.0159292.ref028" class="ref-tip">28</a>]. Using a weak promoter as a base to optimize the system provides a realistic benchmark for wider applicability. All XFPs tested were inserted (<em>ade6p</em>:XFP:<em>ura4t</em>) as a part of a hygromycin resistance cassette in chromosome II between <em>SPBC1711</em>.<em>1</em> and <em>SPBC1711</em>.<em>12</em> (<a href="#pone-0159292-g002">Fig 2A</a>, see <a href="#pone-0159292-t001">Table 1</a> for all strain genotypes). We initially tested the expression of a range of XFPs, with their messages not optimized for codon usage, with focus on proteins reported to be rapidly maturing, and that in combination have significant spectral separation: mTagBFP2[<a href="#pone.0159292.ref029" class="ref-tip">29</a>], turbo GFP[<a href="#pone.0159292.ref030" class="ref-tip">30</a>], SF-GFP[<a href="#pone.0159292.ref013" class="ref-tip">13</a>], YFP-Venus[<a href="#pone.0159292.ref016" class="ref-tip">16</a>], mKO2[<a href="#pone.0159292.ref031" class="ref-tip">31</a>], TagRFP-T[<a href="#pone.0159292.ref014" class="ref-tip">14</a>], mOrange2 [<a href="#pone.0159292.ref014" class="ref-tip">14</a>], mcherry, mKate2[<a href="#pone.0159292.ref032" class="ref-tip">32</a>] and E2Crimson[<a href="#pone.0159292.ref033" class="ref-tip">33</a>]. Of those, mTagBFP2, turbo GFP and, surprisingly, YFP-Venus gave no detectable signal by flow cytometry (data not shown). mKate2 and mOrange2 gave only weak signal, while SF-GFP, TagRFP-T, mKO2, E2Crimson and mCherry have low to intermediate signals (<a href="#pone-0159292-g002">Fig 2B</a>). SF-GFP, mKO2 or TagRFP-T and E2Crinsom form a compatible 3-color detection set. Since in our hands mKO2 was significantly brighter than TagRFP-T, we moved forward with optimizing the coding messages of SF-GFP, mKO2 and E2C XFPs for the codon usage and GC content of <em>S</em>.<em>pombe</em> (proprietary GeneOptimizer algorithm, Life Technologies). We denote these versions with the subscript s.p. The alignment between pre-optimized XFPs and s.p. XFPs can be found in S File 1. Message optimization resulted in a substantial increase in XFP fluorescence as measured by flow cytometry signals (see for mKO2, <a href="#pone-0159292-g002">Fig 2B</a>). Using a 488nm laser to excite SF-GFP, and a 561nm laser to excite both mKO2 and E2C, we obtained about 10-15-fold signal over background (untransformed cells) for SF-GFP<sub>s.p.</sub> and mKO2<sub>s.p.</sub>, but only around 4-fold for E2C<sub>s.p.</sub>. To increase E2C signals to a similar level as the other two XFPs under these detection conditions, we fused three E2C monomers separated by Glycine-Serine linkers between the three units. This triple protein (3xE2C<sub>s.p.</sub>) yielded roughly 2.7x the signal of the monomer, resulting in around 12-fold signal to noise.</p>
<a class="link-target" id="pone-0159292-g002" name="pone-0159292-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.g002" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Detection of <em>ade6</em> promoter-driven SF-GFP<sub>s.p.</sub>, mKO2<sub>s.p.</sub> and 3XE2C<sub>s.p.</sub> by flow cytometry.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p><strong>A.</strong> Constructs driving on of the three XFPs from the weak <em>ade6</em> promoter, terminated by the <em>ura4</em> terminator and marked by a hyg resistance cassette were inserted on chromosome II between <em>SPBC1711</em>.<em>11</em> and <em>SPBC1711</em>.<em>12</em>. A trimerized version of E2C was used. <strong>B.</strong> The three strains were examined using a 488nm to excite SF-GFP<sub>s.p.</sub> and 561nm laser to excite mKO2<sub>s.p.</sub> and 3XE2C<sub>s.p.</sub> The effect of coding message optimization is shown for mKO2, where the non-optimized version (mKO2) is only 25% as bright as the optimized version (mKO2<sub>s.p.</sub>). A 2.7- fold increase in signal by trimerizing E2C (3XE2C<sub>s.p.</sub>) versus the monomeric version ((1XE2C<sub>s.p.</sub>). <strong>C.</strong> Excitation with 488nm for SF-GFP<sub>s.p.</sub> and 532nm for mKO2<sub>s.p.</sub> and 3XE2C<sub>s.p.</sub>. 532 excitation results in a ~10 fold increase of mKO<sub>s.p.</sub> signal versus excitation at 561nm. <strong>D.</strong> Excitation with 488nm for SF-GFP<sub>s.p.,</sub> 561nm for mKO2<sub>s.p.</sub> and 640nm for 3XE2C<sub>s.p.</sub>. The analysis was performed using two flow cytometers, the emission filter sets are denoted below the histograms.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.g002">
              https://doi.org/10.1371/journal.pone.0159292.g002</a></p></div><a class="link-target" id="pone-0159292-t001" name="pone-0159292-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.t001" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> <em>Schizosaccharomyces pombe</em> strain genotypes used in this study.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.t001">
              https://doi.org/10.1371/journal.pone.0159292.t001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>To make the optimized XFPs available to the community in a multiuse format, we produced C-terminal tagging vectors for the monomeric versions of E2C<sub>s.p.</sub>, SF-GFP<sub>s.p.</sub> and mKO2<sub>s.p.</sub>. The general format of the vectors is described in <a href="#pone-0159292-g001">Fig 1</a>: The XFP is terminated by the <em>ura4</em> terminator followed by a hygromycin B (hyg), nourseothricin (nat) or G418 (kan) MX drug resistance cassette [<a href="#pone.0159292.ref006" class="ref-tip">6</a>]. The N terminal ATG is preceded by a glycine rich and common linker sequence, allowing for the amplification of each XFP with a common primer. The corresponding reverse primer anneals to all three drug resistance cassettes. The plasmids have been deposited to Addgene (ID numbers 74080–74087).</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Simultaneous detection of SF-GFPs.p. mKO2s.p. and E2Cs.p. by flow cytometry"></a>
<h3>Simultaneous detection of SF-GFP<sub>s.p.</sub> mKO2<sub>s.p.</sub> and E2C<sub>s.p.</sub> by flow cytometry</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Next, to examine the usefulness of our optimized proteins in three color detection by flow cytometry, we examined the signal to noise in the preferred emission channel (<a href="#pone-0159292-g002">Fig 2</a>) but also the emission bleed from one XFP into the channel of another. We chose three laser excitation scenarios that likely cover most available standard flow cytometer configurations. Briefly, in all experiments, SF-GFP is excited by a 488nm laser, available on most stand cytometers, while mKO2 and E2C proteins are excited by either 532nm, 561nm or 640nm lasers. Under a configuration where SF-GFP<sub>s.p.</sub> is excited by a 488nm laser, and mKO2<sub>s.p.</sub> and 3xE2C<sub>s.p.</sub> are exited with a 561nm laser (488/561/561), we obtain about 10-15x signal for the three XFPs relative to untransformed control cells (<a href="#pone-0159292-g002">Fig 2B</a>). If instead we excited mKO2<sub>s.p.</sub> and 3xE2C<sub>s.p.</sub>with a 532nm laser (488/532/532), we can obtain almost 100x signal over noise for mKO2<sub>s.p.</sub>, at the cost of decrease in signal for 3xE2C<sub>s.p.</sub>(<a href="#pone-0159292-g002">Fig 2C</a>). This configuration is useful if one of the proteins under study is expressed/produced at a lower level than the others. For such a low-abundance protein, mKO2<sub>s.p.</sub> is ideally suited as it is significantly brighter at its optimal excitation than SF-GFP<sub>s.p.</sub> or 3xE2C<sub>s.p.</sub> To assess bleed and compensation, we stayed with the 488/561/561 excitation regime that produces even signal for all three proteins. Neither SF-GFP<sub>s.p.</sub> nor 3xE2C<sub>s.p.</sub> exhibits any significant bleed into the channels for the other XFPs (<a href="#pone.0159292.s001">S1 Fig</a>). On the other hand, mKO2<sub>s.p.</sub> has very minor bleed into the GFP channel, and significant bleed into the E2C channel (<a href="#pone.0159292.s001">S1 Fig</a>, left). This emission light bleed can be efficiently compensated by common cytometry software packages such as the DIVA software on BD flow cytometers (<a href="#pone.0159292.s001">S1 Fig</a>, right), allowing specific detection of all three XFPs simultaneously. It is also possible to detect 3xE2C<sub>s.p.</sub> using a 640nm laser (488/532/640 or 488/561/640). In the 488/561/640 excitation regime (<a href="#pone-0159292-g002">Fig 2D</a>), there is no detectable bleed from mKO2 into the E2C 640nm excited, 670/14 channel. The only compensation required in these conditions therefore is for the slight bleed from mKO2<sub>s.p.</sub> into the GFP channel (<a href="#pone.0159292.s002">S2 Fig</a>). We found that under the conditions of this experiment, the 3xE2C<sub>s.p.</sub> signal was reduced about 40% compared to excitation at 561nm. The 488/561/640nm excitation combination is the most specific, with little compromise in signal-to-noise versus 488/561/561nm. However, if this combination is not available, we have shown here that the 3x-color system can be specifically detected using alternative (488/532/532; 488/561/561) combinations.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Determining maturation times of XFPs in live S.pombe cells"></a>
<h3>Determining maturation times of XFPs in live <em>S</em>.<em>pombe</em> cells</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>SF-GFP[<a href="#pone.0159292.ref013" class="ref-tip">13</a>], E2C[<a href="#pone.0159292.ref033" class="ref-tip">33</a>] and mKO2[<a href="#pone.0159292.ref026" class="ref-tip">26</a>, <a href="#pone.0159292.ref031" class="ref-tip">31</a>] have been described as fast maturing proteins, yet this was established either <em>in vitro</em> or in heterologous systems. It has remained unclear whether maturation kinetics derived in one system or even strain[<a href="#pone.0159292.ref020" class="ref-tip">20</a>] will hold for the intended system of study. To address this uncertainty, we devised an experimental method to determine the <em>in vivo</em> maturation time of XFPs in <em>S</em>.<em>pombe</em>. Conceptually, our system requires only two key factors: The ability to induce expression of an XFP cassette, preferably producing an unstable message, and second, the ability to follow the protein and fluorescence accumulation of the XFP. In that sense, our method should have wide applicability to other biological systems.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2" class="link-target"></a><p>We use the well-described uracil-responsive gene <em>urg1</em> [<a href="#pone.0159292.ref034" class="ref-tip">34</a>, <a href="#pone.0159292.ref035" class="ref-tip">35</a>] in <em>S</em>.<em>pombe</em> to drive production of our XFPs in response to uracil administration. Conceptually, this system is very similar to induction of gene expression in response to auxotrophic media conditions in <em>S</em>.<em>cerevisiae</em> [<a href="#pone.0159292.ref036" class="ref-tip">36</a>]. One striking and useful feature of the <em>urg1</em> system is the rapid decay of the message following withdrawal of uracil [<a href="#pone.0159292.ref035" class="ref-tip">35</a>], allowing for production of a XFP pulse that can be followed. Genes encoding the three fluors C-terminally tagged with a 5x FLAG tag were placed downstream of the <em>urg1</em> promoter, replacing the endogenous <em>Urg1</em> ORF (<a href="#pone-0159292-g003">Fig 3A</a>). Cells were grown to early log–phase, and XFP production was induced with uracil for 20min followed by a uracil-free chase. To assess uracil induction and decay kinetics of our <em>urg1p</em>:XFP:5xFLAG messages, we tracked them by RT-qPCR over time (<a href="#pone-0159292-g003">Fig 3B</a>). We find that all messages are induced strongly by uracil and exhibit a sharp drop following their induction peak. However, while mKO2 and E2C mRNAs are induced rapidly, with messages peaking at 40min and 20min, respectively, induction of SF-GFP mRNA is delayed, and peaks at around 120min. This delay may be due to an altered transcriptional response or chromatin state at the <em>urg1</em> promoter in the <em>urg1p</em>:SF-GFP:5xFLAG strain. While SF-GFP induction exhibits a delayed response to uracil, mKO2 and E2C mRNA exhibit a very rapid response and decay as published for the endogenous <em>urg1</em> gene [<a href="#pone.0159292.ref035" class="ref-tip">35</a>]. Since all three messages respond to uracil with a pulse of mRNA production, it renders them equally useful for our maturation experiments.</p>
<a class="link-target" id="pone-0159292-g003" name="pone-0159292-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.g003" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> </div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p><strong>Determination of maturation kinetics <em>in vivo</em> of SF-GFP<sub>s.p.</sub>, mKO2<sub>s.p.</sub> and E2C<sub>s.p.</sub> A. Overview of approach.</strong> XFPs were placed downstream of the <em>urg1</em> promoter and tagged with a 5XFLAG tag. Transcription was induced by uracil for 20min. Anti-FLAG westerns can detect all stages following translation including folding and maturation (light orange box), while follow cytometry measurements only detect the fully matured protein. <strong>B. The dynamics of XFP:5xFLAG transcripts following the removal of uracil.</strong> E2C<sub>s.p.</sub> / actin, SF-GFP<sub>s.p.</sub> / actin and mKO2<sub>s.p.</sub> / actin mRNA rations are plotted as function of time and the peak set to 1 for each of the fluors. Uracil was administered from 0-20min. E2C<sub>s.p.</sub> transcript peaks at 20min, mKO2<sub>s.p.</sub> peaks at 40min, while SF-GFP<sub>s.p.</sub> peaks at 120min. <strong>C.-E. Time courses of total protein and fluorescence.</strong> The GAPDH normalized FLAG western blot signal (ng protein) is plotted on the right axis, total fluorescence as measured by flow cytometry on the left axis.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.g003">
              https://doi.org/10.1371/journal.pone.0159292.g003</a></p></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3" class="link-target"></a><p>To obtain an estimate for XFP folding and maturation following translation, we reasoned that western blot detection of the XFP:FLAG protein would capture all stages following translation, while fluorescence detection of the XFP would capture only the fully matured state (see scheme <a href="#pone-0159292-g003">Fig 3A</a>). We measured protein production using LiCor western blots, detecting FLAG and cellular GAPDH control signals simultaneously. The same samples were measured at indicated time points for fluorescence using the flow cytometry setup as in <a href="#pone-0159292-g002">Fig 2B</a> (<a href="#pone-0159292-g003">Fig 3C–3E</a>). In order to determine the maturation time of our three XFPs, we took the following approaches:</p>
<a id="article1.body1.sec2.sec3.p4" name="article1.body1.sec2.sec3.p4" class="link-target"></a><p>1. To estimate the delay between protein appearance and fluorescence, the folding maturation half-time (t<sub>1/2 fold/mature</sub>), we subtract the half-time of protein production as estimated by western blots (t<sub>1/2 protein</sub>), from the half-time of appearance of maximal fluorescence ((t<sub>1/2 fluorescence</sub>), i.e. t<sub>1/2 fold/mature</sub> = t<sub>1/2 fluorescence</sub>—t<sub>1/2 protein</sub>. We estimate the half time from the point of apparent maximal FLAG signal of fluorescence signal (see <a href="#pone-0159292-t002">Table 2</a>). Interestingly, we find that t<sub>1/2 protein</sub> is not identical between the three XFPs: It is similar between <em>urg1p</em>:mKO<sub>s.p.</sub> and <em>urg1p</em>:E2C<sub>s.p.</sub>, but apparently slower for <em>urg1p</em>:SF-GFP<sub>s.p.</sub> (<a href="#pone-0159292-g003">Fig 3D</a>, and see below). As noted above, this difference is due to a longer lag in uracil response of the SF-GFP<sub>s.p</sub> mRNA in the <em>urg1p</em>:SF-GFP<sub>s.p.</sub> strain. However, this delay in SF-GFP<sub>s.p.</sub> protein appearance caused by delayed induction of the message, relative to the other two fluors, underscoring the importance of measuring protein translation/folding rates in response to an inducer. If we now subtract this t<sub>1/2 protein</sub> from t<sub>1/2 fluorescence</sub>, we obtain a t<sub>1/2 fold/mature</sub> of 25min for SF-GFP and 40min for E2C. However, the t<sub>1/2 fold/mature</sub> for mKO2<sub>s.p.</sub> is significantly longer, at ~135min. This does not match with its previous characterization as a fast folding protein [<a href="#pone.0159292.ref026" class="ref-tip">26</a>, <a href="#pone.0159292.ref031" class="ref-tip">31</a>] and may indicate that the intracellular environment in <em>S</em>.<em>pombe</em> imposes slower maturation kinetics for mKO2.</p>
<a class="link-target" id="pone-0159292-t002" name="pone-0159292-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.t002" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Halftimes of maximal protein or fluorescence accumulation.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.t002">
              https://doi.org/10.1371/journal.pone.0159292.t002</a></p></div><a id="article1.body1.sec2.sec3.p5" name="article1.body1.sec2.sec3.p5" class="link-target"></a><p>2. Next, we aimed to extract the <em>in vivo</em> intrinsic rate constants of both protein production following induction and folding/maturation from our data (see <a href="#pone-0159292-t003">Table 3</a>). In our initial model, we used two protein states:</p>
<a class="link-target" id="pone-0159292-t003" name="pone-0159292-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.t003" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Fitting-derived parameters.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.t003">
              https://doi.org/10.1371/journal.pone.0159292.t003</a></p></div><a id="article1.body1.sec2.sec3.p6" name="article1.body1.sec2.sec3.p6" class="link-target"></a><p>RNA➔ translated protein (<em>P</em><sub><em>T</em></sub>)➔ folded/matured protein (<em>P</em><sub><em>FM</em></sub>).</p>
<a id="article1.body1.sec2.sec3.p7" name="article1.body1.sec2.sec3.p7" class="link-target"></a><p>We extracted the rate constants of XFP production and folding/maturation by applying a simple set of differential equations with one rate constant for production, <em>k</em><sub><em>P</em></sub>, and one for folding/maturation <em>k</em><sub><em>FM</em></sub> (<a href="#pone-0159292-g004">Fig 4A</a>). For E2C<sub>s.p.</sub> and mKO2 <sub>s.p.</sub>, <em>k</em><sub><em>P</em></sub> could be directly extracted from the data by fitting equations <em>(1) (2)</em> described in the fitting methods section. For both these fluors, the fit of the protein production equations to FLAG western data was excellent, indicating that the transcription and translation of the fluorescent proteins can be explained by the most parsimonious simple linear model with no rate limiting steps (<a href="#pone-0159292-g004">Fig 4A</a> top and <a href="#pone-0159292-g004">4C</a>). By contrast, we could not fit the FLAG western data for SF-GFP<sub>s.p.</sub> with these simple equations. As noted above, the <em>urg1p</em>:SF-GFP<sub>s.p.</sub> strain responded slower to uracil administration, evident in the delay in message (and protein) appearance relative to E2C and mKO2 (Figs <a href="#pone-0159292-g003">3</a> and <a href="#pone-0159292-g004">4</a>). Together with our observations that this reporter gene becomes repressed over long generational times (data not shown), we believe this delay to be due to an equilibrium between an accessible (open) and inaccessible (closed) chromatin state. We therefore introduced an additional equation that modeled explicitly such an equilibrium, resulting in a good fit of the SF-GFP<sub>s.p.</sub> protein data (<a href="#pone-0159292-g004">Fig 4A</a> bottom). We obtained a translation rate <em>k</em><sub><em>P</em></sub> for SF-GFP<sub>s.p.</sub>, E2C<sub>s.p.</sub> and mKO2<sub>s.p.</sub> of 0.0002min<sup>-1</sup>, 0.0004min<sup>-1</sup> and 0.0007min<sup>-1</sup>, respectively.</p>
<a class="link-target" id="pone-0159292-g004" name="pone-0159292-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.g004" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Mathematical fitting of total protein (FLAG/GAPDH) and mature protein (fluorescence) data.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig2.caption1.p1" name="article1.body1.sec2.sec3.fig2.caption1.p1" class="link-target"></a><p><strong>A.</strong> E2C and SF-GFP fits assume a two protein state model (translated and folded/matured). The SF-GFP protein fit assumed an equilibrium between a closed (non-uracil responsive) and open (uracil responsive) chromatin state at the <em>urg1</em> locus. <strong>B.</strong> mKO2 fits assume a three protein state model (translated, folded and matured). The solid lines represents the mean of all possible fits to the model, the gray lines represent all the fits with parameter combinations yielding less than 5% mean fitting error.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.g004">
              https://doi.org/10.1371/journal.pone.0159292.g004</a></p></div><a id="article1.body1.sec2.sec3.p8" name="article1.body1.sec2.sec3.p8" class="link-target"></a><p>To fit the fluorescence data, we opted to feed the measured FLAG western data directly into the fluor folding/maturation equation (<a href="#pone-0159292-g004">Fig 4A</a>). Decoupling the fit of the total protein from that of the fluorescent pool makes the extraction of the folding/maturation parameters independent of the quality of fit of the protein. The simple two-state model generated excellent fits for SF-GFP<sub>s.p.</sub> and E2C<sub>s.p.</sub> (methods), generating folding/maturation rate <em>k</em><sub><em>FM</em></sub> of and 0.019min<sup>-1</sup> and 0.008 min<sup>-1</sup>, respectively, indicating a faster folding/maturation rate for SF-GFP<sub>s.p.</sub>. This is consistent with the shorter t<sub>1/2 fold/mature</sub> for SF-GFP<sub>s.p.</sub>. Importantly, to test the robustness of the extracted parameters, we used a procedure in which we perturbed the fit parameters and simulated the equations with these parameters, keeping only fits that resulted in less than 5% mean fitting error from the data. <a href="#pone.0159292.s003">S3A and S3B Fig</a> shows the distributions of the resulting parameters, illustrating that different parameter combinations are indeed capable of reproducing the data at this tolerance, but that these values were closely centered around the mean fit values which we report.</p>
<a id="article1.body1.sec2.sec3.p9" name="article1.body1.sec2.sec3.p9" class="link-target"></a><p>Using the same procedure, we were not able to find any parameter set that can fit the mKO2 fluorescence data for the two protein state model in which a single step described folding and maturation. On inspection of the raw data, we noticed a long lag between maximal accumulation of translated protein and the appearance of any fluorescence in two replicates of the experiment (<a href="#pone-0159292-g004">Fig 4D</a> and <a href="#pone.0159292.s004">S4A Fig</a>). We reasoned that the observed long lag with zero accumulation of fluorescence is not consistent with a model where the maturation rate is the sole limiting rate following translation, but rather indicates another, non-fluorescent mKO2<sub>s.p.</sub> intermediate formed at a slow production rate. To account for this lag and a possible non-fluorescent intermediate, we proceeded to fit the data to the two protein state model but including a 100 min delay following the translation step. This approach resulted in an acceptable fit (<a href="#pone.0159292.s004">S4B Fig</a>), with a fitted <em>k</em><sub><em>FM*</em></sub> of 0.00015min<sup>-1</sup>. Since we can only obtain a <em>k</em><sub><em>FM</em></sub> with the introduction of a long delay function, we hypothesized that formation of this the non-fluorescent intermediate takes place over significant timescales and thus expanded the model to a three protein state model:</p>
<a id="article1.body1.sec2.sec3.p10" name="article1.body1.sec2.sec3.p10" class="link-target"></a><p>RNA➔ translated protein (<em>P</em><sub><em>T</em></sub>)➔ folded protein (<em>P</em><sub><em>F</em></sub>) ➔ mature protein (<em>P</em><sub><em>M</em></sub>) (Equation in <a href="#pone-0159292-g004">Fig 4B</a>). Using this model, we could fit the data, extracting a separate folding constant <em>k</em><sub><em>F</em></sub> and maturation constant <em>k</em><sub><em>M</em></sub> values of 0.0047min<sup>-1</sup> and 0.0052min<sup>-1</sup>, respectively (<a href="#pone-0159292-g004">Fig 4C</a>). These values were consistent in a full independent repeat of the entire experiment (<a href="#pone.0159292.s004">S4A Fig</a>). Even though the <em>k</em><sub><em>M</em></sub> is not drastically different from above <em>k</em><sub><em>FM</em></sub> values for SF-GFP<sub>s.p.</sub> and E2C<sub>s.p.</sub>, the preceding rate <em>k</em><sub><em>F</em></sub> of formation of the non-fluorescent intermediate is on the same timescale as the separate maturation rate, introducing a delay. Together, folding and maturation yield a much higher t<sub>1/2 fold/mature</sub> for mKO<sub>s.p.</sub>. It is important to note that the fit with the three state protein model was superior to the one with a simple delay (<a href="#pone.0159292.s004">S4B Fig</a>), highlighting that the dynamics of the non-fluorescent intermediate are important to account for. Finally, as in the case for SF-GFP<sub>s.p.</sub> and E2C<sub>s.p.</sub>, the parameters that could reproduce the fits at less than 5% mean fitting error clustered very closely around the reported mean (<a href="#pone.0159292.s003">S3C Fig</a>). All the relevant parameters are summarized in <a href="#pone-0159292-t003">Table 3</a>.</p>
<a id="article1.body1.sec2.sec3.p11" name="article1.body1.sec2.sec3.p11" class="link-target"></a><p>In summary, using simple two or three protein state models, we were able to derive combined folding/maturation rates (SF-GFP<sub>s.p.</sub> and E2C<sub>s.p.</sub>) or separate folding and maturation rates (mKO2<sub>s.p.</sub>) for all three flours in our toolbox, while accounting for the any delays in protein production.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Usefulness of XFPs for imaging of low-abundance protein in S.pombe cells"></a>
<h3>Usefulness of XFPs for imaging of low-abundance protein in <em>S</em>.<em>pombe</em> cells</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>The <em>ade6</em> promoter is fairly weak [<a href="#pone.0159292.ref028" class="ref-tip">28</a>] and thus serves as a useful model of low- to medium low abundant proteins. Using the constructs from <a href="#pone-0159292-g001">Fig 1</a> (<em>ade6p</em>:<em>XFP</em>:<em>ura4t</em>), we imaged live <em>S</em>.<em>pombe</em> cells using a spinning disc confocal microscope to address both signal and intracellular distribution of the XFPs alone (<a href="#pone-0159292-g005">Fig 5</a>). The XFPs were excited by a 488nm laser for SF-GFP, 561nm for mKO2 and 640nm for the E2C constructs. While we imaged each strain alone in <a href="#pone.0159292.s005">S5A Fig</a>, in <a href="#pone-0159292-g005">Fig 5A and 5B</a>, we mixed the three strains and co-visualized them under the same microscope settings. Both <em>ade6p</em> driven-SF-GFP<sub>s.p.</sub> and mKO2<sub>s.p.</sub> yielded bright signals specifically in their respective emission channels (GFP and RFP, <a href="#pone.0159292.s005">S5A Fig</a> and <a href="#pone-0159292-g005">Fig 5A and 5B</a>), and importantly, show a uniform cellular distribution. <em>ade6p</em>:E2C could not be visualized by live-cell microcopy (data not shown), indicating that E2C is less well suited for visualization of low-abundance proteins. In contrast, <em>ade6p</em>:3XE2C could be easily visualized in both the Cy5 channel as well as weakly in the RFP channel (<a href="#pone.0159292.s005">S5A Fig</a> and <a href="#pone-0159292-g005">Fig 5A</a>). This is expected given the emission pre-sets in the microscope used and could be eliminated using different filter sets. 3XE2C<sub>s.p.</sub> yielded some diffuse fluorescence background and is excluded from the nucleus. However, it accumulates in several cellular foci. This does not appear to correlate with cellular toxicity as 3XE2C strains grow at the same rate as the untransformed control (data not shown). This aggregation may be a feature of the 3XE2C<sub>s.p.</sub> protein. To test this, we prepared a construct with a 1XE2C<sub>s.p.</sub> driven by the strong <em>act1</em> promoter (<a href="#pone-0159292-g005">Fig 5B</a>, Cy5). In this case, we observe strong signal coming from <em>act1</em>:1XE2C and importantly, uniform cellular distribution as in the case of SF-GFP<sub>s.p.</sub> and mKO2<sub>s.p.</sub> This observation confirms that the aggregation seen for the <em>ade6p</em>:3XE2C is due to the tandem fusion. To determine whether an endogenous protein will display the correct localization pattern when fused to E2C, we C-terminally tagged the heterochromatin protein Swi6 with 1XE2C<sub>s.p.</sub> using the general construct in <a href="#pone-0159292-g001">Fig 1</a>. Employing the same strategy, we fused Swi6 to SF-GFP<sub>s.p.</sub> as a control, since GFP fusions have been previously validated for Swi6 [<a href="#pone.0159292.ref037" class="ref-tip">37</a>]. We observed that both Swi6:SF-GFP<sub>s.p.</sub> and Swi6:1XE2C<sub>s.p.</sub> display the same and correct localization pattern of several nuclear foci and diffuse nuclear staining with no cytosolic fluorescence [<a href="#pone.0159292.ref038" class="ref-tip">38</a>] (<a href="#pone-0159292-g005">Fig 5C</a>). This observation indicates that 1) 1XE2C C-terminal fusions do not interfere with the normal localization of the native protein and 2) moderately expressed proteins such as Swi6 (~10,000 copies per cell) can be visualized with 1XE2C. To explore co-localization of proteins with these tools within the same cell, we fused Sad1, a spindle pole body (SPB) marker present at ~2000–3000 copies per cell, to mKO2<sub>s.p.</sub> and introduced the <em>sad1</em>:mKO2<sub>s.p.</sub> construct into the <em>swi6</em>:SF-GFP<sub>s.p.</sub> strain. In the RFP channel, we clearly saw a single bright spot marking the SPB (<a href="#pone-0159292-g005">Fig 5D</a>), as expected [<a href="#pone.0159292.ref039" class="ref-tip">39</a>]. In the GFP channel, we saw that Swi6:SF-GFP<sub>s.p.</sub> localizes adjacent to the SPB but also to other nuclear foci that likely represent the heterochromatic telomeric ends; the fusion proteins further exhibits a diffuse nuclear staining (<a href="#pone-0159292-g005">Fig 5D</a>). These data indicate the suitability of our system for co-localization of two or more low- to moderately- expressed proteins in the cell. This is further supported by the little cross bleed we observed even without optimized filter sets for our three fluors.</p>
<a class="link-target" id="pone-0159292-g005" name="pone-0159292-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0159292.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0159292.g005" data-doi="info:doi/10.1371/journal.pone.0159292" data-uri="info:doi/10.1371/journal.pone.0159292.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0159292.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0159292.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0159292.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0159292.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Performance of SF-GFP<sub>s.p.</sub>, mKO2<sub>s.p.</sub> and E2C<sub>s.p.</sub> for low abundance protein detection by microscopy.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p><strong>A.</strong> Untagged <em>ade6</em> promoter driven XFPs. SF-GFP<sub>s.p.,</sub> mKO2<sub>s.p.</sub> and 3XE2C<sub>s.p.</sub> cells were mixed at 1:1:1 and imaged in brightfield and GFP, RFP and Cy5 channels. The merged imaged is shown on the right. Images were taken at 60x magnification. <strong>B.</strong> Untagged <em>ade6p</em>:SF-GFP<sub>s.p.,</sub> <em>ade6p</em>:mKO2<sub>s.p.</sub> <em>act1p</em>:1xE2C<sub>s.p.</sub> promoter driven XFPs. SF-GFP<sub>s.p.,</sub> mKO2<sub>s.p.</sub> and 1XE2C<sub>s.p.</sub> cells were mixed at 1:1:1 and imaged cells were visualized in brightfield, the GFP, RFP and Cy5 channels. The merged imaged is shown on the right. Images were taken at 60x magnification. <strong>C.</strong> Swi6 visualized by SF-GFP<sub>s.p.</sub> or 1XE2C<sub>s.p.</sub> Cells containing either Swi6 C-terminally tagged by SF-GFP<sub>s.p.</sub> or 1XE2C<sub>s.p.</sub> were mixed 1:1 and imaged in brightfield and GFP and Cy5 channels. Arrowheads point to cell nuclei. Images were taken at 60x magnification. <strong>D.</strong> Co-localization of Swi6:SF-GFP<sub>s.p.</sub> and Sad1:mKO2 <sub>s.p.</sub> in the same cell. Cells were visualized in brightfield, the GFP and RFP channels. Arrowheads point to Sad1:mKO2 <sub>s.p.</sub>-marked spindle pole bodies. Image were taken at 100x magnification.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0159292.g005">
              https://doi.org/10.1371/journal.pone.0159292.g005</a></p></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>Our flow cytometry results showed (<a href="#pone-0159292-g001">Fig 1</a>) that consistently, mKO2<sub>s.p.</sub> is the brightest of the three XFPs, and the microscopy analysis indicated that it is readily detected when driven by a low expressed promoter (as is SF-GFP<sub>s.p.</sub>). We wondered whether it can be used to visualize a very low abundance protein that could not be visualized when expressed form its own promoter. Dcr1 is a very low expressed protein and could only be visualized when ectopically expressed from a heterologous promoter that is stronger than the endogenous <em>dcr1</em> promoter [<a href="#pone.0159292.ref040" class="ref-tip">40</a>]. We constructed an N-terminal mKO2 fusion to Dcr1 (<em>dcr1p</em>:mKO2 <sub>s.p.</sub>:<em>Dcr1</em>:<em>dcr1t</em>). We can detect mKO2:Dcr1 over background under this condition, albeit weakly (<a href="#pone.0159292.s005">S5B Fig</a>). This differs from prior attempts, where Dcr1 could only be visualized when expressed from high and intermediate strength versions of a constitutive promoter (<em>nmt1</em>) but not its own promoter or weak constitutive versions of <em>nmt1</em> [<a href="#pone.0159292.ref040" class="ref-tip">40</a>, <a href="#pone.0159292.ref041" class="ref-tip">41</a>]. These data indicate that mKO2 <sub>s.p.</sub>is particularly well suited as a tag for low expressed proteins in <em>S</em>.<em>pombe</em>.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec007" name="sec007" data-toc="sec007" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>In this work we describe an integrated toolbox of XFPs that enables three-color imaging in <em>S</em>. <em>pombe</em>. We also describe a new method to extract the apparent <em>in vivo</em> folding and maturation times for XFPs, which should be widely applicable. We make the following broad observations about the features of the current toolbox: 1. Upon <em>S</em>.<em>pombe</em> codon optimization, all three fluors are easily detectable by flow cytometry when driven by a weak promoter (<em>ade6</em>). 2. The XFPs are spectrally separable, enabling three-color detection in the context of flow cytometry and microscopy. Appropriate filter sets should be easily available to eliminate the remaining bleed from E2C into the RFP channel under the microscope. 3. All three fluors are suitable for localization of endogenous proteins at least when fused C-terminally. 4. SF-GFP and E2C mature relatively quickly <em>in vivo</em> in <em>S</em>.<em>pombe</em>, with maturation halftimes of 25 and 40min, respectively. On the other hand, mKO2 has a longer maturation halftime of around 135min. We believe this is largely due to a long-lived non-fluorescent intermediate state we model as a folded state (see fitting in <a href="#pone-0159292-g004">Fig 4D</a>). 5. SF-GFP and mKO2 are ideal for microscopy imaging for low abundance proteins, as they are easily detected when driven by the <em>ade6</em> promoter. E2C is more appropriate for moderate- to high- abundance proteins as we were not able to detect 1XE2C driven by the <em>ade6</em> promoter by microscopy, but could detect it easily when driven by the <em>act1</em> promoter (high expression) and when fused C-terminally to Swi6 (moderate expression). SF-GFP and E2C are clearly most ideal for experiments aimed at dynamics. For steady state reporter gene approaches, all three XFPs are useful and all can be detected simultaneously. We find that three-color detection can be achieved either with two lasers, where there is some cross bleed particularly from mKO2 into the E2C detection channels (<a href="#pone-0159292-g002">Fig 2B</a> and <a href="#pone.0159292.s001">S1 Fig</a>), or with three lasers, eliminating any significant cross bleed (<a href="#pone-0159292-g002">Fig 2D</a>). SF-GFP and mKO2 are most suitable for steady state microscopic imaging, especially of low abundance proteins. Plasmids allowing straightforward C-terminal XFP tagging of proteins of choice contain kan<sup>R</sup>, nat<sup>R</sup> and hyg<sup>R</sup> cassettes have been made available through Addgene.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>Maturation times between XFPs varies widely, even within the same family [<a href="#pone.0159292.ref010" class="ref-tip">10</a>, <a href="#pone.0159292.ref014" class="ref-tip">14</a>]. The question of XFP maturation times in the system of study is largely unaddressed in most studies, yet knowing the delay between protein production and fluorescence is necessary to interpret time-dependent data. It has widely been assumed the maturation times can be extracted from <em>in vitro</em> measurements, or that maturation parameter derived from one system (typically <em>E</em>.<em>coli</em>) would apply to the system of study. However, given recent findings that there are differences maturation times even between strains of the same organism [<a href="#pone.0159292.ref020" class="ref-tip">20</a>], it is reasonable to consider that maturation times may vary between organisms. This is on one level not surprising, as chromophore formation is pH sensitive [<a href="#pone.0159292.ref022" class="ref-tip">22</a>–<a href="#pone.0159292.ref024" class="ref-tip">24</a>] and widely known to be temperature sensitive [<a href="#pone.0159292.ref021" class="ref-tip">21</a>]. However, at least in the case of GFP, no dependence on other cofactors has been demonstrated [<a href="#pone.0159292.ref042" class="ref-tip">42</a>]. But these two factors alone, pH and growth temperature, can vary considerably between model organisms. Thus, it appears a common method for determining the <em>in vivo</em> maturation rate might be useful to interpret XFP kinetic data. Such a common framework with some variations exists for <em>E</em>.<em>coli</em>, where XFP expression is induced by IPTG and translation is shut off with chloramphenicol [<a href="#pone.0159292.ref012" class="ref-tip">12</a>]. In this pulse-chase scenario, folding and maturation of a defined protein quantity is followed. However, if the protein accumulation is not measured, inducer response and translation will be confounded with folding and maturation. When fluorescence can be measured at high sensitivity, it is also possible to measure maturation times by following the increase over steady state following the addition of a translation block. Yet, this is not always a practical approach as appropriately sensitive equipment is not widely available.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>We utilized a simple approach to measure maturation times <em>in vivo</em> in <em>S</em>.<em>pombe</em>. The approach has the following parts (<a href="#pone-0159292-g002">Fig 2A</a>): Induction of a XFP:5XFLAG message in response to a soluble inducer (uracil) for a short time window (pulse of protein production), measurement of protein and fluorescence accumulation, and finally fitting of folding and maturation kinetics. While conceptually any inducible system should do, a unique and useful feature of the <em>urg1</em> gene system we use is that the message is very unstable following uracil removal (<a href="#pone-0159292-g003">Fig 3</a> and [<a href="#pone.0159292.ref035" class="ref-tip">35</a>]), guaranteeing a defined burst of message production. By simultaneously integrating total protein information into the fluorescence fit (see <a href="#sec008">methods</a>), we are able to account for any delays in induction and consequentially, protein production, which might arise from experiment to experiment or strain to strain variation. The importance of this approach is validated by the slow protein induction of SF-GFP as opposed to E2C or mKO2 (Figs <a href="#pone-0159292-g003">3C</a> and <a href="#pone-0159292-g004">4A</a> bottom). Importantly, using a mathematical approach we were able to show that the slow appearance of mKO2 fluorescence following induction is the result of the slow production of a non-fluorescent intermediate as well as a relatively slow maturation rate. We expanded the model to a three protein state model for mKO2 and we believe such a model delivers meaningful <em>in vivo</em> of <em>k</em><sub><em>F</em></sub> and <em>k</em><sub><em>M</em></sub> values given the highly constrained parameter space these variables can adopt resulting in robust data fits (<a href="#pone.0159292.s003">S3 Fig</a>). Given that the rates obtained here are derived in the context of arbitrary fluorescence units, they cannot be directly compared to data obtained under experimental conditions divergent from those described here. However, they are powerful in allowing quantitative comparison between fluors using internally consistent experimental approach.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>The experimental approach should be easy to replicate for other XFP proteins in <em>S</em>.<em>pombe</em>, and for other biological systems, such as <em>Saccharomyces cerevisiae</em> that for which appropriate induction systems are available.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec008" name="sec008" data-toc="sec008" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Strain source, construction and growth"></a>
<h3>Strain source, construction and growth</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>All parent <em>S</em>. <em>pombe</em> strains used for this study are derivatives of publically available strains. The genotypes of all strains used in this study can be found in <a href="#pone-0159292-t001">Table 1</a>. The <em>ade6p</em>:XFP:<em>ura4t</em>:hygMX and <em>act1p</em>:XFP:<em>ura4t</em>:hygMX cassettes were produced using <em>in vivo</em> recombination techniques and introduced in between SPBC1711.11 and SPBC1711.12 in the BAS99 background. To produce <em>urg1p</em>:XFP:5xFLAG strains, we first disrupted the <em>Urg1</em> open reading frame with a KanMX cassette (Δ<em>urg1</em>::KanMX). We replaced the KanMX cassette with a <em>urg1p</em>:XFP:5xFLAG construct that carried a NatMX cassette. To minimize interfere of the MX cassette with the <em>urg1</em> locus, we included in the construct 1.6kb of <em>urg1</em> 3’ UTR downstream of the XFP insertion site before the beginning of the MX cassette. To produce mKO<sub>s.p.</sub>:Dcr1 fusion proteins driven from the endogenous <em>dcr1</em> promoter, we inserted a KanMX marked construct upstream of <em>Dcr1</em> that contained mKO2<sub>s.p.</sub> inserted between the endogenous <em>dcr1</em> promoter and the <em>Dcr1</em> ORF. To produce <em>swi6</em>::<em>swi6</em>:1XE2C, <em>swi6</em>::<em>swi6</em>:SF-GFP and <em>sad1</em>::<em>sad1</em>:mKO2, the appropriate cassette in <a href="#pone-0159292-g001">Fig 1</a> was PCR amplified with 80bp homology primers and inserted into the endogenous locus by homologous recombination. The double labeled <em>swi6</em>::<em>swi6</em>:SF-GFP; <em>sad1</em>::<em>sad1</em>:mKO2 strain was produced by a genetic cross. For all experiments, <em>S</em>.<em>pombe</em> cells were grown in EMM media at 32°C.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Flow cytometry"></a>
<h3>Flow cytometry</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>All strains were grown in EMM media overnight to about ~OD 1.5 and were back diluted to OD 0.1 and grown for an additional 3-5hrs before flow cytometry. All analysis was performed in the UCSF Parnassus Flow Cytometry core on Beckton Dickinson Flow LSRII or Fortessa cytometers outfitted with 488nm, 532 or 561nm, and 640nm lasers. PMT voltages were adjusted as to yield a fluorescence value of 10<sup>2</sup> for untransformed cells in each channel. Cells were gated for single recently divided cells by their forward and side scatter characteristics and their mean fluorescence values reported.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="Uracil induction"></a>
<h3>Uracil induction</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Cells were grown through the day in EMM-uracil to about OD 0.5 and diluted to 0.01 for overnight growth. The next morning, cells were typically in early log phase and induced for 20min with 0.25mg/ml uracil. Uracil was removed by centrifugation of the induced cultures, one wash in PBS, and reintroduction of EMM-uracil media.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="Western blots"></a>
<h3>Western blots</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>For uracil induction time-courses, cells were flash-frozen and total protein extracted using previously described methods [<a href="#pone.0159292.ref043" class="ref-tip">43</a>]. Low-fluorescence PVDF membranes (Bio-rad) were probed with polyclonal anti-FLAG (Abgent AP1013a) and monoclonal anti GAPDH (Thermo Scientific MA5-15738) primary antisera simultaneously, detected by IR-dye linked 680 and 800nm secondary antibodies and antibody fluorescence analyzed as described [<a href="#pone.0159292.ref044" class="ref-tip">44</a>]. FLAG signals were normalized to GAPDH loading control signals.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="Real-time qPCR"></a>
<h3>Real-time qPCR</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>Total RNA was extracted from yeast cells using the Masterpure yeast total RNA kit (Illumina). 3μg RNA was reverse transcribed with Superscript III reverse transcriptase (Invitrogen). RT-qPCR was performed as described using an actin internal control [<a href="#pone.0159292.ref044" class="ref-tip">44</a>, <a href="#pone.0159292.ref045" class="ref-tip">45</a>]</p>
</div>

<div id="section6" class="section toc-section"><a id="sec014" name="sec014" class="link-target" title="Fitting"></a>
<h3>Fitting</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>For SF-GFP and E2C, we derived the following overall equations for a two state consecutive reaction model following uracil induction:</p>
<a id="article1.body1.sec4.sec6.p2" name="article1.body1.sec4.sec6.p2" class="link-target"></a><p>XFP RNA (<em>RNA</em>) ➔ XFP total protein (<em>P</em><sub><em>T</em></sub>) ➔ XFP folded/matured (<em>P</em><sub><em>FM</em></sub>):</p>
<a id="article1.body1.sec4.sec6.p3" name="article1.body1.sec4.sec6.p3" class="link-target"></a><p>In this two protein state model, <em>P</em><sub><em>T</em></sub> <em>= P</em><sub><em>U</em></sub> <em>+ P</em><sub><em>FM</em></sub>, where <em>P</em><sub><em>T</em></sub> is total protein, <em>P</em><sub><em>U</em></sub> is unfolded, non-fluorescent protein, <em>P</em><sub><em>FM</em></sub> is folded and mature fluroescent protein. We directly measure <em>P</em><sub><em>T</em></sub> and <em>P</em><sub><em>FM</em></sub>. The equations describing these sequential reactions are described by the following Ordinary Differential Equations (ODEs):
<a name="pone.0159292.e001" id="pone.0159292.e001" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e001" class="inline-graphic"></span>
<a name="pone.0159292.e002" id="pone.0159292.e002" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e002" class="inline-graphic"></span>
<a name="pone.0159292.e003" id="pone.0159292.e003" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e003" class="inline-graphic"></span></p>
<a id="article1.body1.sec4.sec6.p4" name="article1.body1.sec4.sec6.p4" class="link-target"></a><p>In this model, <em>u</em> represents the rate constant of mRNA induction by uracil, and <em>α</em><sub><em>m</em></sub> the rate constant of RNA turnover. <em>P</em><sub><em>T</em></sub> denotes total protein as measured by western blotting. <em>k</em><sub><em>P</em></sub> and <em>α</em><sub><em>p</em></sub> are the rate constants of translation and total protein degradation, respectively. <em>P</em><sub><em>M</em></sub> denotes matured protein as measured by fluorescence. <em>k</em><sub><em>FM</em></sub> and <em>β</em><sub><em>FM</em></sub> are the rate constants of folding/maturation and fluorescent protein degradation, respectively. <em>θ</em> is the scaling factor between protein units (FLAG/GAPDH) and fluorescence units (arbitrary fluorescence units).</p>
<a id="article1.body1.sec4.sec6.p5" name="article1.body1.sec4.sec6.p5" class="link-target"></a><p>We fit the total protein time course data to equation <em>(2)</em> and the fluorescence time course data to <em>(3)</em>. In the fitting for <em>(3)</em>, we use the <em>P</em><sub><em>T</em></sub> values from the fit of the total protein data to <em>(2)</em> as inputs.</p>
<a id="article1.body1.sec4.sec6.p6" name="article1.body1.sec4.sec6.p6" class="link-target"></a><p>For SF-GFP, given the delayed kinetics of protein appearance (see <a href="#sec002">results</a>) we introduced a step depicting transitions between closed to open chromatin preceding transcription. In this model, chromatin transition to the open state (forward arrow) is influenced by the uracil input (<em>u</em>), while transition to the closed state (back arrow) is governed by <em>k</em><sub><em>c</em></sub>.</p>
<a name="pone.0159292.e004" id="pone.0159292.e004" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e004" class="inline-graphic"></span><a id="article1.body1.sec4.sec6.p7" name="article1.body1.sec4.sec6.p7" class="link-target"></a><p><em>C</em><sub><em>c</em></sub> is the closed, non-induction competent state, and <em>C</em><sub><em>o</em></sub> is the open, induction competent state. To model this step, we introduce an additional equation for the dynamics of the open chromatin state:
<a name="pone.0159292.e005" id="pone.0159292.e005" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e005" class="inline-graphic"></span></p>
<a id="article1.body1.sec4.sec6.p8" name="article1.body1.sec4.sec6.p8" class="link-target"></a><p>Also using that <em>C</em><sub><em>o</em></sub> + <em>C</em><sub><em>c</em></sub> = 1. Since transcription is only possible from <em>C</em><sub><em>o</em></sub>, We modified equation <em>(1)</em> above for SF-GFP:
<a name="pone.0159292.e006" id="pone.0159292.e006" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e006" class="inline-graphic"></span></p>
<a id="article1.body1.sec4.sec6.p9" name="article1.body1.sec4.sec6.p9" class="link-target"></a><p>Where <em>k</em><sub><em>o</em></sub> is the rate of transcription from the open chromatin state.</p>
<a id="article1.body1.sec4.sec6.p10" name="article1.body1.sec4.sec6.p10" class="link-target"></a><p>For mKO2, since a model aggregating maturation and folding of the fluorescent protein could not reproduce the data, we expanded the model to include separate steps for these processing, resulting in a three protein sequential reaction model following uracil induction:</p>
<a id="article1.body1.sec4.sec6.p11" name="article1.body1.sec4.sec6.p11" class="link-target"></a><p>mKO2 RNA (<em>RNA</em>) ➔ total protein (<em>P</em><sub><em>T</em></sub>) ➔ folded protein (<em>P</em><sub><em>F</em></sub>) ➔ mKO2 mature protein (<em>P</em><sub><em>M</em></sub>)</p>
<a id="article1.body1.sec4.sec6.p12" name="article1.body1.sec4.sec6.p12" class="link-target"></a><p>In this protein state model, <em>P</em><sub><em>T</em></sub> <em>= P</em><sub><em>U</em></sub> <em>+ P</em><sub><em>F</em></sub> <em>+ P</em><sub><em>M</em></sub>, where <em>P</em><sub><em>T</em></sub> is total protein, <em>P</em><sub><em>U</em></sub> is unfolded, non-fluorescent protein, <em>P</em><sub><em>F</em></sub> is folded, non-fluorescent protein and <em>P</em><sub><em>M</em></sub> is mature, fluroescent protein.</p>
<a name="pone.0159292.e007" id="pone.0159292.e007" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e007" class="inline-graphic"></span><a name="pone.0159292.e008" id="pone.0159292.e008" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e008" class="inline-graphic"></span><a name="pone.0159292.e009" id="pone.0159292.e009" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e009" class="inline-graphic"></span><a name="pone.0159292.e010" id="pone.0159292.e010" class="link-target"></a><span class="equation"><img src="article/file?type=thumbnail&amp;id=info:doi/10.1371/journal.pone.0159292.e010" class="inline-graphic"></span><a id="article1.body1.sec4.sec6.p13" name="article1.body1.sec4.sec6.p13" class="link-target"></a><p>The fitting was performed using a customized script within the MatLab (Mathworks) software. Briefly, we sought to find a large set of parameters that can fit the data within a pre-defined error tolerance. In this case, we defined the error tolerance as the mean of the absolute value of all the errors between the fit and individual data points. We set this tolerance to 5%. To generate the model parameters that can fit the data, we fed different initial guesses to the function fminsearch, which uses a nonlinear Nelder-Mead algorithm to find minima of a function. We then used the successful guesses as “seeds” for another algorithm that expands around the successful parameters sets, trying different combinations and storing the ones that satisfy the error tolerance.</p>
</div>

<div id="section7" class="section toc-section"><a id="sec015" name="sec015" class="link-target" title="Microscopy"></a>
<h3>Microscopy</h3>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1" class="link-target"></a><p><em>S</em>.<em>pombe</em> cells were grown as above for flow cytometry measurements. Glass slides were coated with concanavalin A, were cells applied to the slide and imaged using a Yokagawa CSU22 Spinning Disc confocal fitted Nikon Ti-E microscope. In the “DAPI” channel (for DAPI), excitation was performed by a 405nm laser and the emission preset was 460/50nm. In the “GFP” channel (for SF-GFP) excitation was performed by a 488nm laser and the emission preset was 525/50nm. In the “RFP” channel (for mKO2 and E2C) excitation was performed by a 561nm laser and the emission preset was 610/60nm. In the “Cy5” channel (for E2C) excitation was performed by a 640nm laser and the emission preset was 700/75nm.</p>
</div>

<div id="section8" class="section toc-section"><a id="sec016" name="sec016" class="link-target" title="Image processing"></a>
<h3>Image processing</h3>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1" class="link-target"></a><p>Images were false colored in ImageJ software and merged. The brightness adjusted for the separate channels so no background color (intensity outside the cell) dominates in the merged image. Channels were then split and images saved as jpegs. Contrast was adjusted for clarity across the entire image. For mKO2:Dcr1 cell images, the RFP channel was smoothened given the low signal.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="sec017" name="sec017" data-toc="sec017" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0159292"></div><div class="supplementary-material"><a name="pone.0159292.s001" id="pone.0159292.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0159292.s001">S1 Fig. </a>Compensation for cross-bleed using 488nm/561nm/561nm excitation regime.</h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">Shown on the left column are uncompensated, raw histograms of unstained, SF-GFP<sub>s.p.</sub>, mKO2<sub>s.p.</sub> and 3xE2C<sub>s.p.</sub> cells with the emissions filter indicated on the bottom excited with either the 561nm (TOP, BOTTOM) or 488nm (MIDDLE) laser. Significant bleed from mKO2<sub>s.p.</sub> (orange arrows)into the E3C channel and slight bleed into the SF-GFP channel is evident. The right column shows the corresponding histograms following compensation by the Becton Dickinson FACSDiva software. All cross bleed by mKO2<sub>s.p.</sub> is eliminated.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0159292.s001">https://doi.org/10.1371/journal.pone.0159292.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0159292.s002" id="pone.0159292.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0159292.s002">S2 Fig. </a>Compensation for cross-bleed using 488nm/561nm/640nm excitation regime.</h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">As in <a href="#pone.0159292.s001">S1 Fig</a>, expect that excitation was performed by three lasers (488 for SF-GFP, 561 for mKO2 and 640 for 3XE2C). Only slight bleed from mKO2<sub>s.p.</sub> into the SF-GFP channel is evident and eliminated upon compensation.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0159292.s002">https://doi.org/10.1371/journal.pone.0159292.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0159292.s003" id="pone.0159292.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0159292.s003">S3 Fig. </a>Parameter distributions for fits in <a href="#pone-0159292-g004">Fig 4</a>.</h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>A.</strong> Parameters distributions for E2C<sub>s.p.</sub> total protein (LEFT) and fluorescence (RIGHT) fits that yield a fit with &lt;5% mean fitting error. <strong>B.</strong> Parameters distributions for SF-GFP<sub>s.p.</sub> total protein including open and closed chromatin transitions, (LEFT) and fluorescence (RIGHT) fits that yield a fit with &lt;5% mean fitting error. <strong>A.</strong> Parameters distributions for mKO2<sub>s.p.</sub> total protein (LEFT) and fluorescence (RIGHT) fits that yield a fit with &lt;5% mean fitting error. All <em>α</em>, <em>β</em> and <em>k</em> values are in min<sup>-1</sup>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0159292.s003">https://doi.org/10.1371/journal.pone.0159292.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0159292.s004" id="pone.0159292.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0159292.s004">S4 Fig. </a>Second run of mKO2 induction and alternative fitting approach.</h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>A.</strong> Shown is a second repeat of uracil pulse induction of <em>urg1p</em>::mKO2<sub>s.p.</sub> Expression was induced, protein and fluorescence analyzed and fit as in <a href="#pone-0159292-g004">Fig 4B</a> using the three protein state model. The mean fit is shown in color, the gray lines are all possible fits. <strong>B.</strong> Fitting mKO2<sub>s.p.</sub> fluorescence with a 2 protein state model with a delay function. We derived a single <em>k</em><sub><em>F/M*</em></sub> form the same raw data as shown in <a href="#pone-0159292-g003">Fig 3</a> and <a href="#pone-0159292-g004">Fig 4B</a> using a two protein state model and delay.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0159292.s004">https://doi.org/10.1371/journal.pone.0159292.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0159292.s005" id="pone.0159292.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0159292.s005">S5 Fig. </a>Single channel microscopy with SF-GFP, mKO2 and 3XE2C and visualization of Dcr1 with mKO2.</h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>A.</strong> Untagged <em>ade6</em> promoter driven XFPs. <strong>TOP:</strong> SF-GFP<sub>s.p.</sub> cells were visualized in brightfield, the GFP, RFP and Cy5 channels. <strong>Middle:</strong> mKO2<sub>s.p.</sub> cells were visualized in brightfield, the GFP, RFP and Cy5 channels. Images were taken at 60x magnification. <strong>Bottom:</strong> 3XE2C<sub>s.p.</sub> cells were visualized in brightfield, the GFP, RFP and Cy5 channels. <strong>B.</strong> C-terminally Visualization of mKO2:Dcr1 driven by its endogenous promoter. mKO2 was inserted between the <em>dcr1</em> promoter and the <em>dcr1</em> open reading frame and tagged upstream with a G418 resistance cassette. Cells were stained with DAPI and visualized in the DAPI and RFP channel. Two fields of cells are shown. Arrowheads denote location of cell nuclei. Image were taken at 100x magnification.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0159292.s005">https://doi.org/10.1371/journal.pone.0159292.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0159292.s006" id="pone.0159292.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0159292.s006">S1 File. </a>Alignments of <em>S</em>.<em>pombe</em>. (<em>s</em>.<em>p</em>.) codon optimized to non-optimized sequences.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0159292.s006">https://doi.org/10.1371/journal.pone.0159292.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank Jacob Stewart-Ornstein for his gift of several of the initial XFP plasmids. We thank Kurt Thorn and Delaine Larsen at the UCSF Nikon Imaging Center for guidance and training and additionally thank Stéphane Camus for initial confocal microscopy training.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: BA-S RAG HJE-S HDM SB. Performed the experiments: BA-S RAG. Analyzed the data: BA-S HJE-S. Contributed reagents/materials/analysis tools: BA-S HJE-S SB. Wrote the paper: BA-S SB HDM.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0159292.ref001" id="pone.0159292.ref001" class="link-target"></a>Shaner NC, Steinbach PA, Tsien RY. A guide to choosing fluorescent proteins. Nature methods. 2005;2(12):905–9.  pmid:16299475 <ul class="reflinks"><li><a href="#" data-author="Shaner" data-cit="ShanerNC%2C%20SteinbachPA%2C%20TsienRY.%20A%20guide%20to%20choosing%20fluorescent%20proteins.%20Nature%20methods.%202005%3B2%2812%29%3A905%E2%80%939.%20doi%3A%2010.1038%2Fnmeth819%2016299475" data-title="A%20guide%20to%20choosing%20fluorescent%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16299475" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+guide+to+choosing+fluorescent+proteins+Shaner+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0159292.ref002" id="pone.0159292.ref002" class="link-target"></a>Lee S, Lim WA, Thorn KS. Improved blue, green, and red fluorescent protein tagging vectors for S. cerevisiae. PloS one. 2013;8(7):e67902.  pmid:23844123; PubMed Central PMCID: PMC3699464. <ul class="reflinks"><li><a href="#" data-author="Lee" data-cit="LeeS%2C%20LimWA%2C%20ThornKS.%20Improved%20blue%2C%20green%2C%20and%20red%20fluorescent%20protein%20tagging%20vectors%20for%20S.%20cerevisiae.%20PloS%20one.%202013%3B8%287%29%3Ae67902.%20doi%3A%2010.1371%2Fjournal.pone.0067902%2023844123%3B%20PubMed%20Central%20PMCID%3A%20PMC3699464." data-title="Improved%20blue%2C%20green%2C%20and%20red%20fluorescent%20protein%20tagging%20vectors%20for%20S.%20cerevisiae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23844123" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Improved+blue%2C+green%2C+and+red+fluorescent+protein+tagging+vectors+for+S.+cerevisiae+Lee+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0159292.ref003" id="pone.0159292.ref003" class="link-target"></a>Snaith HA, Anders A, Samejima I, Sawin KE. New and old reagents for fluorescent protein tagging of microtubules in fission yeast; experimental and critical evaluation. Methods Cell Biol. 2010;97:147–72.  pmid:20719270. <ul class="reflinks"><li><a href="#" data-author="Snaith" data-cit="SnaithHA%2C%20AndersA%2C%20SamejimaI%2C%20SawinKE.%20New%20and%20old%20reagents%20for%20fluorescent%20protein%20tagging%20of%20microtubules%20in%20fission%20yeast%3B%20experimental%20and%20critical%20evaluation.%20Methods%20Cell%20Biol.%202010%3B97%3A147%E2%80%9372.%20doi%3A%2010.1016%2FS0091-679X%2810%2997009-X%2020719270." data-title="New%20and%20old%20reagents%20for%20fluorescent%20protein%20tagging%20of%20microtubules%20in%20fission%20yeast%3B%20experimental%20and%20critical%20evaluation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20719270" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=New+and+old+reagents+for+fluorescent+protein+tagging+of+microtubules+in+fission+yeast%3B+experimental+and+critical+evaluation+Snaith+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0159292.ref004" id="pone.0159292.ref004" class="link-target"></a>Snaith HA, Samejima I, Sawin KE. Multistep and multimode cortical anchoring of tea1p at cell tips in fission yeast. The EMBO journal. 2005;24(21):3690–9.  pmid:16222337; PubMed Central PMCID: PMC1276721. <ul class="reflinks"><li><a href="#" data-author="Snaith" data-cit="SnaithHA%2C%20SamejimaI%2C%20SawinKE.%20Multistep%20and%20multimode%20cortical%20anchoring%20of%20tea1p%20at%20cell%20tips%20in%20fission%20yeast.%20The%20EMBO%20journal.%202005%3B24%2821%29%3A3690%E2%80%939.%20doi%3A%2010.1038%2Fsj.emboj.7600838%2016222337%3B%20PubMed%20Central%20PMCID%3A%20PMC1276721." data-title="Multistep%20and%20multimode%20cortical%20anchoring%20of%20tea1p%20at%20cell%20tips%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16222337" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Multistep+and+multimode+cortical+anchoring+of+tea1p+at+cell+tips+in+fission+yeast+Snaith+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0159292.ref005" id="pone.0159292.ref005" class="link-target"></a>Bjerling P, Olsson I, Meng X. Quantitative live cell fluorescence-microscopy analysis of fission yeast. J Vis Exp. 2012;(59).  pmid:22297579; PubMed Central PMCID: PMC3462563. <ul class="reflinks"><li><a href="#" data-author="Bjerling" data-cit="BjerlingP%2C%20OlssonI%2C%20MengX.%20Quantitative%20live%20cell%20fluorescence-microscopy%20analysis%20of%20fission%20yeast.%20J%20Vis%20Exp.%202012%3B%2859%29.%20doi%3A%2010.3791%2F3454%2022297579%3B%20PubMed%20Central%20PMCID%3A%20PMC3462563." data-title="Quantitative%20live%20cell%20fluorescence-microscopy%20analysis%20of%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22297579" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Quantitative+live+cell+fluorescence-microscopy+analysis+of+fission+yeast+Bjerling+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0159292.ref006" id="pone.0159292.ref006" class="link-target"></a>Sato M, Dhut S, Toda T. New drug-resistant cassettes for gene disruption and epitope tagging in Schizosaccharomyces pombe. Yeast. 2005;22(7):583–91.  pmid:15942936. <ul class="reflinks"><li><a href="#" data-author="Sato" data-cit="SatoM%2C%20DhutS%2C%20TodaT.%20New%20drug-resistant%20cassettes%20for%20gene%20disruption%20and%20epitope%20tagging%20in%20Schizosaccharomyces%20pombe.%20Yeast.%202005%3B22%287%29%3A583%E2%80%9391.%20doi%3A%2010.1002%2Fyea.1233%2015942936." data-title="New%20drug-resistant%20cassettes%20for%20gene%20disruption%20and%20epitope%20tagging%20in%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15942936" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=New+drug-resistant+cassettes+for+gene+disruption+and+epitope+tagging+in+Schizosaccharomyces+pombe+Sato+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0159292.ref007" id="pone.0159292.ref007" class="link-target"></a>Ahn J, Choi CH, Kang CM, Kim CH, Park HM, Song KB, et al. Generation of expression vectors for high-throughput functional analysis of target genes in Schizosaccharomyces pombe. J Microbiol. 2009;47(6):789–95.  pmid:20127475. <ul class="reflinks"><li><a href="#" data-author="Ahn" data-cit="AhnJ%2C%20ChoiCH%2C%20KangCM%2C%20KimCH%2C%20ParkHM%2C%20SongKB%2C%20et%20al.%20Generation%20of%20expression%20vectors%20for%20high-throughput%20functional%20analysis%20of%20target%20genes%20in%20Schizosaccharomyces%20pombe.%20J%20Microbiol.%202009%3B47%286%29%3A789%E2%80%9395.%20doi%3A%2010.1007%2Fs12275-009-0010-4%2020127475." data-title="Generation%20of%20expression%20vectors%20for%20high-throughput%20functional%20analysis%20of%20target%20genes%20in%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20127475" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Generation+of+expression+vectors+for+high-throughput+functional+analysis+of+target+genes+in+Schizosaccharomyces+pombe+Ahn+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0159292.ref008" id="pone.0159292.ref008" class="link-target"></a>Johnson M, East DA, Mulvihill DP. Formins determine the functional properties of actin filaments in yeast. Current biology: CB. 2014;24(13):1525–30.  pmid:24954052. <ul class="reflinks"><li><a href="#" data-author="Johnson" data-cit="JohnsonM%2C%20EastDA%2C%20MulvihillDP.%20Formins%20determine%20the%20functional%20properties%20of%20actin%20filaments%20in%20yeast.%20Current%20biology%3A%20CB.%202014%3B24%2813%29%3A1525%E2%80%9330.%20doi%3A%2010.1016%2Fj.cub.2014.05.034%2024954052." data-title="Formins%20determine%20the%20functional%20properties%20of%20actin%20filaments%20in%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24954052" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Formins+determine+the+functional+properties+of+actin+filaments+in+yeast+Johnson+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0159292.ref009" id="pone.0159292.ref009" class="link-target"></a>Warren DT, Andrews PD, Gourlay CW, Ayscough KR. Sla1p couples the yeast endocytic machinery to proteins regulating actin dynamics. J Cell Sci. 2002;115(Pt 8):1703–15. pmid:11950888. <ul class="reflinks"><li><a href="#" data-author="Warren" data-cit="WarrenDT%2C%20AndrewsPD%2C%20GourlayCW%2C%20AyscoughKR.%20Sla1p%20couples%20the%20yeast%20endocytic%20machinery%20to%20proteins%20regulating%20actin%20dynamics.%20J%20Cell%20Sci.%202002%3B115%28Pt%208%29%3A1703%E2%80%9315.%2011950888." data-title="Sla1p%20couples%20the%20yeast%20endocytic%20machinery%20to%20proteins%20regulating%20actin%20dynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11950888" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sla1p+couples+the+yeast+endocytic+machinery+to+proteins+regulating+actin+dynamics+Warren+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0159292.ref010" id="pone.0159292.ref010" class="link-target"></a>Iizuka R, Yamagishi-Shirasaki M, Funatsu T. Kinetic study of de novo chromophore maturation of fluorescent proteins. Anal Biochem. 2011;414(2):173–8.  pmid:21459075. <ul class="reflinks"><li><a href="#" data-author="Iizuka" data-cit="IizukaR%2C%20Yamagishi-ShirasakiM%2C%20FunatsuT.%20Kinetic%20study%20of%20de%20novo%20chromophore%20maturation%20of%20fluorescent%20proteins.%20Anal%20Biochem.%202011%3B414%282%29%3A173%E2%80%938.%20doi%3A%2010.1016%2Fj.ab.2011.03.036%2021459075." data-title="Kinetic%20study%20of%20de%20novo%20chromophore%20maturation%20of%20fluorescent%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21459075" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Kinetic+study+of+de+novo+chromophore+maturation+of+fluorescent+proteins+Iizuka+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0159292.ref011" id="pone.0159292.ref011" class="link-target"></a>Baird GS, Zacharias DA, Tsien RY. Biochemistry, mutagenesis, and oligomerization of DsRed, a red fluorescent protein from coral. Proceedings of the National Academy of Sciences of the United States of America. 2000;97(22):11984–9.  pmid:11050229; PubMed Central PMCID: PMC17281. <ul class="reflinks"><li><a href="#" data-author="Baird" data-cit="BairdGS%2C%20ZachariasDA%2C%20TsienRY.%20Biochemistry%2C%20mutagenesis%2C%20and%20oligomerization%20of%20DsRed%2C%20a%20red%20fluorescent%20protein%20from%20coral.%20Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America.%202000%3B97%2822%29%3A11984%E2%80%939.%20doi%3A%2010.1073%2Fpnas.97.22.11984%2011050229%3B%20PubMed%20Central%20PMCID%3A%20PMC17281." data-title="Biochemistry%2C%20mutagenesis%2C%20and%20oligomerization%20of%20DsRed%2C%20a%20red%20fluorescent%20protein%20from%20coral" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11050229" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Biochemistry%2C+mutagenesis%2C+and+oligomerization+of+DsRed%2C+a+red+fluorescent+protein+from+coral+Baird+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0159292.ref012" id="pone.0159292.ref012" class="link-target"></a>Bevis BJ, Glick BS. Rapidly maturing variants of the Discosoma red fluorescent protein (DsRed). Nature biotechnology. 2002;20(1):83–7.  pmid:11753367. <ul class="reflinks"><li><a href="#" data-author="Bevis" data-cit="BevisBJ%2C%20GlickBS.%20Rapidly%20maturing%20variants%20of%20the%20Discosoma%20red%20fluorescent%20protein%20%28DsRed%29.%20Nature%20biotechnology.%202002%3B20%281%29%3A83%E2%80%937.%20doi%3A%2010.1038%2Fnbt0102-83%2011753367." data-title="Rapidly%20maturing%20variants%20of%20the%20Discosoma%20red%20fluorescent%20protein%20%28DsRed%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11753367" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Rapidly+maturing+variants+of+the+Discosoma+red+fluorescent+protein+%28DsRed%29+Bevis+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0159292.ref013" id="pone.0159292.ref013" class="link-target"></a>Pedelacq JD, Cabantous S, Tran T, Terwilliger TC, Waldo GS. Engineering and characterization of a superfolder green fluorescent protein. Nature biotechnology. 2006;24(1):79–88.  pmid:16369541. <ul class="reflinks"><li><a href="#" data-author="Pedelacq" data-cit="PedelacqJD%2C%20CabantousS%2C%20TranT%2C%20TerwilligerTC%2C%20WaldoGS.%20Engineering%20and%20characterization%20of%20a%20superfolder%20green%20fluorescent%20protein.%20Nature%20biotechnology.%202006%3B24%281%29%3A79%E2%80%9388.%20doi%3A%2010.1038%2Fnbt1172%2016369541." data-title="Engineering%20and%20characterization%20of%20a%20superfolder%20green%20fluorescent%20protein" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16369541" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Engineering+and+characterization+of+a+superfolder+green+fluorescent+protein+Pedelacq+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0159292.ref014" id="pone.0159292.ref014" class="link-target"></a>Shaner NC, Lin MZ, McKeown MR, Steinbach PA, Hazelwood KL, Davidson MW, et al. Improving the photostability of bright monomeric orange and red fluorescent proteins. Nature methods. 2008;5(6):545–51.  pmid:18454154; PubMed Central PMCID: PMC2853173. <ul class="reflinks"><li><a href="#" data-author="Shaner" data-cit="ShanerNC%2C%20LinMZ%2C%20McKeownMR%2C%20SteinbachPA%2C%20HazelwoodKL%2C%20DavidsonMW%2C%20et%20al.%20Improving%20the%20photostability%20of%20bright%20monomeric%20orange%20and%20red%20fluorescent%20proteins.%20Nature%20methods.%202008%3B5%286%29%3A545%E2%80%9351.%20doi%3A%2010.1038%2Fnmeth.1209%2018454154%3B%20PubMed%20Central%20PMCID%3A%20PMC2853173." data-title="Improving%20the%20photostability%20of%20bright%20monomeric%20orange%20and%20red%20fluorescent%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18454154" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Improving+the+photostability+of+bright+monomeric+orange+and+red+fluorescent+proteins+Shaner+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0159292.ref015" id="pone.0159292.ref015" class="link-target"></a>Nagai T, Ibata K, Park ES, Kubota M, Mikoshiba K, Miyawaki A. A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications. Nature biotechnology. 2002;20(1):87–90.  pmid:11753368. <ul class="reflinks"><li><a href="#" data-author="Nagai" data-cit="NagaiT%2C%20IbataK%2C%20ParkES%2C%20KubotaM%2C%20MikoshibaK%2C%20MiyawakiA.%20A%20variant%20of%20yellow%20fluorescent%20protein%20with%20fast%20and%20efficient%20maturation%20for%20cell-biological%20applications.%20Nature%20biotechnology.%202002%3B20%281%29%3A87%E2%80%9390.%20doi%3A%2010.1038%2Fnbt0102-87%2011753368." data-title="A%20variant%20of%20yellow%20fluorescent%20protein%20with%20fast%20and%20efficient%20maturation%20for%20cell-biological%20applications" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11753368" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+variant+of+yellow+fluorescent+protein+with+fast+and+efficient+maturation+for+cell-biological+applications+Nagai+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0159292.ref016" id="pone.0159292.ref016" class="link-target"></a>Charvin G, Cross FR, Siggia ED. A microfluidic device for temporally controlled gene expression and long-term fluorescent imaging in unperturbed dividing yeast cells. PloS one. 2008;3(1):e1468.  pmid:18213377; PubMed Central PMCID: PMC2194624. <ul class="reflinks"><li><a href="#" data-author="Charvin" data-cit="CharvinG%2C%20CrossFR%2C%20SiggiaED.%20A%20microfluidic%20device%20for%20temporally%20controlled%20gene%20expression%20and%20long-term%20fluorescent%20imaging%20in%20unperturbed%20dividing%20yeast%20cells.%20PloS%20one.%202008%3B3%281%29%3Ae1468.%20doi%3A%2010.1371%2Fjournal.pone.0001468%2018213377%3B%20PubMed%20Central%20PMCID%3A%20PMC2194624." data-title="A%20microfluidic%20device%20for%20temporally%20controlled%20gene%20expression%20and%20long-term%20fluorescent%20imaging%20in%20unperturbed%20dividing%20yeast%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18213377" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+microfluidic+device+for+temporally+controlled+gene+expression+and+long-term+fluorescent+imaging+in+unperturbed+dividing+yeast+cells+Charvin+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0159292.ref017" id="pone.0159292.ref017" class="link-target"></a>Khmelinskii A, Keller PJ, Bartosik A, Meurer M, Barry JD, Mardin BR, et al. Tandem fluorescent protein timers for in vivo analysis of protein dynamics. Nature biotechnology. 2012;30(7):708–14.  pmid:22729030. <ul class="reflinks"><li><a href="#" data-author="Khmelinskii" data-cit="KhmelinskiiA%2C%20KellerPJ%2C%20BartosikA%2C%20MeurerM%2C%20BarryJD%2C%20MardinBR%2C%20et%20al.%20Tandem%20fluorescent%20protein%20timers%20for%20in%20vivo%20analysis%20of%20protein%20dynamics.%20Nature%20biotechnology.%202012%3B30%287%29%3A708%E2%80%9314.%20doi%3A%2010.1038%2Fnbt.2281%2022729030." data-title="Tandem%20fluorescent%20protein%20timers%20for%20in%20vivo%20analysis%20of%20protein%20dynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22729030" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tandem+fluorescent+protein+timers+for+in+vivo+analysis+of+protein+dynamics+Khmelinskii+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0159292.ref018" id="pone.0159292.ref018" class="link-target"></a>Macdonald PJ, Chen Y, Mueller JD. Chromophore maturation and fluorescence fluctuation spectroscopy of fluorescent proteins in a cell-free expression system. Anal Biochem. 2012;421(1):291–8.  pmid:22093611; PubMed Central PMCID: PMC3367886. <ul class="reflinks"><li><a href="#" data-author="Macdonald" data-cit="MacdonaldPJ%2C%20ChenY%2C%20MuellerJD.%20Chromophore%20maturation%20and%20fluorescence%20fluctuation%20spectroscopy%20of%20fluorescent%20proteins%20in%20a%20cell-free%20expression%20system.%20Anal%20Biochem.%202012%3B421%281%29%3A291%E2%80%938.%20doi%3A%2010.1016%2Fj.ab.2011.10.040%2022093611%3B%20PubMed%20Central%20PMCID%3A%20PMC3367886." data-title="Chromophore%20maturation%20and%20fluorescence%20fluctuation%20spectroscopy%20of%20fluorescent%20proteins%20in%20a%20cell-free%20expression%20system" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22093611" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chromophore+maturation+and+fluorescence+fluctuation+spectroscopy+of+fluorescent+proteins+in+a+cell-free+expression+system+Macdonald+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0159292.ref019" id="pone.0159292.ref019" class="link-target"></a>Aliye N, Fabbretti A, Lupidi G, Tsekoa T, Spurio R. Engineering color variants of green fluorescent protein (GFP) for thermostability, pH-sensitivity, and improved folding kinetics. Appl Microbiol Biotechnol. 2015;99(3):1205–16.  pmid:25112226. <ul class="reflinks"><li><a href="#" data-author="Aliye" data-cit="AliyeN%2C%20FabbrettiA%2C%20LupidiG%2C%20TsekoaT%2C%20SpurioR.%20Engineering%20color%20variants%20of%20green%20fluorescent%20protein%20%28GFP%29%20for%20thermostability%2C%20pH-sensitivity%2C%20and%20improved%20folding%20kinetics.%20Appl%20Microbiol%20Biotechnol.%202015%3B99%283%29%3A1205%E2%80%9316.%20doi%3A%2010.1007%2Fs00253-014-5975-1%2025112226." data-title="Engineering%20color%20variants%20of%20green%20fluorescent%20protein%20%28GFP%29%20for%20thermostability%2C%20pH-sensitivity%2C%20and%20improved%20folding%20kinetics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25112226" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Engineering+color+variants+of+green+fluorescent+protein+%28GFP%29+for+thermostability%2C+pH-sensitivity%2C+and+improved+folding+kinetics+Aliye+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0159292.ref020" id="pone.0159292.ref020" class="link-target"></a>Hebisch E, Knebel J, Landsberg J, Frey E, Leisner M. High variation of fluorescence protein maturation times in closely related Escherichia coli strains. PloS one. 2013;8(10):e75991.  pmid:24155882; PubMed Central PMCID: PMC3796512. <ul class="reflinks"><li><a href="#" data-author="Hebisch" data-cit="HebischE%2C%20KnebelJ%2C%20LandsbergJ%2C%20FreyE%2C%20LeisnerM.%20High%20variation%20of%20fluorescence%20protein%20maturation%20times%20in%20closely%20related%20Escherichia%20coli%20strains.%20PloS%20one.%202013%3B8%2810%29%3Ae75991.%20doi%3A%2010.1371%2Fjournal.pone.0075991%2024155882%3B%20PubMed%20Central%20PMCID%3A%20PMC3796512." data-title="High%20variation%20of%20fluorescence%20protein%20maturation%20times%20in%20closely%20related%20Escherichia%20coli%20strains" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24155882" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=High+variation+of+fluorescence+protein+maturation+times+in+closely+related+Escherichia+coli+strains+Hebisch+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0159292.ref021" id="pone.0159292.ref021" class="link-target"></a>Remington SJ. Fluorescent proteins: maturation, photochemistry and photophysics. Curr Opin Struct Biol. 2006;16(6):714–21.  pmid:17064887. <ul class="reflinks"><li><a href="#" data-author="Remington" data-cit="RemingtonSJ.%20Fluorescent%20proteins%3A%20maturation%2C%20photochemistry%20and%20photophysics.%20Curr%20Opin%20Struct%20Biol.%202006%3B16%286%29%3A714%E2%80%9321.%20doi%3A%2010.1016%2Fj.sbi.2006.10.001%2017064887." data-title="Fluorescent%20proteins%3A%20maturation%2C%20photochemistry%20and%20photophysics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17064887" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Fluorescent+proteins%3A+maturation%2C+photochemistry+and+photophysics+Remington+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0159292.ref022" id="pone.0159292.ref022" class="link-target"></a>Sniegowski JA, Lappe JW, Patel HN, Huffman HA, Wachter RM. Base catalysis of chromophore formation in Arg96 and Glu222 variants of green fluorescent protein. The Journal of biological chemistry. 2005;280(28):26248–55.  pmid:15888441. <ul class="reflinks"><li><a href="#" data-author="Sniegowski" data-cit="SniegowskiJA%2C%20LappeJW%2C%20PatelHN%2C%20HuffmanHA%2C%20WachterRM.%20Base%20catalysis%20of%20chromophore%20formation%20in%20Arg96%20and%20Glu222%20variants%20of%20green%20fluorescent%20protein.%20The%20Journal%20of%20biological%20chemistry.%202005%3B280%2828%29%3A26248%E2%80%9355.%20doi%3A%2010.1074%2Fjbc.M412327200%2015888441." data-title="Base%20catalysis%20of%20chromophore%20formation%20in%20Arg96%20and%20Glu222%20variants%20of%20green%20fluorescent%20protein" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15888441" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Base+catalysis+of+chromophore+formation+in+Arg96+and+Glu222+variants+of+green+fluorescent+protein+Sniegowski+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0159292.ref023" id="pone.0159292.ref023" class="link-target"></a>Shu X, Shaner NC, Yarbrough CA, Tsien RY, Remington SJ. Novel chromophores and buried charges control color in mFruits. Biochemistry. 2006;45(32):9639–47.  pmid:16893165. <ul class="reflinks"><li><a href="#" data-author="Shu" data-cit="ShuX%2C%20ShanerNC%2C%20YarbroughCA%2C%20TsienRY%2C%20RemingtonSJ.%20Novel%20chromophores%20and%20buried%20charges%20control%20color%20in%20mFruits.%20Biochemistry.%202006%3B45%2832%29%3A9639%E2%80%9347.%20doi%3A%2010.1021%2Fbi060773l%2016893165." data-title="Novel%20chromophores%20and%20buried%20charges%20control%20color%20in%20mFruits" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16893165" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Novel+chromophores+and+buried+charges+control+color+in+mFruits+Shu+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0159292.ref024" id="pone.0159292.ref024" class="link-target"></a>Doherty GP, Bailey K, Lewis PJ. Stage-specific fluorescence intensity of GFP and mCherry during sporulation In Bacillus Subtilis. BMC Res Notes. 2010;3:303.  pmid:21073756; PubMed Central PMCID: PMC2994887. <ul class="reflinks"><li><a href="#" data-author="Doherty" data-cit="DohertyGP%2C%20BaileyK%2C%20LewisPJ.%20Stage-specific%20fluorescence%20intensity%20of%20GFP%20and%20mCherry%20during%20sporulation%20In%20Bacillus%20Subtilis.%20BMC%20Res%20Notes.%202010%3B3%3A303.%20doi%3A%2010.1186%2F1756-0500-3-303%2021073756%3B%20PubMed%20Central%20PMCID%3A%20PMC2994887." data-title="Stage-specific%20fluorescence%20intensity%20of%20GFP%20and%20mCherry%20during%20sporulation%20In%20Bacillus%20Subtilis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21073756" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Stage-specific+fluorescence+intensity+of+GFP+and+mCherry+during+sporulation+In+Bacillus+Subtilis+Doherty+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0159292.ref025" id="pone.0159292.ref025" class="link-target"></a>Scott BL, Hoppe AD. Optimizing fluorescent protein trios for 3-Way FRET imaging of protein interactions in living cells. Sci Rep. 2015;5:10270.  pmid:26130463; PubMed Central PMCID: PMC4487001. <ul class="reflinks"><li><a href="#" data-author="Scott" data-cit="ScottBL%2C%20HoppeAD.%20Optimizing%20fluorescent%20protein%20trios%20for%203-Way%20FRET%20imaging%20of%20protein%20interactions%20in%20living%20cells.%20Sci%20Rep.%202015%3B5%3A10270.%20doi%3A%2010.1038%2Fsrep10270%2026130463%3B%20PubMed%20Central%20PMCID%3A%20PMC4487001." data-title="Optimizing%20fluorescent%20protein%20trios%20for%203-Way%20FRET%20imaging%20of%20protein%20interactions%20in%20living%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26130463" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Optimizing+fluorescent+protein+trios+for+3-Way+FRET+imaging+of+protein+interactions+in+living+cells+Scott+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0159292.ref026" id="pone.0159292.ref026" class="link-target"></a>Kaida A, Miura M. Visualizing the effect of hypoxia on fluorescence kinetics in living HeLa cells using the fluorescent ubiquitination-based cell cycle indicator (Fucci). Exp Cell Res. 2012;318(3):288–97.  pmid:22079518. <ul class="reflinks"><li><a href="#" data-author="Kaida" data-cit="KaidaA%2C%20MiuraM.%20Visualizing%20the%20effect%20of%20hypoxia%20on%20fluorescence%20kinetics%20in%20living%20HeLa%20cells%20using%20the%20fluorescent%20ubiquitination-based%20cell%20cycle%20indicator%20%28Fucci%29.%20Exp%20Cell%20Res.%202012%3B318%283%29%3A288%E2%80%9397.%20doi%3A%2010.1016%2Fj.yexcr.2011.10.016%2022079518." data-title="Visualizing%20the%20effect%20of%20hypoxia%20on%20fluorescence%20kinetics%20in%20living%20HeLa%20cells%20using%20the%20fluorescent%20ubiquitination-based%20cell%20cycle%20indicator%20%28Fucci%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22079518" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Visualizing+the+effect+of+hypoxia+on+fluorescence+kinetics+in+living+HeLa+cells+using+the+fluorescent+ubiquitination-based+cell+cycle+indicator+%28Fucci%29+Kaida+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0159292.ref027" id="pone.0159292.ref027" class="link-target"></a>Zhang L, Patel HN, Lappe JW, Wachter RM. Reaction progress of chromophore biogenesis in green fluorescent protein. J Am Chem Soc. 2006;128(14):4766–72.  pmid:16594713. <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangL%2C%20PatelHN%2C%20LappeJW%2C%20WachterRM.%20Reaction%20progress%20of%20chromophore%20biogenesis%20in%20green%20fluorescent%20protein.%20J%20Am%20Chem%20Soc.%202006%3B128%2814%29%3A4766%E2%80%9372.%20doi%3A%2010.1021%2Fja0580439%2016594713." data-title="Reaction%20progress%20of%20chromophore%20biogenesis%20in%20green%20fluorescent%20protein" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16594713" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Reaction+progress+of+chromophore+biogenesis+in+green+fluorescent+protein+Zhang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0159292.ref028" id="pone.0159292.ref028" class="link-target"></a>Audergon PN, Catania S, Kagansky A, Tong P, Shukla M, Pidoux AL, et al. Epigenetics. Restricted epigenetic inheritance of H3K9 methylation. Science. 2015;348(6230):132–5.  pmid:25838386; PubMed Central PMCID: PMC4397586. <ul class="reflinks"><li><a href="#" data-author="Audergon" data-cit="AudergonPN%2C%20CataniaS%2C%20KaganskyA%2C%20TongP%2C%20ShuklaM%2C%20PidouxAL%2C%20et%20al.%20Epigenetics.%20Restricted%20epigenetic%20inheritance%20of%20H3K9%20methylation.%20Science.%202015%3B348%286230%29%3A132%E2%80%935.%20doi%3A%2010.1126%2Fscience.1260638%2025838386%3B%20PubMed%20Central%20PMCID%3A%20PMC4397586." data-title="Epigenetics.%20Restricted%20epigenetic%20inheritance%20of%20H3K9%20methylation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25838386" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Epigenetics.+Restricted+epigenetic+inheritance+of+H3K9+methylation+Audergon+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0159292.ref029" id="pone.0159292.ref029" class="link-target"></a>Subach OM, Cranfill PJ, Davidson MW, Verkhusha VV. An enhanced monomeric blue fluorescent protein with the high chemical stability of the chromophore. PloS one. 2011;6(12):e28674.  pmid:22174863; PubMed Central PMCID: PMC3234270. <ul class="reflinks"><li><a href="#" data-author="Subach" data-cit="SubachOM%2C%20CranfillPJ%2C%20DavidsonMW%2C%20VerkhushaVV.%20An%20enhanced%20monomeric%20blue%20fluorescent%20protein%20with%20the%20high%20chemical%20stability%20of%20the%20chromophore.%20PloS%20one.%202011%3B6%2812%29%3Ae28674.%20doi%3A%2010.1371%2Fjournal.pone.0028674%2022174863%3B%20PubMed%20Central%20PMCID%3A%20PMC3234270." data-title="An%20enhanced%20monomeric%20blue%20fluorescent%20protein%20with%20the%20high%20chemical%20stability%20of%20the%20chromophore" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174863" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=An+enhanced+monomeric+blue+fluorescent+protein+with+the+high+chemical+stability+of+the+chromophore+Subach+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0159292.ref030" id="pone.0159292.ref030" class="link-target"></a>Shagin DA, Barsova EV, Yanushevich YG, Fradkov AF, Lukyanov KA, Labas YA, et al. GFP-like proteins as ubiquitous metazoan superfamily: evolution of functional features and structural complexity. Mol Biol Evol. 2004;21(5):841–50.  pmid:14963095. <ul class="reflinks"><li><a href="#" data-author="Shagin" data-cit="ShaginDA%2C%20BarsovaEV%2C%20YanushevichYG%2C%20FradkovAF%2C%20LukyanovKA%2C%20LabasYA%2C%20et%20al.%20GFP-like%20proteins%20as%20ubiquitous%20metazoan%20superfamily%3A%20evolution%20of%20functional%20features%20and%20structural%20complexity.%20Mol%20Biol%20Evol.%202004%3B21%285%29%3A841%E2%80%9350.%20doi%3A%2010.1093%2Fmolbev%2Fmsh079%2014963095." data-title="GFP-like%20proteins%20as%20ubiquitous%20metazoan%20superfamily%3A%20evolution%20of%20functional%20features%20and%20structural%20complexity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14963095" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=GFP-like+proteins+as+ubiquitous+metazoan+superfamily%3A+evolution+of+functional+features+and+structural+complexity+Shagin+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0159292.ref031" id="pone.0159292.ref031" class="link-target"></a>Sakaue-Sawano A, Kurokawa H, Morimura T, Hanyu A, Hama H, Osawa H, et al. Visualizing spatiotemporal dynamics of multicellular cell-cycle progression. Cell. 2008;132(3):487–98.  pmid:18267078. <ul class="reflinks"><li><a href="#" data-author="Sakaue-Sawano" data-cit="Sakaue-SawanoA%2C%20KurokawaH%2C%20MorimuraT%2C%20HanyuA%2C%20HamaH%2C%20OsawaH%2C%20et%20al.%20Visualizing%20spatiotemporal%20dynamics%20of%20multicellular%20cell-cycle%20progression.%20Cell.%202008%3B132%283%29%3A487%E2%80%9398.%20doi%3A%2010.1016%2Fj.cell.2007.12.033%2018267078." data-title="Visualizing%20spatiotemporal%20dynamics%20of%20multicellular%20cell-cycle%20progression" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18267078" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Visualizing+spatiotemporal+dynamics+of+multicellular+cell-cycle+progression+Sakaue-Sawano+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0159292.ref032" id="pone.0159292.ref032" class="link-target"></a>Shcherbo D, Murphy CS, Ermakova GV, Solovieva EA, Chepurnykh TV, Shcheglov AS, et al. Far-red fluorescent tags for protein imaging in living tissues. Biochem J. 2009;418(3):567–74.  pmid:19143658; PubMed Central PMCID: PMC2893397. <ul class="reflinks"><li><a href="#" data-author="Shcherbo" data-cit="ShcherboD%2C%20MurphyCS%2C%20ErmakovaGV%2C%20SolovievaEA%2C%20ChepurnykhTV%2C%20ShcheglovAS%2C%20et%20al.%20Far-red%20fluorescent%20tags%20for%20protein%20imaging%20in%20living%20tissues.%20Biochem%20J.%202009%3B418%283%29%3A567%E2%80%9374.%20doi%3A%2010.1042%2FBJ20081949%2019143658%3B%20PubMed%20Central%20PMCID%3A%20PMC2893397." data-title="Far-red%20fluorescent%20tags%20for%20protein%20imaging%20in%20living%20tissues" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19143658" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Far-red+fluorescent+tags+for+protein+imaging+in+living+tissues+Shcherbo+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0159292.ref033" id="pone.0159292.ref033" class="link-target"></a>Strack RL, Hein B, Bhattacharyya D, Hell SW, Keenan RJ, Glick BS. A rapidly maturing far-red derivative of DsRed-Express2 for whole-cell labeling. Biochemistry. 2009;48(35):8279–81.  pmid:19658435; PubMed Central PMCID: PMC2861903. <ul class="reflinks"><li><a href="#" data-author="Strack" data-cit="StrackRL%2C%20HeinB%2C%20BhattacharyyaD%2C%20HellSW%2C%20KeenanRJ%2C%20GlickBS.%20A%20rapidly%20maturing%20far-red%20derivative%20of%20DsRed-Express2%20for%20whole-cell%20labeling.%20Biochemistry.%202009%3B48%2835%29%3A8279%E2%80%9381.%20doi%3A%2010.1021%2Fbi900870u%2019658435%3B%20PubMed%20Central%20PMCID%3A%20PMC2861903." data-title="A%20rapidly%20maturing%20far-red%20derivative%20of%20DsRed-Express2%20for%20whole-cell%20labeling" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19658435" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+rapidly+maturing+far-red+derivative+of+DsRed-Express2+for+whole-cell+labeling+Strack+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0159292.ref034" id="pone.0159292.ref034" class="link-target"></a>Watson AT, Werler P, Carr AM. Regulation of gene expression at the fission yeast Schizosaccharomyces pombe urg1 locus. Gene. 2011;484(1–2):75–85.  pmid:21664261. <ul class="reflinks"><li><a href="#" data-author="Watson" data-cit="WatsonAT%2C%20WerlerP%2C%20CarrAM.%20Regulation%20of%20gene%20expression%20at%20the%20fission%20yeast%20Schizosaccharomyces%20pombe%20urg1%20locus.%20Gene.%202011%3B484%281%E2%80%932%29%3A75%E2%80%9385.%20doi%3A%2010.1016%2Fj.gene.2011.05.028%2021664261." data-title="Regulation%20of%20gene%20expression%20at%20the%20fission%20yeast%20Schizosaccharomyces%20pombe%20urg1%20locus" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21664261" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+gene+expression+at+the+fission+yeast+Schizosaccharomyces+pombe+urg1+locus+Watson+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0159292.ref035" id="pone.0159292.ref035" class="link-target"></a>Watt S, Mata J, Lopez-Maury L, Marguerat S, Burns G, Bahler J. urg1: a uracil-regulatable promoter system for fission yeast with short induction and repression times. PloS one. 2008;3(1):e1428.  pmid:18197241; PubMed Central PMCID: PMC2174524. <ul class="reflinks"><li><a href="#" data-author="Watt" data-cit="WattS%2C%20MataJ%2C%20Lopez-MauryL%2C%20MargueratS%2C%20BurnsG%2C%20BahlerJ.%20urg1%3A%20a%20uracil-regulatable%20promoter%20system%20for%20fission%20yeast%20with%20short%20induction%20and%20repression%20times.%20PloS%20one.%202008%3B3%281%29%3Ae1428.%20doi%3A%2010.1371%2Fjournal.pone.0001428%2018197241%3B%20PubMed%20Central%20PMCID%3A%20PMC2174524." data-title="urg1%3A%20a%20uracil-regulatable%20promoter%20system%20for%20fission%20yeast%20with%20short%20induction%20and%20repression%20times" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18197241" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=urg1%3A+a+uracil-regulatable+promoter+system+for+fission+yeast+with+short+induction+and+repression+times+Watt+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0159292.ref036" id="pone.0159292.ref036" class="link-target"></a>Da Silva NA, Srikrishnan S. Introduction and expression of genes for metabolic engineering applications in Saccharomyces cerevisiae. FEMS yeast research. 2012;12(2):197–214. Epub 2011/12/02.  pmid:22129153. <ul class="reflinks"><li><a href="#" data-author="Da%20Silva" data-cit="Da%20SilvaNA%2C%20SrikrishnanS.%20Introduction%20and%20expression%20of%20genes%20for%20metabolic%20engineering%20applications%20in%20Saccharomyces%20cerevisiae.%20FEMS%20yeast%20research.%202012%3B12%282%29%3A197%E2%80%93214.%20Epub%202011%2F12%2F02.%20doi%3A%2010.1111%2Fj.1567-1364.2011.00769.x%2022129153." data-title="Introduction%20and%20expression%20of%20genes%20for%20metabolic%20engineering%20applications%20in%20Saccharomyces%20cerevisiae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22129153" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Introduction+and+expression+of+genes+for+metabolic+engineering+applications+in+Saccharomyces+cerevisiae+Da+Silva+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0159292.ref037" id="pone.0159292.ref037" class="link-target"></a>Matsuyama A, Arai R, Yashiroda Y, Shirai A, Kamata A, Sekido S, et al. ORFeome cloning and global analysis of protein localization in the fission yeast Schizosaccharomyces pombe. Nature biotechnology. 2006;24(7):841–7. Epub 2006/07/11.  pmid:16823372. <ul class="reflinks"><li><a href="#" data-author="Matsuyama" data-cit="MatsuyamaA%2C%20AraiR%2C%20YashirodaY%2C%20ShiraiA%2C%20KamataA%2C%20SekidoS%2C%20et%20al.%20ORFeome%20cloning%20and%20global%20analysis%20of%20protein%20localization%20in%20the%20fission%20yeast%20Schizosaccharomyces%20pombe.%20Nature%20biotechnology.%202006%3B24%287%29%3A841%E2%80%937.%20Epub%202006%2F07%2F11.%20doi%3A%2010.1038%2Fnbt1222%2016823372." data-title="ORFeome%20cloning%20and%20global%20analysis%20of%20protein%20localization%20in%20the%20fission%20yeast%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16823372" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=ORFeome+cloning+and+global+analysis+of+protein+localization+in+the+fission+yeast+Schizosaccharomyces+pombe+Matsuyama+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0159292.ref038" id="pone.0159292.ref038" class="link-target"></a>Cheutin T, Gorski SA, May KM, Singh PB, Misteli T. In Vivo Dynamics of Swi6 in Yeast: Evidence for a Stochastic Model of Heterochromatin. Mol Cell Biol. 2004;24(8):3157–67.  pmid:15060140 <ul class="reflinks"><li><a href="#" data-author="Cheutin" data-cit="CheutinT%2C%20GorskiSA%2C%20MayKM%2C%20SinghPB%2C%20MisteliT.%20In%20Vivo%20Dynamics%20of%20Swi6%20in%20Yeast%3A%20Evidence%20for%20a%20Stochastic%20Model%20of%20Heterochromatin.%20Mol%20Cell%20Biol.%202004%3B24%288%29%3A3157%E2%80%9367.%20doi%3A%2010.1128%2Fmcb.24.8.3157%E2%80%933167.2004%2015060140" data-title="In%20Vivo%20Dynamics%20of%20Swi6%20in%20Yeast%3A%20Evidence%20for%20a%20Stochastic%20Model%20of%20Heterochromatin" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060140" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=In+Vivo+Dynamics+of+Swi6+in+Yeast%3A+Evidence+for+a+Stochastic+Model+of+Heterochromatin+Cheutin+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0159292.ref039" id="pone.0159292.ref039" class="link-target"></a>Hagan I, Yanagida M. The product of the spindle formation gene sad1+ associates with the fission yeast spindle pole body and is essential for viability. J Cell Biol. 1995;129(4):1033–47. pmid:7744953; PubMed Central PMCID: PMCPMC2120497. <ul class="reflinks"><li><a href="#" data-author="Hagan" data-cit="HaganI%2C%20YanagidaM.%20The%20product%20of%20the%20spindle%20formation%20gene%20sad1%2B%20associates%20with%20the%20fission%20yeast%20spindle%20pole%20body%20and%20is%20essential%20for%20viability.%20J%20Cell%20Biol.%201995%3B129%284%29%3A1033%E2%80%9347.%207744953%3B%20PubMed%20Central%20PMCID%3A%20PMCPMC2120497." data-title="The%20product%20of%20the%20spindle%20formation%20gene%20sad1%2B%20associates%20with%20the%20fission%20yeast%20spindle%20pole%20body%20and%20is%20essential%20for%20viability" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7744953" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+product+of+the+spindle+formation+gene+sad1%2B+associates+with+the+fission+yeast+spindle+pole+body+and+is+essential+for+viability+Hagan+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0159292.ref040" id="pone.0159292.ref040" class="link-target"></a>Emmerth S, Schober H, Gaidatzis D, Roloff T, Jacobeit K, Buhler M. Nuclear retention of fission yeast dicer is a prerequisite for RNAi-mediated heterochromatin assembly. Dev Cell. 2010;18(1):102–13.  pmid:20152181. <ul class="reflinks"><li><a href="#" data-author="Emmerth" data-cit="EmmerthS%2C%20SchoberH%2C%20GaidatzisD%2C%20RoloffT%2C%20JacobeitK%2C%20BuhlerM.%20Nuclear%20retention%20of%20fission%20yeast%20dicer%20is%20a%20prerequisite%20for%20RNAi-mediated%20heterochromatin%20assembly.%20Dev%20Cell.%202010%3B18%281%29%3A102%E2%80%9313.%20doi%3A%2010.1016%2Fj.devcel.2009.11.011%2020152181." data-title="Nuclear%20retention%20of%20fission%20yeast%20dicer%20is%20a%20prerequisite%20for%20RNAi-mediated%20heterochromatin%20assembly" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20152181" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Nuclear+retention+of+fission+yeast+dicer+is+a+prerequisite+for+RNAi-mediated+heterochromatin+assembly+Emmerth+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0159292.ref041" id="pone.0159292.ref041" class="link-target"></a>Barraud P, Emmerth S, Shimada Y, Hotz HR, Allain FH, Buhler M. An extended dsRBD with a novel zinc-binding motif mediates nuclear retention of fission yeast Dicer. The EMBO journal. 2011;30(20):4223–35.  pmid:21847092; PubMed Central PMCID: PMC3199388. <ul class="reflinks"><li><a href="#" data-author="Barraud" data-cit="BarraudP%2C%20EmmerthS%2C%20ShimadaY%2C%20HotzHR%2C%20AllainFH%2C%20BuhlerM.%20An%20extended%20dsRBD%20with%20a%20novel%20zinc-binding%20motif%20mediates%20nuclear%20retention%20of%20fission%20yeast%20Dicer.%20The%20EMBO%20journal.%202011%3B30%2820%29%3A4223%E2%80%9335.%20doi%3A%2010.1038%2Femboj.2011.300%2021847092%3B%20PubMed%20Central%20PMCID%3A%20PMC3199388." data-title="An%20extended%20dsRBD%20with%20a%20novel%20zinc-binding%20motif%20mediates%20nuclear%20retention%20of%20fission%20yeast%20Dicer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21847092" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=An+extended+dsRBD+with+a+novel+zinc-binding+motif+mediates+nuclear+retention+of+fission+yeast+Dicer+Barraud+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0159292.ref042" id="pone.0159292.ref042" class="link-target"></a>Heim R, Prasher DC, Tsien RY. Wavelength mutations and posttranslational autoxidation of green fluorescent protein. Proceedings of the National Academy of Sciences of the United States of America. 1994;91(26):12501–4. pmid:7809066; PubMed Central PMCID: PMC45466. <ul class="reflinks"><li><a href="#" data-author="Heim" data-cit="HeimR%2C%20PrasherDC%2C%20TsienRY.%20Wavelength%20mutations%20and%20posttranslational%20autoxidation%20of%20green%20fluorescent%20protein.%20Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America.%201994%3B91%2826%29%3A12501%E2%80%934.%207809066%3B%20PubMed%20Central%20PMCID%3A%20PMC45466." data-title="Wavelength%20mutations%20and%20posttranslational%20autoxidation%20of%20green%20fluorescent%20protein" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7809066" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Wavelength+mutations+and+posttranslational+autoxidation+of+green+fluorescent+protein+Heim+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0159292.ref043" id="pone.0159292.ref043" class="link-target"></a>Knop M, Siegers K, Pereira G, Zachariae W, Winsor B, Nasmyth K, et al. Epitope tagging of yeast genes using a PCR-based strategy: More tags and improved practical routines. Yeast. 1999;15(10B):963–72. pmid:ISI:000081644400007. <ul class="reflinks"><li><a href="#" data-author="Knop" data-cit="KnopM%2C%20SiegersK%2C%20PereiraG%2C%20ZachariaeW%2C%20WinsorB%2C%20NasmythK%2C%20et%20al.%20Epitope%20tagging%20of%20yeast%20genes%20using%20a%20PCR-based%20strategy%3A%20More%20tags%20and%20improved%20practical%20routines.%20Yeast.%201999%3B15%2810B%29%3A963%E2%80%9372.%20ISI%3A000081644400007." data-title="Epitope%20tagging%20of%20yeast%20genes%20using%20a%20PCR-based%20strategy%3A%20More%20tags%20and%20improved%20practical%20routines" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/ISI:000081644400007" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Epitope+tagging+of+yeast+genes+using+a+PCR-based+strategy%3A+More+tags+and+improved+practical+routines+Knop+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0159292.ref044" id="pone.0159292.ref044" class="link-target"></a>Al-Sady B, Madhani HD, Narlikar GJ. Division of Labor between the Chromodomains of HP1 and Suv39 Methylase Enables Coordination of Heterochromatin Spread. Mol Cell. 2013;51(1):80–91. Epub 2013/07/16.  pmid:23849629. <ul class="reflinks"><li><a href="#" data-author="Al-Sady" data-cit="Al-SadyB%2C%20MadhaniHD%2C%20NarlikarGJ.%20Division%20of%20Labor%20between%20the%20Chromodomains%20of%20HP1%20and%20Suv39%20Methylase%20Enables%20Coordination%20of%20Heterochromatin%20Spread.%20Mol%20Cell.%202013%3B51%281%29%3A80%E2%80%9391.%20Epub%202013%2F07%2F16.%20doi%3A%2010.1016%2Fj.molcel.2013.06.013%2023849629." data-title="Division%20of%20Labor%20between%20the%20Chromodomains%20of%20HP1%20and%20Suv39%20Methylase%20Enables%20Coordination%20of%20Heterochromatin%20Spread" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23849629" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Division+of+Labor+between+the+Chromodomains+of+HP1+and+Suv39+Methylase+Enables+Coordination+of+Heterochromatin+Spread+Al-Sady+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0159292.ref045" id="pone.0159292.ref045" class="link-target"></a>Canzio D, Chang EY, Shankar S, Kuchenbecker KM, Simon MD, Madhani HD, et al. Chromodomain-mediated oligomerization of HP1 suggests a nucleosome-bridging mechanism for heterochromatin assembly. Mol Cell. 2011;41(1):67–81. Epub 2011/01/08.  pmid:21211724; PubMed Central PMCID: PMC3752404. <ul class="reflinks"><li><a href="#" data-author="Canzio" data-cit="CanzioD%2C%20ChangEY%2C%20ShankarS%2C%20KuchenbeckerKM%2C%20SimonMD%2C%20MadhaniHD%2C%20et%20al.%20Chromodomain-mediated%20oligomerization%20of%20HP1%20suggests%20a%20nucleosome-bridging%20mechanism%20for%20heterochromatin%20assembly.%20Mol%20Cell.%202011%3B41%281%29%3A67%E2%80%9381.%20Epub%202011%2F01%2F08.%20doi%3A%2010.1016%2Fj.molcel.2010.12.016%2021211724%3B%20PubMed%20Central%20PMCID%3A%20PMC3752404." data-title="Chromodomain-mediated%20oligomerization%20of%20HP1%20suggests%20a%20nucleosome-bridging%20mechanism%20for%20heterochromatin%20assembly" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21211724" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chromodomain-mediated+oligomerization+of+HP1+suggests+a+nucleosome-bridging+mechanism+for+heterochromatin+assembly+Canzio+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

