



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Existing technologies allow isolating antigen-specific monoclonal antibodies (mAbs) from B cells. We devised a direct approach to isolate mAbs with predetermined conformational epitope specificity, using epitope mimetics (mimotopes) that reflect the three-dimensional structure of given antigen subdomains. We performed differential biopanning using bacteriophages encoding random peptide libraries and polyclonal antibodies (Abs) that had been affinity-purified with either native or denatured antigen. This strategy yielded conformational mimotopes. We then generated mimotope-fluorescent protein fusions, which were used as baits to isolate single memory B cells from rhesus monkeys (RMs). To amplify RM immunoglobulin variable regions, we developed RM-specific PCR primers and generated chimeric simian-human mAbs with predicted epitope specificity. We established proof-of-concept of our strategy by isolating mAbs targeting the conformational V3 loop crown of HIV Env; the new mAbs cross-neutralized viruses of different clades. The novel technology allows isolating mAbs from RMs or other hosts given experimental immunogens or infectious agents.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Sholukh AM, Mukhtar MM, Humbert M, Essono SS, Watkins JD, Vyas HK, et al.  (2012) Isolation of Monoclonal Antibodies with Predetermined Conformational Epitope Specificity. PLoS ONE 7(6):
           e38943.
        
        https://doi.org/10.1371/journal.pone.0038943</p><p><strong>Editor: </strong>Shan Lu, University of Massachusetts Medical Center, United States of America</p><p><strong>Received: </strong>January 20, 2012; <strong>Accepted: </strong>May 14, 2012; <strong>Published: </strong> June 21, 2012</p><p><strong>Copyright: </strong> © 2012 Sholukh et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by NIH grants P01 AI048240, P01 AI048240-08s1, R37 AI034266, and P30 AI060354. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have read the journal’s policy and have the following conflicts. The Dana-Farber Cancer Institute has applied for patent coverage of the new technology: US Patent Application number PCT/US2010/046353 for “Mimotopes of HIV and Uses Thereof” was filed on 8/23/2010 in which AMS, MH and RMR are listed as co-inventors; a patent application for “Two Sets of Primers Suitable for Amplification of Rhesus Monkey Immunoglobulin’s Light and Heavy Chain Variable Domains and Constant Regions” listing RMR, AMS, MMM and SSE will be filed shortly. Dana-Farber Cancer Institute requires its employees to assign patent and other intellectual property rights to the Institute; therefore the co-inventors have no competing financial interests to declare, and the authors affirm that their adherence to all the PLoS ONE policies on sharing data and materials is unaffected.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Isolation of naturally induced and matured antibodies (Abs) is of prime importance for analytical vaccinology <a href="#pone.0038943-Zielinski1" class="ref-tip">[1]</a>, <a href="#pone.0038943-Corti1" class="ref-tip">[2]</a>. Three major strategies have been used to interrogate the B-cell repertoire: traditional phage display, high-throughput screening of immortalized B cell or plasma cell cultures, and isolation of antigen-specific B cells to PCR amplify the variable heavy (V<sub>H</sub>) and variable light (V<sub>L</sub>) immunoglobulin (Ig) genes <a href="#pone.0038943-Corti1" class="ref-tip">[2]</a>, <a href="#pone.0038943-Kwong1" class="ref-tip">[3]</a>. Development and refinement of high-throughput screening methods, flow cytometric capabilities and single-cell cloning techniques led to substitution of the traditional phage display techniques by approaches that allow the isolation of naturally selected Igs. Phage display is constrained by the diversity of the library used, by physical-chemical properties of the Ig fragments displayed and involves random combination of V<sub>H</sub>/V<sub>L</sub> pairs. Consequently, it is not known whether Abs isolated by phage display represent natural molecules generated by the host in response to immunization or infection by a pathogen of interest. Even the recently published technique based upon high-throughput DNA sequencing of the plasma cell Ig repertoire with subsequent comprehensive analysis of V<sub>H</sub> and V<sub>L</sub> representation resulting in the assembly of V<sub>H</sub>/V<sub>L</sub> pairs left the possibility that the recovered pair may be artificial <a href="#pone.0038943-Reddy1" class="ref-tip">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Naturally occurring Abs can be isolated via high-throughput screening of immortalized memory B or plasma cell cultures. This approach yielded broad and potent neutralizing monoclonal Abs (nmAbs) against HIV, dengue and influenza viruses that significantly improved our understanding of the specificity and mechanisms of humoral immunity <a href="#pone.0038943-Robinson1" class="ref-tip">[5]</a>–<a href="#pone.0038943-Beltramello1" class="ref-tip">[8]</a>. Isolation of antigen-specific B cells by sorting with a fluorescence-activated flow cytometer (FACSorting), first reported in 1972 <a href="#pone.0038943-Julius1" class="ref-tip">[9]</a>, has led to the recovery of a number of naturally induced and matured Abs against rotavirus and HIV <a href="#pone.0038943-Weitkamp1" class="ref-tip">[10]</a>–<a href="#pone.0038943-Wu1" class="ref-tip">[12]</a>. However, using the whole antigen for B-cell sorting implies a need for extensive screening of newly isolated mAbs, followed by epitope mapping. A more sophisticated approach consists of directly isolating B cells specific for individual epitopes. The success of this approach depends on the identification of epitope mimetics (mimotopes) that correctly reflect the three-dimensional structure of a given antigen subdomain/epitope. Here we report the development of a novel methodology using fluorescently labeled mimotopes as baits to isolate rhesus monkey (RM) memory B cells for subsequent cloning of Ab genes.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Overview of Strategy"></a>
<h3>Overview of Strategy</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>The overall strategy is depicted in <a href="#pone-0038943-g001">Figure 1</a>. Mimotopes were isolated by biopanning using recombinant bacteriophages encoding random peptide libraries. To enrich for conformational mimotopes, we devised a new strategy of differential biopanning that involved positive selection with Abs pre-purified on native antigen followed by negative selection with Abs eluted from denatured antigen (<a href="#pone-0038943-g002">Figure 2</a>). After a total of three rounds of positive/negative selections with these Ab preparations, the resulting mimotopes are expected to show preferential conformation dependence. Next, mimotope sequences are cloned into a vector encoding a fluorescent reporter gene, giving rise to a fluorescent fusion protein (<a href="#pone-0038943-g001">Figure 1</a>). The latter is used for mimotope-specific isolation of single memory B cells by flow cytometry. Single-cell RT-PCR is then used to amplify the V<sub>H</sub> and V<sub>L</sub> Ig genes using newly designed, RM-specific primers. Next, the V<sub>H</sub> and V<sub>L</sub> sequences are cloned into expression vectors encoding the backbone of human IgG1 heavy or light chains, respectively. Cotransfection into eukaryotic cells grown in serum-free medium, will yield recombinant, chimeric simian-human monoclonal Abs (mAbs). Such mAbs will specifically recognize not only the fluorescently labeled mimotope fusion protein used for the selection procedure, but also the corresponding antigen, with the epitope specificity determined by the nature of the mimotope used for the selection.</p>
<a class="link-target" id="pone-0038943-g001" name="pone-0038943-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.g001" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Strategy of isolation of epitope-specific Abs.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>After differential biopanning of phage display peptide library, mimotopes representing conformational epitopes are sequenced, and the inserts plus M13 flanking sequences are cloned into a bacterial expression vector encoding fluorescent protein. After purification of the resulting mimotope fusion proteins, binding assays to assure maintenance of the correct three-dimensional structure of mimotope insert are conducted. This is followed by single-cell sorting for specific IgG-positive memory B cells, single-cell RT-PCR and cloning of RM V<sub>L</sub> and V<sub>H</sub> regions into pFUSE2-type vectors encoding the backbone of human IgG1.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.g001">
              https://doi.org/10.1371/journal.pone.0038943.g001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Isolation of Conformational Mimotopes"></a>
<h3>Isolation of Conformational Mimotopes</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>To validate our new strategy, we sought to isolate mAbs specific for the V3 loop of the HIV envelope glycoprotein, using conformation-dependent mimotopes. The V3 loop is a prime target for nAbs and has conserved structural features in the crown, although the entire loop structure is known for its remarkable sequence variability <a href="#pone.0038943-Jiang1" class="ref-tip">[13]</a>. Anti-V3 mAbs isolated and characterized by others <a href="#pone.0038943-Gorny1" class="ref-tip">[14]</a>–<a href="#pone.0038943-Hioe1" class="ref-tip">[17]</a> allowed us to compare and validate the technology presented here.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>We used RMs that had developed high-titer, broadly neutralizing Abs (nAbs) during chronic infection with simian-human immunodeficiency virus (SHIV) (<a href="#pone.0038943.s007">Table S1</a> and <a href="#pone.0038943.s008">Table S2</a>) for B-cell isolation. First, we affinity-purified polyclonal Abs that contained paratopes specific for conformational epitopes on the original antigen molecule. In parallel, we also affinity-purified polyclonal Abs recognizing the same antigen in denatured form. Towards this end (<a href="#pone-0038943-g002">Figure 2</a>), wells of 96-well plates were coated with HIV gp160 under native or denaturing conditions and incubated with serum of monkey RKl-8. Bound Abs were eluted by pH shift, identical fractions were combined, concentrated and used for biopanning of random peptide phage libraries. “Conformational” Abs (eluted from native, multimeric gp160, shown in blue, <a href="#pone-0038943-g002">Figure 2</a>) were used for positive selection; “linear” Abs (eluted from denatured gp160, shown in green, <a href="#pone-0038943-g002">Figure 2</a>) for negative selection. Three positive/two negative rounds yielded mimotopes that were tested for gp160 specificity and conformational dependence with RM sera. Interestingly, only 12 mimotopes resembling the V3 loop crown of HIV gp120 (<a href="#pone-0038943-g003">Figure 3A</a>) were selected using this method of differential biopanning with Abs specific for either native or denatured gp160. Some of these conformational V3 mimotopes were chosen to test the feasibility of our overall strategy. Notably, our differential biopanning strategy resulted in the selection of mimotopes with strong conformational dependence as shown by ELISA performed with native or denatured phages and probed with serum from RM RKl-8 (<a href="#pone-0038943-g003">Figure 3B</a>). The fact that only V3-like mimotopes were found may be ascribed to the stringent selection conditions. It should be noted that mimotopes positively selected by the “conformational” Abs (shown in blue, <a href="#pone-0038943-g002">Figure 2</a>) are discarded in the subsequent negative selection step if they are also bound to “linear” Abs (shown in green, <a href="#pone-0038943-g002">Figure 2</a>). As a consequence, the only mimotopes that remain are those with high specificity and selective binding to the “conformational” Abs, a condition fulfilled by the sterically constrained V3 loop.</p>
<a class="link-target" id="pone-0038943-g002" name="pone-0038943-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.g002" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Differential biopanning strategy to select conformational mimotopes of desired specificity.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>HIV gp160 was bound to ELISA plates under native or denaturing conditions. After incubation with serum of monkey RKl-8 (which was chronically infected with a clade C simian-human immunodeficiency virus (SHIV) <a href="#pone.0038943-Humbert1" class="ref-tip">[25]</a> and had high titers of anti-HIV nAbs) followed by washing, Abs were eluted. Abs eluted from the native protein antigen (“conformational” Abs) were used for the positive selection rounds; Abs eluted from the denatured protein (“linear” Abs) were used for the negative rounds. A total of three biopanning rounds were performed and mimotopes selected by this strategy were tested for specificity and conformation dependence with rhesus monkey sera.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.g002">
              https://doi.org/10.1371/journal.pone.0038943.g002</a></p></div><a id="article1.body1.sec2.sec2.p3" name="article1.body1.sec2.sec2.p3" class="link-target"></a><p>Next, we cloned, bacterially expressed and purified mWasabi-mimotope fusion proteins (termed mWasabi-mimes, <a href="#pone.0038943.s001">Figure S1</a>). We analyzed the latter by ELISA with sera from animals with high-titer nAbs to confirm specificity and cross-recognition. Mimotope Tc.2 was recognized by serum of different RMs and serum of monkey RJa-9 demonstrated the highest binding (<a href="#pone-0038943-g003">Figure 3C</a>). This mimotope closely resembles the V3 loop crown in amino acid sequence (<a href="#pone-0038943-g003">Figure 3A</a>). Conformational similarity of mimotope and V3 loop is demonstrated by molecular modeling and structural superimposition shown on <a href="#pone-0038943-g003">Figure 3D</a>. Noteworthy, Tc.2 was isolated from cyclic 7mer peptide library and is flanked by two cysteines forming disulfide bridge thus stabilizing the mimotope structure.</p>
<a class="link-target" id="pone-0038943-g003" name="pone-0038943-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.g003" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> Properties of V3 loop mimotopes.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig2.caption1.p1" name="article1.body1.sec2.sec2.fig2.caption1.p1" class="link-target"></a><p>(A) Binding of rhesus monkey RKl-8 serum Abs to native and denatured mimotopes. Wild-type phages (WT) were used as negative control, an anti-phage Ab (anti-M13 pIII) as positive control to show correct phage immobilization. Each data point represents the mean ± s.e.m. (n  = 3). *Mimotope selected for B-cell sorting. (B) Sequence alignment of V3 loop region of gp160 and mimotopes isolated by differential biopanning. Amino acid colors indicate: blue, basic; green, non-polar (hydrophobic); orange, polar (uncharged); magenta (acidic). (C) Binding of RM sera to different V3 mimotopes. Plates were coated with mWasabi-mimotope fusion proteins and parental mWasabi as control and then probed with sera of rhesus monkeys with high nAb titers. Serum from monkey RKl-8 served as positive control; serum from a naïve animal was used as negative control in all experiments. Each data point represents the mean ± s.e.m. (n  = 3). (D) Structural superimposition of Tc.2 mimotope and V3 peptide obtained by <em>in silico</em> molecular modeling. Tc.2 mimotope was modeled using the published V3 peptide structure in complex with Fab2557 <a href="#pone.0038943-Jiang1" class="ref-tip">[13]</a>. The original V3 loop is shown in blue, the Tc.2 mimotope in orange. The cysteine residues and disulfide bridge are in green.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.g003">
              https://doi.org/10.1371/journal.pone.0038943.g003</a></p></div></div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Isolation of Single Memory B Cells"></a>
<h3>Isolation of Single Memory B Cells</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>The cross-reactivity of mimotope Tc.2 with sera from different RMs implied mimicry of a conserved HIV Env epitope by Tc.2. Consequently, we selected the corresponding mWasabi-Tc.2 fusion protein as bait to isolate cognate memory B cells from PBMC of monkey RJa-9 by flow cytometry. Mimotope-specific single memory B cells were sorted into a PCR plate as follows: CD3<sup>−</sup>/CD19<sup>+</sup>/CD27<sup>+</sup>/IgG<sup>+</sup>/mWasabi-mime<sup>+</sup> (<a href="#pone-0038943-g004">Figure 4</a>). For the experiment 5×10<sup>7</sup> PBMC were used and the cytometer was programmed to collect 90 individual CD3<sup>−</sup>/CD19<sup>+</sup>/CD27<sup>+</sup>/IgG<sup>+</sup>/mWasabi-mime<sup>+</sup> cells at one cell per well in the plate. Mimotope-specific cells represented approximately 0.1% of the memory B cells and as many as 8 mimotope-specific IgG-positive memory B cells were observed per million lymphocytes sorted. Control staining with mWasabi yielded only one non-specific cell per million lymphocytes, possibly due to autofluorescence.</p>
<a class="link-target" id="pone-0038943-g004" name="pone-0038943-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.g004" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Isolation of single V3 loop-specific memory B cells by FACSorting.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>Mimotope-specific memory B cells were selected by the following phenotype: CD3<sup>−</sup>/CD19<sup>+</sup>/CD27<sup>+</sup>/IgG<sup>+</sup>/mWasabi-mimotope<sup>+</sup> and were sorted into individual wells of a 96-well PCR plate. The consecutive number of gates is shown in the left upper corner of each gate.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.g004">
              https://doi.org/10.1371/journal.pone.0038943.g004</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Single-cell PCR and mAb Cloning"></a>
<h3>Single-cell PCR and mAb Cloning</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>After cDNA synthesis, light and heavy chain variable Ig genes were amplified by semi-nested PCR with a set of newly developed primers specific to RM Ig genes (<a href="#pone.0038943.s009">Table S3</a>). RM IgG-specific primers published to date do not result in the amplification of V<sub>H</sub> and V<sub>L</sub> genes of all RM subfamilies <a href="#pone.0038943-Margolin1" class="ref-tip">[18]</a>, whereas our new primer set led to the amplification of all RM V<sub>H</sub> and V<sub>L</sub> genes and yielded frequencies similar to their natural usage rate (<a href="#pone.0038943.s010">Table S4</a>). This procedure allowed us to derive V<sub>H</sub> gene fragments in 31 wells and V<sub>L</sub> gene fragments in 19 wells, respectively. In 12 wells, both V<sub>H</sub> and V<sub>L</sub> genes were amplified.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>Next, we sequenced matching pairs of V<sub>H</sub> and V<sub>L</sub> genes obtained after two rounds of PCR to assess productivity and gene rearrangement as well as to obtain sequence information for the beginning of framework region 1 (FR1). After amplification of V<sub>H</sub>/V<sub>L</sub> pairs with cloning primers, we inserted the PCR fragments into vectors of the pFUSE2-family that contain constant region sequences of human Ig light (Igκ or Igλ2) or heavy (Igγ1) chains. This cloning strategy yielded chimeric simian-human IgG1 mAbs.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Functional Characterization of mAbs"></a>
<h3>Functional Characterization of mAbs</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Out of 12 isolated V<sub>H</sub>/V<sub>L</sub> pairs, 11 chimeric mAbs were expressed and purified. Two of them, 33B2 and 33C6, exhibited specific binding to the Tc.2 mimotope and HIV-1<sub>CN54</sub> gp120 (<a href="#pone.0038943.s002">Figure S2</a>) as well as HIV envelopes of different clades by ELISA (<a href="#pone-0038943-g005">Figure 5A</a>, <a href="#pone.0038943.s003">Figure S3</a>). The weaker binding of mAbs to Tc.2 compare to gp120 (<a href="#pone.0038943.s002">Figure S2</a>), especially 33B2, can be explained by the fact that the mimotope represents only part of entire epitope due to the relatively short length.</p>
<a class="link-target" id="pone-0038943-g005" name="pone-0038943-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.g005" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> Properties of mAbs 33B2 and 33C6.</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>(A) Binding to envelope proteins of different HIV clades. Plates were coated with Env proteins and probed with newly isolated mAbs along with isotype controls (positive control, HGN194 <a href="#pone.0038943-Corti2" class="ref-tip">[16]</a>; negative control, Fm-6 <a href="#pone.0038943-Sui1" class="ref-tip">[37]</a>) and cognate rhesus monkey (RJa-9) serum. HIV Env proteins were derived from the following strains: clade A, UG37; B, BaL and IIIB; C, CN54 and 1157ip (the latter is a simian-human immunodeficiency virus (SHIV) strain encoding an HIV clade C envelope <a href="#pone.0038943-Humbert1" class="ref-tip">[25]</a>; D, UG21. SIVmne gp160 was used as negative control. (B) Epitope mapping. Bars show Ab binding by ELISA to consensus HIV clade C peptides representing the V3 loop region. The control peptide represents the scrambled C-terminal sequence of SHIV-1157ip gp120. (C) Virion binding assay. MAbs 33B2 and 33C6 along with negative control (Fm-6) and positive control (HGN194) were captured on the plate coated with goat anti-human IgG Fc-specific Ab and then exposed to SHIV-1157ipEL-p <a href="#pone.0038943-Siddappa1" class="ref-tip">[19]</a> virions. The amount of virus bound to mAbs was assessed by p27 Gag ELISA. (D) Binding of mAbs 33B2 and 33C6 with native and denatured HIV<sub>CN54</sub> gp120. Plates were coated with native, reduced or reduced/denatured HIV Env protein and probed with 33B2 and 33C6 along with Fm-6 (negative control mAb) as well as serum of HIV-positive individual (positive control), and serum of HIV-negative individual (negative control). We used <em>tris</em>(2-carboxyethyl)phosphine (TCEP) to reduce gp120 and TCEP + SDS to reduce and denature, respectively (<a href="#s4">Methods</a>). Each data point represents the mean ± s.e.m. (n  = 3). Experiments were repeated 3 times.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.g005">
              https://doi.org/10.1371/journal.pone.0038943.g005</a></p></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>To confirm the expected V3 loop epitope specificity of mAbs 33B2 and 33C6, peptides representing the V3 loop of consensus HIV clade C gp120 were used for epitope mapping. Peptide binding analysis revealed a common epitope recognized by both mAbs (<a href="#pone-0038943-g005">Figure 5B</a>, <a href="#pone.0038943.s004">Figure S4A</a>): the sequence RKSIRIG, which is located in the V3 loop crown and resembles the epitope of the known anti-V3 mAb, HGN194 <a href="#pone.0038943-Corti2" class="ref-tip">[16]</a>. Interestingly, mAb 33B2 interacted only with peptides 9259 and 9261 bearing the epitope either at the N- or C-termini but did not bind to peptide 9260, where the epitope is in the middle, presumably due to improper conformation of the linear peptide. Peptide 9261 showed the highest degree of inhibition of mAbs binding to gp120 (<a href="#pone.0038943.s004">Figure S4B and S4C</a>). Lower binding of mAb 33B2 compared to binding of 33C6 to linear peptides 9260 and 9261 implies that epitope of 33B2 is more conformation-dependent.</p>
<a id="article1.body1.sec2.sec5.p3" name="article1.body1.sec2.sec5.p3" class="link-target"></a><p>Since the mimotope used as B-cell bait was selected to be strongly conformational, we postulated that 33B2 and 33C6 would preferentially bind to native as opposed to denatured envelope proteins. To confirm binding to native Env in trimeric form, we performed a virion binding assay, in which both mAbs successfully captured virions of the chimeric simian-human immunodeficiency virus (SHIV) encoding <em>env</em> of a recently transmitted HIV clade C, SHIV-1157ipEL-p <a href="#pone.0038943-Siddappa1" class="ref-tip">[19]</a> (<a href="#pone-0038943-g005">Figure 5C</a>). Upon Env denaturation, binding of both mAbs was strongly diminished by ELISA (<a href="#pone-0038943-g005">Figure 5D</a>); denaturation greatly abrogated binding of both mAbs to gp120, whereas reduction of disulfide bridges did not affect binding.</p>
<a id="article1.body1.sec2.sec5.p4" name="article1.body1.sec2.sec5.p4" class="link-target"></a><p>Next, we assessed potency/breadth of virus neutralization of the two mAbs with a panel of neutralization-sensitive (tier 1) and more difficult-to-neutralize (tier 2) viruses. Overall, 33B2 and 33C6 neutralized 75% or 91% of the tier 1 HIV strains of different clades tested (<a href="#pone-0038943-t001">Table 1</a>), respectively. In contrast, few of the tier 2 strains tested were neutralized by the two new mAbs (7% and 15%, respectively). These results are similar to the neutralization profiles of other anti-V3 Abs <a href="#pone.0038943-Gorny2" class="ref-tip">[15]</a>–<a href="#pone.0038943-Hioe1" class="ref-tip">[17]</a>. The fact that we preselected conformation-dependent mimotopes increased our chances of finding neutralizing mAbs since most virus-neutralizing Abs are conformation dependent.</p>
<a class="link-target" id="pone-0038943-t001" name="pone-0038943-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.t001" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Neutralization activity of mAbs 33B2 and 33C6.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.t001">
              https://doi.org/10.1371/journal.pone.0038943.t001</a></p></div></div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Genetic Analysis of mAbs"></a>
<h3>Genetic Analysis of mAbs</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>Genetic analysis of 33B2 and 33C6 V<sub>H</sub> and V<sub>L</sub> genes revealed that these mAbs likely originated from the same parental B cell clone, because they share the same V<sub>H</sub> and J<sub>H</sub> genes, IGHV5-51*01 and IGHJ4*2, respectively (<a href="#pone-0038943-t002">Table 2</a>). Although 33B2 and 33C6 used different D genes, the complementarity-determining region 3 (CDR3) of both mAbs are 16 amino acids long and have very similar amino acid sequences (<a href="#pone-0038943-g006">Figure 6</a>). In contrast, the light chains recruited different genes, IGLV1-50 and IGLV1-47, from the same subfamily (<a href="#pone-0038943-t002">Table 2</a>). The CDR sequence homology indicates convergence of the molecular evolution of both mAbs (<a href="#pone-0038943-g006">Figure 6</a>). Our results are in good agreement with data about Ig gene usage by other anti-V3 Abs <a href="#pone.0038943-Gorny1" class="ref-tip">[14]</a>. Compared to the RM germline, the frequency of amino acid mutations observed in VH was 15% for mAb 33B2 and 18% for 33C6, whereas for VL, the mutation frequencies were 10% and 15%, respectively (<a href="#pone.0038943.s005">Figure S5</a> and <a href="#pone.0038943.s006">Figure S6</a>). These values are significantly lower compared to the recently identified anti-CD4 binding site neutralizing mAbs, but comparable to the parameters of other anti-V3 mAbs and of mAb b12 <a href="#pone.0038943-Scheid1" class="ref-tip">[11]</a>, <a href="#pone.0038943-Wu1" class="ref-tip">[12]</a>, <a href="#pone.0038943-Corti2" class="ref-tip">[16]</a>, <a href="#pone.0038943-Parren1" class="ref-tip">[20]</a>–<a href="#pone.0038943-Zhang1" class="ref-tip">[23]</a>. Neither mAb exhibited autoreactivity in assays involving double-stranded (ds) DNA, Sm, ribonucleoprotein (RNP), SS-A/Ro and SS-B/La antigens, cardiolipin and nuclear antigens (data not shown).</p>
<a class="link-target" id="pone-0038943-g006" name="pone-0038943-g006"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.g006" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 6. </span> VH and VL CDR sequences of V3-specific mAbs.</div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1" class="link-target"></a><p>(A) 33B2 and 33C6 VH CDR amino acid sequences. (B) 33B2 and 33C6 VL CDR amino acid sequences.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.g006">
              https://doi.org/10.1371/journal.pone.0038943.g006</a></p></div><a class="link-target" id="pone-0038943-t002" name="pone-0038943-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0038943.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0038943.t002" data-doi="info:doi/10.1371/journal.pone.0038943" data-uri="info:doi/10.1371/journal.pone.0038943.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0038943.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0038943.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0038943.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0038943.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Genetic characteristics of V3-specific mAbs.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0038943.t002">
              https://doi.org/10.1371/journal.pone.0038943.t002</a></p></div></div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>Our data give proof-of-concept that the novel mAb isolation strategy, together with the newly developed set of primers specific to RM Ig genes, yielded mAbs with the epitope specificity as predicted by the fluorescently labeled mimotope. Our methodology represents a direct approach to isolate naturally occurring, matured Abs as mAbs without the need of extensive screening and provides a practical tool for analytical vaccinology. In contrast to other methods of mAb generation using phage display with Ig fragments that yield randomly recombined Ig chains, isolating single memory B cells will allow probing of natural Ab repertoires. Our focus on isolating peripheral blood memory B cells as preferred source of matured V<sub>H</sub>/V<sub>L</sub> genes has an advantage compared to plasma cells. The latter lack B-cell receptors (BCR), whereas memory B cells bear antigen receptors recognizing only one epitope; these receptors serve as targets that are specifically recognized by fluorescently labeled mimotopes.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>An important aspect of our new technology is the use of conformational mimotopes, which we selected with affinity-purified polyclonal RM Abs specific for either native or denatured antigen. We used the power of differential biopanning to select mimotopes representing conformational epitopes and thus increased the chances of isolating biologically relevant V<sub>H</sub>/V<sub>L</sub> genes, since most Ab paratopes are directed against conformational targets on the antigen.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>Our new approach to isolate conformation-specific mAbs with predetermined epitope specificity is applicable to any antigen. General strategies depicted in <a href="#pone-0038943-g001">Figures 1</a> and <a href="#pone-0038943-g002">2</a> are not restricted to isolating HIV envelope-related mAbs. We chose a V3 mimotope to illustrate the power and practicality of our new approach. By opting to isolate mAbs with a narrow, well defined epitope specificity, we expect to observe far fewer “hits” when isolating memory B cells compared to standard technologies using entire antigens as baits. The benefits of our targeted approach will be that the epitope specificity will be known, thereby obviating a labor-intensive mapping effort. Another approach to zero in on a specific antigen region involved an HIV gp120 that had been redesigned to diminish undesirable antigenic determinants to facilitate the isolation of B cells specific for the CD4-binding site <a href="#pone.0038943-Wu1" class="ref-tip">[12]</a>. The latter is known to encompass a number of different epitopes. This approach required resolving of spatial structure of the antigen as well as sophisticated protein modeling. In contrast, empirical selection of epitope mimetics by differential biopanning of phage display peptide libraries using polyclonal Abs provides a fast and easy technique to generate mimotopes resembling conformational epitopes from infected or immunized hosts.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>It should be pointed out that our single cell-based mAb isolation does not depend on using mimotopes with stringent conformation dependence as baits. In fact, we have performed a number of RM B-cell isolations with mimotopes that had some but not exclusive conformation dependence (unpublished data). In fact, we have performed biopannings with more than 20 RM sera from vaccinated and/or SHIV-infected animals using standard rather than differential biopanning. In such cases, we are generally able to assign the resulting mimotopes to the corresponding entire antigen of interest in approximately 90% of the cases based upon partial amino acid similarity. However, in exceptional cases, as many as 40% to 50% of the mimotopes could not be assigned and are likely representing quaternary conformational regions.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>Our choice of RMs as source for V<sub>H</sub>/V<sub>L</sub> genes will potentially yield mAbs with clinical applicability as prophylactic or therapeutic agents. Of note, RMs and humans have a closely related repertoire of Ig genes, much more so than mice, rabbits or other species used routinely for mAb isolation. Our findings of V<sub>H</sub> gene usage, length of CDR3 and frequency of mutations are similar to anti-V3 Abs derived from HIV-infected individuals, thus confirming this notion.</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6" class="link-target"></a><p>Although we demonstrated the ability to generate mAbs with predetermined epitope specificity using the RM model of immunodeficiency virus infection, our approach can be readily extended to any target of choice, such as experimental or natural infection with potential pathogens, autoimmune diseases and cancer.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="Animals"></a>
<h3>Animals</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>The current in vitro study only used blood samples from animals listed in <a href="#pone.0038943.s007">Table S1</a>. These RMs, chronically infected with clade C SHIV, were part of our ongoing study to establish and optimize a model to test experimental vaccine approaches against the most frequent clade of HIV worldwide, and data from each of these animals have been published previously <a href="#pone.0038943-Siddappa1" class="ref-tip">[19]</a>, <a href="#pone.0038943-AyashRashkovsky1" class="ref-tip">[24]</a>–<a href="#pone.0038943-Song1" class="ref-tip">[27]</a>. Animal experiments were carried out in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the U.S. Public Health Services/National Institutes of Health, as well as according to the recommendations in the Weatherall report on ‘‘The Use of Non-human Primates in Research’’ (<a href="http://www.acmedsci.ac.uk/images/project/nhpdownl.pdf">http://www.acmedsci.ac.uk/images/project/nhpdownl.pdf</a>). The protocol was approved by the Committee on the Ethics of Animal Experiments of Emory University (IACUC ID: YER-2001401-111814; Emory University Animal Welfare Assurance Number A3180-01). The rhesus monkeys were housed at the Yerkes National Primate Research Center (YNPRC, Emory University, Atlanta, GA). The animals were housed indoors with a 12 hour light/dark cycle, in individual cages but in visual and auditory contact with other RMs, were fed monkey chow (Purina) ad libitum supplemented daily with fresh fruit. Standard enrichment was provided by the YNPRC enrichment staff including perches, rubber toys and varied treats such as peanuts and cereals. Blood was collected under ketamine or Telazol anesthesia from the femoral vein. YNPRC facilities are fully accredited by the Association for Assessment and Accreditation of Laboratory Animal Care International. Animal experiments were approved by the Institutional Animal Care and Use Committees at Emory and the Dana-Farber Cancer Institute via a Collaborating Institution Animal Use Agreement. Because the experiments described here involved a virus that may cause an incurable disease, such as AIDS, discomfort, stress and pain may occur. Animals were closely monitored and observed for development of disease at least twice daily. Animals determined to be under stress or in discomfort, were administered appropriate anesthetics and/or analgesics as directed by the clinical veterinary staff. If stress cannot be alleviated, the animals are euthanized.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="Collection of Human PBMC"></a>
<h3>Collection of Human PBMC</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>This study made use of anonymous human blood donor samples for in vitro neutralization tests. These cells were derived from the blood bank at Brigham and Women’s Hospital (Boston, MA), where leukocytes were collected in Trima collars during platelet pheresis. The donors were prescreened and found to be negative for blood-borne pathogens, including HIV-1, hepatitis, and others. The use of Trima-collar derived PBMC for in vitro neutralization assays was reviewed by the IRB; this use of human PBMC was considered to be exempt.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="Envelope Proteins and Peptides"></a>
<h3>Envelope Proteins and Peptides</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>HIV envelope proteins of strains UG37, BaL, IIIB, CN54 and UG21 along with consensus clade C peptides were kindly provided by the NIH AIDS Research and Reference Reagent Program (ARRRP). SHIV-1157ip and SIVmne envelope proteins were generated by recombinant vaccinia virus technology <a href="#pone.0038943-Klaniecki1" class="ref-tip">[28]</a>.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="Differential Biopanning"></a>
<h3>Differential Biopanning</h3>

<div id="section1" class="section toc-section"><a id="s4d1" name="s4d1" class="link-target" title="Affinity purification of polyclonal Abs specific for native vs. denatured antigens"></a><h4>Affinity purification of polyclonal Abs specific for native vs. denatured antigens.</h4><a id="article1.body1.sec4.sec4.sec1.p1" name="article1.body1.sec4.sec4.sec1.p1" class="link-target"></a><p>Affinity purification was performed in 96-well plates coated overnight with HIV gp160<sub>SHIV-1157ip</sub> under native or denaturing conditions. On the next day, plates were washed, blocked and incubated overnight with plasma of monkey RKl-8. The following day, plates were extensively washed, and bound Abs were eluted by pH shift. Identical fractions were combined, concentrated and used for biopanning of random peptide phage libraries.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4d2" name="s4d2" class="link-target" title="Biopanning"></a><h4>Biopanning.</h4><a id="article1.body1.sec4.sec4.sec2.p1" name="article1.body1.sec4.sec4.sec2.p1" class="link-target"></a><p>Biopanning of random peptide phage display libraries with polyclonal clade C SHIV-infected RM serum is described elsewhere <a href="#pone.0038943-Humbert2" class="ref-tip">[29]</a>. Briefly, paramagnetic beads (Dynabeads M-280 tosyl-activated; Invitrogen) were coated with rabbit anti-monkey IgG (Sigma-Aldrich) and pre-incubated with serum Abs affinity-purified using native gp160<sub>SHIV-1157ip</sub> (positive selection). After an overnight incubation with 10 µl of the original phage display peptide libraries (7mer, cyclic 7mer, 12mer; New England Biolabs), beads were intensively washed and bound phages were eluted by a 10 min pH shift using 0.2 M glycine-HCl pH 2.2 supplemented with 1 mg/ml BSA (Sigma-Aldrich) and subsequently neutralized with 1 M Tris-HCl pH 9.1 (Sigma-Aldrich). Eluted phages were used for negative selection with serum Abs that had been affinity-purified with denatured gp160<sub>SHIV-1157ip</sub> (boiled for 3 min in carbonate buffer (pH 9.6) containing 0.05 M TCEP, <em>tris</em>(2-carboxyethyl)phosphine (Pierce) and 2% SDS (Sigma-Aldrich). Phages remaining in the supernatant after negative selection were amplified for 4.5 h in <em>E. coli</em> (ER2738, New England Biolabs), precipitated overnight at 4°C (20% PEG-8000/2.5 M NaCl; Fisher Scientific), collected, precipitated a second time and subjected to two more positive/one more negative selection rounds. Eluted phages from the third positive selection were titered on LB plates and single clones were picked and tested by phage ELISA for specific binding. Single-stranded DNA of positive clones was isolated and sequenced to deduce the sequences of the peptide inserts, which were grouped into motifs and assigned to sequences of the Env-C protein, where possible.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s4e" name="s4e" class="link-target" title="Molecular Modeling and Structure Images"></a>
<h3>Molecular Modeling and Structure Images</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>Protein modeling and energy calculations were performed using Discovery Studio (Accelrys Software, Inc.) based on a sequence alignment of the V3-mimotope with the core structure corresponding to the X-ray structure of ZAM18 V3 peptide in complex with Fab2557 <a href="#pone.0038943-Jiang1" class="ref-tip">[13]</a> (pdb code 3MLU). Energies were calculated using CHARMM (Chemistry at Harvard Macromolecular Mechanics). We introduced solvent factors using the implicit of distance-dependent dielectrics model and performed energy minimization of the mimotope (Steepest Descent following by Conjugate Gradient).</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2" class="link-target"></a><p>Molecular graphics image for antibody <a href="#pone.0038943-Harris1" class="ref-tip">[30]</a> (pdb code 1IGT) was produced using the UCSF Chimera package from the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by NIH P41 RR001081). Model for mWasabi was generated by Swiss-Model Workspace <a href="#pone.0038943-Kiefer1" class="ref-tip">[31]</a> (Swiss Institute of Bioinformatics &amp; the Biozentrum University of Basel) and the image was produced with UCSF Chimera package.</p>
</div>

<div id="section6" class="section toc-section"><a id="s4f" name="s4f" class="link-target" title="Generation of pET-mWasabi.II-mime Vector and mWasabi-mimotope Fusion Expression (Figure S1)"></a>
<h3>Generation of pET-mWasabi.II-mime Vector and mWasabi-mimotope Fusion Expression (<a href="#pone.0038943.s001">Figure S1</a>)</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>First, sequences encoding mWasabi with a N-terminal His<sub>6</sub>-tag were amplified from the parental plasmid pNCS-mWasabi (Allele Biotechnology) with primers to introduce NdeI and EcoRI restriction sites (<a href="#pone.0038943.s011">Table S5</a>) and cloned into the pET-22b(+) vector (Novagen). The resulting plasmid was termed pET-mWasabi.II. Next, mimotopes plus flanking regions were amplified from phage DNA with primers to introduce EcoRI and HindIII restriction sites (<a href="#pone.0038943.s011">Table S5</a>) followed by cloning into pET-mWasabi.II at the C-terminus of the fluorescent protein. The mWasabi-mimotope fusion proteins were expressed in BL21 <em>E.coli</em> and purified on the Ni<sup>2+</sup>-agarose according to the manufacturer instruction.</p>
</div>

<div id="section7" class="section toc-section"><a id="s4g" name="s4g" class="link-target" title="ELISAs"></a>
<h3>ELISAs</h3>
<a id="article1.body1.sec4.sec7.p1" name="article1.body1.sec4.sec7.p1" class="link-target"></a><p>Phage binding ELISA and mimotope fusion protein ELISA were performed as published <a href="#pone.0038943-Humbert2" class="ref-tip">[29]</a>.</p>

<div id="section1" class="section toc-section"><a id="s4g1" name="s4g1" class="link-target" title="ELISAs with cell culture supernatants and purified mAbs"></a><h4>ELISAs with cell culture supernatants and purified mAbs.</h4><a id="article1.body1.sec4.sec7.sec1.p1" name="article1.body1.sec4.sec7.sec1.p1" class="link-target"></a><p>ELISA plates were coated with 50–100 ng/well of antigens (mimotope fusion proteins, HIV proteins or V3 loop consensus peptides) in carbonate buffer, pH 9.6. After washing, plates were blocked with 2% bovine serum albumin (BSA) (Sigma-Aldrich), 0.05% Tween-PBS (blocking buffer). Plates were then incubated with cell supernatants diluted with blocking buffer or purified mAbs diluted at 1 µg/ml with several consecutive 1∶4 dilutions in blocking buffer. After washing, the plates were developed by incubation for 1 h with goat HRP-conjugated anti-human IgG (Jackson ImmunoResearch) and by adding 100 µl of <em>o</em>-phenylenediamine or TMB solution. Optical densities were measured at the appropriate wavelength using an ELISA microplate reader (Mithras LB 940, Berthold Technologies).</p>
</div>

<div id="section2" class="section toc-section"><a id="s4g2" name="s4g2" class="link-target" title="“Denatured” ELISA"></a><h4>“Denatured” ELISA.</h4><a id="article1.body1.sec4.sec7.sec2.p1" name="article1.body1.sec4.sec7.sec2.p1" class="link-target"></a><p>Wells of 96-well plates were coated with gp160<sub>SHIV-1157ip</sub> under native or denaturing conditions. For denaturation, antigen was diluted to at 0.5 µg/ml into carbonate buffer (pH 9.6) containing 0.05 M TCEP and 2% SDS and boiled for 3 min. Antigen in coating buffer served as “native”. Subsequent ELISAs were performed similarly as described above.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4g3" name="s4g3" class="link-target" title="Competitive ELISA"></a><h4>Competitive ELISA.</h4><a id="article1.body1.sec4.sec7.sec3.p1" name="article1.body1.sec4.sec7.sec3.p1" class="link-target"></a><p>The interaction of anti-V3 loop mAbs with solid-phase HIV<sub>CN54</sub> gp120 was inhibited by peptides corresponding to the V3 loop of consensus clade C gp120 sequence (<a href="#pone.0038943.s004">Figure S4A</a>). A negative peptide control consisted of a scrambled C-terminal gp120 peptide (24 amino acids long; synthesized by the Molecular Biology Core Facilities, DFCI).</p>
</div>
</div>

<div id="section8" class="section toc-section"><a id="s4h" name="s4h" class="link-target" title="Primer Design"></a>
<h3>Primer Design</h3>
<a id="article1.body1.sec4.sec8.p1" name="article1.body1.sec4.sec8.p1" class="link-target"></a><p>To design primers specific for RM Ig genes (<a href="#pone.0038943.s009">Table S3</a>), we analyzed RM Ig V gene sequences obtained in our group as well as sequences available through public databases, IMGT, the international ImMunoGeneTics information system® (IMGT/LIGM-DB) and NCBI, The National Center for Biotechnology Information. Sequence analysis and germline gene identification were performed using the IMGT-V-Quest software; sequence alignment was done with DNAStar Lasergene software package (DNAStar Inc). In addition, using known sequences of rare human V genes, the RM genome was blasted to identify potential counterparts. To validate newly designed primers, V<sub>H</sub> and V<sub>L</sub> genes were amplified using cDNA derived from mRNA isolated from the mixture of bone marrow B cells of 10 RMs. The resulting PCR products were cloned into the sequencing plasmid vector, pCR4-TOPO (Invitrogen), and sequenced. Sequences of 300 randomly chosen clones were analyzed.</p>
</div>

<div id="section9" class="section toc-section"><a id="s4i" name="s4i" class="link-target" title="Mimotope-specific Memory B-cell Isolation"></a>
<h3>Mimotope-specific Memory B-cell Isolation</h3>
<a id="article1.body1.sec4.sec9.p1" name="article1.body1.sec4.sec9.p1" class="link-target"></a><p>RM PBMC were obtained from YNPRC by overnight shipping in BD Vacutainer® CPT™ Cell Preparation Tubes (Beckton Dickinson, BD). Upon arrival, cells were washed with RPMI 1640 (Invitrogen) supplemented with 15% fetal calf serum (FCS) (Sigma-Aldrich) followed by PBS containing 0.2% BSA (Sigma-Aldrich). Then, cells were incubated with APC-Cy™7 mouse anti-human CD3 (BD Pharmingen), APC mouse anti-human CD27 (BD Pharmingen), PE-Cy™7 mouse anti-human IgG (BD Pharmingen), anti-CD19-R-Phycoerythrin (Beckman Coulter) and mWasabi-mimotope fusion protein for 30 min on ice in PBS-BSA. Controls were performed with isotype controls and mWasabi. Cells were then washed twice with ice cold PBS-BSA and single-cell sorted (FACSAria II (BD Biosciences)) into 96-well PCR plates as described <a href="#pone.0038943-Tiller1" class="ref-tip">[32]</a>. Live single-cell sorting of RM mimotope-specific memory B cells under appropriate biocontainment was available through the Harvard CFAR Core facility. Plates were snap frozen on dry ice immediately after the completion of sorting.</p>
</div>

<div id="section10" class="section toc-section"><a id="s4j" name="s4j" class="link-target" title="Single B-cell RT-PCR and Expression Vector Cloning"></a>
<h3>Single B-cell RT-PCR and Expression Vector Cloning</h3>
<a id="article1.body1.sec4.sec10.p1" name="article1.body1.sec4.sec10.p1" class="link-target"></a><p>cDNA synthesis and Ig amplification were performed as previously described <a href="#pone.0038943-Tiller1" class="ref-tip">[32]</a>, with following modifications. The frozen plates with single memory B cells were thawed, and reverse transcription was performed by adding 3 µl of random hexamer primers (Applied Biosystems) at 50 µM, 1 µl of 10 mM dNTP mix (Invitrogen), 0.0625 µl of Igepal CA-630 (Sigma), 40 units of RNaseOUT™ (Invitrogen), 1.25 µl of 0.1 DTT (Invitrogen) and 0.25 µl of SuperScript III (Invitrogen) into each well. Reaction conditions for reverse transcription were as follows: 42°C for 10 min, 25°C for 10 min, 50°C for 60 min and 94°C for 5 min. The cDNA plates were stored at −20°C until further use. The IgH, Igλ and Igκ V genes were amplified independently by semi-nested PCR starting from 3 µl of cDNA as a template. All PCRs were performed in 96-well PCR plates in a total volume of 50 µl containing water, 5 µl of 10× buffer, 1 µl of dNTP mix, each at 10 mM, 1 µl of MgCl<sub>2</sub> at 25 mM (Qiagen), by 1 µl of forward primer mix and reverse primer (<a href="#pone.0038943.s009">Table S3</a>) at 10 µM, and 0.4 µl of HotStart Taq DNA polymerase (Qiagen). The PCR thermocycler program was: 95°C for 15 min; 50 cycles (95°C for 30 sec, 50°C for Igκ and Igλ or 54°C for IgH for 30 sec and 72°C for 1 min); 72°C for 10 min. For PCR2, 3 µl of PCR1 product were used as a template. Forward primers were as in PCR1 and reverse primer specific for PCR2 (<a href="#pone.0038943.s009">Table S3</a>). Fifty cycles were used with the same parameters as those for the first round, with annealing temperature of 52°C for Igκ and Igλ and 57°C for IgH for 30 sec. After PCR2, the fragments for matching heavy/light chain pairs were isolated and subjected to direct sequencing with the reverse PCR2 primer. PCR products that represented productive IgH, Igλ or Igκ rearranged sequences were reamplified from PCR1 using “cloning” primers. These primers contained unique restriction digest sites, a Kozak motif, leader peptide coding region and sequences complementary to FR1. After amplification, DNA fragments were gel purified, digested and cloned into pFUSE2-CLIg-hk, pFUSE2-CLIg-hl2 and pFUSE2-CHIg-hG1 (all from Invivogen) containing the backbones of human Igγ1, Igκ and Igλ, respectively, thus giving rise to chimeric simian-human mAbs.</p>
</div>

<div id="section11" class="section toc-section"><a id="s4k" name="s4k" class="link-target" title="Ab Production and Purification"></a>
<h3>Ab Production and Purification</h3>
<a id="article1.body1.sec4.sec11.p1" name="article1.body1.sec4.sec11.p1" class="link-target"></a><p>Full-length IgG1 mAbs were produced by transient cotransfection of the paired heavy and light chain pFUSE plasmids into 293-F cells (Invitrogen) grown in serum-free FreeStyle™ 293 Expression Medium (Gibco® Invitrogen) using the <em>Trans</em>IT-PRO™ Transfection Kit (Mirus Bio). Cells were cultivated for 3 days at 37°C/8% CO<sub>2</sub> with continuous shaking at 135 rpm. Supernatants were collected, filtered through 0.22 µm filters and supplemented with Halt Protease Inhibitor Cocktail (Thermo Fisher) and 100× penicillin-streptomycin solution (Gibco® Invitrogen). Next, supernatants were tested for binding to HIV Env and mimotopes, and positive mAbs were affinity-purified using Protein A agarose (GE Healthcare) according to manufacturer’s instructions. IgG concentrations were determined by measuring absorbance at 280 nm on Nanodrop 1000 (Thermo Scientific) using the IgG default protocol.</p>
</div>

<div id="section12" class="section toc-section"><a id="s4l" name="s4l" class="link-target" title="Virion Binding Assay"></a>
<h3>Virion Binding Assay</h3>
<a id="article1.body1.sec4.sec12.p1" name="article1.body1.sec4.sec12.p1" class="link-target"></a><p>ELISA plates (Nunc) were coated with 5 µg/ml of goat anti-human IgG Fc specific Ab (Jackson ImmunoResearch) overnight at 4°C. After blocking and washing, mAbs were added at 5 µg/ml and incubated for 2 h. The plates were washed, SHIV-1157ipEL-p was added to the mAbs and incubated for 20 h, after which the plates were washed again and incubated with 0.5% Triton X-100 for 1 h to release p27 from the virus bound to the various mAbs. The amount of p27 released was determined using a p27 SIV capture kit (ABL, Inc).</p>
</div>

<div id="section13" class="section toc-section"><a id="s4m" name="s4m" class="link-target" title="Neutralization Assays"></a>
<h3>Neutralization Assays</h3>
<a id="article1.body1.sec4.sec13.p1" name="article1.body1.sec4.sec13.p1" class="link-target"></a><p>The TZM-bl and M7-Luc assays were performed as described elsewhere <a href="#pone.0038943-Montefiori1" class="ref-tip">[33]</a> and the human PBMC assay was performed as reported <a href="#pone.0038943-Watkins1" class="ref-tip">[34]</a>.</p>

<div id="section1" class="section toc-section"><a id="s4m1" name="s4m1" class="link-target" title="A3R5 cell assay"></a><h4>A3R5 cell assay.</h4><a id="article1.body1.sec4.sec13.sec1.p1" name="article1.body1.sec4.sec13.sec1.p1" class="link-target"></a><p>A3R5 (A3.01/R5.6) cells are derived from the CEM human lymphoblastoid cell line A3.01 <a href="#pone.0038943-Folks1" class="ref-tip">[35]</a> and engineered to express CCR5 (McLinden RJ, Chenine AL, LaBranche C, Perfetto S, Ochsenbauer C, Kappes J, Montefiori, DC, Kim JH. “Novel CD4+/CCR5+/CXCR4+ human T-cell line shows enhanced sensitivity of HIV-1 to neutralization by sCD4, mAbs and HIV-1-positive sera”, manuscript in preparation); cells were obtained from Drs. Jerome Kim and Robert McLinden at the US Medical HIV Research Program. The A3R5 assay was performed with Env.IMC.LucR viruses <a href="#pone.0038943-Edmonds1" class="ref-tip">[36]</a>. Neutralization titers reflect the sample concentration at which relative luminescence units (RLU) were reduced by 50% compared to RLU in virus control wells after subtraction of background activity in cell control wells.</p>
</div>
</div>

<div id="section14" class="section toc-section"><a id="s4n" name="s4n" class="link-target" title="Reactivity with Autoantigens"></a>
<h3>Reactivity with Autoantigens</h3>
<a id="article1.body1.sec4.sec14.p1" name="article1.body1.sec4.sec14.p1" class="link-target"></a><p>Autoreactivity was tested with an anti-dsDNA EIA kit, anti-Sm/RNP EIA kit, anti-Sm EIA kit, autoimmune EIA anti-SS-A/Ro Test, autoimmune EIA anti-SS-B/La test, Bio-Rad Kallestad ANA screen (all Bio-Rad) and QUANTA Lite® ACA IgG III (INOVA Diagnostics). Assays were performed on automated PhD System (Bio-Rad) and DSX™ System (Dynex Technologies).</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0038943"></div><div class="supplementary-material"><a name="pone.0038943.s001" id="pone.0038943.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Cloning of mimotopes into the mWasabi backbone.</strong> mWasabi and mimotope fragments were amplified using specific primers to introduce appropriate restriction sites to insert both sequences (boxed) into the pET-22b(+) vector. mWasabi was cloned first and the resulting vector pET-mWasabi.II served as an acceptor of different mimotopes. After transformation of <em>E.coli</em> with the resulting plasmids, mWasabi-mimotope fusion proteins were expressed under IPTG induction and purified from bacterial cell lysates by metal-affinity chromatography. pNCS-mWasabi was obtained from Allele Biotech.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s001">https://doi.org/10.1371/journal.pone.0038943.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s002" id="pone.0038943.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s002">Figure S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Binding analysis of 12 293F cell supernatants to mimotope fusion proteins, mWasabi-Tc.2 and HIV Env.</strong> Supernatants of 293F cells transfected with plasmids encoding cognate heavy and light immunoglobulin chains were collected 72 h post-transfection, diluted with blocking buffer (<a href="#s4">Material and Methods</a>) and incubated with mWasabi-Tc.2, mWasabi or HIV<sub>CN54</sub> gp120. The anti-V3 loop mAb HGN194 <a href="#pone.0038943-Corti2" class="ref-tip">[16]</a> and monkey RJa-9 serum were used as positive controls. Supernatant from non-transfected 293F cells (0 sup.) served as negative control. Each data point represents the mean ± s.e.m. (n  = 3).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s002">https://doi.org/10.1371/journal.pone.0038943.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s003" id="pone.0038943.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s003">Figure S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Binding of mAbs 33B2 and 33C6 to HIV Env of different clades.</strong> Plates were coated with envelope proteins and probed with different dilutions of mAbs 33B2 and 33C6. HIV Env proteins were derived from the following strains: clade A, UG37; B, BaL and IIIB; C, CN54 and 1157ip; D, UG21. SIVmne gp160 was used as negative control. MAb HGN194 served as positive and Fm-6 <a href="#pone.0038943-Sui1" class="ref-tip">[37]</a> as negative isotype controls, respectively. Each data point represents the mean ± s.e.m. (n  = 3).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s003">https://doi.org/10.1371/journal.pone.0038943.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s004" id="pone.0038943.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s004">Figure S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Inhibition of binding of mAbs to HIV<sub>CN54</sub> gp120 by consensus clade C peptides representing the V3 loop region.</strong> ELISA plates were coated with gp120 and exposed to mAbs mixed with V3 loop peptides (9258, 9259, 9260, 9261, 9262, and 9263) or control peptide representing the scrambled C-terminus of HIV gp120. Each data point represents the mean ± s.e.m. (n  = 3). (A) Amino acid sequences of linear consensus clade C peptide representing the V3 loop of gp120; (B) inhibition of binding of mAb 33B2; and (C) inhibition of binding of mAb 33C6.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s004">https://doi.org/10.1371/journal.pone.0038943.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIFF)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s005" id="pone.0038943.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s005">Figure S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Alignment of 33B2 and 33C6 VH with human (HU-IGHV5-51) and rhesus monkey (RM-IGHV-5-51) germline amino acid sequences and calculation of mutation frequency versus rhesus monkey germline.</strong> Red amino acids, divergence from the rhesus monkey germline; green, divergence from the human germline.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s005">https://doi.org/10.1371/journal.pone.0038943.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s006" id="pone.0038943.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s006">Figure S6. </a></h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI"><strong>Alignment of 33B2 and 33C6 VL with human (HU-IGLV1-50 and HU-IGLV1-47) and rhesus monkey (RM-IGLV1-50 and RM-IGLV1-47) germline amino acid sequences and calculation of mutation frequency versus rhesus monkey germline.</strong> Red amino acids, divergence from the rhesus monkey germline; green, divergence from the human germline.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s006">https://doi.org/10.1371/journal.pone.0038943.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s007" id="pone.0038943.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s007">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Treatment history and clinical parameters for cohort of RMs used for the study.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s007">https://doi.org/10.1371/journal.pone.0038943.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s008" id="pone.0038943.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s008">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>IC<sub>50</sub> neutralization titers of RM sera.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s008">https://doi.org/10.1371/journal.pone.0038943.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s009" id="pone.0038943.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s009">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Primers for amplification of rhesus monkey immunoglobulin V heavy and light chain genes.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s009">https://doi.org/10.1371/journal.pone.0038943.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s010" id="pone.0038943.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s010">Table S4. </a></h3><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Frequency of RM V gene usage.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s010">https://doi.org/10.1371/journal.pone.0038943.s010</a></p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0038943.s011" id="pone.0038943.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0038943.s011">Table S5. </a></h3><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1" class="link-target"></a><p class="preSiDOI">
            <strong>Primers for amplification of mWasabi and mimotopes.</strong>
          </p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0038943.s011">https://doi.org/10.1371/journal.pone.0038943.s011</a></p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank Dr. Julie Johnson-McGrath for editorial assistance, Dr. Patricia Polacino for assistance in the preparation of gp160, Dr. Wayne Marasco (Dana-Farber Cancer Institute) for providing the human IgG1 isotype control mAb Fm-6 and Maggie Wesberry, Sebastian Molnar and Brittani Barrows for technical assistance with neutralization assays.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: AMS MH RMR. Performed the experiments: AMS MMM MH SSE JDW HKV VS GH DCM VRP PHS. Analyzed the data: AMS MMM MH RMR. Contributed reagents/materials/analysis tools: AMS SSE MK SLH. Wrote the paper: AMS MMM RMR. Performed differential biopanning and isolated conformation-dependent mimotopes: MH. Performed virion-binding assays and molecular modeling: JDW. Performed neutralization assays: VP DMM. Tested for autoreactivity: PS.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0038943-Zielinski1" id="pone.0038943-Zielinski1" class="link-target"></a>Zielinski CE, Corti D, Mele F, Pinto D, Lanzavecchia A, et al.  (2011) Dissecting the human immunologic memory for pathogens. Immunological Reviews  240: 40–51.  <ul class="reflinks"><li><a href="#" data-author="Zielinski" data-cit="ZielinskiCECortiDMeleFPintoDLanzavecchiaA2011Dissecting%20the%20human%20immunologic%20memory%20for%20pathogens.Immunological%20Reviews2404051" data-title="Dissecting%20the%20human%20immunologic%20memory%20for%20pathogens." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dissecting+the+human+immunologic+memory+for+pathogens.+Zielinski+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0038943-Corti1" id="pone.0038943-Corti1" class="link-target"></a>Corti D, Sallusto F, Lanzavecchia A (2011) High throughput cellular screens to interrogate the human T and B cell repertoires. Curr Opin Immunol  23: 430–435.  <ul class="reflinks"><li><a href="#" data-author="Corti" data-cit="CortiDSallustoFLanzavecchiaA2011High%20throughput%20cellular%20screens%20to%20interrogate%20the%20human%20T%20and%20B%20cell%20repertoires.Curr%20Opin%20Immunol23430435" data-title="High%20throughput%20cellular%20screens%20to%20interrogate%20the%20human%20T%20and%20B%20cell%20repertoires." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=High+throughput+cellular+screens+to+interrogate+the+human+T+and+B+cell+repertoires.+Corti+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0038943-Kwong1" id="pone.0038943-Kwong1" class="link-target"></a>Kwong PD, Mascola JR, Nabel GJ (2009) Mining the B cell repertoire for broadly neutralizing monoclonal antibodies to HIV-1. Cell Host Microbe  6: 292–294.  <ul class="reflinks"><li><a href="#" data-author="Kwong" data-cit="KwongPDMascolaJRNabelGJ2009Mining%20the%20B%20cell%20repertoire%20for%20broadly%20neutralizing%20monoclonal%20antibodies%20to%20HIV-1.Cell%20Host%20Microbe6292294" data-title="Mining%20the%20B%20cell%20repertoire%20for%20broadly%20neutralizing%20monoclonal%20antibodies%20to%20HIV-1." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Mining+the+B+cell+repertoire+for+broadly+neutralizing+monoclonal+antibodies+to+HIV-1.+Kwong+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0038943-Reddy1" id="pone.0038943-Reddy1" class="link-target"></a>Reddy ST, Ge X, Miklos AE, Hughes RA, Kang SH, et al.  (2010) Monoclonal antibodies isolated without screening by analyzing the variable-gene repertoire of plasma cells. Nat Biotech  28: 965–969.  <ul class="reflinks"><li><a href="#" data-author="Reddy" data-cit="ReddySTGeXMiklosAEHughesRAKangSH2010Monoclonal%20antibodies%20isolated%20without%20screening%20by%20analyzing%20the%20variable-gene%20repertoire%20of%20plasma%20cells.Nat%20Biotech28965969" data-title="Monoclonal%20antibodies%20isolated%20without%20screening%20by%20analyzing%20the%20variable-gene%20repertoire%20of%20plasma%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Monoclonal+antibodies+isolated+without+screening+by+analyzing+the+variable-gene+repertoire+of+plasma+cells.+Reddy+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0038943-Robinson1" id="pone.0038943-Robinson1" class="link-target"></a>Robinson JE, Franco K, Elliott DH, Maher MJ, Reyna A, et al.  (2010) Quaternary epitope specificities of anti-HIV-1 neutralizing antibodies generated in rhesus macaques infected by the simian/human immunodeficiency virus SHIVSF162P4. J Virol  84: 3443–3453.  <ul class="reflinks"><li><a href="#" data-author="Robinson" data-cit="RobinsonJEFrancoKElliottDHMaherMJReynaA2010Quaternary%20epitope%20specificities%20of%20anti-HIV-1%20neutralizing%20antibodies%20generated%20in%20rhesus%20macaques%20infected%20by%20the%20simian%2Fhuman%20immunodeficiency%20virus%20SHIVSF162P4.J%20Virol8434433453" data-title="Quaternary%20epitope%20specificities%20of%20anti-HIV-1%20neutralizing%20antibodies%20generated%20in%20rhesus%20macaques%20infected%20by%20the%20simian%2Fhuman%20immunodeficiency%20virus%20SHIVSF162P4." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Quaternary+epitope+specificities+of+anti-HIV-1+neutralizing+antibodies+generated+in+rhesus+macaques+infected+by+the+simian%2Fhuman+immunodeficiency+virus+SHIVSF162P4.+Robinson+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0038943-Walker1" id="pone.0038943-Walker1" class="link-target"></a>Walker LM, Phogat SK, Chan-Hui PY, Wagner D, Phung P, et al.  (2009) Broad and potent neutralizing antibodies from an African donor reveal a new HIV-1 vaccine target. Science  326: 285–289.  <ul class="reflinks"><li><a href="#" data-author="Walker" data-cit="WalkerLMPhogatSKChan-HuiPYWagnerDPhungP2009Broad%20and%20potent%20neutralizing%20antibodies%20from%20an%20African%20donor%20reveal%20a%20new%20HIV-1%20vaccine%20target.Science326285289" data-title="Broad%20and%20potent%20neutralizing%20antibodies%20from%20an%20African%20donor%20reveal%20a%20new%20HIV-1%20vaccine%20target." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Broad+and+potent+neutralizing+antibodies+from+an+African+donor+reveal+a+new+HIV-1+vaccine+target.+Walker+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0038943-Wrammert1" id="pone.0038943-Wrammert1" class="link-target"></a>Wrammert J, Smith K, Miller J, Langley WA, Kokko K, et al.  (2008) Rapid cloning of high-affinity human monoclonal antibodies against influenza virus. Nature  453: 667–671.  <ul class="reflinks"><li><a href="#" data-author="Wrammert" data-cit="WrammertJSmithKMillerJLangleyWAKokkoK2008Rapid%20cloning%20of%20high-affinity%20human%20monoclonal%20antibodies%20against%20influenza%20virus.Nature453667671" data-title="Rapid%20cloning%20of%20high-affinity%20human%20monoclonal%20antibodies%20against%20influenza%20virus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Rapid+cloning+of+high-affinity+human+monoclonal+antibodies+against+influenza+virus.+Wrammert+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0038943-Beltramello1" id="pone.0038943-Beltramello1" class="link-target"></a>Beltramello M, Williams KL, Simmons CP, Macagno A, Simonelli L, et al.  (2010) The human immune response to Dengue virus is dominated by highly cross-reactive antibodies endowed with neutralizing and enhancing activity. Cell Host Microbe  8: 271–283.  <ul class="reflinks"><li><a href="#" data-author="Beltramello" data-cit="BeltramelloMWilliamsKLSimmonsCPMacagnoASimonelliL2010The%20human%20immune%20response%20to%20Dengue%20virus%20is%20dominated%20by%20highly%20cross-reactive%20antibodies%20endowed%20with%20neutralizing%20and%20enhancing%20activity.Cell%20Host%20Microbe8271283" data-title="The%20human%20immune%20response%20to%20Dengue%20virus%20is%20dominated%20by%20highly%20cross-reactive%20antibodies%20endowed%20with%20neutralizing%20and%20enhancing%20activity." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+human+immune+response+to+Dengue+virus+is+dominated+by+highly+cross-reactive+antibodies+endowed+with+neutralizing+and+enhancing+activity.+Beltramello+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0038943-Julius1" id="pone.0038943-Julius1" class="link-target"></a>Julius MH, Masuda T, Herzenberg LA (1972) Demonstration that antigen-binding cells are precursors of antibody-producing cells after purification with a fluorescence-activated cell sorter. Proc Natl Acad Sci  69: 1934–1938.  <ul class="reflinks"><li><a href="#" data-author="Julius" data-cit="JuliusMHMasudaTHerzenbergLA1972Demonstration%20that%20antigen-binding%20cells%20are%20precursors%20of%20antibody-producing%20cells%20after%20purification%20with%20a%20fluorescence-activated%20cell%20sorter.Proc%20Natl%20Acad%20Sci6919341938" data-title="Demonstration%20that%20antigen-binding%20cells%20are%20precursors%20of%20antibody-producing%20cells%20after%20purification%20with%20a%20fluorescence-activated%20cell%20sorter." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Demonstration+that+antigen-binding+cells+are+precursors+of+antibody-producing+cells+after+purification+with+a+fluorescence-activated+cell+sorter.+Julius+1972" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0038943-Weitkamp1" id="pone.0038943-Weitkamp1" class="link-target"></a>Weitkamp JH, Kallewaard N, Kusuhara K, Feigelstock D, Feng N, et al.  (2003) Generation of recombinant human monoclonal antibodies to rotavirus from single antigen-specific B cells selected with fluorescent virus-like particles. J Immunol Methods  275: 223–237.  <ul class="reflinks"><li><a href="#" data-author="Weitkamp" data-cit="WeitkampJHKallewaardNKusuharaKFeigelstockDFengN2003Generation%20of%20recombinant%20human%20monoclonal%20antibodies%20to%20rotavirus%20from%20single%20antigen-specific%20B%20cells%20selected%20with%20fluorescent%20virus-like%20particles.J%20Immunol%20Methods275223237" data-title="Generation%20of%20recombinant%20human%20monoclonal%20antibodies%20to%20rotavirus%20from%20single%20antigen-specific%20B%20cells%20selected%20with%20fluorescent%20virus-like%20particles." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Generation+of+recombinant+human+monoclonal+antibodies+to+rotavirus+from+single+antigen-specific+B+cells+selected+with+fluorescent+virus-like+particles.+Weitkamp+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0038943-Scheid1" id="pone.0038943-Scheid1" class="link-target"></a>Scheid JF, Mouquet H, Ueberheide B, Diskin R, Klein F, et al.  (2011) Sequence and structural convergence of broad and potent HIV antibodies that mimic CD4 binding. Science  333: 1633–1637.  <ul class="reflinks"><li><a href="#" data-author="Scheid" data-cit="ScheidJFMouquetHUeberheideBDiskinRKleinF2011Sequence%20and%20structural%20convergence%20of%20broad%20and%20potent%20HIV%20antibodies%20that%20mimic%20CD4%20binding.Science33316331637" data-title="Sequence%20and%20structural%20convergence%20of%20broad%20and%20potent%20HIV%20antibodies%20that%20mimic%20CD4%20binding." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence+and+structural+convergence+of+broad+and+potent+HIV+antibodies+that+mimic+CD4+binding.+Scheid+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0038943-Wu1" id="pone.0038943-Wu1" class="link-target"></a>Wu X, Yang ZY, Li Y, Hogerkorp CM, Schief WR, et al (2010) Rational design of envelope identifies broadly neutralizing human monoclonal antibodies to HIV-1. Science  329: 856–861.  <ul class="reflinks"><li><a href="#" data-author="Wu" data-cit="WuXYangZYLiYHogerkorpCMSchief%20WR%2C%20et%20al2010Rational%20design%20of%20envelope%20identifies%20broadly%20neutralizing%20human%20monoclonal%20antibodies%20to%20HIV-1.Science329856861" data-title="Rational%20design%20of%20envelope%20identifies%20broadly%20neutralizing%20human%20monoclonal%20antibodies%20to%20HIV-1." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Rational+design+of+envelope+identifies+broadly+neutralizing+human+monoclonal+antibodies+to+HIV-1.+Wu+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0038943-Jiang1" id="pone.0038943-Jiang1" class="link-target"></a>Jiang X, Burke V, Totrov M, Williams C, Cardozo T, et al.  (2010) Conserved structural elements in the V3 crown of HIV-1 gp120. Nat Struct Mol Biol  17: 955–961.  <ul class="reflinks"><li><a href="#" data-author="Jiang" data-cit="JiangXBurkeVTotrovMWilliamsCCardozoT2010Conserved%20structural%20elements%20in%20the%20V3%20crown%20of%20HIV-1%20gp120.Nat%20Struct%20Mol%20Biol17955961" data-title="Conserved%20structural%20elements%20in%20the%20V3%20crown%20of%20HIV-1%20gp120." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Conserved+structural+elements+in+the+V3+crown+of+HIV-1+gp120.+Jiang+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0038943-Gorny1" id="pone.0038943-Gorny1" class="link-target"></a>Gorny MK, Wang X-H, Williams C, Volsky B, Revesz K, et al.  (2009) Preferential use of the VH5–51 gene segment by the human immune response to code for antibodies against the V3 domain of HIV-1. Molecular Immunology  46: 917–926.  <ul class="reflinks"><li><a href="#" data-author="Gorny" data-cit="GornyMKWangX-HWilliamsCVolskyBReveszK2009Preferential%20use%20of%20the%20VH5%E2%80%9351%20gene%20segment%20by%20the%20human%20immune%20response%20to%20code%20for%20antibodies%20against%20the%20V3%20domain%20of%20HIV-1.Molecular%20Immunology46917926" data-title="Preferential%20use%20of%20the%20VH5%E2%80%9351%20gene%20segment%20by%20the%20human%20immune%20response%20to%20code%20for%20antibodies%20against%20the%20V3%20domain%20of%20HIV-1." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Preferential+use+of+the+VH5%E2%80%9351+gene+segment+by+the+human+immune+response+to+code+for+antibodies+against+the+V3+domain+of+HIV-1.+Gorny+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0038943-Gorny2" id="pone.0038943-Gorny2" class="link-target"></a>Gorny MK, Williams C, Volsky B, Revesz K, Cohen S, et al.  (2002) Human monoclonal antibodies specific for conformation-sensitive epitopes of V3 neutralize human immunodeficiency virus type 1 primary isolates from various clades. J Virol  76: 9035–9045.  <ul class="reflinks"><li><a href="#" data-author="Gorny" data-cit="GornyMKWilliamsCVolskyBReveszKCohenS2002Human%20monoclonal%20antibodies%20specific%20for%20conformation-sensitive%20epitopes%20of%20V3%20neutralize%20human%20immunodeficiency%20virus%20type%201%20primary%20isolates%20from%20various%20clades.J%20Virol7690359045" data-title="Human%20monoclonal%20antibodies%20specific%20for%20conformation-sensitive%20epitopes%20of%20V3%20neutralize%20human%20immunodeficiency%20virus%20type%201%20primary%20isolates%20from%20various%20clades." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+monoclonal+antibodies+specific+for+conformation-sensitive+epitopes+of+V3+neutralize+human+immunodeficiency+virus+type+1+primary+isolates+from+various+clades.+Gorny+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0038943-Corti2" id="pone.0038943-Corti2" class="link-target"></a>Corti D, Langedijk JP, Hinz A, Seaman MS, Vanzetta F, et al.  (2010) Analysis of memory B cell responses and isolation of novel monoclonal antibodies with neutralizing breadth from HIV-1-infected individuals. PLoS One  5: e8805.  <ul class="reflinks"><li><a href="#" data-author="Corti" data-cit="CortiDLangedijkJPHinzASeamanMSVanzettaF2010Analysis%20of%20memory%20B%20cell%20responses%20and%20isolation%20of%20novel%20monoclonal%20antibodies%20with%20neutralizing%20breadth%20from%20HIV-1-infected%20individuals.PLoS%20One5e8805" data-title="Analysis%20of%20memory%20B%20cell%20responses%20and%20isolation%20of%20novel%20monoclonal%20antibodies%20with%20neutralizing%20breadth%20from%20HIV-1-infected%20individuals." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+of+memory+B+cell+responses+and+isolation+of+novel+monoclonal+antibodies+with+neutralizing+breadth+from+HIV-1-infected+individuals.+Corti+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0038943-Hioe1" id="pone.0038943-Hioe1" class="link-target"></a>Hioe CE, Wrin T, Seaman MS, Yu X, Wood B, et al.  (2011) Anti-V3 monoclonal antibodies display broad neutralizing activities against multiple HIV-1 subtypes. PLoS ONE  5: e10254.  <ul class="reflinks"><li><a href="#" data-author="Hioe" data-cit="HioeCEWrinTSeamanMSYuXWoodB2011Anti-V3%20monoclonal%20antibodies%20display%20broad%20neutralizing%20activities%20against%20multiple%20HIV-1%20subtypes.PLoS%20ONE5e10254" data-title="Anti-V3%20monoclonal%20antibodies%20display%20broad%20neutralizing%20activities%20against%20multiple%20HIV-1%20subtypes." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Anti-V3+monoclonal+antibodies+display+broad+neutralizing+activities+against+multiple+HIV-1+subtypes.+Hioe+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0038943-Margolin1" id="pone.0038943-Margolin1" class="link-target"></a>Margolin DH, Saunders EH, Bronfin B, de Rosa N, Axthelm MK, et al.  (2006) Germinal center function in the spleen during simian HIV infection in rhesus monkeys. J Immunol  177: 1108–1119.  <ul class="reflinks"><li><a href="#" data-author="Margolin" data-cit="MargolinDHSaundersEHBronfinBde%20RosaNAxthelmMK2006Germinal%20center%20function%20in%20the%20spleen%20during%20simian%20HIV%20infection%20in%20rhesus%20monkeys.J%20Immunol17711081119" data-title="Germinal%20center%20function%20in%20the%20spleen%20during%20simian%20HIV%20infection%20in%20rhesus%20monkeys." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Germinal+center+function+in+the+spleen+during+simian+HIV+infection+in+rhesus+monkeys.+Margolin+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0038943-Siddappa1" id="pone.0038943-Siddappa1" class="link-target"></a>Siddappa NB, Watkins JD, Wassermann KJ, Song R, Wang W, et al.  (2010) R5 clade C SHIV strains with tier 1 or 2 neutralization sensitivity: tools to dissect env evolution and to develop AIDS vaccines in primate models. PLoS One  5: e11689.  <ul class="reflinks"><li><a href="#" data-author="Siddappa" data-cit="SiddappaNBWatkinsJDWassermannKJSongRWangW2010R5%20clade%20C%20SHIV%20strains%20with%20tier%201%20or%202%20neutralization%20sensitivity%3A%20tools%20to%20dissect%20env%20evolution%20and%20to%20develop%20AIDS%20vaccines%20in%20primate%20models.PLoS%20One5e11689" data-title="R5%20clade%20C%20SHIV%20strains%20with%20tier%201%20or%202%20neutralization%20sensitivity%3A%20tools%20to%20dissect%20env%20evolution%20and%20to%20develop%20AIDS%20vaccines%20in%20primate%20models." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=R5+clade+C+SHIV+strains+with+tier+1+or+2+neutralization+sensitivity%3A+tools+to+dissect+env+evolution+and+to+develop+AIDS+vaccines+in+primate+models.+Siddappa+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0038943-Parren1" id="pone.0038943-Parren1" class="link-target"></a>Parren PW, Fisicaro P, Labrijn AF, Binley JM, Yang WP, et al.  (1996) In vitro antigen challenge of human antibody libraries for vaccine evaluation: the human immunodeficiency virus type 1 envelope. J Virol  70: 9046–9050.  <ul class="reflinks"><li><a href="#" data-author="Parren" data-cit="ParrenPWFisicaroPLabrijnAFBinleyJMYangWP1996In%20vitro%20antigen%20challenge%20of%20human%20antibody%20libraries%20for%20vaccine%20evaluation%3A%20the%20human%20immunodeficiency%20virus%20type%201%20envelope.J%20Virol7090469050" data-title="In%20vitro%20antigen%20challenge%20of%20human%20antibody%20libraries%20for%20vaccine%20evaluation%3A%20the%20human%20immunodeficiency%20virus%20type%201%20envelope." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=In+vitro+antigen+challenge+of+human+antibody+libraries+for+vaccine+evaluation%3A+the+human+immunodeficiency+virus+type+1+envelope.+Parren+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0038943-Posner1" id="pone.0038943-Posner1" class="link-target"></a>Posner MR, Hideshima T, Cannon T, Mukherjee M, Mayer KH, et al.  (1991) An IgG human monoclonal antibody that reacts with HIV-1/GP120, inhibits virus binding to cells, and neutralizes infection. J Immunol  146: 4325–4332.  <ul class="reflinks"><li><a href="#" data-author="Posner" data-cit="PosnerMRHideshimaTCannonTMukherjeeMMayerKH1991An%20IgG%20human%20monoclonal%20antibody%20that%20reacts%20with%20HIV-1%2FGP120%2C%20inhibits%20virus%20binding%20to%20cells%2C%20and%20neutralizes%20infection.J%20Immunol14643254332" data-title="An%20IgG%20human%20monoclonal%20antibody%20that%20reacts%20with%20HIV-1%2FGP120%2C%20inhibits%20virus%20binding%20to%20cells%2C%20and%20neutralizes%20infection." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+IgG+human+monoclonal+antibody+that+reacts+with+HIV-1%2FGP120%2C+inhibits+virus+binding+to+cells%2C+and+neutralizes+infection.+Posner+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0038943-Roben1" id="pone.0038943-Roben1" class="link-target"></a>Roben P, Moore JP, Thali M, Sodroski J, Barbas CF, 3rd, et al (1994) Recognition properties of a panel of human recombinant Fab fragments to the CD4 binding site of gp120 that show differing abilities to neutralize human immunodeficiency virus type 1. J Virol  68: 4821–4828.  <ul class="reflinks"><li><a href="#" data-author="" data-cit="Roben%20P%2C%20Moore%20JP%2C%20Thali%20M%2C%20Sodroski%20J%2C%20Barbas%20CF%2C%203rd%2C%20et%20al1994Recognition%20properties%20of%20a%20panel%20of%20human%20recombinant%20Fab%20fragments%20to%20the%20CD4%20binding%20site%20of%20gp120%20that%20show%20differing%20abilities%20to%20neutralize%20human%20immunodeficiency%20virus%20type%201.J%20Virol6848214828" data-title="Recognition%20properties%20of%20a%20panel%20of%20human%20recombinant%20Fab%20fragments%20to%20the%20CD4%20binding%20site%20of%20gp120%20that%20show%20differing%20abilities%20to%20neutralize%20human%20immunodeficiency%20virus%20type%201." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Recognition+properties+of+a+panel+of+human+recombinant+Fab+fragments+to+the+CD4+binding+site+of+gp120+that+show+differing+abilities+to+neutralize+human+immunodeficiency+virus+type+1.++1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0038943-Zhang1" id="pone.0038943-Zhang1" class="link-target"></a>Zhang MY, Xiao X, Sidorov IA, Choudhry V, Cham F, et al.  (2004) Identification and characterization of a new cross-reactive human immunodeficiency virus type 1-neutralizing human monoclonal antibody. J Virol  78: 9233–9242.  <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangMYXiaoXSidorovIAChoudhryVChamF2004Identification%20and%20characterization%20of%20a%20new%20cross-reactive%20human%20immunodeficiency%20virus%20type%201-neutralizing%20human%20monoclonal%20antibody.J%20Virol7892339242" data-title="Identification%20and%20characterization%20of%20a%20new%20cross-reactive%20human%20immunodeficiency%20virus%20type%201-neutralizing%20human%20monoclonal%20antibody." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+characterization+of+a+new+cross-reactive+human+immunodeficiency+virus+type+1-neutralizing+human+monoclonal+antibody.+Zhang+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0038943-AyashRashkovsky1" id="pone.0038943-AyashRashkovsky1" class="link-target"></a>Ayash-Rashkovsky M, Chenine AL, Steele LN, Lee SJ, Song R, et al.  (2007) Coinfection with <em>Schistosoma mansoni</em> reactivates viremia in rhesus macaques with chronic simian-human immunodeficiency virus clade C infection. Infect Immun  75: 1751–1756.  <ul class="reflinks"><li><a href="#" data-author="Ayash-Rashkovsky" data-cit="Ayash-RashkovskyMChenineALSteeleLNLeeSJSongR2007Coinfection%20with%20Schistosoma%20mansoni%20reactivates%20viremia%20in%20rhesus%20macaques%20with%20chronic%20simian-human%20immunodeficiency%20virus%20clade%20C%20infection.Infect%20Immun7517511756" data-title="Coinfection%20with%20Schistosoma%20mansoni%20reactivates%20viremia%20in%20rhesus%20macaques%20with%20chronic%20simian-human%20immunodeficiency%20virus%20clade%20C%20infection." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Coinfection+with+Schistosoma+mansoni+reactivates+viremia+in+rhesus+macaques+with+chronic+simian-human+immunodeficiency+virus+clade+C+infection.+Ayash-Rashkovsky+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0038943-Humbert1" id="pone.0038943-Humbert1" class="link-target"></a>Humbert M, Rasmussen RA, Song R, Ong H, Sharma P, et al.  (2008) SHIV-1157i and passaged progeny viruses encoding R5 HIV-1 clade C env cause AIDS in rhesus monkeys. Retrovirology  5: 94.  <ul class="reflinks"><li><a href="#" data-author="Humbert" data-cit="HumbertMRasmussenRASongROngHSharmaP2008SHIV-1157i%20and%20passaged%20progeny%20viruses%20encoding%20R5%20HIV-1%20clade%20C%20env%20cause%20AIDS%20in%20rhesus%20monkeys.Retrovirology594" data-title="SHIV-1157i%20and%20passaged%20progeny%20viruses%20encoding%20R5%20HIV-1%20clade%20C%20env%20cause%20AIDS%20in%20rhesus%20monkeys." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=SHIV-1157i+and+passaged+progeny+viruses+encoding+R5+HIV-1+clade+C+env+cause+AIDS+in+rhesus+monkeys.+Humbert+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0038943-Rasmussen1" id="pone.0038943-Rasmussen1" class="link-target"></a>Rasmussen RA, Ong H, Song R, Chenine AL, Ayash-Rashkovsky M, et al.  (2007) Efficacy of a multigenic protein vaccine containing multimeric HIV gp160 against heterologous SHIV clade C challenges. AIDS  21: 1841–1848.  <ul class="reflinks"><li><a href="#" data-author="Rasmussen" data-cit="RasmussenRAOngHSongRChenineALAyash-RashkovskyM2007Efficacy%20of%20a%20multigenic%20protein%20vaccine%20containing%20multimeric%20HIV%20gp160%20against%20heterologous%20SHIV%20clade%20C%20challenges.AIDS2118411848" data-title="Efficacy%20of%20a%20multigenic%20protein%20vaccine%20containing%20multimeric%20HIV%20gp160%20against%20heterologous%20SHIV%20clade%20C%20challenges." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Efficacy+of+a+multigenic+protein+vaccine+containing+multimeric+HIV+gp160+against+heterologous+SHIV+clade+C+challenges.+Rasmussen+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0038943-Song1" id="pone.0038943-Song1" class="link-target"></a>Song RJ, Chenine AL, Rasmussen RA, Ruprecht CR, Mirshahidi S, et al.  (2006) Molecularly cloned SHIV-1157ipd3N4: a highly replication- competent, mucosally transmissible R5 simian-human immunodeficiency virus encoding HIV clade C Env. J Virol  80: 8729–8738.  <ul class="reflinks"><li><a href="#" data-author="Song" data-cit="SongRJChenineALRasmussenRARuprechtCRMirshahidiS2006Molecularly%20cloned%20SHIV-1157ipd3N4%3A%20a%20highly%20replication-%20competent%2C%20mucosally%20transmissible%20R5%20simian-human%20immunodeficiency%20virus%20encoding%20HIV%20clade%20C%20Env.J%20Virol8087298738" data-title="Molecularly%20cloned%20SHIV-1157ipd3N4%3A%20a%20highly%20replication-%20competent%2C%20mucosally%20transmissible%20R5%20simian-human%20immunodeficiency%20virus%20encoding%20HIV%20clade%20C%20Env." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecularly+cloned+SHIV-1157ipd3N4%3A+a+highly+replication-+competent%2C+mucosally+transmissible+R5+simian-human+immunodeficiency+virus+encoding+HIV+clade+C+Env.+Song+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0038943-Klaniecki1" id="pone.0038943-Klaniecki1" class="link-target"></a>Klaniecki J, Dykers T, Travis B, Schmitt R, Wain M, et al.  (1991) Cross-neutralizing antibodies in rabbits immunized with HIV-1 gp160 purified from simian cells infected with a recombinant vaccinia virus. AIDS Res Hum Retroviruses  7: 791–798.  <ul class="reflinks"><li><a href="#" data-author="Klaniecki" data-cit="KlanieckiJDykersTTravisBSchmittRWainM1991Cross-neutralizing%20antibodies%20in%20rabbits%20immunized%20with%20HIV-1%20gp160%20purified%20from%20simian%20cells%20infected%20with%20a%20recombinant%20vaccinia%20virus.AIDS%20Res%20Hum%20Retroviruses7791798" data-title="Cross-neutralizing%20antibodies%20in%20rabbits%20immunized%20with%20HIV-1%20gp160%20purified%20from%20simian%20cells%20infected%20with%20a%20recombinant%20vaccinia%20virus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cross-neutralizing+antibodies+in+rabbits+immunized+with+HIV-1+gp160+purified+from+simian+cells+infected+with+a+recombinant+vaccinia+virus.+Klaniecki+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0038943-Humbert2" id="pone.0038943-Humbert2" class="link-target"></a>Humbert M, Rasmussen RA, Ong H, Kaiser FM, Hu SL, et al.  (2008) Inducing cross-clade neutralizing antibodies against HIV-1 by immunofocusing. PLoS One  3: e3937.  <ul class="reflinks"><li><a href="#" data-author="Humbert" data-cit="HumbertMRasmussenRAOngHKaiserFMHuSL2008Inducing%20cross-clade%20neutralizing%20antibodies%20against%20HIV-1%20by%20immunofocusing.PLoS%20One3e3937" data-title="Inducing%20cross-clade%20neutralizing%20antibodies%20against%20HIV-1%20by%20immunofocusing." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Inducing+cross-clade+neutralizing+antibodies+against+HIV-1+by+immunofocusing.+Humbert+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0038943-Harris1" id="pone.0038943-Harris1" class="link-target"></a>Harris LJ, Larson SB, Hasel KW, McPherson A (1997) Refined structure of an intact IgG2a monoclonal antibody. Biochemistry  36: 1581–1597.  <ul class="reflinks"><li><a href="#" data-author="Harris" data-cit="HarrisLJLarsonSBHaselKWMcPhersonA1997Refined%20structure%20of%20an%20intact%20IgG2a%20monoclonal%20antibody.Biochemistry3615811597" data-title="Refined%20structure%20of%20an%20intact%20IgG2a%20monoclonal%20antibody." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Refined+structure+of+an+intact+IgG2a+monoclonal+antibody.+Harris+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0038943-Kiefer1" id="pone.0038943-Kiefer1" class="link-target"></a>Kiefer F, Arnold K, Kunzli M, Bordoli L, Schwede T (2009) The SWISS-MODEL Repository and associated resources. Nucleic Acids Res  37: D387–392.  <ul class="reflinks"><li><a href="#" data-author="Kiefer" data-cit="KieferFArnoldKKunzliMBordoliLSchwedeT2009The%20SWISS-MODEL%20Repository%20and%20associated%20resources.Nucleic%20Acids%20Res37D387392" data-title="The%20SWISS-MODEL%20Repository%20and%20associated%20resources." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+SWISS-MODEL+Repository+and+associated+resources.+Kiefer+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0038943-Tiller1" id="pone.0038943-Tiller1" class="link-target"></a>Tiller T, Meffre E, Yurasov S, Tsuiji M, Nussenzweig MC, et al.  (2008) Efficient generation of monoclonal antibodies from single human B cells by single cell RT-PCR and expression vector cloning. J Immunol Methods  329: 112–124.  <ul class="reflinks"><li><a href="#" data-author="Tiller" data-cit="TillerTMeffreEYurasovSTsuijiMNussenzweigMC2008Efficient%20generation%20of%20monoclonal%20antibodies%20from%20single%20human%20B%20cells%20by%20single%20cell%20RT-PCR%20and%20expression%20vector%20cloning.J%20Immunol%20Methods329112124" data-title="Efficient%20generation%20of%20monoclonal%20antibodies%20from%20single%20human%20B%20cells%20by%20single%20cell%20RT-PCR%20and%20expression%20vector%20cloning." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Efficient+generation+of+monoclonal+antibodies+from+single+human+B+cells+by+single+cell+RT-PCR+and+expression+vector+cloning.+Tiller+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0038943-Montefiori1" id="pone.0038943-Montefiori1" class="link-target"></a>Montefiori DC (2005) Evaluating neutralizing antibodies against HIV, SIV, and SHIV in luciferase reporter gene assays. Curr Protoc Immunol Chapter 12: Unit 12 11.   <ul class="find-nolinks"></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0038943-Watkins1" id="pone.0038943-Watkins1" class="link-target"></a>Watkins JD, Siddappa NB, Lakhashe SK, Humbert M, Sholukh A, et al.  (2011) An anti-HIV-1 V3 loop antibody fully protects cross-clade and elicits T-cell immunity in macaques mucosally challenged with an R5 clade C SHIV. PLoS One  6: e18207.  <ul class="reflinks"><li><a href="#" data-author="Watkins" data-cit="WatkinsJDSiddappaNBLakhasheSKHumbertMSholukhA2011An%20anti-HIV-1%20V3%20loop%20antibody%20fully%20protects%20cross-clade%20and%20elicits%20T-cell%20immunity%20in%20macaques%20mucosally%20challenged%20with%20an%20R5%20clade%20C%20SHIV.PLoS%20One6e18207" data-title="An%20anti-HIV-1%20V3%20loop%20antibody%20fully%20protects%20cross-clade%20and%20elicits%20T-cell%20immunity%20in%20macaques%20mucosally%20challenged%20with%20an%20R5%20clade%20C%20SHIV." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+anti-HIV-1+V3+loop+antibody+fully+protects+cross-clade+and+elicits+T-cell+immunity+in+macaques+mucosally+challenged+with+an+R5+clade+C+SHIV.+Watkins+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0038943-Folks1" id="pone.0038943-Folks1" class="link-target"></a>Folks T, Benn S, Rabson A, Theodore T, Hoggan MD, et al.  (1985) Characterization of a continuous T-cell line susceptible to the cytopathic effects of the acquired immunodeficiency syndrome (AIDS)-associated retrovirus. Proc Natl Acad Sci  82: 4539–4543.  <ul class="reflinks"><li><a href="#" data-author="Folks" data-cit="FolksTBennSRabsonATheodoreTHogganMD1985Characterization%20of%20a%20continuous%20T-cell%20line%20susceptible%20to%20the%20cytopathic%20effects%20of%20the%20acquired%20immunodeficiency%20syndrome%20%28AIDS%29-associated%20retrovirus.Proc%20Natl%20Acad%20Sci8245394543" data-title="Characterization%20of%20a%20continuous%20T-cell%20line%20susceptible%20to%20the%20cytopathic%20effects%20of%20the%20acquired%20immunodeficiency%20syndrome%20%28AIDS%29-associated%20retrovirus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+a+continuous+T-cell+line+susceptible+to+the+cytopathic+effects+of+the+acquired+immunodeficiency+syndrome+%28AIDS%29-associated+retrovirus.+Folks+1985" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0038943-Edmonds1" id="pone.0038943-Edmonds1" class="link-target"></a>Edmonds TG, Ding H, Yuan X, Wei Q, Smith KS, et al.  (2010) Replication competent molecular clones of HIV-1 expressing Renilla luciferase facilitate the analysis of antibody inhibition in PBMC. Virology  408: 1–13.  <ul class="reflinks"><li><a href="#" data-author="Edmonds" data-cit="EdmondsTGDingHYuanXWeiQSmithKS2010Replication%20competent%20molecular%20clones%20of%20HIV-1%20expressing%20Renilla%20luciferase%20facilitate%20the%20analysis%20of%20antibody%20inhibition%20in%20PBMC.Virology408113" data-title="Replication%20competent%20molecular%20clones%20of%20HIV-1%20expressing%20Renilla%20luciferase%20facilitate%20the%20analysis%20of%20antibody%20inhibition%20in%20PBMC." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Replication+competent+molecular+clones+of+HIV-1+expressing+Renilla+luciferase+facilitate+the+analysis+of+antibody+inhibition+in+PBMC.+Edmonds+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0038943-Sui1" id="pone.0038943-Sui1" class="link-target"></a>Sui J, Hwang WC, Perez S, Wei G, Aird D, et al.  (2009) Structural and functional bases for broad-spectrum neutralization of avian and human influenza A viruses. Nat Struct Mol Biol  16: 265–273.  <ul class="reflinks"><li><a href="#" data-author="Sui" data-cit="SuiJHwangWCPerezSWeiGAirdD2009Structural%20and%20functional%20bases%20for%20broad-spectrum%20neutralization%20of%20avian%20and%20human%20influenza%20A%20viruses.Nat%20Struct%20Mol%20Biol16265273" data-title="Structural%20and%20functional%20bases%20for%20broad-spectrum%20neutralization%20of%20avian%20and%20human%20influenza%20A%20viruses." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Structural+and+functional+bases+for+broad-spectrum+neutralization+of+avian+and+human+influenza+A+viruses.+Sui+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

