



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2>
<div id="section1" class="section toc-section"><a id="" name="" class="link-target" title="Background"></a>
<h3>Background</h3>
<a id="article1.front1.article-meta1.abstract1.sec1.p1" name="article1.front1.article-meta1.abstract1.sec1.p1" class="link-target"></a><p>To investigate the functions of Dicer and microRNAs in neural stem (NS) cell self-renewal and neurogenesis, we established neural stem cell lines from the embryonic mouse Dicer-null cerebral cortex, producing neural stem cell lines that lacked all microRNAs.</p>
</div>

<div id="section2" class="section toc-section"><a id="" name="" class="link-target" title="Principal Findings"></a>
<h3>Principal Findings</h3>
<a id="article1.front1.article-meta1.abstract1.sec2.p1" name="article1.front1.article-meta1.abstract1.sec2.p1" class="link-target"></a><p>Dicer-null NS cells underwent normal self-renewal and could be maintained <em>in vitro</em> indefinitely, but had subtly altered cell cycle kinetics and abnormal heterochromatin organisation. In the absence of all microRNAs, Dicer-null NS cells were incapable of generating either glial or neuronal progeny and exhibited a marked dependency on exogenous EGF for survival. Dicer-null NS cells assumed complex differences in mRNA and protein expression under self-renewing conditions, upregulating transcripts indicative of self-renewing NS cells and expressing genes characteristic of differentiating neurons and glia. Underlining the growth-factor dependency of Dicer-null NS cells, many regulators of apoptosis were enriched in expression in these cells. Dicer-null NS cells initiate some of the same gene expression changes as wild-type cells under astrocyte differentiating conditions, but also show aberrant expression of large sets of genes and ultimately fail to complete the differentiation programme. Acute replacement of Dicer restored their ability to differentiate to both neurons and glia.</p>
</div>

<div id="section3" class="section toc-section"><a id="" name="" class="link-target" title="Conclusions"></a>
<h3>Conclusions</h3>
<a id="article1.front1.article-meta1.abstract1.sec3.p1" name="article1.front1.article-meta1.abstract1.sec3.p1" class="link-target"></a><p>The block in differentiation due to loss of Dicer and microRNAs is reversible and the significantly altered phenotype of Dicer-null NS cells does not constitute a permanent transformation. We conclude that Dicer and microRNAs function in this system to maintain the neural stem cell phenotype and to facilitate the completion of differentiation.</p>
</div>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Andersson T, Rahman S, Sansom SN, Alsiö JM, Kaneda M, Smith J, et al.  (2010) Reversible Block of Mouse Neural Stem Cell Differentiation in the Absence of Dicer and MicroRNAs. PLoS ONE 5(10):
           e13453.
        
        https://doi.org/10.1371/journal.pone.0013453</p><p><strong>Editor: </strong>Thomas A. Reh, University of Washington, United States of America</p><p><strong>Received: </strong>April 20, 2010; <strong>Accepted: </strong>September 21, 2010; <strong>Published: </strong> October 18, 2010</p><p><strong>Copyright: </strong> © 2010 Andersson et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This work was supported by grants from the Wellcome Trust (FJL) and the Wenner-Gren Foundation (TA) and by core support to the Gurdon Institute from Cancer Research UK and the Wellcome Trust (<a href="http://www.wellcome.ac.uk">www.wellcome.ac.uk</a>). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Neural stem (NS) cells in the developing vertebrate embryo are multipotent, self-renewing cells that give rise to specific types of neurons in a fixed temporal order. MicroRNAs have been proposed as candidates for regulating many aspects of neural stem cell biology, based on the functions of microRNAs in the control of developmental timing and cell fate specification in a number of biological systems, including stem cells (for reviews, see <a href="#pone.0013453-Bartel1" class="ref-tip">[1]</a>, <a href="#pone.0013453-Ambros1" class="ref-tip">[2]</a>). In the nervous system, microRNAs have been proposed to regulate stem cell self-renewal, neurogenesis, cell fate determination and neuronal differentiation <a href="#pone.0013453-Kosik1" class="ref-tip">[3]</a>, <a href="#pone.0013453-Cheng1" class="ref-tip">[4]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Many microRNAs are expressed in the developing forebrain, including microRNAs expressed in NS cells or during neuronal differentiation <em>in vivo</em> and <em>in vitro</em> <a href="#pone.0013453-LagosQuintana1" class="ref-tip">[5]</a>, <a href="#pone.0013453-Houbaviy1" class="ref-tip">[6]</a>, <a href="#pone.0013453-Krichevsky1" class="ref-tip">[7]</a>, <a href="#pone.0013453-Miska1" class="ref-tip">[8]</a>, <a href="#pone.0013453-Sempere1" class="ref-tip">[9]</a>. Marked changes in microRNA expression occur during the induction of neurogenesis from embryonic stem (ES) cells and gain and loss of functions of single microRNAs can alter the degree of neurogenesis from ES cells <a href="#pone.0013453-Krichevsky2" class="ref-tip">[10]</a>. Functional studies of neurogenesis in the olfactory epithelium have found that the major role of microRNAs and specifically the mir-200 family is to regulate neuronal differentiation in olfactory neurons, rather than regulate neurogenesis <a href="#pone.0013453-Choi1" class="ref-tip">[11]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>One approach to studying the possible functions of microRNAs has been to genetically ablate key proteins required for the biogenesis of microRNAs, most commonly the RNase III enzyme Dicer. Loss of Dicer function in embryonic stem (ES) cells revealed a requirement for Dicer and microRNAs in ES cell differentiation, but not ES cell maintenance and self-renewal <a href="#pone.0013453-Murchison1" class="ref-tip">[12]</a>, <a href="#pone.0013453-Kanellopoulou1" class="ref-tip">[13]</a>. Null mutations in the Drosha-associated RNA binding protein DGCR8 gave a slightly different phenotype, in that ES cells failed to differentiate <em>in vitro</em> and also failed to down-regulate expression of key regulators of ES cell self-renewal <a href="#pone.0013453-Wang1" class="ref-tip">[14]</a>. Studies of loss of Dicer function in the olfactory epithelium <a href="#pone.0013453-Choi1" class="ref-tip">[11]</a> and the developing cerebral cortex <a href="#pone.0013453-DePietriTonelli1" class="ref-tip">[15]</a>, <a href="#pone.0013453-KawaseKoga1" class="ref-tip">[16]</a> have found that Dicer is not required for neurogenesis. Similarly, neurogenesis still takes place in the retina following Dicer ablation <a href="#pone.0013453-Damiani1" class="ref-tip">[17]</a>, <a href="#pone.0013453-Georgi1" class="ref-tip">[18]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>However, <em>in vivo</em> studies of microRNA function by removal of Dicer share a common problem that deletion of Dicer does not immediately result in microRNA loss of function. MicroRNAs have been found to have relatively long half-lives in several different cell types and to persist for many cell cycles after Dicer loss of function <a href="#pone.0013453-Cobb1" class="ref-tip">[19]</a>. In a developing neural stem cell-specific Dicer mutant, microRNAs persisted for several days after Dicer ablation <a href="#pone.0013453-KawaseKoga1" class="ref-tip">[16]</a>, and we have also found that following Dicer deletion in many cases microRNAs can persist at over 40% of wild-type levels during the period of neurogenesis.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5" class="link-target"></a><p>To study the absolute requirement for Dicer-dependent microRNAs and Dicer in neurogenesis and neural stem cell self-renewal, we generated NS cell lines lacking Dicer from the embryonic, Dicer-null cerebral cortex. It has recently been reported that Dicer-null NS cells can be generated from the Dicer-null cortex at low efficiency, and are impaired in their ability to self-renew and generate differentiated progeny <a href="#pone.0013453-KawaseKoga2" class="ref-tip">[20]</a>. We find that Dicer is not required for neural stem cell derivation. We cultured Dicer-null NS cells for several weeks to allow microRNA turnover to remove persisting microRNAs, generating Dicer-null NS cells that lack detectable microRNAs. We found that Dicer and microRNAs are dispensable for self-renewal, but that microRNAs are essential for the neural stem cell differentiation to either neurons or glia. This contrasts with previous <em>in vivo</em> findings <a href="#pone.0013453-Choi1" class="ref-tip">[11]</a>, <a href="#pone.0013453-DePietriTonelli1" class="ref-tip">[15]</a>, but is consistent with studies of embryonic stem cells <a href="#pone.0013453-Murchison1" class="ref-tip">[12]</a>, <a href="#pone.0013453-Kanellopoulou1" class="ref-tip">[13]</a>, <a href="#pone.0013453-Wang1" class="ref-tip">[14]</a>. This difference is likely to be due to the ability to allow microRNAs to reduce to undetectable levels in Dicer-null NS cells in culture, whereas the long half-lives of microRNAs results in persistence of many microRNAs during the period of neurogenesis following Dicer deletion <em>in vivo.</em></p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6" class="link-target"></a><p>Dicer-null NS cells have an abnormal phenotype, combining increased expression of key regulators of stem cell maintenance with inappropriate expression of differentiation genes. Under neuronal differentiation conditions Dicer-null NS cells exhibit growth factor withdrawal-induced apoptosis. In three different glial differentiation conditions, Dicer-null NS cells do not undergo apoptosis, instead initiating but failing to complete the glial differentiation programme. Strikingly, the ability of Dicer-null NS cells to generate both neurons and glia is restored by the acute reintroduction of Dicer, demonstrating that these cells are not irreversibly transformed despite the absence of Dicer and microRNAs. We conclude that removal of Dicer and microRNAs substantially alters the phenotype of NS cells to render them incapable of full differentiation, but that the ability to differentiate is restored by the acute reintroduction of Dicer, indicating that the altered phenotype of Dicer-null NS cells is not a permanent transformation.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Generation of cortex-specific Dicer-null mice"></a>
<h3>Generation of cortex-specific Dicer-null mice</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>Nestin-Cre transgenic mice <a href="#pone.0013453-Tronche1" class="ref-tip">[21]</a> were crossed to Dicer<sup>fl/fl</sup> mice <a href="#pone.0013453-Yi1" class="ref-tip">[22]</a> to produce Nes-Cre; Dicer<sup>fl/+</sup> founder males. Nes-Cre; Dicer<sup>fl/+</sup> males were crossed with Dicer<sup>fl/fl</sup> female mice to generate Dicer-null embryos. All procedures were carried out according to UK Home Office regulations, under UK Home Office Project Licence PPL/182193, and approved by the Gurdon Institute animal use committee. Mice were genotyped for the Nes-Cre transgene and the Dicer alleles by PCR <a href="#pone.0013453-Tronche1" class="ref-tip">[21]</a>, <a href="#pone.0013453-Yi1" class="ref-tip">[22]</a>.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Neural stem cell line derivation, culture and characterization"></a>
<h3>Neural stem cell line derivation, culture and characterization</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Polyclonal neural stem cell lines were established from the E12.5 cortex of Dicer-null and wild-type littermate mice and maintained according to published methods with minor variations <a href="#pone.0013453-Conti1" class="ref-tip">[23]</a>. Briefly, E12.5 cortices were dissociated to single cells and placed in non-adherent culture conditions to promote neurosphere formation in the presence of 10 µg/ml recombinant EGF and 10 µg/ml recombinant FGF2 (Peprotech). Neurospheres were harvested seven days later and transferred to laminin-coated culture dishes to promote adherence and cellular outgrowth. Karyotyping was carried out by DAPI staining of colcemid-treated cells, with at least five clear chromosome spreads counted for each line analysed. Neuronal differentiation of NS cells was initiated by sequential withdrawal of EGF and FGF2, essentially as described <a href="#pone.0013453-Conti1" class="ref-tip">[23]</a>. Glial differentiation was induced either by FGF2 withdrawal or addition of either 2% foetal calf serum (FCS; Sigma-Aldrich) or 10 µg/ml BMP4 (Peprotech) to the culture medium.</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>For immunofluorescent staining, cultures were fixed with 4% paraformaldehyde in PBS or in ice-cold methanol. Primary antibodies used in this study were: Nestin (mouse; BD Biosciences), Sox2 (goat; Santa Cruz Biotechnology), Tuj1 (mouse; Covance), H3K9Me3 (rabbit; Upstate), GFAP (rabbit; Sigma-Aldrich), Hmga2 (rabbit; kind gift of Dr. M. Narita), Cyclin D1 (mouse; Santa Cruz Biotechnology). Primary antibodies were detected with species-specific Alexa-488 or Alexa-546 conjugated secondary antibodies. Immunolabelled cells were visualised by confocal or epifluorescence microscopy.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Microarray analysis of microRNA and mRNA expression"></a>
<h3>Microarray analysis of microRNA and mRNA expression</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Custom microarrays representing 171 microRNAs in the microRNA registry, 3 snRNAs and 4 tRNAs were constructed using 5′-amine-linked antisense oligonucleotides printed on CodeLink slides. Short RNAs were extracted from embryonic cortex and cells in culture (mirVana, Ambion), visualised on a BioAnalyzer (Agilent) and quantified with a Nanodrop spectrophotometer. 250 ng of short RNAs were labelled (Array 900RT, Genisphere) and hybridised overnight before detection according to the manufacturer's instructions. For messenger RNA expression profiling, total RNA isolated from neural stem cell cultures was used to generate cDNA by reverse transcription and PCR amplification using the SMART system (BD) <a href="#pone.0013453-Matz1" class="ref-tip">[24]</a>. Amplified cDNA was labelled by the incorporation of either Alexa 555 or 647-dCTP (Invitrogen), as previously described <a href="#pone.0013453-Livesey1" class="ref-tip">[25]</a>, <a href="#pone.0013453-Livesey2" class="ref-tip">[26]</a>. The microarrays used for this study were arrays of approximately 22,500 65-mer oligonucleotides (Compugen/Sigma-Genosys) printed in-house on CodeLink slides (Amersham) or arrays of the MEEBO oligonucleotide library (Invitrogen). Array slides were scanned in an Axon GenePix 4000B scanner (Axon Instruments), and data from each array extracted using Genepix 4.0 (Axon Instruments). Microarray data were archived and analysed locally in Acuity (Axon Instruments) and in Bioconductor.</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Microarray data analysis and bioinformatics"></a>
<h3>Microarray data analysis and bioinformatics</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>MicroRNA levels in Dicer-null NS cells were normalised to those in wild-type NS cells using the set of control small RNAs on the array (tRNAs, snRNAs). MicroRNA levels were expressed relative to wild-type levels, correcting for local and systemic background signals. Gene expression differences between wild-type and Dicer-null self-renewing NS cells were identified by comparing two wild type and two Dicer-null NSC lines on six dye-swapped MEEBO microarrays: four hybridisations directly compared expression between Dicer-null and wild-type lines, and two hybridisations directly compared the two wild-type lines. Expression analysis was carried out using the Limma package<a href="#pone.0013453-Smyth1" class="ref-tip">[27]</a>. Low intensity features were identified and weighted appropriately, arrays were lowess normalised and then normalised between arrays. Control features and features not representing genes were excluded before the identification of differentially expressed genes using the eBayes algorithm. An adjusted p-value cut-off of 0.05 was used to identify significant genes. Gene ontology analysis of the gene expression changes in Dicer-null NS cells was performed using GOToolBox (<a href="http://crfb.univ-mrs.fr/GOToolBox/home.php">http://crfb.univ-mrs.fr/GOToolBox/home.php</a>) <a href="#pone.0013453-Martin1" class="ref-tip">[28]</a>. GO categories (p&lt;0.01) significantly enriched by comparison with the whole probeset were identified using a Benjamini &amp; Hochberg corrected hypergeometric test.</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>To compare gene expression changes between wild-type and Dicer-null NS cells during glial differentiation, RNA was harvested from wild-type and Dicer-null NS lines 24 hours after the addition of BMP4 or PBS. A set of six array hybridisations was carried out: two hybridisations comparing gene expression between wild-type NS cells with and without BMP4 treatment; two hybridisations comparing gene expression between Dicer-null NS cells with and without BMP4 treatment; and two hybridisations directly comparing BMP-treated wild-type and Dicer-null NS cells. Array data were lowess normalised (in Acuity) and the entire set of six arrays analysed as a group. Data were filtered to remove low-intensity features and features absent on more than two arrays. To identify robust changes in gene expression occurring in at least one of the cell lines, only genes showing at least a 50% change in gene expression in two or more arrays were retained for further analysis. The content of this final set of genes was explored by hierarchical and agglomerative clustering in the Acuity system to identify the major classes of gene expression changes, as reported above. To prospectively identify differences in the response to BMP4 treatment between wild-type and Dicer-null NS cells, we used the significance analysis of microarrays (SAM) algorithm <a href="#pone.0013453-Tusher1" class="ref-tip">[29]</a> to analyse the four microarrays comparing BMP-treated and control Dicer-null and wild-type NS cells. Data were normalised and filtered as before and significant changes in gene expression discovered by performing a two-class SAM, using a false discovery rate (FDR) cut-off of 0.1. Data passing the 0.1 FDR were hierarchically clustered using MEV to aid visualisation <a href="#pone.0013453-Saeed1" class="ref-tip">[30]</a>.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Retroviral expression of Dicer"></a>
<h3>Retroviral expression of Dicer</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Human full-length Dicer (hDicer, kindly provided by Dr P. Provost, Laval) was subcloned into the MMLV-based pFB murine replication-incompetent retrovirus plasmid vector (Stratagene). The pFB-hDicer construct was introduced into 293T cells together with the pCL-Eco packaging plasmid by calcium phosphate transfection. Virus-containing supernatants were collected 36 and 48 hours after transfection, centrifuged and directly added to NS cultures. Cells were washed and the culture media replaced 24 or 48 hours after application of virus to initiate neuronal or glial differentiation. Viral infection and Dicer expression in infected cells was confirmed by genomic PCR and RT-PCR for hDicer using two different sets of primers specific to human Dicer.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="Neural stem cell lines can be established from the Dicer-null brain and self-renew normally"></a>
<h3>Neural stem cell lines can be established from the Dicer-null brain and self-renew normally</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>Microarray analysis of microRNA expression at late stages of neurogenesis in the cerebral cortex of a nervous system-specific Dicer mutant (Nes-Cre/Dicer; embryonic day 15, E15) found that microRNAs were still present in the Dicer mutant cortex at a significant level, typically around 35–40% of wild-type littermate levels (<a href="#pone-0013453-g001">Fig. 1A</a>). In contrast, microRNAs were effectively undetectable in the Dicer-null cortex at birth (<a href="#pone-0013453-g001">Fig. 1A</a>). Therefore, it was not possible to analyse the effects of the complete absence of microRNAs on neural stem cell self-renewal and neurogenesis during development <em>in vivo</em>. To investigate whether microRNAs are required for neural stem and progenitor cell self-renewal, proliferation and neurogenesis, we derived neural stem (NS) cell lines from the cerebral cortex of Dicer mutants and control littermates at E12.</p>
<a class="link-target" id="pone-0013453-g001" name="pone-0013453-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0013453.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0013453.g001" data-doi="info:doi/10.1371/journal.pone.0013453" data-uri="info:doi/10.1371/journal.pone.0013453.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0013453.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Dicer-null cortical neural stem cells can be established from the embryonic cerebral cortex and self-renewal appropriately.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) At E15, several days after Dicer ablation using the neural progenitor Nes-Cre deleter line, Dicer-null cortices still have about 40% of wild-type microRNA levels (average of the relative expression of all microarray–detected microRNAs), whereas microRNAs are almost undetectable in the cortex at birth. MicroRNA levels were measured as described in the methods and <a href="#pone.0013453.s001">Table S1</a>. (B) PCR genotyping of monolayer cultures of cortex-derived NS cell lines demonstrates that lines derived from Nes-Cre;Dicer<sup>fl/fl</sup> E12 cortex are Cre+ and contain only the recombined Dicer allele. (C) Dicer-null NS cells lack expression of microRNAs, compared to wild-type cells, although expression of other short RNAs, such as tRNAs, is normal. Expression for a subset of typical microRNAs and two tRNAs is shown relative to expression in control NS cells, and is the average of six replicate features on the array. (D–G) Neural stem cell lines established from wild-type and Dicer-null E12.5 cortex in monolayer culture both express the NS-cell specific intermediate filament protein Nestin (D, E) and the NS cell transcription factor Sox2 (F, G). While Nestin is clearly expressed, the morphology of Dicer-null NS cells is abnormal and lacks the long radial fibres found in wild-type NS cells. Scale bars, 25 µm. (H, I) Dicer-null and control NS cells both form neurospheres in non-adherent culture, in which the majority of cells express Nestin. Note that cells at the periphery of neurospheres generated by control cells begin to differentiate to astrocytes, as indicated by GFAP expression, whereas no GFAP expression is observed in Dicer-null neurospheres. Scale bar, 25 µm. Nuclei in all images visualised with DAPI.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0013453.g001">
              https://doi.org/10.1371/journal.pone.0013453.g001</a></p></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>NS cell lines were established from Dicer-null cortices at the same efficiency as homozygous floxed Dicer allele control littermates (<a href="#pone-0013453-g001">Fig. 1</a>) and confirmed as being Dicer-null by genotyping (<a href="#pone-0013453-g001">Fig. 1B</a>). Cells of both genotypes were maintained for several weeks and the absence of microRNAs from the Dicer-null cells confirmed by array analysis (<a href="#pone-0013453-g001">Fig. 1C</a>; <a href="#pone.0013453.s001">Table S1</a>). Characterisation of the effects of Dicer and microRNA loss on neural stem cell biology focussed on two independent Dicer-null NS lines and two independent homozygous floxed Dicer allele control lines. Both control and Dicer-null NS lines expressed proteins characteristic of NS cells, including the intermediate filament protein Nestin and the transcription factor Sox2 (<a href="#pone-0013453-g001">Fig. 1D–G</a>). Dicer-null NS cells were morphologically different to control cells, lacking the long, fine radial processes seen in control cells, instead having shorter, twisted processes (<a href="#pone-0013453-g001">Fig. 1D, E</a>).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>The ability of control and Dicer-null NS cells to form primary and secondary neurospheres was confirmed for both low and high passage number cells (<a href="#pone-0013453-g001">Fig. 1H, I</a>). Therefore, we conclude that NS cell lines can be established from the embryonic Dicer-null cerebral cortex, and that those NS cells can self-renew appropriately, despite the lack of Dicer and all microRNAs.</p>
</div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="Dicer-null NS cells show subtly altered cell cycle kinetics and heterochromatin organization"></a>
<h3>Dicer-null NS cells show subtly altered cell cycle kinetics and heterochromatin organization</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>ES cells lacking microRNAs due to deletion of the DGCR8 gene, essential for microRNA biogenesis, have been reported to have changes in cell cycle kinetics, with an increase in the number of cells in G1 phase of the cell cycle <a href="#pone.0013453-Wang1" class="ref-tip">[14]</a>, <a href="#pone.0013453-Wang2" class="ref-tip">[31]</a>. Analysis of the cell cycle distribution of two Dicer-null NS cells found that this was broadly similar to that observed in two control, wild-type NS lines (<a href="#pone-0013453-g002">Fig. 2A–D</a>). However, it is noteworthy that the two wild-type lines show some variation in their cell cycle kinetics (<a href="#pone-0013453-g002">Fig 2A, B</a>). In both null lines, reproducible but subtle changes in cell cycle distributions were observed, most notably a reduction in the proportion of cells in G2/M (<a href="#pone-0013453-g002">Fig. 2E</a>), suggesting that these cells are cycling slightly more slowly than controls. This is consistent with the longer time we observed for Dicer null NS cells to reach confluency in culture.</p>
<a class="link-target" id="pone-0013453-g002" name="pone-0013453-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0013453.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0013453.g002" data-doi="info:doi/10.1371/journal.pone.0013453" data-uri="info:doi/10.1371/journal.pone.0013453.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0013453.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Dicer-null NS cells have minor alterations cell cycle kinetics and significant changes in heterochromatin organisation.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>(A–D) Plots of the cell cycle distributions of wildtype (A, B) and Dicer-null (C, D) cells. In both genotypes the majority (∼60%) of cells are in G1, reflecting the relatively slow cycle times for NS cells. Y-axis, cell counts; x-axis, DNA content (Hoechst 33342 fluorescence). (E) Histogram of the proportions of NS cells in each phase of the cell cycle. There is a significant difference in the numbers of cells in G1 in the two control lines, however there is a consistent decrease in the proportion of Dicer-null NS cells in G2/M, compared with controls. (F, G) High-power, confocal images of wild-type (F) and Dicer-null (G) NS cells stained for H3K9Me3. Representative images are shown of the distribution of H3K9Me3 staining in control cells as a relatively small number of large foci (F, arrows). In contrast, the H3K9Me3 distribution is variable in the Dicer-null NS cells, with the majority of cells containing smaller, irregular foci of H3K9Me3 (G). Scale bar, 10 µm. (H) Dicer-null NS cells are karotypically normal, containing the normal complement of 40 chromosomes for mouse and no obvious translocations.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0013453.g002">
              https://doi.org/10.1371/journal.pone.0013453.g002</a></p></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>Dicer has been found to be important for the maintenance of heterochromatin in several vertebrate cell types, including ES cells <a href="#pone.0013453-Kanellopoulou1" class="ref-tip">[13]</a>, <a href="#pone.0013453-Fukagawa1" class="ref-tip">[32]</a>, <a href="#pone.0013453-Giles1" class="ref-tip">[33]</a>. Immunofluorescence analysis of H3K9Me3 levels and localisation in Dicer-null NS cells found that, although total H3K9Me3 levels were broadly similar to those seen in control NS cells, as previously observed <a href="#pone.0013453-KawaseKoga2" class="ref-tip">[20]</a>, the distribution of nuclear H3K9Me3 was highly abnormal. Instead of a small number of relatively large H3K9Me3 foci, Dicer-null NS nuclei typically contain over twice as many H3K9Me3 foci and these are small and irregular in morphology (<a href="#pone-0013453-g002">Fig. 2F, G</a>), suggesting that Dicer-null NS cells fail to maintain heterochromatin appropriately. However, despite this, karyotype analysis demonstrated that Dicer-null NS cells had the normal complement of 40 chromosomes for mouse with no obvious translocations (<a href="#pone-0013453-g002">Fig. 2H</a>).</p>
</div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="Dicer-null NS cells fail to generate differentiated progeny"></a>
<h3>Dicer-null NS cells fail to generate differentiated progeny</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>To test the ability of Dicer-null NS cells to generate differentiated progeny, we placed NS cells in a range of different conditions that drive gliogenesis or neurogenesis, using approaches that depend either on growth factor withdrawal or on positive, instructive signals. Neurogenesis from NS cell cultures can be induced by sequential withdrawal of two growth factors, EGF and FGF2, from the culture medium <a href="#pone.0013453-Conti1" class="ref-tip">[23]</a>. Using this approach, cells with classic neuronal morphology, expressing neuron-specific tubulin were observed within approximately six days in control cultures (<a href="#pone-0013453-g003">Fig. 3E</a>). The same procedure resulted in apoptosis of Dicer-null NS cells upon withdrawal of the first growth factor, EGF, such that there were very few live cells in the culture within 2–3 days and the few surviving cells did not differentiate into neurons (<a href="#pone-0013453-g003">Fig. 3D, F</a>).</p>
<a class="link-target" id="pone-0013453-g003" name="pone-0013453-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0013453.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0013453.g003" data-doi="info:doi/10.1371/journal.pone.0013453" data-uri="info:doi/10.1371/journal.pone.0013453.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0013453.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> Dicer-null NS cells are incapable of neurogenesis and gliogenesis.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p>(A, B) Under self-renewing conditions, Dicer-null NS cells express Nestin, but at a lower level than control NS cells. (C–F) Withdrawal of EGF and continued exposure to FGF2 promotes neurogenesis in control NS cells (Tuj1+ cells, C), but does not result in neurogenesis in Dicer-null NS cells (D). Shortly after EGF withdrawal most Dicer-null NS cells die and detach from the culture plate. Brightfield image of wild-type (E) and Dicer-null (F) 24 hours after EGF-withdrawal. (G, H) Withdrawal of FGF2 and continued exposure to EGF promotes gliogenesis in control NS cells (GFAP+ cells, G), but does not result in gliogenesis in Dicer-null NS cells (H), based on both GFAP expression and cell morphology. (I, J) Control NS cells generate large numbers of GFAP+ astrocytes 48 hours after exposure to BMP4 (I), whereas Dicer-null NS cells fail to express GFAP or assume an astrocytic morphology (J). (K, L) Addition of foetal calf serum (FCS) promotes gliogenesis from control cells within 48 hours (K), but does not stimulate gliogenesis from Dicer-null NS cells (L). Scale bar for all images, 25 µm. Nuclei in all images visualised with DAPI.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0013453.g003">
              https://doi.org/10.1371/journal.pone.0013453.g003</a></p></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>We used three different methods to induce gliogenesis from NS cells: positive instruction by BMP4 or foetal calf serum (FCS), and selective growth factor withdrawal (removal of FGF2 and maintenance of EGF). In control cultures almost every cell had the classic astrocytic morphology and was strongly positive for the astrocyte intermediate filament protein GFAP within 48 hours of BMP4 or FCS addition to the cultures, or 96 hours of FGF2 withdrawal (<a href="#pone-0013453-g003">Fig. 3G, I, K</a>). In contrast, none of the Dicer-null NS cells expressed GFAP upon addition of BMP4 (<a href="#pone-0013453-g003">Fig. 3J</a>). The use of alternative methods for inducing gliogenesis, by withdrawal of FGF2 or the addition of foetal calf serum, completely failed to induce gliogenesis in one line of Dicer-null NS cells and did so at very low efficiency in the other (<a href="#pone-0013453-g003">Fig. 3H, L</a> and data not shown). It is noteworthy that we did not observe large-scale apoptosis under these differentiation conditions, and particularly upon FGF2 withdrawal, in contrast with the apoptosis observed upon EGF withdrawal.</p>
</div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="Dicer-null NS cells exhibit complex changes in mRNA levels resulting in a substantially altered phenotype"></a>
<h3>Dicer-null NS cells exhibit complex changes in mRNA levels resulting in a substantially altered phenotype</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Given the functions of miRNAs in post-transcriptional gene silencing, loss of Dicer and all microRNAs would be expected to result in considerable alterations in mRNA levels and associated phenotypic changes. To assess whether this is the case, mRNA levels in two independent Dicer-null NS lines were compared with matched control NS lines. To control for differences in mRNA levels that were not due to the absence of Dicer and microRNAs, the analysis included a direct comparison of gene expression between the two control lines.</p>
<a id="article1.body1.sec3.sec4.p2" name="article1.body1.sec3.sec4.p2" class="link-target"></a><p>Array analysis of mRNA expression in Dicer-null NS cells revealed significantly altered expression of 3033 transcripts, 1219 of which were increased and 1814 of which decreased in levels in Dicer-null NS cells (<a href="#pone-0013453-g004">Fig. 4A</a>; <a href="#pone.0013453.s002">Table S2</a>). Notable among the transcripts increased in levels in Dicer-null NS cells were stem cell-related transcripts, such as Prominin-1 (CD133) and Hmga2 (<a href="#pone-0013453-g004">Fig. 4B, C</a>), and a number of positive and negative cell cycle regulators, including cyclins E1 and B2 and Cdkn1c. However, there was also an increase in expression of genes normally specifically expressed in differentiating cortical neurons, including Foxp1, Foxp2, Doublecortin, Mtap1b and Enc1 <a href="#pone.0013453-Ferland1" class="ref-tip">[34]</a>, <a href="#pone.0013453-Allen1" class="ref-tip">[35]</a>. This was accompanied by down-regulation of other cell cycle regulators, including Cdkn1a and CyclinD1 (<a href="#pone-0013453-g004">Fig. 4D, E</a> and <a href="#pone.0013453.s002">Table S2</a>). Together, these indicate that the overall phenotypic change in Dicer-null NS cells does not correspond to a simple shift towards or away from a more differentiated phenotype.</p>
<a class="link-target" id="pone-0013453-g004" name="pone-0013453-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0013453.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0013453.g004" data-doi="info:doi/10.1371/journal.pone.0013453" data-uri="info:doi/10.1371/journal.pone.0013453.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0013453.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Dicer-null NS cells exhibit complex changes in mRNA levels resulting in a substantially altered phenotype.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1" class="link-target"></a><p>(A) Dicer-null NS cells show significant changes in the mRNA levels of 3033 genes, compared with control NS cells. Although a small number of transcripts demonstrated over ten-fold alterations in expression, the overwhelming majority showed changes of two-fold or less. See <a href="#s2">methods</a> for technical details of the array hybridisations and statistical analysis. Detailed gene lists are provided in <a href="#pone.0013453.s002">Table S2</a>. (B–E) Changes in mRNA transcripts were reflected in changes in protein expression in cultures of Dicer-null NS cells. Increased Hmga2 mRNA was found to result in the appearance of a subset of strongly HMGA2-expressing nuclei in Dicer-null NS cells, as indicated by the arrows (B, C). Reduced levels of Cyclin D1 mRNA was reflected in the absence of CyclinD1 protein-positive Dicer-null NS cells, compared with controls (yellow arrows in D, with no Ccnd1+ cells detected in the Dicer-null cells in E). Antibody staining is representative of that seen in at least two cultures in two independent lines for each genotype. Scale bar, 25 µm. Nuclei are visualised with DAPI counterstaining. (F) Key Gene Ontology categories found enriched in the sets of genes up (red) and down-regulated in Dicer-null NS cells, compared to wild-type controls. For each category, the number represents the number of genes identified in that category, and the bar size reflects the fold-enrichment of that category in the up- or down-regulated gene set compared with the whole probeset. Details of all enriched GO categories are provided in <a href="#pone.0013453.s003">Table S3</a>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0013453.g004">
              https://doi.org/10.1371/journal.pone.0013453.g004</a></p></div><a id="article1.body1.sec3.sec4.p3" name="article1.body1.sec3.sec4.p3" class="link-target"></a><p>To investigate in an unbiased manner whether there were clear trends in the cellular processes altered in Dicer-null NS cells, we carried out a Gene Ontology analysis of the sets of increased and decreased transcripts. Gene Ontology analysis found significant enrichments for sets of genes involved in several biological processes, suggesting several possible changes in the phenotype of Dicer-null NS cells (<a href="#pone-0013453-g004">Fig. 4F</a>; full details of the GO analysis are included in <a href="#pone.0013453.s003">Table S3</a>). Positive regulators of apoptosis are enriched in genes upregulated in Dicer-null NS cells, whereas negative regulators of apoptosis are enriched in the genes down-regulated in Dicer-null NS cells (<a href="#pone-0013453-g004">Fig. 4F</a>). This is consistent with the extreme sensitivity these cells display to EGF withdrawal. Underlining the complex nature of the phenotype of Dicer-null NS cells, genes involved in neurogenesis and nervous system development are enriched in both the up- and down-regulated gene sets, as are cell cycle regulators and regulators of the cytoskeleton.</p>
</div>

<div id="section5" class="section toc-section"><a id="s3e" name="s3e" class="link-target" title="Dicer-null NS cells initiate but fail to complete astrocyte differentiation programs"></a>
<h3>Dicer-null NS cells initiate but fail to complete astrocyte differentiation programs</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>Dicer-null NS cells completely fail to undergo neurogenesis and are incapable of gliogenesis. However, Dicer-null NS cells do change in morphology and express GFAP in a very small number of cells when placed in glial differentiation promoting conditions, suggesting that they do respond to differentiation signals and can, at very low frequency, complete this differentiation program. To address this, we studied the global gene expression changes that take place in control and Dicer-null NS cells when induced to differentiate. To do so, we used BMP4-induced astrocytic differentiation as a model, as this is an efficient system, with almost 100% of wild-type cells differentiating to astrocytes over 48 hours. Furthermore, this differentiation procedure does not involve EGF withdrawal, which causes massive apoptosis in Dicer-null NS cells (see above). Within 24 hours of BMP4 addition, Dicer-null NS cells down-regulate expression of the diagnostic neural stem cell gene Nestin, (<a href="#pone-0013453-g005">Fig. 5A–D</a>), indicating that the Dicer-null NS cells can respond to differentiation signals appropriately.</p>
<a class="link-target" id="pone-0013453-g005" name="pone-0013453-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0013453.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0013453.g005" data-doi="info:doi/10.1371/journal.pone.0013453" data-uri="info:doi/10.1371/journal.pone.0013453.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0013453.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> Dicer-null NS cells initiate but fail to complete a differentiation program.</div><p class="caption_target"><a id="article1.body1.sec3.sec5.fig1.caption1.p1" name="article1.body1.sec3.sec5.fig1.caption1.p1" class="link-target"></a><p>(A–D) Following two days of astrocytic differentiation in response to BMP4, Nestin protein is down-regulated in both control and Dicer-null NS cells, whereas the expression of the astrocyte-specific protein GFAP is only induced in control NS cells. Scale bar, 25 µm. (E–H) Microarray analysis of BMP4-induced glial differentiation in control and Dicer-null NS cells. Examples of the four major patterns of gene expression observed within 24 hours of BMP4 exposure are shown. In each cluster, each column represents a microarray experiment, each row a unique gene. Two hybridisations for each comparison are shown: control cells 24 hours after BMP4 exposure compared with untreated control cells (+/+); Dicer-null cells 24 hours after BMP4 exposure compared with untreated Dicer-null cells (−/−); BMP4-treated wild-type compared with BMP4-treated Dicer-null cells (+/+ vs −/−). Clusters of transcripts upregulated in both cell types (E) and down-regulated in both cell types (F) are shown, demonstrating that Dicer-null NS cells do execute components of the astrocytic differentiation program. However, as shown in (G) Dicer-null cells also upregulate a set of genes to an inappropriately high level, and also fail to upregulate a large set of genes that are upregulated in control cells (H). (I) Significance analysis of microarrays (SAM) plot comparing gene expression between BMP4-treated wild-type and Dicer-null NS cells (see <a href="#s2">methods</a> for details). At a 0.1 false discovery rate, two sets of transcripts showing differential expression between the two groups of arrays (BMP4-treated Dicer-null/untreated Dicer-null vs BMP4-treated wild-type/untreated wild-type) were observed: genes more highly expressed in BMP-treated Dicer-null cells compared with the equivalent wild-type cells, and genes more highly expressed in BMP4-treated wild-type cells compared with the equivalent Dicer-null cells. (J, K) Hierarchical clustering of the two sets of transcripts detected by the SAM analysis reported in (I). Within each set, two subclusters were found and are labeled: transcripts higher in BMP4-treated Dicer-null cells contained a cluster of genes down-regulated only in the wild-type cells, but not in the Dicer-nulls (J), and a cluster of genes up-regulated in the Dicer-null cells, but not in the wild-type cells (J); similarly, transcripts higher in BMP4-treated wild-type cells contained a cluster of genes up-regulated only in the wild-type cells, but not in the Dicer-nulls (K), and a small cluster of genes down-regulated in the Dicer-null cells, but not in the wild-type cells (K). Cluster contents are detailed in <a href="#pone.0013453.s004">Table S4</a>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0013453.g005">
              https://doi.org/10.1371/journal.pone.0013453.g005</a></p></div><a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2" class="link-target"></a><p>To explore the progression of the BMP4-induced astrocyte differentiation program in wild-type and Dicer-null NS cells, we applied microarray gene expression profiling to study gene expression in the first 24 hours following BMP4 treatment (<a href="#pone-0013453-g005">Fig. 5E–H</a>). Gene expression in BMP4-treated Dicer-null and wild-type NS cells was compared to their untreated counterparts and the expression data first explored by clustering methods to identify the broad patterns of gene expression changes following BMP4 treatment. The array analysis found that Dicer-null NS cells increase expression of approximately half of the genes that are seen to upregulate in control cells and also down-regulate many of the genes that control cells down-regulate upon differentiation (<a href="#pone-0013453-g005">Fig. 5E, F</a>). However, Dicer-null NS cells fail to activate expression of a large set of genes expressed during glial differentiation, including GFAP (<a href="#pone-0013453-g005">Fig. 5H</a>). Furthermore, there is a set of genes inappropriately increased in expression in Dicer-null NS cells that are not induced in wild-type NS cells (<a href="#pone-0013453-g005">Fig. 5G</a>).</p>
<a id="article1.body1.sec3.sec5.p3" name="article1.body1.sec3.sec5.p3" class="link-target"></a><p>To directly identify the differences in the transcriptional responses of Dicer-null and wild-type NS cells to BMP4, a statistical analysis using the significance analysis of microarrays algorithm (SAM, see <a href="#s2">methods</a> for details) was carried out (<a href="#pone-0013453-g005">Fig. 5I</a>). A set of 616 transcripts showing significant (false discovery rate &lt;0.1) differences between the BMP4-treated Dicer-null and wild-type NS cells was hierarchically clustered to aid visualisation of the different classes of expression differences detected (<a href="#pone-0013453-g005">Fig. 5J, K</a>). Four classes were detected: transcripts down-regulated only in the wildtype cells after 24 hours of BMP4 exposure, with no change in expression in Dicer-null NS cells (227 transcripts); transcripts up-regulated only in the Dicer-null cells after 24 hours of BMP4 exposure, with no or significantly lower change in expression in wild-type NS cells (218 transcripts); transcripts only upregulated in wild-type cells (150 transcripts); transcripts downregulated only in Dicer-null cells (21 transcripts). Details of the contents of the SAM clusters (<a href="#pone-0013453-g005">Fig. 5J, K</a>) are given in the supplementary material (<a href="#pone.0013453.s004">Table S4</a>).</p>
<a id="article1.body1.sec3.sec5.p4" name="article1.body1.sec3.sec5.p4" class="link-target"></a><p>The set of genes showing differential expression between Dicer-null and wild-type NS cells was compared to published data on genes specifically expressed in differentiating astrocytes <a href="#pone.0013453-Cahoy1" class="ref-tip">[36]</a>. Of the top 40 genes reported as having astrocyte-specific expression, 8 (20%) are found in the set of genes upregulated in wild-type cells and not by Dicer-null cells (<a href="#pone.0013453.s005">Table S5</a>), including GFAP, Aqp4, Slc1a2 and AldoC <a href="#pone.0013453-Cahoy1" class="ref-tip">[36]</a>. None of the astrocyte-specific genes were found in the other clusters. Therefore, we conclude that Dicer-null NS cells do respond to differentiation signals appropriately and initiate a differentiation program. However, they fail to complete this programme and do not reach the terminally differentiated phenotype.</p>
</div>

<div id="section6" class="section toc-section"><a id="s3f" name="s3f" class="link-target" title="The Dicer-dependent block in NS cell differentiation is reversed by the reintroduction of Dicer"></a>
<h3>The Dicer-dependent block in NS cell differentiation is reversed by the reintroduction of Dicer</h3>
<a id="article1.body1.sec3.sec6.p1" name="article1.body1.sec3.sec6.p1" class="link-target"></a><p>The phenotype of Dicer-null NS cells reported here is complex in its nature: these cells have an abnormal morphology, disrupted heterochromatin, markedly different gene and protein expression, a dependency on EGF for survival and an inability to differentiate. The gene expression phenotype of Dicer-null NS cells alone indicates that these cells are highly abnormal. Therefore, it is possible that the inability to differentiate reflects a pathological, irreversible transformation of these cells. Of particular concern in this respect is the significant upregulation in Hmga2 mRNA and protein levels, given the oncogenic role of Hmga2 in many tumours <a href="#pone.0013453-Young1" class="ref-tip">[37]</a>.</p>
<a id="article1.body1.sec3.sec6.p2" name="article1.body1.sec3.sec6.p2" class="link-target"></a><p>To test whether Dicer-null NS cells are irreversibly transformed to a proliferative phenotype incapable of differentiation, we acutely introduced human Dicer into the Dicer-null NS cells by retroviral misexpression (<a href="#pone-0013453-g006">Fig. 6</a> A, B). Infected cells were then cultured under differentiating conditions 24 hours later. At this early stage of Dicer reintroduction, we found that the Dicer-null NS cells no longer undergo apoptosis upon EGF withdrawal. Importantly, they regained the ability to differentiate to both neurons and glia at high efficiency (<a href="#pone-0013453-g006">Fig. 6C–H</a>). We conclude that, despite the considerable phenotypic changes in the Dicer-null NS cells, acute reintroduction of Dicer corrected the major features of the phenotype, most notably their ability to differentiate. Therefore, Dicer-null NS cells are not transformed and have not irreversibly altered their neurogenic and gliogenic potential.</p>
<a class="link-target" id="pone-0013453-g006" name="pone-0013453-g006"></a><div class="figure" data-doi="10.1371/journal.pone.0013453.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0013453.g006" data-doi="info:doi/10.1371/journal.pone.0013453" data-uri="info:doi/10.1371/journal.pone.0013453.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0013453.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0013453.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0013453.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0013453.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 6. </span> The differentiation block in Dicer-null NS cells can be reversed by reintroduction of Dicer.</div><p class="caption_target"><a id="article1.body1.sec3.sec6.fig1.caption1.p1" name="article1.body1.sec3.sec6.fig1.caption1.p1" class="link-target"></a><p>(A) Genomic PCR for human Dicer (hDicer) with two different sets of hDicer-specific primers in two lines of Dicer-null NS cells following infection with a luciferase-expressing virus (control, C) or a hDicer-expressing virus (hD) demonstrated the presence of hDicer DNA in the pFB-hDicer infected cells. (B) RT-PCR for hDicer RNA in virally-infected cells demonstrates expression of hDicer mRNA. (C–E) Wildtype control cells (C) differentiate to GFAP-expressing astrocytes 48 hours after exposure to FCS, in contrast with Dicer-null NS cells infected with a luciferase-expressing virus (D). Dicer-null NS cells infected with a hDicer-containing virus 24 hours before FCS addition recover the ability to generate GFAP-expressing astrocytes (E). (F–H) Similarly, wildtype cells (F) differentiate to Tuj1-expressing neurons upon withdrawal of EGF and maintenance of FGF2, in contrast with Dicer-null NS cells infected with a luciferase-expressing virus (G), the majority of which undergo apoptosis. Dicer-null NS cells infected with the hDicer-containing virus recover the ability to generate Tuj1-expressing neurons (H). Scale bar, 25 µm. Nuclei in all panels visualised with DAPI.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0013453.g006">
              https://doi.org/10.1371/journal.pone.0013453.g006</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>While many microRNAs are expressed in the developing nervous system in stem cells and differentiating neurons, the roles of microRNAs in key aspects of neural stem cell biology are still not well understood. MicroRNAs have been shown to positively and negatively regulate a variety of processes in the vertebrate nervous system, including morphogenesis of the nervous system in zebrafish <a href="#pone.0013453-Giraldez1" class="ref-tip">[38]</a>, terminal differentiation of olfactory neurons <a href="#pone.0013453-Choi1" class="ref-tip">[11]</a> and the morphology of dendritic spines <a href="#pone.0013453-Schratt1" class="ref-tip">[39]</a>. In several non-neural stem cell types and in germ cells, microRNAs have been found to be required for stem cell maintenance (for review, see <a href="#pone.0013453-Stadler1" class="ref-tip">[40]</a>). Recently, it was found that loss of Dicer reduces the efficiency of neural stem cell derivation from the mouse cerebral cortex, and the self-renewal and differentiation of those cells <a href="#pone.0013453-KawaseKoga2" class="ref-tip">[20]</a>. Dicer has also now been shown to be required for the developmental change in the competence of retinal progenitor cells <a href="#pone.0013453-Georgi1" class="ref-tip">[18]</a>. We find here that NS cells self-renew normally in the absence of Dicer but fail to differentiate due to marked phenotypic changes that block their ability to fully execute a differentiation programme. Importantly, although Dicer-null cells are highly abnormal in many respects, their ability to differentiate is restored shortly after the reintroduction of Dicer.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>Low amounts of microRNAs appear to be sufficient for the microRNA system to function appropriately: we and others have found that the genesis of differentiated cell types occurs in many systems in the presence of low amounts of microRNAs <a href="#pone.0013453-DePietriTonelli1" class="ref-tip">[15]</a>, <a href="#pone.0013453-KawaseKoga1" class="ref-tip">[16]</a>, <a href="#pone.0013453-Harris1" class="ref-tip">[41]</a>, <a href="#pone.0013453-Cobb2" class="ref-tip">[42]</a>. To address whether microRNAs are absolutely required for neural stem cell self-renewal and neurogenesis, we generated neocortical neural stem cell lines lacking Dicer that also lacked all microRNAs. In contrast to a recent report <a href="#pone.0013453-KawaseKoga2" class="ref-tip">[20]</a>, NS cell lines were derived from the E12 mouse cortex at equal efficiency as from control cortices. We attribute this to the difference in the time of Dicer deletion in the two studies and the subsequent perdurance of microRNAs: we found that deletion of Dicer with Nestin-Cre results in microRNAs being present at approximately 40% of wild-type levels at E15. Therefore, the derivation of NS lines from the E12 Dicer null cortex begins when microRNAs are still present, and those lower levels of microRNAs are sufficient for NS cell derivation.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>Having established Dicer-null NS cell lines, we cultured those cells for several weeks to allow the persisting microRNAs to be turned over, confirming the absence of all microRNAs assayed by microarray profiling. In that system, neural stem cell self-renewal was unaffected by the absence of Dicer and microRNAs, and these cell lines could be maintained indefinitely. However, Dicer-null NS cells were incapable of either gliogenesis or neurogenesis, and were unusually sensitive to EGF withdrawal-induced apoptosis. Regardless of the process used to promote the production of differentiated progeny, whether by growth factor withdrawal or by the use of specific (BMP4) or broad (foetal calf serum) inductive signals, Dicer-null NS cells effectively do not generate differentiated progeny.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Neurogenesis and neuronal differentiation continue in the cerebral cortex and retina following Dicer deletion <a href="#pone.0013453-DePietriTonelli1" class="ref-tip">[15]</a>, <a href="#pone.0013453-KawaseKoga1" class="ref-tip">[16]</a>, <a href="#pone.0013453-Damiani1" class="ref-tip">[17]</a>, <a href="#pone.0013453-Georgi1" class="ref-tip">[18]</a>. However, in both parts of the CNS there is a marked increase in apoptosis in the progenitor cell population <a href="#pone.0013453-DePietriTonelli1" class="ref-tip">[15]</a>, <a href="#pone.0013453-KawaseKoga1" class="ref-tip">[16]</a>, <a href="#pone.0013453-Georgi1" class="ref-tip">[18]</a>. The data presented here provide some clarification for the basis of that apoptosis: neural stem and progenitor cells lacking Dicer and microRNAs can self-renew, but undergo apoptosis when they attempt neurogenesis. We cannot ascertain exactly when this apoptosis occurs, whether it is during the later stages of the cell cycle or upon exiting cell cycle and initiating a neuronal differentiation program. With respect to glial differentiation <em>in vitro</em>, Dicer-null NS cells do not undergo apoptosis under those conditions, but do fail to terminally differentiate.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>Although Dicer-null NS cells are grossly similar to their wild-type counterparts, expressing Nestin and Sox2, for example, loss of Dicer and microRNAs results in considerable changes in mRNA transcript levels in NS cells, with sets of both up- and down-regulated transcripts. These changes in mRNA levels would be predicted to have substantial effects on NS cell self-renewal and differentiation. The overall gene expression phenotype of Dicer-null NS cells is complex and cannot simply be defined as corresponding to a more or less well differentiated neural stem cell type: transcripts found in neurons and glia are at comparatively high levels in Dicer-null NS cells, along with many positive cell cycle regulators and genes that drive self-renewal or repress differentiation. Functionally, however, the dominant phenotype observed <em>in vitro</em> is an inability to generate differentiated progeny, with self-renewal effectively uncompromised in culture.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6" class="link-target"></a><p>Notable among the transcripts upregulated in the Dicer-null NS cells is a number of genes normally associated with stem cell maintenance, including Prominin-1 (CD133) and Hmga2. The transcriptional regulator Hmga2 is normally expressed in neural stem cells early in embryonic development, where it actively drives self-renewal by indirectly repressing expression of negative regulators of cell cycle progression <a href="#pone.0013453-Nishino1" class="ref-tip">[43]</a>. Previous studies have shown that the reduction in Hmga2 during development is mediated by let-7 family microRNAs <a href="#pone.0013453-Nishino1" class="ref-tip">[43]</a>, microRNAs expressed in wild-type NS cells and absent from Dicer-null NS cells. Increased Hmga2 expression or Hmga2 gene amplification initiate pituitary tumour development, and increased Hmga2 expression is found in several tumour types <a href="#pone.0013453-Young1" class="ref-tip">[37]</a>. Therefore, the high levels of Hmga2 found in Dicer-null NS cells would be predicted to bias NS cells to self-renewal and inhibit differentiation, and could potentially reflect a transformation of Dicer-null NS cells.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7" class="link-target"></a><p>A common feature of many tumour types is a global reduction in microRNAs<a href="#pone.0013453-Lu1" class="ref-tip">[44]</a>, <a href="#pone.0013453-Kumar1" class="ref-tip">[45]</a>. Together with the increase in Hmga2 and the inability to differentiate, this raises the question as to whether Dicer-null NS cells are transformed in some manner, or bear similarities with glioma cells. Dicer-null NS cells do not appear to be transformed to an oncogenic phenotype, as they show a marked dependence on growth factor signalling for survival, whereas a classic hallmark of the cancer cell is its growth factor independence <a href="#pone.0013453-Hanahan1" class="ref-tip">[46]</a>. In addition, Dicer-null NS cells express many genes that are normally expressed in differentiating neurons, and not in neural stem cells, such as Doublecortin <a href="#pone.0013453-Gleeson1" class="ref-tip">[47]</a>, and the transcription factors Foxp1 and Foxp2 <a href="#pone.0013453-Ferland1" class="ref-tip">[34]</a>. Furthermore, Dicer-null NS cells are incapable of generating differentiated progeny, whereas glioma cells typically can differentiate to neurons or glia <em>in vitro</em> <a href="#pone.0013453-Pollard1" class="ref-tip">[48]</a>. Finally, and most importantly, the inability of Dicer-null cells to differentiate is reversed by the acute reintroduction of Dicer. Therefore, we conclude that while highly abnormal in their gene expression, with subtle changes in their heterochromatin status and cell cycle kinetics, these cells have not irreversibly altered their phenotype. Interestingly, recent genetic studies in mice and humans have found that Dicer haploinsufficiency increases tumour incidence, but that both alleles of Dicer are rarely deleted in human tumour cells <em>in vivo</em> <a href="#pone.0013453-Kumar2" class="ref-tip">[49]</a>, again suggesting that somatic cells lacking all Dicer function do not become transformed.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8" class="link-target"></a><p>The lack of dependence of NS cell self-renewal on Dicer and microRNAs, combined with the inability of those cells to generate differentiated progeny, has obvious parallels with the functions of Dicer and microRNAs in embryonic stem cells. Dicer or DGCR8 null ES cells can self-renew but fail to differentiate <a href="#pone.0013453-Kanellopoulou1" class="ref-tip">[13]</a>, <a href="#pone.0013453-Wang1" class="ref-tip">[14]</a>. DGCR8 null ES cells maintain inappropriate expression of stem cell regulators under differentiation-promoting culture conditions <a href="#pone.0013453-Wang1" class="ref-tip">[14]</a>. The persistent expression of stem cell regulators under differentiating conditions essentially renders DGCR8 null ES cells resistant to differentiation and this phenotype was attributed to a failure of the microRNA system to reduce the levels of those proteins during differentiation <a href="#pone.0013453-Wang1" class="ref-tip">[14]</a>.</p>
<a id="article1.body1.sec4.p9" name="article1.body1.sec4.p9" class="link-target"></a><p>Data presented here suggest that an additional microRNA function in neural stem cells may be to modulate the levels of regulators of self-renewal to a level compatible with differentiation, effectively lowering the barrier to differentiation given the appropriate inductive signals. Dicer-null NS cells do respond to differentiation-inducing signals, up-regulating and down-regulating sets of transcripts in the same manner as wild-type cells. However, Dicer-null NS cells fail to complete a differentiation program, and in the case of gliogenesis this appears to be the result of not up-regulating expression of a large set of cell-specific transcripts characteristic of the differentiated astrocyte.</p>
<a id="article1.body1.sec4.p10" name="article1.body1.sec4.p10" class="link-target"></a><p>In conclusion, we have found that the dominant phenotype observed upon removal of all microRNAs in NS cells is a failure to maintain cellular homeostasis and to regulate critical transitions during differentiation. In support of this, we found that neural stem cells could proceed through the cell cycle and self-renew in the absence of Dicer and all microRNAs, were substantially different phenotypically from their wild-type counterparts and could not generate differentiated progeny. However, their ability to differentiate is restored by the reintroduction of Dicer, indicating that Dicer-null NS cells are not irreversibly transformed.</p>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0013453"></div><div class="supplementary-material"><a name="pone.0013453.s001" id="pone.0013453.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0013453.s001">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">microRNAs detected reproducibly in cortical NS cells in vitro.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0013453.s001">https://doi.org/10.1371/journal.pone.0013453.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.01 MB XLS)</p>
</div><div class="supplementary-material"><a name="pone.0013453.s002" id="pone.0013453.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0013453.s002">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Limma analysis to identify genes with increased or decreased expression in Dicer-null NS cells compared with wild-type controls.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0013453.s002">https://doi.org/10.1371/journal.pone.0013453.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(1.01 MB XLS)</p>
</div><div class="supplementary-material"><a name="pone.0013453.s003" id="pone.0013453.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0013453.s003">Table S3. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">Gene Ontology analysis of genes with increased or decreased expression in Dicer-null NS cells compared with wild-type controls.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0013453.s003">https://doi.org/10.1371/journal.pone.0013453.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.31 MB XLS)</p>
</div><div class="supplementary-material"><a name="pone.0013453.s004" id="pone.0013453.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0013453.s004">Table S4. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">Contents of clusters of genes showing differential expression between BMP4-treated Dicer-null and wild-type NS cells.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0013453.s004">https://doi.org/10.1371/journal.pone.0013453.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.72 MB XLS)</p>
</div><div class="supplementary-material"><a name="pone.0013453.s005" id="pone.0013453.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0013453.s005">Table S5. </a></h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI">Expression of astrocyte-specific genes in BMP4-treated wild-type and Dicer-null NS cells.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0013453.s005">https://doi.org/10.1371/journal.pone.0013453.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(0.03 MB XLS)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The authors thank Dr P Provost for the gift of the human full-length Dicer construct, Dr M Narita for providing the anti-Hmga2 antibody, Prof A Surani for advice and support and the members of the Livesey lab for comments and feedback on the manuscript.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: TA SR FJL. Performed the experiments: TA SR SNS JMA JS FJL. Analyzed the data: TA SR SNS JMA JS FJL. Contributed reagents/materials/analysis tools: MK DO AT. Wrote the paper: TA FJL.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0013453-Bartel1" id="pone.0013453-Bartel1" class="link-target"></a>Bartel DP, Chen CZ (2004) Micromanagers of gene expression: the potentially widespread influence of metazoan microRNAs. Nat Rev Genet  5: 396–400.  <ul class="reflinks"><li><a href="#" data-author="Bartel" data-cit="BartelDPChenCZ2004Micromanagers%20of%20gene%20expression%3A%20the%20potentially%20widespread%20influence%20of%20metazoan%20microRNAs.Nat%20Rev%20Genet5396400" data-title="Micromanagers%20of%20gene%20expression%3A%20the%20potentially%20widespread%20influence%20of%20metazoan%20microRNAs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Micromanagers+of+gene+expression%3A+the+potentially+widespread+influence+of+metazoan+microRNAs.+Bartel+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0013453-Ambros1" id="pone.0013453-Ambros1" class="link-target"></a>Ambros V (2004) The functions of animal microRNAs. Nature  431: 350–355.  <ul class="reflinks"><li><a href="#" data-author="Ambros" data-cit="AmbrosV2004The%20functions%20of%20animal%20microRNAs.Nature431350355" data-title="The%20functions%20of%20animal%20microRNAs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+functions+of+animal+microRNAs.+Ambros+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0013453-Kosik1" id="pone.0013453-Kosik1" class="link-target"></a>Kosik KS, Krichevsky AM (2005) The Elegance of the MicroRNAs: A Neuronal Perspective. Neuron  47: 779–782.  <ul class="reflinks"><li><a href="#" data-author="Kosik" data-cit="KosikKSKrichevskyAM2005The%20Elegance%20of%20the%20MicroRNAs%3A%20A%20Neuronal%20Perspective.Neuron47779782" data-title="The%20Elegance%20of%20the%20MicroRNAs%3A%20A%20Neuronal%20Perspective." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Elegance+of+the+MicroRNAs%3A+A+Neuronal+Perspective.+Kosik+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0013453-Cheng1" id="pone.0013453-Cheng1" class="link-target"></a>Cheng LC, Tavazoie M, Doetsch F (2005) Stem cells: from epigenetics to microRNAs. Neuron  46: 363–367.  <ul class="reflinks"><li><a href="#" data-author="Cheng" data-cit="ChengLCTavazoieMDoetschF2005Stem%20cells%3A%20from%20epigenetics%20to%20microRNAs.Neuron46363367" data-title="Stem%20cells%3A%20from%20epigenetics%20to%20microRNAs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Stem+cells%3A+from+epigenetics+to+microRNAs.+Cheng+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0013453-LagosQuintana1" id="pone.0013453-LagosQuintana1" class="link-target"></a>Lagos-Quintana M, Rauhut R, Yalcin A, Meyer J, Lendeckel W, et al.  (2002) Identification of tissue-specific microRNAs from mouse. Curr Biol  12: 735–739.  <ul class="reflinks"><li><a href="#" data-author="Lagos-Quintana" data-cit="Lagos-QuintanaMRauhutRYalcinAMeyerJLendeckelW2002Identification%20of%20tissue-specific%20microRNAs%20from%20mouse.Curr%20Biol12735739" data-title="Identification%20of%20tissue-specific%20microRNAs%20from%20mouse." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+tissue-specific+microRNAs+from+mouse.+Lagos-Quintana+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0013453-Houbaviy1" id="pone.0013453-Houbaviy1" class="link-target"></a>Houbaviy HB, Murray MF, Sharp PA (2003) Embryonic stem cell-specific MicroRNAs. Dev Cell  5: 351–358.  <ul class="reflinks"><li><a href="#" data-author="Houbaviy" data-cit="HoubaviyHBMurrayMFSharpPA2003Embryonic%20stem%20cell-specific%20MicroRNAs.Dev%20Cell5351358" data-title="Embryonic%20stem%20cell-specific%20MicroRNAs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Embryonic+stem+cell-specific+MicroRNAs.+Houbaviy+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0013453-Krichevsky1" id="pone.0013453-Krichevsky1" class="link-target"></a>Krichevsky AM, King KS, Donahue CP, Khrapko K, Kosik KS (2003) A microRNA array reveals extensive regulation of microRNAs during brain development. Rna  9: 1274–1281.  <ul class="reflinks"><li><a href="#" data-author="Krichevsky" data-cit="KrichevskyAMKingKSDonahueCPKhrapkoKKosikKS2003A%20microRNA%20array%20reveals%20extensive%20regulation%20of%20microRNAs%20during%20brain%20development.Rna912741281" data-title="A%20microRNA%20array%20reveals%20extensive%20regulation%20of%20microRNAs%20during%20brain%20development." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+microRNA+array+reveals+extensive+regulation+of+microRNAs+during+brain+development.+Krichevsky+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0013453-Miska1" id="pone.0013453-Miska1" class="link-target"></a>Miska EA, Alvarez-Saavedra E, Townsend M, Yoshii A, Sestan N, et al.  (2004) Microarray analysis of microRNA expression in the developing mammalian brain. Genome Biol  5: R68.  <ul class="reflinks"><li><a href="#" data-author="Miska" data-cit="MiskaEAAlvarez-SaavedraETownsendMYoshiiASestanN2004Microarray%20analysis%20of%20microRNA%20expression%20in%20the%20developing%20mammalian%20brain.Genome%20Biol5R68" data-title="Microarray%20analysis%20of%20microRNA%20expression%20in%20the%20developing%20mammalian%20brain." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Microarray+analysis+of+microRNA+expression+in+the+developing+mammalian+brain.+Miska+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0013453-Sempere1" id="pone.0013453-Sempere1" class="link-target"></a>Sempere LF, Freemantle S, Pitha-Rowe I, Moss E, Dmitrovsky E, et al.  (2004) Expression profiling of mammalian microRNAs uncovers a subset of brain-expressed microRNAs with possible roles in murine and human neuronal differentiation. Genome Biol  5: R13.  <ul class="reflinks"><li><a href="#" data-author="Sempere" data-cit="SempereLFFreemantleSPitha-RoweIMossEDmitrovskyE2004Expression%20profiling%20of%20mammalian%20microRNAs%20uncovers%20a%20subset%20of%20brain-expressed%20microRNAs%20with%20possible%20roles%20in%20murine%20and%20human%20neuronal%20differentiation.Genome%20Biol5R13" data-title="Expression%20profiling%20of%20mammalian%20microRNAs%20uncovers%20a%20subset%20of%20brain-expressed%20microRNAs%20with%20possible%20roles%20in%20murine%20and%20human%20neuronal%20differentiation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+profiling+of+mammalian+microRNAs+uncovers+a+subset+of+brain-expressed+microRNAs+with+possible+roles+in+murine+and+human+neuronal+differentiation.+Sempere+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0013453-Krichevsky2" id="pone.0013453-Krichevsky2" class="link-target"></a>Krichevsky AM, Sonntag KC, Isacson O, Kosik KS (2006) Specific microRNAs modulate embryonic stem cell-derived neurogenesis. Stem Cells  24: 857–864.  <ul class="reflinks"><li><a href="#" data-author="Krichevsky" data-cit="KrichevskyAMSonntagKCIsacsonOKosikKS2006Specific%20microRNAs%20modulate%20embryonic%20stem%20cell-derived%20neurogenesis.Stem%20Cells24857864" data-title="Specific%20microRNAs%20modulate%20embryonic%20stem%20cell-derived%20neurogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Specific+microRNAs+modulate+embryonic+stem+cell-derived+neurogenesis.+Krichevsky+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0013453-Choi1" id="pone.0013453-Choi1" class="link-target"></a>Choi PS, Zakhary L, Choi WY, Caron S, Alvarez-Saavedra E, et al.  (2008) Members of the miRNA-200 family regulate olfactory neurogenesis. Neuron  57: 41–55.  <ul class="reflinks"><li><a href="#" data-author="Choi" data-cit="ChoiPSZakharyLChoiWYCaronSAlvarez-SaavedraE2008Members%20of%20the%20miRNA-200%20family%20regulate%20olfactory%20neurogenesis.Neuron574155" data-title="Members%20of%20the%20miRNA-200%20family%20regulate%20olfactory%20neurogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Members+of+the+miRNA-200+family+regulate+olfactory+neurogenesis.+Choi+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0013453-Murchison1" id="pone.0013453-Murchison1" class="link-target"></a>Murchison EP, Partridge JF, Tam OH, Cheloufi S, Hannon GJ (2005) Characterization of Dicer-deficient murine embryonic stem cells. Proc Natl Acad Sci U S A  102: 12135–12140.  <ul class="reflinks"><li><a href="#" data-author="Murchison" data-cit="MurchisonEPPartridgeJFTamOHCheloufiSHannonGJ2005Characterization%20of%20Dicer-deficient%20murine%20embryonic%20stem%20cells.Proc%20Natl%20Acad%20Sci%20U%20S%20A1021213512140" data-title="Characterization%20of%20Dicer-deficient%20murine%20embryonic%20stem%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+Dicer-deficient+murine+embryonic+stem+cells.+Murchison+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0013453-Kanellopoulou1" id="pone.0013453-Kanellopoulou1" class="link-target"></a>Kanellopoulou C, Muljo SA, Kung AL, Ganesan S, Drapkin R, et al.  (2005) Dicer-deficient mouse embryonic stem cells are defective in differentiation and centromeric silencing. Genes Dev  19: 489–501.  <ul class="reflinks"><li><a href="#" data-author="Kanellopoulou" data-cit="KanellopoulouCMuljoSAKungALGanesanSDrapkinR2005Dicer-deficient%20mouse%20embryonic%20stem%20cells%20are%20defective%20in%20differentiation%20and%20centromeric%20silencing.Genes%20Dev19489501" data-title="Dicer-deficient%20mouse%20embryonic%20stem%20cells%20are%20defective%20in%20differentiation%20and%20centromeric%20silencing." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dicer-deficient+mouse+embryonic+stem+cells+are+defective+in+differentiation+and+centromeric+silencing.+Kanellopoulou+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0013453-Wang1" id="pone.0013453-Wang1" class="link-target"></a>Wang Y, Medvid R, Melton C, Jaenisch R, Blelloch R (2007) DGCR8 is essential for microRNA biogenesis and silencing of embryonic stem cell self-renewal. Nat Genet  39: 380–385.  <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangYMedvidRMeltonCJaenischRBlellochR2007DGCR8%20is%20essential%20for%20microRNA%20biogenesis%20and%20silencing%20of%20embryonic%20stem%20cell%20self-renewal.Nat%20Genet39380385" data-title="DGCR8%20is%20essential%20for%20microRNA%20biogenesis%20and%20silencing%20of%20embryonic%20stem%20cell%20self-renewal." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=DGCR8+is+essential+for+microRNA+biogenesis+and+silencing+of+embryonic+stem+cell+self-renewal.+Wang+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0013453-DePietriTonelli1" id="pone.0013453-DePietriTonelli1" class="link-target"></a>De Pietri Tonelli D, Pulvers JN, Haffner C, Murchison EP, Hannon GJ, et al.  (2008) miRNAs are essential for survival and differentiation of newborn neurons but not for expansion of neural progenitors during early neurogenesis in the mouse embryonic neocortex. Development  135: 3911–3921.  <ul class="reflinks"><li><a href="#" data-author="De%20Pietri%20Tonelli" data-cit="De%20Pietri%20TonelliDPulversJNHaffnerCMurchisonEPHannonGJ2008miRNAs%20are%20essential%20for%20survival%20and%20differentiation%20of%20newborn%20neurons%20but%20not%20for%20expansion%20of%20neural%20progenitors%20during%20early%20neurogenesis%20in%20the%20mouse%20embryonic%20neocortex.Development13539113921" data-title="miRNAs%20are%20essential%20for%20survival%20and%20differentiation%20of%20newborn%20neurons%20but%20not%20for%20expansion%20of%20neural%20progenitors%20during%20early%20neurogenesis%20in%20the%20mouse%20embryonic%20neocortex." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miRNAs+are+essential+for+survival+and+differentiation+of+newborn+neurons+but+not+for+expansion+of+neural+progenitors+during+early+neurogenesis+in+the+mouse+embryonic+neocortex.+De+Pietri+Tonelli+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0013453-KawaseKoga1" id="pone.0013453-KawaseKoga1" class="link-target"></a>Kawase-Koga Y, Otaegi G, Sun T (2009) Different timings of Dicer deletion affect neurogenesis and gliogenesis in the developing mouse central nervous system. Dev Dyn  238: 2800–2812.  <ul class="reflinks"><li><a href="#" data-author="Kawase-Koga" data-cit="Kawase-KogaYOtaegiGSunT2009Different%20timings%20of%20Dicer%20deletion%20affect%20neurogenesis%20and%20gliogenesis%20in%20the%20developing%20mouse%20central%20nervous%20system.Dev%20Dyn23828002812" data-title="Different%20timings%20of%20Dicer%20deletion%20affect%20neurogenesis%20and%20gliogenesis%20in%20the%20developing%20mouse%20central%20nervous%20system." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Different+timings+of+Dicer+deletion+affect+neurogenesis+and+gliogenesis+in+the+developing+mouse+central+nervous+system.+Kawase-Koga+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0013453-Damiani1" id="pone.0013453-Damiani1" class="link-target"></a>Damiani D, Alexander JJ, O'Rourke JR, McManus M, Jadhav AP, et al.  (2008) Dicer inactivation leads to progressive functional and structural degeneration of the mouse retina. J Neurosci  28: 4878–4887.  <ul class="reflinks"><li><a href="#" data-author="Damiani" data-cit="DamianiDAlexanderJJO%27RourkeJRMcManusMJadhavAP2008Dicer%20inactivation%20leads%20to%20progressive%20functional%20and%20structural%20degeneration%20of%20the%20mouse%20retina.J%20Neurosci2848784887" data-title="Dicer%20inactivation%20leads%20to%20progressive%20functional%20and%20structural%20degeneration%20of%20the%20mouse%20retina." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dicer+inactivation+leads+to+progressive+functional+and+structural+degeneration+of+the+mouse+retina.+Damiani+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0013453-Georgi1" id="pone.0013453-Georgi1" class="link-target"></a>Georgi SA, Reh TA (2010) Dicer is required for the transition from early to late progenitor state in the developing mouse retina. J Neurosci  30: 4048–4061.  <ul class="reflinks"><li><a href="#" data-author="Georgi" data-cit="GeorgiSARehTA2010Dicer%20is%20required%20for%20the%20transition%20from%20early%20to%20late%20progenitor%20state%20in%20the%20developing%20mouse%20retina.J%20Neurosci3040484061" data-title="Dicer%20is%20required%20for%20the%20transition%20from%20early%20to%20late%20progenitor%20state%20in%20the%20developing%20mouse%20retina." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dicer+is+required+for+the+transition+from+early+to+late+progenitor+state+in+the+developing+mouse+retina.+Georgi+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0013453-Cobb1" id="pone.0013453-Cobb1" class="link-target"></a>Cobb BS, Nesterova TB, Thompson E, Hertweck A, O'Connor E, et al.  (2005) T cell lineage choice and differentiation in the absence of the RNase III enzyme Dicer. J Exp Med  201: 1367–1373.  <ul class="reflinks"><li><a href="#" data-author="Cobb" data-cit="CobbBSNesterovaTBThompsonEHertweckAO%27ConnorE2005T%20cell%20lineage%20choice%20and%20differentiation%20in%20the%20absence%20of%20the%20RNase%20III%20enzyme%20Dicer.J%20Exp%20Med20113671373" data-title="T%20cell%20lineage%20choice%20and%20differentiation%20in%20the%20absence%20of%20the%20RNase%20III%20enzyme%20Dicer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=T+cell+lineage+choice+and+differentiation+in+the+absence+of+the+RNase+III+enzyme+Dicer.+Cobb+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0013453-KawaseKoga2" id="pone.0013453-KawaseKoga2" class="link-target"></a>Kawase-Koga Y, Low R, Otaegi G, Pollock A, Deng H, et al.  (2010) RNAase-III enzyme Dicer maintains signaling pathways for differentiation and survival in mouse cortical neural stem cells. J Cell Sci  123: 586–594.  <ul class="reflinks"><li><a href="#" data-author="Kawase-Koga" data-cit="Kawase-KogaYLowROtaegiGPollockADengH2010RNAase-III%20enzyme%20Dicer%20maintains%20signaling%20pathways%20for%20differentiation%20and%20survival%20in%20mouse%20cortical%20neural%20stem%20cells.J%20Cell%20Sci123586594" data-title="RNAase-III%20enzyme%20Dicer%20maintains%20signaling%20pathways%20for%20differentiation%20and%20survival%20in%20mouse%20cortical%20neural%20stem%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=RNAase-III+enzyme+Dicer+maintains+signaling+pathways+for+differentiation+and+survival+in+mouse+cortical+neural+stem+cells.+Kawase-Koga+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0013453-Tronche1" id="pone.0013453-Tronche1" class="link-target"></a>Tronche F, Kellendonk C, Kretz O, Gass P, Anlag K, et al.  (1999) Disruption of the glucocorticoid receptor gene in the nervous system results in reduced anxiety. Nat Genet  23: 99–103.  <ul class="reflinks"><li><a href="#" data-author="Tronche" data-cit="TroncheFKellendonkCKretzOGassPAnlagK1999Disruption%20of%20the%20glucocorticoid%20receptor%20gene%20in%20the%20nervous%20system%20results%20in%20reduced%20anxiety.Nat%20Genet2399103" data-title="Disruption%20of%20the%20glucocorticoid%20receptor%20gene%20in%20the%20nervous%20system%20results%20in%20reduced%20anxiety." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Disruption+of+the+glucocorticoid+receptor+gene+in+the+nervous+system+results+in+reduced+anxiety.+Tronche+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0013453-Yi1" id="pone.0013453-Yi1" class="link-target"></a>Yi R, O'Carroll D, Pasolli HA, Zhang Z, Dietrich FS, et al.  (2006) Morphogenesis in skin is governed by discrete sets of differentially expressed microRNAs. Nat Genet  38: 356–362.  <ul class="reflinks"><li><a href="#" data-author="Yi" data-cit="YiRO%27CarrollDPasolliHAZhangZDietrichFS2006Morphogenesis%20in%20skin%20is%20governed%20by%20discrete%20sets%20of%20differentially%20expressed%20microRNAs.Nat%20Genet38356362" data-title="Morphogenesis%20in%20skin%20is%20governed%20by%20discrete%20sets%20of%20differentially%20expressed%20microRNAs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Morphogenesis+in+skin+is+governed+by+discrete+sets+of+differentially+expressed+microRNAs.+Yi+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0013453-Conti1" id="pone.0013453-Conti1" class="link-target"></a>Conti L, Pollard SM, Gorba T, Reitano E, Toselli M, et al.  (2005) Niche-independent symmetrical self-renewal of a mammalian tissue stem cell. PLoS Biol  3: e283.  <ul class="reflinks"><li><a href="#" data-author="Conti" data-cit="ContiLPollardSMGorbaTReitanoEToselliM2005Niche-independent%20symmetrical%20self-renewal%20of%20a%20mammalian%20tissue%20stem%20cell.PLoS%20Biol3e283" data-title="Niche-independent%20symmetrical%20self-renewal%20of%20a%20mammalian%20tissue%20stem%20cell." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Niche-independent+symmetrical+self-renewal+of+a+mammalian+tissue+stem+cell.+Conti+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0013453-Matz1" id="pone.0013453-Matz1" class="link-target"></a>Matz M, Shagin D, Bogdanova E, Britanova O, Lukyanov S, et al.  (1999) Amplification of cDNA ends based on template-switching effect and step-out PCR. Nucleic Acids Res  27: 1558–1560.  <ul class="reflinks"><li><a href="#" data-author="Matz" data-cit="MatzMShaginDBogdanovaEBritanovaOLukyanovS1999Amplification%20of%20cDNA%20ends%20based%20on%20template-switching%20effect%20and%20step-out%20PCR.Nucleic%20Acids%20Res2715581560" data-title="Amplification%20of%20cDNA%20ends%20based%20on%20template-switching%20effect%20and%20step-out%20PCR." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Amplification+of+cDNA+ends+based+on+template-switching+effect+and+step-out+PCR.+Matz+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0013453-Livesey1" id="pone.0013453-Livesey1" class="link-target"></a>Livesey FJ, Furukawa T, Steffen MA, Church GM, Cepko CL (2000) Microarray analysis of the transcriptional network controlled by the photoreceptor homeobox gene Crx. Curr Biol  10: 301–310.  <ul class="reflinks"><li><a href="#" data-author="Livesey" data-cit="LiveseyFJFurukawaTSteffenMAChurchGMCepkoCL2000Microarray%20analysis%20of%20the%20transcriptional%20network%20controlled%20by%20the%20photoreceptor%20homeobox%20gene%20Crx.Curr%20Biol10301310" data-title="Microarray%20analysis%20of%20the%20transcriptional%20network%20controlled%20by%20the%20photoreceptor%20homeobox%20gene%20Crx." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Microarray+analysis+of+the+transcriptional+network+controlled+by+the+photoreceptor+homeobox+gene+Crx.+Livesey+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0013453-Livesey2" id="pone.0013453-Livesey2" class="link-target"></a>Livesey FJ, Young TL, Cepko CL (2004) An analysis of the gene expression program of mammalian neural progenitor cells. Proc Natl Acad Sci U S A  101: 1374–1379.  <ul class="reflinks"><li><a href="#" data-author="Livesey" data-cit="LiveseyFJYoungTLCepkoCL2004An%20analysis%20of%20the%20gene%20expression%20program%20of%20mammalian%20neural%20progenitor%20cells.Proc%20Natl%20Acad%20Sci%20U%20S%20A10113741379" data-title="An%20analysis%20of%20the%20gene%20expression%20program%20of%20mammalian%20neural%20progenitor%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+analysis+of+the+gene+expression+program+of+mammalian+neural+progenitor+cells.+Livesey+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0013453-Smyth1" id="pone.0013453-Smyth1" class="link-target"></a>Smyth G (2005)  Limma: linear models for microarray data. In:  Gentleman R, Carey V, Dudoit S, Irizarry R, Huber W, editors. Bioinformatics and Computational Biology Solutions using R and Bioconductor. New York: Springer. pp. 397–420.  <ul class="find-nolinks"></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0013453-Martin1" id="pone.0013453-Martin1" class="link-target"></a>Martin D, Brun C, Remy E, Mouren P, Thieffry D, et al.  (2004) GOToolBox: functional analysis of gene datasets based on Gene Ontology. Genome Biol  5: R101.  <ul class="reflinks"><li><a href="#" data-author="Martin" data-cit="MartinDBrunCRemyEMourenPThieffryD2004GOToolBox%3A%20functional%20analysis%20of%20gene%20datasets%20based%20on%20Gene%20Ontology.Genome%20Biol5R101" data-title="GOToolBox%3A%20functional%20analysis%20of%20gene%20datasets%20based%20on%20Gene%20Ontology." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=GOToolBox%3A+functional+analysis+of+gene+datasets+based+on+Gene+Ontology.+Martin+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0013453-Tusher1" id="pone.0013453-Tusher1" class="link-target"></a>Tusher VG, Tibshirani R, Chu G (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc Natl Acad Sci U S A  98: 5116–5121.  <ul class="reflinks"><li><a href="#" data-author="Tusher" data-cit="TusherVGTibshiraniRChuG2001Significance%20analysis%20of%20microarrays%20applied%20to%20the%20ionizing%20radiation%20response.Proc%20Natl%20Acad%20Sci%20U%20S%20A9851165121" data-title="Significance%20analysis%20of%20microarrays%20applied%20to%20the%20ionizing%20radiation%20response." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Significance+analysis+of+microarrays+applied+to+the+ionizing+radiation+response.+Tusher+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0013453-Saeed1" id="pone.0013453-Saeed1" class="link-target"></a>Saeed AI, Bhagabati NK, Braisted JC, Liang W, Sharov V, et al.  (2006) TM4 microarray software suite. Methods Enzymol  411: 134–193.  <ul class="reflinks"><li><a href="#" data-author="Saeed" data-cit="SaeedAIBhagabatiNKBraistedJCLiangWSharovV2006TM4%20microarray%20software%20suite.Methods%20Enzymol411134193" data-title="TM4%20microarray%20software%20suite." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=TM4+microarray+software+suite.+Saeed+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0013453-Wang2" id="pone.0013453-Wang2" class="link-target"></a>Wang Y, Baskerville S, Shenoy A, Babiarz JE, Baehner L, et al.  (2008) Embryonic stem cell-specific microRNAs regulate the G1-S transition and promote rapid proliferation. Nat Genet  40: 1478–1483.  <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangYBaskervilleSShenoyABabiarzJEBaehnerL2008Embryonic%20stem%20cell-specific%20microRNAs%20regulate%20the%20G1-S%20transition%20and%20promote%20rapid%20proliferation.Nat%20Genet4014781483" data-title="Embryonic%20stem%20cell-specific%20microRNAs%20regulate%20the%20G1-S%20transition%20and%20promote%20rapid%20proliferation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Embryonic+stem+cell-specific+microRNAs+regulate+the+G1-S+transition+and+promote+rapid+proliferation.+Wang+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0013453-Fukagawa1" id="pone.0013453-Fukagawa1" class="link-target"></a>Fukagawa T, Nogami M, Yoshikawa M, Ikeno M, Okazaki T, et al.  (2004) Dicer is essential for formation of the heterochromatin structure in vertebrate cells. Nat Cell Biol  6: 784–791.  <ul class="reflinks"><li><a href="#" data-author="Fukagawa" data-cit="FukagawaTNogamiMYoshikawaMIkenoMOkazakiT2004Dicer%20is%20essential%20for%20formation%20of%20the%20heterochromatin%20structure%20in%20vertebrate%20cells.Nat%20Cell%20Biol6784791" data-title="Dicer%20is%20essential%20for%20formation%20of%20the%20heterochromatin%20structure%20in%20vertebrate%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dicer+is+essential+for+formation+of+the+heterochromatin+structure+in+vertebrate+cells.+Fukagawa+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0013453-Giles1" id="pone.0013453-Giles1" class="link-target"></a>Giles KE, Ghirlando R, Felsenfeld G (2010) Maintenance of a constitutive heterochromatin domain in vertebrates by a Dicer-dependent mechanism. Nat Cell Biol  12: 94–99. sup pp 91–96.  <ul class="reflinks"><li><a href="#" data-author="Giles" data-cit="GilesKEGhirlandoRFelsenfeldG2010Maintenance%20of%20a%20constitutive%20heterochromatin%20domain%20in%20vertebrates%20by%20a%20Dicer-dependent%20mechanism.Nat%20Cell%20Biol129499sup%20pp%2091%E2%80%9396" data-title="Maintenance%20of%20a%20constitutive%20heterochromatin%20domain%20in%20vertebrates%20by%20a%20Dicer-dependent%20mechanism." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Maintenance+of+a+constitutive+heterochromatin+domain+in+vertebrates+by+a+Dicer-dependent+mechanism.+Giles+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0013453-Ferland1" id="pone.0013453-Ferland1" class="link-target"></a>Ferland RJ, Cherry TJ, Preware PO, Morrisey EE, Walsh CA (2003) Characterization of Foxp2 and Foxp1 mRNA and protein in the developing and mature brain. J Comp Neurol  460: 266–279.  <ul class="reflinks"><li><a href="#" data-author="Ferland" data-cit="FerlandRJCherryTJPrewarePOMorriseyEEWalshCA2003Characterization%20of%20Foxp2%20and%20Foxp1%20mRNA%20and%20protein%20in%20the%20developing%20and%20mature%20brain.J%20Comp%20Neurol460266279" data-title="Characterization%20of%20Foxp2%20and%20Foxp1%20mRNA%20and%20protein%20in%20the%20developing%20and%20mature%20brain." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+Foxp2+and+Foxp1+mRNA+and+protein+in+the+developing+and+mature+brain.+Ferland+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0013453-Allen1" id="pone.0013453-Allen1" class="link-target"></a>Allen KM, Walsh CA (1999) Genes that regulate neuronal migration in the cerebral cortex. Epilepsy Res  36: 143–154.  <ul class="reflinks"><li><a href="#" data-author="Allen" data-cit="AllenKMWalshCA1999Genes%20that%20regulate%20neuronal%20migration%20in%20the%20cerebral%20cortex.Epilepsy%20Res36143154" data-title="Genes%20that%20regulate%20neuronal%20migration%20in%20the%20cerebral%20cortex." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genes+that+regulate+neuronal+migration+in+the+cerebral+cortex.+Allen+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0013453-Cahoy1" id="pone.0013453-Cahoy1" class="link-target"></a>Cahoy JD, Emery B, Kaushal A, Foo LC, Zamanian JL, et al.  (2008) A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. J Neurosci  28: 264–278.  <ul class="reflinks"><li><a href="#" data-author="Cahoy" data-cit="CahoyJDEmeryBKaushalAFooLCZamanianJL2008A%20transcriptome%20database%20for%20astrocytes%2C%20neurons%2C%20and%20oligodendrocytes%3A%20a%20new%20resource%20for%20understanding%20brain%20development%20and%20function.J%20Neurosci28264278" data-title="A%20transcriptome%20database%20for%20astrocytes%2C%20neurons%2C%20and%20oligodendrocytes%3A%20a%20new%20resource%20for%20understanding%20brain%20development%20and%20function." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+transcriptome+database+for+astrocytes%2C+neurons%2C+and+oligodendrocytes%3A+a+new+resource+for+understanding+brain+development+and+function.+Cahoy+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0013453-Young1" id="pone.0013453-Young1" class="link-target"></a>Young AR, Narita M (2007) Oncogenic HMGA2: short or small? Genes Dev  21: 1005–1009.  <ul class="reflinks"><li><a href="#" data-author="Young" data-cit="YoungARNaritaM2007Oncogenic%20HMGA2%3A%20short%20or%20small%3FGenes%20Dev2110051009" data-title="Oncogenic%20HMGA2%3A%20short%20or%20small%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Oncogenic+HMGA2%3A+short+or+small%3F+Young+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0013453-Giraldez1" id="pone.0013453-Giraldez1" class="link-target"></a>Giraldez AJ, Cinalli RM, Glasner ME, Enright AJ, Thomson JM, et al.  (2005) MicroRNAs regulate brain morphogenesis in zebrafish. Science  308: 833–838.  <ul class="reflinks"><li><a href="#" data-author="Giraldez" data-cit="GiraldezAJCinalliRMGlasnerMEEnrightAJThomsonJM2005MicroRNAs%20regulate%20brain%20morphogenesis%20in%20zebrafish.Science308833838" data-title="MicroRNAs%20regulate%20brain%20morphogenesis%20in%20zebrafish." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNAs+regulate+brain+morphogenesis+in+zebrafish.+Giraldez+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0013453-Schratt1" id="pone.0013453-Schratt1" class="link-target"></a>Schratt GM, Tuebing F, Nigh EA, Kane CG, Sabatini ME, et al.  (2006) A brain-specific microRNA regulates dendritic spine development. Nature  439: 283–289.  <ul class="reflinks"><li><a href="#" data-author="Schratt" data-cit="SchrattGMTuebingFNighEAKaneCGSabatiniME2006A%20brain-specific%20microRNA%20regulates%20dendritic%20spine%20development.Nature439283289" data-title="A%20brain-specific%20microRNA%20regulates%20dendritic%20spine%20development." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+brain-specific+microRNA+regulates+dendritic+spine+development.+Schratt+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0013453-Stadler1" id="pone.0013453-Stadler1" class="link-target"></a>Stadler BM, Ruohola-Baker H (2008) Small RNAs: keeping stem cells in line. Cell  132: 563–566.  <ul class="reflinks"><li><a href="#" data-author="Stadler" data-cit="StadlerBMRuohola-BakerH2008Small%20RNAs%3A%20keeping%20stem%20cells%20in%20line.Cell132563566" data-title="Small%20RNAs%3A%20keeping%20stem%20cells%20in%20line." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Small+RNAs%3A+keeping+stem+cells+in+line.+Stadler+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0013453-Harris1" id="pone.0013453-Harris1" class="link-target"></a>Harris KS, Zhang Z, McManus MT, Harfe BD, Sun X (2006) Dicer function is essential for lung epithelium morphogenesis. Proc Natl Acad Sci U S A  103: 2208–2213.  <ul class="reflinks"><li><a href="#" data-author="Harris" data-cit="HarrisKSZhangZMcManusMTHarfeBDSunX2006Dicer%20function%20is%20essential%20for%20lung%20epithelium%20morphogenesis.Proc%20Natl%20Acad%20Sci%20U%20S%20A10322082213" data-title="Dicer%20function%20is%20essential%20for%20lung%20epithelium%20morphogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dicer+function+is+essential+for+lung+epithelium+morphogenesis.+Harris+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0013453-Cobb2" id="pone.0013453-Cobb2" class="link-target"></a>Cobb BS, Hertweck A, Smith J, O'Connor E, Graf D, et al.  (2006) A role for Dicer in immune regulation. J Exp Med  203: 2519–2527.  <ul class="reflinks"><li><a href="#" data-author="Cobb" data-cit="CobbBSHertweckASmithJO%27ConnorEGrafD2006A%20role%20for%20Dicer%20in%20immune%20regulation.J%20Exp%20Med20325192527" data-title="A%20role%20for%20Dicer%20in%20immune%20regulation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+role+for+Dicer+in+immune+regulation.+Cobb+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0013453-Nishino1" id="pone.0013453-Nishino1" class="link-target"></a>Nishino J, Kim I, Chada K, Morrison SJ (2008) Hmga2 promotes neural stem cell self-renewal in young but not old mice by reducing p16Ink4a and p19Arf Expression. Cell  135: 227–239.  <ul class="reflinks"><li><a href="#" data-author="Nishino" data-cit="NishinoJKimIChadaKMorrisonSJ2008Hmga2%20promotes%20neural%20stem%20cell%20self-renewal%20in%20young%20but%20not%20old%20mice%20by%20reducing%20p16Ink4a%20and%20p19Arf%20Expression.Cell135227239" data-title="Hmga2%20promotes%20neural%20stem%20cell%20self-renewal%20in%20young%20but%20not%20old%20mice%20by%20reducing%20p16Ink4a%20and%20p19Arf%20Expression." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hmga2+promotes+neural+stem+cell+self-renewal+in+young+but+not+old+mice+by+reducing+p16Ink4a+and+p19Arf+Expression.+Nishino+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0013453-Lu1" id="pone.0013453-Lu1" class="link-target"></a>Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, et al.  (2005) MicroRNA expression profiles classify human cancers. Nature  435: 834–838.  <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuJGetzGMiskaEAAlvarez-SaavedraELambJ2005MicroRNA%20expression%20profiles%20classify%20human%20cancers.Nature435834838" data-title="MicroRNA%20expression%20profiles%20classify%20human%20cancers." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=MicroRNA+expression+profiles+classify+human+cancers.+Lu+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0013453-Kumar1" id="pone.0013453-Kumar1" class="link-target"></a>Kumar MS, Lu J, Mercer KL, Golub TR, Jacks T (2007) Impaired microRNA processing enhances cellular transformation and tumorigenesis. Nat Genet  39: 673–677.  <ul class="reflinks"><li><a href="#" data-author="Kumar" data-cit="KumarMSLuJMercerKLGolubTRJacksT2007Impaired%20microRNA%20processing%20enhances%20cellular%20transformation%20and%20tumorigenesis.Nat%20Genet39673677" data-title="Impaired%20microRNA%20processing%20enhances%20cellular%20transformation%20and%20tumorigenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Impaired+microRNA+processing+enhances+cellular+transformation+and+tumorigenesis.+Kumar+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0013453-Hanahan1" id="pone.0013453-Hanahan1" class="link-target"></a>Hanahan D, Weinberg RA (2000) The hallmarks of cancer. Cell  100: 57–70.  <ul class="reflinks"><li><a href="#" data-author="Hanahan" data-cit="HanahanDWeinbergRA2000The%20hallmarks%20of%20cancer.Cell1005770" data-title="The%20hallmarks%20of%20cancer." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+hallmarks+of+cancer.+Hanahan+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0013453-Gleeson1" id="pone.0013453-Gleeson1" class="link-target"></a>Gleeson JG, Allen KM, Fox JW, Lamperti ED, Berkovic S, et al.  (1998) Doublecortin, a brain-specific gene mutated in human X-linked lissencephaly and double cortex syndrome, encodes a putative signaling protein. Cell  92: 63–72.  <ul class="reflinks"><li><a href="#" data-author="Gleeson" data-cit="GleesonJGAllenKMFoxJWLampertiEDBerkovicS1998Doublecortin%2C%20a%20brain-specific%20gene%20mutated%20in%20human%20X-linked%20lissencephaly%20and%20double%20cortex%20syndrome%2C%20encodes%20a%20putative%20signaling%20protein.Cell926372" data-title="Doublecortin%2C%20a%20brain-specific%20gene%20mutated%20in%20human%20X-linked%20lissencephaly%20and%20double%20cortex%20syndrome%2C%20encodes%20a%20putative%20signaling%20protein." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Doublecortin%2C+a+brain-specific+gene+mutated+in+human+X-linked+lissencephaly+and+double+cortex+syndrome%2C+encodes+a+putative+signaling+protein.+Gleeson+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0013453-Pollard1" id="pone.0013453-Pollard1" class="link-target"></a>Pollard SM, Yoshikawa K, Clarke ID, Danovi D, Stricker S, et al.  (2009) Glioma stem cell lines expanded in adherent culture have tumor-specific phenotypes and are suitable for chemical and genetic screens. Cell Stem Cell  4: 568–580.  <ul class="reflinks"><li><a href="#" data-author="Pollard" data-cit="PollardSMYoshikawaKClarkeIDDanoviDStrickerS2009Glioma%20stem%20cell%20lines%20expanded%20in%20adherent%20culture%20have%20tumor-specific%20phenotypes%20and%20are%20suitable%20for%20chemical%20and%20genetic%20screens.Cell%20Stem%20Cell4568580" data-title="Glioma%20stem%20cell%20lines%20expanded%20in%20adherent%20culture%20have%20tumor-specific%20phenotypes%20and%20are%20suitable%20for%20chemical%20and%20genetic%20screens." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Glioma+stem+cell+lines+expanded+in+adherent+culture+have+tumor-specific+phenotypes+and+are+suitable+for+chemical+and+genetic+screens.+Pollard+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pone.0013453-Kumar2" id="pone.0013453-Kumar2" class="link-target"></a>Kumar MS, Pester RE, Chen CY, Lane K, Chin C, et al.  (2009) Dicer1 functions as a haploinsufficient tumor suppressor. Genes Dev  23: 2700–2704.  <ul class="reflinks"><li><a href="#" data-author="Kumar" data-cit="KumarMSPesterREChenCYLaneKChinC2009Dicer1%20functions%20as%20a%20haploinsufficient%20tumor%20suppressor.Genes%20Dev2327002704" data-title="Dicer1%20functions%20as%20a%20haploinsufficient%20tumor%20suppressor." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dicer1+functions+as+a+haploinsufficient+tumor+suppressor.+Kumar+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

