



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>During meiotic prophase, telomeres cluster, forming the bouquet chromosome arrangement, and facilitate homologous chromosome pairing. In fission yeast, bouquet formation requires switching of telomere and centromere positions. Centromeres are located at the spindle pole body (SPB) during mitotic interphase, and upon entering meiosis, telomeres cluster at the SPB, followed by centromere detachment from the SPB. Telomere clustering depends on the formation of the microtubule-organizing center at telomeres by the linker of nucleoskeleton and cytoskeleton complex (LINC), while centromere detachment depends on disassembly of kinetochores, which induces meiotic centromere formation. However, how the switching of telomere and centromere positions occurs during bouquet formation is not fully understood. Here, we show that, when impaired telomere interaction with the LINC or microtubule disruption inhibited telomere clustering, kinetochore disassembly-dependent centromere detachment and accompanying meiotic centromere formation were also inhibited. Efficient centromere detachment required telomere clustering-dependent SPB recruitment of a conserved telomere component, Taz1, and microtubules. Furthermore, when artificial SPB recruitment of Taz1 induced centromere detachment in telomere clustering-defective cells, spindle formation was impaired. Thus, detachment of centromeres from the SPB without telomere clustering causes spindle impairment. These findings establish novel regulatory mechanisms, which prevent concurrent detachment of telomeres and centromeres from the SPB during bouquet formation and secure proper meiotic divisions.</p>
</div><div class="abstract toc-section"><a id="abstract1" name="abstract1" class="link-target" data-toc="abstract1" title="Author Summary"></a>
<h2>Author Summary</h2>
<a id="article1.front1.article-meta1.abstract2.p1" name="article1.front1.article-meta1.abstract2.p1" class="link-target"></a><p>Meiosis is a type of cell division, that generates haploid gametes and is essential for sexual reproduction. During meiosis, telomeres cluster on a small region of the nuclear periphery, forming a conserved chromosome arrangement referred to as the “bouquet”. Because the bouquet arrangement facilitates homologous chromosome pairing, which is essential for proper meiotic chromosome segregation, it is of great importance to understand how the bouquet arrangement is formed. In fission yeast, the bouquet arrangement requires switching of telomere and centromere positions. During mitosis, centromeres are located at the fungal centrosome called the spindle pole body (SPB). Upon entering meiosis, telomeres cluster at the SPB, and centromeres become detached from the SPB, forming the bouquet arrangement. In this study, we show that centromere detachment is linked with telomere clustering. When telomere clustering was inhibited, centromere detachment was also inhibited. This regulatory relationship depended on a conserved telomere component, Taz1, and microtubules. Furthermore, we show that the regulatory relationship is crucial for proper meiotic divisions when telomere clustering is defective. Our findings reveal a hitherto unknown regulatory relationship between meiotic telomere and centromere positions in bouquet formation, which secures proper meiotic divisions.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Katsumata K, Hirayasu A, Miyoshi J, Nishi E, Ichikawa K, Tateho K, et al.  (2016) A Taz1- and Microtubule-Dependent Regulatory Relationship between Telomere and Centromere Positions in Bouquet Formation Secures Proper Meiotic Divisions. PLoS Genet 12(9):
           e1006304.
        
        https://doi.org/10.1371/journal.pgen.1006304</p><p><strong>Editor: </strong>R. Scott Hawley, Stowers Institute for Medical Research, UNITED STATES</p><p><strong>Received: </strong>April 13, 2016; <strong>Accepted: </strong>August 17, 2016; <strong>Published: </strong> September 9, 2016</p><p><strong>Copyright: </strong> © 2016 Katsumata et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>All relevant data are within the paper and its Supporting Information files.</p><p><strong>Funding: </strong>This work was supported by JSPS KAKENHI Grant Number 16K07248 (Grant-in-Aid for Scientific Research C), and performed under the Cooperative Research Program of Institute for Protein Research, Osaka University, CR-13/14/15/16-03. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Chromosome positioning changes dynamically during development and differentiation, and contributes to various chromosomal events including gene expression and DNA metabolism [<a href="#pgen.1006304.ref001" class="ref-tip">1</a>–<a href="#pgen.1006304.ref005" class="ref-tip">5</a>]. Especially during meiosis, chromosomes adopt a characteristic position called the “bouquet” arrangement, in which telomeres cluster at the nuclear periphery. The bouquet arrangement is highly conserved among eukaryotes [<a href="#pgen.1006304.ref006" class="ref-tip">6</a>, <a href="#pgen.1006304.ref007" class="ref-tip">7</a>], and how it is formed and what functions it has are important questions in the field of meiosis.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Studies of various organisms show that the bouquet arrangement facilitates homologous chromosome pairing [<a href="#pgen.1006304.ref007" class="ref-tip">7</a>–<a href="#pgen.1006304.ref009" class="ref-tip">9</a>]. Bouquet-defective mutants of yeasts and mammals exhibit impaired homologous chromosome pairing and phenotypes associated with the impaired pairing, such as increased non-homologous association, decreased recombination, and defective formation of the synaptonemal complex, a structure that bridges the paired homologous chromosomes [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>–<a href="#pgen.1006304.ref023" class="ref-tip">23</a>]. In <em>Caenorhabditis elegans</em>, special chromosome regions called “pairing centers” cluster instead of telomeres, and impaired clustering of pairing centers causes similar defects [<a href="#pgen.1006304.ref024" class="ref-tip">24</a>–<a href="#pgen.1006304.ref029" class="ref-tip">29</a>]. Homologous chromosome pairing is essential for formation of chiasmata, which physically link homologous chromosomes and enable their segregation at meiosis I (reductional segregation). Consistently, bouquet-defective organisms also exhibit impaired chromosome segregation [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>, <a href="#pgen.1006304.ref014" class="ref-tip">14</a>, <a href="#pgen.1006304.ref030" class="ref-tip">30</a>].</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Among different organisms, the fission yeast <em>Schizosaccharomyces pombe</em> shows the most prominent example of the bouquet arrangement. <em>S</em>. <em>pombe</em> mitotic chromosomes are positioned with their centromeres clustered at the spindle pole body (SPB; a centrosome equivalent in fungi) and their telomeres located away from it (this corresponds to the “Rabl” configuration seen in other organisms) [<a href="#pgen.1006304.ref031" class="ref-tip">31</a>]. Under nitrogen-starved conditions, <em>S</em>. <em>pombe</em> cells enter meiosis through cell conjugation. Around this period, telomeres cluster at the SPB and centromeres become detached from it, forming the bouquet arrangement (<a href="#pgen-1006304-g001">Fig 1A</a>) [<a href="#pgen.1006304.ref032" class="ref-tip">32</a>]. When the bouquet arrangement is formed, the SPB oscillates between the cell ends with the clustered telomeres, generating so-called “horsetail” nuclear movements (<a href="#pgen-1006304-g001">Fig 1A</a>, Horsetail stage). The SPB-led telomere movements promote pairing of homologous chromosomes by inducing their alignment and contact; impairment of either telomere clustering or horsetail movements compromises homologous chromosome pairing [<a href="#pgen.1006304.ref008" class="ref-tip">8</a>, <a href="#pgen.1006304.ref033" class="ref-tip">33</a>].</p>
<a class="link-target" id="pgen-1006304-g001" name="pgen-1006304-g001"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g001" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Changes in chromosome positioning during meiosis and LINC-dependent telocentrosome formation.</div><p class="caption_target"><a id="article1.body1.sec1.fig1.caption1.p1" name="article1.body1.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) Changes in chromosome positioning during meiosis. During mitotic interphase, centromeres (red circles) are located at the SPB, while telomeres (blue circles) are located away from it. When cells enter meiosis by cell conjugation, telomeres cluster at the SPB, which radiates microtubules, and centromeres become detached from it (Karyogamy). After nuclear fusion, the diploid nucleus becomes elongated and oscillates between the cell ends, led by the SPB (Horsetail stage). Thereafter, the nucleus remains in the middle of the cell with several microtubules extending in parallel to the long cell axis until meiotic divisions start (Post-horsetail stage). (B) Factors required for telomere-LINC interaction and telocentrosome formation. γ-TuC: γ-tubulin complex; NE: nuclear envelope.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g001">
              https://doi.org/10.1371/journal.pgen.1006304.g001</a></p></div><a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Studies of <em>S</em>. <em>pombe</em> have unveiled additional functions of the bouquet arrangement. Telomere clustering additionally contributes to spindle formation, and defective telomere clustering causes impairment of spindle formation [<a href="#pgen.1006304.ref034" class="ref-tip">34</a>, <a href="#pgen.1006304.ref035" class="ref-tip">35</a>]. Furthermore, centromere detachment from the SPB induces the formation of meiosis-specific centromeres. During homologous chromosome segregation, homologous chromosomes attach to opposite SPBs (bipolar attachment) with sister chromatids attaching to the same SPB (monopolar attachment) (<a href="#pgen.1006304.s001">S1A Fig</a>) [<a href="#pgen.1006304.ref036" class="ref-tip">36</a>, <a href="#pgen.1006304.ref037" class="ref-tip">37</a>]. At this period, the kinetochores on the sister centromeres face the same side (kinetochore mono-orientation), facilitating monopolar attachment of sister chromatids, while centromere cohesion persists, preventing sister chromatid separation upon their bipolar attachment (<a href="#pgen.1006304.s001">S1B Fig</a>) [<a href="#pgen.1006304.ref038" class="ref-tip">38</a>]. Persistent centromere cohesion also enables sister chromatid segregation (equational segregation) at meiosis II. Without centromere detachment from the SPB, meiotic centromere properties are not properly established and sister chromatids frequently undergo equational segregation at meiosis I [<a href="#pgen.1006304.ref039" class="ref-tip">39</a>, <a href="#pgen.1006304.ref040" class="ref-tip">40</a>]. Centromere detachment also induces meiotic centromere formation in the budding yeast <em>Saccharomyces cerevisiae</em> [<a href="#pgen.1006304.ref041" class="ref-tip">41</a>].</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5" class="link-target"></a><p>The linker of nucleoskeleton and cytoskeleton (LINC) complex formed by the conserved SUN (Sad1/Unc-84) and KASH (Klarsicht/ANC-1/Syne homology) domain nuclear membrane proteins recently emerged as a key player in SPB association of telomeres and centromeres [<a href="#pgen.1006304.ref008" class="ref-tip">8</a>, <a href="#pgen.1006304.ref033" class="ref-tip">33</a>, <a href="#pgen.1006304.ref042" class="ref-tip">42</a>–<a href="#pgen.1006304.ref044" class="ref-tip">44</a>]. In <em>S</em>. <em>pombe</em>, the LINC complex consisting of Sad1 (SUN) and Kms1/Kms2 (KASH) is localized at the SPB. Upon entering meiosis, meiosis-specific Bqt1 and Bqt2 recruit the LINC complex to telomeres, resulting in localization of Sad1 and Kms1/Kms2 at telomeres in addition to the SPB [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>, <a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. The telomere-recruited LINC complexes form the microtubule-organizing center termed the “telocentrosome” (<a href="#pgen-1006304-g001">Fig 1B</a>). Subsequently, microtubule motors gather telomeres at the SPB by moving on SPB- and telocentrosome-nucleated microtubules (<a href="#pgen.1006304.s001">S1C Fig</a>) [<a href="#pgen.1006304.ref033" class="ref-tip">33</a>, <a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. SPB localization of centromeres also depends on the LINC complex; the Csi1-dependent interaction of the outer kinetochore components with Sad1 causes tethering of mitotic centromeres to the SPB [<a href="#pgen.1006304.ref044" class="ref-tip">44</a>]. When the bouquet is formed, the outer kinetochore components delocalize from the centromere, and this kinetochore disassembly causes centromere detachment from the SPB [<a href="#pgen.1006304.ref039" class="ref-tip">39</a>, <a href="#pgen.1006304.ref046" class="ref-tip">46</a>].</p>
<a id="article1.body1.sec1.p6" name="article1.body1.sec1.p6" class="link-target"></a><p>In <em>S</em>. <em>pombe</em>, centromere detachment immediately follows telomere clustering [<a href="#pgen.1006304.ref047" class="ref-tip">47</a>, <a href="#pgen.1006304.ref048" class="ref-tip">48</a>], and both events require mating pheromone-dependent activation of MAP kinase [<a href="#pgen.1006304.ref039" class="ref-tip">39</a>, <a href="#pgen.1006304.ref049" class="ref-tip">49</a>]. Together with the fact that SPB localization of telomeres and centromeres depends on the LINC complex, these facts suggest that regulation of centromere detachment and telomere clustering is related. However, their regulatory relationship remains unclear.</p>
<a id="article1.body1.sec1.p7" name="article1.body1.sec1.p7" class="link-target"></a><p>Here, we show that, when telomere clustering is inhibited, centromere detachment is also inhibited. Efficient centromere detachment requires SPB recruitment of the telomere component Taz1 by telomere clustering and microtubules that drive telomere clustering. We provide evidence indicating that this regulation secures proper spindle formation when telomere clustering is defective. To our knowledge, this is the first example of a regulatory relationship between telomere and centromere positions, which is critical for proper execution of meiotic divisions.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Centromere detachment from the SPB is inhibited in telomere clustering-defective mutants"></a>
<h3>Centromere detachment from the SPB is inhibited in telomere clustering-defective mutants</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p><em>S</em>. <em>pombe</em> cells normally enter meiosis through conjugation of two haploid cells. The bouquet arrangement is formed around the period of cell conjugation [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>, <a href="#pgen.1006304.ref047" class="ref-tip">47</a>], and persists until meiotic divisions start (<a href="#pgen-1006304-g001">Fig 1A</a>) [<a href="#pgen.1006304.ref032" class="ref-tip">32</a>]. To explore the relationship between telomere clustering and centromere detachment in bouquet formation, we examined telomere and centromere positioning during karyogamy and the horsetail stage in telomere clustering-defective cells.</p>
<a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>We first examined telomere positioning by visualizing the telomere-proximal <em>sod2</em> locus on chromosome I using the lacI/lacO recognition system [<a href="#pgen.1006304.ref050" class="ref-tip">50</a>] (<a href="#pgen-1006304-g002">Fig 2A</a>, Mitotic interphase). In wild-type cells, during mitotic interphase, <em>sod2</em> signals were located away from the SPB, which was visualized by an mCherry-tagged, conserved SPB component, Sfi1 [<a href="#pgen.1006304.ref051" class="ref-tip">51</a>]. By contrast, they were mostly juxtaposed with the SPB during karyogamy and the meiotic mononuclear stage (including the horsetail and post-horsetail stages; see <a href="#pgen-1006304-g001">Fig 1A</a>), confirming the occurrence of telomere clustering (<a href="#pgen-1006304-g002">Fig 2A and 2B</a>, WT). Telomere clustering depends on telomere-LINC interaction, and a Sad1 interactor, Bqt1, and a telomere-Bqt1 connector, Rap1, are essential for this interaction (<a href="#pgen-1006304-g001">Fig 1B</a>) [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>, <a href="#pgen.1006304.ref011" class="ref-tip">11</a>, <a href="#pgen.1006304.ref052" class="ref-tip">52</a>]. Consistently, loss of either Bqt1 (<em>bqt1Δ</em>) or Rap1 (<em>rap1Δ</em>) caused almost complete elimination of <em>sod2</em>-SPB juxtaposition (<a href="#pgen-1006304-g002">Fig 2A and 2B</a>). Furthermore, Rap1 interacts with telomeres via two different telomere-binding proteins, Taz1 and Pot1 (<a href="#pgen-1006304-g001">Fig 1B</a>) [<a href="#pgen.1006304.ref011" class="ref-tip">11</a>, <a href="#pgen.1006304.ref052" class="ref-tip">52</a>–<a href="#pgen.1006304.ref054" class="ref-tip">54</a>]. Loss of either Taz1 (<em>taz1Δ</em>) or a Rap1-Pot1 connector, Poz1 (<em>poz1Δ</em>), probably partially compromises telomere-LINC interaction, and loss of both eliminates the interaction completely. Accordingly, introduction of <em>taz1Δ</em> or <em>poz1Δ</em> mutation only reduced or rarely affected <em>sod2</em>-SPB juxtaposition, whereas introduction of both mutations almost completely eliminated their juxtaposition (<a href="#pgen-1006304-g002">Fig 2B</a>, Karyogamy). Thus, telomere-clustering defects are correlated with telomere-LINC interaction defects. Telomere clustering also depends on telocentrosome formation, which requires a γ-tubulin complex(γ-TuC)-recruiting factor, Mto1 (<a href="#pgen-1006304-g001">Fig 1B</a>) [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>], and <em>sod2</em>-SPB juxtaposition was reduced in <em>mto1Δ</em> cells (<a href="#pgen-1006304-g002">Fig 2A and 2B</a>).</p>
<a class="link-target" id="pgen-1006304-g002" name="pgen-1006304-g002"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g002" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Telomere and centromere positioning in telomere clustering-defective cells.</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) Localization patterns of the telomere-adjacent <em>sod2</em> locus. The karyogamy stage was judged by two nuclei with teardrop or deformed shapes and/or with a single SPB (Karyogamy). The mononuclear stage, including both horsetail and post-horsetail stages, was judged by a single nucleus with a non-round, deformed shape (Mononuc). (B) Observation frequencies of different <em>sod2</em> localization patterns. Numbers in parentheses indicate the number of examined cells. (C) Centromere localization during karyogamy and the horsetail stage. Magenta and green arrowheads indicate the SPBs and SPB-co-localized centromeres, respectively. Insets are enlarged images of SPB-associated centromere signals (white bar: 0.5 μm). The karyogamy stage was judged as in (A). The horsetail stage was judged by a single nucleus with an astral microtubule array (Horsetail). (D) Population of cells containing SPB-associated centromeres. Averages of three independent experiments are shown. Error bars indicate standard deviation. More than 30 and 60 cells were examined in each experiment for karyogamy and the horsetail stage, respectively. Lines indicate sets of data that were statistically compared. *p<em>&lt;</em>0.05; **p<em>&lt;</em>0.01; ***p<em>&lt;</em>0.005; ****p&lt;0.001 (by the Student’s t-test); ns: no significant difference.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g002">
              https://doi.org/10.1371/journal.pgen.1006304.g002</a></p></div><a id="article1.body1.sec2.sec1.p3" name="article1.body1.sec2.sec1.p3" class="link-target"></a><p>We next examined centromere positioning by visualizing the centromere-specific histone H3 variant Cnp1 [<a href="#pgen.1006304.ref055" class="ref-tip">55</a>]. In wild-type cells, all centromeres were co-localized with the SPB during mitosis (<a href="#pgen-1006304-g002">Fig 2C</a>, Mitotic interphase), but they were mostly located away from it during karyogamy and the horsetail stage (judged by a single nucleus with an astral microtubule array; see <a href="#pgen-1006304-g001">Fig 1A</a>) (<a href="#pgen-1006304-g002">Fig 2C and 2D</a>, WT). By striking contrast, in all telomere clustering-defective cells, centromeres were often co-localized with the SPB (<a href="#pgen-1006304-g002">Fig 2C and 2D</a>; <a href="#pgen.1006304.s002">S2 Fig</a>). Therefore, centromere detachment from the SPB is commonly inhibited in telomere clustering-defective cells. However, in <em>taz1Δ</em> and <em>mto1Δ</em> cells, although there was a considerable level of telomere-SPB association (<a href="#pgen-1006304-g002">Fig 2B</a>), centromere detachment was inhibited at similar levels to those seen in <em>bqt1Δ</em> and <em>rap1Δ</em> cells during karyogamy (<a href="#pgen-1006304-g002">Fig 2D</a>, Karyogamy; no statistically significant differences among the mutants), in which telomere-SPB association was almost completely lost (<a href="#pgen-1006304-g002">Fig 2B</a>). This indicates that inhibition of centromere detachment is not correlated with loss of telomere-SPB association.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Kinetochore disassembly is inhibited in telomere clustering-defective mutants"></a>
<h3>Kinetochore disassembly is inhibited in telomere clustering-defective mutants</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>Because centromere detachment is induced by outer kinetochore disassembly, we next examined the localization of Nuf2 and Mis12, which are components of the Ndc80 and Mis12 outer kinetochore complexes, respectively. The outer kinetochore components delocalize from the centromere at the time of centromere detachment, and remain delocalized until kinetochore reformation occurs around the end of the horsetail stage (<a href="#pgen-1006304-g003">Fig 3A</a>) [<a href="#pgen.1006304.ref046" class="ref-tip">46</a>]. In wild-type cells, Nuf2 was observed at the centromere-localized SPB during mitotic interphase, and was mostly absent during karyogamy and the horsetail stage (<a href="#pgen-1006304-g003">Fig 3B and 3C</a>, WT). Mis12 showed similar localization patterns, but with higher observation frequencies (<a href="#pgen.1006304.s003">S3A and S3B Fig</a>, WT). The higher observation frequencies probably mean that Mis12 complexes delocalize from the centromere later and re-localize at the centromere earlier than Ndc80 complexes.</p>
<a class="link-target" id="pgen-1006304-g003" name="pgen-1006304-g003"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g003" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Localization of Nuf2 in telomere clustering-defective cells.</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>(A) Kinetochore disassembly and meiosis progression. Green dots indicate intact kinetochores. The intact kinetochores are absent during almost the entire period of karyogamy and the horsetail stage (Kinetochore disassembly stage). (B) Nuf2 localization. The karyogamy and horsetail stages were judged, as in <a href="#pgen-1006304-g002">Fig 2C</a>. Arrowheads in the horsetail stage show SPB positions. (C) Population of cells containing Nuf2 signals. Karyogamy: karyogamy stage; Horsetail: the horsetail stage; Horsetail with Mrc1: the horsetail stage with nuclear Mrc1 signals. Numbers in parentheses show the number of examined cells. Bars show averages of three independent experiments except for those in “Horsetail with Mrc1”. More than 30 and 60 cells were examined in each experiment for karyogamy and the horsetail stage, respectively. Error bars indicate standard deviation. Lines indicate sets of data that were statistically compared. *p<em>&lt;</em>0.05; **p<em>&lt;</em>0.005; ***p<em>&lt;</em>0.0005; ns: no significant difference (by the Student’s t-test); nd: not detected. (D) Population of cells with different Nuf2 localization. At SPB: one SPB-associated signal; Off/at SPB: SPB-associated and SPB-dissociated signals; Off SPB: SPB-dissociated signals. Images in the box show the representative localization of the SPB and signals. (E) A Nuf2 signal dissociated from the centromere in the <em>taz1Δ</em> mutant (arrowhead). The percentage indicates the observation frequency of centromere-dissociated Nuf2 signals.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g003">
              https://doi.org/10.1371/journal.pgen.1006304.g003</a></p></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>By contrast, in all telomere clustering-defective mutants, Nuf2 and Mis12 signals were frequently observed (<a href="#pgen-1006304-g003">Fig 3B–3D</a>; <a href="#pgen.1006304.s003">S3 Fig</a>). The higher observation frequencies in the horsetail stage were not due to enrichment of cells in the late, kinetochore-reformation stage (<a href="#pgen-1006304-g003">Fig 3A</a>) because, when we specified early horsetail-stage cells by nuclear signals of RFP-tagged Mrc1, a DNA replication regulator that was observed only in the early meiotic stage (<a href="#pgen.1006304.s004">S4A Fig</a>), Nuf2 was still observed at higher frequencies (<a href="#pgen-1006304-g003">Fig 3C</a>, Horsetail with Mrc1). In addition, the higher observation frequencies were not caused by centromere clustering-dependent increases in signal intensities because the number of centromere signals significantly increased during the horsetail stage in the <em>bqt1Δ</em> mutant [average number of Cnp1 signals was 3.4 ± 1.1 (n = 162) for wild type and 3.9 ± 1.2 (n = 225) for the <em>bqt1Δ</em> mutant during the horsetail stage; p&lt;0.001 by the Student’s t-test]. This indicates that delocalization of the outer kinetochore components is inhibited in the mutants. The retained signals were localized either at the SPB or at the SPB-detached centromeres (<a href="#pgen-1006304-g003">Fig 3B and 3D</a>; <a href="#pgen.1006304.s003">S3A, S3C</a> and <a href="#pgen.1006304.s004">S4B</a> Figs) [centromere and/or SPB localization of Nuf2 signals was confirmed by co-localization of the Nuf2 signals with either SPB and/or centromere signals in cells in which both the SPB and centromeres were visualized (n = 73)]. Furthermore, the signals were sometimes co-localized with the centromere-free SPB (<a href="#pgen-1006304-g003">Fig 3E</a>, arrowhead). This indicates that, in addition to the dissociation of kinetochore complexes from the centromere, that from the SPB is also inhibited. We also noticed that localization patterns of the retained signals in the <em>mto1Δ</em> mutant were somewhat different from those in other mutants; the signals tended to be localized at SPB-detached centromeres in <em>mto1Δ</em> cells, but at the SPB in other mutant cells (<a href="#pgen-1006304-g003">Fig 3D</a>; <a href="#pgen.1006304.s003">S3C</a> and <a href="#pgen.1006304.s004">S4B</a> Figs).</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Microtubule disruption inhibits centromere detachment from the SPB"></a>
<h3>Microtubule disruption inhibits centromere detachment from the SPB</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Because telomere clustering depends on microtubules [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>], we next examined if microtubule disruption inhibits centromere detachment using a microtubule-depolymerizing drug, methyl 2-benzimidazole carbamate (MBC). To disrupt microtubules before telomere clustering, we synchronously induced meiosis using haploid cells bearing both mating-type genes, as reported previously [<a href="#pgen.1006304.ref040" class="ref-tip">40</a>, <a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. These haploid cells underwent meiosis fairly synchronously after a single mitotic division (shown by a transient increase in binuclear cells) in nitrogen-depleted conditions, forming more than two nuclei (<a href="#pgen.1006304.s005">S5A–S5C Fig</a>). We treated the cells with MBC or its solvent DMSO after mitotic division.</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2" class="link-target"></a><p>In the DMSO-treated control experiment, as the percentage of cells with SPB-associated telomeres increased (<a href="#pgen-1006304-g004">Fig 4A</a>, WT+DMSO), the percentage of those with SPB-associated centromeres (<a href="#pgen-1006304-g004">Fig 4B and 4C</a>, WT+DMSO) or Nuf2 signals (<a href="#pgen-1006304-g004">Fig 4D and 4E</a>, WT+DMSO) conversely decreased. This confirms the occurrence of centromere detachment and kinetochore disassembly in addition to telomere clustering. It should also be noted that loss of Bqt1 or Mto1 caused inhibition of telomere clustering, centromere detachment, and kinetochore disassembly in haploid meiotic cells, as shown by the decrease in cells with SPB-associated telomeres (<a href="#pgen-1006304-g004">Fig 4A</a>, <em>bqt1Δ</em> and <em>mto1Δ</em>) and the increase in those with SPB-associated centromeres (<a href="#pgen-1006304-g004">Fig 4B and 4C</a>, <em>bqt1Δ</em> and <em>mto1Δ</em>) or Nuf2 signals (<a href="#pgen-1006304-g004">Fig 4D and 4E</a>, <em>bqt1Δ</em> and <em>mto1Δ</em>). Thus, haploid meiotic cells mirror diploid meiotic zygotes.</p>
<a class="link-target" id="pgen-1006304-g004" name="pgen-1006304-g004"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g004" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Effects of MBC on centromere detachment and kinetochore disassembly.</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>(A and B) Changes in the population of haploid cells with SPB-associated telomeres (A) or centromeres (B) during meiosis progression. SPB association of a portion of (Partial) or all (Full) Taz1/Cnp1 signals is shown by red or blue, respectively. (C) Centromere positioning in haploid cells. (D) Changes in the population of haploid cells containing Nuf2 signals during meiosis progression. Nuf2 localization is shown as in <a href="#pgen-1006304-g003">Fig 3D</a>. (E) Nuf2 localization in haploid cells. In (C) and (E), images show cells incubated in nitrogen-free medium for 5 h. White lines show cell outlines, and arrowheads show Cnp1 or Nuf2 signals co-localized with SPB signals. Bars: 5 μm. WT+DMSO: wild-type haploid cells treated with DMSO; WT+MBC: wild-type haploid cells treated with MBC.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g004">
              https://doi.org/10.1371/journal.pgen.1006304.g004</a></p></div><a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3" class="link-target"></a><p>MBC treatment inhibited telomere clustering without inhibiting meiosis progression (<a href="#pgen.1006304.s005">S5A–S5C Fig</a>), as shown by a significant decrease in the percentage of cells with SPB-associated telomeres (<a href="#pgen-1006304-g004">Fig 4A</a>, WT+MBC). Notably, MBC also inhibited centromere detachment and kinetochore disassembly, as shown by a significant increase in the percentage of cells with SPB-associated centromeres (<a href="#pgen-1006304-g004">Fig 4B and 4C</a>, WT+MBC) or Nuf2 signals (<a href="#pgen-1006304-g004">Fig 4D and 4E</a>, WT+MBC). The retained Nuf2 signals were often located away from the SPB (<a href="#pgen-1006304-g004">Fig 4D and 4E</a>, WT+MBC). This localization pattern was similar to that seen in <em>mto1Δ</em> cells but different from that seen in <em>bqt1Δ</em> cells, where the signals were mainly at the SPB (<a href="#pgen-1006304-g004">Fig 4D and 4E</a>). These observations suggest that the effects of MBC on kinetochore disassembly are similar to those of <em>mto1Δ</em> mutation but different from those of <em>bqt1Δ</em> mutation. Furthermore, although the effects are similar, those of MBC are perhaps stronger than the <em>mto1Δ</em> mutation because the percentage of Nuf2-retaining cells was larger in MBC-treated cells than in <em>mto1Δ</em> cells (<a href="#pgen-1006304-g004">Fig 4D</a>, WT+MBC and <em>mto1Δ</em>).</p>
</div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Nuf2 delocalization is delayed and its re-localization is advanced in telomere clustering-defective cells"></a>
<h3>Nuf2 delocalization is delayed and its re-localization is advanced in telomere clustering-defective cells</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>To understand the effects of impaired telomere clustering on kinetochore disassembly in greater detail, we next examined the localization dynamics of kinetochore components in haploid meiotic cells. In all wild-type cells, Nuf2 signals disappeared transiently (<a href="#pgen-1006304-g005">Fig 5A and 5B</a>, WT; <a href="#pgen.1006304.s009">S1</a> and <a href="#pgen.1006304.s010">S2</a> Movies). By contrast, in <em>bqt1Δ</em> cells, Nuf2 signals sometimes did not disappear (<a href="#pgen-1006304-g005">Fig 5A</a>, <em>bqt1Δ</em>), confirming inhibition of kinetochore disassembly. Furthermore, even when they disappeared, their disappearance was delayed, and their reappearance was conversely advanced (<a href="#pgen-1006304-g005">Fig 5A–5C</a>; <a href="#pgen.1006304.s011">S3</a> and <a href="#pgen.1006304.s012">S4</a> Movies); as a result, the duration of Nuf2 disappearance was shorter (<a href="#pgen-1006304-g005">Fig 5D</a>). These observations confirm the inhibition of kinetochore disassembly in telomere clustering-defective cells.</p>
<a class="link-target" id="pgen-1006304-g005" name="pgen-1006304-g005"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g005" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Live cell analysis of Nuf2 localization in haploid meiotic cells.</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p>(A) Live cell analysis of Nuf2 localization in haploid wild-type and <em>bqt1Δ</em> cells. Cells were induced to enter meiosis by incubation in nitrogen-free medium at 30°C. After incubation for 4 h, they were observed every 10 min at 25°C. Each bar indicates Nuf2 localization in each meiotic cell. Blue and gray bars indicate periods when Nuf2 dots were present and absent, respectively, and red bars indicate the meiotic division stage. (B) Changes in Nuf2 localization in haploid wild-type and <em>bqt1Δ</em> cells during meiosis. Numbers show time from the start of meiosis I, and arrowheads show Nuf2 signals. White bars: 2 μm. (C) Nuf2 disappearance period relative to the start of meiosis I in haploid cells (grey bars). (D) Duration of Nuf2 disappearance in haploid meiosis. Numbers in parentheses and error bars indicate the number of examined cells and standard deviation, respectively.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g005">
              https://doi.org/10.1371/journal.pgen.1006304.g005</a></p></div></div>

<div id="section5" class="section toc-section"><a id="sec007" name="sec007" class="link-target" title="Defective telomere clustering causes equational segregation of sister chromatids at meiosis I"></a>
<h3>Defective telomere clustering causes equational segregation of sister chromatids at meiosis I</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Kinetochore disassembly is thought to be required for establishment of sister kinetochore mono-orientation and protection of centromere cohesion; therefore, we next examined sister chromatid segregation in telomere clustering-defective cells. For this analysis, we used cells that do not form chiasmata because chiasmata promote monopolar attachment of sister chromatids and obscure centromere properties (<a href="#pgen.1006304.s001">S1B Fig</a>) [<a href="#pgen.1006304.ref038" class="ref-tip">38</a>]. If the properties of meiosis-specific centromeres are impaired, sister chromatids should undergo equational segregation in chiasma-lacking cells.</p>
<a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>GFP visualization of centromeres of one of the homologous chromosomes (<a href="#pgen-1006304-g006">Fig 6A</a>) showed that sister chromatids rarely underwent equational segregation in <em>rec12</em> recombination-deficient, chiasma-lacking zygotes (<a href="#pgen-1006304-g006">Fig 6B</a>, +), as reported previously [<a href="#pgen.1006304.ref038" class="ref-tip">38</a>, <a href="#pgen.1006304.ref040" class="ref-tip">40</a>]. By contrast, introduction of <em>bqt1Δ</em> or <em>mto1Δ</em> mutation significantly increased equational segregation in <em>rec12</em> zygotes (<a href="#pgen-1006304-g006">Fig 6B</a>). This indicates that the centromere properties are compromised. In chiasma-lacking cells, sister centromeres frequently attach to both SPBs despite mono-orientation of sister kinetochores, but protected centromere cohesion prevents sister chromatid separation. In these cells, <em>sgo1Δ</em> mutation, which eliminates cohesion protection, causes frequent equational segregation of sister chromatids (<a href="#pgen-1006304-g006">Fig 6C</a>, +) [<a href="#pgen.1006304.ref038" class="ref-tip">38</a>]. Notably, <em>bqt1Δ</em> mutation further increased equational segregation in the <em>rec12Δ sgo1Δ</em> mutant (<a href="#pgen-1006304-g006">Fig 6C</a>, <em>bqt1Δ</em>), similar to <em>moa1Δ</em> mutation (<a href="#pgen.1006304.s006">S6 Fig</a>), which is thought to compromise kinetochore mono-orientation [<a href="#pgen.1006304.ref056" class="ref-tip">56</a>]. This strongly suggests that <em>bqt1Δ</em> mutation compromises kinetochore mono-orientation. <em>bqt1Δ</em> mutation also increased equational segregation in haploid <em>rec12</em><sup><em>+</em></sup> cells (<a href="#pgen-1006304-g006">Fig 6D and 6E</a>), indicating that the increase was not attributed to loss of Rec12 functions. Furthermore, MBC treatment also increased equational segregation (<a href="#pgen-1006304-g006">Fig 6F</a>). All these results support the idea that impairment of telomere clustering causes inhibition of kinetochore disassembly that is required for establishment of the meiosis-specific centromere properties.</p>
<a class="link-target" id="pgen-1006304-g006" name="pgen-1006304-g006"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g006" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 6. </span> Sister chromatid segregation in telomere clustering-defective cells.</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>(A) Chromosome segregation patterns at meiosis I in <em>rec12Δ</em> diploid zygotes. (B and C) The frequencies of equational segregation of sister chromatids at meiosis I in <em>rec12Δ</em> (B) and <em>rec12Δ sgo1Δ</em> (C) diploid zygotes. (D) Chromosome segregation patterns at meiosis I in haploid cells. (E and F) Effects of <em>bqt1Δ</em> mutation (E) or MBC treatment (F) on sister chromatid segregation in haploid meiotic cells. Sister chromatid segregation was analyzed by visualizing the centromere-proximal region of chromosome II [<a href="#pgen.1006304.ref040" class="ref-tip">40</a>]. +: no mutations otherwise depicted; DMSO: treated with DMSO; MBC: treated with MBC. Averages of three independent experiments are shown. Error bars indicate standard deviation. More than 60 cells were examined in each experiment. Lines indicate pairs of data that were statistically compared. *p&lt;0.05; **p&lt;0.005; ns: no significant difference (by the Student’s t-test).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g006">
              https://doi.org/10.1371/journal.pgen.1006304.g006</a></p></div></div>

<div id="section6" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="SPB-recruited Taz1 promotes centromere detachment from the SPB"></a>
<h3>SPB-recruited Taz1 promotes centromere detachment from the SPB</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>Telomere clustering causes recruitment of telomere-LINC connectors to the SPB (<a href="#pgen.1006304.s007">S7A Fig</a>, Wild type). We therefore hypothesized that an SPB-recruited telomere-LINC connector(s) promotes centromere detachment. Among the telomere-LINC connectors, we suspected that Taz1 contributes to centromere detachment, because in the <em>taz1Δ</em> mutant, although telomere-SPB association was substantially retained (<a href="#pgen-1006304-g002">Fig 2B</a>, <em>taz1Δ</em>), the observed inhibition of centromere detachment was not statistically different from that observed in <em>bqt1Δ</em> or <em>rap1Δ</em> cells during karyogamy (Figs <a href="#pgen-1006304-g002">2D</a> and <a href="#pgen-1006304-g003">3C</a>, <a href="#pgen.1006304.s003">S3B Fig</a>), where telomere-SPB association was almost completely eliminated. To test this possibility, we constructed a Taz1 fragment that lacks the telomeric DNA-binding domain (Myb domain) [<a href="#pgen.1006304.ref011" class="ref-tip">11</a>, <a href="#pgen.1006304.ref057" class="ref-tip">57</a>] and fused it with an mCherry fluorescent protein for its visualization and with the nuclear localization sequence for its nuclear localization (Taz1<em>Δ</em>myb; <a href="#pgen-1006304-g007">Fig 7A</a>). In <em>taz1Δ</em> cells, Taz1<em>Δ</em>myb was expected to be localized at the SPB by interacting with SPB-localized Rap1 [<a href="#pgen.1006304.ref011" class="ref-tip">11</a>], but not with telomeres (<a href="#pgen.1006304.s007">S7A Fig</a>, <em>taz1Δ+</em>Taz1<em>Δ</em>myb). If SPB recruitment of Taz1 is important, it should restore centromere detachment in <em>taz1Δ</em> cells.</p>
<a class="link-target" id="pgen-1006304-g007" name="pgen-1006304-g007"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g007" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 7. </span> Effects of a Taz1 fragment on centromere positioning and kinetochore disassembly.</div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1" class="link-target"></a><p>(A) Schematic diagrams of Taz1, Taz1<em>Δ</em>myb, and Taz1<em>Δ</em>myb-Sad1. Black and blue numbers indicate amino acid numbers of Taz1 and Sad1, respectively. TRFH: TRF homology domain; RBM: Rap1-binding motif; DD: dimerization domain; Myb; Myb DNA-binding domain; NLS: nuclear localization sequence; TM: transmembrane domain; SUN: SUN domain [<a href="#pgen.1006304.ref057" class="ref-tip">57</a>] (<a href="http://www.pombase.org/spombe/result/SPBC12D12.01">http://www.pombase.org/spombe/result/SPBC12D12.01</a>). Note that Sad1 lacks its N-terminal 21 amino acids. (B) Intracellular localization of Taz1<em>Δ</em>myb (left) and Taz1<em>Δ</em>myb-Sad1 (right) during karyogamy. Arrowheads show co-localization of SPB and Taz1<em>Δ</em>myb/Taz1<em>Δ</em>myb-Sad1 signals, and arrows show SPB-dissociated Taz1<em>Δ</em>myb-Sad1 signals. (C and D) Population of cells containing SPB-associated centromeres (C) or Nuf2 signals (D) during karyogamy. Averages of three independent experiments are shown. At least 30 cells were examined in each experiment. Error bars: standard deviation. Shaded bars indicate the data adopted from Figs <a href="#pgen-1006304-g002">2D</a> and <a href="#pgen-1006304-g003">3C</a>. Lines indicate sets of data that were statistically compared. *p&lt;0.05, **p<em>&lt;</em>0.005, ***p&lt;0.0005; ns: no significant difference (p&gt;0.05) (Student’s t-test). In all experiments, the fusion gene of Taz1<em>Δ</em>myb or Taz1<em>Δ</em>myb-Sad1 was integrated at the <em>aur1</em><sup><em>+</em></sup> locus on the chromosome and expressed under the <em>taz1</em><sup><em>+</em></sup> promoter.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g007">
              https://doi.org/10.1371/journal.pgen.1006304.g007</a></p></div><a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2" class="link-target"></a><p>In wild-type cells, Taz1<em>Δ</em>myb was localized in the nucleus during karyogamy with a very low or undetectable level of accumulation at telomeres or the SPB (<a href="#pgen-1006304-g007">Fig 7B</a>, WT). Its weak SPB accumulation was probably due to the presence of endogenous intact Taz1. In <em>taz1Δ</em> cells, by contrast, Taz1<em>Δ</em>myb accumulated at the SPB (<a href="#pgen-1006304-g007">Fig 7B</a>, <em>taz1Δ</em>, arrowheads) without restoring telomere-SPB association (<a href="#pgen.1006304.s007">S7B Fig</a>, +Taz1<em>Δ</em>myb, <em>taz1Δ</em>). Importantly, Taz1<em>Δ</em>myb significantly decreased the percentage of cells with SPB-associated centromeres or Nuf2 signals (<a href="#pgen-1006304-g007">Fig 7C and 7D</a>, +Taz1<em>Δ</em>myb, <em>taz1Δ</em>). This result supports the idea that defective SPB recruitment of Taz1 causes inhibition of centromere detachment in <em>taz1Δ</em> cells.</p>
<a id="article1.body1.sec2.sec6.p3" name="article1.body1.sec2.sec6.p3" class="link-target"></a><p>In <em>bqt1Δ</em> and <em>rap1Δ</em> cells, Taz1<em>Δ</em>myb failed to decrease cells with SPB-associated centromeres or Nuf2 signals (<a href="#pgen-1006304-g007">Fig 7C and 7D</a>, +Taz1<em>Δ</em>myb). However, Taz1<em>Δ</em>myb did not accumulate at the SPB because Taz1 cannot directly interact with the LINC complex without Bqt1 or Rap1 [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>] (<a href="#pgen-1006304-g007">Fig 7B</a>, Taz1<em>Δ</em>myb, <em>bqt1Δ</em>), and it remained unclear if defective SPB recruitment of Taz1 is the cause of the inhibition in these cells. To elucidate this point, we artificially tethered Taz1<em>Δ</em>myb to the SPB by fusing it with almost the entire length of Sad1, a LINC component that constitutively localizes at the SPB (Taz1<em>Δ</em>myb-Sad1; <a href="#pgen-1006304-g007">Fig 7A</a>; <a href="#pgen.1006304.s007">S7A Fig</a>, <em>bqt1Δ+</em>Taz1<em>Δ</em>myb-Sad1). Taz1<em>Δ</em>myb-Sad1 did not affect cell growth (judged by cell viability and the doubling time) and centromere-SPB association during mitosis. Importantly, Taz1<em>Δ</em>myb-Sad1 accumulated at the SPB without restoring telomere-SPB association in <em>bqt1Δ</em> and <em>rap1Δ</em> cells, as well as in <em>taz1Δ</em> cells (<a href="#pgen-1006304-g007">Fig 7B</a> and <a href="#pgen.1006304.s007">S7B Fig</a>, <em>+</em>Taz1<em>Δ</em>myb-Sad1), and significantly decreased the percentage of cells with SPB-associated centromeres or Nuf2 signals (<a href="#pgen-1006304-g007">Fig 7C and 7D</a>, +Taz1<em>Δ</em>myb-Sad1). These observations indicate that a lack of Taz1 at the SPB causes inhibition of centromere detachment and kinetochore disassembly in <em>bqt1Δ</em>, <em>rap1Δ</em>, and <em>taz1Δ</em> cells. Taking all these results together, we conclude that SPB-recruited Taz1 promotes centromere detachment from the SPB. It should be noted, however, that in <em>bqt1Δ</em>, <em>rap1Δ</em>, or <em>taz1Δ</em> cells expressing Taz1<em>Δ</em>myb-Sad1, the percentage of cells with SPB-associated centromeres or Nuf2 signals was slightly higher than that in wild-type cells (<a href="#pgen-1006304-g007">Fig 7C and 7D</a>), suggesting that Taz1<em>Δ</em>myb-Sad1 does not completely restore centromere detachment.</p>
</div>

<div id="section7" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="Microtubules promote centromere detachment from the SPB independently of SPB-recruited Taz1"></a>
<h3>Microtubules promote centromere detachment from the SPB independently of SPB-recruited Taz1</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>In <em>mto1Δ</em> or MBC-treated wild-type cells, although Taz1 is often recruited to the SPB by frequent telomere-SPB association (Figs <a href="#pgen-1006304-g002">2B</a> and <a href="#pgen-1006304-g004">4A</a>) [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>], centromere detachment was substantially inhibited (Figs <a href="#pgen-1006304-g002">2D</a> and <a href="#pgen-1006304-g004">4B</a>). In addition, in <em>mto1Δ</em> zygotes, although Taz1<em>Δ</em>myb-Sad1 was localized at the SPB (<a href="#pgen-1006304-g007">Fig 7B</a>, +Taz1<em>Δ</em>myb-Sad1, <em>mto1Δ</em>, arrowheads), it decreased the percentage of cells with SPB-associated centromeres or Nuf2 signals only slightly, and the decrease was not statistically significant (<a href="#pgen-1006304-g007">Fig 7C and 7D</a>, +Taz1<em>Δ</em>myb-Sad1, <em>mto1Δ</em>). Likewise, in <em>mto1Δ</em> or MBC-treated haploid meiotic cells, Taz1<em>Δ</em>myb-Sad1 did not significantly affect centromere and Nuf2 localization (<a href="#pgen-1006304-g008">Fig 8A</a>) as well as meiosis progression (<a href="#pgen.1006304.s005">S5D and S5E Fig</a>, WT+Taz1<em>Δ</em>myb-Sad1 and <em>mto1Δ+</em>Taz1Δmyb-Sad1). These results suggest that microtubules contribute to the regulation of centromere detachment independently of Taz1.</p>
<a class="link-target" id="pgen-1006304-g008" name="pgen-1006304-g008"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g008"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g008" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g008"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g008" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g008">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g008">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g008">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g008">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g008">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g008">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 8. </span> Taz1-independent inhibition of centromere detachment from the SPB and kinetochore disassembly by microtubule disruption.</div><p class="caption_target"><a id="article1.body1.sec2.sec7.fig1.caption1.p1" name="article1.body1.sec2.sec7.fig1.caption1.p1" class="link-target"></a><p>(A) Effects of Taz1Δmyb-Sad1 on centromere (Cnp1) and Nuf2 (Nuf2) localization in <em>mto1Δ</em> (<em>mto1Δ</em>) or MBC-treated wild-type (WT+MBC) cells. Data of <em>mto1Δ</em> or MBC-treated wild-type cells without Taz1Δmyb-Sad1 expression (+None) are adopted from <a href="#pgen-1006304-g004">Fig 4</a>. +None: without Taz1Δmyb-Sad1 expression; +Taz1Δmyb-Sad1: with Taz1Δmyb-Sad1 expression. (B) Effects of MBC on centromere (Cnp1) and Nuf2 (Nuf2) localization in <em>bqt1Δ</em> cells with (<em>bqt1Δ+</em>Taz1Δmyb-Sad1) or without (<em>bqt1Δ</em>) Taz1Δmyb-Sad1 expression. +DMSO: treated with DMSO; +MBC: treated with MBC.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g008">
              https://doi.org/10.1371/journal.pgen.1006304.g008</a></p></div><a id="article1.body1.sec2.sec7.p2" name="article1.body1.sec2.sec7.p2" class="link-target"></a><p>To test this possibility, we treated haploid <em>bqt1Δ</em> cells, which are completely defective in SPB recruitment of Taz1, with MBC. MBC treatment increased the percentage of cells containing SPB-associated centromeres (<a href="#pgen-1006304-g008">Fig 8B</a>, Cnp1, <em>Δbqt1</em>) without inhibiting meiosis progression (<a href="#pgen.1006304.s005">S5D Fig</a>, <em>bqt1Δ</em>). Even with Taz1Δmyb-Sad1 expression, MBC treatment increased the percentage of cells with Nuf2 signals, as well as those with SPB-associated centromeres (<a href="#pgen-1006304-g008">Fig 8B</a>, <em>bqt1Δ</em>+Taz1Δmyb-Sad1). From these results, we conclude that microtubules promote centromere detachment independently of SPB-recruited Taz1. Microtubules drive SPB movements, which cause horsetail nuclear movements. However, centromere detachment was not dependent on SPB movements because centromere detachment was not inhibited in <em>dhc1Δ</em> cells, which are defective in SPB movements [<a href="#pgen.1006304.ref058" class="ref-tip">58</a>] (<a href="#pgen.1006304.s002">S2 Fig</a>).</p>
</div>

<div id="section8" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Induction of centromere detachment causes spindle impairment in telomere clustering-defective cells"></a>
<h3>Induction of centromere detachment causes spindle impairment in telomere clustering-defective cells</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>Our results show the presence of regulatory mechanisms that inhibit centromere detachment from the SPB in the absence of telomere-SPB association. However, the advantages of the mechanisms in meiosis are unclear. It was recently reported that defective telomere clustering causes impairment of spindle formation with detachment of the SPB from the nuclear membrane, and that centromere-SPB association restores spindle formation in the absence of telomere clustering [<a href="#pgen.1006304.ref034" class="ref-tip">34</a>, <a href="#pgen.1006304.ref035" class="ref-tip">35</a>]. These facts suggest that meiotic spindle formation requires SPB association of either telomeres or centromeres, and that concurrent detachment of telomeres and centromeres from the SPB is harmful for spindle formation. The inhibitory mechanisms of centromere detachment may secure spindle formation by preventing concurrent detachment of centromeres and telomeres. To test this possibility, we observed spindle formation in <em>taz1Δ</em> and <em>bqt1Δ</em> mutants, which are partially and fully defective in telomere clustering, respectively (<a href="#pgen-1006304-g002">Fig 2B</a>), and examined the effects of Taz1Δmyb-Sad1 on spindle formation. If spindle formation requires SPB association of either telomeres or centromeres, Taz1Δmyb-Sad1-dependent elimination of centromere-SPB association should cause impairment of spindle formation in telomere clustering-defective cells, and the extents of the resultant impairments should be correlated with the extents of telomere-clustering defects.</p>
<a id="article1.body1.sec2.sec8.p2" name="article1.body1.sec2.sec8.p2" class="link-target"></a><p>In wild-type cells, meiosis I and II occurred sequentially with the formation of one and two spindles, respectively (<a href="#pgen-1006304-g009">Fig 9A</a>, WT; <a href="#pgen.1006304.s013">S5 Movie</a>), and Taz1Δmyb-Sad1 only marginally compromised spindle formation (<a href="#pgen-1006304-g009">Fig 9B</a>, WT). In <em>taz1Δ</em> cells, spindle impairment was not detected, while, in <em>bqt1Δ</em> cells, impaired spindle formation together with detachment of the SPB from the nuclear periphery were observed in about half of cases, as reported previously [<a href="#pgen.1006304.ref034" class="ref-tip">34</a>] (<a href="#pgen-1006304-g009">Fig 9A and 9B</a>, +None; <a href="#pgen.1006304.s014">S6 Movie</a>) (we cannot exclude the possibility that the detached SPBs remained connected with the nucleus by Cut11-lacking nuclear membrane). Notably, Taz1Δmyb-Sad1 induced or increased spindle impairment together with SPB detachment from the nuclear periphery in <em>taz1Δ</em> and <em>bqt1Δ</em> cells (<a href="#pgen-1006304-g009">Fig 9A and 9B</a>, +Taz1Δmyb-Sad1; <a href="#pgen.1006304.s015">S7 Movie</a>), supporting the importance of centromere-SPB association for spindle formation. In addition, the extents of spindle impairment were correlated with the extents of defective telomere clustering; telomere clustering was more severely impaired in <em>bqt1Δ</em> cells than in <em>taz1Δ</em> cells (<a href="#pgen-1006304-g009">Fig 9B</a>, +Taz1Δmyb-Sad1). These results support the idea that meiotic spindle formation requires SPB association of either telomeres or centromeres, and that the inhibitory mechanisms of centromere detachment secure spindle formation by preventing the concurrent detachment of centromeres and telomeres. The SPB was frequently missing at one (monopolar) or both (nonpolar) spindle poles (<a href="#pgen-1006304-g009">Fig 9A and 9B</a>; <a href="#pgen.1006304.s014">S6</a> and <a href="#pgen.1006304.s015">S7</a> Movies). This may mean that telomere/centromere-SPB association is critical for anchoring the SPB to the spindle pole.</p>
<a class="link-target" id="pgen-1006304-g009" name="pgen-1006304-g009"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g009"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g009" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g009"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g009" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g009">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g009">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g009">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g009">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g009">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g009">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 9. </span> Effects of Taz1Δmyb-Sad1 on spindle formation in telomere clustering-defective cells.</div><p class="caption_target"><a id="article1.body1.sec2.sec8.fig1.caption1.p1" name="article1.body1.sec2.sec8.fig1.caption1.p1" class="link-target"></a><p>(A) Dynamics of meiotic spindles and the SPB. Microtubules, the SPB, and the nuclear periphery were visualized by mCherry-tagged α-tubulin (Atb2), a GFP-tagged SPB component Sid4, and an mCherry-tagged nuclear pore component Cut11 [<a href="#pgen.1006304.ref059" class="ref-tip">59</a>], respectively. Images were taken every 5 min. Arrowheads indicate SPBs that are dissociated from the nuclear periphery. White lines indicate cell outlines, and numbers indicate time in minutes. +Taz1Δmyb-Sad1: with Taz1Δmyb-Sad1 expression. Bar: 5 μm. (B) Observation frequencies of various meiosis-I spindle types and SPB detachment from the nuclear periphery. +None: without Taz1Δmyb-Sad1 expression; +Taz1Δmyb-Sad1: with Taz1Δmyb-Sad1 expression. Bipolar: a normal spindle with SPBs at both poles; Monopolar: a spindle with the SPB at one of the poles; Nonpolar: an SPB-lacking spindle; Other: other types of spindles; No spindle: no spindle formation; SPB detach: percentages of cells with the SPB detached from the nuclear periphery; N: number of examined cells. Typical images of the observed spindles are shown at the bottom, and arrowheads indicate the spindle pole lacking a SPB. Bar: 2 μm.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g009">
              https://doi.org/10.1371/journal.pgen.1006304.g009</a></p></div></div>
</div>

<div id="section3" class="section toc-section"><a id="sec011" name="sec011" data-toc="sec011" class="link-target" title="Discussion"></a><h2>Discussion</h2>
<div id="section1" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="Taz1- and microtubule-dependent regulation of centromere detachment from the SPB"></a>
<h3>Taz1- and microtubule-dependent regulation of centromere detachment from the SPB</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>In this study, we showed that kinetochore disassembly-dependent centromere detachment from the SPB is inhibited when telomere clustering is compromised during bouquet formation in <em>S</em>. <em>pombe</em>. Two lines of evidence show that SPB recruitment of Taz1 promotes centromere detachment (<a href="#pgen-1006304-g010">Fig 10A</a>). First, in <em>taz1Δ</em> cells, despite occasional SPB association of telomeres, centromere detachment was greatly inhibited. Second, Taz1Δmyb-Sad1 alleviated the inhibition without restoring telomere clustering. We also found that in addition to Taz1, microtubules contribute to centromere detachment (<a href="#pgen-1006304-g010">Fig 10A</a>). This conclusion was drawn from the fact that centromere detachment was inhibited by introduction of <em>mto1Δ</em> mutation or MBC treatment.</p>
<a class="link-target" id="pgen-1006304-g010" name="pgen-1006304-g010"></a><div class="figure" data-doi="10.1371/journal.pgen.1006304.g010"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pgen.1006304.g010" data-doi="info:doi/10.1371/journal.pgen.1006304" data-uri="info:doi/10.1371/journal.pgen.1006304.g010"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pgen.1006304.g010" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g010">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pgen.1006304.g010">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g010">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pgen.1006304.g010">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g010">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pgen.1006304.g010">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 10. </span> Regulation of centromere detachment from the SPB and its role in meiosis.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) Regulation of centromere detachment from the SPB. Telomere clustering causes SPB recruitment of Taz1, and SPB-recruited Taz1 promotes centromere detachment from the SPB. SPB- and telocentrosome-nucleated microtubules, which drive telomere clustering, also promote centromere detachment. Ndc80C: Ndc80 complex; Mis12C: Mis12 complex. (B) A role of telomere-dependent regulation of centromere detachment in meiosis. During mitosis centromeres are attached to the SPB, while telomeres are located away from it (Rabl). Centromeres become detached from the SPB after clustering of telomeres at the SPB, forming the bouquet chromosome arrangement (Bouquet). When telomere clustering is impaired, the Taz1- and/or microtubule-dependent regulatory mechanism inhibits centromere detachment, preventing concomitant detachment of centromeres and telomeres from the SPB, which is harmful for spindle formation.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pgen.1006304.g010">
              https://doi.org/10.1371/journal.pgen.1006304.g010</a></p></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>Two facts indicate that the contribution of microtubules is Taz1-independent. First, in cells with a disorganized microtubule network, despite occasional recruitment of Taz1 to the SPB, centromere detachment was inhibited to a level comparable to that in cells in which SPB recruitment of Taz1 was completely defective. Second, Taz1Δmyb-Sad1 failed to restore centromere detachment in <em>mto1Δ</em> or MBC-treated cells. Thus, telomere clustering-dependent SPB recruitment of Taz1 and microtubules independently promote centromere detachment in bouquet formation. Taz1- and microtubule-dependent regulatory mechanisms probably inhibit centromere detachment when telomere clustering is impaired.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>One likely scenario for Taz1-dependent centromere regulation is that Taz1 recruits a factor(s) to the SPB that promotes centromere detachment by inducing dissociation of outer kinetochore complexes from the centromere and SPB. The promoting factor is perhaps activated or induced by mating pheromone-dependent MAP kinase because, in the absence of mating pheromone signaling, centromeres remain attached to the SPB despite clustering of telomeres [<a href="#pgen.1006304.ref039" class="ref-tip">39</a>]. Aurora kinase is a potential candidate for the modifying factor because this kinase induces centromere detachment from the SPB in <em>S</em>. <em>cerevisiae</em> [<a href="#pgen.1006304.ref060" class="ref-tip">60</a>, <a href="#pgen.1006304.ref061" class="ref-tip">61</a>]. Another candidate is Polo-like kinase because it regulates meiotic kinetochore mono-orientation and centromere cohesion [<a href="#pgen.1006304.ref062" class="ref-tip">62</a>]. However, it is currently unclear if these kinases contribute to centromere detachment.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4" class="link-target"></a><p>Although Taz1 is important, we cannot exclude the possibility that a different telomere-associated factor(s) additionally contributes to centromere detachment. Taz1Δmyb-Sad1 did not appear to completely restore centromere detachment in the telomere clustering-defective mutants (<a href="#pgen-1006304-g007">Fig 7C and 7D</a>). Furthermore, the retention level of SPB-centromere association or Nuf2/Mis12 centromere localization in the <em>taz1Δ</em> mutant was slightly lower than that in the <em>bqt1Δ</em> or <em>rap1Δ</em> mutant (Figs <a href="#pgen-1006304-g002">2D</a> and <a href="#pgen-1006304-g003">3C</a>, <a href="#pgen.1006304.s003">S3B Fig</a>, Horsetail). These facts may mean the contribution of a different telomere-associated factor to centromere detachment.</p>
<a id="article1.body1.sec3.sec1.p5" name="article1.body1.sec3.sec1.p5" class="link-target"></a><p>How microtubules contribute to centromere detachment remains elusive. Given the requirement of telomere- and/or SPB-nucleated microtubules for telomere clustering, we speculate that efficient centromere detachment requires the telomere/SPB-nucleated microtubules. In <em>mto1Δ</em> cells, γ-TuC telomere/SPB localization and telomere/SPB-microtubule interaction are not completely eliminated [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>], and the remaining telomere/SPB-nucleated microtubules may account for why inhibition of kinetochore disassembly appeared to be weaker in <em>mto1Δ</em> cells than in MBC-treated cells. It is possible that the telomere/SPB-nucleated microtubules recruit a cytoplasmic factor(s) to telomeres and/or the SPB, which activates a centromere detachment-promoting factor in the nucleus, perhaps via the LINC. Alternatively, microtubule-lacking, LINC-accumulated telomeres may generate inhibitory signals for centromere detachment, as spindle-unattached centromeres do for metaphase-anaphase transition in the spindle assembly checkpoint pathway. In either case, the regulatory mechanism is probably different from the Taz1-dependent mechanism because the localization patterns of retained kinetochore components differ between cells lacking telomere-LINC connectors and those defective in microtubule formation (Figs <a href="#pgen-1006304-g003">3D</a> and <a href="#pgen-1006304-g004">4D</a>, <a href="#pgen.1006304.s003">S3C Fig</a>).</p>
</div>

<div id="section2" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="Impairment of meiotic centromere formation by telomere-clustering defects"></a>
<h3>Impairment of meiotic centromere formation by telomere-clustering defects</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>Kinetochore disassembly is thought to be required for establishment of kinetochore mono-orientation and protection of centromere cohesion [<a href="#pgen.1006304.ref039" class="ref-tip">39</a>, <a href="#pgen.1006304.ref040" class="ref-tip">40</a>]. Our finding that equational segregation of sister chromatids at meiosis I increased in chiasma-lacking, telomere clustering-defective cells, where kinetochore disassembly is inhibited, supported this idea. In telomere clustering-defective cells, kinetochore mono-orientation is probably compromised because equational segregation is increased in <em>sgo1Δ rec12Δ</em> zygotes. It was recently reported that the centromeric histone H3 variant Cnp1 and the heterochromatin protein 1 orthologue Swi6 frequently fail to localize at centromeres in telomere clustering-defective cells [<a href="#pgen.1006304.ref048" class="ref-tip">48</a>]. This may mean that kinetochore disassembly also contributes to the proper localization of central centromeric components, and that impaired centromere localization of the central components may cause kinetochore mono-orientation defects. However, if inhibition of kinetochore disassembly induces Cnp1 delocalization, it probably occurs after the horsetail stage because the number of Cnp1 signals was increased during the horsetail stage in the <em>bqt1Δ</em> mutant.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec014" name="sec014" class="link-target" title="Importance of a regulatory link between centromere detachment and telomere clustering for spindle formation"></a>
<h3>Importance of a regulatory link between centromere detachment and telomere clustering for spindle formation</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>We showed that Taz1Δmyb-Sad1 induces or increases impairment of spindle formation in telomere clustering-defective <em>taz1Δ</em> and <em>bqt1Δ</em> cells. Taz1Δmyb-Sad1 induced centromere detachment in these cells; therefore, retention of centromere-SPB association is probably important for spindle formation in the absence of telomere-SPB association. This finding agrees with previous findings that defective telomere clustering causes spindle impairment and that SPB association of centromeres can substitute for SPB association of telomeres and promote spindle formation [<a href="#pgen.1006304.ref034" class="ref-tip">34</a>, <a href="#pgen.1006304.ref035" class="ref-tip">35</a>]. Considering these facts together with the strong correlation of the extents of spindle defects with those of defective telomere/centromere-SPB association, we conclude that concurrent detachment of both telomeres and centromeres from the SPB causes spindle impairment, and propose that the Taz1- and microtubule-dependent inhibitory mechanisms of centromere detachment secure proper spindle formation by preventing such harmful detachment (<a href="#pgen-1006304-g010">Fig 10B</a>). Microtubule disruption is caused by various environmental factors such as low temperature and osmotic stress; therefore, telomere clustering is probably often inhibited in nature. Despite the inhibition of meiotic centromere formation, cells may have evolved the inhibitory mechanisms of centromere detachment to secure spindle formation.</p>
<a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>How telomere/centromere-SPB association contributes to spindle formation is unclear. It was reported that loss of telomere-SPB association leads to decrease in Sad1 at the SPB, and that forced SPB interaction of centromeres restores SPB localization of Sad1 [<a href="#pgen.1006304.ref035" class="ref-tip">35</a>]. Given these observations, it was suggested that telomere/centromere-SPB association is required for proper Sad1 localization at the SPB. The decrease in Sad1 at the SPB may account for the dissociation of the SPB from the spindle pole as well as from the nuclear periphery observed in our analyses. Because abolishment of SPB movements restores spindle formation in telomere clustering-defective cells [<a href="#pgen.1006304.ref063" class="ref-tip">63</a>], in the absence of telomere/centromere-SPB association, SPB movements may disrupt Sad1 interaction with the nuclear membrane, reducing Sad1 localization at the SPB. Alternatively, defective SPB recruitment of telomere- or centromere-associated factors that regulate Sad1 localization may cause the reduction.</p>
<a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3" class="link-target"></a><p>Several facts raise the possibility that similar regulatory mechanisms are present in other organisms. First, meiotic telomere clustering depends on the LINC complexes in other organisms [<a href="#pgen.1006304.ref008" class="ref-tip">8</a>, <a href="#pgen.1006304.ref042" class="ref-tip">42</a>, <a href="#pgen.1006304.ref043" class="ref-tip">43</a>]. Second, in mouse oocytes, kinetochore disassembly probably also occurs, because some of the outer kinetochore complexes are not localized at centromeres before meiotic divisions [<a href="#pgen.1006304.ref064" class="ref-tip">64</a>]. Third, a telomere-binding protein, Ndj1, which contributes to telomere clustering, regulates meiotic spindle formation by interacting with the SUN-domain protein Mps3 in <em>S</em>. <em>cerevisiae</em> [<a href="#pgen.1006304.ref018" class="ref-tip">18</a>, <a href="#pgen.1006304.ref019" class="ref-tip">19</a>, <a href="#pgen.1006304.ref065" class="ref-tip">65</a>, <a href="#pgen.1006304.ref066" class="ref-tip">66</a>], suggesting the presence of a regulatory relationship between telomeres and spindle formation. Although these facts suggest conservation of the mechanisms, at least in <em>S</em>. <em>cerevisiae</em>, microtubule-dependent regulation of centromere detachment is apparently missing because microtubule disruption causes centromere detachment from the SPB and induces meiotic centromere formation unlike the situation in <em>S</em>. <em>pombe</em> [<a href="#pgen.1006304.ref041" class="ref-tip">41</a>]. Although conservation of the regulatory mechanisms is currently unclear, there is no doubt that our findings contribute to understanding the mechanisms of meiosis, because telomere and centromere positions play crucial roles in proper meiotic chromosome segregation. Furthermore, chromosome positioning changes dynamically during development and differentiation and contributes to various chromosomal events in many different organisms [<a href="#pgen.1006304.ref001" class="ref-tip">1</a>–<a href="#pgen.1006304.ref005" class="ref-tip">5</a>], and our findings may also be relevant for understanding the chromosome positioning-dependent mechanisms that regulate development and differentiation of other organisms.</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="sec015" name="sec015" data-toc="sec015" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="sec016" name="sec016" class="link-target" title="Yeast strains, media, basic genetic methods, and visualization of intracellular components"></a>
<h3>Yeast strains, media, basic genetic methods, and visualization of intracellular components</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>The fission yeast strains used in this study are shown in <a href="#pgen.1006304.s008">S1 Table</a>. The media and basic genetic manipulation methods used in this study were described by Moreno et al. [<a href="#pgen.1006304.ref067" class="ref-tip">67</a>]. The deletion alleles of <em>bqt1</em><sup><em>+</em></sup>, <em>rap1</em><sup><em>+</em></sup>, <em>taz1</em><sup><em>+</em></sup>, and <em>mto1</em><sup><em>+</em></sup> were described previously [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>, <a href="#pgen.1006304.ref011" class="ref-tip">11</a>, <a href="#pgen.1006304.ref013" class="ref-tip">13</a>, <a href="#pgen.1006304.ref068" class="ref-tip">68</a>]. Visualization of the <em>sod2</em><sup><em>+</em></sup> and <em>cen2</em> loci, Cnp1, Nuf2, and microtubules was described previously [<a href="#pgen.1006304.ref040" class="ref-tip">40</a>, <a href="#pgen.1006304.ref045" class="ref-tip">45</a>, <a href="#pgen.1006304.ref050" class="ref-tip">50</a>, <a href="#pgen.1006304.ref069" class="ref-tip">69</a>, <a href="#pgen.1006304.ref070" class="ref-tip">70</a>]. The fusion gene of <em>mis12</em><sup><em>+</em></sup> and GFP was obtained from the Yeast Genetic Resource Center.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2" class="link-target"></a><p>Sfi1 was visualized as follows. A DNA fragment encoding mCherry and the P<sub>TET</sub> terminator was amplified by PCR using the oligonucleotide primers 5´-ACGCGTCGACGAAGATCTTCGGATCCCCGGGTTAATTAAC-3´ and 5´-GGGGTACCATATTACCCTGTTATCCCTAGCG-3´, and an mCherry<em>-</em>bearing plasmid as a template, and inserted between the <em>Sal</em>I and <em>Kpn</em>I sites of an integration vector, pYC36, which transforms <em>lys1-131</em> cells into <em>lys1</em><sup><em>+</em></sup> cells when it is integrated [<a href="#pgen.1006304.ref071" class="ref-tip">71</a>]. The resultant plasmid was digested with <em>Sma</em>I and <em>Sac</em>II and ligated with a DNA fragment coding the <em>sfi1</em><sup><em>+</em></sup> gene and its promoter, which was amplified by PCR using the synthetic oligonucleotide primers, 5´-TCCCCGCGGGGCATTGTATTTGTCAATACCCA-3´ and 5’-AAAGGCCTACGGGTATTAGGAGGTATAGGC-3’, and the fission yeast genomic DNA as a template, and digested with <em>Sac</em>II and <em>Stu</em>I. The resultant plasmid pJM1 was introduced into <em>lys1-131</em> cells, and integrants were selected by the <em>lys</em><sup><em>+</em></sup> phenotype. Alternatively, C-terminal mCherry-tagged Sfi1 was generated by the two-step PCR-based method [<a href="#pgen.1006304.ref072" class="ref-tip">72</a>]. DNA fragments encoding the Sfi1 C-terminus-coding region or the <em>sfi1</em><sup><em>+</em></sup> terminator were amplified by PCR using two sets of synthetic oligonucleotide primers (5´-TTTCAATATTAGTGATTGGAAGCG-3´ and 5´-TTAATTAACCCGGGGATACGGGTATTAGGAGGTATAGGC-3´; 5´-TTTTCGCCTCGACATCATCTCATTTACGATTGACGGAGAGAGT-3´ and 5´-ATACGTATTTCATTTTTGTAAATTTTTC-3´) and genomic DNA as a template. The mCherry module was then amplified by PCR using the two PCR products as primers and pHM22 [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>] as a template. The resulting PCR product was introduced into cells, and integrants were selected by resistance to the antibiotic nourseothricin (Werner Bioagents, Jena, Germany) and confirmed by PCR and microscopic observation.</p>
<a id="article1.body1.sec4.sec1.p3" name="article1.body1.sec4.sec1.p3" class="link-target"></a><p>For Mrc1 visualization, a DNA fragment encoding the <em>mrc1</em> gene and its promoter was PCR-amplified from fission yeast genomic DNA using oligonucleotide primers, 5´-TCCCCGCGGTCGAAAGGGTACACAAGCGGA-3´ and 5´-AAGGCCTGTCAAAGTCCGAGTAATTATTCAA-3´. The amplified fragment was digested with <em>Sac</em>II and <em>Stu</em>I and inserted between the <em>Sac</em>II and <em>Sma</em>I sites of an <em>mCherry-</em>coding integration plasmid, pHM4 [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>], yielding pAH8, which encodes the Mrc1-mCherry fusion. Mrc1 was visualized by introducing pAH8 into cells.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec017" name="sec017" class="link-target" title="Analysis of chromosome positioning and kinetochore complex localization in zygotic meiosis"></a>
<h3>Analysis of chromosome positioning and kinetochore complex localization in zygotic meiosis</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>Cells grown on YES solid medium at 30°C were transferred to ME solid medium and induced to enter meiosis by incubation at 25°C for 14–18 h. Nuclear DNA in meiotic zygotes was stained with DNA-specific Hoechst 33342 dye as described previously [<a href="#pgen.1006304.ref058" class="ref-tip">58</a>]. Images of the cells at seven focal planes were taken through a 60×/1.42 NA Plan Apo oil immersion objective lens using an Olympus IX71 inverted microscope (Olympus Corp., Tokyo, Japan) equipped with a cooled charge-coupled device camera (CoolSNAP-HQ2; Nippon Roper Co. Ltd., Tokyo, Japan). Obtained images were processed by deconvolution and analyzed using MetaMorph (version 7) software (Molecular Devices Japan, Inc., Tokyo, Japan).</p>
</div>

<div id="section3" class="section toc-section"><a id="sec018" name="sec018" class="link-target" title="Synchronous induction of meiosis in haploid cells and MBC treatment"></a>
<h3>Synchronous induction of meiosis in haploid cells and MBC treatment</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Haploid cells bearing both mating-type genes were grown in liquid YES medium. They were then induced to enter meiosis by incubation at 30°C in liquid EMM medium lacking a nitrogen source (EMM-N), and treated with MBC, as described previously [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. For monitoring meiosis progression, 50 μl of the culture was harvested every hour, and nuclear morphology was examined by staining DNA with Hoechst 33342.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec019" name="sec019" class="link-target" title="Analysis of sister chromatid segregation at meiosis I"></a>
<h3>Analysis of sister chromatid segregation at meiosis I</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>Segregation of sister chromatids in zygotic <em>rec12</em> cells was examined as follows. Cells containing the GFP-labeled <em>cen2</em> locus and those lacking the GFP-labeled locus of the opposite mating type were grown on YES solid medium at 30°C and mated on ME solid medium at 25°C. For analyzing sister chromatid segregation in haploid meiotic cells, cells containing both mating-type genes were induced to undergo meiosis, as previously described [<a href="#pgen.1006304.ref038" class="ref-tip">38</a>, <a href="#pgen.1006304.ref040" class="ref-tip">40</a>]. The meiotic cells were stained with Hoechst 33342, and GFP signals were examined in those containing two chromosomal DNA masses. When analyzing <em>mto1Δ</em> zygotes, microtubules were simultaneously visualized by mCherry-tagged <em>atb2</em><sup><em>+</em></sup>, and only those forming a meiosis-I spindle were examined because two meiosis I-like chromosome masses were frequently formed due to defective karyogamy.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec020" name="sec020" class="link-target" title="Expression of an N-terminal portion of Taz1 and its artificial tethering to the SPB"></a>
<h3>Expression of an N-terminal portion of Taz1 and its artificial tethering to the SPB</h3>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1" class="link-target"></a><p>To express an N-terminal portion of Taz1 (Taz1Δmyb), a plasmid bearing a gene encoding Taz1Δmyb was constructed using the integration plasmid pMY23, which encodes an mCherry and <em>taz1</em><sup><em>+</em></sup> fusion and the <em>aur1</em><sup><em>r</em></sup> gene as a selectable marker [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. The region that encodes a myb domain of Taz1 in pMY23 was first deleted using the KOD-Plus-Mutagenesis Kit (Toyobo Co., Ltd., Osaka, Japan), generating pJM4. Briefly, a DNA fragment that encodes Taz1Δmyb and mCherry was amplified by PCR using two synthetic oligonucleotide primers, 5’-TTCTCTTCTCAGATTATCACCCTCT-3’ and 5’-AGGATCCCCGGGTTAATTAACAGCA-3’, and pMY23 as a template. After PCR, the template plasmid was removed by <em>Dpn</em>I digestion, and the PCR product was circularized by self-ligation, generating pJM4. The sequence of pJM4 was confirmed by DNA sequencing. Then, a DNA fragment encoding the NLS together with a FLAG tag and a Swi6 chromo-domain was amplified by PCR using two synthetic oligonucleotide primers, 5’-GAAGATCTGGATCCTAGTCGCTTTGTTAAAT-3’ and 5’-CCTTAATTAAACCCGGGCCTTTCTTCTTTTTG-3’, and the plasmid Swi6CD-TOPO (a gift from Dr. Jun-ichi Nakayama, Nagoya City University, Nagoya, Aichi, Japan) as a template. The PCR product was digested with <em>Bgl</em>II and <em>Pac</em>I and placed at the fusion site of Taz1Δmyb and mCherry by inserting it between the <em>Bam</em>HI and <em>Pac</em>I sites of pJM4. The DNA region encoding a FLAG tag and Swi6 chromo-domain was then removed from the resultant plasmid using the KOD-Plus-Mutagenesis Kit with two synthetic oligonucleotide primers, 5’-CCGCGGGTCGACAGGATCCAAACGGCCT-3’ and 5’-CCTTCTCTTCTCAGATTATCACCCC-3’, as described for pJM4. The resultant plasmid pAW9-1 encodes Taz1Δmyb tagged with NLS and mCherry at its C-terminus. Its sequence was confirmed by DNA sequencing. pAW9-1 was transformed into cells, and integrants were selected by resistance to the antibiotic aureobasidin A (Takara Bio. Inc., Otsu, Japan).</p>
<a id="article1.body1.sec4.sec5.p2" name="article1.body1.sec4.sec5.p2" class="link-target"></a><p>To tether Taz1Δmyb to the SPB, the plasmid pAH16-6, which encodes a fusion of Taz1Δmyb and Sad1, was constructed. pAH16-6 was constructed from a plasmid, pMY35, that encodes Sad1 with the HA epitope tag. pMY35 was constructed as follows. A DNA fragment encoding the HA epitope tag together with the <em>adh1</em> terminator Tadh1 was amplified by PCR using synthetic oligonucleotide primers, 5’-ACGCGTCGACGAAGATCTTCGGATCCCCGGGTTAATTAAC-3’ and 5’-GGGGTACCATATTACCCTGTTATCCCTAGCG-3’, and pFA6a-3HA-kanMX6 [<a href="#pgen.1006304.ref073" class="ref-tip">73</a>] as a template. The PCR fragment was digested with <em>Kpn</em>I and <em>Sal</em>I and inserted between the corresponding sites of the integration plasmid pYC36, yielding pTO5. Then, a DNA fragment encoding Sad1 together with its own promoter was amplified by PCR using synthetic oligonucleotide primers, 5’-tccccgcggatgtatccctaacaaacgcaaaaa-3’ and 5’-ccccgctcgagagatgaatcttgacccgtattct-3’, and the fission yeast genomic DNA as a template, and inserted between the <em>Sac</em>II and <em>Sal</em>I sites of pTO5 after digestion with <em>Sac</em>II and <em>Xho</em>I. A portion of the resultant plasmid that encodes Sad1 fused with the HA epitope tag was amplified by PCR using synthetic oligonucleotide primers, 5’-CCCAGTCACGACGTTGTAAAAC-3’ and 5’- CCCCGCTCGAGATATTACCCTGTTATCCCTAGCG-3’, digested with <em>Sac</em>I and <em>Xho</em>I, and inserted between the <em>Sac</em>I and <em>Sal</em>I sites of the integration plasmid pTO2 bearing the <em>aur1</em><sup><em>r</em></sup> gene as a selectable marker [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>] to yield pMY35. To construct pAH16-6, a fusion gene of Taz1Δmyb and Sad1 was first constructed by inserting a Taz1Δmyb-encoding DNA fragment into pMY35. A portion of pAW9-1 that encodes Taz1Δmyb and the <em>taz1</em><sup><em>+</em></sup> promoter was amplified by PCR using synthetic oligonucleotide primers, 5’-ACGAGCTCATCGACAAGGCATGCGAAGC-3’ and 5’-CCGCTCGAGGATTCTCTTCTCAGATTATCACCC-3’, and inserted between the <em>Sac</em>I and <em>Sal</em>I sites of pMY35 after digestion with <em>Sac</em>I and <em>Xho</em>I, yielding the plasmid pAH15-1. Then, pAH16-6 was constructed by replacing a part of pAH15-1 that encodes the HA epitope tag and Tadh1 with an mCherry<em>-</em> and T<sub>TET</sub>-coding region of the plasmid pHM4 [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. The part of pHM4 was amplified by PCR using synthetic oligonucleotide primers, 5’- CCTTAATTAATAGCAAGGGCGAGGAGGATA-3’ and 5’- TCCCCGCGGGGATCTGCCGGTAGAGGT-3’, and inserted between the <em>Pac</em>I and <em>Sac</em>II sites of pAH15-1 after digestion with the corresponding enzymes, yielding pAH16-6. The sequence of pAH16-6 was confirmed by DNA sequencing. pAH16-6 was transformed into cells, and integrants were selected as described for pAW9-1.</p>
</div>

<div id="section6" class="section toc-section"><a id="sec021" name="sec021" class="link-target" title="Live cell analysis of spindle and Nuf2 dynamics"></a>
<h3>Live cell analysis of spindle and Nuf2 dynamics</h3>
<a id="article1.body1.sec4.sec6.p1" name="article1.body1.sec4.sec6.p1" class="link-target"></a><p>For analysis of spindle dynamics in diploid zygotes, cells were grown on solid YES medium and induced to enter meiosis by incubation for 14–18 h at 25°C on solid ME medium. Then, the cells were suspended in liquid EMM-N medium. For analysis of Nuf2 dynamics, Nuf2-expressing haploid cells bearing both mating-type genes were grown in liquid YES medium and induced to enter meiosis by incubation at 30°C in liquid EMM-N medium, as described previously [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. For analysis of spindle or Nuf2 dynamics, a drop of the cell suspension was placed on the bottom of 35 mm glass-bottom dishes (Matsunami Glass Ind., Ltd.) coated with 5 mg/ml lectin (Sigma-Aldrich Japan, Inc.). The cells were observed through a 60×/1.42 NA Plan Apo oil immersion objective lens (Olympus Corp.) using a DeltaVision microscope system operated by SoftWoRx software or an IX71 inverted microscope operated by MetaMorph software. Time-lapse images of the cells were collected at eight focal planes spaced at 0.4 μm intervals every 5 or 10 min for spindle dynamics and at nine focal planes spaced at 0.5 μm intervals every 10 min for Nuf2 dynamics using a cooled CCD camera. During collection of time-lapse images, the cells were kept at 25°C. All obtained images were processed by deconvolution, and analyzed using MetaMorph or Priism/IVE software (available at <a href="http://www.msg.ucsf.edu/IVE/index.html">http://www.msg.ucsf.edu/IVE/index.html</a>).</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="sec022" name="sec022" data-toc="sec022" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pgen.1006304"></div><div class="supplementary-material"><a name="pgen.1006304.s001" id="pgen.1006304.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s001">S1 Fig. </a>Chromosome segregation at meiosis I and a telomere-clustering mechanism.</h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">(A) Chromosome segregation and spindle attachment of chromosomes at meiosis I. (B) Contribution of meiosis-specific chromosome properties and the chiasma to sister chromatid co-segregation at meiosis I. Kinetochore mono-orientation and the chiasma facilitate monopolar attachment of sister chromatids, promoting co-segregation of sister chromatids. Persistent centromere cohesion also promotes sister chromatid co-segregation by preventing separation of sister centromeres upon bipolar attachment of sister chromatids. (C) A telomere-clustering mechanism. The telomere-localized LINC complexes form the telocentrosome. Oligomerized, minus end-directed microtubule motors crosslink the telocentrosome- and SPB-nucleated microtubules and gather the telomeres by moving along the microtubules toward the nucleation sites (black arrows). The minus end-directed microtubule motors tethered to the telocentrosome also directly transport the telomeres toward the nucleation sites to aid telomere clustering. Purple arrow: telocentrosome/telomere movement. NE: nuclear envelope.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s001">https://doi.org/10.1371/journal.pgen.1006304.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s002" id="pgen.1006304.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s002">S2 Fig. </a>Centromere localization in <em>poz1Δ</em>, <em>poz1Δ taz1Δ</em>, and <em>dhc1Δ</em> cells during karyogamy.</h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Population of cells containing SPB-associated centromeres. Centromere-SPB association was examined during karyogamy as in <a href="#pgen-1006304-g002">Fig 2D</a>. p: p value determined by the Student’s t-test; ns: no significant difference (p&gt;0.05).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s002">https://doi.org/10.1371/journal.pgen.1006304.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s003" id="pgen.1006304.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s003">S3 Fig. </a>Localization of Mis12 in telomere clustering-defective cells.</h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">(A) Mis12 localization during mitotic interphase, karyogamy, and the horsetail stage. Arrowheads indicate SPB positions. Karyogamy: karyogamy stage; Horsetail: the horsetail stage. (B) Population of cells containing Mis12 signals. Karyogamy: karyogamy stage; Horsetail: the horsetail stage. Bars show averages of three independent experiments. (C) Population of cells with different Mis12 localization. At SPB: one SPB-associated signal; Off/at SPB: SPB-associated and SPB-dissociated signals; Off SPB: SPB-dissociated signals. Images in the box show the representative localization of the SPB and signals.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s003">https://doi.org/10.1371/journal.pgen.1006304.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s004" id="pgen.1006304.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s004">S4 Fig. </a>Mrc1 localization during meiosis and Nuf2 localization in Mrc1-positive cells.</h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">(A) Changes in Mrc1 localization in a zygotic cell. Cells expressing mCherry-tagged Mrc1 (magenta) and GFP-tagged Atb2 (green) were induced to enter meiosis on solid ME medium. They were suspended in EMM-N liquid medium and observed every 15 min under a microscope. Note that the Mrc1 nuclear signal was undetectable shortly after nuclear fusion (45 min), indicating that this signal is an indicator of the early meiotic stage. Numbers indicate time in minutes. Bar: 2 μm. (B) Population of cells with different Nuf2 localization. Horsetail with Mrc1: the horsetail stage with nuclear Mrc1 signals. Numbers in parentheses show the number of examined cells.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s004">https://doi.org/10.1371/journal.pgen.1006304.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s005" id="pgen.1006304.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s005">S5 Fig. </a>Effects of MBC treatment, various mutations, or Taz1Δmyb-Sad1 on meiosis progression of haploid cells.</h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI">Meiosis progression of haploid cells expressing GFP-tagged Taz1 (A), Cnp1 (B and D), or Nuf2 (C and E). (A, B, and C) Effects of MBC treatment or <em>bqt1Δ</em> or <em>mto1Δ</em> mutation on haploid meiosis progression. (D and E) Combination effects of MBC treatment, <em>bqt1Δ</em> or <em>mto1Δ</em> mutation, and Taz1Δmyb-Sad1 on haploid meiosis progression. MBC (+MBC) or DMSO (+DMSO) was added 2 h after nitrogen depletion. More than 100 cells were examined at each time point. 1 nuc: mononuclear cells; 2 nuc: binuclear cells; 2&lt; nuc: cells containing three or four nuclei.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s005">https://doi.org/10.1371/journal.pgen.1006304.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s006" id="pgen.1006304.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s006">S6 Fig. </a>Effects of <em>moa1Δ</em> mutation on sister chromatid segregation in chiasma-lacking <em>sgo1Δ</em> zygotes.</h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI">The frequencies of equational segregation of sister chromatids at meiosis I in <em>rec12Δ sgo1Δ</em> diploid zygotes. Sister chromatid segregation was analyzed by visualizing the centromere-proximal <em>lys1</em> locus of chromosome I [<a href="#pgen.1006304.ref074" class="ref-tip">74</a>]. It should be noted that chromosome I undergoes equational segregation more frequently than chromosome II in the <em>rec12Δ sgo1Δ</em> background due probably to different centromere structures and/or chromosome lengths [<a href="#pgen.1006304.ref038" class="ref-tip">38</a>].</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s006">https://doi.org/10.1371/journal.pgen.1006304.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s007" id="pgen.1006304.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s007">S7 Fig. </a>Schematic diagram of the predicted localization of telomere-LINC connectors and Taz1Δmyb fragments and effects of Taz1Δmyb fragments on telomere-SPB association and Nuf2 localization.</h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI">(A) Predicted localization of telomere-LINC connectors and Taz1Δmyb fragments in various types of cells. In wild-type cells, the telomere-LINC connectors are recruited to the SPB by telomere clustering at the SPB (Wild type). In <em>taz1Δ</em> cells, the telomere-LINC connectors other than Taz1 are probably recruited to the SPB by occasional telomere-SPB association. Rap1 interacts with a Sad1 interactor Bqt1 [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>]; therefore, telomere-free Rap1 may also interact with the SPB through the Bqt1-Bqt2 complex. Taz1Δmyb likely localizes at the SPB through interaction with Rap1 but not with telomeres in <em>taz1Δ</em> cells (<em>taz1Δ</em> + Taz1Δmyb). In <em>bqt1Δ</em> cells, the telomere-LINC connectors are probably not recruited to the SPB [<a href="#pgen.1006304.ref010" class="ref-tip">10</a>], but Taz1Δmyb-Sad1 is probably able to localize at the SPB via Sad1 (<em>bqt1Δ</em> + Taz1Δmyb-Sad1). In <em>mto1Δ</em> cells, LINC connectors and the LINC complex accumulate at telomeres, but due to defective telomere microtubule nucleation, telomere clustering is defective [<a href="#pgen.1006304.ref045" class="ref-tip">45</a>]. Taz1Δmyb-Sad1 is probably localized at telomeres as well as the SPB (<em>mto1Δ</em> + Taz1Δmyb-Sad1). Poz1/Tpz1/Pot1: a complex of Poz1, Tpz1, and Pot1; mCh: mCherry molecule. (B) Localization patterns of the telomere-adjacent <em>sod2</em> locus during karyogamy in wild-type and telomere clustering-defective cells expressing Taz1Δmyb (+Taz1Δmyb) or Taz1Δmyb-Sad1 (+Taz1Δmyb-Sad1). Numbers in parentheses indicate the number of examined cells. (C) Population of Taz1Δmyb- (+Taz1Δmyb) or Taz1Δmyb-Sad1-expressing (+Taz1Δmyb-Sad1) cells with different Nuf2 localization. At SPB: one SPB-associated signal; Off/at SPB: SPB-associated and SPB-dissociated signals; Off SPB: SPB-dissociated signals. Images in the box show the representative localization of the SPB and signals.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s007">https://doi.org/10.1371/journal.pgen.1006304.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(EPS)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s008" id="pgen.1006304.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s008">S1 Table. </a>Strains used in this study.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s008">https://doi.org/10.1371/journal.pgen.1006304.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s009" id="pgen.1006304.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s009">S1 Movie. </a>Nuf2 localization dynamics during meiosis in a haploid wild-type cell.</h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI">A haploid wild-type cell bearing both mating-type genes and expressing GFP-tagged Nuf2 was induced to enter meiosis by incubation in EMM-N medium. Images at nine focal planes spaced at 0.5 μm intervals were taken every 10 min, and each set of images was processed by deconvolution and combined to form a maximal projection by Metamorph software. The video plays at a rate of three frames per second.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s009">https://doi.org/10.1371/journal.pgen.1006304.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(MOV)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s010" id="pgen.1006304.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s010">S2 Movie. </a>Nuf2 localization dynamics in relation with the SPB during meiosis in a haploid wild-type cell.</h3><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1" class="link-target"></a><p class="preSiDOI">This movie is the same as <a href="#pgen.1006304.s009">S1 Movie</a>, except that GFP-tagged Nuf2 and mCherry-tagged Sfi1 are simultaneously shown in green and magenta, respectively.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s010">https://doi.org/10.1371/journal.pgen.1006304.s010</a></p><a id="article1.body1.sec5.supplementary-material10.caption1.p2" name="article1.body1.sec5.supplementary-material10.caption1.p2" class="link-target"></a><p class="postSiDOI">(MOV)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s011" id="pgen.1006304.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s011">S3 Movie. </a>Nuf2 localization dynamics during meiosis in a haploid <em>bqt1Δ</em> cell.</h3><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1" class="link-target"></a><p class="preSiDOI">A haploid <em>bqt1Δ</em> cell bearing both mating-type genes and expressing GFP-tagged Nuf2 was induced to enter meiosis by incubation in EMM-N medium. Images at nine focal planes spaced at 0.5 μm intervals were taken every 10 min, and each set of images was processed by deconvolution and combined to form a maximal projection by Metamorph software. The video plays at a rate of three frames per second.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s011">https://doi.org/10.1371/journal.pgen.1006304.s011</a></p><a id="article1.body1.sec5.supplementary-material11.caption1.p2" name="article1.body1.sec5.supplementary-material11.caption1.p2" class="link-target"></a><p class="postSiDOI">(MOV)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s012" id="pgen.1006304.s012" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s012">S4 Movie. </a>Nuf2 localization dynamics in relation with the SPB during meiosis in a haploid <em>bqt1Δ</em> cell.</h3><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1" class="link-target"></a><p class="preSiDOI">This movie is the same as <a href="#pgen.1006304.s011">S3 Movie</a>, except that GFP-tagged Nuf2 and mCherry-tagged Sfi1 are simultaneously shown in green and magenta, respectively.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s012">https://doi.org/10.1371/journal.pgen.1006304.s012</a></p><a id="article1.body1.sec5.supplementary-material12.caption1.p2" name="article1.body1.sec5.supplementary-material12.caption1.p2" class="link-target"></a><p class="postSiDOI">(MOV)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s013" id="pgen.1006304.s013" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s013">S5 Movie. </a>Dynamics of microtubules, the SPB, and the nuclear membrane during meiosis in a wild-type zygote.</h3><a id="article1.body1.sec5.supplementary-material13.caption1.p1" name="article1.body1.sec5.supplementary-material13.caption1.p1" class="link-target"></a><p class="preSiDOI">Images of a wild-type zygote expressing mCherry-tagged Atb2 and Cut11 (magenta) and GFP-tagged Sid4 (green) were taken at eight focal planes spaced at 0.4 μm intervals every 5 min. Each set of images was processed by deconvolution and combined to form a maximal projection by Priism/IVE software. The video plays at a rate of five frames per second.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s013">https://doi.org/10.1371/journal.pgen.1006304.s013</a></p><a id="article1.body1.sec5.supplementary-material13.caption1.p2" name="article1.body1.sec5.supplementary-material13.caption1.p2" class="link-target"></a><p class="postSiDOI">(MOV)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s014" id="pgen.1006304.s014" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s014">S6 Movie. </a>Dynamics of microtubules, the SPB, and the nuclear membrane during meiosis in a <em>bqt1Δ</em> zygote.</h3><a id="article1.body1.sec5.supplementary-material14.caption1.p1" name="article1.body1.sec5.supplementary-material14.caption1.p1" class="link-target"></a><p class="preSiDOI">Images of a <em>bqt1Δ</em> zygote expressing mCherry-tagged Atb2 and Cut11 (magenta) and GFP-tagged Sid4 (green) were taken at eight focal planes spaced at 0.4 μm intervals every 5 min. Each set of images was processed by deconvolution and combined to form a maximal projection by Priism/IVE software. The video plays at a rate of five frames per second.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s014">https://doi.org/10.1371/journal.pgen.1006304.s014</a></p><a id="article1.body1.sec5.supplementary-material14.caption1.p2" name="article1.body1.sec5.supplementary-material14.caption1.p2" class="link-target"></a><p class="postSiDOI">(MOV)</p>
</div><div class="supplementary-material"><a name="pgen.1006304.s015" id="pgen.1006304.s015" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pgen.1006304.s015">S7 Movie. </a>Dynamics of microtubules, the SPB, and the nuclear membrane during meiosis in a <em>bqt1Δ</em> zygote expressing Taz1Δmyb-Sad1.</h3><a id="article1.body1.sec5.supplementary-material15.caption1.p1" name="article1.body1.sec5.supplementary-material15.caption1.p1" class="link-target"></a><p class="preSiDOI">Images of a <em>bqt1Δ</em> zygote expressing Taz1Δmyb-Sad1 together with mCherry-tagged Atb2 and Cut11 (magenta) and GFP-tagged Sid4 (green) were taken at eight focal planes spaced at 0.4 μm intervals every 5 min. Each set of images was processed by deconvolution and combined to form a maximal projection by Priism/IVE software. The video plays at a rate of five frames per second.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pgen.1006304.s015">https://doi.org/10.1371/journal.pgen.1006304.s015</a></p><a id="article1.body1.sec5.supplementary-material15.caption1.p2" name="article1.body1.sec5.supplementary-material15.caption1.p2" class="link-target"></a><p class="postSiDOI">(MOV)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank Junko Kanoh, Yasushi Hiraoka, Jun-ichi Nakayama, and the National Bio-Resource Project of the MEXT, Japan for strains or reagents, and Akira Shinohara and Masahiro Terasawa for use of the DeltaVision microscope system or related technical support. We also thank Kayoko Tanaka, Akira Shinohara, Junko Kanoh, Takashi Ushimaru, and Takanori Oyoshi for critical reading of the manuscript and many helpful comments.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>

<ol class="simple">

<li><strong>Conceptualization:</strong> AY.</li>

<li><strong>Funding acquisition:</strong> AY.</li>

<li><strong>Investigation:</strong> KK AH JM EN KI KT AW HM AY.</li>

<li><strong>Methodology:</strong> AY.</li>

<li><strong>Project administration:</strong> AY.</li>

<li><strong>Supervision:</strong> AY.</li>

<li><strong>Validation:</strong> KK AH JM EN KI KT AW HM AY.</li>

<li><strong>Visualization:</strong> AY.</li>

<li><strong>Writing – original draft:</strong> AY.</li>

<li><strong>Writing – review &amp; editing:</strong> AY.</li>

</ol></p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pgen.1006304.ref001" id="pgen.1006304.ref001" class="link-target"></a>Cremer T, Cremer M. Chromosome territories. Cold Spring Harbor perspectives in biology. 2010;2(3):a003889. Epub 2010/03/20.  pmid:20300217; PubMed Central PMCID: PMC2829961. <ul class="reflinks"><li><a href="#" data-author="Cremer" data-cit="CremerT%2C%20CremerM.%20Chromosome%20territories.%20Cold%20Spring%20Harbor%20perspectives%20in%20biology.%202010%3B2%283%29%3Aa003889.%20Epub%202010%2F03%2F20.%20doi%3A%2010.1101%2Fcshperspect.a003889%2020300217%3B%20PubMed%20Central%20PMCID%3A%20PMC2829961." data-title="Chromosome%20territories" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20300217" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chromosome+territories+Cremer+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pgen.1006304.ref002" id="pgen.1006304.ref002" class="link-target"></a>Meister P, Mango SE, Gasser SM. Locking the genome: nuclear organization and cell fate. Current opinion in genetics &amp; development. 2011;21(2):167–74. Epub 2011/02/25.  pmid:21345665; PubMed Central PMCID: PMC4041333. <ul class="reflinks"><li><a href="#" data-author="Meister" data-cit="MeisterP%2C%20MangoSE%2C%20GasserSM.%20Locking%20the%20genome%3A%20nuclear%20organization%20and%20cell%20fate.%20Current%20opinion%20in%20genetics%20%26%20development.%202011%3B21%282%29%3A167%E2%80%9374.%20Epub%202011%2F02%2F25.%20doi%3A%2010.1016%2Fj.gde.2011.01.023%2021345665%3B%20PubMed%20Central%20PMCID%3A%20PMC4041333." data-title="Locking%20the%20genome%3A%20nuclear%20organization%20and%20cell%20fate" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21345665" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Locking+the+genome%3A+nuclear+organization+and+cell+fate+Meister+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pgen.1006304.ref003" id="pgen.1006304.ref003" class="link-target"></a>Rocha PP, Skok JA. The origin of recurrent translocations in recombining lymphocytes: a balance between break frequency and nuclear proximity. Curr Opin Cell Biol. 2013;25(3):365–71. Epub 2013/03/13.  pmid:23478218; PubMed Central PMCID: PMC3691303. <ul class="reflinks"><li><a href="#" data-author="Rocha" data-cit="RochaPP%2C%20SkokJA.%20The%20origin%20of%20recurrent%20translocations%20in%20recombining%20lymphocytes%3A%20a%20balance%20between%20break%20frequency%20and%20nuclear%20proximity.%20Curr%20Opin%20Cell%20Biol.%202013%3B25%283%29%3A365%E2%80%9371.%20Epub%202013%2F03%2F13.%20doi%3A%2010.1016%2Fj.ceb.2013.02.007%2023478218%3B%20PubMed%20Central%20PMCID%3A%20PMC3691303." data-title="The%20origin%20of%20recurrent%20translocations%20in%20recombining%20lymphocytes%3A%20a%20balance%20between%20break%20frequency%20and%20nuclear%20proximity" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23478218" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+origin+of+recurrent+translocations+in+recombining+lymphocytes%3A+a+balance+between+break+frequency+and+nuclear+proximity+Rocha+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pgen.1006304.ref004" id="pgen.1006304.ref004" class="link-target"></a>Mine-Hattab J, Rothstein R. DNA in motion during double-strand break repair. Trends Cell Biol. 2013;23(11):529–36. Epub 2013/07/23.  pmid:23867212; PubMed Central PMCID: PMC3820367. <ul class="reflinks"><li><a href="#" data-author="Mine-Hattab" data-cit="Mine-HattabJ%2C%20RothsteinR.%20DNA%20in%20motion%20during%20double-strand%20break%20repair.%20Trends%20Cell%20Biol.%202013%3B23%2811%29%3A529%E2%80%9336.%20Epub%202013%2F07%2F23.%20doi%3A%2010.1016%2Fj.tcb.2013.05.006%2023867212%3B%20PubMed%20Central%20PMCID%3A%20PMC3820367." data-title="DNA%20in%20motion%20during%20double-strand%20break%20repair" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23867212" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DNA+in+motion+during+double-strand+break+repair+Mine-Hattab+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pgen.1006304.ref005" id="pgen.1006304.ref005" class="link-target"></a>Aparicio OM. Location, location, location: it's all in the timing for replication origins. Genes Dev. 2013;27(2):117–28. Epub 2013/01/26.  pmid:23348837; PubMed Central PMCID: PMC3566304. <ul class="reflinks"><li><a href="#" data-author="Aparicio" data-cit="AparicioOM.%20Location%2C%20location%2C%20location%3A%20it%27s%20all%20in%20the%20timing%20for%20replication%20origins.%20Genes%20Dev.%202013%3B27%282%29%3A117%E2%80%9328.%20Epub%202013%2F01%2F26.%20doi%3A%2010.1101%2Fgad.209999.112%2023348837%3B%20PubMed%20Central%20PMCID%3A%20PMC3566304." data-title="Location%2C%20location%2C%20location%3A%20it%27s%20all%20in%20the%20timing%20for%20replication%20origins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23348837" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Location%2C+location%2C+location%3A+it%27s+all+in+the+timing+for+replication+origins+Aparicio+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pgen.1006304.ref006" id="pgen.1006304.ref006" class="link-target"></a>Scherthan H. A bouquet makes ends meet. Nat Rev Mol Cell Biol. 2001;2(8):621–7. Epub 2001/08/03.  pmid:11483995. <ul class="reflinks"><li><a href="#" data-author="Scherthan" data-cit="ScherthanH.%20A%20bouquet%20makes%20ends%20meet.%20Nat%20Rev%20Mol%20Cell%20Biol.%202001%3B2%288%29%3A621%E2%80%937.%20Epub%202001%2F08%2F03.%20doi%3A%2010.1038%2F35085086%2011483995." data-title="A%20bouquet%20makes%20ends%20meet" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483995" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+bouquet+makes+ends+meet+Scherthan+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pgen.1006304.ref007" id="pgen.1006304.ref007" class="link-target"></a>Zickler D, Kleckner N. The leptotene-zygotene transition of meiosis. Annual review of genetics. 1998;32:619–97. Epub 1999/02/03.  pmid:9928494. <ul class="reflinks"><li><a href="#" data-author="Zickler" data-cit="ZicklerD%2C%20KlecknerN.%20The%20leptotene-zygotene%20transition%20of%20meiosis.%20Annual%20review%20of%20genetics.%201998%3B32%3A619%E2%80%9397.%20Epub%201999%2F02%2F03.%20doi%3A%2010.1146%2Fannurev.genet.32.1.619%209928494." data-title="The%20leptotene-zygotene%20transition%20of%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9928494" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+leptotene-zygotene+transition+of+meiosis+Zickler+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pgen.1006304.ref008" id="pgen.1006304.ref008" class="link-target"></a>Hiraoka Y, Dernburg AF. The SUN rises on meiotic chromosome dynamics. Dev Cell. 2009;17(5):598–605. Epub 2009/11/20.  pmid:19922865. <ul class="reflinks"><li><a href="#" data-author="Hiraoka" data-cit="HiraokaY%2C%20DernburgAF.%20The%20SUN%20rises%20on%20meiotic%20chromosome%20dynamics.%20Dev%20Cell.%202009%3B17%285%29%3A598%E2%80%93605.%20Epub%202009%2F11%2F20.%20doi%3A%2010.1016%2Fj.devcel.2009.10.014%2019922865." data-title="The%20SUN%20rises%20on%20meiotic%20chromosome%20dynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19922865" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+SUN+rises+on+meiotic+chromosome+dynamics+Hiraoka+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pgen.1006304.ref009" id="pgen.1006304.ref009" class="link-target"></a>Koszul R, Kleckner N. Dynamic chromosome movements during meiosis: a way to eliminate unwanted connections? Trends Cell Biol. 2009;19(12):716–24. Epub 2009/10/27.  pmid:19854056; PubMed Central PMCID: PMC2787882. <ul class="reflinks"><li><a href="#" data-author="Koszul" data-cit="KoszulR%2C%20KlecknerN.%20Dynamic%20chromosome%20movements%20during%20meiosis%3A%20a%20way%20to%20eliminate%20unwanted%20connections%3F%20Trends%20Cell%20Biol.%202009%3B19%2812%29%3A716%E2%80%9324.%20Epub%202009%2F10%2F27.%20doi%3A%2010.1016%2Fj.tcb.2009.09.007%2019854056%3B%20PubMed%20Central%20PMCID%3A%20PMC2787882." data-title="Dynamic%20chromosome%20movements%20during%20meiosis%3A%20a%20way%20to%20eliminate%20unwanted%20connections%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19854056" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Dynamic+chromosome+movements+during+meiosis%3A+a+way+to+eliminate+unwanted+connections%3F+Koszul+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pgen.1006304.ref010" id="pgen.1006304.ref010" class="link-target"></a>Chikashige Y, Tsutsumi C, Yamane M, Okamasa K, Haraguchi T, Hiraoka Y. Meiotic proteins bqt1 and bqt2 tether telomeres to form the bouquet arrangement of chromosomes. Cell. 2006;125(1):59–69. Epub 2006/04/18.  pmid:16615890. <ul class="reflinks"><li><a href="#" data-author="Chikashige" data-cit="ChikashigeY%2C%20TsutsumiC%2C%20YamaneM%2C%20OkamasaK%2C%20HaraguchiT%2C%20HiraokaY.%20Meiotic%20proteins%20bqt1%20and%20bqt2%20tether%20telomeres%20to%20form%20the%20bouquet%20arrangement%20of%20chromosomes.%20Cell.%202006%3B125%281%29%3A59%E2%80%9369.%20Epub%202006%2F04%2F18.%20doi%3A%2010.1016%2Fj.cell.2006.01.048%2016615890." data-title="Meiotic%20proteins%20bqt1%20and%20bqt2%20tether%20telomeres%20to%20form%20the%20bouquet%20arrangement%20of%20chromosomes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16615890" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Meiotic+proteins+bqt1+and+bqt2+tether+telomeres+to+form+the+bouquet+arrangement+of+chromosomes+Chikashige+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pgen.1006304.ref011" id="pgen.1006304.ref011" class="link-target"></a>Chikashige Y, Hiraoka Y. Telomere binding of the Rap1 protein is required for meiosis in fission yeast. Curr Biol. 2001;11(20):1618–23. Epub 2001/10/26. pmid:11676924. <ul class="reflinks"><li><a href="#" data-author="Chikashige" data-cit="ChikashigeY%2C%20HiraokaY.%20Telomere%20binding%20of%20the%20Rap1%20protein%20is%20required%20for%20meiosis%20in%20fission%20yeast.%20Curr%20Biol.%202001%3B11%2820%29%3A1618%E2%80%9323.%20Epub%202001%2F10%2F26.%2011676924." data-title="Telomere%20binding%20of%20the%20Rap1%20protein%20is%20required%20for%20meiosis%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11676924" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Telomere+binding+of+the+Rap1+protein+is+required+for+meiosis+in+fission+yeast+Chikashige+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pgen.1006304.ref012" id="pgen.1006304.ref012" class="link-target"></a>Nimmo ER, Pidoux AL, Perry PE, Allshire RC. Defective meiosis in telomere-silencing mutants of Schizosaccharomyces pombe. Nature. 1998;392(6678):825–8. Epub 1998/05/08. 10.1038/33941. pmid:9572142. <ul class="reflinks"><li><a href="#" data-author="Nimmo" data-cit="NimmoER%2C%20PidouxAL%2C%20PerryPE%2C%20AllshireRC.%20Defective%20meiosis%20in%20telomere-silencing%20mutants%20of%20Schizosaccharomyces%20pombe.%20Nature.%201998%3B392%286678%29%3A825%E2%80%938.%20Epub%201998%2F05%2F08.%2010.1038%2F33941.%209572142." data-title="Defective%20meiosis%20in%20telomere-silencing%20mutants%20of%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9572142" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Defective+meiosis+in+telomere-silencing+mutants+of+Schizosaccharomyces+pombe+Nimmo+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pgen.1006304.ref013" id="pgen.1006304.ref013" class="link-target"></a>Cooper JP, Watanabe Y, Nurse P. Fission yeast Taz1 protein is required for meiotic telomere clustering and recombination. Nature. 1998;392(6678):828–31. Epub 1998/05/08.  pmid:9572143. <ul class="reflinks"><li><a href="#" data-author="Cooper" data-cit="CooperJP%2C%20WatanabeY%2C%20NurseP.%20Fission%20yeast%20Taz1%20protein%20is%20required%20for%20meiotic%20telomere%20clustering%20and%20recombination.%20Nature.%201998%3B392%286678%29%3A828%E2%80%9331.%20Epub%201998%2F05%2F08.%20doi%3A%2010.1038%2F33947%209572143." data-title="Fission%20yeast%20Taz1%20protein%20is%20required%20for%20meiotic%20telomere%20clustering%20and%20recombination" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9572143" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Fission+yeast+Taz1+protein+is+required+for+meiotic+telomere+clustering+and+recombination+Cooper+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pgen.1006304.ref014" id="pgen.1006304.ref014" class="link-target"></a>Conrad MN, Dominguez AM, Dresser ME. Ndj1p, a meiotic telomere protein required for normal chromosome synapsis and segregation in yeast. Science. 1997;276(5316):1252–5. Epub 1997/05/23. pmid:9157883. <ul class="reflinks"><li><a href="#" data-author="Conrad" data-cit="ConradMN%2C%20DominguezAM%2C%20DresserME.%20Ndj1p%2C%20a%20meiotic%20telomere%20protein%20required%20for%20normal%20chromosome%20synapsis%20and%20segregation%20in%20yeast.%20Science.%201997%3B276%285316%29%3A1252%E2%80%935.%20Epub%201997%2F05%2F23.%209157883." data-title="Ndj1p%2C%20a%20meiotic%20telomere%20protein%20required%20for%20normal%20chromosome%20synapsis%20and%20segregation%20in%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9157883" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Ndj1p%2C+a+meiotic+telomere+protein+required+for+normal+chromosome+synapsis+and+segregation+in+yeast+Conrad+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pgen.1006304.ref015" id="pgen.1006304.ref015" class="link-target"></a>Conrad MN, Lee CY, Wilkerson JL, Dresser ME. MPS3 mediates meiotic bouquet formation in Saccharomyces cerevisiae. Proc Natl Acad Sci U S A. 2007;104(21):8863–8. pmid:17495028. <ul class="reflinks"><li><a href="#" data-author="Conrad" data-cit="ConradMN%2C%20LeeCY%2C%20WilkersonJL%2C%20DresserME.%20MPS3%20mediates%20meiotic%20bouquet%20formation%20in%20Saccharomyces%20cerevisiae.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202007%3B104%2821%29%3A8863%E2%80%938.%2017495028." data-title="MPS3%20mediates%20meiotic%20bouquet%20formation%20in%20Saccharomyces%20cerevisiae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17495028" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MPS3+mediates+meiotic+bouquet+formation+in+Saccharomyces+cerevisiae+Conrad+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pgen.1006304.ref016" id="pgen.1006304.ref016" class="link-target"></a>Goldman AS, Lichten M. Restriction of ectopic recombination by interhomolog interactions during Saccharomyces cerevisiae meiosis. Proc Natl Acad Sci U S A. 2000;97(17):9537–42. Epub 2000/08/16. pmid:10944222; PubMed Central PMCID: PMC16900. <ul class="reflinks"><li><a href="#" data-author="Goldman" data-cit="GoldmanAS%2C%20LichtenM.%20Restriction%20of%20ectopic%20recombination%20by%20interhomolog%20interactions%20during%20Saccharomyces%20cerevisiae%20meiosis.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202000%3B97%2817%29%3A9537%E2%80%9342.%20Epub%202000%2F08%2F16.%2010944222%3B%20PubMed%20Central%20PMCID%3A%20PMC16900." data-title="Restriction%20of%20ectopic%20recombination%20by%20interhomolog%20interactions%20during%20Saccharomyces%20cerevisiae%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10944222" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Restriction+of+ectopic+recombination+by+interhomolog+interactions+during+Saccharomyces+cerevisiae+meiosis+Goldman+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pgen.1006304.ref017" id="pgen.1006304.ref017" class="link-target"></a>Kosaka H, Shinohara M, Shinohara A. Csm4-dependent telomere movement on nuclear envelope promotes meiotic recombination. PLoS genetics. 2008;4(9):e1000196. pmid:18818742.  <ul class="reflinks"><li><a href="#" data-author="Kosaka" data-cit="KosakaH%2C%20ShinoharaM%2C%20ShinoharaA.%20Csm4-dependent%20telomere%20movement%20on%20nuclear%20envelope%20promotes%20meiotic%20recombination.%20PLoS%20genetics.%202008%3B4%289%29%3Ae1000196.%2018818742.%20doi%3A%2010.1371%2Fjournal.pgen.1000196" data-title="Csm4-dependent%20telomere%20movement%20on%20nuclear%20envelope%20promotes%20meiotic%20recombination" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18818742" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Csm4-dependent+telomere+movement+on+nuclear+envelope+promotes+meiotic+recombination+Kosaka+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pgen.1006304.ref018" id="pgen.1006304.ref018" class="link-target"></a>Trelles-Sticken E, Dresser ME, Scherthan H. Meiotic telomere protein Ndj1p is required for meiosis-specific telomere distribution, bouquet formation and efficient homologue pairing. J Cell Biol. 2000;151(1):95–106. Epub 2000/10/06. pmid:11018056; PubMed Central PMCID: PMC2189801. <ul class="reflinks"><li><a href="#" data-author="Trelles-Sticken" data-cit="Trelles-StickenE%2C%20DresserME%2C%20ScherthanH.%20Meiotic%20telomere%20protein%20Ndj1p%20is%20required%20for%20meiosis-specific%20telomere%20distribution%2C%20bouquet%20formation%20and%20efficient%20homologue%20pairing.%20J%20Cell%20Biol.%202000%3B151%281%29%3A95%E2%80%93106.%20Epub%202000%2F10%2F06.%2011018056%3B%20PubMed%20Central%20PMCID%3A%20PMC2189801." data-title="Meiotic%20telomere%20protein%20Ndj1p%20is%20required%20for%20meiosis-specific%20telomere%20distribution%2C%20bouquet%20formation%20and%20efficient%20homologue%20pairing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11018056" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Meiotic+telomere+protein+Ndj1p+is+required+for+meiosis-specific+telomere+distribution%2C+bouquet+formation+and+efficient+homologue+pairing+Trelles-Sticken+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pgen.1006304.ref019" id="pgen.1006304.ref019" class="link-target"></a>Wanat JJ, Kim KP, Koszul R, Zanders S, Weiner B, Kleckner N, et al. Csm4, in collaboration with Ndj1, mediates telomere-led chromosome dynamics and recombination during yeast meiosis. PLoS genetics. 2008;4(9):e1000188. Epub 2008/09/27.  pmid:18818741; PubMed Central PMCID: PMC2533701. <ul class="reflinks"><li><a href="#" data-author="Wanat" data-cit="WanatJJ%2C%20KimKP%2C%20KoszulR%2C%20ZandersS%2C%20WeinerB%2C%20KlecknerN%2C%20et%20al.%20Csm4%2C%20in%20collaboration%20with%20Ndj1%2C%20mediates%20telomere-led%20chromosome%20dynamics%20and%20recombination%20during%20yeast%20meiosis.%20PLoS%20genetics.%202008%3B4%289%29%3Ae1000188.%20Epub%202008%2F09%2F27.%20doi%3A%2010.1371%2Fjournal.pgen.1000188%2018818741%3B%20PubMed%20Central%20PMCID%3A%20PMC2533701." data-title="Csm4%2C%20in%20collaboration%20with%20Ndj1%2C%20mediates%20telomere-led%20chromosome%20dynamics%20and%20recombination%20during%20yeast%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18818741" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Csm4%2C+in+collaboration+with+Ndj1%2C+mediates+telomere-led+chromosome+dynamics+and+recombination+during+yeast+meiosis+Wanat+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pgen.1006304.ref020" id="pgen.1006304.ref020" class="link-target"></a>Horn HF, Kim DI, Wright GD, Wong ES, Stewart CL, Burke B, et al. A mammalian KASH domain protein coupling meiotic chromosomes to the cytoskeleton. J Cell Biol. 2013;202(7):1023–39. Epub 2013/09/26.  pmid:24062341. <ul class="reflinks"><li><a href="#" data-author="Horn" data-cit="HornHF%2C%20KimDI%2C%20WrightGD%2C%20WongES%2C%20StewartCL%2C%20BurkeB%2C%20et%20al.%20A%20mammalian%20KASH%20domain%20protein%20coupling%20meiotic%20chromosomes%20to%20the%20cytoskeleton.%20J%20Cell%20Biol.%202013%3B202%287%29%3A1023%E2%80%9339.%20Epub%202013%2F09%2F26.%20doi%3A%2010.1083%2Fjcb.201304004%2024062341." data-title="A%20mammalian%20KASH%20domain%20protein%20coupling%20meiotic%20chromosomes%20to%20the%20cytoskeleton" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24062341" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+mammalian+KASH+domain+protein+coupling+meiotic+chromosomes+to+the+cytoskeleton+Horn+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pgen.1006304.ref021" id="pgen.1006304.ref021" class="link-target"></a>Ding X, Xu R, Yu J, Xu T, Zhuang Y, Han M. SUN1 is required for telomere attachment to nuclear envelope and gametogenesis in mice. Dev Cell. 2007;12(6):863–72. pmid:17543860. <ul class="reflinks"><li><a href="#" data-author="Ding" data-cit="DingX%2C%20XuR%2C%20YuJ%2C%20XuT%2C%20ZhuangY%2C%20HanM.%20SUN1%20is%20required%20for%20telomere%20attachment%20to%20nuclear%20envelope%20and%20gametogenesis%20in%20mice.%20Dev%20Cell.%202007%3B12%286%29%3A863%E2%80%9372.%2017543860." data-title="SUN1%20is%20required%20for%20telomere%20attachment%20to%20nuclear%20envelope%20and%20gametogenesis%20in%20mice" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17543860" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=SUN1+is+required+for+telomere+attachment+to+nuclear+envelope+and+gametogenesis+in+mice+Ding+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pgen.1006304.ref022" id="pgen.1006304.ref022" class="link-target"></a>Shibuya H, Hernandez-Hernandez A, Morimoto A, Negishi L, Hoog C, Watanabe Y. MAJIN Links Telomeric DNA to the Nuclear Membrane by Exchanging Telomere Cap. Cell. 2015;163(5):1252–66. Epub 2015/11/10.  pmid:26548954. <ul class="reflinks"><li><a href="#" data-author="Shibuya" data-cit="ShibuyaH%2C%20Hernandez-HernandezA%2C%20MorimotoA%2C%20NegishiL%2C%20HoogC%2C%20WatanabeY.%20MAJIN%20Links%20Telomeric%20DNA%20to%20the%20Nuclear%20Membrane%20by%20Exchanging%20Telomere%20Cap.%20Cell.%202015%3B163%285%29%3A1252%E2%80%9366.%20Epub%202015%2F11%2F10.%20doi%3A%2010.1016%2Fj.cell.2015.10.030%2026548954." data-title="MAJIN%20Links%20Telomeric%20DNA%20to%20the%20Nuclear%20Membrane%20by%20Exchanging%20Telomere%20Cap" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26548954" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MAJIN+Links+Telomeric+DNA+to+the+Nuclear+Membrane+by+Exchanging+Telomere+Cap+Shibuya+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pgen.1006304.ref023" id="pgen.1006304.ref023" class="link-target"></a>Niwa O, Shimanuki M, Miki F. Telomere-led bouquet formation facilitates homologous chromosome pairing and restricts ectopic interaction in fission yeast meiosis. Embo J. 2000;19(14):3831–40. Epub 2000/07/19.  pmid:10899136; PubMed Central PMCID: PMC313979. <ul class="reflinks"><li><a href="#" data-author="Niwa" data-cit="NiwaO%2C%20ShimanukiM%2C%20MikiF.%20Telomere-led%20bouquet%20formation%20facilitates%20homologous%20chromosome%20pairing%20and%20restricts%20ectopic%20interaction%20in%20fission%20yeast%20meiosis.%20Embo%20J.%202000%3B19%2814%29%3A3831%E2%80%9340.%20Epub%202000%2F07%2F19.%20doi%3A%2010.1093%2Femboj%2F19.14.3831%2010899136%3B%20PubMed%20Central%20PMCID%3A%20PMC313979." data-title="Telomere-led%20bouquet%20formation%20facilitates%20homologous%20chromosome%20pairing%20and%20restricts%20ectopic%20interaction%20in%20fission%20yeast%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10899136" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Telomere-led+bouquet+formation+facilitates+homologous+chromosome+pairing+and+restricts+ectopic+interaction+in+fission+yeast+meiosis+Niwa+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pgen.1006304.ref024" id="pgen.1006304.ref024" class="link-target"></a>Phillips CM, Wong C, Bhalla N, Carlton PM, Weiser P, Meneely PM, et al. HIM-8 binds to the X chromosome pairing center and mediates chromosome-specific meiotic synapsis. Cell. 2005;123(6):1051–63. pmid:16360035. <ul class="reflinks"><li><a href="#" data-author="Phillips" data-cit="PhillipsCM%2C%20WongC%2C%20BhallaN%2C%20CarltonPM%2C%20WeiserP%2C%20MeneelyPM%2C%20et%20al.%20HIM-8%20binds%20to%20the%20X%20chromosome%20pairing%20center%20and%20mediates%20chromosome-specific%20meiotic%20synapsis.%20Cell.%202005%3B123%286%29%3A1051%E2%80%9363.%2016360035." data-title="HIM-8%20binds%20to%20the%20X%20chromosome%20pairing%20center%20and%20mediates%20chromosome-specific%20meiotic%20synapsis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16360035" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=HIM-8+binds+to+the+X+chromosome+pairing+center+and+mediates+chromosome-specific+meiotic+synapsis+Phillips+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pgen.1006304.ref025" id="pgen.1006304.ref025" class="link-target"></a>Penkner A, Tang L, Novatchkova M, Ladurner M, Fridkin A, Gruenbaum Y, et al. The nuclear envelope protein Matefin/SUN-1 is required for homologous pairing in C. elegans meiosis. Dev Cell. 2007;12(6):873–85. pmid:17543861. <ul class="reflinks"><li><a href="#" data-author="Penkner" data-cit="PenknerA%2C%20TangL%2C%20NovatchkovaM%2C%20LadurnerM%2C%20FridkinA%2C%20GruenbaumY%2C%20et%20al.%20The%20nuclear%20envelope%20protein%20Matefin%2FSUN-1%20is%20required%20for%20homologous%20pairing%20in%20C.%20elegans%20meiosis.%20Dev%20Cell.%202007%3B12%286%29%3A873%E2%80%9385.%2017543861." data-title="The%20nuclear%20envelope%20protein%20Matefin%2FSUN-1%20is%20required%20for%20homologous%20pairing%20in%20C.%20elegans%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17543861" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+nuclear+envelope+protein+Matefin%2FSUN-1+is+required+for+homologous+pairing+in+C.+elegans+meiosis+Penkner+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pgen.1006304.ref026" id="pgen.1006304.ref026" class="link-target"></a>MacQueen AJ, Phillips CM, Bhalla N, Weiser P, Villeneuve AM, Dernburg AF. Chromosome sites play dual roles to establish homologous synapsis during meiosis in C. elegans. Cell. 2005;123(6):1037–50. pmid:16360034. <ul class="reflinks"><li><a href="#" data-author="MacQueen" data-cit="MacQueenAJ%2C%20PhillipsCM%2C%20BhallaN%2C%20WeiserP%2C%20VilleneuveAM%2C%20DernburgAF.%20Chromosome%20sites%20play%20dual%20roles%20to%20establish%20homologous%20synapsis%20during%20meiosis%20in%20C.%20elegans.%20Cell.%202005%3B123%286%29%3A1037%E2%80%9350.%2016360034." data-title="Chromosome%20sites%20play%20dual%20roles%20to%20establish%20homologous%20synapsis%20during%20meiosis%20in%20C.%20elegans" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16360034" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chromosome+sites+play+dual+roles+to+establish+homologous+synapsis+during+meiosis+in+C.+elegans+MacQueen+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pgen.1006304.ref027" id="pgen.1006304.ref027" class="link-target"></a>Sato A, Isaac B, Phillips CM, Rillo R, Carlton PM, Wynne DJ, et al. Cytoskeletal forces span the nuclear envelope to coordinate meiotic chromosome pairing and synapsis. Cell. 2009;139(5):907–19. Epub 2009/11/17.  pmid:19913287; PubMed Central PMCID: PMC2825574. <ul class="reflinks"><li><a href="#" data-author="Sato" data-cit="SatoA%2C%20IsaacB%2C%20PhillipsCM%2C%20RilloR%2C%20CarltonPM%2C%20WynneDJ%2C%20et%20al.%20Cytoskeletal%20forces%20span%20the%20nuclear%20envelope%20to%20coordinate%20meiotic%20chromosome%20pairing%20and%20synapsis.%20Cell.%202009%3B139%285%29%3A907%E2%80%9319.%20Epub%202009%2F11%2F17.%20doi%3A%2010.1016%2Fj.cell.2009.10.039%2019913287%3B%20PubMed%20Central%20PMCID%3A%20PMC2825574." data-title="Cytoskeletal%20forces%20span%20the%20nuclear%20envelope%20to%20coordinate%20meiotic%20chromosome%20pairing%20and%20synapsis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19913287" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cytoskeletal+forces+span+the+nuclear+envelope+to+coordinate+meiotic+chromosome+pairing+and+synapsis+Sato+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pgen.1006304.ref028" id="pgen.1006304.ref028" class="link-target"></a>Penkner AM, Fridkin A, Gloggnitzer J, Baudrimont A, Machacek T, Woglar A, et al. Meiotic chromosome homology search involves modifications of the nuclear envelope protein Matefin/SUN-1. Cell. 2009;139(5):920–33. Epub 2009/11/17.  pmid:19913286. <ul class="reflinks"><li><a href="#" data-author="Penkner" data-cit="PenknerAM%2C%20FridkinA%2C%20GloggnitzerJ%2C%20BaudrimontA%2C%20MachacekT%2C%20WoglarA%2C%20et%20al.%20Meiotic%20chromosome%20homology%20search%20involves%20modifications%20of%20the%20nuclear%20envelope%20protein%20Matefin%2FSUN-1.%20Cell.%202009%3B139%285%29%3A920%E2%80%9333.%20Epub%202009%2F11%2F17.%20doi%3A%2010.1016%2Fj.cell.2009.10.045%2019913286." data-title="Meiotic%20chromosome%20homology%20search%20involves%20modifications%20of%20the%20nuclear%20envelope%20protein%20Matefin%2FSUN-1" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19913286" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Meiotic+chromosome+homology+search+involves+modifications+of+the+nuclear+envelope+protein+Matefin%2FSUN-1+Penkner+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pgen.1006304.ref029" id="pgen.1006304.ref029" class="link-target"></a>Labella S, Woglar A, Jantsch V, Zetka M. Polo kinases establish links between meiotic chromosomes and cytoskeletal forces essential for homolog pairing. Dev Cell. 2011;21(5):948–58. Epub 2011/10/25.  pmid:22018921. <ul class="reflinks"><li><a href="#" data-author="Labella" data-cit="LabellaS%2C%20WoglarA%2C%20JantschV%2C%20ZetkaM.%20Polo%20kinases%20establish%20links%20between%20meiotic%20chromosomes%20and%20cytoskeletal%20forces%20essential%20for%20homolog%20pairing.%20Dev%20Cell.%202011%3B21%285%29%3A948%E2%80%9358.%20Epub%202011%2F10%2F25.%20doi%3A%2010.1016%2Fj.devcel.2011.07.011%2022018921." data-title="Polo%20kinases%20establish%20links%20between%20meiotic%20chromosomes%20and%20cytoskeletal%20forces%20essential%20for%20homolog%20pairing" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22018921" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Polo+kinases+establish+links+between+meiotic+chromosomes+and+cytoskeletal+forces+essential+for+homolog+pairing+Labella+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pgen.1006304.ref030" id="pgen.1006304.ref030" class="link-target"></a>Hodgkin J, Horvitz HR, Brenner S. Nondisjunction Mutants of the Nematode CAENORHABDITIS ELEGANS. Genetics. 1979;91(1):67–94. Epub 1979/01/01. pmid:17248881; PubMed Central PMCID: PMC1213932. <ul class="reflinks"><li><a href="#" data-author="Hodgkin" data-cit="HodgkinJ%2C%20HorvitzHR%2C%20BrennerS.%20Nondisjunction%20Mutants%20of%20the%20Nematode%20CAENORHABDITIS%20ELEGANS.%20Genetics.%201979%3B91%281%29%3A67%E2%80%9394.%20Epub%201979%2F01%2F01.%2017248881%3B%20PubMed%20Central%20PMCID%3A%20PMC1213932." data-title="Nondisjunction%20Mutants%20of%20the%20Nematode%20CAENORHABDITIS%20ELEGANS" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17248881" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Nondisjunction+Mutants+of+the+Nematode+CAENORHABDITIS+ELEGANS+Hodgkin+1979" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pgen.1006304.ref031" id="pgen.1006304.ref031" class="link-target"></a>Funabiki H, Hagan I, Uzawa S, Yanagida M. Cell cycle-dependent specific positioning and clustering of centromeres and telomeres in fission yeast. J Cell Biol. 1993;121(5):961–76. Epub 1993/06/01. pmid:8388878; PubMed Central PMCID: PMC2119680. <ul class="reflinks"><li><a href="#" data-author="Funabiki" data-cit="FunabikiH%2C%20HaganI%2C%20UzawaS%2C%20YanagidaM.%20Cell%20cycle-dependent%20specific%20positioning%20and%20clustering%20of%20centromeres%20and%20telomeres%20in%20fission%20yeast.%20J%20Cell%20Biol.%201993%3B121%285%29%3A961%E2%80%9376.%20Epub%201993%2F06%2F01.%208388878%3B%20PubMed%20Central%20PMCID%3A%20PMC2119680." data-title="Cell%20cycle-dependent%20specific%20positioning%20and%20clustering%20of%20centromeres%20and%20telomeres%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8388878" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cell+cycle-dependent+specific+positioning+and+clustering+of+centromeres+and+telomeres+in+fission+yeast+Funabiki+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pgen.1006304.ref032" id="pgen.1006304.ref032" class="link-target"></a>Chikashige Y, Ding DQ, Funabiki H, Haraguchi T, Mashiko S, Yanagida M, et al. Telomere-led premeiotic chromosome movement in fission yeast. Science. 1994;264(5156):270–3. Epub 1994/04/08. pmid:8146661. <ul class="reflinks"><li><a href="#" data-author="Chikashige" data-cit="ChikashigeY%2C%20DingDQ%2C%20FunabikiH%2C%20HaraguchiT%2C%20MashikoS%2C%20YanagidaM%2C%20et%20al.%20Telomere-led%20premeiotic%20chromosome%20movement%20in%20fission%20yeast.%20Science.%201994%3B264%285156%29%3A270%E2%80%933.%20Epub%201994%2F04%2F08.%208146661." data-title="Telomere-led%20premeiotic%20chromosome%20movement%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8146661" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Telomere-led+premeiotic+chromosome+movement+in+fission+yeast+Chikashige+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pgen.1006304.ref033" id="pgen.1006304.ref033" class="link-target"></a>Yamamoto A. Gathering up meiotic telomeres: a novel function of the microtubule-organizing center. Cellular and molecular life sciences: CMLS. 2014;71(11):2119–34. Epub 2014/01/15.  pmid:24413667. <ul class="reflinks"><li><a href="#" data-author="Yamamoto" data-cit="YamamotoA.%20Gathering%20up%20meiotic%20telomeres%3A%20a%20novel%20function%20of%20the%20microtubule-organizing%20center.%20Cellular%20and%20molecular%20life%20sciences%3A%20CMLS.%202014%3B71%2811%29%3A2119%E2%80%9334.%20Epub%202014%2F01%2F15.%20doi%3A%2010.1007%2Fs00018-013-1548-1%2024413667." data-title="Gathering%20up%20meiotic%20telomeres%3A%20a%20novel%20function%20of%20the%20microtubule-organizing%20center" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24413667" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Gathering+up+meiotic+telomeres%3A+a+novel+function+of+the+microtubule-organizing+center+Yamamoto+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pgen.1006304.ref034" id="pgen.1006304.ref034" class="link-target"></a>Tomita K, Cooper JP. The telomere bouquet controls the meiotic spindle. Cell. 2007;130(1):113–26. Epub 2007/07/17.  pmid:17632059. <ul class="reflinks"><li><a href="#" data-author="Tomita" data-cit="TomitaK%2C%20CooperJP.%20The%20telomere%20bouquet%20controls%20the%20meiotic%20spindle.%20Cell.%202007%3B130%281%29%3A113%E2%80%9326.%20Epub%202007%2F07%2F17.%20doi%3A%2010.1016%2Fj.cell.2007.05.024%2017632059." data-title="The%20telomere%20bouquet%20controls%20the%20meiotic%20spindle" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17632059" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+telomere+bouquet+controls+the+meiotic+spindle+Tomita+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pgen.1006304.ref035" id="pgen.1006304.ref035" class="link-target"></a>Fennell A, Fernandez-Alvarez A, Tomita K, Cooper JP. Telomeres and centromeres have interchangeable roles in promoting meiotic spindle formation. J Cell Biol. 2015;208(4):415–28. Epub 2015/02/18.  pmid:25688135; PubMed Central PMCID: PMC4332249. <ul class="reflinks"><li><a href="#" data-author="Fennell" data-cit="FennellA%2C%20Fernandez-AlvarezA%2C%20TomitaK%2C%20CooperJP.%20Telomeres%20and%20centromeres%20have%20interchangeable%20roles%20in%20promoting%20meiotic%20spindle%20formation.%20J%20Cell%20Biol.%202015%3B208%284%29%3A415%E2%80%9328.%20Epub%202015%2F02%2F18.%20doi%3A%2010.1083%2Fjcb.201409058%2025688135%3B%20PubMed%20Central%20PMCID%3A%20PMC4332249." data-title="Telomeres%20and%20centromeres%20have%20interchangeable%20roles%20in%20promoting%20meiotic%20spindle%20formation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25688135" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Telomeres+and+centromeres+have+interchangeable+roles+in+promoting+meiotic+spindle+formation+Fennell+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pgen.1006304.ref036" id="pgen.1006304.ref036" class="link-target"></a>Hauf S, Watanabe Y. Kinetochore orientation in mitosis and meiosis. Cell. 2004;119(3):317–27. Epub 2004/10/28.  pmid:15507205. <ul class="reflinks"><li><a href="#" data-author="Hauf" data-cit="HaufS%2C%20WatanabeY.%20Kinetochore%20orientation%20in%20mitosis%20and%20meiosis.%20Cell.%202004%3B119%283%29%3A317%E2%80%9327.%20Epub%202004%2F10%2F28.%20doi%3A%2010.1016%2Fj.cell.2004.10.014%2015507205." data-title="Kinetochore%20orientation%20in%20mitosis%20and%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15507205" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Kinetochore+orientation+in+mitosis+and+meiosis+Hauf+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pgen.1006304.ref037" id="pgen.1006304.ref037" class="link-target"></a>Petronczki M, Siomos MF, Nasmyth K. Un menage a quatre: the molecular biology of chromosome segregation in meiosis. Cell. 2003;112(4):423–40. Epub 2003/02/26. pmid:12600308. <ul class="reflinks"><li><a href="#" data-author="Petronczki" data-cit="PetronczkiM%2C%20SiomosMF%2C%20NasmythK.%20Un%20menage%20a%20quatre%3A%20the%20molecular%20biology%20of%20chromosome%20segregation%20in%20meiosis.%20Cell.%202003%3B112%284%29%3A423%E2%80%9340.%20Epub%202003%2F02%2F26.%2012600308." data-title="Un%20menage%20a%20quatre%3A%20the%20molecular%20biology%20of%20chromosome%20segregation%20in%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12600308" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Un+menage+a+quatre%3A+the+molecular+biology+of+chromosome+segregation+in+meiosis+Petronczki+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pgen.1006304.ref038" id="pgen.1006304.ref038" class="link-target"></a>Hirose Y, Suzuki R, Ohba T, Hinohara Y, Matsuhara H, Yoshida M, et al. Chiasmata promote monopolar attachment of sister chromatids and their co-segregation toward the proper pole during meiosis I. PLoS genetics. 2011;7(3):e1001329. Epub 2011/03/23.  pmid:21423721; PubMed Central PMCID: PMC3053323. <ul class="reflinks"><li><a href="#" data-author="Hirose" data-cit="HiroseY%2C%20SuzukiR%2C%20OhbaT%2C%20HinoharaY%2C%20MatsuharaH%2C%20YoshidaM%2C%20et%20al.%20Chiasmata%20promote%20monopolar%20attachment%20of%20sister%20chromatids%20and%20their%20co-segregation%20toward%20the%20proper%20pole%20during%20meiosis%20I.%20PLoS%20genetics.%202011%3B7%283%29%3Ae1001329.%20Epub%202011%2F03%2F23.%20doi%3A%2010.1371%2Fjournal.pgen.1001329%2021423721%3B%20PubMed%20Central%20PMCID%3A%20PMC3053323." data-title="Chiasmata%20promote%20monopolar%20attachment%20of%20sister%20chromatids%20and%20their%20co-segregation%20toward%20the%20proper%20pole%20during%20meiosis%20I" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21423721" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chiasmata+promote+monopolar+attachment+of+sister+chromatids+and+their+co-segregation+toward+the+proper+pole+during+meiosis+I+Hirose+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pgen.1006304.ref039" id="pgen.1006304.ref039" class="link-target"></a>Asakawa H, Hayashi A, Haraguchi T, Hiraoka Y. Dissociation of the Nuf2-Ndc80 complex releases centromeres from the spindle-pole body during meiotic prophase in fission yeast. Mol Biol Cell. 2005;16(5):2325–38. Epub 2005/02/25.  pmid:15728720; PubMed Central PMCID: PMC1087238. <ul class="reflinks"><li><a href="#" data-author="Asakawa" data-cit="AsakawaH%2C%20HayashiA%2C%20HaraguchiT%2C%20HiraokaY.%20Dissociation%20of%20the%20Nuf2-Ndc80%20complex%20releases%20centromeres%20from%20the%20spindle-pole%20body%20during%20meiotic%20prophase%20in%20fission%20yeast.%20Mol%20Biol%20Cell.%202005%3B16%285%29%3A2325%E2%80%9338.%20Epub%202005%2F02%2F25.%20doi%3A%2010.1091%2Fmbc.E04-11-0996%2015728720%3B%20PubMed%20Central%20PMCID%3A%20PMC1087238." data-title="Dissociation%20of%20the%20Nuf2-Ndc80%20complex%20releases%20centromeres%20from%20the%20spindle-pole%20body%20during%20meiotic%20prophase%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728720" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Dissociation+of+the+Nuf2-Ndc80+complex+releases+centromeres+from+the+spindle-pole+body+during+meiotic+prophase+in+fission+yeast+Asakawa+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pgen.1006304.ref040" id="pgen.1006304.ref040" class="link-target"></a>Yamamoto A, Hiraoka Y. Monopolar spindle attachment of sister chromatids is ensured by two distinct mechanisms at the first meiotic division in fission yeast. Embo J. 2003;22(9):2284–96. Epub 2003/05/03.  pmid:12727894; PubMed Central PMCID: PMC156094. <ul class="reflinks"><li><a href="#" data-author="Yamamoto" data-cit="YamamotoA%2C%20HiraokaY.%20Monopolar%20spindle%20attachment%20of%20sister%20chromatids%20is%20ensured%20by%20two%20distinct%20mechanisms%20at%20the%20first%20meiotic%20division%20in%20fission%20yeast.%20Embo%20J.%202003%3B22%289%29%3A2284%E2%80%9396.%20Epub%202003%2F05%2F03.%20doi%3A%2010.1093%2Femboj%2Fcdg222%2012727894%3B%20PubMed%20Central%20PMCID%3A%20PMC156094." data-title="Monopolar%20spindle%20attachment%20of%20sister%20chromatids%20is%20ensured%20by%20two%20distinct%20mechanisms%20at%20the%20first%20meiotic%20division%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12727894" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Monopolar+spindle+attachment+of+sister+chromatids+is+ensured+by+two+distinct+mechanisms+at+the+first+meiotic+division+in+fission+yeast+Yamamoto+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pgen.1006304.ref041" id="pgen.1006304.ref041" class="link-target"></a>Miller MP, Unal E, Brar GA, Amon A. Meiosis I chromosome segregation is established through regulation of microtubule-kinetochore interactions. eLife. 2012;1:e00117. Epub 2013/01/01.  pmid:23275833; PubMed Central PMCID: PMC3525924. <ul class="reflinks"><li><a href="#" data-author="Miller" data-cit="MillerMP%2C%20UnalE%2C%20BrarGA%2C%20AmonA.%20Meiosis%20I%20chromosome%20segregation%20is%20established%20through%20regulation%20of%20microtubule-kinetochore%20interactions.%20eLife.%202012%3B1%3Ae00117.%20Epub%202013%2F01%2F01.%20doi%3A%2010.7554%2FeLife.00117%2023275833%3B%20PubMed%20Central%20PMCID%3A%20PMC3525924." data-title="Meiosis%20I%20chromosome%20segregation%20is%20established%20through%20regulation%20of%20microtubule-kinetochore%20interactions" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23275833" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Meiosis+I+chromosome+segregation+is+established+through+regulation+of+microtubule-kinetochore+interactions+Miller+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pgen.1006304.ref042" id="pgen.1006304.ref042" class="link-target"></a>Fridkin A, Penkner A, Jantsch V, Gruenbaum Y. SUN-domain and KASH-domain proteins during development, meiosis and disease. Cellular and molecular life sciences: CMLS. 2009;66(9):1518–33. Epub 2009/01/07.  pmid:19125221. <ul class="reflinks"><li><a href="#" data-author="Fridkin" data-cit="FridkinA%2C%20PenknerA%2C%20JantschV%2C%20GruenbaumY.%20SUN-domain%20and%20KASH-domain%20proteins%20during%20development%2C%20meiosis%20and%20disease.%20Cellular%20and%20molecular%20life%20sciences%3A%20CMLS.%202009%3B66%289%29%3A1518%E2%80%9333.%20Epub%202009%2F01%2F07.%20doi%3A%2010.1007%2Fs00018-008-8713-y%2019125221." data-title="SUN-domain%20and%20KASH-domain%20proteins%20during%20development%2C%20meiosis%20and%20disease.%20Cellular%20and%20molecular%20life%20sciences" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19125221" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=SUN-domain+and+KASH-domain+proteins+during+development%2C+meiosis+and+disease.+Cellular+and+molecular+life+sciences+Fridkin+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pgen.1006304.ref043" id="pgen.1006304.ref043" class="link-target"></a>Razafsky D, Hodzic D. Bringing KASH under the SUN: the many faces of nucleo-cytoskeletal connections. J Cell Biol. 2009;186(4):461–72. Epub 2009/08/19.  pmid:19687252; PubMed Central PMCID: PMC2733748. <ul class="reflinks"><li><a href="#" data-author="Razafsky" data-cit="RazafskyD%2C%20HodzicD.%20Bringing%20KASH%20under%20the%20SUN%3A%20the%20many%20faces%20of%20nucleo-cytoskeletal%20connections.%20J%20Cell%20Biol.%202009%3B186%284%29%3A461%E2%80%9372.%20Epub%202009%2F08%2F19.%20doi%3A%2010.1083%2Fjcb.200906068%2019687252%3B%20PubMed%20Central%20PMCID%3A%20PMC2733748." data-title="Bringing%20KASH%20under%20the%20SUN%3A%20the%20many%20faces%20of%20nucleo-cytoskeletal%20connections" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19687252" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Bringing+KASH+under+the+SUN%3A+the+many+faces+of+nucleo-cytoskeletal+connections+Razafsky+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pgen.1006304.ref044" id="pgen.1006304.ref044" class="link-target"></a>Hou H, Zhou Z, Wang Y, Wang J, Kallgren SP, Kurchuk T, et al. Csi1 links centromeres to the nuclear envelope for centromere clustering. J Cell Biol. 2012;199(5):735–44. Epub 2012/11/21.  pmid:23166349; PubMed Central PMCID: PMC3514793. <ul class="reflinks"><li><a href="#" data-author="Hou" data-cit="HouH%2C%20ZhouZ%2C%20WangY%2C%20WangJ%2C%20KallgrenSP%2C%20KurchukT%2C%20et%20al.%20Csi1%20links%20centromeres%20to%20the%20nuclear%20envelope%20for%20centromere%20clustering.%20J%20Cell%20Biol.%202012%3B199%285%29%3A735%E2%80%9344.%20Epub%202012%2F11%2F21.%20doi%3A%2010.1083%2Fjcb.201208001%2023166349%3B%20PubMed%20Central%20PMCID%3A%20PMC3514793." data-title="Csi1%20links%20centromeres%20to%20the%20nuclear%20envelope%20for%20centromere%20clustering" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23166349" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Csi1+links+centromeres+to+the+nuclear+envelope+for+centromere+clustering+Hou+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pgen.1006304.ref045" id="pgen.1006304.ref045" class="link-target"></a>Yoshida M, Katsuyama S, Tateho K, Nakamura H, Miyoshi J, Ohba T, et al. Microtubule-organizing center formation at telomeres induces meiotic telomere clustering. J Cell Biol. 2013;200(4):385–95. Epub 2013/02/13.  pmid:23401002; PubMed Central PMCID: PMC3575533. <ul class="reflinks"><li><a href="#" data-author="Yoshida" data-cit="YoshidaM%2C%20KatsuyamaS%2C%20TatehoK%2C%20NakamuraH%2C%20MiyoshiJ%2C%20OhbaT%2C%20et%20al.%20Microtubule-organizing%20center%20formation%20at%20telomeres%20induces%20meiotic%20telomere%20clustering.%20J%20Cell%20Biol.%202013%3B200%284%29%3A385%E2%80%9395.%20Epub%202013%2F02%2F13.%20doi%3A%2010.1083%2Fjcb.201207168%2023401002%3B%20PubMed%20Central%20PMCID%3A%20PMC3575533." data-title="Microtubule-organizing%20center%20formation%20at%20telomeres%20induces%20meiotic%20telomere%20clustering" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23401002" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Microtubule-organizing+center+formation+at+telomeres+induces+meiotic+telomere+clustering+Yoshida+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pgen.1006304.ref046" id="pgen.1006304.ref046" class="link-target"></a>Hayashi A, Asakawa H, Haraguchi T, Hiraoka Y. Reconstruction of the kinetochore during meiosis in fission yeast Schizosaccharomyces pombe. Mol Biol Cell. 2006;17(12):5173–84. Epub 2006/10/13.  pmid:17035632; PubMed Central PMCID: PMC1679682. <ul class="reflinks"><li><a href="#" data-author="Hayashi" data-cit="HayashiA%2C%20AsakawaH%2C%20HaraguchiT%2C%20HiraokaY.%20Reconstruction%20of%20the%20kinetochore%20during%20meiosis%20in%20fission%20yeast%20Schizosaccharomyces%20pombe.%20Mol%20Biol%20Cell.%202006%3B17%2812%29%3A5173%E2%80%9384.%20Epub%202006%2F10%2F13.%20doi%3A%2010.1091%2Fmbc.E06-05-0388%2017035632%3B%20PubMed%20Central%20PMCID%3A%20PMC1679682." data-title="Reconstruction%20of%20the%20kinetochore%20during%20meiosis%20in%20fission%20yeast%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17035632" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Reconstruction+of+the+kinetochore+during+meiosis+in+fission+yeast+Schizosaccharomyces+pombe+Hayashi+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pgen.1006304.ref047" id="pgen.1006304.ref047" class="link-target"></a>Chikashige Y, Ding DQ, Imai Y, Yamamoto M, Haraguchi T, Hiraoka Y. Meiotic nuclear reorganization: switching the position of centromeres and telomeres in the fission yeast <em>Schizosaccharomyces pombe</em>. Embo J. 1997;16(1):193–202. Epub 1997/01/02.  pmid:9009280; PubMed Central PMCID: PMC1169626. <ul class="reflinks"><li><a href="#" data-author="Chikashige" data-cit="ChikashigeY%2C%20DingDQ%2C%20ImaiY%2C%20YamamotoM%2C%20HaraguchiT%2C%20HiraokaY.%20Meiotic%20nuclear%20reorganization%3A%20switching%20the%20position%20of%20centromeres%20and%20telomeres%20in%20the%20fission%20yeast%20Schizosaccharomyces%20pombe.%20Embo%20J.%201997%3B16%281%29%3A193%E2%80%93202.%20Epub%201997%2F01%2F02.%20doi%3A%2010.1093%2Femboj%2F16.1.193%209009280%3B%20PubMed%20Central%20PMCID%3A%20PMC1169626." data-title="Meiotic%20nuclear%20reorganization%3A%20switching%20the%20position%20of%20centromeres%20and%20telomeres%20in%20the%20fission%20yeast%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9009280" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Meiotic+nuclear+reorganization%3A+switching+the+position+of+centromeres+and+telomeres+in+the+fission+yeast+Schizosaccharomyces+pombe+Chikashige+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pgen.1006304.ref048" id="pgen.1006304.ref048" class="link-target"></a>Klutstein M, Fennell A, Fernandez-Alvarez A, Cooper JP. The telomere bouquet regulates meiotic centromere assembly. Nat Cell Biol. 2015;17(4):458–69. Epub 2015/03/17.  pmid:25774833. <ul class="reflinks"><li><a href="#" data-author="Klutstein" data-cit="KlutsteinM%2C%20FennellA%2C%20Fernandez-AlvarezA%2C%20CooperJP.%20The%20telomere%20bouquet%20regulates%20meiotic%20centromere%20assembly.%20Nat%20Cell%20Biol.%202015%3B17%284%29%3A458%E2%80%9369.%20Epub%202015%2F03%2F17.%20doi%3A%2010.1038%2Fncb3132%2025774833." data-title="The%20telomere%20bouquet%20regulates%20meiotic%20centromere%20assembly" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25774833" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+telomere+bouquet+regulates+meiotic+centromere+assembly+Klutstein+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="pgen.1006304.ref049" id="pgen.1006304.ref049" class="link-target"></a>Yamamoto TG, Chikashige Y, Ozoe F, Kawamukai M, Hiraoka Y. Activation of the pheromone-responsive MAP kinase drives haploid cells to undergo ectopic meiosis with normal telomere clustering and sister chromatid segregation in fission yeast. J Cell Sci. 2004;117(Pt 17):3875–86. Epub 2004/07/22.  pmid:15265989. <ul class="reflinks"><li><a href="#" data-author="Yamamoto" data-cit="YamamotoTG%2C%20ChikashigeY%2C%20OzoeF%2C%20KawamukaiM%2C%20HiraokaY.%20Activation%20of%20the%20pheromone-responsive%20MAP%20kinase%20drives%20haploid%20cells%20to%20undergo%20ectopic%20meiosis%20with%20normal%20telomere%20clustering%20and%20sister%20chromatid%20segregation%20in%20fission%20yeast.%20J%20Cell%20Sci.%202004%3B117%28Pt%2017%29%3A3875%E2%80%9386.%20Epub%202004%2F07%2F22.%20doi%3A%2010.1242%2Fjcs.01248%2015265989." data-title="Activation%20of%20the%20pheromone-responsive%20MAP%20kinase%20drives%20haploid%20cells%20to%20undergo%20ectopic%20meiosis%20with%20normal%20telomere%20clustering%20and%20sister%20chromatid%20segregation%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15265989" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Activation+of+the+pheromone-responsive+MAP+kinase+drives+haploid+cells+to+undergo+ectopic+meiosis+with+normal+telomere+clustering+and+sister+chromatid+segregation+in+fission+yeast+Yamamoto+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="pgen.1006304.ref050" id="pgen.1006304.ref050" class="link-target"></a>Ding DQ, Yamamoto A, Haraguchi T, Hiraoka Y. Dynamics of homologous chromosome pairing during meiotic prophase in fission yeast. Dev Cell. 2004;6(3):329–41. Epub 2004/03/20. pmid:15030757. <ul class="reflinks"><li><a href="#" data-author="Ding" data-cit="DingDQ%2C%20YamamotoA%2C%20HaraguchiT%2C%20HiraokaY.%20Dynamics%20of%20homologous%20chromosome%20pairing%20during%20meiotic%20prophase%20in%20fission%20yeast.%20Dev%20Cell.%202004%3B6%283%29%3A329%E2%80%9341.%20Epub%202004%2F03%2F20.%2015030757." data-title="Dynamics%20of%20homologous%20chromosome%20pairing%20during%20meiotic%20prophase%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15030757" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Dynamics+of+homologous+chromosome+pairing+during+meiotic+prophase+in+fission+yeast+Ding+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="pgen.1006304.ref051" id="pgen.1006304.ref051" class="link-target"></a>Kilmartin JV. Sfi1p has conserved centrin-binding sites and an essential function in budding yeast spindle pole body duplication. J Cell Biol. 2003;162(7):1211–21. Epub 2003/09/25.  pmid:14504268; PubMed Central PMCID: PMC2173958. <ul class="reflinks"><li><a href="#" data-author="Kilmartin" data-cit="KilmartinJV.%20Sfi1p%20has%20conserved%20centrin-binding%20sites%20and%20an%20essential%20function%20in%20budding%20yeast%20spindle%20pole%20body%20duplication.%20J%20Cell%20Biol.%202003%3B162%287%29%3A1211%E2%80%9321.%20Epub%202003%2F09%2F25.%20doi%3A%2010.1083%2Fjcb.200307064%2014504268%3B%20PubMed%20Central%20PMCID%3A%20PMC2173958." data-title="Sfi1p%20has%20conserved%20centrin-binding%20sites%20and%20an%20essential%20function%20in%20budding%20yeast%20spindle%20pole%20body%20duplication" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14504268" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sfi1p+has+conserved+centrin-binding+sites+and+an+essential+function+in+budding+yeast+spindle+pole+body+duplication+Kilmartin+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="pgen.1006304.ref052" id="pgen.1006304.ref052" class="link-target"></a>Kanoh J, Ishikawa F. spRap1 and spRif1, recruited to telomeres by Taz1, are essential for telomere function in fission yeast. Curr Biol. 2001;11(20):1624–30. Epub 2001/10/26. pmid:11676925. <ul class="reflinks"><li><a href="#" data-author="Kanoh" data-cit="KanohJ%2C%20IshikawaF.%20spRap1%20and%20spRif1%2C%20recruited%20to%20telomeres%20by%20Taz1%2C%20are%20essential%20for%20telomere%20function%20in%20fission%20yeast.%20Curr%20Biol.%202001%3B11%2820%29%3A1624%E2%80%9330.%20Epub%202001%2F10%2F26.%2011676925." data-title="spRap1%20and%20spRif1%2C%20recruited%20to%20telomeres%20by%20Taz1%2C%20are%20essential%20for%20telomere%20function%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11676925" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=spRap1+and+spRif1%2C+recruited+to+telomeres+by+Taz1%2C+are+essential+for+telomere+function+in+fission+yeast+Kanoh+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="pgen.1006304.ref053" id="pgen.1006304.ref053" class="link-target"></a>Fujita I, Tanaka M, Kanoh J. Identification of the functional domains of the telomere protein Rap1 in Schizosaccharomyces pombe. PLoS One. 2012;7(11):e49151. Epub 2012/11/08.  pmid:23133674; PubMed Central PMCID: PMC3487762. <ul class="reflinks"><li><a href="#" data-author="Fujita" data-cit="FujitaI%2C%20TanakaM%2C%20KanohJ.%20Identification%20of%20the%20functional%20domains%20of%20the%20telomere%20protein%20Rap1%20in%20Schizosaccharomyces%20pombe.%20PLoS%20One.%202012%3B7%2811%29%3Ae49151.%20Epub%202012%2F11%2F08.%20doi%3A%2010.1371%2Fjournal.pone.0049151%2023133674%3B%20PubMed%20Central%20PMCID%3A%20PMC3487762." data-title="Identification%20of%20the%20functional%20domains%20of%20the%20telomere%20protein%20Rap1%20in%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23133674" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+the+functional+domains+of+the+telomere+protein+Rap1+in+Schizosaccharomyces+pombe+Fujita+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="pgen.1006304.ref054" id="pgen.1006304.ref054" class="link-target"></a>Miyoshi T, Kanoh J, Saito M, Ishikawa F. Fission yeast Pot1-Tpp1 protects telomeres and regulates telomere length. Science. 2008;320(5881):1341–4. Epub 2008/06/07.  pmid:18535244. <ul class="reflinks"><li><a href="#" data-author="Miyoshi" data-cit="MiyoshiT%2C%20KanohJ%2C%20SaitoM%2C%20IshikawaF.%20Fission%20yeast%20Pot1-Tpp1%20protects%20telomeres%20and%20regulates%20telomere%20length.%20Science.%202008%3B320%285881%29%3A1341%E2%80%934.%20Epub%202008%2F06%2F07.%20doi%3A%2010.1126%2Fscience.1154819%2018535244." data-title="Fission%20yeast%20Pot1-Tpp1%20protects%20telomeres%20and%20regulates%20telomere%20length" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18535244" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Fission+yeast+Pot1-Tpp1+protects+telomeres+and+regulates+telomere+length+Miyoshi+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="pgen.1006304.ref055" id="pgen.1006304.ref055" class="link-target"></a>Takahashi K, Chen ES, Yanagida M. Requirement of Mis6 centromere connector for localizing a CENP-A-like protein in fission yeast. Science. 2000;288(5474):2215–9. Epub 2000/06/24. 8597 [pii]. pmid:10864871. <ul class="reflinks"><li><a href="#" data-author="Takahashi" data-cit="TakahashiK%2C%20ChenES%2C%20YanagidaM.%20Requirement%20of%20Mis6%20centromere%20connector%20for%20localizing%20a%20CENP-A-like%20protein%20in%20fission%20yeast.%20Science.%202000%3B288%285474%29%3A2215%E2%80%939.%20Epub%202000%2F06%2F24.%208597%20%5Bpii%5D.%2010864871." data-title="Requirement%20of%20Mis6%20centromere%20connector%20for%20localizing%20a%20CENP-A-like%20protein%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10864871" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Requirement+of+Mis6+centromere+connector+for+localizing+a+CENP-A-like+protein+in+fission+yeast+Takahashi+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="pgen.1006304.ref056" id="pgen.1006304.ref056" class="link-target"></a>Yokobayashi S, Watanabe Y. The kinetochore protein Moa1 enables cohesion-mediated monopolar attachment at meiosis I. Cell. 2005;123(5):803–17. Epub 2005/12/06.  pmid:16325576. <ul class="reflinks"><li><a href="#" data-author="Yokobayashi" data-cit="YokobayashiS%2C%20WatanabeY.%20The%20kinetochore%20protein%20Moa1%20enables%20cohesion-mediated%20monopolar%20attachment%20at%20meiosis%20I.%20Cell.%202005%3B123%285%29%3A803%E2%80%9317.%20Epub%202005%2F12%2F06.%20doi%3A%2010.1016%2Fj.cell.2005.09.013%2016325576." data-title="The%20kinetochore%20protein%20Moa1%20enables%20cohesion-mediated%20monopolar%20attachment%20at%20meiosis%20I" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16325576" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+kinetochore+protein+Moa1+enables+cohesion-mediated+monopolar+attachment+at+meiosis+I+Yokobayashi+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="pgen.1006304.ref057" id="pgen.1006304.ref057" class="link-target"></a>Chen Y, Rai R, Zhou ZR, Kanoh J, Ribeyre C, Yang Y, et al. A conserved motif within RAP1 has diversified roles in telomere protection and regulation in different organisms. Nature structural &amp; molecular biology. 2011;18(2):213–21. Epub 2011/01/11.  pmid:21217703; PubMed Central PMCID: PMC3688267. <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenY%2C%20RaiR%2C%20ZhouZR%2C%20KanohJ%2C%20RibeyreC%2C%20YangY%2C%20et%20al.%20A%20conserved%20motif%20within%20RAP1%20has%20diversified%20roles%20in%20telomere%20protection%20and%20regulation%20in%20different%20organisms.%20Nature%20structural%20%26%20molecular%20biology.%202011%3B18%282%29%3A213%E2%80%9321.%20Epub%202011%2F01%2F11.%20doi%3A%2010.1038%2Fnsmb.1974%2021217703%3B%20PubMed%20Central%20PMCID%3A%20PMC3688267." data-title="A%20conserved%20motif%20within%20RAP1%20has%20diversified%20roles%20in%20telomere%20protection%20and%20regulation%20in%20different%20organisms" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217703" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+conserved+motif+within+RAP1+has+diversified+roles+in+telomere+protection+and+regulation+in+different+organisms+Chen+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref58"><span class="order">58.
            </span><a name="pgen.1006304.ref058" id="pgen.1006304.ref058" class="link-target"></a>Yamamoto A, West RR, McIntosh JR, Hiraoka Y. A cytoplasmic dynein heavy chain is required for oscillatory nuclear movement of meiotic prophase and efficient meiotic recombination in fission yeast. J Cell Biol. 1999;145(6):1233–49. Epub 1999/06/15. pmid:10366596; PubMed Central PMCID: PMC2133150. <ul class="reflinks"><li><a href="#" data-author="Yamamoto" data-cit="YamamotoA%2C%20WestRR%2C%20McIntoshJR%2C%20HiraokaY.%20A%20cytoplasmic%20dynein%20heavy%20chain%20is%20required%20for%20oscillatory%20nuclear%20movement%20of%20meiotic%20prophase%20and%20efficient%20meiotic%20recombination%20in%20fission%20yeast.%20J%20Cell%20Biol.%201999%3B145%286%29%3A1233%E2%80%9349.%20Epub%201999%2F06%2F15.%2010366596%3B%20PubMed%20Central%20PMCID%3A%20PMC2133150." data-title="A%20cytoplasmic%20dynein%20heavy%20chain%20is%20required%20for%20oscillatory%20nuclear%20movement%20of%20meiotic%20prophase%20and%20efficient%20meiotic%20recombination%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10366596" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+cytoplasmic+dynein+heavy+chain+is+required+for+oscillatory+nuclear+movement+of+meiotic+prophase+and+efficient+meiotic+recombination+in+fission+yeast+Yamamoto+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref59"><span class="order">59.
            </span><a name="pgen.1006304.ref059" id="pgen.1006304.ref059" class="link-target"></a>West RR, Vaisberg EV, Ding R, Nurse P, McIntosh JR. <em>cut11</em><sup><em>+</em></sup>: A gene required for cell cycle-dependent spindle pole body anchoring in the nuclear envelope and bipolar spindle formation in Schizosaccharomyces pombe. Mol Biol Cell. 1998;9(10):2839–55. Epub 1998/10/08. pmid:9763447; PubMed Central PMCID: PMC25557. <ul class="reflinks"><li><a href="#" data-author="West" data-cit="WestRR%2C%20VaisbergEV%2C%20DingR%2C%20NurseP%2C%20McIntoshJR.%20cut11%2B%3A%20A%20gene%20required%20for%20cell%20cycle-dependent%20spindle%20pole%20body%20anchoring%20in%20the%20nuclear%20envelope%20and%20bipolar%20spindle%20formation%20in%20Schizosaccharomyces%20pombe.%20Mol%20Biol%20Cell.%201998%3B9%2810%29%3A2839%E2%80%9355.%20Epub%201998%2F10%2F08.%209763447%3B%20PubMed%20Central%20PMCID%3A%20PMC25557." data-title="cut11%2B%3A%20A%20gene%20required%20for%20cell%20cycle-dependent%20spindle%20pole%20body%20anchoring%20in%20the%20nuclear%20envelope%20and%20bipolar%20spindle%20formation%20in%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9763447" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=cut11%2B%3A+A+gene+required+for+cell+cycle-dependent+spindle+pole+body+anchoring+in+the+nuclear+envelope+and+bipolar+spindle+formation+in+Schizosaccharomyces+pombe+West+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref60"><span class="order">60.
            </span><a name="pgen.1006304.ref060" id="pgen.1006304.ref060" class="link-target"></a>Meyer RE, Kim S, Obeso D, Straight PD, Winey M, Dawson DS. Mps1 and Ipl1/Aurora B act sequentially to correctly orient chromosomes on the meiotic spindle of budding yeast. Science. 2013;339(6123):1071–4. Epub 2013/02/02.  pmid:23371552; PubMed Central PMCID: PMC3604795. <ul class="reflinks"><li><a href="#" data-author="Meyer" data-cit="MeyerRE%2C%20KimS%2C%20ObesoD%2C%20StraightPD%2C%20WineyM%2C%20DawsonDS.%20Mps1%20and%20Ipl1%2FAurora%20B%20act%20sequentially%20to%20correctly%20orient%20chromosomes%20on%20the%20meiotic%20spindle%20of%20budding%20yeast.%20Science.%202013%3B339%286123%29%3A1071%E2%80%934.%20Epub%202013%2F02%2F02.%20doi%3A%2010.1126%2Fscience.1232518%2023371552%3B%20PubMed%20Central%20PMCID%3A%20PMC3604795." data-title="Mps1%20and%20Ipl1%2FAurora%20B%20act%20sequentially%20to%20correctly%20orient%20chromosomes%20on%20the%20meiotic%20spindle%20of%20budding%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23371552" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mps1+and+Ipl1%2FAurora+B+act+sequentially+to+correctly+orient+chromosomes+on+the+meiotic+spindle+of+budding+yeast+Meyer+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref61"><span class="order">61.
            </span><a name="pgen.1006304.ref061" id="pgen.1006304.ref061" class="link-target"></a>Kim S, Meyer R, Chuong H, Dawson DS. Dual mechanisms prevent premature chromosome segregation during meiosis. Genes Dev. 2013;27(19):2139–46. Epub 2013/10/12.  pmid:24115770; PubMed Central PMCID: PMC3850097. <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimS%2C%20MeyerR%2C%20ChuongH%2C%20DawsonDS.%20Dual%20mechanisms%20prevent%20premature%20chromosome%20segregation%20during%20meiosis.%20Genes%20Dev.%202013%3B27%2819%29%3A2139%E2%80%9346.%20Epub%202013%2F10%2F12.%20doi%3A%2010.1101%2Fgad.227454.113%2024115770%3B%20PubMed%20Central%20PMCID%3A%20PMC3850097." data-title="Dual%20mechanisms%20prevent%20premature%20chromosome%20segregation%20during%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24115770" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Dual+mechanisms+prevent+premature+chromosome+segregation+during+meiosis+Kim+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref62"><span class="order">62.
            </span><a name="pgen.1006304.ref062" id="pgen.1006304.ref062" class="link-target"></a>Kim J, Ishiguro K, Nambu A, Akiyoshi B, Yokobayashi S, Kagami A, et al. Meikin is a conserved regulator of meiosis-I-specific kinetochore function. Nature. 2015;517(7535):466–71. Epub 2014/12/24.  pmid:25533956. <ul class="reflinks"><li><a href="#" data-author="Kim" data-cit="KimJ%2C%20IshiguroK%2C%20NambuA%2C%20AkiyoshiB%2C%20YokobayashiS%2C%20KagamiA%2C%20et%20al.%20Meikin%20is%20a%20conserved%20regulator%20of%20meiosis-I-specific%20kinetochore%20function.%20Nature.%202015%3B517%287535%29%3A466%E2%80%9371.%20Epub%202014%2F12%2F24.%20doi%3A%2010.1038%2Fnature14097%2025533956." data-title="Meikin%20is%20a%20conserved%20regulator%20of%20meiosis-I-specific%20kinetochore%20function" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25533956" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Meikin+is+a+conserved+regulator+of+meiosis-I-specific+kinetochore+function+Kim+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref63"><span class="order">63.
            </span><a name="pgen.1006304.ref063" id="pgen.1006304.ref063" class="link-target"></a>Chikashige Y, Yamane M, Okamasa K, Mori C, Fukuta N, Matsuda A, et al. Chromosomes rein back the spindle pole body during horsetail movement in fission yeast meiosis. Cell structure and function. 2014;39(2):93–100. Epub 2014/06/24. pmid:24954111. <ul class="reflinks"><li><a href="#" data-author="Chikashige" data-cit="ChikashigeY%2C%20YamaneM%2C%20OkamasaK%2C%20MoriC%2C%20FukutaN%2C%20MatsudaA%2C%20et%20al.%20Chromosomes%20rein%20back%20the%20spindle%20pole%20body%20during%20horsetail%20movement%20in%20fission%20yeast%20meiosis.%20Cell%20structure%20and%20function.%202014%3B39%282%29%3A93%E2%80%93100.%20Epub%202014%2F06%2F24.%2024954111." data-title="Chromosomes%20rein%20back%20the%20spindle%20pole%20body%20during%20horsetail%20movement%20in%20fission%20yeast%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24954111" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chromosomes+rein+back+the+spindle+pole+body+during+horsetail+movement+in+fission+yeast+meiosis+Chikashige+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref64"><span class="order">64.
            </span><a name="pgen.1006304.ref064" id="pgen.1006304.ref064" class="link-target"></a>Parra MT, Gomez R, Viera A, Llano E, Pendas AM, Rufas JS, et al. Sequential assembly of centromeric proteins in male mouse meiosis. PLoS genetics. 2009;5(3):e1000417. Epub 2009/03/14.  pmid:19283064; PubMed Central PMCID: PMC2652116. <ul class="reflinks"><li><a href="#" data-author="Parra" data-cit="ParraMT%2C%20GomezR%2C%20VieraA%2C%20LlanoE%2C%20PendasAM%2C%20RufasJS%2C%20et%20al.%20Sequential%20assembly%20of%20centromeric%20proteins%20in%20male%20mouse%20meiosis.%20PLoS%20genetics.%202009%3B5%283%29%3Ae1000417.%20Epub%202009%2F03%2F14.%20doi%3A%2010.1371%2Fjournal.pgen.1000417%2019283064%3B%20PubMed%20Central%20PMCID%3A%20PMC2652116." data-title="Sequential%20assembly%20of%20centromeric%20proteins%20in%20male%20mouse%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19283064" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sequential+assembly+of+centromeric+proteins+in+male+mouse+meiosis+Parra+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref65"><span class="order">65.
            </span><a name="pgen.1006304.ref065" id="pgen.1006304.ref065" class="link-target"></a>Li P, Shao Y, Jin H, Yu HG. Ndj1, a telomere-associated protein, regulates centrosome separation in budding yeast meiosis. J Cell Biol. 2015;209(2):247–59. Epub 2015/04/22.  pmid:25897084; PubMed Central PMCID: PMC4411264. <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiP%2C%20ShaoY%2C%20JinH%2C%20YuHG.%20Ndj1%2C%20a%20telomere-associated%20protein%2C%20regulates%20centrosome%20separation%20in%20budding%20yeast%20meiosis.%20J%20Cell%20Biol.%202015%3B209%282%29%3A247%E2%80%9359.%20Epub%202015%2F04%2F22.%20doi%3A%2010.1083%2Fjcb.201408118%2025897084%3B%20PubMed%20Central%20PMCID%3A%20PMC4411264." data-title="Ndj1%2C%20a%20telomere-associated%20protein%2C%20regulates%20centrosome%20separation%20in%20budding%20yeast%20meiosis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25897084" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Ndj1%2C+a+telomere-associated+protein%2C+regulates+centrosome+separation+in+budding+yeast+meiosis+Li+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref66"><span class="order">66.
            </span><a name="pgen.1006304.ref066" id="pgen.1006304.ref066" class="link-target"></a>Conrad MN, Lee CY, Chao G, Shinohara M, Kosaka H, Shinohara A, et al. Rapid telomere movement in meiotic prophase is promoted by NDJ1, MPS3, and CSM4 and is modulated by recombination. Cell. 2008;133(7):1175–87. Epub 2008/07/01.  pmid:18585352. <ul class="reflinks"><li><a href="#" data-author="Conrad" data-cit="ConradMN%2C%20LeeCY%2C%20ChaoG%2C%20ShinoharaM%2C%20KosakaH%2C%20ShinoharaA%2C%20et%20al.%20Rapid%20telomere%20movement%20in%20meiotic%20prophase%20is%20promoted%20by%20NDJ1%2C%20MPS3%2C%20and%20CSM4%20and%20is%20modulated%20by%20recombination.%20Cell.%202008%3B133%287%29%3A1175%E2%80%9387.%20Epub%202008%2F07%2F01.%20doi%3A%2010.1016%2Fj.cell.2008.04.047%2018585352." data-title="Rapid%20telomere%20movement%20in%20meiotic%20prophase%20is%20promoted%20by%20NDJ1%2C%20MPS3%2C%20and%20CSM4%20and%20is%20modulated%20by%20recombination" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18585352" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Rapid+telomere+movement+in+meiotic+prophase+is+promoted+by+NDJ1%2C+MPS3%2C+and+CSM4+and+is+modulated+by+recombination+Conrad+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref67"><span class="order">67.
            </span><a name="pgen.1006304.ref067" id="pgen.1006304.ref067" class="link-target"></a>Moreno S, Klar A, Nurse P. Molecular genetic analysis of fission yeast Schizosaccharomyces pombe. Methods in enzymology. 1991;194:795–823. Epub 1991/01/01. pmid:2005825. <ul class="reflinks"><li><a href="#" data-author="Moreno" data-cit="MorenoS%2C%20KlarA%2C%20NurseP.%20Molecular%20genetic%20analysis%20of%20fission%20yeast%20Schizosaccharomyces%20pombe.%20Methods%20in%20enzymology.%201991%3B194%3A795%E2%80%93823.%20Epub%201991%2F01%2F01.%202005825." data-title="Molecular%20genetic%20analysis%20of%20fission%20yeast%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2005825" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+genetic+analysis+of+fission+yeast+Schizosaccharomyces+pombe+Moreno+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref68"><span class="order">68.
            </span><a name="pgen.1006304.ref068" id="pgen.1006304.ref068" class="link-target"></a>Samejima I, Lourenco PC, Snaith HA, Sawin KE. Fission yeast mto2p regulates microtubule nucleation by the centrosomin-related protein mto1p. Mol Biol Cell. 2005;16(6):3040–51. Epub 2005/01/22.  pmid:15659644; PubMed Central PMCID: PMC1142446. <ul class="reflinks"><li><a href="#" data-author="Samejima" data-cit="SamejimaI%2C%20LourencoPC%2C%20SnaithHA%2C%20SawinKE.%20Fission%20yeast%20mto2p%20regulates%20microtubule%20nucleation%20by%20the%20centrosomin-related%20protein%20mto1p.%20Mol%20Biol%20Cell.%202005%3B16%286%29%3A3040%E2%80%9351.%20Epub%202005%2F01%2F22.%20doi%3A%2010.1091%2Fmbc.E04-11-1003%2015659644%3B%20PubMed%20Central%20PMCID%3A%20PMC1142446." data-title="Fission%20yeast%20mto2p%20regulates%20microtubule%20nucleation%20by%20the%20centrosomin-related%20protein%20mto1p" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659644" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Fission+yeast+mto2p+regulates+microtubule+nucleation+by+the+centrosomin-related+protein+mto1p+Samejima+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref69"><span class="order">69.
            </span><a name="pgen.1006304.ref069" id="pgen.1006304.ref069" class="link-target"></a>Takayama Y, Sato H, Saitoh S, Ogiyama Y, Masuda F, Takahashi K. Biphasic incorporation of centromeric histone CENP-A in fission yeast. Mol Biol Cell. 2008;19(2):682–90. Epub 2007/12/14.  pmid:18077559; PubMed Central PMCID: PMC2230595. <ul class="reflinks"><li><a href="#" data-author="Takayama" data-cit="TakayamaY%2C%20SatoH%2C%20SaitohS%2C%20OgiyamaY%2C%20MasudaF%2C%20TakahashiK.%20Biphasic%20incorporation%20of%20centromeric%20histone%20CENP-A%20in%20fission%20yeast.%20Mol%20Biol%20Cell.%202008%3B19%282%29%3A682%E2%80%9390.%20Epub%202007%2F12%2F14.%20doi%3A%2010.1091%2Fmbc.E07-05-0504%2018077559%3B%20PubMed%20Central%20PMCID%3A%20PMC2230595." data-title="Biphasic%20incorporation%20of%20centromeric%20histone%20CENP-A%20in%20fission%20yeast" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18077559" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Biphasic+incorporation+of+centromeric+histone+CENP-A+in+fission+yeast+Takayama+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref70"><span class="order">70.
            </span><a name="pgen.1006304.ref070" id="pgen.1006304.ref070" class="link-target"></a>Nabetani A, Koujin T, Tsutsumi C, Haraguchi T, Hiraoka Y. A conserved protein, Nuf2, is implicated in connecting the centromere to the spindle during chromosome segregation: a link between the kinetochore function and the spindle checkpoint. Chromosoma. 2001;110:322–34. pmid:11685532 <ul class="reflinks"><li><a href="#" data-author="Nabetani" data-cit="NabetaniA%2C%20KoujinT%2C%20TsutsumiC%2C%20HaraguchiT%2C%20HiraokaY.%20A%20conserved%20protein%2C%20Nuf2%2C%20is%20implicated%20in%20connecting%20the%20centromere%20to%20the%20spindle%20during%20chromosome%20segregation%3A%20a%20link%20between%20the%20kinetochore%20function%20and%20the%20spindle%20checkpoint.%20Chromosoma.%202001%3B110%3A322%E2%80%9334.%2011685532" data-title="A%20conserved%20protein%2C%20Nuf2%2C%20is%20implicated%20in%20connecting%20the%20centromere%20to%20the%20spindle%20during%20chromosome%20segregation%3A%20a%20link%20between%20the%20kinetochore%20function%20and%20the%20spindle%20checkpoint" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/11685532" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=A+conserved+protein%2C+Nuf2%2C+is+implicated+in+connecting+the+centromere+to+the+spindle+during+chromosome+segregation%3A+a+link+between+the+kinetochore+function+and+the+spindle+checkpoint+Nabetani+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref71"><span class="order">71.
            </span><a name="pgen.1006304.ref071" id="pgen.1006304.ref071" class="link-target"></a>Chikashige Y, Kurokawa R, Haraguchi T, Hiraoka Y. Meiosis induced by inactivation of Pat1 kinase proceeds with aberrant nuclear positioning of centromeres in the fission yeast Schizosaccharomyces pombe. Genes Cells. 2004;9(8):671–84. Epub 2004/08/10.  pmid:15298676. <ul class="reflinks"><li><a href="#" data-author="Chikashige" data-cit="ChikashigeY%2C%20KurokawaR%2C%20HaraguchiT%2C%20HiraokaY.%20Meiosis%20induced%20by%20inactivation%20of%20Pat1%20kinase%20proceeds%20with%20aberrant%20nuclear%20positioning%20of%20centromeres%20in%20the%20fission%20yeast%20Schizosaccharomyces%20pombe.%20Genes%20Cells.%202004%3B9%288%29%3A671%E2%80%9384.%20Epub%202004%2F08%2F10.%20doi%3A%2010.1111%2Fj.1356-9597.2004.00760.x%2015298676." data-title="Meiosis%20induced%20by%20inactivation%20of%20Pat1%20kinase%20proceeds%20with%20aberrant%20nuclear%20positioning%20of%20centromeres%20in%20the%20fission%20yeast%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15298676" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Meiosis+induced+by+inactivation+of+Pat1+kinase+proceeds+with+aberrant+nuclear+positioning+of+centromeres+in+the+fission+yeast+Schizosaccharomyces+pombe+Chikashige+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref72"><span class="order">72.
            </span><a name="pgen.1006304.ref072" id="pgen.1006304.ref072" class="link-target"></a>Krawchuk MD, Wahls WP. High-efficiency gene targeting in Schizosaccharomyces pombe using a modular, PCR-based approach with long tracts of flanking homology. Yeast. 1999;15(13):1419–27. Epub 1999/10/06.  pmid:10509024; PubMed Central PMCID: PMC3190138. <ul class="reflinks"><li><a href="#" data-author="Krawchuk" data-cit="KrawchukMD%2C%20WahlsWP.%20High-efficiency%20gene%20targeting%20in%20Schizosaccharomyces%20pombe%20using%20a%20modular%2C%20PCR-based%20approach%20with%20long%20tracts%20of%20flanking%20homology.%20Yeast.%201999%3B15%2813%29%3A1419%E2%80%9327.%20Epub%201999%2F10%2F06.%20doi%3A%2010.1002%2F%28SICI%291097-0061%2819990930%2915%3A13%3C1419%3A%3AAID-YEA466%3E3.0.CO%3B2-Q%2010509024%3B%20PubMed%20Central%20PMCID%3A%20PMC3190138." data-title="High-efficiency%20gene%20targeting%20in%20Schizosaccharomyces%20pombe%20using%20a%20modular%2C%20PCR-based%20approach%20with%20long%20tracts%20of%20flanking%20homology" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10509024" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=High-efficiency+gene+targeting+in+Schizosaccharomyces+pombe+using+a+modular%2C+PCR-based+approach+with+long+tracts+of+flanking+homology+Krawchuk+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref73"><span class="order">73.
            </span><a name="pgen.1006304.ref073" id="pgen.1006304.ref073" class="link-target"></a>Bahler J, Wu JQ, Longtine MS, Shah NG, McKenzie A 3rd, Steever AB, et al. Heterologous modules for efficient and versatile PCR-based gene targeting in Schizosaccharomyces pombe. Yeast. 1998;14(10):943–51. Epub 1998/08/26.  pmid:9717240. <ul class="reflinks"><li><a href="#" data-author="Bahler" data-cit="BahlerJ%2C%20WuJQ%2C%20LongtineMS%2C%20ShahNG%2C%20McKenzieA3rd%2C%20SteeverAB%2C%20et%20al.%20Heterologous%20modules%20for%20efficient%20and%20versatile%20PCR-based%20gene%20targeting%20in%20Schizosaccharomyces%20pombe.%20Yeast.%201998%3B14%2810%29%3A943%E2%80%9351.%20Epub%201998%2F08%2F26.%20doi%3A%2010.1002%2F%28SICI%291097-0061%28199807%2914%3A10%3C943%3A%3AAID-YEA292%3E3.0.CO%3B2-Y%209717240." data-title="Heterologous%20modules%20for%20efficient%20and%20versatile%20PCR-based%20gene%20targeting%20in%20Schizosaccharomyces%20pombe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9717240" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Heterologous+modules+for+efficient+and+versatile+PCR-based+gene+targeting+in+Schizosaccharomyces+pombe+Bahler+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref74"><span class="order">74.
            </span><a name="pgen.1006304.ref074" id="pgen.1006304.ref074" class="link-target"></a>Nabeshima K, Nakagawa T, Straight AF, Murray A, Chikashige Y, Yamashita YM, et al. Dynamics of centromeres during metaphase-anaphase transition in fission yeast: Dis1 is implicated in force balance in metaphase bipolar spindle. Mol Biol Cell. 1998;9(11):3211–25. Epub 1998/11/05. pmid:9802907; PubMed Central PMCID: PMC25611. <ul class="reflinks"><li><a href="#" data-author="Nabeshima" data-cit="NabeshimaK%2C%20NakagawaT%2C%20StraightAF%2C%20MurrayA%2C%20ChikashigeY%2C%20YamashitaYM%2C%20et%20al.%20Dynamics%20of%20centromeres%20during%20metaphase-anaphase%20transition%20in%20fission%20yeast%3A%20Dis1%20is%20implicated%20in%20force%20balance%20in%20metaphase%20bipolar%20spindle.%20Mol%20Biol%20Cell.%201998%3B9%2811%29%3A3211%E2%80%9325.%20Epub%201998%2F11%2F05.%209802907%3B%20PubMed%20Central%20PMCID%3A%20PMC25611." data-title="Dynamics%20of%20centromeres%20during%20metaphase-anaphase%20transition%20in%20fission%20yeast%3A%20Dis1%20is%20implicated%20in%20force%20balance%20in%20metaphase%20bipolar%20spindle" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9802907" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Dynamics+of+centromeres+during+metaphase-anaphase+transition+in+fission+yeast%3A+Dis1+is+implicated+in+force+balance+in+metaphase+bipolar+spindle+Nabeshima+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

