



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Red Junglefowls (RJFs), the wild progenitor of modern day chickens (DCs), are believed to be in genetic endangerment due to introgression of domestic genes through opportunistic matings with domestic or feral chickens. Previous studies from India reported rare hybridization of RJFs in the wild. However, RJF population genetic structure, pattern of gene flow and their admixture with DC populations are poorly understood at the landscape level. We conducted this study with a large sample size, covering the predicted natural distribution range of RJFs in India. We documented strong evidence of directional gene flow from DCs to free-ranging wild RJFs, with the Northeastern RJF population exhibiting the most genetic variants in their nuclear and mitochondrial genomes, indicating it to be the ancestral population from which early radiation may have occurred. The results provide evidence that landscape features do not act as a barrier to gene flow and the distribution pattern could not be explored due to physical sharing or exchange of wild birds in the past when forests were continuous across RJF range in India.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Thakur M, Fernandes M, Sathyakumar S, Singh SK, Vijh RK, Han J, et al.  (2018) Understanding the cryptic introgression and mixed ancestry of Red Junglefowl in India. PLoS ONE 13(10):
           e0204351.
        
        https://doi.org/10.1371/journal.pone.0204351</p><p><strong>Editor: </strong>Gyaneshwer Chaubey, Banaras Hindu University, INDIA</p><p><strong>Received: </strong>April 20, 2018; <strong>Accepted: </strong>September 6, 2018; <strong>Published: </strong> October 11, 2018</p><p><strong>Copyright: </strong> © 2018 Thakur et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>Data available from the Dryad Digital Repository: <a href="https://doi.org/10.5061/dryad.rv38cp4">https://doi.org/10.5061/dryad.rv38cp4</a>.</p><p><strong>Funding: </strong>The project was funded through the grant in aid scheme of Wildlife Institute of India, Dehradun. Funds were raised by Dr. S. Sathyakumar.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>The polyphyletic origins of Domestic Chickens (DCs, <em>Gallus gallus domesticus</em>) is a reason to speculate that gene flow between RJFs (<em>Gallus gallus murghi</em>) and DCs is widespread and more frequent than supposed by previous studies [<a href="#pone.0204351.ref001" class="ref-tip">1</a>,<a href="#pone.0204351.ref002" class="ref-tip">2</a>,<a href="#pone.0204351.ref003" class="ref-tip">3</a>] while domestication may have occurred at multiple locations in South and South-East Asia [<a href="#pone.0204351.ref004" class="ref-tip">4</a>,<a href="#pone.0204351.ref005" class="ref-tip">5</a>,<a href="#pone.0204351.ref006" class="ref-tip">6</a>,<a href="#pone.0204351.ref007" class="ref-tip">7</a>]. However, cryptic introgression from domestic or feral DCs to RJFs or viceversa and the transport of DCs amongst different regions obscure the history of these two species. Several studies have suggested physical mixing and gene flow between RJF in the wild and DC populations [<a href="#pone.0204351.ref004" class="ref-tip">4</a>,<a href="#pone.0204351.ref008" class="ref-tip">8</a>,<a href="#pone.0204351.ref009" class="ref-tip">9</a>, <a href="#pone.0204351.ref010" class="ref-tip">10</a>,<a href="#pone.0204351.ref011" class="ref-tip">11</a>]. Interestingly, Gering <em>et al</em>. [<a href="#pone.0204351.ref012" class="ref-tip">12</a>] reported feralisation of <em>Kauai</em> chicken through invasive genetics and further raised the issue of <em>'domestication in reverse'</em>. In general, hybridization in the absence of reproductive isolation is an inevitable phenomenon and cannot be avoided in cases where domestic and wild congenerics are sympatric [<a href="#pone.0204351.ref013" class="ref-tip">13</a>,<a href="#pone.0204351.ref014" class="ref-tip">14</a>,<a href="#pone.0204351.ref015" class="ref-tip">15</a>,<a href="#pone.0204351.ref016" class="ref-tip">16</a>]. The situation gets complex when hybrid offsprings are reproductively viable and participate subsequent mating across the species. Allendorf <em>et al</em>. [<a href="#pone.0204351.ref017" class="ref-tip">17</a>] stated that 5% or less proportion of hybridization in RJFs is an effect of admixture or natural selection whereas another study, based largely on birds reared in captivity and released into the wild, reported rare hybridization between RJFs and DCs in the wild in India [<a href="#pone.0204351.ref005" class="ref-tip">5</a>]. Berthouly <em>et al</em>. [<a href="#pone.0204351.ref006" class="ref-tip">6</a>] postulated that their observation of low genetic exchange might be due to sampling bias and reported a fair gene flow from RJF to local Vietnamese chicken populations.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>RJFs in India are widely distributed across 51 x 10<sup>5</sup> km<sup>2</sup> in 21 States [<a href="#pone.0204351.ref018" class="ref-tip">18</a>]. Further, based on our field observations and monitoring on RJFs in the wild, we often encountered RJFs and DCs feeding in the same flocks in the vicinity of forest habitats [<a href="#pone.0204351.ref019" class="ref-tip">19</a>]. We believe that the threat of hybridization to RJFs with DCs has not been addressed appropriately at a landscape level. In addition, the extent of hybridization between wild RJFs and DCs is stressed to be of importance in the International Union for Conservation of Nature (IUCN) Action Plan for Pheasants (2000). However, IUCN listing of RJFs as “Least Concern”, the non-listing of RJFs on the Convention of International Trade in Endangered Species of Wild Fauna and Flora (CITES) [<a href="#pone.0204351.ref020" class="ref-tip">20</a>], and the present inclusion of RJFs in the Wildlife (Protection) Act, 1972 of India has no provisions to assess the hybridization threat to this speciesdespite a multi-billion dollar poultry industry has evolved through wild RJF. Recent poultry epidemics, such as the one in Hong Kong in 1998 and the ‘bird flu’ in India and other parts of S.E. Asia, could spell doom to the poultry industry and the only fallback option the poultry farmers would eventually be the ‘wild’ RJF [<a href="#pone.0204351.ref018" class="ref-tip">18</a>].</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Further, one of the primary premises in present-day conservation programs is to maximize the conservation of genetic diversity available for potential future use. If hybridization of RJFs with and DCs occurrs and continues, it would produce populations which may not be valued for future breeding and conservation purposes under the IUCN guidelines. Considering the importance of conservation concern to safeguard the wild ancestor of DCs, we undertook this study to answer two important questions:</p>


<ol class="order">

<li>Whether or not, the threat of hybridization and genetic exchange between RJF and DC in India is significant or rare as documented by earlier studies.</li>

<li>If such hybridization occurs or has occurred in India in the past, whether it is localized with specific distribution pattern and how does it affect the current population genetic structure of RJF?</li>

</ol></div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Materials and methods"></a><h2>Materials and methods</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="Sample collection and DNA extraction"></a>
<h3>Sample collection and DNA extraction</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>For sampling, we divided India into five zones based on the availability of continuous habitats for RJF viz., <em>North</em> the States of Jammu and Kashmir, Himachal Pradesh, Uttarakhand, Haryana, Punjab and Uttar Pradesh, <em>Central</em> the States of Madhya Pradesh and Chhattisgarh, <em>East</em> the States of Bihar, Jharkhand, West Bengal and Sikkim, <em>Southeast</em> the States of Odisha and Andhra Pradesh and <em>Northeast</em> the States of Assam, Arunachal Pradesh, Nagaland, Mizoram, Manipur, Tripura and Meghalaya. We collected blood samples from 57 wild RJF and 79 DC individuals across India (<a href="#pone-0204351-g001">Fig 1</a>). The wild RJF were live trapped and approximately 500μl blood from each bird was withdrawn from the brachial vein and stored in DNA zol BD (Invitrogen TM, Carlsbad, CA, USA). All DC samples were collected in the vicinity of wild RJFs adjacent to the forests. The genomic DNA from whole blood was extracted following Mackey <em>et al</em>. [<a href="#pone.0204351.ref021" class="ref-tip">21</a>].</p>
<a class="link-target" id="pone-0204351-g001" name="pone-0204351-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g001" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> India geographic map showing sampling locations of RJFs.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g001">
              https://doi.org/10.1371/journal.pone.0204351.g001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="Microsatellite genotyping and sequencing of D-loop region"></a>
<h3>Microsatellite genotyping and sequencing of D-loop region</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>We genotyped samples with 30 microsatellite loci (ADL0268, MCW0206, LEI0166, MCW0020, MCW0037, ADL0112, MCW0295, MCW0067, MCW0104, MCW0111, MCW0034, MCW0222, LEI0094, MCW0216, MCW0081, MCW0330, LEI0234, MCW0103, MCW0098, MCW0069, MCW0016, MCW0078, MCW0123, MCW0165, MCW0248, ADL0278, LEI0192, MCW0014, MCW0183 and MCW0284). We scrutinized these loci based on their wide coverage on the genome and high degree of polymorphism [<a href="#pone.0204351.ref022" class="ref-tip">22</a>,<a href="#pone.0204351.ref023" class="ref-tip">23</a>,<a href="#pone.0204351.ref024" class="ref-tip">24</a>]. The PCR composition and cycling conditions followed standard procedures outlined elsewhere [<a href="#pone.0204351.ref025" class="ref-tip">25</a>,<a href="#pone.0204351.ref026" class="ref-tip">26</a>]. The fluorescent-based genotyping was performed through capillary electrophoresis on an ABI 3130 Genetic Analyzer (Applied Biosystems, Foster City, CA, USA). The allele scoring was done manually using GeneMapper (ver. 3.7) (Applied Biosystems, Foster City, CA, USA).</p>
<a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>To amplify mitochondrial DNA D-loop hypervariable region (HV1), specific primers L16750 (<span class="monospace">5′-AGGACTACGGCTTGAAAAGC-3′</span>) as theforward primer and H547 (<span class="monospace">5′-ATGTGCCTGACCGAGGAACCAG-3′</span>) as reverse primer were used. This primer pair amplified a 550 bp fragment between sites 16750 (GenBank accession number NC_001323; [<a href="#pone.0204351.ref027" class="ref-tip">27</a>]) and 547 (GenBank accession number AB098668; [<a href="#pone.0204351.ref028" class="ref-tip">28</a>]). The PCRs were conducted in 10 μl reaction—0.25 mM MgCl<sub>2</sub>, 1 μM each of forward and reverse primers, 0.25 mM dNTPs and 1 unit <em>Taq</em> polymerase. Thermocycling conditions of PCR consisted of an initial denaturation at 94°C for 2 min, followed by 35 cycles of denaturation at 94°C for 30 seconds, an annealing step at 52°C for 20 seconds, and an extension at 72°C for 60 seconds and finished by a final extension at 72°C for 10 min. PCR products were purified using the QIAquick PCR purification kit (QIAGEN, GmbH, Germany) according to the manufacturer’s protocol. Direct sequencing of a HV1 segment of the D-loop region was performed using two internal primers CR-for (<span class="monospace">5′-TCTATATTCCACATTTCTC-3′) a</span>nd CR-rev (<span class="monospace">5′-GCGAGCATAACCAAATGG-3′</span>). Sequencing was done using the BigDye Terminator (ver. 3.1) Cycle Sequencing Kit (Applied Biosystems, Foster City, CA, USA) and the purified sequencing products were electrophoresed on the ABI 3130 Genetic Analyzer.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Statistical analysis"></a>
<h3>Statistical analysis</h3>

<div id="section1" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="(a) Microsatellites"></a><h4>(a) Microsatellites.</h4><a id="article1.body1.sec2.sec3.sec1.p1" name="article1.body1.sec2.sec3.sec1.p1" class="link-target"></a><p><span class="underline">Genetic variability, inbreeding and population differentiation.</span> Genetic diversity estimates <em>i</em>.<em>e</em>. observed (Na) andeffective numbers of alleles (ne), observed (Ho), expected (He) and unbiased expected heterozygosity (UHe) and pair‐wise <em>F</em><sub><em>ST</em></sub> were estimated using GENEALEX (ver. 6.5) [<a href="#pone.0204351.ref029" class="ref-tip">29</a>]. The mean allelic richness (A<sub>R</sub>), which is an estimate of an independent sample size for each population and the polymorphic information content (PIC)were estimated using FSTAT (ver.2.9.3.2) [<a href="#pone.0204351.ref030" class="ref-tip">30</a>] and Cervus (ver. 3.0) [<a href="#pone.0204351.ref031" class="ref-tip">31</a>], respectively. We tested any deviation from Hardy–Weinberg equilibrium (HWE) at each locus following probability test approach involving 10,000 dememorizations, 500 batches and 10,000 iterations per batch in GENEPOP (ver. 4.2) [<a href="#pone.0204351.ref032" class="ref-tip">32</a>, <a href="#pone.0204351.ref033" class="ref-tip">33</a>]. The <em>F</em><sub><em>IS</em></sub> following Weir and Cockerham [<a href="#pone.0204351.ref034" class="ref-tip">34</a>] method and Linkage disequilibrium (LD) were tested using the a log likelihood ratio statistic in GENEPOP (ver. 4.2) [<a href="#pone.0204351.ref033" class="ref-tip">33</a>].</p>
<a id="article1.body1.sec2.sec3.sec1.p2" name="article1.body1.sec2.sec3.sec1.p2" class="link-target"></a><p><span class="underline">Population genetic structure.</span> Population genetic structure of RJF in India was inferred using the Bayesian clustering algorithm implemented in program STRUCTURE (ver.2.3.3) [<a href="#pone.0204351.ref035" class="ref-tip">35</a>] following a model assuming <em>K</em> populations (e.g. 1 to 10). We selected an admixture model with a burn‐in period of 50,000 and 500,000 Markov chain Monte Carlo repetitions as well as a model of correlated allele frequencies without prior sampling location information. Twenty independent runs at each <em>K</em> value were performed. The most appropriate <em>K</em> value was determined by calculating ad hoc quantity (ΔK) as proposed by Evanno <em>et al</em>. [<a href="#pone.0204351.ref036" class="ref-tip">36</a>] and each individual was assigned to the inferred clusters using a threshold proportion of membership (q), <em>i</em>.<em>e</em>. q ≥ 0.80, following Mukesh <em>et al</em>. [<a href="#pone.0204351.ref026" class="ref-tip">26</a>, <a href="#pone.0204351.ref037" class="ref-tip">37</a>, <a href="#pone.0204351.ref038" class="ref-tip">38</a>]. The clustering results of STRUCTURE were visualized over ClumpaK (<a href="http://clumpak.tau.ac.il/index.html">http://clumpak.tau.ac.il/index.html</a>), a web server that provides a full pipeline for clustering, summarizing and visualizing the STRUCTURE results.</p>
<a id="article1.body1.sec2.sec3.sec1.p3" name="article1.body1.sec2.sec3.sec1.p3" class="link-target"></a><p><span class="underline">Spatial genetics and barrier detection.</span> As sampling was done across the species distribution in India, we performed a spatial Bayesian clustering analysis in GENELAND (ver.) 4.0.3 [<a href="#pone.0204351.ref039" class="ref-tip">39</a>] to detect barriers in the gene flow between free-ranging RJF populations. Unlike population assignment in STRUCTURE, this analysis also takes into account the spatial coordinates (<em>i</em>.<em>e</em>. latitude and longitude) of each individual along with their multi-locus genotypes for specifically identifying genetic breaks while generating maps of population ranges. We employed the spatial model to infer the most likely number of clusters indicated as ‘K’ with either correlated or uncorrelated allele frequency model. In these analyses, we conducted 30 independent runs for each K ranging from 1 to 10, with 1,000,000 iterations and 1,000 thinning. Other parameters were set to default values (maximum rate of the Poisson process, 100; uncertainty on spatial coordinates, 0; maximum number of nuclei, 300; null allele model, FALSE). The top three runs showing the highest average logarithm of the posterior probability were selected and post-processing was conducted using 100 x 100 pixels in the spatial domain with a burnin period of 200.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec007" name="sec007" class="link-target" title="(b) Mitochondrial D-loop region"></a><h4>(b) Mitochondrial D-loop region.</h4><a id="article1.body1.sec2.sec3.sec2.p1" name="article1.body1.sec2.sec3.sec2.p1" class="link-target"></a><p><span class="underline">Genetic variability and demographic history.</span> We checked all raw sequences using Sequencher ver. 4.7 (<a href="http://www.genecode.com/">www.genecode.com</a>) and any ambiguity, if encountered, was manually corrected. The cleaned sequences were aligned using CLUSTAL W-multiple sequence alignment algorithm, implemented in BioEdit version 7.2.5 (<a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html">http://www.mbio.ncsu.edu/BioEdit/bioedit.html</a>). Using DnaSP (ver.5.10) [<a href="#pone.0204351.ref040" class="ref-tip">40</a>], diversity estimates <em>i</em>.<em>e</em>. polymorphic sites (S), number of haplotypes (H), nucleotide diversity (π), haplotype diversity (Hd), average number of nucleotide differences (K) and mismatch distribution test for demographic expansion, equilibrium or bottleneck [<a href="#pone.0204351.ref041" class="ref-tip">41</a>] were computed. Neutrality tests, <em>i</em>.<em>e</em>. Tajima’s D [<a href="#pone.0204351.ref042" class="ref-tip">42</a>], Fu’s Fs [<a href="#pone.0204351.ref043" class="ref-tip">43</a>] and Fu and Li’s F and D [<a href="#pone.0204351.ref044" class="ref-tip">44</a>] were carried out to evaluate the demographic effects.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="(c) Genetic introgression and clustering of RJFs with DCs"></a><h4>(c) Genetic introgression and clustering of RJFs with DCs.</h4><a id="article1.body1.sec2.sec3.sec3.p1" name="article1.body1.sec2.sec3.sec3.p1" class="link-target"></a><p>Program STRUCTURE with the same parameters as described above was used with the inclusion of DC samples collected in the vicinity of RJFs to assign the admixed individuals. To evaluate the direction of gene flow, haplotypes shared, and their frequencies of occurrence between RJFs and DCs were examined using the phylogenetic network based on the median-joining method [<a href="#pone.0204351.ref045" class="ref-tip">45</a>] implemented inNETWORK ver. 4.5.1 (<a href="http://www.fluxusengineering.com/">http://www.fluxusengineering.com</a>). In NETWORK, we assigned equal weights to all variable sites and applied default values for the epsilon parameter (epsilon = 0). The microsatellite and sequencing data phylogenetic trees constructed were used to investigate the recent and long-term admixture and the genetic relationship between wild RJF and DC populations. For microsatellite data, a genetic distance matrix was calculated using Program GENALEX (ver. 6.5) [<a href="#pone.0204351.ref029" class="ref-tip">29</a>], and a phylogenetic tree was constructed following the maximum likelihood (ML) method in MEGA program version 7.0 [<a href="#pone.0204351.ref046" class="ref-tip">46</a>]. For sequencing data, maximum likelihood method and the Tamura 3 parameter, the most fit substitution model, were considered to reconstruct the phylogenetic tree using MEGA program version 7.0 program [<a href="#pone.0204351.ref046" class="ref-tip">46</a>].</p>
</div>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec009" name="sec009" data-toc="sec009" class="link-target" title="Results"></a><h2>Results</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>Among the 57 RJF and 79 DC samples collected, there was 68 (32 RJFs and 36 DCs) samples from North, 25 (9 RJFs and 16 DCs) from East, 5 (2 RJFs and 3 DCs) from Central, 5 (4 RJFs and 1 DC) from South East and 33 (10 RJFs and 23 DCs) from Northeast of India. Since, we can trap only a few wild RJFs in Central and Southeast zones; we pooled samples of these zones to create a Cent-Southeast population for further analysis. During analysis, two birds- RJ1 (Cent-Southeast) and RJ9 (Northeast) were excluded due to uncertainty in the GPS locations of their sample points.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>We genotyped all samples three times or repeated the process until we generated consensus genotypes for all samples. However, four loci <em>i</em>.<em>e</em>. LEI0192, MCW0014, MCW0183 and MCW0284 exhibited a considerable amount of missing values for few a samples even after multiple repetition, so we removed them from further analysis. We manually checked allelic data, and found no indication of any genotyping error (Data available on the Dryad Digital Repository on <a href="https://doi.org/10.5061/dryad.rv38cp4">https://doi.org/10.5061/dryad.rv38cp4</a>).</p>

<div id="section1" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="Genetic diversity indices and demographic history"></a>
<h3>Genetic diversity indices and demographic history</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>All 26 microsatellite loci were polymorphic, yielding a total of 692 alleles across the four RJF populations. The average Na and ne ranged from 4.27±0.34 (Cent-Southeast) to 9.69±0.76 (North) and from 3.11±0.27 (Cent-Southeast) to 5.02±0.35 (North), respectively. The Ho ranged from 0.52 (±0.04 North; ±0.06 Cent-Southeast) to 0.58 (±0.05 East; ± 0.04 Northeast) and did not differ significantly among the four populations. The mean PIC values were greater than 0.5 for all the four RJF populations. The UHeranged from 0.67±0.04 in Cent-Southeast to 0.81±0.02 in Northeast population, and the average A<sub>R</sub> (rarefaction after two diploid individuals per population) ranged from 2.59±0.13 in Cent-Southeast to 3.04±0.07 in Northeast population. Total count of N<sub>P</sub> was the highest in North (85 alleles) but the lowest in Cent-Southeast (12 alleles) RJF populations. The <em>F</em><sub><em>IS</em></sub> estimates in the four RJF populations were significantly positive (P&lt;0.001), suggesting that all the populations were slightly inbred. The number of loci deviated from HWE (P&lt;0.05) ranged from four in Cent-Southeast population to 22 in North population with the majority of them showing a heterozygote deficit. Nevertheless only two loci (MCW0295 and MCW0123) deviated from HWE across the four populations (P&lt;0.05; <a href="#pone.0204351.s001">S1 Table</a>). Several pairs of loci showed significant LD (P&lt;0.05; <a href="#pone.0204351.s002">S2 Table</a>), but none of them was present across the four populations.</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>For D-loop data, we analysed 100 samples, <em>i</em>.<em>e</em>.38 wild RJF (15 North, 7 East, 6 Cent-Southeast and 10 Northeast) and 62 DC collected in the vicinity of the RJF. Wild RJF yielded 37 polymorphic sites which formed 31 haplotypes with an average of 6.48 nucleotide differences. All analyzed samples yielded 68 haplotypes, among which 37 were found in DCs. Most haplotypes were present in one sample each. Haplotypes # 1, 12 and 13 were shared among the individuals of the same population while haplotype 3 was shared between North and Northeast populations (<a href="#pone.0204351.s006">S1 Fig</a>). Novel haplotypes were deposited in the GenBank database (Accession no. MG053424 to MG053491). Northeast population represented the highest diversity estimates with 26 polymorphic sites, 10 haplotypes and 8.71 nucleotide differences. The overall haplotype (Hd) and nucleotide diversity (π) were 0.98 ±0.014 and 0.01±0.001, respectively (<a href="#pone-0204351-t001">Table 1</a>). All four RJF populations showed a multimodal pattern of mismatch distribution, supporting these populations to be under demographic equilibrium and without any bottleneck (<a href="#pone.0204351.s007">S2 Fig</a>). The estimates of neutrality tests, in general, showed a similar pattern revealed by mismatch distribution curve while negative estimates of Tajima’s D and Fu’s Fs statistic tests implied that rare alleles were more common than expected. However, the observed estimates were not significant (P&gt;0.10), suggesting that populations have not undergone any expansion. Fu and Li’s D and F tests also indicated no significant departure from neutrality (P&gt;0.10).</p>
<a class="link-target" id="pone-0204351-t001" name="pone-0204351-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.t001" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Summary of molecular genetic diversity indices of wild RJF populations.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.t001">
              https://doi.org/10.1371/journal.pone.0204351.t001</a></p></div><a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>Pair-wise <em>F</em><sub><em>ST</em></sub> values between all four RJF populations were significant (P&lt;0.01; <a href="#pone-0204351-t002">Table 2</a>) for both the microsatellite and D-loop data. The highest differentiation was found between North and Cent-Southeast populations (0.10 for microsatellites and 0.30 for D loop data). However, similar estimates were observed between North and Northeast populations for microsatellites (0.05) and D-loop sequences (0.06).</p>
<a class="link-target" id="pone-0204351-t002" name="pone-0204351-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.t002" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Comparison of pair-wise <em>F</em><sub><em>ST</em></sub> estimates between four wild RJF and four DC populations.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.t002">
              https://doi.org/10.1371/journal.pone.0204351.t002</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="Inference of population genetic structure and clustering patterns on the spatial scale"></a>
<h3>Inference of population genetic structure and clustering patterns on the spatial scale</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>The <em>ad hoc</em> quantity value was the highest at K = 2 (ΔK -139.8081; <a href="#pone-0204351-g002">Fig 2B</a>; <a href="#pone-0204351-t003">Table 3</a>). All individuals were successfully assigned to two clusters, <em>cluster 1–</em>54.38% and <em>cluster 2–</em>45.61% based on their posterior probability values <em>i</em>.<em>e</em>. q ≥ 0.80. A further increase in the <em>K</em> value did not split the samples into additional clusters (<a href="#pone-0204351-g003">Fig 3</a>). Interestingly, birds from North and Northeast populations were assigned to both clusters, indicating their shared ancestry in the past. However, nearly all birds of East and Cent-Southeast populations were assigned to <em>cluster</em> 1.The global performance of STRUCTURE in assigning all individuals at K = 2 was 100% (<a href="#pone-0204351-t003">Table 3</a>).</p>
<a class="link-target" id="pone-0204351-g002" name="pone-0204351-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g002" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Graphical representation of the true number of the cluster in wild RJF populations.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>(A) Mean L(K) (±SD) over 20 independent runs for each <em>K</em>-value. (B) The ad hoc quantity (ΔK).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g002">
              https://doi.org/10.1371/journal.pone.0204351.g002</a></p></div><a class="link-target" id="pone-0204351-g003" name="pone-0204351-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g003" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> Bayesian clustering patterns of wild RJF populations (n = 57).</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g003">
              https://doi.org/10.1371/journal.pone.0204351.g003</a></p></div><a class="link-target" id="pone-0204351-t003" name="pone-0204351-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.t003" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> Genetic assignment of wild RJF (n = 57) populations through Bayesian clustering analysis.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.t003">
              https://doi.org/10.1371/journal.pone.0204351.t003</a></p></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>In a GENELAND analysis assuming uncorrelated allele frequency model, four genetic clusters (K = 4) were inferred in 20 out of 30 runs (66.66%; <a href="#pone.0204351.s004">S4 Table</a>, the first three columns,) based on the highest average logarithm of the posterior probability (<a href="#pone.0204351.s008">S3 Fig</a>). In contrast, when assuming correlated allele frequency model, nearly all runs (29/30) indicated 10 clusters (<a href="#pone.0204351.s004">S4 Table</a>, the last three columns) which were not interpretable. Therefore, we considered only the results of the analysis assuming uncorrelated frequency model. The results of the top three runs showing the highest average logarithm of the posterior probability were largely consistent. The spatial distribution of these four inferred clusters showed that the boundaries of these clusters were generally coincided with multiple landscape features and individuals were assigned to these four clusters as cluster-I (green; 36.36%), cluster-II (light green; 7.27%), cluster-III (orange; 5.45%) and cluster- IV (white; 50.90%) (<a href="#pone-0204351-g004">Fig 4E</a>). Interestingly, all the samples collected from the State Uttar Pradesh (Pilibhit Tiger Reserve, Dudhwa National Park, Katerniaghat &amp; Kishanpur Wildlife Sanctuary) were assigned to cluster- IV while samples collected from the State Uttarakhand (Terai Forest Division, Corbett National Park, Rajaji National Park &amp; Wildlife Institute of India campus) were assigned to cluster- I. All the samples collected from Manipur were grouped in cluster-III (<a href="#pone.0204351.s004">S4 Table</a>). Only three individuals (one each from Khelma Forest Division-Nagaland, Hailakandi Forest Division -Assam and Nameri National Park -Assam) were grouped in cluster- II. Samples collected from several locations were assigned to cluster-IV, and this can plausibly be interpreted based on the possible historic gene flow and physical exchange of the birds in the past. Each individual’s estimated proportions of membership obtained from GENELAND analysis (around 0.50 at maximum, <a href="#pone.0204351.s005">S5 Table</a>) were lower than those obtained using the STRUCTURE analysis. The results provided indications that the present landscape features did not act as a barrier to gene flow or the actual geo-spatial clustering trend in the landscape remained masked by the fact of physical sharing/exchange of wild birds in the past as the forested areas and most preferred habitats for the species may be connected.</p>
<a class="link-target" id="pone-0204351-g004" name="pone-0204351-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g004" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 4. </span> Results of Bayesian model based clustering in GENELAND using wild RJF individuals (n = 57; K = 4 as the best run showing the highest average logarithm of the posterior probability).</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig3.caption1.p1" name="article1.body1.sec3.sec2.fig3.caption1.p1" class="link-target"></a><p>(a—d) Maps of posterior probability for each inferred cluster show the spatial location of genetic discontinuities of clusters I,II,III and IV; (e) A map of estimated cluster membership shows spatial distribution of the four inferred genetic clusters across the study areas. Sample locations are represented by <em>black dots</em>. <em>X</em> and <em>Y spaces</em> indicate longitude (E) and latitude (N). <em>Light yellow coloring</em> corresponds to high posterior probability and <em>red</em> corresponds to low posterior probability.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g004">
              https://doi.org/10.1371/journal.pone.0204351.g004</a></p></div></div>

<div id="section3" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="Genetic introgression and identification of admixed individuals"></a>
<h3>Genetic introgression and identification of admixed individuals</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>The results indicated a multi-modal value of delta <em>K</em> as each peak breaking out quite clearly at <em>K</em> = 2 and <em>K</em> = 4 (<a href="#pone-0204351-g005">Fig 5B</a>). Individual assignment was &gt;98% at <em>K</em> = 2 (<a href="#pone-0204351-g006">Fig 6</a>; <a href="#pone.0204351.s003">S3 Table</a>) and ~82% at <em>K</em> = 4 (<a href="#pone-0204351-g006">Fig 6</a>; <a href="#pone-0204351-t004">Table 4</a>) considering a posterior probability value q ≥ 0.80. At <em>K</em> = 2, all the RJFs but one bird of East population (RJ-110_SK) were assigned to any of these two clusters while at <em>K</em> = 4 nine RJFs were admixed (RJ-76_UK, RJ-84_UK, RJ-116_UP, RJ-55_UP, RJ-58_UP, RJ-63_UP, RJ-105_BH, RJ-110_SK and RJ-113_AS).</p>
<a class="link-target" id="pone-0204351-g005" name="pone-0204351-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g005" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 5. </span> Graphical representation of the true number of the cluster in wild RJF and DC populations.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p>(A) Mean L(K) (±SD) over 20 independent runs for each <em>K</em>value. (B) The ad hoc quantity (ΔK).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g005">
              https://doi.org/10.1371/journal.pone.0204351.g005</a></p></div><a class="link-target" id="pone-0204351-g006" name="pone-0204351-g006"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g006" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 6. </span> Bayesian clustering patterns of wild RJF (n = 57) and DC(n = 79) populations.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g006">
              https://doi.org/10.1371/journal.pone.0204351.g006</a></p></div><a class="link-target" id="pone-0204351-t004" name="pone-0204351-t004"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.t004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.t004" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.t004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.t004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.t004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.t004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.t004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 4. </span> Assignment of admixed individuals between wild RJF (n = 57) and DC (n = 79) populations through Bayesian clustering analysis at <em>K</em> = 4.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.t004">
              https://doi.org/10.1371/journal.pone.0204351.t004</a></p></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>A Maximum Likelihood (ML) tree obtained from the microsatellite data formed several groups of possible hybrids between the RJFs and DCs (represented by a red triangle with pink shadow in <a href="#pone-0204351-g007">Fig 7</a>. There were 13 RJFs (22.8% of total RJF samples; RJ-123, RJ-116, RJ-66, RJ-98, RJ-99, RJ-115, RJ-105, RJ-108, RJ-112, RJ-6 and RJ-70, RJ-9 and RJ-11) being grouped with DCs representing sharing of similar alleles between RJFs and DCs. This clustering pattern as revealed by ML tree based on a distance matrix of microsatellite data exhibited the recent event of admixture. While ML tree based on D-loop sequences identified several RJFs <em>i</em>.<em>e</em>. RJ123, RJ101, RJ104 and RJ1 sharing identical haplotypes present in DCs and several other RJFs being grouped with DCs (<a href="#pone-0204351-g008">Fig 8</a>).</p>
<a class="link-target" id="pone-0204351-g007" name="pone-0204351-g007"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g007" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 7. </span> A maximum likelihood phylogenetic tree constructed using microsatellite data of wild RJF and DC samples.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig3.caption1.p1" name="article1.body1.sec3.sec3.fig3.caption1.p1" class="link-target"></a><p>(The genetic distance matrix was calculated using GeneAlex and phylogenetic tree was constructed in Mega version 7.0. Scale indicates the allele frequency. The possible hybrids as revealed by sharing similar alleles are represented with red triangle under pink shadow).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g007">
              https://doi.org/10.1371/journal.pone.0204351.g007</a></p></div><a class="link-target" id="pone-0204351-g008" name="pone-0204351-g008"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g008"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g008" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g008"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g008" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g008">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g008">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g008">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g008">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g008">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g008">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 8. </span> Molecular phylogenetic analysis by Maximum Likelihood method based on the Hasegawa-Kishino-Yano model (HKY+G+I to be the best fit model).</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig4.caption1.p1" name="article1.body1.sec3.sec3.fig4.caption1.p1" class="link-target"></a><p>(The tree is drawn to scale, with branch lengths measured in the number of substitutions per site. The analysis involved 108 nucleotide sequences. All positions containing gaps and missing data were eliminated. There were a total of 452 positions in the final dataset. The possible hybrids as revealed by sharing the same haplotypes <em>i</em>.<em>e</em>. Hap# 10, 14 and 19 are represented with red triangle, supporting a longer term admixture).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g008">
              https://doi.org/10.1371/journal.pone.0204351.g008</a></p></div><a id="article1.body1.sec3.sec3.p3" name="article1.body1.sec3.sec3.p3" class="link-target"></a><p>The median-joining network analysis of 68 haplotypes suggested a strong geospatial structure in the wild RJFs and DCs (<a href="#pone-0204351-g009">Fig 9</a>). To understand the direction of gene flow, we prioritize shared haplotypes over the haplotypes unique to individual populations. We assigned shared haplotypes into three categories <em>viz</em>. shared—between RJF and DC populations (Hap#10, 14 and 19), between RJF populations (Hap#6) and between DC populations (Hap#37, 40, 41, 45, 50 and 53). Among three haplotypes shared between RJFs and DCs, Hap#14 and 19 were predominant in DCs at 60% and75%, respectively. While Hap#10 was shared by 50% in RJFs or DCs. In the second group, Hap#6 which was shared between RJF populations of North (Uttarakhand) and Northeast (Assam) might be the ancestral haplotype reflecting historic gene flow between these two populations. Interestingly, it was well supported by the results of STRUCTURE (50% of assignment of Northeast RJFs with North RJFs; <a href="#pone-0204351-t003">Table 3</a>) and GENELAND (One RJF sample collected from Assam being grouped in cluster-IV along with RJFs collected from Uttarakhand; <a href="#pone.0204351.s005">S5 Table</a>) analyses. However, we did not give much attention to the haplotypes shared between DCs as this could be due to the transport of DCs across the country for the local benefit or commercial purposes.</p>
<a class="link-target" id="pone-0204351-g009" name="pone-0204351-g009"></a><div class="figure" data-doi="10.1371/journal.pone.0204351.g009"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0204351.g009" data-doi="info:doi/10.1371/journal.pone.0204351" data-uri="info:doi/10.1371/journal.pone.0204351.g009"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0204351.g009" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g009">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0204351.g009">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g009">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0204351.g009">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g009">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0204351.g009">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 9. </span> Median joining haplotype network of 136 birds (57 wild RJFs and 79 DCs) belonged to 68 haplotypes.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig5.caption1.p1" name="article1.body1.sec3.sec3.fig5.caption1.p1" class="link-target"></a><p>(Haplotypes of RJFs and DCs are represented in different colored circles. The size of the circle is proportional to the haplotype frequency. Important haplotyoes shared within wild RJF populations, between wild RJFs and DCs and within DC populations are numbered).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0204351.g009">
              https://doi.org/10.1371/journal.pone.0204351.g009</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="sec013" name="sec013" data-toc="sec013" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>All the 26 loci were polymorphic and informative for population genetic analysis as the majority of loci showed a highPIC value &gt; 0.5. Several loci were deviated individually from HWE in different populations e.g. four loci in Cent-Southeast and 22 loci in North RJF populations due to heterozygote deficit following possible inbreeding at regional level. Two loci were deviated from HWE across all four RJF populations attributed to the significant heterozygote deficit. None of the locus pairs showed significant LD, and independent assortment across the four RJF populations. The diversity indices, <em>e</em>.<em>g</em>. Na, ne, He, average PIC values and N<sub>P,</sub> were found to be the highest in the North RJF population. However, most of these genetic attributes in a population are likely influenced by the sample size. UHe and A<sub>R</sub> which are independent of sample size, were were the highest in Northeast population which was also corroborated by the D-loop data. Statham <em>et al</em>. [<a href="#pone.0204351.ref047" class="ref-tip">47</a>] reported that the parental population of a widespread free-ranging animal species, like Russian silver fox (<em>Vulpes vulpes</em>), tended to represent most of its genetic diversity across the entire range. Thus our study warrants that Northeast RJF population would be the wild progenitor due to its abundance of genetic variants in both nuclear and mitochondrial genomes from where early radiation of RJFs throughout India would have occurred.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>The Bayesian analysis of population assignment grouped all the samples into two clusters and the shared assignment of Northeast RJFs with North RJFs (50% being assigned in <em>cluster-I</em>; <a href="#pone-0204351-t003">Table 3</a>) indicated their common ancestry probably due to the habitat connectivity among the two populations along Bhutan, the <em>Duars</em> region of North Bengal and the <em>Terai</em> region of Nepal and India. However, the GENELAND clustering pattern showed four clusters c (<a href="#pone-0204351-g004">Fig 4E</a>). Interestingly, samples of North RJF population were assigned to both clusters-I and IV (<a href="#pone.0204351.s005">S5 Table</a>). This result was also observed in the STRUCTURE analysis. Further, cluster-II (Manipur) was largely separated from cluster-IV, because of the urban areas while, cluster-III present in a small island, might have been separated from the main range by the Brahmaputra River, a significant zoogeographic barrier as evident for the restricted movement of many terrestrial and arboreal mammals. Only two samples, one each from Sikkim and Assam, were assigned into cluster-III. Cluster-IV revealed interesting observations as individuals were collected from several locations, <em>i</em>.<em>e</em>. S1 (J&amp;K, HP and UK), S2 (Bihar), S3 (Chhattisgarh), S4 (Odisha), S5 (Assam) and S6 (Mizoram and Nagaland), reflecting this cluster-IV to be a genetic reservoir and perhaps a source of radiation either due to historic gene flow or physical exchange of the birds.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>Interestingly, RJF samples from the State of Uttar Pradesh (Pilibhit Tiger Reserve, Dudhwa National Park, Katerniaghat and Kishanpur Wildlife Sanctuary) were assigned to cluster-IV and RJF samples from the State of Uttarakhand (Terai Forest Division, Corbett National Park, Rajaji National Park and Wildlife Institute of India campus) were assigned to cluster-I, irrespective of their physical proximity. There may be several explanations for this unusual observation, one possiblity might be the restricted exchange of wild RJFs within the region by the locals for upgrading their DCs or the other possible reason could be the anthropogenic activities in this region, particularly in and around Terai East and West along the Gola river which might have affected the movement of free-ranging RJFs within the region, as also demonstrated for other animal species such as tigers and leopards [<a href="#pone.0204351.ref048" class="ref-tip">48</a>]. After identifying the admixed individuals between RJFs and DCs, we obtained individual assignment &gt;98% at K = 2 (<a href="#pone-0204351-g005">Fig 5B</a>; <a href="#pone.0204351.s003">S3 Table</a>) but we considered assignment above 82% at <em>K =</em> .4 yielding a relatively high resolution of individual ancestry for identifying admixed individuals (<a href="#pone-0204351-g005">Fig 5B</a>; <a href="#pone-0204351-t004">Table 4</a>). At K = 4, nine RJFs <em>i</em>.<em>e</em>. RJ-76, RJ-84, RJ-116, RJ-55, RJ-58, RJ-63, RJ-105, RJ-110 and RJ-113 were found to be admixed. Since microsatellites have high mutation rates, they provide information about recent evolutionary history as compared to the slow mutating genes, for example the mitochondrial genes that provide data about ancient history [<a href="#pone.0204351.ref045" class="ref-tip">45</a>]. To reconstruct the recent past, we used both these marker systems to address the issue of genetic swapping between wild RJFs and DCs. A ML tree based on microsatellites distance matrix identified 13 RJFs (~23% of the sampled RJFs) being grouped with DCs as plausible hybrids. However, the phylogenetic relationship based on the D loop sequences identified several RJFs <em>i</em>.<em>e</em>. RJ123, RJ101, RJ104 and RJ1 sharingidentical haplotypes with DCs. These result indicated that hybridization of RJF is not a recent phenomenon but may have persisted since the domestication of junglefowls. By cross-checking the hybrids identified based on microsatellites with the Dloop based phylogeny, we identified RJF <em>i</em>.<em>e</em>. RJ123 to be a hybrid. This observation was not surprising as the microsatellites and D loop follow different modes of genetic inheritance, where the former are involved in recombinations due to their bi-parental inheritance while D-loop located in mitochondria follows a maternal inheritance. Network analysis has also identified three haplotypes (Hap#10, 14 and 19) to be shared between RJFs and DCs, supporting a longer-term admixture of the species. Among these three haplotypes, frequencies of Hap#14 and 19 were 60% and 75% in DCs suggesting gene flow between RJFs and DCs. This trend, however was supported by a small number of samples probably derived from male RJFs crossing with DC hens. This explanation was highly reasonable since we have encountered the mixing behaviour of RJF males with DC hens even in the presence of DC cocks [<a href="#pone.0204351.ref019" class="ref-tip">19</a>]. Our results are in the accordance with the observations made by Gering <em>et al</em>. [<a href="#pone.0204351.ref012" class="ref-tip">12</a>] about the origin of <em>Kauai</em> chickens whose behaviour and morphology overlap with those of DCs and RJFs, implying these chickens to be feral <em>G</em>. <em>gallus</em> descendent from recent invasion(s) of DCs into an ancient RJF reservoir.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>To infer on the threat status of the species undergoing hybridization, we believe it is necessary to sample across the species range as the phenomenon may alter with the compromised samples or the sampling locality. Kanginakudru <em>et al</em>. [<a href="#pone.0204351.ref005" class="ref-tip">5</a>] concluded that hybridization is a rare phenomenon, but we believe that such underestimated patterns was due to the limited number of samples, mostly collected from captivity. This study is the first attempt to address the phenomenon of hybridization across the distribution of the RJF species, and our results did not find landscape features contributing to the population genetic structure, possibly due to the exchange of wild RJFs in the past when forests were continuous across India.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>RJF wild populations across the range are under extreme pressure of poaching for local consumption, habitat loss and genetic hybridization with domestic/feral chickens, it is necessary to monitor the presence/absence and abundance of this species in different parts of the distribution range within as well as outside Protected Area network. Conservation breeding of ‘genetically pure’ RJFs may be initiated with inclusion of genetic parameters and captive stocks are required to be scientifically managed for future use such as research and reintroductions.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec014" name="sec014" data-toc="sec014" class="link-target" title="Supporting information"></a><h2>Supporting information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0204351"></div><div class="supplementary-material"><a name="pone.0204351.s001" id="pone.0204351.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s001">S1 Table. </a>Hardy–Weinberg equilibrium (HWE) test.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s001">https://doi.org/10.1371/journal.pone.0204351.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0204351.s002" id="pone.0204351.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s002">S2 Table. </a>Linkage disequilibrium (LD) test.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s002">https://doi.org/10.1371/journal.pone.0204351.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0204351.s003" id="pone.0204351.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s003">S3 Table. </a>Assignment of admixed individuals between wild RJF (n = 57) and DC (n = 79) populations through Bayesian clustering analysis at <em>K</em>2.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s003">https://doi.org/10.1371/journal.pone.0204351.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0204351.s004" id="pone.0204351.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s004">S4 Table. </a>The most likely number of clusters (<em>K</em>) and average logarithm of the posterior probability for each of the 30 runs in GENELAND analysis assuming either uncorrelated or correlated allele frequency models.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s004">https://doi.org/10.1371/journal.pone.0204351.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0204351.s005" id="pone.0204351.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s005">S5 Table. </a>The individual’s proportion of membership of the wild RJFs (n = 55) in each of the four clusters (<em>K</em> = 4) inferred by GENELAND analysis using uncorrelated allele frequency model.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s005">https://doi.org/10.1371/journal.pone.0204351.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="postSiDOI">(DOC)</p>
</div><div class="supplementary-material"><a name="pone.0204351.s006" id="pone.0204351.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s006">S1 Fig. </a>Median joining haplotype network of 38 wild RJFs that represented 31 haplotypes.</h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI">(Haplotypes of four RJF populations are represented in different colored circles. The size of the circle is proportional to the haplotype frequency. Haplotype 3 is shared between North and Northeast populations).</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s006">https://doi.org/10.1371/journal.pone.0204351.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0204351.s007" id="pone.0204351.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s007">S2 Fig. </a>Demographic mismatch distribution curve of four RJF populations.</h3><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="preSiDOI">A, North; B, East; C, Central-Southeast and D,Northeast.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s007">https://doi.org/10.1371/journal.pone.0204351.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p2" name="article1.body1.sec5.supplementary-material7.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0204351.s008" id="pone.0204351.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0204351.s008">S3 Fig. </a>The number of cluster (<em>K</em>) simulated from the posterior distribution of the best run showing the highest average logarithm of the posterior probability in GENELAND analysis.</h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI">Analysis was conducted under the spatial model with uncorrelated allele frequencyincorporating latitude/longitude.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0204351.s008">https://doi.org/10.1371/journal.pone.0204351.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank all the State Forest / Wildlife Departments, Principal Chief Conservator of Forests and Chief Wildlife Wardens for granting permission for live-trapping and collection of RJF samples in the wild and the logistics provided during field surveys. We are thankful to Director, Dean, Research Coordinator and faculties of Wildlife Institute of India, Dehradun for their encouragement and support. This study was funded by the in-house research grant of Wildlife Institute of India, Dehradun. We acknowledge Dr William J. McShea (Smithsonian Conservation Biology Institute, Virginia) for editing manuscript and two reviewers for helping in improving the contents of the present manuscript.</p>
</div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0204351.ref001" id="pone.0204351.ref001" class="link-target"></a>Eriksson J., Larson G., Gunnarsson U., Bed’hom B., Tixier-Boichard M., Strömstedt L. et al. (2008). Identification of the yellow skin gene reveals a hybrid origin of the domestic chicken. <em>PLoS Genet</em> 4: e1000010.  pmid:18454198 <ul class="reflinks"><li><a href="#" data-author="Eriksson" data-cit="ErikssonJ.%2C%20LarsonG.%2C%20GunnarssonU.%2C%20Bed%E2%80%99homB.%2C%20Tixier-BoichardM.%2C%20Str%C3%B6mstedtL.%20et%20al.%20%282008%29.%20Identification%20of%20the%20yellow%20skin%20gene%20reveals%20a%20hybrid%20origin%20of%20the%20domestic%20chicken.%20PLoS%20Genet%204%3A%20e1000010.%20doi%3A%2010.1371%2Fjournal.pgen.1000010%2018454198" data-title="Identification%20of%20the%20yellow%20skin%20gene%20reveals%20a%20hybrid%20origin%20of%20the%20domestic%20chicken" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18454198" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+the+yellow+skin+gene+reveals+a+hybrid+origin+of+the+domestic+chicken+Eriksson+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0204351.ref002" id="pone.0204351.ref002" class="link-target"></a>Fumihito A., Miyake T., Sumi S., Takada M., Ohno S. and Kondo N. (1994). One subspecies of the red junglefowl <em>Gallus gallus gallus</em>. suffices as the matriarchic ancestor of all domestic breeds. <em>Proc Natl Acad Sci USA</em>, 91:12505–12509. pmid:7809067 <ul class="reflinks"><li><a href="#" data-author="Fumihito" data-cit="FumihitoA.%2C%20MiyakeT.%2C%20SumiS.%2C%20TakadaM.%2C%20OhnoS.%20and%20KondoN.%20%281994%29.%20One%20subspecies%20of%20the%20red%20junglefowl%20Gallus%20gallus%20gallus.%20suffices%20as%20the%20matriarchic%20ancestor%20of%20all%20domestic%20breeds.%20Proc%20Natl%20Acad%20Sci%20USA%2C%2091%3A12505%E2%80%9312509.%207809067" data-title="One%20subspecies%20of%20the%20red%20junglefowl%20Gallus%20gallus%20gallus.%20suffices%20as%20the%20matriarchic%20ancestor%20of%20all%20domestic%20breeds" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7809067" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=One+subspecies+of+the+red+junglefowl+Gallus+gallus+gallus.+suffices+as+the+matriarchic+ancestor+of+all+domestic+breeds+Fumihito+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0204351.ref003" id="pone.0204351.ref003" class="link-target"></a>Fumihito A., Miyake T., Takada M., Shingu R., Endo T., Gojobori T. et al. (1996). Monophyletic origin and unique dispersal patterns of domestic fowls. <em>Proc Natl Acad Sci USA</em>. 93: 792–6795. <ul class="reflinks"><li><a href="#" data-author="Fumihito" data-cit="FumihitoA.%2C%20MiyakeT.%2C%20TakadaM.%2C%20ShinguR.%2C%20EndoT.%2C%20GojoboriT.%20et%20al.%20%281996%29.%20Monophyletic%20origin%20and%20unique%20dispersal%20patterns%20of%20domestic%20fowls.%20Proc%20Natl%20Acad%20Sci%20USA.%2093%3A%20792%E2%80%936795." data-title="Monophyletic%20origin%20and%20unique%20dispersal%20patterns%20of%20domestic%20fowls" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Monophyletic+origin+and+unique+dispersal+patterns+of+domestic+fowls+Fumihito+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0204351.ref004" id="pone.0204351.ref004" class="link-target"></a>Liu Y.P., Wu G.S., Yao Y.G., Miao Y.W., Luikart G, Baig M et al. (2006). Multiple origins of chickens: out of Asian jungles. ‎<em>Mol Phylogenetics Evol</em>, 38: 12–19. <ul class="reflinks"><li><a href="#" data-author="Liu" data-cit="LiuY.P.%2C%20WuG.S.%2C%20YaoY.G.%2C%20MiaoY.W.%2C%20LuikartG%2C%20BaigM%20et%20al.%20%282006%29.%20Multiple%20origins%20of%20chickens%3A%20out%20of%20Asian%20jungles.%20%E2%80%8EMol%20Phylogenetics%20Evol%2C%2038%3A%2012%E2%80%9319." data-title="Multiple%20origins%20of%20chickens%3A%20out%20of%20Asian%20jungles" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multiple+origins+of+chickens%3A+out+of+Asian+jungles+Liu+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0204351.ref005" id="pone.0204351.ref005" class="link-target"></a>Kanginakudru S., Murlidhar M., Jakati R. D. and Nagaraju J. (2008). Genetic evidence from Indian red junglefowl corroborates multiple domestication of modern day chicken. <em>BMC Evol Biol</em>, 8: 174.  pmid:18544161 <ul class="reflinks"><li><a href="#" data-author="Kanginakudru" data-cit="KanginakudruS.%2C%20MurlidharM.%2C%20JakatiR.%20D.%20and%20NagarajuJ.%20%282008%29.%20Genetic%20evidence%20from%20Indian%20red%20junglefowl%20corroborates%20multiple%20domestication%20of%20modern%20day%20chicken.%20BMC%20Evol%20Biol%2C%208%3A%20174.%20doi%3A%2010.1186%2F1471-2148-8-174%2018544161" data-title="Genetic%20evidence%20from%20Indian%20red%20junglefowl%20corroborates%20multiple%20domestication%20of%20modern%20day%20chicken" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18544161" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+evidence+from+Indian+red+junglefowl+corroborates+multiple+domestication+of+modern+day+chicken+Kanginakudru+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0204351.ref006" id="pone.0204351.ref006" class="link-target"></a>Berthouly C, Leroy G, Van TN, Thanh HH, Bed’Hom B, Nguyen BT et al. (2009). Genetic analysis of local Vietnamese chickens provides evidence of gene flow from wild to domestic populations. <em>BMC Genet</em> 10: 1.  pmid:19133138 <ul class="reflinks"><li><a href="#" data-author="Berthouly" data-cit="BerthoulyC%2C%20LeroyG%2C%20VanTN%2C%20ThanhHH%2C%20Bed%E2%80%99HomB%2C%20NguyenBT%20et%20al.%20%282009%29.%20Genetic%20analysis%20of%20local%20Vietnamese%20chickens%20provides%20evidence%20of%20gene%20flow%20from%20wild%20to%20domestic%20populations.%20BMC%20Genet%2010%3A%201.%20doi%3A%2010.1186%2F1471-2156-10-1%2019133138" data-title="Genetic%20analysis%20of%20local%20Vietnamese%20chickens%20provides%20evidence%20of%20gene%20flow%20from%20wild%20to%20domestic%20populations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19133138" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+analysis+of+local+Vietnamese+chickens+provides+evidence+of+gene+flow+from+wild+to+domestic+populations+Berthouly+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0204351.ref007" id="pone.0204351.ref007" class="link-target"></a>Miao Y.W., Peng M.S., Wu G.S., Ouyang Y.N., Yang Z.Y., Yu N. et al. Chicken domestication: an updated perspective based on mitochondrial genomes. (2013). <em>Heredity</em>, 110, 277–282.  pmid:23211792 <ul class="reflinks"><li><a href="#" data-author="Miao" data-cit="MiaoY.W.%2C%20PengM.S.%2C%20WuG.S.%2C%20OuyangY.N.%2C%20YangZ.Y.%2C%20YuN.%20et%20al.%20Chicken%20domestication%3A%20an%20updated%20perspective%20based%20on%20mitochondrial%20genomes.%20%282013%29.%20Heredity%2C%20110%2C%20277%E2%80%93282.%20doi%3A%2010.1038%2Fhdy.2012.83%2023211792" data-title="Chicken%20domestication%3A%20an%20updated%20perspective%20based%20on%20mitochondrial%20genomes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23211792" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chicken+domestication%3A+an+updated+perspective+based+on+mitochondrial+genomes+Miao+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0204351.ref008" id="pone.0204351.ref008" class="link-target"></a>Peterson A.T., and Brisbin I. L. Jr. (1999). Genetic endangerment of wild Red Junglefowl <em>Gallus</em> gallus? <em>Bird Conserv Int</em> 8: 387–394. <ul class="reflinks"><li><a href="#" data-author="Peterson" data-cit="PetersonA.T.%2C%20and%20BrisbinI.%20L.Jr.%20%281999%29.%20Genetic%20endangerment%20of%20wild%20Red%20Junglefowl%20Gallus%20gallus%3F%20Bird%20Conserv%20Int%208%3A%20387%E2%80%93394." data-title="Genetic%20endangerment%20of%20wild%20Red%20Junglefowl%20Gallus%20gallus%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+endangerment+of+wild+Red+Junglefowl+Gallus+gallus%3F+Peterson+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0204351.ref009" id="pone.0204351.ref009" class="link-target"></a>Brisbin I. L. Jr., Peterson A.T., Okimoto R., and Amoto G. (2002). Characterisation of the genetic status of populations of Red Junglefowl. <em>J</em>. <em>Bombay Nat Hist Soc</em>, 99: 217–223. <ul class="reflinks"><li><a href="#" data-author="Brisbin" data-cit="BrisbinI.%20L.Jr.%2C%20PetersonA.T.%2C%20OkimotoR.%2C%20and%20AmotoG.%20%282002%29.%20Characterisation%20of%20the%20genetic%20status%20of%20populations%20of%20Red%20Junglefowl.%20J.%20Bombay%20Nat%20Hist%20Soc%2C%2099%3A%20217%E2%80%93223." data-title="Characterisation%20of%20the%20genetic%20status%20of%20populations%20of%20Red%20Junglefowl" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterisation+of+the+genetic+status+of+populations+of+Red+Junglefowl+Brisbin+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0204351.ref010" id="pone.0204351.ref010" class="link-target"></a>Nishibori M., Shimogiri T., Hayashi T., and Yasue H. (2005). Molecular evidence for hybridization of species in the genus Gallus except for <em>Gallus varius</em>. <em>Ani Gen</em>, 36: 367–375. <ul class="reflinks"><li><a href="#" data-author="Nishibori" data-cit="NishiboriM.%2C%20ShimogiriT.%2C%20HayashiT.%2C%20and%20YasueH.%20%282005%29.%20Molecular%20evidence%20for%20hybridization%20of%20species%20in%20the%20genus%20Gallus%20except%20for%20Gallus%20varius.%20Ani%20Gen%2C%2036%3A%20367%E2%80%93375." data-title="Molecular%20evidence%20for%20hybridization%20of%20species%20in%20the%20genus%20Gallus%20except%20for%20Gallus%20varius" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+evidence+for+hybridization+of+species+in+the+genus+Gallus+except+for+Gallus+varius+Nishibori+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0204351.ref011" id="pone.0204351.ref011" class="link-target"></a>Brisbin I. L. Jr., and Peterson A. T. (2007). Playing chicken with Red Junglefowl: identifying phenotypic markers of genetic purity in <em>Gallus gallus</em>. <em>Ani Cons</em>, 10: 429–435. <ul class="reflinks"><li><a href="#" data-author="Brisbin" data-cit="BrisbinI.%20L.Jr.%2C%20and%20PetersonA.%20T.%20%282007%29.%20Playing%20chicken%20with%20Red%20Junglefowl%3A%20identifying%20phenotypic%20markers%20of%20genetic%20purity%20in%20Gallus%20gallus.%20Ani%20Cons%2C%2010%3A%20429%E2%80%93435." data-title="Playing%20chicken%20with%20Red%20Junglefowl%3A%20identifying%20phenotypic%20markers%20of%20genetic%20purity%20in%20Gallus%20gallus" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Playing+chicken+with+Red+Junglefowl%3A+identifying+phenotypic+markers+of+genetic+purity+in+Gallus+gallus+Brisbin+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0204351.ref012" id="pone.0204351.ref012" class="link-target"></a>Gering E., Johnsson M., Willis P., Getty T. and Wright D. (2015). Mixed ancestry and admixture in Kauai’s feral chickens: invasion of domestic genes into ancient Red Junglefowl reservoirs. <em>Mol Ecol</em>, 24, 2112–2124.  pmid:25655399 <ul class="reflinks"><li><a href="#" data-author="Gering" data-cit="GeringE.%2C%20JohnssonM.%2C%20WillisP.%2C%20GettyT.%20and%20WrightD.%20%282015%29.%20Mixed%20ancestry%20and%20admixture%20in%20Kauai%E2%80%99s%20feral%20chickens%3A%20invasion%20of%20domestic%20genes%20into%20ancient%20Red%20Junglefowl%20reservoirs.%20Mol%20Ecol%2C%2024%2C%202112%E2%80%932124.%20doi%3A%2010.1111%2Fmec.13096%2025655399" data-title="Mixed%20ancestry%20and%20admixture%20in%20Kauai%E2%80%99s%20feral%20chickens%3A%20invasion%20of%20domestic%20genes%20into%20ancient%20Red%20Junglefowl%20reservoirs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25655399" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mixed+ancestry+and+admixture+in+Kauai%E2%80%99s+feral+chickens%3A+invasion+of+domestic+genes+into+ancient+Red+Junglefowl+reservoirs+Gering+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0204351.ref013" id="pone.0204351.ref013" class="link-target"></a>Flamand J.R.B., Vankan D., Gairhe K.P., Duong H. and Barker J.S.F. (2003). Genetic identification of wild Asian water buffalo in Nepal. <em>Ani Cons</em>, 6, 3: 265–270. <ul class="reflinks"><li><a href="#" data-author="Flamand" data-cit="FlamandJ.R.B.%2C%20VankanD.%2C%20GairheK.P.%2C%20DuongH.%20and%20BarkerJ.S.F.%20%282003%29.%20Genetic%20identification%20of%20wild%20Asian%20water%20buffalo%20in%20Nepal.%20Ani%20Cons%2C%206%2C%203%3A%20265%E2%80%93270." data-title="Genetic%20identification%20of%20wild%20Asian%20water%20buffalo%20in%20Nepal" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+identification+of+wild+Asian+water+buffalo+in+Nepal+Flamand+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0204351.ref014" id="pone.0204351.ref014" class="link-target"></a>Randi E. (2008). Detecting hybridization between wild species and their domesticated relatives. <em>Mol Ecol</em>, 17, 285–93.  pmid:18173502 <ul class="reflinks"><li><a href="#" data-author="Randi" data-cit="RandiE.%20%282008%29.%20Detecting%20hybridization%20between%20wild%20species%20and%20their%20domesticated%20relatives.%20Mol%20Ecol%2C%2017%2C%20285%E2%80%9393.%20doi%3A%2010.1111%2Fj.1365-294X.2007.03417.x%2018173502" data-title="Detecting%20hybridization%20between%20wild%20species%20and%20their%20domesticated%20relatives" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/18173502" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Detecting+hybridization+between+wild+species+and+their+domesticated+relatives+Randi+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0204351.ref015" id="pone.0204351.ref015" class="link-target"></a>Šprem N., Salajpal K., Safner T., Đikić D., Jurić J., Curik I. et al. (2014). Genetic analysis of hybridization between domesticated endangered pig breeds and wild boar, <em>Livest Sci</em>, 162, 1–4. <strong><a href="https://doi.org/10.1016/j.livsci.2013.12.010">https://doi.org/10.1016/j.livsci.2013.12.010</a></strong>. <ul class="reflinks"><li><a href="#" data-author="%C5%A0prem" data-cit="%C5%A0premN.%2C%20SalajpalK.%2C%20SafnerT.%2C%20%C4%90iki%C4%87D.%2C%20Juri%C4%87J.%2C%20CurikI.%20et%20al.%20%282014%29.%20Genetic%20analysis%20of%20hybridization%20between%20domesticated%20endangered%20pig%20breeds%20and%20wild%20boar%2C%20Livest%20Sci%2C%20162%2C%201%E2%80%934.%20https%3A%2F%2Fdoi.org%2F10.1016%2Fj.livsci.2013.12.010." data-title="Genetic%20analysis%20of%20hybridization%20between%20domesticated%20endangered%20pig%20breeds%20and%20wild%20boar" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+analysis+of+hybridization+between+domesticated+endangered+pig+breeds+and+wild+boar+%C5%A0prem+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0204351.ref016" id="pone.0204351.ref016" class="link-target"></a>Torres R.T., Ferreira E., Rocha R.G. and Fonseca C., (2017). Hybridization between wolf and domestic dog: First evidence from an endangered population in central Portugal, <em>Mamm Biol</em>, 86, 70–74. <ul class="reflinks"><li><a href="#" data-author="Torres" data-cit="TorresR.T.%2C%20FerreiraE.%2C%20RochaR.G.%20and%20FonsecaC.%2C%20%282017%29.%20Hybridization%20between%20wolf%20and%20domestic%20dog%3A%20First%20evidence%20from%20an%20endangered%20population%20in%20central%20Portugal%2C%20Mamm%20Biol%2C%2086%2C%2070%E2%80%9374." data-title="Hybridization%20between%20wolf%20and%20domestic%20dog%3A%20First%20evidence%20from%20an%20endangered%20population%20in%20central%20Portugal" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hybridization+between+wolf+and+domestic+dog%3A+First+evidence+from+an+endangered+population+in+central+Portugal+Torres+2017" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0204351.ref017" id="pone.0204351.ref017" class="link-target"></a>Allendorf W. F., Leary R. F., Spruell P., and Wenburg K. J. (2001). The problems with hybrids: setting conservation guidelines. <em>Trends Ecol Evolut</em>, 16: 613–622. <ul class="reflinks"><li><a href="#" data-author="Allendorf" data-cit="AllendorfW.%20F.%2C%20LearyR.%20F.%2C%20SpruellP.%2C%20and%20WenburgK.%20J.%20%282001%29.%20The%20problems%20with%20hybrids%3A%20setting%20conservation%20guidelines.%20Trends%20Ecol%20Evolut%2C%2016%3A%20613%E2%80%93622." data-title="The%20problems%20with%20hybrids%3A%20setting%20conservation%20guidelines" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+problems+with+hybrids%3A+setting+conservation+guidelines+Allendorf+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0204351.ref018" id="pone.0204351.ref018" class="link-target"></a>Sathyakumar S., Fernandes M., Mukesh , Kaul R. and Kalsi R.S., (2012). Conservation of Red Junglefowl <em>Gallus gallus</em> in India. Final Project Report. Wildlife Institute of India, Dehradun. 98 Pp. <ul class="find-nolinks"></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0204351.ref019" id="pone.0204351.ref019" class="link-target"></a>Thakur M. and Schättin E.W., (2014). Die Ahnen des Haushuhns sind gefährdet. Geflügel, TIERWELT / 1/ 2. <ul class="reflinks"><li><a href="#" data-author="Thakur" data-cit="ThakurM.%20and%20Sch%C3%A4ttinE.W.%2C%20%282014%29.%20Die%20Ahnen%20des%20Haushuhns%20sind%20gef%C3%A4hrdet.%20Gefl%C3%BCgel%2C%20TIERWELT%20%2F%201%2F%202." data-title="Die%20Ahnen%20des%20Haushuhns%20sind%20gef%C3%A4hrdet.%20Gefl%C3%BCgel" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Die+Ahnen+des+Haushuhns+sind+gef%C3%A4hrdet.+Gefl%C3%BCgel+Thakur+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0204351.ref020" id="pone.0204351.ref020" class="link-target"></a>Fuller R. A. and Garson P.J. (eds) (2000). Pheasants: status survey and conservation action plan 2000–2004. WPA/Birdlife/SSC Pheasant Specialist Group. IUCN, Gland. Switzerland and Cambridge, U.K. and the World Pheasant Association, Reading, U.K. <ul class="find-nolinks"></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0204351.ref021" id="pone.0204351.ref021" class="link-target"></a>Mackey K., Williams P., Seim S., and Chomczynski P. (1996). The use of DNAzol for the rapid isolation of genomic DNA from whole blood. <em>Biomedical Products Supplement</em>, 13–15. <ul class="reflinks"><li><a href="#" data-author="Mackey" data-cit="MackeyK.%2C%20WilliamsP.%2C%20SeimS.%2C%20and%20ChomczynskiP.%20%281996%29.%20The%20use%20of%20DNAzol%20for%20the%20rapid%20isolation%20of%20genomic%20DNA%20from%20whole%20blood.%20Biomedical%20Products%20Supplement%2C%2013%E2%80%9315." data-title="The%20use%20of%20DNAzol%20for%20the%20rapid%20isolation%20of%20genomic%20DNA%20from%20whole%20blood" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+use+of+DNAzol+for+the+rapid+isolation+of+genomic+DNA+from+whole+blood+Mackey+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0204351.ref022" id="pone.0204351.ref022" class="link-target"></a>FAO. (2011). Molecular genetics characterization of animal genetic resources. FAO Animal Production and Health Guidelines. No. 9. Rome. <ul class="find-nolinks"></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0204351.ref023" id="pone.0204351.ref023" class="link-target"></a>FAO/MoDAD. (2004). Secondary Guidelines. Measurement of Domestic Animal Diversity (MoDAD): Recommended Microsatellite Markers. Available at <a href="http://fao.org/dad-is">http://fao.org/dad-is</a>. <ul class="find-nolinks"></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0204351.ref024" id="pone.0204351.ref024" class="link-target"></a>Mwacharo J.M., Nomura K., Hanada H., Amano T., Han H.J. and Hanotte O. (2013). Reconstructing the origin and dispersal patterns of village chickens across East Africa: insights from autosomal markers. <em>Mol</em>. <em>Ecol</em>., 22: 2683–2697.  pmid:23611649 <ul class="reflinks"><li><a href="#" data-author="Mwacharo" data-cit="MwacharoJ.M.%2C%20NomuraK.%2C%20HanadaH.%2C%20AmanoT.%2C%20HanH.J.%20and%20HanotteO.%20%282013%29.%20Reconstructing%20the%20origin%20and%20dispersal%20patterns%20of%20village%20chickens%20across%20East%20Africa%3A%20insights%20from%20autosomal%20markers.%20Mol.%20Ecol.%2C%2022%3A%202683%E2%80%932697.%20doi%3A%2010.1111%2Fmec.12294%2023611649" data-title="Reconstructing%20the%20origin%20and%20dispersal%20patterns%20of%20village%20chickens%20across%20East%20Africa%3A%20insights%20from%20autosomal%20markers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23611649" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Reconstructing+the+origin+and+dispersal+patterns+of+village+chickens+across+East+Africa%3A+insights+from+autosomal+markers+Mwacharo+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0204351.ref025" id="pone.0204351.ref025" class="link-target"></a>Mukesh , Kalsi R.S., Mandhan R.P. and Sathyakumar S. (2011). Genetic diversity studies of red jungle fowl across its distribution range in Northern India. <em>Asian J Biot</em>, 3: 293–301. <ul class="reflinks"><li><a href="#" data-author="Mukesh" data-cit="Mukesh%2C%20KalsiR.S.%2C%20MandhanR.P.%20and%20SathyakumarS.%20%282011%29.%20Genetic%20diversity%20studies%20of%20red%20jungle%20fowl%20across%20its%20distribution%20range%20in%20Northern%20India.%20Asian%20J%20Biot%2C%203%3A%20293%E2%80%93301." data-title="Genetic%20diversity%20studies%20of%20red%20jungle%20fowl%20across%20its%20distribution%20range%20in%20Northern%20India" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetic+diversity+studies+of+red+jungle+fowl+across+its+distribution+range+in+Northern+India+Mukesh+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0204351.ref026" id="pone.0204351.ref026" class="link-target"></a>Mukesh , Fernandes M., Jianlin H. and Sathyakumar S. (2013). Genetics driven interventions for ex situ conservation of red junglefowl (Gallus gallus murghi) populations in India. Zoo Biol, 32(5):476–83. <ul class="reflinks"><li><a href="#" data-author="Mukesh" data-cit="Mukesh%2C%20FernandesM.%2C%20JianlinH.%20and%20SathyakumarS.%20%282013%29.%20Genetics%20driven%20interventions%20for%20ex%20situ%20conservation%20of%20red%20junglefowl%20%28Gallus%20gallus%20murghi%29%20populations%20in%20India.%20Zoo%20Biol%2C%2032%285%29%3A476%E2%80%9383." data-title="Genetics%20driven%20interventions%20for%20ex%20situ%20conservation%20of%20red%20junglefowl%20%28Gallus%20gallus%20murghi%29%20populations%20in%20India" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genetics+driven+interventions+for+ex+situ+conservation+of+red+junglefowl+%28Gallus+gallus+murghi%29+populations+in+India+Mukesh+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0204351.ref027" id="pone.0204351.ref027" class="link-target"></a>Desjardins P and Morais R. (1990). Sequence and gene organization of the chicken mitochondrial genome. A novel gene order in higher vertebrates. <em>J Mol Biol</em> 212: 599–634.  pmid:2329578 <ul class="reflinks"><li><a href="#" data-author="Desjardins" data-cit="DesjardinsP%20and%20MoraisR.%20%281990%29.%20Sequence%20and%20gene%20organization%20of%20the%20chicken%20mitochondrial%20genome.%20A%20novel%20gene%20order%20in%20higher%20vertebrates.%20J%20Mol%20Biol%20212%3A%20599%E2%80%93634.%20doi%3A%2010.1016%2F0022-2836%2890%2990225-B%202329578" data-title="Sequence%20and%20gene%20organization%20of%20the%20chicken%20mitochondrial%20genome.%20A%20novel%20gene%20order%20in%20higher%20vertebrates" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2329578" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence+and+gene+organization+of+the+chicken+mitochondrial+genome.+A+novel+gene+order+in+higher+vertebrates+Desjardins+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0204351.ref028" id="pone.0204351.ref028" class="link-target"></a>Komiyama T., Ikeo K. and Gojobori T. (2003). Where is the origin of the Japanese gamecocks? <em>Gene</em> 317: 195–202. pmid:14604808 <ul class="reflinks"><li><a href="#" data-author="Komiyama" data-cit="KomiyamaT.%2C%20IkeoK.%20and%20GojoboriT.%20%282003%29.%20Where%20is%20the%20origin%20of%20the%20Japanese%20gamecocks%3F%20Gene%20317%3A%20195%E2%80%93202.%2014604808" data-title="Where%20is%20the%20origin%20of%20the%20Japanese%20gamecocks%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/14604808" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Where+is+the+origin+of+the+Japanese+gamecocks%3F+Komiyama+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0204351.ref029" id="pone.0204351.ref029" class="link-target"></a>Peakall R and Smouse P.E. (2012) GenAlEx 6.5: genetic analysis in Excel. Population genetics software for teaching and research-an update. <em>Bioinformatics</em>, 28:2537–2539.  pmid:22820204 <ul class="reflinks"><li><a href="#" data-author="Peakall" data-cit="PeakallR%20and%20SmouseP.E.%20%282012%29%20GenAlEx%206.5%3A%20genetic%20analysis%20in%20Excel.%20Population%20genetics%20software%20for%20teaching%20and%20research-an%20update.%20Bioinformatics%2C%2028%3A2537%E2%80%932539.%20doi%3A%2010.1093%2Fbioinformatics%2Fbts460%2022820204" data-title="GenAlEx%206.5%3A%20genetic%20analysis%20in%20Excel.%20Population%20genetics%20software%20for%20teaching%20and%20research-an%20update" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22820204" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=GenAlEx+6.5%3A+genetic+analysis+in+Excel.+Population+genetics+software+for+teaching+and+research-an+update+Peakall+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0204351.ref030" id="pone.0204351.ref030" class="link-target"></a>Goudet J. (2001). FSTAT, a program to estimate and test gene diversities and fixation indices (Version 2.9.3.2). Available online at: <a href="http://www2.unil.ch/popgen/softwares/fstat.htm">http://www2.unil.ch/popgen/softwares/fstat.htm</a> [last accessed February 2012]. <ul class="find-nolinks"></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0204351.ref031" id="pone.0204351.ref031" class="link-target"></a>Kalinowski S.T., Taper M.L. and Marshall T.C. (2007) Revising how the computer program CERVUS accommodates genotyping error increases success in paternity assignment. <em>Mol Ecol</em> 16:1099–1106.  pmid:17305863 <ul class="reflinks"><li><a href="#" data-author="Kalinowski" data-cit="KalinowskiS.T.%2C%20TaperM.L.%20and%20MarshallT.C.%20%282007%29%20Revising%20how%20the%20computer%20program%20CERVUS%20accommodates%20genotyping%20error%20increases%20success%20in%20paternity%20assignment.%20Mol%20Ecol%2016%3A1099%E2%80%931106.%20doi%3A%2010.1111%2Fj.1365-294X.2007.03089.x%2017305863" data-title="Revising%20how%20the%20computer%20program%20CERVUS%20accommodates%20genotyping%20error%20increases%20success%20in%20paternity%20assignment" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17305863" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Revising+how+the+computer+program+CERVUS+accommodates+genotyping+error+increases+success+in+paternity+assignment+Kalinowski+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0204351.ref032" id="pone.0204351.ref032" class="link-target"></a>Guo S.W. and Thompson E.A. (1992). Performing the exact test of Hardy–Weinberg proportion for multiple alleles. <em>Biometrics</em> 48:361–372. pmid:1637966 <ul class="reflinks"><li><a href="#" data-author="Guo" data-cit="GuoS.W.%20and%20ThompsonE.A.%20%281992%29.%20Performing%20the%20exact%20test%20of%20Hardy%E2%80%93Weinberg%20proportion%20for%20multiple%20alleles.%20Biometrics%2048%3A361%E2%80%93372.%201637966" data-title="Performing%20the%20exact%20test%20of%20Hardy%E2%80%93Weinberg%20proportion%20for%20multiple%20alleles" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1637966" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Performing+the+exact+test+of+Hardy%E2%80%93Weinberg+proportion+for+multiple+alleles+Guo+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0204351.ref033" id="pone.0204351.ref033" class="link-target"></a>Raymond M. and Rousset F. (1995). GENEPOP (Version 1.2): population genetics software for exact tests and ecumenicism. <em>J Hered</em> 86:248–249. <ul class="reflinks"><li><a href="#" data-author="Raymond" data-cit="RaymondM.%20and%20RoussetF.%20%281995%29.%20GENEPOP%20%28Version%201.2%29%3A%20population%20genetics%20software%20for%20exact%20tests%20and%20ecumenicism.%20J%20Hered%2086%3A248%E2%80%93249." data-title="GENEPOP%20%28Version%201.2%29%3A%20population%20genetics%20software%20for%20exact%20tests%20and%20ecumenicism" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=GENEPOP+%28Version+1.2%29%3A+population+genetics+software+for+exact+tests+and+ecumenicism+Raymond+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0204351.ref034" id="pone.0204351.ref034" class="link-target"></a>Weir B.S. and Cockerham C.C. (1984). Estimating F-statistics for the analysis of population structure. <em>Evolution</em> 38:1358–1370.  pmid:28563791 <ul class="reflinks"><li><a href="#" data-author="Weir" data-cit="WeirB.S.%20and%20CockerhamC.C.%20%281984%29.%20Estimating%20F-statistics%20for%20the%20analysis%20of%20population%20structure.%20Evolution%2038%3A1358%E2%80%931370.%20doi%3A%2010.1111%2Fj.1558-5646.1984.tb05657.x%2028563791" data-title="Estimating%20F-statistics%20for%20the%20analysis%20of%20population%20structure" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/28563791" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Estimating+F-statistics+for+the+analysis+of+population+structure+Weir+1984" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0204351.ref035" id="pone.0204351.ref035" class="link-target"></a>Pritchard J.K., Stephens M. and Donnelly P. (2000). Inference of population structure using multi-locus genotype data. <em>Genetics</em>, 155:945–959. pmid:10835412 <ul class="reflinks"><li><a href="#" data-author="Pritchard" data-cit="PritchardJ.K.%2C%20StephensM.%20and%20DonnellyP.%20%282000%29.%20Inference%20of%20population%20structure%20using%20multi-locus%20genotype%20data.%20Genetics%2C%20155%3A945%E2%80%93959.%2010835412" data-title="Inference%20of%20population%20structure%20using%20multi-locus%20genotype%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Inference+of+population+structure+using+multi-locus+genotype+data+Pritchard+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0204351.ref036" id="pone.0204351.ref036" class="link-target"></a>Evanno G., Regnaut S. and Goudet J. (2005). Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study. <em>Mol Ecol</em> 14:2611–2620.  pmid:15969739 <ul class="reflinks"><li><a href="#" data-author="Evanno" data-cit="EvannoG.%2C%20RegnautS.%20and%20GoudetJ.%20%282005%29.%20Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study.%20Mol%20Ecol%2014%3A2611%E2%80%932620.%20doi%3A%2010.1111%2Fj.1365-294X.2005.02553.x%2015969739" data-title="Detecting%20the%20number%20of%20clusters%20of%20individuals%20using%20the%20software%20STRUCTURE%3A%20a%20simulation%20study" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15969739" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Detecting+the+number+of+clusters+of+individuals+using+the+software+STRUCTURE%3A+a+simulation+study+Evanno+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0204351.ref037" id="pone.0204351.ref037" class="link-target"></a>Mukesh , Kumar V.P., Sharma L.K., Shukla M., and a S. (2015a). Pragmatic Perspective on Conservation genetics and demographic history of the last surviving population of Kashmir red deer (Cervus elaphus hanglu) in India. PLoS ONE 10(2): e0117069.  pmid:25671567 <ul class="reflinks"><li><a href="#" data-author="Mukesh" data-cit="Mukesh%2C%20KumarV.P.%2C%20SharmaL.K.%2C%20ShuklaM.%2C%20and%20aS.%20%282015a%29.%20Pragmatic%20Perspective%20on%20Conservation%20genetics%20and%20demographic%20history%20of%20the%20last%20surviving%20population%20of%20Kashmir%20red%20deer%20%28Cervus%20elaphus%20hanglu%29%20in%20India.%20PLoS%20ONE%2010%282%29%3A%20e0117069.%20doi%3A%2010.1371%2Fjournal.pone.0117069%2025671567" data-title="Pragmatic%20Perspective%20on%20Conservation%20genetics%20and%20demographic%20history%20of%20the%20last%20surviving%20population%20of%20Kashmir%20red%20deer%20%28Cervus%20elaphus%20hanglu%29%20in%20India" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25671567" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Pragmatic+Perspective+on+Conservation+genetics+and+demographic+history+of+the+last+surviving+population+of+Kashmir+red+deer+%28Cervus+elaphus+hanglu%29+in+India+Mukesh+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0204351.ref038" id="pone.0204351.ref038" class="link-target"></a>Mukesh , Sharma L.K., Charoo S.A. and Sathyakumar S. (2015b). Conflict Bear Translocation: Investigating Population Genetics and Fate of Bear Translocation in Dachigam National Park, Jammu and Kashmir, India. PLoS ONE 10(8): e0132005.  pmid:26267280 <ul class="reflinks"><li><a href="#" data-author="Mukesh" data-cit="Mukesh%2C%20SharmaL.K.%2C%20CharooS.A.%20and%20SathyakumarS.%20%282015b%29.%20Conflict%20Bear%20Translocation%3A%20Investigating%20Population%20Genetics%20and%20Fate%20of%20Bear%20Translocation%20in%20Dachigam%20National%20Park%2C%20Jammu%20and%20Kashmir%2C%20India.%20PLoS%20ONE%2010%288%29%3A%20e0132005.%20doi%3A%2010.1371%2Fjournal.pone.0132005%2026267280" data-title="Conflict%20Bear%20Translocation%3A%20Investigating%20Population%20Genetics%20and%20Fate%20of%20Bear%20Translocation%20in%20Dachigam%20National%20Park%2C%20Jammu%20and%20Kashmir%2C%20India" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/26267280" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Conflict+Bear+Translocation%3A+Investigating+Population+Genetics+and+Fate+of+Bear+Translocation+in+Dachigam+National+Park%2C+Jammu+and+Kashmir%2C+India+Mukesh+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0204351.ref039" id="pone.0204351.ref039" class="link-target"></a>Guillot G, Mortier F. and Estoup A. (2005) GENELAND: a computer package for landscape genetics. <em>Mol Ecol Notes</em> 5:712–715. <ul class="reflinks"><li><a href="#" data-author="Guillot" data-cit="GuillotG%2C%20MortierF.%20and%20EstoupA.%20%282005%29%20GENELAND%3A%20a%20computer%20package%20for%20landscape%20genetics.%20Mol%20Ecol%20Notes%205%3A712%E2%80%93715." data-title="GENELAND%3A%20a%20computer%20package%20for%20landscape%20genetics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=GENELAND%3A+a+computer+package+for+landscape+genetics+Guillot+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0204351.ref040" id="pone.0204351.ref040" class="link-target"></a>Librado P. and Rozas J. (2009). DnaSP v5: A software for comprehensive analysis of DNA polymorphism data. <em>Bioinformatics</em>, 25:1451–1452.  pmid:19346325 <ul class="reflinks"><li><a href="#" data-author="Librado" data-cit="LibradoP.%20and%20RozasJ.%20%282009%29.%20DnaSP%20v5%3A%20A%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data.%20Bioinformatics%2C%2025%3A1451%E2%80%931452.%20doi%3A%2010.1093%2Fbioinformatics%2Fbtp187%2019346325" data-title="DnaSP%20v5%3A%20A%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/19346325" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=DnaSP+v5%3A+A+software+for+comprehensive+analysis+of+DNA+polymorphism+data+Librado+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0204351.ref041" id="pone.0204351.ref041" class="link-target"></a>Rogers A.R. and Harpending H. (1992) Population growth makes waves in the distribution of pairwise genetic differences. <em>Mol Biol Evol</em> 9:552–569.  pmid:1316531 <ul class="reflinks"><li><a href="#" data-author="Rogers" data-cit="RogersA.R.%20and%20HarpendingH.%20%281992%29%20Population%20growth%20makes%20waves%20in%20the%20distribution%20of%20pairwise%20genetic%20differences.%20Mol%20Biol%20Evol%209%3A552%E2%80%93569.%20doi%3A%2010.1093%2Foxfordjournals.molbev.a040727%201316531" data-title="Population%20growth%20makes%20waves%20in%20the%20distribution%20of%20pairwise%20genetic%20differences" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1316531" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Population+growth+makes+waves+in+the+distribution+of+pairwise+genetic+differences+Rogers+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0204351.ref042" id="pone.0204351.ref042" class="link-target"></a>Tajima H. (1989) Statistical method for testing the neutral mutation hypothesis by DNA polymorphism. <em>Genetics</em>, 123: 585–595. pmid:2513255 <ul class="reflinks"><li><a href="#" data-author="Tajima" data-cit="TajimaH.%20%281989%29%20Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism.%20Genetics%2C%20123%3A%20585%E2%80%93595.%202513255" data-title="Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Statistical+method+for+testing+the+neutral+mutation+hypothesis+by+DNA+polymorphism+Tajima+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0204351.ref043" id="pone.0204351.ref043" class="link-target"></a>Fu Y. (1997) Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection. <em>Genetics</em>, 147: 915–925. pmid:9335623 <ul class="reflinks"><li><a href="#" data-author="Fu" data-cit="FuY.%20%281997%29%20Statistical%20tests%20of%20neutrality%20of%20mutations%20against%20population%20growth%2C%20hitchhiking%20and%20background%20selection.%20Genetics%2C%20147%3A%20915%E2%80%93925.%209335623" data-title="Statistical%20tests%20of%20neutrality%20of%20mutations%20against%20population%20growth%2C%20hitchhiking%20and%20background%20selection" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9335623" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Statistical+tests+of+neutrality+of+mutations+against+population+growth%2C+hitchhiking+and+background+selection+Fu+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="pone.0204351.ref044" id="pone.0204351.ref044" class="link-target"></a>Fu Y and Li W. (1993) Statistical tests on neutrality of mutations. <em>Genetics</em> 133, 693–709. pmid:8454210 <ul class="reflinks"><li><a href="#" data-author="Fu" data-cit="FuY%20and%20LiW.%20%281993%29%20Statistical%20tests%20on%20neutrality%20of%20mutations.%20Genetics%20133%2C%20693%E2%80%93709.%208454210" data-title="Statistical%20tests%20on%20neutrality%20of%20mutations" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8454210" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Statistical+tests+on+neutrality+of+mutations+Fu+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="pone.0204351.ref045" id="pone.0204351.ref045" class="link-target"></a>Bandelt HJ, Forster P, and Ro¨hl A.(1999) Median-joining networks for inferring intraspecific phylogenies. <em>Mol Biol Evol</em> 16:37–48.  pmid:10331250 <ul class="reflinks"><li><a href="#" data-author="Bandelt" data-cit="BandeltHJ%2C%20ForsterP%2C%20and%20Ro%C2%A8hlA.%281999%29%20Median-joining%20networks%20for%20inferring%20intraspecific%20phylogenies.%20Mol%20Biol%20Evol%2016%3A37%E2%80%9348.%20doi%3A%2010.1093%2Foxfordjournals.molbev.a026036%2010331250" data-title="Median-joining%20networks%20for%20inferring%20intraspecific%20phylogenies" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10331250" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Median-joining+networks+for+inferring+intraspecific+phylogenies+Bandelt+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="pone.0204351.ref046" id="pone.0204351.ref046" class="link-target"></a>Kumar S., Stecher G. and Tamura K. (2016). MEGA7: Molecular Evolutionary Genetics Analysis version 7.0 for bigger datasets. <em>Mol Biol Evol</em>, 33:1870–1874  pmid:27004904 <ul class="reflinks"><li><a href="#" data-author="Kumar" data-cit="KumarS.%2C%20StecherG.%20and%20TamuraK.%20%282016%29.%20MEGA7%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20version%207.0%20for%20bigger%20datasets.%20Mol%20Biol%20Evol%2C%2033%3A1870%E2%80%931874%20doi%3A%2010.1093%2Fmolbev%2Fmsw054%2027004904" data-title="MEGA7%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20version%207.0%20for%20bigger%20datasets" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MEGA7%3A+Molecular+Evolutionary+Genetics+Analysis+version+7.0+for+bigger+datasets+Kumar+2016" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="pone.0204351.ref047" id="pone.0204351.ref047" class="link-target"></a>Statham M. J., Trut L. N., Sacks B. N., Kharlamova A. V., Oskina I. N., Gulevich R. G. et al. (2011). On the origin of a domesticated species: Identifying the parent population of Russian silver foxes (<em>Vulpes vulpes)</em>. <em>Biol J Linn Soc Lond</em> 103(1):168–175.  pmid:21625363 <ul class="reflinks"><li><a href="#" data-author="Statham" data-cit="StathamM.%20J.%2C%20TrutL.%20N.%2C%20SacksB.%20N.%2C%20KharlamovaA.%20V.%2C%20OskinaI.%20N.%2C%20GulevichR.%20G.%20et%20al.%20%282011%29.%20On%20the%20origin%20of%20a%20domesticated%20species%3A%20Identifying%20the%20parent%20population%20of%20Russian%20silver%20foxes%20%28Vulpes%20vulpes%29.%20Biol%20J%20Linn%20Soc%20Lond%20103%281%29%3A168%E2%80%93175.%20doi%3A%2010.1111%2Fj.1095-8312.2011.01629.x%2021625363" data-title="On%20the%20origin%20of%20a%20domesticated%20species%3A%20Identifying%20the%20parent%20population%20of%20Russian%20silver%20foxes%20%28Vulpes%20vulpes%29" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21625363" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=On+the+origin+of+a+domesticated+species%3A+Identifying+the+parent+population+of+Russian+silver+foxes+%28Vulpes+vulpes%29+Statham+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="pone.0204351.ref048" id="pone.0204351.ref048" class="link-target"></a>Johnsingh A.J.T and Negi A.S. (2003). Status of tiger and leopard in Rajaji–Corbett Conservation Unit, Northern India. Biological Conservation. 111, 385–393 <ul class="reflinks"><li><a href="#" data-author="Johnsingh" data-cit="JohnsinghA.J.T%20and%20NegiA.S.%20%282003%29.%20Status%20of%20tiger%20and%20leopard%20in%20Rajaji%E2%80%93Corbett%20Conservation%20Unit%2C%20Northern%20India.%20Biological%20Conservation.%20111%2C%20385%E2%80%93393" data-title="Status%20of%20tiger%20and%20leopard%20in%20Rajaji%E2%80%93Corbett%20Conservation%20Unit%2C%20Northern%20India" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Status+of+tiger+and+leopard+in+Rajaji%E2%80%93Corbett+Conservation+Unit%2C+Northern+India+Johnsingh+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

