



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>RNA silencing or interference (RNAi) is a gene regulation mechanism in eukaryotes that controls cell differentiation and developmental processes via expression of microRNAs. RNAi also serves as an innate antiviral defence response in plants, nematodes, and insects. This antiviral response is triggered by virus-specific double-stranded RNA molecules (dsRNAs) that are produced during infection. To overcome antiviral RNAi responses, many plant and insect viruses encode RNA silencing suppressors (RSSs) that enable them to replicate at higher titers. Recently, several human viruses were shown to encode RSSs, suggesting that RNAi also serves as an innate defence response in mammals. Here, we demonstrate that the Ebola virus VP35 protein is a suppressor of RNAi in mammalian cells and that its RSS activity is functionally equivalent to that of the HIV-1 Tat protein. We show that VP35 can replace HIV-1 Tat and thereby support the replication of a Tat-minus HIV-1 variant. The VP35 dsRNA-binding domain is required for this RSS activity. Vaccinia virus E3L protein and influenza A virus NS1 protein are also capable of replacing the HIV-1 Tat RSS function. These findings support the hypothesis that RNAi is part of the innate antiviral response in mammalian cells. Moreover, the results indicate that RSSs play a critical role in mammalian virus replication.</p>
</div><div class="abstract toc-section"><a id="abstract1" name="abstract1" class="link-target" data-toc="abstract1" title="Author Summary"></a>
<h2>Author Summary</h2>

<div id="section1" class="section toc-section">

<a id="article1.front1.article-meta1.abstract2.sec1.p1" name="article1.front1.article-meta1.abstract2.sec1.p1" class="link-target"></a><p>Cells have evolved mechanisms to protect themselves from virus infection. A well-known antiviral mechanism in mammals is the interferon (IFN) response of the innate immune system. In plants, insects, and worms, RNA silencing or RNA interference (RNAi) is a strong antiviral defence mechanism. It is still debated whether RNAi is also used as an antiviral mechanism in mammals. Many mammalian viruses encode essential factors that suppress the innate antiviral responses of the host. Such innate immunity suppressor proteins, or IFN antagonists, have recently been reported to also suppress RNAi in mammalian cells. We now demonstrate that the Ebola virus VP35 protein, a known IFN antagonist, suppresses RNAi in human cells. In addition, VP35 restores the production of an HIV-1 variant with a defective RNAi suppressor Tat protein. These results indicate that RNAi is part of the innate antiviral defence response in mammals and that viruses need to counteract this response in order to replicate. Whereas RNAi and INF act in concert to prevent the infection of mammalian cells, the invading viruses encode a protein that counteracts both defence mechanisms.</p>
</div>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Haasnoot J, de Vries W, Geutjes E-J, Prins M, de Haan P, Berkhout B (2007) The Ebola Virus VP35 Protein Is a Suppressor of RNA Silencing. PLoS Pathog 3(6):
           e86.
        
        https://doi.org/10.1371/journal.ppat.0030086</p><p><strong>Editor: </strong>Yoshihiro Kawaoka, University of Wisconsin-Madison, United States of America</p><p><strong>Received: </strong>August 31, 2006; <strong>Accepted: </strong>May 10, 2007; <strong>Published: </strong> June 22, 2007</p><p><strong>Copyright: </strong> © 2007 Haasnoot et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research is supported by Senter-NOVEM.</p><p><strong>Competing interests: </strong>  The authors have declared that no competing interests exist.</p><p><strong>Abbreviations:
          </strong>CMV,
            cytomegalovirus; dox,
            doxycycline; dsRNA,
            double-stranded RNA; EBOV,
            Ebola virus; E3L,
            vaccinia virus E3L; GFP,
            green fluorescent protein; HIV-1,
            human immunodeficiency virus type 1; IFN,
            interferon; LTR,
            long terminal repeat; NS1,
            influenza A virus NS1; PFV-1,
            primate foamy virus type 1; PKR,
            protein kinase R; RIG-I/MDA5,
            retinoic-acid-inducible protein I/melanoma-differentiation-associated gene 5; RNAi,
            RNA interference; RSS,
            RNA silencing suppressor; shRNA,
            short hairpin RNA; shLuc,
            firefly luciferase–specific shRNA; siRNA,
            small interfering RNA</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>An important criterion for productive virus infection is that the virus evades host antiviral immune responses. In plants, insects, and nematodes, the basis of these protective immune responses is formed by the RNA interference (RNAi) mechanism [<a href="#ppat-0030086-b001" class="ref-tip">1</a>–<a href="#ppat-0030086-b004" class="ref-tip">4</a>]. During virus infection, RNAi against the virus is activated by the production of virus-specific double-stranded RNAs (dsRNAs). These virus-specific dsRNAs are processed into small interfering RNAs (siRNAs; a 21-nucleotide dsRNA duplex) by the RNAse III–like endonuclease-denoted Dicer. Subsequently, one strand of the siRNA duplex, the guide-strand, is incorporated into the RNA-induced silencing complex (RISC) to target viral mRNAs bearing complementary sequences for destruction. To overcome this antiviral RNAi response, viruses encode RNA silencing suppressors (RSSs) [<a href="#ppat-0030086-b005" class="ref-tip">5</a>]. For plant viruses, RSSs were first described as pathogenicity factors that contribute to high virus accumulation and disease. One of the best-characterized suppressors is the tombusvirus-encoded P19 protein. This protein, which suppresses RNAi both in plants and mammalian cells, blocks RNAi by binding siRNAs via its dsRNA-binding domain, thereby sequestering the siRNAs from the RNAi pathway [<a href="#ppat-0030086-b006" class="ref-tip">6</a>]. Another way to block RNAi is via inhibition of Dicer activity. For example, the turnip crinkle virus P83 protein was recently shown to specifically block the activity of the Dicer-like 4 protein [<a href="#ppat-0030086-b007" class="ref-tip">7</a>].</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Activation of RNAi in mammalian cells, either by transfection of synthetic siRNAs or by endogenous expression of short hairpin RNAs (shRNAs), is a potent new antiviral tool [<a href="#ppat-0030086-b008" class="ref-tip">8</a>]. These findings support the idea that RNAi is part of the innate immune system in mammals. However, in most cases, virus-specific siRNAs could not be detected in virus-infected mammalian cells [<a href="#ppat-0030086-b009" class="ref-tip">9</a>]. So far, virus-specific siRNAs have only been identified in human cells for human immunodeficiency virus type 1 (HIV-1) and the LINE-1 retrotransposon [<a href="#ppat-0030086-b010" class="ref-tip">10</a>–<a href="#ppat-0030086-b012" class="ref-tip">12</a>]. It has been argued that mammalian cells do not need RNAi-based antiviral responses because they have acquired the interferon (IFN) response [<a href="#ppat-0030086-b013" class="ref-tip">13</a>]. However, all other eukaryotes also evolved innate antiviral defence responses. For instance, plants have pattern recognition receptors, and virus recognition leads to apoptosis and the systemic acquired resistance response that is analogous to the IFN response in mammalian cells [<a href="#ppat-0030086-b014" class="ref-tip">14</a>]. Similar to RNAi, the IFN pathway is triggered by cytoplasmic viral dsRNAs and acts as a sensitive and potent antiviral response that is involved in innate and subsequent adaptive immunity.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>If RNAi has an antiviral function in mammals, then the infecting viruses should encode RSSs as they do in plant and insect viruses. Recently, several mammalian viruses have been shown to encode viral factors that exhibit RSS activity in animal cells. These factors include the influenza A virus NS1 (NS1), vaccinia virus E3L (E3L), hepatitis C virus Core, primate foamy virus type 1 (PFV-1) Tas, and the HIV-1 Tat proteins, as well as the adenovirus virus-associated RNAs I and II (VAI and VAII) [<a href="#ppat-0030086-b011" class="ref-tip">11</a>,<a href="#ppat-0030086-b015" class="ref-tip">15</a>–<a href="#ppat-0030086-b018" class="ref-tip">18</a>]. Like plant virus RSSs, these suppressors block induced RNAi against reporter gene constructs. Moreover, NS1 and E3L were able to replace the RNAi suppression function of the b2 protein encoded by Flock house virus and to support virus replication in insect cells [<a href="#ppat-0030086-b017" class="ref-tip">17</a>]. Although most of these viral proteins/RNAs are essential for virus replication, their mode of action is largely unknown. We and others have shown that the adenovirus virus-associated RNAs inhibit RNAi by acting as decoy substrates for Exportin 5, Dicer, and RISC [<a href="#ppat-0030086-b016" class="ref-tip">16</a>,<a href="#ppat-0030086-b019" class="ref-tip">19</a>]. HIV-1 Tat is thought to block Dicer activity, whereas NS1 and E3L may sequester dsRNAs and siRNAs [<a href="#ppat-0030086-b011" class="ref-tip">11</a>,<a href="#ppat-0030086-b017" class="ref-tip">17</a>,<a href="#ppat-0030086-b020" class="ref-tip">20</a>]. Strikingly, all RSS proteins from mammalian viruses possess IFN or protein kinase R (PKR) antagonistic properties, suggesting that RNAi and other innate antiviral responses are interrelated [<a href="#ppat-0030086-b021" class="ref-tip">21</a>–<a href="#ppat-0030086-b024" class="ref-tip">24</a>].</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Although there is consensus about the fact that viral factors can indeed block RNAi in mammalian cells, there is an ongoing debate about the significance for viral replication [<a href="#ppat-0030086-b013" class="ref-tip">13</a>]. Because non-viral dsRNA-binding proteins such as the bacterial RNase-III protein can also act as RSSs in plant cells, it is important to determine the contribution of RNAi suppression to virus replication [<a href="#ppat-0030086-b025" class="ref-tip">25</a>]. Here, we show that Tat-mediated RNAi suppression is required for HIV-1 virus production. Using defective Tat-minus HIV-1 mutants, we identified the Ebola virus (EBOV) VP35 protein as a potent RNAi suppressor that is functionally equivalent to the HIV-1 Tat RSS function. Furthermore, we show that the E3L and NS1 proteins can also functionally complement the Tat RSS function and rescue virus production of the HIV-1 Tat-minus mutants. These data support the role of RNAi in innate antiviral responses in mammalian cells and the essential role of RNAi suppression in HIV-1 replication.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Testing Viral IFN Antagonistic Proteins for RNAi Suppression Activity"></a>
<h3>Testing Viral IFN Antagonistic Proteins for RNAi Suppression Activity</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>At present, all RSS proteins identified in mammalian viruses have IFN/PKR antagonistic properties. Previously, it was shown that the EBOV IFN antagonist protein VP35 is capable of restoring the replication of an influenza A virus mutant with a large deletion in the NS1 gene [<a href="#ppat-0030086-b021" class="ref-tip">21</a>,<a href="#ppat-0030086-b026" class="ref-tip">26</a>], raising the possibility that VP35 acts as an RSS in mammalian cells. To investigate the RNAi suppressor activity of EBOV VP35, we assayed whether this protein is capable of suppressing shRNA-mediated silencing of a luciferase reporter. HEK293T cells were co-transfected with expression vectors for luciferase and firefly luciferase–specific shRNA (shLuc) [<a href="#ppat-0030086-b027" class="ref-tip">27</a>], and with an EBOV VP35 expression plasmid containing the human EF1α promoter. Co-transfection of luciferase with shLuc resulted in a strong decrease of luciferase expression (<a href="#ppat-0030086-g001">Figure 1</a>A). Addition of the VP35 expression plasmid suppressed the shRNA-mediated RNAi in a dose-dependent manner and restored luciferase expression. To exclude that the observed effect is caused by the IFN antagonistic properties of VP35, its RSS activity was also determined in African green monkey kidney Vero cells that have a defect (IFN<sup>−</sup>) in the IFN pathway [<a href="#ppat-0030086-b028" class="ref-tip">28</a>]. As in HEK293T (IFN<sup>+</sup>) cells, VP35 expression suppressed RNAi in Vero (IFN<sup>−</sup>) cells and rescued luciferase expression (<a href="#ppat-0030086-g001">Figure 1</a>B).</p>
<a class="link-target" id="ppat-0030086-g001" name="ppat-0030086-g001"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g001" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> RNAi Suppression by VP35, NS1, and E3L in Mammalian Cells</div><p class="caption_target"><a id="article1.body1.sec2.sec1.fig1.caption1.p1" name="article1.body1.sec2.sec1.fig1.caption1.p1" class="link-target"></a><p>HEK293T cells or Vero cells were transfected with an expression plasmid for firefly luciferase (Luc), a vector expressing an shRNA against luciferase (shLuc), expression constructs for various RSSs (VP35, NS1, E3L), or a control GFP expression plasmid. Luciferase expression was measured at 2–3 d post transfection.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p2" name="article1.body1.sec2.sec1.fig1.caption1.p2" class="link-target"></a><p>(A and B) Effect of VP35 (100, 300, and 600 ng) on the expression of the silenced luciferase reporter in HEK293T cells (A) and (B) Vero cells.</p>
<a id="article1.body1.sec2.sec1.fig1.caption1.p3" name="article1.body1.sec2.sec1.fig1.caption1.p3" class="link-target"></a><p>(C and D) Effect of NS1, E3L, and GFP (600 ng) on expression of the silenced luciferase reporter in HEK293T cells (C) and in (D) Vero cells.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g001">
              https://doi.org/10.1371/journal.ppat.0030086.g001</a></p></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2" class="link-target"></a><p>Next, we compared the RSS activity of VP35 with that of NS1 and E3L, which were previously demonstrated to have RSS activity in insect and plant cells [<a href="#ppat-0030086-b017" class="ref-tip">17</a>,<a href="#ppat-0030086-b020" class="ref-tip">20</a>,<a href="#ppat-0030086-b029" class="ref-tip">29</a>]. Because insect and plant cells lack the IFN pathway, the RSS activity was determined both in HEK293T (IFN<sup>+</sup>) and Vero (IFN<sup>−</sup>) cells. E3L suppressed RNAi in HEK293T cells, whereas NS1 protein did not (<a href="#ppat-0030086-g001">Figure 1</a>C). However, both E3L and NS1 were able to suppress RNAi in Vero cells. The control expression plasmid encoding green fluorescent protein (GFP) did not suppress RNAi-mediated inhibition of luciferase expression (<a href="#ppat-0030086-g001">Figure 1</a>D). These data show that VP35, E3L, and NS1 are able to suppress shRNA-induced RNAi in mammalian cells. Possibly, the cell type–dependent RSS activity of NS1 follows from differences in the expression level of certain cellular RNAi co-factors.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="VP35 dsRNA-Binding Capacity Is Required for RSS Activity"></a>
<h3>VP35 dsRNA-Binding Capacity Is Required for RSS Activity</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>VP35 has a dsRNA-binding motif with high similarity to the dsRNA-binding domain of the NS1 protein [<a href="#ppat-0030086-b030" class="ref-tip">30</a>]. Mutational analysis has shown that this VP35 domain is important for suppression of type I IFN responses [<a href="#ppat-0030086-b030" class="ref-tip">30</a>–<a href="#ppat-0030086-b032" class="ref-tip">32</a>]. Because binding of dsRNA is one of the strategies to suppress RNA silencing, we wanted to investigate the importance of this domain in VP35-mediated RSS activity [<a href="#ppat-0030086-b033" class="ref-tip">33</a>,<a href="#ppat-0030086-b034" class="ref-tip">34</a>]. The VP35 dsRNA-binding domain has been identified as <sup>304</sup>PRACQKSLRPV<sup>314</sup>. We tested two mutants containing a single amino acid substitution, K309A and R312A, and a double mutant, K309A/R312A. Furthermore, we tested the C-terminal deletion mutant R300T, which lacks 40 amino acids. Mutants K309A and K312A have a defect in dsRNA binding, whereas R300T lacks the complete dsRNA-binding domain [<a href="#ppat-0030086-b030" class="ref-tip">30</a>,<a href="#ppat-0030086-b031" class="ref-tip">31</a>]. The wild-type VP35 (VP35<sup>wt</sup>) and the various mutants were cloned in the expression vector pcDNA3.1 containing the cytomegalovirus (CMV) promoter. HEK293T cells were co-transfected with expression vectors for firefly luciferase, shLuc, the wild-type EBOV VP35<sup>wt</sup>, or the various mutants. In addition, a renilla luciferase expression plasmid (pRL-CMV) was co-transfected as internal control. Co-transfection of the firefly luciferase with shLuc resulted in a strong decrease of luciferase expression (<a href="#ppat-0030086-g002">Figure 2</a>A). Addition of the VP35<sup>wt</sup> expression plasmid suppressed shRNA-mediated RNAi and restored luciferase expression. Interestingly, all mutants, in particular R312A, K309A/R312A, and R300T, were unable to rescue luciferase expression (<a href="#ppat-0030086-g002">Figure 2</a>A). The internal renilla luciferase control was not affected by the various VP35 expression plasmids, and the firefly/renilla luciferase ratio showed a similar trend as the firefly luciferase data. These results indicate that dsRNA binding is essential for VP35-mediated RNAi suppression in the luciferase assay.</p>
<a class="link-target" id="ppat-0030086-g002" name="ppat-0030086-g002"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g002" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> VP35 dsRNA-Binding Capacity Is Required for RSS Activity</div><p class="caption_target"><a id="article1.body1.sec2.sec2.fig1.caption1.p1" name="article1.body1.sec2.sec2.fig1.caption1.p1" class="link-target"></a><p>(A) HEK293T cells were co-transfected with expression vectors for firefly luciferase (Luc), shLuc, the EBOV VP35<sup>wt</sup>, or the various mutants (K309A, R312A, and K309/R12A), or a C-terminal deletion mutant R300T (50 ng). The renilla expression plasmid pRL-CMV was co-transfected as internal control. Luciferase expression was measured at 2–3 d post transfection. From the top down, the panels show the relative expression of firefly luciferase, renilla luciferase, and the ratio firefly/renilla.</p>
<a id="article1.body1.sec2.sec2.fig1.caption1.p2" name="article1.body1.sec2.sec2.fig1.caption1.p2" class="link-target"></a><p>(B) HEK293T cells were co-transfected with expression vectors for firefly luciferase, renilla luciferase, and either the control shLucR, or the active shLuc in the presence and absence (−) of VP35<sup>wt</sup> expression plasmid. Luciferase expression was measured at 2–3 d post transfection. Shown is relative luciferase expression corrected for the internal renilla control (firefly/renilla).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g002">
              https://doi.org/10.1371/journal.ppat.0030086.g002</a></p></div><a id="article1.body1.sec2.sec2.p2" name="article1.body1.sec2.sec2.p2" class="link-target"></a><p>To ensure that the effect of VP35<sup>wt</sup> is specific for the silenced luciferase, we tested the effect of VP35<sup>wt</sup> on luciferase expression in the presence of an inactive shRNA against luciferase, shLucR. In shLucR, the hairpin cassette has been inserted in reversed orientation, making it inactive as RNAi inducer [<a href="#ppat-0030086-b027" class="ref-tip">27</a>] (<a href="#ppat-0030086-g002">Figure 2</a>B). Co-transfection of the luciferase expression plasmid with shLucR did not affect luciferase expression, whereas shLuc induced a marked reduction in luciferase expression. Co-transfection of the VP35<sup>wt</sup> expression plasmid did not stimulate luciferase expression in the presence of shLucR, whereas it did enhance luciferase expression in the presence of shLuc. These results show that VP35-stimulated reporter expression is specific for an RNAi-silenced reporter.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="A Second Function of HIV-1 Tat Is Required for Virus Production"></a>
<h3>A Second Function of HIV-1 Tat Is Required for Virus Production</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>RSS proteins derived from plant or insect viruses, but also NS1 and E3L, have been identified by their ability to functionally complement attenuated viruses that lack their own RSS [<a href="#ppat-0030086-b017" class="ref-tip">17</a>,<a href="#ppat-0030086-b035" class="ref-tip">35</a>]. Since it has recently been shown that the HIV-1 Tat protein has RSS activity, we wanted to determine whether EBOV VP35 is capable of functionally replacing the Tat RSS activity in HIV-1. To do this, one needs a Tat-minus virus, but such variants are completely replication impaired due to a transcription defect. However, we previously constructed the HIV-rtTA virus, in which Tat-mediated transcription transactivation is inactivated and replaced by the Tet-On system (<a href="#ppat-0030086-g003">Figure 3</a>A) [<a href="#ppat-0030086-b036" class="ref-tip">36</a>,<a href="#ppat-0030086-b037" class="ref-tip">37</a>]. This virus is fully dependent on doxycycline (dox) for gene expression and replication. To test whether the Tat function can be removed in this context, we introduced a frame shift at Tat codon 20 by a single nucleotide deletion in the Tat open reading frame. Virus production was measured in HEK293T cells transfected with the HIV-rtTA molecular clone encoding Tat<sup>wt</sup> (HIV-rtTA-Tat<sup>wt</sup>) or the frame shift mutant Tat<sup>fs</sup> (HIV-rtTA-Tat<sup>fs</sup>). Both constructs produced virus in a dox-dependent manner, but Tat<sup>fs</sup> produced much less virus than HIV-rtTA-Tat<sup>wt</sup> (<a href="#ppat-0030086-g003">Figure 3</a>B). Because HIV-rtTA is not dependent on Tat-mediated transcriptional activation, this result indicates that Tat is needed for an additional function.</p>
<a class="link-target" id="ppat-0030086-g003" name="ppat-0030086-g003"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g003" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> The Tat-Minus HIV-1 Complementation System</div><p class="caption_target"><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1" class="link-target"></a><p>(A) Schematic of the HIV-rtTA genome. In HIV-rtTA, the Tat-TAR transcriptional axis has been inactivated by mutation of both the Tat protein and the TAR hairpin (as indicated), and replaced by the tetracycline-inducible tetO-rtTA system. For this, the rtTA gene was inserted in place of the Nef gene and eight tetO sites were inserted in the LTR promoter. Upon administration of dox, rtTA can bind to tetO and activate transcription and viral replication. A frame shift mutation was introduced at codon 20 of the Tat gene.</p>
<a id="article1.body1.sec2.sec3.fig1.caption1.p2" name="article1.body1.sec2.sec3.fig1.caption1.p2" class="link-target"></a><p>(B) Virus production in HEK293T cells transfected with HIV-rtTA-Tat<sup>wt</sup> or HIV-rtTA-Tat<sup>fs</sup> in the absence (−) or presence (+) of dox. CA-p24 in the culture supernatant was measured at 3 d post transfection.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g003">
              https://doi.org/10.1371/journal.ppat.0030086.g003</a></p></div></div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="HIV-1 Tat-Mediated Transactivation of Transcription and RNAi Suppression"></a>
<h3>HIV-1 Tat-Mediated Transactivation of Transcription and RNAi Suppression</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>To investigate whether the observed production defect of HIV-rtTA-Tat<sup>fs</sup> is caused by inactivation of Tat RSS activity, we studied rescue of HIV-rtTA-Tat<sup>fs</sup> by co-transfection of expression plasmids for Tat<sup>wt</sup> and two Tat mutants, Tat<sup>F32A</sup> and Tat<sup>Y26A</sup> [<a href="#ppat-0030086-b038" class="ref-tip">38</a>]. We first determined the transcriptional activity of Tat<sup>F32A</sup> and Tat<sup>Y26A</sup>. For this, we used TZM-bl cells that contain the firefly luciferase reporter gene under control of the HIV-1 long terminal repeat (LTR), which is Tat responsive. Only transfection of Tat<sup>wt</sup> expression plasmid activated luciferase expression, whereas the Tat mutants Tat<sup>F32A</sup> and Tat<sup>Y26A</sup> did not stimulate luciferase expression (<a href="#ppat-0030086-g004">Figure 4</a>A). We next determined the RSS activity of HIV-1 Tat<sup>wt</sup>, Tat<sup>F32A</sup>, and Tat<sup>Y26A</sup> using the luciferase RNAi assay. Tat<sup>wt</sup> and Tat<sup>Y26A</sup> were capable of suppressing shLuc-mediated silencing of the luciferase expression, but mutant Tat<sup>F32A</sup> did not suppress RNAi (<a href="#ppat-0030086-g004">Figure 4</a>B). Because both mutants express stable proteins that lack transactivation capacity [<a href="#ppat-0030086-b038" class="ref-tip">38</a>], this result indicates that RSS activity can be separated from transcriptional activity, which is in agreement with the results described by Bennasser et al. in 2005 [<a href="#ppat-0030086-b011" class="ref-tip">11</a>].</p>
<a class="link-target" id="ppat-0030086-g004" name="ppat-0030086-g004"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g004" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Tat-Mediated RNAi Suppression Is Essential for HIV-rtTA Virus Production</div><p class="caption_target"><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1" class="link-target"></a><p>(A) TZM-bl cells containing a Tat-responsive firefly luciferase reporter gene under control of the HIV-1-LTR were transfected with pBluescript (−), Tat<sup>wt</sup>, Tat<sup>F32A</sup>, or Tat<sup>Y26A</sup>. Luciferase expression was determined 2 d post transfection.</p>
<a id="article1.body1.sec2.sec4.fig1.caption1.p2" name="article1.body1.sec2.sec4.fig1.caption1.p2" class="link-target"></a><p>(B) Transfection of HEK293T cells with the luciferase reporter expression plasmid (Luc), the vector expressing an shRNA against luciferase (Luc + shLuc), and expression plasmids for Tat<sup>wt</sup> , Tat<sup>F32A</sup>, or Tat<sup>Y26A</sup>. Luciferase expression was determined 2 d post transfection.</p>
<a id="article1.body1.sec2.sec4.fig1.caption1.p3" name="article1.body1.sec2.sec4.fig1.caption1.p3" class="link-target"></a><p>(C) Virus production 2 d post transfection in HEK293T cells transfected with HIV-rtTA-Tat<sup>fs</sup> and expression plasmids for Tat<sup>wt</sup>, Tat<sup>F32A</sup>, or Tat<sup>Y26A</sup>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g004">
              https://doi.org/10.1371/journal.ppat.0030086.g004</a></p></div><a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>We next determined whether the production defect of HIV-rtTA-Tat<sup>fs</sup> in HEK293T cells could be restored by co-expression of Tat<sup>wt</sup>, Tat<sup>F32A</sup>, or Tat<sup>Y26A</sup>. Tat<sup>wt</sup> and Tat<sup>Y26A</sup>, which are capable of suppressing shRNA-mediated silencing of luciferase expression, rescued the production of HIV-rtTA-Tat<sup>fs</sup> (<a href="#ppat-0030086-g004">Figure 4</a>C). However, Tat<sup>F32A</sup>, which did not suppress RNAi, could not restore virus production. These data indicate that the defect in HIV-rtTA-Tat<sup>fs</sup> virus production is caused by a <em>trans</em>-acting RSS defect, rather than a <em>cis</em>-acting defect, of the frame shift mutation.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="Complementation of HIV-rtTA-Tatfs by Heterologous RNAi Suppressor Proteins"></a>
<h3>Complementation of HIV-rtTA-Tat<sup>fs</sup> by Heterologous RNAi Suppressor Proteins</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>We subsequently tested whether the HIV-rtTA-Tat<sup>fs</sup> production defect could also be restored by co-transfection of the EF1α expression plasmids for VP35, NS1, or E3L. Indeed, VP35 rescued virus production of HIV-rtTA-Tat<sup>fs</sup> (<a href="#ppat-0030086-g005">Figure 5</a>A). Little RSS activity was measured for E3L and NS1 (<a href="#ppat-0030086-g005">Figure 5</a>A), but increasing activity was measured when more RSS DNA was used in the transfection (<a href="#ppat-0030086-g005">Figure 5</a>B). In the latter case, a GFP reporter was included as a negative control. To exclude that VP35, E3L, NS1, and Tat<sup>wt</sup> stimulate HIV-rtTA-Tat<sup>fs</sup> virus production by inducing promoter activity, we transfected cells with a firefly luciferase reporter under transcriptional control of the 8tetO promoter (the same promoter as in HIV-rtTA-Tat<sup>fs</sup>) with an rtTA expression plasmid and with the different RSS vectors. We measured dox-dependent promoter activity resulting in luciferase expression (<a href="#ppat-0030086-g005">Figure 5</a>C). Co-expression of Tat<sup>wt</sup>, E3L, VP35, NS1, or GFP did not further increase luciferase expression, indicating that these proteins do no activate the 8tetO promoter <em>in trans</em>. The various RSS proteins also did not activate expression of a luciferase reporter under control of a wild-type HIV-1 LTR promoter in TZM-bl cells (<a href="#ppat-0030086-g005">Figure 5</a>D). This shows that the observed rescue of the Tat function by the heterologous RSSs is independent of promoter activation.</p>
<a class="link-target" id="ppat-0030086-g005" name="ppat-0030086-g005"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g005" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> Functional Complementation of the Tat RSS Function by VP35, NS1, and E3L</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig1.caption1.p1" name="article1.body1.sec2.sec5.fig1.caption1.p1" class="link-target"></a><p>(A and B) HEK293T cells were transfected with HIV-rtTA-Tat<sup>wt</sup>, HIV-rtTA-Tat<sup>fs</sup>, and expression plasmids for (A) Tat<sup>wt</sup>, VP35, E3L, and NS1 (50 ng) or (B) E3L, NS1, and GFP (10, 100, and 500 ng). Virus production was determined 2 d post transfection.</p>
<a id="article1.body1.sec2.sec5.fig1.caption1.p2" name="article1.body1.sec2.sec5.fig1.caption1.p2" class="link-target"></a><p>(C) To measure transcriptional transactivation capacity of the various proteins, HEK293T cells were transfected with a firefly luciferase reporter under control of the 8tetO promoter (similar to the promoter in the HIV-rtTA constructs), renilla expression vector pRL-CMV, an expression plasmid for rtTA to activate the promoter in the presence of dox, and the indicated amounts of RSSs. Luciferase expression (plotted as firefly/renilla) was determined 2 d post transfection. −, pBluescript (negative control).</p>
<a id="article1.body1.sec2.sec5.fig1.caption1.p3" name="article1.body1.sec2.sec5.fig1.caption1.p3" class="link-target"></a><p>(D) Transcriptional transactivation capacity of the various RSS proteins was measured using TZM-bl cells that contain a Tat-responsive firefly luciferase reporter gene under control of the HIV-1-LTR. The cells transfected with 0.2 μg of Tat or the indicated RSS expression plasmids. Two to three days after transfection, luciferase expression was measured. −, pBluescript (negative control).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g005">
              https://doi.org/10.1371/journal.ppat.0030086.g005</a></p></div><a id="article1.body1.sec2.sec5.p2" name="article1.body1.sec2.sec5.p2" class="link-target"></a><p>To confirm that the observed rescue of virus production corresponds with an increase in viral mRNAs, we analysed viral RNA accumulation by Northern blot analyses. First, rescue of HIV-rtTA-Tat<sup>fs</sup> virus production by Tat<sup>wt</sup> and VP35 was determined by measuring CA-p24 production in the supernatant (<a href="#ppat-0030086-g006">Figure 6</a>A). The amount of full-length genomic RNA of HIV-rtTA-Tat<sup>fs</sup> accumulating in transfected cells was reduced when compared to that of HIV-rtTA-Tat<sup>wt</sup> (<a href="#ppat-0030086-g006">Figure 6</a>B). In accordance with the CA-p24 values, co-transfection of Tat and VP35 restored the amount of full-length HIV-1 RNA.</p>
<a class="link-target" id="ppat-0030086-g006" name="ppat-0030086-g006"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g006"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g006" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g006"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g006" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g006">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g006">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g006">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g006">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g006">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g006">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 6. </span> Analyses of HIV-rtTA-Tat<sup>fs</sup> Genomic RNA Accumulation Complemented with Tat and VP35</div><p class="caption_target"><a id="article1.body1.sec2.sec5.fig2.caption1.p1" name="article1.body1.sec2.sec5.fig2.caption1.p1" class="link-target"></a><p>(A) HEK293T cells were transfected with HIV-rtTA-Tat<sup>wt</sup>, HIV-rtTA-Tat<sup>fs</sup>, and HIV-rtTA-Tat<sup>fs</sup> in combination with Tat<sup>wt</sup> or increasing amounts of VP35. Two to three days after transfection, virus production was measured via CA-p24 ELISA, and total RNA was isolated.</p>
<a id="article1.body1.sec2.sec5.fig2.caption1.p2" name="article1.body1.sec2.sec5.fig2.caption1.p2" class="link-target"></a><p>(B) Total RNA (10 μg) was run on a denaturing agarose gel, and after blotting, probed with a Nef-specific radiolabeled probe. Relative genomic RNA accumulation was quantified by phosphorimaging and is indicated below. The RNA accumulation of the HIV-rtTA-Tat<sup>wt</sup> was set at 100%.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g006">
              https://doi.org/10.1371/journal.ppat.0030086.g006</a></p></div></div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="VP35 dsRNA-Binding Capacity Is Required for trans Complementation of HIV-rtTA-Tatfs"></a>
<h3>VP35 dsRNA-Binding Capacity Is Required for <em>trans</em> Complementation of HIV-rtTA-Tat<sup>fs</sup></h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>Next, we wanted to determine whether the VP35 dsRNA-binding capacity is required for the complementation of HIV-rtTA-Tat<sup>fs</sup>. HEK293T cells were transfected with either HIV-rtTA-Tat<sup>wt</sup> or HIV-rtTA-Tat<sup>fs</sup>, and expression plasmids for Tat<sup>wt</sup>, VP35<sup>wt</sup>, and the VP35 mutants K309A, R312A, K309A/R312A, and R300T (<a href="#ppat-0030086-g007">Figure 7</a>). Tat<sup>wt</sup>, VP35<sup>wt</sup>, and K309A were able to rescue HIV-rtTA-Tat<sup>fs</sup> production, and co-transfection of mutant R312A partially restored HIV-rtTA-Tat<sup>fs</sup> virus production. In contrast, mutants K309/R312A and R300T were unable to rescue virus production. These results correlate with the ability of the various VP35 mutants to suppress RNAi in the luciferase assay. Thus, the data confirm the importance of RNAi suppression in HIV-1 production and that the dsRNA-binding capacity of VP35 is essential for this activity.</p>
<a class="link-target" id="ppat-0030086-g007" name="ppat-0030086-g007"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g007"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g007" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g007"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g007" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g007">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g007">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g007">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g007">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g007">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g007">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 7. </span> VP35 dsRNA-Binding Capacity Is Required for <em>trans</em> Complementation of HIV-rtTA-Tat<sup>fs</sup></div><p class="caption_target"><a id="article1.body1.sec2.sec6.fig1.caption1.p1" name="article1.body1.sec2.sec6.fig1.caption1.p1" class="link-target"></a><p>HEK293T cells were transfected with HIV-rtTA-Tat<sup>wt</sup>, HIV-rtTA-Tat<sup>fs</sup>, and expression plasmids for Tat<sup>wt</sup>, VP35<sup>wt</sup>, or the various VP35 mutants (K309A, R312A, and K309/R12A), or a C-terminal deletion mutant R300T (50 ng). Virus production was determined 2 d post transfection.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g007">
              https://doi.org/10.1371/journal.ppat.0030086.g007</a></p></div></div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1" class="link-target"></a><p>Although the importance of antiviral RNAi responses in plants, nematodes, and insects is firmly established, it is still under debate whether RNAi has a similar function in mammalian cells. If RNAi is a protective antiviral response in mammals, then virus infection should trigger the production of virus-specific siRNAs; inhibition of RNAi should affect virus replication, and viruses should have evolved factors that suppress RNAi [<a href="#ppat-0030086-b013" class="ref-tip">13</a>]. To date, there are only three examples of virus-specific siRNAs accumulating in infected mammalian cells. The human retrotransposon LINE-1 was shown to be inhibited by transposon-specific siRNAs, which is similar to what was previously described for transposon silencing in <span class="genus-species">Caenorhabditis elegans</span> [<a href="#ppat-0030086-b010" class="ref-tip">10</a>,<a href="#ppat-0030086-b012" class="ref-tip">12</a>,<a href="#ppat-0030086-b039" class="ref-tip">39</a>]. Furthermore, there is data suggesting accumulation of virus-specific siRNAs during HIV-1 replication [<a href="#ppat-0030086-b011" class="ref-tip">11</a>]. However, virus-specific siRNA accumulation in mammalian cells appears to be relatively low compared to that in plant and insect cells. The reason for this has remained unclear. One explanation could be the lack of host RNA-dependent RNA polymerase (RdRp) activity in mammalian cells. In plants and insects, RdRp activity is responsible for amplification of the RNAi signal. In combination with viral RSS activity, the absence of RdRp activity could explain the low siRNA levels in mammalian cells in comparison to those in plant, insect, or nematode cells. Another possibility is that antiviral RNAi in mammalian cells is initiated by cellular microRNAs rather than by de novo production of siRNAs [<a href="#ppat-0030086-b015" class="ref-tip">15</a>]. This has been suggested for the primate retrovirus PFV-1, where the cellular miR-32 was found to target PFV-1 sequences. PFV-1 overcomes microRNA-mediated antiviral pressure by the expressing the RSS Tas protein. In principle, viruses could also escape from antiviral microRNA pressure by acquisition of one or a few point mutations within the target sequence [<a href="#ppat-0030086-b040" class="ref-tip">40</a>,<a href="#ppat-0030086-b041" class="ref-tip">41</a>]. It is therefore likely that there is a more general RNAi response against PFV infection that necessitates the presence of an RSS.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2" class="link-target"></a><p>Little is known about enhanced virus replication in cells with a defective RNAi mechanism. However, recently it has been reported that HIV-1 replication is increased in human cells in which Dicer and Drosha expression is inhibited [<a href="#ppat-0030086-b042" class="ref-tip">42</a>]. This suggests that RNAi indeed plays a central role in anti-HIV-1 defence responses in human cells. Another report describes enhanced accumulation of the mammalian vesicular stomatitis virus in <span class="genus-species">C. elegans</span> with a defective RNAi machinery, which confirms that RNAi has an antiviral activity in nematodes [<a href="#ppat-0030086-b004" class="ref-tip">4</a>]. The other indication for RNAi-mediated antiviral activity in mammals is the fact that a number of mammalian viruses encode potent RSSs. However, it is still unclear to what extent these suppressors contribute to virus replication, because these factors have multiple functions that are difficult to separate. Moreover, the relevance of RSS activity measured in reporter assays has been questioned because non-specific binding of siRNAs by overexpression by dsRNA-binding proteins might also result in RNAi suppression [<a href="#ppat-0030086-b013" class="ref-tip">13</a>]. Therefore, it is essential to test the importance of RSS activity in a viral context instead of in reporter assays. Using the HIV-rtTA virus as a tool to investigate RNAi suppression, we were able to separate the transactivation function of HIV-1 Tat from its RSS function. We showed that Tat-mediated RSS activity is essential for HIV-1 production without possible side effects of Tat-mediated transcriptional transactivation. These data are in agreement with data published by Bennasser et al. in 2005 [<a href="#ppat-0030086-b011" class="ref-tip">11</a>]. The HIV-rtTA virus was used as a tool to identify EBOV VP35 protein as an RSS that can complement this Tat function. Both HIV-1 Tat and EBOV VP35 were found to complement HIV-rtTA-Tat<sup>fs</sup> production efficiently at low concentrations, indicating specific RSS activity. Moreover, RSSs encoded by other mammalian viruses are also able to complement the Tat RSS activity. The results presented here support the importance of RNAi in innate antiviral defence responses in mammalian cells and the essential function of RSSs in mammalian virus replication.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3" class="link-target"></a><p>In plants, insects, and nematodes, antiviral RNAi responses are triggered by virus-specific dsRNAs. In mammalian cells, virus-specific dsRNAs induce the IFN pathway via members of the Toll-like receptor family, or via a replication-dependent pathway involving the cytoplasmic dsRNA sensors RIG-I/MDA5 (retinoic-acid-inducible protein I/melanoma-differentiation-associated gene 5) [<a href="#ppat-0030086-b043" class="ref-tip">43</a>,<a href="#ppat-0030086-b044" class="ref-tip">44</a>]. Other antiviral proteins that are induced by dsRNA include the 2′-5′ oligoadenylate cyclase (2′-5′ OAS)/RNAseL and PKR [<a href="#ppat-0030086-b045" class="ref-tip">45</a>,<a href="#ppat-0030086-b046" class="ref-tip">46</a>]. Since RNAi, IFN responses, and 2′-5′ OAS/RNAseL/PKR are triggered by dsRNA, it is likely that these pathways cooperate in the innate antiviral defence response. The helicases RIG-I/MDA5 are candidate proteins that could link antiviral RNAi and IFN responses because they can be activated by siRNAs [<a href="#ppat-0030086-b044" class="ref-tip">44</a>,<a href="#ppat-0030086-b047" class="ref-tip">47</a>].</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4" class="link-target"></a><p>In agreement with this model is the recent observation that EBOV VP35 inhibits RIG-I induced activation of type I IFN responses [<a href="#ppat-0030086-b031" class="ref-tip">31</a>]. In part, this inhibition is dependent on dsRNA-binding capacity of VP35 [<a href="#ppat-0030086-b031" class="ref-tip">31</a>]. The K309A mutant was shown to have a partial defect in IFN antagonistic activity, whereas the other mutants, R312A, K309A/R312A, and R300T, were severely impaired in blocking IFN responses. Similarly, our assays show that mutation of the dsRNA-binding domain results in a loss of RNAi suppression activity. Only mutant K309A showed partial RSS activity and was able to complement HIV-rtTA-Tat<sup>fs</sup> production. These data suggest that VP35-mediated RNAi suppression and VP35-dsRNA-dependent IFN antagonistic properties are linked. We therefore propose that VP35 is able to sequester virus-specific siRNAs or dsRNA precursors of siRNA, resulting in suppression of an antiviral RNAi response that acts upstream of the 2′-5′ OAS/RNAseL/PKR and IFN pathways. In this scenario, the amount of cytoplasmic virus–specific siRNAs needs to reach a certain threshold level before RIG-I/MDA5 and the IFN pathway are activated (<a href="#ppat-0030086-g008">Figure 8</a>). In this way, virus-specific siRNAs would function as signal molecules for activation of the IFN response.</p>
<a class="link-target" id="ppat-0030086-g008" name="ppat-0030086-g008"></a><div class="figure" data-doi="10.1371/journal.ppat.0030086.g008"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.ppat.0030086.g008" data-doi="info:doi/10.1371/journal.ppat.0030086" data-uri="info:doi/10.1371/journal.ppat.0030086.g008"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.ppat.0030086.g008" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g008">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.ppat.0030086.g008">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g008">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.ppat.0030086.g008">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g008">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.ppat.0030086.g008">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 8. </span> Antiviral RNAi Activity and IFN Responses Function together in Innate Antiviral Defences</div><p class="caption_target"><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1" class="link-target"></a><p>During virus replication, virus-specific dsRNA molecules are generated that are recognized by Dicer and processed into siRNAs. One strand of the siRNA, the guide-strand, is loaded into RISC, which targets viral RNAs for destruction. To escape this antiviral pressure, viruses encode RSS factors that block the RNAi pathway. When virus-specific dsRNA molecules and siRNAs accumulate above a certain threshold, and consequently can no longer be masked by the viral RSS, the cytoplasmatic dsRNA sensors RIG-I/MDA5, PKR, and 2′5′ OAS/RNAseL are activated. This results in the activation of range antiviral responses, including the production of type I IFNs, general translational inhibition, and RNA degradation. This model clarifies why RSSs like NS1, VP35, and E3L have been identified as IFN antagonists.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.ppat.0030086.g008">
              https://doi.org/10.1371/journal.ppat.0030086.g008</a></p></div><a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5" class="link-target"></a><p>Virus-mediated RNAi suppression therefore has two functions. First, it suppresses the RNAi response that acts as the first line antiviral defence. Second, the inhibition of siRNA accumulation prevents RIG-I/MDA5-mediated activation of the IFN pathway. Important in this respect are the immature plasmacytoid dendritic cells (PDCs), which play a predominant role in antiviral immunity. If the accumulation of virus-specific siRNAs in these cells is blocked by viral RSS activity, there will be no trigger to induce the secretion of class I IFNs that in turn induce a range of antiviral genes, resulting in the antiviral state of the neighbouring cells, dendritic cell maturation, and subsequent adaptive immunity [<a href="#ppat-0030086-b048" class="ref-tip">48</a>]. This results in a delayed onset of the immune responses, and neighbouring cells will remain permissive to the invading virus for a prolonged time [<a href="#ppat-0030086-b049" class="ref-tip">49</a>].</p>
<a id="article1.body1.sec3.p6" name="article1.body1.sec3.p6" class="link-target"></a><p>Evasion of antiviral immune responses is a key process during virus replication. The results presented here suggest that RNAi plays an important role in innate antiviral defences and that HIV-1 needs to counter this mechanism in order to replicate. The fact that many RSS proteins also have IFN antagonistic properties [<a href="#ppat-0030086-b050" class="ref-tip">50</a>–<a href="#ppat-0030086-b056" class="ref-tip">56</a>] supports the idea that RNAi and IFN responses work together against invading viruses. It remains to be determined how the RNAi and IFN responses cooperate in mechanistic terms. In addition, future experiments should reveal the precise viral signature that activates the antiviral RNAi response in mammals.</p>
<a id="article1.body1.sec3.p7" name="article1.body1.sec3.p7" class="link-target"></a><p>Although viral RSS factors will be able to counter RNAi during natural infections, these factors will likely not reduce the effectiveness of RNAi-based antiviral therapeutics. High concentrations of exogenous synthetic siRNAs, for example, to block EBOV replication will easily saturate the VP35 RSS activity, rendering VP35 ineffective, and subsequently still inhibit virus replication [<a href="#ppat-0030086-b057" class="ref-tip">57</a>]. Similarly, plant viruses that encode RSS factors can also be silenced by RNAi, as can viruses such as HIV-1, influenza, hepatitis C virus, and Flock house virus that encode RSS factors.</p>
<a id="article1.body1.sec3.p8" name="article1.body1.sec3.p8" class="link-target"></a><p>Besides revealing novel aspects of the virus–host interaction, the <em>in trans</em> complementation of viruses lacking their RSS by heterologous RSSs creates opportunities for improving production of virus particles in mammalian cells. Moreover, this phenomenon could be used to develop a new generation of live attenuated viral vaccines that improves upon the current antiviral measures.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s4a" name="s4a" class="link-target" title="Construction of expression plasmids."></a>
<h3>Construction of expression plasmids.</h3>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1" class="link-target"></a><p>The NS1 (from influenza A virus strain PR8 [<a href="#ppat-0030086-b020" class="ref-tip">20</a>]), VP35 (from Ebola virus strain Zaire), E3L (from vaccinia virus strain Ankara), and EGFP open reading frames were cloned into the mammalian expression vector pEF5-V5-DEST containing human EF1α promoter using GATEWAY technology (Invitrogen, <a href="http://www.invitrogen.com">http://www.invitrogen.com</a>). The constructs for VP35<sup>wt</sup> and the various VP35 mutants contain the CMV promoter (pcDNA3.1-VP35<sup>wt</sup>, K309A, R312A, K309A/R312A, and R300T) and were a kind gift from S. T. Nichol [<a href="#ppat-0030086-b030" class="ref-tip">30</a>]. The HIV-1 Tat expression plasmids, pcDNA3-wtTat/pKV-wtTat, and the mutants, pcDNA3-Y26A and pKV-F32A, were described previously [<a href="#ppat-0030086-b038" class="ref-tip">38</a>]. HIV-rtTA-Tat<sup>fs</sup> was created by deletion of A58 in the Tat open reading frame of HIV-rtTA-Tat<sup>Y26A</sup> (variant KYK in [<a href="#ppat-0030086-b037" class="ref-tip">37</a>]), thus creating a frame shift at codon 20.</p>
</div>

<div id="section2" class="section toc-section"><a id="s4b" name="s4b" class="link-target" title="Cell culture and transfections."></a>
<h3>Cell culture and transfections.</h3>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1" class="link-target"></a><p>Human embryonic kidney (HEK293T) cells and African green monkey kidney Vero cells were grown as a monolayer in DMEM (Gibco BRL, <a href="http://www.invitrogen.com">http://www.invitrogen.com</a>) supplemented with 10% fetal calf serum (FCS) (Hyclone, <a href="http://www.hyclone.com">http://www.hyclone.com</a>), minimal essential medium with non-essential amino acids, penicillin (100 U/ml), and streptomycin (100 μg/ml) at 37 °C and 5% CO<sub>2</sub>. One day before transfection, cells were trypsinized, resuspended in DMEM, and seeded in 24-well plates at a density of 1.5 × 10<sup>5</sup> cells per well. At the time of transfection, the cells were 60%–70% confluent. The transfection was performed in duplicates using Lipofectamine 2000 (Invitrogen) according to the instructions of the manufacturer. For the luciferase RNAi assay, cells were transfected with 100 ng of luciferase-expressing plasmid pGL3 (Promega, <a href="http://www.promega.com">http://www.promega.com</a>) and 10 ng of expression plasmid shLuc (pShh1-Ff1), from which an shRNA against luciferase is expressed under control of the U6 promoter [<a href="#ppat-0030086-b027" class="ref-tip">27</a>]. Cells were lysed 2–3 d post transfection in 150 μl of 1x Passive Lysis Buffer (Promega) by shaking for 30 min at room temperature. The cell lysate was centrifuged for 5 min at 1,500 rpm, and luciferase expression was measured in 10 μl of supernatant with the luciferase reporter assay system (Promega).</p>
<a id="article1.body1.sec4.sec2.p2" name="article1.body1.sec4.sec2.p2" class="link-target"></a><p>For the complementation studies, 100 ng of HIV-rtTA-Tat<sup>fs</sup> was transfected with the indicated amounts of RSS plasmid. The total amount of DNA was brought to 1 μg using pBluescript (Stratagene, <a href="http://www.stratagene.com">http://www.stratagene.com</a>). Two to three days after transfection, virus production was determined by measuring CA-p24 levels by enzyme-linked immunosorbent assay (ELISA).</p>
<a id="article1.body1.sec4.sec2.p3" name="article1.body1.sec4.sec2.p3" class="link-target"></a><p>For the testing of the effect of Tat and the various RSS proteins on the 8tetO promoter, we transfected HEK293T cells with 20 ng of 8tetO-luc expression plasmid containing a luciferase reporter under control of the 8tetO promoter (the same promoter as in HIV-rtTA-Tat<sup>fs</sup>), 0.4 ng of rtTA expression plasmid, 0.5 ng of pRL-CMV, and the different RSS vectors. Two to three days after transfection, luciferase expression was measured.</p>
</div>

<div id="section3" class="section toc-section"><a id="s4c" name="s4c" class="link-target" title="Transactivation assay."></a>
<h3>Transactivation assay.</h3>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1" class="link-target"></a><p>Transcriptional transactivation capacity of Tat, Tat mutants, and the various RSS proteins was measured using TZM-bl cells. These cells were obtained from the NIH AIDS Research and Reference Reagent Program (also termed JC53-BL cells; catalog number 8129; <a href="https://www.aidsreagent.org">https://www.aidsreagent.org</a>). TZM-bl cells are genetically modified Hela cells that express CD4, CXCR4, and CCR5 and contain the firefly reporter gene under control of the HIV-1 LTR, which is Tat responsive. The cells were seeded similarly to the HEK293T cells and transfected with 0.2 μg of Tat or RSS expression plasmid, and luciferase expression was measured 2–3 d after transfection.</p>
</div>

<div id="section4" class="section toc-section"><a id="s4d" name="s4d" class="link-target" title="Northern blot analyses."></a>
<h3>Northern blot analyses.</h3>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1" class="link-target"></a><p>HEK293T cells (T25 flask) were transfected with 6.6 μg of HIV-rtTA-Tat<sup>wt</sup>, HIV-rtTA-Tat<sup>fs</sup>, and HIV-rtTA-Tat<sup>fs</sup> in combination with 0.132 μg of Tat<sup>wt</sup> or 0.132/0.66/1.32/6.6 μg of VP35 expression plasmid. Dox was added to the medium to activate HIV-rtTA virus production. Two to three days post transfection, virus production was measured via CA-p24 ELISA, and total RNA was isolated using the mirVana RNA isolation kit (Ambion, <a href="http://www.ambion.com">http://www.ambion.com</a>). For detection of genomic HIV-1 RNAs, gel electrophoresis of 10 μg of total RNA was performed on a denaturing formaldehyde, 1% agarose gel. RNA was transferred to a positively charged nylon membrane (Boehringer, <a href="http://www.boehringer-ingelheim.com">http://www.boehringer-ingelheim.com</a>) via capillary blotting and crosslinked to the membrane with a UV crosslinker (Stratagene). A 19-nt LNA-modified oligonucleotide complementary to the HIV-1 Nef gene was used as a probe, which was 5′ end labeled using the kinaseMax kit (Ambion) in the presence of 1 μl of [γ-<sup>32</sup>P]ATP (0.37 MBq/μl; Amersham Biosciences, <a href="http://www.gelifesciences.com">http://www.gelifesciences.com</a>) and purified over a MicroSpin G-25 column (Amersham Biosciences). Prehybridization and hybridization was performed in Ultrahyb buffer (Ambion) at 60 °C for 30 min and 18 h, respectively. The membrane was washed twice for 15 min at 60 °C with high-stringency buffer (0.2x SSC, 0.2% SDS). Images were obtained using the Typhoon Trio phosphor imager (Amersham Biosciences).</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2>
<div id="section1" class="section toc-section"><a id="s5a" name="s5a" class="link-target" title="Accession Numbers"></a>
<h3>Accession Numbers</h3>
<a id="article1.body1.sec5.sec1.p1" name="article1.body1.sec5.sec1.p1" class="link-target"></a><p>The GenBank (<a href="http://www.ncbi.nlm.nih.gov/Genbank/index.html">http://www.ncbi.nlm.nih.gov/Genbank/index.html</a>) accession numbers for the proteins discussed in this article are E3L (from vaccinia virus) (U94848), NS1 (from influenza A virus) (EF467817), and VP35 (from Ebola virus) (AF086833).</p>
</div>
</div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>H. D. Klenk, Institut für Virology, Phillips Universität, Marburg, Germany, is thanked for donating the VP35 clone. S. T. Nichol, Centers for Disease Control and Prevention, Atlanta, United States of America, is thanked for donating the VP35<sup>wt</sup> and VP35 mutant expression plasmids. Stephan Heynen is thanked for CA-p24 ELISA.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>JH, WdV, PdH, and BB conceived and designed the experiments. JH, WdV, and EJG performed the experiments. JH and WdV analyzed the data. JH, WdV, MP, PdH, and BB contributed reagents/materials/analysis tools. JH, WdV, PdH, and BB contributed to writing the paper.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="ppat-0030086-b001" id="ppat-0030086-b001" class="link-target"></a>Galiana-Arnoux D, Dostert C, Schneemann A, Hoffmann JA, Imler JL (2006) Essential function in vivo for Dicer-2 in host defense against RNA viruses in <em>Drosophila</em>. Nat Immunol  7: 590–597.  <ul class="reflinks"><li><a href="#" data-author="Galiana-Arnoux" data-cit="Galiana-ArnouxDDostertCSchneemannAHoffmannJAImlerJL2006Essential%20function%20in%20vivo%20for%20Dicer-2%20in%20host%20defense%20against%20RNA%20viruses%20in%20Drosophila.Nat%20Immunol7590597" data-title="Essential%20function%20in%20vivo%20for%20Dicer-2%20in%20host%20defense%20against%20RNA%20viruses%20in%20Drosophila." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Essential+function+in+vivo+for+Dicer-2+in+host+defense+against+RNA+viruses+in+Drosophila.+Galiana-Arnoux+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="ppat-0030086-b002" id="ppat-0030086-b002" class="link-target"></a>Wang XH, Aliyari R, Li WX, Li HW, Kim K, et al.  (2006) RNA interference directs innate immunity against viruses in adult <em>Drosophila</em>. Science  312: 452–454.  <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangXHAliyariRLiWXLiHWKimK2006RNA%20interference%20directs%20innate%20immunity%20against%20viruses%20in%20adult%20Drosophila.Science312452454" data-title="RNA%20interference%20directs%20innate%20immunity%20against%20viruses%20in%20adult%20Drosophila." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=RNA+interference+directs+innate+immunity+against+viruses+in+adult+Drosophila.+Wang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="ppat-0030086-b003" id="ppat-0030086-b003" class="link-target"></a>Schott DH, Cureton DK, Whelan SP, Hunter CP (2005) An antiviral role for the RNA interference machinery in <span class="genus-species">Caenorhabditis elegans</span>. Proc Natl Acad Sci U S A  102: 18420–18424.  <ul class="reflinks"><li><a href="#" data-author="Schott" data-cit="SchottDHCuretonDKWhelanSPHunterCP2005An%20antiviral%20role%20for%20the%20RNA%20interference%20machinery%20in%20Caenorhabditis%20elegans.Proc%20Natl%20Acad%20Sci%20U%20S%20A1021842018424" data-title="An%20antiviral%20role%20for%20the%20RNA%20interference%20machinery%20in%20Caenorhabditis%20elegans." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=An+antiviral+role+for+the+RNA+interference+machinery+in+Caenorhabditis+elegans.+Schott+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="ppat-0030086-b004" id="ppat-0030086-b004" class="link-target"></a>Wilkins C, Dishongh R, Moore SC, Whitt MA, Chow M, et al.  (2005) RNA interference is an antiviral defence mechanism in <span class="genus-species">Caenorhabditis elegans</span>. Nature  436: 1044–1047.  <ul class="reflinks"><li><a href="#" data-author="Wilkins" data-cit="WilkinsCDishonghRMooreSCWhittMAChowM2005RNA%20interference%20is%20an%20antiviral%20defence%20mechanism%20in%20Caenorhabditis%20elegans.Nature43610441047" data-title="RNA%20interference%20is%20an%20antiviral%20defence%20mechanism%20in%20Caenorhabditis%20elegans." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=RNA+interference+is+an+antiviral+defence+mechanism+in+Caenorhabditis+elegans.+Wilkins+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="ppat-0030086-b005" id="ppat-0030086-b005" class="link-target"></a>Voinnet O (2005) Induction and suppression of RNA silencing: Insights from viral infections. Nat Rev Genet  6: 206–220.  <ul class="reflinks"><li><a href="#" data-author="Voinnet" data-cit="VoinnetO2005Induction%20and%20suppression%20of%20RNA%20silencing%3A%20Insights%20from%20viral%20infections.Nat%20Rev%20Genet6206220" data-title="Induction%20and%20suppression%20of%20RNA%20silencing%3A%20Insights%20from%20viral%20infections." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Induction+and+suppression+of+RNA+silencing%3A+Insights+from+viral+infections.+Voinnet+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="ppat-0030086-b006" id="ppat-0030086-b006" class="link-target"></a>Scholthof HB (2006) The tombusvirus-encoded P19: From irrelevance to elegance. Nat Rev Microbiol  4: 405–411.  <ul class="reflinks"><li><a href="#" data-author="Scholthof" data-cit="ScholthofHB2006The%20tombusvirus-encoded%20P19%3A%20From%20irrelevance%20to%20elegance.Nat%20Rev%20Microbiol4405411" data-title="The%20tombusvirus-encoded%20P19%3A%20From%20irrelevance%20to%20elegance." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+tombusvirus-encoded+P19%3A+From+irrelevance+to+elegance.+Scholthof+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="ppat-0030086-b007" id="ppat-0030086-b007" class="link-target"></a>Deleris A, Gallego-Bartolome J, Bao J, Kasschau KD, Carrington JC, et al.  (2006) Hierarchical action and inhibition of plant Dicer-like proteins in antiviral defense. Science  313: 68–71.  <ul class="reflinks"><li><a href="#" data-author="Deleris" data-cit="DelerisAGallego-BartolomeJBaoJKasschauKDCarringtonJC2006Hierarchical%20action%20and%20inhibition%20of%20plant%20Dicer-like%20proteins%20in%20antiviral%20defense.Science3136871" data-title="Hierarchical%20action%20and%20inhibition%20of%20plant%20Dicer-like%20proteins%20in%20antiviral%20defense." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hierarchical+action+and+inhibition+of+plant+Dicer-like+proteins+in+antiviral+defense.+Deleris+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="ppat-0030086-b008" id="ppat-0030086-b008" class="link-target"></a>Haasnoot PCJ, Berkhout B (2006) RNA interference: Its use as antiviral therapy. Handbook of experimental pharmacology. Heidelberg: Springer-Verlag. pp. 117–150. <ul class="find-nolinks"></ul></li><li id="ref9"><span class="order">9.
            </span><a name="ppat-0030086-b009" id="ppat-0030086-b009" class="link-target"></a>Pfeffer S, Zavolan M, Grasser FA, Chien M, Russo JJ, et al.  (2004) Identification of virus-encoded microRNAs. Science  304: 734–736.  <ul class="reflinks"><li><a href="#" data-author="Pfeffer" data-cit="PfefferSZavolanMGrasserFAChienMRussoJJ2004Identification%20of%20virus-encoded%20microRNAs.Science304734736" data-title="Identification%20of%20virus-encoded%20microRNAs." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+virus-encoded+microRNAs.+Pfeffer+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="ppat-0030086-b010" id="ppat-0030086-b010" class="link-target"></a>Soifer HS, Zaragoza A, Peyvan M, Behlke MA, Rossi JJ (2005) A potential role for RNA interference in controlling the activity of the human LINE-1 retrotransposon. Nucleic Acids Res  33: 846–856.  <ul class="reflinks"><li><a href="#" data-author="Soifer" data-cit="SoiferHSZaragozaAPeyvanMBehlkeMARossiJJ2005A%20potential%20role%20for%20RNA%20interference%20in%20controlling%20the%20activity%20of%20the%20human%20LINE-1%20retrotransposon.Nucleic%20Acids%20Res33846856" data-title="A%20potential%20role%20for%20RNA%20interference%20in%20controlling%20the%20activity%20of%20the%20human%20LINE-1%20retrotransposon." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+potential+role+for+RNA+interference+in+controlling+the+activity+of+the+human+LINE-1+retrotransposon.+Soifer+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="ppat-0030086-b011" id="ppat-0030086-b011" class="link-target"></a>Bennasser Y, Le SY, Benkirane M, Jeang KT (2005) Evidence that HIV-1 encodes an siRNA and a suppressor of RNA silencing. Immunity  22: 607–619.  <ul class="reflinks"><li><a href="#" data-author="Bennasser" data-cit="BennasserYLeSYBenkiraneMJeangKT2005Evidence%20that%20HIV-1%20encodes%20an%20siRNA%20and%20a%20suppressor%20of%20RNA%20silencing.Immunity22607619" data-title="Evidence%20that%20HIV-1%20encodes%20an%20siRNA%20and%20a%20suppressor%20of%20RNA%20silencing." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Evidence+that+HIV-1+encodes+an+siRNA+and+a+suppressor+of+RNA+silencing.+Bennasser+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="ppat-0030086-b012" id="ppat-0030086-b012" class="link-target"></a>Yang N, Kazazian HH Jr (2006) L1 retrotransposition is suppressed by endogenously encoded small interfering RNAs in human cultured cells. Nat Struct Mol Biol  13: 763–771.  <ul class="reflinks"><li><a href="#" data-author="Yang" data-cit="YangNKazazianHHJr2006L1%20retrotransposition%20is%20suppressed%20by%20endogenously%20encoded%20small%20interfering%20RNAs%20in%20human%20cultured%20cells.Nat%20Struct%20Mol%20Biol13763771" data-title="L1%20retrotransposition%20is%20suppressed%20by%20endogenously%20encoded%20small%20interfering%20RNAs%20in%20human%20cultured%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=L1+retrotransposition+is+suppressed+by+endogenously+encoded+small+interfering+RNAs+in+human+cultured+cells.+Yang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="ppat-0030086-b013" id="ppat-0030086-b013" class="link-target"></a>Cullen BR (2006) Is RNA interference involved in intrinsic antiviral immunity in mammals? Nat Immunol  7: 563–567.  <ul class="reflinks"><li><a href="#" data-author="Cullen" data-cit="CullenBR2006Is%20RNA%20interference%20involved%20in%20intrinsic%20antiviral%20immunity%20in%20mammals%3FNat%20Immunol7563567" data-title="Is%20RNA%20interference%20involved%20in%20intrinsic%20antiviral%20immunity%20in%20mammals%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Is+RNA+interference+involved+in+intrinsic+antiviral+immunity+in+mammals%3F+Cullen+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="ppat-0030086-b014" id="ppat-0030086-b014" class="link-target"></a>Dangl JL, Jones JD (2001) Plant pathogens and integrated defence responses to infection. Nature  411: 826–833.  <ul class="reflinks"><li><a href="#" data-author="Dangl" data-cit="DanglJLJonesJD2001Plant%20pathogens%20and%20integrated%20defence%20responses%20to%20infection.Nature411826833" data-title="Plant%20pathogens%20and%20integrated%20defence%20responses%20to%20infection." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Plant+pathogens+and+integrated+defence+responses+to+infection.+Dangl+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="ppat-0030086-b015" id="ppat-0030086-b015" class="link-target"></a>Lecellier CH, Dunoyer P, Arar K, Lehmann-Che J, Eyquem S, et al.  (2005) A cellular microRNA mediates antiviral defense in human cells. Science  308: 557–560.  <ul class="reflinks"><li><a href="#" data-author="Lecellier" data-cit="LecellierCHDunoyerPArarKLehmann-CheJEyquemS2005A%20cellular%20microRNA%20mediates%20antiviral%20defense%20in%20human%20cells.Science308557560" data-title="A%20cellular%20microRNA%20mediates%20antiviral%20defense%20in%20human%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+cellular+microRNA+mediates+antiviral+defense+in+human+cells.+Lecellier+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="ppat-0030086-b016" id="ppat-0030086-b016" class="link-target"></a>Lu S, Cullen BR (2004) Adenovirus VA1 noncoding RNA can inhibit small interfering RNA and microRNA biogenesis. J Virol  78: 12868–12876.  <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuSCullenBR2004Adenovirus%20VA1%20noncoding%20RNA%20can%20inhibit%20small%20interfering%20RNA%20and%20microRNA%20biogenesis.J%20Virol781286812876" data-title="Adenovirus%20VA1%20noncoding%20RNA%20can%20inhibit%20small%20interfering%20RNA%20and%20microRNA%20biogenesis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Adenovirus+VA1+noncoding+RNA+can+inhibit+small+interfering+RNA+and+microRNA+biogenesis.+Lu+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="ppat-0030086-b017" id="ppat-0030086-b017" class="link-target"></a>Li WX, Li H, Lu R, Li F, Dus M, et al.  (2004) Interferon antagonist proteins of influenza and vaccinia viruses are suppressors of RNA silencing. Proc Natl Acad Sci U S A  101: 1350–1355.  <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiWXLiHLuRLiFDusM2004Interferon%20antagonist%20proteins%20of%20influenza%20and%20vaccinia%20viruses%20are%20suppressors%20of%20RNA%20silencing.Proc%20Natl%20Acad%20Sci%20U%20S%20A10113501355" data-title="Interferon%20antagonist%20proteins%20of%20influenza%20and%20vaccinia%20viruses%20are%20suppressors%20of%20RNA%20silencing." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Interferon+antagonist+proteins+of+influenza+and+vaccinia+viruses+are+suppressors+of+RNA+silencing.+Li+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="ppat-0030086-b018" id="ppat-0030086-b018" class="link-target"></a>Wang Y, Kato N, Jazag A, Dharel N, Otsuka M, et al.  (2006) Hepatitis C virus core protein is a potent inhibitor of RNA silencing-based antiviral response. Gastroenterology  130: 883–892.  <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangYKatoNJazagADharelNOtsukaM2006Hepatitis%20C%20virus%20core%20protein%20is%20a%20potent%20inhibitor%20of%20RNA%20silencing-based%20antiviral%20response.Gastroenterology130883892" data-title="Hepatitis%20C%20virus%20core%20protein%20is%20a%20potent%20inhibitor%20of%20RNA%20silencing-based%20antiviral%20response." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Hepatitis+C+virus+core+protein+is+a+potent+inhibitor+of+RNA+silencing-based+antiviral+response.+Wang+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="ppat-0030086-b019" id="ppat-0030086-b019" class="link-target"></a>Andersson MG, Haasnoot PCJ, Xu N, Berenjian S, Berkhout B, et al.  (2005) Suppression of RNA interference by adenovirus virus-associated RNA. J Virol  79: 9556–9565.  <ul class="reflinks"><li><a href="#" data-author="Andersson" data-cit="AnderssonMGHaasnootPCJXuNBerenjianSBerkhoutB2005Suppression%20of%20RNA%20interference%20by%20adenovirus%20virus-associated%20RNA.J%20Virol7995569565" data-title="Suppression%20of%20RNA%20interference%20by%20adenovirus%20virus-associated%20RNA." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Suppression+of+RNA+interference+by+adenovirus+virus-associated+RNA.+Andersson+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="ppat-0030086-b020" id="ppat-0030086-b020" class="link-target"></a>Bucher E, Hemmes H, de Haan P, Goldbach R, Prins M (2004) The influenza A virus NS1 protein binds small interfering RNAs and suppresses RNA silencing in plants. J Gen Virol  85: 983–991.  <ul class="reflinks"><li><a href="#" data-author="Bucher" data-cit="BucherEHemmesHde%20HaanPGoldbachRPrinsM2004The%20influenza%20A%20virus%20NS1%20protein%20binds%20small%20interfering%20RNAs%20and%20suppresses%20RNA%20silencing%20in%20plants.J%20Gen%20Virol85983991" data-title="The%20influenza%20A%20virus%20NS1%20protein%20binds%20small%20interfering%20RNAs%20and%20suppresses%20RNA%20silencing%20in%20plants." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+influenza+A+virus+NS1+protein+binds+small+interfering+RNAs+and+suppresses+RNA+silencing+in+plants.+Bucher+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="ppat-0030086-b021" id="ppat-0030086-b021" class="link-target"></a>Basler CF, Wang X, Muhlberger E, Volchkov V, Paragas J, et al.  (2000) The Ebola virus VP35 protein functions as a type I IFN antagonist. Proc Natl Acad Sci U S A  97: 12289–12294.  <ul class="reflinks"><li><a href="#" data-author="Basler" data-cit="BaslerCFWangXMuhlbergerEVolchkovVParagasJ2000The%20Ebola%20virus%20VP35%20protein%20functions%20as%20a%20type%20I%20IFN%20antagonist.Proc%20Natl%20Acad%20Sci%20U%20S%20A971228912294" data-title="The%20Ebola%20virus%20VP35%20protein%20functions%20as%20a%20type%20I%20IFN%20antagonist." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Ebola+virus+VP35+protein+functions+as+a+type+I+IFN+antagonist.+Basler+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="ppat-0030086-b022" id="ppat-0030086-b022" class="link-target"></a>Garcia-Sastre A, Egorov A, Matassov D, Brandt S, Levy DE, et al.  (1998) Influenza A virus lacking the NS1 gene replicates in interferon-deficient systems. Virol  252: 324–330.  <ul class="reflinks"><li><a href="#" data-author="Garcia-Sastre" data-cit="Garcia-SastreAEgorovAMatassovDBrandtSLevyDE1998Influenza%20A%20virus%20lacking%20the%20NS1%20gene%20replicates%20in%20interferon-deficient%20systems.Virol252324330" data-title="Influenza%20A%20virus%20lacking%20the%20NS1%20gene%20replicates%20in%20interferon-deficient%20systems." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Influenza+A+virus+lacking+the+NS1+gene+replicates+in+interferon-deficient+systems.+Garcia-Sastre+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="ppat-0030086-b023" id="ppat-0030086-b023" class="link-target"></a>Kitajewski J, Schneider RJ, Safer B, Munemitsu SM, Samuel CE, et al.  (1986) Adenovirus VAI RNA antagonizes the antiviral action of interferon by preventing activation of the interferon-induced eIF-2 alpha kinase. Cell  45: 195–200.  <ul class="reflinks"><li><a href="#" data-author="Kitajewski" data-cit="KitajewskiJSchneiderRJSaferBMunemitsuSMSamuelCE1986Adenovirus%20VAI%20RNA%20antagonizes%20the%20antiviral%20action%20of%20interferon%20by%20preventing%20activation%20of%20the%20interferon-induced%20eIF-2%20alpha%20kinase.Cell45195200" data-title="Adenovirus%20VAI%20RNA%20antagonizes%20the%20antiviral%20action%20of%20interferon%20by%20preventing%20activation%20of%20the%20interferon-induced%20eIF-2%20alpha%20kinase." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Adenovirus+VAI+RNA+antagonizes+the+antiviral+action+of+interferon+by+preventing+activation+of+the+interferon-induced+eIF-2+alpha+kinase.+Kitajewski+1986" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="ppat-0030086-b024" id="ppat-0030086-b024" class="link-target"></a>Brand SR, Kobayashi R, Mathews MB (1997) The Tat protein of human immunodeficiency virus type 1 is a substrate and inhibitor of the interferon-induced, virally activated protein kinase, PKR. J Biol Chem  272: 8388–8395.  <ul class="reflinks"><li><a href="#" data-author="Brand" data-cit="BrandSRKobayashiRMathewsMB1997The%20Tat%20protein%20of%20human%20immunodeficiency%20virus%20type%201%20is%20a%20substrate%20and%20inhibitor%20of%20the%20interferon-induced%2C%20virally%20activated%20protein%20kinase%2C%20PKR.J%20Biol%20Chem27283888395" data-title="The%20Tat%20protein%20of%20human%20immunodeficiency%20virus%20type%201%20is%20a%20substrate%20and%20inhibitor%20of%20the%20interferon-induced%2C%20virally%20activated%20protein%20kinase%2C%20PKR." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Tat+protein+of+human+immunodeficiency+virus+type+1+is+a+substrate+and+inhibitor+of+the+interferon-induced%2C+virally+activated+protein+kinase%2C+PKR.+Brand+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="ppat-0030086-b025" id="ppat-0030086-b025" class="link-target"></a>Lichner Z, Silhavy D, Burgyan J (2003) Double-stranded RNA-binding proteins could suppress RNA interference-mediated antiviral defences. J Gen Virol  84: 975–980.  <ul class="reflinks"><li><a href="#" data-author="Lichner" data-cit="LichnerZSilhavyDBurgyanJ2003Double-stranded%20RNA-binding%20proteins%20could%20suppress%20RNA%20interference-mediated%20antiviral%20defences.J%20Gen%20Virol84975980" data-title="Double-stranded%20RNA-binding%20proteins%20could%20suppress%20RNA%20interference-mediated%20antiviral%20defences." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Double-stranded+RNA-binding+proteins+could+suppress+RNA+interference-mediated+antiviral+defences.+Lichner+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="ppat-0030086-b026" id="ppat-0030086-b026" class="link-target"></a>Enterlein S, Warfield KL, Swenson DL, Stein DA, Smith JL, et al.  (2006) VP35 knockdown inhibits Ebola virus amplification and protects against lethal infection in mice. Antimicrob Agents Chemother  50: 984–993.  <ul class="reflinks"><li><a href="#" data-author="Enterlein" data-cit="EnterleinSWarfieldKLSwensonDLSteinDASmithJL2006VP35%20knockdown%20inhibits%20Ebola%20virus%20amplification%20and%20protects%20against%20lethal%20infection%20in%20mice.Antimicrob%20Agents%20Chemother50984993" data-title="VP35%20knockdown%20inhibits%20Ebola%20virus%20amplification%20and%20protects%20against%20lethal%20infection%20in%20mice." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=VP35+knockdown+inhibits+Ebola+virus+amplification+and+protects+against+lethal+infection+in+mice.+Enterlein+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="ppat-0030086-b027" id="ppat-0030086-b027" class="link-target"></a>Paddison PJ, Caudy AA, Bernstein E, Hannon GJ, Conklin DS (2002) Short hairpin RNAs (shRNAs) induce sequence-specific silencing in mammalian cells. Genes Dev  16: 948–958.  <ul class="reflinks"><li><a href="#" data-author="Paddison" data-cit="PaddisonPJCaudyAABernsteinEHannonGJConklinDS2002Short%20hairpin%20RNAs%20%28shRNAs%29%20induce%20sequence-specific%20silencing%20in%20mammalian%20cells.Genes%20Dev16948958" data-title="Short%20hairpin%20RNAs%20%28shRNAs%29%20induce%20sequence-specific%20silencing%20in%20mammalian%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Short+hairpin+RNAs+%28shRNAs%29+induce+sequence-specific+silencing+in+mammalian+cells.+Paddison+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="ppat-0030086-b028" id="ppat-0030086-b028" class="link-target"></a>Diaz MO, Ziemin S, Le Beau MM, Pitha P, Smith SD, et al.  (1988) Homozygous deletion of the alpha- and beta 1-interferon genes in human leukemia and derived cell lines. Proc Natl Acad Sci U S A  85: 5259–5263.  <ul class="reflinks"><li><a href="#" data-author="Diaz" data-cit="DiazMOZieminSLe%20BeauMMPithaPSmithSD1988Homozygous%20deletion%20of%20the%20alpha-%20and%20beta%201-interferon%20genes%20in%20human%20leukemia%20and%20derived%20cell%20lines.Proc%20Natl%20Acad%20Sci%20U%20S%20A8552595263" data-title="Homozygous%20deletion%20of%20the%20alpha-%20and%20beta%201-interferon%20genes%20in%20human%20leukemia%20and%20derived%20cell%20lines." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Homozygous+deletion+of+the+alpha-+and+beta+1-interferon+genes+in+human+leukemia+and+derived+cell+lines.+Diaz+1988" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="ppat-0030086-b029" id="ppat-0030086-b029" class="link-target"></a>Delgadillo MO, Saenz P, Salvador B, Garcia JA, Simon-Mateo C (2004) Human influenza virus NS1 protein enhances viral pathogenicity and acts as an RNA silencing suppressor in plants. J Gen Virol  85: 993–999.  <ul class="reflinks"><li><a href="#" data-author="Delgadillo" data-cit="DelgadilloMOSaenzPSalvadorBGarciaJASimon-MateoC2004Human%20influenza%20virus%20NS1%20protein%20enhances%20viral%20pathogenicity%20and%20acts%20as%20an%20RNA%20silencing%20suppressor%20in%20plants.J%20Gen%20Virol85993999" data-title="Human%20influenza%20virus%20NS1%20protein%20enhances%20viral%20pathogenicity%20and%20acts%20as%20an%20RNA%20silencing%20suppressor%20in%20plants." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+influenza+virus+NS1+protein+enhances+viral+pathogenicity+and+acts+as+an+RNA+silencing+suppressor+in+plants.+Delgadillo+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="ppat-0030086-b030" id="ppat-0030086-b030" class="link-target"></a>Hartman AL, Towner JS, Nichol ST (2004) A C-terminal basic amino acid motif of Zaire ebolavirus VP35 is essential for type I interferon antagonism and displays high identity with the RNA-binding domain of another interferon antagonist, the NS1 protein of influenza A virus. Virol  328: 177–184.  <ul class="reflinks"><li><a href="#" data-author="Hartman" data-cit="HartmanALTownerJSNicholST2004A%20C-terminal%20basic%20amino%20acid%20motif%20of%20Zaire%20ebolavirus%20VP35%20is%20essential%20for%20type%20I%20interferon%20antagonism%20and%20displays%20high%20identity%20with%20the%20RNA-binding%20domain%20of%20another%20interferon%20antagonist%2C%20the%20NS1%20protein%20of%20influenza%20A%20virus.Virol328177184" data-title="A%20C-terminal%20basic%20amino%20acid%20motif%20of%20Zaire%20ebolavirus%20VP35%20is%20essential%20for%20type%20I%20interferon%20antagonism%20and%20displays%20high%20identity%20with%20the%20RNA-binding%20domain%20of%20another%20interferon%20antagonist%2C%20the%20NS1%20protein%20of%20influenza%20A%20virus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+C-terminal+basic+amino+acid+motif+of+Zaire+ebolavirus+VP35+is+essential+for+type+I+interferon+antagonism+and+displays+high+identity+with+the+RNA-binding+domain+of+another+interferon+antagonist%2C+the+NS1+protein+of+influenza+A+virus.+Hartman+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="ppat-0030086-b031" id="ppat-0030086-b031" class="link-target"></a>Cardenas WB, Loo YM, Gale M Jr, Hartman AL, Kimberlin CR, et al.  (2006) Ebola virus VP35 protein binds double-stranded RNA and inhibits alpha/beta interferon production induced by RIG-I signaling. J Virol  80: 5168–5178.  <ul class="reflinks"><li><a href="#" data-author="Cardenas" data-cit="CardenasWBLooYMGaleMJrHartmanALKimberlinCR2006Ebola%20virus%20VP35%20protein%20binds%20double-stranded%20RNA%20and%20inhibits%20alpha%2Fbeta%20interferon%20production%20induced%20by%20RIG-I%20signaling.J%20Virol8051685178" data-title="Ebola%20virus%20VP35%20protein%20binds%20double-stranded%20RNA%20and%20inhibits%20alpha%2Fbeta%20interferon%20production%20induced%20by%20RIG-I%20signaling." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Ebola+virus+VP35+protein+binds+double-stranded+RNA+and+inhibits+alpha%2Fbeta+interferon+production+induced+by+RIG-I+signaling.+Cardenas+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="ppat-0030086-b032" id="ppat-0030086-b032" class="link-target"></a>Hartman AL, Dover JE, Towner JS, Nichol ST (2006) Reverse genetic generation of recombinant Zaire Ebola viruses containing disrupted IRF-3 inhibitory domains results in attenuated virus growth in vitro and higher levels of IRF-3 activation without inhibiting viral transcription or replication. J Virol  80: 6430–6440.  <ul class="reflinks"><li><a href="#" data-author="Hartman" data-cit="HartmanALDoverJETownerJSNicholST2006Reverse%20genetic%20generation%20of%20recombinant%20Zaire%20Ebola%20viruses%20containing%20disrupted%20IRF-3%20inhibitory%20domains%20results%20in%20attenuated%20virus%20growth%20in%20vitro%20and%20higher%20levels%20of%20IRF-3%20activation%20without%20inhibiting%20viral%20transcription%20or%20replication.J%20Virol8064306440" data-title="Reverse%20genetic%20generation%20of%20recombinant%20Zaire%20Ebola%20viruses%20containing%20disrupted%20IRF-3%20inhibitory%20domains%20results%20in%20attenuated%20virus%20growth%20in%20vitro%20and%20higher%20levels%20of%20IRF-3%20activation%20without%20inhibiting%20viral%20transcription%20or%20replication." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Reverse+genetic+generation+of+recombinant+Zaire+Ebola+viruses+containing+disrupted+IRF-3+inhibitory+domains+results+in+attenuated+virus+growth+in+vitro+and+higher+levels+of+IRF-3+activation+without+inhibiting+viral+transcription+or+replication.+Hartman+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="ppat-0030086-b033" id="ppat-0030086-b033" class="link-target"></a>Lakatos L, Csorba T, Pantaleo V, Chapman EJ, Carrington JC, et al.  (2006) Small RNA binding is a common strategy to suppress RNA silencing by several viral suppressors. EMBO J  25: 2768–2780.  <ul class="reflinks"><li><a href="#" data-author="Lakatos" data-cit="LakatosLCsorbaTPantaleoVChapmanEJCarringtonJC2006Small%20RNA%20binding%20is%20a%20common%20strategy%20to%20suppress%20RNA%20silencing%20by%20several%20viral%20suppressors.EMBO%20J2527682780" data-title="Small%20RNA%20binding%20is%20a%20common%20strategy%20to%20suppress%20RNA%20silencing%20by%20several%20viral%20suppressors." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Small+RNA+binding+is+a+common+strategy+to+suppress+RNA+silencing+by+several+viral+suppressors.+Lakatos+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="ppat-0030086-b034" id="ppat-0030086-b034" class="link-target"></a>Merai Z, Kerenyi Z, Kertesz S, Magna M, Lakatos L, et al.  (2006) Double-stranded RNA binding may be a general plant RNA viral strategy to suppress RNA silencing. J Virol  80: 5747–5756.  <ul class="reflinks"><li><a href="#" data-author="Merai" data-cit="MeraiZKerenyiZKerteszSMagnaMLakatosL2006Double-stranded%20RNA%20binding%20may%20be%20a%20general%20plant%20RNA%20viral%20strategy%20to%20suppress%20RNA%20silencing.J%20Virol8057475756" data-title="Double-stranded%20RNA%20binding%20may%20be%20a%20general%20plant%20RNA%20viral%20strategy%20to%20suppress%20RNA%20silencing." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Double-stranded+RNA+binding+may+be+a+general+plant+RNA+viral+strategy+to+suppress+RNA+silencing.+Merai+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="ppat-0030086-b035" id="ppat-0030086-b035" class="link-target"></a>Li H, Li WX, Ding SW (2002) Induction and suppression of RNA silencing by an animal virus. Science  296: 1319–1321.  <ul class="reflinks"><li><a href="#" data-author="Li" data-cit="LiHLiWXDingSW2002Induction%20and%20suppression%20of%20RNA%20silencing%20by%20an%20animal%20virus.Science29613191321" data-title="Induction%20and%20suppression%20of%20RNA%20silencing%20by%20an%20animal%20virus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Induction+and+suppression+of+RNA+silencing+by+an+animal+virus.+Li+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="ppat-0030086-b036" id="ppat-0030086-b036" class="link-target"></a>Berkhout B, Silverman RH, Jeang KT (1989) Tat trans-activates the human immunodeficiency virus through a nascent RNA target. Cell  59: 273–282.  <ul class="reflinks"><li><a href="#" data-author="Berkhout" data-cit="BerkhoutBSilvermanRHJeangKT1989Tat%20trans-activates%20the%20human%20immunodeficiency%20virus%20through%20a%20nascent%20RNA%20target.Cell59273282" data-title="Tat%20trans-activates%20the%20human%20immunodeficiency%20virus%20through%20a%20nascent%20RNA%20target." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Tat+trans-activates+the+human+immunodeficiency+virus+through+a+nascent+RNA+target.+Berkhout+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="ppat-0030086-b037" id="ppat-0030086-b037" class="link-target"></a>Verhoef K, Marzio G, Hillen W, Bujard H, Berkhout B (2001) Strict control of human immunodeficiency virus type 1 replication by a genetic switch: Tet for Tat. J Virol  75: 979–987.  <ul class="reflinks"><li><a href="#" data-author="Verhoef" data-cit="VerhoefKMarzioGHillenWBujardHBerkhoutB2001Strict%20control%20of%20human%20immunodeficiency%20virus%20type%201%20replication%20by%20a%20genetic%20switch%3A%20Tet%20for%20Tat.J%20Virol75979987" data-title="Strict%20control%20of%20human%20immunodeficiency%20virus%20type%201%20replication%20by%20a%20genetic%20switch%3A%20Tet%20for%20Tat." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Strict+control+of+human+immunodeficiency+virus+type+1+replication+by+a+genetic+switch%3A+Tet+for+Tat.+Verhoef+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="ppat-0030086-b038" id="ppat-0030086-b038" class="link-target"></a>Verhoef K, Koper M, Berkhout B (1997) Determination of the minimal amount of Tat activity required for human immunodeficiency virus type 1 replication. Virol  237: 228–236.  <ul class="reflinks"><li><a href="#" data-author="Verhoef" data-cit="VerhoefKKoperMBerkhoutB1997Determination%20of%20the%20minimal%20amount%20of%20Tat%20activity%20required%20for%20human%20immunodeficiency%20virus%20type%201%20replication.Virol237228236" data-title="Determination%20of%20the%20minimal%20amount%20of%20Tat%20activity%20required%20for%20human%20immunodeficiency%20virus%20type%201%20replication." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Determination+of+the+minimal+amount+of+Tat+activity+required+for+human+immunodeficiency+virus+type+1+replication.+Verhoef+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="ppat-0030086-b039" id="ppat-0030086-b039" class="link-target"></a>Sijen T, Plasterk RH (2003) Transposon silencing in the <span class="genus-species">Caenorhabditis elegans</span> germ line by natural RNAi. Nature  426: 310–314.  <ul class="reflinks"><li><a href="#" data-author="Sijen" data-cit="SijenTPlasterkRH2003Transposon%20silencing%20in%20the%20Caenorhabditis%20elegans%20germ%20line%20by%20natural%20RNAi.Nature426310314" data-title="Transposon%20silencing%20in%20the%20Caenorhabditis%20elegans%20germ%20line%20by%20natural%20RNAi." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transposon+silencing+in+the+Caenorhabditis+elegans+germ+line+by+natural+RNAi.+Sijen+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="ppat-0030086-b040" id="ppat-0030086-b040" class="link-target"></a>Das AT, Brummelkamp TR, Westerhout EM, Vink M, Madiredjo M, et al.  (2004) Human immunodeficiency virus type 1 escapes from RNA interference-mediated inhibition. J Virol  78: 2601–2605.  <ul class="reflinks"><li><a href="#" data-author="Das" data-cit="DasATBrummelkampTRWesterhoutEMVinkMMadiredjoM2004Human%20immunodeficiency%20virus%20type%201%20escapes%20from%20RNA%20interference-mediated%20inhibition.J%20Virol7826012605" data-title="Human%20immunodeficiency%20virus%20type%201%20escapes%20from%20RNA%20interference-mediated%20inhibition." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Human+immunodeficiency+virus+type+1+escapes+from+RNA+interference-mediated+inhibition.+Das+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="ppat-0030086-b041" id="ppat-0030086-b041" class="link-target"></a>Westerhout EM, Ooms M, Vink M, Das AT, Berkhout B (2005) HIV-1 can escape from RNA interference by evolving an alternative structure in its RNA genome. Nucleic Acids Res  33: 796–804.  <ul class="reflinks"><li><a href="#" data-author="Westerhout" data-cit="WesterhoutEMOomsMVinkMDasATBerkhoutB2005HIV-1%20can%20escape%20from%20RNA%20interference%20by%20evolving%20an%20alternative%20structure%20in%20its%20RNA%20genome.Nucleic%20Acids%20Res33796804" data-title="HIV-1%20can%20escape%20from%20RNA%20interference%20by%20evolving%20an%20alternative%20structure%20in%20its%20RNA%20genome." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=HIV-1+can+escape+from+RNA+interference+by+evolving+an+alternative+structure+in+its+RNA+genome.+Westerhout+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="ppat-0030086-b042" id="ppat-0030086-b042" class="link-target"></a>Triboulet R, Mari B, Lin YL, Chable-Bessia C, Bennasser Y, et al.  (2007) Suppression of microRNA-silencing pathway by HIV-1 during virus replication. Science  315: 1579–1582.  <ul class="reflinks"><li><a href="#" data-author="Triboulet" data-cit="TribouletRMariBLinYLChable-BessiaCBennasserY2007Suppression%20of%20microRNA-silencing%20pathway%20by%20HIV-1%20during%20virus%20replication.Science31515791582" data-title="Suppression%20of%20microRNA-silencing%20pathway%20by%20HIV-1%20during%20virus%20replication." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Suppression+of+microRNA-silencing+pathway+by+HIV-1+during+virus+replication.+Triboulet+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="ppat-0030086-b043" id="ppat-0030086-b043" class="link-target"></a>Kato H, Takeuchi O, Sato S, Yoneyama M, Yamamoto M, et al.  (2006) Differential roles of MDA5 and RIG-I helicases in the recognition of RNA viruses. Nature  441: 101–105.  <ul class="reflinks"><li><a href="#" data-author="Kato" data-cit="KatoHTakeuchiOSatoSYoneyamaMYamamotoM2006Differential%20roles%20of%20MDA5%20and%20RIG-I%20helicases%20in%20the%20recognition%20of%20RNA%20viruses.Nature441101105" data-title="Differential%20roles%20of%20MDA5%20and%20RIG-I%20helicases%20in%20the%20recognition%20of%20RNA%20viruses." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Differential+roles+of+MDA5+and+RIG-I+helicases+in+the+recognition+of+RNA+viruses.+Kato+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref44"><span class="order">44.
            </span><a name="ppat-0030086-b044" id="ppat-0030086-b044" class="link-target"></a>Marques JT, Devosse T, Wang D, Zamanian-Daryoush M, Serbinowski P, et al.  (2006) A structural basis for discriminating between self and nonself double-stranded RNAs in mammalian cells. Nat Biotechnol  24: 559–565.  <ul class="reflinks"><li><a href="#" data-author="Marques" data-cit="MarquesJTDevosseTWangDZamanian-DaryoushMSerbinowskiP2006A%20structural%20basis%20for%20discriminating%20between%20self%20and%20nonself%20double-stranded%20RNAs%20in%20mammalian%20cells.Nat%20Biotechnol24559565" data-title="A%20structural%20basis%20for%20discriminating%20between%20self%20and%20nonself%20double-stranded%20RNAs%20in%20mammalian%20cells." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+structural+basis+for+discriminating+between+self+and+nonself+double-stranded+RNAs+in+mammalian+cells.+Marques+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref45"><span class="order">45.
            </span><a name="ppat-0030086-b045" id="ppat-0030086-b045" class="link-target"></a>Stark GR, Kerr IM, Williams BR, Silverman RH, Schreiber RD (1998) How cells respond to interferons. Annu Rev Biochem  67: 227–264.  <ul class="reflinks"><li><a href="#" data-author="Stark" data-cit="StarkGRKerrIMWilliamsBRSilvermanRHSchreiberRD1998How%20cells%20respond%20to%20interferons.Annu%20Rev%20Biochem67227264" data-title="How%20cells%20respond%20to%20interferons." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=How+cells+respond+to+interferons.+Stark+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref46"><span class="order">46.
            </span><a name="ppat-0030086-b046" id="ppat-0030086-b046" class="link-target"></a>Zhou A, Paranjape J, Brown TL, Nie H, Naik S, et al.  (1997) Interferon action and apoptosis are defective in mice devoid of 2′,5′-oligoadenylate-dependent RNase L. EMBO J  16: 6355–6363.  <ul class="reflinks"><li><a href="#" data-author="Zhou" data-cit="ZhouAParanjapeJBrownTLNieHNaikS1997Interferon%20action%20and%20apoptosis%20are%20defective%20in%20mice%20devoid%20of%202%E2%80%B2%2C5%E2%80%B2-oligoadenylate-dependent%20RNase%20L.EMBO%20J1663556363" data-title="Interferon%20action%20and%20apoptosis%20are%20defective%20in%20mice%20devoid%20of%202%E2%80%B2%2C5%E2%80%B2-oligoadenylate-dependent%20RNase%20L." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Interferon+action+and+apoptosis+are+defective+in+mice+devoid+of+2%E2%80%B2%2C5%E2%80%B2-oligoadenylate-dependent+RNase+L.+Zhou+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref47"><span class="order">47.
            </span><a name="ppat-0030086-b047" id="ppat-0030086-b047" class="link-target"></a>Meylan E, Curran J, Hofmann K, Moradpour D, Binder M, et al.  (2005) Cardif is an adaptor protein in the RIG-I antiviral pathway and is targeted by hepatitis C virus. Nature  437: 1167–1172.  <ul class="reflinks"><li><a href="#" data-author="Meylan" data-cit="MeylanECurranJHofmannKMoradpourDBinderM2005Cardif%20is%20an%20adaptor%20protein%20in%20the%20RIG-I%20antiviral%20pathway%20and%20is%20targeted%20by%20hepatitis%20C%20virus.Nature43711671172" data-title="Cardif%20is%20an%20adaptor%20protein%20in%20the%20RIG-I%20antiviral%20pathway%20and%20is%20targeted%20by%20hepatitis%20C%20virus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cardif+is+an+adaptor+protein+in+the+RIG-I+antiviral+pathway+and+is+targeted+by+hepatitis+C+virus.+Meylan+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref48"><span class="order">48.
            </span><a name="ppat-0030086-b048" id="ppat-0030086-b048" class="link-target"></a>Hilleman MR (2004) Strategies and mechanisms for host and pathogen survival in acute and persistent viral infections. Proc Natl Acad Sci U S A  101(Suppl 2): 14560–14566.  <ul class="reflinks"><li><a href="#" data-author="Hilleman" data-cit="HillemanMR2004Strategies%20and%20mechanisms%20for%20host%20and%20pathogen%20survival%20in%20acute%20and%20persistent%20viral%20infections.Proc%20Natl%20Acad%20Sci%20U%20S%20A101Suppl%2021456014566" data-title="Strategies%20and%20mechanisms%20for%20host%20and%20pathogen%20survival%20in%20acute%20and%20persistent%20viral%20infections." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Strategies+and+mechanisms+for+host+and+pathogen+survival+in+acute+and+persistent+viral+infections.+Hilleman+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref49"><span class="order">49.
            </span><a name="ppat-0030086-b049" id="ppat-0030086-b049" class="link-target"></a>Fernandez-Sesma A, Marukian S, Ebersole BJ, Kaminski D, Park MS, et al.  (2006) Influenza virus evades innate and adaptive immunity via the NS1 protein. J Virol  80: 6295–6304.  <ul class="reflinks"><li><a href="#" data-author="Fernandez-Sesma" data-cit="Fernandez-SesmaAMarukianSEbersoleBJKaminskiDParkMS2006Influenza%20virus%20evades%20innate%20and%20adaptive%20immunity%20via%20the%20NS1%20protein.J%20Virol8062956304" data-title="Influenza%20virus%20evades%20innate%20and%20adaptive%20immunity%20via%20the%20NS1%20protein." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Influenza+virus+evades+innate+and+adaptive+immunity+via+the+NS1+protein.+Fernandez-Sesma+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref50"><span class="order">50.
            </span><a name="ppat-0030086-b050" id="ppat-0030086-b050" class="link-target"></a>Katze MG, He Y, Gale M Jr (2002) Viruses and interferon: A fight for supremacy. Nat Rev Immunol  2: 675–687.  <ul class="reflinks"><li><a href="#" data-author="Katze" data-cit="KatzeMGHeYGaleMJr2002Viruses%20and%20interferon%3A%20A%20fight%20for%20supremacy.Nat%20Rev%20Immunol2675687" data-title="Viruses%20and%20interferon%3A%20A%20fight%20for%20supremacy." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Viruses+and+interferon%3A+A+fight+for+supremacy.+Katze+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref51"><span class="order">51.
            </span><a name="ppat-0030086-b051" id="ppat-0030086-b051" class="link-target"></a>Xiang Y, Condit RC, Vijaysri S, Jacobs B, Williams BR, et al.  (2002) Blockade of interferon induction and action by the E3L double-stranded RNA binding proteins of vaccinia virus. J Virol  76: 5251–5259.  <ul class="reflinks"><li><a href="#" data-author="Xiang" data-cit="XiangYConditRCVijaysriSJacobsBWilliamsBR2002Blockade%20of%20interferon%20induction%20and%20action%20by%20the%20E3L%20double-stranded%20RNA%20binding%20proteins%20of%20vaccinia%20virus.J%20Virol7652515259" data-title="Blockade%20of%20interferon%20induction%20and%20action%20by%20the%20E3L%20double-stranded%20RNA%20binding%20proteins%20of%20vaccinia%20virus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Blockade+of+interferon+induction+and+action+by+the+E3L+double-stranded+RNA+binding+proteins+of+vaccinia+virus.+Xiang+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref52"><span class="order">52.
            </span><a name="ppat-0030086-b052" id="ppat-0030086-b052" class="link-target"></a>Chang HW, Watson JC, Jacobs BL (1992) The E3L gene of vaccinia virus encodes an inhibitor of the interferon-induced, double-stranded RNA-dependent protein kinase. Proc Natl Acad Sci U S A  89: 4825–4829.  <ul class="reflinks"><li><a href="#" data-author="Chang" data-cit="ChangHWWatsonJCJacobsBL1992The%20E3L%20gene%20of%20vaccinia%20virus%20encodes%20an%20inhibitor%20of%20the%20interferon-induced%2C%20double-stranded%20RNA-dependent%20protein%20kinase.Proc%20Natl%20Acad%20Sci%20U%20S%20A8948254829" data-title="The%20E3L%20gene%20of%20vaccinia%20virus%20encodes%20an%20inhibitor%20of%20the%20interferon-induced%2C%20double-stranded%20RNA-dependent%20protein%20kinase." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+E3L+gene+of+vaccinia+virus+encodes+an+inhibitor+of+the+interferon-induced%2C+double-stranded+RNA-dependent+protein+kinase.+Chang+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref53"><span class="order">53.
            </span><a name="ppat-0030086-b053" id="ppat-0030086-b053" class="link-target"></a>Lu Y, Wambach M, Katze MG, Krug RM (1995) Binding of the influenza virus NS1 protein to double-stranded RNA inhibits the activation of the protein kinase that phosphorylates the elF-2 translation initiation factor. Virol  214: 222–228.  <ul class="reflinks"><li><a href="#" data-author="Lu" data-cit="LuYWambachMKatzeMGKrugRM1995Binding%20of%20the%20influenza%20virus%20NS1%20protein%20to%20double-stranded%20RNA%20inhibits%20the%20activation%20of%20the%20protein%20kinase%20that%20phosphorylates%20the%20elF-2%20translation%20initiation%20factor.Virol214222228" data-title="Binding%20of%20the%20influenza%20virus%20NS1%20protein%20to%20double-stranded%20RNA%20inhibits%20the%20activation%20of%20the%20protein%20kinase%20that%20phosphorylates%20the%20elF-2%20translation%20initiation%20factor." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Binding+of+the+influenza+virus+NS1+protein+to+double-stranded+RNA+inhibits+the+activation+of+the+protein+kinase+that+phosphorylates+the+elF-2+translation+initiation+factor.+Lu+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref54"><span class="order">54.
            </span><a name="ppat-0030086-b054" id="ppat-0030086-b054" class="link-target"></a>Min JY, Krug RM (2006) The primary function of RNA binding by the influenza A virus NS1 protein in infected cells: Inhibiting the 2′-5′ oligo (A) synthetase/RNase L pathway. Proc Natl Acad Sci U S A  103: 7100–7105.  <ul class="reflinks"><li><a href="#" data-author="Min" data-cit="MinJYKrugRM2006The%20primary%20function%20of%20RNA%20binding%20by%20the%20influenza%20A%20virus%20NS1%20protein%20in%20infected%20cells%3A%20Inhibiting%20the%202%E2%80%B2-5%E2%80%B2%20oligo%20%28A%29%20synthetase%2FRNase%20L%20pathway.Proc%20Natl%20Acad%20Sci%20U%20S%20A10371007105" data-title="The%20primary%20function%20of%20RNA%20binding%20by%20the%20influenza%20A%20virus%20NS1%20protein%20in%20infected%20cells%3A%20Inhibiting%20the%202%E2%80%B2-5%E2%80%B2%20oligo%20%28A%29%20synthetase%2FRNase%20L%20pathway." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+primary+function+of+RNA+binding+by+the+influenza+A+virus+NS1+protein+in+infected+cells%3A+Inhibiting+the+2%E2%80%B2-5%E2%80%B2+oligo+%28A%29+synthetase%2FRNase+L+pathway.+Min+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref55"><span class="order">55.
            </span><a name="ppat-0030086-b055" id="ppat-0030086-b055" class="link-target"></a>Rivas C, Gil J, Melkova Z, Esteban M, Diaz-Guerra M (1998) Vaccinia virus E3L protein is an inhibitor of the interferon (i.f.n.)-induced 2–5A synthetase enzyme. Virol  243: 406–414.  <ul class="reflinks"><li><a href="#" data-author="Rivas" data-cit="RivasCGilJMelkovaZEstebanMDiaz-GuerraM1998Vaccinia%20virus%20E3L%20protein%20is%20an%20inhibitor%20of%20the%20interferon%20%28i.f.n.%29-induced%202%E2%80%935A%20synthetase%20enzyme.Virol243406414" data-title="Vaccinia%20virus%20E3L%20protein%20is%20an%20inhibitor%20of%20the%20interferon%20%28i.f.n.%29-induced%202%E2%80%935A%20synthetase%20enzyme." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Vaccinia+virus+E3L+protein+is+an+inhibitor+of+the+interferon+%28i.f.n.%29-induced+2%E2%80%935A+synthetase+enzyme.+Rivas+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref56"><span class="order">56.
            </span><a name="ppat-0030086-b056" id="ppat-0030086-b056" class="link-target"></a>Mibayashi M, Martinez-Sobrido L, Loo YM, Cardenas WB, Gale M Jr, et al.  (2007) Inhibition of retinoic acid-inducible gene I-mediated induction of beta interferon by the NS1 protein of influenza A virus. J Virol  81: 514–524.  <ul class="reflinks"><li><a href="#" data-author="Mibayashi" data-cit="MibayashiMMartinez-SobridoLLooYMCardenasWBGaleMJr2007Inhibition%20of%20retinoic%20acid-inducible%20gene%20I-mediated%20induction%20of%20beta%20interferon%20by%20the%20NS1%20protein%20of%20influenza%20A%20virus.J%20Virol81514524" data-title="Inhibition%20of%20retinoic%20acid-inducible%20gene%20I-mediated%20induction%20of%20beta%20interferon%20by%20the%20NS1%20protein%20of%20influenza%20A%20virus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Inhibition+of+retinoic+acid-inducible+gene+I-mediated+induction+of+beta+interferon+by+the+NS1+protein+of+influenza+A+virus.+Mibayashi+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref57"><span class="order">57.
            </span><a name="ppat-0030086-b057" id="ppat-0030086-b057" class="link-target"></a>Geisbert TW, Hensley LE, Kagan E, Yu EZ, Geisbert JB, et al.  (2006) Postexposure protection of guinea pigs against a lethal ebola virus challenge is conferred by RNA interference. J Infect Dis  193: 1650–1657.  <ul class="reflinks"><li><a href="#" data-author="Geisbert" data-cit="GeisbertTWHensleyLEKaganEYuEZGeisbertJB2006Postexposure%20protection%20of%20guinea%20pigs%20against%20a%20lethal%20ebola%20virus%20challenge%20is%20conferred%20by%20RNA%20interference.J%20Infect%20Dis19316501657" data-title="Postexposure%20protection%20of%20guinea%20pigs%20against%20a%20lethal%20ebola%20virus%20challenge%20is%20conferred%20by%20RNA%20interference." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Postexposure+protection+of+guinea+pigs+against+a+lethal+ebola+virus+challenge+is+conferred+by+RNA+interference.+Geisbert+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

