



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Methicillin-resistant <em>Staphylococcus aureus</em> (MRSA) is intrinsically cross-resistant to virtually all β-lactam antibiotics. The central determinant for the MRSA phenotype is the <em>mecA</em> gene, whose transcriptional control may be mediated by a repressor (<em>mecI</em>) and a sensor/inducer (<em>mecR1</em>). The <em>mecI-mecR1-</em>mediated induction of <em>mecA</em> takes several hours rendering the strains phenotypically susceptible in spite of the presence of the resistance gene. Therefore, it has been proposed that the full resistance to β-lactams observed in many contemporary clinical MRSA strains requires a non-functional <em>mecI-mecR1</em> regulatory system. The <em>mecA</em> gene is embedded in a large chromosomal cassette (the SCC<em>mec</em> element) for which several structural types have been described. Some epidemic MRSA clones, typically expressing full β-lactam resistance, carry SCC<em>mec</em> elements that contain an intact <em>mecI-mecR1 locus</em> (e.g. SCC<em>mec</em> types II and III). We have addressed this apparent contradiction by first sequencing the <em>mecI</em> coding region and <em>mecA</em> promoter sequences in a collection of prototype MRSA strains characterized by different SCC<em>mec</em> types. A conserved non-sense mutation within <em>mecI</em> was detected in all SCC<em>mec</em> type III strains tested, presumably responsible for a non-functional truncated MecI protein and, therefore, explaining the full resistance phenotype. In SCC<em>mec</em> type II strains no conserved mutations were found. We next transformed a collection of prototype MRSA epidemic strains with a recombinant plasmid overexpressing a wild-type copy of <em>mecI</em>. Surprisingly, for the great majority of the strains no significant alterations in the phenotypic expression of β-lactam resistance could be detected. These findings were confirmed and further explored, challenging the currently accepted mechanism of <em>mecA</em> transcriptional control. Our observations suggest the existence of yet unidentified additional determinants involved in the transcriptional control of <em>mecA</em> gene and point to a revision of the <em>mecA</em> regulatory mechanism in contemporary MRSA strains.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Oliveira DC, de Lencastre H (2011) Methicillin-Resistance in <em>Staphylococcus aureus</em> Is Not Affected by the Overexpression in Trans of the <em>mecA</em> Gene Repressor: A Surprising Observation. PLoS ONE 6(8):
           e23287.
        
        https://doi.org/10.1371/journal.pone.0023287</p><p><strong>Editor: </strong>Laurence Van Melderen, Universite Libre de Bruxelles, Belgium</p><p><strong>Received: </strong>May 18, 2011; <strong>Accepted: </strong>July 12, 2011; <strong>Published: </strong> August 2, 2011</p><p><strong>Copyright: </strong> © 2011 Oliveira, de Lencastre. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>Partial support for this study was provided by Projects POCI/BIA-MIC/60320/2004 and PTDC/BIA-MIC/64071/2006 from Fundação para a Ciência e Tecnologia (FCT), Lisbon, Portugal, awarded to D. C. Oliveira; Project TROCAR, contract number 223031 from the European Commission awarded to H. de Lencastre; and grant RO1AI045738 from the U.S. Public Health Service awarded to A. Tomasz. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Methicillin-resistant <em>Staphylococcus aureus</em> (MRSA) is a leading cause of nosocomial infections worldwide and has also emerged as a community-associated pathogen <a href="#pone.0023287-Chambers1" class="ref-tip">[1]</a>. MRSA are inherently cross-resistant to virtually all β-lactam antibiotics, the most effective and widely used class of antimicrobials. Moreover, MRSA clinical strains are quite often multi-drug resistant, reducing significantly the therapeutic options for the treatment of staphylococcal infections.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>The MRSA characteristic phenotype is due to the presence of <em>mecA</em>, which encodes a penicillin-binding protein (PBP), PBP2a, with reduced affinity for β-lactams <a href="#pone.0023287-Song1" class="ref-tip">[2]</a>, <a href="#pone.0023287-Hartman1" class="ref-tip">[3]</a>. <em>mecA</em> is embedded in a large heterologous chromosomal cassette, the SCC<em>mec</em> element <a href="#pone.0023287-Ito1" class="ref-tip">[4]</a>. Some MRSA strains carry upstream to the <em>mecA</em> gene the regulatory genes <em>mecI-mecR1</em> encoding for a repressor and a sensor/inducer of the <em>mecA</em> expression, respectively <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>. This genetic organization is similar to the β-lactamase locus that encodes for penicillin-resistance only, and contains the structural gene (<em>blaZ</em>), a repressor (<em>blaI</em>) and a sensor/inducer (<em>blaR1</em>). Apart from the identical structural organization between <em>mec</em> and <em>bla</em> systems, there is also good homology between the inducers (61% identity at the amino acid level) and the β-lactam binding domain of the repressors (44% identity at the amino acid level). In fact, there is a cross-talk between both regulatory systems, as each one alone is able to control the transcription of <em>mecA</em> and <em>bla</em>Z <a href="#pone.0023287-Hackbarth1" class="ref-tip">[6]</a>, <a href="#pone.0023287-Ryffel1" class="ref-tip">[7]</a>. However, the two regulatory systems differ remarkably in the induction efficiency: the <em>blaI-blaR1</em> system induces <em>mecA</em> in a few minutes, whereas the <em>mecI-mecR1</em> system takes several hours <a href="#pone.0023287-Hackbarth1" class="ref-tip">[6]</a>, <a href="#pone.0023287-Ryffel1" class="ref-tip">[7]</a>, <a href="#pone.0023287-McKinney1" class="ref-tip">[8]</a>. Actually, the <em>mecI</em>-<em>mecR1</em> mediated induction of <em>mecA</em> expression is so slow that, in clinical terms, fully functional <em>mecI</em> and <em>mecR1</em> genes render the cell phenotypically susceptible in spite of the presence of <em>mecA</em> – the so-called “pre-MRSA” phenotype <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>, <a href="#pone.0023287-KuwaharaArai1" class="ref-tip">[9]</a>. In agreement with this observation, the <em>in vitro</em> deletion of the <em>mecI</em> gene has been shown to increase the resistance levels to β-lactams in staphylococci <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>, <a href="#pone.0023287-KuwaharaArai1" class="ref-tip">[9]</a>, <a href="#pone.0023287-Dickinson1" class="ref-tip">[10]</a>.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Based on those observations, it has been postulated that full resistance to β-lactams, characteristic of many contemporary MRSA clinical strains, implies a non-functional <em>mecI-mecR1</em> regulatory system <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>, <a href="#pone.0023287-KuwaharaArai1" class="ref-tip">[9]</a>. As a matter of fact, the absence of <em>mecI</em> or the accumulation of point mutations in the <em>mecI</em> coding sequence, <em>mecI</em> ribosomal binding site or <em>mecA</em> gene promoter, have been found in several MRSA strains <a href="#pone.0023287-Archer1" class="ref-tip">[11]</a>, <a href="#pone.0023287-HurlimannDalel1" class="ref-tip">[12]</a>, <a href="#pone.0023287-Kobayashi1" class="ref-tip">[13]</a>, <a href="#pone.0023287-Suzuki1" class="ref-tip">[14]</a>, <a href="#pone.0023287-Weller1" class="ref-tip">[15]</a>. Nevertheless, in spite of the negative effect of the presence of MecI on the phenotypic expression of resistance, there is no clear direct correlation between the cellular amounts of <em>mecA</em> transcript or PBP2a protein and the phenotypic level of resistance (i.e. the minimum inhibitory concentration, MIC, for the strain) <a href="#pone.0023287-Hartman2" class="ref-tip">[16]</a>, <a href="#pone.0023287-Murakami1" class="ref-tip">[17]</a>. Finally, the existence of other unknown determinants involved in <em>mecA</em> regulation, for instance mediating the signal transduction between the activated MecR1 and the promoter bound MecI, has been postulated based on contradictory experimental observations <a href="#pone.0023287-Kobayashi1" class="ref-tip">[13]</a>, <a href="#pone.0023287-Weller1" class="ref-tip">[15]</a>, <a href="#pone.0023287-Archer2" class="ref-tip">[18]</a> and critical structural data analysis <a href="#pone.0023287-MallorquFernndez1" class="ref-tip">[19]</a>, <a href="#pone.0023287-GarcaCastellanos1" class="ref-tip">[20]</a>. Early studies on the β-lactamase regulon in <em>S. aureus</em> by Cohen and Sweeney have also suggested the existence of the <em>blaR2</em> locus, unlinked to <em>blaI-blaR1</em> and able to mediate the constitutive expression of <em>blaZ</em> <a href="#pone.0023287-Cohen1" class="ref-tip">[21]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>Currently, MRSA clones are defined not only based on the type of genetic lineage but also on the type of SCC<em>mec</em> element they carry, as the same lineage may be associated with several SCC<em>mec</em> types <a href="#pone.0023287-Oliveira1" class="ref-tip">[22]</a>, <a href="#pone.0023287-Enright1" class="ref-tip">[23]</a>. Several structural types of the SCC<em>mec</em> element have been described differing in size and genetic content <a href="#pone.0023287-IWGSCC1" class="ref-tip">[24]</a>. The genetic organization of the <em>mecA</em> vicinity (<em>mec</em> gene complex) is one of the key characteristics used to define SCC<em>mec</em> types. In <em>S. aureus</em> three major <em>mec</em> classes have been described based mainly on the presence of insertion sequences and intact or disrupted <em>mecI-mecR1</em> sequences: class A has intact sequences for <em>mecI-mecR1</em>, whereas classes B and C have no <em>mecI</em> and partially deleted <em>mecR1</em> due to the integration of insertion sequences in the regulatory region of the <em>mecA</em>. Considering the eight major SCC<em>mec</em> types described so far in <em>S. aureus</em>, the <em>mec</em> gene complex class A characterizes SCC<em>mec</em> types II, III and VIII; class B, SCC<em>mec</em> types I, IV and VI; and class C, SCC<em>mec</em> types V and VII. According to the current model of <em>mecA</em> transcriptional control <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>, <a href="#pone.0023287-KuwaharaArai1" class="ref-tip">[9]</a>, it is tempting to interpret the disruption of the <em>mecI-mecR1</em> regulatory system in SCC<em>mec</em> types I and IV–VII, as a strategy to overcome the tight <em>mecA</em> repression mediated by <em>mecI-mecR1</em> system in MRSA strains. However, it seems that there is no clear correlation between resistance levels and <em>mecI-mecR1</em> functionality, as some strains lacking <em>mecI-mecR1</em> have a very low resistance level, whereas some strains with complete <em>mecI-mecR1</em> locus are highly resistant <a href="#pone.0023287-Suzuki1" class="ref-tip">[14]</a>, <a href="#pone.0023287-Weller1" class="ref-tip">[15]</a>, <a href="#pone.0023287-Shukla1" class="ref-tip">[25]</a>. In addition, the epidemicity of MRSA strains does not correlate with the <em>mecI-mecR1</em> functionality as well. For example, two major pandemic nosocomial MRSA clones – the New York/Japan (or ST5-II) and the Brazilian (or ST239-III) clones – are characterized by SCC<em>mec</em> types II and III, respectively, and have a complete <em>mecI-mecR1</em> locus <a href="#pone.0023287-Oliveira1" class="ref-tip">[22]</a>, <a href="#pone.0023287-Enright1" class="ref-tip">[23]</a>.</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5" class="link-target"></a><p>In this study, we have addressed the puzzling lack of correlation between SCC<em>mec</em> type (i.e. <em>mec</em> gene class) and resistance phenotype in order to clarify how the <em>mecA</em> transcription is controlled in pandemic MRSA strains. We first characterized by DNA sequencing the <em>mecA</em> regulatory locus in prototype strains of SCC<em>mec</em> types II and III (i.e. with a complete <em>mecI-mecR1</em> locus). We found a conserved point mutation within the <em>mecI</em> coding-sequence among SCC<em>mec</em> type III strains, which introduces a premature stop codon resulting in truncated MecI repressor. Among the SCC<em>mec</em> type II strains we could not find any conserved sequence alteration either in the <em>mecI</em> coding sequence or in the <em>mecA</em> promoter that could suggest a non-functional <em>mecI-mecR1</em> system and, as such, justify the high-resistance levels to β-lactams. We thus decided to challenge the current model of <em>mecA</em> regulation by over-expressing <em>in trans</em> the wild-type MecI in a collection of prototype MRSA clinical strains. To our surprise, in most strains we could not detect any significant alterations in the phenotypic expression of oxacillin resistance. These observations suggest that other yet uncharacterized factors are involved in the control of the expression of β-lactam resistance in MRSA, namely by interfering with the <em>mecI</em>-mediated repression of <em>mecA</em>, and point to a revision of the current model for the transcriptional control of the resistance determinant in contemporary strains.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Bacterial strains and culture conditions"></a>
<h3>Bacterial strains and culture conditions</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>MRSA strains used in this study and their relevant characteristics are listed in <a href="#pone-0023287-t001">Tables 1</a> and <a href="#pone-0023287-t002">2</a>. All strains have been selected from large international collections and have been previously characterized in detail for genetic background by pulsed-field gel electrophoresis (PFGE), multi-locus sequence typing (MLST), and <em>spa</em> typing and also for SCC<em>mec</em> type. In <a href="#pone-0023287-t001">Table 1</a>, we have included prototype strains for three major pandemic MRSA lineages characterized by SCC<em>mec</em> types I–III <a href="#pone.0023287-Oliveira1" class="ref-tip">[22]</a>: the Iberian or ST247-I clone, the New York/Japan or ST5-II clone, and the Brazilian or ST239-III clone. In addition, in <a href="#pone-0023287-t001">Table 1</a> we have also included two reference strains extensively used in studies addressing the β-lactam resistance mechanisms in <em>S. aureus</em>: strains COL <a href="#pone.0023287-deLencastre1" class="ref-tip">[26]</a> and N315 <a href="#pone.0023287-KuwaharaArai1" class="ref-tip">[9]</a>. Strain COL is highly and homogenously resistant to oxacillin, has a non-functional <em>mecI-mecR1</em> system, it naturally lacks β-lactamase, and constitutively expresses <em>mecA</em> <a href="#pone.0023287-deLencastre1" class="ref-tip">[26]</a>, <a href="#pone.0023287-deLencastre2" class="ref-tip">[27]</a>. Strain N315, has a very heterogeneous oxacillin-resistance phenotypic profile, has wild-type <em>mecI-mecR1</em> sequences, is β-lactamase positive and has an inducible expression of <em>mecA</em> <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>, <a href="#pone.0023287-KuwaharaArai1" class="ref-tip">[9]</a>. In <a href="#pone-0023287-t002">Table 2</a>, we list further MRSA strains that were selected to extended our initial collection in order to confirm the experimental observations. For this purpose, we have included more representative SCC<em>mec</em> type II strains and also strains characterized by SCC<em>mec</em> types IV–VI <a href="#pone.0023287-Milheirico1" class="ref-tip">[28]</a>, <a href="#pone.0023287-Milheirico2" class="ref-tip">[29]</a>, <a href="#pone.0023287-Oliveira2" class="ref-tip">[30]</a>. <em>S. aureus</em> strains were routinely grown overnight at 37°C under aerobic conditions on tryptic soy agar or tryptic soy broth (Difco, BD). <em>Escherichia coli</em> strains were grown in Luria-Bertani broth (Difco, BD) with aeration at 37°C. Chloramphenicol (20 µg/ml) and ampicillin (100 µg/ml) (Sigma-Aldrich) were used for selection and maintenance of <em>S. aureus</em> and <em>E. coli</em> transformants, respectively.</p>
<a class="link-target" id="pone-0023287-t001" name="pone-0023287-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0023287.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0023287.t001" data-doi="info:doi/10.1371/journal.pone.0023287" data-uri="info:doi/10.1371/journal.pone.0023287.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0023287.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> Characteristics of the prototype MRSA strains used in this study.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0023287.t001">
              https://doi.org/10.1371/journal.pone.0023287.t001</a></p></div><a class="link-target" id="pone-0023287-t002" name="pone-0023287-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0023287.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0023287.t002" data-doi="info:doi/10.1371/journal.pone.0023287" data-uri="info:doi/10.1371/journal.pone.0023287.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0023287.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> Characteristics of the extended collection of representative MRSA strains.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0023287.t002">
              https://doi.org/10.1371/journal.pone.0023287.t002</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="DNA manipulations"></a>
<h3>DNA manipulations</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>DNA manipulations were performed by standard methods <a href="#pone.0023287-Ausubel1" class="ref-tip">[31]</a>, <a href="#pone.0023287-Sambrook1" class="ref-tip">[32]</a>. Restriction enzymes were used as recommended by the manufacturer (New England Biolabs). Routine PCR amplification was performed with AmpliTaq DNA polymerase (Applied Biosystems) according to the manufacturer's recommendations. Wizard Plus Minipreps and Midipreps (Promega) purification systems were used for plasmid extraction. PCR and digestion products were purified with Wizard PCR Preps and Wizard DNA Clean-up systems (Promega). Ligation reactions were performed with the Rapid DNA Ligation Kit (Roche). DNA sequencing was performed by the Rockefeller University Protein/DNA Technology Center or by Macrogen (<a href="http://www.macrogen.com">www.macrogen.com</a>).</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="β-lactamase detection"></a>
<h3>β-lactamase detection</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>Detection of functional β-lactamase locus was performed either with BBL™ DrySlide™ Nitrocefin (BD) (strains listed in <a href="#pone-0023287-t001">Table 1</a>) or with nitrocefin disks (Sigma-Aldrich) (strains listed in <a href="#pone-0023287-t002">Table 2</a>) according to the manufacturer's recommendations for cultures grown overnight in TSB with and without induction with oxacillin at 0.5 µg/ml. Results were recorded after 30 min. incubation at room temperature. In the case of strains negative for the Nitrocefin assay, the absence of β-lactamase genes was confirmed by PCR with three pairs of primers targeting internal fragments of ca. 500 bp of <em>blaZ</em>, <em>bla</em>I, and <em>blaR1</em>. Primers were designed based on the available sequence at GenBank for Tn<em>552</em> of <em>S. aureus</em> (accession number: X52734); primer sequences were as follows (5′→3′): blaZ F, <span class="gene">GAT AAG AGA TTT GCC TAT GC</span>; blaZ R, <span class="gene">GCA TAT GTT ATT GCT TGA CC</span>; blaI F, <span class="gene">GCA AGT TGA AAT ATC TAT GG</span>; blaI R, <span class="gene">GAA AGG ATC CAT TTT CTG TAC ACT CTC ATC</span>; blaR1 F, <span class="gene">CAT GAC AAT GAA GTA GAA GC</span>; and blaR1 R, <span class="gene">CTT ATG ATT CCA TGA CAT ACG</span>.</p>
</div>

<div id="section4" class="section toc-section"><a id="s2d" name="s2d" class="link-target" title="Phenotypic analysis"></a>
<h3>Phenotypic analysis</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>Initial susceptibility screening to oxacillin was determined for all parental and recombinant strains with 1 mg oxacillin diffusion disks prepared in-house <a href="#pone.0023287-deLencastre3" class="ref-tip">[33]</a>. For the MIC determination oxacillin E-test strips (AB Biodisk) were used for all strains. Overnight TSB cultures were adjusted to an optical density at 620 nm (OD<sub>620</sub>) of 0.08 (equivalent to 0.5 McFarland), plated onto Muller Hinton agar (Difco) plates supplemented with 2% NaCl, and incubated at 37°C for 24 h. Strains listed in <a href="#pone-0023287-t001">Table 1</a> were also tested by population analysis profiles (PAPs), as previously described <a href="#pone.0023287-Jett1" class="ref-tip">[34]</a>, <a href="#pone.0023287-Pinho1" class="ref-tip">[35]</a>. In short, 10 µl drops of 10<sup>0</sup>, 10<sup>−1</sup>, 10<sup>−2</sup>, 10<sup>−3</sup>, 10<sup>−4</sup>, 10<sup>−5</sup>, and 10<sup>−6</sup> dilutions of an overnight culture were plated on TSA plates containing 0, 0.75, 1.5, 3, 6, 12.5, 25, 50, 100, 200, 400, 800 µg/ml of oxacillin (Sigma). For each oxacillin concentration, colonies were counted for the first dilution with non-confluent growh after 24 h and 48 h of incubation at 37°C.</p>
</div>

<div id="section5" class="section toc-section"><a id="s2e" name="s2e" class="link-target" title="DNA sequencing of mecI and the mecA promoter"></a>
<h3>DNA sequencing of <em>mecI</em> and the <em>mecA</em> promoter</h3>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1" class="link-target"></a><p>Based on the sequence of the <em>mecA</em> regulatory region for the reference strain N315 (accession number D86934) two primer sets were designed for the amplification and sequencing of the <em>mecI</em> coding region and the promoter region of the <em>mecA</em> gene (P<em>mecA</em>). Primer sequences were as follows (5′→3′): mecI F, <span class="gene">TTA CGC TTA CCG CTT TTTCG</span>; mecI R, <span class="gene">ATC AAG ACT TGC ATT CAG GC</span>; PmecA F, <span class="gene">GTA ACA GAT GAT TGT TGA CC</span>; PmecA R, <span class="gene">AAG ATG AAG TGG TAA TAG CG</span>. DNA sequencing raw data analysis and multi-sequence alignments were performed using the DNA Star software package (Lasergene). All sequences have been deposited in GenBank with accession numbers JF946491–JF946513.</p>
</div>

<div id="section6" class="section toc-section"><a id="s2f" name="s2f" class="link-target" title="Recombinant strains"></a>
<h3>Recombinant strains</h3>
<a id="article1.body1.sec2.sec6.p1" name="article1.body1.sec2.sec6.p1" class="link-target"></a><p>A DNA fragment containing the wild-type <em>mecI</em> coding region and the putative ribosomal binding site from the prototype strain N315 was amplified by PCR with the high-fidelity <em>Pfu</em> Turbo DNA polymerase (Stratagene, La Jolla, CA) with primers <span class="gene">5′-ATG G<span class="underline">GA ATT C</span>AG CAC AAC AAA TTT CTG AGC-3′</span> (forward) and <span class="gene">5′-AGA G<span class="underline">GG GAT CC</span>T CAA CGA CTT GAT TGT TTC C-3′</span> (reverse) containing the underlined recognition sequences for endonucleases <em>Eco</em>RI and <em>Bam</em>HI, respectively. Using the same strategy a PCR fragment containing a non-functional <em>mecI</em> coding sequence (<em>mecI*</em>) was obtained from strain ANS46, a prototype strain for SCC<em>mec</em> type III strain with a non-sense mutation at Gln<sub>68</sub> in the <em>mecI</em> coding sequence <a href="#pone.0023287-Oliveira1" class="ref-tip">[22]</a>, <a href="#pone.0023287-Ito2" class="ref-tip">[36]</a>. (CAA→TAA) . A DNA fragment containing the wild-type <em>mecI-mecR1</em> coding regions and the putative ribosomal binding sites from the prototype strain N315 was also amplified by PCR with <em>Pfu</em> Turbo DNA polymerase with forward primer <span class="gene">5′-GTT C<span class="underline">GA ATT C</span>TT CTA CTT CAC CAT TAT CGC-3′</span> (containing the underlined recognition sequences for endonuclease <em>Eco</em>RI) and the same reverse primer described above. After double digestion and purification, the inserts were directionally cloned into the multiple cloning site of pGC2. pGC2 is a high-copy number <em>E. coli-S. aureus</em> shuttle plasmid with resistance determinants to ampicillin (<em>E. coli</em>) and chloramphenicol (<em>S. aureus</em>), obtained from P. Matthews, in which the multiple cloning site is flanked by the strong SP6 and T7 bacteriophage promoters. After ligation the recombinant plasmids were transformed and propagated in <em>E. coli</em> strain DH5α. The integrity of the <em>mecI</em><sup>WT</sup>, <em>mecI*</em>, and <em>mecImecR1</em> insert sequences was verified by DNA restriction analysis and sequencing. The recombinant plasmids were then introduced into the restriction-deficient <em>S. aureus</em> strain RN4220 (R. Novick) by electroporation with a Gene Pulser apparatus (Bio-Rad, Hercules, Calif.) essentially as previously described <a href="#pone.0023287-Kraemer1" class="ref-tip">[37]</a> and then transduced into the MRSA clinical strains by using phage 80α, as previously described <a href="#pone.0023287-Oshida1" class="ref-tip">[38]</a>.</p>
<a id="article1.body1.sec2.sec6.p2" name="article1.body1.sec2.sec6.p2" class="link-target"></a><p>For the electrophoretic mobility shift assay, the wild-type coding sequences of MecI and BlaI were fused to a histidine tag at the N-terminal. Inserts were obtained by PCR with the <em>Pfu</em> Turbo DNA polymerase using DNA from the prototype strain N315 as template and the following primers (5′→3′): mecI Q1, <span class="gene">TCA G<span class="underline">GG ATC C</span>GA TAA TAA AAC GTA TGA AAT ATC ATC TGC</span> (forward); mecI Q2, <span class="gene">GAG G<span class="underline">AA GCT T</span>TC AAC GAC TTG ATT GTT TCC</span> (reverse); blaI Q1, <span class="gene">GTC T<span class="underline">GG ATC C</span>GC CAA TAA GCA AGT TGA ATA TCT ATG G-3</span> (forward); and blaI Q2, <span class="gene">GAC A<span class="underline">AA GCT T</span>AT TTT CTG TAC ACT CTC ATC</span> (reverse), containing the underlined recognition sequences for endonucleases <em>Bam</em>HI (forward) and <em>Hind</em>III (reverse). After double digestion and purification, the inserts were directionally cloned into the multiple cloning site of pQE-30 (Qiagen), an <em>E. coli</em> expression vector containing the coding sequence for a tag of histidines upstream the <em>Bam</em>H1 restriction site. After ligation the recombinant plasmids were transformed and propagated in <em>E. coli</em> strain M15 (Qiagen). The integrity of insert sequences was verified by DNA restriction analysis and sequencing.</p>
</div>

<div id="section7" class="section toc-section"><a id="s2g" name="s2g" class="link-target" title="Total RNA isolation and Northern blot analysis"></a>
<h3>Total RNA isolation and Northern blot analysis</h3>
<a id="article1.body1.sec2.sec7.p1" name="article1.body1.sec2.sec7.p1" class="link-target"></a><p>Overnight cultures were grown in TSB, supplemented with chloramphenicol (10 µg/ml) when appropriate, and then diluted 1∶50 in fresh TSB. After cells were grown to the mid-log phase (OD<sub>620</sub>∼0.7), they were pelleted and processed with the FastRNA Blue isolation kit (Bio101, QBiogen) in combination with FastPrep FP120 (Bio101-Savant, QBiogen), according to the manufacturer's recommendations. For the analysis of the <em>mecA</em> induction profile, after cultures were grown to OD<sub>620</sub>∼0.7, oxacillin at 0.5 µg/ml was added and cultures were incubated for an additional 60 minutes. Samples were taken at 0, 5, 15, 30, and 60 minutes, pelleted and kept on ice until being simultaneously processed with the FastRNA Blue isolation kit. Total RNA (5 µg) was resolved through a 1.2% agarose-0.66 M formaldehyde gel in MOPS (morpholine propanesulfonic acid) running buffer (Sigma). Blotting of RNA onto Hybond N+ membranes (Amersham) was performed with Turboblotter alkaline transfer systems (Schleicher &amp; Schuell). For detection of <em>mecA</em> specific transcripts, a DNA probe was constructed by PCR amplification with primers (5′→3′): <em>mecA</em> P1, <span class="gene">AAA TCG ATG TAA AGG TTG GC</span> and <em>mecA</em> P2, <span class="gene">GTT CTG CAG TAC CGG ATT TG</span>. After purification the probe was labeled with a Ready To Go labeling kit (Amersham) by using [a-<sup>32</sup>P]dCTP (Amersham) and was hybridized under high-stringency conditions. The blots were subsequently washed and autoradiographed.</p>
</div>

<div id="section8" class="section toc-section"><a id="s2h" name="s2h" class="link-target" title="Electrophoretic mobility shift assay"></a>
<h3>Electrophoretic mobility shift assay</h3>
<a id="article1.body1.sec2.sec8.p1" name="article1.body1.sec2.sec8.p1" class="link-target"></a><p>Expression and purification of MecI and BlaI N-terminal histidine tag fusions (His-MecI and His-BlaI, respectively) was performed in native conditions for 200 ml induced cultures using a Ni-NTA matrix (Qiagen), as recommended by the manufacturer. The purification procedure and recombinant protein purity was evaluated by SDS-PAGE analysis. The concentrations of purified His-MecI and His-BlaI proteins were estimated using the BCA Protein Assay Kit (Pierce), as recommended by the manufacturer. For the electrophoretic mobility shift assay we used the chemiluminescent-based DIG Gel Shift Kit (Roche), following the manufacturer's recommendations. As DNA target we used a c.a. 200 bp fragment encompassing the <em>mecA</em> promoter and operator sequences from prototype strain COL obtained by PCR amplification with primers (5′→3′): <em>mecA</em> PF1, <span class="gene">ATA TCG TGA GCA ATG AAC TG</span> (forward) and <em>mecA</em> PR1, <span class="gene">TAT ATA CCA AAC CCG ACA AC</span> (reverse).</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="Characterization of the prototype MRSA strains"></a>
<h3>Characterization of the prototype MRSA strains</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>The resistance-level to oxacillin of the prototype MRSA strains listed in <a href="#pone-0023287-t001">Table 1</a> was checked with the oxacillin E-test. All strains were classified as fully resistant to oxacillin: MIC≥32 µg/ml. The presence of inducible and functional expression of β-lactamase was also checked through the hydrolysis of the chromogenic substrate nitrocefin for overnight cultures grown in the presence or absence of an inducer (oxacillin at sub-MIC concentration of 0.5 µg/ml). All strains in <a href="#pone-0023287-t001">Table 1</a>, except the reference strain COL, were positive for β-lactamase and its expression was found to be inducible in the presence of oxacillin, suggesting a complete and functional <em>bla</em> locus (<em>blaI-blaR1-blaZ</em>).</p>
<a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>The <em>mecI</em> sequence was determined for a total of 11 previously characterized strains positive for the <em>mecI-mecR1 locus</em> <a href="#pone.0023287-Oliveira1" class="ref-tip">[22]</a>: three classified as ST5, SCC<em>mec</em> type II (including strains N315 and BK2464 listed in <a href="#pone-0023287-t001">Table 1</a>) and eight classified as ST239, SCC<em>mec</em> type III (including strains HU25 and BK2421 listed in <a href="#pone-0023287-t001">Table 1</a>). A conserved point mutation was found in all SCC<em>mec</em> type III strains. The mutation introduced a premature stop codon at Gln<sub>68</sub> (CAA→TAA) originating a truncated repressor protein (<em>mecI</em>*). Among the SCC<em>mec</em> type II strains, a point mutation (GGAG→GGA<span class="underline">A</span>) in the <em>mecI</em> ribosomal binding site (RBS) was detected in one strain (BK2464). No mutations were found within the <em>mecI</em> coding region for the two other SCC<em>mec</em> type II strains. A total of 10 previously characterized strains <a href="#pone.0023287-Oliveira1" class="ref-tip">[22]</a>, including all strains listed in <a href="#pone-0023287-t001">Table 1</a>, were also sequenced for the <em>mecA</em> promoter region. Except for one mutation in the position −5 detected in a ST5, SCC<em>mec</em> type II strain, no point mutations were found in the promoter sequence of the <em>mecA</em> gene, when compared to the published sequence for the prototype strain N315 <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>.</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>The induction profile of <em>mecA</em> transcription was checked by northern blotting analysis upon exposure to oxacillin for three prototype strains - PER34, HU25 and N315 - representing different SCC<em>mec</em> types and, therefore, with functional and non-functional <em>mecI-</em>mecR1 regulatory locus (<a href="#pone-0023287-t001">Table 1</a> and <a href="#pone-0023287-g001">Figure 1</a>). The three strains tested were shown to have an inducible <em>mecA</em> expression independently of being positive or negative for a functional copy of the <em>mecI-mecR1</em> locus although there was a remarkable difference in the induction efficiencies: strains PER34 and HU25 (<em>mecI</em> negative and truncated <em>mecI</em>, respectively) showed a complete induction of <em>mecA</em> after 15 minutes, whereas for strain N315 (wild-type sequence for <em>mecI-mecR1</em>) the induction was not complete even after 60 minutes of induction. Since the three strains were positive for the β-lactamase locus, these observations confirm that <em>mecA</em> transcription can in fact be under the control of the <em>blaI-blaR1</em> regulatory <em>locus</em> in clinical MRSA strains and are also in agreement with the previous reported differences of induction efficiency of <em>mecA</em> between the <em>mecI-mecR1</em> and the <em>blaI-blaR1</em> systems.</p>
<a class="link-target" id="pone-0023287-g001" name="pone-0023287-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0023287.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0023287.g001" data-doi="info:doi/10.1371/journal.pone.0023287" data-uri="info:doi/10.1371/journal.pone.0023287.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0023287.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Northern blot analysis of the <em>mecA</em> induction profile in three prototype MRSA strains.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p>Induction of <em>mecA</em> transcription for three prototype strains upon exposure to oxacillin at 0′, 5′, 15′, 30′, and 60′. Strain relevant characteristics are as follows: strain PER34 - SCC<em>mec</em> type I, <em>mecI</em> negative, Δ<em>mecR1</em>; strain HU25 - SCC<em>mec</em> type III, <em>mecI</em>*, <em>mecR1</em><sup>WT</sup>; strain N315 - SCC<em>mec</em> type II, <em>mecI</em><sup>WT</sup><em>mecR1</em><sup>WT</sup>. The three strains are β-lactamase positive.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0023287.g001">
              https://doi.org/10.1371/journal.pone.0023287.g001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="Overexpression of the mecA repressor in prototype MRSA strains"></a>
<h3>Overexpression of the <em>mecA</em> repressor in prototype MRSA strains</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>From the above characterization of the prototype MRSA strains listed in <a href="#pone-0023287-t001">Table 1</a>, we were able to justify the high-level oxacillin resistance for strains carrying SCC<em>mec</em> types I and III. These were found to have no functional <em>mecI-mecR1</em> regulatory loci either due to the characteristic presence of an insertion sequence (SCC<em>mec</em> type I) or due to the conserved non-sense mutation within the coding region of <em>mecI</em> (SCC<em>mec</em> type III) identified in this study. However, for SCC<em>mec</em> type II strains we could not explain the high-resistance phenotype since, except for the point mutation in the <em>mecI</em> RBS of strain BK2464, no consistent alteration within the <em>mecI</em> coding region or <em>mecA</em> promoter sequence was detected.</p>
<a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>To explore these puzzling observations, we have overexpressed <em>in trans</em> the <em>mecA</em> repressor in the prototype MRSA strains and compared the phenotypic expression of β-lactam resistance between the parental and transformed strains. For this purpose, all strains listed in <a href="#pone-0023287-t001">Table 1</a> were transformed with a high-copy plasmid containing a wild-type copy of <em>mecI</em> of strain N315 (pGC2-<em>mecI</em>). Strain COL transformed with pGC2-mecI showed a dramatic decrease of the oxacillin resistance level according to the PAP profile (see <a href="#pone-0023287-g002">Figure 2</a>, panel A) and the MIC as determined by E-test dropped from &gt;256 to 1.5 µg/ml. Strains N315 and BK2464 (SCC<em>mec</em> type II) showed a slight decrease of the oxacillin resistance level according to the PAP profiles (see <a href="#pone-0023287-g002">Figure 2</a>, panel C for PAP profile of strain BK2464), although by E-test there was no detectable decrease for strain BK2464 (MIC&gt;256 µg/ml for both parental and recombinant strains) and only a slight decrease for strain N315 (from 32 to 16 µg/ml). Strains PER34 and HPV107 (SCC<em>mec</em> type I) and HU25 and BK2421 (SCC<em>mec</em> type III) showed no significant alteration of the oxacillin resistance phenotypic expression profile (see <a href="#pone-0023287-g002">Figure 2</a>, panels B and D, for PAP profiles of strains PER34 and HU25, respectively) nor in the MIC as determined by the E-test (MIC&gt;256 µg/ml for all cases).</p>
<a class="link-target" id="pone-0023287-g002" name="pone-0023287-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0023287.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0023287.g002" data-doi="info:doi/10.1371/journal.pone.0023287" data-uri="info:doi/10.1371/journal.pone.0023287.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0023287.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Population analysis profiles of representative parental strains and recombinant strains.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>Panel A – strain COL and its recombinants strains COL+pGC2-<em>mecI</em><sup>WT</sup>, COL+pGC2-<em>mecI</em>*, and COL+pGC2-<em>mecImecR1</em> grown overnight with and without oxacillin before plating onto TSA plates. Panel B – strain PER34 (SCC<em>mec</em> type I) and PER34+pGC2-<em>mecI</em><sup>WT</sup>. Panel C – strain BK2464 (SCC<em>mec</em> type II) and BK2464+pGC2-<em>mecI</em><sup>WT</sup>. Panel D – strain HU25 (SCC<em>mec</em> type III) and HU25+pGC2-<em>mecI</em><sup>WT</sup>.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0023287.g002">
              https://doi.org/10.1371/journal.pone.0023287.g002</a></p></div><a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3" class="link-target"></a><p>A control experiment was done with the mutated <em>mecI</em> sequence found in SCC<em>mec</em> type III strains (<em>mecI*</em>). The cloning strategy was exactly the same used for obtaining the recombinant plasmid pGC2-<em>mecI</em>. When strain COL was transformed with pGC2-<em>mecI*</em> no significant alteration of the oxacillin resistance level was detected – see <a href="#pone-0023287-g002">Figure 2</a>, Panel A. Moreover, the northern blot analysis of the <em>mecA</em> transcription showed no differences between COL and COL+pGC2-<em>mecI</em>*, whereas no <em>mecA</em> transcript could be detected for COL+pGC2-<em>mecI</em> – see <a href="#pone-0023287-g003">Figure 3</a>, lanes 1–3.</p>
<a class="link-target" id="pone-0023287-g003" name="pone-0023287-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0023287.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0023287.g003" data-doi="info:doi/10.1371/journal.pone.0023287" data-uri="info:doi/10.1371/journal.pone.0023287.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0023287.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> Northern blotting analysis of <em>mecA</em> transcription in COL transformants.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1" class="link-target"></a><p>Lane 1, parental strain COL; lane 2, COL+pGC2-<em>mecI</em><sup>WT</sup>; lane 3, COL+pGC2-<em>mecI</em>*; lane 4, COL+pGC2-<em>mecImecR1</em> uninduced; lane 5, COL+pGC2-<em>mecImecR1</em> induced with oxacillin.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0023287.g003">
              https://doi.org/10.1371/journal.pone.0023287.g003</a></p></div><a id="article1.body1.sec3.sec2.p4" name="article1.body1.sec3.sec2.p4" class="link-target"></a><p>Since strain COL is negative for the <em>mecI</em> and has a truncated <em>mecR1</em>, the phenotype observed for COL+pGC2-<em>mecI</em> could be explained by the lack of an inducer of the <em>mecA</em> transcription able to release the <em>mecI</em>-mediated repression. To test this hypothesis, we have transformed COL with a recombinant plasmid containing the wild-type sequences for both <em>mecImecR1</em> (pGC2-<em>mecImecR1</em>). Although by northern blotting analysis the <em>mecA</em> transcription was restored after overnight growth in the presence of oxacillin (see <a href="#pone-0023287-g003">Figure 3</a>, lanes 4–5), suggesting a fully functional <em>mecI-mecR1</em> regulatory system, no significant increase on the oxacillin MIC was detected when compared to COL+pGC2-<em>mecI</em>. Moreover, when grown in liquid culture, this recombinant strain exhibited an atypical aggregation of cellular mass in solution (data not shown), suggesting a cellular toxic effect due to the overexpression of MecR1.</p>
</div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="Overexpression of mecI in an extended collection of representative MRSA strains"></a>
<h3>Overexpression of <em>mecI</em> in an extended collection of representative MRSA strains</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>In order to confirm the previous observations, all strains listed in <a href="#pone-0023287-t002">Table 2</a> were also transformed with the high-copy plasmid containing the wild-type <em>mecI</em> (pGC2-<em>mecI</em>). This collection includes four additional SCC<em>mec</em> type II strains, 13 SCC<em>mec</em> type IV strains, and isolates for SCC<em>mec</em> types V and VI. SCC<em>mec</em> types IV–VI are characterized by deletions in the <em>mecImecR1</em> locus due to the presence of insertion sequences. Once again, we could not detect significant alterations in the phenotypic expression of oxacillin resistance upon the overexpression of the <em>mecA</em> repressor, except for one strain negative for the nitrocefin assay (VNG17). However, for another nitrocefin negative strain (RJP17) the resistance phenotype was stable. The absence of β-lactamase was confirmed for these two strains by PCR amplification of internal fragments of <em>blaZ</em>, <em>blaI</em> and <em>blaR1</em>. No amplification signals were detected. DNA sequencing of the <em>mecA</em> promoter region for both strains revealed no mutations in the operator sequences but a point mutation in the <em>mecA</em> ribosomal-binding site was identified by comparison to the wild-type sequence of strain N315 (GGA GGA→GGA G<span class="underline">T</span>A).</p>
</div>

<div id="section4" class="section toc-section"><a id="s3d" name="s3d" class="link-target" title="Relative affinity of MecI and BlaI for the mecA promoter sequence"></a>
<h3>Relative affinity of MecI and BlaI for the <em>mecA</em> promoter sequence</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>In an attempt to explain the lack of effect on the resistance phenotype upon the overexpression of <em>mecI</em>, we have evaluated the <em>in vitro</em> relative binding affinity of MecI and BlaI repressors to the <em>mecA</em> operator sequences by an electrophoretic mobility shift assay (EMSA). For this purpose, we have expressed in <em>E. coli</em> N-terminal histidine-tag fusions to the wild-type protein sequences of prototype strain N315 (His-MecI and His-BlaI) and evaluated the binding of purified proteins to a DNA fragment containing the <em>mecA</em> promoter sequences of prototype strain COL. As illustrated in <a href="#pone-0023287-g004">Figure 4</a>, by using equivalent concentrations of the purified MecI and BlaI repressors, lower amounts of recombinant MecI protein are required to induce an electrophoretic shift of the DNA fragment containing the <em>mecA</em> promoter; i.e., suggesting that the cognate repressor of <em>mecA</em> has an increased affinity for its promoter sequences.</p>
<a class="link-target" id="pone-0023287-g004" name="pone-0023287-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0023287.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0023287.g004" data-doi="info:doi/10.1371/journal.pone.0023287" data-uri="info:doi/10.1371/journal.pone.0023287.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0023287.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0023287.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0023287.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0023287.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Electrophoretic mobility shift assay of relative MecI and BlaI affinities for the <em>mecA</em> promoter sequence.</div><p class="caption_target"><a id="article1.body1.sec3.sec4.fig1.caption1.p1" name="article1.body1.sec3.sec4.fig1.caption1.p1" class="link-target"></a><p>Binding reaction was performed with a labeled 200 bp DNA fragment encompassing the <em>mecA</em> promoter mixed with increasing amounts of purified MecI (panel A) and BlaI (panel B). Lanes are as follows: lane 1 – no protein (control); lane 2 – 0,001 µg; lane 3 – 0,01 µg; lane 4 – 0,05 µg; lane 5 – 0,1 µg; lane 6 – 0,25 µg; lane 7 – 0,5 µg; lane 8 – 1 µg; lane 9 – 0,1 ug of protein with a 125 molar excess of unlabelled DNA (specific competition).</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0023287.g004">
              https://doi.org/10.1371/journal.pone.0023287.g004</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>The transcription of <em>mecA</em>, the gene responsible for the “broad-spectrum” β-lactam resistance in <em>S. aureus</em>, may be controlled by two regulatory systems: <em>mecI-mecR1</em> and <em>blaI-blaR1</em>. Due to the extremely slow induction of <em>mecA</em> expression mediated by <em>mecI-mecR1</em> regulators, it is believed that the high-level β-lactam resistance characteristic of many contemporary clinical MRSA strains requires a non-functional <em>mecI-mecR1</em> regulatory locus <a href="#pone.0023287-Hiramatsu1" class="ref-tip">[5]</a>, <a href="#pone.0023287-KuwaharaArai1" class="ref-tip">[9]</a>.</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>In this study we aimed to understand how the <em>mecA</em> transcription is regulated in contemporary pandemic MRSA strains, which in many cases express full β-lactam resistance. Based on previous studies by us and many others on the global epidemiology of MRSA, which included the characterization of the SCC<em>mec</em> element, we could, based on the current model of the <em>mecA</em> regulation, justify the potential for the expression of high-level β-lactam resistance in many MRSA clones; i.e. all those clones characterized by SCC<em>mec</em> types I and IV–VII, which have no-functional <em>mecI-mecR1</em> genes due to the presence of insertion sequences. However, for MRSA clones characterized by SCC<em>mec</em> types II, III and VIII, which carry a complete <em>mecI-mecR1</em> region, we could not explain the high-level β-lactam resistance phenotype. In addition, as illustrated among the MRSA strains selected for this study, no clear relationship exists between SCC<em>mec</em> type (i.e. <em>mecI-mecR1</em> functionality) and the level of resistance. For instance, low-level resistance is detected among SCC<em>mec</em> type IV strains that have no functional <em>mecI-mecR1</em> locus.</p>
<a id="article1.body1.sec4.p3" name="article1.body1.sec4.p3" class="link-target"></a><p>Upon DNA sequencing of the <em>mecI</em> coding region, we found a conserved non-sense mutation at Gln68 in all SCC<em>mec</em> type III strains tested. This mutation leads to a truncated MecI protein of 67 a.a. (instead of the 123 a.a. of the wild-type). This mutation appears to be conserved in the ST239-III lineage since it has also been detected in all isolates of a diverse international collection of c.a. 60 ST239-III strains whose genomic sequences have been fully determined <a href="#pone.0023287-Harris1" class="ref-tip">[39]</a>. In addition, this very same mutation has been detected in a cluster of 12 clinical MRSA strains and found to be statistically associated to an increase in the <em>mecA</em> transcription, suggesting a non-functional MecI protein <a href="#pone.0023287-Rosato1" class="ref-tip">[40]</a>. In our northern blotting experiments, overexpression of this mutated <em>mecI</em> failed to repress the <em>mecA</em> transcription on the prototype strain COL that is characterized by a constitutive expression of the resistance determinant. Altogether, these observations suggest that this conserved non-functional version of <em>mecI</em> accounts for the high-level resistance phenotype of SCC<em>mec</em> type III strains.</p>
<a id="article1.body1.sec4.p4" name="article1.body1.sec4.p4" class="link-target"></a><p>Concerning SCC<em>mec</em> type II strains, we detected in strain BK2464 a mutation within the ribosomal binding site (RBS) of <em>mecI</em>, from GGAG to GGA<span class="underline">A</span>. This mutation has also been previously detected among clinical MRSA strains and showed to be statistically associated to a decrease of <em>mecA</em> repression <a href="#pone.0023287-Rosato1" class="ref-tip">[40]</a>, which could explain high-levels of β-lactam resistance. However, since we could not detect this mutation in the other two SCC<em>mec</em> type II strains analyzed, neither in all genomic and SCC<em>mec</em> sequences of type II strains available at GenBank (e.g. strains MRSA252, Mu3, Mu50, JH1, JH9, etc.), it seems that this alteration of the <em>mecI</em> RBS <em>per se</em> does not justify the β-lactam resistance phenotype of SCC<em>mec</em> type II strains. Rosato <em>et al</em> have also noticed that the <em>mecI</em> RBS differences could not entirely explain the difference in <em>mecA</em> transcription between isolates <a href="#pone.0023287-Rosato1" class="ref-tip">[40]</a>. Regarding the <em>mecA</em> promoter, we identified a point mutation found in one single strain in the position −5, which theoretically may affect the binding of MecI <a href="#pone.0023287-Sharma1" class="ref-tip">[41]</a>. However, this mutation was not detected either in the other SCC<em>mec</em> type II strains or in the available genomic sequences and, as such, it also does not justify the resistance phenotype of SCC<em>mec</em> type II strains. As we have not sequenced <em>mecR1</em>, the hypothesis that the inducer is mutated and able to induce <em>mecA</em> transcription more efficiently cannot be excluded. Nevertheless, this hypothesis seems unlikely since no mutations within the <em>mecR1</em> coding sequence were detected among genomic sequences available at Genbank, which include SCC<em>mec</em> type II and fully oxacillin-resistant strains. In addition, in a large study which addressed the allelic variation of <em>mecA</em> regulators only two silent mutations were found within the <em>mecR1</em> coding sequence <a href="#pone.0023287-Shukla1" class="ref-tip">[25]</a>.</p>
<a id="article1.body1.sec4.p5" name="article1.body1.sec4.p5" class="link-target"></a><p>In order to further explore the puzzling observations described above, we have cloned the wild-type sequence of <em>mecI</em> from the reference strain N315 in a high-copy number plasmid with strong bacteriophage promoters flanking the cloning site and we have introduced this recombinant plasmid into prototype and representative MRSA strains. By over-expressing the repressor <em>in trans</em>, according to the current model for the <em>mecA</em> transcriptional control, we were expecting to see a significant decrease in the resistance level to oxacillin, particularly for strains naturally negative for <em>mecI</em> (i.e. containing SCC<em>mec</em> types I and IV–VII). To our surprise, except for strains COL and VNG17, the resistant phenotype to oxacillin showed no significant changes in all strains. Since we could see an effect in strains COL and VNG17, we were confident about the functionality of the cloned <em>mecI</em>. Nevertheless, we have also transformed COL with the truncated <em>mecI</em> found in SCC<em>mec</em> type III strains cloned into the same plasmid using exactly the same cloning strategy. As expected, the overexpression of the truncated <em>mecI</em> failed to cause any phenotypic alterations. Northern blotting analysis of <em>mecA</em> in COL and its transformants also showed that only the wild-type <em>mecI</em> was able to repress the <em>mecA</em> constitutive transcription of the parental strain. Because COL has no functional <em>mecR1</em>, the phenotype observed could be explained by the lack of an inducer able to release the <em>mecI-</em>mediated repression on <em>mecA</em>, although this would also apply to all other SCC<em>mec</em> types I and IV–VI tested for which there was no effect upon the overexpression of <em>mecI</em> (except for strain VNG17). Nevertheless, we have transformed COL with a recombinant plasmid containing the full wild-type <em>mecI-mecR1</em> system. Interestingly, although we could see induction of <em>mecA</em> transcription in the presence of oxacillin, the resistance phenotype of the parental strain could not be restored. Moreover, the transformed strain presented clear physiological perturbations when grown in liquid medium, which suggests that the overexpression of MecR1, a trans-membranar protein, might be toxic for the cell, as previously observed <a href="#pone.0023287-Rosato2" class="ref-tip">[42]</a>.</p>
<a id="article1.body1.sec4.p6" name="article1.body1.sec4.p6" class="link-target"></a><p>The two strains for which the <em>mecI</em> overexpression caused a decrease in the oxacillin-resistance level (COL and VNG17) were negative for the β-lactamase locus, suggesting that this <em>locus</em> might be responsible for the observed “resistance” of MRSA strains to the <em>mecI</em> overexpression. Indeed, it has been shown that <em>blaI-blaR1</em> can efficiently repress and induce <em>mecA</em> transcription <a href="#pone.0023287-Hackbarth1" class="ref-tip">[6]</a>, <a href="#pone.0023287-Ryffel1" class="ref-tip">[7]</a>; we also confirmed these observations by northern blotting analysis of the <em>mecA</em> induction profile in three prototype strains (<a href="#pone-0023287-g001">Figure 1</a>). However, two lines of evidence may reject this hypothesis. First, in another β-lactamase negative strain (RJP17), belonging to the same clone of VNG17 and isolated in the same country and time period, we could not detect significant alterations in the oxacillin resistance phenotype upon the overexpression of <em>mecI</em>. Second, although <em>blaI-blaR1</em> can efficiently induce <em>mecA</em> transcription, the <em>blaI-blaR1</em> and <em>mecI-mecR1</em> systems are not inter-changeable; i.e. BlaR1 is not able to release the MecI-mediated repression on <em>mecA</em> <a href="#pone.0023287-McKinney1" class="ref-tip">[8]</a>. That is to say, there is no evidence so far supporting the hypothesis that the <em>blaI-blaR1</em> system is able to out-compete the MecI-mediated repression on <em>mecA</em>, which could eventually account for the lack of effect on the resistance phenotype observed in our overexpression experiments. Moreover, in our experimental system the <em>blaI-blaR1</em> expressed from its native promoter would have to out-compete MecI expressed constitutively from a strong bacteriophage promoter.</p>
<a id="article1.body1.sec4.p7" name="article1.body1.sec4.p7" class="link-target"></a><p>Inspired by the fact that the <em>mecI-mecR1</em> and <em>blaI-blaR1</em> systems differ remarkably in the induction efficiency of <em>mecA</em>, we have tested the hypothesis that our observations could be explained by differences in the relative affinities of MecI and BlaI for the <em>mecA</em> promoter sequences. We have addressed this hypothesis <em>in vitro</em> through an electrophoretic mobility shift assay, which clearly showed that lower concentrations of purified MecI are required for detecting a binding to the DNA fragment containing the <em>mecA</em> promoter sequences, suggesting an increased affinity when compared to BlaI. Therefore, our observations could not be explained by an increased affinity of BlaR1 for the <em>mecA</em> promoter, which eventually would “protect” against the increased cellular amounts of MecI. Actually, this hypothesis would make little sense if one takes into account that the induction process involves the proteolysis of BlaR1 and BlaI and that the sustained expression of the resistance gene requires the continuous expression of both regulatory proteins from their common promoter <a href="#pone.0023287-Zhang1" class="ref-tip">[43]</a>. In our experimental system, this signal transduction cascade would have to out-compete the binding of MecI (overexpressed constitutively) to the <em>mecA</em> promoter.</p>
<a id="article1.body1.sec4.p8" name="article1.body1.sec4.p8" class="link-target"></a><p>In short, this study has shown that unexpectedly the β-lactam resistance phenotype of MRSA strains is not affected by the overexpression <em>in trans</em> of the <em>mecA</em> repressor, even in strains negative for or with non-functional <em>mecI</em> gene. This puzzling observation, besides contradicting the current model for the <em>mecA</em> transcriptional control in contemporary MRSA strains, strongly suggests that other yet unidentified determinants are involved directly or indirectly in the transcriptional control of the <em>mecA</em> gene and, consequently in the phenotypic expression of β-lactam resistance. Elucidation of the nature of these determinants is under way and will be of paramount clinical relevance, since the full understanding of the molecular mechanisms controlling the phenotypic expression of the “broad-spectrum” β-lactam resistance in clinical MRSA strains may contribute to the design of new therapeutic strategies, which may extend the clinical utility of β-lactams.</p>
</div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We thank A. Tomasz for all the encouragement and stimulating discussions. We thank E. Feil for checking the ST239 genomic sequences for the Gln68 mutation in <em>mecI</em>. We also thank T. Ito and W.B. Grubb for having kindly provided some prototype MRSA strains used in this study.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: DCO. Performed the experiments: DCO. Analyzed the data: DCO HdL. Contributed reagents/materials/analysis tools: DCO HdL. Wrote the paper: DCO HdL.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0023287-Chambers1" id="pone.0023287-Chambers1" class="link-target"></a>Chambers HF, Deleo FR (2009) Waves of resistance: Staphylococcus aureus in the antibiotic era. Nat Rev Microbiol  7: 629–641.  <ul class="reflinks"><li><a href="#" data-author="Chambers" data-cit="ChambersHFDeleoFR2009Waves%20of%20resistance%3A%20Staphylococcus%20aureus%20in%20the%20antibiotic%20era.Nat%20Rev%20Microbiol7629641" data-title="Waves%20of%20resistance%3A%20Staphylococcus%20aureus%20in%20the%20antibiotic%20era." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Waves+of+resistance%3A+Staphylococcus+aureus+in+the+antibiotic+era.+Chambers+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0023287-Song1" id="pone.0023287-Song1" class="link-target"></a>Song MD, Wachi M, Doi M, Ishino F, Matsuhashi M (1987) Evolution of an inducible penicillin-target protein in methicillin-resistant Staphylococcus aureus by gene fusion. FEBS Lett  221: 167–171.  <ul class="reflinks"><li><a href="#" data-author="Song" data-cit="SongMDWachiMDoiMIshinoFMatsuhashiM1987Evolution%20of%20an%20inducible%20penicillin-target%20protein%20in%20methicillin-resistant%20Staphylococcus%20aureus%20by%20gene%20fusion.FEBS%20Lett221167171" data-title="Evolution%20of%20an%20inducible%20penicillin-target%20protein%20in%20methicillin-resistant%20Staphylococcus%20aureus%20by%20gene%20fusion." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Evolution+of+an+inducible+penicillin-target+protein+in+methicillin-resistant+Staphylococcus+aureus+by+gene+fusion.+Song+1987" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0023287-Hartman1" id="pone.0023287-Hartman1" class="link-target"></a>Hartman B, Tomasz A (1981) Altered penicillin-binding proteins in methicillin-resistant strains of Staphylococcus aureus. Antimicrob Agents Chemother  19: 726–735.  <ul class="reflinks"><li><a href="#" data-author="Hartman" data-cit="HartmanBTomaszA1981Altered%20penicillin-binding%20proteins%20in%20methicillin-resistant%20strains%20of%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother19726735" data-title="Altered%20penicillin-binding%20proteins%20in%20methicillin-resistant%20strains%20of%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Altered+penicillin-binding+proteins+in+methicillin-resistant+strains+of+Staphylococcus+aureus.+Hartman+1981" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0023287-Ito1" id="pone.0023287-Ito1" class="link-target"></a>Ito T, Katayama Y, Hiramatsu K (1999) Cloning and nucleotide sequence determination of the entire mec DNA of pre-methicillin-resistant Staphylococcus aureus N315. Antimicrob Agents Chemother  43: 1449–1458.  <ul class="reflinks"><li><a href="#" data-author="Ito" data-cit="ItoTKatayamaYHiramatsuK1999Cloning%20and%20nucleotide%20sequence%20determination%20of%20the%20entire%20mec%20DNA%20of%20pre-methicillin-resistant%20Staphylococcus%20aureus%20N315.Antimicrob%20Agents%20Chemother4314491458" data-title="Cloning%20and%20nucleotide%20sequence%20determination%20of%20the%20entire%20mec%20DNA%20of%20pre-methicillin-resistant%20Staphylococcus%20aureus%20N315." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cloning+and+nucleotide+sequence+determination+of+the+entire+mec+DNA+of+pre-methicillin-resistant+Staphylococcus+aureus+N315.+Ito+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0023287-Hiramatsu1" id="pone.0023287-Hiramatsu1" class="link-target"></a>Hiramatsu K, Asada K, Suzuki E, Okonogi K, Yokota T (1992) Molecular cloning and nucleotide sequence determination of the regulator region of mecA gene in methicillin-resistant Staphylococcus aureus (MRSA). FEBS Lett  298: 133–136.  <ul class="reflinks"><li><a href="#" data-author="Hiramatsu" data-cit="HiramatsuKAsadaKSuzukiEOkonogiKYokotaT1992Molecular%20cloning%20and%20nucleotide%20sequence%20determination%20of%20the%20regulator%20region%20of%20mecA%20gene%20in%20methicillin-resistant%20Staphylococcus%20aureus%20%28MRSA%29.FEBS%20Lett298133136" data-title="Molecular%20cloning%20and%20nucleotide%20sequence%20determination%20of%20the%20regulator%20region%20of%20mecA%20gene%20in%20methicillin-resistant%20Staphylococcus%20aureus%20%28MRSA%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+cloning+and+nucleotide+sequence+determination+of+the+regulator+region+of+mecA+gene+in+methicillin-resistant+Staphylococcus+aureus+%28MRSA%29.+Hiramatsu+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0023287-Hackbarth1" id="pone.0023287-Hackbarth1" class="link-target"></a>Hackbarth CJ, Chambers HF (1993) blaI and blaR1 regulate beta-lactamase and PBP 2a production in methicillin-resistant Staphylococcus aureus. Antimicrob Agents Chemother  37: 1144–1149.  <ul class="reflinks"><li><a href="#" data-author="Hackbarth" data-cit="HackbarthCJChambersHF1993blaI%20and%20blaR1%20regulate%20beta-lactamase%20and%20PBP%202a%20production%20in%20methicillin-resistant%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother3711441149" data-title="blaI%20and%20blaR1%20regulate%20beta-lactamase%20and%20PBP%202a%20production%20in%20methicillin-resistant%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=blaI+and+blaR1+regulate+beta-lactamase+and+PBP+2a+production+in+methicillin-resistant+Staphylococcus+aureus.+Hackbarth+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0023287-Ryffel1" id="pone.0023287-Ryffel1" class="link-target"></a>Ryffel C, Kayser FH, Berger-Bachi B (1992) Correlation between regulation of mecA transcription and expression of methicillin resistance in staphylococci. Antimicrob Agents Chemother  36: 25–31.  <ul class="reflinks"><li><a href="#" data-author="Ryffel" data-cit="RyffelCKayserFHBerger-BachiB1992Correlation%20between%20regulation%20of%20mecA%20transcription%20and%20expression%20of%20methicillin%20resistance%20in%20staphylococci.Antimicrob%20Agents%20Chemother362531" data-title="Correlation%20between%20regulation%20of%20mecA%20transcription%20and%20expression%20of%20methicillin%20resistance%20in%20staphylococci." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Correlation+between+regulation+of+mecA+transcription+and+expression+of+methicillin+resistance+in+staphylococci.+Ryffel+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0023287-McKinney1" id="pone.0023287-McKinney1" class="link-target"></a>McKinney TK, Sharma VK, Craig WA, Archer GL (2001) Transcription of the gene mediating methicillin resistance in Staphylococcus aureus (mecA) is corepressed but not coinduced by cognate mecA and beta-lactamase regulators. J Bacteriol  183: 6862–6868.  <ul class="reflinks"><li><a href="#" data-author="McKinney" data-cit="McKinneyTKSharmaVKCraigWAArcherGL2001Transcription%20of%20the%20gene%20mediating%20methicillin%20resistance%20in%20Staphylococcus%20aureus%20%28mecA%29%20is%20corepressed%20but%20not%20coinduced%20by%20cognate%20mecA%20and%20beta-lactamase%20regulators.J%20Bacteriol18368626868" data-title="Transcription%20of%20the%20gene%20mediating%20methicillin%20resistance%20in%20Staphylococcus%20aureus%20%28mecA%29%20is%20corepressed%20but%20not%20coinduced%20by%20cognate%20mecA%20and%20beta-lactamase%20regulators." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcription+of+the+gene+mediating+methicillin+resistance+in+Staphylococcus+aureus+%28mecA%29+is+corepressed+but+not+coinduced+by+cognate+mecA+and+beta-lactamase+regulators.+McKinney+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0023287-KuwaharaArai1" id="pone.0023287-KuwaharaArai1" class="link-target"></a>Kuwahara-Arai K, Kondo N, Hori S, Tateda-Suzuki E, Hiramatsu K (1996) Suppression of methicillin resistance in a mecA-containing pre-methicillin-resistant Staphylococcus aureus strain is caused by the mecI-mediated repression of PBP 2′ production. Antimicrob Agents Chemother  40: 2680–2685.  <ul class="reflinks"><li><a href="#" data-author="Kuwahara-Arai" data-cit="Kuwahara-AraiKKondoNHoriSTateda-SuzukiEHiramatsuK1996Suppression%20of%20methicillin%20resistance%20in%20a%20mecA-containing%20pre-methicillin-resistant%20Staphylococcus%20aureus%20strain%20is%20caused%20by%20the%20mecI-mediated%20repression%20of%20PBP%202%E2%80%B2%20production.Antimicrob%20Agents%20Chemother4026802685" data-title="Suppression%20of%20methicillin%20resistance%20in%20a%20mecA-containing%20pre-methicillin-resistant%20Staphylococcus%20aureus%20strain%20is%20caused%20by%20the%20mecI-mediated%20repression%20of%20PBP%202%E2%80%B2%20production." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Suppression+of+methicillin+resistance+in+a+mecA-containing+pre-methicillin-resistant+Staphylococcus+aureus+strain+is+caused+by+the+mecI-mediated+repression+of+PBP+2%E2%80%B2+production.+Kuwahara-Arai+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0023287-Dickinson1" id="pone.0023287-Dickinson1" class="link-target"></a>Dickinson TM, Archer GL (2000) Phenotypic expression of oxacillin resistance in Staphylococcus epidermidis: roles of mecA transcriptional regulation and resistant-subpopulation selection. Antimicrob Agents Chemother  44: 1616–1623.  <ul class="reflinks"><li><a href="#" data-author="Dickinson" data-cit="DickinsonTMArcherGL2000Phenotypic%20expression%20of%20oxacillin%20resistance%20in%20Staphylococcus%20epidermidis%3A%20roles%20of%20mecA%20transcriptional%20regulation%20and%20resistant-subpopulation%20selection.Antimicrob%20Agents%20Chemother4416161623" data-title="Phenotypic%20expression%20of%20oxacillin%20resistance%20in%20Staphylococcus%20epidermidis%3A%20roles%20of%20mecA%20transcriptional%20regulation%20and%20resistant-subpopulation%20selection." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phenotypic+expression+of+oxacillin+resistance+in+Staphylococcus+epidermidis%3A+roles+of+mecA+transcriptional+regulation+and+resistant-subpopulation+selection.+Dickinson+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0023287-Archer1" id="pone.0023287-Archer1" class="link-target"></a>Archer GL, Niemeyer DM, Thanassi JA, Pucci MJ (1994) Dissemination among staphylococci of DNA sequences associated with methicillin resistance. Antimicrob Agents Chemother  38: 447–454.  <ul class="reflinks"><li><a href="#" data-author="Archer" data-cit="ArcherGLNiemeyerDMThanassiJAPucciMJ1994Dissemination%20among%20staphylococci%20of%20DNA%20sequences%20associated%20with%20methicillin%20resistance.Antimicrob%20Agents%20Chemother38447454" data-title="Dissemination%20among%20staphylococci%20of%20DNA%20sequences%20associated%20with%20methicillin%20resistance." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dissemination+among+staphylococci+of+DNA+sequences+associated+with+methicillin+resistance.+Archer+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0023287-HurlimannDalel1" id="pone.0023287-HurlimannDalel1" class="link-target"></a>Hurlimann-Dalel RL, Ryffel C, Kayser FH, Berger-Bachi B (1992) Survey of the methicillin resistance-associated genes mecA, mecR1-mecI, and femA-femB in clinical isolates of methicillin-resistant Staphylococcus aureus. Antimicrob Agents Chemother  36: 2617–2621.  <ul class="reflinks"><li><a href="#" data-author="Hurlimann-Dalel" data-cit="Hurlimann-DalelRLRyffelCKayserFHBerger-BachiB1992Survey%20of%20the%20methicillin%20resistance-associated%20genes%20mecA%2C%20mecR1-mecI%2C%20and%20femA-femB%20in%20clinical%20isolates%20of%20methicillin-resistant%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother3626172621" data-title="Survey%20of%20the%20methicillin%20resistance-associated%20genes%20mecA%2C%20mecR1-mecI%2C%20and%20femA-femB%20in%20clinical%20isolates%20of%20methicillin-resistant%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Survey+of+the+methicillin+resistance-associated+genes+mecA%2C+mecR1-mecI%2C+and+femA-femB+in+clinical+isolates+of+methicillin-resistant+Staphylococcus+aureus.+Hurlimann-Dalel+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0023287-Kobayashi1" id="pone.0023287-Kobayashi1" class="link-target"></a>Kobayashi N, Taniguchi K, Urasawa S (1998) Analysis of diversity of mutations in the mecI gene and mecA promoter/operator region of methicillin-resistant Staphylococcus aureus and Staphylococcus epidermidis. Antimicrob Agents Chemother  42: 717–720.  <ul class="reflinks"><li><a href="#" data-author="Kobayashi" data-cit="KobayashiNTaniguchiKUrasawaS1998Analysis%20of%20diversity%20of%20mutations%20in%20the%20mecI%20gene%20and%20mecA%20promoter%2Foperator%20region%20of%20methicillin-resistant%20Staphylococcus%20aureus%20and%20Staphylococcus%20epidermidis.Antimicrob%20Agents%20Chemother42717720" data-title="Analysis%20of%20diversity%20of%20mutations%20in%20the%20mecI%20gene%20and%20mecA%20promoter%2Foperator%20region%20of%20methicillin-resistant%20Staphylococcus%20aureus%20and%20Staphylococcus%20epidermidis." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+of+diversity+of+mutations+in+the+mecI+gene+and+mecA+promoter%2Foperator+region+of+methicillin-resistant+Staphylococcus+aureus+and+Staphylococcus+epidermidis.+Kobayashi+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0023287-Suzuki1" id="pone.0023287-Suzuki1" class="link-target"></a>Suzuki E, Kuwahara-Arai K, Richardson JF, Hiramatsu K (1993) Distribution of mec regulator genes in methicillin-resistant Staphylococcus clinical strains. Antimicrob Agents Chemother  37: 1219–1226.  <ul class="reflinks"><li><a href="#" data-author="Suzuki" data-cit="SuzukiEKuwahara-AraiKRichardsonJFHiramatsuK1993Distribution%20of%20mec%20regulator%20genes%20in%20methicillin-resistant%20Staphylococcus%20clinical%20strains.Antimicrob%20Agents%20Chemother3712191226" data-title="Distribution%20of%20mec%20regulator%20genes%20in%20methicillin-resistant%20Staphylococcus%20clinical%20strains." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Distribution+of+mec+regulator+genes+in+methicillin-resistant+Staphylococcus+clinical+strains.+Suzuki+1993" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0023287-Weller1" id="pone.0023287-Weller1" class="link-target"></a>Weller TM (1999) The distribution of mecA, mecR1 and mecI and sequence analysis of mecI and the mec promoter region in staphylococci expressing resistance to methicillin. J Antimicrob Chemother  43: 15–22.  <ul class="reflinks"><li><a href="#" data-author="Weller" data-cit="WellerTM1999The%20distribution%20of%20mecA%2C%20mecR1%20and%20mecI%20and%20sequence%20analysis%20of%20mecI%20and%20the%20mec%20promoter%20region%20in%20staphylococci%20expressing%20resistance%20to%20methicillin.J%20Antimicrob%20Chemother431522" data-title="The%20distribution%20of%20mecA%2C%20mecR1%20and%20mecI%20and%20sequence%20analysis%20of%20mecI%20and%20the%20mec%20promoter%20region%20in%20staphylococci%20expressing%20resistance%20to%20methicillin." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+distribution+of+mecA%2C+mecR1+and+mecI+and+sequence+analysis+of+mecI+and+the+mec+promoter+region+in+staphylococci+expressing+resistance+to+methicillin.+Weller+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0023287-Hartman2" id="pone.0023287-Hartman2" class="link-target"></a>Hartman BJ, Tomasz A (1986) Expression of methicillin resistance in heterogeneous strains of Staphylococcus aureus. Antimicrob Agents Chemother  29: 85–92.  <ul class="reflinks"><li><a href="#" data-author="Hartman" data-cit="HartmanBJTomaszA1986Expression%20of%20methicillin%20resistance%20in%20heterogeneous%20strains%20of%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother298592" data-title="Expression%20of%20methicillin%20resistance%20in%20heterogeneous%20strains%20of%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Expression+of+methicillin+resistance+in+heterogeneous+strains+of+Staphylococcus+aureus.+Hartman+1986" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0023287-Murakami1" id="pone.0023287-Murakami1" class="link-target"></a>Murakami K, Tomasz A (1989) Involvement of multiple genetic determinants in high-level methicillin resistance in Staphylococcus aureus. J Bacteriol  171: 874–879.  <ul class="reflinks"><li><a href="#" data-author="Murakami" data-cit="MurakamiKTomaszA1989Involvement%20of%20multiple%20genetic%20determinants%20in%20high-level%20methicillin%20resistance%20in%20Staphylococcus%20aureus.J%20Bacteriol171874879" data-title="Involvement%20of%20multiple%20genetic%20determinants%20in%20high-level%20methicillin%20resistance%20in%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Involvement+of+multiple+genetic+determinants+in+high-level+methicillin+resistance+in+Staphylococcus+aureus.+Murakami+1989" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0023287-Archer2" id="pone.0023287-Archer2" class="link-target"></a>Archer GL, Bosilevac JM (2001) Signaling antibiotic resistance in staphylococci. Science  291: 1915–1916.  <ul class="reflinks"><li><a href="#" data-author="Archer" data-cit="ArcherGLBosilevacJM2001Signaling%20antibiotic%20resistance%20in%20staphylococci.Science29119151916" data-title="Signaling%20antibiotic%20resistance%20in%20staphylococci." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Signaling+antibiotic+resistance+in+staphylococci.+Archer+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0023287-MallorquFernndez1" id="pone.0023287-MallorquFernndez1" class="link-target"></a>Mallorquí-Fernández G, Marrero A, García-Piquè S, García-Castellanos R, Gomis-Rüth FX (2004) Staphylococcal methicillin resistance: fine focus on folds and functions. FEMS Microbiol Lett  235: 1–8.  <ul class="reflinks"><li><a href="#" data-author="Mallorqu%C3%AD-Fern%C3%A1ndez" data-cit="Mallorqu%C3%AD-Fern%C3%A1ndezGMarreroAGarc%C3%ADa-Piqu%C3%A8SGarc%C3%ADa-CastellanosRGomis-R%C3%BCthFX2004Staphylococcal%20methicillin%20resistance%3A%20fine%20focus%20on%20folds%20and%20functions.FEMS%20Microbiol%20Lett23518" data-title="Staphylococcal%20methicillin%20resistance%3A%20fine%20focus%20on%20folds%20and%20functions." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Staphylococcal+methicillin+resistance%3A+fine+focus+on+folds+and+functions.+Mallorqu%C3%AD-Fern%C3%A1ndez+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0023287-GarcaCastellanos1" id="pone.0023287-GarcaCastellanos1" class="link-target"></a>García-Castellanos R, Mallorquí-Fernández G, Marrero A, Potempa J, Coll M, et al.  (2004) On the transcriptional regulation of methicillin resistance: MecI repressor in complex with its operator. J Biol Chem  279: 17888–17896.  <ul class="reflinks"><li><a href="#" data-author="Garc%C3%ADa-Castellanos" data-cit="Garc%C3%ADa-CastellanosRMallorqu%C3%AD-Fern%C3%A1ndezGMarreroAPotempaJCollM2004On%20the%20transcriptional%20regulation%20of%20methicillin%20resistance%3A%20MecI%20repressor%20in%20complex%20with%20its%20operator.J%20Biol%20Chem2791788817896" data-title="On%20the%20transcriptional%20regulation%20of%20methicillin%20resistance%3A%20MecI%20repressor%20in%20complex%20with%20its%20operator." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=On+the+transcriptional+regulation+of+methicillin+resistance%3A+MecI+repressor+in+complex+with+its+operator.+Garc%C3%ADa-Castellanos+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0023287-Cohen1" id="pone.0023287-Cohen1" class="link-target"></a>Cohen S, Sweeney HM (1968) Constitutive penicillinase formation in Staphylococcus aureus owing to a mutation unlinked to the penicillinase plasmid. J Bacteriol  95: 1368–1374.  <ul class="reflinks"><li><a href="#" data-author="Cohen" data-cit="CohenSSweeneyHM1968Constitutive%20penicillinase%20formation%20in%20Staphylococcus%20aureus%20owing%20to%20a%20mutation%20unlinked%20to%20the%20penicillinase%20plasmid.J%20Bacteriol9513681374" data-title="Constitutive%20penicillinase%20formation%20in%20Staphylococcus%20aureus%20owing%20to%20a%20mutation%20unlinked%20to%20the%20penicillinase%20plasmid." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Constitutive+penicillinase+formation+in+Staphylococcus+aureus+owing+to+a+mutation+unlinked+to+the+penicillinase+plasmid.+Cohen+1968" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0023287-Oliveira1" id="pone.0023287-Oliveira1" class="link-target"></a>Oliveira DC, Tomasz A, de Lencastre H (2001) The evolution of pandemic clones of methicillin resistant Staphylococcus aureus: identification of two ancestral genetic backgrounds and the associated mec elements. Microb Drug Resist  7: 349–361.  <ul class="reflinks"><li><a href="#" data-author="Oliveira" data-cit="OliveiraDCTomaszAde%20LencastreH2001The%20evolution%20of%20pandemic%20clones%20of%20methicillin%20resistant%20Staphylococcus%20aureus%3A%20identification%20of%20two%20ancestral%20genetic%20backgrounds%20and%20the%20associated%20mec%20elements.Microb%20Drug%20Resist7349361" data-title="The%20evolution%20of%20pandemic%20clones%20of%20methicillin%20resistant%20Staphylococcus%20aureus%3A%20identification%20of%20two%20ancestral%20genetic%20backgrounds%20and%20the%20associated%20mec%20elements." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+evolution+of+pandemic+clones+of+methicillin+resistant+Staphylococcus+aureus%3A+identification+of+two+ancestral+genetic+backgrounds+and+the+associated+mec+elements.+Oliveira+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0023287-Enright1" id="pone.0023287-Enright1" class="link-target"></a>Enright MC, Robinson DA, Randle G, Feil EJ, Grundmann H, et al.  (2002) The evolutionary history of methicillin-resistant Staphylococcus aureus (MRSA). Proc Natl Acad Sci U S A  99: 7687–7692.  <ul class="reflinks"><li><a href="#" data-author="Enright" data-cit="EnrightMCRobinsonDARandleGFeilEJGrundmannH2002The%20evolutionary%20history%20of%20methicillin-resistant%20Staphylococcus%20aureus%20%28MRSA%29.Proc%20Natl%20Acad%20Sci%20U%20S%20A9976877692" data-title="The%20evolutionary%20history%20of%20methicillin-resistant%20Staphylococcus%20aureus%20%28MRSA%29." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+evolutionary+history+of+methicillin-resistant+Staphylococcus+aureus+%28MRSA%29.+Enright+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0023287-IWGSCC1" id="pone.0023287-IWGSCC1" class="link-target"></a>IWG-SCC (2009) Classification of staphylococcal cassette chromosome mec (SCCmec): guidelines for reporting novel SCCmec elements. Antimicrob Agents Chemother  53: 4961–4967.  <ul class="reflinks"><li><a href="#" data-author="" data-cit="IWG-SCC2009Classification%20of%20staphylococcal%20cassette%20chromosome%20mec%20%28SCCmec%29%3A%20guidelines%20for%20reporting%20novel%20SCCmec%20elements.Antimicrob%20Agents%20Chemother5349614967" data-title="Classification%20of%20staphylococcal%20cassette%20chromosome%20mec%20%28SCCmec%29%3A%20guidelines%20for%20reporting%20novel%20SCCmec%20elements." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Classification+of+staphylococcal+cassette+chromosome+mec+%28SCCmec%29%3A+guidelines+for+reporting+novel+SCCmec+elements.++2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0023287-Shukla1" id="pone.0023287-Shukla1" class="link-target"></a>Shukla SK, Ramaswamy SV, Conradt J, Stemper ME, Reich R, et al.  (2004) Novel Polymorphisms in mec Genes and a New mec Complex Type in Methicillin-Resistant Staphylococcus aureus Isolates Obtained in Rural Wisconsin. Antimicrob Agents Chemother  48: 3080–3085.  <ul class="reflinks"><li><a href="#" data-author="Shukla" data-cit="ShuklaSKRamaswamySVConradtJStemperMEReichR2004Novel%20Polymorphisms%20in%20mec%20Genes%20and%20a%20New%20mec%20Complex%20Type%20in%20Methicillin-Resistant%20Staphylococcus%20aureus%20Isolates%20Obtained%20in%20Rural%20Wisconsin.Antimicrob%20Agents%20Chemother4830803085" data-title="Novel%20Polymorphisms%20in%20mec%20Genes%20and%20a%20New%20mec%20Complex%20Type%20in%20Methicillin-Resistant%20Staphylococcus%20aureus%20Isolates%20Obtained%20in%20Rural%20Wisconsin." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Novel+Polymorphisms+in+mec+Genes+and+a+New+mec+Complex+Type+in+Methicillin-Resistant+Staphylococcus+aureus+Isolates+Obtained+in+Rural+Wisconsin.+Shukla+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0023287-deLencastre1" id="pone.0023287-deLencastre1" class="link-target"></a>de Lencastre H, Tomasz A (1994) Reassessment of the number of auxiliary genes essential for expression of high-level methicillin resistance in Staphylococcus aureus. Antimicrob Agents Chemother  38: 2590–2598.  <ul class="reflinks"><li><a href="#" data-author="de%20Lencastre" data-cit="de%20LencastreHTomaszA1994Reassessment%20of%20the%20number%20of%20auxiliary%20genes%20essential%20for%20expression%20of%20high-level%20methicillin%20resistance%20in%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother3825902598" data-title="Reassessment%20of%20the%20number%20of%20auxiliary%20genes%20essential%20for%20expression%20of%20high-level%20methicillin%20resistance%20in%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Reassessment+of+the+number+of+auxiliary+genes+essential+for+expression+of+high-level+methicillin+resistance+in+Staphylococcus+aureus.+de+Lencastre+1994" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0023287-deLencastre2" id="pone.0023287-deLencastre2" class="link-target"></a>de Lencastre H, Wu SW, Pinho MG, Ludovice AM, Filipe S, et al.  (1999) Antibiotic resistance as a stress response: complete sequencing of a large number of chromosomal loci in Staphylococcus aureus strain COL that impact on the expression of resistance to methicillin. Microb Drug Resist  5: 163–175.  <ul class="reflinks"><li><a href="#" data-author="de%20Lencastre" data-cit="de%20LencastreHWuSWPinhoMGLudoviceAMFilipeS1999Antibiotic%20resistance%20as%20a%20stress%20response%3A%20complete%20sequencing%20of%20a%20large%20number%20of%20chromosomal%20loci%20in%20Staphylococcus%20aureus%20strain%20COL%20that%20impact%20on%20the%20expression%20of%20resistance%20to%20methicillin.Microb%20Drug%20Resist5163175" data-title="Antibiotic%20resistance%20as%20a%20stress%20response%3A%20complete%20sequencing%20of%20a%20large%20number%20of%20chromosomal%20loci%20in%20Staphylococcus%20aureus%20strain%20COL%20that%20impact%20on%20the%20expression%20of%20resistance%20to%20methicillin." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Antibiotic+resistance+as+a+stress+response%3A+complete+sequencing+of+a+large+number+of+chromosomal+loci+in+Staphylococcus+aureus+strain+COL+that+impact+on+the+expression+of+resistance+to+methicillin.+de+Lencastre+1999" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0023287-Milheirico1" id="pone.0023287-Milheirico1" class="link-target"></a>Milheirico C, Oliveira DC, de Lencastre H (2007) Multiplex PCR strategy for subtyping the staphylococcal cassette chromosome mec type IV in methicillin-resistant Staphylococcus aureus: ‘SCCmec IV multiplex’. J Antimicrob Chemother  60: 42–48.  <ul class="reflinks"><li><a href="#" data-author="Milheirico" data-cit="MilheiricoCOliveiraDCde%20LencastreH2007Multiplex%20PCR%20strategy%20for%20subtyping%20the%20staphylococcal%20cassette%20chromosome%20mec%20type%20IV%20in%20methicillin-resistant%20Staphylococcus%20aureus%3A%20%E2%80%98SCCmec%20IV%20multiplex%E2%80%99.J%20Antimicrob%20Chemother604248" data-title="Multiplex%20PCR%20strategy%20for%20subtyping%20the%20staphylococcal%20cassette%20chromosome%20mec%20type%20IV%20in%20methicillin-resistant%20Staphylococcus%20aureus%3A%20%E2%80%98SCCmec%20IV%20multiplex%E2%80%99." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multiplex+PCR+strategy+for+subtyping+the+staphylococcal+cassette+chromosome+mec+type+IV+in+methicillin-resistant+Staphylococcus+aureus%3A+%E2%80%98SCCmec+IV+multiplex%E2%80%99.+Milheirico+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0023287-Milheirico2" id="pone.0023287-Milheirico2" class="link-target"></a>Milheirico C, Oliveira DC, de Lencastre H (2007) Update to the multiplex PCR strategy for assignment of mec element types in Staphylococcus aureus. Antimicrob Agents Chemother  51: 3374–3377.  <ul class="reflinks"><li><a href="#" data-author="Milheirico" data-cit="MilheiricoCOliveiraDCde%20LencastreH2007Update%20to%20the%20multiplex%20PCR%20strategy%20for%20assignment%20of%20mec%20element%20types%20in%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother5133743377" data-title="Update%20to%20the%20multiplex%20PCR%20strategy%20for%20assignment%20of%20mec%20element%20types%20in%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Update+to+the+multiplex+PCR+strategy+for+assignment+of+mec+element+types+in+Staphylococcus+aureus.+Milheirico+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0023287-Oliveira2" id="pone.0023287-Oliveira2" class="link-target"></a>Oliveira DC, de Lencastre H (2002) Multiplex PCR strategy for rapid identification of structural types and variants of the mec element in methicillin resistant isolates of Staphylococcus aureus. Antimicrob Agents Chemother  46: 2155–2161.  <ul class="reflinks"><li><a href="#" data-author="Oliveira" data-cit="OliveiraDCde%20LencastreH2002Multiplex%20PCR%20strategy%20for%20rapid%20identification%20of%20structural%20types%20and%20variants%20of%20the%20mec%20element%20in%20methicillin%20resistant%20isolates%20of%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother4621552161" data-title="Multiplex%20PCR%20strategy%20for%20rapid%20identification%20of%20structural%20types%20and%20variants%20of%20the%20mec%20element%20in%20methicillin%20resistant%20isolates%20of%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multiplex+PCR+strategy+for+rapid+identification+of+structural+types+and+variants+of+the+mec+element+in+methicillin+resistant+isolates+of+Staphylococcus+aureus.+Oliveira+2002" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0023287-Ausubel1" id="pone.0023287-Ausubel1" class="link-target"></a>Ausubel FM, Brent R, Kingston RE, Moore DD, Seidman JG, et al.  (1992) Short protocols in Molecular Biology. New York: Green Publishing Associates and John Wiley &amp; Sons.   <ul class="find-nolinks"></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0023287-Sambrook1" id="pone.0023287-Sambrook1" class="link-target"></a>Sambrook JE, Fritsch EF, Maniatis T (1989) Molecular cloning: a laboratory manual. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory.   <ul class="find-nolinks"></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0023287-deLencastre3" id="pone.0023287-deLencastre3" class="link-target"></a>de Lencastre H, Sa Figueiredo AM, Urban C, Rahal J, Tomasz A (1991) Multiple mechanisms of methicillin resistance and improved methods for detection in clinical isolates of Staphylococcus aureus. Antimicrob Agents Chemother  35: 632–639.  <ul class="reflinks"><li><a href="#" data-author="de%20Lencastre" data-cit="de%20LencastreHSa%20FigueiredoAMUrbanCRahalJTomaszA1991Multiple%20mechanisms%20of%20methicillin%20resistance%20and%20improved%20methods%20for%20detection%20in%20clinical%20isolates%20of%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother35632639" data-title="Multiple%20mechanisms%20of%20methicillin%20resistance%20and%20improved%20methods%20for%20detection%20in%20clinical%20isolates%20of%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Multiple+mechanisms+of+methicillin+resistance+and+improved+methods+for+detection+in+clinical+isolates+of+Staphylococcus+aureus.+de+Lencastre+1991" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0023287-Jett1" id="pone.0023287-Jett1" class="link-target"></a>Jett BD, Hatter KL, Huycke MM, Gilmore MS (1997) Simplified agar plate method for quantifying viable bacteria. Biotechniques  23: 648–650.  <ul class="reflinks"><li><a href="#" data-author="Jett" data-cit="JettBDHatterKLHuyckeMMGilmoreMS1997Simplified%20agar%20plate%20method%20for%20quantifying%20viable%20bacteria.Biotechniques23648650" data-title="Simplified%20agar%20plate%20method%20for%20quantifying%20viable%20bacteria." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Simplified+agar+plate+method+for+quantifying+viable+bacteria.+Jett+1997" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0023287-Pinho1" id="pone.0023287-Pinho1" class="link-target"></a>Pinho MG, de Lencastre H, Tomasz A (1998) Transcriptional analysis of the Staphylococcus aureus penicillin binding protein 2 gene. J Bacteriol  180: 6077–6081.  <ul class="reflinks"><li><a href="#" data-author="Pinho" data-cit="PinhoMGde%20LencastreHTomaszA1998Transcriptional%20analysis%20of%20the%20Staphylococcus%20aureus%20penicillin%20binding%20protein%202%20gene.J%20Bacteriol18060776081" data-title="Transcriptional%20analysis%20of%20the%20Staphylococcus%20aureus%20penicillin%20binding%20protein%202%20gene." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptional+analysis+of+the+Staphylococcus+aureus+penicillin+binding+protein+2+gene.+Pinho+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0023287-Ito2" id="pone.0023287-Ito2" class="link-target"></a>Ito T, Katayama Y, Asada K, Mori N, Tsutsumimoto K, et al.  (2001) Structural comparison of three types of staphylococcal cassette chromosome mec integrated in the chromosome in methicillin-resistant Staphylococcus aureus. Antimicrob Agents Chemother  45: 1323–1336.  <ul class="reflinks"><li><a href="#" data-author="Ito" data-cit="ItoTKatayamaYAsadaKMoriNTsutsumimotoK2001Structural%20comparison%20of%20three%20types%20of%20staphylococcal%20cassette%20chromosome%20mec%20integrated%20in%20the%20chromosome%20in%20methicillin-resistant%20Staphylococcus%20aureus.Antimicrob%20Agents%20Chemother4513231336" data-title="Structural%20comparison%20of%20three%20types%20of%20staphylococcal%20cassette%20chromosome%20mec%20integrated%20in%20the%20chromosome%20in%20methicillin-resistant%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Structural+comparison+of+three+types+of+staphylococcal+cassette+chromosome+mec+integrated+in+the+chromosome+in+methicillin-resistant+Staphylococcus+aureus.+Ito+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0023287-Kraemer1" id="pone.0023287-Kraemer1" class="link-target"></a>Kraemer GR, Iandolo JJ (1990) High-frequency transformation of Staphylococcus aureus by electroporation. Current Microbiology  21: 373–376.  <ul class="reflinks"><li><a href="#" data-author="Kraemer" data-cit="KraemerGRIandoloJJ1990High-frequency%20transformation%20of%20Staphylococcus%20aureus%20by%20electroporation.Current%20Microbiology21373376" data-title="High-frequency%20transformation%20of%20Staphylococcus%20aureus%20by%20electroporation." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=High-frequency+transformation+of+Staphylococcus+aureus+by+electroporation.+Kraemer+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0023287-Oshida1" id="pone.0023287-Oshida1" class="link-target"></a>Oshida T, Tomasz A (1992) Isolation and characterization of a Tn551-autolysis mutant of Staphylococcus aureus. J Bacteriol  174: 4952–4959.  <ul class="reflinks"><li><a href="#" data-author="Oshida" data-cit="OshidaTTomaszA1992Isolation%20and%20characterization%20of%20a%20Tn551-autolysis%20mutant%20of%20Staphylococcus%20aureus.J%20Bacteriol17449524959" data-title="Isolation%20and%20characterization%20of%20a%20Tn551-autolysis%20mutant%20of%20Staphylococcus%20aureus." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Isolation+and+characterization+of+a+Tn551-autolysis+mutant+of+Staphylococcus+aureus.+Oshida+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0023287-Harris1" id="pone.0023287-Harris1" class="link-target"></a>Harris SR, Feil EJ, Holden MTG, Quail MA, Nickerson EK, et al.  (2010) Evolution of MRSA during hospital transmission and intercontinental spread. Science  327: 469–474.  <ul class="reflinks"><li><a href="#" data-author="Harris" data-cit="HarrisSRFeilEJHoldenMTGQuailMANickersonEK2010Evolution%20of%20MRSA%20during%20hospital%20transmission%20and%20intercontinental%20spread.Science327469474" data-title="Evolution%20of%20MRSA%20during%20hospital%20transmission%20and%20intercontinental%20spread." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Evolution+of+MRSA+during+hospital+transmission+and+intercontinental+spread.+Harris+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0023287-Rosato1" id="pone.0023287-Rosato1" class="link-target"></a>Rosato AE, Craig WA, Archer GL (2003) Quantitation of mecA Transcription in Oxacillin-Resistant Staphylococcus aureus Clinical Isolates. Journal of Bacteriology  185: 3446–3452.  <ul class="reflinks"><li><a href="#" data-author="Rosato" data-cit="RosatoAECraigWAArcherGL2003Quantitation%20of%20mecA%20Transcription%20in%20Oxacillin-Resistant%20Staphylococcus%20aureus%20Clinical%20Isolates.Journal%20of%20Bacteriology18534463452" data-title="Quantitation%20of%20mecA%20Transcription%20in%20Oxacillin-Resistant%20Staphylococcus%20aureus%20Clinical%20Isolates." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Quantitation+of+mecA+Transcription+in+Oxacillin-Resistant+Staphylococcus+aureus+Clinical+Isolates.+Rosato+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref41"><span class="order">41.
            </span><a name="pone.0023287-Sharma1" id="pone.0023287-Sharma1" class="link-target"></a>Sharma VK, Hackbarth CJ, Dickinson TM, Archer GL (1998) Interaction of native and mutant MecI repressors with sequences that regulate mecA, the gene encoding penicillin binding protein 2a in methicillin-resistant staphylococci. J Bacteriol  180: 2160–2166.  <ul class="reflinks"><li><a href="#" data-author="Sharma" data-cit="SharmaVKHackbarthCJDickinsonTMArcherGL1998Interaction%20of%20native%20and%20mutant%20MecI%20repressors%20with%20sequences%20that%20regulate%20mecA%2C%20the%20gene%20encoding%20penicillin%20binding%20protein%202a%20in%20methicillin-resistant%20staphylococci.J%20Bacteriol18021602166" data-title="Interaction%20of%20native%20and%20mutant%20MecI%20repressors%20with%20sequences%20that%20regulate%20mecA%2C%20the%20gene%20encoding%20penicillin%20binding%20protein%202a%20in%20methicillin-resistant%20staphylococci." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Interaction+of+native+and+mutant+MecI+repressors+with+sequences+that+regulate+mecA%2C+the+gene+encoding+penicillin+binding+protein+2a+in+methicillin-resistant+staphylococci.+Sharma+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref42"><span class="order">42.
            </span><a name="pone.0023287-Rosato2" id="pone.0023287-Rosato2" class="link-target"></a>Rosato AE, Kreiswirth BN, Craig WA, Eisner W, Climo MW, et al.  (2003) mecA-blaZ Corepressors in Clinical <em>Staphylococcus aureus</em> Isolates. Antimicrobial Agents and Chemotherapy  47: 1460–1463.  <ul class="reflinks"><li><a href="#" data-author="Rosato" data-cit="RosatoAEKreiswirthBNCraigWAEisnerWClimoMW2003mecA-blaZ%20Corepressors%20in%20Clinical%20Staphylococcus%20aureus%20Isolates.Antimicrobial%20Agents%20and%20Chemotherapy4714601463" data-title="mecA-blaZ%20Corepressors%20in%20Clinical%20Staphylococcus%20aureus%20Isolates." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=mecA-blaZ+Corepressors+in+Clinical+Staphylococcus+aureus+Isolates.+Rosato+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref43"><span class="order">43.
            </span><a name="pone.0023287-Zhang1" id="pone.0023287-Zhang1" class="link-target"></a>Zhang HZ, Hackbarth CJ, Chansky KM, Chambers HF (2001) A proteolytic transmembrane signaling pathway and resistance to beta-lactams in staphylococci. Science  291: 1962–1965.  <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangHZHackbarthCJChanskyKMChambersHF2001A%20proteolytic%20transmembrane%20signaling%20pathway%20and%20resistance%20to%20beta-lactams%20in%20staphylococci.Science29119621965" data-title="A%20proteolytic%20transmembrane%20signaling%20pathway%20and%20resistance%20to%20beta-lactams%20in%20staphylococci." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=A+proteolytic+transmembrane+signaling+pathway+and+resistance+to+beta-lactams+in+staphylococci.+Zhang+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

