



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>Ribosomal RNA (rRNA) genes, whose activity results in nucleolus formation, constitute an extremely important part of genome. Despite the extensive exploration into avian genomes, no complete description of avian rRNA gene primary structure has been offered so far. We publish a complete chicken rRNA gene cluster sequence here, including <em>5’ETS</em> (1836 bp), <em>18S rRNA</em> gene (1823 bp), <em>ITS1</em> (2530 bp), <em>5</em>.<em>8S rRNA</em> gene (157 bp), <em>ITS2</em> (733 bp), <em>28S rRNA</em> gene (4441 bp) and <em>3’ETS</em> (343 bp). The rRNA gene cluster sequence of 11863 bp was assembled from raw reads and deposited to GenBank under KT445934 accession number. The assembly was validated through <em>in situ</em> fluorescent hybridization analysis on chicken metaphase chromosomes using computed and synthesized specific probes, as well as through the reference assembly against <em>de novo</em> assembled rRNA gene cluster sequence using sequenced fragments of BAC-clone containing chicken NOR (nucleolus organizer region). The results have confirmed the chicken rRNA gene cluster validity.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Dyomin AG, Koshel EI, Kiselev AM, Saifitdinova AF, Galkina SA, Fukagawa T, et al.  (2016) Chicken rRNA Gene Cluster Structure. PLoS ONE 11(6):
           e0157464.
        
        https://doi.org/10.1371/journal.pone.0157464</p><p><strong>Editor: </strong>Emanuele Buratti, International Centre for Genetic Engineering and Biotechnology, ITALY</p><p><strong>Received: </strong>March 10, 2016; <strong>Accepted: </strong>May 31, 2016; <strong>Published: </strong> June 14, 2016</p><p><strong>Copyright: </strong> © 2016 Dyomin et al. This is an open access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Data Availability: </strong>NGS data are available from NCBI, SRA accession numbers—DRX001863, SRP069763. rRNA gene cluster sequence is available from NCBI, Nucleotide accession number—KT445934. All other relevant data are within the paper and its Supporting Information files.</p><p><strong>Funding: </strong>The research has been financed by 1. Saint-Petersburg State University Grant #1.50.1043. 2014 (AD EK) <a href="http://english.spbu.ru/">http://english.spbu.ru/</a>; 2. Russian Basic Research Foundation Grant #15-04-05684 (EG SG AD EK) <a href="http://www.rfbr.ru/rffi/eng">http://www.rfbr.ru/rffi/eng</a>; 3. RF Government Grant #074-U01 (AnK ArK) <a href="http://government.ru/en/">http://government.ru/en/</a>. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="sec001" name="sec001" data-toc="sec001" class="link-target" title="Background"></a><h2>Background</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Ribosomal gene arrays constitute one of the most important genome components [<a href="#pone.0157464.ref001" class="ref-tip">1</a>]. They form nucleolus organizing regions (NOR) in chromosomes, and the functional status of NOR is an indicator of the physiological status of cells, tissues and the entire organism at various ontogenetic stages [<a href="#pone.0157464.ref001" class="ref-tip">1</a>–<a href="#pone.0157464.ref006" class="ref-tip">6</a>]. In a commonly accepted approach that we follow here, the term “ribosomal gene array” refers to the sequence composed of repeated rRNA gene clusters separated by intergenic spacers (<em>IGS</em>) [<a href="#pone.0157464.ref007" class="ref-tip">7</a>]. All animal rRNA gene clusters are known to have fundamentally similar structures (<a href="#pone-0157464-g001">Fig 1</a>). Each cluster comprises sequences of rRNA encoding conservative genes (<em>18S rRNA</em>, <em>5</em>.<em>8S rRNA</em> and <em>28S rRNA</em>) separated by internal transcribed spacers (<em>ITS1</em> and <em>ITS2</em>), and external transcribed spacers (<em>5’ETS</em> and <em>3’ETS</em>) situated downstream of <em>18S rRNA</em> gene and upstream of <em>28S rRNA</em> gene respectively. All these elements transcribe into a single RNA precursor, pre-rRNA. <em>18S rRNA</em>, <em>5</em>.<em>8S rRNA</em> and <em>28S rRNA</em> genes are quite conserved between organisms. Both internal and external transcribed spacers feature high structural variability, which accounts for ribosome cluster length variation within a wide range from 8 to 14 thousand base pairs (bp), promoter and terminator regions for RNA polymerase I (RNApolI) being located in the <em>IGS</em> [<a href="#pone.0157464.ref007" class="ref-tip">7</a>].</p>
<a class="link-target" id="pone-0157464-g001" name="pone-0157464-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0157464.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0157464.g001" data-doi="info:doi/10.1371/journal.pone.0157464" data-uri="info:doi/10.1371/journal.pone.0157464.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0157464.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 1. </span> Ribosomal DNA structure (after Singer and Berg, 1991).</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0157464.g001">
              https://doi.org/10.1371/journal.pone.0157464.g001</a></p></div><a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>In general, rRNA gene cluster structure is still underinvestigated [<a href="#pone.0157464.ref001" class="ref-tip">1</a>]. Despite recent vertebrate genome-wide extensive studies [<a href="#pone.0157464.ref008" class="ref-tip">8</a>], decoding of NOR structure and rRNA gene clusters remains a complicated task, primarily due to the high repetitivity and extensive length of the clusters as well as a faster spacer sequence evolution [<a href="#pone.0157464.ref001" class="ref-tip">1</a>]. So far GenBank [<a href="#pone.0157464.ref009" class="ref-tip">9</a>] has contained annotated complete rRNA gene cluster sequences only for a limited number of vertebrate species, in particular <em>Homo sapiens</em> (GenBank accession number: U13369), <em>Mus musculus</em> (GenBank accession number: BK000964) and <em>Nothobranchius furzeri</em> (GenBank accession number: EU780557). Yet, no description of the complete rRNA gene cluster sequence has been offered so far for such a major taxon as Aves.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3" class="link-target"></a><p>Chicken rRNA gene cluster deserves special research focus among bird class representatives. Chicken plays an important role in agriculture and it has been widely used in an extensive range of biomedical research, including such areas as developmental biology, genetics, cell biology, histology, virology and so on [<a href="#pone.0157464.ref010" class="ref-tip">10</a>, <a href="#pone.0157464.ref011" class="ref-tip">11</a>]. Chicken genome was the first to have been sequenced in avian and among the first in vertebrata genomes [<a href="#pone.0157464.ref012" class="ref-tip">12</a>]. <em>Gallus gallus 4</em>.<em>0</em> assembly has been used as a reference genome for assembling sequenced genomes of other birds [<a href="#pone.0157464.ref013" class="ref-tip">13</a>]. Chicken NOR location on chromosome 16 (GGA16) [<a href="#pone.0157464.ref014" class="ref-tip">14</a>–<a href="#pone.0157464.ref017" class="ref-tip">17</a>] has been recently confirmed by fluorescent <em>in situ</em> hybridization (FISH) using plasmid <em>ETS</em> [<a href="#pone.0157464.ref018" class="ref-tip">18</a>] and NOR containing WAG137G04 BAC-clone [<a href="#pone.0157464.ref019" class="ref-tip">19</a>] as probes. Noteworthy, only 5% of GGA16 chromosome sequence has been decoded according to the published assembly of <em>Gallus gallus 4</em>.<em>0</em> genome [<a href="#pone.0157464.ref012" class="ref-tip">12</a>], and such decoding does not include the NOR region [<a href="#pone.0157464.ref019" class="ref-tip">19</a>].</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4" class="link-target"></a><p>GenBank contains annotated sequences for individual fragments of chicken <em>18S rRNA</em> and <em>28S rRNA</em> genes. There are also two incompatible sequence variants, both annotated in GenBank as <em>ITS1-5</em>.<em>8S-ITS2</em> (Accession numbers: DQ018752 –DQ018755; FJ008990).</p>
<a id="article1.body1.sec1.p5" name="article1.body1.sec1.p5" class="link-target"></a><p>In the course of “ChIP-sequencing with CENP-A from chicken cells containing neocentromeres on Z chromosome” project [<a href="#pone.0157464.ref020" class="ref-tip">20</a>], we obtained multiple raw chicken sequences. Based on these sequences as well as sequences of unannotated Whole Genome Shotgun (WGS) contigs from <em>Gallus gallus 4</em>.<em>0</em> genome assembly (NCBI WGS accession number: AADN00000000), we assembled and described the complete structure of chicken rRNA gene cluster. The assembled sequence relation to GGA16 NOR was confirmed by serial FISH to chicken mitotic chromosomes. For validation of rRNA gene cluster assembly a NOR-containing BAC clone [<a href="#pone.0157464.ref019" class="ref-tip">19</a>], was sequenced and re-assembled on the basis of the <em>de novo</em> assembled chicken rRNA gene cluster.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec002" name="sec002" data-toc="sec002" class="link-target" title="Material and Methods"></a><h2>Material and Methods</h2>
<div id="section1" class="section toc-section"><a id="sec003" name="sec003" class="link-target" title="SRA data and BAC clone sample"></a>
<h3>SRA data and BAC clone sample</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p>To assemble a complete chicken rRNA gene cluster <em>de novo</em> we used a sequence library [<a href="#pone.0157464.ref020" class="ref-tip">20</a>] which had been earlier generated of raw reads (NCBI SRA accession number: DRX001863) in the course of “ChIP-sequencing with CENP-A from chicken cells containing neocentromeres on Z chromosome” project (NCBI BioProject accession number: PRJDB2279). For assembly verification and adjustment we used 19 unannotated sequences of the WGS contigs from the <em>Gallus gallus 4</em>.<em>0</em> genome assembly (NCBI WGS accession number: AADN00000000.3) [<a href="#pone.0157464.ref012" class="ref-tip">12</a>] and 11 sequences from Nucleotide database [<a href="#pone.0157464.ref021" class="ref-tip">21</a>] annotated by the authors as chicken ribosomal cluster fragments (<a href="#pone-0157464-t001">Table 1</a>). WAG137G04 BAC clone containing chicken NOR was used as a probe to NOR [<a href="#pone.0157464.ref019" class="ref-tip">19</a>].</p>
<a class="link-target" id="pone-0157464-t001" name="pone-0157464-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0157464.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0157464.t001" data-doi="info:doi/10.1371/journal.pone.0157464" data-uri="info:doi/10.1371/journal.pone.0157464.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0157464.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> WGS contigs and annotated sequences used to verify the chicken ribosomal cluster assembly.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0157464.t001">
              https://doi.org/10.1371/journal.pone.0157464.t001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="sec004" name="sec004" class="link-target" title="BAC clone sequencing"></a>
<h3>BAC clone sequencing</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>WAG137G04 BAC clone high-throughput sequencing was performed using Illumina MiSeq sequencing machine and MiSeq Reagent Kit v3 2x300. DNA quality was analyzed using Nanodrop 1000 spectrophotometer (Thermo, USA). DNA was quantified with Quantifluor-ST fluorometer (Promega, USA) and dsDNA kit (Promega, USA). DNA fragment library was prepared from 1 ng of BAC clone DNA using NexteraXT kit (Illumina, USA) according to manufacturer’s guideline. Library quality was estimated using Agilent Bioanalyzer 2100 instrument (Agilent, USA) and RT-PCR according to Illumina Sequencing Library qPCR Quantification Guide.</p>
</div>

<div id="section3" class="section toc-section"><a id="sec005" name="sec005" class="link-target" title="Cluster assembling and annotating"></a>
<h3>Cluster assembling and annotating</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p><em>De novo</em> assembling of chicken rRNA gene cluster sequence from SRA sequence library DRX00186 and searching for WGS contigs homologous to the rRNA gene cluster elements was performed using BLAST [<a href="#pone.0157464.ref024" class="ref-tip">24</a>] and Unipro UGENE v. 1.16.1 software package [<a href="#pone.0157464.ref025" class="ref-tip">25</a>].</p>
<a id="article1.body1.sec2.sec3.p2" name="article1.body1.sec2.sec3.p2" class="link-target"></a><p>Chicken rRNA gene cluster reference assembling was performed on the basis of BAC clone raw read library using Geneious 9.0.5 software package [<a href="#pone.0157464.ref026" class="ref-tip">26</a>]. The <em>de novo</em> assembled sequence was used as a reference. For sequence alignment and nucleotide structure determination Mega 6.06 [<a href="#pone.0157464.ref027" class="ref-tip">27</a>] was applied. Repeat searching and typing were performed in Repeatmasker 4.0.5 [<a href="#pone.0157464.ref028" class="ref-tip">28</a>]. Nucleotide sequence secondary structure was computed in Mfold [<a href="#pone.0157464.ref029" class="ref-tip">29</a>].</p>
<a id="article1.body1.sec2.sec3.p3" name="article1.body1.sec2.sec3.p3" class="link-target"></a><p>To define the boundaries between spacer (<em>5’ETS</em>, <em>ITS1</em>, <em>ITS2</em>, <em>3’ETS</em>) sequences and <em>rRNA</em> gene (<em>18S</em>, <em>5</em>.<em>8S</em>, <em>28S</em>) sequences we used annotated fragments of ribosomal clusters of <em>H</em>. <em>sapiens</em> (GenBank accession number: U13369), <em>Rattus norvegicus</em> (GenBank accession number: NR_046239), <em>M</em>. <em>musculus</em> (GenBank accession number: NR_046233), <em>Xenopus laevis</em> (GenBank accession number: X02995) and <em>Crocodylus porosus</em> (GenBank accession number: EU727191).</p>
</div>

<div id="section4" class="section toc-section"><a id="sec006" name="sec006" class="link-target" title="Fluorescent in situ hybridization"></a>
<h3>Fluorescent <em>in situ</em> hybridization</h3>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1" class="link-target"></a><p>To verify the relation of <em>de novo</em> assembled sequence to the NOR on GGA16 a serial FISH was applied to chicken mitotic chromosomes using the assembled sequence fragments and WAG137G04 BAC clone DNA as probes. Metaphase spreads were obtained from primary chicken embryo cell cultures according to standard procedures. Briefly, a chicken embryo of 4 day incubation was aseptically removed from the egg, washed in PBS and fragmented. Embryo fragmrnts were mechanically macerated and cell suspension has been placed into a tissue culture flask with Dulbecco's modified Eagle's medium (DMEM, Paneco) containing 10% fetal bovine serum (PAA Laboratories), 1% Streptomycin and 1% L-Glutamine. Settled cells were cultured overnight at 37°C in humidified 5% CO<sub>2</sub> atmosphere and arrested with 0.05 μg/ml colcemid (CaryoMax, Invitrogen), followed by hypotonic treatment and fixation. Fertile eggs were provided by the Federal State Unitary research farm “Gene Pool”, Pushkin, Leningrad region, Russia (<a href="http://genofund.narod.ru/index.htm">http://genofund.narod.ru/index.htm</a>). Experimental procedures involving chicken embryos were conducted in accordance with International Guiding Principles for Biomedical Research Involving Animals established by Council for International Organizations of Medical Sciences (CIOMS) and approved by Saint-Petersburg State University Ethics Committee (statement # 131-03-2).</p>
<a id="article1.body1.sec2.sec4.p2" name="article1.body1.sec2.sec4.p2" class="link-target"></a><p>To prepare FISH probes, PCR primers were designed from the <em>de novo</em> assembled rRNA gene cluster using Unipro UGENE 1.16.1 software package; their identity to the related regions of the assembled rRNA gene cluster was validated by standard sequencing. WAG137G04 BAC clone probe was produced by standard DOP-PCR amplification using 6MW primers [<a href="#pone.0157464.ref030" class="ref-tip">30</a>].</p>
<a id="article1.body1.sec2.sec4.p3" name="article1.body1.sec2.sec4.p3" class="link-target"></a><p>PCR amplification of the selected sequence areas was carried out in 20 μl of reaction mix as follows: Taq-pol 5U/μl (Sileks, Russia) – 0,5 μl; 10X Taq Buffer (Sileks, Russia) – 2 μl; MgCl2 25 mM (Sileks, Russia) – 2 μl; 10mM dNTP (2.5 mM each) (Sileks, Russia) – 1.6 μl; 10μM primer (10 pmol/μl) – 1 μl each; DNA – 0,5 μl; H<sub>2</sub>O – 11.4 μl. PCR protocol: 94°С–5’; (94°С–20”, 60°C–15”, 72°C–20”)35 сycles; 72°С–5’; 4°C–hold.</p>
<a id="article1.body1.sec2.sec4.p4" name="article1.body1.sec2.sec4.p4" class="link-target"></a><p>Fluorescent probes were generated by labeling PCR products with modified biotin-16-dUTP nucleotide (Sileks, Russia) during amplification procedure. Biotin was detected with streptavidin-Alexa488 (Invitrogen, USA). FISH was carried out according to a published protocol [<a href="#pone.0157464.ref031" class="ref-tip">31</a>]. The preps were additionally stained with 4’,6-diamidino-2-phenylindole (DAPI) in concentration of 1μg/ml. For establishing whether the fluorescent probe hybridization had taken place at NOR on GGA16, re-FISH with WAG137G04 BAC clone probe was carried out on the same preparations.</p>
<a id="article1.body1.sec2.sec4.p5" name="article1.body1.sec2.sec4.p5" class="link-target"></a><p>FISH results were analyzed using DM4000В (Leica Microsystems CMS, GmbH) epifluorescent microscope. The images were processed using LAS AF software (Leica Microsystems CMS, GmbH) and superposed in Adobe Photoshop CS5.1 (Adobe Systems, Inc, USA).</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="sec007" name="sec007" data-toc="sec007" class="link-target" title="Results and Discussion"></a><h2>Results and Discussion</h2>
<div id="section1" class="section toc-section"><a id="sec008" name="sec008" class="link-target" title="De novo rRNA gene cluster assembly"></a>
<h3><em>De novo</em> rRNA gene cluster assembly</h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>We analyzed all GenBank sequences annotated as chicken rRNA gene cluster elements. Eight of them were completely rejected because of their short lengths or high homology to ribosomal DNA of other organisms (<a href="#pone-0157464-t002">Table 2</a>). Veracity of nine others and related annotations were also under doubt because in GenBank were represented only in unique copies. Among the annotated chicken rDNA sequences, 18S rDNA fragment was only deposited to GenBank in several identical copies. We found neither <em>ITS2/3’ETS</em> nor the major part of <em>28S rRNA</em> gene annotated sequences related to chicken in Genbank.</p>
<a class="link-target" id="pone-0157464-t002" name="pone-0157464-t002"></a><div class="figure" data-doi="10.1371/journal.pone.0157464.t002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0157464.t002" data-doi="info:doi/10.1371/journal.pone.0157464" data-uri="info:doi/10.1371/journal.pone.0157464.t002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0157464.t002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 2. </span> The rejected sequences of chicken rDNA.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0157464.t002">
              https://doi.org/10.1371/journal.pone.0157464.t002</a></p></div><a id="article1.body1.sec3.sec1.p2" name="article1.body1.sec3.sec1.p2" class="link-target"></a><p>We selected the sequences reliably annotated as chicken rRNA gene cluster fragments to identify homologous WGS contigs in GenBank. <em>Gallus gallus</em> WGS project (accession number AADN00000000.3) in total contains 19 contigs. These contigs appeared to belong to chicken rRNA gene cluster. However, in the proposed rRNA gene cluster sequence, contig overlappings were absent at three sites and no contigs containing either <em>28S rRNA</em> gene 3’end or <em>3’ETS</em> sequence were found. The overlapping contig length was 9230 bp. The total length of overlapping annotated sequences and contigs from the transcription initial site was 9931 bp (<a href="#pone.0157464.s010">S1 Table</a>).</p>
<a id="article1.body1.sec3.sec1.p3" name="article1.body1.sec3.sec1.p3" class="link-target"></a><p>The maximum overlapped area was found in a <em>18S rRNA</em> gene sequence of 388 bp assembled from ten homologous sequences (six contigs and four annotated sequences). This 388 bp sequence served as a primary reference for our rRNA gene cluster <em>de novo</em> assembly using raw reads obtained earlier (NCBI SRA accession number DRX001863) [<a href="#pone.0157464.ref020" class="ref-tip">20</a>]. 388 bp DNA fragment homologous reads were identified in BLAST and aligned in Unipro UGENE v. 1.16.1. The hanging ends of the consensus sequence were tiled in Geneious 9.0.5. As a result, we assembled a sequence of 12053 bp, a 11994 bp sequence being upstream of a pre-rRNA transcription initiator site representing a uninterrupted contig.</p>
<a id="article1.body1.sec3.sec1.p4" name="article1.body1.sec3.sec1.p4" class="link-target"></a><p>In total, we used 941443 forward and reverse reads. Sequence mean coverage with reads was 1767 per site.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec009" name="sec009" class="link-target" title="FISH co-localization assembled sequence fragments with NOR"></a>
<h3>FISH co-localization assembled sequence fragments with NOR</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>To verify the assembled sequence relation to the NOR on GGA16, we applied serial re-FISH to chicken mitotic chromosomes. The probes to areas from 1829 to 2024 bp and from 6189 to 6358 bp, presumably associated with <em>5’ETS–18S</em> rDNA and <em>ITS1–5</em>.<em>8S</em> rDNA boundaries respectively within the assembled sequence, were obtained from genome DNA by primer synthesis (<a href="#pone-0157464-t003">Table 3</a>). Probe were 196 bp and 170 bp in length respectively. The primers were computed based on the assembled rRNA gene cluster. Probe adequacy was validated by standard sequencing. The NOR probe was produced by DOP-PCR amplification of WAG137G04 BAC-clone known to contain a chicken GGA16 fragment comprising NOR [<a href="#pone.0157464.ref019" class="ref-tip">19</a>]. On chicken mitotic plates, all probes were hybridized to similar positions on two homologous microchromosomes (<a href="#pone-0157464-g002">Fig 2</a>). Our findings reliably confirm that our assembly strongly matches chicken NOR.</p>
<a class="link-target" id="pone-0157464-g002" name="pone-0157464-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0157464.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0157464.g002" data-doi="info:doi/10.1371/journal.pone.0157464" data-uri="info:doi/10.1371/journal.pone.0157464.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0157464.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 2. </span> Localization of assembled rRNA gene cluster fragments on chicken mitotic chromosomes.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>FISH with (A) <em>5’ETS–18S</em> rDNA fragment probe, (D) <em>ITS1–5</em>.<em>8S</em> rDNA fragment probe, (B, E) WAG137G04 BAC probe, which contains NOR, marker of GGA16, (C, F) merge. A, D–green fluorescence; B, E–red fluorescence; C, F–merge. Chromosomes are counterstained with DAPI (blue). Bar – 5 μm.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0157464.g002">
              https://doi.org/10.1371/journal.pone.0157464.g002</a></p></div><a class="link-target" id="pone-0157464-t003" name="pone-0157464-t003"></a><div class="figure" data-doi="10.1371/journal.pone.0157464.t003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0157464.t003" data-doi="info:doi/10.1371/journal.pone.0157464" data-uri="info:doi/10.1371/journal.pone.0157464.t003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0157464.t003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 3. </span> The list of primers for amplification of target regions of the assembled ribosomal cluster sequence.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0157464.t003">
              https://doi.org/10.1371/journal.pone.0157464.t003</a></p></div></div>

<div id="section3" class="section toc-section"><a id="sec010" name="sec010" class="link-target" title="BAC clone sequencing and reference assembly of rRNA gene cluster"></a>
<h3>BAC clone sequencing and reference assembly of rRNA gene cluster</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>WAG137G04 BAC clone -containing chicken NOR was sequenced using Illumina MiSeq to check whether the assembled sequence was really a chicken rRNA gene cluster. In total, 941780 forward and reverse reads were obtained. The mean read length was 239 bp. The BAC clone raw reads library was submitted to SRA database (GenBank accession number: SRP069763; BioProject accession number PRJNA311164). The <em>de novo</em> assembled rRNA gene cluster was used as a reference sequence. The assembling was carried out in Geneious 9.0.5. 91444 reads were found to be homologous to the reference sequence. Mean reference coverage was 1713. A total of 98.3% of the reference sequence overlapped the reads. The <em>de novo</em> and re-assembled variants of the rRNA gene cluster sequence demonstrate 99.7% identity in the overlap area (<a href="#pone.0157464.s012">S3 Table</a>). Therefore, our results demonstrate that the assembled <em>de novo</em> sequence of 12053 bp represents a chicken rRNA gene cluster.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec011" name="sec011" class="link-target" title="Chicken rRNA gene cluster sequence annotation"></a>
<h3>Chicken rRNA gene cluster sequence annotation</h3>
<a id="article1.body1.sec3.sec4.p1" name="article1.body1.sec3.sec4.p1" class="link-target"></a><p>Chicken rRNA gene cluster sequence was described through comparison with the most completely annotated Tetrapoda rDNA sequences in Genbank: <em>H</em>.<em>sapiens</em>, <em>R</em>.<em>norvegicus</em>, <em>M</em>.<em>musculus</em>, <em>X</em>.<em>laevis</em> and <em>C</em>.<em>porosus</em> (<a href="#pone.0157464.s011">S2 Table</a>).</p>

<div id="section1" class="section toc-section"><a id="sec012" name="sec012" class="link-target" title="5’ETS"></a><h4><em>5’ETS</em>.</h4><a id="article1.body1.sec3.sec4.sec1.p1" name="article1.body1.sec3.sec4.sec1.p1" class="link-target"></a><p>The validity of our chicken rRNA <em>5’ETS</em> assembly was confirmed using seven WGS contigs (<a href="#pone-0157464-t001">Table 1</a>). Additionally, we used a chicken sequence (GenBank accession number: DQ112354) containing the entire RNApolI promoter and a pre-rRNA initiation site [<a href="#pone.0157464.ref011" class="ref-tip">11</a>]. These WGS contigs and annotated sequences overlapped 63.9% and 14.8% of the assembly respectively (<a href="#pone.0157464.s010">S1 Table</a>). Within the designated boundaries (<a href="#pone.0157464.s001">S1 Fig</a>) the length of <em>5’ETS</em> is 1836 bp (from 1 to 1836 bp) (<a href="#pone.0157464.s011">S2 Table</a>). Within the assembled <em>5’ETS</em>, repeating sequences account for 6.4% of its size and are composed of (GCGA)n and (GAGA)n nucleotide combinations and (CGG)n and (GCC)n reverse repeats, the latter being responsible for formation of hairpins related to rRNA processing. The <em>5’ETS</em> sequence was also found to feature increased GC pair content – 75.0%. At the same time, CpG content in chicken <em>5'ETS</em> (15.7%) is comparable to CpG content in <em>5'ETS</em> of human (16.6%) and xenopus (15.5%).</p>
<a id="article1.body1.sec3.sec4.sec1.p2" name="article1.body1.sec3.sec4.sec1.p2" class="link-target"></a><p>Despite significant variation in <em>5'ETS</em> length among these three species (1961, 3656 and 713 bp, respectively) and lack of a clear homology (<a href="#pone.0157464.s011">S2 Table</a>), their CpG content variability does not exceed 1.1%. The CpG dinucleotide content similarity in varying sequences from quite distant animal taxa probably suggests the importance of CpG role in promoter and spacer functions of <em>5'ETS</em>. It might also be indicative of the existence of a stabilizer for spacer nucleotide content.</p>
</div>

<div id="section2" class="section toc-section"><a id="sec013" name="sec013" class="link-target" title="18S rRNA gene"></a><h4><em>18S rRNA</em> gene.</h4><a id="article1.body1.sec3.sec4.sec2.p1" name="article1.body1.sec3.sec4.sec2.p1" class="link-target"></a><p>To confirm the accuracy of our assembly, six WGS contigs and five annotated sequences of fragments of <em>18S rRNA</em> gene were used in addition to raw sequences (<a href="#pone-0157464-t001">Table 1</a>). These WGS contigs and annotated sequences overlapped 100% and 99.4% of the assembly respectively (<a href="#pone.0157464.s010">S1 Table</a>). Within the designated boundaries (<a href="#pone.0157464.s002">S2 Fig</a>) the length of complete <em>18S rRNA</em> sequences was 1823 bp (from 1837 to 3659 bp starting from the transcription initiation site; <a href="#pone.0157464.s011">S2 Table</a>), which is virtually the same as the length of <em>X</em>. <em>laevis 18S rRNA</em> (1825 bp). GC pair content is 54.5% and is also comparable to other taxon data: <em>H</em>. <em>sapiens</em> (56.1%), <em>X</em>. <em>laevis</em> (53.8%), <em>C</em>. <em>porosus</em> (49.5%).</p>
</div>

<div id="section3" class="section toc-section"><a id="sec014" name="sec014" class="link-target" title="ITS1"></a><h4><em>ITS1</em>.</h4><a id="article1.body1.sec3.sec4.sec3.p1" name="article1.body1.sec3.sec4.sec3.p1" class="link-target"></a><p>ITS1 sequence deposited to GenBank by Tang et al. (DQ018754, complete sequence; DQ018752, partial sequence) and Chen et al. (FJ008990, complete sequence) were found to be mismatched. We did define chicken ITS1 structure more precisely through tiling assembly of raw sequences and 5 WGS contigs. WGS contigs overlapped 83% of ITS1 assembly (<a href="#pone.0157464.s010">S1 Table</a>). Within the designated boundaries (<a href="#pone.0157464.s003">S3 Fig</a>) the length of chicken 1TS1 is 2530 bp (from 3660 to 6189 bp) (<a href="#pone.0157464.s011">S2 Table</a>). Our entire ITS1 sequences are homologous to the version offered by Tang et al., with the exception of spacer boundaries and 430 bp insertion in position 4898–5328 bp starting from the transcription initiation site. ITS1 nucleotide content analysis resulted in high GC pair content – 82.2% and high CpG dinucleotide content – 20.1%. Repeats account for 13.4% of ITS1 size and are composed of the following nucleotide combinations: (CCGAGG)n, (CCGGT)n, (GC)n, (GGGGGCC)n, (GAG)n и (GGCGCG)n. High GC pair content in ITS1 sequence leads to formation of multiple hairpin structures. In accordance with the models produced for T – 60°C, [K<sup>+</sup>] – 50 mM PCR conditions, complete hairpin dissociation does not take place (<a href="#pone.0157464.s004">S4 Fig</a>). Under such conditions, the average increment of Gibbs free energy (ΔG) in ITS1 sequence in ssDNA form is – 183.23.</p>
</div>

<div id="section4" class="section toc-section"><a id="sec015" name="sec015" class="link-target" title="5.8S rRNA gene"></a><h4><em>5</em>.<em>8S rRNA</em> gene.</h4><a id="article1.body1.sec3.sec4.sec4.p1" name="article1.body1.sec3.sec4.sec4.p1" class="link-target"></a><p>For assembly validation two WGS contigs and an annotated sequence (GenBank accession number: DQ018754) (<a href="#pone-0157464-t001">Table 1</a>) were used. These WGS contigs and annotated sequences overlapped 100% and 66.2% of the assembly respectively (<a href="#pone.0157464.s010">S1 Table</a>). Upon alignment (<a href="#pone.0157464.s005">S5 Fig</a>) the length of chicken <em>5</em>.<em>8S rRNA</em> gene was 157 bp (from 6190 to 6346 bp) and similar to the equivalent human gene (<a href="#pone.0157464.s011">S2 Table</a>). The number of GC pars in chicken <em>5</em>.<em>8S rRNA</em> sequence was also similar to human (57.3% and 57.4% respectively).</p>
</div>

<div id="section5" class="section toc-section"><a id="sec016" name="sec016" class="link-target" title="ITS2"></a><h4><em>ITS2</em>.</h4><a id="article1.body1.sec3.sec4.sec5.p1" name="article1.body1.sec3.sec4.sec5.p1" class="link-target"></a><p>Chicken <em>ITS2</em> assembly was validated using two overlapping WGS contigs (<a href="#pone-0157464-t001">Table 1</a>), assembly overlapped being 100% (<a href="#pone.0157464.s010">S1 Table</a>). We excluded from the analysis sequences DQ018753, DQ018755 and FJ008990, which are annotated as containing chicken <em>ITS2</em>. The reason was that the related gene flank sequences were found non-homologous to 3’-end of <em>5</em>.<em>8S rRNA</em> sequence and to <em>5</em>’-end of chicken <em>28S rRNA</em>. Within the designated boundaries (<a href="#pone.0157464.s006">S6 Fig</a>) the length of chicken <em>1TS2</em> was 733 bp (from 6347 to 7079 bp) (<a href="#pone.0157464.s011">S2 Table</a>). Repeats account for 24.3% of spacer size. These repeats are composed of the following nucleotide combinations: (CCGT)n, (GCGCG)n, (CGTT)n, (CCGT)n, (GCG)n. The sequence features increased GC pair content – 82.0% and CpG dinucleotide content – 20.7% which is virtually similar to <em>ITS1</em> values (see above). Similarly to <em>ITS1</em>, <em>ITS2</em> also feature formation of extended hairpins with high melting temperature. According to our results obtained under standard PCR parameters, ΔG = -62.5 (<a href="#pone.0157464.s007">S7 Fig</a>).</p>
</div>

<div id="section6" class="section toc-section"><a id="sec017" name="sec017" class="link-target" title="28S rRNA gene"></a><h4><em>28S rRNA</em> gene.</h4><a id="article1.body1.sec3.sec4.sec6.p1" name="article1.body1.sec3.sec4.sec6.p1" class="link-target"></a><p>The assembled chicken <em>28S rRNA</em> sequence was verified using six WGS contigs and five annotated sequences from GenBank (<a href="#pone-0157464-t001">Table 1</a>). These WGS contigs and annotated sequences overlapped 76.7% and 69.7% of the assembly respectively (<a href="#pone.0157464.s010">S1 Table</a>). Within the designated boundaries (<a href="#pone.0157464.s008">S8 Fig</a>) chicken <em>28S rRNA</em> gene length is 4441 bp (from 7080 to 11520 bp) (<a href="#pone.0157464.s011">S2 Table</a>). GC pair content in <em>28S rRNA</em> chicken gene is much higher than in <em>18S</em> and <em>5</em>.<em>8S rRNA</em> genes and achieves as much as 68.0%.</p>
</div>

<div id="section7" class="section toc-section"><a id="sec018" name="sec018" class="link-target" title="3’ETS"></a><h4><em>3’ETS</em>.</h4><a id="article1.body1.sec3.sec4.sec7.p1" name="article1.body1.sec3.sec4.sec7.p1" class="link-target"></a><p>Chicken <em>3’ETS</em> sequence was assembled from raw reads. The length of the initial assembly was 474 bp. Yet we were unable to define precisely the position of the 3’-end of <em>3’ETS</em> due to lack of data on the sequence of Sal1-box, the variable element of RNApolI transcription termination [<a href="#pone.0157464.ref035" class="ref-tip">35</a>].</p>
<a id="article1.body1.sec3.sec4.sec7.p2" name="article1.body1.sec3.sec4.sec7.p2" class="link-target"></a><p>Based on comparison of <em>3’ETS</em> length and nucleotide content in other organisms (345 bp / 82.9% GC in human, 521 bp / 75.62% GC in mouse, 236 bp / 84.32% GC in xenopus), we can assume that in chicken pre-rRNA transcription termination could occur at 11863 bp position. In chicken <em>3’ETS</em> sequence, the region between 11864 bp and 11994 bp is a poly-T region (<a href="#pone.0157464.s010">S1 Table</a>). This region has not been reported to exist in <em>3’ETS</em> of <em>H</em>.<em>sapiens</em>, <em>M</em>.<em>musculus</em>, and <em>X</em>.<em>laevis</em> (<a href="#pone.0157464.s009">S9 Fig</a>). An analysis of 343 bp (from 11521 to 11863 bp) of the <em>3’ETS</em> region has resulted in 79.9% and 18.9% of GC pair and CpG dinucleotide content respectively. (GC)n, (CGTT)n and (CGGC)n repeats constitute about 24.78% of the analyzed sequence. These results indicate a certain similarity between the 5’<em>ETS</em> and internal ribosomal cluster spacers. We believe that this phenomenon could be related to the existence of a general evolutionary mechanism supporting this stability of spacer nucleotide content within the ribosomal cluster sequence in birds.</p>
</div>
</div>

<div id="section5" class="section toc-section"><a id="sec019" name="sec019" class="link-target" title="ITS1 and ITS2 features"></a>
<h3><em>ITS1</em> and <em>ITS2</em> features</h3>
<a id="article1.body1.sec3.sec5.p1" name="article1.body1.sec3.sec5.p1" class="link-target"></a><p>We have shown chicken <em>ITS1</em> sequence to be more extended than that of most animals, with the exception of marsupials [<a href="#pone.0157464.ref036" class="ref-tip">36</a>]. Chicken <em>ITS2</em> has also proved to be far more extended than reported earlier (GenBank accession numbers: DQ018753, DQ018755, FJ008990). Our attempts to amplify chicken <em>ITS1</em> and <em>ITS2</em> using routine PCR with different thermostable polymerases and GC-enhancers as well as to obtain their complete sequences have been unsuccessful. Our findings suggest that the key problem in avian <em>ITS1</em> and <em>ITS2</em> amplification and sequencing related to high CG pair content and secondary structure formation. These factors impact polymerase effect in PCR process and increase the probability of AT-enriched regions non-specific amplification. This is quite likely to be the main reason for the poor representation of annotated avian ribosomal cluster sequences in the current version of GenBank (<a href="#pone-0157464-t004">Table 4</a>).</p>
<a class="link-target" id="pone-0157464-t004" name="pone-0157464-t004"></a><div class="figure" data-doi="10.1371/journal.pone.0157464.t004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0157464.t004" data-doi="info:doi/10.1371/journal.pone.0157464" data-uri="info:doi/10.1371/journal.pone.0157464.t004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0157464.t004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.t004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.t004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.t004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 4. </span> GenBank representation of <em>ITS1</em> and <em>ITS2</em> sequences.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0157464.t004">
              https://doi.org/10.1371/journal.pone.0157464.t004</a></p></div><a id="article1.body1.sec3.sec5.p2" name="article1.body1.sec3.sec5.p2" class="link-target"></a><p>Notably, among the eight avian sequences deposited to GenBank, only three actually belong to ribosomal clusters, namely to <em>ITS1</em> sequence. At the same time, complete decoding of the ribosomal cluster sequence in different groups of organisms is of a great value for various fields of biology, primarily systematics, phylogeny, and ecology. Our alignment of chicken rRNA gene complete cluster might be useful for comparative research in these fields. In the majority of animals, including birds, the order of alternation of coding and spacer sequences within rRNA gene clusters is conserved. Yet spacer sequences, particularly <em>ITS1</em> and <em>ITS2</em>, are characterized by high variability. Due to this they are used extensively as DNA barcodes as well as nuclear markers of micro- and macro-evolutionary events [<a href="#pone.0157464.ref037" class="ref-tip">37</a>–<a href="#pone.0157464.ref040" class="ref-tip">40</a>]. However, the features of avian <em>ITS1</em> and <em>ITS2</em> do not allow treating their sequences as easily accessible and effective DNA barcodes or phylogenetic markers for this class.</p>
</div>
</div>

<div id="section4" class="section toc-section"><a id="sec020" name="sec020" data-toc="sec020" class="link-target" title="Conclusions"></a><h2>Conclusions</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>In this work, we determined, verified, and featured the complete sequence of chicken ribosomal cluster <strong>(</strong><a href="#pone-0157464-g003">Fig 3</a><strong>)</strong>. The assembled chicken cluster is 11863 bp in length and contains complete <em>5’ETS</em>, <em>18S rRNA</em>, <em>ITSI</em>, <em>5</em>.<em>8S rRNA</em>, <em>ITS2</em>, <em>28S rRNA</em> sequences as well as <em>3’ETS</em> sequence 5’-end. This cluster sequence was deposited to GenBank under accession number KT445934. Chicken <em>18S</em>, <em>5</em>.<em>8S</em> and <em>28S rRNA</em> gene sequences are typical for orther tetrapods. However both <em>ITS1</em> and <em>ITS2</em> were found to be more extended in size and higher in GC content. As a result, they have a complicated secondary structure that prevents their PCR amplification and consequently their use as phylogenetic markers. It seems more promising to use a relatively less GC-enriched <em>5’ETS</em> sequence for phylogeny purposes. Meanwhile, <em>ITS1</em>, <em>ITS2</em> and <em>3’ETS</em> sequences revealed similarities in the GC, CpG and repeated sequence contents. Thus it seems possible to suggest the existence of a general evolutionary mechanism supporting the spacer constant nucleotide proportions within avian rDNA genome.</p>
<a class="link-target" id="pone-0157464-g003" name="pone-0157464-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0157464.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0157464.g003" data-doi="info:doi/10.1371/journal.pone.0157464" data-uri="info:doi/10.1371/journal.pone.0157464.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0157464.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0157464.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0157464.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0157464.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Fig 3. </span> The chicken rRNA gene cluster structure and features.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0157464.g003">
              https://doi.org/10.1371/journal.pone.0157464.g003</a></p></div><a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>Chicken rRNA gene cluster structure knowledge extends the use of birds as a model object for exploration of NOR regulation, particularly in ontogenesis and cell differentiation. Our results could be useful for developing optimal solutions in the areas of population genetics and evolution.</p>
</div>

<div id="section5" class="section toc-section"><a id="sec021" name="sec021" data-toc="sec021" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0157464"></div><div class="supplementary-material"><a name="pone.0157464.s001" id="pone.0157464.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s001">S1 Fig. </a>Chicken <em>5’ETS</em> sequence boundaries resulted from a multiple sequence alignment.</h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">Black frames outline the elements of the ribosomal cluster. Cluster members are schematically indicated by colored boxes. Asterisk–chicken pre-rRNA transcription initiation site. Arrow–transcription direction. Double slashes schematically point the hidden area of alignment. Extreme right and left–accession numbers and taxa abbreviations relating to the sequences located before and after the hidden area of the alignment, respectively. G. gal–<em>Gallus gallus</em>; R. nor–<em>Rattus norvegicus</em>; C. por–<em>Crocodylus porosus</em>; X. laev–<em>Xenopus laevis</em>; H. sap–<em>Homo sapiens</em>; M. mus–<em>Mus musculus</em>. KT445934.2_G. gal–sequence assembled by raw read tiling.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s001">https://doi.org/10.1371/journal.pone.0157464.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s002" id="pone.0157464.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s002">S2 Fig. </a><em>18S rRNA</em> gene sequence boundaries resulted from a multiple sequence alignment.</h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">Indications are the same as in <a href="#pone.0157464.s001">S1 Fig</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s002">https://doi.org/10.1371/journal.pone.0157464.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s003" id="pone.0157464.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s003">S3 Fig. </a>Chicken <em>ITS1</em> sequence boundaries resulted from a multiple sequence alignment.</h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">Indications are the same as in <a href="#pone.0157464.s001">S1 Fig</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s003">https://doi.org/10.1371/journal.pone.0157464.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s004" id="pone.0157464.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s004">S4 Fig. </a>Chicken <em>ITS1</em> ssDNA secondary structures under the conditions of T = 60°C, [K<sup>+</sup>] = 50 mM.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s004">https://doi.org/10.1371/journal.pone.0157464.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s005" id="pone.0157464.s005" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s005">S5 Fig. </a>Chicken <em>5</em>.<em>8S rRNA</em> gene sequence boundaries resulted from a multiple sequence alignment.</h3><a id="article1.body1.sec5.supplementary-material5.caption1.p1" name="article1.body1.sec5.supplementary-material5.caption1.p1" class="link-target"></a><p class="preSiDOI">Indications are the same as in <a href="#pone.0157464.s001">S1 Fig</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s005">https://doi.org/10.1371/journal.pone.0157464.s005</a></p><a id="article1.body1.sec5.supplementary-material5.caption1.p2" name="article1.body1.sec5.supplementary-material5.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s006" id="pone.0157464.s006" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s006">S6 Fig. </a>Chicken <em>ITS2</em> sequence boundaries resulted from a multiple sequence alignment.</h3><a id="article1.body1.sec5.supplementary-material6.caption1.p1" name="article1.body1.sec5.supplementary-material6.caption1.p1" class="link-target"></a><p class="preSiDOI">Indications are the same as in <a href="#pone.0157464.s001">S1 Fig</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s006">https://doi.org/10.1371/journal.pone.0157464.s006</a></p><a id="article1.body1.sec5.supplementary-material6.caption1.p2" name="article1.body1.sec5.supplementary-material6.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s007" id="pone.0157464.s007" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s007">S7 Fig. </a>Chicken <em>ITS2</em> ssDNA secondary structures under the conditions of T = 60°C, [K<sup>+</sup>] = 50 mM.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s007">https://doi.org/10.1371/journal.pone.0157464.s007</a></p><a id="article1.body1.sec5.supplementary-material7.caption1.p1" name="article1.body1.sec5.supplementary-material7.caption1.p1" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s008" id="pone.0157464.s008" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s008">S8 Fig. </a>Chicken <em>28S rRNA</em> gene sequence boundaries resulted from a multiple sequence alignment.</h3><a id="article1.body1.sec5.supplementary-material8.caption1.p1" name="article1.body1.sec5.supplementary-material8.caption1.p1" class="link-target"></a><p class="preSiDOI">Indications are the same as in <a href="#pone.0157464.s001">S1 Fig</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s008">https://doi.org/10.1371/journal.pone.0157464.s008</a></p><a id="article1.body1.sec5.supplementary-material8.caption1.p2" name="article1.body1.sec5.supplementary-material8.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s009" id="pone.0157464.s009" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s009">S9 Fig. </a>Chicken <em>3’ETS</em> sequence boundaries resulted from a multiple sequence alignment.</h3><a id="article1.body1.sec5.supplementary-material9.caption1.p1" name="article1.body1.sec5.supplementary-material9.caption1.p1" class="link-target"></a><p class="preSiDOI">Indications are the same as in <a href="#pone.0157464.s001">S1 Fig</a>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s009">https://doi.org/10.1371/journal.pone.0157464.s009</a></p><a id="article1.body1.sec5.supplementary-material9.caption1.p2" name="article1.body1.sec5.supplementary-material9.caption1.p2" class="link-target"></a><p class="postSiDOI">(TIF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s010" id="pone.0157464.s010" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s010">S1 Table. </a>Raw read assembly verification using WGS contigs and annotated sequences.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s010">https://doi.org/10.1371/journal.pone.0157464.s010</a></p><a id="article1.body1.sec5.supplementary-material10.caption1.p1" name="article1.body1.sec5.supplementary-material10.caption1.p1" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s011" id="pone.0157464.s011" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s011">S2 Table. </a>Multiple alignment matrix for chicken rRNA gene cluster boundaries search using annotated fragments of ribosomal clusters of <em>Homo sapiens</em>, <em>Rattus norvegicus</em>, <em>Mus musculus</em>, <em>Xenopus laevis</em> and <em>Crocodylus porosus</em>.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s011">https://doi.org/10.1371/journal.pone.0157464.s011</a></p><a id="article1.body1.sec5.supplementary-material11.caption1.p1" name="article1.body1.sec5.supplementary-material11.caption1.p1" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0157464.s012" id="pone.0157464.s012" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0157464.s012">S3 Table. </a>De novo and re-assembled rRNA gene cluster sequence alignment.</h3><p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0157464.s012">https://doi.org/10.1371/journal.pone.0157464.s012</a></p><a id="article1.body1.sec5.supplementary-material12.caption1.p1" name="article1.body1.sec5.supplementary-material12.caption1.p1" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>We are grateful to Romain Solinhac and Valerie Fillon for kindly provided WAG137G04 BAC-clone. We acknowledge “Chromas” and “Molecular and Cell Technology” Saint-Petersburg State University Resource Centers.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: AD AS EG. Performed the experiments: AD EK AMK. Analyzed the data: AD EG. Contributed reagents/materials/analysis tools: TF AAK. Wrote the paper: AD SG EG.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0157464.ref001" id="pone.0157464.ref001" class="link-target"></a>Shaw P, Brown J. Nucleoli: composition, function, and dynamics. Plant Physiol. 2012; 158(1): 44–51.  pmid:22082506 <ul class="reflinks"><li><a href="#" data-author="Shaw" data-cit="ShawP%2C%20BrownJ.%20Nucleoli%3A%20composition%2C%20function%2C%20and%20dynamics.%20Plant%20Physiol.%202012%3B%20158%281%29%3A%2044%E2%80%9351.%20doi%3A%2010.1104%2Fpp.111.188052%2022082506" data-title="Nucleoli%3A%20composition%2C%20function%2C%20and%20dynamics" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22082506" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Nucleoli%3A+composition%2C+function%2C+and+dynamics+Shaw+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0157464.ref002" id="pone.0157464.ref002" class="link-target"></a>Smirnov E, Kalmárová M, Koberna K, Zemanová Z, Malínský J, Masata M, et al. NORs and their transcription competence during the cell cycle. Folia Biol. 2006; 52(3): 59–70. <ul class="reflinks"><li><a href="#" data-author="Smirnov" data-cit="SmirnovE%2C%20Kalm%C3%A1rov%C3%A1M%2C%20KobernaK%2C%20Zemanov%C3%A1Z%2C%20Mal%C3%ADnsk%C3%BDJ%2C%20MasataM%2C%20et%20al.%20NORs%20and%20their%20transcription%20competence%20during%20the%20cell%20cycle.%20Folia%20Biol.%202006%3B%2052%283%29%3A%2059%E2%80%9370." data-title="NORs%20and%20their%20transcription%20competence%20during%20the%20cell%20cycle" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=NORs+and+their+transcription+competence+during+the+cell+cycle+Smirnov+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0157464.ref003" id="pone.0157464.ref003" class="link-target"></a>Hernandez-Verdun D, Roussel P, Thiry M, Sirri V, Lafontaine DL. The nucleolus: structure/function relationship in RNA metabolism. Wiley Interdiscip Rev RNA. 2010; 1(3): 415–431.  pmid:21956940 <ul class="reflinks"><li><a href="#" data-author="Hernandez-Verdun" data-cit="Hernandez-VerdunD%2C%20RousselP%2C%20ThiryM%2C%20SirriV%2C%20LafontaineDL.%20The%20nucleolus%3A%20structure%2Ffunction%20relationship%20in%20RNA%20metabolism.%20Wiley%20Interdiscip%20Rev%20RNA.%202010%3B%201%283%29%3A%20415%E2%80%93431.%20doi%3A%2010.1002%2Fwrna.39%2021956940" data-title="The%20nucleolus%3A%20structure%2Ffunction%20relationship%20in%20RNA%20metabolism" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21956940" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+nucleolus%3A+structure%2Ffunction+relationship+in+RNA+metabolism+Hernandez-Verdun+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0157464.ref004" id="pone.0157464.ref004" class="link-target"></a>Bozinovic G, Sit TL, Hinton DE, Oleksiak MF. Gene expression throughout a vertebrate's embryogenesis. BMC Genomics. 2011 Feb 28;12:132.  pmid:21356103 <ul class="reflinks"><li><a href="#" data-author="Bozinovic" data-cit="BozinovicG%2C%20SitTL%2C%20HintonDE%2C%20OleksiakMF.%20Gene%20expression%20throughout%20a%20vertebrate%27s%20embryogenesis.%20BMC%20Genomics.%202011%20Feb%2028%3B12%3A132.%20doi%3A%2010.1186%2F1471-2164-12-132%2021356103" data-title="Gene%20expression%20throughout%20a%20vertebrate%27s%20embryogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21356103" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Gene+expression+throughout+a+vertebrate%27s+embryogenesis+Bozinovic+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0157464.ref005" id="pone.0157464.ref005" class="link-target"></a>Zheng Z, Jia JL, Bou G, Hu LL, Wang ZD, Shen XH, et al. rRNA genes are not fully activated in mouse somatic cell nuclear transfer embryos. J Biol Chem. 2012;287(24): 19949–19960.  pmid:22467869 <ul class="reflinks"><li><a href="#" data-author="Zheng" data-cit="ZhengZ%2C%20JiaJL%2C%20BouG%2C%20HuLL%2C%20WangZD%2C%20ShenXH%2C%20et%20al.%20rRNA%20genes%20are%20not%20fully%20activated%20in%20mouse%20somatic%20cell%20nuclear%20transfer%20embryos.%20J%20Biol%20Chem.%202012%3B287%2824%29%3A%2019949%E2%80%9319960.%20doi%3A%2010.1074%2Fjbc.M112.355099%2022467869" data-title="rRNA%20genes%20are%20not%20fully%20activated%20in%20mouse%20somatic%20cell%20nuclear%20transfer%20embryos" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/22467869" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=rRNA+genes+are+not+fully+activated+in+mouse+somatic+cell+nuclear+transfer+embryos+Zheng+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0157464.ref006" id="pone.0157464.ref006" class="link-target"></a>Tafforeau L, Zorbas C, Langhendries JL, Mullineux ST, Stamatopoulou V, Mullier R, et al. The complexity of human ribosome biogenesis revealed by systematic nucleolar screening of Pre-rRNA processing factors. Mol Cell. 2013;51(4): 539–551.  pmid:23973377 <ul class="reflinks"><li><a href="#" data-author="Tafforeau" data-cit="TafforeauL%2C%20ZorbasC%2C%20LanghendriesJL%2C%20MullineuxST%2C%20StamatopoulouV%2C%20MullierR%2C%20et%20al.%20The%20complexity%20of%20human%20ribosome%20biogenesis%20revealed%20by%20systematic%20nucleolar%20screening%20of%20Pre-rRNA%20processing%20factors.%20Mol%20Cell.%202013%3B51%284%29%3A%20539%E2%80%93551.%20doi%3A%2010.1016%2Fj.molcel.2013.08.011%2023973377" data-title="The%20complexity%20of%20human%20ribosome%20biogenesis%20revealed%20by%20systematic%20nucleolar%20screening%20of%20Pre-rRNA%20processing%20factors" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23973377" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+complexity+of+human+ribosome+biogenesis+revealed+by+systematic+nucleolar+screening+of+Pre-rRNA+processing+factors+Tafforeau+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0157464.ref007" id="pone.0157464.ref007" class="link-target"></a>Singer M, Berg P. Genes and Genomes, a Changing Perspective. Mill Valley: University Science Books; 1991. <ul class="find-nolinks"></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0157464.ref008" id="pone.0157464.ref008" class="link-target"></a>Koepfli KP, Paten B. Genome 10K Community of Scientists, O'Brien SJ. The Genome 10K Project: a way forward. Annu Rev Anim Biosci. 2015;3: 57–111.  pmid:25689317 <ul class="reflinks"><li><a href="#" data-author="Koepfli" data-cit="KoepfliKP%2C%20PatenB.%20Genome%2010K%20Community%20of%20Scientists%2C%20O%27BrienSJ.%20The%20Genome%2010K%20Project%3A%20a%20way%20forward.%20Annu%20Rev%20Anim%20Biosci.%202015%3B3%3A%2057%E2%80%93111.%20doi%3A%2010.1146%2Fannurev-animal-090414-014900%2025689317" data-title="The%20Genome%2010K%20Project%3A%20a%20way%20forward" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25689317" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+Genome+10K+Project%3A+a+way+forward+Koepfli+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0157464.ref009" id="pone.0157464.ref009" class="link-target"></a>NCBI GenBank database. Available: <a href="http://www.ncbi.nlm.nih.gov/genbank">http://www.ncbi.nlm.nih.gov/genbank</a> <ul class="find-nolinks"></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0157464.ref010" id="pone.0157464.ref010" class="link-target"></a>Zhang X, Yelbuz TM, Cofer GP, Choma MA, Kirby ML, Johnson GA. Improved preparation of chick embryonic samples for magnetic resonance microscopy. Magn Reson Med. 2003;49(6): 1192–1195. pmid:12768599 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangX%2C%20YelbuzTM%2C%20CoferGP%2C%20ChomaMA%2C%20KirbyML%2C%20JohnsonGA.%20Improved%20preparation%20of%20chick%20embryonic%20samples%20for%20magnetic%20resonance%20microscopy.%20Magn%20Reson%20Med.%202003%3B49%286%29%3A%201192%E2%80%931195.%2012768599" data-title="Improved%20preparation%20of%20chick%20embryonic%20samples%20for%20magnetic%20resonance%20microscopy" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12768599" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Improved+preparation+of+chick+embryonic+samples+for+magnetic+resonance+microscopy+Zhang+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0157464.ref011" id="pone.0157464.ref011" class="link-target"></a>Massin P, Rodrigues P, Marasescu M, van der Werf S, Naffakh N. Cloning of the chicken RNA polymerase I promoter and use for reverse genetics of influenza A viruses in avian cells. J Virol. 2005;79(21):13811–6. pmid:16227302 <ul class="reflinks"><li><a href="#" data-author="Massin" data-cit="MassinP%2C%20RodriguesP%2C%20MarasescuM%2C%20van%20der%20WerfS%2C%20NaffakhN.%20Cloning%20of%20the%20chicken%20RNA%20polymerase%20I%20promoter%20and%20use%20for%20reverse%20genetics%20of%20influenza%20A%20viruses%20in%20avian%20cells.%20J%20Virol.%202005%3B79%2821%29%3A13811%E2%80%936.%2016227302" data-title="Cloning%20of%20the%20chicken%20RNA%20polymerase%20I%20promoter%20and%20use%20for%20reverse%20genetics%20of%20influenza%20A%20viruses%20in%20avian%20cells" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/16227302" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Cloning+of+the+chicken+RNA+polymerase+I+promoter+and+use+for+reverse+genetics+of+influenza+A+viruses+in+avian+cells+Massin+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0157464.ref012" id="pone.0157464.ref012" class="link-target"></a>International Chicken Genome Sequencing Consortium. Sequence and comparative analysis of the chicken genome provide unique perspectives on vertebrate evolution. Nature. 2004;432(7018): 695–716. pmid:15592404 <ul class="reflinks"><li><a href="#" data-author="" data-cit="International%20Chicken%20Genome%20Sequencing%20Consortium.%20Sequence%20and%20comparative%20analysis%20of%20the%20chicken%20genome%20provide%20unique%20perspectives%20on%20vertebrate%20evolution.%20Nature.%202004%3B432%287018%29%3A%20695%E2%80%93716.%2015592404" data-title="Sequence%20and%20comparative%20analysis%20of%20the%20chicken%20genome%20provide%20unique%20perspectives%20on%20vertebrate%20evolution" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15592404" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Sequence+and+comparative+analysis+of+the+chicken+genome+provide+unique+perspectives+on+vertebrate+evolution++2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0157464.ref013" id="pone.0157464.ref013" class="link-target"></a>Zhang G, Li C, Li Q, Li B, Larkin DM, Lee C, et al. Comparative genomics reveals insights into avian genome evolution and adaptation. Science. 2014;346(6215): 1311–1320.  pmid:25504712 <ul class="reflinks"><li><a href="#" data-author="Zhang" data-cit="ZhangG%2C%20LiC%2C%20LiQ%2C%20LiB%2C%20LarkinDM%2C%20LeeC%2C%20et%20al.%20Comparative%20genomics%20reveals%20insights%20into%20avian%20genome%20evolution%20and%20adaptation.%20Science.%202014%3B346%286215%29%3A%201311%E2%80%931320.%20doi%3A%2010.1126%2Fscience.1251385%2025504712" data-title="Comparative%20genomics%20reveals%20insights%20into%20avian%20genome%20evolution%20and%20adaptation" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25504712" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Comparative+genomics+reveals+insights+into+avian+genome+evolution+and+adaptation+Zhang+2014" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0157464.ref014" id="pone.0157464.ref014" class="link-target"></a>Auer H, Mayr B, Lambrou M, Schleger W. An extended chicken karyotype, including the NOR chromosome. Cytogenet Cell Genet. 1987;45(3–4): 218–221. pmid:3691189 <ul class="reflinks"><li><a href="#" data-author="Auer" data-cit="AuerH%2C%20MayrB%2C%20LambrouM%2C%20SchlegerW.%20An%20extended%20chicken%20karyotype%2C%20including%20the%20NOR%20chromosome.%20Cytogenet%20Cell%20Genet.%201987%3B45%283%E2%80%934%29%3A%20218%E2%80%93221.%203691189" data-title="An%20extended%20chicken%20karyotype%2C%20including%20the%20NOR%20chromosome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/3691189" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=An+extended+chicken+karyotype%2C+including+the+NOR+chromosome+Auer+1987" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0157464.ref015" id="pone.0157464.ref015" class="link-target"></a>Miller MM, Goto RM, Taylor RL Jr, Zoorob R, Auffray C, Briles RW, et al. Assignment of Rfp-Y to the chicken major histocompatibility complex/NOR microchromosome and evidence for high-frequency recombination associated with the nucleolar organizer region. Proc Natl Acad Sci USA. 1996;93(9): 3958–3962. pmid:8632997 <ul class="reflinks"><li><a href="#" data-author="Miller" data-cit="MillerMM%2C%20GotoRM%2C%20TaylorRLJr%2C%20ZoorobR%2C%20AuffrayC%2C%20BrilesRW%2C%20et%20al.%20Assignment%20of%20Rfp-Y%20to%20the%20chicken%20major%20histocompatibility%20complex%2FNOR%20microchromosome%20and%20evidence%20for%20high-frequency%20recombination%20associated%20with%20the%20nucleolar%20organizer%20region.%20Proc%20Natl%20Acad%20Sci%20USA.%201996%3B93%289%29%3A%203958%E2%80%933962.%208632997" data-title="Assignment%20of%20Rfp-Y%20to%20the%20chicken%20major%20histocompatibility%20complex%2FNOR%20microchromosome%20and%20evidence%20for%20high-frequency%20recombination%20associated%20with%20the%20nucleolar%20organizer%20region" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/8632997" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Assignment+of+Rfp-Y+to+the+chicken+major+histocompatibility+complex%2FNOR+microchromosome+and+evidence+for+high-frequency+recombination+associated+with+the+nucleolar+organizer+region+Miller+1996" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0157464.ref016" id="pone.0157464.ref016" class="link-target"></a>Wain HM, Toye AA, Hughes S, Bumstead N. Targeting of marker loci to chicken chromosome 16 by representational difference analysis. Anim Genet. 1998;29(6):446–452. pmid:9883506 <ul class="reflinks"><li><a href="#" data-author="Wain" data-cit="WainHM%2C%20ToyeAA%2C%20HughesS%2C%20BumsteadN.%20Targeting%20of%20marker%20loci%20to%20chicken%20chromosome%2016%20by%20representational%20difference%20analysis.%20Anim%20Genet.%201998%3B29%286%29%3A446%E2%80%93452.%209883506" data-title="Targeting%20of%20marker%20loci%20to%20chicken%20chromosome%2016%20by%20representational%20difference%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/9883506" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Targeting+of+marker+loci+to+chicken+chromosome+16+by+representational+difference+analysis+Wain+1998" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0157464.ref017" id="pone.0157464.ref017" class="link-target"></a>Masabanda JS, Burt DW, O'Brien PC, Vignal A, Fillon V, Walsh PS, et al. Molecular cytogenetic definition of the chicken genome: the first complete avian karyotype. Genetics. 2004;166(3): 1367–1373. pmid:15082555 <ul class="reflinks"><li><a href="#" data-author="Masabanda" data-cit="MasabandaJS%2C%20BurtDW%2C%20O%27BrienPC%2C%20VignalA%2C%20FillonV%2C%20WalshPS%2C%20et%20al.%20Molecular%20cytogenetic%20definition%20of%20the%20chicken%20genome%3A%20the%20first%20complete%20avian%20karyotype.%20Genetics.%202004%3B166%283%29%3A%201367%E2%80%931373.%2015082555" data-title="Molecular%20cytogenetic%20definition%20of%20the%20chicken%20genome%3A%20the%20first%20complete%20avian%20karyotype" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/15082555" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+cytogenetic+definition+of+the+chicken+genome%3A+the+first+complete+avian+karyotype+Masabanda+2004" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0157464.ref018" id="pone.0157464.ref018" class="link-target"></a>Delany ME, Gessaro TM, Rodrigue KL, Daniels LM. Chromosomal mapping of chicken mega-telomere arrays to GGA9, 16, 28 and W using a cytogenomic approach. Cytogenet Genome Res. 2007;117(1–4): 54–63. pmid:17675845 <ul class="reflinks"><li><a href="#" data-author="Delany" data-cit="DelanyME%2C%20GessaroTM%2C%20RodrigueKL%2C%20DanielsLM.%20Chromosomal%20mapping%20of%20chicken%20mega-telomere%20arrays%20to%20GGA9%2C%2016%2C%2028%20and%20W%20using%20a%20cytogenomic%20approach.%20Cytogenet%20Genome%20Res.%202007%3B117%281%E2%80%934%29%3A%2054%E2%80%9363.%2017675845" data-title="Chromosomal%20mapping%20of%20chicken%20mega-telomere%20arrays%20to%20GGA9%2C%2016%2C%2028%20and%20W%20using%20a%20cytogenomic%20approach" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17675845" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chromosomal+mapping+of+chicken+mega-telomere+arrays+to+GGA9%2C+16%2C+28+and+W+using+a+cytogenomic+approach+Delany+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0157464.ref019" id="pone.0157464.ref019" class="link-target"></a>Solinhac R, Leroux S, Galkina S, Chazara O, Feve K, Vignoles F, et al. Integrative mapping analysis of chicken microchromosome 16 organization. BMC Genomics. 2010 Nov 4;11:616.  pmid:21050458 <ul class="reflinks"><li><a href="#" data-author="Solinhac" data-cit="SolinhacR%2C%20LerouxS%2C%20GalkinaS%2C%20ChazaraO%2C%20FeveK%2C%20VignolesF%2C%20et%20al.%20Integrative%20mapping%20analysis%20of%20chicken%20microchromosome%2016%20organization.%20BMC%20Genomics.%202010%20Nov%204%3B11%3A616.%20doi%3A%2010.1186%2F1471-2164-11-616%2021050458" data-title="Integrative%20mapping%20analysis%20of%20chicken%20microchromosome%2016%20organization" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21050458" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Integrative+mapping+analysis+of+chicken+microchromosome+16+organization+Solinhac+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0157464.ref020" id="pone.0157464.ref020" class="link-target"></a>Shang WH, Hori T, Martins NM, Toyoda A, Misu S, Monma N, et al. Chromosome engineering allows the efficient isolation of vertebrate neocentromeres. Dev Cell. 2013;24(6): 635–648.  pmid:23499358 <ul class="reflinks"><li><a href="#" data-author="Shang" data-cit="ShangWH%2C%20HoriT%2C%20MartinsNM%2C%20ToyodaA%2C%20MisuS%2C%20MonmaN%2C%20et%20al.%20Chromosome%20engineering%20allows%20the%20efficient%20isolation%20of%20vertebrate%20neocentromeres.%20Dev%20Cell.%202013%3B24%286%29%3A%20635%E2%80%93648.%20doi%3A%2010.1016%2Fj.devcel.2013.02.009%2023499358" data-title="Chromosome%20engineering%20allows%20the%20efficient%20isolation%20of%20vertebrate%20neocentromeres" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/23499358" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Chromosome+engineering+allows+the+efficient+isolation+of+vertebrate+neocentromeres+Shang+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0157464.ref021" id="pone.0157464.ref021" class="link-target"></a>NCBI Nucleotide database. Available: <a href="http://www.ncbi.nlm.nih.gov/nuccore">http://www.ncbi.nlm.nih.gov/nuccore</a>. <ul class="find-nolinks"></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0157464.ref022" id="pone.0157464.ref022" class="link-target"></a>van Tuinen M, Sibley CG, Hedges SB. The early history of modern birds inferred from DNA sequences of nuclear and mitochondrial ribosomal genes. Mol Biol Evol. 2000 Mar;17(3): 451–457. pmid:10723745 <ul class="reflinks"><li><a href="#" data-author="van%20Tuinen" data-cit="van%20TuinenM%2C%20SibleyCG%2C%20HedgesSB.%20The%20early%20history%20of%20modern%20birds%20inferred%20from%20DNA%20sequences%20of%20nuclear%20and%20mitochondrial%20ribosomal%20genes.%20Mol%20Biol%20Evol.%202000%20Mar%3B17%283%29%3A%20451%E2%80%93457.%2010723745" data-title="The%20early%20history%20of%20modern%20birds%20inferred%20from%20DNA%20sequences%20of%20nuclear%20and%20mitochondrial%20ribosomal%20genes" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/10723745" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+early+history+of+modern+birds+inferred+from+DNA+sequences+of+nuclear+and+mitochondrial+ribosomal+genes+van+Tuinen+2000" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0157464.ref023" id="pone.0157464.ref023" class="link-target"></a>Paśko Ł, Ericson PG, Elzanowski A. Phylogenetic utility and evolution of indels: a study in neognathous birds. Mol Phylogenet Evol. 2011;61(3): 760–771.  pmid:21843647 <ul class="reflinks"><li><a href="#" data-author="Pa%C5%9Bko" data-cit="Pa%C5%9Bko%C5%81%2C%20EricsonPG%2C%20ElzanowskiA.%20Phylogenetic%20utility%20and%20evolution%20of%20indels%3A%20a%20study%20in%20neognathous%20birds.%20Mol%20Phylogenet%20Evol.%202011%3B61%283%29%3A%20760%E2%80%93771.%20doi%3A%2010.1016%2Fj.ympev.2011.07.021%2021843647" data-title="Phylogenetic%20utility%20and%20evolution%20of%20indels%3A%20a%20study%20in%20neognathous%20birds" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/21843647" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogenetic+utility+and+evolution+of+indels%3A+a+study+in+neognathous+birds+Pa%C5%9Bko+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0157464.ref024" id="pone.0157464.ref024" class="link-target"></a>NCBI BLAST databases. Available: <a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>. <ul class="find-nolinks"></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0157464.ref025" id="pone.0157464.ref025" class="link-target"></a>Unipro UGENE. Available: <a href="http://ugene.net">http://ugene.net</a>. <ul class="find-nolinks"></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0157464.ref026" id="pone.0157464.ref026" class="link-target"></a>Geneious. Available: <a href="http://www.geneious.com">http://www.geneious.com</a>. <ul class="find-nolinks"></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0157464.ref027" id="pone.0157464.ref027" class="link-target"></a>Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. MEGA6: Molecular Evolutionary Genetics Analysis version 6.0. Molecular Biology and Evolution. 2013;30: 2725–2729.  pmid:24132122 <ul class="reflinks"><li><a href="#" data-author="Tamura" data-cit="TamuraK%2C%20StecherG%2C%20PetersonD%2C%20FilipskiA%2C%20KumarS.%20MEGA6%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20version%206.0.%20Molecular%20Biology%20and%20Evolution.%202013%3B30%3A%202725%E2%80%932729.%20doi%3A%2010.1093%2Fmolbev%2Fmst197%2024132122" data-title="MEGA6%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20version%206.0" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=MEGA6%3A+Molecular+Evolutionary+Genetics+Analysis+version+6.0+Tamura+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0157464.ref028" id="pone.0157464.ref028" class="link-target"></a>RepeatMasker. Available: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>. <ul class="find-nolinks"></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0157464.ref029" id="pone.0157464.ref029" class="link-target"></a>Zuker M. Mfold web server for nucleic acid folding and hybridization prediction. Nucleic Acids Res. 2003;31(13): 3406–3415. pmid:12824337 <ul class="reflinks"><li><a href="#" data-author="Zuker" data-cit="ZukerM.%20Mfold%20web%20server%20for%20nucleic%20acid%20folding%20and%20hybridization%20prediction.%20Nucleic%20Acids%20Res.%202003%3B31%2813%29%3A%203406%E2%80%933415.%2012824337" data-title="Mfold%20web%20server%20for%20nucleic%20acid%20folding%20and%20hybridization%20prediction" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824337" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Mfold+web+server+for+nucleic+acid+folding+and+hybridization+prediction+Zuker+2003" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0157464.ref030" id="pone.0157464.ref030" class="link-target"></a>Telenius H, Carter NP, Bebb CE, Nordenskjold M, Ponder BAJ, Tunnacliffe A. Degenerate oligonucleotide-primed PCR: General amplification of target DNA by a single degenerate primer. Genomics.1992;13: 718–725. pmid:1639399 <ul class="reflinks"><li><a href="#" data-author="Telenius" data-cit="TeleniusH%2C%20CarterNP%2C%20BebbCE%2C%20NordenskjoldM%2C%20PonderBAJ%2C%20TunnacliffeA.%20Degenerate%20oligonucleotide-primed%20PCR%3A%20General%20amplification%20of%20target%20DNA%20by%20a%20single%20degenerate%20primer.%20Genomics.1992%3B13%3A%20718%E2%80%93725.%201639399" data-title="Degenerate%20oligonucleotide-primed%20PCR%3A%20General%20amplification%20of%20target%20DNA%20by%20a%20single%20degenerate%20primer" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/1639399" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Degenerate+oligonucleotide-primed+PCR%3A+General+amplification+of+target+DNA+by+a+single+degenerate+primer+Telenius+1992" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0157464.ref031" id="pone.0157464.ref031" class="link-target"></a>Galkina S, Deryusheva S, Fillon V, Vignal A, Crooijmans R, Groenen M, et al. FISH on avian lampbrush chromosomes produces higher resolution gene mapping. Genetica. 2006;128: 241–251. pmid:17028954 <ul class="reflinks"><li><a href="#" data-author="Galkina" data-cit="GalkinaS%2C%20DeryushevaS%2C%20FillonV%2C%20VignalA%2C%20CrooijmansR%2C%20GroenenM%2C%20et%20al.%20FISH%20on%20avian%20lampbrush%20chromosomes%20produces%20higher%20resolution%20gene%20mapping.%20Genetica.%202006%3B128%3A%20241%E2%80%93251.%2017028954" data-title="FISH%20on%20avian%20lampbrush%20chromosomes%20produces%20higher%20resolution%20gene%20mapping" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/17028954" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=FISH+on+avian+lampbrush+chromosomes+produces+higher+resolution+gene+mapping+Galkina+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0157464.ref032" id="pone.0157464.ref032" class="link-target"></a>Chikuni K., Minaka N., Ikenaga H. Molecular phylogeny of some Passeriformes, based on cytochrome b sequences. J. Yamashina Inst. for Ornithology. 1995;28: 1–8. <ul class="reflinks"><li><a href="#" data-author="Chikuni" data-cit="ChikuniK.%2C%20MinakaN.%2C%20IkenagaH.%20Molecular%20phylogeny%20of%20some%20Passeriformes%2C%20based%20on%20cytochrome%20b%20sequences.%20J.%20Yamashina%20Inst.%20for%20Ornithology.%201995%3B28%3A%201%E2%80%938." data-title="Molecular%20phylogeny%20of%20some%20Passeriformes%2C%20based%20on%20cytochrome%20b%20sequences" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+phylogeny+of+some+Passeriformes%2C+based+on+cytochrome+b+sequences+Chikuni+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0157464.ref033" id="pone.0157464.ref033" class="link-target"></a>Azad AA, Deacon NJ. The 3'-terminal primary structure of five eukaryotic 18S rRNAs determined by the direct chemical method of sequencing. The highly conserved sequences include an invariant region complementary to eukaryotic 5S rRNA. Nucleic Acids Res. 1980;8(19): 4365–4376. pmid:7433112 <ul class="reflinks"><li><a href="#" data-author="Azad" data-cit="AzadAA%2C%20DeaconNJ.%20The%203%27-terminal%20primary%20structure%20of%20five%20eukaryotic%2018S%20rRNAs%20determined%20by%20the%20direct%20chemical%20method%20of%20sequencing.%20The%20highly%20conserved%20sequences%20include%20an%20invariant%20region%20complementary%20to%20eukaryotic%205S%20rRNA.%20Nucleic%20Acids%20Res.%201980%3B8%2819%29%3A%204365%E2%80%934376.%207433112" data-title="The%203%27-terminal%20primary%20structure%20of%20five%20eukaryotic%2018S%20rRNAs%20determined%20by%20the%20direct%20chemical%20method%20of%20sequencing.%20The%20highly%20conserved%20sequences%20include%20an%20invariant%20region%20complementary%20to%20eukaryotic%205S%20rRNA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7433112" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=The+3%27-terminal+primary+structure+of+five+eukaryotic+18S+rRNAs+determined+by+the+direct+chemical+method+of+sequencing.+The+highly+conserved+sequences+include+an+invariant+region+complementary+to+eukaryotic+5S+rRNA+Azad+1980" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0157464.ref034" id="pone.0157464.ref034" class="link-target"></a>Hedges SB, Moberg KD, Maxson LR. Tetrapod phylogeny inferred from 18S and 28S ribosomal RNA sequences and a review of the evidence for amniote relationships. Mol Biol Evol. 1990;7(6): 607–633. pmid:2283953 <ul class="reflinks"><li><a href="#" data-author="Hedges" data-cit="HedgesSB%2C%20MobergKD%2C%20MaxsonLR.%20Tetrapod%20phylogeny%20inferred%20from%2018S%20and%2028S%20ribosomal%20RNA%20sequences%20and%20a%20review%20of%20the%20evidence%20for%20amniote%20relationships.%20Mol%20Biol%20Evol.%201990%3B7%286%29%3A%20607%E2%80%93633.%202283953" data-title="Tetrapod%20phylogeny%20inferred%20from%2018S%20and%2028S%20ribosomal%20RNA%20sequences%20and%20a%20review%20of%20the%20evidence%20for%20amniote%20relationships" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/2283953" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Tetrapod+phylogeny+inferred+from+18S+and+28S+ribosomal+RNA+sequences+and+a+review+of+the+evidence+for+amniote+relationships+Hedges+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0157464.ref035" id="pone.0157464.ref035" class="link-target"></a>Evers R, Grummt I. Molecular coevolution of mammalian ribosomal gene terminator sequences and the transcription termination factor TTF-I. Proc Natl Acad Sci USA. 1995;92(13): 5827–5831. pmid:7597036 <ul class="reflinks"><li><a href="#" data-author="Evers" data-cit="EversR%2C%20GrummtI.%20Molecular%20coevolution%20of%20mammalian%20ribosomal%20gene%20terminator%20sequences%20and%20the%20transcription%20termination%20factor%20TTF-I.%20Proc%20Natl%20Acad%20Sci%20USA.%201995%3B92%2813%29%3A%205827%E2%80%935831.%207597036" data-title="Molecular%20coevolution%20of%20mammalian%20ribosomal%20gene%20terminator%20sequences%20and%20the%20transcription%20termination%20factor%20TTF-I" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/7597036" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Molecular+coevolution+of+mammalian+ribosomal+gene+terminator+sequences+and+the+transcription+termination+factor+TTF-I+Evers+1995" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0157464.ref036" id="pone.0157464.ref036" class="link-target"></a>Coleman AW. Analysis of mammalian rDNA internal transcribed spacers. PLoS One. 2013 Nov 19;8(11):e79122.  pmid:24260162 <ul class="reflinks"><li><a href="#" data-author="Coleman" data-cit="ColemanAW.%20Analysis%20of%20mammalian%20rDNA%20internal%20transcribed%20spacers.%20PLoS%20One.%202013%20Nov%2019%3B8%2811%29%3Ae79122.%20doi%3A%2010.1371%2Fjournal.pone.0079122%2024260162" data-title="Analysis%20of%20mammalian%20rDNA%20internal%20transcribed%20spacers" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/24260162" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Analysis+of+mammalian+rDNA+internal+transcribed+spacers+Coleman+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref37"><span class="order">37.
            </span><a name="pone.0157464.ref037" id="pone.0157464.ref037" class="link-target"></a>Bellemain E, Carlsen T, Brochmann C, Coissac E, Taberlet P, Kauserud H. ITS as an environmental DNA barcode for fungi: an in silico approach reveals potential PCR biases. BMC Microbiol. 2010 Jul 9;10:189.  pmid:20618939 <ul class="reflinks"><li><a href="#" data-author="Bellemain" data-cit="BellemainE%2C%20CarlsenT%2C%20BrochmannC%2C%20CoissacE%2C%20TaberletP%2C%20KauserudH.%20ITS%20as%20an%20environmental%20DNA%20barcode%20for%20fungi%3A%20an%20in%20silico%20approach%20reveals%20potential%20PCR%20biases.%20BMC%20Microbiol.%202010%20Jul%209%3B10%3A189.%20doi%3A%2010.1186%2F1471-2180-10-189%2020618939" data-title="ITS%20as%20an%20environmental%20DNA%20barcode%20for%20fungi%3A%20an%20in%20silico%20approach%20reveals%20potential%20PCR%20biases" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20618939" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=ITS+as+an+environmental+DNA+barcode+for+fungi%3A+an+in+silico+approach+reveals+potential+PCR+biases+Bellemain+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref38"><span class="order">38.
            </span><a name="pone.0157464.ref038" id="pone.0157464.ref038" class="link-target"></a>Yao H, Song J, Liu C, Luo K, Han J, Li Y, et al. Use of ITS2 region as the universal DNA barcode for plants and animals. PLoS One. 2010 Oct 1;5(10). pii: e13102.  <ul class="reflinks"><li><a href="#" data-author="Yao" data-cit="YaoH%2C%20SongJ%2C%20LiuC%2C%20LuoK%2C%20HanJ%2C%20LiY%2C%20et%20al.%20Use%20of%20ITS2%20region%20as%20the%20universal%20DNA%20barcode%20for%20plants%20and%20animals.%20PLoS%20One.%202010%20Oct%201%3B5%2810%29.%20pii%3A%20e13102.%20doi%3A%2010.1371%2Fjournal.pone.0013102" data-title="Use%20of%20ITS2%20region%20as%20the%20universal%20DNA%20barcode%20for%20plants%20and%20animals" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Use+of+ITS2+region+as+the+universal+DNA+barcode+for+plants+and+animals+Yao+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref39"><span class="order">39.
            </span><a name="pone.0157464.ref039" id="pone.0157464.ref039" class="link-target"></a>Chen S, Yao H, Han J, Liu C, Song J, Shi L, et al. Validation of the ITS2 region as a novel DNA barcode for identifying medicinal plant species. PLoS One. 2010 Jan 7;5(1):e8613.  pmid:20062805 <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenS%2C%20YaoH%2C%20HanJ%2C%20LiuC%2C%20SongJ%2C%20ShiL%2C%20et%20al.%20Validation%20of%20the%20ITS2%20region%20as%20a%20novel%20DNA%20barcode%20for%20identifying%20medicinal%20plant%20species.%20PLoS%20One.%202010%20Jan%207%3B5%281%29%3Ae8613.%20doi%3A%2010.1371%2Fjournal.pone.0008613%2020062805" data-title="Validation%20of%20the%20ITS2%20region%20as%20a%20novel%20DNA%20barcode%20for%20identifying%20medicinal%20plant%20species" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/20062805" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=Validation+of+the+ITS2+region+as+a+novel+DNA+barcode+for+identifying+medicinal+plant+species+Chen+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref40"><span class="order">40.
            </span><a name="pone.0157464.ref040" id="pone.0157464.ref040" class="link-target"></a>Wang XC, Liu C, Huang L, Bengtsson-Palme J, Chen H, Zhang JH, Cai D, et al. ITS1: a DNA barcode better than ITS2 in eukaryotes? Mol Ecol Resour. 2015;15(3): 573–586.  pmid:25187125 <ul class="reflinks"><li><a href="#" data-author="Wang" data-cit="WangXC%2C%20LiuC%2C%20HuangL%2C%20Bengtsson-PalmeJ%2C%20ChenH%2C%20ZhangJH%2C%20CaiD%2C%20et%20al.%20ITS1%3A%20a%20DNA%20barcode%20better%20than%20ITS2%20in%20eukaryotes%3F%20Mol%20Ecol%20Resour.%202015%3B15%283%29%3A%20573%E2%80%93586.%20doi%3A%2010.1111%2F1755-0998.12325%2025187125" data-title="ITS1%3A%20a%20DNA%20barcode%20better%20than%20ITS2%20in%20eukaryotes%3F" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://www.ncbi.nlm.nih.gov/pubmed/25187125" target="_new" title="Go to article in PubMed">
                        PubMed/NCBI
                      </a></li><li><a href="http://scholar.google.com/scholar?q=ITS1%3A+a+DNA+barcode+better+than+ITS2+in+eukaryotes%3F+Wang+2015" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

