



<div class="abstract toc-section"><a id="abstract0" name="abstract0" data-toc="abstract0" class="link-target" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1" class="link-target"></a><p>RNAi (RNA interference) relies on the production of small RNAs (sRNAs) from double-stranded RNA and comprises a major pathway in eukaryotes to restrict the propagation of selfish genetic elements. Amplification of the initial RNAi signal by generation of multiple secondary sRNAs from a targeted mRNA is catalyzed by RNA-dependent RNA polymerases (RdRPs). This phenomenon is known as transitivity and is particularly important in plants to limit the spread of viruses. Here we describe, using a genome-wide approach, the distribution of sRNAs in the glaucophyte alga <em>Cyanophora paradoxa</em>. <em>C. paradoxa</em> is a member of the supergroup Plantae (also known as Archaeplastida) that includes red algae, green algae, and plants. The ancient (&gt;1 billion years ago) split of glaucophytes within Plantae suggests that <em>C. paradoxa</em> may be a useful model to learn about the early evolution of RNAi in the supergroup that ultimately gave rise to plants. Using next-generation sequencing and bioinformatic analyses we find that sRNAs in <em>C. paradoxa</em> are preferentially associated with mRNAs, including a large number of transcripts that encode proteins arising from different functional categories. This pattern of exonic sRNAs appears to be a general trend that affects a large fraction of mRNAs in the cell. In several cases we observe that sRNAs have a bias for a specific strand of the mRNA, including many instances of antisense predominance. The genome of <em>C. paradoxa</em> encodes four sequences that are homologous to RdRPs in <em>Arabidopsis thaliana</em>. We discuss the possibility that exonic sRNAs in the glaucophyte may be secondarily derived from mRNAs by the action of RdRPs. If this hypothesis is confirmed, then transitivity may have had an ancient origin in Plantae.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Gross J, Wajid S, Price DC, Zelzion E, Li J, Chan CX, et al.  (2013) Evidence for Widespread Exonic Small RNAs in the Glaucophyte Alga <em>Cyanophora paradoxa</em>. PLoS ONE 8(7):
           e67669.
        
        https://doi.org/10.1371/journal.pone.0067669</p><p><strong>Editor: </strong>Stefan Maas, NIGMS, NIH, United States of America</p><p><strong>Received: </strong>February 7, 2013; <strong>Accepted: </strong>May 21, 2013; <strong>Published: </strong> July 3, 2013</p><p><strong>Copyright: </strong> © 2013 Gross et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was made possible by grants from the National Science Foundation (MGSP 0625440, MCB 0946528) awarded to DB. Partial support was provided to DB by Rutgers University, New Brunswick. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests: </strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section toc-section"><a id="s1" name="s1" data-toc="s1" class="link-target" title="Introduction"></a><h2>Introduction</h2><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1" class="link-target"></a><p>Eukaryotic genomes are “parasitized” by self-replicating genetic entities such as viruses and transposable elements (TEs, comprising transposons and retrotransposons). To suppress the intragenomic spread of such selfish replicons, eukaryotes have evolved diverse defensive strategies, among the most important of which is RNAi <a href="#pone.0067669-Obbard1" class="ref-tip">[1]</a>, <a href="#pone.0067669-Mallory1" class="ref-tip">[2]</a>. This system relies on the recognition of double-stranded RNA (dsRNA) species, found in intermediary replication forms of virus, hairpin structures, or complementary antisense transcripts associated with TEs <a href="#pone.0067669-Obbard1" class="ref-tip">[1]</a>, <a href="#pone.0067669-Caplen1" class="ref-tip">[3]</a>, <a href="#pone.0067669-Hammond1" class="ref-tip">[4]</a>. Such dsRNAs are processed by Dicer-type RNase III proteins to produce sRNAs that are usually 20–23 nucleotides in length <a href="#pone.0067669-Hammond2" class="ref-tip">[5]</a>. The resulting sRNAs are loaded into a RNA-induced silencing complex (RISC), formed by Argonaute-type proteins (AGO), that recognize viral RNAs or TE-derived transcripts based on the sRNA-to-target base pair complementarity <a href="#pone.0067669-Mallory1" class="ref-tip">[2]</a>. The silencing is accomplished by diverse mechanisms such as inhibition of transcription (e.g., by DNA or histone methylation), cleavage of the mRNA, or by blocking transcript translation <a href="#pone.0067669-Hammond1" class="ref-tip">[4]</a>, <a href="#pone.0067669-Kawasaki1" class="ref-tip">[6]</a>, <a href="#pone.0067669-Lytle1" class="ref-tip">[7]</a>, <a href="#pone.0067669-Fabian1" class="ref-tip">[8]</a>. The RNAi process, whereby sRNAs are formed from perfect dsRNAs and act <em>in cis</em> by pairing to their cognate producing transcripts is referred to as the small interfering RNA (siRNA) pathway <a href="#pone.0067669-Obbard1" class="ref-tip">[1]</a>. MicroRNAs (miRNA) also represent a class of sRNAs widespread in eukaryotic genomes that probably evolved from the ancestral siRNA pathway <a href="#pone.0067669-Obbard1" class="ref-tip">[1]</a>, <a href="#pone.0067669-Piriyapongsa1" class="ref-tip">[9]</a>. However miRNAs differ from siRNAs by being derived from self-complementary fold-back precursor RNAs in which a few mismatches are included. These sRNAs recognize their targets by near-perfect complementarity and act primarily <em>in trans</em> to silence endogenous host protein-encoding transcripts <a href="#pone.0067669-Lytle1" class="ref-tip">[7]</a>, <a href="#pone.0067669-Fabian1" class="ref-tip">[8]</a>, <a href="#pone.0067669-Piriyapongsa1" class="ref-tip">[9]</a>. A further class of sRNA found in plants, nematodes, and in the fungus <em>Mucor circinelloides</em> is associated with the amplification of the RNAi signal that is mediated by RNA-dependent RNA polymerases (RdRPs) in a phenomenon referred to as transitivity <a href="#pone.0067669-Calo1" class="ref-tip">[10]</a>, <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>. The process is initiated by primary siRNAs or miRNAs that in an AGO-dependent fashion target viral RNAs or specific transcripts in the cell (e.g., trans-acting siRNA [tasiRNA] transcripts in plants), triggering the recruitment of an RdRP to drive the synthesis of a complementary (antisense) strand to the targeted transcript <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Chen1" class="ref-tip">[12]</a>, <a href="#pone.0067669-Manavella1" class="ref-tip">[13]</a>, <a href="#pone.0067669-Allen1" class="ref-tip">[14]</a>. The resulting duplex is subsequently processed by Dicer homologs into diverse secondary siRNAs that usually cover the length of the original transcript in patterns of phased registers. Secondary siRNAs reinforce silencing of the primary targeted RNA or, in the case of tasiRNA in higher plants, modulate expression of protein-coding mRNAs in <em>trans</em> <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Allen1" class="ref-tip">[14]</a>. Therefore, transitivity yields the spread and amplification of the RNAi pathway that results in a robust form of defense against viral RNAs.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2" class="link-target"></a><p>Differentially expressed sRNAs in response to fluctuating environmental conditions are already known in some prokaryotes such as marine strains of the cyanobacterium <em>Synechococcus</em> <a href="#pone.0067669-Gierga1" class="ref-tip">[15]</a>. In eukaryotes, RNAi is believed to be an ancient process that probably evolved before the basal split of the major supergroups <a href="#pone.0067669-Shabalina1" class="ref-tip">[16]</a>, <a href="#pone.0067669-Cerutti1" class="ref-tip">[17]</a>. In some lineages, reductive evolution resulted in partial or complete loss of the RNAi pathway, whereas in many multicellular organisms, amplification of the molecular components involved in RNAi contributed to the evolution of novel and intricate regulatory functions <a href="#pone.0067669-Obbard1" class="ref-tip">[1]</a>, <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Shabalina1" class="ref-tip">[16]</a>, <a href="#pone.0067669-Cerutti1" class="ref-tip">[17]</a>. For example, <em>Arabidopsis thaliana</em> encodes 4 Dicers, 10 Argonautes, and 6 RdRP homologs, reflecting an elaborate RNAi system that operates in plants <a href="#pone.0067669-Mallory1" class="ref-tip">[2]</a>, <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>. These data raise an important question: was RNAi co-opted from its original role as a molecular immune system against selfish elements to ultimately foster increasingly complex regulatory networks in the cell? RNAi and transitivity are also found in the moss <em>Physcomitrella patens</em> and the green alga <em>Chlamydomonas reinhardtii</em> <a href="#pone.0067669-TalmorNeiman1" class="ref-tip">[18]</a>, <a href="#pone.0067669-Molnar1" class="ref-tip">[19]</a>, <a href="#pone.0067669-Arif1" class="ref-tip">[20]</a>. Other algae, such as <em>Phaeodactylum tricornutum</em>, <em>Thalassiosira pseudonana</em>, <em>Pyropia yezoensis</em>, <em>Ectocarpus siliculosus</em>, and diverse protists also produce siRNAs and miRNAs that potentially regulate molecular processes in the cell <a href="#pone.0067669-Cerutti1" class="ref-tip">[17]</a>, <a href="#pone.0067669-Huang1" class="ref-tip">[21]</a>, <a href="#pone.0067669-NordenKrichmar1" class="ref-tip">[22]</a>, <a href="#pone.0067669-Liang1" class="ref-tip">[23]</a>, <a href="#pone.0067669-Cock1" class="ref-tip">[24]</a>. Therefore, the extent to which protists in general, and in particular algae already have a full breadth of regulatory systems based on RNAi remains an open question. To address this issue, we carried out a genome-wide analysis of sRNAs in the glaucophyte alga <em>Cyanophora paradoxa</em>. Because glaucophytes putatively occupy a basal phylogenetic position in the supergroup Plantae (also known as Archaeplastida) the RNAi system in this lineage may reflect ancestral and/or divergent features of the RNAi system described in green algae and plants <a href="#pone.0067669-Price1" class="ref-tip">[25]</a>, <a href="#pone.0067669-ReyesPrieto1" class="ref-tip">[26]</a>. Therefore <em>C. paradoxa</em> offers an important model to study the evolution of sRNA-guided silencing in Plantae and more generally to understand the complexity of RNAi pathways in protists.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2" name="s2" data-toc="s2" class="link-target" title="Materials and Methods"></a><h2>Materials and Methods</h2>
<div id="section1" class="section toc-section"><a id="s2a" name="s2a" class="link-target" title="Growth Conditions"></a>
<h3>Growth Conditions</h3>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1" class="link-target"></a><p><em>Cyanophora paradoxa</em> (CCMP329) was cultivated at the Provasoli-Guillard National Center for Marine Algae and Microbiota (formerly CCMP). Using normal, replete growth conditions, cells were propagated under continuous light in standard DY-V medium to high density. Cells were then harvested by centrifugation and frozen in liquid nitrogen. To apply a high light shock, aliquots of high-density cultures were further exposed to daylight irradiance for 40 minutes before being frozen. Others aliquots were taken after 300 mM salt (NaCl) treatment for 30 minutes, and exposure to –2°C temperature for 40 minutes. These shock conditions represent strong environmental stimuli during relatively brief time intervals to ensure that differential gene expression was effectively elicited by the stresses.</p>
</div>

<div id="section2" class="section toc-section"><a id="s2b" name="s2b" class="link-target" title="Library Preparation and Sequencing"></a>
<h3>Library Preparation and Sequencing</h3>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1" class="link-target"></a><p>The <em>TRIzol</em> method (Life Technologies, Grand Island, NY USA) was used to isolate total RNA from the replete-grown cells and from cultures submitted to light-, salt-, and cold-shock. RNA extracted from each treatment condition was fractionated by electrophoresis in 12% denaturing (7 M urea) polyacrylamide gel. Excision of a gel band in the range of 15–35 nt was used to select the fraction containing the cellular sRNAs of <em>C. paradoxa</em>. The <em>miRCat Small RNA Cloning Kit</em> (IDT, Coralville, IA, USA) was used to prepare sRNA libraries for each treatment condition according to the manufacturer’s instructions. A further PCR amplification step added indexed-<em>Illumina</em> (San Diego, CA, USA) adaptors necessary for sequencing on the GAIIx instrument. The mRNAs were purified from total RNA using <em>Dynabeads</em> (Life Technologies, Grand Island, NY USA). Degradome libraries for each treatment condition were prepared by ligation of a RNA linker to the 5′ phosphate of cleaved mRNAs <a href="#pone.0067669-AddoQuaye1" class="ref-tip">[27]</a>. The sequence of this linker (5′-ACACTCTTTCCCTACACGACGCTCTTCCGATCTNNNN-3′) corresponded to the <em>Illumina</em> Adapter 1 containing a customized index to identify the library. Reverse transcription was performed using an oligo containing a poly-d(T) fused to the Adapter 2 of <em>Illumina</em> (<span class="gene">5′-CTCGGCATTCCTGCTGAACCGCTCTTCCGATCTTTTTTTTTTTTTTTTTTTTTTTT-3′</span>). An enrichment step was performed using standard <em>Illumina</em> PCR primers. For preparation of RNA-seq libraries, cDNA was first synthesized from mRNAs of normal- and salt-treated cultures using the <em>Mint cDNA</em> synthesis kit (Evrogen, Moscow, Russia), and then converted to RNA-seq libraries by transposon tagmentation using the <em>Nextera</em> (Illumina, San Diego, CA, USA) DNA sample preparation kit. All libraries were sequenced on the <em>Illumina</em> GAIIx platform according to the manufacturer’s instructions.</p>
</div>

<div id="section3" class="section toc-section"><a id="s2c" name="s2c" class="link-target" title="Bioinformatics"></a>
<h3>Bioinformatics</h3>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1" class="link-target"></a><p>A bioinformatics pipeline for filtering sRNAs was established to demultiplex the sequenced sRNA libraries, trim the adaptor and barcodes, select sRNAs sequences of size 16–32 nt, and eliminate sRNAs that mapped to databases of rRNA and tRNAs and the plastid genome of <em>C. paradoxa</em>. Sequences of sRNAs and degradome tags were deposited at NCBI under the BioProject accession number PRJNA12793. Using the <em>CLC Genomics Workbench 5.5</em> (CLC bio, Aarhus, Denmark) the filtered sRNAs were mapped against the genome contigs, EST contigs, and predicted protein-coding sequences (CDSs) of <em>C. paradoxa</em>. Only full-length matches with 100% identity were considered for downstream analysis and gene expression studies. These sequences represent therefore, nuclear-encoded sRNAs in <em>C. paradoxa</em>. Degradome libraries were sequenced using the 50 bp single-read chemistry of <em>Illumina</em>. The resulting degradome data were also filtered to eliminate adaptors, library indexes, and rRNA/plastome hits. In addition, poly-A tails were removed from the degradome data when necessary. The same criterion of 100% identity was used to align degradome tags to the genome data, ESTs, and CDSs using the mapping tools of the <em>CLC Genomics Workbench.</em> This bioinformatics platform was also used for several other analysis reported here, such as quantification of sRNAs and degradome expression and sRNA redundancy analysis. The miRNAs were identified using <em>miRDeep</em> <a href="#pone.0067669-Friedlander1" class="ref-tip">[28]</a>, and sRNA targets in the CDS of <em>C. paradoxa</em> were predicted using <em>CleaveLand 3</em> <a href="#pone.0067669-AddoQuaye2" class="ref-tip">[29]</a>.</p>
</div>
</div>

<div id="section3" class="section toc-section"><a id="s3" name="s3" data-toc="s3" class="link-target" title="Results"></a><h2>Results</h2>
<div id="section1" class="section toc-section"><a id="s3a" name="s3a" class="link-target" title="Discovery of sRNAs in Cyanophora paradoxa"></a>
<h3>Discovery of sRNAs in <em>Cyanophora paradoxa</em></h3>
<a id="article1.body1.sec3.sec1.p1" name="article1.body1.sec3.sec1.p1" class="link-target"></a><p>We recently produced a draft genome of <em>C. paradoxa</em> <a href="#pone.0067669-Price1" class="ref-tip">[25]</a>. Homology searches using BLAST <a href="#pone.0067669-Altschul1" class="ref-tip">[30]</a> against the assembled genome contigs of <em>C. paradoxa</em> reveal the presence of molecular components putatively involved in RNAi. When used as a query, the <em>A. thaliana</em> Dicer protein (AtDCL1, accession number NP_171612) retrieves a significant BLASTP hit (<em>e</em>-value 4×10<sup>−9</sup>) with a gene model predicted within the genome contig 53704, whereas sequences homologous to Argonaut (AtAGO1, AAC18440) are found on contigs 10646 and 53267, respectively. Surprisingly, the genome of <em>C. paradoxa</em> also encodes several putative homologs of <em>A. thaliana</em> RNA-dependent RNA polymerase (AtRDR1, NP_172932) located on genome contigs 42, 27852, 39851, and 9284. The presence in the <em>C. paradoxa</em> genome of genes homologous to key molecular components involved in RNAi provides strong evidence of an analogous system operating in this glaucophyte alga <a href="#pone.0067669-Cerutti1" class="ref-tip">[17]</a>. We sought therefore to characterize sRNAs from <em>C. paradoxa</em>. To accomplish this aim, four cDNA libraries were prepared from size-selected sRNAs extracted from cold-, light-, and salt-stressed algal cultures, and from cells grown under normal, replete conditions. These libraries were sequenced using the <em>Illumina</em> GAIIx platform. Upon filtering a total of 4,739,151 reads matched against genome data, EST contigs, and the predicted CDS from <em>C. paradoxa</em> (<a href="#pone-0067669-t001">Table 1</a> and <a href="#pone-0067669-g001">Figure 1A</a>). Mapped sRNAs had the predominant size of 21 nt (<a href="#pone-0067669-g001">Figure 1B</a>) and a slight overrepresentation of adenine and uracil in the first base (<a href="#pone-0067669-g001">Figure 1C</a>). The widespread abundance of sRNAs and common characteristics suggest that <em>C. paradoxa</em> sRNAs have features also present in plants <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>.</p>
<a class="link-target" id="pone-0067669-g001" name="pone-0067669-g001"></a><div class="figure" data-doi="10.1371/journal.pone.0067669.g001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0067669.g001" data-doi="info:doi/10.1371/journal.pone.0067669" data-uri="info:doi/10.1371/journal.pone.0067669.g001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0067669.g001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 1. </span> Mapping and composition features of sRNAs discovered in <em>Cyanophora paradoxa</em>.</div><p class="caption_target"><a id="article1.body1.sec3.sec1.fig1.caption1.p1" name="article1.body1.sec3.sec1.fig1.caption1.p1" class="link-target"></a><p>(A) The results of mapping redundant (above the x-axis) and unique (below the x-axis) sRNAs to genomic contigs, EST contigs, and CDSs. The numbers on the top of the colored bars correspond to the number of genomic contigs (blue), EST contigs (red) and CDSs (green) associated with the range of sRNAs counts displayed at the top of the panel. (B) Size distribution of redundant sRNAs in <em>C. paradoxa</em> indicates the predominance of the 21nt length class. (C) Composition of the 5′ nucleotide of unique sRNAs.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0067669.g001">
              https://doi.org/10.1371/journal.pone.0067669.g001</a></p></div><a class="link-target" id="pone-0067669-t001" name="pone-0067669-t001"></a><div class="figure" data-doi="10.1371/journal.pone.0067669.t001"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0067669.t001" data-doi="info:doi/10.1371/journal.pone.0067669" data-uri="info:doi/10.1371/journal.pone.0067669.t001"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0067669.t001" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.t001">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.t001">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.t001">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.t001">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.t001">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.t001">
                original image
              </a></li></ul></div><div class="figcaption"><span>Table 1. </span> The distribution of 4,739,151 sRNAs that were mapped to genome contigs, CDS sequences, and EST contigs.</div><p class="caption_target"></p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0067669.t001">
              https://doi.org/10.1371/journal.pone.0067669.t001</a></p></div></div>

<div id="section2" class="section toc-section"><a id="s3b" name="s3b" class="link-target" title="Transcriptome-wide Mapping Reveals a Pathway for Exonic sRNA Production"></a>
<h3>Transcriptome-wide Mapping Reveals a Pathway for Exonic sRNA Production</h3>
<a id="article1.body1.sec3.sec2.p1" name="article1.body1.sec3.sec2.p1" class="link-target"></a><p>We mapped the four sRNA libraries to a data set of genome contigs, predicted CDSs, and EST contigs with size greater than 199 bp (<a href="#pone-0067669-t001">Table 1</a>, <a href="#pone-0067669-g001">Figure 1A</a>, and <a href="#pone.0067669.s003">Table S1</a>). More than one-half of the genome contigs (36,241 contigs) have at least one sRNA mapped to them. Surprisingly, a large fraction of the sRNAs is associated with a few regions of the genome. For instance, 3,749,680 redundant sRNAs map to only 305 of the total number of 60,119 genome contigs (<a href="#pone-0067669-g001">Figure 1A</a>). These 305 contigs, which encompass only 2.3% of the sequenced genome, are therefore associated with 83.13% of the total sRNAs, and likely represent hotspots for sRNA production. The mapping also reveals that sRNA production is primarily associated with transcribed regions. A total of 2,302,640 sRNAs (approximately 48.5% of the total) were mapped to over 70.0% of the selected EST contigs (<a href="#pone-0067669-g001">Figure 1A</a>, red bar in the middle). In addition, about 75.0% of the predicted CDSs have sRNAs associated with them, indicating that a large fraction of the sRNAs derive from exonic regions (<a href="#pone-0067669-g001">Figure 1A</a>, <a href="#pone.0067669.s003">Table S1</a>). In many instances, sRNAs that are associated with transcripts are biased to a specific DNA strand (e.g., genome contig 37295 and EST contig 56375 shown in <a href="#pone-0067669-g002">Figures 2A and 2B</a>, respectively). The correct transcriptional orientation can be determined using the degradome-seq data (<a href="#pone.0067669.s004">Table S2</a>) because the libraries were 5′ to 3′ directionally cloned <a href="#pone.0067669-AddoQuaye1" class="ref-tip">[27]</a>. In addition, the orientation of transcription is apparent using the predicted CDS. Based on these approaches, we found that many of the ESTs and CDSs have sRNAs mapped in an antisense orientation, ruling out the possibility that strand bias of sRNAs represents degradation of highly expressed mRNAs (<a href="#pone-0067669-g003">Figure 3</a>). Consistent with this idea, available mRNA-seq data from <em>C. paradoxa</em> indicate that some exonic sRNAs are associated with CDSs that have no expression data (<a href="#pone.0067669.s001">Figure S1</a>). Conversely, genes with high expression and degradation turnover do not necessarily give rise to sRNAs. Taken together these observations support the idea that a pathway for sRNA production from exons is present in <em>C. paradoxa</em>.</p>
<a class="link-target" id="pone-0067669-g002" name="pone-0067669-g002"></a><div class="figure" data-doi="10.1371/journal.pone.0067669.g002"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0067669.g002" data-doi="info:doi/10.1371/journal.pone.0067669" data-uri="info:doi/10.1371/journal.pone.0067669.g002"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0067669.g002" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g002">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g002">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g002">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g002">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g002">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g002">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 2. </span> Two examples of sRNA and degradome mapping to <em>C. paradoxa</em> genome data.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig1.caption1.p1" name="article1.body1.sec3.sec2.fig1.caption1.p1" class="link-target"></a><p>(A) Mapping of sRNAs (red) and degradome tags (black) to genome contig 37295. (B) Mapping of sRNAs (red) and degradome tags (black) to EST contig 56375.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0067669.g002">
              https://doi.org/10.1371/journal.pone.0067669.g002</a></p></div><a class="link-target" id="pone-0067669-g003" name="pone-0067669-g003"></a><div class="figure" data-doi="10.1371/journal.pone.0067669.g003"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0067669.g003" data-doi="info:doi/10.1371/journal.pone.0067669" data-uri="info:doi/10.1371/journal.pone.0067669.g003"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0067669.g003" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g003">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g003">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g003">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g003">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g003">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g003">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 3. </span> The sRNAs associated with CDSs show orientation bias.</div><p class="caption_target"><a id="article1.body1.sec3.sec2.fig2.caption1.p1" name="article1.body1.sec3.sec2.fig2.caption1.p1" class="link-target"></a><p>The percent of sRNAs that mapped in the sense (blue curve) or antisense (red curve) direction of 6,519 CDS selected on the basis of prevalent (&gt;75%) degradome sense transcription or absence of antisense degradome transcription.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0067669.g003">
              https://doi.org/10.1371/journal.pone.0067669.g003</a></p></div><a id="article1.body1.sec3.sec2.p2" name="article1.body1.sec3.sec2.p2" class="link-target"></a><p>Notably, sRNAs in <em>C. paradoxa</em> are associated with protein-coding genes (<a href="#pone.0067669.s003">Table S1</a>). To get a better picture of this unusual pattern of sRNA distribution, we selected 6,519 CDSs for which degradome tags corroborate the orientation of the protein-coding gene (<a href="#pone-0067669-g003">Figure 3</a>). All these 6,519 CDSs have ≥10 sRNAs, many CDSs are associated with a large number of mapped sRNAs (the top 16 CDS in <a href="#pone.0067669.s003">Table S1</a> have more than 10,000 mapped sRNAs). We found that 1,377 CDSs display a bias for antisense sRNA production (<a href="#pone-0067669-g003">Figure 3</a>). At least 107 CDSs have only antisense sRNA associated with them and over 900 CDSs have &gt;20 antisense sRNAs. The top CDSs with the most sRNAs mapped to them show a strong predominance of antisense mapping (<a href="#pone.0067669.s003">Table S1</a>). In contrast, there are over 5,000 CDSs in which sRNAs are predominantly mapped in the sense orientation (<a href="#pone-0067669-g003">Figure 3</a>). About 3,500 of these CDSs also have antisense associated sRNAs and therefore may represent actual sites for sRNA production rather than a product of mRNA degradation.</p>
<a id="article1.body1.sec3.sec2.p3" name="article1.body1.sec3.sec2.p3" class="link-target"></a><p>This transcriptome-wide pattern of exonic sRNA with biased orientation remains unaffected when mapping unique (non-redundant) sRNAs to the 6,519 CDS (<a href="#pone.0067669.s002">Figure S2</a>), confirming that the production of sRNAs tends to spread across the transcript. We feel it is unlikely that these results are an artifact of cDNA over-amplification of a few sRNAs that map to particular sites of the CDS. Production of sRNAs across the length of an mRNA is consistent with the action of RdRPs <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Chen1" class="ref-tip">[12]</a>, four of which are potentially encoded in the genome of <em>C. paradoxa</em>. Although in some instances, we observed a pattern of blocks of mapped sRNAs over separate regions of the mRNA (e.g., EST contig 56375; <a href="#pone-0067669-g002">Figure 2B</a>), we did not detect a phasing distribution that is a typical signature of secondary siRNAs in plants <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Chen1" class="ref-tip">[12]</a>. The pattern of exonic sRNAs seems to be a general trend affecting a large fraction of cellular mRNAs. For example, predicted genes with associated sRNAs encode proteins of diverse cellular functions including translation, signal transduction, and varied metabolic and cell biological functions (<a href="#pone.0067669.s003">Table S1</a>). Interestingly, the top CDSs with abundant sRNA associated with them do not have BLAST hits at NCBI therefore their identity and function are unknown (<a href="#pone.0067669.s003">Table S1</a>). This is also the case for many of the remaining sRNA-associated CDSs.</p>
</div>

<div id="section3" class="section toc-section"><a id="s3c" name="s3c" class="link-target" title="Degradome Analysis Indicates the Predominance of a 5′ to 3′ mRNA Turnover Pathway"></a>
<h3>Degradome Analysis Indicates the Predominance of a 5′ to 3′ mRNA Turnover Pathway</h3>
<a id="article1.body1.sec3.sec3.p1" name="article1.body1.sec3.sec3.p1" class="link-target"></a><p>To gain insights into possible sRNA mediation of gene expression, we generated 5,166,650 degradome tags that were mapped to the genome contigs, CDSs, and EST contigs of <em>C. paradoxa</em> (<a href="#pone.0067669.s004">Table S2</a>). Of these, 836,255 degradome tags (from 25–45 nucleotides in length) had perfect matches to the predicted CDSs, thereby allowing for identification of reading-frame orientation. These tags were also tentatively used for the identification of possible sites of sRNA-mediated cleavage <a href="#pone.0067669-AddoQuaye2" class="ref-tip">[29]</a>. For this purpose, the degradome dataset was remapped to determine the 5′-phosphorylated position of the tag captured by our cloning procedure. This allowed us to generate for each CDS a graphic plot of peaks that represents the putative sites of mRNA cleavage (e.g., CDS 7658.3 in <a href="#pone-0067669-g004">Figure 4A</a>). The tendency for degradome peaks to be concentrated at 3′ ends of transcripts was revealed by plotting the relative frequency as a function of the distance to the stop codon of CDSs (<a href="#pone-0067669-g004">Figure 4B</a>). This bias in the distribution of degradome peaks is also found in <em>A. thaliana</em> and likely indicates overrepresentation of 5′ to 3′ exonucleolytic mRNA turnover products <a href="#pone.0067669-AddoQuaye1" class="ref-tip">[27]</a>, <a href="#pone.0067669-German1" class="ref-tip">[31]</a>.</p>
<a class="link-target" id="pone-0067669-g004" name="pone-0067669-g004"></a><div class="figure" data-doi="10.1371/journal.pone.0067669.g004"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0067669.g004" data-doi="info:doi/10.1371/journal.pone.0067669" data-uri="info:doi/10.1371/journal.pone.0067669.g004"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0067669.g004" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g004">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g004">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g004">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g004">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g004">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g004">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 4. </span> Profile of the <em>C. paradoxa</em> degradome tags mapped to predicted CDSs.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig1.caption1.p1" name="article1.body1.sec3.sec3.fig1.caption1.p1" class="link-target"></a><p>(A) Example of degradome peaks distributed over CDS 7658.3. The blue peaks are tag counts &lt;3x of the mean total number of peaks. The red peaks are tag counts ≥3x the mean total number of peaks. (B) Average contribution of total degradome peaks as a function of relative distance from the stop codon of CDSs. For this analysis, all individual CDSs tag counts for each position were numbered from the stop codon and normalized as a relative frequency to the total tag counts in that CDS. Then the total relative frequency for each position (relative to the stop codon) was calculated as an average number; i.e., the sum of individual relative frequencies in that position of all CDSs divided by the total number of CDSs. (C) For each <em>C. paradoxa</em> CDS, peaks with relative frequency of degradome tags ≥3x the mean total number of counts were retrieved (e.g., the red peaks for CDS 7658.6 on panel A). This resulted in 7,995 peaks from all CDSs (first bar on the left). The high-frequency degradome peaks are displayed as a function of the factor (3x–13x) by which their relative frequency surmounts the mean tag count found on the respective CDSs.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0067669.g004">
              https://doi.org/10.1371/journal.pone.0067669.g004</a></p></div><a id="article1.body1.sec3.sec3.p2" name="article1.body1.sec3.sec3.p2" class="link-target"></a><p>In plants, the peaks of the degradome frequency distribution are usually significantly higher than the background level of exonucleolytic degradation <a href="#pone.0067669-German1" class="ref-tip">[31]</a>, <a href="#pone.0067669-Pantaleo1" class="ref-tip">[32]</a>. Therefore, to capture in <em>C. paradoxa</em> the potential signal from degradome peaks that may represent sRNA-mediated endonucleolytic products, we selected all peaks that were above background frequencies for each CDS. This was done by removing peaks that were below three-fold of the mean frequency of normalized counts per CDS. This procedure resulted in 7,995 peaks for downstream analysis (<a href="#pone-0067669-g004">Figure 4C</a>). We used a BLAST alignment approach to search for possible significant overlaps between sRNAs and the 7,995 high-frequency peaks. As a negative control, we generated 7,995 random peaks that were aligned to sRNAs using the same BLAST procedure. This analysis did not show a significant difference between the high-frequency group and the randomized dataset with respect to the number of sRNAs overlapping the degradome tags (results not shown). This analysis indicates that no obvious correlation exists between sRNAs and degradome peaks in <em>C. paradoxa</em>. To investigate the possibility that a few individual sRNAs may be associated with degradome peaks, we analyzed 100 highly expressed sRNAs in greater detail. We also used the <em>miRDeep</em> program <a href="#pone.0067669-Friedlander1" class="ref-tip">[28]</a> to identify 32 putative miRNAs in the genome of <em>C. paradoxa</em>. By running a targeting prediction under <em>CleaveLand 3</em> <a href="#pone.0067669-AddoQuaye2" class="ref-tip">[29]</a> we did not detect significant matches between our data set of 100 highly expressed sRNAs and 32 miRNAs to degradome tags when mapped to the CDSs. Our inability to find sRNA targets using different bioinformatic approaches suggests the possibility that in <em>C. paradoxa</em> sRNAs do not mediate mRNA cleavage. However, by analyzing expression profiles, we observed that exonic sRNAs related to CDSs appear to be differentially expressed between normal and stress conditions (i.e., salt, cold, and high light; <a href="#pone-0067669-g005">Figure 5</a>). To ensure that our analysis reflects meaningful differences in gene expression we constrained the analyzed dataset to CDSs that have 100 or more normalized sRNA counts in at least one of the four libraries (<a href="#pone-0067669-g005">Figure 5</a>). We found that 312 CDSs have at least 10-fold sRNA expression differences between the normal and one of the stress libraries, suggesting that the regulation of sRNA expression is influenced by environmental factors.</p>
<a class="link-target" id="pone-0067669-g005" name="pone-0067669-g005"></a><div class="figure" data-doi="10.1371/journal.pone.0067669.g005"><div class="img-box"><a title="Click for larger image" href="article/figure/image?size=medium&amp;id=info:doi/10.1371/journal.pone.0067669.g005" data-doi="info:doi/10.1371/journal.pone.0067669" data-uri="info:doi/10.1371/journal.pone.0067669.g005"><img src="article/figure/image?size=inline&amp;id=info:doi/10.1371/journal.pone.0067669.g005" alt="thumbnail" class="thumbnail"></a><div class="expand"></div></div><div class="figure-inline-download">
          Download:
          <ul><li><div class="definition-label"><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g005">
                  PPT
                </a></div><a href="article/figure/powerpoint?id=info:doi/10.1371/journal.pone.0067669.g005">
                PowerPoint slide
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g005">
                  PNG
                </a></div><a href="article/figure/image?download&amp;size=large&amp;id=info:doi/10.1371/journal.pone.0067669.g005">
                larger image
              </a></li><li><div class="definition-label"><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g005">
                  TIFF
                </a></div><a href="article/figure/image?download&amp;size=original&amp;id=info:doi/10.1371/journal.pone.0067669.g005">
                original image
              </a></li></ul></div><div class="figcaption"><span>Figure 5. </span> Expression analysis of sRNAs associated with CDSs.</div><p class="caption_target"><a id="article1.body1.sec3.sec3.fig2.caption1.p1" name="article1.body1.sec3.sec3.fig2.caption1.p1" class="link-target"></a><p>(A) Heat map showing results of cluster analysis of 312 filtered CDSs with differential expression of sRNAs. The filtering criteria were at least 100 normalized (reads per kilobase per million, RPKM) counts in one of the conditions and a difference of 10x fold or more of expression in at least one stress condition when compared to the normal. (B) Bars indicating how many differentially expressed CDSs are up-regulated or down-regulated under normal versus stress conditions, how many have known/unknown function, and the number of CDSs that have predominantly sense or antisense associated sRNAs.</p>
</p><p class="caption_object"><a href="https://doi.org/10.1371/journal.pone.0067669.g005">
              https://doi.org/10.1371/journal.pone.0067669.g005</a></p></div></div>
</div>

<div id="section4" class="section toc-section"><a id="s4" name="s4" data-toc="s4" class="link-target" title="Discussion"></a><h2>Discussion</h2><a id="article1.body1.sec4.p1" name="article1.body1.sec4.p1" class="link-target"></a><p>Here we describe genome-wide analysis of sRNAs in the glaucophyte alga <em>C. paradoxa</em>. The sRNAs we isolated mapped to over 70% of the EST contigs and to 75% of the predicted CDSs, demonstrating that sRNA production is associated with mRNA sequences in this species. The following lines of evidence indicate that the identified sRNAs are authentic exonic sRNAs rather than degradation products of abundantly expressed mRNA captured by our library construction procedure: (i) we found a global pattern of antisense sRNA biogenesis (<a href="#pone-0067669-g003">Figure 3</a>, <a href="#pone.0067669.s003">Table S1</a>). In many cases, individual CDSs were associated with thousands of sRNAs that have a predominant antisense orientation (<a href="#pone-0067669-t001">Table 1</a>). (ii) The mRNA-seq data indicate that some highly expressed genes do not produce a detectable sRNA (<a href="#pone.0067669.s001">Figure S1</a>). (iii) Conversely, some poorly expressed genes are the templates for significant sRNA production (<a href="#pone.0067669.s001">Figure S1</a>).</p>
<a id="article1.body1.sec4.p2" name="article1.body1.sec4.p2" class="link-target"></a><p>A possible explanation for the significant levels of transcript-derived sRNAs is the production of secondary siRNA by the action of RdRP during amplificatory cascades of the RNAi signal <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Chen1" class="ref-tip">[12]</a>. BLAST analysis indicates that <em>C. paradoxa</em> putatively encodes 4 loci with sequence identity to <em>A. thaliana</em> RdRP homologs, suggesting that transitivity may exist in this alga. The phenomenon of secondary siRNA production from RNAs that were targeted by a miRNA or primary siRNA is very important in plants to efficiently suppress viral infection in the cell <a href="#pone.0067669-Obbard1" class="ref-tip">[1]</a>, <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>. In addition, in plants RdRP production of secondary siRNA has been co-opted for regulatory networks involving a few protein-encoding genes in the cell. In <em>A. thaliana</em> there are 5 TAS <em>loci</em> that are targeted by specific miRNAs, triggering recruitment of RdRP and the production of multiple secondary siRNAs <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Chen1" class="ref-tip">[12]</a>. These tasiRNAs target and regulate the expression of genes involved in the auxin system and modulate developmental processes <a href="#pone.0067669-Ghildiyal1" class="ref-tip">[11]</a>, <a href="#pone.0067669-Yoon1" class="ref-tip">[33]</a>. TasiRNAs also target pentatricopeptide repeat (PPR) protein encoding transcripts <a href="#pone.0067669-Howell1" class="ref-tip">[34]</a>. Many of these PPR mRNAs show prolific production of secondary siRNAs that are functionally dependent on the catalysis of RdRPs. Despite the overrepresentation of sRNAs among the PPR family, only about 20% of protein-coding genes in <em>A. thaliana</em> have sRNAs associated with them <a href="#pone.0067669-Kasschau1" class="ref-tip">[35]</a>. From this pool, approximately 75% of the genes (i.e., 15% of all protein encoding genes) have ≤5 sRNA counts. These numbers suggest that the association of sRNAs with protein-coding transcripts is modest even among multicellular taxa such as <em>A. thaliana</em> that encodes 6 RdRP homologs. However exonic sRNAs are abundant in the fungus <em>Mucor circinelloides</em> where they are produced by RdRPs, mediating the silencing of targeted mRNAs <a href="#pone.0067669-Calo1" class="ref-tip">[10]</a>, <a href="#pone.0067669-Nicolas1" class="ref-tip">[36]</a>. The lack of genetic tools in <em>C. paradoxa</em> precludes identification of the molecular circuitry involved in the production of exonic sRNAs to advance our understanding of the role of these molecules in regulating algal gene expression. However our finding that sRNAs are predominantly associated with mRNAs and the identification of putative RdRPs in <em>C. paradoxa</em> suggests that a mechanism must exist for the production of secondary siRNAs. If this is confirmed then we speculate that transitivity represents an ancestral trait present in the common ancestor of Plantae. The likely presence of transitivity in <em>C. paradoxa</em> and in <em>M. circinelloides</em> also suggests that a complex RNAi system was present in the ancestor of all eukaryotes.</p>
</div>

<div id="section5" class="section toc-section"><a id="s5" name="s5" data-toc="s5" class="link-target" title="Supporting Information"></a><h2>Supporting Information</h2><div class="figshare_widget" doi="10.1371/journal.pone.0067669"></div><div class="supplementary-material"><a name="pone.0067669.s001" id="pone.0067669.s001" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0067669.s001">Figure S1. </a></h3><a id="article1.body1.sec5.supplementary-material1.caption1.p1" name="article1.body1.sec5.supplementary-material1.caption1.p1" class="link-target"></a><p class="preSiDOI">The frequency of sRNA counts and gene expression levels are not correlated. (A) From RNA-seq analysis (data not shown) we observed 4,700 CDSs that do not have gene expression counts while displaying different levels of sRNA expression (shown on the x-axis). (B) In contrast, there are 3,867 CDSs that have RNA-seq counts (i.e. they are expressed) but do not result in sRNAs. Note that in 16 cases, highly expressed genes (&gt;1,000 RNA-seq counts) do not produce any sRNAs.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0067669.s001">https://doi.org/10.1371/journal.pone.0067669.s001</a></p><a id="article1.body1.sec5.supplementary-material1.caption1.p2" name="article1.body1.sec5.supplementary-material1.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0067669.s002" id="pone.0067669.s002" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0067669.s002">Figure S2. </a></h3><a id="article1.body1.sec5.supplementary-material2.caption1.p1" name="article1.body1.sec5.supplementary-material2.caption1.p1" class="link-target"></a><p class="preSiDOI">The percentages of strand prevalence of both redundant and unique sRNAs to transcriptionally oriented CDSs and ESTs. (A) For the sake of comparison, a similar plotting to <a href="#pone-0067669-g003">Figure 3</a> is shown. The 6,519 CDS were selected on the basis of prevalent (&gt;75%) degradome sense transcription or absence of antisense degradome transcription. (B) From the previous 6,519 CDSs, 4,720 CDSs were selected that produced 10 or more unique sRNAs (non-redundant). The same trend of skewed strand distribution of sRNAs is still maintained and therefore is not an artifact of overamplification of redundant sRNAs species. (C) Mapping done against EST contigs (&gt;199 bp length). The 2,995 ESTs that have 70% or more degradome tags that determine the transcriptional direction were ordered on the x-axis. The percent of sense/antisense sRNA indicates the strand specificity of many sRNAs. (D) The same scenario of strand bias is maintained when choosing 2,465 ESTs that have 10 or more sRNAs associated with them.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0067669.s002">https://doi.org/10.1371/journal.pone.0067669.s002</a></p><a id="article1.body1.sec5.supplementary-material2.caption1.p2" name="article1.body1.sec5.supplementary-material2.caption1.p2" class="link-target"></a><p class="postSiDOI">(PDF)</p>
</div><div class="supplementary-material"><a name="pone.0067669.s003" id="pone.0067669.s003" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0067669.s003">Table S1. </a></h3><a id="article1.body1.sec5.supplementary-material3.caption1.p1" name="article1.body1.sec5.supplementary-material3.caption1.p1" class="link-target"></a><p class="preSiDOI">Transcriptome-wide mapping of exonic sRNAs in <em>C. paradoxa</em>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0067669.s003">https://doi.org/10.1371/journal.pone.0067669.s003</a></p><a id="article1.body1.sec5.supplementary-material3.caption1.p2" name="article1.body1.sec5.supplementary-material3.caption1.p2" class="link-target"></a><p class="postSiDOI">(XLSX)</p>
</div><div class="supplementary-material"><a name="pone.0067669.s004" id="pone.0067669.s004" class="link-target"></a><h3 class="siTitle title-small"><a href="article/file?type=supplementary&amp;id=info:doi/10.1371/journal.pone.0067669.s004">Table S2. </a></h3><a id="article1.body1.sec5.supplementary-material4.caption1.p1" name="article1.body1.sec5.supplementary-material4.caption1.p1" class="link-target"></a><p class="preSiDOI">Degradome mapping to genomic contigs, EST contigs &gt;199 bp in length, and predicted CDSs from <em>C. paradoxa</em>.</p>
<p class="siDoi"><a href="https://doi.org/10.1371/journal.pone.0067669.s004">https://doi.org/10.1371/journal.pone.0067669.s004</a></p><a id="article1.body1.sec5.supplementary-material4.caption1.p2" name="article1.body1.sec5.supplementary-material4.caption1.p2" class="link-target"></a><p class="postSiDOI">(DOCX)</p>
</div></div>





<div class="section toc-section"><a id="ack" name="ack" data-toc="ack" title="Acknowledgments" class="link-target"></a><h2>Acknowledgments</h2>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1" class="link-target"></a><p>The authors are grateful to Bob Andersen and the staff at the Provasoli-Guillard National Center for Culture of Marine Phytoplankton (formerly CCMP) for preparing the cultures of <em>C. paradoxa</em> used for this project. We are grateful to staff members of the Rutgers Genome Cooperative for producing the <em>Illumina</em> sequence data.</p>
</div><div class="contributions toc-section"><a id="authcontrib" name="authcontrib" data-toc="authcontrib" title="Author Contributions"></a><h2>Author Contributions</h2><p>Conceived and designed the experiments: JG CXC DB. Performed the experiments: JG. Analyzed the data: JG SW DCP EZ JL CXC. Contributed reagents/materials/analysis tools: SW EZ JL. Wrote the paper: JG DB.</p></div><div class="toc-section"><a id="references" name="references" class="link-target" data-toc="references" title="References"></a><h2>References</h2><ol class="references"><li id="ref1"><span class="order">1.
            </span><a name="pone.0067669-Obbard1" id="pone.0067669-Obbard1" class="link-target"></a>Obbard DJ, Gordon KH, Buck AH, Jiggins FM (2009) The evolution of RNAi as a defence against viruses and transposable elements. Philos Trans R Soc Lond B Biol Sci 364: 99–115. <ul class="reflinks"><li><a href="#" data-author="Obbard" data-cit="ObbardDJ%2C%20GordonKH%2C%20BuckAH%2C%20JigginsFM%20%282009%29%20The%20evolution%20of%20RNAi%20as%20a%20defence%20against%20viruses%20and%20transposable%20elements.%20Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci%20364%3A%2099%E2%80%93115." data-title="The%20evolution%20of%20RNAi%20as%20a%20defence%20against%20viruses%20and%20transposable%20elements" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+evolution+of+RNAi+as+a+defence+against+viruses+and+transposable+elements+Obbard+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref2"><span class="order">2.
            </span><a name="pone.0067669-Mallory1" id="pone.0067669-Mallory1" class="link-target"></a>Mallory A, Vaucheret H (2010) Form, function, and regulation of ARGONAUTE proteins. Plant Cell 22: 3879–3889. <ul class="reflinks"><li><a href="#" data-author="Mallory" data-cit="MalloryA%2C%20VaucheretH%20%282010%29%20Form%2C%20function%2C%20and%20regulation%20of%20ARGONAUTE%20proteins.%20Plant%20Cell%2022%3A%203879%E2%80%933889." data-title="Form%2C%20function%2C%20and%20regulation%20of%20ARGONAUTE%20proteins" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Form%2C+function%2C+and+regulation+of+ARGONAUTE+proteins+Mallory+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref3"><span class="order">3.
            </span><a name="pone.0067669-Caplen1" id="pone.0067669-Caplen1" class="link-target"></a>Caplen NJ, Parrish S, Imani F, Fire A, Morgan RA (2001) Specific inhibition of gene expression by small double-stranded RNAs in invertebrate and vertebrate systems. Proc Natl Acad Sci U S A 98: 9742–9747. <ul class="reflinks"><li><a href="#" data-author="Caplen" data-cit="CaplenNJ%2C%20ParrishS%2C%20ImaniF%2C%20FireA%2C%20MorganRA%20%282001%29%20Specific%20inhibition%20of%20gene%20expression%20by%20small%20double-stranded%20RNAs%20in%20invertebrate%20and%20vertebrate%20systems.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%2098%3A%209742%E2%80%939747." data-title="Specific%20inhibition%20of%20gene%20expression%20by%20small%20double-stranded%20RNAs%20in%20invertebrate%20and%20vertebrate%20systems" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Specific+inhibition+of+gene+expression+by+small+double-stranded+RNAs+in+invertebrate+and+vertebrate+systems+Caplen+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref4"><span class="order">4.
            </span><a name="pone.0067669-Hammond1" id="pone.0067669-Hammond1" class="link-target"></a>Hammond SM, Caudy AA, Hannon GJ (2001) Post-transcriptional gene silencing by double-stranded RNA. Nat Rev Genet 2: 110–119. <ul class="reflinks"><li><a href="#" data-author="Hammond" data-cit="HammondSM%2C%20CaudyAA%2C%20HannonGJ%20%282001%29%20Post-transcriptional%20gene%20silencing%20by%20double-stranded%20RNA.%20Nat%20Rev%20Genet%202%3A%20110%E2%80%93119." data-title="Post-transcriptional%20gene%20silencing%20by%20double-stranded%20RNA" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Post-transcriptional+gene+silencing+by+double-stranded+RNA+Hammond+2001" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref5"><span class="order">5.
            </span><a name="pone.0067669-Hammond2" id="pone.0067669-Hammond2" class="link-target"></a>Hammond SM (2005) Dicing and slicing: the core machinery of the RNA interference pathway. FEBS Lett 579: 5822–5829. <ul class="reflinks"><li><a href="#" data-author="Hammond" data-cit="HammondSM%20%282005%29%20Dicing%20and%20slicing%3A%20the%20core%20machinery%20of%20the%20RNA%20interference%20pathway.%20FEBS%20Lett%20579%3A%205822%E2%80%935829." data-title="Dicing%20and%20slicing%3A%20the%20core%20machinery%20of%20the%20RNA%20interference%20pathway" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dicing+and+slicing%3A+the+core+machinery+of+the+RNA+interference+pathway+Hammond+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref6"><span class="order">6.
            </span><a name="pone.0067669-Kawasaki1" id="pone.0067669-Kawasaki1" class="link-target"></a>Kawasaki H, Taira K (2005) Transcriptional gene silencing by short interfering RNAs. Curr Opin Mol Ther 7: 125–131. <ul class="reflinks"><li><a href="#" data-author="Kawasaki" data-cit="KawasakiH%2C%20TairaK%20%282005%29%20Transcriptional%20gene%20silencing%20by%20short%20interfering%20RNAs.%20Curr%20Opin%20Mol%20Ther%207%3A%20125%E2%80%93131." data-title="Transcriptional%20gene%20silencing%20by%20short%20interfering%20RNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Transcriptional+gene+silencing+by+short+interfering+RNAs+Kawasaki+2005" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref7"><span class="order">7.
            </span><a name="pone.0067669-Lytle1" id="pone.0067669-Lytle1" class="link-target"></a>Lytle JR, Yario TA, Steitz JA (2007) Target mRNAs are repressed as efficiently by microRNA-binding sites in the 5′ UTR as in the 3′ UTR. Proc Natl Acad Sci U S A 104: 9667–9672. <ul class="reflinks"><li><a href="#" data-author="Lytle" data-cit="LytleJR%2C%20YarioTA%2C%20SteitzJA%20%282007%29%20Target%20mRNAs%20are%20repressed%20as%20efficiently%20by%20microRNA-binding%20sites%20in%20the%205%E2%80%B2%20UTR%20as%20in%20the%203%E2%80%B2%20UTR.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20104%3A%209667%E2%80%939672." data-title="Target%20mRNAs%20are%20repressed%20as%20efficiently%20by%20microRNA-binding%20sites%20in%20the%205%E2%80%B2%20UTR%20as%20in%20the%203%E2%80%B2%20UTR" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Target+mRNAs+are+repressed+as+efficiently+by+microRNA-binding+sites+in+the+5%E2%80%B2+UTR+as+in+the+3%E2%80%B2+UTR+Lytle+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref8"><span class="order">8.
            </span><a name="pone.0067669-Fabian1" id="pone.0067669-Fabian1" class="link-target"></a>Fabian MR, Sonenberg N, Filipowicz W (2010) Regulation of mRNA translation and stability by microRNAs. Annu Rev Biochem 79: 351–379. <ul class="reflinks"><li><a href="#" data-author="Fabian" data-cit="FabianMR%2C%20SonenbergN%2C%20FilipowiczW%20%282010%29%20Regulation%20of%20mRNA%20translation%20and%20stability%20by%20microRNAs.%20Annu%20Rev%20Biochem%2079%3A%20351%E2%80%93379." data-title="Regulation%20of%20mRNA%20translation%20and%20stability%20by%20microRNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Regulation+of+mRNA+translation+and+stability+by+microRNAs+Fabian+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref9"><span class="order">9.
            </span><a name="pone.0067669-Piriyapongsa1" id="pone.0067669-Piriyapongsa1" class="link-target"></a>Piriyapongsa J, Jordan IK (2008) Dual coding of siRNAs and miRNAs by plant transposable elements. RNA 14: 814–821. <ul class="reflinks"><li><a href="#" data-author="Piriyapongsa" data-cit="PiriyapongsaJ%2C%20JordanIK%20%282008%29%20Dual%20coding%20of%20siRNAs%20and%20miRNAs%20by%20plant%20transposable%20elements.%20RNA%2014%3A%20814%E2%80%93821." data-title="Dual%20coding%20of%20siRNAs%20and%20miRNAs%20by%20plant%20transposable%20elements" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Dual+coding+of+siRNAs+and+miRNAs+by+plant+transposable+elements+Piriyapongsa+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref10"><span class="order">10.
            </span><a name="pone.0067669-Calo1" id="pone.0067669-Calo1" class="link-target"></a>Calo S, Nicolas FE, Vila A, Torres-Martinez S, Ruiz-Vazquez RM (2012) Two distinct RNA-dependent RNA polymerases are required for initiation and amplification of RNA silencing in the basal fungus <em>Mucor circinelloides</em>. Mol Microbiol 83: 379–394. <ul class="reflinks"><li><a href="#" data-author="Calo" data-cit="CaloS%2C%20NicolasFE%2C%20VilaA%2C%20Torres-MartinezS%2C%20Ruiz-VazquezRM%20%282012%29%20Two%20distinct%20RNA-dependent%20RNA%20polymerases%20are%20required%20for%20initiation%20and%20amplification%20of%20RNA%20silencing%20in%20the%20basal%20fungus%20Mucor%20circinelloides.%20Mol%20Microbiol%2083%3A%20379%E2%80%93394." data-title="Two%20distinct%20RNA-dependent%20RNA%20polymerases%20are%20required%20for%20initiation%20and%20amplification%20of%20RNA%20silencing%20in%20the%20basal%20fungus%20Mucor%20circinelloides" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Two+distinct+RNA-dependent+RNA+polymerases+are+required+for+initiation+and+amplification+of+RNA+silencing+in+the+basal+fungus+Mucor+circinelloides+Calo+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref11"><span class="order">11.
            </span><a name="pone.0067669-Ghildiyal1" id="pone.0067669-Ghildiyal1" class="link-target"></a>Ghildiyal M, Zamore PD (2009) Small silencing RNAs: an expanding universe. Nat Rev Genet 10: 94–108. <ul class="reflinks"><li><a href="#" data-author="Ghildiyal" data-cit="GhildiyalM%2C%20ZamorePD%20%282009%29%20Small%20silencing%20RNAs%3A%20an%20expanding%20universe.%20Nat%20Rev%20Genet%2010%3A%2094%E2%80%93108." data-title="Small%20silencing%20RNAs%3A%20an%20expanding%20universe" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Small+silencing+RNAs%3A+an+expanding+universe+Ghildiyal+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref12"><span class="order">12.
            </span><a name="pone.0067669-Chen1" id="pone.0067669-Chen1" class="link-target"></a>Chen HM, Chen LT, Patel K, Li YH, Baulcombe DC, et al. (2010) 22-Nucleotide RNAs trigger secondary siRNA biogenesis in plants. Proc Natl Acad Sci U S A 107: 15269–15274. <ul class="reflinks"><li><a href="#" data-author="Chen" data-cit="ChenHM%2C%20ChenLT%2C%20PatelK%2C%20LiYH%2C%20BaulcombeDC%2C%20et%20al.%20%282010%29%2022-Nucleotide%20RNAs%20trigger%20secondary%20siRNA%20biogenesis%20in%20plants.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20107%3A%2015269%E2%80%9315274." data-title="22-Nucleotide%20RNAs%20trigger%20secondary%20siRNA%20biogenesis%20in%20plants" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=22-Nucleotide+RNAs+trigger+secondary+siRNA+biogenesis+in+plants+Chen+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref13"><span class="order">13.
            </span><a name="pone.0067669-Manavella1" id="pone.0067669-Manavella1" class="link-target"></a>Manavella PA, Koenig D, Weigel D (2012) Plant secondary siRNA production determined by microRNA-duplex structure. Proc Natl Acad Sci U S A 109: 2461–2466. <ul class="reflinks"><li><a href="#" data-author="Manavella" data-cit="ManavellaPA%2C%20KoenigD%2C%20WeigelD%20%282012%29%20Plant%20secondary%20siRNA%20production%20determined%20by%20microRNA-duplex%20structure.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20109%3A%202461%E2%80%932466." data-title="Plant%20secondary%20siRNA%20production%20determined%20by%20microRNA-duplex%20structure" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Plant+secondary+siRNA+production+determined+by+microRNA-duplex+structure+Manavella+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref14"><span class="order">14.
            </span><a name="pone.0067669-Allen1" id="pone.0067669-Allen1" class="link-target"></a>Allen E, Howell MD (2010) miRNAs in the biogenesis of trans-acting siRNAs in higher plants. Semin Cell Dev Biol 21: 798–804. <ul class="reflinks"><li><a href="#" data-author="Allen" data-cit="AllenE%2C%20HowellMD%20%282010%29%20miRNAs%20in%20the%20biogenesis%20of%20trans-acting%20siRNAs%20in%20higher%20plants.%20Semin%20Cell%20Dev%20Biol%2021%3A%20798%E2%80%93804." data-title="miRNAs%20in%20the%20biogenesis%20of%20trans-acting%20siRNAs%20in%20higher%20plants" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miRNAs+in+the+biogenesis+of+trans-acting+siRNAs+in+higher+plants+Allen+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref15"><span class="order">15.
            </span><a name="pone.0067669-Gierga1" id="pone.0067669-Gierga1" class="link-target"></a>Gierga G, Voss B, Hess WR (2012) Non-coding RNAs in marine <em>Synechococcus</em> and their regulation under environmentally relevant stress conditions. ISME J 6: 1544–1557. <ul class="reflinks"><li><a href="#" data-author="Gierga" data-cit="GiergaG%2C%20VossB%2C%20HessWR%20%282012%29%20Non-coding%20RNAs%20in%20marine%20Synechococcus%20and%20their%20regulation%20under%20environmentally%20relevant%20stress%20conditions.%20ISME%20J%206%3A%201544%E2%80%931557." data-title="Non-coding%20RNAs%20in%20marine%20Synechococcus%20and%20their%20regulation%20under%20environmentally%20relevant%20stress%20conditions" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Non-coding+RNAs+in+marine+Synechococcus+and+their+regulation+under+environmentally+relevant+stress+conditions+Gierga+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref16"><span class="order">16.
            </span><a name="pone.0067669-Shabalina1" id="pone.0067669-Shabalina1" class="link-target"></a>Shabalina SA, Koonin EV (2008) Origins and evolution of eukaryotic RNA interference. Trends Ecol Evol 23: 578–587. <ul class="reflinks"><li><a href="#" data-author="Shabalina" data-cit="ShabalinaSA%2C%20KooninEV%20%282008%29%20Origins%20and%20evolution%20of%20eukaryotic%20RNA%20interference.%20Trends%20Ecol%20Evol%2023%3A%20578%E2%80%93587." data-title="Origins%20and%20evolution%20of%20eukaryotic%20RNA%20interference" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Origins+and+evolution+of+eukaryotic+RNA+interference+Shabalina+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref17"><span class="order">17.
            </span><a name="pone.0067669-Cerutti1" id="pone.0067669-Cerutti1" class="link-target"></a>Cerutti H, Casas-Mollano JA (2006) On the origin and functions of RNA-mediated silencing: from protists to man. Curr Genet 50: 81–99. <ul class="reflinks"><li><a href="#" data-author="Cerutti" data-cit="CeruttiH%2C%20Casas-MollanoJA%20%282006%29%20On%20the%20origin%20and%20functions%20of%20RNA-mediated%20silencing%3A%20from%20protists%20to%20man.%20Curr%20Genet%2050%3A%2081%E2%80%9399." data-title="On%20the%20origin%20and%20functions%20of%20RNA-mediated%20silencing%3A%20from%20protists%20to%20man" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=On+the+origin+and+functions+of+RNA-mediated+silencing%3A+from+protists+to+man+Cerutti+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref18"><span class="order">18.
            </span><a name="pone.0067669-TalmorNeiman1" id="pone.0067669-TalmorNeiman1" class="link-target"></a>Talmor-Neiman M, Stav R, Klipcan L, Buxdorf K, Baulcombe DC, et al. (2006) Identification of trans-acting siRNAs in moss and an RNA-dependent RNA polymerase required for their biogenesis. Plant J 48: 511–521. <ul class="reflinks"><li><a href="#" data-author="Talmor-Neiman" data-cit="Talmor-NeimanM%2C%20StavR%2C%20KlipcanL%2C%20BuxdorfK%2C%20BaulcombeDC%2C%20et%20al.%20%282006%29%20Identification%20of%20trans-acting%20siRNAs%20in%20moss%20and%20an%20RNA-dependent%20RNA%20polymerase%20required%20for%20their%20biogenesis.%20Plant%20J%2048%3A%20511%E2%80%93521." data-title="Identification%20of%20trans-acting%20siRNAs%20in%20moss%20and%20an%20RNA-dependent%20RNA%20polymerase%20required%20for%20their%20biogenesis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+trans-acting+siRNAs+in+moss+and+an+RNA-dependent+RNA+polymerase+required+for+their+biogenesis+Talmor-Neiman+2006" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref19"><span class="order">19.
            </span><a name="pone.0067669-Molnar1" id="pone.0067669-Molnar1" class="link-target"></a>Molnar A, Schwach F, Studholme DJ, Thuenemann EC, Baulcombe DC (2007) miRNAs control gene expression in the single-cell alga <em>Chlamydomonas reinhardtii.</em>. Nature 447: 1126–1129. <ul class="reflinks"><li><a href="#" data-author="Molnar" data-cit="MolnarA%2C%20SchwachF%2C%20StudholmeDJ%2C%20ThuenemannEC%2C%20BaulcombeDC%20%282007%29%20miRNAs%20control%20gene%20expression%20in%20the%20single-cell%20alga%20Chlamydomonas%20reinhardtii..%20Nature%20447%3A%201126%E2%80%931129." data-title="miRNAs%20control%20gene%20expression%20in%20the%20single-cell%20alga%20Chlamydomonas%20reinhardtii." target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=miRNAs+control+gene+expression+in+the+single-cell+alga+Chlamydomonas+reinhardtii.+Molnar+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref20"><span class="order">20.
            </span><a name="pone.0067669-Arif1" id="pone.0067669-Arif1" class="link-target"></a>Arif MA, Frank W, Khraiwesh B (2013) Role of RNA Interference (RNAi) in the Moss <em>Physcomitrella patens</em>. Int J Mol Sci 14: 1516–1540. <ul class="reflinks"><li><a href="#" data-author="Arif" data-cit="ArifMA%2C%20FrankW%2C%20KhraiweshB%20%282013%29%20Role%20of%20RNA%20Interference%20%28RNAi%29%20in%20the%20Moss%20Physcomitrella%20patens.%20Int%20J%20Mol%20Sci%2014%3A%201516%E2%80%931540." data-title="Role%20of%20RNA%20Interference%20%28RNAi%29%20in%20the%20Moss%20Physcomitrella%20patens" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Role+of+RNA+Interference+%28RNAi%29+in+the+Moss+Physcomitrella+patens+Arif+2013" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref21"><span class="order">21.
            </span><a name="pone.0067669-Huang1" id="pone.0067669-Huang1" class="link-target"></a>Huang A, He L, Wang G (2011) Identification and characterization of microRNAs from <em>Phaeodactylum tricornutum</em> by high-throughput sequencing and bioinformatics analysis. BMC Genomics 12: 337. <ul class="reflinks"><li><a href="#" data-author="Huang" data-cit="HuangA%2C%20HeL%2C%20WangG%20%282011%29%20Identification%20and%20characterization%20of%20microRNAs%20from%20Phaeodactylum%20tricornutum%20by%20high-throughput%20sequencing%20and%20bioinformatics%20analysis.%20BMC%20Genomics%2012%3A%20337." data-title="Identification%20and%20characterization%20of%20microRNAs%20from%20Phaeodactylum%20tricornutum%20by%20high-throughput%20sequencing%20and%20bioinformatics%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+and+characterization+of+microRNAs+from+Phaeodactylum+tricornutum+by+high-throughput+sequencing+and+bioinformatics+analysis+Huang+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref22"><span class="order">22.
            </span><a name="pone.0067669-NordenKrichmar1" id="pone.0067669-NordenKrichmar1" class="link-target"></a>Norden-Krichmar TM, Allen AE, Gaasterland T, Hildebrand M (2011) Characterization of the small RNA transcriptome of the diatom, <em>Thalassiosira pseudonana</em>. PLoS One 6: e22870. <ul class="reflinks"><li><a href="#" data-author="Norden-Krichmar" data-cit="Norden-KrichmarTM%2C%20AllenAE%2C%20GaasterlandT%2C%20HildebrandM%20%282011%29%20Characterization%20of%20the%20small%20RNA%20transcriptome%20of%20the%20diatom%2C%20Thalassiosira%20pseudonana.%20PLoS%20One%206%3A%20e22870." data-title="Characterization%20of%20the%20small%20RNA%20transcriptome%20of%20the%20diatom%2C%20Thalassiosira%20pseudonana" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Characterization+of+the+small+RNA+transcriptome+of+the+diatom%2C+Thalassiosira+pseudonana+Norden-Krichmar+2011" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref23"><span class="order">23.
            </span><a name="pone.0067669-Liang1" id="pone.0067669-Liang1" class="link-target"></a>Liang C, Zhang X, Zou J, Xu D, Su F, et al. (2010) Identification of miRNA from <em>Porphyra yezoensis</em> by high-throughput sequencing and bioinformatics analysis. PLoS One 5: e10698. <ul class="reflinks"><li><a href="#" data-author="Liang" data-cit="LiangC%2C%20ZhangX%2C%20ZouJ%2C%20XuD%2C%20SuF%2C%20et%20al.%20%282010%29%20Identification%20of%20miRNA%20from%20Porphyra%20yezoensis%20by%20high-throughput%20sequencing%20and%20bioinformatics%20analysis.%20PLoS%20One%205%3A%20e10698." data-title="Identification%20of%20miRNA%20from%20Porphyra%20yezoensis%20by%20high-throughput%20sequencing%20and%20bioinformatics%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+miRNA+from+Porphyra+yezoensis+by+high-throughput+sequencing+and+bioinformatics+analysis+Liang+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref24"><span class="order">24.
            </span><a name="pone.0067669-Cock1" id="pone.0067669-Cock1" class="link-target"></a>Cock JM, Sterck L, Rouze P, Scornet D, Allen AE, et al. (2010) The <em>Ectocarpus</em> genome and the independent evolution of multicellularity in brown algae. Nature 465: 617–621. <ul class="reflinks"><li><a href="#" data-author="Cock" data-cit="CockJM%2C%20SterckL%2C%20RouzeP%2C%20ScornetD%2C%20AllenAE%2C%20et%20al.%20%282010%29%20The%20Ectocarpus%20genome%20and%20the%20independent%20evolution%20of%20multicellularity%20in%20brown%20algae.%20Nature%20465%3A%20617%E2%80%93621." data-title="The%20Ectocarpus%20genome%20and%20the%20independent%20evolution%20of%20multicellularity%20in%20brown%20algae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=The+Ectocarpus+genome+and+the+independent+evolution+of+multicellularity+in+brown+algae+Cock+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref25"><span class="order">25.
            </span><a name="pone.0067669-Price1" id="pone.0067669-Price1" class="link-target"></a>Price DC, Chan CX, Yoon HS, Yang EC, Qiu H, et al. (2012) <em>Cyanophora paradoxa</em> genome elucidates origin of photosynthesis in algae and plants. Science 335: 843–847. <ul class="reflinks"><li><a href="#" data-author="Price" data-cit="PriceDC%2C%20ChanCX%2C%20YoonHS%2C%20YangEC%2C%20QiuH%2C%20et%20al.%20%282012%29%20Cyanophora%20paradoxa%20genome%20elucidates%20origin%20of%20photosynthesis%20in%20algae%20and%20plants.%20Science%20335%3A%20843%E2%80%93847." data-title="Cyanophora%20paradoxa%20genome%20elucidates%20origin%20of%20photosynthesis%20in%20algae%20and%20plants" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Cyanophora+paradoxa+genome+elucidates+origin+of+photosynthesis+in+algae+and+plants+Price+2012" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref26"><span class="order">26.
            </span><a name="pone.0067669-ReyesPrieto1" id="pone.0067669-ReyesPrieto1" class="link-target"></a>Reyes-Prieto A, Bhattacharya D (2007) Phylogeny of nuclear-encoded plastid-targeted proteins supports an early divergence of glaucophytes within Plantae. Mol Biol Evol 24: 2358–2361. <ul class="reflinks"><li><a href="#" data-author="Reyes-Prieto" data-cit="Reyes-PrietoA%2C%20BhattacharyaD%20%282007%29%20Phylogeny%20of%20nuclear-encoded%20plastid-targeted%20proteins%20supports%20an%20early%20divergence%20of%20glaucophytes%20within%20Plantae.%20Mol%20Biol%20Evol%2024%3A%202358%E2%80%932361." data-title="Phylogeny%20of%20nuclear-encoded%20plastid-targeted%20proteins%20supports%20an%20early%20divergence%20of%20glaucophytes%20within%20Plantae" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Phylogeny+of+nuclear-encoded+plastid-targeted+proteins+supports+an+early+divergence+of+glaucophytes+within+Plantae+Reyes-Prieto+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref27"><span class="order">27.
            </span><a name="pone.0067669-AddoQuaye1" id="pone.0067669-AddoQuaye1" class="link-target"></a>Addo-Quaye C, Eshoo TW, Bartel DP, Axtell MJ (2008) Endogenous siRNA and miRNA targets identified by sequencing of the <em>Arabidopsis</em> degradome. Curr Biol 18: 758–762. <ul class="reflinks"><li><a href="#" data-author="Addo-Quaye" data-cit="Addo-QuayeC%2C%20EshooTW%2C%20BartelDP%2C%20AxtellMJ%20%282008%29%20Endogenous%20siRNA%20and%20miRNA%20targets%20identified%20by%20sequencing%20of%20the%20Arabidopsis%20degradome.%20Curr%20Biol%2018%3A%20758%E2%80%93762." data-title="Endogenous%20siRNA%20and%20miRNA%20targets%20identified%20by%20sequencing%20of%20the%20Arabidopsis%20degradome" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Endogenous+siRNA+and+miRNA+targets+identified+by+sequencing+of+the+Arabidopsis+degradome+Addo-Quaye+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref28"><span class="order">28.
            </span><a name="pone.0067669-Friedlander1" id="pone.0067669-Friedlander1" class="link-target"></a>Friedlander MR, Chen W, Adamidi C, Maaskola J, Einspanier R, et al. (2008) Discovering microRNAs from deep sequencing data using miRDeep. Nat Biotechnol 26: 407–415. <ul class="reflinks"><li><a href="#" data-author="Friedlander" data-cit="FriedlanderMR%2C%20ChenW%2C%20AdamidiC%2C%20MaaskolaJ%2C%20EinspanierR%2C%20et%20al.%20%282008%29%20Discovering%20microRNAs%20from%20deep%20sequencing%20data%20using%20miRDeep.%20Nat%20Biotechnol%2026%3A%20407%E2%80%93415." data-title="Discovering%20microRNAs%20from%20deep%20sequencing%20data%20using%20miRDeep" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Discovering+microRNAs+from+deep+sequencing+data+using+miRDeep+Friedlander+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref29"><span class="order">29.
            </span><a name="pone.0067669-AddoQuaye2" id="pone.0067669-AddoQuaye2" class="link-target"></a>Addo-Quaye C, Miller W, Axtell MJ (2009) CleaveLand: a pipeline for using degradome data to find cleaved small RNA targets. Bioinformatics 25: 130–131. <ul class="reflinks"><li><a href="#" data-author="Addo-Quaye" data-cit="Addo-QuayeC%2C%20MillerW%2C%20AxtellMJ%20%282009%29%20CleaveLand%3A%20a%20pipeline%20for%20using%20degradome%20data%20to%20find%20cleaved%20small%20RNA%20targets.%20Bioinformatics%2025%3A%20130%E2%80%93131." data-title="CleaveLand%3A%20a%20pipeline%20for%20using%20degradome%20data%20to%20find%20cleaved%20small%20RNA%20targets" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=CleaveLand%3A+a+pipeline+for+using+degradome+data+to+find+cleaved+small+RNA+targets+Addo-Quaye+2009" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref30"><span class="order">30.
            </span><a name="pone.0067669-Altschul1" id="pone.0067669-Altschul1" class="link-target"></a>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ (1990) Basic local alignment search tool. J Mol Biol 215: 403–410. <ul class="reflinks"><li><a href="#" data-author="Altschul" data-cit="AltschulSF%2C%20GishW%2C%20MillerW%2C%20MyersEW%2C%20LipmanDJ%20%281990%29%20Basic%20local%20alignment%20search%20tool.%20J%20Mol%20Biol%20215%3A%20403%E2%80%93410." data-title="Basic%20local%20alignment%20search%20tool" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Basic+local+alignment+search+tool+Altschul+1990" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref31"><span class="order">31.
            </span><a name="pone.0067669-German1" id="pone.0067669-German1" class="link-target"></a>German MA, Pillay M, Jeong DH, Hetawal A, Luo S, et al. (2008) Global identification of microRNA-target RNA pairs by parallel analysis of RNA ends. Nat Biotechnol 26: 941–946. <ul class="reflinks"><li><a href="#" data-author="German" data-cit="GermanMA%2C%20PillayM%2C%20JeongDH%2C%20HetawalA%2C%20LuoS%2C%20et%20al.%20%282008%29%20Global%20identification%20of%20microRNA-target%20RNA%20pairs%20by%20parallel%20analysis%20of%20RNA%20ends.%20Nat%20Biotechnol%2026%3A%20941%E2%80%93946." data-title="Global%20identification%20of%20microRNA-target%20RNA%20pairs%20by%20parallel%20analysis%20of%20RNA%20ends" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Global+identification+of+microRNA-target+RNA+pairs+by+parallel+analysis+of+RNA+ends+German+2008" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref32"><span class="order">32.
            </span><a name="pone.0067669-Pantaleo1" id="pone.0067669-Pantaleo1" class="link-target"></a>Pantaleo V, Szittya G, Moxon S, Miozzi L, Moulton V, et al. (2010) Identification of grapevine microRNAs and their targets using high-throughput sequencing and degradome analysis. Plant J 62: 960–976. <ul class="reflinks"><li><a href="#" data-author="Pantaleo" data-cit="PantaleoV%2C%20SzittyaG%2C%20MoxonS%2C%20MiozziL%2C%20MoultonV%2C%20et%20al.%20%282010%29%20Identification%20of%20grapevine%20microRNAs%20and%20their%20targets%20using%20high-throughput%20sequencing%20and%20degradome%20analysis.%20Plant%20J%2062%3A%20960%E2%80%93976." data-title="Identification%20of%20grapevine%20microRNAs%20and%20their%20targets%20using%20high-throughput%20sequencing%20and%20degradome%20analysis" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Identification+of+grapevine+microRNAs+and+their+targets+using+high-throughput+sequencing+and+degradome+analysis+Pantaleo+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref33"><span class="order">33.
            </span><a name="pone.0067669-Yoon1" id="pone.0067669-Yoon1" class="link-target"></a>Yoon EK, Yang JH, Lim J, Kim SH, Kim SK, et al. (2010) Auxin regulation of the microRNA390-dependent transacting small interfering RNA pathway in <em>Arabidopsis</em> lateral root development. Nucleic Acids Res 38: 1382–1391. <ul class="reflinks"><li><a href="#" data-author="Yoon" data-cit="YoonEK%2C%20YangJH%2C%20LimJ%2C%20KimSH%2C%20KimSK%2C%20et%20al.%20%282010%29%20Auxin%20regulation%20of%20the%20microRNA390-dependent%20transacting%20small%20interfering%20RNA%20pathway%20in%20Arabidopsis%20lateral%20root%20development.%20Nucleic%20Acids%20Res%2038%3A%201382%E2%80%931391." data-title="Auxin%20regulation%20of%20the%20microRNA390-dependent%20transacting%20small%20interfering%20RNA%20pathway%20in%20Arabidopsis%20lateral%20root%20development" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Auxin+regulation+of+the+microRNA390-dependent+transacting+small+interfering+RNA+pathway+in+Arabidopsis+lateral+root+development+Yoon+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref34"><span class="order">34.
            </span><a name="pone.0067669-Howell1" id="pone.0067669-Howell1" class="link-target"></a>Howell MD, Fahlgren N, Chapman EJ, Cumbie JS, Sullivan CM, et al. (2007) Genome-wide analysis of the RNA-DEPENDENT RNA POLYMERASE6/DICER-LIKE4 pathway in <em>Arabidopsis</em> reveals dependency on miRNA- and tasiRNA-directed targeting. Plant Cell 19: 926–942. <ul class="reflinks"><li><a href="#" data-author="Howell" data-cit="HowellMD%2C%20FahlgrenN%2C%20ChapmanEJ%2C%20CumbieJS%2C%20SullivanCM%2C%20et%20al.%20%282007%29%20Genome-wide%20analysis%20of%20the%20RNA-DEPENDENT%20RNA%20POLYMERASE6%2FDICER-LIKE4%20pathway%20in%20Arabidopsis%20reveals%20dependency%20on%20miRNA-%20and%20tasiRNA-directed%20targeting.%20Plant%20Cell%2019%3A%20926%E2%80%93942." data-title="Genome-wide%20analysis%20of%20the%20RNA-DEPENDENT%20RNA%20POLYMERASE6%2FDICER-LIKE4%20pathway%20in%20Arabidopsis%20reveals%20dependency%20on%20miRNA-%20and%20tasiRNA-directed%20targeting" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+analysis+of+the+RNA-DEPENDENT+RNA+POLYMERASE6%2FDICER-LIKE4+pathway+in+Arabidopsis+reveals+dependency+on+miRNA-+and+tasiRNA-directed+targeting+Howell+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref35"><span class="order">35.
            </span><a name="pone.0067669-Kasschau1" id="pone.0067669-Kasschau1" class="link-target"></a>Kasschau KD, Fahlgren N, Chapman EJ, Sullivan CM, Cumbie JS, et al. (2007) Genome-wide profiling and analysis of <em>Arabidopsis</em> siRNAs. PLoS Biol 5: e57. <ul class="reflinks"><li><a href="#" data-author="Kasschau" data-cit="KasschauKD%2C%20FahlgrenN%2C%20ChapmanEJ%2C%20SullivanCM%2C%20CumbieJS%2C%20et%20al.%20%282007%29%20Genome-wide%20profiling%20and%20analysis%20of%20Arabidopsis%20siRNAs.%20PLoS%20Biol%205%3A%20e57." data-title="Genome-wide%20profiling%20and%20analysis%20of%20Arabidopsis%20siRNAs" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Genome-wide+profiling+and+analysis+of+Arabidopsis+siRNAs+Kasschau+2007" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li><li id="ref36"><span class="order">36.
            </span><a name="pone.0067669-Nicolas1" id="pone.0067669-Nicolas1" class="link-target"></a>Nicolas FE, Moxon S, de Haro JP, Calo S, Grigoriev IV, et al. (2010) Endogenous short RNAs generated by Dicer 2 and RNA-dependent RNA polymerase 1 regulate mRNAs in the basal fungus <em>Mucor circinelloides</em>. Nucleic Acids Res 38: 5535–5541. <ul class="reflinks"><li><a href="#" data-author="Nicolas" data-cit="NicolasFE%2C%20MoxonS%2C%20de%20HaroJP%2C%20CaloS%2C%20GrigorievIV%2C%20et%20al.%20%282010%29%20Endogenous%20short%20RNAs%20generated%20by%20Dicer%202%20and%20RNA-dependent%20RNA%20polymerase%201%20regulate%20mRNAs%20in%20the%20basal%20fungus%20Mucor%20circinelloides.%20Nucleic%20Acids%20Res%2038%3A%205535%E2%80%935541." data-title="Endogenous%20short%20RNAs%20generated%20by%20Dicer%202%20and%20RNA-dependent%20RNA%20polymerase%201%20regulate%20mRNAs%20in%20the%20basal%20fungus%20Mucor%20circinelloides" target="_new" title="Go to article in CrossRef">
                      View Article
                    </a></li><li><a href="http://scholar.google.com/scholar?q=Endogenous+short+RNAs+generated+by+Dicer+2+and+RNA-dependent+RNA+polymerase+1+regulate+mRNAs+in+the+basal+fungus+Mucor+circinelloides+Nicolas+2010" target="_new" title="Go to article in Google Scholar">
                      Google Scholar
                    </a></li></ul></li></ol></div>

